CREATE DATABASE  IF NOT EXISTS `BASP` /*!40100 DEFAULT CHARACTER SET utf8 */ /*!80016 DEFAULT ENCRYPTION='N' */;
USE `BASP`;
-- MySQL dump 10.13  Distrib 8.0.28, for macos11 (x86_64)
--
-- Host: 127.0.0.1    Database: BASP
-- ------------------------------------------------------
-- Server version	8.0.28

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `Author`
--

DROP TABLE IF EXISTS `Author`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `Author` (
  `idAuthor` int NOT NULL AUTO_INCREMENT,
  `FirstName` varchar(45) DEFAULT NULL,
  `LastName` varchar(45) DEFAULT NULL,
  PRIMARY KEY (`idAuthor`)
) ENGINE=InnoDB AUTO_INCREMENT=827 DEFAULT CHARSET=utf8mb3;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Author`
--

LOCK TABLES `Author` WRITE;
/*!40000 ALTER TABLE `Author` DISABLE KEYS */;
INSERT INTO `Author` VALUES (1,'Tariq','Madani'),(2,'Aisha','Al-Ghamdi'),(3,'Tariq','Madani'),(4,'Aisha','Al-Ghamdi'),(5,'Albert','Van Der Vliet'),(6,'Jason','Eiserich'),(7,'Carroll','Cross'),(8,'Erika','Crouch'),(9,'Joseph','Domachowske'),(10,'Cynthia','Bonville'),(11,'Helene','Rosenberg'),(12,'Alexander','Pasternak'),(13,'Erwin','Van Den Born'),(14,'Willy','Spaan'),(15,'Eric','Snijder'),(16,'Gonzalo','Alvarez'),(17,'Paul','Hébert'),(18,'Sharyn','Szick'),(19,'Jonathan','Ball'),(20,'Richard','Venn'),(21,'Dirk-Jan','Slebos'),(22,'Stefan','Ryter'),(23,'Augustine','Choi'),(24,'Fu-Chiang','Tsui'),(25,'Jeremy','Espino'),(26,'Virginia','Dato'),(27,'Per','Gesteland'),(28,'Judith','Hutman'),(29,'Michael','Wagner'),(30,'Senya','Ivanov'),(31,'','Matsufuji'),(32,'Yasuko','Murakami'),(33,'Raymond','Gesteland'),(34,'John','Atkins'),(35,'Stephanie','Shi'),(36,'Peiyong','Huang'),(37,'Hsin-Pai','Li'),(38,'Michael','Lai'),(39,'Julia','Pridgeon'),(40,'Thangiah','Geetha'),(41,'Marie','Wooten'),(42,'Aspasia','Ploubidou'),(43,'Violaine','Moreau'),(44,'Keith','Ashman'),(45,'Inge','Reckmann'),(46,'Cayetano','Gonza'),(47,'Â','Lez'),(48,'Michael','Way'),(49,'Biehuoy','Shieh'),(50,'Ching','Li'),(51,'Joanne','Verheij'),(52,'Johan','Ab'),(53,'','Groeneveld'),(54,'Albertus','Beishuizen'),(55,'Arthur','Van Lingen'),(56,'Alberdina','Simoons-Smit'),(57,'Rob','Jm'),(58,'Strack','Van Schijndel'),(59,'Travis','Porco'),(60,'Karen','Holbrook'),(61,'Susan','Fernyak'),(62,'Diane','Portnoy'),(63,'Randy','Reiter'),(64,'Tomás','Aragón'),(65,'Ted','Kremer'),(66,'Mikael','Rinne'),(67,'Yi','Xu'),(68,'Ming','Chen'),(69,'Mark','Kelley'),(70,'Marit','Bratlie'),(71,'Finn','Drabløs'),(72,'Genevieve','Howse'),(73,'Yves','Le Loir'),(74,'Vasco','Azevedo'),(75,'Sergio','Oliveira'),(76,'Daniela','Freitas'),(77,'Anderson','Miyoshi'),(78,'Luis','Bermúdez-Humarán'),(79,'Sébastien','Nouaille'),(80,'Luciana','Ribeiro'),(81,'Sophie','Leclercq'),(82,'Jane','Gabriel'),(83,'Valeria','Guimaraes'),(84,'Maricê','Oliveira'),(85,'Cathy','Charlier'),(86,'Michel','Gautier'),(87,'Philippe','Langella'),(88,'Christine','Dufraigne'),(89,'Bernard','Fertil'),(90,'Sylvain','Lespinats'),(91,'Alain','Giron'),(92,'Patrick','Deschavanne'),(93,'Mazen','Karaman'),(94,'Susan','Groshen'),(95,'Chi-Chiang','Lee'),(96,'Brian','Pike'),(97,'Joseph','Hacia'),(98,'Hu','Zhu'),(99,'Hsiao','Chan'),(100,'Zuoming','Zhou'),(101,'Jianming','Li'),(102,'Hui','Zhu'),(103,'Lanlan','Yin'),(104,'Ming','Xu'),(105,'Lijun','Cheng'),(106,'Jiahao','Sha'),(107,'Willemijntje','Hoogerwerf'),(108,'Kelly','Gondesen'),(109,'Shu-Yuan','Xiao'),(110,'John','Winston'),(111,'William','Willis'),(112,'Pankaj','Pasricha'),(113,'Remy','Froissart'),(114,'Denis','Roze'),(115,'Marilyne','Uzest'),(116,'Lionel','Galibert'),(117,'Stephane','Blanc'),(118,'Yannis','Michalakis'),(119,'Ewan','Plant'),(120,'Jonathan','Dinman'),(121,'K','Jacobs'),(122,'J','Harger'),(123,'A','Meskauskas'),(124,'J','Jacobs'),(125,'J','Baxter'),(126,'A','Petrov'),(127,'J','Dinman'),(128,'Brock','Binkowski'),(129,'Kathryn','Richmond'),(130,'James','Kaysen'),(131,'Michael','Sussman'),(132,'Peter','Belshaw'),(133,'Sandrine','Imbeaud'),(134,'Esther','Graudens'),(135,'Virginie','Boulanger'),(136,'Xavier','Barlet'),(137,'Patrick','Zaborski'),(138,'Eric','Eveno'),(139,'Odilo','Mueller'),(140,'Andreas','Schroeder'),(141,'Charles','Auffray'),(142,'Katri','Mä Kelä Inen'),(143,'Kristiina','Mä Kinen'),(144,'Vivian','Lin'),(145,'Priscilla','Robinson'),(146,'Open','Access'),(147,'Stephen','Berger'),(148,'Greg','Martin'),(149,'Tsutomu','Suzuki'),(150,'Takuya','Ueda'),(151,'Kimitsuna','Watanabe'),(152,'Magdalena','Alejska'),(153,'Magdalena','Figlerowicz'),(154,'Nelli','Malinowska'),(155,'Anna','Urbanowicz'),(156,'Marek','Figlerowicz'),(157,'Joseph','Golden'),(158,'Leslie','Schiff'),(159,'Elzbieta','Kierzek'),(160,'Anna','Ciesielska'),(161,'Karol','Pasternak'),(162,'David','Mathews'),(163,'Douglas','Turner'),(164,'Ryszard','Kierzek'),(165,'Shea','Gardner'),(166,'Marisa','Lam'),(167,'Jason','Smith'),(168,'Clinton','Torres'),(169,'Tom','Slezak'),(170,'Muthuraman','Sathiamurthy'),(171,'Bjoern','Peters'),(172,'Huynh-Hoa','Bui'),(173,'John','Sidney'),(174,'John','Mokili'),(175,'Stephen','Wilson'),(176,'Ward','Fleri'),(177,'Deborah','Mcguinness'),(178,'Philip','Bourne'),(179,'Alessandro','Sette'),(180,'Pål','Olsvik'),(181,'Kai','Lie'),(182,'Ann-Elise','Jordal'),(183,'Tom','Nilsen'),(184,'Ivar','Hordvik'),(185,'G','Rohde'),(186,'I','Borg'),(187,'U','Arinir'),(188,'J','Kronsbein'),(189,'R','Rausse'),(190,'T','Bauer'),(191,'A','Bufe'),(192,'G','Schultze-Werninghaus'),(193,'Manami','Inoue'),(194,'Motoki','Iwasaki'),(195,'Tetsuya','Otani'),(196,'Shizuka','Sasazuki'),(197,'Shoichiro','Tsugane'),(198,'Yasuyoshi','Mori'),(199,'Tsuyoshi','Hirano'),(200,'Tsugunori','Notomi'),(201,'* -Yasuyoshi_Mori@eiken','Jp'),(202,';','Tsuyoshi'),(203,'Hirano -Tsuyoshi_Hirano@eiken','Jp'),(204,';','Tsugunori'),(205,'Notomi -Tsugunori_Notomi@eiken','Jp'),(206,'Han-Zhu','Qian'),(207,'Joseph','Schumacher'),(208,'Huey','Chen'),(209,'Yu-Hua','Ruan'),(210,'Theodore','Oliphant'),(211,'Michael','Engle'),(212,'Grant','Nybakken'),(213,'Chris','Doane'),(214,'Syd','Johnson'),(215,'Ling','Huang'),(216,'Sergey','Gorlatov'),(217,'Erin','Mehlhop'),(218,'Anantha','Marri'),(219,'Kyung','Chung'),(220,'Gregory','Ebel'),(221,'Laura','Kramer'),(222,'Daved','Fremont'),(223,'Michael','Diamond'),(224,'Ran','Balicer'),(225,'Saad','Omer'),(226,'Daniel','Barnett'),(227,'George','Everly'),(228,'Carly','Ruderman'),(229,'Shawn','Tracy'),(230,'Cécile','Bensimon'),(231,'Mark','Bernstein'),(232,'Laura','Hawryluck'),(233,'Randi','Zlotnik Shaul'),(234,'Ross','Upshur'),(235,'Mepur','Ravindranath'),(236,'Thiruverkadu','Saravanan'),(237,'Clarence','Monteclaro'),(238,'Naftali','Presser'),(239,'Xing','Ye'),(240,'Senthamil','Selvan'),(241,'Stanley','Brosman'),(242,'Luigi','Franciosi'),(243,'Clive','Page'),(244,'Bartolome','Celli'),(245,'Mario','Cazzola'),(246,'Michael','Walker'),(247,'Meindert','Danhof'),(248,'Klaus','Rabe'),(249,'Oscar','Della Pasqua'),(250,'Yi-Shin','Pan'),(251,'Yun-Shien','Lee'),(252,'Yung-Lin','Lee'),(253,'Wei-Chen','Lee'),(254,'Sen-Yung','Hsieh'),(255,'Ulisses','Braga-Neto'),(256,'Ernesto','Marques'),(257,'Vibha','Bhatnagar'),(258,'Michael','Stoto'),(259,'Sally','Morton'),(260,'Rob','Boer'),(261,'Samuel','Bozzette'),(262,'Gérard','Krause'),(263,'Bonita','Brodhun'),(264,'Doris','Altmann'),(265,'Hermann','Claus'),(266,'Justus','Benzler'),(267,'Emine','Alp'),(268,'Andreas','Voss'),(269,'Yuwana','Podin'),(270,'Edna','Gias'),(271,'Flora','Ong'),(272,'Yee-Wei','Leong'),(273,'Siew-Fung','Yee'),(274,'Mohd','Yusof'),(275,'David','Perera'),(276,'Bibiana','Teo'),(277,'Thian-Yew','Wee'),(278,'Sik-Chi','Yao'),(279,'Sik-King','Yao'),(280,'Andrew','Kiyu'),(281,'Mohd','Taha Arif'),(282,'Mary','Cardosa'),(283,'Jalan','Abang'),(284,'Haji','Openg'),(285,'Kwang-Il','Lim'),(286,'Tobias','Lang'),(287,'Vy','Lam'),(288,'John','Yin'),(289,'Achim','Aigner'),(290,'Julien','Bobe'),(291,'Jerôme','Montfort'),(292,'Thaovi','Nguyen'),(293,'Alexis','Fostier'),(294,'Brian','Hanley'),(295,'Scott','Johnson'),(296,'David','Marshall'),(297,'Gerda','Harms'),(298,'Christie','Miller'),(299,'Christopher','Sherrill'),(300,'Edward','Beaty'),(301,'Scott','Lederer'),(302,'Eric','Roesch'),(303,'Gary','Madsen'),(304,'Gary','Hoffman'),(305,'Ronald','Laessig'),(306,'Greg','Kopish'),(307,'Mei','Wang Baker'),(308,'Steven','Benner'),(309,'Philip','Farrell'),(310,'James','Prudent'),(311,'Clark','Henderson'),(312,'Christine','Anderson'),(313,'Michael','Howard'),(314,'Beatrice','Cobucci-Ponzano'),(315,'Fiorella','Conte'),(316,'Dario','Benelli'),(317,'Paola','Londei'),(318,'Angela','Flagiello'),(319,'Maria','Monti'),(320,'Piero','Pucci'),(321,'Mosè','Rossi'),(322,'Marco','Moracci'),(323,'Kuan-Teh','Jeang'),(324,'Venkat','Yedavalli'),(325,'Jian','Huang'),(326,'Alex','Gutteridge'),(327,'Wataru','Honda'),(328,'Minoru','Kanehisa'),(329,'Tariq','Madani'),(330,'Aisha','Al-Ghamdi'),(331,'Albert','Van Der Vliet'),(332,'Jason','Eiserich'),(333,'Carroll','Cross'),(334,'Erika','Crouch'),(335,'Joseph','Domachowske'),(336,'Cynthia','Bonville'),(337,'Helene','Rosenberg'),(338,'Alexander','Pasternak'),(339,'Erwin','Van Den Born'),(340,'Willy','Spaan'),(341,'Eric','Snijder'),(342,'Gonzalo','Alvarez'),(343,'Paul','Hébert'),(344,'Sharyn','Szick'),(345,'Jonathan','Ball'),(346,'Richard','Venn'),(347,'Dirk-Jan','Slebos'),(348,'Stefan','Ryter'),(349,'Augustine','Choi'),(350,'Fu-Chiang','Tsui'),(351,'Jeremy','Espino'),(352,'Virginia','Dato'),(353,'Per','Gesteland'),(354,'Judith','Hutman'),(355,'Michael','Wagner'),(356,'Senya','Ivanov'),(357,'','Matsufuji'),(358,'Yasuko','Murakami'),(359,'Raymond','Gesteland'),(360,'John','Atkins'),(361,'Stephanie','Shi'),(362,'Peiyong','Huang'),(363,'Hsin-Pai','Li'),(364,'Michael','Lai'),(365,'Julia','Pridgeon'),(366,'Thangiah','Geetha'),(367,'Marie','Wooten'),(368,'Aspasia','Ploubidou'),(369,'Violaine','Moreau'),(370,'Keith','Ashman'),(371,'Inge','Reckmann'),(372,'Cayetano','Gonza'),(373,'Â','Lez'),(374,'Michael','Way'),(375,'Biehuoy','Shieh'),(376,'Ching','Li'),(377,'Joanne','Verheij'),(378,'Johan','Ab'),(379,'','Groeneveld'),(380,'Albertus','Beishuizen'),(381,'Arthur','Van Lingen'),(382,'Alberdina','Simoons-Smit'),(383,'Rob','Jm'),(384,'Strack','Van Schijndel'),(385,'Travis','Porco'),(386,'Karen','Holbrook'),(387,'Susan','Fernyak'),(388,'Diane','Portnoy'),(389,'Randy','Reiter'),(390,'Tomás','Aragón'),(391,'Ted','Kremer'),(392,'Mikael','Rinne'),(393,'Yi','Xu'),(394,'Ming','Chen'),(395,'Mark','Kelley'),(396,'Marit','Bratlie'),(397,'Finn','Drabløs'),(398,'Genevieve','Howse'),(399,'Yves','Le Loir'),(400,'Vasco','Azevedo'),(401,'Sergio','Oliveira'),(402,'Daniela','Freitas'),(403,'Anderson','Miyoshi'),(404,'Luis','Bermúdez-Humarán'),(405,'Sébastien','Nouaille'),(406,'Luciana','Ribeiro'),(407,'Sophie','Leclercq'),(408,'Jane','Gabriel'),(409,'Valeria','Guimaraes'),(410,'Maricê','Oliveira'),(411,'Cathy','Charlier'),(412,'Michel','Gautier'),(413,'Philippe','Langella'),(414,'Christine','Dufraigne'),(415,'Bernard','Fertil'),(416,'Sylvain','Lespinats'),(417,'Alain','Giron'),(418,'Patrick','Deschavanne'),(419,'Mazen','Karaman'),(420,'Susan','Groshen'),(421,'Chi-Chiang','Lee'),(422,'Brian','Pike'),(423,'Joseph','Hacia'),(424,'Hu','Zhu'),(425,'Hsiao','Chan'),(426,'Zuoming','Zhou'),(427,'Jianming','Li'),(428,'Hui','Zhu'),(429,'Lanlan','Yin'),(430,'Ming','Xu'),(431,'Lijun','Cheng'),(432,'Jiahao','Sha'),(433,'Willemijntje','Hoogerwerf'),(434,'Kelly','Gondesen'),(435,'Shu-Yuan','Xiao'),(436,'John','Winston'),(437,'William','Willis'),(438,'Pankaj','Pasricha'),(439,'Remy','Froissart'),(440,'Denis','Roze'),(441,'Marilyne','Uzest'),(442,'Lionel','Galibert'),(443,'Stephane','Blanc'),(444,'Yannis','Michalakis'),(445,'Ewan','Plant'),(446,'Jonathan','Dinman'),(447,'K','Jacobs'),(448,'J','Harger'),(449,'A','Meskauskas'),(450,'J','Jacobs'),(451,'J','Baxter'),(452,'A','Petrov'),(453,'J','Dinman'),(454,'Brock','Binkowski'),(455,'Kathryn','Richmond'),(456,'James','Kaysen'),(457,'Michael','Sussman'),(458,'Peter','Belshaw'),(459,'Sandrine','Imbeaud'),(460,'Esther','Graudens'),(461,'Virginie','Boulanger'),(462,'Xavier','Barlet'),(463,'Patrick','Zaborski'),(464,'Eric','Eveno'),(465,'Odilo','Mueller'),(466,'Andreas','Schroeder'),(467,'Charles','Auffray'),(468,'Katri','Mä Kelä Inen'),(469,'Kristiina','Mä Kinen'),(470,'Vivian','Lin'),(471,'Priscilla','Robinson'),(472,'Open','Access'),(473,'Stephen','Berger'),(474,'Greg','Martin'),(475,'Tsutomu','Suzuki'),(476,'Takuya','Ueda'),(477,'Kimitsuna','Watanabe'),(478,'Magdalena','Alejska'),(479,'Magdalena','Figlerowicz'),(480,'Nelli','Malinowska'),(481,'Anna','Urbanowicz'),(482,'Marek','Figlerowicz'),(483,'Joseph','Golden'),(484,'Leslie','Schiff'),(485,'Elzbieta','Kierzek'),(486,'Anna','Ciesielska'),(487,'Karol','Pasternak'),(488,'David','Mathews'),(489,'Douglas','Turner'),(490,'Ryszard','Kierzek'),(491,'Shea','Gardner'),(492,'Marisa','Lam'),(493,'Jason','Smith'),(494,'Clinton','Torres'),(495,'Tom','Slezak'),(496,'Muthuraman','Sathiamurthy'),(497,'Bjoern','Peters'),(498,'Huynh-Hoa','Bui'),(499,'John','Sidney'),(500,'John','Mokili'),(501,'Stephen','Wilson'),(502,'Ward','Fleri'),(503,'Deborah','Mcguinness'),(504,'Philip','Bourne'),(505,'Alessandro','Sette'),(506,'Pål','Olsvik'),(507,'Kai','Lie'),(508,'Ann-Elise','Jordal'),(509,'Tom','Nilsen'),(510,'Ivar','Hordvik'),(511,'G','Rohde'),(512,'I','Borg'),(513,'U','Arinir'),(514,'J','Kronsbein'),(515,'R','Rausse'),(516,'T','Bauer'),(517,'A','Bufe'),(518,'G','Schultze-Werninghaus'),(519,'Manami','Inoue'),(520,'Motoki','Iwasaki'),(521,'Tetsuya','Otani'),(522,'Shizuka','Sasazuki'),(523,'Shoichiro','Tsugane'),(524,'Yasuyoshi','Mori'),(525,'Tsuyoshi','Hirano'),(526,'Tsugunori','Notomi'),(527,'* -Yasuyoshi_Mori@eiken','Jp'),(528,';','Tsuyoshi'),(529,'Hirano -Tsuyoshi_Hirano@eiken','Jp'),(530,';','Tsugunori'),(531,'Notomi -Tsugunori_Notomi@eiken','Jp'),(532,'Han-Zhu','Qian'),(533,'Joseph','Schumacher'),(534,'Huey','Chen'),(535,'Yu-Hua','Ruan'),(536,'Theodore','Oliphant'),(537,'Michael','Engle'),(538,'Grant','Nybakken'),(539,'Chris','Doane'),(540,'Syd','Johnson'),(541,'Ling','Huang'),(542,'Sergey','Gorlatov'),(543,'Erin','Mehlhop'),(544,'Anantha','Marri'),(545,'Kyung','Chung'),(546,'Gregory','Ebel'),(547,'Laura','Kramer'),(548,'Daved','Fremont'),(549,'Michael','Diamond'),(550,'Ran','Balicer'),(551,'Saad','Omer'),(552,'Daniel','Barnett'),(553,'George','Everly'),(554,'Carly','Ruderman'),(555,'Shawn','Tracy'),(556,'Cécile','Bensimon'),(557,'Mark','Bernstein'),(558,'Laura','Hawryluck'),(559,'Randi','Zlotnik Shaul'),(560,'Ross','Upshur'),(561,'Mepur','Ravindranath'),(562,'Thiruverkadu','Saravanan'),(563,'Clarence','Monteclaro'),(564,'Naftali','Presser'),(565,'Xing','Ye'),(566,'Senthamil','Selvan'),(567,'Stanley','Brosman'),(568,'Luigi','Franciosi'),(569,'Clive','Page'),(570,'Bartolome','Celli'),(571,'Mario','Cazzola'),(572,'Michael','Walker'),(573,'Meindert','Danhof'),(574,'Klaus','Rabe'),(575,'Oscar','Della Pasqua'),(576,'Yi-Shin','Pan'),(577,'Yun-Shien','Lee'),(578,'Yung-Lin','Lee'),(579,'Wei-Chen','Lee'),(580,'Sen-Yung','Hsieh'),(581,'Ulisses','Braga-Neto'),(582,'Ernesto','Marques'),(583,'Vibha','Bhatnagar'),(584,'Michael','Stoto'),(585,'Sally','Morton'),(586,'Rob','Boer'),(587,'Samuel','Bozzette'),(588,'Gérard','Krause'),(589,'Bonita','Brodhun'),(590,'Doris','Altmann'),(591,'Hermann','Claus'),(592,'Justus','Benzler'),(593,'Emine','Alp'),(594,'Andreas','Voss'),(595,'Yuwana','Podin'),(596,'Edna','Gias'),(597,'Flora','Ong'),(598,'Yee-Wei','Leong'),(599,'Siew-Fung','Yee'),(600,'Mohd','Yusof'),(601,'David','Perera'),(602,'Bibiana','Teo'),(603,'Thian-Yew','Wee'),(604,'Sik-Chi','Yao'),(605,'Sik-King','Yao'),(606,'Andrew','Kiyu'),(607,'Mohd','Taha Arif'),(608,'Mary','Cardosa'),(609,'Jalan','Abang'),(610,'Haji','Openg'),(611,'Kwang-Il','Lim'),(612,'Tobias','Lang'),(613,'Vy','Lam'),(614,'John','Yin'),(615,'Achim','Aigner'),(616,'Julien','Bobe'),(617,'Jerôme','Montfort'),(618,'Thaovi','Nguyen'),(619,'Alexis','Fostier'),(620,'Brian','Hanley'),(621,'Scott','Johnson'),(622,'David','Marshall'),(623,'Gerda','Harms'),(624,'Christie','Miller'),(625,'Christopher','Sherrill'),(626,'Edward','Beaty'),(627,'Scott','Lederer'),(628,'Eric','Roesch'),(629,'Gary','Madsen'),(630,'Gary','Hoffman'),(631,'Ronald','Laessig'),(632,'Greg','Kopish'),(633,'Mei','Wang Baker'),(634,'Steven','Benner'),(635,'Philip','Farrell'),(636,'James','Prudent'),(637,'Clark','Henderson'),(638,'Christine','Anderson'),(639,'Michael','Howard'),(640,'Beatrice','Cobucci-Ponzano'),(641,'Fiorella','Conte'),(642,'Dario','Benelli'),(643,'Paola','Londei'),(644,'Angela','Flagiello'),(645,'Maria','Monti'),(646,'Piero','Pucci'),(647,'Mosè','Rossi'),(648,'Marco','Moracci'),(649,'Kuan-Teh','Jeang'),(650,'Venkat','Yedavalli'),(651,'Jian','Huang'),(652,'Alex','Gutteridge'),(653,'Wataru','Honda'),(654,'Minoru','Kanehisa'),(655,'Regina','Armstrong'),(656,'Tuan','Le'),(657,'Nicole','Flint'),(658,'Adam','Vana'),(659,'Yong-Xing','Zhou'),(660,'Nina','Mccrate'),(661,'Mychel','Varner'),(662,'Kenneth','Kim'),(663,'Maria','Nagan'),(664,'[','Uuu'),(665,'M','Yarian'),(666,'E','Marszalek'),(667,'A','Sochacka'),(668,'R','Malkiewicz'),(669,'A','Guenther'),(670,'P','Miskiewicz'),(671,'Igor','Kutyavin'),(672,'Dave','Milesi'),(673,'Yevgeniy','Belousov'),(674,'Mikhail','Podyminogin'),(675,'Alexei','Vorobiev'),(676,'Vladimir','Gorn'),(677,'Eugeny','Lukhtanov'),(678,'Nicolaas','Vermeulen'),(679,'Walt','Mahoney'),(680,'Alison','Thompson'),(681,'Karen','Faith'),(682,'Jennifer','Gibson'),(683,'Ross','Upshur'),(684,'Kuo-Chin','Kao'),(685,'Ying-Huang','Tsai'),(686,'Yao-Kuang','Wu'),(687,'Ning-Hung','Chen'),(688,'Meng-Jer','Hsieh'),(689,'Shiu-Feng','Huang'),(690,'Chung-Chi','Huang'),(691,'J','Malpica'),(692,'S','Sacristán'),(693,'A','Fraile'),(694,'F','García-Arenal'),(695,'Don','Klinkenberg'),(696,'Christophe','Fraser'),(697,'Hans','Heesterbeek'),(698,'P','Caley'),(699,'N','Becker'),(700,'D','Philp'),(701,'Wendelien','Oswald'),(702,'Thomas','Geisbert'),(703,'Kelly','Davis'),(704,'Joan','Geisbert'),(705,'Nancy','Sullivan'),(706,'Peter','Jahrling'),(707,'Paul','Parren'),(708,'Dennis','Burton'),(709,'Douglas','Moore'),(710,'Alan','Shiell'),(711,'Tom','Noseworthy'),(712,'Margaret','Russell'),(713,'Gerald','Predy'),(714,'Marina','Smirnova'),(715,'John','Birchall'),(716,'Jeffrey','Pearson'),(717,'Eric','Milbrandt'),(718,'Derek','Angus'),(719,'Xiaowei','Zhu'),(720,'Mark','Gerstein'),(721,'Michael','Snyder'),(722,'Michael',''),(723,'Timothy','Reddy'),(724,'Boris','Shakhnovich'),(725,'Daniel','Roberts'),(726,'Shelley','Russek'),(727,'Charles','Delisi'),(728,'T','Garske'),(729,'P','Clarke'),(730,'A','Ghani'),(731,'Martin','Eichner'),(732,'Markus','Schwehm'),(733,'Hans-Peter','Duerr'),(734,'Stefan','Brockmann'),(735,'J','Evans'),(736,'K','Aronstein'),(737,'Y','Chen'),(738,'C','Hetru'),(739,'J.-L','Imler'),(740,'H','Jiang'),(741,'M','Kanost'),(742,'G','Thompson'),(743,'Z','Zou'),(744,'D','Hultmark'),(745,'A','Catic'),(746,'S','Misaghi'),(747,'G','Korbel'),(748,'H','Ploegh'),(749,'Silvia','Villullas'),(750,'Darryl','Hill'),(751,'Richard','Sessions'),(752,'Jon','Rea'),(753,'Mumtaz','Virji'),(754,'David','Murdoch'),(755,'Willem','Df Venter'),(756,'Annelies','Rie'),(757,'Charles','Feldman'),(758,'R','Sampath'),(759,'K','Russell'),(760,'C','Massire'),(761,'M','Eshoo'),(762,'V','Harpin'),(763,'Irina','Chis Ster'),(764,'Neil','Ferguson'),(765,'Kimberly','Dyer'),(766,'Ingrid','Schellens'),(767,'Cynthia','Bonville'),(768,'Brittany','Martin'),(769,'Joseph','Domachowske'),(770,'Helene','Rosenberg'),(771,'David','Dausey'),(772,'James','Buehler'),(773,'Nicole','Lurie'),(774,'Hui','Yu'),(775,'Wang','†2'),(776,'Kang','Tu'),(777,'Lu','Xie'),(778,'Yuan-Yuan','Li'),(779,'Yi-Xue','Li'),(780,'Ulla','Schwertassek'),(781,'Yves','Balmer'),(782,'Marcus','Gutscher'),(783,'Lars','Weingarten'),(784,'Marc','Preuss'),(785,'Johanna','Engelhard'),(786,'Monique','Winkler'),(787,'Tobias','Dick'),(788,'André','Catic'),(789,'Edda','Fiebiger'),(790,'Gregory','Korbel'),(791,'','Danië L Blom'),(792,'Paul','Galardy'),(793,'Hidde','Ploegh'),(794,'Jin','Li'),(795,'Ross','Berbeco'),(796,'Robert','Distel'),(797,'Pasi','Jä Nne'),(798,'Lilin','Wang'),(799,'G','Makrigiorgos'),(800,'V','Villard'),(801,'G','Agak'),(802,'G','Frank'),(803,'A','Jafarshad'),(804,'C','Servis'),(805,'Guoqing','Lu'),(806,'Thaine','Rowley'),(807,'Rebecca','Garten'),(808,'Ruben','Donis'),(809,'Hans','Duerr'),(810,'Stefan','Brockmann'),(811,'Isolde','Piechotowski'),(812,'Markus','Schwehm'),(813,'Martin','Eichner'),(814,'Eric','Dumonteil'),(815,'Christophe','Fraser'),(816,'Scott','Robbins'),(817,'Gilles','Bessou'),(818,'Amé','Lie Cornillon'),(819,'Nicolas','Zucchini'),(820,'Brigitte','Rupp'),(821,'Zsolt','Ruzsics'),(822,'Torsten','Sacher'),(823,'Elena','Tomasello'),(824,'Eric','Vivier'),(825,'Ulrich','Koszinowski'),(826,'Marc','Dalod');
/*!40000 ALTER TABLE `Author` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Paper`
--

DROP TABLE IF EXISTS `Paper`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `Paper` (
  `idPaper` varchar(200) NOT NULL,
  `title` text,
  `abstract` text,
  `body` text,
  `year` datetime DEFAULT NULL,
  `doi` varchar(45) DEFAULT NULL,
  `word_count` int DEFAULT NULL,
  `unique_word_count` int DEFAULT NULL,
  PRIMARY KEY (`idPaper`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Paper`
--

LOCK TABLES `Paper` WRITE;
/*!40000 ALTER TABLE `Paper` DISABLE KEYS */;
INSERT INTO `Paper` VALUES ('025339bfce1cb8efa81c5accdabefe04dcdac9d2.json','Managing emerging infectious diseases: Is a federal system an impediment to effective laws?','In the 1980\'s and 1990\'s HIV/AIDS was the emerging infectious disease. In 2003–2004 we saw the emergence of SARS, Avian influenza and Anthrax in a man made form used for bioterrorism. Emergency powers legislation in Australia is a patchwork of Commonwealth quarantine laws and State and Territory based emergency powers in public health legislation. It is time for a review of such legislation and time for consideration of the efficacy of such legislation from a country wide perspective in an age when we have to consider the possibility of mass outbreaks of communicable diseases which ignore jurisdictional boundaries.',' management infectious disease increasingly complex world mass international travel globalization terrorism heighten challenge Federal State Territory government ensure Australia law sufficiently flexible address type problem emerge HIV AIDS late emerge infectious disease considerable thought legislative response number australian jurisdiction particular attention give unique feature disease method transmission kind people risk protection need community infected population well manage care infect minimize new infection health worker researcher begin find effective strategy far find help promote reduction spread HIV involve adoption law policy protect right people risk infection good example legislative response adopt approach find section Victorian Health Act section emphasize need protect privacy infected individual undertake stage response proportional risk present infected individual legislation effective HIV praise progressive approach community face emergence new infectious disease SARS Anthrax whilst case disease Australia threat possible outbreak acknowledge response plan Anthrax new infectious disease human infect anthrax handle product infected animal breathe anthrax spore infected animal product like wool example People infected gastrointestinal anthrax eat undercooked meat infected animal manufacture use weapon bioterrorism force rethink management new context infectious disease different feature HIV spread transmission infect bodily fluid blood semen SARS contrast transmit droplet infected case result coughing carry virus close contact infection profile SARS require plan possible overrun Intensive Care Units likely infection number ICU staff affect morale capacity cope Anthrax raise different problem include possible investigation terrorist suspect alongside investigation outbreak infectious disease difficulty raise likelihood public panic flooding public health official report suspicious white powder early medium report spread avian influenza South East Asia disease different feature HIV AIDS SARS approach australian outbreak different main difference source transmission virus infected bird human little difference ordinary influenza symptom vary severity treatment virus early predict emerge infectious disease current spread give rise concern possibility Australia federal system parallel set law operation Commonwealth Constitution set legislative power Commonwealth specific power list Commonwealth constitution state constitution broad power cover matter peace order good governance Commonwealth specific power legislate respect health quarantine power national legislative scheme public health rely cooperative approach States Territories cumbersome difficult specific head power Commonwealth limit ability legislate respect health legislative power Commonwealth specify Constitution include expressly activity comprise field public health reason Commonwealth emergency health power quarantine power quarantine power currently restrict isolation border country people plant animal prevent spread disease real possibility quarantine law broad scope depend widely High Court interpret section Commonwealth Constitution quarantine law override state law long remain law respect quarantine power potentially colossus far expansion legislative authority field public health concern quarantine power likely candidate head power base development commonwealth law management public health emergency possibility external affair power relevant treaty international agreement give effect domestic law legislation limit law give effect treaty state territory range emergency power available exist public health legislation relatively old example Health Act WA Public Health Act NT base ordinance act currently review health emergency power vary jurisdiction include power support disease surveillance contact tracing order restrict behavior movement individual infectious disease certain circumstance power recall food search premise seize property close building range substantial intrusive power suggest time consider state territory public health legislation contain sufficient measure manage outbreak infectious disease modern environment include mass travel swift spread infection additional complexity raise fear bioterrorism currently public health emergency cause spread emerge infectious disease Australia need rely patchwork legislative measure assist cope commonwealth quarantine law State Territory power public health legislation need address problem outbreak occur border area jurisdiction doubt airspace offshore state territory response require addition quarantine measure confusion jurisdiction application State Territory power State Territory public health act adequately provide interjurisdictional communication cooperation difficulty infectious disease cause overseas death people State Territory circumstance australian coronial investigation consider desirable situation jurisdiction australian coroner trigger State Territory coronial law apply different inquest different law undertake different coroner incident suggest time look efficiency emergency power law Australia map law jurisdiction Commonwealth quarantine law consider effectiveness face outbreak fast moving easily spread infectious disease efficacy Australia law consider relation bioterrorism infection anthrax despite great deal medium coverage infection death responsible legislature ought acknowledge possibility ensure law ready support swift effective response consider individual piece legislation task manage outbreak newly emerge infectious disease jurisdiction currently review public health legislation doubt proper consideration issue review think legislation jurisdiction Commonwealth quarantine fit power enable communication cooperation jurisdiction outbreak infectious disease kind opportunity coordinated response public health order jurisdiction travel jurisdiction infected individual travel arrangement outbreak occur close interstate border outbreak bus carry passenger Victoria South Australia northern encouraging note specific legislation mechanism achieve communication cooperation jurisdiction Communicable Disease Network Australia CDNA Network fact successful foster regular communication Communicable Disease Units country involve coordinated action number multistate outbreak despite existence network good work relationship government official agency different jurisdiction outbreak communicable disease require existence legislative power public health emergency generate confusion panic clarity power way power interact crucial emergency apparent Bali tragedy coroner jurisdiction trigger differently different jurisdiction act support communication cooperation inquest need death people ordinarily resident jurisdiction time find shortcoming legislation crisis review efficacy law work protect public health Australians undertake possible overcome hangover federation betterment Australians relation corporation law doubt recently raise constitutional basis corporation law scheme States Territories able cooperate refer necessary power Commonwealth provide certainty law govern corporation public health important governance corporation cooperate certainty flexibility consistent approach protect right subject broad States Territories generally reluctant refer power Commonwealth time seriously discuss referral power context health emergency power time Commonwealth States Territories recognise need law work set law protect country simply individual law protect individual jurisdiction work internationally area model State Emergency Health Powers Act develop preamble Act rationale development set wake tragic event September nation realize Government foremost responsibility protect health safety wellbeing citizen new emerge danger include emergent resurgent infectious disease incident civilian mass casualty immediate threat population renew focus prevention detection management containment public health emergency call like Australia Federal system model intend take state wish date pass half state bill excellent starting point development australian model number legislative mechanism support nationally uniform approach health emergency power legislation Australia development adoption model food legislation provide useful model cooperative uniform approach model act develop consultation jurisdiction cover area agree core area Act ought subject national approach provision consider administrative adopt discretion jurisdiction intergovernmental agreement sign mechanism protect uniformity legislation agreement set Ministerial Council support Food Regulation Standing Committee Council responsibility decide proposal amend model decision favor amendment States Territories use good endeavor submit respective Parliaments legislation give effect amendment law important tool support management outbreak infectious disease existence Federal system mean different approach State Territory Commonwealth control quarantine newly emerge infectious disease create real threat public health era easy mass travel present threat bioterrorism mean time Australia examine law contain manage infectious disease outbreak law examine effectiveness area cover ought enable response protect health Australians crosse border easily SARS avian influenza','2004-11-19 00:00:00','http://dx.doi.org/10.1186/1743-8462-1-7',1130,497),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json','Recombination Every Day: Abundant Recombination in a Virus during a Single Multi-Cellular Host Infection','Viral recombination can dramatically impact evolution and epidemiology. In viruses, the recombination rate depends on the frequency of genetic exchange between different viral genomes within an infected host cell and on the frequency at which such co-infections occur. While the recombination rate has been recently evaluated in experimentally co-infected cell cultures for several viruses, direct quantification at the most biologically significant level, that of a host infection, is still lacking. This study fills this gap using the cauliflower mosaic virus as a model. We distributed four neutral markers along the viral genome, and co-inoculated host plants with marker-containing and wild-type viruses. The frequency of recombinant genomes was evaluated 21 d post-inoculation. On average, over 50% of viral genomes recovered after a single host infection were recombinants, clearly indicating that recombination is very frequent in this virus. Estimates of the recombination rate show that all regions of the genome are equally affected by this process. Assuming that ten viral replication cycles occurred during our experiment—based on data on the timing of coat protein detection—the per base and replication cycle recombination rate was on the order of 2 × 10(−5) to 4 × 10(−5). This first determination of a virus recombination rate during a single multi-cellular host infection indicates that recombination is very frequent in the everyday life of this virus.',' increase number length viral sequence available recombinant mosaic virus recognize frequently recombination event demonstrate associate virus expand host range increase virulence accompany origin major change virus adaptation remain unclear recombination event represent highly frequent significant phenomenon everyday life virus virus exchange genetic material different viral genome co infect host cell Progeny hybrid different mechanism reassortment segment parental genome fragment intra molecular recombination polymerase switch template RNA virus homologous non homologous recombination RNA DNA virus quantification viral recombination multicellular organism attempt distinct experimental approach vitro cell culture vivo live host vitro approach far apply animal virus allow establishment intrinsic recombination rate experimentally co infected cell cell culture necessarily reflect situation entire live host frequency coinfected cell poorly know depend factor size pathogen population relative frequency distribution different variant host defense mechanism prevent secondary infection cell vivo experimental approach close biological condition informative actually happen real world discuss numerous experimental constraint far preclude actual quantification baseline rate recombination experimental design extreme positive selection recombinant genome viable study use complementation technique detect recombinant PCR infected host tissue provide information presence frequency viral population far quantitative PCR quantitative method apply evaluate number recombinant appear experimentally infected live host finally recent method base sequence analysis infer population recombination rate individual recombination rate result method certainly vivo recombination account caveat isolate collect different host different geographical region selective neutrality sequence variation estimate base clearly establish estimate study essence address estimation recombination rate different evolutionary scale take currently available information indicate viral recombination rate estimate directly time space scale correspond single multi cellular host infection level significant biology evolution virus study intend fill gap evaluate recombination frequency cauliflower mosaic virus CaMV single passage host plant turnip Brassica pararetrovirus major grouping contain hepadnavirus hepatitis virus badnaviruse banana streak virus caulimoviruse CaMV pararetroviruse characterize non segmented double strand dna genome enter host cell nucleus viral dna accumulate minichromosome transcription ensure host RNA polymerase II CaMV genome consist approximately bp encode viral gene product detect planta figure viral protein express major transcript RNA encode RNA correspond entire genome serve mrna protein pre genomic RNA matrix protein product gene reverse transcribe genome genomic DNA concomitantly encapsidate detection CaMV recombinant turnip host report numerous time study demonstrate appearance infectious recombinant viral genome inoculation host plant infectious non infectious parental clone ii transgenic plant contain CaMV transgene CaMV genome miss correspond genomic region reveal inter genomic viral recombination demonstrate CaMV recombine transgene host genome study base phylogenetic analysis CaMV strain clearly suggest different origin different genomic region multiple recombination event evolution virus indirect experimental evidence indicate case CaMV recombination occur host nucleus different viral minichromosome presumably action DNA repair cellular machinery mechanism template switching reverse transcription predominant retrovirus certainly apply pararetroviruse reason basis numerous experimental datum CaMV generally believe recombine cytoplasm host cell legal template switching pre genomic RNA molecule illegal template switching RNA hypothesis recombination CaMV consider operate linear template reverse transcription extremity later ligate circularize genomic dna position figure cite study clearly demonstrate CaMV able recombine study base complementation technique non quantitative detection phylogenetically base inference recombination inform recombination exceptional event everyday process shape genetic composition CaMV population present work aim answer question end construct CaMV genome genetic marker demonstrate neutral competition experiment co inoculating host plant equal amount wild type marker contain CaMV particle generate mixed population impressive proportion recombinant distribute different class correspond exchange different genomic region detect quantify altogether recombinant genome average population analysis datum assume number viral replication infection period range indicate nucleotide replication cycle indicate origin replication reverse transcription occur direction indicate dot outermost circle like arrow reverse transcription accomplish viral reverse transcriptase RNA template DOI recombination rate CaMV order magnitude order entire genome provide quantification knowledge recombination rate virus population single passage single host figure suppose marker contain genomic region recombine predict detection quantification seven class recombinant genotype þbcd aþþþ aþcd þbþþ abþd þþcþ abcþ þþþd þþcd abþþ aþþd þbcþ aþcþ þbþd class detect frequency CaMV population analyze summarize Table proportion recombinant genome find mixed viral population astonishingly high similar co infected plant analyze table column range plant plant mean frequency standard error result indicate recombination event frequent invasion host plant CaMV represent knowledge direct quantification viral recombination infection live multi cellular host inferred generation recombination interference rate assume CaMV undergo replication cycle infection sampling give plant table recombination rate adjacent marker large order take distance nucleotide marker account yield average recombination rate nucleotide generation order recombination rate vary genome Kruskal Wallis test relax assumption number replication calculate recombination parameter assume generation effect number generation estimate linear double number generation result halving recombination rate detailed result show example average recombination rate assume generation equal respectively compare value table yield nucleotide generation recombination rate Table show order interference coefficient general negative indicate crossing genomic segment increase probability crossing occur genomic segment second order coefficient parameter average value close zero large variance mechanism lead result discuss following section major breakthrough work present lie space time scale experiment perform process occur course single infection multi cellular host obvious biological relevance disease previous study viral recombination suffer major drawback respect base conclusion experiment rely complementation non infectious virus virus undetermined relative fitness phylogenetically base analysis experiment cell culture reason detail introduction method provide information frequency recombination occurrence address question different temporal spatial scale result cell culture hand impose cell coinfection different viral variant potentially ing frequency recombination event study circumvent limitation analyze viral genotype sample infected plant course single infection invasion co infection cell organ tissue close natural half genome Table present CaMV population single passage host plant identify recombinant datum allow infer nucleotide generation recombination rate order time length generation time require give genome replication totally unknown plant virus experimental datum available CaMV base kinetic gene expression infected protoplast capsid protein produce reverse transcription encapsidation genomic dna couple phenomenon judge reasonable assume generation time average generation experiment case estimate mistaken verify linear relationship number generation allow immediate adjustment CaMV generation time precisely establish point consider clone genome successive replication event potentially allow timing experiment previously show CaMV mature virus particle accumulate compact inclusion body sequester long time inclusion frequent infected cell include leave invade virus population week viral population present age structure bias estimation recombination rate order minimize bias clone analyze collect young newly form leaf chance find genome unsequestered line assume high case data analysis conservative age structure lead underestimation recombination rate result interference pair locus negative recombination event locus apparently increase probability recombination pair loci believe parsimonious explanation negative interference base way infection build plant host divide infected host cell infect single virus genotype infect viral genotype analogous clonal propagation recombination undetectable recombination detectable result indicate frequent sample consist virus result mixture type host cell infection contain virus recombination virus recombination event yield impression negative interference conceptual argument support mathematical model easy detailed result show proportion population reproduce clonally analogous single infection remain reproduce panmictically negative interference infer exist example assume locus model real recombination rate interference apparent recombination interference parameter interestingly example show estimate recombination rate conservative fraction host cell singly infect multiply infect lead underestimation recombination rate judge calculate marker respectively find evidence recombination entire CaMV genome value remarkably similar recombination site evenly distribute genome consider template switching model major way recombinant create CaMV mention introduction hot spot template switching predict position extremity RNAs recombination mechanism associate second strand dna synthesis host cell dna repair machinery act significantly hot parameter follow recombination rate marker recombination rate marker recombination rate marker interference crossover segment interference crossover segment interference crossover segment second order interference account residual interference recombination rate maximum likelihood estimate confidence interval interference parameter obtain numerically explain Materials Methods DOI spot expect position sequence interruption design experiment position marker information putative hot spot position correspond end RNA nucleotide position putative hot spot end RNA nucleotide position respectively fall marker pair respectively result indicate hot spot quantitatively equivalent predict different recombination mechanism likely simply exist explanation observe CaMV exchange portion genome gene thereof astonishingly high frequency course single host infection knowledge viral recombination rate previously quantify experimentally plant virus contrast retrovirus particularly extensively investigate sense discuss case quantification intrinsic recombination rate carry artificially coinfecte cell culture estimated intrinsic nucleotide generation recombination rate order order magnitude high estimation CaMV reason detail probably underestimate host CaMV recombination rate believe intrinsic recombination rate CaMV high order pararetroviruse plant badnaviruse vertebrate hepadnavirus similar cycle host cell include step nuclear minichromosome genomic size RNA synthesis reverse transcription encapsidation vertebrate hepadnavirus hepatitis virus infect host different plant biology physiology lead totally different frequency cell co infection development virus population result informative pararetroviruse virus share biological characteristic extrapolate vertebrate pararetroviruse caution viral isolate plasmid complete genome CaMV isolate Cabb clone plasmid unique SalI restriction site analyze recombination different region genome introduce genetic marker position respectively approximately cardinal point CaMV circular double strand dna bp figure marker correspond single nucleotide change introduce PCR direct mutagenesis result duplication previously unique restriction site BsiWI PstI MluI saci plasmid designate pMark study target possible exchange gene viral genome marker introduce code region correspond open reading frame IV VI respectively important concern quantify recombination absence selection create neutral marker consequently marker consist synonymous mutation viral particle co inoculation generate parental virus particle plasmid pMark mechanically inoculate individual plant previously describe plant turnip rapa cv right grow glasshouse condition light dark photoperiod thirty day post inoculation symptomatic leave harvest viral particle purify describe early result preparation parental virus designate Cabb Mark quantify spectrometry formula describe Hull et al fix initial frequency marker value solution contain mg ml virus particle Cabb Mark ratio prepared plantlet co infect mechanical inoculation leave ll virus solution abrasive Celite AFA Fluka Ronkonkoma New York United States mixed CaMV population allow grow systemic infection estimation marker frequency mixed virus population design experimental protocol quantify marker frequency mixed Cabb Mark virus population single passage host plant individual plant inoculate equal amount Cabb Mark harvest post inoculation symptom fully develop viral DNA purify mg young newly form infected leave accord protocol describe previously precipitation step protocol viral dna resuspend purify Wizard DNA clean kit Promega Fitchburg Wisconsin United States TE mM Tris HCl mm EDTA ph aliquot viral dna preparation digest restriction enzyme correspond marker submit agarose gel electrophoresis color ethydium bromide expose uv individual restriction enzyme cut Cabb DNA twice Mark generate dna fragment different size attributable mixed population CaMV genome scan agarose gel estimate relative frequency genotype viral dna preparation marker position densitometry NIH image program statistical analysis frequency marker describe isolation individual CaMV genome identification recombinant identify quantify recombinant camv mixed population aliquot viral DNA preparation describe digest restriction enzyme sali directly clone correspond site viral population analyze genome length clone digest separately BsiWI PstI MluI saci test presence marker respectively experiment marker represent additional restriction site easily distinguish Cabb Mark genotype marker position agarose gel electrophoresis digest clone clone marker parental genotype clone harbor marker clearly recombinant high number recombinant detect marker eventually appear disappear spontaneous mutation neglect statistical analysis present different method quantify recombination CaMV genome method assume different marker neutral discuss assumption dataset test neutrality marker result plant co infect ratio Mark Cabb consist viral dna densitometry datum derive plant describe plant estimate frequency marker genome population second consist restriction individual genome length viral clone obtain co infected plant describe yield estimate frequency marker repeat different plant frequency different marker marker dataset second dataset test frequency significantly different expect value neutrality test dataset normality reject seven case Wilcoxon sign rank non parametric test marker dataset case value large cautionary remark analysis case find excess marker unfortunately dataset regard independent method frequency estimate obtain different plant second dataset subset plant independent estimate case minimal power detect significant deviation neutrality small sample size note stage deviation expect value cause slight deviation ratio infecting mixed solution deviation ratio frequency viral particle actually plant second relatively small sample size low statistical power test present detect large deviation result clearly precisely marker large effect recombination estimate affect slightly hypothetical selective effect marker fact introduce marker provoke silent substitution CaMV genome assume marker effectively neutral rest analysis dataset estimate recombination frequency consist genome length viral clone co infected plant individually genotype marker discuss detail Results recombination frequent concern marker approximately half genotype clone exhibit recombinant genotype meaningful try obtain quantitative estimate recombination datum aim analyze viral recombination live host consequently deal fact viral replication cycle occur infection last experiment wait long disease develop able recover sufficient amount viral dna infection base kinetic gene expression postulate replication cycle last seven cycle occur infection sampling time case assumption incorrect calculation assume seven replication cycle show result affect qualitatively slightly quantitatively important note assume recombination occur template switching mechanism recombination point view CaMV genome linear reverse transcription start finish position figure point circularization dna genome imply change contiguous marker consider true recombination simply stem circularization dna synthesis polymerase switch template estimate recombination rate marker write recurrence equation describe change frequency genotype generation assume random mating selection standard Wright Fisher population genetic model express frequency possible genotype generation later function initial frequency recombination parameter subsequently calculate maximum likelihood estimate recombination parameter asymptotic variance give initial frequency assume parental genotype Mark Cabb equal initial frequency genotype initial frequency zero frequency generation observe frequency state different value algebraic numerical calculation carry software Mathematica recombination parameter recombination rate adjacent loci recombination rate marker interference coefficient interference recombination event segment marker define parameter follow Christiansen particular recombination distribution locus respectively Tables important realize give definition parameter estimator recombination rate loci affect number loci consider word obtain estimation recombination rate marker consider genotypic frequency locus frequency loci plus locus complete information access fourmarker genotype information additional loci affect estimate interference coefficient prove impossible carry calculation locus algebraically instead computer program calculate expect genotypic frequency loci generation give state initial frequency specify recombination parameter combination recombination parameter calculate euclidean distance vector expect genotypic frequency observed genotypic frequency consider estimate recombination parameter yield minimal euclidean distance case estimate recombination rate pair locus equal second decimal estimate algebraically datum locus','2005-03-01 00:00:00','http://dx.doi.org/10.1371/journal.pbio.003008',2322,775),('06ced00a5fc04215949aa72528f2eeaae1d58927.json','Surfactant protein-D and pulmonary host defense','Surfactant protein-D (SP-D) participates in the innate response to inhaled microorganisms and organic antigens, and contributes to immune and inflammatory regulation within the lung. SP-D is synthesized and secreted by alveolar and bronchiolar epithelial cells, but is also expressed by epithelial cells lining various exocrine ducts and the mucosa of the gastrointestinal and genitourinary tracts. SP-D, a collagenous calcium-dependent lectin (or collectin), binds to surface glycoconjugates expressed by a wide variety of microorganisms, and to oligosaccharides associated with the surface of various complex organic antigens. SP-D also specifically interacts with glycoconjugates and other molecules expressed on the surface of macrophages, neutrophils, and lymphocytes. In addition, SP-D binds to specific surfactant-associated lipids and can influence the organization of lipid mixtures containing phosphatidylinositol in vitro. Consistent with these diverse in vitro activities is the observation that SP-D-deficient transgenic mice show abnormal accumulations of surfactant lipids, and respond abnormally to challenge with respiratory viruses and bacterial lipopolysaccharides. The phenotype of macrophages isolated from the lungs of SP-D-deficient mice is altered, and there is circumstantial evidence that abnormal oxidant metabolism and/or increased metalloproteinase expression contributes to the development of emphysema. The expression of SP-D is increased in response to many forms of lung injury, and deficient accumulation of appropriately oligomerized SP-D might contribute to the pathogenesis of a variety of human lung diseases.',' surfactant protein SP member collagenous subfamily calcium dependent lectin collectin include pulmonary surfactant protein SP serum mannose bind lectin collectin inter act wide variety microorganism lipid organic particulate antigen modulate function immune effector cell response ligand article review currently know site production structure function regulated expression SP Emphasis place functional attribute know ligand interaction structure function relationship believe important host defense additional information SP member collectin family reader refer recent review synthesize secrete airspace lung respiratory epithelium alveolar level SP constitutively synthesize secrete alveolar type II cell proximally lung SP secrete subset bronchiolar epithelial cell non ciliated Clara cell SP store secretory granule Clara cell likely SP subject regulated secretion granule exocy tosis level respiratory tract specie SP synthesize epithelial cell submucosal gland associate bronchi trachea alveolar macrophage strong cytoplasmic membrane labeling antibody SP contain detectable sp message lung major site SP production increase evidence extrapulmonary site expression assess monoclonal affinity purify antibody reverse transcriptase mediate PCR RT PCR hybridization assay tissue human large mammal summarize Table difficult entirely exclude crossreaction amplification related sequence localization site human tissue confirm monoclonal antibody combination RT PCR sequence amplified product non pulmonary expression largely restrict cell line epithelial surface duct certain glandular epithelial cell direct indirect continuity environment notable exception generalization include heart brain pancreatic islet testicular Leydig cell sp identify amnionic epithelial cell immunohistochemistry unclear synthesize locally derive lung way amniotic fluid interestingly site SP microscopically co localize SP bind protein putative SP receptor extrapulmonary expression describe small mammal rat sp message identify RNA extract skin blood vessel protein message identify gastric mucosa mesentery RT PCR SP message identify mouse stomach heart kidney kda reduce consist discrete structural domain short terminal domain relatively long collagenous domain short amphipathic connect peptide coil coil neck domain terminal Ctype lectin carbohydrate recognition domain CRD molecule consist trimeric subunit kDa associate termini Fig preparation SP contain predominance dodecamer trimeric subunit proportion oligomer vary specie example rat lavage recombinant rat sp exclusively assemble dodecamer trimer recombinant human SP secrete trimer dodecamer high order multimer sp isolate lavage patient alveolar proteinosis consist predominantly high order multimer contain trimeric subunit Fig crystallographic mutagenesis study suggest structural determinant saccharide binding similar originally describe mannose bind lectin bind calcium ion intrachain disulfide crosslink stabilize require tertiary structure CRD participate glucose mannose type recognition interaction glycolipid ligand phosphatidylinositol PI require participation terminal end protein trimeric cluster crd necessary high affinity binding carbohydrate ligand crystal structure human SP suggest spatial distribution crd trimeric subunit permit simultaneous cooperative interaction glycoconjugate display surface particulate ligand furthermore solid phase bind study show monomeric crd approximately fold low binding affinity multivalent ligand trimeric crd crystallographic study human SP suggest spatial organization crd trimer stabilize interaction terminal sequence trimeric neck domain interestingly crd deviation threefold asymmetry suggest flexibility crd relation neck dependence binding PI terminal sequence reflect conformational effect direct participation sequence ligand interaction collagen domain length SP highly conserve lack interruption repeat Gly sequence different amino acid collagenous protein domain enrich imino acid contain hydroxyproline unlike SP sp contain hydroxylysine collagen domain rat human bovine mouse SP lack cysteine residue cdna sequencing identify codon cysteine collagen domain pig SP suggest possibility alternative pattern chain association oligomeric assembly pig SP translate exon SP contain highly conserve unusually hydrophilic Gly sequence show little homology remainder collagen sequence functional significance region unknown suggest region contribute oligomer assembly mediate interaction cellular receptor collagen domain determine maximal spatial separation trimeric terminal lectin domain SP molecule contribute normal oligomeric assembly secretion example deletion entire collagen domain rat SP result secretion trimer dodecamer addition dipyridyl inhibitor prolyl hydroxylation interfere formation stable collagen helix cause intracellular accumulation kDa monomer dimer case complete conservation number Gly triplet suggest spatial separation trimeric crd critical normal SP function terminal peptide mature protein contain conserve cysteine residue position residue participate interchain disulfide crosslink stabilize trimer terminal association trimeric subunit stable oligomerization trimeric subunit permit cooperative bridging interaction spatially separate bind site surface different particle process form interchain disulfide bond complex appropriate crosslinking terminal domain rate limit secretion subcellular fractionation study suggest interchain bond form initially chain trimeric subunit subsequent rearrangement rough endoplasmic reticulum allow covalent crosslinke single chain subunit crosslinked chain associated elimination free thiol group mutant protein contain unpaired terminal cysteine residue secrete unclear result abnormality disulfide bond failure stabilize require terminal conformation collagen domain contain hydroxylysyl derive glycoside single link oligosaccharide specie human rat mouse cow site link glycosylation locate near terminal end collagenous domain recently show pig SP additional potential site link glycosylation CRD rat human lung lavage sp sialylate suggest charge heterogeneity cleavage highly purified neuraminidase preparation human amniotic fluid bovine lavage sp recover amniotic fluid show predominantly complex type biantennary structure sialic acid variant form SP kda identify lavage subset human lavage sample protein show link glycosylation threonyl residue terminal peptide domain present functional significance sugar know presence link glycosylation terminal domain predict interfere normal dodecamer assembly regard glycosylate kDa form human SP recover trimeric subunit small specie glycoprotein functional role attached carbohydrate unknown mutational analysis show link sugar rat sp require secretion dodecamer formation interaction variety microorganism designation mannose type type lectin sp preferentially bind simple complex saccharide contain mannose glucose inositol sp interact specific constituent pulmonary surfactant include PI glucosylceramide bind glucosylceramide involve interaction carbohydrate bind sequence CRD glucosyl moiety interaction SP PI involve interaction lipid CRD dependent interaction inositol moiety surface diverse complex array polysaccharide glycoconjugate class microorganism contain sugar recognize SP outcome interaction depend specific organism modify condition microbial growth potential consequence interaction include follow vary degree lectin dependent aggregation microbial agglutination enhance binding microorganism microbial aggregate receptor host cell phagocyte activation opsonic enhancement phagocytosis kill potentially involve cellular receptor SP Binding organism suspension degree aggregation sp bind purify lipopolysaccharide LPS isolate variety Gram negative organism addition LPS major cell wall component label lectin blotting outer membrane isolate Escherichia coli interaction involve recognition core oligosaccharide domain contain glucose heptose sp interact preferentially purify LPS molecule characterize short absent antigen preferentially agglutinate bacterial strain express predominance rough antigen deficient LPS core oligosaccharide domain LPS constitute major ligand SP Gram negative bacteria mechanism interaction group microorganism probably heterogeneous sp bind smooth unencapsulated strain Gram negative bacteria immunofluorescence mechanism uncertain quantity quality bind differ observe rough strain necessarily result agglutination LPS molecule surface bacteria heterogeneity extent maturation possible interaction mediate subpopulation LPS deficient antigen density bind site low high affinity binding recognition surface glycoconjugate gramnegative bacteria SP depend expression lectin specific residue give strain specie accessibility residue example SP bind inefficiently core region LPS encapsulate Klebsiella efficiently agglutinate correspond unencapsulated phase variant interaction SP core oligosaccharide Gram negative organism influence number repeat saccharide unit associate terminal antigen LPS potential ligand include antigen domain LPS certain capsular polysaccharide membraneassociate glycoprotein regard sp bind di mannose contain antigen express subset Klebsiella serotype ofek Sahly EC Crouch unpublished datum type lectin specifically SP mannose receptor interact specific capsular polysaccharide specific interaction SP capsular glycoconjugate exopolysaccharide describe mechanism interaction Gram positive organism elucidate lipoteichoic acid major glycolipid associate Gram positive cell wall detectably compete LPS bind SP Ofek Mesika Kalina Keisari Chang McGregor EC Crouch manuscript submit preliminary study observe binding compete partly maltose EDTA raise possibility binding complex Gram negative organism similar effect observe neutrophil preincubate SP slight enhancement uptake bacteria incubate human SP wash addition neutrophil notably extent binding internalization dependent extent multimerization human SP multimer demonstrate high potency difference cell type extent SP multimerization difference size organization bacterial aggregate account apparent inconsistency LPS mediate binding SP Gram negative bacteria SP bind study author suggest fungal aggregation inhibit phagocytosis interestingly SP binding directly inhibit fungal growth decrease outgrowth pseudohyphae invasive form fungus absence phagocytic cell possible effect secondary agglutination possibly result nutrient deprivation purify rat human sp inhibit infectivity hemagglutination activity influenza sp interact host cell directly indirectly indicate SP enhance phagocytosis killing certain microorganism enhance oxidant response microbial binding present study suggest enhancement phagocytosis SP involve participation opsonic receptor furthermore enhanced uptake IAV mediate viral aggregation enhanced interaction virus natural receptor host cell case SP interact directly host cell case influence behavior sp chemotactic haptotactic neutrophil certain mononuclear phagocyte elicit directional actin polymerization alveolar macrophage regard sp considerably potent sp early study natural protein isolate silicotic animal report direct effect oxidant metabolism isolated alveolar macrophage effect probably attribute endotoxin contamination aggregation purified dodecamer significantly increase production nitric oxide proinflammatory cytokine tumor necrosis factor Kesari Wang Mesika Crouch Ofek unpublished datum interestingly purify sp report increase production metalloproteinase absence significant effect proinflammatory cytokine production ability SP modulate variety cellular function little currently know potential cellular receptor protein compartment unclear uptake receptor dependent SP internalize association specific ligand class binding host cell CRD dependent CRD independent study demonstrate CRD dependent binding phagocyte inhibit EDTA compete saccharide vitro vivo indicate ability SP elicit chemotaxis neutrophilic monocytic cell depend lectin activity SP addition Kuan coworker report extract formaldehyde fix alveolar macrophage detergent largely eliminate binding purify SP suggest membrane associate ligand glycolipid receptor Dong Wright extend finding suggest PI contribute SP binding alveolar macrophage interest sp bind recombinant interaction link oligosaccharide membrane associate form widely express host cell possible serve bind site macrophage cell type phagocytic uptake certain bacteria neutrophil inhibit calcium chelation compete sugar result inhibition microbial agglutination lectin dependent interaction phagocyte Wang et al suggest SP bind lymphocytic cell lectin dependent mechanism regard interesting note glucosylceramide ligand SP vitro abundant neutral glycolipid express lymphoid cell reid co worker present evidence lectin independent binding study suggest binding involve know collectin receptor putative receptor protein widely express member scavenger receptor superfamily bind CRD SP calcium dependent manner require lectin activity SP protein immunolocalize alveolar macrophage membrane distribute SP different human tissue show mediate binding SP cell participate signal transduction event cdnas isolate lung show membrane span region protein abundant soluble component BAL give highly multimerize protein contain numerous potential ligand binding domain Fig possible protein cooperate SP neu tralization clearance certain ligand specifically mediate interaction SP host cell Wright co worker demonstrate binding SP isolated type II pneumocyte mechanism distinct binding macrophage binding dependent concentration time temperature require calcium sensitive protease treatment PI phospholipase internalized sp degrade recycle lamellar body sp binding alter uptake surfactant lipid sp demonstrate comparatively direct effect metabolism host cell situation self aggregation endotoxin contamination exclude possible explanation modulation cellular function require prior interaction SP ligand numerous potential physiological advantage presence active protein restrict site microbial antigenic deposition binding complex multivalent particulate antigen crd markedly alter conformation SP molecule respect spatial orientation arm relation terminal crosslinke domain respect spatial orientation crd give trimeric subunit charge SP particulate ligand lead local distant conformational change expose cryptic bind site cellular receptor preliminary evidence consistent notion interaction SP ligand alter capacity activate host cell table discuss sp isolate different multimeric form proteinosis lavage produce chinese hamster ovary cell transfecte human SP cDNA describe previously effect SP neutrophil response influenza virus highly dependent ability SP agglutinate viral particle agglutination activity directly correlate extent multimerization trimer bind virus little capacity modulate neutrophil interaction contrast highly multimerize protein great activity dodecamer observation factor favor enhanced oligomerization lead accumulation trimeric subunit promote influence sp function example liberation active trimer hypothetical microbial protease lead accumulation molecule inhibit aggregation dependent activity SP contrast recombinant trimeric crd stimulate chemotaxis decrease viral infectivity high order oligomer SP self aggregate precipitate presence calcium vitro functional consequence know lectin activity SP decrease nitric oxide dependent nitration tyrosine residue nitration decrease ability SP enhance adherence Pneumocystis alveolar macrophage similar finding report SP Conditions mildly acidic pH find endocytic compartment predict disrupt lectin dependent activity SP proteolytic degradation remain important possibility SP highly resistant degradation wide variety neutral protease vitro degradation product show accumulate pathological condition vivo glucose concentration level encounter diabete interfere SP ability interact specific strain IAV microorganism vitro microorganism release cell wall polysaccharide glycoconjugate interfere binding collectin organism regard sp recover rat instillation LPS airway show decrease lectin activity attribute occupancy CRD LPS reasonable speculate organism compete organism bind SP situation conceivably predispose secondary infection lastly potential inhibitory effect compete saccharide ligand present important methodological consideration experiment carbohydrate contain cell culture medium buffer Non type lectin difficult predict function SP airspace lectin overlap specificity present level mannose bind lectin probably low absence increase vascular permeability SP macrophage mannose receptor conceivably interact ligand distal airway alveolus interaction lead antagonistic cooperative effect furthermore little knowledge microanatomic distribution molecule specific circumstance vivo SP probably associate insoluble phase alveolar lining material macrophage mannose receptor membrane associate distribution alter setting lung injury model SP deficiency detectable anatomical physiological abnormality birth animal gradually develop patchy subpleural alveolar lipidosis associate type II cell hypertrophy accumulation enlarged foamy macrophage apparent expansion peribronchial lymphoid tissue interestingly mouse eventually develop distal acinar emphysema area subpleural fibrosis reflect continue inflammatory reaction associate abnormal oxidant metabolism metalloproteinase activity contrast SP deficient mouse essentially normal respiratory function surfactant lipid metabolism numerous apparent host defense abnormality capacity SP bind specific strain influenza vitro highly correlate capacity virus proliferate mouse vivo specifically strain oligosaccharide attachment ha preferentially neutralize SP vitro decrease proliferation mouse administration mannan virus increase replication IAV lung involvement mannose type type lectin implicate SP sensitive IAV strain replicate high titer lung diabetic mouse nondiabetic control replication virus positively correlate blood glucose level decrease response insulin treatment significantly blood glucose level comparable measure diabetic mouse sufficient inhibit interaction SP viral strain vitro strain interact SP interact SP replicate extent diabetic control mouse sp level increase association certain infection example SP level level serum mannose bind lectin increase markedly IAV infection impressive increase SP observe murine model Pneumocystis carinii aeruginosa infection deficient mouse extensively characterize respect host defense function decrease viral clearance enhanced inflammation challenge respiratory syncytial virus IAV Levine personal communication addition increase inflammation increase oxidant production decrease macrophage phagocytosis response intratracheally instill group streptococcus Haemophilus influenzae Levine personal communication overexpression wild type sp type II pneumocyte SP deficient mouse prevent lipidosis inflammatory change ability overexpresse wild type SP exogenous SP ameliorate abnormality describe coexist pulmonary abnormality complicate interpretation challenge model example macrophage activation enhance kill offset decrease result directly SP deficiency sp deficiency modify host response instill LPS decrease lung injury inflammatory cell recruitment bind potential antigen deliver macrophage antigen present cell contribute development acquire immunity regard publish observation suggest possible role development humoral cellular immunity response microorganism complex organic antigen example sp decrease lymphocyte proliferation interestingly single arm mutant potent intact dodecamer mediate effect sp bind oligosaccharide associate dust mite allergen inhibit binding specific IgE allergen possibly direct CRD dependent binding lymphocyte alteration level SP state oligomerization influence development immunological response contribute pathogenesis asthma hypersensitivity disorder potential interplay humoral immunity collectin regard antimicrobial host defense example increase glycosylation IAV coat protein adaptation believe help virus evade antibody mediate neutralization associate increase reactivity SP collectin relative potential importance antibody collectin mediate host defense influence subtle variation structure microbial surface little recent information developmental regulation SP expression general sp increase rapidly late gestation production SP increase culture fetal lung explant expression increase glucocorticoid exposure fetal rat glucocorticoid vivo lead precocious expression increase number SP express cell increase cellular level SP message sp produce constitutively lung protein accumulation gene expression inducible increase SP expression observe number disease state model Tables general synthesis secretion SP increase association lung injury activation respiratory epithelium example level SP mrna SP accumulation increase intratracheal instillation LPS SP expression alveolar bronchiolar epithelial cell increase exposure rat Keratinocyte growth factor KGF increase SP expression protein production association pneumocyte hyperplasia injury cause bleomycin addition level SP increase markedly response overexpression certain cytokine response microbial challenge upstream regulatory region SP gene demonstrate increase promoter activity presence glucocorticoid consistent finding vivo lung organ culture functional glucocorticoid response element identify effect dexamethasone secondary involve effect transregulatory molecule activity human SP promoter dependent conserve activator element bind member fos jun family transcriptional factor addition promoter contain multiple functional bind site CCAAT enhancer bind protein EBP transcription factor mutagenesis experiment suggest require basal stimulate promoter activity promoter activity markedly increase cell co transfection EBPβ cDNA YC crouch unpublished datum importance conserve element presence multiple bind site EBP transcription factor consistent observed modulation SP expression setting tissue injury sp promoter activity dependent binding thyroid transcription factor promoter activity dependent interact forkhead bind site upstream downstream element site bind hepatic nuclear apparently forkhead box protein lung adenocarcinoma nuclear extract comparison genomic cdna sequence suggest existence genetic polymorphism SP code sequence include terminal propeptide domain compare mature protein additional difference collagen domain position substitution conservative extent expect disrupt collagen helix Floros Table Increased SP accumulation expression animal modelssilicosis Rat Hyperoxia Rat Endotoxin LPS Rat Challenge aeruginosa Mouse Challenge IAV Pneumocystis carinii SCID mouse Rat overexpression Mouse SCID severe combine immunodeficiency co worker recently confirm existence polymorphism position mature protein potential biological significance know interestingly kda variant SP show link glycosylation suggest polymorphism associate altered glycosylation interestingly kda variant recover trimeric subunit raise possibility difference glycosylation residue immediately terminal influence multimerization capacity SP participate bridging interaction increase evidence SP interact specifically wide variety respiratory pathogen modulate leukocyte response organism participate aspect pulmonary immune inflammatory regulation table sp influence activity phagocyte CRD dependent CRD independent interaction effect SP result aggregation enhanced binding agglutinated ligand natural receptor lung major site SP expression likely protein generalize role host defense acute response infection tissue injury','2000-08-25 00:00:00','http://dx.doi.org/10.1186/rr19',2864,961),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json','Evaluation of potential reference genes in real-time RT-PCR studies of Atlantic salmon','BACKGROUND: Salmonid fishes are among the most widely studied model fish species but reports on systematic evaluation of reference genes in qRT-PCR studies is lacking. RESULTS: The stability of six potential reference genes was examined in eight tissues of Atlantic salmon (Salmo salar), to determine the most suitable genes to be used in quantitative real-time RT-PCR analyses. The relative transcription levels of genes encoding 18S rRNA, S20 ribosomal protein, β-actin, glyceraldehyde-3P-dehydrogenase (GAPDH), and two paralog genes encoding elongation factor 1A (EF1A(A )and EF1A(B)) were quantified in gills, liver, head kidney, spleen, thymus, brain, muscle, and posterior intestine in six untreated adult fish, in addition to a group of individuals that went through smoltification. Based on calculations performed with the geNorm VBA applet, which determines the most stable genes from a set of tested genes in a given cDNA sample, the ranking of the examined genes in adult Atlantic salmon was EF1A(B)>EF1A(A)>β-actin>18S rRNA>S20>GAPDH. When the same calculations were done on a total of 24 individuals from four stages in the smoltification process (presmolt, smolt, smoltified seawater and desmoltified freshwater), the gene ranking was EF1A(B)>EF1A(A)>S20>β-actin>18S rRNA>GAPDH. CONCLUSION: Overall, this work suggests that the EF1A(A )and EF1A(B )genes can be useful as reference genes in qRT-PCR examination of gene expression in the Atlantic salmon.',' real time RT PCR expression level target gene interest estimate basis endogenous control housekeeping gene ribosomal RNA rRNA total RNA commonly reference gene expression analysis today purpose control remove reduce difference sampling difference RNA quantity quality ideal endogenous control express constant level different tissue organism stage development unaffected experimental treatment express roughly level RNA study datum normalization real time RT PCR remain real problem especially absolute quantification numerous study reveal single universal gene constant expression level developmental experimental situation good choice reference gene use endogenous control vary depend tissue interest experiment large number gene reason select normalization mrna expression datum select reference gene fluctuate randomly sample small difference expression gene interest miss gene expression coefficient variation CV different group individual ideally low possible general stability potential reference gene test examined tissue cell different experimental design increase number paper discuss selection reference gene real time RT PCR analysis commonly reference gene encode dehydrogenase GAPDH actin recently use gene endogenous control scrutinize study document GAPDH actin gene caution control GAPDH mammal know play role broad range cellular mechanism review Sirover include key enzyme glycolysis overall GAPDH mrna level regulate large number physiological state use reference inappropriate experimental condition Actin major component protein scaffold support cell determine shape abundant intracellular protein eukaryotic cell commonly reference application actin gene recently characterize historical carryover northern blot conventional RT PCR general discussion use classic reference gene like GAPDH actin Huggett et al eukaryotic elongation factor elongation factor alpha play important role translation catalyze gtp dependent binding aminoacyl trna acceptor site ribosome protein involve broad diversity function constitute total cytoplasmic protein content cell human cdnas actively transcribe isoform clone review Thornton et al paralog gene recently apply reference real time qrt PCR Atlantic salmon plausible assume presence highly similar gene result tetraploidization event occur salmonid ancestor comparatively recent past rRNA gene consider ideal internal control qrt PCR analysis Ambion ribosomal RNA constitute total cellular RNA study show rRNA vary condition affect expression mrnas discuss Bustin Nolan question raise use ribosomal RNA gene reference Vandesompele et al stress fact imbalance rrna mrna fraction different sample make gene encode ribosomal RNAs unsuitable reference meet challenge accurate interpretation realtime qrt PCR datum author suggest index stable housekeeping gene normalization develop genorm VBA applet Microsoft Excel regard similar software tool BestKeeper develop Pfaffl et al tool find stable reference gene different experimental condition genorm software determine individual stability gene pool gene stability calculate accord similarity expression profile pair wise comparison geometric mean normalizing factor gene high stable gene suggest exclude stepwise fashion stable gene determine index suggest base good gene genorm select stable reference gene recent study aim work evaluate usefulness potential reference gene atlantic salmon salmonid fish widely study model fish specie general extensive basic information different aspect biology collect large scale dna sequence project salmon initiate laboratory work select classic reference gene encode GAPDH actin gene encode rrna ribosomal protein paralog gene encode elongation factor evaluate usefulness reference gene RNA tissue adult salmon subject real time PCR relative transcription level gene estimate phase young salmon go smoltification order check stability physiological stressful condition ranking examine gene analyze genorm show Table tissue muscle liver gill head kidney spleen thymus gene emerge stable gene rank number brain actin gene rank number intestine rrna gene rank bad gene tissue surprisingly GAPDH gene rank bad tissue liver head kidney spleen brain thymus confirm general skepticism use gene reference combine total ranking read rRNA GAPDH analyze datum Bestkeeper software analyze reference gene virus infect cell Radonic et al conclude bestkeeper tool give result slightly deviate correspond obtain genorm able evaluate gene stability stressful condition mrna expression select gene examine gill salmon go smoltification prior seawater entry juvenile anadromous salmon undergo parr smolt transformation characterize behavioral morphological physiological change know challenge fish physiological alteration include increase seawater tolerance olfactory sensitivity metabolic rate scope growth change hemoglobin visual pigment select examine gill smoltification tissue play major role ionic osmotic regulation adaptation hyperosmotic seawater figure show raw Ct value study gene gill smoltification smoltifie seawater desmoltifie freshwater figure datum present normalize index calculate genorm stable gene actin base value genorm rank stability gene fish go smoltification follow order actin rRNA GAPDH figure Figure see gene low individual raw Ct variation individual raw Ct variation studied gene see presmolt smolt group characteristic drop expression see gene smolt group compare presmolt group transfer seawater individual raw Ct variation decrease gene overall raw Ct datum suggest physiological challenge smoltification process affect expression gene datum normalize index stable gene actin relative expression level alter gene ribosomal gene emerge second bad paralog gene stable fact genorm rank co express gene weakness consider evaluate paralog gene likely co regulated gene identify important oncogene show differently express human tissue Hamalainen et al find gene good reference gene real time RT PCR examination similar finding report Frost Nilsen salmon louse show gene valid candidate reference rrna gapdh gene unsuitable current finding base genorm evaluation question recommend application ribosomal gene reference suggest example Ambion reference line early warning use rrna gene reference avoid normalization gene actin index partly base expression gene include index instead mean normalize expression gene calculate new index pattern expression approximately gene see figure suggest gene stability measure find appropriate reference gene correlation absorbance NanoDrop PCR efficiency datum show tend PCR efficiency high sample low ratio sample treat DNase solution ratio usually drop DNase treatment ratio increase sample table evaluation usefulness potential reference gene tissue atlantic salmon rank genorm software good bad individual analyze gene tissue qRT PCR analysis gene gill atlantic salmon go smoltification presmolt smoltification smolt smoltification smoltifie finish smoltifie seawater desmolt desmoltification freshwater figure qrt PCR analysis gene gill atlantic salmon go smoltification presmolt smoltification smolt smoltification smoltifie finish smoltifie seawater desmolt desmoltification freshwater number indicate raw Ct PCR analysis gene gill smoltifye Atlantic salmon figure qrt PCR analysis gene gill smoltifye Atlantic salmon datum figure normalize index good gene actin calculate genorm software group analyze Kruskal Wallis test significant overall value give graph actin significant difference presmolt smoltified group presmolt desmolt group smolt desmolt group significant difference presmolt desmolt group significant difference smolt smoltified group asterisk denote significant difference time ratio drop DNase treatment add substance RNA solution increase absorbance nm decrease nm absorbance add substance salt component inhibit RT reaction PCR reaction call PCR poisoning see ratio low sample inadequate PCR efficiency slope especially RNA head kidney thymus intestine tissue gradient standard curve table reason obtain well amplification rate efficiency diluted sample inhibitor dilute effective level obvious way problem dilute cdna PCR reaction alternatively cleanup column purify concentrate level examine gene coefficient variance CV different tissue vary considerably mrna level tissue regulate numerous endogenous exogenous stimulus transcription rate metabolic active tissue regulated compare active tissue inter tissue variation degradation rate mrnas example affect mrna stability result reveal muscle low cv study gene compare high cv active tissue like thymus head kidney spleen thymus intestine head kidney gill brain liver spleen gene low cv base raw Ct value tissue intestine GAPDH gene high cv thymus elongation factor gene relatively similar expression examine tissue expression likely co express examine tissue favor genorm calculation result demonstrate assay optimize tissue organism necessarily work equally tissue tissue study work intestine head kidney spleen troublesome datum base genorm calculation suggest Atlantic salmon gene test present study good candidate reference gene GAPDH gene unsuitable reference quantitative real time RT PCR regard rRNA gene apply caution tool like genorm applet Microsoft Excel useful help select stable gene experiment tissue individual collect range individual separate base sex size sampling time treat heterogeneous group examine width mrna expression study gene different organ group fish handle feed accord normal aquacultural management individual expose particular treatment examine physiological stress alter gene expression gill total individual collect term presmolt termed smolt smoltification smoltification group keep desmoltifie freshwater term desmolt FW group transfer seawater term smoltifie SW group atlantic salmon examine smoltification anadromous population Vosso river Vosso Southwestern Norway Nilsen et al detail fish treat fish treat euthanize accord norwegian national legislation laboratory animal sample organ gill liver brain head kidney spleen thymus white muscle posterior intestine dissect immediately freeze cryo tube liquid store RNA extraction RNA extract spleen head kidney tissue sample low quality redo sampling individual tissue sample Stability gene gill atlantic salmon smoltifi cation calculate genorm software store RNA later Ambion processing RNA isolate phenol chloroform extraction describe Chomczynski Sacchi store µl RNase free milliq total RNA extract Trizol reagent Invitrogen Life Technologies accord manufacturer instruction Genomic DNA eliminate sample DNase treatment accord manufacturer description Ambion RNA store processing quality RNA assess NanoDrop uv Vis Spectrophotometer NanoDrop Technologies Wilmington DE USA Agilent Bioanalyzer Agilent Technologies Palo Alto USA nm absorbance ratio indicate pure RNA sample RNA Nano LabChip kit Agilent Technologies Palo Alto USA evaluate integrity RNA RNeasy MinElute Cleanup kit Qiagen purify troublesome sample kit ratio increase average PCR primer TaqMan MGB probe sequence quantification gene encode rrna ribosomal protein actin GAPDH show Table gene actin reference real time RT PCR analysis atlantic salmon recent study primer amplify PCR product basepair bp long range bp suggest Applied Biosystems Taq Man assays qpcr assay design Primer Express software Applied Biosystems Foster City USA select appropriate primer probe sequence know atlantic salmon gene mrna sequence encode ribosomal protein GAPDH obtain GenBank accession number respectively exon exon border consider assay base cdna clone report GenBank previously assay base EST alignment zebrafish indicate exon exon border choose primer subsequently confirm salmon gene contain intron site deduce alignment zebrafish PCR product contain intron clone topo vector Invitrogen sequence sequence provide request PCR primer actin base atlantic salmon design span exon exon border gene deduce corresponding gene human zebrafish rRNA PCR primer probe design Atlantic salmon sequence place conserve region gene base comparison human gene RNA sample subject DNase treatment avoid genomic dna contamination amplify PCR product actual cdnas sequence ensure correct mrna sequence quantify fragment sequence BigDye version fluorescent chemistry Applied Biosystems run ABI prism dna apparatus University Bergen Sequencing Facility step real time RT PCR protocol develop measure mrna level study gene tissue atlantic salmon evaluation potential reference gene raw Ct value present genorm VBA applet Microsoft Excel determine stable gene set test gene Ct value transform quantity standard curve accord genorm manual gene expression stability calculate genorm applet gene rank well bad base value GraphPad Prism software GraphPad Software statistical analysis work linear regression determine PCR efficiency base dilution curve non parametric Kruskal Wallis test compare difference group salmon go smoltification PAO responsible experiment data analysis draft manuscript KKL conduct real time RT PCR analysis contribute experimental process AEOJ construct qpcr assay gene TON provide cdna smoltification experiment IH participate supervisor study design analysis writing','2005-11-17 00:00:00','http://dx.doi.org/10.1186/1471-2199-6-21',1774,640),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json','Transmission patterns of smallpox: systematic review of natural outbreaks in Europe and North America since World War II','BACKGROUND: Because smallpox (variola major) may be used as a biological weapon, we reviewed outbreaks in post-World War II Europe and North America in order to understand smallpox transmission patterns. METHODS: A systematic review was used to identify papers from the National Library of Medicine, Embase, Biosis, Cochrane Library, Defense Technical Information Center, WorldCat, and reference lists of included publications. Two authors reviewed selected papers for smallpox outbreaks. RESULTS: 51 relevant outbreaks were identified from 1,389 publications. The median for the effective first generation reproduction rate (initial R) was 2 (range 0–38). The majority outbreaks were small (less than 5 cases) and contained within one generation. Outbreaks with few hospitalized patients had low initial R values (median of 1) and were prolonged if not initially recognized (median of 3 generations); outbreaks with mostly hospitalized patients had higher initial R values (median 12) and were shorter (median of 3 generations). Index cases with an atypical presentation of smallpox were less likely to have been diagnosed with smallpox; outbreaks in which the index case was not correctly diagnosed were larger (median of 27.5 cases) and longer (median of 3 generations) compared to outbreaks in which the index case was correctly diagnosed (median of 3 cases and 1 generation). CONCLUSION: Patterns of spread during Smallpox outbreaks varied with circumstances, but early detection and implementation of control measures is a most important influence on the magnitude of outbreaks. The majority of outbreaks studied in Europe and North America were controlled within a few generations if detected early.',' anthrax attack follow event September focus attention threat terrorism biological agent include variola major causative agent smallpox concern attack smallpox result death susceptibility population number issue virus release number people expose need consider order estimate impact bioterrorist attack smallpox key factor determine size outbreak result attack transmissibility smallpox difficult estimate measure transmissibility effective reproductive rate equal expect number infection produce infectious host real world set determine straightforward manner individual epidemic relate circumstance population vaccination status individual epidemic occur consider intentional attack great interest lie initial spread occur outbreak recognize reason focus transmission index case generation secondary case term initial datum study obtain systematic literature review report smallpox outbreak limit review report smallpox outbreak occur Europe North America society similar modern United States term demographic social physical structure pre world War II western develop society smallpox outbreak report emanate population immunity relatively similar contemporary population expect priori early post war european north american population generally vaccinate childhood vaccination rate generally low adult remote vaccination immunity naturally acquire infection similarly low population outbreak rare terrorist use smallpox st century substantially different epidemiological characteristic discussion outbreak analyze paper close actual population experience help partially guide modern smallpox control effort following database search literature smallpox National Library Medicine keyword smallpox variola Embase Biosis keyword smallpox variola virus human vitro Cochrane Controlled Clinical Trials Library keyword smallpox variola Defense Technical Information Center keyword smallpox search later update PubMed search keyword smallpox variola duplicate title remove secondary search include specific search outbreak specific country site review reference list publication identify search figure language restriction author SB VB independently scan title relevance publication choose reviewer review author VB scan select article identify smallpox outbreak Europe North America abstractor RB research associate VB independently extract datum identify outbreak discrepancy reviewer resolve consensus non english publication abstract author RB research assistant author vb direction include analysis following available date occurrence number index case total case case generation generation following outbreak detail extract available number case acquire hospital household distance case acquire household hospital contact number miss case case diagnose smallpox time illness death generation outbreak identify clinical presentation index case index case classify having typical smallpox atypical smallpox milder illness see people prior smallpox vaccination rash characteristic smallpox hemorrhagic smallpox usually fatal illness characterize internal hemorrhage petechiae follow datum efficacy control measure extract available use ring vaccination vaccination people close contact patient smallpox case isolation separation suspect case smallpox general public quarantine restriction movement region expose smallpox mass vaccination vaccination entire region prophylactic vaccinia immune globulin administration number outbreak period study small total outbreak circumstance vary widely descriptive qualitative analysis emphasize analysis initial meas ured divide total number generation case number index zero th generation case addition outbreak sufficient detail classify set hospital centre outbreak setting define accord proportion secondary case non index case acquire hospital examination distribution case location outbreak hospital outbreak define secondary case hospital acquire mixed outbreak secondary case hospital acquire community outbreak secondary case hospitalacquire base result primary search title scan select despite initial de duplication duplicate figure secondary search identify additional publication result publication obtain journal citation select publica tion government report select publication unobtainable european north american outbreak identify thirty include single World Health Organization publication prior work Mack outbreak document multiple publication Tables outbreak exclude source outbreak confirm outbreak result importation smallpox rug Union Soviet Socialist Republics second outbreak occur unique circumstance occur aboard ship Poland result test aerosolize smallpox fourth outbreak exclude number case generation document initial reproductive rate derive datum United States remain original outbreak occur thirty year period different country different circumstance majority report outbreak occur Tables result involved index case import endemic country outbreak result laboratory exposure publish list single outbreak outbreak include multiple epidemiologically distinct component occur case travel different region result secondary outbreak new area reference column footnote table component outbreak treat separate outbreak effective total outbreak analyze outbreak available information include total number generation case number case generation duration outbreak number death Tables thirty outbreak categorize set detailed information generation outbreak identify control measure clinical presentation index case number miss case table median initial reproduction rate outbreak range table figure half initial third initial median duration measure number generation range table figure extend index generation nearly quarter last few generation median outbreak size measure total number case range case table Figure half involve few case third involve few case median number death range table figure fifth involve death quarter involve few death outbreak great initial value tend large long example outbreak initial versus median value total number case generation versus versus respectively temporal trend detail report thirty outbreak report detail typically large median last generation median total case median death median compare lessdetaile outbreak table figure detailed outbreak index case atypical mild hemorrhagic disease clinical presentation smallpox common index case miss diagnose smallpox time illness majority index case infect median initial detailed outbreak range table figure initial third initial outbreak remain community low initial median hospital base generation median table Figure mixed outbreak generally start community later spread hospital contact intermediate initial median table Figure initial small index typical versus atypical hemorrhagic presentation median respectively index case identify smallpox versus unrecognized median table figure effect pronounce hospital outbreak initial identify versus unidentified index case hospital outbreak versus value community mixed outbreak versus versus respectively median duration outbreak generation range generation table figure extend generation Table analyze Outbreak Marseille France break component entry Cardiff component Outbreak Poland break component entry Kosovo Yugoslavia break component entry footnote table nature index case clear text limited information total number case list consistent text second reference total number case list list World Health Organization Table proportion hospital acquire case list nurse infect result household exposure setting exclude hospital contact total number case list list World Health Organization Table last second generation outbreak remain community last median generation hospital base last median generation mix last median generation table outbreak tend short index case typical compare atypical hemorrhagic median number generation respectively index case identify median number generation table duration hospital outbreak affect identification index case duration mixed community outbreak last generation respectively index case unidentified median outbreak size measure total number case range case table Figure approximately equal proportion small total number case large total outbreak table outbreak remain community involve median case hospital base involve median case mix median case table outbreak tend small index case typical versus atypical hemorrhagic median number case index case identify median number case table outbreak size similarly small setting index case identify similarly large setting median number death range table figure fifth involve death fifth involve few death table community hospital base mixed outbreak median death respectively Table case fatality rate number death small index case identify median equal table case fatality rate high index case present hemorrhagic type smallpox strong association typical presentation early identification index case index case identify smallpox outbreak presentation typical atypical hemorrhagic relationship aid understanding outbreak severity example outbreak great case index case identify large outbreak index case atypical hemorrhagic index case identify outbreak case regardless index case typical atypical hemorrhagic description smallpox transmission important biosecurity planning especially planning effort posit outbreak result number case thousand large relative clinical experience shortly eradication natural disease systematic review design provide understanding smallpox transmission pattern identify analyze effectively smallpox outbreak Europe North America outbreak find median total case death characteristic outbreak vary greatly outbreak initial involve people cause death generation spread outbreak initial involve case last generation outbreak small contain generation percent involve case percent cause death percent initial generation outbreak category setting identification index case clinical characteristic provide additional insight review original publication able categorize analyze outbreak set example able review original publication outbreak document identify thirteen additional outbreak outbreak epidemiologically distinct component result effective outbreak analysis hospital outbreak find mention additional outbreak hospital outbreak high initial reproductive rate community mixed outbreak case death community type interaction identification set studied outbreak reveal pattern similar see frequently unrecognize infectious disease tuberculosis recently SARS index case identify control establish early median initial number generation similarly low setting index case unidentified large increase median initial see hospital setting large increase number generation see community mixed setting consistent observation close quarter institution transmission smallpox relatively effective identification containment measure effective identification impression reinforce examination shipboard break exclude analysis outbreak initial respectively recognition follow control generation community outbreak transmission intrinsically effective consequence miss initial identification community elevated initial increase length outbreak long outbreak decrease effectiveness containment measure initial spread community possibly great difficulty track manage case setting nonetheless outbreak report detail bring control generally case isolation ring vaccination measure rarely mass vaccination New York City outbreak mass vaccination quarantine Kosovo outbreak limited information efficacy additional measure note new treatment modality Cidofovir show protective animal study vaccinia orthopox virus monkeypox cowpox exposure begin atypical hemorrhagic versus typical presentation index case case high initial transmission rate somewhat long relate ease initial diagnosis non typical case mis identify outbreak index case mis identify generally large deadly long median initial compare case identify speculate large difference initial index case identify parameter effectively initial reproductive rate early institution control measure clear type outbreak see develop country second half century quickly control despite search database broad search scan nearly title review paper language requirement include epidemic involve post world War II western population limit identify separate outbreak divide analysis majority outbreak country suggest outbreak certain country likely document outbreak report detail large briefly report probable small outbreak routinely publish detail outbreak relatively common smallpox endemic part world apparent publication bias suggest unpublished small outbreak analysis overestimate transmission potential smallpox base publish report recent literature contain estimate reproductive rate smallpox difficult directly compare finding estimate initial initial effective reproduction rate smallpox study study use contemporary western outbreak estimate basic reproductive rate theoretical parameter define expect number new infected host infectious host produce large randomly mixed population susceptible individual effective reproduction rate specifically Gani Leach epidemic modelling datum Century european smallpox outbreak estimate Eichner Dietz obtain datum outbreak Nigeria Meltzer et al examine outbreak country world estimate estimate overall initial comparable Meltzer et al use single point estimate initial policy analysis ignore main lesson review work clearly show pattern speed duration outbreak spread vary widely accord specific circumstance surround outbreak different smallpox release scenario expect yield different result parameter modify natural history outbreak vary scenario example previous work model outcome terrorist attack different scenario scenario include community base outbreak result infected case ride busy transit system intentional release aerosolized smallpox airport terminal mixed outbreak result release aerosolized smallpox large office building adjustment poorly vaccinated population healthcare workforce understanding smallpox transmission pattern hospital mixed community setting provide important insight smallpox spread different attack scenario contemporary population expect transmission introduce smallpox high find typical population density high mobility great similar think immunity contemporary population likely somewhat low population study healthcare workforce poorly vaccinate inexperienced smallpox decrease human resource available care containment increase likelihood initial case miss counterbalance fact index case vaccinate recently likelihood easily recognize typical case high outbreak scenario great concern involve intentional release outbreak large obvious review involve strain select engineer virulent contagious finally outbreak Kosovo deserve notice region develop wealthy country study outbreak large identify review combine hospital mixed component hospital mixed component outbreak second high high initial reproductive rate hospital component last generation median number generation hospital outbreak suggest smallpox outbreak develop country limited resource healthcare disease surveillance case isolation potentially devastating bioterrorist attack Western industrialize country clear post world War II outbreak western country small low transmission rate uniformly case index case present typical smallpox recognize early exception common heterogeneous particularly usually atypical hemorrhagic index case initially recognize initially unrecognized community outbreak last generation despite low initial value suggest control measure effective setting difficulty trace vaccinate case contact contrast unrecognized hospital outbreak control quickly despite high initial value suggest control measure work closed contain setting early detection particularly patient present typical smallpox couple early implementation control measure decrease duration size outbreak setting publish Bio Med Central scientist read work free charge','2006-05-05 00:00:00','http://dx.doi.org/10.1186/1471-2458-6-126',1990,592),('12de3563e18c3c6c5943fa8216e0eea970a590cd.json','Pandemic influenza preparedness: an ethical framework to guide decision-making','BACKGROUND: Planning for the next pandemic influenza outbreak is underway in hospitals across the world. The global SARS experience has taught us that ethical frameworks to guide decision-making may help to reduce collateral damage and increase trust and solidarity within and between health care organisations. Good pandemic planning requires reflection on values because science alone cannot tell us how to prepare for a public health crisis. DISCUSSION: In this paper, we present an ethical framework for pandemic influenza planning. The ethical framework was developed with expertise from clinical, organisational and public health ethics and validated through a stakeholder engagement process. The ethical framework includes both substantive and procedural elements for ethical pandemic influenza planning. The incorporation of ethics into pandemic planning can be helped by senior hospital administrators sponsoring its use, by having stakeholders vet the framework, and by designing or identifying decision review processes. We discuss the merits and limits of an applied ethical framework for hospital decision-making, as well as the robustness of the framework. SUMMARY: The need for reflection on the ethical issues raised by the spectre of a pandemic influenza outbreak is great. Our efforts to address the normative aspects of pandemic planning in hospitals have generated interest from other hospitals and from the governmental sector. The framework will require re-evaluation and refinement and we hope that this paper will generate feedback on how to make it even more robust.',' paper present ethical framework pandemic influenza planning ethical framework develop expertise clinical organisational public health ethic validate stakeholder engagement process ethical framework include substantive procedural element ethical pandemic influenza planning incorporation ethic pandemic planning help senior hospital administrator sponsor use have stakeholder vet framework design identify decision review process discuss merit limit apply ethical framework hospital decision making robustness framework need reflection ethical issue raise spectre pandemic influenza outbreak great effort address normative aspect pandemic planning hospital generate interest hospital governmental sector framework require evaluation refinement hope paper generate feedback robust world prepare emergence pandemic strain influenza trans national national local organisation agency design plan manage community outbreak addition medical commu nity identify scientific research priority need relate anticipate pandemic need examine ethical issue arise plan public health crisis magnitude limited supply antiviral health care worker duty bind care ill pandemic compete familial obligation prioritize scarce ventilated hospital bed hospital cancel elective surgery restrict hospital visitation date bioethic community slow respond public health issue general pandemic influenza planning particular paper discuss need ethic pandemic influenza planning discuss ethical framework develop guide pandemic planning hospital article find depth analysis ethic pandemic planning Kotalik offer ethical analysis pandemic plan country argument primarily ethic pandemic planning effort oppose ethic pandemic planning example argue persuasively problematic country plan accept particular condition resource scarcity plan assumption Kotalik raise important issue ethic pandemic planning article ethical framework focus specifically provide guidance decision maker ethical issue pandemic planning include provide guidance design ethical process decision making provide guide ethical value consideration substantive issue framework propose example practical ethic attempt provide decision maker introduction articulation generally accept ethical principle value significance ethical framework unique collaborative approach take development involve ethicist different area expertise variety health care stakeholder fill important need pandemic planning ethical framework guide decision making unmet pandemic planning process world wide characteristic public health crisis health need overwhelm available human material resource difficult decision resource allocate medical science provide valuable information help decision science insufficient consider resource allocation decision kind decision decision maker face prepare get influenza pandemic scholar begin point pandemic planning need ethical consideration seriously allow urgency logistical scientific need sideline discussion ethical consideration argue discourse health care scientific moral dimension pandemic influenza plan presuppose certain ethical value principle norm interest preference important presupposition explicit SARS experience Toronto teach health care organisation cost address ethical concern severe loss public trust low hospital staff morale confusion role responsibility stigmatization vulnerable community misinformation key insight SARS overlook peril time crisis guidance incomplete consequence uncertain information constantly change hour hour decision involve life death fairness important emphasis add shall argue fairness consideration procedurally substantively important need fair decision make process equitable distribution scarce human material resource example triage ventilated bed ICU theory decision maker rely scientific evidence determine good maximise benefit allocation ventilated bed science tell initial decision maximise benefit notion maximise benefit derive reflection value ethical analysis require determine utilitarian approach triage maximisation benefit preferable assignment ventilated bed different basis example great need utilitarian maximisation benefit think ethically sound implement system base criterion ethically straightforward require ethical reflection count good stewardship moral obligation demonstrate transparency accountability fairness trustworthiness allocation scarce resource importance ethic pandemic planning application value judgement science especially embed planning assumption practice medicine example ethic little contribute understand mechanism influenza virus transmission significant contribution debate level harm public prepared accept burden negative outcome distribute population resource invest stockpile antiviral medication use ethical framework guide decision making help mitigate unintended unavoidable collateral damage influenza pandemic Kotalik argue incorporation ethic pan demic plan help instrument build mutual trust solidarity time likely present major challenge society ethical framework help guide decision offer great assurance value instantiate accountability transparency trust carefully think decision making review decision stakeholder key lesson Toronto SARS experience health care institution staff benefit development ethical framework decision making intention section systematically derivate normative theory value principle framework paper narrow focus example apply practical ethic attempt introduce articulate value commonly accept intention comprehensively defend value framework area scholarship draw articulate relevance pandemic planning demonstrate discursive legitimacy process stakeholder engagement vetting knowledge pandemic planning process attempt develop ethical framework guide pandemic influenza planning assess ethical framework robustness resonance community intended user significance procedural element development framework minimize insight glean implement framework health care organisation governmental setting build key lesson SARS emergency ethic literature draw expertise clinical organisational public health ethic identify key ethical process value relevant health care organisation value present vet variety health care stakeholder framework product iterative inclusive process Ontario need guidance ethical issue pertain influenza pandemic widely acknowledge word work ethical framework Sunnybrook Women College Health Science Centre know invite join hospital pandemic planning effort broad sectoral interest ethic invite join Ontario Ministry Health Long Term Care MOHLTC effort design pandemic plan working group form response pandemic planning initiative take place early hospital Clinical Ethics Centre invite provide ethic support planning initiative soon apparent scope issue go purview clinical ethic include organisational public health ethic expertise organisational public health ethic quickly procure University Toronto Joint Centre bioethic partnership University sixteen affiliated healthcare organization include partner subsequently de amalgamate Sunnybrook Health Sciences Centre Women College Hospital ethical framework currently implement Sunnybrook framework take shape invite join MOHLTC planning effort begin work Vaccine Antiviral working group MOHLTC adapt work meet related distinct challenge face government work MOHLTC begin Vaccine Antiviral working group ethical framework develop MOHLTC eventually include Ontario Health Pandemic Influenza Plan annex section vaccine antiviral originally anticipate ethical framework plan expertise clinical ethic important development framework knowledge skill experience clinical ethicist need address dilemma challenge find daily clinical arena obvious challenge integrate expertise public health ethic framework design guide decision making clinical health care setting related challenge thoughtfully integrate generally accept principle value clinical ethic public health ethic order meet challenge author turn respective ethic literature SARS experience Toronto hospital health care provider review SARS literature public health ethic generally guide integration public health clinical ethic perspective Toronto experience SARS demonstrate organisation face unique ethical challenge deal public health crisis ethic literature identify need great forethought organisation foster ethical decision making time crisis reason legitimacy framework enhance include insight analysis recent public health crisis like surprisingly literature clinical ethic little disaster preparedness decision thing triage extraordinary circumstance ethic literature bioterrorism battle field triage inform thinking call attention important issue duty care reciprocity equity good stewardship importance have ethically robust criterion policy develop advance pandemic influenza outbreak underscore literature critical decision like individual case case basis physician place position individually decide deny treatment patient guidance policy protocol robust disaster preparedness require practise preventive ethic ethical framework vet Pandemic Planning Committee Joint Centre Bioethics Clinical Ethics Group comprise affiliated health care organization clinical ethicist MOHLTC Vaccine Antiviral Working Group MOHLTC pandemic planning committee process refine framework grateful group valuable insight ethical framework intend inform decisionmake replace intend encourage reflection important value discussion review ethical concern arise public health crisis intend means improve accountability decisionmake require revision feedback circumstance require framework divide distinct part begin premise planning decision pandemic influenza outbreak ought guide ethical decision make process inform ethical value ethical process help improve accountability hope extent possible ethical process produce ethical outcome substantive ethical quality decision enhance recognise ethical process guarantee ethical outcome identify key ethical value guide decision making address substantive ethical dimension decision making context plan pandemic influenza crisis difficult decision fraught ethical challenge framework ethical proc esse base accountability reasonableness model develop Daniels Sabin adapt Gibson Martin Singer model provide useful mean identify key element ethical decision make process extensive literature develop Daniels Sabin accountability reasonableness framework Daniels Sabin framework broad applicability institutional setting priority set situation Daniels Sabin framework apply deliberative theory democratic justice specific problem health care priority setting unique regard feel promote kind deliberative approach pandemic planning ethical framework intend support table outline characteristic ethical decision make process stakeholder able accept difficult decision pandemic influenza crisis decisionmake process perceive ethical legitimacy second framework identify key ethical value inform pandemic influenza planning process decision making outbreak value intend provide guidance important consider value relevant situation hallmark challenging ethical decision tension clear answer privilege make decision value tension importance have ethical decision make process reinforce value identify ethical framework base initially previous research finding ethic SARS University Toronto Joint Centre Bioethics JCB work fund Canadian Institutes Health Research grant lead key publication ethical dimension SARS particular singer et al seminal British Medical Journal article begin identify key ethical value relevance SARS epidemic Toronto value articulate working group adapt pandemic influenza planning context discursive process stakeholder consultation public health specialist ministry official pandemic influenza committee Clinical Ethics Group JCB augment value include new value stewardship trust refine definition value light anticipate demand pandemic influenza crisis compare hospital base epidemic SARS substantive value identify articulate framework intend exhaustive set underdetermine good achieve overall goal pandemic planning generally include minimization morbidity mortality societal disruption procedural engagement overall goal pandemic response benefit ethical framework guide shape debate description value guide decision making find Table framework hot button ethical issue identify work Toronto hospital MOHLTC issue follow hot button issue intend exhaustive serve illustrate value ethical framework identify key ethical aspect decision making let issue target prioritize population vaccine antiviral illustrate value ethical framework help guide decision making value solidarity protect public harm require priority set maximize capacity help society ensure ill care pandemic furthermore proportionality require decision maker consider community vulnerable contagion likely benefit immunization inform public conversant value ethical framework aware expertise inform ranking priority immunisation consistent value trust principle transparency lastly know use framework inform decision making vital ensure framework useful identify necessary exhaustive element successful integration ethic hospital pandemic planning process element sponsorship ethical framework senior hospital administration vetting framework key stakeholder decision review process mechanism place ensure ethical decision making sustain crisis decision explicitly stakeholder view mind opportunity stakeholder engage decision make process example decision make relate staff deployment include input affected staff decision publicly defensible mean process decision open scrutiny basis decision publicly accessible affected stakeholder example communication plan develop advance ensure information effectively disseminate affect stakeholder stakeholder know need information decision base reason evidence principle value stakeholder agree relevant meet health need pandemic influenza crisis people credible accountable example decision maker provide rationale prioritise particular group antiviral medication limit access elective surgery service opportunity revisit revise decision new information emerge crisis mechanism address dispute complaint example elective surgery cancel postpone formal mechanism stakeholder voice concern decision duty provide Care duty provide care respond suffering inherent health care professional code ethic influenza pandemic demand health care provider institution work overwhelm resource health care provider weigh demand professional role compete obligation health family friend health care worker face significant challenge relate resource allocation scope practice professional liability workplace condition decision maker work collaboratively stakeholder professional college advance influenza pandemic establish practice guideline work collaboratively develop fair accountable process resolve dispute provide support ease moral burden duty care Develop mean institution handle appeal complaint especially regard work exemption vaccination prophylaxis staff Health care worker increase risk care patient influenza weigh familial obligation obligation self professional duty care addition comply vaccination antiviral regimen prophylaxis conflict individual liberty principle equity hold thing equal patient equal claim receive need health care influenza pandemic tough decision need health service maintain defer extraordinary circumstance measure take contain spread deadly disease inevitably cause considerable collateral damage influenza pandemic extend cessation elective surgery limit provision emergent necessary service allocate scarce resource value equity guide develop fair criterion allocation consideration give compensation meet inclusion criterion entitle receive care individual liberty value enshrine health care practice principle respect autonomy usual circumstance health care provider balance respect individual autonomy duty protect individual patient harm public health crisis restriction individual liberty necessary protect public harm patient staff member public affect proportional risk public harm necessary relevant protect public good employ restrictive mean necessary achieve public health goal apply discrimination Social distancing strategy employ visitor restriction hospital necessary protection public proportionate threat allay individual right privacy health care public health crisis necessary override right protect public harm proportionate response need private information require release intrusive mean protect public disclose private information relevant achieve legitimate necessary public health goal release private information intrusive mean protect public health determine good intend significant justify potential harm come suspend privacy right harm stigmatization individual particular community provide public education correct misconception disease transmission offset misattribution blame particular communitiesThe need conduct contact tracing possibly infect people require particular group individual identify publicly need weigh potential harm expose community individual stigmatization proportionality require restriction individual liberty measure take protect public harm exceed necessary address actual level risk critical need community decision maker use restrictive coercive measure limit restrict liberty entitlement use coercive measure circumstance restrictive measure fail achieve appropriate public health end decision close emergency room consider potential harm keep emergency room open significant warrant closure foundational principle public health ethic obligation protect public harm principle require citizen comply impose restriction order ensure public wellbeing safety protect public harm hospital require restrict public access service area restrict visit hour limit availability service elective surgery impose infectious control practice mask Weigh medical moral imperative compliance Ensure stakeholder aware medical moral reason public health measure Ensure stakeholder aware benefit compliance consequence non compliance establish mechanism review decision public health situation change address stakeholder concern complaint make decision quarantine individual protection public harm weigh individual liberty note ethical value individual liberty tension protection public harm individual interest minimize harm reciprocity require society support face disproportionate burden protect public good take step minimise impact far possible influenza pandemic measure protect public good likely impose disproportionate burden health care worker patient family health care worker face expand duty increase workplace risk physical emotional stress isolation peer family case infection lead hospitalization death similarly quarantine individual family ill patient experience significant social economic emotional ease burden health care worker patient patient family hospital coordination health care organization ensure safety worker especially redeploy staff area usual scope practiceThe provision antiviral medication vaccination hospital staff prophylaxis way hospital ensure safety worker expose great usual risk discharge duty care SARS heighten global awareness interdependence health system need solidarity systemic institutional boundary stem contagious disease influenza pandemic require global solidarity require vision solidarity health care institution Solidarity Good open honest communication open collaboration spirit common purpose health care institution share public health information coordinate health care delivery transfer patient deployment human material resource Territoriality hospital department health care institution need overcome good communication sense common purpose order provide equitable care jurisdictionsIn society institution individual entrust governance scarce resource vaccine antiviral ventilator hospital bed health care worker pandemic influenza outbreak difficult decision allocate material human resource collateral damage result allocation decision entrust governance role guide notion stewardship inherent stewardship notion trust ethical behaviour good decision making decision maker responsibility avoid reduce collateral damage result resource allocation decision Maximize benefit allocate resource protect develop resource possible consider good outcome benefit public good equity fair distribution benefit hospital decision stock pile antiviral medication consider effective way protect staff infection money stockpiling come money well use trust essential component relationship clinician patient staff organization public health care provider organization health system public health crisis stakeholder perceive public health measure betrayal trust access need care deny abandonment time great need decision maker confront challenge maintain stakeholder trust time stem influenza pandemic control measure take time build step build trust stakeholder crisis hit swing ensure decision making process ethical transparent affect stakeholder early engagement stakeholder distance justify stakeholder confidence decision maker trustworthiness value trust respect promote follow ethical process outline ethical framework inform decision making health care institution depend large extent people senior position organisation see relevance decision make process dependant robust framework require willingness frame pandemic planning issue normative nature argue value framework stringent impractical implement crisis condition especially find Ethical Processes framework Table certainly crisis condition place constraint extent principle act effort action full extent possible circumstance experience possible support senior administrator senior administration Pandemic Planning Committee previous experience accountability reasonableness framework decision making pandemic influenza planning committee familiar Ethical Processes framework receptive idea guide ethical framework senior administrator receptive ethical framework learn SARS organisation decision make process honour value ethical process SARS deal legacy collateral damage staff patient form distrust low morale reason senior administrator play important role vet ethical framework ensure institutional sponsor favour adopt ethical framework important gain widespread support ethical framework decision making ensure ethical framework look good remain unused order obtain support buy ethical framework important key stakeholder institution vet framework require careful consideration key stakeholder institution include responsibility decision making affect decision take vetting process intend create buy decrease likelihood interest issue morally relevant pandemic planning neglect overlook enhance moral legitimacy value framework addition process stakeholder vetting increase likelihood value instantiate framework resonate stakeholder community experience value framework resonate pandemic planner share ethical framework primarily pragmatic justification selection value framework mean framework provisional ought subject revision light compelling argument empirical evidence stakeholder feedback important note iterative inclusive process value framework deliberate stakeholder group lend form discursive ethical legitimacy help justify inclusion ethical framework intend framework invite dialogue legitimacy adequacy return issue final section paper ideally vetting process include people represent interest patient family volunteer hospital constituency patient relation human resource occupational health representative provide guidance feedback development framework direct input patient family representative obtain limitation framework vet important stakeholder importance solidarity management public health crisis suggest public health care organisation consider stakeholder hospital pandemic planning pragmatic hospital undertake broad public consultation vetting process pandemic plan general ethical framework particular solidarity equity suggest broad stakeholder interest relevant pandemic planning consequently opportunity broad ethical dialogue pandemic planning need encourage order ensure support key stakeholder maintain outbreak need effective communication mechanism place important aspect responsive decision make process ensure formal opportunity revisit revise decision new information emerge ethical framework formulate template decision review process adapt Gibson JL formal decision review process template unpublished aid organisation identify exist establish new mechanism formal review decision believe decision review mechanism essential ethical decision making public health crisis way value ethical framework action formal mechanism review decision need order capture feedback stakeholder key decision resolve dispute challenge process important ensure decision good possible circumstance give change information engage stakeholder constructively difficult decision give unpredictable nature public health emergency difficulty pose charge planning decision making reasonable assume decision revise pandemic influenza crisis dispute challenge arise restriction requirement impose staff patient family pandemic influenza outbreak decision review process essential argue stringent measure time crisis argue review decision take place regardless likely ad hoc manner formalize process increase fairness moral legitimacy exist mechanism handle kind review important distinguish different type ethical analysis order explain approach take development ethical framework discuss Callahan Jennings draw useful distinction apply ethic critical ethic ethical framework example apply ethic framework identifie rely general principle apply real world example professional conduct decision making design professional guidance client general public standard use assess professional conduct certainly need critical ethical analysis pay attention problem result institutional arrangement prevail structure cultural attitude social power expect ethical framework design guide clinical decision making explicitly address kind issue ethical framework address kind issue critical ethical analysis address example framework promote value process seek redress power disparitie institution section framework deal ethical process particular challenge institutional decision typically example value inclusiveness process principle essential redress power difference key stakeholder ethical framework product apply ethical analysis evaluate light strength redress Callahan Jennings characterize critical ethic problem power disparity institution pluralistic society different ethical perspective exist simultaneously issue global public individual health ethical framework guide decision making robust extent reflect value belief decision maker refer value belief affect decision take framework rely heavily Toronto experience SARS surface examine ethical value important public health crisis influenza pandemic likely present particular ethical challenge different SARS predict severity contagion spread community important uncritically adopt framework use basis continued reflection evaluation ensure relevance responsiveness unfold health crisis important consider extent ethical framework reflective community lesson SARS experience China likely surface different ethical value emphasise different aspect framework Callahan Jennings argue submit rich discourse ethic public health advance relate background value general society particular community carry previously maintain issue relate pandemic influenza planning raise critical ethical analysis require broad public debate kind issue require public debate take place societal level ethical pandemic planning require organisation agency foster internal dialogue value instantiate ethical framework imperative value outline framework resonate member organisation community serve procedural aspect framework provide means ensure value community reflect decision making procedural principle inclusiveness responsiveness important recognise value static circumstance evolve rapidly pandemic influenza outbreak ethical framework require evaluation revision challenge continue recognise importance moral reflection circumstance conducive encourage process evaluation strive assess result decision consistent value framework intend promote reason imperative start ethical dialogue advance find way encourage consideration ethical issue stage decision making hope paper way advance objective paper stimulate discussion ethical issue value pervade pandemic planning believe framework unique blending clinical public health organizational ethic strength draw lesson recent public health crisis SARS Toronto extent empirically ground strength product inclusive process development include stakeholder vetting unique attempt provide guidance decision maker face public health crisis hope framework acceptance hospital provincial government Ontario signal change way decision take institution charge make decision life death consequence good pandemic planning require reflection value scientific information drive decision development ethical framework hospital pandemic planning call expertise clinical organisational public health Stakeholder engagement essential ethical framework relevant ethical framework contain procedural substantive ethical value guide decision key element integration ethic pandemic planning sponsorship senior hospital administration vet stakeholder decision review ethical framework robust extent pandemic influenza planning decision see ethically legitimate affect order increase robustness pandemic planning general timely public debate ethical issue essential','2006-12-04 00:00:00','http://dx.doi.org/10.1186/1472-6939-7-12',3587,987),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json','Heterogeneous nuclear ribonucleoprotein A1 regulates RNA synthesis of a cytoplasmic virus','Heterogeneous nuclear ribonucleoprotein (hnRNP A1) is involved in pre-mRNA splicing in the nucleus and translational regulation in the cytoplasm. In the present study, we demonstrate that hnRNP A1 also participates in the transcription and replication of a cytoplasmic RNA virus, mouse hepatitis virus (MHV). Overexpression of hnRNP A1 accelerated the kinetics of viral RNA synthesis, whereas the expression in the cytoplasm of a dominant-negative hnRNP A1 mutant that lacks the nuclear transport domain significantly delayed it. The hnRNP A1 mutant caused a global inhibition of viral mRNA transcription and genomic replication, and also a preferential inhibition of the replication of defective-interfering RNAs. Similar to the wild-type hnRNP A1, the hnRNP A1 mutant complexed with an MHV polymerase gene product, the nucleocapsid protein and the viral RNA. However, in contrast to the wild-type hnRNP A1, the mutant protein failed to bind a 250 kDa cellular protein, suggesting that the recruitment of cellular proteins by hnRNP A1 is important for MHV RNA synthesis. Our findings establish the importance of cellular factors in viral RNA-dependent RNA synthesis.',' introduction hnRNP RNA bind protein contain RNA bind domain RBDs glycine rich domain responsible interaction involve pre mrna splicing transport cellular RNAs review Dreyfuss et al predominantly locate nucleus shuttle nucleus cytoplasm Pin Äol Roma Dreyfuss signal mediate shuttle identi ed amino acid sequence term locate near terminus hnRNP amino acid Michael et al Siomi Dreyfuss Weighardt et al yeast hybrid screening bait result discovery novel transportin mediate pathway nuclear import hnRNP Pollard et al Fridell et al Siomi et al function cytoplasmic hnRNP de ned study show cytoplasmic nuclear hnRNP exhibit different RNA bind pro le cytoplasmic hnRNP capable high af nity bind AU rich element modulate mrna turnover translation Hamilton et al Hamilton et al Henics et al show promote ribosome binding mrnas cap mediate mechanism prevent spurious initiation aberrant translation start site Svitkin et al belong Coronaviridae family positivesense single strand RNA virus MHV replication transcription occur exclusively cytoplasm infected cell viral RNA dependent RNA polymerase RdRp review Lai Cavanagh initially gene viral genome translate viral RdRp replicate viral genomic RNAs negative strand RNAs subsequently negative strand rna template transcribe mrnas include genomic sized RNA nest set subgenomic mrna transcript identical non translated leader sequence nucleotide co terminal polyadenylate end subgenomic mrna transcription MHV utilize unique discontinuous mechanism leader sequence derive different molecule fuse RNAs intergenic IG site transcription initiation site generate subgenomic mrnas Jeong Makino Liao Lai Zhang et al exact mechanism mrnas controversial show process discontinuous RNA transcription regulate viral RNA element include cis trans acting leader RNA Zhang et al IG sequence Makino et al untranslated sequence Lin et al considerable biochemical evidence suggest possible direct indirect interaction RNA regulatory element hnRNP bind MHV negative leader IG sequence Furuya Lai Li et al site direct mutagenesis IG sequence demonstrate extent binding hnRNP IG sequence correlate ef ciency transcription IG site Zhang Lai Li et al immunostaine hnRNP show hnRNP relocate cytoplasm MHV infect cell viral RNA synthesis occur Li et al hnRNP mediate formation ribonucleoprotein complex contain MHV leader IG sequence result suggest hnRNP serve protein mediator distant RNA region interact heterogeneous nuclear ribonucleoprotein regulate RNA synthesis cytoplasmic virus EMBO Journal Vol pp European Molecular Biology Organization cellular protein include calreticulin Singh et al polypyrimidine tract bind protein PTB Hellen et al Wu Baer et al La protein Pardigon Strauss McBride et al protein Parsley et al nucleolin Waggoner Sarnow implicate involve viral RNA transcription replication addition MHV hnRNP report interact human cytomegalovirus immediate early gene protein play important role regulation virus replication Wang et al furthermore yeast protein relate human core RNA splicing factor show require ef cient replication brome mosaic virus RNA Diez et al recently reddy colleague demonstrate inhibition HIV replication dominant negative mutant reddy et al cellular protein show experimentally participate directly RNA dependent RNA synthesis order demonstrate involvement hnRNP MHV RNA replication transcription establish DBT cell line stably express wildtype wt hnRNP terminus truncate mutant lack sequence glycine rich domain show mutant hnRNP localize predominantly cytoplasm exhibit dominant negative effect viral genomic RNA replication subgenomic mrna transcription contrast overexpression wt hnRNP accelerate synthesis viral RNAs result provide strong evidence hnRNP directly indirectly involve MHV RNA synthesis cytoplasm terminal protein important function nde reveal novel function hnRNP cytoplasm characterization stable cell line express wt terminus truncate hnRNP explore potential role hnRNP MHV RNA synthesis establish murine DBT cell line stably express Flag tag wt hnRNP DBT mutant hnRNP amino acid deletion terminus DBT figure mutant lack glycine rich domain sequence responsible shuttle hnRNP nucleus cytoplasm immunoblot cell lysate anti flag antibody detect kDa protein DBT cell kda protein independent clone DBT cell figure protein cross reactive anti flag antibody control cell line stably transfecte vector DBT VEC amount Flagtagged wt truncate hnRNP comparable cell line chicken polyclonal antibody hnRNP detect endogenous hnRNP isoform hnRNP relate protein cell lysate cell line band kDa overlap Flag tag wt hnRNP DBT cell slight increase overall hnRNP DBT cell compare DBT VEC cell indicate exogenous hnRNP constitute small fraction total hnRNP cell DBT cell additional band small size kda correspond mutant hnRNP detect overall expression level exogenous hnRNP hnRNP fold low endogenous hnRNP cell lysate figure similar endogenous hnRNP protein Pin Äol Roma Dreyfuss Flag tag wt hnRNP localize exclusively nucleus Figure mutant hnRNP localize predominantly cytoplasm Figure consistent previous nde nuclear localization signal necessary localize hnRNP nucleus Weighardt et al hnRNP abundant endogenous hnRNP cytoplasm expression level wt mutant hnRNP vary individual cell base stain Figure growth rate Figure cell morphology datum show similar different cell line effect overexpression wt mutant hnRNP syncytium formation virus production rst assess effect hnRNP overexpression morphological change induce MHV infection different clone DBT cell line Virus infection perform multiplicity infection detect subtle morphological difference different cell line Syncytia appear post infection DBT VEC cell early DBT cell syncytia signi cantly large spread DBT cell DBT VEC cell Figure similar difference observe additional clone DBT cell datum show contrast syncytium observe different clone DBT cell DBT cell detach plate DBT VEC cell remain plate datum show remarkably sign syncytium formation DBT cell virus infection overall morphology cell similar DBT VEC cell datum show DBT cell eventually kill suggest inhibition viral replication result disruption MHV receptor correspondingly virus production cell line signi cantly different virus production DBT cell fold DBT VEC DBT cell figure DBT cell produce twice virus DBT VEC cell time period relocalization hnRNP MHV infection MHV RNA synthesis occur exclusively cytoplasm infected cell order hnRNP participate directly viral transcription recruit site RNA synthesis hnRNP shuttle nucleus cytoplasm normal cell Pin Äol Roma Dreyfuss level cytoplasmic hnRNP low demonstrate previously hnRNP relocate nucleus cytoplasm MHV infect cell Li et al determine overexpressed hnRNP participate MHV RNA synthesis perform immunostaine experiment anti flag antibody localize flag tag hnRNP DBT cell signi cant increase cytoplasmic level hnRNP corresponding decrease nuclear hnRNP observe virus infect cell syncytia figure cell express MHV nucleocapsid protein cytoplasm Figure comparison uninfected cell protein stain hnRNP predominantly localize nucleus arrow Figure DBT cell cell stain positive MHV protein figure Signi cantly viral protein detect cell stain weakly Flag hnRNP Figure suggest expression high level hnRNP interfere viral replication effect wt mutant hnRNP MHV protein production investigate effect wt mutant hnRNP production MHV structural nonstructural protein cytoplasmic protein extract infected cell line different time point infection immunoblot analysis detect open reading frame ORF product Lu et al protein expression DBT VEC cell clearly detect peak figure DBT cell appear peak DBT cell protein detect similar pattern difference observe protein cell line actin level different cell line remain relatively constant infection DBT cell actin detect loss dead cell figure result clearly demonstrate overexpression wt hnRNP accelerate viral protein production expression mutant hnRNP delay perform staining protein con rm western blot result represent image show Figure DBT cell stain positive protein DBT VEC cell cell find express protein DBT cell protein appear doublet lane Figure result varied experiment experiment protein know phosphorylate Stohlman Lai post translationally modi ed know figure DBT cell show signi cantly high level incorporation peak DBT cell detectable level incorporation radioactivity result suggest hnRNP regulate MHV RNA synthesis assess production genomic subgenomic MHV RNAs cell line northern blot analysis genomic subgenomic RNA specie detect DBT VEC DBT cell signi cantly high steady state level RNA specie DBT cell Figure contrast viral RNA detect DBT cell time point MHV RNA level DBT VEC DBT cell decrease generally loss dead cell small subgenomic rna detectable DBT cell viral RNA specie detectable DBT cell figure lane DBT cell dead lane result con rme synthesis viral RNA specie accelerate overexpression wt hnRNP delay dominant negative mutant hnRNP analysis detect additional RNA specie arrow Figure determined defective interfere DI RNA northern blot analysis probe represent region leader present genomic DI RNAs datum show interestingly DI RNA inhibit great extent RNA specie DBT cell result suggest replication DI RNAs sensitive dominant negative inhibition cytoplasmic hnRNP demonstrate MHV RNA transcription machinery defective cell express mutant hnRNP study transcription MHV DI RNA contain transcription promoter derive IG sequence mRNA chloramphenicol acetyltransferase CAT reporter gene CAT activity express serum free medium replace virus growth medium contain NCS mg ml actinomycin mci ml add infected cell labeling cytoplasmic extract prepare precipitate TCA TCA precipitable count measure scintillation counter northern blot analysis MHV genomic subgenomic RNA synthesis DBT cell cytoplasmic RNA extract MHV infect cell northern blot analysis naturally occur DI RNA MHV indicate arrow DI RNA subgenomic mrna contain CAT sequence produce RNA transfecte MHV infect cell infection CAT activity DBT cell signi cantly high DBT VEC cell figure hand CAT activity low DBT cell CAT activity DBT cell slightly low DBT VEC cell loss dead DBT cell CAT activity DBT signi cantly low DBT VEC DBT cell result establish mrna transcription DI RNA inhibit hnRNP result show Figure suggest DI RNA replication sensitive inhibitory effect hnRNP mutant con rm result study replication DI RNA serial virus passage DBT cell infect MHV transfecte DIssE RNA derive JHM virus Makino Lai virus release passage twice DBT cell generate virus DBT cell infect virus cytoplasmic RNA extract northern blot analysis glyoxalate RNA well resolution small RNAs DBT cell RNA extract viral RNA synthesis delay cell line cell infect virus yield detectable amount DIssE contain naturally occur DI RNA replication inhibit strongly synthesis MHV genomic subgenomic rna DBT cell figure lane lane DI RNA detectable cell infect virus figure lane contrast DIssE appear cell infect virus increase cell infect virus indicate replication small DIssE inhibitory effect replication large DI RNA Jeong Makino similar DI RNA replication DIssE RNA strongly inhibit MHV genomic subgenomic rna DBT cell figure lane result suggest MHV DI RNA replication dependent function cytoplasmic hnRNP mechanism dominant negative inhibition terminal deletion mutant hnRNP understand underlying mechanism inhibition MHV RNA transcription terminal deletion mutant hnRNP rst examine RNA protein bind property mutant protein electrophoretic mobility shift assay demonstrate hnRNP retain ability bind MHV leader RNA form multimer similar wt hnRNP datum show consistent fact rbd intact figure furthermore uv crosslinke experiment show increase amount puri ed glutathione transferase ef ciently compete endogenous hnRNP binding MHV leader RNA Figure indicate binding hnRNP RNA affect result suggest RNA bind property hnRNP intact examine protein bind property hnRNP hnRNP show interact protein participate MHV RNA synthesis Compton et al Wang Zhang rst determine dominantnegative mutant hnRNP retain ability interact protein vitro GST pull assay truncation mutant hnRNP show protein bind terminal domain aa hnRNP figure binding hnRNP equivalent hnRNP protein affect examine vivo interaction wt mutant hnRNP MHV ORF product show co localize de novo synthesize viral RNA virus passage twice wt DBT cell obtain virus cytoplasmic RNA extract DBT cell infect virus treat glyoxal electrophoresis northern blot analysis label mrna probe DI RNA DIssE RNA indicate arrow unpublished result associate viral replicase complex Gibson Bost et al cytoplasmic extract MHV infect cell immunoprecipitate anti flag antibody conjugate bead follow western blotting rabbit polyclonal antibody co precipitate Flag tag hnRNP DBT cell precipitation observe DBT VEC cell figure DBT cell co immunoprecipitation perform abundant MHV protein synthesize show co precipitate hnRNP indicate hnRNP form complex viral polymerase gene product result suggest ability hnRNP interact polymerase protein alter investigate mutant hnRNP de cient interaction cellular protein complex label protein MHV infect cell mock infect cell different time point infection immunoprecipitate anti flag antibody Signi cantly cellular protein kda show associate wt hnRNP mutant hnRNP Figure suggest hnRNP bind protein terminal domain propose cellular protein important component MHV RNA transcription replication complex accumulate body evidence signify importance cellular factor RNA synthesis RNA virus review Lai previous study show hnRNP bind cis act sequence MHV template RNA interaction correlate transcription ef ciency viral RNA vivo Zhang Lai Li et al addition hnRNP implicate viral RNA replication recent nde hnRNP interact positive negative strand MHV RNA unpublished result hnRNP modulate cytoplasmic viral RNA synthesis functional importance hnRNP viral RNA synthesis far directly demonstrate present study establish MHV RNA transcription replication enhance overexpression wt hnRNP protein inhibit expression dominant negative hnRNP mutant DBT cell line result suggest hnRNP host protein involve formation cytoplasmic replication complex viral RNA synthesis represent novel function hnRNP cytoplasm result indicate inhibitory effect MHV replication exhibit dominant negative mutant hnRNP relatively prominent enhancement effect overexpression wt hnRNP consistent subcellular localization pattern wt mutant hnRNP protein overexpressed exogenous wt hnRNP DBT cell predominantly localize nucleus similar endogenous hnRNP Figure terminal deletion mutant localize mainly cytoplasm level hnRNP high endogenous wt hnRNP cytoplasm DBT cell MHV replication occur result explain hnRNP strong dominant negative inhibitory effect despite fact express low level endogenous hnRNP Figure effect expression wt mutant hnRNP virus production figure viral protein synthesis Figure viral RNA synthesis figure correlate furthermore hnRNP cause global inhibition genomic RNA replication subgenomic mrna transcription preferential inhibition DI RNA specie result suggest inhibition MHV replication hnRNP mutant likely direct effect viral RNA synthesis indirect effect aspect cellular viral function hnRNP bind directly cis act MHV RNA sequence critical MHV RNA transcription Li et al replication Huang unpublished result likely hnRNP participate formation transcription replication complex datum hnRNP interact directly indirectly protein gene product probably associate viral transcription replication complex Compton et al Wang Zhang Gibson Bost et al hnRNP participate directly viral RNA synthesis similar role transcription factor DNAdependent RNA synthesis maintain favorable RNA conformation RNA synthesis alternatively hnRNP modulate MHV RNA transcription replication participate processing transport control stability viral RNAs report RNA processing retrovirus human cell leukemia virus type Black et al Black et al alter binding hnRNP viral RNA regulatory element possible hnRNP participate MHV RNA synthesis indirectly affect production host cell protein turn regulate MHV RNA synthesis hnRNP dose dependent altern ative splicing factor Caceres et al small change intracellular level hnRNP alter splicing cellular protein regardless mechanism study establish importance cellular factor viral RNA dependent RNA synthesis transcription RNA strongly inhibit dominant negative mutant hnRNP show CAT assay figure addition replication naturally occur DI RNA arti cial DIssE RNA completely abolish figure surprisingly replication MHV DI RNAs suffer strong inhibition dominantnegative mutant hnRNP synthesis MHV genomic subgenomic rna suggest DI RNA replication dependent hnRNP DI RNAs contain cis act replication signal essential replication normal cell Kim Makino small size DI RNA cause require hnRNP maintain critical RNA structure show different DI RNAs require different cis act signal RNA replication Kim Makino result demonstrate terminal domain hnRNP include sequence glycinerich region important MHV RNA transcription replication mechanism dominant negative effect hnRNP clear hnRNP retain RNA bind self association ability capable bind viral protein associate transcription replication complex possible hnRNP productive inability interact viral cellular protein involve MHV RNA synthesis find protein kDa bind wt mutant hnRNP Figure remain show cellular protein involve MHV RNA synthesis preliminary study find MHV replicate erythroleukemia cell line report lack detectable hnRNP expression result retrovirus integration allele loss allele Ben David et al hnRNP protein involve variety important cellular function include RNA splicing transport turnover translation conceivable redundant gene product substitute function hnRNP cell uv crosslinke assay cell extract detect protein comparable hnRNP size interact MHV negative strand leader RNA datum show protein represent hnRNP relate protein hnrnps exist cell Buvoli et al Burd et al multiple cellular protein capacity involve MHV RNA synthesis base previous nding Kim Makino Zhang Lai Li et al result study propose model regulation transcription replication MHV RNA hnRNP hypothesize hnRNP component MHV RNA transcription replication complex crosstalk hnRNP viral cellular RNA bind protein designate Figure essential MHV replication transcription protein bind terminus hnRNP cooperate hnRNP recruit protein form transcription replication complex terminaldeletion mutant hnRNP lose ability interact protein bring initiation complex result inhibition MHV RNA transcription replication residual replication transcription activity MHV RNA absence functional hnRNP limited af nity protein cis act signal present MHV genomic RNA site hand DI RNAs lack cis act signal crosstalk protein hnRNP abolish dominant negative mutant hnRNP protein long participate formation initiation complex result complete loss DI RNA replication summary datum provide direct experimental evidence hnRNP involve directly indirectly MHV RNA synthesis probably participate formation RNA transcription replication complex nding reveal novel cytoplasmic function hnRNP viruse DBT cell mouse astrocytoma cell line Hirano et al culture Eagle minimal essential medium MEM supplement newborn calf serum NCS tryptone phosphate broth MHV strain robb Bond propagate DBT cell maintain virus growth medium contain construction establishment DBT stable cell line cdna murine hnRNP gene ampli ed RNA extract DBT cell set primer represent hnRNP code region clone Invitrogen Carlsbad amino acid Flag tag attach terminus hnRNP include Flag tag forward PCR primer truncated hnRNP similarly construct PCR ampli ed fragment represent hnRNP aa establishment permanent DBT cell line plasmid contain Flag tag hnRNP hnRNP transfecte DBT cell DOTAP accord manufacturer instruction Boehringer Mannheim Indianapolis transfecte cell select DBT cell medium contain mg ml Geneticin omega Scienti Tarzana day single colony collect culture individually additional day screen expression flag tag protein polyclonal rabbit antibody gift Dr Susan Loyola University IL chicken polyclonal antibody hnRNP produce Aves Labs tigard immunize chicken puri ed mouse hnRNP protein express bacteria polyclonal anti flag antibody purchase Af nity bioreagent Golden CO goat polyclonal antibody actin obtain Santa Cruz Biotechnology Santa Cruz mouse monoclonal antibody protein describe previously fleme et al growth rate permanent DBT cell equal number DBT VEC DBT DBT cell plate cm culture plate maintain culture medium day cell trypsinize stain Trypan Blue Gibco BRL Grand Island NY count interval hemacytometer Hausser Scienti Horsham assay DBT cell cm plate infect MHV virus adsorption cell wash time serum free MEM replace virus growth medium contain serum ml medium take plate plaque assay labeling MHV RNA Cells plate plate infect MHV mg ml actinomycin add virus growth medium inhibit cellular RNA synthesis label newly synthesize MHV RNA mCi ml NEN Boston MA add medium hourly interval labeling cell wash twice ice cold PBS scrape plate ml PBS cell collect centrifugation incubate ml NTE buffer mM NaCl mm Tris pH mM EDTA contain mm dithiothreitol DTT ml RNasin ice min centrifugation ml cytoplasmic extract spot piece mm paper incubate trichloroacetic acid TCA radioactivity remain mm paper measure scintillation counter northern blot analysis DBT cell infect MHV cytoplasmic extract prepare describe subject phenol chloroform extraction ethanol precipitation purify cytoplasmic RNA approximately mg RNA separate electrophoresis formaldehyde contain agarose gel transfer nitrocellulose membrane well resolution DIssE RNA figure RNA glyoxalate electrophorese agarose gel vitro transcribe label negative strand mrna MHV JHM probe detect MHV genomic subgenomic rna detect DI RNA specie RNA blot probe RNA represent sequence complementary sequence region MHV JHM RNA exclude leader sequence western blot analysis DBT cell plate infect MHV cytoplasmic extract prepare describe previously Li et al hnRNP modulate cytoplasmic viral RNA synthesis time point extract electrophorese polyacrylamide gel transfer nitrocellulose membrane western blotting stain Cells wash phosphate buffer saline PBS xe formaldehyde min room temperature follow min acetone primary antibody dilute bovine serum albumin incubate cell room temperature wash conjugate secondary antibody add cell dilution room temperature fitc tritcconjugated secondary antibody generate green red uorescence cell wash PBS mount Vectashield Vector Laboratories Burlingame crosslinke assay uv crosslinke assay perform describe previously Huang Lai brief DBT cell extract mg protein mg ml tRNA vitro transcribe label negativestrand MHV RNA bp incubate min increase amount puri ed GST ng recombinant fusion protein ng include reaction compete endogenous hnRNP bind reaction mixture place ice uv irradiate uv stratalinker Stratagene min follow digestion mg ml RNase min complex separate gel visualize autoradiography GST pull assay GST pull perform describe previously Tu et al brief fusion protein glutathione bead Pierce Rockford IL incubate vitro translate label protein ml GST bind buffer contain bead wash ve time GST bind buffer contain protein bind bead elute boil laemmli buffer min separate polyacrylamide gel labeling immunoprecipitation DBT cell infect MHV cell incubate methionine free medium min labeling label mci ml start label time point cell harvest protein extraction describe previously Li et al protein extract immunoprecipitate anti flag antibody conjugate bead Sigma St Louis MO Tm buffer mm pH KCl mM MgCl mM EDTA glycerol mm DTT mm immunoprecipitate wash separate gradient gel visualize autoradiography Plasmid linearize xbai vitro transcribe RNA polymerase produce DI RNA DI RNA transfecte MHV infect DBT cell DOTAP describe previously Huang Lai DBT cell infect MHV cell transfecte mg vitro transcribe DI RNA incubate desire length time amplify DI RNA virus passage twice wt DBT cell generate virus cell harvest lyse freeze thaw time centrifugation min supernatant CAT assay describe previously Lin et al','2000-09-01 00:00:00','http://dx.doi.org/10.1093/emboj/19.17.4701',3402,832),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json','Towards standardization of RNA quality assessment using user-independent classifiers of microcapillary electrophoresis traces','While it is universally accepted that intact RNA constitutes the best representation of the steady-state of transcription, there is no gold standard to define RNA quality prior to gene expression analysis. In this report, we evaluated the reliability of conventional methods for RNA quality assessment including UV spectroscopy and 28S:18S area ratios, and demonstrated their inconsistency. We then used two new freely available classifiers, the Degradometer and RIN systems, to produce user-independent RNA quality metrics, based on analysis of microcapillary electrophoresis traces. Both provided highly informative and valuable data and the results were found highly correlated, while the RIN system gave more reliable data. The relevance of the RNA quality metrics for assessment of gene expression differences was tested by Q-PCR, revealing a significant decline of the relative expression of genes in RNA samples of disparate quality, while samples of similar, even poor integrity were found highly comparable. We discuss the consequences of these observations to minimize artifactual detection of false positive and negative differential expression due to RNA integrity differences, and propose a scheme for the development of a standard operational procedure, with optional registration of RNA integrity metrics in public repositories of gene expression data.',' purity integrity RNA critical element overall success RNA base analysis include gene expression profiling method assess expression level thousand gene single assay start low quality RNA strongly compromise result downstream application labor intensive time consume highly expensive spite need standardization RNA sample quality control presently real consensus good classification criterion conventional method sensitive specific single strand RNA susceptible interference contaminant present sample instance spectrophotometer ratio absorbance nm great usually consider acceptable indicator RNA purity measurement compromise presence genomic dna lead estimation actual RNA concentration hand measurement estimate presence protein provide hint possible residual organic contaminant consider nm pure RNA equal second check involve electrophoresis analysis routinely perform agarose gel electrophoresis RNA stain ethidium bromide EtBr sensitive SYBR Green dye proportion ribosomal band conventionally view primary indicator RNA integrity ratio consider typical high quality intact RNA method highly sample consume mg total RNA sensitive detect slight RNA degradation today microfluidic capillary electrophoresis Agilent bioanalyzer Agilent Technologies USA widely particularly gene expression profiling platform require small RNA sample low pg use size standard electrophoresis allow estimation size RNA band measurement appear relatively unaffected contaminant integrity correspondence address tel online version article publish open access model user entitle use reproduce disseminate display open access version article non commercial purpose provide original authorship properly fully attribute Journal Oxford University Press attribute original place publication correct citation detail give article subsequently reproduce disseminate entirety derivative work clearly indicate commercial use contact RNA assess visualization ribosomal RNA band figure elevated threshold baseline decrease ratio indicative degradation broad band show dna contamination Figure apparent review literature standard rrna ratio difficult meet especially RNA derive clinical sample appear relationship rrna profile mrna integrity somewhat unclear hand reflect unspecific damage RNA include sample mishandling postmortem degradation massive apoptosis necrosis reflect specific regulatory process external factor living cell altogether appear total RNA low rrna ratio necessarily poor quality especially degradation product observe electrophoretic trace Figure reason development reliable fully integrate automate system appropriate numeric evaluation RNA integrity highly desirable standardized RNA quality assessment allow reliable comparison experiment facilitate exchange biological information scientific community prospect mind aim anticipate future standard pre normative research identify test software package recently develop gauge integrity RNA sample user independent strategy open source degradometer software calculation degradation factor true ratio base peak height freely available RIN algorithm Agilent expert software base computation RNA Integrity Number RIN tool develop separately extract information RNA integrity microcapillary electrophoretic trace produce userindependent metric tool assess purity integrity RNA sample derive different human adult tissue cell line represent tumor result compare conventional RNA quality measurement approach highly expert human interpretation evaluate simplicity user examine potential accuracy efficiency method contribute standardization RNA integrity assessment upstream biological assay procedure validate real time RT PCR quantitation expression level housekeeping gene RNA sample different level degradation total RNA prepare human cell line especially ATCC bio resource center tissue sample clinical sample different human adult tissue type blood brain breast colon epithelium kidney lymphoma lung liver muscle prostate rectum thyroid RNA purification perform cesium chloride ultracentrifugation accord Chomczynski Sacchi phenol base extraction method TRIzol reagent Invitrogen USA silica gel base purification method RNeasy Mini Kit Qiagen Germany Strataprep kit Stratagene USA SV RNA isolation kit Promega USA accord manufacturer instruction modification material maintain minimal handling RNA extraction carry RNase free environment Supplementary Table online commercially available RNA sample Universal Human Reference distribute Stratagene USA human brain muscle RNAs supply Clontech extract RNA concentration purity verify uv measurement pro Amersham Biosciences USA mm cuvette absorbance spectra measure nm determine ratio calculate microcapillary electrophoresis measurement Agilent bioanalyzer Agilent Technologies USA conjunction RNA Nano RNA Pico LabChip kit total assay run accordance manufacturer instruction Supplementary Notes online evaluate reliability classifier system describe study replicate run set RNA sample load different chip result datum point sample Human RNA integrity categorization RNA integrity checking perform expert operator classify total RNA sample predefine discrete category examine integrity RNA electropherogram Supplementary Table online low number indicate high integrity reference criterion parameter include ribosomal peak definition baseline flatness existence additional noise peak ribosomal peak low molecular weight species contamination genomic dna presence suspicion smearing peak decrease intensity ratio indicate degradation RNA sample result classification high category evaluate robustness human interpretation highly experienced operator train catalog step separately classify subset sample breast cancer include sample varying level integrity intact RNA low quality sample wide range degradation electrophoretic datum export degradometer software folder format comparison sample original datum scale classifier system time axis compensate difference migration time fluorescence intensity axis compensate variation total RNA result fluorescence curve shape peak height scale Degradation Factors DegFact correct ratio calculate Supplementary Table online mathematical model develop Auer et al examine additional degradation peak signal appear low molecular weight range compare ribosomal peak height calculation DegFact base numbering continuous metric range increase DegFact value correspond degradation new group integrity define graduation step classification RNA sample complete group integrity display show alert warn indicative measurable degradation Yellow Orange red non reliable datum come form fourth group Black introduce fifth class label White alert produce software software manual freely available Degradometer version release software Bioanalyzer electrophoretic size file format collect biosize software version release March import Agilent expert software RIN beta release RIN algorithm allow calculation RNA integrity train artificial neural network base determination informative feature extract electrophoretic trace feature identify signal analysis select feature collectively catch information integrity level include total RNA ratio ratio area ribosomal band total area electropherogram height peak fast area ratio ratio area fast region total area electropherogram height low marker total electropherogram RNA sample tissue mammalian specie human mouse rat show vary level degradation adaptive learning approach order assign weight factor relevant feature describe RNA integrity RIN number compute RNA profile Supplementary Table online result classification RNA sample numerically predefine category integrity output RIN decimal integer number range RIN return completely degraded RNA sample RIN achieve intact RNA sample case measure electropherogram signal unusual shape show example peak unexpected migration time spike abnormal fluctuation baseline case reliable RIN computation possible separate neural network train recognize anomaly display warning user suppress display RIN number combine result neural network RIN computation neural network detect anomaly RIN algorithm achieve mean square error mean absolute error independent test set beta release software manual freely available Agilent expert version release November software expression level housekeeping gene GUSB TFRC measure quantitative PCR TaqMan Gene Expression Assays accord manufacturer instruction Applied Biosystems USA sixteen aliquot unique batch RNA sample Universal Human Reference RNA Stratagene USA level integrity cf table test influence RNA quality relative expression gene parallel comparison separate GUSB TFRC TaqMan probe homogeneous quantity mg RNA sample subject reverse transcription step highcapacity cdna archive kit Applied Biosystems USA describe manufacturer single strand cdna product analyze real time PCR TaqMan Gene Expression Assays accord manufacturer instruction Applied Biosystems USA single strand cdna product analyze ABI prism Sequence Detector Applied Biosystems USA efficiency reproducibility reverse transcription test rRNA TaqMan probe assay case duplicate threshold cycle ct value obtain average expression level evaluate relative quantification method fold change test HKG target gene normalize rRNA reference gene compare high quality sample calibrator sample follow formula fold change ÀDDCt DDCt target reference sample target reference calibrator sample sample correspond sample target gene normalize reference gene calibrator sample represent expression level target gene normalize reference gene consider high quality sample mean ÀDDCt SD calculate consider sample individually group quality metric category base RIN metric DegFact value low upper bind mean Intervals Confidence IC analysis expression level HKG affect RNA degradation value mean fold change condition close statistic execute XLSTAT software version Addinsoft USA mean sd coefficient variation variation cv group sample calculate measure dispersion range inter quartile range quartile evaluation low upper bind mean interval Confidence IC comparative statistical analysis group complete non parametric statistical test independent Mann Whitney test independent Kruskal Wallis test analyze total RNA sample profile human tissue cell line tumoral normal origin varying level RNA integrity Supplementary Table online detail significant difference ratio observe specific group sample tumoral versus normal tissue versus cell line instance RNA extract normal sample display improved ratio fall desire range Figure contrast distribution ratio find correlate purification method tissue origin RNA integrity assess resolve ribosomal RNA band Agilent bioanalyzer RNA protocol analysis RNA profile datum sample obtain device problem run system automatically provide ratio profile figure show distribution compute value median ratio variation mean IC addition significant degree variability ratio find identical sample replicate run time RNA sample ratio great rare value measure theoretically desire range sample prepare cultured cell figure integration fail remain case display atypical migration clear rrna band ratio compute datum show expert operator categorize set RNA sample inspect electrophoretic trace successful assay RNA profile check score predefine category figure good Human Categorization regular hc level moderate hc level low hc level degrade hc level remain flag display temperature sensitive profile RNA sample initially find intact highly degraded heat RNase contamination observe datum robustness cataloging comparison qualify criterion set breast cancer sample Materials Methods integrity sample evaluate independently expert operator categorization find highly reliable coefficient variation CV low consider individual interpretation involve explain fact experienced operator accomplish scoring base clearly define set instruction limit frequently observe subjective visual interpretation inconsistency human categorization predictably ratio denote high quality majority RNA sample classify HC level total RNAs low baseline rrna marker classify HC level Figure consider suitable downstream application RNA degradation assess degradometer software Materials Methods RNA profile check score predefine class figure altogether Degradation factor DegFact value compute remain RNA sample display profile interpret reliably DegFact value score sample flag black category Figure correspond sample previously classify operator degrade hc level remain case average degradation factor ic large variation entire set sample mean range low variability persistently find identical sample replicate run consider result observe DegFact value cv addition statistically significant difference find DegFact value sample sort type high DegFact value find characteristic tissue sample display DegFact compare cell line datum find significant linear relationship DegFact value distribution explicit human categorization HC class correspond unambiguous DegFact distribution figure HClevels form single class hc level mean DegFact sd ic HC level mean Deg Fact sd ic HC level mean DegFact sd ic HC level mean DegFact sd ic worth mention normalize height peak interval rescale gradually decrease reverse increase degradation figure RNA sample measure parallel base RNA Integrity Number metric artificial neural network train distinguish different RNA integrity level examine shape microcapillary electrophoretic trace Materials Methods RNA profile check score successfully figure average RIN ic remain sample associate unexpected signal disturb computation RIN default anomaly detection parameter case flag alert add correspond critical anomaly include unexpected datum sample type ribosomal ratio baseline signal region datum categorization find regular variability replicate run compare method consistently small CV expect high RIN characteristic cell line sample display RIN compare tissue sample datum group correspond RNA profile analyze default setting RIN system low threshold RNA quantity load ng reliable detection anomaly recommend manufacturer ng significant linear relationship find RIN number explicit human classification provide operator Figure RNA degradation characterization integrity RNA sample profile score degradometer software total RNA profile successfully categorize predefine alert class mathematical model quantify RNA degradation compute degradation factor DegFact class White Yellow Orange Red associate different level degradation fifth class black alert correspond sample system able qualify accuracy distribution represent number record class comparative analysis human evaluation axis base electrophoresis analysis reference RNA integrity classification observation rrna peak height DegFact value take HC level histogram refer mean rrna peak height confidence interval fluorescence intensity leave scale Mean DegFact value confidence interval arbitrary unit right scale plot mean join DegFact value calculate degradometer software figure distinct HC class correspond explicit RIN number HC level form single class hc level mean RIN sd ic HC level mean RIN sd IC HC level mean RIN sd ic HC level mean RIN sd ic remain sample assay ng RNA separate group consider sample compute RIN sample group derive RNA Nano assay mean RNA quantity load ng ng low limit quantitation indicate manufacturer sample estimate operator poor quality hc level degrade hc level flag Black degradometer software DegFact value score RNA profile interpret reliably possibly low RNA concentration unusual migration behavior shift baseline value degraded sample automate system disagreement sample human interpretation case agreement RIN system inconsistency second group sample profile derive RNA Pico assay RNA quantity load average ng ng manufacturer specification estimate operator range high hc level correct hc level quality level addition RNA profile score degradometer software display alert flag slight degradation detect associate low mean DegFact value IC automate system human interpretation agree case inconsistency influence RNA quality categorization obtain user independent classifier gene expression profiling explore real time RT PCR expression level housekeeping gene GUSB TFRC measure aliquot unique RNA display integrity metric table mean correlation coefficient threshold cycle ct sample quality metric find high consider RIN metric consider DegFact value value mean fold change calculate accord ÀDDCt quantification method Materials Methods find low correspond expression level sample exhibit high RNA quality table Figure consider HKG expression measure relative reference sample obvious decline relative expression level observe sample categorize accord RIN metric figure DegFact value Figure result indicate fold difference expect relative expression level gene sample differ quality table fold difference large measure RNA sample comparable integrity consistently low table Figure addition unambiguous gap distribution define figure distinguish RNA sample high quality category RIN DegFact value low category RIN DegFact value expect measure expression intact mrna yield approximately equal result regardless region probe mrna fragmentation occur sequence abundant test effect PCR probe location RNAs GUSB probe separate nt associate highly similar threshold cycle ct measure parameter figure similar result obtain TFRC probe separate nt parameter datum show region probe source variation result universally accept RNA purity integrity foremost importance ensure reliability reproducibility downstream application biomedical literature PubMed November article relate RNA include respectively quality integrity term find contain RNA quality RNA integrity term interestingly half publish propose standard operational procedure RNA quality assessment scientific community study report base year old method indicate represent established currently method result strongly challenge reliability usefulness conventional method demonstrate inconsistency evaluate RNA quality ratio reflect RNA purity informative integrity RNA available RNA extraction purification method yield highly pure RNA little dna contamination result ratio sample find degraded poor quality high ratio indicative limited protein contamination high ratio indicative absence residual contamination organic compound phenol sugar alcohol highly detrimental downstream application nonetheless sample display low ratio exhibit inhibition downstream application cDNA synthesis labeling vitro transcription datum show second lack reliability rrna ratio gold standard assess RNA integrity ribosomal ratio calculate identical sample independent run large degree variability CV observe biosize software find rrna ratio evaluation compromise fact calculation base area measurement heavily dependent definition start end point peak case system unable localize ribosomal peak ratio compute remain sample clear correlation ratio RNA integrity find RNAs usually high quality RNAs study display consider good quality interestingly Auer et al study tissue seven organism report objective measurement RNA integrity possibly comparison scale peak height correspond area actually observe linear relationship RNA integrity difference normalize peak height increase degradation result significant decrease scale correct height ribosomal peak inversion ratio highly degraded stage cf figure comparison area computation rrna scale peak height measurement produce consistent value cv reduce display clear concentration independent value Supplementary Tables online human evaluation integrity RNA visual inspection electrophoresis profile provide consistent datum variability classification produce independent expert operator CV low automate management conventional control area value CV time consume strongly dependent individual competence highly train specialist set RNA sample allocate predefine category correspond profile consider expert atypical display temperature sensitive shape datum strategy appear unsuitable standardization quality control RNA integrity assessment require simple consistent expert independent classification facilitate information exchange laboratory value correspond number sample category mean quality metric RIN DegFact mean fold change ÀDDCt relative reference sample indicate confidence interval observe technical variation IC rep specify consider duplicate gene target sample replicate gene calibrator sample measure reference sample exhibit RIN DegFact value default mean fold change set observed decrease expression relative expression relative reference sample calculate fold difference refer fold ratio expect expression level gene category category give sample differ quality category category consider RNA comparable integrity fold ratio technical variation expect chance gene expression level technical reason consider investigate performance recently develop user independent software algorithm degradometer software provide reliable evaluation RNA integrity base identification additional degradation peak signal integration mathematical calculation ribosomal peak height allow characterization integrity sample test alert flag find fairly informative strongly reduce complexity metric introduce distinct class label Yellow Orange Red straightforward simple filtering step degradation factor DegFact metric yield precise measure cv valuable flag alert purpose standardization true RNA Integrity Number RIN software allow characterization integrity RNA sample test RIN value RNA sample profile cv general good agreement human classification degradation factor RIN figure provide cross validation user independent qualification system test result refinement human interpretation validate statistically relevant class sample good hc level moderate hc level poor hc level degrade hc level RNA sample previously flag operator display atypical temperature sensitive shape unambiguously assign category sample RIN ic DegFact IC datum show altogether find degradometer RIN algorithm highly reliable user independent method automate assessment RNA degradation integrity RIN system slightly informative tool able compute assessment metric RNA profile compare degradometer software remain flag respectively black alert sample available low limit ng RIN system provide figure workflow operational procedure RNA quality assessment integrity RNA extract purify cell line clinical biological tissue sample control widely bioanalyzer electrophoretic trace standard Agilent analysis software RIN metric calculate score RNA sample numerically predefine category integrity RIN threshold value independent control degradation factor metric DegFact threshold value optionally allocate RNA sample bioanalyzer independent degradometer software standard operating procedure RIN DegFact metric standard exchange language document RNA integrity degradation second classify RNA homogeneous group finally select sample comparable RNA integrity improve scheme meaningful downstream experiment standard operating procedure benefit feedback information help user define threshold integrity metric value base result RNA base analysis metric value compare degradometer software similarly RIN system able provide metric value poor quality sample include low quality degraded sample degradometer software classify advantage RIN classifier critical anomaly detect include genomic dna contamination wavy baseline etc threshold setting change reliable RIN value compute case RNA sample profile successfully classify system intact RNA obviously constitute good representation natural state transcriptome situation gene expression analysis desirable partially degrade RNA study report collection reasonable microarray datum RNA sample impair quality lead meaningful result carefully Auer et al recently conclude degradation preclude microarray analysis comparison sample comparable RNA integrity confirm direct influence RNA quality distribution gene expression level detect PCR significant fold difference relative expression gene sample slightly decrease RNA integrity large variation comparable RNA quality category cf figure Table correlate ratio discrepancy gene expression experiment false positive false negative rate differential gene expression compare sample compute reliable metric RNA integrity RNA find partially degrade highly valuable straight unambiguous relationship establish human interpretation RIN DegFact distribution indicate metric possible distinguish specific sample disparate include comparative gene expression analysis compromise result information provide user independent classifier guarantee successful downstream experiment give comprehensive picture sample safeguard perform useless costly experiment RIN system simple metric easily integrate sample track information system definition standard operating procedure quality assurance follow scheme describe Figure context suggest grow number laboratory perform RNA Quality Control microcapillary electrophoresis offer option report objective RNA quality metric Minimum information Microarray Experiment MIAME standard registration RNA profile public electronic repository standardized information enable facilitate comparison RNA base bioassay perform laboratory RNA sample similar quality way sequence trace compare','2005-03-30 00:00:00','http://dx.doi.org/10.1093/nar/gni054',3244,896),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json','Injection drug use and HIV/AIDS in China: Review of current situation, prevention and policy implications','Illicit drug abuse and HIV/AIDS have increased rapidly in the past 10 to 20 years in China. This paper reviews drug abuse in China, the HIV/AIDS epidemic and its association with injection drug use (IDU), and Chinese policies on illicit drug abuse and prevention of HIV/AIDS based on published literature and unpublished official data. As a major drug trans-shipment country with source drugs from the \"Golden Triangle\" and \"Gold Crescent\" areas in Asia, China has also become an increasingly important drug consuming market. About half of China\'s 1.14 million documented drug users inject, and many share needles. IDU has contributed to 42% of cumulatively reported HIV/AIDS cases thus far. Drug trafficking is illegal in China and can lead to the death penalty. The public security departments adopt \"zero tolerance\" approach to drug use, which conflict with harm reduction policies of the public health departments. Past experience in China suggests that cracking down on drug smuggling and prohibiting drug use alone can not prevent or solve all illicit drug related problems in the era of globalization. In recent years, the central government has outlined a series of pragmatic policies to encourage harm reduction programs; meanwhile, some local governments have not fully mobilized to deal with drug abuse and HIV/AIDS problems seriously. Strengthening government leadership at both central and local levels; scaling up methadone substitution and needle exchange programs; making HIV voluntary counseling and testing available and affordable to both urban and rural drug users; and increasing utilization of outreach and nongovernmental organizations are offered as additional strategies to help cope with China\'s HIV and drug abuse problem.',' Illicit drug abuse increase public health social concern past decade worldwide drug abuse cause problem individual society include loss productivity transmission infectious disease crime family social disorder excessive health care expenditure human immuno deficiency virus associate injection drug use IDU needle sharing large extent stunning tragedy human history cause million death million people live HIV worldwide far Africa afflict continent populous country world China observe rapidly increase drug abuse HIV AIDS occurrence past year China shape course epidemic need swiftly great resolve paper review global illicit drug trafficking drug abuse association HIV AIDS epidemic China chinese policy illicit drug abuse prevention HIV AIDS offer additional strategy governmental crack drug smuggling drug use prohibition help cope China HIV drug abuse problem search English chinese language literature Medline China National Knowledge Infrastructure review unpublished official datum include national report illicit drug control HIV AIDS sentinel surveillance datum paper report review key database include drug abuse China trace late Qing Dynasty british colonist forcefully bring indian opium China exchange silk tea cash Opium locally plan founding new China million chinese people opium addict represent total population short nationwide antidrug campaign drug abuse report eliminate mainland early decade China believe drug free nation drug reemerge China China adopt open door policy reemergence mainly connect global drug trafficking activity Asia major opiate produce region Golden Triangle comprise southeast asian country Myanmar Burma Laos Thailand Golden Crescent include southwest asian nation Afghanistan Iran Pakistan majority heroin opium current chinese market bring Myanmar Yunnan Province Viet Nam Guangxi Province transship inland trafficking route Sichuan Guizhou Gansu Xinjiang Guangdong Shanghai Beijing small potion heroin opium traffic Xinjiang Golden Crescent drug penetrate province late increase amount amphetamine typestimulant ATS chemically related synthetic drug include amphetamine methamphetamine ecstasy locally manufacture consume China number drug user document officially chinese public security department increase million estimate number million lifetime prevalence rate illicit drug use resident age year old high prevalence chinese city increase year prevalence rate increase period main drug choice China heroin accord report National Narcotic Control Commission NNCC drug user abuse heroin abuse ATS MDMA methylenedioxymethamphetamine ecstasy popular city night club recent year farmer live rural border area Yunnan Guangxi province constitute large fraction drug user early urban resident use illicit drug majority drug abuser young people low education level limited job skill small proportion urban user regard drug indicator high social class nncc datum show age year experimentation peer pressure relaxation commonly cite reason begin use drug initially drug take primarily sniff snort smoke cigarette mix drug injection drug increasingly common drug user probably result increase price illicit drug great cost effectiveness inject achieve desire effect national behavioral surveillance datum show median prevalence IDU drug user increase April April median prevalence needle sharing idu increase period buy drug grocery store buy hospital common route obtain injection needle syrinx reason share needle include care hook think clean difficult buy obtain share select person money buy follow drug user IDU needle sharing risk behavior include unprotected commercial sex drug user ultimately non drug user high risk HIV believe female drug user sell sex drug limited publish research provide support evidence impact interaction IDU commercial sex HIV risk clear consider dramatic increase commercial sex industry China potential bridging role dual risk behavior transmit HIV high risk group general population research interaction IDU commercial sex give high priority AIDS case detect China cumulatively report HIV infection far HIV AIDS epidemic mainly concentrated specific geographical area sub group population IDU large contributor report epidemic HIV outbreak idu observe Ruili City Yunnan Province border Myanmar province autonomous region municipality mainland China report HIV infection idu infection China estimate IDU national sentinel site IDU detect HIV infection detect prevalence site detect positive average prevalence increase national average HIV prevalence idu varied dramatic geographic difference HIV prevalence observe Yunnan Xinjiang province severe HIV infection rate idu province close drug trafficking road Guangxi Sichuan Guizhou Hunan Jiangxi moderate epidemic HIV begin spread rapidly Yunnan province late early high prevalence idu detect Urumqi City Yining City Xinjiang Province later select sentinel site IDU province include Guangxi sentinel datum Jiangxi Sichuan Guizhou Hunan HIV national sentinel site IDU detect HIV infection HIV prevalence report select site Xinjiang Sichuan Guizhou Guangxi Hunan official HIV prevalence rate geographic area remain report reflect reality failure detection subtype major circulate recombinant form spread China prevalent strain subtype usually subtype main subtype idu co circulating multiple subtype China imply possibility interclade recombination characterization genetic variability help track epidemic generate subtype specific immunological reagent develop vaccine choose antiretroviral therapy regimen drug trafficking abuse illegal China offender sentence prison smuggle gram heroin receive death penalty smuggle gram heroin Regulations Prohibition Narcotics enact level penalty apply drug user time offender fine allow voluntary detoxification center receive day methadone treatment manage Ministry Public Health cost treatment chinese yuan cost consider expensive drug user family drug user go voluntary detoxification program catch drug send compulsory rehabilitation center CRC administer Ministry Public Security month drug user relapse user go CRC send reeducation labor center RELC administer Justice Department year chinese history drug abuse prostitution consider social evil chinese government typically take crackdown measure try eradicate phenomenon China achieve success story Illicit drug abuse prostitution eradicate national anti drug anti prostitution campaign success repeat past decade possible explanation China expand market economy open outside world situation impossible completely stem manufacture highly profitable illicit drug import drug China long porous land border drug produce country currently chinese government adopt pragmatic policy take measure target root surface drug abuse problem measure target root include continuously crack drug smuggling activity discourage new user anti drug education campaign chinese government actively seek collaborate neighboring country prevent drug smuggling border help Myanmar example reduce opium poppy cultivation replace crop plantation chinese mass medium increase anti drug education general population anti drug education include curricula primary secondary school student measure target surface include provide drug detoxification harm reduction service drug user China voluntary detoxification clinic crc relc accommodate patient western medication traditional chinese medicine detoxification community therapy provide heavily affect area late National Working Group Community base Methadone Maintenance Therapy establish collectively Ministry Health Ministry Public Security State China shun generalize epidemic HIV AIDS largely dependent China deal IDU risk factor break bridge IDU heterosexual transmission past experience China suggest solely crack drug smuggling prohibit drug use prevent solve illicit drug relate problem era globalization governmental support harm reduction program voluntary counseling testing utilization non governmental organization recommend China strong central government government support imaginable achieve success campaign drug use spread HIV AIDS late chinese central government step HIV AIDS control effort include set national policy framework respond HIV AIDS increase funding input expand collaboration international organization Chinese National Medium long term Strategic Plan AIDS Prevention Control formulate set goal health education prevent HIV AIDS std carry detoxification center education center jail Action Plan call create drug free community drug prohibition education drug detoxification activity active promotion healthy life style behavior harm reduction drug user hand substantial autonomy provincial level area response drug use HIV AIDS epidemic vary significantly provincial low administrative level example Yunnan Guangxi province far province support implement advocate harm reduction intervention idu local government fully motivated confront drug abuse HIV AIDS problem government leader ignore cover problem far interested economic growth HIV AIDS control wish administrative area economy province economy city AIDS province AIDS city believe helpful career promotion advocacy support government leader administrative level harm reduction community base prevention need harm reduction include strategy methadone maintenance needle exchange dispense drug outreach service harm reduction controversial issue compare abstinencebase philosophy harm reduction encourage tolerance social phenomenon undesirable hazardous result social turpitude instead abstinence consider proper way address drug problem United States law policy restrict use federal fund support needle injection equipment distribution project study refute concern access sterile syrinx endorsement IDU likely result increase injection initiation injection harm reduction project China pilot phase scholar health official similar concern western country believe needle exchange service send wrong signal encourage drug abuse drug user public consider methadone substitution unethical use drug replace drug drug use HIV AIDS spread rapidly country urgent find support domestic evidence harm reduction reduce HIV transmission evaluate harm reduction project ultimately scale harm reduction effort easy access sterile needle syrinx urban area China legally sell available pharmacy medical clinic drug user live rural area addition drug user share needle syrinx buy night time money buy methadone orally administer metha substitution reduce injection needle sharing opiate drug addict methadone maintenance therapy costly require drug user attend clinic regular basis methadone substitution needle exchange service available affordable convenient time urban rural setting especially community heavy drug use addition cost availability factor affect acceptability methadone maintenance therapy concern safety efficacy therapy greater retention treatment find result great decrease drug use criminal activity unemployment length drug treatment positive association well post treatment outcome limited experience methadone maintenance therapy MMT China show high rate dropout international study show motivational enhancement therapy motivational interviewing enhance treatment initiation retention outcome MMT program add behavioral intervention component MMT program increase abstinence reduce HIV risk behavior policy orient operational research need China well understand increase effectiveness MMT harm reduction intervention chinese context persistent conflict policy legal landscape central government give explicit support harm reduction example state Medium Long Term Strategic Plan Action Plan program implement successfully China country public health public security authority frequently approach drug abuse different perspective lead conflict approach local level crackdown philosophy detention drug user China reflect inconsistent interpretation harm reduction present challenge public health official implement methadone substitution needleexchange program drug user reluctant participate program fear catch police officer impossible completely solve dilemma near future conflict expect gradually reduce follow reason chinese national policy HIV prevention control pragmatic past year MMT needle exchange program unimaginable year ago ready expand country expect open policy trend continue chinese economy increasingly merge international market trend favor harm reduction program Fur thermore China centralized government achieve advantage promote public health policy policy believe correct second inter agency coordination public health crisis enhance central local governmental level SARS outbreak reduce potential conflict public health policy public health worker provide policy advocacy public security authority help change traditional norm illicit drug control obtain support harm reduction operational research need provide evidence benefit harm reduction program convince policy enforcer lead revision unfavorable policy component HIV voluntary counseling testing VCT consider step initiate prevention therapy strategy address AIDS epidemic people opportunity know HIV status precaution avoid spread receive early therapy infect develop country atrisk people VCT national british survey show idu VCT past year fourth million infected people United States remain unaware HIV positive China large discrepancy report estimate million cumulative HIV AIDS case far risk individual seek standard HIV counseling testing service stigma associate drug use HIV AIDS fear arrest know positive result major barrier access VCT survey pregnant woman health professional Yunnan Province epicenter AIDS epidemic China prevalent negative attitude HIV AIDS percent health professional pregnant woman think HIV disease low class illegal people health professional pregnant woman think HIV positive individual allow marry health professional willing treat HIV positive individual cost traditional VCT low awareness risk factor HIV infection distance inconvenience time prevent access VCT possible solution include development outreach program offer anonymous testing counseling heighten risk HIV infection adoption new technology rapid saliva testing counseling strategy improve outreach efficacy program non governmental organization NGO play critical role delivery HIV prevention service assistance person live AIDS flexibility NGOs enable respond quickly fill gap health care social service NGOs government agency willing dofor example reach perceive threat idu marginalize sub group behavior stigmatize high risk HIV AIDS recent survey NGOs Central Eastern Europe show NGOs target injection drug user provide needle exchange HIV prevention peer education deliver AIDS presentation distribute educational material Africa main transmission occur heterosexual activity NGOs likely direct attention general public youth provide peer education community outreach Thailand Brazil success observe control HIV AIDS epidemic reduce AIDS mortality NGO believe play key role program lack rigorous systemic evaluation NGOs face difficulty lack financial resource lack communication governmental organization governmental indifference opposition AIDSrelated stigma large number government organize NGOs GONGOs include Family Planning Associations Women Federation Red Cross Youth League trade union diverse academic association member GONGOs formal position governmental organization volunteer GONGOs true NGO relationship government emerge China exist chinese GONGOs getting involve sexually transmit disease AIDS prevention chinese government encourage participate AIDS control government sponsor NGOs support HIV AIDS education academic publication participate AIDS research education foreign governmental non governmental partner serve powerful aid chinese government achieve goal stop spread drug use HIV AIDS epidemic spite potential greatly contribute HIV prevention China little literature area exception recent study Chen Liao Women Federation HIV prevention program south China study show Women Federation able deliver culturally orient multi level intervention program target female drug user datum indicate program successfully increase knowledge HIV AIDS increase condom use decrease needle syringe sharing female drug user project study systematically evaluate implementation effectiveness NGO base intervention program greatly need future declare compete interest HZQ JES conceive study write draft manuscript HZQ HTC YHR collect datum author participate datum interpretation manuscript revision','2006-02-01 00:00:00','http://dx.doi.org/10.1186/1477-7517-3-4',2177,882),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json','The gene of an archaeal α-l-fucosidase is expressed by translational frameshifting','The standard rules of genetic translational decoding are altered in specific genes by different events that are globally termed recoding. In Archaea recoding has been unequivocally determined so far only for termination codon readthrough events. We study here the mechanism of expression of a gene encoding for a α-l-fucosidase from the archaeon Sulfolobus solfataricus (fucA1), which is split in two open reading frames separated by a −1 frameshifting. The expression in Escherichia coli of the wild-type split gene led to the production by frameshifting of full-length polypeptides with an efficiency of 5%. Mutations in the regulatory site where the shift takes place demonstrate that the expression in vivo occurs in a programmed way. Further, we identify a full-length product of fucA1 in S.solfataricus extracts, which translate this gene in vitro by following programmed −1 frameshifting. This is the first experimental demonstration that this kind of recoding is present in Archaea.',' translation optimally accurate correspondence nucleotide protein sequence consider immutable dogma genetic code universal certain organelle small number organism meaning different codon reassign mrnas decode accordingly surprisingly standard rule genetic decoding alter specific gene different event globally term recode case translational recoding occur competition normal decoding proportion ribosome obey universal rule translational recoding identify prokaryote eukaryote crucial role regulation gene expression include stop codon readthrough ribosome hopping program frameshifting review stop codon readthrough stop codon decode trna carry unusual amino acid translational release factor specific stimulatory element downstream stop codon regulate process hop ribosome stop translation particular site mrna start nucleotide downstream rare event study detail bacteriophage programmed frameshifting ribosome induce shift alternative overlap reading frame nt frameshifting frameshifting mrna process regulate frequency vary different gene program frameshifting study extensively virus retrotransposon insertion element case document instead phenomenon far common cellular gene single case program frameshifting know prokaryote eukaryote include human gene regulate recode event describe previously reference compare frameshifting frameshifting widespread example prokaryote eukaryote program frameshifting trigger element mrna slippery sequence show XXY YYZ motif base usually base function favour trna misalignment site shift take place frameshifting stimulate element flank slippery sequence codon low abundance trna stop codon Shine dalgarno sequence mrna secondary structure correspondence address tel fax email Open Access article distribute term Creative Commons Attribution non commercial License permit unrestricted non commercial use distribution reproduction medium provide original work properly element combination enhance frameshifting pause translating ribosome slippery sequence know case recode Archaea recently review limit termination codon readthrough event regulate incorporation amino acid selenocysteine pyrrolysine respectively archaeal gene regulate translational program frameshifting ribosome hopping identify experimentally far compare domain life study translational recoding Archaea dawn show fucosidase gene crenarchaeon Sulfolobus solfataricus putatively express program frameshifting gene name organize open reading frame orf amino acid respectively separate frameshifte base overlap figure report previously region overlap orf characteristic feature gene express program frameshifting include slippery heptanucleotide AAA AAT codon show zero frame flank putative stem loop rare codon CAC Figure resemble prokaryotic stem loop hairpin Shine Dalgarnolike site show frameshifting obtain mutate site direct mutagenesis gene exactly position predict slippery site produce length gene name encode polypeptide amino acid figure mutant gene express Escherichia coli fully functional fucosidase name Ssa fuc thermophilic thermostable unusual nonameric structure recently determine reaction mechanism function residue active site mutant enzyme functionality product mutant gene provide direct experimental evidence program frameshifting occur vivo address issue report study expression wild type split gene mutant slippery sequence demonstrate express program frameshifte experimental demonstration kind recode present Archaea domain life relevance program frameshifting Archaea discuss figure fucosidase gene Region overlap wild type split gene terminal ORF zero frame terminal ORF fragment show frame slippery heptameric sequence underline rare codon box arrow indicate stem putative mrna secondary structure amino acid involve program frameshifting codon translate event frame shadow Fragment length mutant gene small arrow indicate mutate nucleotide analysis fucosidase expression cell grow cell extract obtain describe previously expression strain wild type gene mutant gene previously name FrameFuc sm tm fusion glutathione transferase GST purification recombinant protein perform report previously nomenclature paper different fucosidase gene list Table western blot study equal amount culture express wild type mutant gene normalize OD resuspend SDS PAGE loading buffer contain Tris HCl buffer pH SDS glycerol mercaptoethanol blue bromophenol sample incubate min indicate directly load gel western blot analysis perform blot SDS PAGEs concentration indicate Hybond polyvinylidenfluorid filter Amersham Biosciences Uppsala Sweden polyclonal anti ssa fuc antibodie rabbit PRIMM Milan Italy anti gst antibody Amersham Biosciences dilute respectively filter wash incubate ImmunoPure anti rabbit IgG antibody conjugate horseradish peroxidase HRP Pierce Biotechnology Rockford IL USA filter develop ECL plus Western Blotting Detection system Amersham Biosciences follow manufacturer indication molecular weight marker western blot analysis ECL streptavidin hrp conjugate Amersham protein concentration sample measure method Bradford amount sample load SDS PAGEs indicate quantification band identify western blot perform program Quantity ChemiDoc EQ System Bio Rad Hercules USA volume analysis tool frameshifte efficiency calculate ratio intensity band frameshifted product frameshifte product termination product mutant slippery sequence wild type gene prepare site direct mutagenesis vector describe previously synthetic oligonucleotide PRIMM follow rev fwd rev fwd mismatched nucleotide mutagenic primer underline direct sequencing identify plasmid contain desire mutation mutant gene name sm tm completely sequence mutant Ssa fuc prepare site direct mutagenesis vector site direct mutagenesis kit describe synthetic oligonucleotide rev describe follow mutagenic oligonucleotide Fuc mismatched nucleotide underline direct sequencing identify plasmid contain desire mutation mutant gene name completely sequence enzymatic characterization Ssa fuc perform describe previously protein express wildtype gene mutant sm purify describe fractionate SDS PAGE protein band excise gel wash mm ammonium bicarbonate pH acetonitrile reduce mm DTT min alkylate mm iodoacetamide min room temperature dark gel piece wash time buffer resuspend mm ammonium bicarbonate incubate ng trypsin overnight supernatant contain peptide analyse MALDIMS Applied Biosystem Voyager DE PRO mass spectrometer hydroxycynnamic acid matrix mass calibration perform standard mixture provide manufacturer liquid chromatography online tandem mass spectrometry LCMSMS analysis perform TOF hybrid mass spectrometer Micromass Waters Milford MA USA couple CapLC capillary chromatographic system Waters Peptide ion select collision cell fragmented analysis daughter ion spectra lead reconstruction peptide sequence genomic DNA strain prepare describe previously dna fragment nt contain complete gene prepare PCR follow synthetic oligonucleotide Genenco Florence Italy fwd TACGACTCACTATAGGTCAGTTAAATGTCACAAAA rev sequence correspond genome underline fwd primer sequence promoter RNA polymerase boldface sequence bsshii site show italic PCR amplification perform describe previously amplification product clone bsshii site plasmid pBluescript II gene completely sequence check undesired mutation introduce PCR recombinant vector obtain name pBlu translation vitro experiment plasmid express mutant gene sm tm experiment translation vitro prepare substitute KpnI NcoI wild type fragment contain slippery site isolate mutant check result plasmid correct sequence mutant gene completely sequence mrnas encode wild type mutant obtain vitro run transcription mg plasmid linearize bsshii incubate RNA polymerase min transcription mixture treat DNAseI RNAse free min transcribed RNAs recover extract sample twice phenol pH phenol chloroform follow precipitation ethanol mrnas resuspend DEPC treat approximate concentration pmol ml vitro translation assay perform essentially describe Condò et al sample ml final volume contain ml cell extract mm KCl mm Tris HCl pH mm Mg acetate mM ATP mM GTP mg bulk tRNA ml ci mmol mci ml pmol mrna mixture incubate min time synthesize protein resolve electrophoresis acrylamide SDS gel reveal autoradiography dry gel Instant Imager apparatus cell strain grow minimal salt culture medium supplement yeast extract casamino acid plus glucose YGM sucrose YSM extraction total RNA perform report previously total RNA extensively digest DNAse Ambion Austin TX USA absence DNA assess lack PCR amplification set primer describe RT PCR experiment perform report previously primer describe previously allow amplification region nt position ATG codon number overlap orf real time PCR experiment total cDNA obtain kit Quantitect RT Qiagen GmbH Hilden Germany ng preparation RNA describe cdna amplify Bio Rad LightCycler dynamo HS Syber Green qPCR kit Finnzymes Oy Espoo Finland synthetic oligonucleotide PRIMM amplification region ORF follow gene rrna gene amplification gene cdna amplification rRNA control template cDNA include PCR condition min initial denaturation follow cycle final step min Product purity control melt point analysis setpoint temperature increase PCR product analyse agarose gel visualize ethidium bromide stain expression value gene normalize value determine rrna gene absolute expression level calculate ratio YSM YGM cell respectively relative mrna expression level YSM YGM ratio calculate ratio YGM ratio YSM cDNA triplicate amplification wild type gene express GST fuse protein produce trace amount fucosidase activity unit mg removal GST suggest programmed frameshifting occur enzyme purify GST purification system analyse SDS PAGE reveal major protein band Figure sample control band excise gel digest situ trypsin directly analyse matrix assist laser ionization mass spectrometry MALDIMS show figure spectra reveal occurrence identical mass signal correspond peptide Peptide encompass overlap region orf result confirm liquid chromatography online tandem mass spectrometry LCMSMS analysis peptide mixture fragmentation spectra signal show common sequence Asn Phe Gly pro val Thr Asp Phe Gly Tyr Lys amino acid ORF underline result unequivocally demonstrate protein contain Peptide produce frameshifte event occur exactly slippery heptamer predict analysis DNA sequence region overlap orf figure MALDIMS analysis product wild type gene reveal presence second Peptide absent spectra Ssa fuc control protein figure sequence Peptide obtain LCMSMS figure Lys Phe Gly pro val Thr Asp Phe Gly Tyr Lys sequence differ amino acid Peptide demonstrating interrupted gene express fulllength protein originate different frameshifting event Polypeptide result shift site identical Ssa fuc prepare site direct mutagenesis suggest expression occur simultaneous site slippage instead polypeptide name Ssa fuc generate frameshifte second site result single site slippage Figure measure global efficiency frameshifting site wild type gene analyse total extract western blot anti gst antibody figure band mark different electrophoretic mobility observe polypeptide kda migrate like GST Ssa fuc fusion identify originate frameshifte site protein kda express mutant gene show electrophoretic mobility compatible GST fuse polypeptide encode ORF solely kda respectively polypeptide originate translational termination ribosome OCH codon terminal ORF figure calculated ratio frameshifting termination product test length fucosidase produce frameshifting event site Ssa fuc result single site slippage different property Ssa fuc sequence arise simultaneous site slippage prepare enzyme site direct mutagenesis slippery sequence AAA AAT mutate AAG AAG mutation underline new mutant gene name produce conservative mutation disrupt slippery sequence reduce shift efficiency second insert produce frameshifting result amino acid sequence Peptide sequence length mutant gene differ region slippery sequence AAG AAT TTC GGC AAG AAG TTC GGC respectively mutation underline nucleotide boldface originally frame table recombinant Ssa fuc purify Materials Methods Gel filtration chromatography demonstrate native condition Ssa fuc nonameric structure Ssa fuc identical molecular weight kDa datum show addition Ssa fuc high substrate selectivity Ssa fuc enzyme high affinity nitrophenyl fucoside fuc substrate identical experimental error mm cat Ssa fuc Ssa fuc addition nitrophenyl arabinoside nitrophenyl glucoside substrate Ssa fuc show previously Ssa fuc suggest different amino acid sequence significantly affect active site enzyme show identical profile specific activity versus temperature optimal temperature high datum show heat stability ph dependence Ssa fuc Ssa fuc report figure optimal growth temperature half life Ssa fuc min fold low Ssa fuc Figure enzyme show different behaviour pH Ssa fuc barely active stable figure enzyme show similar value specific activity phs close intracellular pH slippery sequence experimental datum report indicate predict slippery heptanucleotide region overlap orf wildtype gene regulate cis frameshifte event observe test hypothesis mutate sequence AAA AAT AAG AAT AAG AAC mutation underline obtain single mutant sm triple mutant tm gene respectively worth note mutation disrupt slippery sequence maintain frameshift orf table surprisingly expression sm produce length polypeptide purification affinity chromatography removal GST protein show electrophoretic migration Ssa fuc Ssa fuc Figure protein characterize mass spectrometry analysis follow situ tryptic digestion interestingly MALDI spectra reveal presence single peptide encompass overlap region orf mass value Da peptide Figure sequence peptide determine fragmentation spectra obtain lcmsms analysis Glu Phe Gly pro val Thr Asp Phe Gly Tyr Lys figure remarkably apart Glu residue sequence identical peptide produce indicate mutant gene sm frameshifte event observe wild type gene occur presence Glu instead Lys unexpected mutation AAA AAG AAT sm conservative zero frame ORF encode Lys produce mutation frame ORF worth note frameshifte efficiency gene sm calculate western blot describe fold high compare figure indicate mutation cancel frameshifting site time enhance frameshifting efficiency site contrast triple mutant tm produce low molecular weight band result translational termination figure length protein detect western blot probe anti GST Figure anti ssa fuc antibody figure datum disruption heptameric slippery sequence completely abolish frameshifting confirm sequence direct role control frameshifting vivo test express analyse extract cell grow yeast extract sucrose casaminoacid medium YSM accurate assay show extract contain unit mg fucosidase activity low amount hamper purification enzyme extract cell grow YSM reveal western blot band molecular mass kDa signal detect pre immune serum confirm specificity anti ssa fuc antibody figure different molecular mass result post translational modification occur archaeon incomplete denaturation protein complex particular event unusual enzyme hyperthermophilic archaea test hypothesis appropriate cellular extract analyse western blot extend incubation interestingly treatment shift high molecular mass band kda figure differ recombinant Ssa fuc kda leave question origin difference unsolved try shed light immunoprecipitate extract anti ssa fuc antibody analyse major protein band MALDIMS unfortunately observe peptide compatible fucosidase heavy IgG chain co migrate band expected molecular weight datum test scarce amount fucosidase extract result reduce expression transcriptional level perform northern blot analysis total RNA extract cell grow YSM YGM medium observe signal probe match ORF datum show result suggest produce rare transcript analyse level mrna RT PCR real time PCR band correspond region overlap orf observe RNA extract cell grow YSM mg product gene sm mg Ssa fuc mg band fast electrophoretic mobility result proteolytic cleavage length protein Partial MALDIMS spectrum tryptic digest mutant sm express mass signal correspond peptide encompass overlap region indicate lcmsms analysis peptide amino acid sequence infer fragmentation spectra report western blot cellular extract express wild type sm tm gene Materials Methods blot probe anti GST antibody western blot partially purify protein sample express fuse GST wild type mutant gene cellular extract load GST Sepharose matrix washing equal amount slurry ml ml denaturate load SDS PAGE extract cell express parental plasmid negative control pGEX blot probe anti ssa fuc antibody YGM medium demonstrate condition orf co transcribed figure experiment real time PCR show Figure demonstrate amplify cycle amplification mrna observe cycle despite fact fold cdna amplification indicate gene transcribe low level significant difference mrna level observe cell grow YSM YGM medium confirm analysis western blot extract cell prepare total RNAs reveal equal amount fucosidase extract figure low fucosidase activity observe condition test result poor transcription gene determine efficiency frameshifting perform ribosomes mrnas obtain vitro transcription clone wild type gene mutant thereof program vitro translation system prepare describe Condò et al aim promoter polymerase insert ahead gene interest obtain RNA transcript endow short region nt observe natural mrna autoradiography SDS PAGE translation product figure reveal wild type transcript produce tiny clear band molecular weight correspond length Ssa fuc obtain site direct mutagenesis translate efficiently cell free system spite encode quasi leaderless mrna judge relative intensity signal give translation product wild type length mutant efficiency frameshifte homologous system signal correspond polypeptide expect separated orfs kda respectively observe note product synthesize small detect gel system employ experiment large product ORF hand certainly absent datum unequivocally demonstrate ribosome decode split gene program frameshifte considerable efficiency produce length polypeptide orf condition drive expression length protein observe product sm tm construct datum demonstrate integrity heptanucleotide essential expression gene confirm gene decode program frameshifte organism addition lack expression sm translation vitro contrast efficient expression mutant indicate organism recognize different sequence regulate translational frameshifting identification gene expression regulate recode event serendipitous framework study glycosidase hyperthermophile identify genome archaeon split gene encode putative fucosidase express program frameshifting tackle issue study expression overcome problem connect scarcity expression fucosidase gene manipulation hyperthermophile report fact common strategy study recode event different organism expression wild type split gene lead production frameshifting length polypeptide efficiency value high observe gene express translational frameshifting heterologous system protein gpG gpgt gene express low level condition test particular transcriptional analysis gene reveal express low level YSM YGM medium similarly difference medium find western blot probe anti ssa fuc antibody indicate low expression enzyme result scarce transcription suppress translation western blot allow identify specific band kda heavy recombinant Ssa fuc experiment translation vitro show wild type gene express length polypeptide exhibit molecular mass recombinant protein demonstrate translational machinery fully competent perform program frameshifting likely observe discrepancy molecular mass arise post translational modification produce translation vitro experiment require characterize fucosidase identify MALDIMS lcmsms analysis product wild type split gene demonstrate independent frameshifting event occur vivo propose slippery site particular sequence obtain LCMSMS demonstrate peptide result simultaneous backward slippage site trnas Figure instead sequence peptide result positioning frame site trna fact incorporate amino acid specify codon new frame Figure expression frameshifting wild type gene follow model propose ribosomal frameshifting confirm significance slippery heptanucleotide promote program frameshifting vivo mutate putative regulatory sequence triple mutant tm give length product presumably mutation site slippery sequence dramatically reduce efficiency frameshifte observe previously metazoan result confirm intact slippery sequence wild type gene absolutely necessary expression contrast surprisingly single mutant sm show increase frequency frameshifting compare wild type produce polypeptide shift specifically site explain result observe mutation site slippery sequence AAA AAG AAT create novel slippery sequence AAG identical control expression program frameshifting transposase gene apparently single mutation inactivate simultaneous site trna positioning time foster shift efficiency trna site worth note instead simultaneous slippage effective figure single mutation slippery sequence sm completely annul expression gene indicate sequence essential archaeon program frameshifting exploit different mechanism furthermore difference enzyme produce frameshifting site Ssa fuc Ssa fuc respectively stability physiological temperature functionality Ssa fuc archaeon appear questionable reason regulate program frameshifting know physiological significance program frameshifting assign minority cellular gene uncertain reference mechanism recoding exploit set ratio polypeptide subunit DNA polymerase III holoenzyme alternatively program frameshifting balance expression protein bacterial translational release factor eukaryotic ornithine decarboxylase antizyme reference case polypeptide encode small ORF detect western blot analysis addition modelling Ssa fuc high resolution crystal structure fucosidase thermotoga maritima show terminal polypeptide independent domain show recently include essential catalytic residue exclude possibility functional fucosidase obtain ORF line evidence allow exclude program frameshifting set ratio polypeptide fucosidase probably translational mechanism require control expression level fucosidase gene express program frameshifting carbohydrate active enzyme example expression recode mechanism report gene encoding Helicobacter pylori interrupt frameshifting case expression program frameshifting lead functional enzyme synthesize component surface lipopolysaccharide evade human immune defensive system hard parallel model monosaccharide fucose involve variety biological function fucosidase play role metabolism fucosylate oligosaccharide experiment currently progress knockout wild type gene insert constitutive functional mutant gene archaeal fucosidase gene identify far probably result horizontal gene transfer event fucosidase gene regulate programmed frameshifting Bacteria Eukarya tempting speculate sophisticated mechanism translational regulation preexist apply fucosidase gene physiological reason identification gene interrupt frameshift open possibility regulate program frameshifting recently computational analysis prokaryotic genome reveal seven Archaea harbour interrupt code sequence include study computational analysis archaeal genome reveal interrupted gene present genome gene compose orf separate frameshifting express recode Cobucci Ponzano Rossi Moracci manuscript experimentally show time program frameshifting present Archaea domain finding miss piece puzzle phylogenetic distribution program frameshifting demonstrating mechanism universally conserve','2006-08-18 00:00:00','http://dx.doi.org/10.1093/nar/gkl574',3042,935),('281b4272005dbc998e36b034b7fb528c9ba038cd.json','s-RT-MELT for rapid mutation scanning using enzymatic selection and real time DNA-melting: new potential for multiplex genetic analysis','The rapidly growing understanding of human genetic pathways, including those that mediate cancer biology and drug response, leads to an increasing need for extensive and reliable mutation screening on a population or on a single patient basis. Here we describe s-RT-MELT, a novel technology that enables highly expanded enzymatic mutation scanning in human samples for germline or low-level somatic mutations, or for SNP discovery. GC-clamp-containing PCR products from interrogated and wild-type samples are hybridized to generate mismatches at the positions of mutations over one or multiple sequences in-parallel. Mismatches are converted to double-strand breaks using a DNA endonuclease (Surveyor™) and oligonucleotide tails are enzymatically attached at the position of mutations. A novel application of PCR enables selective amplification of mutation-containing DNA fragments. Subsequently, melting curve analysis, on conventional or nano-technology real-time PCR platforms, detects the samples that contain mutations in a high-throughput and closed-tube manner. We apply s-RT-MELT in the screening of p53 and EGFR mutations in cell lines and clinical samples and demonstrate its advantages for rapid, multiplexed mutation scanning in cancer and for genetic variation screening in biology and medicine.',' screen genetic change unveil molecular attribute human specimen important variety medical application include genotype inherit disorder prediction pathologic behavior malignancy identification cancer biomarker affect treatment decision individual patient example mutation gene like EGFR profoundly influence chemotherapeutic response lung cancer response modulate mutation gene signal pathway ras need efficient high throughput mutation screening multiple gene identify signal transduction pathway tumor sample large portion cancer cause genetic change remain unknown occur numerous position tumor suppressor gene ATM PTEN mutation scan detection specific mutation frequently require molecular cancer profiling sequencing consider gold standard comprehensive mutation analysis multi capillary electrophoresis sequence array pyrosequence provide platform highly parallel genetic analysis expense associate technique currently high instrumentation runningcost somatic mutation gene relatively rare event inefficient scan mutation expensive approach case provide unnecessary datum issue direct sequencing sequencing array difficulty correspondence address tel fax email Open Access article distribute term Creative Commons Attribution non commercial License permit unrestricted non commercial use distribution reproduction medium provide original work properly detect small fraction mutate allele presence high excess normal allele frequently case clinical cancer sample expensive alternative rapid pre screen method SSCP DGGE dHPLC CCM CDCE hr melting widely utilize identify dna fragment contain mutation prior perform sequencing enzymatic mutation detection base mismatch scan enzyme like MutY TDG endonuclease VII mutation pre screening employ albeit modest success enzyme detect possible mutation deletion substantial activity homoduplex DNA recently enzymatic mutation scan method base Surveyor TM CELI II nuclease combine dHPLC gel electrophoresis detection introduce show satisfactory selectivity reliability mutant wild type allele detectable identify base substitution small deletion important cancer biotechnology plant genetic application tilling method reliable use dHPLC examine Surveyor TM DNA fragment slow endpoint detection method restrict examine single dna fragment time result dna fragment sequence limit analysis cancer specimen numerous sample genetic region need screen introduce new approach enable Surveyor TM scan mutation PCR product simultaneously selectively amplify isolate mutation contain DNA linkermediate PCR selectively amplify mutationcontaine dna wild type fragment present approach de couple enzymatic mutation scan endpoint detection step result follow enzymatic action mismatch choose dna detection method real time PCR gel capillary electrophoresis microarray base detection potentially identify mutate dna fragment simplex multiplex fashion utilize real time PCR couple melt curve analysis Surveyor tmmediate real Time Melting RT MELT validate new technology demonstrate approach increase mutation scan throughput order magnitude sample pre scan mutation enable mutation scan PCR fragment simultaneously mutationpositive sample directly sequence somatic mutation low level mutantto wild type ratio surgical cancer specimen genomic DNA cell line define mutation exon exon exon exon exon extract cell line purchase American Type Culture Collection ATCC purchase purified DNA available surgical colon lung cancer tumor sample obtain Massachusetts General Hospital Tumor Bank follow Internal Review Board approval dna EGFR mutation positive cell line formalin fixedparaffin embed lung cancer sample obtain Lowe Center Thoracic Oncology Dana Farber Cancer Institute follow Internal Review Board approval isolate genomic dna DNeasy TM tissue Kit primer contain clamp Sequences GC clamp portion primer gene specific portion primer investigation list Supplementary Table GC clamp sequence add end forward reverse gene specific primer respectively vice versa microlit PCR reaction perform genomic DNA final concentration reagent follow JumpStart TM buffer Sigma mm dNTP mm forward reverse primer JumpStart TM Taq polymerase Sigma PCR cycling Perkin Elmer PCR machine cycling condition min cycle anneal temperature decrease cycle touch PCR min cycle min PCR program link program denaturation annealing PCR product min microliter PCR product ng mix ml Enhancer TM ml Surveyor TM Transgenomic incubate min follow add ml stop solution manufacturer protocol inactivated Surveyor TM product purify PCR QiaQuick TM purification kit Qiagen elute ml water experiment PCR product mix approximately equal PCR product wild type dna prior form cross hybridized sequence facilitate detection homozygous mutation addition polyA tail follow purification Surveyor TM sample poly adenine tail add dna fragment reaction add ml purify surveyor digest PCR product final volume ml final concentration reaction CoCL mm datp Terminal Transferase New England Biolabs reaction incubate min inactivate heat min real time PCR amplification perform Titanium Taq TM polymerase BD Biosciences Smart Cycler Cepheid real time PCR machine reaction add ml polya tail dna final volume ml final concentration Titanium buffer mm dNTP LCGreen Idaho Technologies mm primer mM oligodT anchor mix GACCACGCGTATCGATGTCG ACTTTTTTTTTTTTTTTTV represent oligodT anchor concentration mm race protocol Titanium TM polymerase Clontech BD Biosciences thermocycle program follow cycle min cycle read fluorescence temperature titration perform different denaturation temperature experimentally determine condition selectively enable mutation contain fragment amplify real time PCR step immediately follow real time differential melting curve analysis SmartCycler TM machine dna melting perform immediately follow PCR Smart Cycler machine sample heat differential fluorescent intensity curve ÀdF dT smooth identification melting peak real time PCR product examine dhplc chromatography WAVE TM system Transgenomic mutation positive PCR product purify PCR purification kit Qiagen sequence primer experiment repeat time independent run genomic OpenArray TM high throughput massively parallel real time PCR platform BioTrove test compatibility RT MELT sample exon PCR product generate different lung adenocarcinoma sample mutation contain cell line process hybridization enzymatic step RT MELT real time PCR OpenArray TM platform perform LightCycler FastStart TM DNA Master SYBR Green TM Roche mm mM oligodT anchor mix primer pre positioned array hole polya tail DNA template cycling condition follow cycle min cycle read fluorescence high sensitivity imaging camera real time PCR step immediately follow real time differential melting curve analysis raw datum export Excel software analysis OpenArray TM experiment repeat twice company headquarters estimate min PCR denaturation temperature PCR efficient assume initial approximation hypochromicity present PCR work give sequence completely denature form immediately temperature lower reaction inhibit primer binding percent melt temperature relation GC clamp contain PCR product Surveyor TM activity generate product estimate POLAND algorithm thermodynamic parameter determine Blake Delcourt mM NaCl solution order force agreement single point predict observe value exon sequence contain short GC clamp normalize shift account influence min NaCl content reaction presence SYBR GREEN LC GREEN dye proprietary composition PCR buffer min PCR product estimate semi empirically determine parameter enriched PCR method Behn et al sequence codon mutation exon sample wild type sample addition second method Amplification Primer Ligation Mutation distinguish mutant wild type sample virtue de novo Nla iii site generate mutant sample codon mutation RT MELT assay convert PCR fragment generate position mutation Surveyor TM enzyme fully amplifiable sequence enable selective PCR amplification subsequent quantitative PCR detection method follow denaturation annealing PCR product lead formation cross hybridized sequence position mutation figure sample expose Surveyor TM endonuclease recognize base pair mismatch small loop high specificity generate break dna strand mismatch result dna fragment participate terminal transferase TdT reaction lead polynucleotide tailing sequential addition adenine poly tail real time PCR reaction subsequently perform adjust condition enable selective amplification mutantonly fragment follow real time melting curve analysis identification mutation presence SYBR GREEN TM LC GREEN TM DNA dye enable selective amplification mutationcontaining fragment real time PCR step modify primer employ original amplification genomic DNA Figure forward primer contain region specific target gene high melting domain GC clamp reverse primer contain region specific target gene tail vice versa follow TdT tailing reaction primer real time PCR conjunction primer bind poly tail denaturation temperature real time PCR reaction lower enable PCR amplification fragment contain GC clamp PCR product escape digestion Surveyor TM contain GC clamps figure fragment amplify efficiently PCR enable selective amplification Surveyor TM fragment effective purification mutation contain fragment subsequent close tube melting curve analysis enable clear separation true mutant sequence PCR dimer artifact RT MELT require size separation identification enzymatically generate fragment sequence scan parallel unknown mutation single tube reaction Surveyor TM simple procedure enable specificity Surveyor TM enzyme combine throughput convenience real time PCR rapid mutation scanning finally amplify mutate sequence contain define primer end direct sequencing enzymatically select PCR product readily possible follow real time melting step enable sequencing low level mutation identify Surveyor TM provide initial proof principle unknown mutation scan RT MELT utilize cell line tumor sample contain sequence identify mutation position exon figure depict dhplc chromatogram product obtain sample contain exon mutation wild type sample standard Surveyor TM approach employ identify mutation follow PCR amplification exon genomic DNA result dhplc trace contain single product wild type product mutation contain sequence figure curve respectively RT MELT screen exon sequence follow PCR amplification Simplex Multiplex PCR amplification exonspcr self hybridize cross hybridize wild type dna generate mismatch position mutation PCR fragment Scan mismatch fragment simultaneously CEL TM enzyme CEL TM enzyme use TdT enzyme add oligonucleotide tail oligo dA end serve primer anchorun digest fragment digest fragmentsTdT tailing DNA end amplify mutate couple real time melt analysis mutation closed tube high throughput melting curve analysis positive sequence amplify mutate dna fragment GC modify primer cross hybridized PCR product expose Surveyor TM TdT tail subsequent real time PCR run different denaturation temperature product examine dhplc real time melting curve analysis standard denaturation temperature mutation contain sample contain peak correspond anticipated amplification Surveyor TM un digest fragment figure curve PCR denaturation temperature lower single PCR product generate mutant sample wild type sample demonstrate product Figure curve Figure real time differential melting curve PCR reaction run depict peak correspond PCR product mutant sample clearly evident absent wildtype sample finally figure depict sequence RT MELT generate PCR fragment direct sequencing genomic DNA mutation evident sample RT MELT product anticipated addition poly tail mutation illustrate examine selectivity RT MELT dilution mutant wild type DNA perform dna cell harbor exon homozygous mutation real time PCR reaction perform mutant wild type ratio distinguish wild type dHPLC figure melt curve analysis Figure sample direct di deoxy sequence identify mutation ratio mutant towild type datum show hand sequence RT MELT product possible include low dilution figure sRT melt sequence generate trace poly tail depict presence position mutation exact nucleotide change clear exon position base mutation confuse mutation reason base ambiguity exact position mutation probably understand PCR perform follow poly tail addition contain equimolar mixture reverse primer end depend exact nucleotide mutation correct primer theory prefer primer allow efficient polymerase extension mismatch practice allele specific PCR step occasionally allow primer extension enable version primer amplify position mutation alternatively incorporation poly tail occur base exact position mutation conclude certain case sRT MELT indicate position mutation base exon indicate position actual nucleotide change exon amplify dna group surgical lung adenocarcinoma sample RT MELT screening unknown mutation melt curve analysis mutation different position exon present clinical sample indicate shift melt profile obtain figure subsequently verify sequence set sample sRT melt sequence detect low level mutation colon cancer speciman direct sequencing fail identify figure Figure sequence sample indicate position poly tail addition base actual nucleotide change difficult identify exclude possibility false positive independent RFLP base method verify presence mutation position poly tail addition know figure codon exon mismatched primer approach Behn et al introduce mlui restriction site wild type sample codon mutant subsequent restriction MluI enzyme follow PCR generate product mutation sample wild type sample Supplementary Figure Frame additional verification low level mutation observe mutation generate de novo Nla iii site position mutation accordingly apply amplification Primer Ligation Mutation method describe previously ligate primer Nla iii digest site preferentially amplify mutant fragment second PCR sequenced PCR product identify mutation Supplementary Figure Frame conclusion sRT MELT identify correctly codon low level mutation miss regular sequencing significant exon mutation codon associate bad prognosis cancer table Supplementary Data depict good agreement standard Surveyor screening RT MELT screening di deoxy sequence low level mutation discover sample RT MELT RT MELT sequence trace sample exon mutation depict datum figure indicate lack substantial PCR amplification denaturation temperature fragment contain GC clamp selective amplification mutation contain fragment different mutation position exon estimate influence GC clamp length PCR efficiency versus temperature PCR amplification fragment generate mutation lie different position sequence calculation base POLAND algorithm perform predict minimum temperature substantial PCR amplification plot versus experimentally observe value possibility simulate GC clamp nucleotide nt GC clamp nt GC clamp dna fragment correspond mutation position exon simulate compare experimentally observe minimum temperature generate PCR product sample contain mutation different position exon result figure indicate agreement theoretical prediction experimental observation denaturation temperature region combination nt GC clamp available mutation exon predict result selective amplification mutation contain fragment inhibition GC clamp contain fragment prediction consistent experimental result obtain PCR temperature titration experiment figure develop calculation algorithm predict appropriate PCR denaturation temperature additional PCR fragment GC clamp combination validation RT MELT utilize method identify mutation additional exon figure depict chromatograph obtain mixture dna cell homozygous mutation exon wild type cell screen real time PCR reaction perform different denaturation temperature product examine dhplc melt curve analysis comparison observe exon denaturation temperature Surveyor TM undigested PCR product amplify real time PCR Figure curve mutant wild type respectively lower denaturation temperature single PCR product obtain mutant product primer dimer obtain wild type sample figure curve figure depict melting curve obtain follow real time PCR denaturation temperature mutant wild type sample RT MELT subsequently apply manner screen mutation exon cell line surgical colon sample harbor sequence identify mutation include single base frameshift mutation exon list Supplementary Table melting curve mutant wild type sample exon depict figure datum indicate result similar obtain exon obtain exon mutation EGFR exon particular clinical interest alteration modulate response EGFR inhibitor lung adenocarcinoma patient figure depict application RT MELT screen dna lung cancer cell line harbor dHPLC identify alteration EGFR exon include codon deletion del exon ability RT MELT detect low level EGFR mutation evaluate perform dna dilution heterozygous EGFR exon homozygous sample mutant wildtype ratio detectable dilution experiment Figure finally application RT MELT detect mutation dna formalin fix paraffinembedde FFPE sample examine screen clinical FFPE lung adenocarcinoma specimen sample know harbor EGFR exon mutation negative mutation independently evaluate dhplc figure demonstrate identification mutational status sample PCR RT MELT OpenArray TM RT MELT increase throughput mutation scan significant potential advantage enzymatic mutation scan ability screen sequence simultaneously mutation demonstrate RT MELT parallel scanning mutation PCR product mix equimolar amount PCR product exon contain mutation exon exon form cross hybridized sequence screen mixture mutation exon single tube RT MELT depict figure follow real time PCR melt curve analysis exon exon mutant clearly distinguish wild type sample figure curve mutant exon dna sample dilute fold wild type exon equimolar mixture exon prepare screen single tube RT MELT exon mutation distinguish wild type mixture exon figure curve mutation human tumor encounter exon multiplex single tube RT MELT reaction identify mutation encounter clinical tumor sample combine multiplex PCR directly genomic DNA approach result convenient high throughput method mutation scan adopt real time PCR platform endpoint detection RT MELT throughput mutation scan increase drastically mutation prescreene approach utilize dHPLC capillary gel electrophoresis demonstrate well point highly parallel nano technology platform adopt real time PCR step RT MELT enable array nl volume real time PCR reaction OpenArray TM system carry simultaneously follow differential melting curve analysis proof principle compatibility RT MELT OpenArray TM exon PCR product generate different lung adenocarcinoma sample mutation contain cell line process hybridization enzymatic step RT MELT sample dispense replicate nano liter volume reaction OpenArray TM plate pre fabricate contain appropriate primer amplify real time PCR reaction denaturation temperature presence SYBR GREEN dye melt curve subsequently obtain OpenArray TM melt curve analysis mode PCR growth curve smoothed differential melting curve obtain distinguish clearly mutation contain sample wild type sample figure representative result reaction furthermore identification mutation contain sample good agreement conventional nano technology platform figure versus figure datum indicate RT MELT compatible high throughput nano technology detection format reiterate advantage de couple enzymatic selection detection step Comparison throughput conventional pre screen method dhplc dHPLC Surveyor TM RT MELT Table indicate RT MELT order magnitude fast large number sample screen mutation multiplex RT MELT format adopt throughput increase intrinsic potential enzymatic mutation scan parallel identification mutation principle high enzyme operate numerous distinct mismatch contain sequence molecule tomolecule basis provide highly parallel mutation scan past selectivity enzyme endpoint detection method limit realization potential enable Surveyor TM endonuclease recognize selectively mismatch form mutation small deletion follow cross hybridized sequence formation generate mutation specific dna fragment amplify screen differential melting curve analysis replacement size separation method capillary gel electrophoresis dhplc real time PCR technology endpoint detection platform ability scan sequence parallel result highly increase throughput RT MELT retain ability detect diverse mutation low level Cel II endonuclease know exonuclease activity dna end reason RT MELT design attach oligonucleotide linker end terminal transferase TdT instead end exonuclease activity tend degrade attached clamp RT MELT eliminate influence reduce amplification un digested fragment find exposure dna cross hybridized sequence Surveyor TM limit min substantial degradation clamp avoid multiplexe mutation detection PCR product simultaneously size GC clamp PCR amplicon need individually adjust ensure mutation sequence position PCR product include mixture screen single real time PCR temperature undigested fragment amplify calculational tool develop work guide individual design GC clamp RT MELT detect heterozygous snp mutation mutation prescreene technique presence SNP concurrently mutation difficult identify perform sequencing snp occur fix position melt peak originate snp reproducible pattern melting temperature case distinguishable mutation finally noteworthy RT MELT general methodology apply isolate mutation mismatch cut enzyme Surveyor TM enzyme satisfactory property mutation detection available detection platform real time PCR melting dna microarraybase envision follow enzymatic mutation selection summary develop new method rapid mutation scanning RT MELT utilize Cel II Surveyor TM terminal deoxy nucleotide transferase TdT enzyme isolate amplify mutation contain dna fragment requirement dna sizedependent technique enable highly increase throughput multiplexe mutation screening direct sequencing identify mutant dna fragment RT MELT retain advantage Surveyor endonuclease alternative pre screen method reliability identification genetic alteration present low fraction sample RT MELT provide significant advancement unknown mutation scan cancer research diagnostic general medical biological biotechnology application','2007-06-01 00:00:00','http://dx.doi.org/10.1093/nar/gkm403',2936,777),('2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json','Transcript-level annotation of Affymetrix probesets improves the interpretation of gene expression data','BACKGROUND: The wide use of Affymetrix microarray in broadened fields of biological research has made the probeset annotation an important issue. Standard Affymetrix probeset annotation is at gene level, i.e. a probeset is precisely linked to a gene, and probeset intensity is interpreted as gene expression. The increased knowledge that one gene may have multiple transcript variants clearly brings up the necessity of updating this gene-level annotation to a refined transcript-level. RESULTS: Through performing rigorous alignments of the Affymetrix probe sequences against a comprehensive pool of currently available transcript sequences, and further linking the probesets to the International Protein Index, we generated transcript-level or protein-level annotation tables for two popular Affymetrix expression arrays, Mouse Genome 430A 2.0 Array and Human Genome U133A Array. Application of our new annotations in re-examining existing expression data sets shows increased expression consistency among synonymous probesets and strengthened expression correlation between interacting proteins. CONCLUSION: By refining the standard Affymetrix annotation of microarray probesets from the gene level to the transcript level and protein level, one can achieve a more reliable interpretation of their experimental data, which may lead to discovery of more profound regulatory mechanism.',' Microarray technology invent rapidly profile quantity mrna transcript particular cellular context application universal biomedical research mrna actually detect microarray experiment mrna direct relationship protein methodology algorithm datum analysis commonly gene base evident probe annotation file provide Affymetrix gene level annotation widely accept fail discriminate multiple mrnas transcribe gene result analysis result usually summarize gene level differentially express gene gene set recent work integrate protein protein interaction datum microarray datum analysis unit reduce gene instead mRNA practice attribute fact functional knowledge gene level instead transcript level functional difference mrna variant transcribe gene seldom clear gene level analysis ignore difference mrnas variant strategy biologically meaningful consider diversity gene high transcript encode gene establish alternative splicing increase mrna diversity human gene involve mechanism addition RNA processing event RNA editing account increase diversity mrna level event enable gene encode multiple protein functionally heterogeneous feel necessary separate transcript level synonymous probeset gene level synonymous one probeset hybridize transcript variant gene refer gene level synonymous probeset one hybridize single variant name transcript level synonymous probeset notice transcript level synonymous probeset tend similar expression profile gene level synonymous probeset distinct expression profile fact individual report demonstrate expression transcript variant totally different phenomenon indicate gene level strategy microarray datum analysis imprecise overlook expressional inconsistency gene level synonymous probeset matter fact Affymetrix suffix probeset ID accord probeset specificity example probeset recognize unique transcript variant suffix probeset recognize multiple alternative transcript single gene suffix suffix system give hint varied specificity probeset consider endeavor try away customer worry gene level data analysis strategy correctness suffix system doubt reliable perform transcriptlevel analysis basis imperfect suffix system standard annotation file analysis algorithm gene orient analyst average expression heterogeneity gene deal probeset level datum paper link probeset widely Affymetrix array International Protein Indexes IPIs proper association rigorous alignment procedure demonstrate statistically significant advantage interpret microarray datum transcript level protein level result view precise annotation Affymetrix array probeset achieve reliable interpretation experimental datum application new annotation substantially increase expression correlation interact protein Affymetrix array HG choose study putative protein code probeset array subject alignment investigation table candidate probeset mrna relationship compile base probeset gene mapping information specify standard Affymetrix annotation file gene mrna mapping relationship provide separate mrna transcript source rigorous blast procedure similar describe perform filter candidate probeset mrna relationship finally mrna target pass filter criterion link protein id IPI database rigorous association alignment obtain precise annotation probeset Affymetrix array hg respectively additional file annotation discriminate alternative mrna variant transcribe gene transcript level oppose standard gene level annotation file provide Affymetrix worth note transcript datum redundant conceptual transcript variant represent multiple redundant transcript accession transcript database transcriptlevel annotation file conceptual transcript identify IPI ID investigate probeset associate protein code transcript statistic non control investigate annotate probeset number involve gene protein show Table evident proportion gene cover annotate probeset cover non control probeset gene retain percentage Table HG high correspond probeset retain percentage indicate gene coverage slightly reduce filtering procedure observation support primary goal refine gene level probeset annotation transcript level simply remove poor quality gene level annotation furthermore classify probeset base way link protein check alignment result probeset annotation table divide group annotation table probeset link single protein probeset link multiple protein alternative splicing gene rest investigate probeset link protein categorize group null statistic group show Table know Affymetrix suffix probeset ID accord probeset specificity type probeset suffix categorize group transcript specific gene specific suboptimal probeset cross hybridize unrelated sequence group comparable probeset respectively find total investigate probeset hg respectively table type account significantly low probeset respective array suggest probeset specific transcript clarify issue probeset specificity group investigate probeset varied Affymetrix suffix categorize system table evident overall tomany majority group group respectively disagreement classification example Affymetrix group probeset null probeset suggest call transcript specific probeset expect high specificity treat caution datum analysis hand Affymetrix group contain probeset attribute mainly trimming table probeset annotation table annotate probeset probeset investigate investigate probeset null probeset investigated probeset originally false transcript update sequence information lead absence transcript target probeset similarly large number probeset socalled Affy group HG probeset mapping identify transcript target strategy base premise probeset specific certain gene blast limit candidate transcript associate probeset entire transcript pool subgroup investigate probeset different Affymetrix suffix indicate imperfection Affymetrix suffix system affirm necessity transcript level protein level probeset annotation fact research group address misleading nature Affymetrix suffix system imperfection standard annotation file include annotation work array hg discuss relate work detail section probeset link transcript protein rigorous association alignment procedure expression profile transcript level synonymous probeset suppose consistent gene level synonymous probeset take basis evaluation annotation download expression dataset assay Gene Expression Omnibus database GEO calculate Pearson correlation coefficient PCCs expression profile synonymous probeset gene level transcript level transcript level synonymous probeset work recognize single protein code transcript variant identify unique IPI ID transcript level synonymous probeset name protein level synonymous one practically group gene level synonymous probeset accord gene ID protein level one accord IPI ID probeset protein mapping table provide NetAffx official protein level annotation Affymetrix probeset comparison probeset correspond single protein accord NetAffx designate Affy protein level synonymous probeset level synonymy synonymous group pair wise pcc calculate summarize value indicate expression correlation group expression correlation synonymous group level average overall value dataset Method section detail correlation synonymous group dataset different level depict parallel figure noticeable synonymous probeset Affy protein level high correlation gene level synonymous one base protein level annotation high correlation consistently dataset dataset student test additional file detail comparison prove rationality necessity protein level probeset annotation result support argument reliable interpret microarray datum transcript level gene level inferiority Affy protein level annotation annotation attribute technical detail alignment association procedure representative mrna sequence consensus exemplar sequence probeset instead probe sequence query sequence alignment second align GenBank non redundant protein database comprehensive pool mrna sequence microarray experiment hybridization take place probe sequence immobilize array cDNA sequence sample deduce alignment representative mrna sequence protein sequence precisely simulate hybridization probe mrnas finally NetAffx filter blast result accord value indicate likelihood observed alignment chance frequently adopt sequence homology analysis closely relate specie value sensitive grade align target specie practice take probe sequence query mrna sequence alignment target adopt matching nucleotide proportion filtering criterion similar comparison conduct HG array involve transcript level annotation Harbig et al dataset annotation Harbig et al show advantage Affy protein level annotation transcript level annotation perform good dataset Student test figure additional file work year later Harbig et al update mrna sequence probe mrna alignment factor contribute increase performance contribute factor different approach identify mapping probe mrna target Harbig et al perform phase blast blast probeset target sequence mrna sequence pool blast probe sequence retrieve mrna sequence efficient phase blast strategy reduce alignment precision compare direct probe mrna blast strategy Harbig et al accept mrna high average probe match target probeset high value suboptimal accord datum puzzling fact probeset gene variable expression profile clarify view datum protein level gene likely involve alternative transcript variant constitutively different expression level look gene example show Figure probeset dataset map mouse Prlr gene GeneID probeset correlate protein PRL correlate protein PRL Prlr gene report encode seven isoform prolactin receptor precursor mouse probeset correspond protein show similar expression profile expect pcc value probeset correspond isoform show significantly different expression profile pcc separately Figure datum consistent previous report isoform predominantly express liver kidney PRL highly express PRL weakly express notice shape expression curve PRL somewhat comparable PRL PRL probeset present overall high expression level phenomenon suggest PRL PRL partially co regulated comparable expression pattern express different level probably diverse role regulatory element show Figure probeset dataset map mouse gene GeneID correspond correspond protein indicate IPI homolog yeast asparagine link glycosylation information notice interesting phenomenon case probeset correlate protein similar expression profile instead behave like probeset correlate different variant characterize pcc value issue factor cause microarray hybridization efficiency shift secondly accord calculation expression variant negatively correlate PCC value far report expression regulation transcript variant gene function report correspond protein isoform negative correlation suggest protein perform different role distinctly annotate recent year considerable effort devote identify characterize protein protein interaction PPI investigation molecular event involve PPI functional annotation unclassified protein accord interact partner important topic bold infer protein function accord majority rule utilize PPI network structure study integrate data source functional characterization PPI gene expression datum favorite work assume interact protein pair characterize high expression correlation random one previous investigation indicate relationship expression correlation PPI weak genomic scale recent work strengthen association integrate cross species conservation information notice genome scale study PPI information convert gene pair Pearson correlation probeset pair correspond gene pair evaluate analysis target expand real interact protein pair possible cross gene protein pair interaction exist illustrate figure suppose gene abbreviate Ga gene Gb Ga encode protein protein Gb encode protein protein protein variant protein interact possible cross gene interaction include happen probeset recognize transcript variant respectively produce protein conventional genome scale study mention probeset pair correspond real interact protein pair cross gene pair include blunt expression cor expression profile synonymous probeset Figure Expression profile synonymous probeset probeset array associate mouse gene Prlr probeset correlate protein PRL probeset point protein PRL GEO dataset analyze probeset array associate mouse gene probeset point protein probeset correlate protein GEO dataset real interact entity accord precede observation propose partly explain weak coherency PPI expression correlation order validate proposition investigate relationship protein interaction expression correlation gene level protein level perspective interact protein pair HPRD microarray dataset assay HG GEO depict figure probeset pair correspond possible cross gene protein pair term GGI pair probeset pair correspond real PPI PPI pair calculate pcc PPI pair GGI pair available interaction evaluate statistical significance PCC value tail test estimate correspond false discovery rate FDR SPLOSH FDR estimation method dataset observe strengthen expression correlation interact protein PPI schema relative GGI schema take dataset include array example figure respectively significantly small Kolmogorov Smirnov test value summarize comparison positive negative conclude expression correlation interact protein strengthen non interact protein pair exclude interact gene pair PPI PCC calculation place conventional GGI PCC calculation Figure notice negative correlation strengthen PPI PCC calculation positive correlation PPI pair positively correlate negatively correlate exclusively co express previous publication report phenomenon evident examine PCC value couple PPI pair GGI pair figure show scatter plot PPI PCC value versus correspond GGI PCC value evident point fall quadrant indicate pair PPI PCC value GGI PCC value tend sign scatter plot suggest linear relationship PPI pcc GGI pcc linear regression formula high confidence estimated coefficient far conclude absolute PCC value PPI pair large correspond GGI pair PPI PCC calculation preserve original positive negative correlation tendency reveal conventional GGI PCC calculation strengthen large absolute correlation value correlation tendency interact protein especially negative one likely submerge figure Illustration Entities terminology involve protein protein interaction relationship gene transcript protein probeset type pair mention text PPI pair indicate probeset pair correlate real interact protein pair GGI pair probeset pair correlate possible cross gene pair PPI exist cross gene correlation random pair non interact protein pair include analysis similar observation dataset additional file additional file give result dataset able compare PPI PCC calculation GGI PCC calculation high level FDR threshold list significant PPI pair GGI pair dataset determine base list count significantly correlate ppi GGIs term significant ppi significant ggi dataset dataset PPI GGI demonstrate significant correlation term ppisignificant dataset ggi significant dataset table show Table table show fully additional file partly Table summary table find statistic PPI PCC calculation large counterpart GGI PCC calculation indicate PPI PCC calculation detect correlated PPI pair certain experiment setting table detect correlation tendency experiment setting Table experiment implement PPI pair extract IntAct database protein pair false discovery rate pcc value deem significantly correlate pair significant ppi pair detect PPI calculation Significant ggi detect GGI calculation ilar conclusion obtain detail find additional file additional file work annotate probeset widely Affymetrix array hg proper association rigorous alignment procedure transcript perspective demonstrate necessity advantage explore microarray datum transcript protein level instead conventional gene level Affymetrix utilize complete information available time array design tremendous progress genome sequencing annotation recent year render exist probeset design target identification suboptimal recent year continuous report systematic false expression signal Affymetrix probeset spurious expression correlation cause cross hybridization expressional inconsistency different microarray platform different generation platform research group perform probe mrna blast similar center UniGene improve accuracy annotation gene level major trend effort redefine probeset probe match molecular target place custom probeset propose author point transcript target probeset transcript specific probeset target transcript gene share probe probe organization strategy distinction level gene level transcript splice variant strategy take probe level intensity file CEL file prerequisite half expression dataset deposit public database like GEO find CEL file order distinction precisely transcript level preserve classical Affymetrix probeset distinguish alternatively splice transcript target accord consistent alignment probe date mrna sequence annotation table clearly divide Affymetrix probeset group increase transcript level specificity reliability null probeset recognize transcript probeset hybridize multiple alternative transcript variant intended gene probeset hybridize unique alternative transcript variant intended gene PPI significant dataset ggi significant dataset PCNA PBK PPI significant dataset dataset false discovery rate pcc value PPI pair GGI significant dataset dataset false discovery rate pcc value GGI pair PPI pair PPI significant dataset dataset show table provide additional file intend alternative transcript variant Affymetrix probeset base NetAffx gene level annotation time give fact exist solution accompany imperfection consensus reach overwhelming strategy alternative solution problematic standard annotation point new way improve interpretation exploitation Affymetrix microarray datum transcript collection identical reannotation strategy differ similar discovery previous report example Harbig et al find number probeset detect transcript attribute phenomenon elimination target sequence process sequence update study altogether non control probeset hg array find transcript target pool GenBank RefSeq Ensembl new large collection transcript sequence obtain similar statistic percentage multiple target probeset estimate foregoing study specifically hg Table significant mutual agreement relate research justify necessity set improved annotation mechanism Affymetrix probe face continual growth genomic transcriptomic knowledge ideally transcript protein level past year analysis alternative splicing emerge important new field bioinformatic recent large scale study show alternative splicing analyze high throughput manner dna microarray method study array particularly manufacture analyze alternative splicing genomic tile array exon exon junction array construct priori knowledge possible exon content genomic sequence genomic tile array principle capable detect novel alternative splicing event diverse type doubt datum readily interpretable successful experience remain insufficient hand design particularly address alternative splicing issue exon exon exon junction array report plague problematic probe specificity unsatisfying hybridization efficiency necessity probe coverage length gene include end question reproducibility amplification protocol quantitative accuracy data analysis need address prerequisite reliable quantitative analysis splicing array give current imperfection splicing array technique inconvenience deci phere generate datum economic way large scale investigation alternative transcribing event standard gene expression array provide recognize target probe rigorously define transcript level Hu et al propose primitive analysis method explore alternative splicing Affymetrix gene expression array regret alternative splicing bias gene detect way present paper conduct large scale alignment probe sequence traditional gene expression array currently comprehensive collection transcript sequence highlight probeset mapping unique alternative transcript unambiguously Affymetrix expression array test study find probeset map transcript oneto manner work strongly validate feasible analyze alternative splicing traditional gene expression array foregoing work contribute Hu et al remain qualitative analysis method aim detect novel alternative splicing event work give explicitly relationship probeset currently know alternative transcript variant immediately exploit facilitate quantitative analysis alternative variant mapping relationship define standard probeset traditional gene expression array conveniently exploit standard NetAffx annotation information ad hoc influence widely apply experiment protocol routine datum processing algorithm demonstrative implementation novel annotation table actually observe example negatively correlate alternative variant figure shed light study expression regulation alternative transcript variant sum annotate popular Affymetrix gene expression array HG transcript level perspective aim identify probeset detect target precisely transcript level previous work address similar issue rigorously link exist Affymetrix probeset specific transcript target corresponding protein arm new annotation examine number previous study dataset dataset HG GEO reveal increase expression consistency synonymous probeset close expression correlation interact protein transcript level annotation Affymetrix probeset allow reliable gene expression data analysis accessible protein level correlation study Affymetrix eukaryotic gene expression analysis array HG select study probe sequence file corresponding annotation file annotate annotate download Affymetrix website download NetAffx probesetprotein mapping file file update HG update blast result representative mrna sequence probe protein sequence database ensembl transcript mouse transcript sequence obtain Ensembl database release mapping table Ensembl Gene ID Ensembl Transcript ID Ensembl Peptide ID obtain Ensembl martview entry mapping external protein accession number acquire International Protein Index IPI database release obtain mapping relation IPI number transcript id GenBank RefSeq Ensembl counterpart file human download release dataset download Gene Expression Ominibus April Array index associate sample GEO rank second register Affymetrix mouse array GDS dataset profile analysis contain sample Array HG index associate sample GEO rank register Affymetrix human array analysis download GDS dataset large sample size analysis exclude data format inconsistent detail dataset find additional file total probeset array HG respectively control probeset firstly remove remain probeset associate gene accord probeset gene mapping information provide Affymetrix standard annotation file probeset transcript mapping relationship obtain base gene mrna mapping table GeneBank RefSeq Ensembl process include probeset identify Entrez Gene ID Ensembl gene ID ignore probeset associate multiple entity entity gene centric database gene level specificity appear doubtful standard annotation file filter non control probeset HG respectively rest probeset link candidate transcript target correspond protein entry IPI database IPI currently redundant complete protein database featured specie protein sequence match transcript variant probeset transcript tar get protein counterpart exclude follow blast validation order focus attention transcript characterize function protein level remain probeset HG select blast procedure filter candidate probeset mrna mapping relationship align probe sequence probeset correspond transcript probe blast candidate mrna target program probe transcript match accept mismatch find level probeset probeset transcript match accept probe probeset probe typical probeset map transcript orientation probeset transcript protein link relate probeset pass blast filter retrieve reduce redundancy information multiple transcript correspond IPI finally obtain rigorous probeset annotation file Affymetrix array HG type probeset new annotation file probeset probeset map IPI ID probeset probeset map IPI id probeset subsequent analysis group gene level synonymous probeset accord gene ID gene level protein level one accord IPI ID protein level additionally probeset correspond single protein accord NetAffx probeset protein mapping table Materials group Affy protein level synonymous probeset case hg array include fourth level Harbig protein level comparison Harbig protein level reflect probeset protein association transform recent alternative annotation Affymetrix plus array propose transcript level perspective downloaded annotation file map probeset GenBank mrna accession find HG array map GenBank mrna accession probeset mrna association relate HG array link IPI id finally give rise probeset IPI mapping relationship follow treatment implement standard gene level annotation NetAffx Protein annotation novel annotation expression dataset select GEO respectively hg original intensity datum GEO dataset GDS transform log base normalize constant median sample synonymous group expression value probeset sample large constant median value probeset group regard moderately express expression profile informative keep synonymous group expression value constant median value similar filter procedure Tian et al remain synonymous group Pearson correlation coefficient PCCs calculate expression profile probeset pair minimum value pcc take measurement expression coherence group minimum aggregation gene level synonymous probeset give rise protein pcc theoretically high protein pcc theoretically low identical result protein level synonymous group setting maximum result difference average aggregation sensitive minimum term differentiate group adopt minimum aggregation finally mean expression coherence synonymous group dataset calculate way obtain evaluation expression consistency synonymous probeset microarray dataset compare expression consistency level different microarray dataset give protein protein interaction datum HPRD IntAct transform binary relation protein accession IPI IPI pair get correspond Gene Gene pair PPI assemble PPI probeset pair GGI probeset pair illustrate figure PPI pair associate interact IPI id GGI pair associate correspond Gene id probeset pair associate IPI IPI pair PPI pair associate correspond gene gene pair GGI pair pcc calculate average PPI PCC GGI PCC respectively pcc interact pair calculate obtain accompany false discovery rate SPLOSH FDR estimation method distribution PPI pcc GGI pcc plot figure contrast Figure addition background distribution pcc ran dom probeset pair overlaid figure let number random pair equal number PPI GGI pair repeat process calculate random pcc distribution time average separate random distribution order cut random fluctuation run calculate random pcc distribution randomly compile HPRD IntAct respectively pair probeset probeset pair form group pcc group firstly average group level PCC distribution calculate group level pcc group level averaging devise mimic counterpart operation PPI PCC GGI PCC calculation','2007-06-11 00:00:00','http://dx.doi.org/10.1186/1471-2105-8-194',3405,804),('305cf1019ca9c81126b2e527fedf79969c2c522f.json','ElaD, a Deubiquitinating Protease Expressed by E. coli','BACKGROUND: Ubiquitin and ubiquitin-like proteins (Ubl) are designed to modify polypeptides in eukaryotes. Covalent binding of ubiquitin or Ubls to substrate proteins can be reversed by specific hydrolases. One particular set of cysteine proteases, the CE clan, which targets ubiquitin and Ubls, has homologs in eukaryotes, prokaryotes, and viruses. FINDINGS: We have cloned and analyzed the E. coli protein elaD, which is distantly related to eukaryotic CE clan members of the ULP/SENP protease family that are specific for SUMO and Nedd8. Previously misannotated as a putative sulfatase/phosphatase, elaD is an efficient and specific deubiquitinating enzyme in vitro. Interestingly, elaD is present in all intestinal pathogenic E. coli strains, but conspicuously absent from extraintestinal pathogenic strains (ExPECs). Further homologs of this protease can be found in Acanthamoeba Polyphaga Mimivirus, and in Alpha-, Beta-and Gammaproteobacteria. CONCLUSION: The expression of ULP/SENP-related hydrolases in bacteria therefore extends to plant pathogens and medically relevant strains of Escherichia coli, Legionella pneumophila, Rickettsiae, Chlamydiae, and Salmonellae, in which the elaD ortholog sseL has recently been identified as a virulence factor with deubiquitinating activity. As a counterpoint, our phylogenetic and functional examination reveals that ancient eukaryotic ULP/SENP proteases also have the potential of ubiquitin-specific hydrolysis, suggesting an early common origin of this peptidase clan.',' ubiquitin Ubls SUMO protein exclusively express eukaryote ubiquitination control cellular process include degradation protein proteasome intracellular trafficking ubiquitin like modifier regulate rate extent ubiquitination involve regulation transcription factor nuclear import attachment ubiquitin ubiquitin like modifier substrate protein covalent reversible large family eukaryotic cysteine protease involve generation protein precursor removal modified substrate pathogen tamper ubiquitin proteasome system cripple cell defense instance ubiquitination proteasomal degradation initiate Human Papillomavirus protein stabilization Ikb Yersinia deubiquitinase describe continue discovery new deubiquitinate protease virus broadly hint important pathogen seize control posttranslational modification host cell focus cysteine protease CE clan define MEROPS database CE peptidase express virus bacteria eukaryote eukaryote clan represent family Ubl specific protease ULP SENP remove SUMO substrate protein viral homolog ULP senp act deubiquitinase cleave unrelated protein long glycine motif present terminus substrate example bacterial deubiquitinase include yopj chladubs yopj protease secrete host cell pestis homolog peptidase find bacteria injection yopj eventually suppress inflammatory response affected cell precise molecular mechanism yopj family peptidase solve lack hallmark tryptophan follow active site histidine vitro vivo datum strongly suggest protein protease specificity ubiquitin SUMO effectiveness virulence factor depend additional function acetylation second example bacterial CE peptidase find pathogenic Chlamydiae show pathogenic Chlamydiae nonpathogenic environmental strain express protease specifically recognize ubiquitin presumably remove modifier target protein host cell Chlamydiaelike bacteria possess ubiquitin homolog finding CE peptidase bacteria consider functional similarity eukaryotic ULP SENP protease explore additional bacterial homolog deubiquitinate activity exist search sequence genome bacteria new member CE protease clan employ PHI BLAST find protein typical catalytic triad histidine aspartate glutamate asparagine active site cysteine candidate protein subject second round analysis exclude protein hallmark oxyanionstabilize group consist glutamine asparagine close active site cysteine lastly predict secondary structure candidate protease compare solve structure eukaryotic CE clan homolog apart YopJ ULP SENP homolog Chlamydiae find potential CE peptidase Gammaproteobacteria figure bacteria branch find peptidase homolog giant Acanthamoeba Polyphaga Mimivirus African Swine Fever Virus virus ubiquitin conjugating enzyme discover deubiquitinate activity suggest bacteria identify share close symbiotic pathogenic relationship eukaryote virus CE protease possess dsdna genome sequence variation peptidase extensive allow significant bootstrap value possible faithfully infer phylogeny dataset true eukaryotic ULP senp fall functional class specific SUMO group group good reciprocal blast hit consensus phylogram tree tend correctly predict functional class particular ULP SENP homolog belong bootstrap support generally significant especially interested protein elad belong group II figure express coli ortholog Legionella pneumophila currently sequence strain Salmonella figure coli abundant commensal human gut relevant pathogen nonetheless little know gene define pathogenicity coli strain apart encode obvious toxin furthermore choose elad represent distantly relate hit bioinformatic screen aim test robustness prediction examine protein function figure Phylogram representation CE clan protease virus bacteria eukaryote eukaryotic peptidase blue belong subfamily separate group include mammalian protease include group propose specificity SUMO group group respectively bacterial protein indicate precede viral protein divide microbial protease homolog color green biochemically test protease red indicate absence publish datum function putative protease yellow group represent elad ortholog family contain Adenovirus protease deubiquitinate activity comprise bacterial yopj homolog Vacciniavirus peptidase base sequence similarity bacterial family group distinguish group Proteobacteria locate appear closely relate fungal homolog common node indicate circle bootstrap Chlamydiae previously show presence deubiquitinate deneddylating activity additional group assign specific CE clan subfamily MEROPS database include Mimivirus group Gammaproteobacteria group II Rickettsiae group III african Swine Fever Virus protease Vacciniavirus protease test deubiquitinate ublspecific activity require glycine base motif terminus substrate find ubiquitin Ubls unrelated cd clan peptidase Clostripain outgroup phylogram clarity tree contain ortholog paralog different group family sequence information provide Table goal experiment confirm protease activity determine substrate specificity relationship ULP senp hypothesize elad recognize Ubls ubiquitin express elad vitro transcription translation rabbit reticulocyte lysate incubate metabolically label polypeptide electrophilic probe Michael acceptor add terminus ubiquitin Ubls show figure elad readily form covalent adduct ubiquitin probe less extent probe detectably mutate putative active site cysteine position serine covalent binding electrophile abolish indicate cysteine residue elad essential catalytic activity observe yopj Chlamydia protease eukaryotic ULP SENPs date specific labeling putative protease activity base probe show excellent correlation enzymatic activity assess enzyme kinetic measure hydrolysis fluorogenic substrate derive ubiquitin experiment express elad coli growth rate bacteria unaffected overexpresse elad recover wildtype protein compare amount mutant Figure demonstrate release fluorophore methylcoumarin AMC elad cleave ubiquitin AMC AMC AMC mutant elad fail cleave substrate figure datum show initial rate hydrolysis nM ubiquitin AMC nM elad order minute define elad moderately active deubiquitinase compare rapid Isopeptidase rate minute slow ubiquitinprotease rate minute datum show note assess max enzymatic rate increase linearly substrate concentration test mM ubiquitin AMC nm elad hydrolyze ubiquitin amc initial rate minute similar observation SARS virus deubiquitinase overall functional analysis confirm prediction bioinformatic screen define elad deubiquitinate protease eukaryotic CE peptidase specificity distinct prokaryotic counterpart indicate profound discrepancy hint separate origin protease group set challenge presently hold notion SENP protease exhibit ubiquitin specific activity test bacterial homolog apparently end choose biochemically define CE clan member deeply branched class eukaryote Pezizomycotina subgroup fungus include fumigatus nidulan grisea crassa zeae encode putative protease relate amino acid sequence group uncharacterized bacterial homolog figure phylogram common node respective prokaryote group homolog Pezizomycotina replicate bootstrap support clone express test putative protease zeae representative Pezizomycotina unlike mammalian zeae ortholog display dual activity ubiquitin similar previously define CE clan protease figure sequence comparison homolog human pathogenic bacteria multiple sequence alignment catalytic core region human NCBI protein sequence identifi GI show residue homolog trachomatis GI residue coli elad GI residue pneumophila GI residue typhi ssel GI residue arrow indicate active site histidine aspartate asparagine catalytic cysteine oxyanion stabilize group predict secondary structure show confirm solve structure Figure biochemical assay substrate specificity methionine label vitro translate wildtype elad form covalent adduct suicide inhibitor base ubiquitin ubiquitin vinylmethylester VME vinylsulfone VS probe base probe test activity bona fide substrate show mutation active site cysteine position serine abolishe adduct formation elad electrophilic probe sample resolve reduce SDS PAGE visualize fluorography indicate right molecular mass kDa prokaryote trachomatis figure result indicate ubiquitin specificity restrict bacterial viral CE peptidase exist ancient eukaryotic member protease clan find previously undescribe CE peptidase homolog bacterial viral specie figure furthermore prove distantly relate homolog protein elad coli act deubiquitinate enzyme vitro early show relate Chlamydia protease deubiquitinate activity similar observation YopJ protease family suggest undefined bacterial CE peptidase display ubiquitin ubl specificity especially consider close sequence relationship eukaryotic ULP SENP protease finding raise important question protease specific ubiquitin vivo simply measure offtarget artifact true substrate merely resemble ubiquitin second protease present eukaryote widely distribute bacteria virus learn genetic argue bacterial CE clan peptidase distinct specificity bacterial substrate instance propose origin ubiquitin system predate split eukaryote prokaryote factor distantly resemble Ubls conjugating deconjugate enzyme find bacteria manner cleavage ubiquitin elad artificial byproduct assay true substrate bacterial origin similarly Adenovirus CE protease relax specificity consensus site present ubiquitin certain viral substrate extend analysis elad specificity ubiquitin homolog share significant sequence similarity ubiquitin identical critical terminal region observe reactivity elad vinylsulfone binding elad significantly weak ubiquitin probe furthermore detect hydrolysis terminal peptide bond ubiquitin amc AMC feature clearly distinguish elad promiscuous viral CE peptidase give similarity primary secondary tertiary structure Ubls conclude hydrolysis ubiquitin elad reflect highly specific interaction exception avenae bacterial strain dataset encode homolog ubiquitin make likely substrate elad eukaryotic ubiquitin ortholog elad Salmonella ssel recently show virulence factor display deubiquitinate activity vitro vivo enzyme encode currently sequence Salmonellae present pseudogene Shigellae likewise elad essential coli laboratory condition comparison genome sequence coli strain reveal elad present commensal coli strain intestinal pathogenic strain EAEC EHEC EIEC EPEC ETEC absent ExPEC strain APEC NMEC UPEC table answer second question raise fact CE protease signature motif catalytic domain substrate specificity ubiquitin Ubls relate product suggest common genetic functional origin protease viral bacterial organism express CE protease share intimate relationship eukaryote commensal symbiont pathogen additionally lateral gene transfer propose eukaryote dsDNA virus eukaryote bacteria Chlamydiae Rickettsiae pneumophila notion dynamic horizontal gene transfer involve deubiquitinase underscore distribution ubiquitin specific protease Chlamydiae pathogenic strain express CE peptidase pneumoniae instead encode homolog eukaryotic otubain type deubiquitinating enzyme group bacterial peptidase particular cluster close homolog Pezizomycotina figure sequence relationship share habitat organism plant symbiont phytopathogen environmental fungus raise possibility gene transfer functional perspective homolog Pezizomycotina exert ubiquitin specific hydrolase activity like previously characterize bacterial CE protease figure dual specificity fungus zeae ubiquitin trivial arguably close relative ubiquitin sequence difference require distinct conjugating machinery deconjugate protease particular position arginine ubiquitin alanine act compatibility switch single replacement chain cause ubiquitin mimic vice versa respect recognition ubiquitin homolog zeae contrast observe mammalian possible explanation observation CE protease originally derive deubiquitinase protease specific conserve eukaryotic protein CE peptidase eukaryote structurally diversified accommodate evolve family Ubls protease counterpart bacteria virus deeply branched eukaryote retain specificity ur substrate ubiquitin publish literature datum support notion clan CE protease acquire bacteria virus horizontal gene transfer eukaryote family enzyme form particularly attractive substrate genetic exchange intriguing question distribution CE protease symbiotic pathogenic prokaryote virus suggestive general role host microbe interaction exemplify Salmonella protease ssel dna sequence datum obtain National Center Biotechnology Information Institute Genomic Research University Wisconsin coli Genome Project protein sequence identifier list Table sequence contain surround catalytic core protease align ClustalX algorithm default parameter manually edit Genedoc secondary structure predict JPred Phylograms construct MEGA software package Neighbor Joining Method Poisson correction substitution homogeneous pattern distribution pairwise deletion gap figure show consensus tree base bootstrap replication cloning expression biochemical analysis elad zeae length elad gene NCBI protein sequence identifi GI amplify PCR strain Novagen clone Novagen putative protease separate group show figure specie subdivision Proteobacteria list complete term ortholog paralog bacterial specie','2007-04-18 00:00:00','http://dx.doi.org/10.1371/journal.pone.000038',1590,662),('30a4842a2e257f725cc041e32fd682c495111a50.json','Multi-faceted, multi-versatile microarray: simultaneous detection of many viruses and their expression profiles','There are hundreds of viruses that infect different human organs and cause diseases. Some fatal emerging viral infections have become serious public health issues worldwide. Early diagnosis and subsequent treatment are therefore essential for fighting viral infections. Current diagnostic techniques frequently employ polymerase chain reaction (PCR)-based methods to quickly detect the pathogenic viruses and establish the etiology of the disease or illness. However, the fast PCR method suffers from many drawbacks such as a high false-positive rate and the ability to detect only one or a few gene targets at a time. Microarray technology solves the problems of the PCR limitations and can be effectively applied to all fields of molecular medicine. Recently, a report in Retrovirology described a multi-virus DNA array that contains more than 250 open reading frames from eight human viruses including human immunodeficiency virus type 1. This array can be used to detect multiple viral co-infections in cells and in vivo. Another benefit of this kind of multi-virus array is in studying promoter activity and viral gene expression and correlating such readouts with the progression of disease and reactivation of latent infections. Thus, the virus DNA-chip development reported in Retrovirology is an important advance in diagnostic application which could be a potent clinical tool for characterizing viral co-infections in AIDS as well as other patients.',' hundred virus infect different human organ cause disease fatal emerge viral infection public health issue worldwide early diagnosis subsequent treatment essential fight viral infection current diagnostic technique frequently employ polymerase chain reaction method quickly detect pathogenic virus establish etiology disease illness fast PCR method suffer drawback high false positive rate ability detect gene target time Microarray technology solve problem PCR limitation effectively apply field molecular medicine recently report Retrovirology describe multi virus dna array contain open reading frame human virus include human immunodeficiency virus type array detect multiple viral co infection cell vivo benefit kind multi virus array study promoter activity viral gene expression correlate readout progression disease reactivation latent infection virus DNA chip development report Retrovirology important advance diagnostic application potent clinical tool characterize viral co infection AIDS patient Microarray technology prove powerful tool great potential biological medical use technique recombinant dna fragment synthesize oligonucleotide affix surface glass slide nylon membrane detect complementary nucleic acid sequence frequently represent gene express sequence tag genotype microorganism profile gene expression pattern cell high organism new report Ghedin et al Retrovirology describe successful use multi virus array term multivi rus chip detect multiple viral co infection cultured cell study viral gene expression promoter activity figure Ghedin multivirus chip contain gene human virus include human immunodeficiency virus type conceptually chip detect viral co infection AIDS patient frequently render susceptible additional opportunistic infection develop multivirus chip Ghedin et al test orf human cell leukemia virus type hepatitis virus HCV Epstein Barr virus EBV human herpesvirus Kaposi sarcoma associate herpesvirus KSHV PCR amplify spot glass slide hybridize slide genomic dna cdnas derive virus infect cell multivirus chip find highly specific sensitive detect different viral genomic sequence cell line chip detect effect drug viral gene expression instance cell line latently infect KSHV generate profile viral gene expression presence cyclin dependent schematic drawing multivirus chip possess multiple function kinase inhibitor CKI Roscovitine apply cell suppress reactivation latently infect virus Ghedin et al study role cellular chromatin structure viral gene expression multivirus chip employ chromatin immunoprecipitation technique ChIP isolate cellular dna fragment bind phosphorylate histone dna fragment hybridize viral orf contain multivirus chip investigate role phospho viral gene expression show transcriptionally active silent chromatin state play role regulate expression KSHV gene different cellular context current routine clinical diagnostic employ PCR southern blotting northern blotting dna sequencing microarray hybridization detect characterize gene interest biomedicine PCR generally regard sensitive diagnostic method Iyer et al show sensitivity cdna chip hybridization comparable TaqMan drive quantitative PCR assay microarray hybridization technique likely complicate high false positive rate carry contamination furthermore microarray viral gene transcript infected cell easily detect hybridization prior amplification step microarray technique require experimental material compare Southern northern blotting provide high sensitivity setting large throughput view multivirus chip describe Retrovirology hold advantage commonly technique PCR dna sequence diagnosis viral infection chip provide accurate diagnosis viral infection simultaneously evaluate transcription viral gene use cumulative datum correlate infection clinical disease manifestation second high throughput flexible synthesis nature dna microarray construction allow scientist tailor rapidly alter array match evolve emergence new pathogen SARS genome chip NIAID NIH good example diagnostic array develop quickly timely manner finally novel application describe Ghedin et al use microarray correlate cellular histone code promoter activity KSHV usually transcription gene locate chromosomal DNA influence cis act ele ment dna bind motif structure chromatin vary depend post translational modification histone protein methylation acetylation phosphorylation certain amino acid residue amino terminal tail histone influence chromatin structure accumulate evidence show chromatin associate protein modification play vital role physiological process growth differentiation development mammal plant fungus study dna array technology investigate viral gene expression genotype viral isolate technique study influence cellular chromatin structure viral gene expression Ghedin et al demonstrate dna fragment derive ChIP latent cell genomic DNA capture phospho antibody bind specifically KSHV ORF multivirus chip result suggest latent KSHV genome cell pack nucleosomal structure histone protein near viral promoter phosphorylate serine dna promoter region tightly pack histone easily accessible transcription factor conclusion multivirus chip improvement develop Ghedin et al provide versatile clinical basic use near future chip likely detect viral co infection different clinical setting','2004-05-26 00:00:00','http://dx.doi.org/10.1186/1742-4690-1-11',682,335),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json','Multiplexed Genetic Analysis Using an Expanded Genetic Alphabet','BACKGROUND: All states require some kind of testing for newborns, but the policies are far from standardized. In some states, newborn screening may include genetic tests for a wide range of targets, but the costs and complexities of the newer genetic tests inhibit expansion of newborn screening. We describe the development and technical evaluation of a multiplex platform that may foster increased newborn genetic screening. METHODS: MultiCode(®) PLx involves three major steps: PCR, target-specific extension, and liquid chip decoding. Each step is performed in the same reaction vessel, and the test is completed in ~3 h. For site-specific labeling and room-temperature decoding, we use an additional base pair constructed from isoguanosine and isocytidine. We used the method to test for mutations within the cystic fibrosis transmembrane conductance regulator (CFTR) gene. The developed test was performed manually and by automated liquid handling. Initially, 225 samples with a range of genotypes were tested retrospectively with the method. A prospective study used samples from >400 newborns. RESULTS: In the retrospective study, 99.1% of samples were correctly genotyped with no incorrect calls made. In the perspective study, 95% of the samples were correctly genotyped for all targets, and there were no incorrect calls. CONCLUSIONS: The unique genetic multiplexing platform was successfully able to test for 31 targets within the CFTR gene and provides accurate genotype assignments in a clinical setting.',' multiplexed technology sufficient specificity identify small change human genome available clinical use line probe linear array technology use nitrocellulose paper strip support matrix benefit line probe expensive instrumentation require technology term oligonucleotide ligation assay use target specific ligation generate mutationspecific product analyze gel electrophoresis genetic analysis system couple microparticle flow cytometry limitation method include complicated procedure require extensive training washing centrifugation transfer long completion time expensive instrumentation technician dependent result analysis addition technology easily amenable automation hospital clinic currently send molecular diagnostic test large reference laboratory attributable limitation describe benefit have local laboratory perform diagnostic test appear need molecular testing method circumvent limitation pose current testing method construct step platform technology call MultiCode PLx eliminate technically challenging step involve multiplexe genetic analysis compare line probe assay clinical setting MultiCode PLx use additional nucleobase pair construct complementary nucleobase deoxy isoguanosine dig iso cytosine iC nucleobase specifically recognize base different pattern hydrogen bonding choose pair pair available commercially chemistry explore example iG iC successfully molecular recognition site specific incorporation addition iG iC recognition orthogonal naturally occur nucleobase pair strand DNA contain iC iG component construct hybridize natural dna complex reaction high concentration natural dna known unknown sequence exist orthogonal attribute allow specific molecular recognition place interference additional nucleobase step PLx process PCR extension labeling liquid decode PCR primer design target specific contain single ics amplicon act labeling template target specific extension TSE step step label attach dig triphosphate diGTP incorporate site specifically code target specific extender Fig tag short sequence typically nucleotide length assemble use mixture natural nonnatural basis tag design hybridize perfect complement encode color address microsphere final step decode extension reaction accomplish room temperature capture code extender address microsphere reading reporter signal Luminex instrument step carry reaction vessel transfer washing demonstrate approach develop screening test carrier cystic fibrosi CF test mutation reflex target widely panel test second prevalent mutation population nordic heritage important Wisconsin testing perform add line evidence indicate disease cause mutation replace Wisconsin State Laboratory Hygiene conjunction University Wisconsin choose major testing site long history screen newborn CF study screen method March dna testing daily immunoreactive trypsinogen result expand include mutation recommend American College Medical Genetics American College Obstetricians gynecologist report present MultiCode PLx test datum variety genomic DNAs genotype prepare sample type presence absence mutation cystic fibrosis transmembrane conductance regulator CFTR gene finally datum present prospective study compare testing result obtain freshly prepare blood spot PLx assay commercially available CF Gold Linear Array TM system Roche Molecular DNA extract blood dry blood spot Guthrie card dry blood spot store room temperature day receipt routine testing complete long term storage dry blood spot store dna extraction Guthrie card perform Generation Capture card Kit Gentra Systems accord manufacturer guideline dna purification elution DNA extract blood use Puregene DNA Isolation Kit Gentra Systems Whatman FTA system Whatman Clifton NJ genomic dna preparation dilute water add PCR reaction ng total dna cross reference prospective study genotype determine CF Gold linear array dna sequence discrepancy detect PCR TSE oligonucleotide manufacture inhouse column DNA Synthesizer Northwest Engineering Applied Biosystems standard phosphoramidite chemistry sequence information Tables Data Supplement accompany online version article Synthesis sequence contain iC iG purchase IDT Eurogentec EraGen demonstration assay encompass specific system purchase EraGen Biosciences amino label EraCode oligonucleotide manufacture ABI DNA Synthesizer Applied Biosystems PT Amino Modifier CPG Glen Research target specific extender primer manufacture spacer phosphoramidite Glen Research place EraCode complement target specific region isog isoc respectively couple deprotecte condition natural base phosphoramidite postsynthesis work consist ammonium hydroxide deprotection follow ethanol precipitation PCR TSE oligonucleotide propanol precipitation EraCode bead couple conjugation reaction mixture contain carboxylated microsphere Luminex Corporation centrifuge min supernatant remove bead resuspend mol MES pH resuspension mol amine modify EraCode DNA oligonucleotide freshly prepare EDC Pierce Chemical add reaction mix thoroughly incubate room temperature dark min second EDC add mixture incubate min era Code microsphere wash addition mL mmol Tris pH contain mL Tween pellete centrifugation min supernatant remove EraCode microsphere wash second time sodium dodecyl sulfate mmol Tris pH resuspend bead buffer mmol MOPS pH mmol EDTA sodium dodecyl sulfate mmol sodium chloride sonicate herre sperm dna EraCoded microsphere combine dilute bead buffer generate code mixture concentration microsphere class microsphere capture room temperature tag specific complementary EraCode test use model extension system target iC construct tag interchange targetspecific extender sequence EraCodes construct use basis comprise ics effort minimize interference naturally occur sequence EraCodes design naturally occur basis row effort minimize code code cross hybridization code allow contain series nucleobase row EraCodes validate use extender contain identical target specific sequence extender individually label target standard concentration naturally occur deoxynucleotide triphosphate TiTaq digtp biotin mol EraCode sequence separately couple specific color code fluorescent microsphere type Luminex Corp microsphere type contain unique color signature determine Luminex coupling EraCode bead conjugate pool create equalized bead master mixture bead array different EraCode label bead bead array distribute separate reaction vessel vessel specific extension reaction add room temperature streptavidin phycoerythrin read Luminex extension reaction excess template oligonucleotide ic nmol combine control extension sequence nmol EraGen MC TSE solution prod EraGen Biosciences mol deoxynucleotide triphosphate mol diGTP biotin Titanium Taq Clontech control extension oligonucleotide compose sequence complementary EraCode sequence carbon spacer sequence complementary template oligonucleotide CACAGC reaction subject follow thermal cycling protocol min extension reaction hybridize EraCode bead mixture accord describe hybridization protocol addition mimic MultiCode condition unextended TSEs EraCodes add hybridization extension product reaction perform mix EraGen MC Hyb Solution prod EraGen Biosciences EraCoded bead mixture hybridization reaction streptavidin phycoerythrin conjugate Prozyme plus sheath fluid Luminex add final reaction inject Luminex pcr PCR primer design Primer Hyther Primer pair design flank interest melt temperature Table online Data Supplement single iC add end PCR primer PCR reaction perform final volume reaction contain EraGen MC PCR solution prod system Clinical Chemistry EraGen EraGen CFTR PCR primer mixture Titanium Taq diluted genomic target dna reaction cycle accord following profile cycle cycle primer mixture contain forward reverse primer subset mutation concentration TSE determine exact mutation present TSE require TiTaq diGTP biotin extender contain target specific basis code specific EraCode bead describe TSE TiTaq incorporate label extender specific target have ics present nonnatural base pair form iG iC successfully molecular recognition site specific enzymatic incorporation date covalently couple series different reporter group digtp succinimidyl ester chemistry confirm enzymatic incorporation nucleic acid use commercially available polymerase datum scope report case single amplicon span region incorporate multiple mutation template multiple extender TSE EraCode label bead streptavidin phycoerythrin add reaction analyze Luminex datum report median fluorescence intensity polymorphism pair TSE oligonucleotide primer design use Primer Hyther Table online Data Supplement pair design balance melting temperature range TSE oligonucleotide compose component sequence tag complementary EraCodes carbon spacer target specific sequence fully complementary allele interest polymorphic sequence pair TSE oligonucleotide base extension reaction perform bring PCR reaction addition EraGen MC TSE solution prod EraGen Biosciences include primer system prod EraGen CFTR TSE primer mixture final concentration TSEs nmol Table online Data Supplement reaction cycle accord following profile cycle cycle min report assignment define genotype give target CFTR gene give sample sample test possible assignment target sample analyze rapidly analyze datum obtain develop software present raw datum instrument software directly import Luminex datum organize target mutation fraction wild type wild typeplus mutant raw MFI signal visualize graphically Fig program set default fraction window wild type sample heterozygous sample homozygous mutant sample default window change end user area window designate window program set amplification window wild type mutant mfi set Luminex LabMap software require importation analysis software achieve drag Luminex file folder Analysis Software datum file parse result raw MFI value organize target sample datum acquisition clinical sample test default cutoff window target empirically determine set blind fashion determination report generate offline analysis analysis software write Java compatible Windows Mac OS Linux operate system datum importation achievable Luminex LabMap software version bio Rad BioPlex software display scatter plot fraction plot give target Automation perform Tecan Genesis Model robotic workstation trp start PCR amplicon formation isolated genomic dna continue datum analysis sample subset initial manual testing instruction automation program write Gemini software Tecan initialization launch bead read protocol Luminex perform automatically use custom image recognition command script program write Java read result remotely analyze MultiCode Analysis software Genomic DNA purify manually automation describe trp fit low volume channel dispense pipette system TeStack supply disposable tip MJ Research thermocycler equip remote alpha dock Power Bonnet robotic manipulator arm liquid reagent disposable tip place deck run start reaction addition automate process describe manual method note reaction perform low profile hard skirt polypropylene microplate allow robotic arm grip plate need reaction overlaid light mineral oil Sigma prevent evaporation thermal cycling robotic manipulator transfer plate thermocycler block Luminex mount deck initialization launch bead read protocol Luminex perform automatically use custom image recognition command script pro gram write Java bead reading reaction result analyze MultiCode Analysis software describe schematic diagram MultiCode system show Fig MultiCode process step step region target interest amplify PCR primer contain single second step amplicon step template TSEs iC direct specific digtp biotin incorporation TSEs code end sequence specifically recognize short complementary sequence EraCodes attach address solid matrix molecular recognition give code EraCode specific increase temperature incubation step washing require typical requirement hybridization capture naturally occur base chemistry second step label place site specifically tse process require presence ic contain amplicon complementary TSE correct target available extension reaction continue end amplicon diGTP biotin insert biotin TSE specificity signal consistency arise code tse covalently attach single biotin reporter presence correct target final step tse capture solid matrix detect presence target determine reporter locate spatially liquid matrix create differentially color EraCode specific complementary tag bead contain specific EraCode report high MFI result correct complementary code specific signal MFI mean sd MFI table online Supplemental Data signal generate microsphere contain era Codes completely complementary varied MFI mean SD MFI set give signal noise ratio range bad case specific signal compare nonspecific signal microsphere present high noise understanding report genetic base molecular recognition system successfully implement room temperature washing mutation analysis manual cf multicode system PCR amplicon generate region CFTR gene tag extender analyze mutation reflex target improve robustness overall result test divide reaction additional test target reflex target additional target include late development phase include test transfer Wisconsin State Laboratory Hygiene newborn CFTR genetic screening ongoing multimutation testing CFTR specific PLx assay positive signal intensity varied MFI background signal varied MFI variation signal depend strictly target sequence observe determination window varied wild type sample heterozygous sample homozygous mutant sample calling result summarize Table Table online Data Supplement additional detail sample include wild type sample heterozygous sample represent type compound heterozygous sample type homozygous mutant sample type sample test manually sample report correct assignment failure rate likely step tag target specific extender site specifically label diGTP label correct amplicon present study label biotin streptavidin phycoerythrin signal purpose step tag capture EraCode modify Luminex microsphere capture reaction analyze Luminex system product dna quality assay sensitivity state Materials Methods dna sample fresh signal intensity indicate gross failure consistent PCR failure discrepancy genotype classification sample total potential assignment mutation reflex test include correct assignment time separate specific reflex test sample test include separate sample possibility Table Fig online Data Supplement test result show zero incorrect determination amplification analysis automate cf multicode systemthe final automate process CFTR gene conduct total sample result report separate process PCR datum acquisition sample TSE datum acquisition sample Table Table online Data Supplement sample test information workstation space limit Tecan Genesis Model tip usage high disposable tip implement amplicon carryover concern process apply high throughput scenario PCR step implement workstation manually process sample report call correct process sample report correct assignment incorrect determination assay validation include testing mutation sample type PLx assay CFTR gene analysis directly compare CF Gold linear array method freshly obtain blood card newborn immunoreactive trypsinogen score upper test analyze CFTR recommend core panel mutation reflex target testing perform sample concordance record incorrect assignment Table column sample test sample complete list call obtain PLx system failure rate document error result pipette error sample reteste correct assignment possible potential assignment correct run time sample preparation CF Gold test compare PLx system total hand time CF Gold test compare PLx system datum output CF Gold test visual inspection nitrocellulose strip PLx system computerized calling system describe describe novel high throughput platform incorporate nucleic acid chemistry simplify genetic analysis PLx use additional base pair find nature combination microsphere flow cytometry technology eliminate cumbersome step commonly find establish multiplexe system EraCode molecular recognition system describe particularly suit Luminex instrumentation solid support dispense complex reaction decoding room temperature washing create true Liquid chip type system study indicate chemistry transferable clinical setting addition require process perform robotic workstation high throughput demonstrate principle chemistry use essential target select mutation find CFTR gene recommend lead medical organization datum Wisconsin State Laboratory Hygiene confirm chemistry accurate example incorrect determination percentage correct determination high datum present tabulate use broad sample set include mutation possible panel target exception sample entire manual process take obtain final result upgrade prototype version CFTR test mutation analyze single take complete analysis patient sample validation datum available demonstrate system fully automate mode elect use tecan genesis liquid handling system entire automate process take prepared sample final read entire plate sample present datum possibility automation start PCR step start TSE step need perform PCR offline believe expand genetic alphabet enable eliminate step require technology base base pair provide nature step eliminate include solid support washing transfer aspiration high temperature hybridization specifically believe reason extra base pair beneficial example short code sequence capture step decode complex extension mixture possible code set construct optimization surely difficult cross hybridization sequence complement close complement find nature additional informational content iC iG build DNA allow great diversity compare natural counterpart specifically naturally occur basis library possibility contrast letter system generate unique demon Clinical Chemistry strate specific assembly occur condition room temperature effect amenable simplify molecular diagnostic demonstrate time biotin label attach diGTP site specifically incorporate extension primer generally demonstrate label diGTP incorporate enzymatically DNA allow broad use compound molecular biology reporter group require specific site amplicon transcript extension product use additional base pair helpful recently report site specific enzymatic incorporation quencher attach diGTP create new highly sensitive real time PCR system polymerase incorporation label site expand genetic alphabet basis position use end labeling polymerase incorporation EraCodes PCR amplicon report EraCode system use additional nonnatural base pairing advantage dna base molecular recognition system short dna code simple produce specifically hybridize room temperature require wash step amplify standard polymerase cross hybridize natural dna take believe expansion genetic alphabet true paradigm shift way molecular diagnostic develop future','2004-11-01 00:00:00','http://dx.doi.org/10.1373/clinchem.2004.03433',2427,815),('323ece9056ef7a1d81faf3422e98def1efd38f70.json','Immune pathways and defence mechanisms in honey bees Apis mellifera','Social insects are able to mount both group-level and individual defences against pathogens. Here we focus on individual defences, by presenting a genome-wide analysis of immunity in a social insect, the honey bee Apis mellifera. We present honey bee models for each of four signalling pathways associated with immunity, identifying plausible orthologues for nearly all predicted pathway members. When compared to the sequenced Drosophila and Anopheles genomes, honey bees possess roughly one-third as many genes in 17 gene families implicated in insect immunity. We suggest that an implied reduction in immune flexibility in bees reflects either the strength of social barriers to disease, or a tendency for bees to be attacked by a limited set of highly coevolved pathogens.',' evident social organism range human bird Brown Brown Masuda et al impact sociality disease especially vivid social insect colony typically thousand individual interact close quarter density far exceed crowded vertebrate social group Wilson density couple relatively homeostatic nest environment presence store resource make social insect attractive target disease agent Schmid Hempel expect base parasite pathogen pressure social insect evolve individual group strategy combat disease Grooming nest hygiene behavioural trait find social insect reduce impact pathogenic bacteria fungus parasitic mite example hygienic behaviour describe honey bee Rothenbuhler classical example social defence worker identify remove infect larvae healthy brood Spivak Reuter defence enable sociality include construction nest antimicrobial material Christe et al raising offspring sterile nursery Burgett social fever response disease Starks et al transference immune trait Traniello et al Sadd et al heighten risk taking infected individual Schmid Hempel like eukaryote colony member possess individual defence include immune response disease agent Casteels Josson et al Evans recent sequencing honey bee genome Honey Bee Genome Sequencing Consortium allow global analysis immune component honey bee second opportunity human use genomic insight well understand disease resistance highly social organism insect diverse mechanism combat infection pathogen insect protect layer antimicrobial secretion exterior gut environment hostile pathogen pathogen defence epithelium sufficient stop progress pathogen defeat morphological defence insect meet efficient cellular humoral immune defence insect immunity show parallel innate immune response human vertebrate involve diverse set action include secretion antimicrobial peptide phagocytosis melanization enzymatic degradation pathogen Hoffmann Hultmark insect immune pathway share overall architecture specific orthologous component innate immune system vertebrate Beutler suggest share root immune pathway selection conserve component hundred million year paper propose honey bee model non autonomous pathway implicate inducible host defence toll Imd Janus kinase JNK Boutros et al base primarily extensive search orthologue study fruit fly mosquito moth specie pathway engage cross talk direct immune effector define structure interaction set well tackled individual entity honey bee component pathway remain validate functional test feel present model serve important purpose point likely orthologue involve stage immune response set stage postgenomic functional work honey bee immunity second model intriguing difference specie canonical immune pathway respect gene loss duplication immune gene family bee appear reduce number compare Drosophila Anopheles genome wide analysis honey bee identify gene family reduced diversity Honey Bee Genome Sequencing Consortium reduction appear especially pervasive immune system reduction hold stage immunity recognition signal immune effector couple gene family datum datum specific orthologue test miss gene represent current draft honey bee genome assembly present genome immune relate gene bee diverge especially quickly sequence level escape annotation base sequence similarity specie honey be enact immune response pathway component currently identify immune player insect honey bee target small set coevolve pathogen immune system tune pathogen expense responsive wide range threat social defence barrier honey bee colony effective reduce pathogen pressure bee reliant insect individual immune response honey bee possess apparent orthologue core member pathway implicate immunity Figs Supplementary Material Table precise orthology honey bee fly Drosophila melanogaster anophele gambiae evident pathway member especially intracellular component dozen describe actor signal pathway predict play role insect immunity protein appear completely absent bee genome ligand unpaire JAK STAT pathway presence JAK STAT cytokine receptor domeless member pathway Fig suggest JAK STAT remain functional honey bee trigger currently unrecognized ligand Insect Toll toll like receptor TLRs transmembrane signal transducing protein play critical role immunity development orthologous mammalian tlr implicate immunity Beutler Drosophila toll signal pathway enact cytokine like molecule Spaetzle bind extracellular domain transmembrane receptor toll Drosophila genome encode family Spaetzle relate molecule believe function ligand Drosophila Toll receptor Parker et al plausible spaetzle orthologue evident bee genome Fig functional test need determine act toll bind cytokine follow conformational change activate receptor intracellular death domain dd contain protein recruit form receptor complex activation complex lead degradation NF kappa inhibitor Cactus subsequent nuclear translocation NF transcription factor Dorsal Dorsal relate immune factor Dif Drosophila Royet et al homologue Dorsal find honey bee genome Fig orthologous Dif lend support view Dif highly derive branch find brachyceran fly absent insect mosquito Shin et al arguably honey bee Dorsal call mosquito functional alternate Dif functional test help determine dorsal paralogue key transcription factor pathway intracellular component Tollip Pellino Cactin TNF receptor associate believe aid main player pathway appear present fly specie honey bee candidate effector immune relate toll pathway honey bee include compliment antimicrobial peptide melanize agent phenoloxidase lysozyme confirm effector trigger toll pathway oppose pathway describe evident bee effector responsive pathogen mechanical wounding bee Fig candidate honey bee member toll pathway name give Drosophila pathway component vertebrate orthologue parenthesis Honey bee match give name genome project Honey bee name italic refer gene close paralogue readily distinguish respect pathway component Drosophila Underlining indicate gene show transcriptionally regulated immune challenge candidate honey bee member Imd JNK JAK STAT pathway name Drosophila pathway component vertebrate orthologue parenthesis Honey bee match present name genome project Honey bee name italic refer gene close paralogue readily distinguish respect pathway component Drosophila Underlining indicate gene show transcriptionally regulated immune challenge toll signal fly serve dual purpose development immunity signal process activate peptidoglycan recognition protein Imd specific antimicrobial defence dispensable normal development Hultmark NF like transcription factor Relish signal induce transcription major antimicrobial effector peptide Drosophila Drosophila Imd signalling say specific Gram negative bacteria Gram positive bacteria diaminopimelic acid type peptidoglycan strong elicitor weak response see type peptidoglycan fungi Hultmark Werner et al Stenbak et al broad specificity cause alternative splice form Drosophila PGRP LC carry different peptidoglycan recognition domain Werner et al Mellroth et al Chang et al interestingly Imd signal pathway highly conserve honey bee plausible orthologue component Fig strongly suggest Imd signalling similar fly bee necessary imply similar biological role activation Relish Imd signal lead activation component JNK signal pathway Boutros et al recent evidence indicate pathway provide positive negative feedback expression antimicrobial peptide Wojda et al plausible orthologue major component JNK signalling identify honey bee genome Fig JAK STAT signal pathway contribute innate immunity induction complement like factor overproliferation haemocyte JAK STAT appear initiate cytokine like molecule blood cell Agaisse Perrimon fly extracellular glycosylated protein Upd act ligand activate JAK STAT pathway turn promote phagocytic activity haemocyte JAK STAT pathway recently show participate antiviral response Drosophila Dostert et al Honey bee homologue Drosophila JAK STAT signal pathway Fig comprise cytokine receptor domeless Dom JAK tyrosine kinase Hopscotch transcription factor transcript abundance immune candidate gene adult worker injection escherichia coli Ec saline buffer bee pathogen Paenibacillus larvae control leave column column right transcript abundance instar larvae challenge orally infective dose larvae unchallenged control Cluster gene strongly regulate adult injection wounding Cluster gene regulate infected larvae Cluster gene regulated minimally change challenge bee negative pathway regulator SOCS suppressor cytokine signal PIAS protein inhibitor activate STAT orthologue recently identify component pathway Baeg et al Muller et al tyrosine phosphatase bromo domain contain protein present honey bee key ligand Upd JAK STAT pathway find honey bee genome presence cytokine receptor homologue Domeless member signal pathway indicate mechanism common insect intact honey bee fly addition regulate complement like thiolestercontaine protein TEPs Lagueux et al Boutros et al JAK STAT pathway Drosophila regulate expression Turandot tot gene encode humoral factor induce severe stress Agaisse et al Tot factor Tot apparent honey bee honey bee appear maintain know insect immune relate pathway appear reduced number paralogous member compare set gene family functional group implicate immune responsiveness Christophides et al honey bee substantially low paralogue count Drosophila Anopheles table gene place group honey bee sharp contrast find Drosophila Anopheles respectively bee low gene count family tie low count time Drosophila Anopheles low family defensin dorsal respectively contrast Drosophila Anopheles high paralogue count triad seven time respectively versus bee toll pathway candidate cactus copy ranking significantly different ordinal contingency table analysis reflect difference gene involve pathogen recognition signal effector pgrp major player pathogen recognition Hultmark Steiner Royet et al diverse honey bee versus fly insect genomic datum exist moth Bombyx mori pgrp honey bee genome compare seven Drosophila Anopheles respectively Fig Table bee capacity splice variation contribution diversify peptidoglycan recognition specificity fly mosquito specifically single membrane bind PGRP bee PGRP LC similar fly mosquito PGRP LC lack potential insert alternative peptidoglycan recognition domain alternative splicing factor recognition breadth fly PGRP LC PGRP regulated honey bee disease challenge fly suggest product gene play defensive role Fig Anopheles class Scavenger Receptor SR honey bee group diversify member Drosophila selective sign suggestive immune role Lazzaro Class scavenger receptor bee number roughly similar fly mosquito Fig Table recognition class reduce honey bee include recognition protein grp galectin fibrinogen relate protein table fibrinogen domain gene especially striking absence bee high lineage specific diversification find mosquito Drosophila result domain family member respectively christophide et al PGRP LC SR receptor recently find participate phagocytosis infectious non self Drosophila DSCAM Eater DSCAM long implicate neuronal development show likely role binding bacteria Drosophila haemocyte Watson et al gene potential splice variant honey bee thank set highly interchangeable exon Graveley et al interesting candidate determine extent bee well tune immune response specific pathogen table gene count subset gene family implicate insect immunity anophele gambiae Drosophila melanogaster count base Christophides et al Christophides et al recently identify protein involve Drosophila cellular immunity Eater phagocytic receptor characterize repeat EGF motif extracellular domain Kocks et al gene EGF motif present bee genome Supplementary Material Table fly orthology Eater unclear bee likely engage encapsulation endoparasite pathogen typical integrin Honey Bee Genome Sequencing Consortium implicate lamellocyte encapsulation cellular immunity component appear diverse bee fly specie Supplementary Material Table addition function digestion food serine protease SPs insect participate regulatory cascade pathway embryonic development immune response Kanost Clarke haemolymph sp serine protease homologue SPHs implicate immunity contain clip domain amino terminus regulate localize immune response stimulate protease cascade Jiang Kanost SP relate protein honey bee genome sp sph contain clip domain significantly few Drosophila sp SPHs Ross et al anophele sp sph small number clip domain sp identify date expression datum Manduca sexta Jiang et al additional phylogenetic functional relationship insect sp sph discuss Zou et al serine protease inhibitor serpin superfamily regulate protease cascade mammal arthropod Reichhart insect haemolymph serpin inhibit activate protease maintain homeostasis prevent unregulated activation immune response melanization toll mediate antimicrobial protein synthesis Kanost Clarke honey bee genome seven annotated gene encode serpin protein serpin like region number serpin gene honey bee low Drosophila anophele mirror reduced size protease gene family bee toll relate receptor gene know Drosophila genome anophele Tauszig et al Christophides et al Drosophila toll primary family member implicate immune relate function Lemaitre et al appear Drosophila paralogue structurally similar mammalian tlr involve immune relate signalling Ooi et al Bilak et al identify toll relate gene honey bee Trex Tollo toll member Drosophila apparently reflect gene duplication event fly lineage Fig case arguably loss honey bee lepidopteran lineage instance ancestral toll appear diverge different group fly contrast remain member clade honey bee similarly clade represent single honey bee homologue Aronstein Saldivar presumably ancestral toll gene duplicate fly follow divergence mya lineage lead honey bee protein name Apis mellifera Kanzok et al likely orthologue toll receptor present mellifera genome present sequence genome insect belong order Diptera Lepidoptera Coleoptera exception example orthologue find melanogaster gambiae Diptera Bombyx mori Lepidoptera Tribolium castaneum Coleoptera appear absent genome lepidopteran insect mori appear lose fruit fly suggest gene encode basic set toll receptor present common ancestor insect receptor highly express dynamic tissue specific manner Drosophila embryogenesis Kambris et al note adjacent melanogaster mellifera genome immune effector total honey bee antimicrobial peptide contrast find Drosophila Anopheles respectively table abaecin apidaecin class prolinerich antimicrobial peptide conventional defensin apisimin hymenoptaecin distinct recognize antimicrobial peptide genomic analysis reveal gene encode apidaecin consist conserve terminus follow exon encode complete amino acid peptide Peptide cdna evidence gene predict mechanism ratchet expression apidaecin response bacterial challenge Casteels Josson et al genomic structure apidaecin raise possibility mechanism generate specific response pathogen splice variation exon functional distinct antimicrobial peptide conceivable splice variation locus refine gene immune effector Apidaecin exon differ greatly individual honey bee number encode amino acid sequence bee haplotype sequence project differ sequence exon number differ previously describe apidaecin cDNAs casteel et al collectively haplotype sequence describe Casteels et al encode range secrete peptide peptide share haplotype gene surprisingly high level sequence variation peptide express haplotype reflect different amino acid variant exception identify genomic sequence generate project Klaudiny et al honey bee antimicrobial peptide characterize protein sequence Casteels Josson et al fact belie difficulty discover gene sequence similarity million year separate insect specie evidence close paralogue honey bee antimicrobial peptide contrast pattern insect gene rich cecropin family bee antimicrobial peptide regulate diverse immune challenge Fig Evans honey bee possess prophenoloxidase proPO gene versus Drosophila Anopheles respectively like proPO honey bee propo lack signal peptide consensus sequence NRFG activation site gene encode proPO express strongly old honey bee larvae pupae Lourenço et al gene regulate challenge experiment gene identify propo activator regulate natural infection Fig lysozyme honey bee genome type type class lysozyme regulate challenge honey bee Fig few thiolester contain protein tep bee genome expect base fly table Anopheles genome encode tep originate species specific expansion Christophides et al versus member gene family Drosophila Agaisse Perrimon tep induce septic injury promote phagocytosis mosquito Blandin Levashina Moita et al play central role vertebrate innate immunity complement factor Anopheles find promote phagocytosis Gram negative bacteria major player host response plasmodium infection member group implicate recognition protein effector opsonin insect Levashina et al Moita et al Stroschein Stevenson et al social animal honey bee considerable risk parasite pathogen specifically increase genetic relatedness high population density typify honey bee society strongly favour pathogen spread epizootic outbreak social cost honey bee immunity offset social defence include mate strategy multiple mating queen Tarpy mutual grooming maintenance sheltered environment colony member give study natural pathogen immune pathway model current annotate draft genome unique genetic trait honey bee join fly moth specie important system understand genetic cause immunity disease join human organism great interest understand social driver disease information improve host survival analysis indicate basic set molecule define insect host defence system present honey bee include intact pathway key process implicate immunity development single orthologue assign pathway member potential bee gene work need confirm role interestingly Drosophila Anopheles host appear diversify molecular arsenal species biased gene duplication table find example gene expansion honey bee despite have wideranging parasite pathogen tremendous loss pest domesticate setting Morse Flottum bee appear relatively diminish capacity respond defend pathogen hypothesis help explain observation bee gene family systematically small insect support evidence systematic downward bias paralogue count bee genome level see immune gene family Honey Bee Genome Sequencing Consortium hypothesis bee gene simply miss sequence divergence apply particularly immune gene short subject limited sequence constraint encode antimicrobial peptide peptide identify bee mean discover directly protein level case support idea sequence level search miss additional family member member remain gene poor family comprise especially short gene gene divergent miss completely comparison level insect order fact significant bee Drosophila anophele orthologue present remain discuss family indicate sufficient sequence level conservation identification bee counterpart hypothesis bee simply undescribed mechanism broaden immune efficacy testable point surprising give similarity immune actor diverse insect order study date initial hypothesis bee face diverse set successful parasite pathogen societal defence bee lessen pathogen pressure well support datum hand compare level bee pathogen diverse range Gram positive Gram negative bacteria fungi RNA viruse microsporidia amoebae Morse Flottum bee parasitize mite arthropod raise risk pathogen infection Kanbar Engels Chen et al low ability combat disease Gregory et al Yang Cox Foster despite have pest range kingdom common disease agent bee fact restrict pathogen bacterium Paenibacillus larvae fungus Ascosphaera apis predominant functional datum specificity response pathogen available gene expression change challenge appear especially precise respect honey bee pathogen exotic Escherichia coli stress generally Fig compelling evidence point bee immune response channel small set true pathogen respect final hypothesis bee like social insect relentlessly hygienic remove alien organism nest secrete antimicrobial substance reduce viability growth pathogen colony bee raise young individual cell food source substance strongly antimicrobial property royal jelly Albert Klaudiny testament hygiene fact face severe colony level infection bacterial pathogen Paenibacillus larvae spore normally fatal young larvae Brodsgaard et al vast majority larvae sign exposure Evans Pettis Social barrier reduce exposure minor opportunistic pathogen saprophyte propose generalize target insect immune defence Hultmark bee carry assemblage microbe bacteria particular Gilliam exposure microbe arguably low free live Drosophila decay plant material larvae adult anophele septic aqueous environment larvae bacteria find bee colony rarely associate disease pathology despite extensive study fact resident bacteria colony appear add external defence inhibition bee pathogen Evans Armstrong genomic work help reveal specie highly social insect include ant wasp bee termite appear simplistic innate defence system generally social solitary insect expose life history predict great number high functional diversity immunepathway gene end product compare sister taxa sheltered datum parasite pathogen abundance social Boomsma et al solitary insect surrogate disease load different taxa field epidemiological datum need assess relative fitness impact disease efficacy different line defence analysis present plausible model immune pathway social insect honey bee nearly complete conservation candidate gene pathway bee consistently undercut number gene embellish pathway insect genome wide expression study newly available bee prove success gene knockdown technique RNA inactivation Lourenço et al allow refinement role play pathway member discovery completely novel player honey bee immunity discovery combine analysis specie social solitary bee help determine observation describe unique highly social honey bee longstanding agricultural interest help generate wealth datum honey bee pathology Morse Flottum possible connect datum immune trait help limit pathogen efficacy connection honey bee provide valuable tractable model disease transmission immunity socialize medicine pathology Immune gene candidate insect way query honey bee genome primarily blast family search function search initiate BLASTP query consensus protein list derive HBGP assembly BLASTP algorithm BLOSUM PAM variant appropriate gene size structure Honey bee orthologue identify search honey bee genome assembly directly TBLASTN local database BeeBase blast server search miss gene carry small coverage set honey bee contig short include assembly unassembled read project give honey bee candidate search repeat hope identify paralogue miss interspecific comparison psi BLAST identify honey bee gene basis conserve domain follow RPS BLAST confirm significance domain match putative match align case serine protease scavenger receptor type lectin screen additional motif pfam category tentative match align check gene prediction error case gene official protein list annotation immune candidate gene Honey Bee Genome Project Honey Bee Genome Sequencing Consortium protein match port Apis mellifera assembly alignment program BLAT Jim Kent University California Santa Cruz establish scaffold location Supplementary Material Table match honey bee sequence align counterpart melanogaster gambiae insect sufficient genome level datum available amino acid sequence alignment carry GONNET series weight matrix program Chenna et al alignment propose phylogenetic relationship maximum parsimony neighbour join algorithm program PAUP Sinauer Sunderland MA PHYLIP PGRP tree base conserve domain region alignment edit manually reduce remove ambiguous region scavenger receptor class protein human mouse protein add alignment honey bee scavenger receptor include relatively short predict length apparently result intercontig gap preclude unambiguous alignment alignment available request experiment carry screen immune relate transcript change adult worker bee single local mellifera ligustica colony remove inject abdominally dilute phosphate buffer saline saline solution contain live cell coli vegetative spore honey bee bacterial pathogen Paenibacillus larvae bee uninjected control maintain high humidity immediately freeze prior RNA extraction assess immune response follow natural infection instar larval bee stock give os challenge larvae food describe Evans maintain high humidity parallel control larvae give food bacterial spore sample freeze follow incubation RNA extract abdomen adult bee standard TRIzol Invitrogen Carlsbad USA procedure RNA extract individual larvae rnaqueous kit Ambion Austin TX RNAs pool sample duration larvae challenge bacterial pathogen larvae control prior cdna synthesis give RNA pool DNA remove extract strand cdna synthesize describe Evans transcript abundance cdnas assay quantitative real time PCR Icycler real time PCR machine bio Rad Hercules USA Primer pair design amplify bp section honey bee immune relate gene derive Supplementary Material Table ribosomal protein control gene primer Supplementary Material Table Primer sequence modify necessary run duplicate plate fix thermal protocol consist min cycle step protocol consist min Evans reaction carry µg cdna Taq provide PCR buffer Roche Applied Sciences Indianapolis USA mm dNTP mix mm add MgCl µm primer concentration SYBR Green dye Applied Biosystems Foster City USA nm fluorescein µl reaction volume amplification follow meltcurve dissociation program order confirm expect product size threshold calculate individually target gene plate adult bee sample datum pool replicate single bee injection treatment control result screen appropriate dissociation melt curve value agarose gel order ensure primer artefact presence dna contamination evident numerous primer span exon Immune gene transcript normalize relative expression level gene encode ribosomal protein gene consistent expression honey bee life stage disease status Evans Wheeler Evans display purpose transcript abundance value CTcontrol CTtarget gene median normalize panel gene cluster average linkage clustering Cluster Eisen present relative grey scale value Treeview Eisen','2006-10-01 00:00:00','http://dx.doi.org/10.1111/j.1365-2583.2006.00',3477,1155),('348055649b6b8cf2b9a376498df9bf41f7123605.json','Role of endothelin-1 in lung disease','Endothelin-1 (ET-1) is a 21 amino acid peptide with diverse biological activity that has been implicated in numerous diseases. ET-1 is a potent mitogen regulator of smooth muscle tone, and inflammatory mediator that may play a key role in diseases of the airways, pulmonary circulation, and inflammatory lung diseases, both acute and chronic. This review will focus on the biology of ET-1 and its role in lung disease.',' Xenopus laevis ETA receptor normal lung find great abundance vascular airway smooth muscle etb receptor find endothelium Clearance circulation mediate etb receptor primarily lung kidney liver ETA etb receptor smooth muscle cell lead vasoconstriction etb receptor activation lead bronchoconstriction activation ETB receptor locate endothelial cell lead vasodilation increase nitric oxide production mitogenic inflammatory modulator function primarily mediate ETA receptor activity bind ligand receptor result coupling cell specific protein activate inhibit adenylate cyclase stimulate phosphatidyl inositol specific phosholipase open voltage gate calcium potassium channel varied effect receptor activation depend protein signal transduction pathway active cell interest grow number receptor antagonist exist variable selectivity receptor subtype regulation level transcription stimulus include shear stress hypoxia cytokine tumor necrosis factor IFN etc lipopolysaccharide growth factor transform growth factor platelet derive growth factor epidermal growth factor etc induce transcription mrna secretion protein act autocrine fashion increase expression expression decrease stimulus additionally enhance mrna stability lead increased sustain expression number ETA ETB receptor cell specific regulate variety growth factor receptor expression influence diverse physical biochemical mechanism role pathologic state difficult define address subsequent part article airway localize primarily bronchial smooth muscle low expression epithelium cellular subset epithelium secrete include mucous cell serous cell Clara cell et binding site find bronchial smooth muscle alveolar septae endothelial cell parasympathetic ganglia expression airway previously note regulate inflammatory mediator eosinophilic airway inflammation see severe asthma associate increase level lung secretion act autocrine paracrine fashion ETA receptor lead increase transepithelial potential difference ciliary beat frequency exert mitogenic effect airway epithelium smooth muscle cell endothelin cause bronchoconstriction intact airway potent denude bronchi constrict equally endothelin suggest considerable modulation effect epithelium vast majority bind site bronchial smooth muscle etb receptor bronchoconstriction human bronchi inhibit ETA antagonist augment etb receptor agonist cultured airway epithelium secrete equal amount equivalent affinity etb receptor bronchoconstriction mediate endothelin stimulate release multiple cytokine important airway inflammation enhance secretion histamine leukotriene increase prostaglandin release inhibition cyclo oxygenase effect bronchoconstriction suggest despite release multiple mediator mediate bronchoconstriction direct effect activation ETB receptor ETA mediate bronchoconstriction important follow etb receptor desensitization denudation airway epithelium occur airway inflammation late sustain airway response inhaled antigen interestingly heterozygous knockout mouse reduction peptide airway hyperresponsiveness remodel suggest decrease modulate bronchoconstriction activity functional mechanism possibly decrease basal production inflammatory airway disease characterize bronchoconstriction hyperreactivity influx inflammatory cell mucus production edema airway thickening important role process cause immediate bronchoconstriction increase bronchial reactivity inhaled antigen influx inflammatory cell increase cytokine production airway edema airway remodel airway inflammation lead increase synthesis possibly perpetuate inflammation bronchoconstriction release cultured peripheral mononuclear bronchial epithelial cell asthmatic increase inhibition ETA combine ETA etb receptor additionally lead decrease airway inflammation antigen challenge animal suggest proinflammatory effect airway mediate ETA receptor asthma increase circulating level adult asthmatic normal level attack acute attack elevate serum level correlate inversely airflow measurement decrease treatment Bronchoalveolar lavage BAL asthmatic similarly increase concentration cause bronchoconstriction inversely correlate forced expiratory volume FEV cultured epithelial cell find equal amount BAL fluid asthmatic relative increase ETB versus ETA receptor expression asthmatic patient contribute increase bronchoconstriction asthmatic increase patient nocturnal asthma decrease BAL level treatment acute asthma exacerbation steroid beta adrenergic agonist phosphodiesterase inhibitor result decrease BAL immunostaining situ hybridization biopsy specimen asthmatic show increase bronchial epithelium correlate asthma symptom smoking lead increase circulate patient chronic obstructive pulmonary disease absence pulmonary hypertension hypoxemia increase plasma increase urinary instead correlate decrease oxygenation possibly hypoxic release kidney smoker impair mediate vasodilation correlate bronchial hyperresponsiveness contribute pulmonary hypertension implicate pathogenesis bronchiectasis ability promote neutrophil chemotaxis adherence activation Sputum level increase patient cystic fibrosis sputum correlate Pseduomonas infection noncystic fibrosis relate bronchiectasis implicate pathogenesis bronchiolitis obliteran BO characterize injury small conducting airway result formation proliferative collagen rich tissue obliterate airway architecture BO lead cause late mortality lung transplantation increase lung allograft pro inflammatory mitogenic property airway lead speculation involve formation lesion support increase BAL lung allograft vivo gene transfer airway epithelium hemagglutinate virus Japan rat recently result pathologic change distal airway identical see human BO specimen change nonspecific effect hemagglutinate virus Japan attribute presence gene localize airway epithelium hyperplastic lesion alveolar cell pulmonary hypertension rare progressive disease characterize increase normally low pulmonary vascular tone pulmonary vascular remodeling progressive right heart failure implicate mediator change see pulmonary hypertension pulmonary vasculature find primarily endothelial cell less extent vascular smooth muscle cell endothelium secrete primarily basolateral surface cell secretion increase variety stimulus include cytokine catecholamine physical force shear stress decrease prostaglandin oxidant stress Hypoxia report increase effect decrease release endothelial cell receptor pulmonary vasculature lead vasodilation vasoconstriction depend cell type receptor lung ETA receptor abundant localize medial layer artery decrease intensity peripheral circulation etb receptor find medium pulmonary vessel increase intensity distal circulation intimal etb receptor localize large elastic artery distribution receptor important implication understand regulation vascular tone Vascular receptor increase factor include angiotensin hypoxia act vasodilator vasoconstrictor pulmonary circulation generation opening ATP sensitive potassium channel lead hyperpolarization result vasodilation mediate etb receptor pulmonary endothelium hypertensive chronically hypoxic lung increase etb receptor expression augment vasodilation increase etb mediate release inhibit hypoxic ventilation inhibition synthesis lead increase mediate vasoconstriction ETA etb receptor conversely act vascular smooth muscle mediate induce vasoconstriction normal lung cause vasoconstriction primarily activation ETA receptor large conduct vessel lung small resistance vessel lung etb receptor medium predominate responsible induce vasoconstriction interestingly preconstriction pulmonary circulation result shift primarily ETA mediate etb mediate vasoconstriction overall effect vascular tone depend dose pre exist tone lung administration acute hypoxic vasoconstriction result transient pulmonary vasodilation effect dose dependent low dose lead vasodilation high repetitive dose cause vasoconstriction follow initial brief vasodilation role acute hypoxic vasoconstriction lung certain ETA receptor antagonism attenuate hypoxic pulmonary vasoconstriction specie implicate mechanism acute hypoxic response inhibition ATP channel line evidence suggest importance chronic hypoxic pulmonary hypertension increase plasma lung rat follow exposure hypoxia treatment ETA combine ETA etb receptor antagonist additionally attenuate development hypoxic pulmonary hypertension implicate vascular remodeling associate chronic hypoxia mitogenic effect vascular smooth muscle cell implicate animal model pulmonary hypertension increase fawn hood rat develop severe pulmonary hypertension raise condition mild hypoxia monocrotaline treat rat increase form pulmonary hypertension contribute increase vascular tone vascular remodeling interestingly transgenic mouse overexpresse human gene modestly increase lung level develop pulmonary hypertension normoxic condition exaggerated response chronic hypoxia pulmonary hypertension classify primary unexplained secondary cardiopulmonary disease connective tissue disease ie scleroderma hallmark disease include progressive increase pulmonary vascular resistance pulmonary vascular remodeling thickening medial layer small pulmonary arteriole formation complex plexiform lesion circulate increase human pulmonary hypertension primary cardiopulmonary disease level high patient primary pulmonary hypertension lung major source clearance circulation increase arterio venous ratio see primary pulmonary hypertension suggest decrease clearance increase production lung increase lung patient pulmonary hypertension great increase see small resistance artery plexiform lesion correlate pulmonary vascular resistance interestingly treatment continuous infusion prostacyclin result clinical improvement decrease arterio venous ratio possibly decrease synthesis endothelial cell study receptor antagonist treatment primary pulmonary hypertension underway offer hope patient disease inhibit pluripotent peptide effect vascular tone remodeling line evidence suggest importance lung allograft survival rejection peptide implicate important factor ischemia reperfusion injury time transplant acute chronic rejection allograft circulate increase human undergo lung transplant immediately follow perfusion allograft Plasma increase threefold minute remain high hour follow transplantation decline near normal level hour increase correlate increase pulmonary vascular resistance occur hour post transplantation suggest release circulation mediate event BAL fluid recipient lung allograft similarly increase fold remains elevate year post transplant recipient single lung transplant increase fold BAL fluid transplanted lung compare native lung suggest increase graft underlie disease require transplant BAL fluid correlate episode infection rejection cellular source lung allograft unknown expression nontransplante human lung low find primarily vascular endothelium transbronchial biopsy specimen obtain surveillance clinical suspicion infection rejection follow transplantation reveal presence airway epithelium alveolar macrophage occasionally see lymphocyte endothelium pneumocyte localization different surveillance specimen compare infected reject lung change time transplantation study suggest source increase BAL transplanted lung increase number alveolar inflammatory cell de novo expression airway epithelium biologic importance inflammatory cell support observation peripheral mononuclear cell dog mild moderate lung allograft rejection cause vasoconstriction pulmonary arterial ring attenuate ETA blocker bind activity fail transplant human lung suggest bind activity different compare normal lung lung parenchyma bronchial smooth muscle perivascular infiltrate binding decrease small muscular artery pulmonary artery bronchial artery fail transplant suggest role impair vasoregulation transplanted lung reperfusion injury lead cause early post operative graft failure death severe manifestation increase pulmonary vascular resistance hypoxia pulmonary edema lead cor pulmonale death implicate mediator event increase pulmonary vascular resistance observe human recipient lung allograft follow increase circulate fall decrease circulate similar pattern see dog subject allotransplantation conscious dog left pulmonary allograft demonstrate increase rest pulmonary perfusion pressure acute pulmonary vasoconstrictor response hypoxia administration ETA selective combine ETA ETB receptor blocker change resting tone etb receptor mediate hypoxic pulmonary vasoconstriction appear increase allograft recipient study administration mixed ETA ETB receptor antagonist dog reperfusion allograft result marked increase oxygenation decrease pulmonary arterial pressure improved survival compare control animal model ischemia reperfusion inhibitor ECE additionally attenuate increase circulate severity lung injury receptor antagonist completely eliminate ischemia reperfusion injury suggest change vasoactive mediator increase thromboxane decrease prostaglandin decrease contribute increase pulmonary vascular resistance administration donor donor recipient dog lung transplantation reduce pulmonary arterial pressure lung edema inflammation improve survival suggest reduction follow transplantation partly responsible early graft failure treatment donor associate decrease plasma level acute rejection manifest diffuse infiltrate hypoxia airflow limitation lead respiratory insufficiency death BAL increase dog episode acute rejection decrease immunosuppressive treatment acute episode rejection human associate increase BAL chronic rejection allograft manifest BO major cause morbidity mortality long term lung transplant survivor etiology BO follow transplant unclear relate repeat episode acute rejection chronic low grade rejection organize pneumonia discuss early chronic increase see lung allograft contribute bronchospasm proliferative bronchiolitis obliteran bronchoconstrictor smooth muscle mitogenic effect support increase BAL transplanted lung susceptible BO native lung recipient single lung transplant mitogenic effect play role development pulmonary malignancy metastasis lung human tumor cell line include prostate breast gastric ovary colon etc produce importance lie mitogenic effect tumor growth survival suggest blockade ETA receptor result decrease mitogenic effect prostate cancer colorectal cell line receptor tumor cell alter increase ETA receptor downregulation etb receptor tumor increase etb receptor blockade etb result decrease tumor growth tumor cell result altered balance lose ability respond regulatory signal environment additionally protect Fas ligand mediate apoptosis detect immunohistochemistry situ hybridization pulmonary adenocarcinoma squamous cell tumor less extent small cell carcinoid tumor situ hybridization demonstrate similar pattern mrna expression non neuroendocrine tumor receptor find variety pulmonary tumor cell line ETA receptor find small cell tumor adenocarcinoma large cell tumor etb receptor express primarily adenocarcinoma small cell tumor ECE convert big committed step biosynthesis find human lung tumor adjacent normal lung finding combine presence lung tumor suggest possible autocrine loop sustain support growth lung tumor recent study suggest ETA detectable lung tumor gene downregulate compare normal bronchial epithelial cell line propose role lung tumor autocrine factor paracrine growth factor stroma vasculature surround tumor allow angiogenesis tumor angiogenesis necessary continued growth tumor limit oxygen diffusion growth vessel tumor important metastatic potential tumor play important role angiogenesis tumor growth survival available online commentary review report primary research induction vascular endothelial growth factor expression sprouting new vessel tumor surround tissue bind activity find blood vessel vascular stroma surround lung tumor time resection markedly surround squamous cell tumor production augment hypoxic environment find large solid tumor metastasis dependent neo vascularization important mediator phenomenon receptor antagonist useful role treatment neoplastic disease inhibit growth metastatic potential human tumor experimental lung injury different type result increase circulate BAL lung tissue level human increase sepsis burn disseminate intravascular coagulation acute lung injury acute respiratory distress syndrome ARDS increase correlate poor outcome multiple organ failure increase pulmonary arterial pressure increase airway pressure decrease pio clinical improvement correlate decrease level arterio venous ratio increase patient ARDS clear increase secretion lung decrease clearance patient succumb ARDS marked increase tissue immunostaine vascular endothelium alveolar macrophage smooth muscle airway epithelium compare lung patient die ARDS interestingly patient decrease immunostaine endothelial nitric oxide synthase inducible nitric oxide synthase lung ARDS characterize presence inflammatory cell lung act immune modulator increase contribute lung injury induce expression cytokine include tumor necrosis factor cytokine turn stimulate production inflammatory mediator lead lung injury additionally activate neutrophil increase neutrophil migration trap lung hallmark ARDS disruption dysfunction pulmonary vascular endothelium lead accumulation lung water role endothelin formation pulmonary edema uncertain infusion raise pulmonary vascular pressure uncertain increase pulmonary protein fluid transport lung act synergistically mediator lead pulmonary edema fibrosis final outcome variety injurious process involve lung parenchyma final common pathway response injury alveolar wall involve recruitment inflammatory cell release inflammatory mediator resolution reparative phase occasionally disorder result progressive fibrosi lung important initial event lung injury activate neutrophil aggregate release elastase oxygen radical increase neutrophil adherence activate mast cell induce cytokine production monocyte cytokine induce important mediate pulmonary fibrosi transform growth factor tumor necrosis factor profibrotic stimulate fibroblast replication migration contraction collagen synthesis secretion decrease collagen degradation additionally enhance conversion fibroblast contractile myelofibroblast increase fibronectin production bronchial epithelial cell finally mitogenic effect vascular airway smooth muscle play important role initial injury eventual fibrotic reparative process inflammatory event lung line evidence importance pulmonary fibrosi available Plasma BAL level increase idiopathic pulmonary fibrosi lung biopsy patient idiopathic pulmonary fibrosi additionally increase immunostaine airway epithelial cell type II pneumocyte correlate disease activity Scleroderma commonly associate pulmonary hypertension pulmonary fibrosi Plasma BAL increase patient unclear presence pulmonary hypertension pulmonary fibrosi increase level BAL fluid patient scleroderma increase proliferation cultured lung fibroblast inhibit ETA receptor antagonist suggest airspace contribute significantly fibrotic response increase bind report lung tissue patient scleroderma associate pulmonary fibrosi pulmonary inflammatory cell appear prime production cultured alveolar macrophage patient scleroderma lung involvement secrete increase amount response stimulation lipopolysaccharide observation collectively suggest augment release contribute perpetuate inflammatory process Bleomycin induce pulmonary fibrosi animal associate increase expression alveolar macrophage epithelium increase proceed development pulmonary fibrosi use receptor antagonist produce mixed result limit development bleomycin induce fibrosis decrease fibroblast replication secretion extracellular matrix protein vitro decrease lung collagen content vivo show ETA combine ETA etb receptor antagonist bleomycin group observe decrease fibrotic area lung rat follow bleomycin treat mixed ETA ETB receptor antagonist correlate pulmonary fibrosi remain uncertain increase cause consequence lung disease pulmonary fibrosi recently report mouse constitutively overexpress human mouse know develop progressive nephrosclerosis absence systemic hypertension transgene localize lung strong expression bronchial wall lung mouse develop age dependent accumulation collagen accumulation lymphocyte perivascular space observation suggest increase lung play causative role development pulmonary fibrosi discovery year ago evidence accumulate biologic activity potential role variety disease respiratory track compelling evidence causal relationship activity disease complete increase use ECE endothelin receptor antagonist experimental human respiratory disorder help clarify role pluripotent peptide health disease','2001-02-22 00:00:00','http://dx.doi.org/10.1186/rr44',2436,735),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json','Logistics of community smallpox control through contact tracing and ring vaccination: a stochastic network model','BACKGROUND: Previous smallpox ring vaccination models based on contact tracing over a network suggest that ring vaccination would be effective, but have not explicitly included response logistics and limited numbers of vaccinators. METHODS: We developed a continuous-time stochastic simulation of smallpox transmission, including network structure, post-exposure vaccination, vaccination of contacts of contacts, limited response capacity, heterogeneity in symptoms and infectiousness, vaccination prior to the discontinuation of routine vaccination, more rapid diagnosis due to public awareness, surveillance of asymptomatic contacts, and isolation of cases. RESULTS: We found that even in cases of very rapidly spreading smallpox, ring vaccination (when coupled with surveillance) is sufficient in most cases to eliminate smallpox quickly, assuming that 95% of household contacts are traced, 80% of workplace or social contacts are traced, and no casual contacts are traced, and that in most cases the ability to trace 1–5 individuals per day per index case is sufficient. If smallpox is assumed to be transmitted very quickly to contacts, it may at times escape containment by ring vaccination, but could be controlled in these circumstances by mass vaccination. CONCLUSIONS: Small introductions of smallpox are likely to be easily contained by ring vaccination, provided contact tracing is feasible. Uncertainties in the nature of bioterrorist smallpox (infectiousness, vaccine efficacy) support continued planning for ring vaccination as well as mass vaccination. If initiated, ring vaccination should be conducted without delays in vaccination, should include contacts of contacts (whenever there is sufficient capacity) and should be accompanied by increased public awareness and surveillance.',' concern intentional release smallpox prompt extensive preparation improve ability detect respond outbreak smallpox factor contribute public health challenge understanding prepare smallpox include age quality epidemiological datum native smallpox smallpox vaccine difficulty extrapolate datum current population possible terrorist use altered smallpox ignorance terrorist method release relatively high risk adverse event cause smallpox vaccine Centers Disease Control Prevention CDC establish ring vaccination selective epidemiological control strategy contact case identify vaccinate preferred control measure event smallpox outbreak interim plan successful use ring vaccination smallpox eradication campaign logical emphasis case contact immediate vaccination support use attribution success eradication program ring vaccination challenge Health Officers initiate ring vaccination identification case smallpox legitimate concern ability public health practitioner mount quick comprehensive successful ring vaccination program particularly face moderatesized large smallpox outbreak guide preparation effort inform incident decision making attempt identify outbreak characteristic response capacity significantly impact ability ring vaccination control smallpox outbreak determine ring vaccination useful presence mass vaccination campaign analysis use newly develop mathematical model continuous time event drive network simulation model smallpox ring vaccination mathematical model advance understanding smallpox outbreak progress mathematical computer model address question smallpox transmission model appear conclude ring vaccination effective treat response logistic detail model linear treat depletion susceptible epidemic progress appropriate assess control early epidemic number infect small compare number susceptible innovative model Kaplan et al emphasize importance resource limitation logistic smallpox response assume infectiousness begin onset symptom subsequent identification removal separately monitor close epidemiological contact patient great risk easy find vac cinate conclusion highly critical ring vaccination model Halloran et al stochastic discrete time network model omit explicit inclusion response logistic parameter value similar Kaplan et al inclusion residual immunity individual vaccinate prior discontinuation routine vaccination lead favorable view ring vaccination model Bozzette et al assume ring vaccination reduce number transmission focus health care worker explicitly include network structure population response logistic ring vaccination model Eichner explicitly include network structure population logistic ring vaccination use parameter base datum outbreak Nigeria distinguish close casual contact case isolation surveillance contact conclude case isolation contact tracing prevent spread smallpox finally individual base model Epstein et al present scenario illustrate certain alternative pure mass vaccination ring vaccination contact case prevent smallpox transmission small population individual model include homogeneity assumption analyze tracing contact contact available model include network structure explicit contact tracing response logistic limit number available disease control investigator include feature continuous time event drive network simulation model smallpox ring vaccination specifically model develop include follow feature expose individual vaccinate time result race trace ambulatory case smallpox spread disease case hard recognize vaccination individual prior discontinuation routine vaccination provide possibly considerable protection infection result mild case hard detect public awareness lead rapid detection case use model determine factor promote hinder success ring vaccination smallpox outbreak ring vaccination useful presence mass vaccination campaign particular goal paper examine control smallpox contact tracing ring vaccination network model include response logistic history smallpox briefly review relevant feature natural history epidemiology smallpox follow infection variola virus individual exhibit incubation period approximately day typical sudden onset fever malaise accompany headache backache begin initial pre eruptive phase smallpox day individual common form ordinary type smallpox develop characteristic focal rash precede case oropharyngeal lesion fatal case ordinary smallpox death occur tenth sixteenth day symptom survivor scab separate day illness course smallpox vary widely individual different clinical classification develop consideration clinical feature severity smallpox important standpoint mathematical transmission modeling clinical feature affect ease diagnosis case identification severe form smallpox result transmission vaccinated individual develop severe disease utilize modified simplify version classification system develop Rao mathematical model classify smallpox category early hemorrhagic flat late hemorrhagic ordinary modify mild clinical feature severity smallpox different population affect underlie host factor difference viral strain difference infectious dose owe different prevail mode transmission robust precise quantitative estimate effect pre post exposure vaccination result smallpox severity infectivity difference individual exhibit different form smallpox available significance difference reveal sensitivity analysis detail give Appendix additional file vaccinia virus provide substantial protection infection Dixon assess risk infection individual successfully vaccinate year prior exposure infection risk unvaccinated individual smallpox vaccination apply instance individual experience repeat vaccination failure develop severe smallpox exposure probability successful depend vaccination method assume rate addition protection infection vaccination case modify course infection reduce severity vaccine protection wane time individual vaccinate year prior exposure believe half infection probability unvaccinated person protection severe manifestation smallpox Dixon believe vaccine protection component decay different rate purpose paper assume severity smallpox previously recently vaccinated individual follow distribution vaccinated subject see case series observe Rao Madras vaccinated subject develop smallpox mild diagnose special surveillance awareness observe vaccinated case study Rao vaccinated point life exposure exposure smallpox Smallpox largely disease close contact spread primarily face face contact infected person occasionally contaminated clothing individual incubation period smallpox infectious long term carrier exist patient believe infectious follow development oropharyngeal lesion precede rash hour patient believe infectious week rash Dixon believe patient infectious onset acute viremia evidence suggest little transmission occur prior development rash severe case infectious appear mild case believe little infectiousness scab contain infectious material patient consider infectious scab fall practice patient highly infectious scabbing phase importantly patient vaccinate find cause few secondary case severe case hemorrhagic flat smallpox occasionally result considerable transmission owe diagnostic difficulty mild case patient remain ambulant course disease cause considerable spread household family dwelling secondary attack rate unvaccinated susceptible depend time place occasionally approach drier condition believe favor transmission low rate transmission derive tropical region applicable temperate zone number secondary case result give importation Europe vary widely importation yield case occasional large outbreak see mathematically represent course smallpox accord figure distinguish epidemiologically relevant state follow exposure time vaccination afford complete protection disease period day vaccination prevent disease reduce severity disease prior development symptom late vaccination beginning pre eruptive period patient exhibit fever malaise possibly symptom infectious short period prior appearance rash appearance oropharyngeal lesion permit variola transmission week rash time patient infectious succeed stage rash time patient infectious state assume conditional survive wait time stage choose uniform distribution indicate Appendix additional file incubation period time infection Stage derive estimate incubation distribution smallpox base importation case Europe Appendix additional file detail choose sample uniform distribution simple way ensure minimum waiting time state alternative choice possible simulate transmission smallpox smallworld network highly clustered short characteristic path length specifically assume person locate single household transmission rate great household assume fraction population group workplace social group transmission occur low rate unit time household contact finally assume small probability individual transmit infection individual population casual general network structure model number secondary case cause index case completely susceptible population useful index epidemic potential simple example instance individual infect household cause widespread epidemic household transmission sufficiently frequent construct appropriate generalize basic reproduction number model lead highly cumbersome expression choose alternative ad hoc index epidemic potential give scenario interest simulate introduction index case random population size operationally define containment occur final size epidemic case day show discussion figure Smallpox stage simulation model figure Smallpox stage simulation model flat ordinary smallpox rash indicate dot modified mild smallpox suggest potentially great infectiousness hemorrhagic smallpox indicate horizontal line shading detail provide Table nearly case day window differ little day window simulate disease finite duration finite nonrenewe population epidemic extinction occur finite time assume absence specific case investigation presence smallpox symptom prompt patient diagnose assume high diagnosis rate form ordinary smallpox severe flat hemorrhagic form mild form assume individual diagnose household workplace contact investigate detect probability assume high fraction household contact assume traceable assume fraction workplace social contact traceable fraction household contact traceable assume casual contact traceable high contact finding rate plausible examine San Francisco Department Public Health record contact investigation meningococcal disease like smallpox potentially fatal disease rapid intervention prevent mortality morbidity record available December April investigation period result identification household contact contact workplace social contact identify contact model assume identify asymptomatic contact vaccinate quarantine monitor symptom development symptomatic patient isolate treat necessary model intervention include ring vaccination finally include possibility contact symptomatic asymptomatic trace procedure apply contact contact investigate assume uninfected asymptomatic individual visit trace individual diagnose rapidly trace fact individual isolate able continue chain transmission follow previous model assume limited vaccination capability day ring vaccination assume strategy contact tracing trace direct contact diagnose case tracing contact contact diagnose case direct contact contact structure network illustrate Figure observe individual household contact individual individual smallpox case attempt find vaccinate individual household contact case individual case attempt find individual model effect multiple contact finding attempt conservatively sense attempt find individual household contact case contact determined fail attempt maintain failure rate contact tracing look standpoint find individual large household similar consideration apply workplace social group figure Network structure show household join thick line size workplace social group size join thin line small portion network show Individual household contact workplace social contact individual smallpox case household contact high risk acquire smallpox follow workplace social contact individual population low risk casual transmission individual Case investigation individual identify direct contact probability depend contact household workplace social individual identify vaccinate contact contact trace investigation subsequently identify individual analyze model way select Latin Hypercube sample parameter choose uniformly parameter range give Appendix additional file simulate transmission control smallpox determine parameter important contact tracing ring vaccination effective second Latin Hypercube Sample input parameter assume disease control effort inactive parameter simulate smallpox transmission iteratively select transmission parameter new infection result casual random transmission index case result secondary case think plausible historic smallpox suggest secondary case result smallpox parameter set stochastic simulation set determine daily ring vaccination case tracking capacity need contain simulated smallpox epidemic total number case day finally choose parameter value yield moderately large smallpox epidemic index case cause approximately secondary case present illustrative scenario ring vaccination scenario intend complement simulation calibrate historic smallpox characteristic smallpox deliberate release know important realize model case find time determine fraction contact infected model parameter choose yield rapid transmission close contact reality transmission natural smallpox occur sickbe route occur modern setting sense model err considerably caution pessimism determine input parameter important determine total number smallpox case select Latin Hypercube sample size input parameter range indicate Appendix additional file simulate mean number case day population compute partial rank correlation coefficient prcc Appendix additional file input parameter number smallpox case prcc close zero value parameter little relation simulation output prcc close value parameter highly important determine simulation output neglect number index case directly relate number new case parameter PRCC exceed show Table parameter identify important relate density available contact mean household size prior vaccination fraction protection prior vaccination transmission rate infectivity include length pre eruptive infectious period stage figure note speed ring vaccination household tracing delay fast diagnosis awareness outbreak important parameter additionally infectivity mild case appear important parameter explore factor contribute success ring vaccination choose smallpox scenario result severe fast move epidemic absence disease control simulate epidemic considerably severe believe likely present circumstance parameter simulate smallpox epidemic begin case variety level ring vaccination capacity day contact tracing capacity day show Figure figure assume population size epidemic begin infected individual mean household size assume mean size workplace social contact group contact contact trace assume day number contact trace vaccinate result case investigation day probability find workplace social contact assume table assume nonzero diagnosis probability prodromal period individual Figure repeat simulation assume diagnosis prodromal period individual specific surveillance result nearly identical assume contact tracing ring vaccination day find scenario Figure exhibit decontainment assume diagnosis prodromal period assume contact tracing day find scenario show loss containment Figure repeat scenario figure assume diagnosis prodromal period figure illustrate control epidemic parameter identical Figure workplace social group size instead Figure probability find social contact instead Figure case large size workplace social group low contact find probability make necessary high ring vaccination capacity attain high probability contain epidemic average take long eradication finally occur finally figure control epidemic population begin initial infective keep parameter Figure curve correspond indicate number possible ring vaccination day figure show assume sufficient capacity ring vaccination principle capable contain epidemic begin infected individual mass vaccination case justified far large number individual risk inability perform extensive contact tracing Figure compare effect trace contact contact describe Appendix additional file different level ring vaccination capacity thin figure φProb find household contact Delay trace household contact day expand severe smallpox epidemic figure severe smallpox epidemic begin initial case assume possible ring vaccination day household size workplace social group size assume household contact traceable mean delay day workplace social contact traceable mean delay day assume population protection infection result vaccination prior discontinuation routine vaccination assume infection transmit close contact mean time day person infective cause average casual untraceable infection day assume individual infectious day appearance rash week rash individual infectious infectious week rash prodromal period assume diagnosis rate increase factor smallpox know community assume individual contact investigation additional diagnosis removal rate day follow onset symptom reflect enhanced surveillance contact isolation important parameter summarize Table set parameter choice outline Tables Appendix additional file diagnosis time discuss Appendix additional file expand severe smallpox epidemic inadequate ring vaccination show parameter identical Figure workplace social group size instead probability trace workplace social contact instead severe smallpox epidemic control ring vaccination despite large number initial case parameter identical Figure index case inaugurate attack scenario ring vaccination capacity great indicate recommend ring vaccination ultimately halt epidemic begin index case sufficient vaccination capacity available contact finding feasible follow sufficient contact contact red beneficial sufficient contact tracing ring vaccination capacity exist dotted line scenario parameter Figure number contact tracing possible day day contact contact trace scenario direct contact case trace low level ring vaccination day trace contact contact harmful high level day ring vaccination beneficial trace contact contact contact tracing ring vaccination capacity small adequately cover contact case diversion resource contact contact harmful provide sufficient capacity exist trace contact contact help outrun chain transmission line correspond average realization average number infect day plot figure figure illustrate ring vaccination capacity low trace contact contact model yield severe average epidemic ring vaccination capacity large trace contact contact result severe average epidemic contact tracing ring vaccination capacity low cover adequately contact contact addition contact case extension trace contact contact second ring harmful sufficient capacity cover contact contact tracing contact contact beneficial finally figure illustrate considerable variability see simulation simulation figure show simulation contact contact trace stochastic variability realization considerable parameter hold constant variability expect limit ability inference base observation single realization process baseline hazard infection individual large expect naturally occur smallpox examine effect realistic value hazard particular choose different level ring vaccination capacity relative hazard workplace social contact choose value baseline hazard infection vary day mean time infection day day mean time infection half day Table estimate decontainment probability different level ring vaccination capacity Kr relative hazard infection workplace social contact different level baseline hazard infection household contact base replication simulation level scenario index case introduce population size parameter Figure define decontainment mean total number case index case eventually exceed day introduce index case population repeat time report fraction scenario number infection ultimately exceed choose cutoff indicate ultimate escape containment epidemic result show Table support idea ring vaccination easily control introduce smallpox provide sufficient capacity efficacy tracing considerable uncertainty model parameter choose collection parameter value estimate containment probability operationally define few total case result index case day estimate containment probability simulate smallpox epidemic time parameter value compute frequency realization few index case result day window produce slightly small containment estimate parameter set choice difference great maximum difference see mean absolute difference case containment case day window case day thousand scenario choose Latin Hypercube sample analyze indicate choose hazard close contact transmission hazard random transmission guarantee secondary case case occur case attributable random transmission refer set calibrate scenario text having choose collection parameter set consider level different control parameter apply parameter set simulate different control condition control parameter probability workplace social group contact finding choose value parameter household contact finding probability case second control parameter rate diagnosis effective removal community case develop previously identify trace contact initially asymptomatic refer monitored diagnosis rate assume low level correspond mean diagnosis time day onset symptom high level correspond mean time hour onset symptom high level monitored diagnosis rate correspond effectively isolation contact finally assume doubling diagnosis rate beginning widespread community awareness smallpox compute containment fraction different level ring vaccination capacity contact tracing capacity day scenario parameter set choice assign workplace social group contact trace success probability monitored diagnosis rate Appendix additional file contact tracing ring vaccination capacity day perform realization begin index case compute containment fraction fraction show few case day begin index case choice value examine obtain value containment fraction use result distribution figure average containment fraction Figure display minimum value containment figure plot mean containment fraction average containment fraction scenario ring vaccination capacity vary level workplace social group contact finding probability level monitored diagnosis rate initially asymptomatic contact day day low level ring vaccination traceable contact day epidemic contain ring vaccination level near day index case day average containment fraction figure Stochastic variability illustrate plot number infective time multiple replication example simulation exhibit rapid containment smallpox mean number case average simulation influence small number simulation exhibit uncontained epidemic parameter Figure contact contact trace replication mean containment probability figure mean containment probability increase number ring vaccination day increase figure calibrate parameter set choose parameter set realization simulate fraction epidemic contain few case determine average containment fraction plot vertical axis assume household contact find probability diagnosis rate double community awareness epidemic consider high level workplace social contact finding moderate level consider level diagnosis smallpox investigate alert contact high level correspond hour mean delay indicate high contact isolation moderate level correspond day delay indicate contact isolation figure show condition high workplace social contact find probability high contact isolation moderate workplace social contact find probability high contact isolation high workplace social contact find probability contact isolation moderate workplace social contact find probability contact isolation parameter value choose uncertainty analysis calibrate parameter set figure contact isolation refer monitored diagnosis rate rate previously asymptomatic contact subsequently develop disease diagnose table Table minimum containment probability scenario choose figure figure average simulate containment frequency realization scenario figure determine scenario lead low containment frequency plot single bad containment frequency different level ring vaccination capacity condition figure high social contact find probability high contact isolation effective hour delay follow symptom moderate workplace social contact find probability high contact isolation high workplace social contact find probability contact isolation effective day delay moderate workplace social contact find probability contact isolation parameter figure household contact find probability scenario diagnosis rate double onset community awareness figure contact isolation refer monitored diagnosis rate rate previously asymptomatic contact subsequently develop disease diagnose close average conceal fact scenario parameter set choose calibrate uncertainty analysis control remain difficult impossible high level ring vaccination Figure plot single low containment fraction see compute focus single bad scenario reveal different picture show isolation asymptomatic contact high probability find workplace social contact need control smallpox pessimistic parameter choice rapid contact tracing ring vaccination play important role suppress epidemic long take trace contact likely vaccine efficacious opportunity infected individual transmit disease finally locate isolate vaccinate appropriate illustrate possibility figure examine scenario show early figure household size workplace social group size household contact traceable workplace social group traceable average time infection household contact infective give day assume case contact trace quickly day household contact day social contact contact average find slowly day household contact day workplace social contact assume ring vaccination traceable contact possible day scenario epidemic severe containment define likely contact tracing slow fast scenario infection occur average estimate containment probability slow scenario average infection occur estimate containment probability figure illustrate possibility rapid contact tracing decisive importance scenario parameter set choice case parameter set probability trace contact household workplace social low transmission rate high rapid contact tracing difference conversely parameter set smallpox transmission rate low smallpox easily contain slow contact tracing rapid contact tracing harmful overall typical result figure rapid contact tracing important ensure efficacy ring vaccination address question simulate growth smallpox calibrate scenario figure assume initial case figure ring vaccination possible day simulate epidemic assume day find household contact day find workplace social contact simulate epidemic assume take day find household contact day find workplace social contact figure scenario calculate fraction simulation total number case day containment fraction nearly scenario parameter set choice containment fraction small small contact finding time fast fast contact finding equal improve smallpox control illustrate figure scenario parameter set choice examine difference absolute term smallpox contain contain depend factor rapid contact tracing difference hand scenario examine absolute difference containment probability substantial difference containment probability occa fast contact trace figure fast contact tracing improve efficacy ring vaccination assume baseline parameter figure household size workplace social group size household contact traceable workplace social contact traceable ring vaccination available day contact contact trace fast scenario correspond average day delay household day workplace social contact Figure slow scenario correspond average day delay household day delay social contact figure show average realization start index case effect rapid diagnosis public awareness smallpox lead rapid isolation identification play important role eliminate epidemic illustrate scenario figure assume ring vaccination possible day capacity small contain epidemic absence increase surveillance diagnosis black line figure show steeply rise average number case day surveillance public awareness symptom smallpox increase diagnosis rate multiplie baseline diagnosis rate containment possible blue line doubling diagnosis rate red line peak number case low scenario increase diagnostic rate markedly improve ability ring vaccination control epidemic suggest ring vaccination effort accompany increase public awareness surveillance case rapid diagnosis require ring vaccination effective simulate smallpox epidemic calibrate scenario perform realization begin index case compute fraction scenario epidemic contain define early assume change diagnosis rate assume ring vaccinator day contact find probability household workplace social contact Figure assumption scenario epidemic contain total case realization change diagnosis rate uncertainty analysis calibrate scenario base fraction replication show decontainment reveal important parameter predict failure ring vaccination rapid diagnosis find early uncertainty analysis high fraction vaccinate epidemic small household workplace social group transmissibility lower infectivity prior rash rapid diagnosis high rate diagnosis alert individual contribute great containment probability overall increase diagnosis rate assume individual contact investigation individual diagnose quickly subsequently develop symptom transmission assume rapid smallpox assume highly contagious individual infect identify contact tracing infective scenario examine Figure continue surveillance contact essential component effective ring vaccination design control rapidly spread smallpox smallpox contact diagnose quickly non contact containment ring vaccination require contact find probability household social contact unlimited number ring vaccinator available containment guarantee add additional ring vaccination capacity contact finding rate low follow contact insufficient smallpox transmit rapidly contact containable low contact find probability result calibrate scenario parameter set choice explore level contact find probability need contain epidemic define mean few case ultimately result initial case table scenario assume traceable contact follow rapid diagnosis figure rapid diagnosis public awareness increase surveillance lead far effective epidemic control assume baseline parameter Figure average realization epidemic begin index case assume ring vaccination capacity day contact contact black line diagnosis rate case change case identify multipli blue line diagnosis rate increase multipli case identify Figure result substantially few case red line diagnosis rate double multipli case identify result few case quickly hour case arise previously contact person transmit infection choose different level household workplace social contact find probability different level ring vaccination capacity perform replication different scenario Table report fraction scenario replication exhibit containment scenario smallpox highly contagious require high contact find probability ensure containment epidemic transmission prior rash make epidemic control difficult Figure expand smallpox epidemic assume differ level infectivity prior rash add increase infectivity prior rash keep constant infectivity rash assume parameter figure ring vaccination capacity day infectivity prior rash model relative infectivity short day period oropharyngeal lesion prior rash compare infectivity week rash relative infectivity prodromal period relative period prior rash consider scenario relative infectivity entire period infectivity prodromal period prior rash week rash relative infectivity prior rash week rash prodromal period figure value relative infectivity prior rash infectivity week rash prodromal period value figure show increase infectivity prior rash lead large epidemic compare case high infectivity prior onset rash loss containment occur time realization show case low infectivity prior rash scenario infectivity entire prodromal period show loss control realization increase ring vaccination capacity day day result show lead containment realization high infectivity prior rash low infectivity prodromal period case difference infectivity high prodromal period rash case intuitively add additional infectiousness increase number secondary case control difficult result illustrate small increase infectiousness prior rash diagnosis difficult substantially increase difficulty smallpox control finally figure present scenario parameter modify baseline value Figure assume contact tracing ring vaccination possible day line figure specifically assume severe smallpox hemorrhagic flat average take time long diagnose isolate ordinary smallpox case Table containment severe smallpox different level contact finding column assume level probability find household contact probability find workplace social contact number contact tracing ring vaccination possible day column express percentage result probability containment give assumed contact find probability contact tracing capacity containment probability give containment probability contact case trace column contact containment probability contact contact case trace addition contact case second column contact Contacts parameter Figure population prior vaccination protection discontinuation routine vaccination case smallpox mild diagnose contagious case compare baseline finally vaccine completely ineffective case scenario discuss scenario motivate possibility individual severe form smallpox difficult diagnose remain infectious community long despite great degree illness patient patient infectious particular case quadruple mean diagnosis time lead additional replication containment achieve compare baseline assume community awareness smallpox lead relative rate increase diagnosis severe case ordinary case severe form relatively rare addition scenario show figure replicate calibrate simulation assume case contact tracing day possible diagnosis time severe case time ordinary case finally repeat calibrate scenario time assume long diagnosis time severe case make assumption find difference decontainment fraction large result illustrate vaccination prior discontinuation routine vaccination play role smallpox control ring vaccination decontainment scenario prior Transmission prior rash figure Transmission prior rash make epidemic control difficult figure show expand smallpox epidemic assume differ level infectivity prior rash assume parameter figure ring vaccination capacity day infectivity prior rash model relative infectivity short day period oropharyngeal lesion prior rash compare infectivity week rash relative infectivity prodromal period relative period prior rash scenario relative infectivity prodromal period prior rash week rash scenario relative infectivity prior rash week rash prodromal period Figure value scenario relative infectivity prior rash infectivity week rash prodromal period value parameter Figure additional scenario assume ring vaccination con tact tracing possible day contact contact trace parameter identical Figure indicate Figure additional scenario assume ring vaccination contact tracing possible day contact contact trace parameter identical Figure indicate figure show average replication scenario case repeat result Figure reference number parenthesis legend corresponding fraction scenario decontainment occur case assume flat hemorrhagic smallpox case take time long average diagnose ordinary case case assume population prior protection oppose Figure case assume additional individual instead develop mild smallpox develop ordinary smallpox instead Figure case assume vaccine completely ineffective provide protection infection protection exist population result suggest prior vaccination aid control smallpox strictly necessary control scenario replication exhibit containment Figure assume individual protection vaccination prior discontinuation routine vaccination scenario figure assume fraction zero scenario demonstrate smallpox infection absolute term compare Figure lead mild case individual prior protection epidemic difficult contain replication show loss scenario demonstrate containment possible vaccine completely ineffective case isolation isolation contact contact contact contact tracing possible day replication exhibit containment vaccine offer protection contact tracing possible day replication exhibit containment assume vaccine protection efficient ring vaccination sense vaccination deliver eliminate infection comprehensive mass vaccination follow introduction smallpox sufficient eliminate infection figure probability achieve containment define few total case result index case different level ring vaccination vaccination day mass vaccination compare day jurisdiction United States plan vaccinate specifically level ring vaccination mass vaccination parameter set Figure perform simulated epidemic parameter set vertical axis plot fraction scenario simulated epidemic contain compute fraction scenario simulated epidemic contain indicate colored segment small pie chart symbol mass vaccination rate day mean number death average scenario simulation scenario ring vaccination level day respectively population increase mass vaccination level average death occur ring vaccination fall death ring vaccination day available assume population index case mass vaccination level day average death occur ring vaccination death occur ring vaccination day possible mass vaccination rate day containment define achieve replication scenario ring vaccination mass vaccination rate day containment achieve replication scenario result wide range simulated epidemic seemingly small level ring vaccination couple follow substantial effect prevent epidemic spread reduce death smallpox mass vaccination campaign note jurisdiction United States plan mass vaccination campaign reach population day far great mass vaccination level examine interesting note mass vaccination Figure Mass ring vaccination low level mass vaccination program improve substantially addition ring vaccination shaded pie segment represent fraction scenario containment define text realize vertical position pie chart represent fraction calibrate scenario containment achieve fraction population mass vaccinated increase ring vaccination capacity increase probability containment increase paign effective prevent widespread epidemic low level plan feasible rapid mass vaccination rapidly eliminate smallpox transmission model vaccination contact beneficial assume early vaccination yield great probability prevent ameliorate infection result construct simple network model smallpox transmission address question circumstance contribute success ring vaccination campaign design control smallpox analysis focus use contact tracing ring vaccination prevent widespread epidemic follow deliberate release conduct sensitivity analysis base particular reasonable range unknown parameter result consistent prior vaccination model identify prior vaccination ring vaccination capacity significant factor determine spread smallpox unsurprisingly find household size ring vaccination speed particularly important parameter result intuitively plausible contact find probability appear important analysis narrow range value choose illustrate smallpox control present scenario base control moderately severe smallpox epidemic find swift aggressive contact tracing ring vaccination usually sufficient bring infection control provide sufficient capacity vaccination contact contact beneficial result few infected individual rapid elimination infection investigate contact contact allow chain transmission outrun extent ring vaccination capacity small diversion crucial resource away contact harmful contact contact trace vaccinate provide resource divert away contact case increase surveillance isolation contact improved rate diagnosis community awareness play important role smallpox control note case lower diagnosis rate severe case contribute small extent loss epidemic control suggest public awareness campaign include information help public aware spectrum clinical feature smallpox limitation analysis choose explicitly incorporate specific epidemiology health care worker mortuary worker likely expose infected individual smallpox epidemic infect member community see recent outbreak SARS transmission health care worker consider amplify initial attack simply account exposure consider approximate behavior model health care worker household contact likelihood traceable contact ring vaccination contact tracing identify halt chain transmission model disruption smallpox control patient care occur account analysis cause model sense err optimism appropriateness pre event vaccination health care worker responder address analysis scope model analyze effect contact tracing case contact isolation ring vaccination mass vaccination real smallpox epidemic practice control effort unlikely limit strictly vaccinate contact health care worker likely contact isolate case make vaccine available individual believe live near case voluntary basis occur smallpox control effort past vaccination individual harm disease control effort hinder delay diagnosis case investigation vaccination contact result relatively low level vaccination general population beneficial effect prevent epidemic escape control possibility individual vaccinate isolate miss occur individual institution cooperate disease control effort individual simply find analysis suggest ring vaccination need perfect successfully contain epidemic condition high rate infection contact relatively high rate casual transmission high rate contact finding excess increase surveillance contact isolation need contain epidemic finally vaccination individual low risk contracting smallpox cause harm adverse event vaccine model assume death rate vaccination small compare probability death smallpox play essentially role analysis practice individual suspect high risk vaccine complication relatively low risk contracting smallpox simply isolate closely monitor outbreak presence individual population high risk vaccine complication increase death rate outbreak individual unlikely impair containment epidemic primary focus result support ring vaccination epidemic smallpox assume high rate transmission close contact note stochastically severe rapidly transmissible smallpox scenario loss control see result widespread epidemic scenario transmission potential smallpox small loss control scenario occur frequently result show mass vaccination campaign conduct quickly high coverage result loss control model few death smallpox result ring vaccination conduct mass vaccination simulated smallpox epidemic ring vaccination suggest aggressive fast ring vaccination control epidemic smallpox smallpox identify quickly contact vaccinate promptly identify public awareness smallpox prompt identification case major factor smallpox control simulation play role significant ring vaccination find uncertainty transmission mild case household size casual transmission contribute overall uncertainty epidemic size parameter number infection highly sensitive prior vaccination fraction parameter relate infectiousness parameter relate transmission prior rash model combine network structure response logistic result support complement result investigator result support notion prior vaccine protection play important role slow epidemic despite possibility vaccinated individual develop mild case hard identify transmit disease likewise result provide support view ring vaccination play central smallpox control initiate ring vaccination conduct delay vaccination include contact contact sufficient capacity cover contact case accompany vigorous campaign public awareness facilitate rapid identification isolation case assume ring vaccination fast little delay identification case vaccination contact effective nearly household contact find workplace social contact available sufficient capacity effective ring vaccination planning yield system capable meet benchmark able assess number contact vaccination possible day plan place identify contact work individual employer school community representative authority business access information facilitate contact tracing rapidly investigate vaccinate individual field team manage central dispatch important realize highrisk transient unstably house population reliable contact tracing impossible conclusion model present apply important note model suggest ring vaccination contact tracing isolation likely successful find scenario smallpox transmissible relatively transmissible rash epidemic containment require ring vaccination increase public awareness isolation contact tracing contact contact scenario smallpox transmissible epidemic containment possible low contact finding probability simulation suggest contact tracing ring vaccination need perfect succeed uncertainty knowledge behavior bioterrorist smallpox impossible know advance good high contact finding rate mass public awareness lead early identification case isolation contact investigation contact contact conduct maximum effectiveness reduce probability widespread epidemic possibility smallpox uncontrollable ring vaccination mass vaccination preparation wise mass vaccination unavoidable event deliberate release smallpox believe ring vaccination essential case individual recently expose smallpox protect vaccinate promptly contact identify potentially lie immediate future transmission chain standpoint epidemic control far valuable vaccinate individual transmission chain vaccinate person result support idea ring vaccination case isolation case eliminate smallpox mass vaccination support planning mass vaccination vastly costly difficult policy mass vaccination available event explosive epidemic face unknown multiple redundant prep aration appropriate case investigation isolation control smallpox vaccine work mass vaccination useful event explosive epidemic case tracking impossible','2004-08-06 00:00:00','http://dx.doi.org/10.1186/1471-2458-4-34',5689,921),('3bb07ea10432f7738413dff9816809cc90f03f99.json','Debate: Transfusing to normal haemoglobin levels will not improve outcome','Recent evidence suggests that critically ill patients are able to tolerate lower levels of haemoglobin than was previously believed. It is our goal to show that transfusing to a level of 100 g/l does not improve mortality and other clinically important outcomes in a critical care setting. Although many questions remain, many laboratory and clinical studies, including a recent randomized controlled trial (RCT), have established that transfusing to normal haemoglobin concentrations does not improve organ failure and mortality in the critically ill patient. In addition, a restrictive transfusion strategy will reduce exposure to allogeneic transfusions, result in more efficient use of red blood cells (RBCs), save blood overall, and decrease health care costs.',' anaemia common condition critically ill patient RBC transfusion treatment management patient population fact study report critically ill patient receive RBC transfusion laboratory study examine physiological response ie compensatory mechanism body anaemia include follow increase cardiac output decrease blood viscosity capillary change increase oxygen extraction tissue adaptation meet oxygen requirement critically ill patient affect number factor predispose adverse consequence anaemia persistence condition particular concern cause compensatory mechanism patient impaired risk oxygen deprivation vital organ critically ill patient increase risk adverse effect RBC transfusion pulmonary oedema volume overload immune suppression result increase risk infection microcirculatory injury poorly deformable rbc aim present commentary justify statement transfusing normal haemoglobin concentration improve outcome define normal haemoglobin great woman great man evidence literature justify maintain high concentration use RBC transfusion anaemic patient debate adopt transfusion threshold normal transfusion threshold obviously reduce number allogeneic RBCs transfuse goal impress reader transfuse level equal great improve mortality clinically important outcome critical care setting explore reason reduction total number allogeneic blood transfusion beneficial second examine current evidence low transfusion strategy specifically employ Transfusion Requirements Critical care TRICC trial RBC transfusion inherent risk categorize follow transfusion transmit infection immune relate reaction acute delayed haemolytic reaction febrile allergic anaphylactic reaction graft versus host disease nonimmunerelate reaction fluid overload hypothermia electrolyte toxicity iron improvement donor screening procedure laboratory testing dramatically improve safety blood supply currently risk transmit infectious agent blood transfusion range hepatitis virus HIV Canadian Blood Services personal communication important threat blood supply remain new unknown pathogen recently concern focus potential transmission prion RBCs infectious agent long latency period pose particular risk young individual require RBCs multiple trauma victim risk benefit ratio year old trauma victim year life expectancy differ markedly person age year undergo coronary artery bypass surgery summary risk transmit disease blood supply strive use rbc accord good available evidence order ensure good harm patient long stand observation blood transfusion associate immune suppression clinical phenomenon observe renal transplant patient receive blood transfusion dialysis transplant observe repeatedly transplant centre world recently opelz et al report multicentre clinical trial renal allograft recipient receive modern immunosuppressive regimen patient allocate receive allogeneic RBC unit renal transplant year graft survival rate compare patient transfuse datum suggest long term immunosuppressive effect follow transfusion nonleukocyte reduce allogeneic rbc large number study suggest allogeneic transfusion accelerate cancer growth altered immune surveillance altered immune response allogeneic RBC transfusion predispose critically ill transfusion recipient nosocomial infection increase rate multiplesystem organ failure ultimately result high mortality rate study examine association cancer recurrence postoperative infection transfusion provide weak causal inference poor study design lack independence allogeneic RBC transfusion potential complication recent meta analysis combine result seven rct unable detect clinically important decrease mortality postoperative infection add result new rct van de Watering et al meta analysis relative risk allcause mortality confidence interval confidence interval postoperative infection meta analysis exclude positive rct significant statistical heterogeneity introduce study available rct combine ignore heterogeneity relative risk difference postoperative infection study confidence interval available evidence suggest universal prestorage leukoreduction clinical effect range decrease rate infection high risk patient summary despite convince laboratory evidence supportive clinical study clinical significance immunosuppressive effect allogeneic RBC transfusion clearly establish importantly impact leukoreduction programme study large population patient expect significant exposure allogeneic rbc majority complication allogeneic RBC transfusion frequent intensive care setting patient population possible exception pulmonary oedema hypothermia electrolyte disturbance hypothermia electrolyte disturbance occur frequently massive transfusion critically ill patient optimal effective circulatory volume difficult determine consequence pulmonary oedema frequent complication RBC transfusion patient population explain significantly high rate pulmonary oedema patient transfuse threshold restrictive transfusion group versus liberal transfusion group report TRICC trial alternative explanation frequent use rbc result frequent episode transfusion relate acute lung injury liberal strategy group restrictive strategy versus liberal strategy report TRICC trial clinical evidence insufficient definitively establish correlation age rbc transfuse patient mortality laboratory evidence show storage relate change result impairment blood flow oxygen delivery microcirculatory level Marik et al demonstrate association fall gastric intramucosal pH transfusion rbc store long day addition ample laboratory evidence prolong RBC storage adversely affect rbc potentially result generation cytokine alter host immune function study Fitzgerald et al animal model transfusion consistently observe lack efficacy transfuse store rat blood improve tissue oxygen consumption compare fresh cell blood substitute retrospective clinical study test association age transfuse blood duration stay intensive care unit ICU mortality Martin et al observe statistically significant association transfusion aged blood day old increase duration ICU stay critically ill patient patient receive transfusion aged rbc predictor duration stay survivor median age blood predictive duration stay purdy et al demonstrate negative correlation proportion RBC unit give age transfuse survivor increase age rbc patient admit ICU diagnosis severe sepsis investigator note unit likely old recently report study Vamvakas Carven evaluate effect duration RBC storage postoperative pneumonia consecutive patient undergo coronary artery bypass grafting investigator note adjusted increase risk postoperative pneumonia day average increase duration RBC storage transfuse patient study note patient receive large number RBC unit high mortality risk relatively small view collectively significant consider critically ill patient Canada transfuse ICU stay recently physician depend clinical judgement decide haemoglobin level transfuse critically ill patient result significant variation show exist transfusion practice canadian critical care physician largely lack publish datum subject arbitrary haemoglobin level historically threshold transfuse critically ill patient observational study investigate importance anaemia transfusion practice setting large cohort study perform different patient population intensive care coronary artery bypass surgery hip fracture reach different conclusion RBC transfusion particular improve outcome critically ill patient cardiovascular disease increase risk myocardial infarction coronary artery bypass surgery patient Transfusion impact short term long term mortality hip fracture patient small study evaluate relationship anaemia adverse outcome vascular disease patient increase number ischaemic event observe anaemic patient validity study uncertain give decision transfuse patient correlate illness burden patient possible comorbidity adequately adjust study transfusion threshold compare randomized clinical trial clinical setting varied trial randomize patient transfuse basis conservative liberal strategy definition conservative liberal strategy varied actually overlap study trial include patient trial evaluate impact transfusion symptom trial patient undergo elective coronary artery bypass surgery difference perioperative haemoglobin level small event rate low difference outcome second trial patient undergo knee arthroplasty randomly assign receive autologous blood transfusion immediately surgery receive autologous blood haemoglobin level fall dl difference outcome observe trial hip fracture patient undergo surgical repair find difference outcome death record day surgery symptomatic group death occur dl group number patient trial small evaluate effect low transfusion trigger clinically important outcome mortality morbidity functional status Hebert et al report result TRICC trial patient randomize restrictive strategy haemoglobin concentration maintain trigger set liberal strategy haemoglobin concentration maintain trigger date TRICC trial large study investigate parameter group comparable baseline average daily haemoglobin concentration range restrictive group liberal group average number transfusion reduce restrictive group versus rbc patient cardiac event primarily pulmonary oedema myocardial infarction frequent liberal strategy Table examination composite outcome number patient multiorgan failure find substantially increase group result marginally well restrictive strategy group versus Table overall restrictive transfusion group show low day mortality versus Fig Kaplan Meier survival curve significantly different subgroup patient Acute Physiology Chronic Health Evaluation II score Fig addition day mortality versus ICU mortality versus statistically significant consistent trend term absolute value favour restrictive strategy key observation TRICC trial restrictive strategy well worst equivalent liberal strategy good superior liberal strategy juncture preclinical clinical evidence support adoption restrictive transfusion strategy critically ill patient remain divergent view risk benefit treat anaemia patient cardiovascular disease laboratory base study suggest patient cardiovascular disease require high haemoglobin concentration maintain oxygen delivery partially occlude diseased coronary artery study demonstrate anaemia affect contractile function left ventricle rarely show important effect haemoglobin concentration important address underlie pathophysiological cause acute coronary syndrome prove therapy aspirin blocker treat mild moderate anaemia initial step effect RBC transfusion limit increase debate use allogeneic rbc show associate immunomodulation alteration delivery oxygen microcirculation result increase rate infection organ failure clinical study attempt elucidate risk benefit ratio anaemia transfusion cardiac patient small rct examine transfusion practice patient undergo coronary artery bypass grafting conclude conservative approach administration rbc safe recent cohort study suggest anaemia increase risk mortality critical illness follow surgery patient cardiovascular disease patient restrictive liberal transfusion group respectively difference calculate subtract mean value restrictive group liberal group patient lose day mortality rate nonsurvivor consider organ fail date death change MOD score baseline incorporate adjustment death datum Hébert et al study Jehovah witness group refuse RBC transfusion religious ground undergo surgical procedure note mortality significantly increase patient cardiac disease decrease haemoglobin level study patient cardiac disease similar change haemoglobin level show increase mortality accordance result TRICC trial study Hébert et al critically ill patient correlation Critical Care Vol Alvarez et al Meier estimate survival day admission ICU restrictive liberal transfusion strategy group patient datum Hébert et al Meier estimate survival day admission ICU restrictive liberal transfusion strategy group patient APACHE II score datum Hébert et al mortality rate observe investigator find risk anaemia appear decrease RBC transfusion patient cardiac disease patient cardiac disease mortality increase haemoglobin concentration compare anaemic patient diagnosis versus subgroup patient cardiac disease increase haemoglobin value anaemic patient associate improved survival odd ratio increase possible explanation discrepancy TRICC trial observational study attend physician recruit patient study enter patient consider severe cardiac disease hébert et al seek examine restrictive transfusion strategy effective liberal strategy critically ill patient cardiac disease subgroup patient cardiovascular disease TRICC trial investigator suggest haemodynamically stable critically ill patient cardiovascular disease transfuse haemoglobin concentration fall hemoglobin concentration maintain average daily haemoglobin concentration restrictive transfusion group liberal transfusion group patient cardiovascular disease day mortality rate restrictive transfusion group versus liberal group confidence interval difference mortality rate include day versus ICU versus hospital mortality versus significantly different group Kaplan Meier survival curve compare time death demonstrate similar trend group Fig multiple organ dysfunction MOD score entire study period significantly different group versus change MOD score baseline value significantly low restrictive group liberal group versus measure morbidity mortality composite outcome examine patient die give score total MOD score group different change MOD score significantly different baseline versus specific subset cardiac patient ischaemic heart disease discernible difference day day ICU mortality rate nonsignificant decrease overall survival rate restrictive group note patient confirm ischaemic heart disease severe peripheral vascular disease severe comorbid cardiac disease Fig conclusion restrictive RBC transfusion strategy generally appear safe critically ill patient cardiovascular disease possible exception patient experience acute myocardial infarction unstable angina survival day patient ischemic heart disease restrictive liberal allogeneic RBC transfusion strategy group graph illustrate Kaplan Meier survival curve patient ischemic heart disease study group difference mortality patient restrictive group dash line compare liberal group solid line need reduce allogeneic blood transfusion order reduce associate risk firmly establish rbc associate clinically important immune suppression store rbc show cause adverse microcirculatory effect result increase organ failure question time critically ill patient able tolerate low level haemoglobin deleterious effect reduce exposure allogeneic transfusion large rct Hébert et al establish difference mortality rate restrictive liberal transfusion strategy noncardiac critically ill patient investigator able convincing trend liberal strategy fact deleterious term absolute value statistical significance achieve fact difference strategy achieve great importance mean total number transfusion reduce approximately half impact mortality addition finding easily clinical practice question remain TRICC trial laboratory clinical study establish transfusing normal haemoglobin concentration improve organ failure mortality critically ill patient restrictive transfusion strategy reduce exposure allogeneic transfusion result efficient use rbc save blood overall decrease health care cost','2001-03-08 00:00:00','http://dx.doi.org/10.1186/cc987',1888,611),('40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json','From Functional Genomics to Functional Immunomics: New Challenges, Old Problems, Big Rewards','The development of DNA microarray technology a decade ago led to the establishment of functional genomics as one of the most active and successful scientific disciplines today. With the ongoing development of immunomic microarray technology—a spatially addressable, large-scale technology for measurement of specific immunological response—the new challenge of functional immunomics is emerging, which bears similarities to but is also significantly different from functional genomics. Immunonic data has been successfully used to identify biological markers involved in autoimmune diseases, allergies, viral infections such as human immunodeficiency virus (HIV), influenza, diabetes, and responses to cancer vaccines. This review intends to provide a coherent vision of this nascent scientific field, and speculate on future research directions. We discuss at some length issues such as epitope prediction, immunomic microarray technology and its applications, and computation and statistical challenges related to functional immunomics. Based on the recent discovery of regulation mechanisms in T cell responses, we envision the use of immunomic microarrays as a tool for advances in systems biology of cellular immune responses, by means of immunomic regulatory network models.',' past decade highly successful field functional genomic experience huge growth result development dna microarray technology possible time measure RNA expression thousand gene parallel single assay Immune response complex phenomenon supervene genomic immune response ultimately depend expression gene inside variety cell explain function immune system term gene expression cell constitute reductionist approach study immune system term genomic important goal function immune system antigen processing epitopespecific immune response well understand integrate approach take account property immune system quote immunome detailed map immune reaction give host interact foreign antigen immunomic study immunome functional genomic strive identify role gene cellular process paradigm hybridization mrna complementary DNA functional immunomic aim identify role chemical biological target involve immunological process paradigm specific cellular humoral immune response elicit antigen present immune system effort promise great reward term basic understanding immune system term disease diagnosis prognosis design vaccine combat variety human infirmity range pathogenic infection allergy cancer enable technology functional genomic possible significant advance previously sequential genomic include massive effort require identify genome wide dna sequence computational method parse align sequence sequential genomic datum deposit large public access databank GenBank researcher company dna microarray use sequence database probe similar fashion field functional immunomic come age result advance sequential immunomic consist method catalogue chemical biological target capable elicit immune response know epitope computational statistical method available automate large scale epitope prediction subsection addition classical immunoassay ELISPOT tetramer stain flow cytometry enable high throughput identification epitope recently coordinated effort initiate National Institute Allergy Infectious Disease National Institutes Health auspex large Scale Antibody Cell Epitope Discovery Program author paper participate create integrate immunome database resource toolbox epitope prediction method initiative design identify immune epitope select infectious agent information freely available scientist worldwide Immune Epitope Database Analysis Resource IEDB sequence information produce epitope mapping effort carry essential construction immunomic microarray discuss detail lead experimental paradigm similar employ functional genomic computational epitope prediction method immune system recognize antigen binding antibody humoral response cell receptor cellular response self foreign protein cell epitope correspond general feature surface antigen recognition immune system occur continuous linear epitope sequential fragment protein sequence discontinuous conformational epitope compose fragment scatter protein sequence bring spatial proximity protein fold humoral response target mainly conformational epitope represent total cell response make prediction cell epitope hard problem cell response virtually restrict immunoglobulin access epitope cell receptor activation self versus self discrimination rule immune system ideally cell prediction system use surface model protein antigen measure surface energy interaction variable region immunoglobulin correlate cell activation far cell prediction system estimation probability primary peptide sequence present surface protein base hydrophilicity secondary structure cellular response hand restrict binding cell receptor short linear peptide bind specific groove main class major histocompatibility complex MHC molecule present surface cell cell receptor cell bind affinity peptide MHC molecule necessary requirement effective cellular immune response complicate factor highly polymorphic nature MHC molecule display large variability human population experimental affinity datum deposit public database training datum researcher develop statistical method predict MHC affinity give unknown peptide typically computational epitope prediction method scan length pathogen self immunogenic protein sequence take consecutive overlap peptide addition method predict promiscuous epitope one bind large class different MHC allele know supertype Computational MHC bind prediction method essential systematic search epitope situation technique ELISPOT flow cytometry effectively impractical large number peptide assay early MHC bind study identify characteristic amphipathic chemical pattern bind peptide enhanced version system continue use association method today basic MHC bind prediction method base identification specific amino acid commonly find particular position call bind motif peptide bind specific MHC molecule amino acid peptide bind MHC groove normally amino acid MHC amino acid MHC II potentially play positive negative role binding complex method assign positive negative value amino acid position peptide combine value define score predict binding quantitative matrix approach successful limitation quantitative matrix approach consideration influence interaction amino acid different peptide position epitope value interaction difficult measure explore limited fashion combine pair wise interaction peptide position combination independent binding calculate quantitative matrix coefficient derive pair wise interaction provide well prediction quantitative matrix score generate HLA allele study HLA sequence homology allow development virtual quantitative matrix applicable HLA allele general method MHC bind prediction system artificial neural network statistical model Hidden Markov incorporate nonlinear complex interaction MHC molecule peptide epitope evolve datum include training set general method show potentially superior previous one great challenge general method reliable require large peptide bind datum strategy query committee approach compare prediction different training set identify informative peptide bind datum determine biochemical bind assay order efficiently build representative training dataset recently collection peptide bind affinity class molecule public development prediction model greatly accelerate community resource benchmark remark epitope recognition immune system involve receptor ligand problem MHC molecule peptide biological process involve preparation peptide load MHC molecule Epitope prediction system continue evolve step antigen processing presentation require development cognate immune response model combine rational prediction system great current challenge development model incorporate rule cell cell receptor engagement possible outcome growth boom immunomic growth boom genomic take place field begin enter mature stage development similar growth boom immunomic likely place remain year current decade recently search PubMed database Sente follow query immunomic immunomic immunome antigen microarray functional epitope microarray remove irrelevant article output list add article new journal Immunome Research obtain list article cover year present Figure clear interest field accelerate support expectation continue boom growth expect number publication increase exponential pace immunomic microarray commercially available research use immunomic array technology evolve expect immunomic array small number feature eventually design specific clinical diagnostic purpose regularly medical practice clinical application distant future basic function principle microarray technology binding subsequent measurement target biological specimen interest complementary probe array spatially addressable fashion typically planar surface glass slide support array spot contain probe consequence spatially addressable probe large number different target measure single experiment example case dna microarray technology provide basic enable technology functional genomic target fluorescent mrna molecule indicator genomic expression hybridize gene specific DNA probe immobilize planar surface similar fashion enable technology functional immunomic immunomic microarray basic technology immunomic microarray consider detail paper antibody peptide peptide MHC microarray Table summary technology functional immunomic approach include dissociable antibody microarray cell microarray serum microarray peptide library serological analysis cdna expression library SEREX significant technological challenge inherent fabricate immunomic microarray include identification workable surface coating glass appropriate probe concentration target incubation time suitable spot size interdistance microarray consist antibody probe antigen target measure concentration antigen antibody probe specific antibody microarray useful proteomic application proteomic profiling cancer antigen antibody microarray propose post translational functional genomic rationale directly measure protein concentration traditional dna microarray format existence evidence poor correlation concentration mRNA correspond protein reflect post translational modification protein give possibility measure antigen protein associate foreign agent antibody microarray employ functional immunomic application application actually cell display target protein marker surface general rule antibody microarray data drive functional immunomic application problematic main reason approach require production specific antibody set use define antigen target development large number interrogative feature antibody set tremendous challenge humoral response broad MHC restrict cell response highly conformationally dependent develop great variety chemical biological element present biological fluid include small molecule Peptide microarray use opposite technical approach use antigen peptide fix probe serum antibody target format promise functional immunomic application publish study peptide microarray include application autoimmune disease allergy cell epitope mapping vaccine study detection assay serum diagnostic characterization weak protein interaction analysis antibody specificity Peptide microarray essentially correspond highthroughput parallelize ELISA assay reveal repertoire status antigen specific cell antibody response cell response highly dependent cell immune response peptide microarray ideally parallel extensive analysis cell response peptide MHC microarray pioneer study good depict usefulness peptide immunomic technology perform array protein antigen search specific antibody reactivity pattern serum normal health volunteer compare pattern diabete mellitus patient simple classifier design discriminate healthy diabetic patient overall sensitivity specificity subsequent study feature array able identify prognostic signature predict susceptibility healthy animal develop diabete important report describe use panel select peptide protein antigen know recognize autoimmune disease patient autoimmune peptide array study profile autoantibody reactivity pattern rheumatoid arthritis RA patient RA study serum RA patient healthy control recently diagnose RA patient find early clinical prognostic marker able predict patient likely develop severe RA marker identify group patient milder form disease array develop panel peptide derive allergenic peanut protein establish patient respond great diversity peptide peanut epitope bad allergic reaction importance breadth antibody response simian human immunodeficiency virus SHIV experimental nonhuman primate model human immunodeficiency virus HIV demonstrate study perform array peptide derive simian immunodeficiency virus SIV HIV amino acid sequence study indicate reduction repertoire antibody response associate development acquire immunodeficiency syndrome AIDS example underscore great potential peptide microarray identify valuable clinical marker recent technology propose peptide MHC microarray artificial antigen present chip case recombinant peptide MHC complex costimulatory molecule immobilize surface population cell incubate microarray spot effectively act artificial antigen present cell contain define MHC restrict peptide different method propose detect cell express receptor affinity specific peptide MHC complex microarray include simple inspection cell cluster bind spot identification activate cell secrete specific cytokine cytokine specific capture antibodie Peptide MHC microarray correspond high throughput parallelize ELISPOT assay particularly low density cell case direct counting activate cell possible quantitation involve cell count detect cytokine intensity combination cell count score adjust intensity score benefit peptide MHC microarray map MHC restrict cell epitope involve helper regulatory function immune system conjunction peptide base cell epitope microarray study adaptive system figure illustrate functioning peptide MHC microarray Figure peptide MHC microarray depict inset show probe molecule deposit microarray spot figure depict cell bind activate specific peptide MHC complex help co stimulatory antibody cell secrete cytokine capture specific detection antibody finally depict Figure cell excess cytokine wash away bound cytokine reveal fluorescent antibody method employ reveal cytokine peptide MHC spot design take account element peptide MHC complex detection antibody specific particular cytokine want measure element kind cell population helper CTL target incubate microarray choice different element lead vast number immunomic response measure exciting feature distinguish immunomic dna microarray datum possibility measure signal simultaneously determine single feature epitope case dna microarray response value obtain gene sample concentration mrna produce gene note twodye experiment use mrna different sample case peptide MHC chip single epitope generate different response value correspond different cytokine different target cell population different antibody isotype case peptide microarray word case genomic microarray parameter measure level transcription individual gene case immunomic microarray possible measure parameter immune response single epitope instance single cell epitope recognize different isotype immunoglobulin IgE case intensity antibody response measure quality antibody response aspect relevant high IgE titer relation associate allergy opposite high titer relation IgE epitope situation significant case peptide MHC array peptide MHC epitope induce different cytokine response multicolor peptide MHC microarray counterpart multicolor ELISPOT assays currently use know combine effect multiple cytokine essential control immune response describe suggestive term cytokine chord give family epitope want simultaneously measure inflammatory effector anti inflammatory regulatory cell response know associate concentration IFN respectively case spot microarray contain epitope peptide MHC complex use distinct cytokine antibody detection figure result analysis real value profile obtain functional genomic microarray vector value profile profile call multispectral profile section datum analysis adopt term multicolor refer immunomic datum fact term similar setting ELISPOT assays technological challenge mention previously connection antibody peptide microarray complex case peptide MHC microarray fact involve element previous technology presentation peptide antibody detection secrete cytokine technology peptide MHC microarray infancy view simple economical method screen cell repertoire host hold great potential clinical research application peptide MHC microarray technology study correlate protection effect experimental therapeutic cancer vaccine patient melanoma immunize peptide vaccine immune response examine peptide MHC microarray contain seven type peptide MHC epitope probe secrete factor peptide MHC array show sensitivity detect peptide MHC specific cell cell incubate array theory peptide MHC array detect distinct reactive feature reach minimum frequency analysis peptide MHC microarray response pattern demonstrate patient present IFN TNF secretory response specific epitope remain free melanoma complexity statistical analysis regard immunomic microarray datum different level genomic microarray datum total number gene human estimate comparison total number different cell receptor human generate somatic recombination recognize peptide context major histocompatibility molecule estimate order number cell clonotype generate somatic recombination gene estimate order dna base letter alphabet consist nucleotide basis peptide epitope base letter alphabet consist amino acid know involve life process clearly combinatorial complexity case functional immunomic order magnitude high functional genomic addition functional genomic number interrogative feature need build microarray order functional immunomic total number interrogative feature include microarray analysis large estimate follow case peptide MHC microarray analysis MHC peptide bind motif suggest core amino acid peptide sufficient characterization cell epitope total number interrogative feature correspond number possible letter word base letter alphabet fortunately able bind MHC molecule make number interrogative feature manageable addition cell cell process clonal selection leukocyte react react strongly eliminate dangerous clone react self antigen delete anergize process immune tolerance number feature large method reduce number essential method include prediction method immunomic database mention previously connection epitope mapping selection peptide specific genome pathogen allergen self antigen involve human infirmity tumor antigen diabete autoimmune disease additional procedure group screen genome pathogen putative bind peptide compare candidate epitope know host protein sequence case find peptide identical host peptide important critical disease cause pathogen molecular mimicry disease diabetes dengue hemorrhagic fever Guillian Barret syndrome hypothesize result infection induce self reactive pathogenic immune response important state goal immunomic array test possible naïve cell cell clone generate somatic VJ recombination possible combination peptide simply patient blood consider relevant pursuit goal immunomic array identify prime cell reach reasonable level precursor frequency expect biological relevance frequency circulate cell peripheral blood expect biological relevance small contrast cell frequency addition cell bind affinity peptide MHC cell specific bind one affinity current report limit detection peptide MHC immunomic array cell cell theory peptide MHC array incubate purify cell ml blood detect distinct reactive feature reach minimum frequency expect able detect rare clone low frequency cell need requirement overcome large amount blood unreasonable collect ml blood add cell expansion step grow cell population ex vivo incubate array technique cell clonal amplification commonly detect rare population cell flow cytometry ELISPOT apply immunomic study normally human pbmc ELISPOT assay peptide limit detection estimate fold fold sensitive flow cytometry ELISPOT distinct experimentally immunomic microarray ELISPOT cell apc present mixture cell activate close contact APC peptide MHC array spot surface equivalent large define APC completely load specific peptide epitope reactive cell bind affinity adhere specific peptide MHC spot spot author cell compare limit detection array flow cytometry turn similar limit sensitivity approximately cell possible future peptide MHC spot surface improve sensitivity peptide MHC array great current ELISPOT assay great challenge polymorphism HLA gene particular HLA class II combination different alpha beta chain approach useful limit number feature selection specific allele frequently find population broad screening array second approach use supertype prototype HLA molecule compatible set HLA allele approach customization array have different array single allele combine accord HLA type individual test issue set immunomic microarray datum apart availability vector value response profile case peptide MHC microarray statistical challenge reminiscent datum analysis problem engineering field remote sensing different material characteristic vector response call spectral signature case immunomic datum analogous notion spectral signature cytokine profile associate give epitope cell population figure illustration simple technique address datum analysis problem multicolor immunomic datum combine response long feature vector juxtapose individual cytokine response profile epitope caveat systematic correlation feature result feature vector large number feature measure simultaneously microarray technology present challenge hand likely large number irrelevant feature present hand scientist like work small number strong relevant feature diagnostic prognostic panel basis biochemical validation study mechanism involve problem feature selection arise fundamental limitation statistic call curse dimensionality accord existence large number feature necessitate large exponentially large number sample achieve consistent accurate result number patient microarray base study severely restrict factor cost technology difficulty patient enrollment case small number sample available small number feature time consider recommend approach feature selection consider combination feature time classifier design base feature set consideration use estimate probability error performance score application type analysis immunomic microarray datum allow identification set epitope specific immune response associate example distinct disease state feature selection present explosive combinatorial problem example exhaustive selection set feature initial feature total number feature set assess equal initial set feature instead number feature set size search large complexity feature selection especially crucial functional immunomic application case number initial feature consider huge use high performance computing architecture large computer cluster mandatory give set feature select able design classifier take input microarray datum unknown sample generate output predict class label clinical outcome kind infection condition figure illustrate approach hypothetical functional immunomic application case class label correspond control protect patient situation protection achieve immunization attenuate virus vaccine give infectious disease objective identify epitope discriminatory response group prime target rational epitope base vaccine design set feature correspond epitope identify feature selection thousand microarray probe instance imagine situation protect individual present high tnf response epitope high IFN response epitope base response value observe patient note patient correspond point plane linear classifier design classifier correspond simply decision region separate line future unknown patient response value fall upper decision region likely protect patient provide classifier small probability error case response epitope term TNF IFN cytokine characterize immunological memory induce attenuate virus vaccine large response value epitope indicate protect patient note response epitope epitope good discriminator example indicate need consider multivariate combine effect response note Figure apparent error rate number misclassifie sample divide total number sample actual probability classification error future datum typically exceed apparent error rate biology make use mathematical modeling order provide theoretical core biology analogous way mathematical theory provide core physics century success engineering computational methodology physical realm predictive capability mathematical modeling quote predictive mathematical model necessary biology direction predictive science necessary application engineering method translate biological knowledge therapy mathematical computational basis large complexity biological system comparison physical system make urgent application mathematical computational modeling technique dynamical system provide natural language need describe integrated behavior system coordinate action element capable display emerge self order massively disorganized complexity believe fundamental feature life follow describe notion immunomic regulatory network dynamical system model immune regulation important recent development immunology discovery regulatory cell cell suppress immune response help stem runaway inflammatory process avoid autoimmune disease think beneficial suppression activity turn deleterious take advantage pathogen lead chronic abnormal infectious process observe regulatory cell antigenspecific call natural regulatory cell addition exist regulatory cell antigen specific epitope drive give suppressive action epitope drive regulatory cell conjunction promote activity epitope drive helper cell follow immunological response give epitope suppress promote immunological response epitope notion regulatory network arise fundamental concept understand functioning immune system fact human disease result unbalance immune system homeostasis functional genomic dna microarray datum infer genomic regulatory network biological efficiency reason gene expression quantize level multivariate method classification feature selection discuss previous section prove essential inference Boolean binary regulatory network token immunomic microarray datum infer immunomic regulatory network true gene expression quantize epitope response measure immunomic microarray level immunogenic responder inference boolean immunomic regulatory network general case node immunomic regulatory network represent combination epitope cytokine response measure cell population target case node relationship single physical spot immunomic microarray experiment give cell population edge node represent putative regulatory relationship cell respond respective epitope figure depict simple example quantize boolean response peptide MHC microarray conjunction kind cell target helper cell regulatory cell cytotoxic cell Epitope specific helper cell promote response epitope specific effector cell produce actual protective mechanism addition epitope activate regulatory cell suppress effector response epitope produce antiinflammatory response practice model derive microarray datum automatic epitope feature selection determination predictive relationship classifier design example effector response activate presence help epitope absence regulatory response epitope suppress response epitope promote presence response epitope provide negative feedback mechanism relationship represent wiring diagram transition rule depict figure response quantize value epitope total number possible state system figure depict state transition table obtain transition rule figure table determine attractor system state set state system stay long run absence external disruption attractor associate basin attraction contain attractor transient state set state lead attractor context biological system attractor mathematical model homeostasis minimalist example different behavior correspond distinct basin attraction Figure respective attractor indicate dash rectangle see behavior system depend state response epitope help system pass irrelevant transient state tend rest single state attractor absence activity diagram left Figure response epitope help activity system correspond cyclic attractor effector DOI figure example Simple Immunomic Network Consisting Epitopes Epitope promoter specific helper cell epitope suppressor specific regulatory cell epitope produce effector response specific cytotoxic cell promote suppress response epitope negative feedback response turn cyclically diagram right Figure situation correspond modulation effector response regulation inflammatory response mean negative feedback mechanism note cease response epitope system jump basin attraction tend rest state immune response epitope example determine behavior system function master overall immunological response individual immune response epitope slave concept master slave regulatory unit important understanding complex regulatory system fact consider mechanism genomic regulation Michael Bittner unpublished immunomic regulatory network immunomic microarray datum constitute proper validation computational knowledge discovery subtle epistemological issue involve data drive computer base methodology obtain scientific knowledge Karl Popper explain classic book Logic Scientific Discovery scientific theory consist initial irrational act creativity induction follow rigorous logical consequence deduction testing initial hypothesis initial act creativity word scientific hypothesis computational knowledge discovery computer clearly capable irrational creativity consider science maintain answer yes fact initial irrational act creativity involve step experiment design selection patient sample choice statistical methodology settle actual data analysis purely logic deduction machinery mathematical operation prescribe Popper deductive stage computer play critical role facilitate application complex computational method scientist statistician bias fact unavoidable scientific discipline particular term data drive misnomer describe computational knowledge discovery functional immunomic promise great reward term basic understanding immune system disease diagnosis prognosis rational epitope drive vaccine design research basic biology statistical method associate functional immunomic experiment lead advancement medical science public health functional immunomic early stage development review attempt provide coherent vision nascent field speculate future research direction technology discussion compare contrasted immunomic genomic immunomic supervene genomic epistemological sense immunology ultimately depend functioning gene inside cell immunomic independent character property manner cell pattern gene expression define unique cellular property reaction cognate immune system antigen pattern epitope specific response define final outcome exist large collection mathematical model describe immune system propose boolean immunomic regulatory network new mathematical model immune system regulation dynamical system model parameter estimate immunomic microarray datum somewhat similar concept suggest network model cytokine action propose albeit explicit reference large scale immunomic technology regulatory cell response addition immune system regulatory network previously discuss context genomic immunomic regulatory network model useful computational knowledge discovery simulation regulatory mechanism immune system health disease lead advance practical application vaccine design basic scientific understanding immune system','2006-07-28 00:00:00','http://dx.doi.org/10.1371/journal.pcbi.002008',3752,1062),('40e41f0c52b39669dee24e37875d7a9fabc38636.json','Factors affecting translation at the programmed −1 ribosomal frameshifting site of Cocksfoot mottle virus RNA in vivo','The ratio between proteins P27 and replicase of Cocksfoot mottle virus (CfMV) is regulated via a −1 programmed ribosomal frameshift (−1 PRF). A minimal frameshift signal with a slippery U UUA AAC heptamer and a downstream stem–loop structure was inserted into a dual reporter vector and directed −1 PRF with an efficiency of 14.4 ± 1.9% in yeast and 2.4 ± 0.7% in bacteria. P27-encoding CfMV sequence flanking the minimal frameshift signal caused ∼2-fold increase in the −1 PRF efficiencies both in yeast and in bacteria. In addition to the expected fusion proteins, termination products ending putatively at the frameshift site were found in yeast cells. We propose that the amount of premature translation termination from control mRNAs played a role in determining the calculated −1PRF efficiency. Co-expression of CfMV P27 with the dual reporter vector containing the minimal frameshift signal reduced the production of the downstream reporter, whereas replicase co-expression had no pronounced effect. This finding allows us to propose that CfMV protein P27 may influence translation at the frameshift site but the mechanism needs to be elucidated.',' principal mechanism translation accurate decoding triplet codon sequence reading frame mrna specific signal build mrna sequence cause deviation rule virus exploit translational recode mechanism include translational hopping stop codon readthrough program ribosomal frameshifting PRF review regulate protein produce polyprotein positive strand RNA virus PRF prevail recode mechanism essential determinant stoichiometry synthesize viral protein viral PRF signal regulate production replication associate protein depend virus efficiency PRF vary change efficiency inhibit virus assembly replication PRF regard potential target antiviral agent development efficient antiviral drug hinder little know trans acting factor biophysical parameter affect PRF efficiency Database search identify putative frameshift signal substantial number chromosomally encode eukaryotic mrnas PRF impact complexity proteome eukaryotic organism cis act signal slippery heptamer XXY YYZ incoming reading frame indicate downstream secondary structure direct slippage essential event PRF take place accommodation step slippery sequence simultaneous slippage trnas overlap frame XXX YYY sequence heptamer allow postslippage base pairing non wobble basis trnas new frame codon mrna Downstream RNA secondary structure review force ribosome pause place ribosomal site correctly slippery sequence pausing ribosome sufficient PRF occur fact duration halt necessarily correlate level PRF observe Crystallographic molecular biochemical genetic study suggest pseudoknot restrict movement mrna trna accommodation step elongation fill entrance ribosomal mrna tunnel restriction ease unwind pseudoknot allow mrna forward slippage mrna nucleotide backwards chemical agent correspondence address tel fax email Author publish Oxford University Press right reserve online version article publish open access model user entitle use reproduce disseminate display open access version article non commercial purpose provide original authorship properly fully attribute Journal Oxford University Press attribute original place publication correct citation detail give article subsequently reproduce disseminate entirety derivative work clearly indicate commercial use contact antibiotic certain mutation translation apparatus translation elongation factor change translation fidelity kinetic show influence PRF efficiency review parameter know contribute efficiency PRF sequence slippery heptamer downstream secondary structure length sequence spacer cis act signal downstream sequence termination codon vicinity PRF signal kilobase away affect PRF efficiency specific sequence Barley yellow dwarf virus BYDV untranslated region UTR kb downstream slippage site vital PRF stimulating effect achieve formation tertiary structure complementary nucleotide UTR base pair single strand bulge cis act stem loop human immunodeficiency virus HIV show require complex secondary structure instead simple stemloop optimal PRF vivo investigation suggest PRF study carry minimal frameshift signal lead inaccurate estimate stoichiometry synthesize viral protein product infection cocksfoot mottle virus cfmv genus Sobemovirus infect monocotyledonous plant specie barley oats wheat monopartite single strand nt long positive sense RNA genome polyprotein CfMV translate overlap open reading frame orf PRF mechanism study want determine vivo PRF efficiency guide CfMV UUA aac heptamer stem loop structure addition minimal signal decide test effect flank CfMV sequence ability contribute PRF find surround viral sequence promote efficient PRF minimal signal sequence vivo measure dual reporter vector system develop Stahl et al carry expression pattern deletion analysis understand molecular basis observed upregulation addition critically analyse suitability implement experimental system type recode study interesting possibility viral protein produce PRF regulate PRF hypothesis test co express CfMV protein replicase dual reporter vector region CfMV polyprotein orf Figure clone NheI bcli site lacz luc orf dual reporter vector generous gift Dr Rousset Universite Paris Sud France insert sequence region region region amplify PCR template primer introduce nhei bglii site flank end insert NheI digestion remove lacz ORF reintroduce plasmid final cloning step result plasmid name pac pAC pAC inframe control nucleotide add slippery heptamer generate PCR base mutagenesis exsite Stratagene name pac pAC Bm pAC Cm respectively deletion plasmid pAC AB ABm pAC AC ACm pAC BA BAm pac CAm generate target sequence show figure base numbering refer cfmv genome Transcription drive promoter plasmid encode leucine lactamase ampicillin resistance selective marker plasmid transform Saccharomyces cerevisiae MATa dual reporter plasmid inframe control contain bp sequence frameshift region positive control monitor PRF efficiency analyse protein produce PRF lacz Bm Cm Fluc fragment clone inframe terminal tag NT KpnI XhoI site Invitrogen reporter fusion amplify PCR pac pAC Bm pAC Cm template result plasmid name NT Bm Cm protein expression regulate promoter CfMV encode protein terminal end replicase clone Invitrogen translation initiation codon introduce oligonucleotide PCR result plasmid name pyes pyes Control plasmid lack translation initiation codon prepare PCR base mutagenesis pyes pyes repdaug result plasmid verify sequence plasmid encode auxotrophic marker uracil cloning step perform standard protocol plasmid amplify Escherichia coli purify Qiagen column insert verify sequence yeast transformation LiAc method transformant select synthetic minimal define medium SC lack correspond auxotrophic encode Bacteria grow LB medium contain ampicillin yeast cell grow YPD SC medium protein expression promoter repress growth SC medium contain glucose Expression induce replace glucose galactose raffinose reporter fusion express Invitrogen overnight protein fusion purify denature condition Ni NTA agarose Qiagen analyse SDS PAGE gel protein visualize Coomassie stain antisera raise CfMV polyprotein region encode CfMV VPg protein antibody complex visualize horseradish peroxidase conjugate anti rabbit antibody Sigma ECL chemiluminescent reagent pyes pyes rep pyes pyes repdaug co express pAC correspond pac inframe control strain MATa Invitrogen transformant grow overnight SC Leu Ura medium non induce condition inoculate induction medium cell harvest late logarithmic phase expression cfmv protein confirm western blotting polyclonal antisera CfMV ORF protein determine enzymatic activity describe monitor effect cfmv replicase vitro analysis lacz gene pAC Bm cm vector replace PCR amplify Renilla luciferase rluc gene prlnull vector Promega result pacrf plasmid template PCR order add promoter upstream Rluc gene PCR product RNA synthesis RiboMax kit Promega transcript treat DNase Promega purify Qiagen RNeasy column quantify spectrophotometrically integrity transcript check agarose gel vitro translation carry CfMV PRF test control sequence clone galactosidase LacZ firefly luciferase Luc gene dual reporter vector inframe control construct extra nucleotide insert slippery heptamer fuse reporter reading frame translation inframe control result production galactosidase cfmv firefly luciferase fusion translation test construct incoming frame yield galactosidase cfmv fusion PRF produce galactosidase cfmv firefly luciferase fusion identical produce inframe control PRF efficiency calculate firefly luciferase activity galactosidase normalization give formula cfmv polyprotein encode overlap orf PRF sequence region test dual reporter vector activity promote PRF indicate numbering refer CfMV RNA sequence publish wheat germ extract WGE accord manufacturer protocol Promega reaction incubate room temperature min stop ice prior enzymatic measurement cell culture start independent clone grow late exponential phase cell collect centrifugation freeze liquid nitrogen store Bacterial cell lyse sonication yeast vortexing glass bead vol min lysate clear centrifugation enzymatic activity determine immediately total protein concentration measure Bradford protein assay reagent Bio Rad Galactosidase LacZ firefly Renilla luciferase LUC RUC activity measure commercial kit Promega accord manufacturer instruction lacz activity determine colour intensity nm luciferase activity measure relative light unit RLUs luminometer Biohit ThermoLabsystems PRF efficiency calculate normalize firefly luciferase activity follow formula luc activity test ruc activity test activity inframe ruc activity inframe control cfmv motif PRF slippery heptamer UUA AAC stem loop structure nt downstream efficiency PRF direct cfmv cis act signal assay vivo dual reporter vector system figure reporter produce single mrna factor affect stability mrna rate translation initiation similar influence expression reporter variation monitor change activity upstream reporter quantify PRF compare galactosidase normalize firefly luciferase activity derive test construct PRF obtain inframe control identical galactosidase cfmv firefly luciferase fusion produce PRF add nucleotide slippery heptamer Figure similar vector show detect small change recode efficiency result alteration cis trans acting factor insert varied length CfMV polyproteinencoding region introduce reporter figure region bp short represent approximately minimal frameshift signal prove functional vitro long region insert bp start kDa viral genome link protein vpg gene continue end region encode CfMV protein unknown function minimal requirement functional frameshift signal vivo know intermediate bp sequence select analysis characterize bp frameshift cassette derive RNA positive control result HIV PRF efficiency bacteria yeast figure corroborate publish early indicate dual reporter system fully Galactosidase show retain specific activity irrespective terminal fusion important reporter serve control variation abundance translation rate studied mrnas addition change specific activity heterologous fusion cause alteration solubility conformation expose cryptic protease target site reduce stability protein reliable quantification PRF important test equimolar amount fusion produce corresponding test control construct similar enzymatic activity inframe control analogous test construct equal absolute galactosidase activity table comparable result obtain activity normalize total protein concentration datum show result indicate length fusion affect specific activity galactosidase activity pac inframe control comparable activity obtain enzyme fusion datum show support view observe variation galactosidase activity likely result change translatability stability transcript addition pAC Cm inframe control pac pAC ACm show low galactosidase activity compare equivalent test construct table indicate productivity construct reduce take galactosidase fit reporter normalization factor vivo experiment study cfmv frameshift signal generate significant PRF yeast PRF level measure pAC fold high HIV RNA Figure extent PRF direct minimal region yeast level report cfmv minimal frameshift signal vitro contrast early vitro observation long cfmv sequence upregulate PRF vivo yeast level upregulation fold pac PRF frequency fold pac result efficiency close Figure extremely high value compare value publish early cfmv frameshift signal direct PRF low level bacteria yeast figure extent PRF direct region bacteria yeast long region stimulate PRF fold bacteria compare pac region improve PRF program PRF similar level pac percentage pAC pAC identify responsible enhancement PRF vivo deletion analysis carry flank region delete pAC Bm pAC Cm indicate figure generate vector pAC AB ABm pAC BA BAm pac ACm pAC CAm PRF frequency determine yeast figure increase PRF observe deletion construct comparison PRF direct region BA AB region promote PRF efficiently region region AC well region good region word presence nucleotide downstream nucleotide sufficient increase PRF level direct region deletion analysis identify single specific sequence region responsible increase PRF frequency expression pattern test control construct analyse understand basis observed upregulation yeast cassette contain reporter studied intercistronic sequence express purify terminal histidine fusion allow capture terminally intact product affinity purify protein separate SDS PAGE gel visualize Coomassie stain datum show western blotting cfmv specific anti vpg antibody expect galactosidase cfmv fusion terminate end frame test construct detect long transframe galactosidase cfmv firefly luciferase fusion protein present test inframe construct figure Comparison Coomassie stain gel western blot reveal antisera recognize product terminate cfmv encode region well transframe product furthermore small size cfmv specific region NT decrease binding antibody inframe control fusion datum suitable quantitative analysis PRF interestingly additional protein react cfmv specific antisera co purified cell express NT Bm NT Cm inframe control figure size fusion suggest translation terminate approximately site PRF signal putative termination product present cell express test construct correctly terminate frame product western blot mask product close look absolute galactosidase firefly luciferase activity reveal firefly luciferase expression pAC Cm clearly reduce datum show fact expression inframe control comparable corresponding pac test construct obvious firefly luciferase activity normalize total protein set activity pac relative value corresponding value pAC Bm pAC Cm galactosidase measurement table suggest overall translatability pAC Cm mrna reduce extent explain decrease firefly luciferase expression partially light finding extremely high PRF frequency estimate calculate region explain frequent translation termination frameshift signal pAC Cm mrna reduce firefly luciferase activity relation galactosidase PRF assay vitro WGE lacz encode gene suitable vivo study unsuitable reporter vitro determination PRF efficiency big size good agreement observe unexpected product vitro translation program LacZ cfmv luc mrnas datum show renilla luciferase show retain specific activity irrespective terminal fusion decide use Rluc cfmv luc transcript determine PRF efficiency cellfree system verify suitability Renilla luciferase intend vitro experiment describe transcript encode monocistronic Renilla luciferase Renilla luciferase fuse firefly luciferase rluc Cm luc mix different ratio program vitro translation increase concentration transcript encode rluc luc fusion result linearly grow firefly luciferase activity time renilla luciferase activity remain constant show enzymatic activity sensitive terminal fusion figure similar result obtain Rluc Cm luc mrna datum show PRF efficiency determine transcript contain cfmv region correspond inframe control case slightly high PRF frequency obtain vivo nice correlation vivo result enhance PRF observe region effect weak vivo context region differ minimal region capacity program PRF figure ratio CfMV replicase regulate PRF cfmv infection study protein regulate PRF process replicase expression vector co express yeast dual reporter vector contain minimal PRF test inframe control region intergenic sequence pac replicase expression verify western blot analysis Figure faint band have nearly mobility replicase detect cell grow repress condition small size difference protein regard replicase enzymatic activity measure yeast lysate prepare induced culture measurement show comparable level galactosidase sample indicate replicase expression affect stability dual reporter mrna translatability reporter table effect replicase expression monitor compare reporter activity ratio measure cell harbour expression plasmid table co expression CfMV replicase affect normalize firefly luciferase expression LUC LacZ inframe control slightly increase luciferase expression test construct observe contrast expression reduce firefly luciferase expression test inframe construct effect strong presence inframe control normalize firefly luciferase level reach expression measure vector control verify observed difference firefly luciferase production depend study cfmv protein co express dual reporter vector plasmid have translation initiation codon replicase delete pyes pyes repdaug western blot analysis antisera detect protein produce vector datum show obtain LUC LacZ ratio compare measure cell express cfmv protein pyes pyes Rep LUC LacZ ratio measure cell express replicase slightly low ratio calculate cell harbour pyes repdaug plasmid co express pAC co express pac presence lacz ratio pAC reach expression measure cell transform pyes effect expression evident pac inframe control expression reduce LUC LacZ ratio half compare corresponding value measure cell harbour pyes verify CfMV able reduce downstream reporter expression dual reporter mrnas CfMV proportionally strong effect firefly luciferase production inframe control mrnas comparison test mrnas table calculate PRF efficiency increase PRF study affect huge number different parameter easy task determine real ratio protein produce mechanism vivo viral system efficiency PRF essential determinant stoichiometry synthesize viral protein product rigidly maintain efficient propagation virus example frameshifte retrovirus determine ratio structural Gag enzymatic Gag Pol protein play critical role viral particle assembly study capacity cfmv frameshift signal direct efficient PRF analyse vivo dual reporter vector length cfmv sequence clearly affect actual efficiency percent vivo PRF efficiency elevate long viral sequence direct PRF deletion analysis identify specific region solely responsible enhancement downstream sequence nearby far away cis act signal report enhance PRF virus HIV human cell leukaemia virus BYDV frame stop codon show influence PRF frequency vitro retrovirus CfMV study spacer sequence locate cis act signal show high slippage frequency obtain nucleotide important CfMV spacer start UAC partially explain capacity cfmv sequence promote high slippage level study observed enhancement PRF cause sequence immediate vicinity figure co expression cfmv replicase simultaneously minimal frameshift signal construct pAC correspond inframe control pac yeast yeast total protein sample separate SDS PAGE gel transfer pvdf membrane immunocomplexe detect ECL chemiluminescent system cfmv expression verify western blotting antisera raise cfmv replicase expression detect antisera raise abbreviation repress induce pmal vpg kda baculovirus express cfmv replicase slippery sequence indicate CfMV sequence away influence level frameshifte vivo conclude reliable estimate PRF consequently replicase versus frame translation product obtain length viral sequence reality study hamper non quantitative nature western blot analysis presence different polyprotein processing intermediate difference stability end product infected cell overall competence cfmv signal direct PRF high compare related plant virus Potato leaf roll virus BYDV PRF value report virus measure reporter base assay hypothesize reason high efficiency slippery trna asn encode aac triplet CfMV heptamer Equal UUA AAC slippery heptamer measure induce PRF diversity animal virus review low fitness CfMV PRF signal bacteria agreement poor functioning eukaryotic slippery heptamer order XXA AAC prokaryote IBV RNA have identical shifty heptamer show direct PRF similar level bacteria recent study report XXXAAAC heptamer dictate PRF occur slippage adjacent trnas place heptamer irrespective host eukaryote prokaryote inability prokaryotic translation system direct efficient PRF heptamer inherit property prokaryotic tRNA Asn result difference ribosome ribosome pass PRF site frameshifting resumption frame translation termination Transient polypeptide intermediate result pausing ribosome slippery sequence observe IBV virus polyprotein synthesis pseudoknot structure form IBV mRNA cause translational pause fix position upstream secondary structure regardless slippery heptamer present absent base finding study propose certain percent ribosomes stall secondary structure frameshift site inframe control test mrnas yeast lead prematurely terminate product observe inframe control construct pAC Bm unambiguously prove study high frequency termination translation especially frameshift site pAC Cm mrna nicely explain extremely high calculate PRF efficiency factor change translation fidelity kinetic show influence PRF efficiency review autoregulation frameshifting mammalian ornithine decarboxylase antizyme report mechanism allow modulation frameshifte frequency accord cellular concentration polyamine speculate regulation mechanism useful adjust amount replicationassociate protein match requirement different phase viral replication cycle hypothesis study express cfmv protein replicase pAC pac yeast cell galactosidase production remain constant regardless presence absence cfmv protein interfere translation initiation pAC mrnas se expression cause reduction firefly luciferase production especially inframe control replicase production slightly increase firefly luciferase production pAC pac replicase expression faint effect normalize firefly luciferase production PRF conclusion CfMV replicase pronounce effect translation frameshift site co expression non translatable form dual reporter vector verify truly affect firefly luciferase expression protein level propose cfmv protein influence translation frameshift site cfmv interfere viral protein synthesis cfmv infection mechanism specificity possible biological role need elucidate future','2005-04-20 00:00:00','http://dx.doi.org/10.1093/nar/gki521',2823,854),('418f1b85e589f3ac37ffc37075b6e558e510fa74.json','Designing and conducting tabletop exercises to assess public health preparedness for manmade and naturally occurring biological threats','BACKGROUND: Since 2001, state and local health departments in the United States (US) have accelerated efforts to prepare for high-impact public health emergencies. One component of these activities has been the development and conduct of exercise programs to assess capabilities, train staff and build relationships. This paper summarizes lessons learned from tabletop exercises about public health emergency preparedness and about the process of developing, conducting, and evaluating them. METHODS: We developed, conducted, and evaluated 31 tabletop exercises in partnership with state and local health departments throughout the US from 2003 to 2006. Participant self evaluations, after action reports, and tabletop exercise evaluation forms were used to identify aspects of the exercises themselves, as well as public health emergency responses that participants found more or less challenging, and to highlight lessons learned about tabletop exercise design. RESULTS: Designing the exercises involved substantial collaboration with representatives from participating health departments to assure that the scenarios were credible, focused attention on local preparedness needs and priorities, and were logistically feasible to implement. During execution of the exercises, nearly all health departments struggled with a common set of challenges relating to disease surveillance, epidemiologic investigations, communications, command and control, and health care surge capacity. In contrast, performance strengths were more varied across participating sites, reflecting specific attributes of individual health departments or communities, experience with actual public health emergencies, or the emphasis of prior preparedness efforts. CONCLUSION: The design, conduct, and evaluation of the tabletop exercises described in this report benefited from collaborative planning that involved stakeholders from participating health departments and exercise developers and facilitators from outside the participating agencies. While these exercises identified both strengths and vulnerabilities in emergency preparedness, additional work is needed to develop reliable metrics to gauge exercise performance, inform follow-up action steps, and to develop re-evaluation exercise designs that assess the impact of post-exercise interventions.',' state local health department accelerate effort prepare bioterrorism high impact public health emergency activity spur federal funding guidance Centers Disease Control Prevention CDC Health Resources Services Administration HRSA time emphasis guidance expand bioterrorism include terrorism non terrorism event include infectious disease environmental occupational related emergency pandemic influenza locality rarity major public health emergency necessitate use practice base exercise simulate real life experience order develop improve skill assess response capability time Federal Emergency Management Agency FEMA describe level exercise increase complexity informational seminar minimally exercise response capacity simulation mimic reality exercise participant capacity implement emergency response function progression tabletop exercise FEMA describe facilitated group analysis emergency situation practice public health considerable variability tabletop exercise design conduct tabletop exercise structure discussion evolve event unstructured reaction short scenario participant limit public health staff involve representative partner agency organization scenario range simple complex facilitation range minimally directive allow participant assume responsibility manage discussion role play highly directive enable facilitator assure specific question address recognize need exercise public health emergency response enable funding directive CDC HRSA health department implement exercise program exercise program vary goal include build relationship stakeholder train staff evaluate preparedness level variety purpose include identify gap preparedness recommendation improve preparedness identify variation preparedness health department exercise involve diverse group stakeholder involve public health preparedness representative public health health care agriculture emergency medical service despite commonality preparedness response compo nent variety biological threat exercise design single use focus single disease smallpox pandemic influenza novel virus exercise focus attention interaction preparedness goal exercise strategy illuminate strength vulnerability public health emergency decision making response capacity increase utilization tabletop exercise health department accompany parallel increase knowledge sharing lesson learn regard identify common challenge confront health department strategy effective exercise design management literature deal tabletop exercise date consist entirely case study description single exercise single disease paper describe lesson learn public health researcher RAND collaborator process develop conduct tabletop exercise collaboration state local health department implication public health emergency preparedness datum paper come relate project conduct take project involve develop conducting evaluate tabletop exercise state local health department different size structure different state northeast south mid west west region country table participate health department incur expense involvement exercise staff time require participate project California Georgia involve conduct exercise multiple jurisdiction state California Little Hoover Commission bipartisan independent state body ask RAND assess California public health infrastructure key component project describe great detail development tabletop exercise simulate smallpox outbreak exercise conduct seven local health department California Georgia RAND collaborate Georgia Division Public Health Rollins School Public Health Emory University develop conduct evaluate series tabletop exercise focus different biologic agent seven local health department Georgia exercise focus state level remain project fund Department Health Human Services HHS involve participation multiple local health department project involve develop different tabletop exercise template format focus local public health response bioterrorist agent test local health department different state second project involve develop tabletop exercise examine interface local health department health care system hypothetical influenza pandemic exercise test local health department different state great detail structure tabletop exercise tabletop exercise template find exercise focus relate objective training relationship building evaluation structure design tabletop exercise vary project project objective somewhat different key domain cover outline Table level facilitator involvement vary exercise objective extreme facilitator role limit introduce exercise scenario periodically interject update exercise participant encourage lead discussion base respective role agency organization extreme facilitator take active role lead discussion interject question prompt exercise facilitator turn discussion participant occasionally join discussion request clarification participant assure issue critical exercise objective discuss despite difference exercise share common element include evolve hypothetical scenario facilitated group discussion level collective decision make participant emphasize small medium large mild involvement exercise role play participant little intervention direction facilitator moderate involvement exercise role play issue discussion facilitator insert additional probe ensure discussion stay track Active exercise discussion base facilitator ask question identify issue subsequently role local health department recognize initiate response emergency scenario typically begin single case report series case report herald nascent disease outbreak require public health assessment situation exercise internal communication coordination discipline health department communication coordination partner agency organization health care facility emergency medical service agency exercise extend initial response include scenario progress day week outbreak require great interaction local state level authority attention health care surge capacity exercise conclude hot wash participant discuss collective performance identify strength weakness relevant relate performance experience actual outbreak crisis exercise participant prompt develop initial action plan address key vulnerability identify exercise facilitator subsequently generate write Action Report AAR summarize exercise experience highlight observe strength area improvement addition participant complete exercise evaluation form consist series structured semistructured question ask participant discuss learn exercise evaluate aspect exercise structure conduct example participant ask identify key gap preparedness occur exercise identify useful thing learn tabletop exercise observation report base review action report participant evaluation internal team discussion consensus follow exercise debriefing performance health department participate tabletop exercise vary agency agency consistent theme emerge agency regardless structure biologic agent disease discuss nearly agency struggle common set challenge challenge summarize Table describe represent critical dimension outbreak response local health department structured process notify solicit case report health care provider community hospital largely reliable contact information private provider sure mean reach rapidly instance local health department good relation staff local hospital emergency department staff infection control practitioner appear similar work relationship non hospital base practitioner local public health official unsure direct role follow suspect ill patient collect ship clinical sample laboratory testing example frequently confusion responsibility local health department state health department medical personnel hospital collect laboratory sample sample collect confusion responsibility transport sample site local law enforcement surprised find responsible party related issue ability health department realistically generate surge capacity public health workforce investigate respond large event especially encompass multiple jurisdiction state limit state health department ability shift manpower resource jurisdiction health department conduct exercise proactive contact medium wait contact medium begin communicate public consequence passive approach public health official respond defensively early unexpected medium request turn trouble quickly formulate initial message public clear informative alleviated anxiety health department consistently express uncertainty effectively communicate vulnerable underrepresented population group jurisdiction establish relationship community leader organization serve messenger communication channel group site law enforcement EMS personnel present exercise great familiarity group help identify trust community messenger community health department limit language capacity sufficiently familiar community leader communicate effectively group communicate fully effectively response partner law enforcement EMS occupational health risk personal protection challenge local public health particular public health official usually quick notify response partner soon determine event significant response partner case feel public health official slow provide critical information disease question risk action protect result response partner frequently report feeling leave process express concern continue work risk clarify ensure safety job use National Incident Management System NIMS associated Incident Command Structure ICS structure relatively new public health evident exercise nearly health department difficulty decide implement ICS process identify party serve incident commander similar challenge see decision process relate open Emergency Operations Center EOC result exercise local public health official delay take step preferred maintain informal management process approach prefer outbreak progressively large stress informal network outbreak evolve lack clarity local health department hand control state health department responsibility jointly share local state authority federal agency CDC involved exercise state health department surprise level assistance request local health department especially early stage outbreak rare example state health department surprise local health department assume role responsibility local health department regard regardless general consensus local public health participant exercise CDC staff ground help fairly quickly particularly situation bioterrorism consider likely local health department articulate type plan increase medical surge capacity develop alternative care site instance plan unable hold modest scrutiny exercise superficial lack sufficient detail necessary rapid implementation relate issue local health department frequently report local health care worker manage site create example participant note pop tent bed increase capacity pop people staff obtain census available staff member turn challenge health care participant note staff likely double count particularly nurse security officer work institution increase staff capacity use community volunteer include retire medical personnel recognize potential solution staffing shortage prove extremely difficult actually implement public health participant universally recognize importance volunteer learn plan recruit train mobilize large number volunteer vague lack concrete actionable step realistic application real emergency nearly exercise identify number strength participate health department far commonality strength observe area improvement universally observe public health leader staff commit struggle right thing commonly observe strength strong relationship epidemiologist hospital infection control practitioner public health worker emergency coordinator instance prior experience emergency planning response involvement health department emergency service agency coastal area prepare respond hurricane associate strong facile interaction health department official partner agency exercise conduct period year conduct exercise health department employ experimental design assess change time strike performance health department overall improve time compare early exercise local health department appear far sophisticated early internal process relate notification enhance surveillance large outbreak investigation addition end exercise period health department consider plan surge capacity participate hospital explicit plan cancel emergency surgery discharge severely ill patient appear acutely aware challenge assure adequate number staff provide care large number tabletop exercise conduct allow test compare different strategy design conduct tabletop exercise comparison enable modify exercise time build lesson learn previous exercise briefly highlight lesson learn experience exercise design achieve specific objective develop tabletop exercise assumption single exercise achieve multiple objective training relationship building evaluation objective interrelated opportunity exist achieve concomitantly exercise critical define priority objective exercise different objective different implication exercise design example exercise participant outline response flawed problematic exercise primarily focus objective training appropriate facilitator pause help participant think approach hand objective exercise evaluation type facilitator involvement lead participant choose different course action bias overall outcome evaluate take exercise design build relationship link discipline agency facilitator intervention imply participant mistake embarrassing diminish person credibility depend level trust participant exercise realistic possible remain logistically feasible take optimal mix design element represent balance exercise objective logistic feasibility ideal balance assure sufficient realism provide meaningful experience minimize distraction associate necessary artifice exercise scenario departure reality inadvertent scenario develop insufficient attention local routine force participant sidestep usual procedure seemingly minor design error outdated hospital time course disease inconsistent know epidemiology undermine credibility exercise distract participant role disrupt flow exercise desire realistic exercise scenario lead development tabletop exercise scenario issue area base local preparedness need priority ensure participant address important issue area broad mix challenge relate give scenario address simultaneously range conduct epidemiologic environmental investigation implement modifying intervention information available communicate agency communicate political leader public introduce set task exercise scenario way meaningfully exercise relevant capacity unlikely align exercise objective different stakeholder want address different issue area frustrated expectation meet important stakeholder agree limited number priority issue area exercise focus design scenario area example set exercise design focus pandemic influenza preparedness local health department infeasible exercise entire pandemic plan single scenario single exercise develop scenario exercise meeting local stakeholder decide issue area cover exercise issue area include disease surveillance medical surge capacity non pharmacological disease control use antiviral medication scenario customize unfold deal issue area key decision discussion point develop facilitator probe instruction base specific objective exercise design facilitate properly exercise lack focus resolution leave participant wonder exactly accomplish exercise paramount exercise design focus issue area require concrete decision limited period time example exercise deal simulated smallpox outbreak unfold time involve discussion period deal movement restriction participant ask question school close point participant give limited time discuss issue decision facilitator job discussion focus issue area specific hand ensure end discussion participant collectively task exercise design multiple issue area discussion scenario unfold depend goal objective exercise exercise involve narrow wide range potential participant broad inclusion likely realistic inclusiveness need weigh logistic effectively manage large number participant potential adverse effect inclusion example participant comfortable discuss idea take risk make mistake depend room constraint impede exercise process undermine achievement exercise objective solution problem sequentially stage involvement different participant physically separate different group way closely mimic actual situation example conversation involve air uncertainty weigh difficult alternative normally involve limited group people member group comfortable exercise conversation apart colleague agency organization disadvantage approach substantially difficult logistically diminish opportunity people different group gain understanding role approach problem exercise certain participant notably law enforcement join scenario different stage feedback generally critical participant feel stage participation diminish learning teambuilding opportunity solution split exercise participant group allow participate place people similar level responsibility group conclude exercise session bring share learn exercise describe report represent collaboration people familiar local circumstance people outside participate jurisdiction expertise exercise design facilitation test alternative approach exclusively involve local personnel difficult generalize experience value engage people outside participate agency nonetheless impression certain point development facilitation feedback step value involve people personally invest local relationship situation offer seemingly independent advice perspective tabletop exercise provide useful insight strength vulnerability public health preparedness important recognize exercise outcome influence way design conduct exercise describe report emphasize vary dimension public health preparedness reflect difference state local priority prioritize exercise objective example emphasize early response initial report suspect illness emphasize management surge demand health care service likely occur later emergency scenario give intellectual emotional demand participation exercise participant facilitator energetic later early stage exercise scenario affect perceive capacity execute different element response potential gap observed actual preparedness consider interpret action report evaluate exercise utility tabletop exercise tool identify area improvement improvement problem evolve ability evaluate exercise performance hamper lack evidence base constitute optimal performance lack standard assess public health preparedness need qualitative performance measure one quantifiable measure time quantifiable measure range simple checklist Likert rating scale scorecard example series exercise checklist assess performance health department relate surveillance risk communication function fairly consistent observation health department identify gap identify prior exercise actual experience address reason include lack time lack knowledge change conclude exercise have health department prioritize challenge observe exercise develop initial action plan relate priority item important limitation work interpretation recognize nature exercise change time number important dimension include scenario priority objective facilitation exercise designer facilitator attention begin action plan hot wash result variation unable provide numerical tabulation number health department struggle gap display give strength second employ methodology conclusively assess change time certain improvement identify truly reflective improvement inclusion sophisticated health department exercise period doubt case give national emphasis preparedness planning way health department participate later year qualitatively describe improvement furthermore similar observation improvement public health preparedness time period recently report asse potential influence external event hurricane outbreak health department time period work noteworthy exercise conclude Hurricane Katrina strike addition exercise conduct random sample health department finding generalizable health department finally discuss evidence base determine good practice design conduct exercise extremely thin share experience hope help propose recommendation constitute good prove practice develop conducting evaluate tabletop exercise require considerable planning perspective variety stakeholder tabletop exercise identify strength vulnerability emergency preparedness additional work need develop reliable metric gauge exercise performance inform followup action step develop evaluation exercise design assess impact post exercise intervention','2007-05-29 00:00:00','http://dx.doi.org/10.1186/1471-2458-7-92',2462,832),('435baf47b4f78fc06a8376f451993059407d2972.json','Global Surveillance of Emerging Influenza Virus Genotypes by Mass Spectrometry','BACKGROUND: Effective influenza surveillance requires new methods capable of rapid and inexpensive genomic analysis of evolving viral species for pandemic preparedness, to understand the evolution of circulating viral species, and for vaccine strain selection. We have developed one such approach based on previously described broad-range reverse transcription PCR/electrospray ionization mass spectrometry (RT-PCR/ESI-MS) technology. METHODS AND PRINCIPAL FINDINGS: Analysis of base compositions of RT-PCR amplicons from influenza core gene segments (PB1, PB2, PA, M, NS, NP) are used to provide sub-species identification and infer influenza virus H and N subtypes. Using this approach, we detected and correctly identified 92 mammalian and avian influenza isolates, representing 30 different H and N types, including 29 avian H5N1 isolates. Further, direct analysis of 656 human clinical respiratory specimens collected over a seven-year period (1999–2006) showed correct identification of the viral species and subtypes with >97% sensitivity and specificity. Base composition derived clusters inferred from this analysis showed 100% concordance to previously established clades. Ongoing surveillance of samples from the recent influenza virus seasons (2005–2006) showed evidence for emergence and establishment of new genotypes of circulating H3N2 strains worldwide. Mixed viral quasispecies were found in approximately 1% of these recent samples providing a view into viral evolution. CONCLUSION/SIGNIFICANCE: Thus, rapid RT-PCR/ESI-MS analysis can be used to simultaneously identify all species of influenza viruses with clade-level resolution, identify mixed viral populations and monitor global spread and emergence of novel viral genotypes. This high-throughput method promises to become an integral component of influenza surveillance.',' influenza virus cause global economic public health burden annual influenza epidemic result death year United States periodic pandemic result significantly high death toll emergence new influenza virus strain cause antigenic shift result reassortment gene segment include type antigenic drift result continue accumulation mutation gene pathogenic virus jump specie acquire ability infect transmit human pandemic recent outbreak highly pathogenic avian influenza virus HPAI originate Southeast Asia spread globally result death confirm human case February int global emergence virus bring renew urgency effort track spread evolution influenza virus currently rapid method influenza virus diagnosis rely antigen specific antibody probe real time reverse transcription PCR RT PCR analysis matrix gene identification viral species follow subtype specific RT PCR assay determination viral subtype subtype significant intra inter subtype sequence variation method identify subtype likely identify reassortant newly emerge genetic variant current surveillance method provide information relevant track antigenically novel strain emerge year distinguish multiple lineage influenza virus co circulate persist popula tion secondary genome sequence comparison phylogenetic analysis necessary fully understand multiple lineage virus recognize newly emergent influenza variant monitor global spread virus instance analysis human influenza virus sequence reveal major clade influenza virus circulation influenza season difference multiple reassortment event share common gene lineage similar wholegenome study avian influenza virus reveal presence multiple region specific sub lineage HPAI virus Southeast Asia spread Europe Africa develop method base broad range RT PCR follow electrospray ionization mass spectrometry RT ESI MS rapid accurate detection influenza virus subspecie characterization early identification genetic change circulate virus method previously apply detection pathogen human clinical sample unique capability advantage influenza surveillance highthroughput assay incorporate parallel RT PCR reaction follow ESI ms analysis simultaneously survey specie influenza virus provide clade level resolution identify mixed viral population sample detect reassortant facilitate monitoring viral evolution integral component broad influenza surveillance measure breadth coverage resolution offer panel primer describe Methods detail Table test characterize influenza virus isolate collect human avian animal specie despite extensive genetic diversity sample set broad range primer generate amplicon isolate base composition signature distinguish isolate figure isolate show base composition consistent expect signature correspond sub type base bioinformatic analysis exist sequence datum isolate show previously unknown base composition primer loci suggest novel influenza virus type note unknown figure composition signature provide multidimensional fingerprint genome virus determine cluster related specie sub type representation figure show base composition datum derive PA NP gene segment analyze individual axis importantly influenza primer pair figure visualize dimensional plot strong agreement bioinformatic analysis sequence datum GenBank experimental measurement base composition signature figure Human virus cluster independently avian human virus assess utility RT PCR ESI ms assay surveillance influenza virus human population analyze blind clinical sample collect seven year period result compare conventional analysis sample virus culture serology real time RT PCR method sample influenza positive RT PCR ESI MS conventional assay sample positive RT PCR ESI ms sample positive culture real time RT PCR correspond approximately sensitivity specificity influenza positive sample RT PCR ESI MS analysis identify influenza virus influenza virus complete agreement conventional typing method base composition analysis multiple RNA segment enable categorization isolate previously establish clade determine sequence detail show Table influenza sample determine subtype subtype sample distinguish virus probably arise recent reassortment gene virus gene segment virus importantly prediction agree completely serology direct RT PCR analysis segment sample nonetheless base composition analysis NP PA NS gene segment highly predictive type direct analysis segment necessary unambiguous subtyping potential viral reassortment addition identification specie typing RT PCR ESI MS provide quantitative estimate number viral genome copy original patient sample achieve include fix copy internal RNA calibration standard PCR reaction genome copy number influenza sample range low genome copy intermediate copy high copy base sample linear range calibration standard estimate average genome load copy PCR reaction represent average viral load genome material extract original swab mL nasal aspirate sample require serial dilution obtain viral concentration quantifiable spectrum range genome swab influenza positive patient respiratory symptom observe range order magnitude viral RNA shed demonstrate capability RT PCR ESI MS track evolution circulate influenza virus create tree representation influenza virus sequence Genbank figure black describe Methods experimentally determine base composition map tree figure blue Analysis reveal distribution similar sequence derive clade Holmes et al branch terminate clade represent dominant branch year clade branch co circulate time period strikingly clade isolate detect time Table confirm finding describe visual display likely relationship isolate show Figure collectively result demonstrate richness genetic information provide direct RT PCR ESI ms analysis human clinical material figure spectra sample contain mixture parent BCtype AADFAA single nucleotide variation AAHFAA AADFBB respectively provide snapshot endure fit quasispecie contributor potential viral evolution action dynamic range mixed RT PCR ESI ms detection previously determine approximately allow detection viral variant low abundance mixed population figure result obtain clinical sample contain mixture virus limit detection mixed population demonstrate peak truly represent mixed viral population PCR amplicon clone independent colony sequence clone predict mutation correlate measured amplitude low abundance peak total non overlap nucleotide exclude primer region analyze genetic loci target influenza primer correspond influenza virus genome human sample analyze season show evidence mixed viral population loci correspond sample assume mutation rate broad viral genome region analyze PCR ESI MS human virus sample mixed population virus choose molecular method available pandemic influenza surveillance require consideration practical issue broad availability convenience cost throughput scientific issue relevant public health sensitivity breath coverage depth value information provide end spectrum conventional RT PCR test specific primer probe provide highlyspecific sensitive rapid convenient quantitative relatively inexpensive high throughput format provide valuable surveillance information test optimal surveillance exact nature pandemic virus know supplemental nucleic acid sequencing conventional RT PCR base test capable signal appearance new genetic variant potentially demonstrate loss sensitivity single RT PCR test achieve single presence absence analysis limit specific target design discrimination know variant influenza level resolution describe require hundred independent RT PCR reaction end spectrum virus isolation culture method follow complete genome sequencing require prior knowledge virus sequence provide cladelevel resolution highly detailed information virus evolution unfortunately method slow labor intensive expensive low throughput render ineffective public health arena require rapid response work describe novel method employ good property discuss technique supply additional valuable information provide technique example method identify mix black font type determine sequence analysis blue font experimentally determine base composition type red font experimentally determine base composition type season rare sequence type uniquely discernable base composition analysis amplicon analysis subtype produce BC type BC type indicate asterisk population virus viral quasispecie previously illustrate development drift strain co infection circulate strain potential development shift strain recent advance ESI MS bench mass spectrometer enable analysis PCR amplicon sufficient mass accuracy nucleotide base composition count PCR amplicon unambiguously determine approach outline provide important advantage surveillance broad range primer target highly conserve site influenza virus family flank highly variable information rich region amplify sequence highly diverse virus assay measured base composition allow identification viral specie high degree resolution approach select primer pair target core gene segment PA NS NP conserve region span know influenza virus resultant base composition infer influenza virus type silico analysis influenza virus sequence GenBank database show approach detect know influenza virus distinguish specie type overcome limitation directly target segment require specific primer pair know type segment evolve rapidly newly emerge strain readily detect traditional approach second mixed viral population sample differ single mutation present low virus population identify provide early insight viral evolution integral component surveillance information rich result provide throughput consumable cost conventional sequence specific RT PCR assay result influenza sample season northern hemisphere particularly interesting reveal viral evolution single season virus detect appear seed abundant BC type circulate previous season southern hemisphere majority sample identical BC type probably arise single founder previous season southern hemisphere remain sample differ founder BC type additional point mutation target region describe surprisingly mutation occur fix rapidly viral population sample season show evidence mixed population sequence reveal mutation silent transition codon position summary RT PCR ESI ms method capacity provide rapid diagnosis human influenza individual patient respiratory symptom public health surveillance emerge potentially pandemic strain include novel reassortant use RT PCR ESI MS human animal surveillance offer potential new insight viral evolution scale cost previously possible benchtop mass spectrometer capable analyze complex PCR product rate approximately reaction product minute make RT PCR ESI MS technology practical large scale analysis clinical specimen animal surveillance demonstrate influenza virus method provide sensitive detection directly patient specimen specificity approximate sequence level resolution ability quantitate virus shed detect low abundance mixed infection identify new genetic variant prior knowledge viral sequence technology ideally suit monitor emergence drug resistance mutation therapy identify newly emerge antigenic variant Sample process Viral stock sample consist cultured virus stock obtain ATCC prepare analysis Qiagen QiaAmp Virus kit Valencia manual mini spin kit BioRobot kit Robotic base isolation Qiagen MDx Qiagen BioRobot platform clinical swab sample store Viral Transport Media VTM VTM solution ml pass micron filter subject bead beat small lysis buffer result viral lysate follow protocol base analysis multiple influenza sequence alignment pan influenza virus RT PCR primer set develop capable amplify influenza virus specie subtype HxNy different animal host human avian swine distinguish essential molecular feature base composition signature additional primer pair design broadly amplify know member particular specie cross amplify member different specie pan influenza pan influenza primer surveillance panel primer pair Table select comprise pan influenza primer pair target segment pan influenza primer pair target NP PA NS segment pan influenza primer pair target NP segment primer study thymine nucleotide end minimize addition non templated adenosine amplification Taq polymerase sensitivity RT PCR primer pair determine know quantity synthetic calibrant RNA template describe previously primer pair sensitive copy calibrant RNA primer sensitive copy Table follow RT PCR cycling condition min min min min follow cycle second second second anneal temperature increase cycle PCR continue additional cycle second second second RT PCR cycle end final extension min follow hold RT PCR product analyze Ibis universal biosensor platform Ibis Biosciences Carlsbad perform automate post pcr desalting ESI MS signal acquisition spectral analysis datum report describe previously briefly step follow mL aliquot PCR reaction desalt purify weak anion exchange protocol describe accurate mass ppm high resolution fwhm mass spectra acquire sample high throughput ESI MS protocol describe previously sample approximately mL analyte solution consume second spectral acquisition raw mass spectra post calibrate internal mass standard deconvolve monoisotopic molecular masse unambiguous base composition derive exact mass measurement complementary single strand oligonucleotide quantitative result obtain compare peak height internal PCR calibration standard present PCR molecule demonstrate capability RT PCR ESI MS track evolution circulate influenza virus bioinformatic approach develop framework display RT ESI MS result obtain virus complete genome sequence human influenza virus available GenBank analyze total genome include infer phylogeny influenza virus nucleotide concatenate sequence locus analyze RT PCR ESI MS construct nonredundant alignment sequence type base composition determine genome segment locus analyze unique base composition loci assign letter accord decrease number occurrence letter represent common allele identify locus concatenation base composition letter NP PA locus sequence virus available GenBank yield letter code refer base composition type BC type predominant type label AAAAAA topology tree deduce alignment non redundant sequence program dnadist neighbor Phylip package sequence type differ discrete single mutation original computergenerate tree extensively edit graphic tool place label intermediate type tree lieu zero length branch readability improve sort parallel branch chronologically Table Influenza virus primer pair study Genbank reference sequence segment indicate primer sequence identical reference sequence describe Methods limit detection RT PCR primer pair determine know quantity synthetic calibrant RNA template find mb pdf Table Distribution BC type observe influenza positive human respiratory sample unique base composition genome segment locus analyze assign letter code concatenation letter code loci analyze yield BC type sample assign BC type experimentally determine BC type mark RT PCR ESI MS Analysis result compare BC type signature information sequence currently available GenBank close matching strain show right pane column show comparison clade designation describe Holmes et al find mb pdf','2007-05-30 00:00:00','http://dx.doi.org/10.1371/journal.pone.000048',1982,683),('44102e3e69e70ad2a73e753133283334ce1f8736.json','Vaccinia virus infection disrupts microtubule organization and centrosome function','We examined the role of the microtubule cytoskeleton during vaccinia virus infection. We found that newly assembled virus particles accumulate in the vicinity of the microtubule-organizing centre in a microtubule- and dynein–dynactin complex-dependent fashion. Microtubules are required for efficient intracellular mature virus (IMV) formation and are essential for intracellular enveloped virus (IEV) assembly. As infection proceeds, the microtubule cytoskeleton becomes dramatically reorganized in a fashion reminiscent of overexpression of microtubule-associated proteins (MAPs). Consistent with this, we report that the vaccinia proteins A10L and L4R have MAP-like properties and mediate direct binding of viral cores to microtubules in vitro. In addition, vaccinia infection also results in severe reduction of proteins at the centrosome and loss of centrosomal microtubule nucleation efficiency. This represents the first example of viral-induced disruption of centrosome function. Further studies with vaccinia will provide insights into the role of microtubules during viral pathogenesis and regulation of centrosome function.',' intracellular bacterial viral pathogen evolve numerous mechanism appropriate exploit different system host life cycle order facilitate spread entry exit host Cudmore et al Finlay Cossart Dramsi Cossart case virus well study example exploitation actin cytoskeleton vaccinia virus exit infected cell Cudmore et al Vaccinia virus large dna virus genome kb encode open reading frame orf close relative variola virus causative agent smallpox Johnson et al Massung et al Vaccinia virus morphogenesis complex process occur cytoplasm infected cell result formation intracellular mature virus IMV intracellular envelop virus IEV IMV consist viral core dna protein envelop membrane cisterna derive intermediate compartment Sodeik et al IMV core contain ve major protein Vanslyke Hruby Jensen et al protein associate membrane virus particle Jensen et al Betakova et al depend virus strain cell type proportion IMV enwrap membrane cisterna derive trans Golgi apparatus rise IEV particle Schmelz et al date iev speci protein Roper et al Duncan Smith Parkinson Smith Payne Norrby Shida Engelstad et al Isaacs et al hirt et al identi ed study recombinant virus show play important role IEV assembly Blasco Moss Engelstad Smith Wolffe et al Wolffe et al Roper et al Sanderson et al Ro Èttger et al Vaccinia virus think leave cell fusion outer iev membrane plasma membrane rise extracellular envelop virus EEV Morgan Payne Blasco Moss cell associate envelop virus CEV remain associate outer surface plasma membrane Blasco Moss complex vaccinia infection process actin cytoskeleton dramatically reorganize numerous actin comet like tail induce IEV particle Cudmore et al Ro Èttger et al actin polymerization drive force IEV particle propel actin tail contact plasma membrane extend outward facilitate infection neighbouring cell Cudmore et al addition vaccinia infection result stimulation cell motility loss contact inhibition change cell adhesion Sanderson Smith Sanderson et al Vaccinia virus induce cell motility subdivide cell migration extension neurite like projection dependent microtubule Sanderson et al dependence neurite like projection formation microtubule suggest microtubule cytoskeleton play role life cycle vaccinia virus recently vaccinia protein microtubule show require ef cient IMV dispersion Sanderson et al furthermore absence vaccinia actin base motility cell cell spread occur ef cient Wolffe et al Sanderson et al suggest additional transport mechanism exist give observation wonder microtubule cytoskeleton function life Vaccinia virus infection disrupt microtubule organization centrosome function EMBO Journal Vol pp cycle vaccinia virus report microtubule cytoskeleton complex play important role early stage vaccinia infection later infection cycle loss centrosome function accumulation viral encode microtubule associate protein map result dramatic rearrangement microtubule cytoskeleton Vaccinia localization vicinity MTOC depend microtubule complex Indirect labelling show post infection majority vaccinia virus particle concentrate area coincide centre microtubule aster figure examine localization microtubule dependent infect cell pre treat nocodazole depolymerize microtubule absence microtubule virus particle distribute cytoplasm figure accumulation virus particle area centre microtubule aster suggest microtubule minus end direct motor involve establish position virus location examine possibility infect cell overexpresse dynamitin act dominantnegative function echeverri et al find cell overexpresse dynamitin virus particle accumulate centre microtubule aster cytoplasm occur absence microtubule compare figure Figure vaccinia morphogenesis involve wrap host membrane possible effect nocodazole dynamitin virus localization fact disruption intermediate compartment Golgi apparatus reagent Burkhardt et al independent experiment show case cell infect absence microtubule Golgi apparatus vaccinia virus particle disperse cytoplasm co localize figure secondly vaccinia particle remain vicinity microtubule organize centre MTOC Golgi microtubule disrupt treatment brefeldin figure similar result obtain marker vaccinia galactosyltransferase Golgi intermediate compartment datum show take datum indicate microtubule cytoskeleton require localization newly assemble virus particle vicinity MTOC vaccinia infection formation functional IEV IMV microtubule dependent give requirement microtubule vaccinia localization subsequently examine localization role morphogenesis different intracellular form vaccinia virus IMV IEV electron microscopic examination cell infect presence nocodazole clear IMV particle morphologically indistinguishable control form figure IMV particle assemble absence microtubule wonder number reduce form infectious integrity intermediate compartment depend microtubule Burkhardt et al address question independent virus stock prepare presence absence nocodazole simplify interpretation datum recombinant vaccinia virus mutant unable form IEV Blasco Moss nal concentration virus particle produce Vaccinia use abuse microtubule cytoskeleton determine method Joklik particle ml presence microtubule particle ml absence microtubule fold decrease number virus particle form absence microtubule particle form infectious datum infectious IMV form absence microtubule find evidence IEV formation base electron microscope examination cell infect presence nocodazole figure observe IMV particle partially wrap tran golgi membrane probably process abortive iev formation figure give datum examine indirect low amount IEV particle form absence microtubule nd evidence colocalization IEV protein marker vaccinia particle form presence nocodazole Figure find evidence IEV formation base ability nucleate actin tail Figure IEV particle assembly involve wrapping Golgi apparatus Schmelz et al examine effect disrupt membrane compartment brefeldin nd evidence IEV formation base co localization IEV protein marker virus particle actin tail cell infect presence brefeldin Figure brefeldin treat cell IEV membrane protein require assembly observe endoplasmic reticulum tran Golgi Figure summary datum indicate microtubule cytoskeleton require ef cient IMV assembly essential iev formation course experiment obvious Golgi apparatus progressively disperse infection co concominantly disruption microtubule network Figure analysis show infection normal morphology microtubule cytoskeleton replace morphologically aberrant microtubule form vary common absence discrete MTOC Figure aberrant form broadly classi ed type cell disorganized microtubule network microtubule randomly orient Figure ii cell microtubule form ring nucleus cytoplasm Figure iii cell long projection consist microtubule bundle figure quanti e appearance different morphological form ve independent infection experiment cell count time point experiment figure small compact cell represent post infection respectively microtubule cytoskeleton morphology evident include analysis post infection virus particle assembly occur normal aster microtubule con guration disrupt replace majority cell microtubule obvious organization MTOC Figure cell microtubule ring cell long projection time point figure infection proceed microtubule progressively disrupt bundle figure observation obvious connection disruption change actin microtubule cytoskeleton figure reorganization microtubule network occur cell infect vaccinia deletion mutant actin tail datum show effect vaccinia virus infection reorganization microtubule cytoskeleton observe cell line examine RK Swiss vary degree datum show datum vaccinia infection result severe disruption normal morphology microtubule cytoskeleton formation microtubule bundle loss organization MTOC vaccinia infect cell strongly reminiscent phenotype observe cell overexpresse MAP weisshaar et al Togel et al overexpression map stabilize microtubule examine microtubule cytoskeleton infected cell resistant depolymerization nocodazole cold treatment Figure case suggest virus genome encode viral protein MAP like property identify viral protein exhibit microtubule bind property perform microtubule co sedimentation assay extract prepare uninfected vaccinia infect cell figure initial experiment reveal intact virus particle extract prone pellet absence microtubule make identi cation viral map impossible avoid problem prepare extract cell infect presence rifampicin drug inhibit vaccinia virus particle assembly affect viral protein expression Moss et al Tan McAuslan morphological effect vaccinia infection microtubule cytoskeleton presence absence rifampicin datum show Comparison protein present pellet microtubule co sedimentation assay reveal number additional prominent minor band present extract prepare vaccinia infect uninfected cell figure co sedimentation assays Vaccinia use abuse microtubule cytoskeleton perform presence nocodazole coldtreate extract reveal majority additional band disappear absence microtubule identify viral protein co sediment microtubule perform gel protease digestion follow analysis result peptide MALDI mass spectrometry approach identi e number potential vaccinia encode map structural protein dna bind protein associate viral core Vanslyke Hruby Jensen et al Klemperer et al conserve poxvirus genome unknown function figure associate microtubule vivo mediate binding viral core microtubule vitro available antibody examine localization infected cell associate microtubule vivo addition essential role virus core Vanslyke Hruby Jensen et al negative control examine localization core protein identi ed prominent kda protein pellete absence microtubule figure indirect immunouorescence analysis show associate microtubule presence absence rifampicin figure expect associate viral particle datum show contrast observe association microtubule regardless xation condition localize viral factory viral particle respectively datum show interestingly associate microtubule co localize subset acetylated microtubule figure association virus particle microtubule raise question role microtubule bind activity infection wonder protein mediate interaction incoming viral core microtubule beginning infection core virus particle release cytoplasm start infection cycle Ichihashi Vanderpasschen et al Pedersen et al examine possibility investigate puri ed viral core bind microtubule vitro find viral core able bind microtubule protease treat core show association figure pre incubation puri ed viral core antibody speci cally inhibit interaction viral core microtubule figure contrast IgG antibody inhibitory effect figure take datum suggest MAP like property play role mediate interaction incoming viral core microtubule dramatic rearrangement microtubule cytoskeleton occur vaccinia infection unlikely attribute exclusively action associate subset microtubule figure furthermore loss microtubule organization precede detectable association microtubule occur post infection wonder vaccinia infection disrupt centrosome function give loss microtubule aster con guration infection Figure microtubule nucleate centrosome animal cell examine vaccinia infection affect tubulin critically require process Stearns Kirschner observe tubulin labelling centrosome greatly reduce early post infection figure result obtain infect cell stably express protein tubulin Khodjakov Rieder addition centrosomal centriolar component pericentrin Nap Nek centrin reduce centrosome centriole vaccinia infect cell figure furthermore reduction centrosomal marker require viral protein synthesis level affect cell infect presence cycloheximide datum dramatic reduction tubulin centrosome imply vaccinia infection perturb centrosome Vaccinia use abuse microtubule cytoskeleton function test hypothesis examine centrosome vaccinia infect cell nucleate microtubule follow depolymerization nocodazole find post infection reduction tubulin microtubule nucleation centrosome inef cient compare uninfected control indicate vaccinia centrosome function Figure late time post infection microtubule nucleation ef ciency centrosome low datum show follow nocodazole washout microtubule eventually repolymerize cytoplasm infected cell display organization MTOC control compare figure size virus particle unlikely cell diffusion suggest movement require interaction host cytoskeleton previous datum show vaccinia virus disrupt hijack actin cytoskeleton facilitate movement intracellular envelop form vaccinia virus Cudmore et al Ro Èttger et al infected cell Sanderson Smith Sanderson et al datum describe vaccinia use subsequently disrupt microtubule cytoskeleton infection cycle clear experiment previous observation Ulaeto et al microtubule require maintain integrity Golgi apparatus turn require IMV wrap form IEV Schmelz et al contrast IMV assemble absence microtubule albeit reduce level microtubule require IMV assembly require complex virion accumulation vicinity microtubule aster envisage minus enddirected microtubule dependent movement IMV particle site assembly viral factory MTOC complex enhance possibility wrap Golgi apparatus subsequent iev formation recently show IMV protein microtubule require ef cient IMV dispersion viral factory Sanderson et al absence mature IMV particle accumulate periphery virus factory subsequently wrap form IEV presumably unable microtubule Sanderson et al microtubule dependent accumulation vaccinia vicinity MTOC analogous microtubule dependent movement require herpe simplex virus adenovirus reach site replication nucleus Sodeik et al Suomalainen et al Leopold et al case protein Vaccinia encode protein co sediment microtubule analysis pellet vitro microtubule co sedimentation assays perform protein extract vaccinia infected inf uninfected uninf cell twice pellet load control assays perform absence microtubule nocodazole protein co sediment microtubule present extract infected cell indicate asterisk identity protein determine gel proteolysis MALDI mass spectrometry indicate associate incoming nucleocapsid interact intermediate chain cytoplasmic dynein ye et al report incoming nucleocapsid pseudorabie virus alphaherpe virus closely relate associate dependent microtubule movement nucleus Kaelin et al interaction mediate protein minor essential component capsid co localize microtubule accumulate MTOC Kaelin et al accumulation MTOC consistent possible interaction dynactin motor complex know localize MTOC echeverri et al surprising base observation pseudorabie microtubule involve establish infection cycle cytomegalovirus CMV virus varicella zoster virus herpe virus clear example microtubule dependent virus movement establishment infection incoming human foamy virus HFV dependent microtubule presumably minus end direct microtubule motor nuclear replication site Saib et al absence protein expression HFV Gag protein associate viral genome accumulate centrosome microtubuledependent fashion prior nuclear import Saib et al centrosomal accumulation Gag protein HFV appear unique class retrovirus similar localization report human immunode ciency virus HIV retrovirus hand Gag protein murine leukaemia virus HIV show interact microtubule plus end direct kinesin motor vitro vivo Kim et al Tang et al suggest additional role exist microtubule motor outward movement virus particle vaccinia virus particle able reach cell periphery absence actin base motility image Wolffe et al Sanderson et al Sanderson et al Ro Èttger et al suggest viral particle microtubule Sanderson et al microtubule dependent motordriven movement virus particle represent ef cient mechanism achieve peri nuclear localization require facilitate entry nucleus establishment infection provide excellent way newly assemble virus particle reach cell periphery facilitate continued spread infection datum vaccinia virus use microtubule cytoskeleton achieve peri nuclear localization microtubule Golgi organization disrupt later infection process interestingly CMV report disrupt microtubule cytoskeleton Golgi organization infection cycle Avitabile et al fish et al disruption microtubule network rst sight appear bene cial virus actually hinder viral spread enhance extensive virus assembly spread cell periphery occur time microtubule cytoskeleton Golgi organization disrupt secondly disruption microtubule organization overcome potential microtubule motor anchor effect MTOC allow viral spread periphery occur easily lastly formation long projection mm support extensive microtubule bundle provide means achieve long range spread virus particle Sanderson et al clear disruption reorganization microtubule cytoskeleton vaccinia virus mediate combine effect viral protein MAP like property loss microtubule organize function MTOC true disruption centrosome function remain establish late infection microtubule organize bundle nucleus MTOCorchestrated organization Avitabile et al identi cation viral protein MAP like property unique vaccinia virus tegument protein co localize microtubule infected cell induce microtubule bundle express uninfected cell Elliott example viral map base vivo localization Fig Vaccinia core bind directly microtubule vitro Puri e viral core label DAPI green bind rhodamine label microtubule red absence xation bind microtubule observe core pre treat protease pre incubate antibody protein contrast pre incubation puri ed viral core control IgG antibody protein inhibit interaction microtubule scale bar mm Vaccinia use abuse microtubule cytoskeleton vitro association microtubule protein murine coronavirus Kalicharran Dales movement protein tobamoviru Heinlein et al aphid transmission factor mosaic virus Blanc et al protein pseudorabie virus Kaelin et al spike protein rotavirus Nejmeddine et al protein vesicular stomatitis virus VSV Melki et al identi cation viral core protein MAP like protein unexpected give previously characterized role viral morphogenesis Vanslyke Hruby interaction microtubule vivo vitro microtubule bind datum suggest potential mechanism association viral core microtubule envisage viral core release cytoplasm beginning infection Ichihashi Vanderpasschen et al Pedersen et al bind directly microtubule manner analogous adenovirus nucleocapsid work require determine incoming core fact MTOC dynactin complex use complex anchor microtubule loss centrosome function enhance disruption microtubule cytoskeleton infection loss microtubule organization MTOC precede detectable association microtubule occur post infection Vaccinia induce loss centrosomal protein inhibit cycloheximide indicate viral protein expression require disruption centrosome microtubule nucleation activity knowledge vaccinia virus infection represent rst example virus induce disruption centrosome function predict similar effect mechanism vaccinia virus disrupt centrosome require study clear understand molecular basis disruption provide important insight regulation stability centrosome function currently subject intense research Ohta et al Lane Nigg Karsenti cell ATCC infect wild type vaccinia virus strain Western Reserve WR vaccinia deletion mutant Blasco Moss Parkinson Smith multiplicity infection plaque form unit cell describe previously Ro Èttger et al Nocodazole dissolve dimethyl sulfoxide DMSO brefeldin dissolve ethanol add culture medium nal concentration mm mg ml respectively state non treat control equal volume DMSO ethanol add cell transfecte myc tag dynamitin expression construct echeverri et al infect later WR subsequently xed postinfection experiment describe repeat time following antibody kindly provide anti tubulin Dr anti centrin Professor Sanders Salisbury Paoletti et al anti anti Professor Fry et al anti myc anti Dr Fu Èllekrug et al antibodie correspond vaccinia protein Professor Vanslyke Hruby Professor Klemperer et al Dr Rodriguez et al Ro Èttger et al addition following antibody obtain commercial source anti tubulin Amersham International UK anti acetylate tubulin Sigma USA anti tubulin Sigma anti pericentrin anti BAbCO USA rabbit IgG Sigma Actin visualize label phalloidin derivative Molecular Probes xe methanol paraformaldehyde mm PIPES pH mM MgCl mM EGTA follow Triton permeabilization fixed cell process view image record describe previously Ro Èttger et al cell pre incubate mM nocodazole medium depolymerize microtubule prior infection vaccinia Nocodazole keep medium infection equal volume DMSO add control post infection cell scrape medium sediment centrifugation min cell membrane disrupt nucleus remove centrifugation result post nuclear supernatant centrifuge sucrose cushion min virus pellet resuspend mm Tris pH virus collect centrifugation min resuspend mm Tris pH store concentration virus elemental body determine od measurement Joklik fig Vaccinia infection reduce centrosome microtubule nucleation ef ciency uninfected cell microtubule nucleate centrosome nocodazole washout time indicate contrast infection vaccinia microtubule nucleate inef ciently centrosome image collect identical camera setting allow comparison uorescence intensity centrosome insert adjust Figure facilitate visualization weak tubulin centrosomal labelling arrowhead indicate position centrosome scale bar mm vitro microtubule bind assay Puri ed EEV particle prepare describe previously Ro Èttger et al subsequently prepare virus core follow method Cudmore et al rhodamine label microtubule prepare accord Hyman et al Vaccinia virus core incubate rhodamine label microtubule buffer contain mm taxol min room temperature DAPI add subsequently nal concentration mg ml label virus core finally mixture dilute antifade solution mg ml catalase mg ml glucose oxidase mm glucose view xation Proteinase trypsin treatment core particle prior incubation microtubule perform describe previously roo et al Anti control IgG antibody incubate puri ed core room temperature prior incubation microtubule cell extract microtubule co sedimentation assay extract HeLa cell infect uninfected control maintain presence mg ml rifampicin prepare describe previously Ro Èttger et al extract clari ed centrifugation min cytochalasin add nal concentration mg ml depolymerize actin lament endogenous tubulin extract polymerize step procedure extract supernatant supplement protease inhibitor mM MgGTP mm taxol incubate min room temperature subsequently additional mm taxol add mix reaction incubate min control taxol add stage microtubule polymerization inhibit addition nocodazole nal concentration mm maintain extract experiment follow microtubule assembly mg extract reaction dilute fold buffer contain protease inhibitor mm taxol centrifuge sucrose cushion contain protease inhibitor mm taxol min microtubule pellet solubilize sample buffer analyse gel proteolytic cleavage perform automatically describe Houthaeve et al Genomic Solutions Cambridge peptide obtain analyse Bruker REFLEX MALDI mass spectrometer Bruker Analytik Germany Jensen et al protein identi ed peptide mass ngerprinting Jensen et al program PeptideSearch PeptideSearch post infection nocodazole add culture medium nal concentration mM depolymerize microtubule postinfection cell wash time warm medium remove nocodazole washed cell incubate medium nocodazole indicate time initiate microtubule polymerization wash warm phosphate buffer saline PBS immediately xe parallel sample remove time point rinse ice cold PBS xed process con rm complete microtubule depolymerization initiation microtubule assembly uninfected control HeLa cell treat process identical fashion number image integrate identical camera setting allow direct comparison infected uninfected sample experiment','2000-08-01 00:00:00','http://dx.doi.org/10.1093/emboj/19.15.3932',3047,819),('4994fa72322bbf19120592304d92629226948d8e.json','Rapid Identification of Malaria Vaccine Candidates Based on α-Helical Coiled Coil Protein Motif','To identify malaria antigens for vaccine development, we selected α-helical coiled coil domains of proteins predicted to be present in the parasite erythrocytic stage. The corresponding synthetic peptides are expected to mimic structurally “native” epitopes. Indeed the 95 chemically synthesized peptides were all specifically recognized by human immune sera, though at various prevalence. Peptide specific antibodies were obtained both by affinity-purification from malaria immune sera and by immunization of mice. These antibodies did not show significant cross reactions, i.e., they were specific for the original peptide, reacted with native parasite proteins in infected erythrocytes and several were active in inhibiting in vitro parasite growth. Circular dichroism studies indicated that the selected peptides assumed partial or high α-helical content. Thus, we demonstrate that the bioinformatics/chemical synthesis approach described here can lead to the rapid identification of molecules which target biologically active antibodies, thus identifying suitable vaccine candidates. This strategy can be, in principle, extended to vaccine discovery in a wide range of other pathogens.',' human Plasmodium falciparum pf infection dramatic public health problem today approximately percent world population risk malaria Malaria cause million acute clinical case million death annually ninety percent malaria death occur sub saharan african country young child pregnant woman urgent need malaria vaccine vaccine discovery general particularly malaria empirical process fact protective antigen bear structural physico chemical sequence relate characteristic allow identification protective humoral response recognize characteristic antigen antigenicity immunogenicity accessibility addition lack surrogate marker protection render vaccine discovery process difficult time consume spite constant impressive progress molecular biology technique antigen identification expression vaccine discovery labor intensive make approach fastidious costly poorly adapt highthroughput screening proper protein folding solubility remain limitation numerous case overcome bottleneck manufacturing identification fragment protein possible target protective immune response constitute scientific technical challenge address challenge combine bioinformatic chemical peptide synthesis functional protection assay focus search helical coil coil motif general exhibit folding problem target effective antibody current malaria vaccine candidate eg pathogen lead malaria vaccine contain predict helical coil coil region unpublished result choice base follow consideration helical coil coil motif bear characteristic seven amino acid residue repeat abcdefg hydrophobic residue locate position hydrophilic residue generally motif easily identify bioinformatic analysis secondly important know characteristic ahelical coil coil domain take separately protein frequently readily fold stable oligomeric structure thirdly reason helical coil coil fragment frequently recognize conformational dependent antibody similarly elicit antibodie reactive structurally native epitope addition domain short residue rapidly produce chemical synthesis furthermore antibody anti parasite biological activity designate correspond fragment potential novel vaccine candidate develop assess focus Pf parasite erythrocytic stage target protective antibody describe straightforward rapid procedure base bioinformatic analysis helical coil coil motif peptide synthesis screening Pf genome generalize sequence profile identify protein contain putative helical coil coil motif proteome transcriptome datum assess molecule express Pf parasite erythrocytic stage combine analysis assessment identify segment associate stage display putative helical coil coil motif high probability score Table helical coil coil fragment general amino acid long present protein different one chemically synthesize HPLC purify long peptide amino acid contain helical coil coil domain synthesize antigens table select antigen test ELISA assay reactivity panel sera obtain adult donor Burkina Faso Tanzania Colombia respectively surprise helical coil coil fragment antigenic prevalence responder varied greatly Tables manner protein identify length varied amino acid peptide high prevalence responder ELISA mean od value select study variation recognition panel sera difference genetic background host parasite likely distinct malaria transmission condition region high level recognition helical coil coil motif explain fact take separately protein fragment readily fold stable structure aqueous solution circular dichroism cd study select peptide associate biological activity Tables indicate predominantly assume helical conformation water Peptides Figure exhibit cd pattern characteristic high helical content remain peptide cd profile similar show peptide Figure intermediate show Figures characteristic partial helical organization analyze size exclusion chromatography fplc column peptide present elution profile exhibit chymotrypsin ribonuclease mw respectively cd size exclusion chromatography result suggest peptide adopt helical coil coil structure need unambiguously ascertain NMR ultra centrifugation study test biological activity peptide specific antibody purify affinity chromatography serum pool obtain Papua New guinean adult serum pool test ELISA assays peptide antigenic table peptide specific antibody purify positive serum pool test ELISA antibody react parasite native protein infected red blood cell show IFAT figure Table reactivity restrict blood stage antibody react sporozoite stage datum show reactivity peptide specific show IFAT competition assays correspond peptide figure specificity antibody obtain investigate detail particularly peptide contain glutamic acid sequence know generate cross reactivity malarial Glu rich protein cross reaction systematically investigate affinitypurified antibody peptide result thatwith exception antibody preferentially recognize peptide antibody affinity purify specific correspond peptide Table determine non specific antibody binding solid phase adsorb antigen responsible rare cross reactivity detect ELISA competition assay perform end bind antibody solid phase adsorb antigen compete increase concentration homologous cross react peptide homologous peptide compete good peptide have sequence similarity figure high concentration Figure include short Glu rich peptide derive Cterminus peptide figure finally pattern recognition peptide sera test differ markedly datum show confirm result specificity antibody correspond peptide antibody correspond select peptide test direct cell mediate anti parasite activity clinical experiment show Antibody Dependent Cellmediated Inhibition ADCI falciparum malaria represent mechanism control parasitemia clinical manifestation human peptide specific antibody prove able induce strong intermediate low monocyte dependent parasite kill table absence monocyte direct effect antibody parasite growth observe effect range observe antibody african adult high natural protection know malaria peptidespecific human affinity purify antibody functionally effective show ability react parasite protein inhibit parasite growth vitro functional assay peptide specific antibody elicit natural exposure parasite induce protective mechanism effective malaria sixteen peptide target ADCI positive antibody control immunize mouse table elicit intermediate high antibody response recognize parasite protein infected erythrocyte determine IFAT figure Table see human antibody recognition restrict blood stage sporozoite negative IFAT assay datum show IFAT competition assays correspond peptide Figure anti peptide mouse antibody positive IFAT specific homologous peptide sequence relate peptide table peptide choose propensity form ahelical coil coil induce production antibody recognize epitope present native protein improvement immunogenicity structural specificity remain peptide achieve future short elongation terminal end stabilize helix suggest Cooper et al Lu Hodges use adjuvant genetic polymorphism current vaccine candidate major limitation vaccine development available genotype study peptide sequence parasite isolate worldwide origin indicate limited polymorphism plasmodb unpublished sequencing datum peptide dna sequence deletion entire heptad repeat short region preserve potential helical coil coil formation regard structural feature cellular location prediction protein correspond peptide select ADCI assay table protein contain pentapeptide conform PEXEL consensus position amino acid sequence conform location know active PEXEL motif Materials Methods membrane segment GPI anchor protein contain signal sequence fourteen protein predict cytoplasm nucleus mitochondria peroxysome Table prediction sub cellular localization protein take caution gene annotation constantly update protein trafficking parasite complex fully elucidated investigation require determine actual localization correspond antigen predict localization priori surprising molecule able trigger adci activity recent study show addition merozoite surface protein soluble protein release time schizont rupture equally effective trigger adci provide define epitope case helical coil coil heptad repeat molecule express trans membrane domain export parasite host cell membrane molecule present cytoplasm mature schizont release burst schizont trigger antibody cross link Fc receptor monocyte achieve Pf parasite killing conclusion approach combine genome wide search bioinformatic helical coil coil protein motif chemical synthesis lead rapid identification development new malaria vaccine candidate fact approach straightforward easy scale vaccine formulation comprise mixture peptide single construct epitope principle strategy extend discovery protein vaccine candidate complex pathogen Pf genome bioinformatic analysis generalize sequence profile method pftool package search short helical coil coil domain coil coil profile construct alignment amino acid sequence correspond know coil coil domain profile contain heptad repeat analysis cut level profile choose test perform sequence database protein know structure subsequently coil coil figure immunofluorescence microscopy analysis Pf parasite peptide specific antibody acetone methanol fix schizont merozoite react human peptide specific affinity purify antibody obtain peptide table sera mouse immunize peptide table Grey bright field image blue staining indicate DAPI nuclear staining schizont stage parasite red staining show labeling peptide specific antibody conjugate anti human anti mouse IgG specific antibody Merge picture overlay blue red fluorescence channel region select approach test manually presence characteristic heptad repeat protein analyze COILS program select helical coil coil contain protein test possible surface location GPI anchor follow program identification potential signal peptide secretomep SignalP transmembrane span region TMPRED TMHMM TMHMM GPI anchor protein prediction sub cellular localization ptarget identify PEXEL like motif sequence select protein follow pattern deq represent combination PEXEL pattern indicate recent paper presence identify protein asexual erythrocytic stage check publish datum transcriptome proteome stage development falciparum synthesize Advanced ChemTech Hatley St George UK AC Omega multi channel synthesizer Applied Biosystem synthesizer Foster City solid phase Fmoc chemistry crude peptide purify RP HPLC preparative column analyze mass spectrometry MALDI TOF Applied Biosystem Foster City chemical solvent peptide synthesis purchase Fluka Buchs Switzerland Novabiochem Laufelfinger dichroism CD spectra peptide record jasco spectrometer jasco corporation Tokyo Japan equip temperature controller cm path length cuvette measurement water pH peptide concentration mg ml sera Burkina Faso collect village Goundry locate central Mossi Plateau km north capital Ouagadougou province Oubritenga climate characteristic area sudanese savannah dry season November rainy season June October Malaria transmission high rainy season markedly seasonal ethical clearance obtain Ministry Health Burkina Faso obtain informed consent parent caretaker heparinized venous blood sample collect crosssectional survey malaria low transmission season Tanzanian sera come large scale community base study undertake Kikwalila village Kilombero District Morogoro Region blood sample adult year take finger prick serum keep use research ethical clearance study obtain Tanzanian Commission Science Technology colombian sera collect Buenaventura main port Colombian Pacific Coast human informed consent cross sectional survey carry February framework project support Colombian Research Council COLCIENCIAS area unstable transmission falciparum vivax malaria ethical clearance draw blood human volunteer Human purify antibody mg mL IFAT perform ring stage Valle blood take venipuncture tube contain EDTA sera fractionate store frozen use sera adult Papua New Guinea PNG pool affinity purification collect Maprik district East Sepik Province cross sectional survey July framework Malaria vaccine Epidemiology Evaluation Project MVEEP support United States Agency International Development area highly endemic malaria ethical clearance MVEEP obtain PNG Medical Research Advisory Committee blood take venipuncture tube contain pool immune african globulin piag ADCI prepare immune individual live endemic area negative control IgG IgG obtain pool french adult donor history malaria briefly IgG fraction positive negative control purify size exclusion trisacrylh pall BioSepraH Pall life Sciences NY column follow ionic exchange DEAE Ceramic HyperDH column Pall BioSepraH purify IgG extensively dialyze RPMI keep use mouse inject time mg indicate peptide Montanide ISA base tail day bleeding perform day second immunization ELISA perform accord Lopez et al anti human IgG anti mouse IgG conjugate alkaline phosphatase Sigma St Louis MO second antibody individual human sera adult donor Burkina Faso Tanzania Colombia respectively dilution Serum consider positive optical density od reading high mean OD standard deviation sd negative control individual serum sample naïve swiss donor OD ratio mean duplicate experimental value mean OD negative control high mouse sera end point value determine dilution mean OD standard deviation sd negative control non immune competition assay perform incubate select human affinity purify antibody antibody elicit mouse antigen indicate range concentration minute room temperature prior addition ELISA peptide coated plate well Antigen Sepharose conjugate preparation mg antigen dissolve mL couple buffer NaHCO contain NaCl pH CNBr sepharose Amersham Bioscience AB Uppsala Sweden activate swell mm HCl wash couple buffer antigen solution add gel mixture stir RT coupling reaction excess antigen wash away couple buffer unreacted activate group block treatment ethanolamine pH min RT gel wash sodium acetate buffer pH follow couple buffer antigensepharose bead store PBS contain mm azide isolation specific antibody pooled human serum dilute time PBS contain sodium chloride mix antigen sepharose conjugate mixture stir gently wheel centrifugation supernatant collect store use antigen sepharose bead wash mL trizma base TRIS mm contain NaCl pH mL TRIS mm pH elution bind antibody achieve glycine pH fraction obtain instantly neutralize TRIS pH dialyze phosphate buffer pH antibody concentration determine absorbance solution nm slide coat Pf sporozoite dry RT minute fix acetone minute wash time Tween dry carefully block bovine serum albumin BSA minute RT slide coat Pf merozoite fix acetone minute dry RT appropriate antibody serum dilution prepare BSA distribute mL incubate RT humid chamber wash goat anti human goat anti mouse polyvalent immunoglobulin conjugate Molecular Probes dilute BSA anti human IgG Fc specific FITC conjugate Sigma dilute Evans blue solution add mL slide incubate RT humid chamber dark slide wash cover glycerol seal read fluorescence microscope Leica DMIRB Uganda Palo Alto strain FUP culture supplement albumax GibcoBRL Invitrogen San Diego ADCI assay blood stage parasite culture synchronize successive sorbitol treatment follow maturation floatation porcine skin gelatin type monocyte MN prepare cytapheresis sample obtain healthy blood donor previous history malaria Lecourbe Blood Bank Paris France peripheral blood mononuclear cell PBMC separate Ficoll density gradient PREP TechGen Les Ulis France wash Mg free hbss buffer mm hepe GibcoBRL Invitrogen cell distribute polystyrene flat bottomed culture plate TPP Trasadingen Switzerland adherent MN select incubation humidified co atmosphere adherent cell obtain manner MN estimate non specific esterase test naphtyl acetate esterase Sigma MN donor test prior adci assay direct inhibitory effect assays well contain MN purify describe ml asynchronous parasite culture parasitemia hematocrit add Wells supplement test control antibody ab total volume adjust ml culture medium ml culture medium add ADCI assay stop final parasitemia determine light microscopy Giemsa stain smear count red blood cell Ab test duplicate well include follow control non specific monocytic inhibition direct inhibition control test IgG test piag IgG final concentration mg ml positive negative control respectively immunopurified test Abs mg ml Specific Growth inhibitory Index SGI consider parasite growth inhibition effect test Abs cooperate MN calculate follow SGI parasitemia MN test parasitemia test parasitemia MN parasitemia IgG figure cd spectra peptide find MB TIF figure ELISA inhibition assay anti human peptide specific antibody bind peptide specific antibody peptide absorb ELISA plate inhibit incubate specific antibody mg ml peptide peptide respectively Material Methods Peptides share NNM MNN sequence similarity peptide exhibit apparent sequence similarity table Structural feature cellular location prediction protein contain peptide specific antibody test ADCI Table find mb DOC','2007-07-25 00:00:00','http://dx.doi.org/10.1371/journal.pone.000064',2204,938),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json','Selective redox regulation of cytokine receptor signaling by extracellular thioredoxin-1','The thiol-disulfide oxidoreductase thioredoxin-1 (Trx1) is known to be secreted by leukocytes and to exhibit cytokine-like properties. Extracellular effects of Trx1 require a functional active site, suggesting a redox-based mechanism of action. However, specific cell surface proteins and pathways coupling extracellular Trx1 redox activity to cellular responses have not been identified so far. Using a mechanism-based kinetic trapping technique to identify disulfide exchange interactions on the intact surface of living lymphocytes, we found that Trx1 catalytically interacts with a single principal target protein. This target protein was identified as the tumor necrosis factor receptor superfamily member 8 (TNFRSF8/CD30). We demonstrate that the redox interaction is highly specific for both Trx1 and CD30 and that the redox state of CD30 determines its ability to engage the cognate ligand and transduce signals. Furthermore, we confirm that Trx1 affects CD30-dependent changes in lymphocyte effector function. Thus, we conclude that receptor–ligand signaling interactions can be selectively regulated by an extracellular redox catalyst.',' Disulfide bond long recognize structural element stabilize protein harsh extracellular environment recently additional concept emerge disulfide bond operate dynamic scaffold capable regulated rearrangement variety functional form Jordan Gibbins consistent notion cell surface process long know depend catalyzed thiol disulfide exchange include cell adhesion Essex uptake bacterial toxin de Paiva et al viral fusion host membrane Sanders variety cell surface signal receptor appear exist thiol disulfide configuration example Greene et al case catalyst drive thioldisulfide exchange functional difference redox form elucidate number thiol disulfide oxidoreductase know secrete act cell surface redox catalyst protein disulfide isomerase PDI member thioredoxin trx superfamily cell surface pdi find act transmembrane surface associate protein include envelope protein cause fusogenic conformation Markovic et al integrin mediate platelet adhesion Lahav et al thiol disulfide oxidoreductase associate extracellular function well know intracellular role reduce transiently form disulfide bond cytosolic nuclear target protein participate multitude fundamental process range oxidant scavenging dna synthesis regulation apoptosis cell proliferation Powis Montfort addition release extracellular environment variety normal neoplastic cell Rubartelli et al human purify cytokine like factor supernatant virally transform lymphocyte initially name adult cell leukemia derive factor Tagaya et al Tac induce factor Tagaya et al cell stimulatory factor cell Wakasugi et al extracellular present circulation healthy subject level increase inflammatory condition include viral infection Nakamura et al circulatory act chemoattractant monocyte neutrophil lymphocyte Bertini et al inhibit neutrophil migration inflammatory site vitro vivo Nakamura et al recently find secrete dendritic cell cognate cell recognition contribute subsequent cell activation Angelini et al present extracellular influence cellular behavior remain unknown report extracellular activity depend functional active site appear likely catalyze thiol disulfide exchange cell surface target protein enzymatic activity thiol disulfide exchange reaction highly specific transient detect conventional technique date single cell surface receptor member immunoglobulin superfamily show susceptible redox activity Matthias et al cell surface protein target enzymatic activity await identification study address question cell surface receptor express lymphocyte specifically interact extracellular way disulfide bond exchange kinetic trapping technique enable detection isolation short live reaction intermediate surface intact cell identify validate tumor necrosis factor receptor superfamily member principal target molecule infected transform lymphocyte cell surface activity highly selective discriminate different member tnfr superfamily mediate thiol disulfide exchange lead structural change ectodomain detect conformationsensitive antibody demonstrate disulfide exchange interfere binding ligand cognate receptor affect dependent change lymphocyte effector function implicate stimulatory apoptotic signaling finding suggest interact modulate lymphocyte behavior survival condition infection inflammation identify candidate target protein cell surface lymphoid cell apply trap technique base reaction mechanism approach make use finding mutant thiol dependent oxidoreductase lack terminal cysteine CXXC active site motif form long live mixed disulfide intermediate target protein target protein remain covalently link mutant oxidoreductase amenable isolation analysis principle show Figure kinetic trapping apply previously identify interaction partner Trx family member plant Motohashi et al secretory pathway human lymphocyte study CXXC base trap technique identify establish novel target protein subsequently confirm independent technique demonstrate competence technique identify bona fide interaction partner determine kinetic trapping apply human create recombinant wild type mutant Trx protein equip terminal dual affinity tag compose streptavidin bind peptide SBP hexahistidine tag create trap mutant second cysteine CXXC motif replace serine harbor additional cysteine residue distal active site cysteine residue dispensable catalytic activity cause oxidative inactivation intra intermolecular disulfide bond formation Casagrande et al Watson et al create mutant contain amino acid substitution additional cysteine figure CCCCC CCAAA CSCCC CSAAA SSAAA annotate identity residue test trapping capable identify know target protein allow react cytosolic protein release digitoninpermeabilize cell incubation lead formation reproducible pattern distinct mixed disulfide conjugate visualize silver staining SDS PAGE gel nonreducing condition figure lane accordance trap mechanism conjugation strictly depend availability terminal thiol concurrent absence terminal thiol wild type cysteine free capture protein figure lane pattern trap protein significantly influence presence absence non catalytic cysteine datum show main cytosolic interaction partner identify liquid chromatography tandem mass spectrometry LC MS MS association confirm immunoblotte datum show disulfide link conjugate maintain non reduce condition figure lane cleave monomer constituent reduce condition figure lane test undergo authentic interaction condition typical extracellular environment allow react human plasma protein avoid nonspecific absorbance high abundance serum protein apply kda plasma ultrafiltrate lead capture small number protein visualize colloidal Coomassie stain Figure LC MS MS principal interaction partner plasma ultrafiltrate identify establish target protein recently find present human plasma Chen et al experiment provide proof principle evidence kinetic trapping capable capture identify prove target protein human intra extracellular environment investigate capture protein trx specific directly compare correspond trap mutant member Trx superfamily csaaa annotate identity residue like use active site thiol act disulfide reductase cytosolic environment contrast specialize recognition reduction protein glutathione mixed disulfide bond form mixed disulfide intermediate glutathione protein Yang et al Peltoniemi et al expect detect trapping peroxiredoxin interact protein silver gel figure lane immunoblotte datum show activity thiol reactivity trap mutant confirm independent experiment datum show demonstrate mere availability active site thiol explain profile protein capture trap mutant instead result support notion Trx mediate reducing activity steer distinct target disulfide bond specific protein protein interaction having establish kinetic trap approach soluble target protein ask kinetic trap technique apply surface intact cell culture give previous indication disulfide bond exchange wild type Matthias et al ask kinetic trapping enable detect interaction surface positive promyelocytic cell line brief allow mutant interact surface live cell remove unreacted oxidoreductase washing collect disulfide link complex cellular lysate non trap SSAAA mutant incubate kda fraction prepare fresh human serum Disulfide link complex analyze colloidal Coomassie stain non reduce reduce condition conjugate dimer monomeric indicate kinetic trapping mediate specific protein protein interaction cytosolic protein digitonin permeabilize Jurkat cell incubate sav sepharose Disulfide link complex analyze silver stain nonreduce reduce condition conjugate form indicate band correspond additional cytosolic protein interact streptavidin sav affinity purification find cell surface form mixed disulfide exogenously add Supplementary Figure leave panel dissociate DTT treatment Supplementary Figure right panel result confirm kinetic trapping identify specific reactive cell surface protein allow de novo identification previously unknown cell surface target protein human identify autocrine factor secrete act virally transform lymphoid cell line apply cell surface trap human EBV transform lymphoblastoid cell line derive parental cell line cell line originally description costimulatory factor later identify Wakasugi et al mixed disulfide complex form surface cell isolate analyze specific immunoblotting visualize overall mixed disulfide conjugate interestingly find predominantly engage single protein lymphoblastoid surface suggest highly selective interaction Figure lane expect trap product mixed disulfide conjugate kda susceptible reduction Figure lane kda conjugate form surface EBV negative Burkitts lymphoma cell line figure contrast conjugate size find form surface CCRF CEM cell figure YT large granular lymphoma cell datum show pretreatment cell surface alkylate agent iodoacetamide IAA interfere formation kda mixed disulfide conjugate confirm conjugate form expect disulfide exchange mechanism de novo disulfide bond formation thiol identify unknown target protein perform cell surface trap large scale cell purify interact surface protein sav affinity purification visualize protein colloidal Coomassie stain figure leave panel kda band absent control precipitation correspond band non reduce reduce lane subject tryptic digestion LC MS MS analysis sample unknown protein identify member tnfr superfamily validate direct covalent interaction aliquot trap complex experiment separate non reduce reduce condition subject immunoblotte analysis anti figure middle panel anti antibody figure right panel respectively observe mobility difference non reduce NR reduce lane demonstrate formation mixed disulfide conjugate Figure right panel additional immunoblotting experiment demonstrate low nanomolar concentration sufficient detect interaction Figure confirm trapping depend terminal cysteine CXXC motif figure lane application trap mutant condition lead conjugation Figure Ectodomains TNFR superfamily protein typically compose cysteine rich domain CRDs normally harbor disulfide bond Bodmer et al exclude possibility interact crd uniformly test discriminate distinct member tnfr superfamily show Figure form mixed disulfide conjugate cell identical condition result obtain epidermal growth factor receptor EGFR contain total ectodomain disulfide bond express substantial level cell Gill Lazar figure finding indicate reactivity cell surface receptor selective determine presence crd number ectodomain disulfide bond exclude reactivity receptor determine limit surface expression level ectopically overexpresse TNFR superfamily member control promoter HeLa cell analyze cell surface trap indirect immunofluorescence mock transfecte HeLa cell capture surface figure low row expression lead strong surface association colocalization protein figure upper row contrast expression Figure NGFR datum show promote interaction cell surface strengthen notion reactivity specific property domain structure human differ member TNFR superfamily murine homologue presence additional crd arise internal duplication exon Burgess et al ask unusual feature confer reactivity human test short murine interact show Figure murine express Rauscher murine leukemia virus induce cell lymphoma line RMA efficiently target demonstrate additional crd human require reactivity result suggest enzymatic affinity conserve mammalian evolution demonstrate independent method attack break disulfide bond thiol specific cell surface biotinylation verify activity generate free thiol ectodomain Supplementary Figure subsequent analysis cell surface expression flow cytometry fluorescence microscopy reveal brief treatment cell wild type lead complete loss recognition antibody figure upper panel Figure second column similar result obtain monoclonal antibody Systems Clone examine expression figure middle panel condition recognition Ber antibody slightly affect figure low panel Figure column indicate mediate disulfide bond reduction possibly rearrangement induce structural alteration ectodomain disrupt conceal epitope pursuant observation recognition antibody affect redox state flow cytometry analyze response complex analyze anti egfr immunoblotting cellular lysate include control HeLa cell transiently transfecte expression construct human vector incubate analyze immunofluorescence microscopy specific antibody scale bar mm HeLa cell transiently transfecte expression construct human treat describe analyze immunofluorescence microscopy specific antibody scale bar mm RMA mouse lymphoma cell incubate Disulfide link complex analyze anti mouse immunoblotte non reduce reduce condition disulfide link complex monomeric indicate additional band high molecular weight characterize represent conjugate dimer reduction great detail addition oxidize influence antibody reactivity datum show reduce apply absence regenerate system conformational change observe remain incomplete datum show complete sustained loss antibody reactivity require source reduce equivalent oxidoreductase DTT Trx reductase find effective regenerate system importantly DTT TrxR NADPH effect apply absence figure middle panel datum show wild type figure leave panel redox inactive mutant figure middle panel alter redox sensitive epitope cell surface receptor example analyze parallel cell unaffected treatment figure right panel Loss antibody recognition typically occur minute figure upper panel test influence concentration condition consistent effect conformation evident concentration nM Figure low observation antibody bind influence suggest redox dependent conformational change ectodomain test mediate reduction influence binding ligand analyze interaction recombinant soluble cell surface flow cytometry brief incubation Hodgkin lymphoma cell wild type lead substantial loss binding cell surface figure similar result obtain large granular lymphoma Figure effect evident absence exogenously add reduce system Supplementary Figure demonstrate give condition substrate interaction limit oxidative inactivation show fluorescence microscopy bind surface transfecte HeLa cell colocalize receptor figure upper row treatment HeLa cell figure middle row redox inactive mutant figure low row strongly interfere binding colocalization give influence ligand binding reason interfere mediate signaling test mediate conformational alteration affect dependent cellular response use YT large granular lymphoma line previously study signal emanate define gene regulate signal Muta et al consistent previous result Bowen et al observe stimulation agonistic antibody lead upregulation chain figure low panel compare column treatment YT cell redox inactive prevent upregulation figure low panel column concomitant redox change Figure upper panel column demonstrate redox interaction pronounce influence mediate gene expression YT cell respond signal downregulate expression cytotoxic effector molecule include FasL decrease cytotoxicity Fas express target cell Bowen et al Muta et al test influence mediate suppression cytotoxicity treat YT cell stimulation agonistic anti antibody quantify cytotoxicity Cr label Raji cell stimulation cytotoxicity markedly reduce figure compare column decrease cytotoxicity completely reverse catalytically inactive mutant SSAAA figure column judge RT PCR change cytotoxicity correlate change FasL mrna expression figure low panel result confirm catalytic activity modulate dependent change cellular behavior function accumulate evidence indicate reduction rearrangement disulfide bond constitute mechanism control protein function cell surface Hogg idea disulfide bond act dynamical redox switch specifically operate secrete redox catalyst represent novel concept signal transduction Jordan Gibbins technical difficulty detect analyze individual disulfide rearrangement cell surface progress slow recognize important regulator cellular organismal redox homeostasis Gromer et al particular intracellular counteract oxidative stress promote cell growth inhibit apoptosis condition oxidative stress release cell accumulate site inflammation Nordberg Arner numerous study report secretory influence effector function proliferation lymphocyte Nakamura et al protein pathway couple extracellular redox activity define cellular response remain unknown study address question lymphocyte surface receptor target regulate redox activity extracellular purpose use mechanism base kinetic trap approach capture mixed disulfide intermediate form exogenous target protein cell surface living cell activity base technique offer opportunity identify interaction short lived detectable conventional method knowledge report application kinetic trap identify novel target protein mammalian application technique surface intact cell demonstrate interact intraand extracellular target protein highly selective manner guide specific protein protein recognition random encounter disulfide bond apply approach surface cell line representative lymphoid lineage observe basically target single cell surface protein subsequently identify TNF receptor superfamily member know pronounced preference particular target protein surprising fact assess reactivity protein embed natural microenvironment intact surface active plasma membrane living cell conceivable protein disulfide exchange interaction limit control native context location scrutinize specificity observed interaction ask preference cause unusual density disulfide bond exceptional cell surface expression level ectodomain harbor significant number predict disulfide bridge crd appear unusual term disulfide bond composition density compare member superfamily test experimentally fail interact CRD contain protein include EGFR feature total ectodomain disulfide bond preference explain exceptional surface expression level Hodgkin disease cell line typically express high level cell line include CCRF CEM fold low expression determine flow cytometry selective targeting observe conversely ectopic overexpression related TNFR superfamily member HeLa cell lead interaction strongly interact cell condition consistent finding recent experiment demonstrate target particular site ectodomain Balmer TP Dick unpublished datum facilitate identification low abundance cell surface protein vitro trap experiment typically perform concentration disulfide exchange subsequently test low concentration concentration low nanomolar range nm find rise formation proportional amount mixed disulfide intermediate figure demonstrate observed interaction compatible expect physiological concentration range secretory type know act multiple turnover catalyst suitable reduce system electron source provide regeneration agreement consideration observe sustained reduction cell culture require regenerate system flow cytometry monitor conformational change ectodomain find respond concentration nanomolar range start nM figure low panel minimal concentration require sustain reduction substantially low specific environment efficient deliver reduce equivalent prevent oxidative inactivation present clear extracellular regenerate vivo despite overall oxidizing character extracellular compartment reductive process know place cell surface hand longstanding evidence existence transplasma membrane redox system deliver electron cell surface Crane et al hand regenerate co secrete reductant secretion DC co culture accompany release reduce cysteine creation reduce microenvironment interact cell Angelini et al addition TrxR find secrete activate monocyte extracellular reduce system Soderberg et al concentration human plasma low nanomolar range nM find elevate fold inflammatory condition Yoshida et al plasma oxidize appear represent systemic dilution previously release tissue accordingly local tissue concentration secretory example activate lymph node expect markedly high plasma overall concentration mammalian tissue high mm Gromer et al certain secrete cell type include macrophage dendritic cell distinctly upregulate expression activation Angelini et al vitro study secretion suggest substantial fraction intracellular release hour Rubartelli et al direct measurement extracellular tissue available physiologically relevant extracellular concentration reach upper nanomolar low micromolar range find mediate disulfide reduction change conformation functional property ectodomain reduce state lose ability interact cognate ligand agonistic antibody presence catalytically active impede dependent signal YT lymphoma cell line demonstrate effect FasL expression influence cytotoxicity Fas express target cell physiological role system remain unclear vitro study focus lymphoid malignancy show trigger signal induce proliferation activation growth arrest apoptosis depend cell type stimulatory condition Schneider Hubinger vivo cell surface expression appear tightly regulate restrict lymphocyte undergo activation lymphoid tissue propose provide proliferation survival signal lymphocyte response Croft inflammatory condition expression markedly induce vivo activation monitor release shed plasma membrane binding Hansen et al similar serum serum increase infection autoimmunity allergy example systemic lupus erythematosus rheumatoid arthritis atopic dermatitis Horie Watanabe appear play role regulation antiviral inflammatory response secretion expression associate virally transform lymphocyte elevated level occur viral infection likewise viral infection lead elevate plasma level study indicate secrete modulate antiviral inflammatory process Nakamura et al Nakamura et al study identify enzyme substrate relationship receptor activate lymphocyte involve regulation inflammation lymphocyte migrate different microenvironment conceivable catalyze disulfide exchange dynamically activate inactivate pathway response redox environment interaction represent regulatory link oxidative stress lymphocyte function understanding relationship vivo await generation suitable experimental tool cell culture CCRF CEM Jurkat RMA cell culture RPMI Gibco supplement heatinactivated fetal bovine serum mm glutamine ml penicillin mg ml streptomycin Gibco YT cell culture IMDM Gibco HeLa cell DMEM Gibco supplement depend type experiment recombinant trapping mutant apply cytosolic preparation human serum ultrafiltrate intact cell detailed description different substrate trap protocol provide supplementary information reduction cell surface cell incubate mm mm DTT nM human Trx reductase mm NADPH min monitor reduction cell surface cell stain anti human monoclonal antibody Systems anti human monoclonal antibody Santa Cruz anti human monoclonal antibody Ber DakoCytomation follow incubation PE conjugate goat anti mouse Ig Biosource control cell stain PE conjugate anti human monoclonal antibody BD Pharmingen cell analyze FACSCalibur Becton Dickinson CellQuest software total cell incubate mM SBP CCCCC mm DTT min wash time incubate ng ml recombinant Systems min RT washing cell stain anti polyhis monoclonal antibody Sigma follow incubation PE conjugate goat anti mouse Ig cell seed coverslip transfecte expression construct CaCl precipitation day transfecte cell fix formaldehyde sucrose PBS min RT fixed cell wash time PBS incubate different construct recombinant Systems protein visualize appropriate primary antibody anti polyclonal antibody Systems anti monoclonal antibodie Santa Cruz Ber DakoCytomation anti monoclonal antibody kind gift Dr Krammer anti polyclonal antibody Preuss TP Dick unpublished follow FITC conjugate anti goat IgG FITC conjugate anti rabbit IgG TRITC conjugate anti mouse IgG analyze Nikon confocal microscope supplementary datum available EMBO Journal Online','2007-07-11 00:00:00','http://dx.doi.org/10.1038/sj.emboj.7601746',2961,913),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json','Relevance of human metapneumovirus in exacerbations of COPD','BACKGROUND AND METHODS: Human metapneumovirus (hMPV) is a recently discovered respiratory virus associated with bronchiolitis, pneumonia, croup and exacerbations of asthma. Since respiratory viruses are frequently detected in patients with acute exacerbations of COPD (AE-COPD) it was our aim to investigate the frequency of hMPV detection in a prospective cohort of hospitalized patients with AE-COPD compared to patients with stable COPD and to smokers without by means of quantitative real-time RT-PCR. RESULTS: We analysed nasal lavage and induced sputum of 130 patients with AE-COPD, 65 patients with stable COPD and 34 smokers without COPD. HMPV was detected in 3/130 (2.3%) AE-COPD patients with a mean of 6.5 × 10(5 )viral copies/ml in nasal lavage and 1.88 × 10(5 )viral copies/ml in induced sputum. It was not found in patients with stable COPD or smokers without COPD. CONCLUSION: HMPV is only found in a very small number of patients with AE-COPD. However it should be considered as a further possible viral trigger of AE-COPD because asymptomatic carriage is unlikely.',' respiratory virus play important role exacerbation COPD increasingly recognise application molecular detection method prevalent virus detect polymerase chain reaction far respiratory syncytial virus RSV Influenza Rhinovirus Parainfluenza human metapneumovirus hmpv recently discover respiratory virus isolate dutch child low respiratory tract infection LRTI world wide distribution probable isolate North HMPV recognize member Paramyxoviridae like RSV associate bronchiolitis case pneumonia croup exacerbation asthma disease share feature COPD date report hmpv adult scarce general canadian population patient age group acute respiratory tract infection hMPV positive thirty percent hmpv infect patient hospitalize hospitalization rate significantly high patient year year age prospective cohort adult hmpv detect illness asymptomatic subject HMPV prevalent young adult child frail elderly hmpv infection severe virus isolate lung previously healthy man die acute pneumonia role hmpv acute exacerbation COPD AE COPD study recently outpatient low frequency observe prevalence hmpv patient hospitalize AE COPD unknown aim investigate frequency detection hmpv prospective cohort hospitalize patient AE COPD compare result patient stable COPD smoker different group study group consist hospitalize patient acute exacerbation COPD AE COPD second group subject stable COPD group compose smoker COPD group define previously publish briefly AE COPD patient suffer COPD define GOLD acute exacerbation characterize worsen dyspnea cough expectoration routine posterior anterior chest radiograph evaluate admission expert radiologist exclude reason increase symptom pneumonia tuberculosis pulmonary fibrosi bronchiectasis bronchial carcinoma congestive heart failure stable copd patient exacerbation day prior hospital admission change therapy day include inhaled oral medication admit medical reason department internal medicine pulmonary care copd subject recruit ratio month order prevent seasonal selection bias smoker smoke pack year chronic symptom like cough phlegm report dyspnea bronchial obstruction FEV FEV predict smoker history copd asthma systemic topic pulmonary medication smoker recruit smoking cessation initiative newspaper advertisement study approve ethical committee Ruhr University Bochum Germany write informed consent obtain patient control subject inclusion study clinical evaluation spirometric test nasal lavage induce sputum speciman process viral ribonucleic acid RNA extraction carry describe Rohde et al elution volume µl cDNA generate random hexamer primer previously hmpv specific real time RT PCR design evaluate Maertzdorf et al primer probe localize nucleoprotein gene NL presence degenerate base probe allow detection genetic lineage assay perform TaqMan PCR Core Kit final volume µl contain nM forward primer NL forward nm reverse primer NL reverse nM probe NL probe TGYAATGATGAGGGTGTCACTGCGGTTG residue Nuclease free water negative control plasmid contain gene hmpv kindly provide James Simon VIRONOVATIVE EUR Holding Erasmus University Rotterdam positive control PCR run cycling parameter follow min cycle min amplification detection RNA virus isolate clinical specimen perform GeneAmp Sequence Detection System Applied Biosystems real time PCR product clone QIAGEN PCR cloning kit QIAGEN Hilden Germany standard plasmid DNA absolute quantification hmpv viral load calculation perform previously describe absolute quantification RSV viral load primary objective study compare frequency hmpv detection respiratory specimen COPD patient acute exacerbation smoker datum check normal distribution Kolmogorov Smirnov test datum non parametrical distribution result express median range difference group assess Kruskal Wallis test analyse significant difference individual group pair wise comparison side Mann Whitney test perform significance level set datum analyse process SPSS Version Windows XP operating system total subject investigate October June patient AE COPD patient stable copd smoker COPD clinical characteristic lung function measurement summarize table FEV FEV predict value FEV normal smoker significantly decrease stable COPD patient significantly decrease AE COPD patient compare stable COPD compare detect subject subject AE COPD patient prevalence hmpv AE COPD patient virus simultaneously detect nasal lavage induce sputum patient viral load time high nasal lavage induce sputum patient overall viral load nasal lavage time high compare induce sputum detail table hMPV positive patient differ significantly AE COPD patient clinical parameter lung function analyse hmpv detect winter season main finding control study investigate incidence hmpv subject COPD smoker COPD recently discover respiratory virus detectable exacerbation COPD frequency detection low positive case viral load considerable detection patient stable COPD smoker Vicente et al report incidence hmpv copd patient patient hmpv positive patient transfer hospital control study detail study available fact datum publish form letter result support finding incidence hmpv study low compare respiratory virus similar previous study find Picornaviruses detectable AE COPD patient Influenza respiratory syncytial virus prospective cohort study adult hmpv detect illness HMPV prevalent young adult child frail elderly long term care facility unfortunately report specify elderly patient suffer COPD asymptomatic smoker COPD hmpv detect recent study investigate nasal secretion adult respiratory illness find hmpv ill patient control subject strongly support datum recent study find hmpv adult patient present emergency department AE COPD spring season Quebec Canada support finding american study investigate clinical sample collect hmpv detect patient indicate important geographical seasonal difference hmpv prevalence take result present keep study adult add important information prevalence hmpv hospitalize AE knowledge study analyse viral load hmpv COPD patient find mean viral copy ml nasal lavage viral copy ml induce sputum value indicate hmpv infectious agent trigger exacerbation patient viral load cut value infectivity copd exacerbation study detail need investigation viral load copy ml Cytomegalovirus infection lung transplant patient ml SARS consider indicate infectious disease hmpv find acute exacerbation stable disease smoker copd support trigger role AE infection severe isolate lung previously healthy man die acute pneumonia hmpv positive patient differ clinical characteristic lung function AE COPD patient indicate severe course AE COPD patient take control study relevance hmpv hospitalize AE COPD HMPV detect low frequency noticeable viral load AE COPD patient give asymptomatic carriage hmpv unlikely consider possible trigger AE COPD AE COPD considerable impact course disease regional outbreak hmpv possible include future diagnostic therapeutic consideration declare compete interest','2005-12-21 00:00:00','http://dx.doi.org/10.1186/1465-9921-6-150',924,412),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json','Model-Based Design of Growth-Attenuated Viruses','Live-virus vaccines activate both humoral and cell-mediated immunity, require only a single boosting, and generally provide longer immune protection than killed or subunit vaccines. However, growth of live-virus vaccines must be attenuated to minimize their potential pathogenic effects, and mechanisms of attenuation by conventional serial-transfer viral adaptation are not well-understood. New methods of attenuation based on rational engineering of viral genomes may offer a potentially greater control if one can link defined genetic modifications to changes in virus growth. To begin to establish such links between genotype and growth phenotype, we developed a computer model for the intracellular growth of vesicular stomatitis virus (VSV), a well-studied, nonsegmented, negative-stranded RNA virus. Our model incorporated established regulatory mechanisms of VSV while integrating key wild-type infection steps: hijacking of host resources, transcription, translation, and replication, followed by assembly and release of progeny VSV particles. Generalization of the wild-type model to allow for genome rearrangements matched the experimentally observed attenuation ranking for recombinant VSV strains that altered the genome position of their nucleocapsid gene. Finally, our simulations captured previously reported experimental results showing how altering the positions of other VSV genes has the potential to attenuate the VSV growth while overexpressing the immunogenic VSV surface glycoprotein. Such models will facilitate the engineering of new live-virus vaccines by linking genomic manipulations to controlled changes in virus gene-expression and growth.',' infection cause virus persistently threaten human health example million million million people world carry human immunodeficiency virus type hepatitis virus HBV hepatitis virus HCV respectively annually global population infect influenza result death protection viral infection provide inoculation live virus kill virus subunit vaccine live virus vaccine offer advantage activate humoral cellmediate immunity require single boosting generally provide long immune protection form vaccine adequately attenuate growth minimize possibility vaccine induce pathogenic effect retain immunogenicity attenuation live virus traditionally achieve serially passage virus tissue cell culture adapt grow new cell type elevated reduced temperature process tend reduce replicative ability virulence human animal attenuation historically highly empirical process mechanism know elucidated decade emergence reverse genetic technique create unprecedented opportunity well control viral attenuation reverse genetic enable production RNA virus cloned cDNA specific mutation relatively easily introduce virus challenge engineering virus attenuation shift create variant predict specific genetic change define correlate measurable effect virus growth challenge address development quantitative mechanistic model map genome level change dynamic virus growth different environmental condition model intracellular virus growth aim predict rapidly virus infect cell produce virus progeny input model include rate constituent process entry virus cell transcription viral mrnas translation viral protein replication viral genome assembly intermediate finally production release viral progeny decade detailed biochemical biophysical genetic study diverse virus contribute level mechanistic understanding viral function interaction intracellular model virus growth develop phage Qb phage influenza virus detailed model intracellular growth virus explore behavior mutant virus encode alternative design starting point create alternative design reorder rearrange wildtype gene regulatory element genomic change alter timing level expression different viral gene impact growth production viral progeny depend dynamic expression viral gene preliminary model alternative genome design use language wild type virus retain parameter characterize wild type molecular interaction wild type average rate viral polymerase elongation wild type composition progeny virus apply manner reflect reordering wild type component engineer genome example timing expression gene phage infection map closely sequential order genome relocate essential early gene encode RNA polymerase downstream position delay initiation transcription highly efficient RNA polymerase attenuate phage growth preliminary model growth phage carry alter genome base wild type parameter capture overall observe trend attenuation expand approach mammalian virus biomedical agricultural relevance vesicular stomatitis virus prototype negative sense single strand RNA Mononegavirales virus member family Rhabdoviridae include rabie virus VSV particle single copy kb genome carry gene encode nucleocapsid phospho matrix envelope polymerase protein VSV economically important cause symptom like foot mouth disease livestock offer advantage vaccine vector include low seropositivity human capacity accommodate foreign gene genome size established reverse genetic recombinant form VSV carry foreign virus gene encode immunogenic protein propose potential vaccine HIV influenza respiratory syncytial virus pathogenic immunogenic VSV base vaccine infection VSV virus seek develop silico model VSV infection cycle incorporate know regulatory interaction mechanism relevant quantitative datum literature past year interaction correspond equation formulation describe detail model development section Materials Methods model quantitatively analyze intracellular growth wild type VSV direct host biosynthetic resource VSV gene expression synthesis progeny genome pathway switch synthesis VSV intermediate production VSV progeny reveal model capture experimental result show progressive attenuation virus growth associate move gene downstream wild type position finally use model predict alter position VSV gene promoter attenuate growth VSV increase potential capacity activate adaptive immune response model establish parameter set Tables analyze quantitatively systematically intracellular growth VSV regulate improved understanding virus infection model base analysis guide identify key regulatory component manipulate develop virus mutant possible vaccine vector candidate attenuation mechanism lead unequally distribute synthesis viral mrnas protein partial transcription termination mechanism attenuation common virus mechanism important satisfy different need viral protein infection cycle attenuation factor intergenic region VSV genome Table equation obtain literature incorporate model owe step wise release polymerase gene junction simulation estimate gradual decrease VSV mrna synthesis order figure compare abundant mrna mrna scanty infected baby hamster kidney BHK cell fold relative production level protein match relative availability correspond mrnas figure different protein degrade different rate table different level incorporation protein single virion particle define protein stoichiometry table relative level free viral protein cytoplasm develop differently mrna level figure protein abundant owe low content virion protein abundant persistently low level protein relate immediate complexation nascent genome antigenome nucleocapsid particle replication step owe cyclic switching transcription replication encapsidation process protein level predict oscillate show figure engineering viral genome provide way explore viral regulatory mechanism genomic level produce recombinant virus serve vaccine gene delivery vector oncolytic tumor killing agent complexity interaction viral cellular component involve life cycle virus challenge anticipate altering viral component influence overall behavior virus Lim Lang Lam Yin develop computer model begin mechanistically account key virus cell interaction prediction viral intracellular development Lim et al model able capture experimentally observe effect gene rearrangement level timing viral protein expression virus progeny production aspect important design live virus vaccine refinement extension approach current virus system potential advance application virus therapeutic agent post infection simulation predict significant decrease free protein arise dominance virion assembly process deplete protein compare transcription replication infected murine delay brain tumor DBT cell similar distribution viral mrnas protein obtain unpublished demand genome satisfied strong promoter anti genome template relative genome template anti genome template utilize amplify genome genome template amplify anti genome mrnas incorporate virion progeny particle high demand genome multiple task satisfied strong promoter anti genome template compare genome template polymerase bind strong promoter anti genome ultimately enhance production genome anti genomes figure model parameter prom measure strength anti genomic promoter relative genomic promoter table production ratio genome anti genome estimate dynamically change vary wild type VSV case prom Figure oscillatory change production ratio show Figure follow use genome transcription replication arise owe staggered shifting dominant template genome anti genome replication large value prom favor use anti genome template replicate genome genome template accumulate large excess relative anti genome template successfully compete replication resource Synthesis anti genome dominate accumulate serve dominant template virion production rate BHK cell maximum post infection infected DBT cell similar simulation result obtain synthesis genomesize viral rna continue long time active post infection Figure utilization genomic nucleocapsid genomic nucleocapsid template RNA synthesis incorporate progeny virion fate depend level polymerase protein respectively favor RNA synthesis virion production pathway extent association nucleocapsid protein dominate association polymerase describe parameter cond model table RNA synthesis virion production essential process infection extreme value cond favor process pol nt Strength anti genomic promoter relative genomic promoter prom rate constant ratio association protein polymerase genomic nucleocapsid cond protein bind plasma membrane cond Attenuation factor Degradation rate constant protein dp sec protein dp sec protein dp sec protein dp sec protein concentration protein concentration protein dp sec concentration protein concentration protein dp sec mrna mrna nucleocapsid nc host dependent parameter Number ribosome host cell nrib Elongation rate ribosome rib aa space neighbor ribosome BHK DBT Srib owe increase stability encapsidation nucleocapsid degradable naked mrnas assume nucleocapsid VSV degrade fold slow mrnas simulation result insensitive parameter DOI detrimental growth excessively large cond newly synthesize genomic nucleocapsid tend prematurely incorporate virion particle serve template transcription replication hand extremely small cond genomic nucleocapsid utilize primarily produce viral RNA package viral progeny intermediate parameter value expect optimal viral growth estimate possible range wild type value cond fit simulation result previous experimental observation Figure simulation indicate range cond near optimal optimal VSV growth BHK DBT cell respectively unpublished datum diversion inhibition host translation machinery create time window opportunity translation viral protein infection cycle virus actively passively compete host limited translation resource inhibit host transcription amplify viral mrnas respectively viral leader mrna protein play key role inhibition viral component accumulate cytoplasm initiation infection increase fraction host ribosome available viral mrnas figure fraction ribosomes associate viral mrnas define equation inhibition host macromolecular synthesis cause failure supply accessory factor need initiation elongation step translation result reduction fraction active ribosome time figure describe dec equation mechanism create time window active ribosome maximally available viral translation infected cell figure refer term dec equation abundance viral mrnas limitation impose ribosomal spacing determine fraction active ribosome involve translate viral mrnas occupy ribosomes figure refer Equations model occupy active ribosome available one case number free active ribosome viral translation fully support limitation host machinery early infection stage post infection BHK DBT cell respectively host machinery excess figure later time viral translation limited host resource case number free active ribosome limitation cause transition replication dominant assembly dominant infection stage replication require continuous protein synthesis show Figure small fraction ribosome active form ribosome Table utilize viral translation experiment simulation VSV gene order mutant vaccine application seek minimize viral pathogenicity maximize immunogenicity base observed correlation vitro vivo result assume pathogenicity immunogenicity virus directly link level progeny production protein expression infected cell respectively previous section show VSV regulatory mechanism involve maintain balance infection viral synthesis process indirectly indicate importance balance viral growth perturbation balance genetic genomic manipulation provide way obtain viral phenotype favorable vaccine application test predictive ability model compare simulate protein expression growth gene rearrange VSV strain experimental result later employ model predict genomic manipulation attenuate virus growth increase protein expression protein expression rate gene rearrange virus stepwise decline transcription gene distant end region promoter highlight gene order affect gene expression VSV advance reverse genetic possible create gene rearrange virus strain transcriptional attenuation mechanism create altered level gene expression study internal gene permute result possible VSV strain characterize relative rate viral protein expression BHK cell experimentally measure base incorporation methionine hour window post infection extend model simulate experiment mutant represent geneorder permutation compare model prediction publish result show Figure rate express relative synthesis rate protein corresponding gene position close end genome strain case expression gene position minimal simulation experiment expression gene feature experimental datum simulation capture point fall close parity line indicate agreement simulation experiment Noteworthy subset point circle point exception general rule gene order determine level gene expression gene second position genome express essentially rate gene position result highlight expression rate protein affect gene order correspond rate mrna production length degradation rate fix average rate translational elongation long gene product tend produce slowly net rate protein production reflect rate protein synthesis protein degradation model account contribution circle gene accounting appear capture unexpected high translation level gene second position measure Ball et al second set point show box indicate mismatch figure challenging scenario reflect unknown strain specific mechanism present general gene permutation model note experiment base labeling quantifying protein post infection relatively early time point allow minimize potentially confound influence virion particle assembly production cytoplasmic level viral protein represent point majority viral protein figure gene rearrange virus employ model predict growth VSV strain have gene position genome grow well follow result consistent previously suggest hypothesis relocation gene distal position genome efficient way attenuate VSV vaccine use reduction growth follow move gene likely reflect imbalance replication transcription insufficient production protein reduce extent encapsidation nascent anti genome allow transcription dominate genome replication simulation match growth rank quantitatively match experimental datum predict variation virion production fold decrease small experimentally observe variation fold decrease potential source quantitative difference neglect mass action effect protein encapsidation process model encapsidation simulate instantaneous process free protein available show model capture major effect gene rearrangement viral growth protein expression effect relative promoter strength viral growth genome anti genome VSV synthesize unequal amount determine differ strength promoter strong promoter antigenome allow viral polymerase produce genome meet demand component virion particle template transcription replication explore VSV growth influence difference relative strength genomic anti genomic promoter predict yield virus BHK DBT cell broad range prom show figure small prom virus grow polymerase associate genome tend synthesize primarily anti genome example prom equal infect BHK DBT cell fold fold few progeny wild type VSV infect cell respectively large prom virus grow polymerase preferentially bind newly synthesize anti genomes produce anti genome need template amplify genome simulation predict value prom optimal VSV growth BHK DBT cell respectively figure estimate wild type value prom give VSV yield high maximum yield cell type figure speculate rational way attenuate pathogenicity live wild type virus swap promoter give prom promoter swap predict virion production decrease fold infected BHK DBT cell respectively relative wild type extent growth attenuation promoter swapping high model prediction swapping perturb viral transcription virion assembly budding process modulate signal encode termini genome sufficiently define include simulation rational vaccine attenuation double genomic manipulation variant VSV strain include Ball Wertz aim generate potentially broad diversity growth phenotype create test silico VSV mutant combine gene relocation range prom computationally simple task experimentally nontrivial example VSV prom produce simulate fold few virus progeny wild type BHK Figure interesting note fold degree growth attenuation great product constituent attenuation calculate assume effect uncoupled nonmultiplicative effect double genomic manipulation growth challenge predict absence quantitative model modulation VSV immunogenicity gene shuffling elicit systematic immune response live viral vaccine present display neutralizing epitope typically expression viral surface protein high level antigen expression find correlate rapid potent induction anti viral antibody Flanagan et al suggest gene encode move position VSV genome modulate expression VSV surface glycoprotein generate silico gene shuffle VSV strain have gene order internal gene MPG MGP PGM GMP GPM simulated level protein BHK cell infect strain simulation result consistent idea location gene affect production protein GPM strain give high concentration protein cytoplasm fold high wild type Figure case effect gene rearrangement difficult anticipate complexity involve interaction viral component example PGM strain show level protein expression similar PMG wt MPG strain gene early position strain figure vaccine use aim maximize immunogenicity VSV VSV base vector expression increase VSV gene insert foreign gene minimize potential pathogenicity growth attenuation give design goal specifically VSV vaccine prefer strain GPM show high expression protein Figure low production virion favorable feature Flanagan et al previously construct VSV strain have follow gene order genome construction base intuitive idea translocation gene gene early later position respectively compare wild type increase expression protein attenuate virus growth idea support experimental result detailed correlation location gene viral phenotype relevant vaccine application simulate silico growth mutant retain gene order wild type allow criterion define possible geneorder permutation viral growth level protein infect BHK cell mainly depend location gene gene respectively figure consistent experimental result Flanagan et al gene fix move gene close promoter predict increase protein expression Figure consistent prediction Flanagan et al observe high protein expression strain strain enhanced replicative ability VSV strain locate gene early position genome contribute increase level protein gene gene locate fifth position level virus growth protein expression low figure genome organization stoichiometric amount protein require replication assembly table reach simulation BHK cell parameter Table overall capture experimentally establish relative growth VSV strain cell growth strain significantly overestimate compare experimental result table relevant mechanism large discrepancy simulation experimental result remain elucidate change protein expression level gene shuffling rational mean modify viral feature vaccine use robust synthesis antigen highly attenuate strain appear effective vaccine strategy Flanagan et al previously suggest addition control attenuation virus growth potent vaccine ideally elicit strong humoral cell mediate immune response era highly advanced genetic technology witness turning point development live viral vaccine conventional empirical vaccine development process replace rational reversegenetic base one trend attention focus mechanism base design pathogenic immunogenic virus stain mathematical model intracellular virus growth support design process provide tool systematically analyze viral infection regulatory network identify critical regulatory mechanism component redirect viral phenotype reverse engineer desirable phenotype step infection cell monolayer cell harvest resuspend growth medium plate plate concentration cell ml plate cell return incubator allow grow overnight day representative cell monolayer harvest count approximate number cell monolayer incubate ll virus inoculum MOI allow virus adsorption plate rock gently min evenly distribute virion monolayer adsorption step adsorption period monolayer rinse twice ml HBSS place ml infection medium incubation medium sample ll include virion particle take postinoculation sample keep freeze analysis plaque assay plaque assay BHK cell plate plate culture confluence culture medium remove replace ll serially dilute viral sample inoculate monolayer return incubator allow virus adsorption plate rock gently min end adsorption period inoculum remove monolayer sample replace ml agar overlay agar overlay consist weight volume agar Agar Nobel Difco BD Diagnostic Systems deoxyuridine Sigma add lg ml agar overlay infect sample enhance plaque formation follow agar addition plate allow cool room temperature min return incubator incubate sample fix ml fixative room temperature fixative consist paraformaldehyde VWR sucrose Sigma mm phosphate buffer saline PBS Sigma pH agar overlay remove sample rinse twice ml PBS gentian violet dilute methanol Sigma ml stain sample Model development algebraic differential form equation mathematical model aim account establish molecular processing step development VSV model parameter extract literature parameter obtain fit simulation result experimental datum literature experiment key model parameter give Table detailed description model parameter estimation process provide Protocol figure respectively Virus binding penetration show Figure VSV initiate infection bind receptor phosphatidyl serine lipid component plasma membrane bind step VSV particle endocytose clathrin coated pit penetrate intracellular vesicle endosome membrane fusion penetration lead release encapsidated negative sense viral genome virus protein cytoplasm host cell assume bind step ratedetermine lump early step binding penetration order expression ex concentration bound extracellular virus particle respectively time apparent rate constant virus binding bind assume bind virus immediately endocytose fuse genome protein component instantaneously release cytoplasm expense fuse virus particle protein stoichiometry single VSV particle length viral gene protein summarize Table population distribution polymerase nucleocapsid follow release encapsidated genome protein cytoplasm VSV transcription initiate viral transcription assume independent host cell function replication instead viral complex protein stoichiometry take function polymerase transcription replication absence protein protein bind genome anti genome bind promoter region genomic anti genomic template viral polymerase start synthesize RNA transcription replication product elongate template transcription fraction elongate polymerase terminate transcription dissociate template encounter regulatory signal intergenic region addition regulated polymerase dissociation time dependent concentration change polymerase viral template cytoplasm influence distribution polymerase viral template transcription replication distribution polymerase continuously vary viral template ultimately determine relative synthesis level mrnas genome size RNAs simulate transcription replication process consider spatial temporal distribution template associate polymerase partition viral genome anti genome template segment exclude end region leader Le trailer region Tr genome complementary trailer Trc complementary leader region lec anti genome respectively Figure genome template transcription replication specially group segment gene figure choose minimum number total segment allow gene split specific integer number segment proportional length gene consider mechanism interaction polymerase intergenic regulatory sequence template describe Transcription section simulate polymerase flux segment time elapse initiation transcription template correlate level polymerase occupy geneencoding section template synthesis rate correspond viral mrna similar way distribution polymerase replication template correlate synthesis rate viral genome sized RNA explicit treatment polymerase spatial distribution viral genome antigenome template central model growth wild type gene rearrange virus strain treatment systematically account polymerization associate time delay polymerase flux template segment estimate polymerase flux need figure polymerase complex protein compete bind genomic nucleocapsid polymerase bind nucleocapsid subsequently distribute possible task transcription replication genome replication anti genome model assume genomic template negative sense nucleocapsid promoter leader region free polymerase available association free polymerase protein assume association free genomic template protein polymerase place instantaneously new pol new concentration total genomic nucleocapsid subset genomic nucleocapsid promoter newly occupy protein polymerase respectively pol spacing neighboring polymerase genomic anti genomic template pol concentration polymerase bind promoter region Le genomic template length promoter region specifically second term left hand equation denote concentration genomic template promoter currently occupy polymerase model concentration genomic nucleocapsid promoter bind polymerase concentration polymerase bind promoter genomic nucleocapsid interchangeable factor pol pol respectively binding protein polymerase initiate reaction lead virion assembly RNA synthesis respectively initiation RNA synthesis polymerase require finite time space adjacent polymerase template pol maintain infection assume fix elongation rate consideration expect time concentration nucleocapsid available new binding free protein inversely proportional concentration polymerase currently bind leader region genomic nucleocapsid pol polymerase space pol show second term equation ratio new pol new determine ratio association rate protein polymerase genomic nucleocapsid function rate constant relative amount correspond free component cytosol asso asso pol rate association protein polymerase genomic nucleocapsid respectively pol rate constant association reaction respectively cond denote ratio rate constant pol unlike protein synthesize protein associate plasma membrane equation cond fraction protein associate plasma membrane tran fraction protein satisfy polymerase stoichiometry protein pol total total concentration polymerase associate time nucleocapsid total concentration protein assemble viral progeny concentration protein large fold concentration protein tran equal tran equal model protein compete free genomic nucleocapsid condense nucleocapsid owe association protein utilize transcription replication Equations newly occupy nucleocapsid polymerase pol new calculate way give pol concentration positive sense antigenomic nucleocapsid available bind polymerase trc pol trc total concentration antigenomic nucleocapsid pol trc concentration polymerase bind promoter region Trc anti genomes trc length promoter region anti genome strong promoter genome quantify prom model prom polymerase bind promoter anti genome genome limitation free polymerase complex concentration polymerase newly binding promoter genome anti genomes pol term new describe follow pol trc new concentration polymerase newly binding complementary trailer region promoter anti genome polymerase newly binding bind promoter genome anti genome start viral RNA synthesis transcription replication process transcription viral polymerase leader region genome start transcription replication sufficient protein transcription inhibit encapsidation nascent positive sense RNAs protein replication dominate transcription contrast insufficient free protein transcription dominate replication model correlate extent transcription dominance availability protein introduce factor encap factor define ratio number free protein number require encapsidate available nascent genome sized viral RNAs nocap encap polymerase bind genomic promoter start transcription dpol dt nocap pol pol pol concentration polymerase locate segment gene figure pol elongation rate polymerase attenuation factor gene sec total number segment gene mRNA length mRNA table genome segment continuously charge incoming polymerase discharge outgoing polymerase rate pol Equations polymerase input leader region genome decrease owe lack free polymerase polymerase concentration downstream leader region subsequently reduce figure conserve intergenic sequence involve let fraction viral polymerase release genome template intergenic section transcription call partial transcription termination attenuation figure transcription initiate end promoter attenuation mechanism cause gene proximal end highly express ultimately lead unequal concentration distribution viral mrnas extent partial transcription termination quantify attenuation factor model Figure leader intergenic region respectively fraction polymerase release intergenic region Equations simulate polymerase flux gene segment proportional elongation rate polymerase inversely proportional extent attenuation dpol dt pol pol dpol dt ðpol pol pol concentration polymerase locate jth segment gene indicate prior gene gene newly synthesize mrnas gene determine concentration polymerase occupy gene section genome template decay rate mrnas mRNA concentration mrnas gene mRNA decay rate constant mrna viral mrnas pol total concentration polymerase occupy ith gene formulation transcription assume synthesis viral mrnas rate control transcription initiation elongation polymerase transcription initiation rate parameterize spacing neighboring polymerase model give polymerase elongation rate large polymerase spacing indicate low rate transcription initiation transcription initiation modulate input polymerase leader region genome translation consider translation initiation polypeptide chain elongation contribute rate viral protein synthesis translation initiation rate parameterize ribosomal spacing model calculate number ribosome involve viral translation consider maximum concentration ribosome bind viral mrnas fix ribosomal spacing rib rib avail concentration ribosome actually involve viral translation ribosome available viral translation respectively rib spacing neighboring ribosome ribosome involve viral translation rib allocate type viral mrnas accord length abundance assume viral mrna efficiency translation initiation rib concentration ribosome assign mRNA synthesis rate viral protein depend elongation rate ribosome linear density ribosome correspond mrna order decay rate dp dt rib concentration protein rib elongation rate ribosome length protein dp decay rate constant protein account consumption free protein encapsidation genome length nascent rna assume degradation nucleocapsid yield intact protein dp dt rib ðencap pol tr tr pol lec lecwhere stoichiometry protein single nucleocapsid virion progeny table pol tr pol lec concentration polymerase locate trailer complementary leader region genome anti genomes respectively lec length complementary leader region nc decay rate constant nucleocapsid progeny virion assemble concentration protein reduce correspond stoichiometry single virion particle replication assume protein regulate switch role polymerase transcription replication encapsidate newly synthesize RNAs polymerase start replication leader region genome require supply protein skip attenuation signal gene junction complete round replication depend availability protein encap fraction polymerase terminate replication gene junction model dpol dt ðencap pol pol pol concentration replicate polymerase segment gene section negative sense genomic nucleocapsid dpol dt ðencap pol pol pol pol concentration replicate polymerase segment gene segment gene respectively level polymerase scan genome pol tr determine newly synthesize anti genomic nucleocapsid dpol tr dt pol pol tr tr dðþÞnc dt encap pol tr tr ðþþnc tr trc length trailer region genome consider order kinetic decay anti genomic nucleocapsid synthesis decay genomic nucleocapsid describe way anti genomic nucleocapsid polymerase anti genomic template release intergenic region dpol dt ðencap pol trc trc sec trc lec pol pol concentration replicate polymerase jth segment positive sense anti genomic nucleocapsid total length genome sec total number segment genome Figure dpol dt sec trc lec ðpol pol sec dpol lec dt sec trc lec pol pol lec lec dðàþnc dt ncap pol lec lec ðÀÞnc model non encapsidated nascent genome anti genome fragment release polymerase immediately degrade polymerase leave promoter region move downstream sequence concentration polymerase promoter decrease dynamic change polymerase concentration promoter genomic antigenomic template finally describe respectively pol pol pol new pol pol leave concentration polymerase leave genomic promoter concentration component time time numerical integration time time Dt respectively pol pol pol new pol pol trc leave concentration polymerase leave anti genomic promoter assembly bud assume condensation negative sense nucleocapsid protein initiate virion assembly condensed nucleocapsid degrade manner virion progeny requirement stoichiometric amount protein satisfied progeny virion instantaneously assemble release extracellular space time require condensation negative sense nucleocapsid assembly budding progeny virion assume negligible relative precede step host cell model host cell provide unlimited building block nucleoside triphosphate amino acid growth virus viral component accumulate course infection key host component translation initiation elongation factor deplete main viral product leader mrna protein contribute deficiency inhibit synthesis host macromolecule transcription level leader mrna start accumulate soon initiation infection small component trigger inhibition pool host factor continuously reduce onset infection quantify reduction single decay rate constant specific type host cell dec level host translation factor time relative initial state cell infection host decay rate constant inhibition leader mrna cause order decay host factor result shortage ribosome equip accessory factor viral translation late infection stage model unlike viral transcription replication viral translation directly affect decay host factor depend entirely host machinery early infection host mrnas outnumber viral mrnas successfully compete host translation machinery newly synthesize protein inhibit host transcription initiation export host mrnas nucleus cytoplasm cause gradual shift translation host mrnas viral mrnas model assume potency inhibition protein independent type cell differentiation state develop empirical formula available experimental datum literature account competition host viral mrnas ribosome lyle al cotransfecte host cell VSV mRNA chloramphenicol acetyl transferase CAT plasmid DNA quantify expression CAT base activity function expression VSV protein experiment gene expression CAT reduced high protein expression level assume decrease expression CAT activity decrease proportional decrease occupancy host mrnas translation machinery experimental datum occupancy host mrnas translation machinery correlate number newly synthesize protein cytoplasm fraction translation machinery associate host mrnas cell total number newly synthesize protein cell consider decay host factor competition host viral mrnas derive formula quantify number fully functional ribosome available viral protein synthesis time post infection nrib avail nrib denote total concentration ribosome incorporate require accessory factor translation function ribosome distribute membrane bind cytoplasmic form class support synthesis viral protein viral protein respectively treat ribosome model population initial condition simulation initial condition simulation set fix number infectious extracellular virus particle cell ex time zero number bind virus particle level viral component cell zero model binding extracellular virus particle cell reduce level equation encapsidated genome stoichiometric amount viral protein table immediately release bind virus particle cytoplasm specifically assume protein bind virus particle release form encapsidated genome complex Downstream process begin transcription initiate simulation viral infection start simulation show result key model parameter simulation summarize Table addition nomenclature list show Table figure ratio Genome Anti genome Simulated VSV infection depend Relative Promoter Strengths relative strength anti genomic promoter relative genomic promoter give prom PI stand time post infection figure Productivity Ranking Gene Shuffled Viruses provide Parameter Constraint value cond correspond host parameter value host rib virus productivity gene shuffle virus BHK cell determine normalize high productivity simulated ranking strain match experimentally observe ranking narrow range cond indicate bar initial number infectious extracellular virus particle cell find DOI KB Parameter Estimation','2006-09-01 00:00:00','http://dx.doi.org/10.1371/journal.pcbi.002011',4677,1050),('4db3931c4b24ae432d68b7e24f593b1272445523.json','Differentially profiling the low-expression transcriptomes of human hepatoma using a novel SSH/microarray approach','BACKGROUND: The main limitation in performing genome-wide gene-expression profiling is the assay of low-expression genes. Approaches with high throughput and high sensitivity for assaying low-expression transcripts are urgently needed for functional genomic studies. Combination of the suppressive subtractive hybridization (SSH) and cDNA microarray techniques using the subtracted cDNA clones as probes printed on chips has greatly improved the efficiency for fishing out the differentially expressed clones and has been used before. However, it remains tedious and inefficient sequencing works for identifying genes including the great number of redundancy in the subtracted amplicons, and sacrifices the original advantages of high sensitivity of SSH in profiling low-expression transcriptomes. RESULTS: We modified the previous combination of SSH and microarray methods by directly using the subtracted amplicons as targets to hybridize the pre-made cDNA microarrays (named as \"SSH/microarray\"). mRNA prepared from three pairs of hepatoma and non-hepatoma liver tissues was subjected to the SSH/microarray assays, as well as directly to regular cDNA microarray assays for comparison. As compared to the original SSH and microarray combination assays, the modified SSH/microarray assays allowed for much easier inspection of the subtraction efficiency and identification of genes in the subtracted amplicons without tedious and inefficient sequencing work. On the other hand, 5015 of the 9376 genes originally filtered out by the regular cDNA microarray assays because of low expression became analyzable by the SSH/microarray assays. Moreover, the SSH/microarray assays detected about ten times more (701 vs. 69) HCC differentially expressed genes (at least a two-fold difference and P < 0.01), particularly for those with rare transcripts, than did the regular cDNA microarray assays. The differential expression was validated in 9 randomly selected genes in 18 pairs of hepatoma/non-hepatoma liver tissues using quantitative RT-PCR. The SSH/microarray approaches resulted in identifying many differentially expressed genes implicated in the regulation of cell cycle, cell death, signal transduction and cell morphogenesis, suggesting the involvement of multi-biological processes in hepato-carcinogenesis. CONCLUSION: The modified SSH/microarray approach is a simple but high-sensitive and high-efficient tool for differentially profiling the low-expression transcriptomes. It is most adequate for applying to functional genomic studies.',' Microarray powerful technique simultaneously determine expression thousand gene study quickly yield genome wide description mrna expression call transcriptome give cell tissue give physiologic pathologic condition main challenge genome wide gene expression profiling difficult inspection gene rare transcript hand PCR base suppressive subtractive hybridization SSH technique highly sensitive identify difference gene expression potential SSH assay dynamic change gene expression minute level address addition SSH technique restrict term limited specificity difficulty identify enriched gene combine SSH technique highthroughput screening harvest clone use cdna microarray greatly reduce tedious work northern blot analysis likelihood false positive clone enrich SSH combine approach print clone obtain SSH amplicone chip successfully profile differentiation gene expression approach gene subtract clone remain sequence identification large portion redundancy enriched amplicon identify target hybridize amplicon clone print chip un enriched cdna pool subtract enriched clone approach increase sensitivity detection low expression gene report modification subtract amplicon target hybridize pre prepared microarray chip SSH microarray analysis probe microarray chip characterized hundred thousand gene subtract amplicon determine single hybridization relative expression status compare tissue gene augment easily determine combine use target prepare forward reverse subtraction SSH name modify approach SSH microarray human hepatocellular carcinoma model demonstrate feasibility approach SSH cDNA microarray versus regular cdna microarray approach conduct comparative transcriptomic study human hepatoma particularly gene low expre sion level set approach show Figure PCR base suppressive subtractive hybridization enrich differentially express cdna clone human hepatoma non hepatoma liver tissue SSH conduct forwards RNA prepare hepatoma non hepatoma liver tissue tester driver respectively yield hepatoma subtract amplicon reversely mRNA prepare non hepatoma hepatoma liver tissue tester driver respectively yield non hepatoma subtract amplicon instead previously report combination cdna microarray approach clone subtract cdna library clone print chip directly label subtract amplicon target hybridize pre microarray chip print know cdna clone name modify approach SSH microarray tissue RNA pair cdna microarray chip regular cdna microarray assay comparison human hepatoma generally occur patient chronic hepatitis cirrhosis identify gene implicate common biological process lead hepatocellular carcinogenesis select hepatoma patient different underlying liver disease chronic hepatitis cirrhosis second chronic hepatitis cirrhosis chronic hepatitis clinical histological evidence cirrhosis figure demonstrate representative result obtain correspond regular cdna microarray SSH microarray assay note SSH microarray assay gene subtract amplicon readily identify tremendous sequencing work require original SSH microarray combination assay addition clone include house keep gene actin GAPDH unremarkable difference hybridization intensity regular cdna microarray assay efficiently exclude subtract amplicon figure indicate white circle indicate efficiency subtraction hybridization importantly clone hybridization intensity evaluation threshold regular cdna microarray approach hybridization intensity low close noise detectable microarray method figure indicate red circle observation suggest sensitive detection low abundance transcript SSH microarray assay figure experimental flowchart Messenger RNA prepare hepatoma correspond non hepatoma liver tissue subject PCR base suppressive subtractive hybridization SSH follow result subtract cDNA library target cdna microarray analysis SSH microarray conventional cdna microarray analysis SSH perform forward tester reverse tester direction enrich regulated amplicon regulated transcriptome amplicon human hepatoma respectively subtract amplicon label fluorescent cy dyes target microarray analysis dye swap approach result obtain compare obtain conventional cdna microarray assay differentially express gene categorize group detect conventional cDNA microarray approach II identify conventional cdna microarray SSH microarray approach iii obtain SSH microarray assay result confirm qrt PCR gene randomly select group III differentially express gene label hybridize chipsTo address SSH microarray approach able assay expression gene rare transcript selectively inspect gene low expression total clone chip clone filter analysis regular cdna microarray study low hybridization intensity intensity unit noise subtract figure select examination hybridization intensity SSH microarray assay figure clone enrich SSH microarray assay analyze differential expression intensity unit noise intensity subtract figure SSH microarray approach allow gene low abundant subsequent analysis SSH specifically amplify difference gene expression SSH microarray approach able identify differentially express gene hepatoma non hepatoma liver tissue originally undetectable regular cdna microarray method show Fig lowexpression gene identify SSH microarray assay identify additional gene fold decrease increase hepatoma tissue respectively examine sensitivity detection differential expression approach compare ratio intensity clone obtain regular cdna microarray assay obtain SSH microarray assay show Figure total gene ratio low fold log cDNA microarray assay ratio great fold log SSH microarray assay representative result regular cdna microarray vs figure representative result regular cdna microarray SSH microarray assay RNA prepare hepatoma para hepatoma liver tissue patient HCC leave panel representative result obtain regular cdna microarray assay Cy label aRNA generate hepatoma non hepatoma liver tissue respectively right panel representative result obtain SSH microarray assay label forward reverse subtract amplicon respectively low panel close view upper panel white circle indicate clone equal hybridization intensity conventional cdna microarray assay subtract away subtract amplicon red circle mark clone low hybridization intensity conventional cdna microarray present differential expression subtract amplicon SSH microarray figure gene indicate red box respectively modify SSH microarray approach result differentially express gene particularly rare transcript regular cdna microarray approach total gene differentially express human hepatoma finally identify base datum derive tissue pair duplicate log regular cdna microarray assay SSH microarray assay respec tively Figure additional file microarray assay detect fold differentially express gene particularly low expression gene conventional cdna microarray confirm differential expression gene low expression identify SSH microarray assay include regulate gene respectively quantify relative expression level randomly select gene differential expression detect SSH microarray assay pair HCC match non hcc liver tissue find result consistent obtain SSH microarray assay figure differential low expression transcriptome SSH microarray figure Detection differential low expression transcriptome SSH microarray distinct gene chip gene hybridization intensity low unit define low intensity threshold exclusion microarray analysis representative set result conventional cdna microarray assay low expression gene subject calibration intensity SSH microarray assay gene detectable SSH assay blue spot distribution low expression gene identify SSH microarray assay include gene regulate hepatoma tissue respectively datum present figure base result derive patient case chronic hepatitis cirrhosis hepatoma compare HCC regulate transcriptome identify SSH microarray assay accordance potential molecular function implicate biological process sub cellular localization present Table Microarray technique limit detection gene low expression subtractive hybridization method restrict specificity tremendous work need validate result sequence identify gene report microarray technique rapid high throughput validation subtraction specificity SSH spot enriched clone chip membrane approach require tremendously sequence work identify gene include great number redundant clone subtract amplicon lose sensitivity SSH technique detect low abundance transcript sensitivity microarray analysis determine target hybridize chip probe print chip report modify approach instead subtract clone probe spot chip directly label enriched amplicon target hybridize pre microarray chip microarray analysis name SSH microarray report approach allow readily identify gene tremendous sequencing work subtract amplicon regardless redundant clone easily evaluate subtraction efficiency specificity SSH microarray approach possible conduct transcriptomewide identification differentially express gene particularly low expression report absolute expression level crucial determinant identify gene relative difference expression level impact gene recover subtractive hybridization report SSH microarray approach identify time differentially express gene particularly low expression human hepatoma finding successfully demonstrate transcriptome wide assay differentially express gene low abundance simple powerful approach greatly facilitate future research functional genomic main concern SSH technique specificity combine microarray figure Comparison relatively expression regular cdna microarray SSH microarray distribution ratio intensity obtain conventional cDNA microarray assay SSH microarray assay show clone significant intensity assay include analysis clone inside red box log ratio conventional cDNA microarray assay log ratio SSH microarray assay datum present base derive patient case chronic hepatitis cirrhosis hepatoma log SSH Log cdna nique subtraction efficiency readily evaluate show figure subtraction efficiency determine rare presence clone equal hybridization intensity forward reverse amplicon consistent exclusion housekeeping gene subtract amplicon specificity confirm quantification differential gene expression hepatoma para hepatoma differentially express gene Figure differentially express gene gene differentially expression human hepatoma identify conventional cdna microarray assay gene SSH microarray assay gene gene result derive pair hepatoma liver tissue fold difference gene expression TTN MEST VLDLR MYOG AHSG ttid SYK LMNA FALZ NEUGRIN relate regulation cell differentiation embryonic development gene AHSG PTPRF NCSTN SYK CSNKIE PNOC FGR associate signal transduction response extra cellular proliferation growth stimulus find finding differentially express gene relate multi biological process suggest complexity molecular mechanism hepatocellular carcinogenesis study modify previously method combination suppressive subtractive hybridization microarray technique differentially profile low expression transcriptome human hepatocellular carcinoma directly label reciprocal subtract amplicon target cdna microarray assay compare SSH previous SSH conjunction microarray approach modify approach provide additional advantage easy inspection subtraction efficiency avoidance tremendous sequencing work gene identification high sensitiv ity identify low expression differentially express gene approach allow detection time differentially express gene regular cdna microarray approach human hepatoma particularly low expression approach identify gene potentially implicate human hepatocarcinogenesis identify high efficiency high sensitivity SSH microarray approach powerful rapid differentially profile lowexpression transcriptome adequate apply functional genomic study SSH cdna microarray analysis hepatoma correspond non cancerous liver tissue obtain patient liver surgery Chang Gung Memorial Hospital reverse transcription real time PCR hepatoma match non hepatoma liver tissue obtain additional patient hepatoma diagnosis HCC non hepatoma liver tissue base histo pathologic finding Internal Review Board Medical Ethics Chang Gung Memorial Hospital approve speciman collection procedure informed consent obtain subject subject family total RNA hepatoma para hepatoma liver tissue RNA prepare describe SSH perform Clontech PCR Select cDNA Subtraction kit Clontech Laboratories Palo Alto describe manufacturer follow modification start material consist μg hepatoma mrna tester μg nonhepatoma liver tissue mrna driver vice versa primary secondary PCR condition alter increase specificity amplification accord plan reduce extension time number cycle primary PCR min cycle respectively primary PCR product dilute prior use secondary PCR aspect plan instruction manufacturer Plan diverge plan way initial cycle primary PCR perform annealing extension time reduce min respectively second subsequent cycle denature time increase anneal extension time reduce min respectively study GMRCL Human set Version chip previously describe contain sequence verify human cdna clone map distinct gene sample regular cdna microarray SSH microarray assay perform dye swap microarray design minimize labeling bias statistical variance datum regular cdna microarray experiment μg total RNA labeling hybridization Array Detection kit Genisphere PA SSH experiment μl subtractive PCR product label Cy dctp NEN Boston MA random primer unincorporated fluorescent nucle otide remove Qiaquick PCR purification kit Qiagen fluorescent label dna mix μg human DNA Invitrogen μg yeast tRNA precipitate resuspend μl Microarray Hybridization Buffer Version Amersham Pharmacia hybridization solution heat min denature dna incubate min allow yeast tRNA block repetitive sequence genome probe probe hybridize human cdna microarray GMRCL Human scan slide confocal scanner ChipReader Virtek Canada acquiredquantification relatively gene expression human hepatoma figure Quantification relatively gene expression human hepatoma total differentially express gene include GBA PTPRF OGDHL belong Class III identify SSH microarray assay belong class II identify SSH microarray regular cdna microarray assay assay relative expression HCC correspond non hcc liver tissue patient hepatoma real time semi quantitative RT PCR result present log value gene expression ratio HCC match non hcc liver tissue left panel relatively expression level gene initially determine SSH microarray regular cdna microarray assay respectively note difference expression gene belong Class III originally identify SSH microarray assay regular cdna microarray assay spot background intensity GenePix Pro software Axon Instruments USA slide normalization program write MATLAB software MathWorks MA validate result obtain SSH microarray assay randomly select differentially express gene identify SSH microarray assay differentially express gene identify regular cdna microarray SSH microarray assay comparison gene expression hepatoma correspond non hepatoma liver tissue eighteen patient HCC reverse transcription realtime PCR qrt PCR total RNA extract tissue Trizol reagent reverse transcribe SuperScript III strand synthesis system Invitrogen Carlsbad qRT PCR conduct ABI PRISM sequence detection system Applied Biosystems pre design Assays Demand TaqMan probe primer pair gene obtain Applied Biosystems incoporated ABI Foster City gene set Taq Man probe primer test probe contain carboxyfluorescein phosphoramidite FAM dye label end gene minor groove binder non fluorescent quencher end design hybridize exon junction result fluorescent signal generate assay genomic DNA substrate confirm assay measure mrna equal amount RNA qrt PCR reaction perform triplicate ribosomal rna internal control datum derive pair HCC non hcc liver tissue dye swap approach filter regulate regulate gene HCC fold difference value include study removal un annotate gene total HCC regulated gene subject subsequent gene ontology analysis list differentially express gene analyze online software FatiGo comparative gene ontology category include molecular function biological process cellular component import line software pathway mapping statistical significance define HCC regulated gene group Fisher exact test','2006-05-31 00:00:00','http://dx.doi.org/10.1186/1471-2164-7-131',2040,584),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json','Protection of pulmonary epithelial cells from oxidative stress by hMYH adenine glycosylase','BACKGROUND: Oxygen toxicity is a major cause of lung injury. The base excision repair pathway is one of the most important cellular protection mechanisms that responds to oxidative DNA damage. Lesion-specific DNA repair enzymes include hOgg1, hMYH, hNTH and hMTH. METHODS: The above lesion-specific DNA repair enzymes were expressed in human alveolar epithelial cells (A549) using the pSF91.1 retroviral vector. Cells were exposed to a 95% oxygen environment, ionizing radiation (IR), or H(2)O(2). Cell growth analysis was performed under non-toxic conditions. Western blot analysis was performed to verify over-expression and assess endogenous expression under toxic and non-toxic conditions. Statistical analysis was performed using the paired Student\'s t test with significance being accepted for p < 0.05. RESULTS: Cell killing assays demonstrated cells over-expressing hMYH had improved survival to both increased oxygen and IR. Cell growth analysis of A549 cells under non-toxic conditions revealed cells over-expressing hMYH also grow at a slower rate. Western blot analysis demonstrated over-expression of each individual gene and did not result in altered endogenous expression of the others. However, it was observed that O(2 )toxicity did lead to a reduced endogenous expression of hNTH in A549 cells. CONCLUSION: Increased expression of the DNA glycosylase repair enzyme hMYH in A549 cells exposed to O(2 )and IR leads to improvements in cell survival. DNA repair through the base excision repair pathway may provide an alternative way to offset the damaging effects of O(2 )and its metabolites.',' oxidative stress lead overproduction free radical lung present clinical situation clinical setting include acute respiratory distress syndrome ARDS infant prematurity go develop bronchopulmonary dysplasia BPD pathogenesis chronic obstructive pulmonary disease COPD asthma cystic fibrosis ischemia reperfusion injury druginduce lung toxicity cancer aging use oxygen clinically indicate hypoxemic situation consider potential longterm toxic effect example know oxygen create cellular damage variety mechanism normal cellular metabolism oxygen involve transfer electron NADH molecule form water normal partial pressure oxygen molecule reduce partially reduce toxic byproduct normal metabolism mitochondria metabolite include superoxide anion hydrogen peroxide hydroxyl radical oh know Reactive Oxygen Species ROS exposure condition hyperoxia ionize radiation IR lead increase amount ROS damaging effect ROS know attack lipid protein nucleic acid cell tissue lipid include pulmonary surfactant react ROS produce lipid peroxide cause increase membrane permeability inactivation surfactant inhibition normal cellular enzyme process protein react ROS result decrease protein synthesis inhibition ribosomal translation destruction form protein ultimately lead inactivation intracellular enzyme transport protein result impaired cellular metabolism accumulation cellular waste product lastly ROS cause damage nucleic acid lead modified purine pyrimidine basis apurinic AP apyrimidinic site dna protein cross link lead single strand break defense mechanism exist combat damaging effect ROS intracellular enzymatic system include superoxide dismutase eliminate superoxide anion catalase catalyze reduction directly production hydroxyl radical glutathione peroxidase directly reduce lipid peroxide free radical scavenger stop free radical chain reaction accept electron include tocopheral vitamin ascorbic acid vitamin niacin vitamin riboflavin vitamin vitamin ceruloplasmin system usually provide protection oxygen metabolism normal condition deplete condition increase oxidative stress defense mechanism interest paper involve repair oxidative damage human DNA base excision repair pathway BER BER important cellular protection mechanism remove Base excision repair pathway oxidative DNA damage oxidative dna damage damaged basis excise replace multi step process lesion specific dna glycosylase repair gene initiate process removal damage base result AP site cleave APendonuclease generate phosphate drp polymerase possess dRPase activity cleave drp residue generate nucleotide gap fill single nucleotide gap final nick seal DNA ligase figure oxidative repair gene analyze study include oxoguanine dna glycosylase human Mut homologue hMYH human Mut homologue hMTH endonuclease III hNTH present human cell involve protection dna oxidative damage repair enzyme purine oxidation glycosylase recognize excise oxoguanine lesion pair cytosine pair cytosine adenine dna replication repair occur transversion result repair enzyme hmyh oxoguanine mismatch glycosylase remove adenine misincorporate opposite oxog lesion arise dna replication error repair enzyme hMTH hydrolyze oxidize purine nucleoside triphosphate oxo dgtp oxo GTP oxo datp hydroxy datp effectively remove nucleotide pool prevent incorporation DNA figure lastly repair gene endonuclease III hNTH pyrimidine oxidation hydration glycosylase recognize wide range damage pyrimidine hnth show similar dna glycosylase AP lyase activity remove oxog oxog oxog oxog mispair subsequent step follow hNTH identical follow figure previous study show expression DNA repair gene lead reduce hyperoxiainduce dna damage human alveolar epithelial cell primary goal present study compare protective effect main lesion specific dna glycosylase repair gene individually overexpresse lung cell determine provide great degree protection condition increase oxidative stress human alveolar epithelial cell line year old caucasian male purchase ATCC Cat cell grow DMEM Gibco Grand Island NY supplement fetal bovine serum FBS HyClone Logan UT penicillin streptomycin µg ml Gibco Grand Island cell perform day cell grow confluency cm cell culture dish Corning Incorporated Corning NY cell keep humidified incubator retroviral vector gift Baum University Hamburg Germany construct internal ribosome entry site IRES upstream gene express enhanced green fluorescent protein egfp previously describe dna repair gene individually ligate retroviral vector initially amplify PCR primer introduce kozak sequence end digestion product ecori sali perform subclone digest plasmid vector DNA ligase dna sequencing perform confirm integrity hmth hMYH gift McCullough University Texas Medical School Galveston TX hmth clone Kelley lab Plasmid DNA prepare digestion ecori sali ligate sequence perform confirm integrity gene wild type gene gift Mitra University Texas Medical School Galveston TX digestion BamHI sali perform wt fragment ligate plasmid vector wt fragment excise side flank ecori restriction site ligate proper orientation gene confirm sequence perform determine integrity gene cell suspend viral supernatant plate plate polybrene Sigma Louis MO exposure perform hour day day approximately day beginning infection infected cell analyze flow cytometry sort egfp expression cell pellet sorted cell resuspend NuPage buffer Invitrogen Carlsbad protein concentration determine DC protein assay bio Rad Hercules ug protein load individual lane NuPage Bis Tris Gel Invitrogen Carlsbad gel transfer nitrocellulose paper Osmonics Inc Gloucester MA membrane block blocking solution Roche Diagnostics Indianapolis hour room temperature incubate overnight rabbit polyclonal antibody Novus Biologicals Littleton CO hMTH Novus Biologicals Littleton CO hMYH Oncogene Research Products Darmstadt Germany hNTH Proteintech Group Inc Chicago IL dilution hNTH dilute wash time TBST time blocking solution minute wash membrane incubate anti rabbit secondary antibody hour room temperature lastly membrane wash time TBST minute wash membrane briefly soak BM chemiluminescence blot substrate Roche Diagnostics Indianapolis expose high performance autoradiography film Amersham Biosciences Buckinghamshire England Kodak Digital Science Image Analysis software utilize quantify region interest ROI band mass individual band film visualized difference detect sort egfp positive cell infect dna repair gene count seed plate density cell well gene hour seed individual plate place oxygen chamber supply Haneline Wells Center Research Indianapolis locate incubator oxygen chamber infuse co individual plate remove hour exposure Control cell incubate normal CO incubator concentration monitor MAXO analyzer Maxtec Salt Lake City UT day beginning exposure cell assess cell growth survival sulforhodamine assay SRB srb assay Sigma Louis MO develop National Cancer Institute provide sensitive measure drug induce cytotoxicity colorimetric endpoint non destructive indefinitely stable visible naked eye assay assess cell growth survival express cell cold TCA fix cell plate incubation hour individual well rinse water air dry srb solution add cell allow stain minute acetic acid wash rinse unincorporated dye incorporate dye solubilize µl mM Tris Absorbance measure tunable microplate reader Molecular Devices Sunnyvale wavelength nm background absorbance measure nm subtract measurement nm sort egfp positive cell seed plate density cell hour seed individual plate expose radiation dose Rads plate place CO incubator hour day plate remove cell fix analyze srb assay look cell growth non toxic condition growth curve exponential growth equation determined look double time DT cell infect repair gene interest compare vector infect uninfected wild type cell drug exposure experiment perform time individual drug dose include well group infected cell analysis cell growth exponential growth equation determine Microsoft Excel experiment involve drug exposure normalize zero dose datum express mean SE significance difference calculate pair Student test significance accept dna repair gene hmyh hmth hNTH ligate retroviral vector figure vector derive murine stem cell virus backbone individual repair gene transfection phoenix amphotropic cell viral supernatant collect stably infect analysis cell express individual repair gene effect endogenous glycosylase level contain gene interest integrate dna datum blot analysis perform sorted cell order verify expression gene interest hMYH hmth hNTH detect correct position western blot datum analysis utilize assess overexpression individual repair gene result altered endogenous expression repair gene non toxic toxic condition hr Rad cell express repair gene hmyh hmth hnth lead altered expression endogenous repair gene condition compare vector control cell figure endogenous expression level detection pattern endogenous expression hnth consistent condition compare cell express hMYH hmth reduce expression hnth exposure note lastly assess endogenous expression individual repair gene cell infect follow non toxic toxic condition hr Rad hr onset exposure endogenous hMYH hMTH express degree endogenous expression level detection western analysis result show analyze endogenous hnth expression note hyperoxia hr hrs result reduce protein expression respectively small increase expression hnth note Rad day post exposure baseline day post exposure ROI band mass quantification demonstrate finding figure replicate perform western analysis determine consistency result cell express hmyh demonstrate increased survival exposure condition elevated level oxygen compare cell express vector figure result highly significant time point hour reach highly significant value difference hMYH varied hour exposure hour exposure cell express hMYH demonstrate increased survival exposure dose radiation comparison Figure result highly significant dose radiation Rads reach highly significant value difference hMYH varied dose radiation note experiment vector control cell demonstrate experiment look effect cell express repair gene demonstrate increase survival repair gene compare vector control cell figure datum demonstrate expression hMYH ability improve cellular survival condition hyperoxia radiation able overcome toxicity growth normal condition ascertain determine expression repair gene cause alteration growth cell absence oxidative stress wild type cell cell express hnth hMTH appear grow similar rate double time range cell express hmyh slow growth rate result significant difference cell number day calculated double time cell express hmyh hrs long cell repair gene vector figure slowing growth allow time repair cell survival analysis follow IR treatment oxidative stress lung lead cellular dna damage evidence release specific gene product know regulate dna base excision repair pathway alteration pro inflammatory mediator transcription factor related gene product observe injury show associate feature cellular necrosis apoptosis resultant cellular inflammation death oxidative stress dramatic impact outcome patient clinical setting current clinical therapy oxidative stress lung involve supportive measure prevention research deal oxidative lung injury focus mainly enhance antioxidant enzymatic process free radical scavenger ability alter cellular survival increase specific dna repair mechanism add approach treatment oxidant mediate lung injury investigator hydrogen peroxide substitute hyperoxia know metabolite produce metabolism oxygen ros produce hyperoxia clearly lead dna damage question exist lead deleterious effect dna hyperoxia analysis growth curve exposure comparison hyperoxia IR clearly indicate cellular protection oxidative dna repair gene specific agent protection observe expression repair gene follow exposure speculate damage cause dissimilar damage involve oxidize basis include form DNA lipid protein damage correct oxidative dna repair gene alternatively type damage evoke correct express repair gene form stress know induce damage formation ROS IR radiation induce free radical damage DNA substantial overlap oxidative damage protection provide specific oxidative dna repair gene condition IR notable experiment repair enzyme primary agent utilize induce formation ROS oxygen rich environment use oxygen stressor lead formation ROS offer distinct advantage IR mimic clinical situation constant exposure hyperoxia lead cumulative cellular damage compromise repair determine survival cell enhance small degree increase expression repair enzyme hmyh unexpected finding anticipate repair gene demonstrate great protection response oxidative stress base previous study experiment utilize colony form assay CFA detect improvement survival additionally cfa provide different result compare SRB assay allow growth analysis short window time furthermore study look repair enzyme hMYH impact survival study investigate repair function hmyh myh deficient murine cell demonstrate transfection myh deficient cell wild type myh expression vector increase efficiency repair interesting observation note experiment lead look individual growth characteristic cell express oxidative repair enzyme cell express repair enzyme hmyh clearly grow slow rate compare enzyme mechanism understand point time repair action Cell growth curve associate doubling time DT Figure cell growth curve associate doubling time dt hMYH know remove adenine misincorporate opposite oxog lesion lesion occur lesion allow replicate correct repair hMYH final corrective measure product hmyh activity lesion allow opportunity remove oxog opposite cytosine know cell possess gene base previous study demonstrate presence gene amplification genomic PCR demonstrate endogenous activity cell oxoguanine bioactivity assay explanation result slow growth create hMYH provide wide window opportunity repair process place ultimately grant endogenous opportunity remove oxog lesion create oxidative stress note method section SRB assay provide sensitive measure drug induce cytotoxicity assess cell proliferation survival reduced cell proliferation cell express hmyh nontoxic condition likely underestimate magnitude protective effect particular repair enzyme fact result significant recent study discover hereditary variation glycosylase hmyh predispose familial colorectal cancer look hMYH variant lung cancer patient identify clear pathogenic biallelic hmyh mutation overrepresentation hMYH polymorphism cell line demonstrate somatic mutation hMYH single nucleotide polymorphism SNPs note impact function SNP unknown appear function hMYH important prevent somatic mutation lead cancer gastrointestinal tract study date demonstrate relationship lung cancer know lung subject large quantity ROS certain condition discuss early formation mutation oxidative stress deleterious effect cell include cellular death necrosis apoptosis tissue viability dependent mutation correction replication survive cell replace die ability enhance cellular survival specific oxidative exposure evident increase production hmyh repair gene experiment additionally want determine level endogenous expression glycosylase repair gene pulmonary epithelial cell line demonstrate different stressor lead alteration endogenous production specific repair gene example show endogenous gene expression elevate follow exposure crocidolite asbestos know cause increase oxog level previously report treatment cell sodium dichromate prooxidant lead reduction protein expression observe additional study demonstrate dose dependent regulation protein expression rat lung exposure cadmium know carcinogen associate formation intracellular ROS experiment able demonstrate hyperoxia IR appear impact endogenous expression hmyh hmth hour follow exposure note endogenous hnth reduce hyperoxia hour onset exposure speculate reduction endogenous hnth secondary hyperoxia relate decrease production increase destruction response exposure expression repair enzyme result improvement survival exposure base experiment endogenous level adequate correct specific mutational burden experiment furthermore previous study determine cell express specific repair gene impact endogenous expression oxidative BER gene normal oxidative stress condition able demonstrate endogenous expression glycosylase repair gene alter condition secondary expression gene important finding interpretation survival datum protection cell overexpression specific gene enhancement endogenous repair enzyme level gene study condition limitation exist lung carcinoma cellline likely differ proliferative property response oxidative stress comparison primary epithelial cell enhanced cell growth observe cell line reflective undifferentiated alveolar type II cell likely replace terminally differentiate alveolar type cell death oxidative stress true reflection growth non toxic condition little cell division occur inherent problem observe compare cell line primary cell result need interpret way consider difficult know translate pulmonary epithelial cell vivo stage certainly appear protection observe modest degree pulmonary epithelial cell line experiment assess function repair enzyme hMYH model important perform order delineate finding slow growth normal condition improve survivability condition IR research look potential combination therapy include dna repair mechanism conjunction antioxidant defense mechanism approach enhance cell survival lead well clinical outcome alternatively cell survival important end point hyperoxia study give oxog leave unrepaire lead transversion increase mutational burden cell reflect cell survival experiment study impact mutation production underway ultimately experiment need animal model determine translation vivo pulmonary cell summary demonstrate expression DNA glycosylase repair enzyme hmyh enhance survival pulmonary epithelial cell line exposure condition IR hyperoxia demonstrate expression hMYH lead slowing growth cell non toxic condition play role enhancement survival provide wide window opportunity repair oxidize lesion occur lastly demonstrate expression lead alter endogenous expression repair gene understanding dna repair mechanism continue grow evolution gene therapy take place treatment option available clinical setting help disease process include damaging effect oxygen metabolite','2004-09-27 00:00:00','http://dx.doi.org/10.1186/1465-9921-5-16',2390,841),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json','A novel endonuclease IV post-PCR genotyping system','Here we describe a novel endonuclease IV (Endo IV) based assay utilizing a substrate that mimics the abasic lesions that normally occur in double-stranded DNA. The three component substrate is characterized by single-stranded DNA target, an oligonucleotide probe, separated from a helper oligonucleotide by a one base gap. The oligonucleotide probe contains a non-fluorescent quencher at the 5′ end and fluorophore attached to the 3′ end through a special rigid linker. Fluorescence of the oligonucleotide probe is efficiently quenched by the interaction of terminal dye and quencher when not hybridized. Upon hybridization of the oligonucleotide probe and helper probe to their complementary target, the phosphodiester linkage between the rigid linker and the 3′ end of the probe is efficiently cleaved, generating a fluorescent signal. In this study, the use of the Endo IV assay as a post-PCR amplification detection system is demonstrated. High sensitivity and specificity are illustrated using single nucleotide polymorphism detection.',' nucleic acid assay utilize cleavage enzyme generate fluorescent signal dual dye label oligonucleotide probe extensively diagnostic assay ability fluorescent dye transfer energy absorb light nearby molecule form basis homogeneous nucleic acid base assay unhybridized state fluorophore quencher close proximity allow quencher absorb energy fluorophore affect quench use fluorogenic probe polymerase assay probe enzymatically cleave release fluorophore known signal amplification reaction utilize invasive cleavage structure specific method require anneal oligonucleotide call upstream oligonucleotide probe target sequence result formation unique substrate DNAzyme PCR strategy primer contain target specific sequence harbor antisense sequence dnazyme amplification amplicon produce contain active sense copy dnazyme cleave reporter substrate include reaction mixture target nucleic acid sequence amplify exponentially vitro isothermal condition enzymatic activity essential retroviral replication reverse transcriptase RNase RNA cleavage dna dependent RNA polymerase cycling probe technology represent simple method detection dna target sequence utilize chimeric DNA RNA DNA probe cleave RNase hybridize complementary target apyrimidinic AP site DNA arise spontaneous mutagen induce hydrolysis glycosylic bond repair activity dna glycosylase leave unrepaire AP site potentially lethal mutagenic cope deleterious consequence AP site organism possess AP endonuclease initiate repair dna lesion Class II AP endonuclease initiate repair catalyze hydrolysis abasic site generate group phosphate residue substrate specificity human apurinic endonuclease synthetic substrate analog report previously describe novel endonuclease IV Endo IV assay substrate figure mimic abasic lesion normally occur double strand dna figure lesion figure typical abasic apurinic apyrimidinic site generate spontaneous enzymatic loss nucleic acid base second lesion figure atypical abasic site appear result inherent instability bond abasic deoxyribose lesion cleavage class AP endonuclease show arrow AP endonuclease cleave phosphodiester linkage abasic site novel Endo IV assay short probe fluorophore link phosphate end short enhancer oligonucleotide generate artificial lesion type Endo IV substrate arrangement allow specific efficient cleavage phosphodiester bond probe Endo IV release fluorescent dye Figure addition quencher end probe allow quench fluorescence uncleaved probe specific cleavage phosphate bond generation fluorescence basis new assay study endonuclease IV Escherichia coli Endo IV enzyme function Class II AP endonuclease terminal phosphodiesterase Endo IV enzymatically similar exonuclease III major AP endonuclease differ enzyme mainly lack associate exonuclease activity find Endo IV fairly heat stable characterization compound describe find Supplementary Data total unrelated Centre Etude Polymorphism Humaine DNA sample obtain Coriell Institute Medical Research specify DNA sample research purpose list template available synthesis fluorophore modify control pore support Supplementary Data fluorogenic probe synthesize fluorophore linkerbase cpg support scheme standard Eclipse Dark Quencher introduce end correspond phosphoramidite www probe purify reverse phase high performance liquid chromatography HPLC dry dissolve TE buffer near neighbor model apply calculate extinction coefficient oligonucleotide measurement PBS pH ambient temperature assume random coil dna structure solution Eclipse Quencher fluorophore fluorophore fluorophore substitution correction cm respectively preparation label probe aminohexanoylextended hydroxyprolinol linker FI table modify probe precursor synthesize modified CPG support Eclipse Quencher phosphoramidite mmol scale probe purify reversephase HPLC gradient acetonitrile triethylammonium bicarbonate buffer pH follow dry SpeedVac evaporator probe precursor redissolve ml dry DMSO treat mmol pentafluorophenyl aminohexanoate FMOC aminohexanoyl modify probe isolate reverse phase HPLC deprotecte treatment concentrated ammonium hydroxide room temperature deprotected oligonucleotide purify reverse phase HPLC triethylammonium bicarbonate buffer describe dry dissolve ml dry dmso solution mmol compound scheme ml triethylamine add keep room temperature reaction combine ml solution NaClO acetone precipitate crude conjugate purify reverse phase HPLC dry reconstitute TE buffer additional molar absorbance cm probe concentration calculation correct presence dye synthesis special CPG support preparation require oligonucleotide probe show Schemes Methyl dihydroxyphenyl propanoate scheme synthesize start convert substituted cinnamic acid Knoevenagel condensation Intermediate hydrogenate presence Pd yield substitute phenyl propionic acid demethylate HBr HOAc convert methyl ester Synthesis pentafluorophenyl ester dye intermediate require synthesize dye modify CPG support show Scheme Compounds react substitute phthalic anhydride presence AlCl benzophenone react resorcinol analog trifluoroacetic methanesulfonic acid yield carboxyethyl substitute dye treatment compound trifluoroacetic anhydride generate lactone react dmt protect hydroxyprolinol trimethylacetic anhydride afford protect dyes reaction intermediate succinic anhydride pentafluorophenyl trifluoroacetate yield PFP ester reaction PFP ester long chain aminoalkyl CPG LCAA CPG yield desire CPG support show scheme IV cleavage activity measure real time show figure model system Figure determine optimum Endo IV Trevigen Gaithersburg MD concentration magnesium concentration pH determined mM Tris buffer pH endo IV reaction prepare add ml Endo IV Master Mixture ml PCR mixture Reaction incubate min fluorescent detection perform ABI channel typically detect FAM VIC utilize ABI software Allelic Discrimination alternatively fluorescence detect figure respectively reaction mixture contain Endo IV ml concentration probe enhancer nm target experiment perform LightCycler LightCycler Real Time PCR System assays indicate perform triplicate thermodynamic parameter duplex formation derive Hoff analysis method shape melting curve fit state model linear base line nonlinear square program state dna duplex measure concentration PCR buffer contain mM Tris HCl pH mM MgCl mM NaCl describe early MGB Eclipse Design Software design primer PCR calculate enhancer probe example new Endo IV base assay show figure probe enhancer hybridize complementary synthetic target base gap signal substantially diminish absence enhancer rigid hydroxyprolinol linker figure oligonucleotide fluorophore critical sensitivity specificity experiment result figure illustrate importance component assay cleavage fluorophore highly target dependent rate amplify presence enhancer oligonucleotide low non target dependent cleavage important feature Endo IV assay effect pH cation Endo IV activity determine probe enhancer target Figure show Endo IV cleavage probe relatively flat ph optimum range pH compatibility post pcr Endo IV amplification assay buffer pH choose similar PCR master mixture inclusion divalent magnesium cation buffer range mm result fold increase cleavage rate optimum range concentration mm choose Endo IV assay determine monovalent cation Li Na Rb inhibit Endo IV cleavage activity concentration study range inhibition observe mm monovalent cation concentration increase mm concentration optimum probe concentration determined nm Supplementary Figures figure show model Endo IV assay require target complementary enhancer probe label end Eclipse Dark Quencher fluorescent dye end show probe calculate enhancer calculate separate base mimic natural abasic site probe different fluorescent dye linker figure evaluate performance new assay fluorophore probe II iii similar quantum yield contain hydroxyprolinol linker probe sequence core dye fluorophore probe IV analogous probe III additional aminocaproic spacer introduce hydroxylprolinol moiety dye purpose spacer distance fluorophore cleavage site probe specificity evaluate compare cleavage rate presence absence target Targetdependent specific target independent non specific cleavage rate probe measure percent target cleave minute result summarize Table Probes III demonstrate similar specific cleavage fold decrease rate non specific cleavage probe II iii versus probe observe low non specific cleavage rate translate high specific non specific cleavage ratio appear chloro methyl substitution fluorophore probe II III respectively beneficial reduction non specific cleavage possible explanation reduction effect increase hydrophobicity enzyme activity case probe IV extended hydroxyprolinol linker rate specific cleavage double corresponding increase rate non specific cleavage time high observe probe III substantially compromise ratio specific non specific cleavage rate result imply existence inhibit interaction Endo IV enzymatic activity fluorophore adjust move fluorophore away active center rigid hydroxyprolinol linker low non specific cleavage rate utilize probe III identify linker choice Endo IV assay use flexible straight chain linker instead hydroxyprolinol base linker increase rate nonspecific cleavage high degree extended hydroxyprolinol linker reduce specific nonspecific ratio unacceptable level datum appear structure sterically constrain hydroxyprolinol linker proximity bulky dye moiety cleavage site moderate cleavability phosphodiester bond practically eliminate non template dependent cleavage figure show target probe enhancer align target indicate different gap enhancer typically choose high probe cleavage rate presence enhancer show base gap figure expect base natural abasic substrate requirement show figure base gap give high cleavage rate probe absence enhancer show cleavage Table effect dye linker rate ratio specific non specific cleavage Probe nos Dye Substitution Figure Cleavage rate Specific non specific rate ratio Target specific target non specific rate nearly equivalent base gap presence enhancer show Figure maximum cleavage rate achieve concentration enhancer great practice concentration enhancer typically equivalent large probe cleavage observe enhancer present ii Cl iii Cl CH IV Cl CH CTCGGCCCTT DyeProbe optimization figure show cleavage rate probe different change temperature probe length range investigate show Figure calculate probe show bell shape relationship cleavage rate temperature expect cleavage rate probe increase increase temperature relatively sharp cleavage rate optimum observe probe calculate respectively appear good probe cleavage occur close calculate slightly perform assay temperature slightly high Figure show optimum activity appear allow probe cleavage cycling important feature assay ability Endo IV assay detect different target concentration show Figure observe case nM target concentration probe base fluorescence plateaue presence nm target fluorescence plateau reach limit detection assay nm clearly distinguishable stable background figure stable background characteristic feature assay maintain period property observe assay nucleic acid target ability Endo IV assay discriminate mismatch different position probe illustrate figure mismatch introduce probe time position Endo IV assay show excellent specificity different mismatch base base large match mismatch signal ratio show Figure mismatch signal NTC slightly high exception difficult mismatch match mismatch ratio observe respectively ratio adequate differentiate match mismatch sequence appear datum show discrimination position largely determine substrate requirement discrimination position determine thermodynamic consideration experiment large number assay indicate probe satisfactory discrimination observe mismatch position basis end exquisite specificity Endo IV assay appear result factor strict Relative Flourescence intrinsic substrate specificity requirement Endo IV enzyme appear require rigid prolinol linker figure probe III second use short probe give Endo IV distinct advantage assay probe typically long application Endo IV cleavage post pcr detection system show figure follow asymmetric PCR primer produce predominantly single strand amplicon allele specific probe enhancer Endo IV enzyme add amplification mixture incubate isothermally hour cleaved fluorescence allele specific probe label different fluorescent dye plot scatter diagram show figure genotype analysis single nucleotide polymorphism SNPs immediate importance cancer scatter diagram polymorphism agouti signal molecular epidemiology study protein adenomatous polyposis coli show figure respectively case probe specific wild type allele label probe specific mutant allele label probe set design analyze polymorphism assay unrelated human dna sample obtain Coriell Institute case target mismatch situate basis end probe case mismatch basis end probe experience spacing allele cluster figure comparable well scatter plot observe method exquisite specificity achieve Endo IV enzyme combine short probe high signal accumulation give assay advantage SNP assay comparison signal accumulate assay Endo IV assay similar level high signal accumulation use short probe high specificity assay case hybridization base assay Endo IV assay signal accumulation advantage use short probe specificity advantage addition short probe allow flexibility assay design compare method long probe disclose novel nucleic acid signal detection system base cleavage phosphodiester linkage dualdye label oligonucleotide probe Endo IV enzyme parameter influence activity Endo IV enzyme optimize optimize Endo IV detection assay require addition oligonucleotide probe substrate fluorophore attach relatively rigid linker oligonucleotide helper oligonucleotide rigid linker convey exquisite specificity Endo IV cleavage phosphodiester linkage probe show high specific non specific rate ratio synthesis fluorophore analog couple rigid linker solid support synthesis label oligonucleotide disclose excellent mismatch discrimination observe position probe ability mismatch position allow flexibility probe design genotype assay sequence secondary structural constraint consider method allow use probe short typically method contribute improve mismatch discrimination combination Endo IV assay PCR post amplification detection system demonstrate Post PCR Endo IV genotyping show excellent spacing scatter plot endpoint signal template control wildtype mutant heterozygous sample allow clear allele differentiation contrast endpoint genotype method suffer overlap endpoint signal PCR efficiency issue Endo IV post pcr genotyping successfully industry set genotype assay','2006-09-29 00:00:00','http://dx.doi.org/10.1093/nar/gkl679',1875,670),('53dbd21da3e3a026ab15e946639b45970d6304de.json','Public health preparedness in Alberta: a systems-level study','BACKGROUND: Recent international and national events have brought critical attention to the Canadian public health system and how prepared the system is to respond to various types of contemporary public health threats. This article describes the study design and methods being used to conduct a systems-level analysis of public health preparedness in the province of Alberta, Canada. The project is being funded under the Health Research Fund, Alberta Heritage Foundation for Medical Research. METHODS/DESIGN: We use an embedded, multiple-case study design, integrating qualitative and quantitative methods to measure empirically the degree of inter-organizational coordination existing among public health agencies in Alberta, Canada. We situate our measures of inter-organizational network ties within a systems-level framework to assess the relative influence of inter-organizational ties, individual organizational attributes, and institutional environmental features on public health preparedness. The relative contribution of each component is examined for two potential public health threats: pandemic influenza and West Nile virus. DISCUSSION: The organizational dimensions of public health preparedness depend on a complex mix of individual organizational characteristics, inter-agency relationships, and institutional environmental factors. Our study is designed to discriminate among these different system components and assess the independent influence of each on the other, as well as the overall level of public health preparedness in Alberta. While all agree that competent organizations and functioning networks are important components of public health preparedness, this study is one of the first to use formal network analysis to study the role of inter-agency networks in the development of prepared public health systems.',' international national event bring critical attention canadian public health system prepared system respond type con temporary public health threat threat result emerge infectious disease bioterrorism public health system responsible protect health population Outbreak response occur local regional level potential impact threat overall canadian population differ significantly depend capacity public health system respond example follow SARS outbreak Toronto Canadian National Advisory Committee NAC SARS Public Health hold fortunate SARS outbreak strike primarily Toronto part Canada capacity combat public health threat limit general renewal public health infrastructure capacity respond order effective coordination agency regional provincial federal level necessary assure comprehensive surveillance management public health threat Ralph Klein Premier Alberta comment Albertans Canadians understand disease outbreak like SARS West Nile virus region affect part country coordinated approach help deal public health threat commitment federal provincial agency Canada improve public health system performance develop overall public health preparedness little know current system level state public health preparedness Canada coordination see necessary element public health preparedness measure inter organizational collaboration relevant public health preparedness develop assess relation feature organizational inter organizational environment study protocol present premise conceptual model method measure assess public health preparedness Alberta Canada provincial public health agency embed federal global public health preparedness system focus primarily organizational environment public health preparedness Alberta develop contextually relevant measure evaluation technique address issue relate potential cross alberta variation public health preparedness Alberta public health emergency response procedure primarily govern Disaster Services Act Public Health Act provincial response public health disaster event action organization unit jurisdiction local municipality regional health authority emergency management district provincial government number organizational actor involve preparation response recovery phase public health disaster include department portfolio agency potential actor provincial public health emergency response network consider overall public health emergency management network consist poten tial organizational actor involve respond public health threat precise set organization come particular public health threat depend nature threat example pandemic influenza West Nile virus emerge infectious disease nature transmission differ distinctive set organizational actor place confront case West Nile Virus Alberta Department Environment Sustainable Resource Development integrally involve mosquito surveillance control integral event pandemic influenza outbreak assess potential difference public health threat Alberta public health preparedness examine compare potential public health threat pandemic influenza West Nile Virus embed multiple case study integrate qualitative quantitative method project propose measure empirically degree seamless coordination exist municipal regional district level provincial public health emergency management agency Alberta federal authority agency play important role provincial level response public health emergency purpose research focus inter organizational linkage organizational environment provincial sub provincial actor relevant federal level organization identify future investigation seamless coordination activity task characterize overall federal provincial regional linkage characterize intraprovincial linkage fluid communication resource flow diverse organization administrative authority require integrated coordinated network actor research premise empirical research show effective coordinated approach public health threat require integrate responsive inter organizational system appropriate method analyze inter organizational relation degree integration base network analysis strong inter organizational tie effective preparedness organization involve necessary individual capacity respond public health threat regard fluid communication integrate organizational connection element overall system level approach assess public health preparedness show Figure view public health preparedness product interrelated factor individual organizational attribute inter organizational network exist relevant organizational actor institutional environment organization inter organizational network locate threat emerge integrate coordination specific action set actionset refer case subgroup organization network come purposefully address specific public health threat factor influence system level public health preparedness target population research public health emergency management relate organization Alberta construction project sampling frame base stratified multistage cluster design strata administrative division provinciallevel agency sub provincial administrative agency emergency management district disaster service regional health authority metropolitan area town local area strata sample frame include public health emergency management organization agency department unit strata fourth stratum substratifie accord Alberta Emergency Management district division geographical area northwestern Alberta northeastern Alberta northcentral Alberta central Alberta south central Alberta southern Alberta geographical region draw proportionate random sub sample town stratum sample frame consist public health emergency managementrelate organization town randomly select project frame population consist organization list strata sampling frame list stratum sampling frame study conduct phase phase qualitative phase quantitative research phase help identify organization responsible public health emergency response relevant relationship initial qualitative research consist analysis official document report interview key organizational representative government document report analyze help identify agency involve response system determine formal task assign agency initially assess overall level preparedness responsiveness case study interview Attributes key informant health official determine additional organization involve response system note official document type relationship characterize inter organizational tie actionset develop relevant measure institutional environment public health preparedness responsiveness law mandate certain organization involve emergency response specific public health threat key organizational representative identify agency play important albeit informal role local emergency response system addition helping identify organization absent official document interview investigate perceive characteristic inter organizational relationship mandated relationship example tend characterize low level perceive cooperation organization characteristic investigate quantitative method phase qualitative research reveal point conflict cooperation emerge course public health emergency response provide basis question organizational questionnaire final focus phase interview centre development measure institutional environment network effectiveness public health preparedness resource key agency representative identify critical important organization network process respondent use evaluate organization system phase draw non proportional sample project sampling frame interview organizational representative provincial level stratum representative sub provincial level strata representative local level stratum phase interview respondent organizational representative randomly select strata choose participate interview tape record transcribe content analysis thematic inter textual analysis conduct interview transcript code indicator organization involve public health emergency management preparedness type information sharing organization type resource flow exist organization perception local provincial public health preparedness content analysis identify relevant organization identify documentary research thematic analysis identify critical theme emerge interview inter textual analysis compare response interviewee strata dimension jurisdiction domain geographical location phase involve administration inter organizational questionnaire Alberta base public health emergency management agency information organization inter organizational network collect web base telephone questionnaire phase intend administer questionnaire representative organization list project sampling frame describe early questionnaire consist component organizational attribute component inter organizational network component organizational environment network assessment component organizational attribute component consist question general characteristic organization staff size training background specific occupational role organization sectoral operation budget emergency response relate question agency formally assess epidemiology capacity inter organizational network component consist item ask representative identify organization organization tie specific dimension specific content dimension network map determine follow phase analysis candidate dimension organizational network study include information sharing ii resource sharing include staff iii joint planning critical dimension identify phase respondent present list organization relevant action set ask share information share resource component questionnaire consist respondent subjective assessment institutional environment inter organizational network impact perceive public health preparedness example representative assess environment operate resource easily available scarce prepared perceive organization local system public health threat mandate relation involve sequential interdependence important question organization perceive task transition organization function smoothly actual question component questionnaire determine follow phase datum analysis ethical approval research grant Conjoint Health Research Ethics Board Calgary Health Region University Calgary Faculty Medicine Capital Health Region University Alberta capacity organization prepare public health threat influence system organization locate preparedness overall system influence strength weakness single organization cross case intra provincial comparative design allow research discriminate different system element examine independent influence organizational attribute institutional environment inter organizational network public health preparedness addition mixed method approach enable research analyse role informal formal relationship emergence inter organization network development prepared responsive public health system far aware project assess public health preparedness systemslevel account critical role inter organizational relationship play emergence cohesive responsive system decipher independent influence inter organizational tie overall system research provide measure help researcher policy maker evaluate degree coordination organizational actor gap system linkage organization need central preparedness activity question research design answer','2006-12-28 00:00:00','http://dx.doi.org/10.1186/1471-2458-6-313',1283,430),('555bbbbd23da02c97c6a4238937e31b5781438a9.json','Influenza pandemic intervention planning using InfluSim: pharmaceutical and non- pharmaceutical interventions','BACKGROUND: Influenza pandemic preparedness plans are currently developed and refined on national and international levels. Much attention has been given to the administration of antiviral drugs, but contact reduction can also be an effective part of mitigation strategies and has the advantage to be not limited per se. The effectiveness of these interventions depends on various factors which must be explored by sensitivity analyses, based on mathematical models. METHODS: We use the freely available planning tool InfluSim to investigate how pharmaceutical and non-pharmaceutical interventions can mitigate an influenza pandemic. In particular, we examine how intervention schedules, restricted stockpiles and contact reduction (social distancing measures and isolation of cases) determine the course of a pandemic wave and the success of interventions. RESULTS: A timely application of antiviral drugs combined with a quick implementation of contact reduction measures is required to substantially protract the peak of the epidemic and reduce its height. Delays in the initiation of antiviral treatment (e.g. because of parsimonious use of a limited stockpile) result in much more pessimistic outcomes and can even lead to the paradoxical effect that the stockpile is depleted earlier compared to early distribution of antiviral drugs. CONCLUSION: Pharmaceutical and non-pharmaceutical measures should not be used exclusively. The protraction of the pandemic wave is essential to win time while waiting for vaccine development and production. However, it is the height of the peak of an epidemic which can easily overtax general practitioners, hospitals or even whole public health systems, causing bottlenecks in basic and emergency medical care.',' recent spread highly pathogenic avian influenza Asia Europe transmission human intensify concern emergence novel strain influenza pandemic potential inter pandemic stage nation plan pandemic contingency follow recommendation national influenza preparedness plan constantly refine aim mitigate effect pandemic influenza national regional local level absence pandemic strain seasonal influenza cause substantial morbidity mortality seasonal outbreak pressure general practitioner strain hospital resource lead bottleneck outpatient treatment hospital admission capacity intervention strategy reduce impact influenza individual public health system interpandemic phase vaccination important tool reduce morbidity mortality potent vaccine probably generally available initial phase pandemic control strategy like pharmaceutical antiviral non pharmaceutical intervention reduction contact rate implement use antiviral drug pandemic treatment choice present country afford stockpile drug furthermore concern reliance pharmaceutical solution express epidemic mitigate reduce contact rate general population decrease infectivity case reduction achieve measure like quarantine case isolation closing day care centre school cancel mass gathering event voluntary self isolation general behavioural change public increase social distance effectiveness intervention depend factor prospectively explore sensitivity analysis base mathematical model use freely available Java applet InfluSim investigate effectively pharmaceutical non pharmaceutical intervention contribute mitigate influenza pandemic vaccine available particular examine intervention delay determine course pandemic constrict success intervention InfluSim deterministic compartment model base system thousand differential equation extend classic seir model clinical demographic parameter relevant pandemic preparedness planning detail simulation discussion standard parameter value describe previously summarize description model provide Appendix program source code publicly available offer transparency reproducibility simulation produce time course cumulative number influenza case outpatient visit apply antiviral treatment dose neuraminidase inhibitor hospitalization death work day lose sickness associate financial loss analysis present base InfluSim demographic public health parameter represent situation Germany intervention include antiviral treatment isolation patient social distancing measure closing day care centre school cancel mass gathering event standard set InfluSim parameter freely accessible infected individual expect severely ill seek medical help patient seek medical help refer outpatient paper exponential distribution model delay onset symptom seek medical help average patient visit doctor hour patient seek medical help hour onset symptom give antiviral treatment stockpile antiviral exhaust antiviral treatment reduce duration degree infectivity case number hospitalization table detailed description Appendix non pharmaceutical intervention examine paper contact reduction measure isolation case effectively lead reduce contact rate individual scenario present assume everybody population avoid give percentage contact improved hygiene wear mask behavioural change sick patient isolate reduce contact rate moderately sick severely sick non hospitalize hospitalize case respectively intervention comprise closing day care centre school cancelling mass gathering event examine detail separate paper assume basic reproduction number standard parameter set InfluSim epidemic population individual reach peak day introduction infection practically week intervention perform figure epidemic population infected seek medical help hospitalize die figure show pharmaceutical non pharmaceutical intervention mitigate scenario contact reduction isolation case Appendix protract peak epidemic week distribution antiviral additional contact reduction measure delay epidemic approximately day hardly sufficient provide substantial delay combination antiviral treatment isolation case social distancing general population necessary delay epidemic order week example furthermore show efficient mitigation epidemic necessarily associate significant reduction number infection information proportion infected people outpatient legend figure mitigating effect antiviral strongly depend onset distribution figure antiviral delay epidemic distribute early case exist population late distribution antiviral start day lead paradoxical effect stockpile exhaust quick compare early distribution shaded area und curve figure additionally mitigating effect intervention drastically diminish benefit restrict lower peak epidemic unrestricted availability drug grey curve figure lead epidemic asymptomatic moderately sick case eligible treatment ii patient visit doctor average hour onset symptom highly infectious iii antiviral fully prevent infectivity figure contrast mitigating effect negligible antiviral distribute delay figure independent delay distribution antiviral quantitative availability affect height peak epidemic hardly mitigation epidemic figure consideration final size epidemic summary delay epidemic depend early action lower peak depend quantitative availability antiviral contact reduction measure comprise social distancing isolation case effective mitigation strategy advantage antiviral treatment limit se continue sufficiently long period time figure examine effect isolation case social distancing measure figure caption detail absence antiviral treatment peak epidemic protract day percent contact reduction intervention start immediately introduction infection peak shift possible early action degree contact reduction contact reduction initiate later peak shift diminishe proportionality remain example intervention start week introduction infection peak epidemic mitigate half day contact reduction Figure premature cessation contact reduction measure restore infection rate pre intervention value fuel epidemic lead delayed course high total number infection involve plateau second peak epidemic figure precede example intervention base antiviral contact reduction yield peak delay order week month require vaccine development production demand combine intervention scheme Figure examine optimistic scenario antiviral distribute immediately infection introduce dark bar Figure vary onset social distancing measure antiviral stockpile last long social distancing measure initiate early pale bar Figure immediate initiation contact reduction protract epidemic month delay initiation lead plateau epidemic curve time antiviral intervention population infect course epidemic cumulative number outpatient reach reflect assumption approximately infected individual sufficiently sick seek medical help outcome remain surprisingly stable intervention assume optimistic resource cf footnote figure instance immediate unlimited availability antiviral reduce fraction figure minor effect reason case seek medical help receive antiviral treatment infection pass case seek medical help antiviral treatment fully prevent transmission disadvantage apply contact reduction measure instance reduction contact reduce fraction figure combination antiviral treatment contact reduction reduce value Figure precede analysis assume parameter value precisely know real world scenario uncertainty arise biological variability stochastic influence heterogeneity etc illustrate concluding example extent simulate epidemic affect uncertainty parameter value show Figure epidemic highly variable parameter varied moderate range vary parameter increase variability intervention parameter variation consider cumulative number outpatient range thousand thousand inset Figure parameter strong predictor number outpatient analysis show strongly determine quickly antiviral exhausted simulation randomly choose parameter combination involve value lead minor outbreak give intervention scheme cumulative number outpatient escalate antiviral stockpile exhausted proportion susceptible large allow propagation infective case epidemic curve proceed second wave plateau pandemic influenza expect unexpected historical report frequently mention surprising speed pandemic wave travel population predict course future pandemic cause virus unknown characteristic base substantial uncertainty rely sensitivity analysis perform mathematical model like InfluSim short serial interval influenza timely action essential different control measure regard complementary compete antiviral treatment non pharmaceutical measure exclusively mitigate pandemic influenza wave infectious disease model suggest upcoming influenza epidemic low basic reproduction number contain source target use antiviral drug publish scenario concern phase inter pandemic alert period assume outbreak start rural area low population density expect pandemic virus introduce Europe local epidemic phase communitybase prophylaxis limited use reason high prevalence infection phase wide distribution require enormous number antiviral course available stockpile virtually impossible locally contain pandemic target antiviral prophylaxis development resistance limited production capacity extremely high cost limitation strategy population wide prophylaxis recommend final phase pandemic discussion pandemic influenza preparedness planning frequently focusse amount drug stockpile supply currently stockpile antiviral drug fully effective pandemic strain use able sufficiently prevent spread influenza transmission infection occur onset clinical symptom assume InfluSim model ii asymptomatic moderately sick case usually treat despite contribute transmission iii occurrence case influenza like illness cause pathogen lead accelerated depletion antiviral stockpile likewise moderately sick case healthy people seek medical help succeed receive antiviral treatment deplete stockpile factor reduce efficacy pharmaceutical control measure indicate demand extend strategy non pharmaceutical intervention measure especially antiviral limited supply early possible distribution delay case abundant resource quickly exhaust have impact spread disease figure confirm antiviral need strongly depend number infection present intervention initiate antiviral drug extremely limited preferably treat severe casesonset sustainability antiviral intervention Intervention limited amount antiviral figure Intervention limited amount antiviral number outpatient expect pandemic wave varied availability antiviral parameter value base InfluSim standard configuration list end legend indicate superscript antiviral availability range antiviral available dash curve antiviral available population Figure Effects contact reduction measure number outpatient expect pandemic wave contact reduction measure implement additionally isolation case parameter value base InfluSim standard configuration list end legend indicate superscript dash curve show epidemic intervention contact reduction involve social distancing isolation case curve effect cause social distancing contact reduce grey curve step black curve left right bar graph illustrate period contact reduction day end delay day end temporarily day day parameter modification give following term italic refer term InfluSim user interface InfluSim output cumulative proportion population infect cumulative proportion outpatient population need hospitalization practically effect pandemic wave se help reduce death toll population result rely pharmaceutical solution pandemic preparedness involve non pharmaceu tical measure early self isolation social distancing measure highly effective show SARS epidemic global alert implementation massive infection control measure effective reproduction number Hong Kong Vietnam Singapore Canada fall unity rigorous social distancing measure entire population Parameter value base InfluSim standard configuration list end legend indicate superscript sensitivity analysis extend scenario show Figure antiviral available population distribute day zero contact reduction measure include isolation case initiate week introduction infection scenario day right panel parameter value realization sample independently normal distribution show mean give bold value lie range specify dotted line truncate basic reproduction number cumulative infectivity half symptomatic period relative infectivity asymptomatic case antiviral treatment reduce infectivity factor parameter increase value aggravate epidemic large plot random realization select extreme epidemic epidemic homogeneously place epidemic cause parameter value draw left tail correspond distribution epidemic cause parameter value draw right tail correspond distribution right panel epidemic curve plateau second wave antiviral stockpile exhaust proportion susceptible large allow propagation infective thin curve black optimistic parameter combination small available stockpile period intervention epidemic curve proceed plateau bold curve grey inset distribution cumulative number outpatient obtain random realization parameter modification give following term italic refer term InfluSim user interface tax social economic structure population willing able reduce contact course pandemic wave Figure assume contact reduction measure improve hygiene wear mask behavioural change add reduce contact study SARS outbreak suggest preventative effect wear mask compliance availability mask effectiveness influenza infection remain unknown factor stockpile surgical mask population result exorbitant high number feasible individual stockpiling impossible economic limitation especially crisis situation specific effect behavio ral change remain uncertain model contribution general reduction contact rate contrast SARS able rely isolate hospitalize case new influenza pandemic emerge standard parameter setting InfluSim expect total population hospitalize bad case scenario Pandemic Preparedness Plan value time large wide majority infected individual hospitalize influenza rely self isolation moderately sick case bed ride patient stay home case form majority infection exert high force infection moderate reduction contact general population substantially change pandemic wave time essence control infectious disease spread high speed intervention effective beginning people infect timely application antiviral drug limited supply quick implementation contact reduction measure notably protract peak epidemic substantially reduce height pandemic influenza wave protraction pandemic wave essential win time wait vaccine development production height peak pandemic wave easily overtax general practitioner hospital public health system lead dangerous bottleneck basic emergency medical care vaccinate small fraction population pre pandemic vaccine similar effect course epidemic reduce basic reproduction number percentage immunize individual sensitivity analysis end Results section show planning intervention strategy base single parameter value address variability detailed analysis present subsequent publication mathematical model like InfluSim predict specific outcome explore good bad case scenario declare compete interest InfluSim deterministic compartment model base system differential equation extend classic seir model clinical demographic parameter relevant pandemic preparedness planning allow produce time course cumulative number influenza case outpatient visit apply antiviral treatment dose hospitalization death work day lose sickness associate economic aspect software program Java open access operate platform independent execute regular desktop computer model structure InfluSim represent figure description give susceptible individual infect rate depend age intervention apply current time infected individual incubate infection mean duration day obtain realistic distribution duration incubation period model stage yield gamma distribute incubation period coefficient variation incubation stage regard early infectious period patient spread infection account average time half day standard set parameter pass incubation stage infected individual fully infective fraction develop clinical symptom figure course disease depend age risk group remain asymptomatic show moderate course disease moderately sick remain severe course disease sick small fraction show extremely severe course disease extremely sick need hospitalization rationale distinguish extremely sick case die disease need hospitalize aspect group severe case identical period infectivity gamma distribute depend course disease age case allow infectivity change course disease apply weight factor depend stage infectivity standard value result infectivity high immediately onset symptom decline geometric progression time figure case seek medical help average day onset symptom wait time visit doctor exponentially distribute sick extremely sick patient visit doctor offer antiviral treatment sick patient advise withdraw home disease extremely sick case immediately hospitalize death rate extremely sick hospitalize case age dependent asymptomatic moderately sick patient pass duration infectivity consider healthy immune sick extremely sick patient convalescent resume ordinary life gamma distribute mean day coefficient variation fully recover patient pass period convalescence join group healthy immune work adult return work child visit day care centre school antiviral treatment severe extremely severe case visit doctor day onset symptom offer antiviral treatment give supply exhaust antiviral treatment reduce patient infectivity percent duration disease risk hospitalization percent extremely sick patient hospitalization prevent treatment send home join group treat sick patient contact rate general population reduce increase social distance close school day care centre cancel mass gathering event behavioural change isolation case reduce contact rate contact necessarily reduce specify user standard scenario consider reduction moderately sick case sick case home extremely sick case hospitalize respectively mixing age class employ whoacquire infection matrix WAIFW matrix give relative frequency contact infective individual age InfluSim assume bi directional contact child total number contact adult adult child order match user specify basic reproduction number diseasespecific infectivity duration infectivity matrix incorporate result generation matrix matrix multiply scale factor choose large eigenvalue equal choose value force infection give product number infective individual correspond age dependent contact rate start simulation infection introduce fully susceptible population avoid bias simulation initial infection distribute age risk class','2007-07-13 00:00:00','http://dx.doi.org/10.1186/1471-2334-7-76',2275,672),('575e5af469e1a43638c3dac9fdafda882fbe9d7a.json','Ventilator associated pneumonia and infection control','Ventilator associated pneumonia (VAP) is the leading cause of morbidity and mortality in intensive care units. The incidence of VAP varies from 7% to 70% in different studies and the mortality rates are 20–75% according to the study population. Aspiration of colonized pathogenic microorganisms on the oropharynx and gastrointestinal tract is the main route for the development of VAP. On the other hand, the major risk factor for VAP is intubation and the duration of mechanical ventilation. Diagnosis remains difficult, and studies showed the importance of early initiation of appropriate antibiotic for prognosis. VAP causes extra length of stay in hospital and intensive care units and increases hospital cost. Consequently, infection control policies are more rational and will save money.',' nosocomial pneumonia NP define parenchymal lung infection occur hour hospital admission account hospitalacquire infection lead cause death nosocomial infection major threat patient admit intensive care unit ICU receive mechanical ventilation MV recent study show ventilator associate pneumonia VAP common infectious complication patient admit ICU result high mortality morbidity prolong length hospitalisation increase cost hospitalisation mortality rate VAP range accord study population average excess cost pneumonia estimate patient patient recent study patient clinical experience major advance diagnostic technique management VAP remain significant problem intensivist review epidemiology diagnosis mainly infection control VAP discuss different study incidence VAP report different depend definition type hospital ICU population study type rate calculate vary large database day point prevalence study conduct european icu pneumonia account nosocomial infection National Nosocomial Infections Surveillance System NNIS NP account nosocomial infection ICU NNIS datum medical icu account recent study report device associate incidence rate ventilator day generally rate VAP surgical ICU high medical icu depend difference patient population surgical disorder proportion patient need MV duration ventilation Kollef et al report incidence NP patient admit cardiothoracic ICU surgical ICU medical occur route haematogenous spread distant focus infection contiguous spread inhalation infectious aerosol aspiration aspiration pathogenic gram positive gram negative bacteria colonize oropharynx gastrointestinal tract main route role route rare microorganism reach distal lung multiply cause invasive disease host defence include filtration humidification air upper airway epiglottic cough reflex ciliary transport respiratory epithelium phagocyte opsonin distal lung systemic cell mediate humoral immunity prevent bacterial invasion ICU host defence patient usually alter underlie disease device cough efficiently sedation underlie disease intubate endotracheal tube hold vocal cord open facilitate aspiration important risk factor NP tracheal intubation associate fold risk increase risk cause sinusitis trauma nasopharynx nasotracheal tube impair swallowing secretion act reservoir bacterial proliferation increase bacterial adherence colonization airway require presence foreign body traumatize oropharyngeal epithelium cause ischemia secondary cuff pressure impair ciliary clearance cough cause leakage secretion cuff require suction remove secretion microorganism adhere surface endotracheal tube specie exude exopolysaccharide act slime like adhesive microbial biofilm tube surface provide reservoir microorganism greatly resistant action antimicrobial host defence patient require MV expose device nebulizer humidifier source microorganism duration MV increase risk infection cook et al report cumulative increase risk VAP time day week MV day second week day week study similarly show risk pneumonia increase duration MV high risk day need reintubation urgent intubation document massive aspiration associate high incidence VAP effect prior antibiotic therapy controversial Sirvent et al report short course cephalosporin prophylaxis associate low rate VAP patient structural coma investigator show antibiotic administer day reduce risk early onset ventilator associate pneumonia prior antibiotic exposure predispose patient subsequent colonization infection resistant pathogen tube impair function gastroesophageal sphincter increase risk maxillary sinusitis oropharyngeal colonization reflux lead migration bacteria pneumonia enteral nutrition give nasogastric tube associate increase risk VAP predispose VAP elevate gastric pH lead gastric colonization increase risk reflux aspiration cause gastric distension patient transportation find risk factor VAP facilitate aspiration contaminated secretion upper airway ventilator circuit supine position independent risk factor VAP show Table identify risk factor guide prevention measure causative organism vary accord patient demographic ICU method diagnosis duration hospital ICU stay antibiotic policy Gram negative bacteria common pathogen cause VAP study NNIS datum frequent reported isolate Staphylococcus aureus report isolate gram negative common gram negative specie Pseudomonas aeruginosa enterobacter specie Klebsiella pneumoniae recent year gram positive bacteria common ICU EPIC study aureus account case identify microorganism nnis datum medical ICU report high percentage aureus polymicrobial infection rate usually high VAP duration MV prior exposure antimicrobial significantly influence distribution pattern etiologic agent early onset VAP day methicillinsensitive aureus Streptococcus pneumoniae Haemophilus influenzae common pathogen methicillin resistant aureus MRSA aeruginosa acinetobacter baumannii Stenotrophomonas maltophilia frequent late onset VAP day MRSA aeruginosa baumannii multi resistant gram negative pathogen common pathogen patient expose prior antibiotic special factor predispose patient infection specific microorganism summarize table determine risk factor microorganism help select appropriate antimicrobial treatment improve outcome diagnosis pneumonia mechanically ventilated patient difficult gold standard diagnostic method usually base combination clinical radiological microbiological criterion define Centers Disease Control CDC table criterion low sensitivity specificity systemic sign fever leukocytosis etc infection see condition ICU pulmonary edema pulmonary infarction surgery trauma devascularized tissue open wound etc investigator report clinical diagnosis VAP associate falsenegative false positive result ICU patient systemic sign infection underlie disease chronic renal failure immunosuppresion etc radiological infiltration limited value mimic cardiogenic pulmonary edema noncardiogenic pulmonary edema adult respiratory distress syndrome ARDS atelectasis pulmonary contusion uncommon ICU autopsy prove VAP study Wunderink et al report radiographic sign diagnostic accuracy great presence air bronchogram sign correlate pneumonia correctly predict pneumonia entire group upper respiratory tract patient colonize potential pulmonary pathogen hour intubation consequently isolation pathogen tracheal secretion indicate pulmonary infection positive Gram stain guide initial antibiotic therapy prior antibiotic corticosteroid therapy reduce sensitivity technique et al propose combine seven variable temperature leukocyte tracheal aspirate volume purulence tracheal secretion chest ray oxygenation pao semiquantitative culture tracheal aspirate diagnosis VAP define clinical pulmonary infection score CPIS table score varied point report CPIS associate sensitivity specificity diagnosis pneumonia post mortem study Papazian colleague report high diagnostic accuracy CPIS threshold sensitivity specificity original scoring system limitation require hour result tracheal aspirate culture identify pulmonary infiltrate progression depend intensivist experience Singh et al modify CPIS calculate baseline clinical variable CPIS hour base variable score antibiotic stop patient persistent low score day therapy avoid unnecessary use antibiotic patient discontinue therapy improve recent study Fartoukh et al report modify CPIS perform well clinical suspicion pneumonia high propose incorporate result specimen gram stain add point gram stain positive modify CPIS increase sensitivity score physician diagnostic accuracy qualitative culture tracheal aspirate TA specific diagnostic method low respiratory tract colonization high percentage false positive result investigator report quantitative culture TA equal diagnostic accuracy invasive technique recent study quantitative culture TA compare plug telescope catheter PTC specificity TA similar PTC cut point cfu mL high sensitivity TA cfu mL low PTC cut point mL sensitivity TA statistically different PTC quantitative culture TA non invasive inexpensive simple method risk cut value cfu mL sensitivity low patient VAP identify cut value cfu mL unnecessary antibiotic treatment give low specificity recent year investigator favour invasive technique diagnosis pneumonia protect speci man brush bronchoalveolar lavage diagnostic accuracy PSB mL secretion collect presence cfu mL bacteria sensitivity specificity diagnosis VAP BAL large proportion lung sample diagnostic threshold cfu sensitivity specificity BAL respectively Heyland colleague propose PSB BAL increase physician confidence diagnosis management VAP allow great ability limit discontinue antibiotic therapy study patient undergo bronchoscopy PSB BAL low mortality rate compare patient undergo bronchoscopy recent meta analysis conclude regular use bronchoscopy diagnosis VAP alter mortality directly affect initial antibiotic prescription disadvantage invasive technique prior antibiotic use decrease sensitivity accuracy method recent study Souweine et al report current antibiotic treatment prescribe prior infectious disease VAP diagnostic accuracy protect speciman brush bronchoalveolar lavage change technique base quantitative culture result culture require hour miss early case information appropriate initial antibiotic therapy invasive test worsen patient status cardiac arrhythmia hypoxemia bleed etc increase cost care prove use invasive technique lead decrease patient mortality spread microorganism blood pleural space blood pleural effusion culture low sensitivity specificity luna colleague demonstrate positive predictive value blood culture detect etiologic microorganism sensitivity blood culture follow criterion rectal temperature blood leucocytosis leave shift blood leukopenia leukocyte Gram stain tracheal aspirate high power field positive culture endotracheal aspirate new persistent progressive radiographical infiltrate clude blood culture patient VAP useful suspicion probable infectious condition isolation microorganism blood confirm microorganism pathogen cause VAP set blood sample culture tap pleural effusion mm perform patient suspect VAP conclusion microbiological testing perform decide appropriate initial empirical antibiotic therapy clinician choose optimal diagnostic test specific patient clinical setting long duration mechanical ventilation long stay ICU increase use antibiotic high cost healthcare importantly increase mortality prevention VAP major priority despite advance pathogenesis VAP intensivist struggle prevention strategy basic hygiene principle infection control hand washing disinfection patient contact use glove sterile equipment remain important prevention VAP healthcare worker HCW spread microorganism patient patient hand easily hcw realize importance handwashing disinfection compliance low especially compliance rate low activity carry high risk transmission ICU high workload decrease compliance wrist watch bangle jewellery act reservoir organism inhibit effective hand cleaning staff wrist watch jewellery achieve effective hand cleaning use gown glove appropriate change wash disinfect hand patient Bedside hand antiseptic alcohol base handrub solution easy access sink availability washing equipment decrease workload communication education tool poster feedback improve compliance decrease cross transmission nosocomial infection internal machinery mechanical ventilator important risk factor VAP filter inspiratory phase circuit patient necessary furthermore importance filter expiratory limb mechanical ventilator circuit prevent cross comtamination know need investigation device respiratory tract come contact mucous membrane clean highlevel disinfection minute reusable equipment require resuscitation bag spirometer oxygen analyzer clean disinfect patient avoid cross transmission study routine change ventilator circuit recommend replacement require gross soilage mechanical malfunction condensate fluid ventilator circuit contain high concentration pathogenic bacteria risk factor VAP remove regularly accidental drainage condensate patient airway contamination caregiver ventilator disconnection disposal condensate avoid line device way valve place disposable circuit empty regularly recommend collect condensate humidification inspired air important care ventilator management humidification achieve active humidifier bubble wick passive humidifier hygroscopic condens artificial nose heat moisture exchanger humidification formation condensate tubing colonization condensate microorganism important risk factor VAP HME recycle heat moisture reduce condensate formation bacterial colonization circuit bacterial filtration characteristic need change daily hour week advantage reduce nurse workload reduce financial cost well safety hme favourable device icu investigator report low rate VAP HME group conventional heat water humidification system effect HME prevention VAP controversial recent study show significant difference VAP rate furthermore hme increase dead space resistance breathing cause airway occlusion tenacious secretion additional study need identify benefit hme infection control medication humidification air insert inspiratory phase mechanical ventilator circuit contaminate condensate tube contaminated solution inoculate infectious aerosol particle directly lung parenchyma cause outbreak icu recommendation infection control nebulizer fill immediately use sterile water drug refill liquid nebulise cleaning disinfect receptacle daily sterile water rinse allow dry patient specific large volume nebulizer patient specific mask mouthpiece connect piece medicine cup secretion trachea approach VAP prevention type tracheal suction catheter ventilated patient open single use catheter closed multiple use catheter single use system hcw use sterile solution rinse catheter care aseptic technique suction endotracheal secretion closed suctioning system secretion suction removal mechanical ventilation support cause hypoxia hypotension arrhythmia environmental contamination similar VAP rate closed open system suggest early trial Combes colleague report time great risk VAP open suctioning system closed suctioning system recent study closed suction catheter extension ventilator circuit daily change catheter necessary infection control study significant difference VAP rate report daily change compare routine change decrease cost use closed suction system recommend VAP prevention program relationship use invasive device nosocomial pneumonia direct investigator use noninvasive ventilation reduce VAP rate study low risk VAP antibiotic use short length ICU stay low mortality report use non invasive ventilation care take use non invasive mechanical ventilation reduce frequency tracheal intubation endotracheal tube alter host defence impairs mechanical clearance respiratory tract cause local trauma inflammation allow pool secretion cuff pressure endotracheal tube cuff sufficient prevent leakage colonized subglottic secretion low airway continuous intermittent suctioning oropharyngeal upper respiratory tract secretion endotracheal cuff prevent aspiration endotracheal tube separate dorsal luman cuff design suction subglottic secretion continuously find able decrease rate early onset VAP randomised trial benefit continous subglottic suction overall VAP frequency find reduce eliminate volume fluid aspirate lung lack effect prevention late onset pneumonia high cost tube restrict usage microbial biofilm endotracheal tube surface reservoir pathogen prevent microorganism action antibiotic adair colleague propose high concentration antibiotic endotracheal luminal surface achieve nebuliser endotracheal surface modification expect prevent biofilm formation endotracheal tube role reduce incidence VAP minimise patient exposure systemic antibiotic nasotracheal intubation increase risk nosocomial sinusitis predispose VAP aspiration infected secretion nasal sinus fore endotracheal intubation prefer decrease risk nasotracheal intubation nasogastric tube cause orophanryngeal colonization nosocomial sinusitis impair function upper oesophagus sphincter facilitate reflux bacteria gut result increase risk VAP randomized study gastroesophageal reflux microaspiration gastric content low airway influence size nasogastric tube potential complication tracheal malposition coil clog high cost small bore nasogastric tube routinely recommend prevention VAP nutritional state hypoalbuminemia contribute development VAP reason early initiation enteral nutrition preventive effect mechanically ventilate patient help maintain gastrointestinal epithelium reduce need stress bleed prophylaxis nasogastric tube alkalinization stomach content feed gastric colonization gastrooesophageal reflux aspiration pneumonia promote recent study postpyloric enteral access placement improve tube feed tolerance reduce rate VAP Heyland et al acidify feed critically ill patient demonstrate dramatic reduction bacterial growth aspirate stomach content low rate gram negative bacterial growth tracheal secretion patient receive acidify feed significant reduction nosocomial pneumonia patient active gastro intestinal bleeding acidemia renal failure furthermore small size study research effect prevention need practice recent year selective decontamination digestive tract SDD extensively study prevention strategy VAP SDD topical non absorb antimicrobial usually combine polymyxin aminoglycoside amphotericin prevent gastrointestinal colonisation pathogenic microorganism selectively eradicate potential pathogenic microorganism gram negative aerobic intestinal bacteria aureus fungus affect anaerobic flora elimination anaerobic flora lead increase colonization gram negative aerobic flora investigator topical antibiotic apply oropharynx nasogastric tube add systemic therapy broad spectrum cefotaxime day prevent early infection pneumoniae influenzae aureus recent meta analysis search randomize control trial publish significant reduction incidence respiratory tract infection total mortality determine meta analysis recent prospective randomized study mention topical antibiotic reduce respiratory infection influence survival threat SDD lead selection overgrowth antibiotic resistant microorganism recent study Netherlands incidence MRSA vancomycin resistant enterococcus VRE low reduction frequency colonization resistant gram negative bacteria effect acquisition MRSA report study icu MRSA endemic increase incidence MRSA report icu high incidence multi resistant microorganism SDD hand trauma surgical patient SDD effective medical patient colonisation conclusion routine use SDD icu recommend decide accord patient population study characteristic addition colonization oral cavity pathogen important risk factor develoment VAP unclear oral care chlorhexidine reduce VAP concern chlorhexidine relate increase colonization gram negative bacteria consider position prevent aspiration passage bacteria airway prefer ICU patient contraindication bed Continous Lateral Rotational Therapy CLRT turn continuously slowly change patient position investigator believe help drainage pulmonary secretion bed expensive effectiveness demonstrate routine use bed recommend chest physiotherapy improve clearance secretion prevention VAP recommend lack prove benefit associated risk arterial oxygen desaturation ulcer prophylaxis propose risk factor alkalinization gastric content effect stress ulcer prophylaxis antacid VAP controversial study use sucralfate associate decrease incidence VAP report support efficient anti ulcer prophylaxis sucralfate choice agent prophylaxis accord patient cost effectiveness reduce aspiration oropharyngeal content use sedative avoid Kress et al report reduce use sedative daily interruption sedative drug infusion patient awake decrease duration mechanical ventilation length stay ICU','2006-04-06 00:00:00','http://dx.doi.org/10.1186/1476-0711-5-7',2431,872),('5806726a24dc91de3954001effbdffd7a82d54e2.json','The 21st International Symposium on Intensive Care and Emergency Medicine, Brussels, Belgium, 20-23 March 2001','The 21st International Symposium on Intensive Care and Emergency Medicine was dominated by the results of recent clinical trials in sepsis and acute respiratory distress syndrome (ARDS). The promise of extracorporeal liver replacement therapy and noninvasive ventilation were other areas of interest. Ethical issues also received attention. Overall, the \'state of the art\' lectures, pro/con debates, seminars and tutorials were of a high standard. The meeting was marked by a sense of renewed enthusiasm that positive progress is occurring in intensive care medicine.',' year symposium dominate result recent clinical trial year magic bullet trial sepsis number successful therapeutic option emerge addition recent advance understanding soup mediator observe sepsis offer tantalizing target new therapy contrast eagerly await result Italy prone positioning trial ARDS dishearten epidemiology sepsis ARDS impact clinical study future provision critical care hot topic era extracorporeal liver replacement therapy considerable early promise probability wide availability finally ethic remain area interest report summarize discuss presentation topic Angus Pittsburgh PA USA present group work epidemiology sepsis USA accept publication Critical Care Medicine develop method identify hospitalize patient sepsis base criterion widely record code system hospital prospective testing method find sensitive reliable apply representative selection hospital result indicate intensive care unit ICU patient systemic inflammatory response syndrome approximately progress severe sepsis mortality severe sepsis great demographically extreme age represent risk group mortality high datum provide reminder increase demand health care resource cause age population predict exceed intensive care provision International Symposium Intensive Care Emergency Medicine Brussels Belgium March year finally investigator find striking demographic peak patient age year attribute largely human immunodeficiency virus long stand debate school sepsis theory dys autoregulation versus cellular dysfunction sign resolution new technique study tissue oxygen tension present Ince Amsterdam Netherlands provide evidence microcirculatory dys autoregulation result significant shunting occur predominantly submucosal serosal portion organ early event study macroscopic restoration global oxygen delivery fail improve oxygen consumption mucosa hyperoxic submucosa serosa remain hypoxic somewhat counterintuitively reverse face resistant hypotension vasodilator animal model cellular dysfunction camp somewhat doubtful importance microcirculatory finding clearly establish championed mechanism mitochondrial failure late crucial event evolution sepsis Fink Pittsburgh PA USA present evidence mitochondrial failure septic cell trigger activation enzyme poly adenosine diphosphate ribose polymerase enzyme represent significant target novel therapy apparently development debate toxicity oxygen formation free radical continue despite absence demonstrate effectiveness scavenge therapy testament incomplete understanding area round table conference precede year symposium concentrate distil current knowledge microscopic event critically ill patient explanation macroscopic multiorgan failure commonly encounter conclusion conference appear relate future direction research particular study organ organ interaction Marshall Toronto Canada propose development alternative malign physiological scoring system base staging system widely field oncology propose mediator level addition physiological variable soon usefully characterize septic patient suggest light recent successful mediator trial sepsis future therapy direct manner analogous control glucose diabetic patient natural anticoagulant antithrombin III III tissue factor pathway inhibitor TFPI activate protein APC cytokine tumour necrosis factor late inflammatory mediator target large multicentre clinical trial attempt improve current dismal outcome patient severe sepsis KyberSept III study recruit patient centre high Simplified Acute Physiology Scale score median mortality nearly unfortunately overall benefit show III placebo result encouraging analysis subgroup patient receive III heparin know inhibit III interestingly improvement quality life score see survivor receive III comparison receive placebo suggest morbidity reduce analysis subgroup patient III group receive concomitant heparin significantly high incidence bleed event outcome worsen dose heparin increase explanation failure study include inhibitory effect heparin failure achieve iii activity level great baseline treatment population level establish require therapeutic benefit phase ii trial phase ii clinical trial result TFPI tfpi placebo unpublished datum mortality benefit sicker sepsis patient coagulation problem result phase III multicentre study expect present International Symposium Intensive Care Emergency Medicine Brussels trial anti tnf formulation variable date include North american sepsis trial NORASEPT international sepsis trial INTERSEPT NORASEPT II possible reason include lack biological activity anti tnf formulation study inappropriate timing therapy redundancy proinflammatory mediator hetereogeneity patient population Monoclonal Anti TNF Randomized control Sepsis Trial MONARCS study different anti tnf formulation fragment murine monoclonal antibody human TNF stratified patient base demonstrable abnormality immunological pathway highly elevate level circulate cytokine induce TNF unpublished result reveal day mortality rate anti TNF placebo group respectively patient high level recruitment study anti TNF placebo represent relative mortality reduction relative mortality reduction patient anti TNF placebo independent baseline level Recombinant Human Activated Protein Worldwide Evaluation Severe Sepsis PROWESS study hot press presentation result congress allow carry unopened New England Journal Medicine issue briefcase catch total site country recruit patient severe sepsis trial prematurely stop follow second safety analysis day cause mortality rate placebo APC respectively relative risk reduction resolution cardiovascular respiratory function rapid survivor receive APC ICU hospital stay differ trend increase bleed event APC group APC versus placebo event primarily trauma instrumentation exciting breakthrough recognize APC reach market place seriously stretch ICU finance especially appear mediator horizon encourage use combination fight inflammatory oppose epidemiological view ARDS present Lemaire Créteil France Evans London UK broad agreement exist incidence condition order significant variation country argue variation result availability ventilated bed high incidence apparent country great provision emphasize condition consider result intensive care intervention speaker physician induce lung early result Acute Lung Injury Verification European Epidemiology ALIVE study unpublished datum sponsor European Society Intensive Care Medicine odd recent trial finding alive study include patient survey find day mortality compare control group recent trial Pneumonia common cause responsible case sepsis identify cause astonishingly study find ratio arterial oxygen tension fractional inspire oxygen ICU admission highly predictive mortality despite continue controversy measurement diverse range view present International Consensus Conference ARDS unpublished datum hold Barcelona late year decision change define criterion condition remain unresolved debate surround chest ray criterion use ratio arterial oxygen tension fractional inspire oxygen level utility pulmonary artery wedge pressure measurement continue addition debate arise ARDS unilateral process coexist cardiac failure appear increase recognition ARDS represent small subset patient acute lung failure approximately surprisingly little know remainder large group contrast alive study centre report day mortality represent improvement year ago argue day follow period short clinical trial long term quality life patient ARDS poor compare critically ill patient condition result suggest recovery lung function good overall dependent severity treatment recommendation include universal adoption National Institutes Health protective lung ventilation strategy general agreement recruitment manoeuvre beneficial employ remain controversial Rouby Paris France forward new classification ARDS base compute tomography finding observe patient split group depend appearance upper lobe group upper lobe normal positive end expiratory pressure PEEP little benefit result significant distension survival group approximately group upper lobe abnormal PEEP dramatic benefit survival approximately group mixed patchy abnormality effect PEEP predictable group survival approximately Milan Italy present result longawaite italian multicentre randomize control trial prone positioning ARDS unpublished datum trial terminate year despite having recruit patient enrollment patient originally plan order achieve sufficient statistical power trial outset recruitment encumber lack familiarity scepticism procedure centre end study participant unwilling enter patient trial feel unethical deny intervention trial protocol result patient treatment group prone average day period overall difference mortality control treatment group day time ICU discharge month interestingly analysis subgroup reveal significant difference outcome day patient severe disease disappear ICU discharge retrospect design ambitious trial flaw failure establish optimal utilization manoeuvre opening session report make progress support fail liver Wendon London UK current optimism probably limit extracorporeal method molecular adsorbent recirculating system essentially extracorporeal albumin dialysis show beneficial clinical effect improved survival small randomized control trial equipment familiar use dialytic therapy undoubtedly hear system year slide patient read newspaper transparent helmet receive noninvasive ventilation NIV resemble picture NASA astronaut report tolerate prolonged period significantly reduce complication associate NIV pressure area tolerance mask recent Consensus Conference examine weaning aspect NIV emphasize reduced weaning time avoidance reintubation call randomized control trial finally continuous positive airway pressure show beneficial pulmonary oedema caution advise use bilevel positive airway pressure reporting myocardial infarction study group study unmatched start point different conclusion draw randomized control trial result available area attend session accord Levy Providence RI USA complete contrast interest show USA subject new datum session emphasis strategy lawsuit welcome suggestion include statement scientific society national international level open reporting medical file decision withdraw withhold treatment family involvement decision making ultimately involve well medium education day year symposium sadly abandon belgian rail strike despite usual convivial atmosphere congress abundant overall state art lecture pro con debate seminar tutorial usual high standard access symposium venue limit lack capacity secondary room International Symposium mark sense renew enthusiasm real positive progress occur coal face intensive care','2001-05-02 00:00:00','http://dx.doi.org/10.1186/cc1013',1329,726),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json','Technical Description of RODS: A Real-time Public Health Surveillance System','This report describes the design and implementation of the Real-time Outbreak and Disease Surveillance (RODS) system, a computer-based public health surveillance system for early detection of disease outbreaks. Hospitals send RODS data from clinical encounters over virtual private networks and leased lines using the Health Level 7 (HL7) message protocol. The data are sent in real time. RODS automatically classifies the registration chief complaint from the visit into one of seven syndrome categories using Bayesian classifiers. It stores the data in a relational database, aggregates the data for analysis using data warehousing techniques, applies univariate and multivariate statistical detection algorithms to the data, and alerts users of when the algorithms identify anomalous patterns in the syndrome counts. RODS also has a Web-based user interface that supports temporal and spatial analyses. RODS processes sales of over-the-counter health care products in a similar manner but receives such data in batch mode on a daily basis. RODS was used during the 2002 Winter Olympics and currently operates in two states—Pennsylvania and Utah. It has been and continues to be a resource for implementing, evaluating, and applying new methods of public health surveillance.',' unfortunately conventional public health disease surveillance rely physician laboratory reporting manual analysis surveillance datum ill equip timely detection threat reportable disease system rely health care professional recognize diagnose report case suspect outbreak public health official unlikely event alert raise index suspicion physician attribute early symptom sign disease bioattack victim appropriately report case key limitation current system lone physician blind case colleague nearby hospital see knowledge lead physician consider uncommon disease strongly diagnostic reasoning mandatory laboratory report illequippe early detection take time test order specimen obtain transport process result sufficiently early detection biological attack accomplish surveillance scheme detect infected individual early disease process completeness note biosensor develop deploy detect organism air type detection feasible occur fundamentally early delay introduce incubation period disease eliminate surveillance system approach face unsolved technical problem analysis contaminated specimen norm air sampling biosensor need right place person lapel street corner hallway provide complete surveillance coverage surveillance method detect disease early stage important research direction public health surveillance method generally refer syndromic surveillance goal recognition outbreak base symptom sign infection effect human behavior prior contact health care system datum syndromic surveillance system establish specific diagnosis particular individual syndromic surveillance system design detect signature pattern disease population achieve sufficient specificity example absurd use symptom fever attempt establish work diagnosis inhalational anthrax individual reasonable establish work diagnosis anthrax release community observe pattern individual fever distribute linear streak urban region consistent prevail wind direction day early reasonable fact imperative establish work diagnosis public health emergency present information recent example form syndromic surveillance drop surveillance stationing public health worker emergency department ed special clinic high profile event Super Bowl capture datum patient present symptom potentially indicative bioterrorism major disadvantage approach cost round clock staffing manual datum collection expensive approach take Realtime Outbreak Disease Surveillance RODS system detection base datum collect routinely purpose example datum include absenteeism datum sale counter OTC health care product chief complaint ed expense manual datum collection avoid datum obtain typically noisy approximation obtain direct interviewing patient case individual level datum approach play complementary role current method public health surveillance assist physician public health official continuously update picture health status population focus research syndromic surveillance free text chief complaint routinely collect triage nurse ed acute care clinic patient registration deploy type surveillance Winter Olympics States Pennsylvania Utah describe previous version RODS system system undergo considerable subsequent development architecturally functionally report provide detailed description current version RODS example computer base public health surveillance system adhere National Electronic Disease Surveillance System NEDSS specification Centers Disease Control Prevention CDC BackgroundThe role public health surveillance collect analyze interpret datum biological agent disease risk factor health event provide timely dissemination collect information decision maker conventionally public health surveillance rely manual operation line analysis exist syndromic surveillance system include CDC drop surveillance system Early Notification Community base epidemic ESSENCE Lightweight Epidemiology Advanced Detection Emergency Response System leader Rapid Syndrome Validation Project RSVP system discuss Lober et al Lober et al summarize desirable characteristic syndromic surveillance system analyze extent system existence characteristic limitation system ESSENCE child Hospital Boston University Washington batch transfer datum delay detection long time interval periodicity batch transfer example surveillance system daily batch transfer delay day detection outbreak system require manual datum input CDC dropin surveillance system RSVP leader labor intensive bad case require round theclock staffing manual data input feasible mid long term solution approach add item exist encounter form item ignore busy limitation exist surveillance system system exploit exist standard communication protocol like Heath Level available data type commonly surveillance system symptom diagnosis patient ED physician office visit type datum identify study include emergency center nurse advice line type datum include sale thecounter health care product prescription telephone volume health care provider drug store absenteeism conduct study demonstrate free text chief complaint datum use correlate outbreak Design ObjectivesThe overall design objective RODS similar early warning system missile defense collect datum require achieve early detection wide area necessary analyze datum way effectively decision maker require analysis close real time design objective complex difficult operationalize large number organism large number possible route dissemination require potentially different type datum detection different algorithm different time urgency reason focus begin project specific problem detect large scale outbreak outdoor outside building aerosol release anthrax additional design objective adherence NEDSS standard ensure future interoperability type public health surveillance system scalability system rely manual datum entry focused way response system analysis passively collect datum report describe RODS completely rewrite Java Enterprise Edition application previous publication describe RODS multidata type enable mean time series datum incorporate database user interface deploy RODS system currently display analyze health care delivery site registration separately monitor sale OTC health care product overview RODS use clinical datum collect health care provider system registration process patient arrive ed InstaCare Utah registration clerk triage nurse elicit patient reason visit chief complaint age gender home zip code datum enter datum registration computer registration computer generate ADT admission discharge transfer message transmit health system message router call integration engine usually message router health system hospital facility process routine exist business activity need create de novo public health surveillance figure show flow clinical datum RODS hospital message router receipt message registration computer delete identifiable information message transmit RODS secure virtual private network VPN lease line Winter Olympics utilize type connection facility fault tolerance RODS listener maintain connection health system message router parse message describe detail pass chief complaint portion message bayesian text classifier assign free text chief complaint seven syndromic category eighth category database store category datum application detection algorithm user interface datum sale OTC health care product process separately National Retail Data Monitor discuss detail article issue JAMIA processing keep separate intentionally future server National Retail Data Monitor operate different physical location RODS RODS user interface display sale OTC health care product discuss user interface connect National Retail Data Monitor prior September RODS receive datum hospital associate UPMC Health System effort recruit hospital meet resistance terrorist attack include anthrax Fall hospital agree participate datum project de identify certain information number ED visit zip code consider proprietary information health system Health Insurance Portability Accountability Act HIPAA concern prominent discussion data share agreement execute participate health system address concern additional precaution RODS project member meet annually University Pittsburgh council review obligation require sign agreement year state understand term data share agreement agree abide term RODS begin research project University Pittsburgh function IRB approval time health care facility send admission discharge transfer ADT message RODS patient visit ed walk clinic minimal data set send show Figure qualify HIPAA Limited Data Set currently datum element age date birth gender home zip code free text chief complaint listener receive message message router locate health system listener pass receive message parser bean Enterprise JavaBean EJB RODS business logic tier parser bean use regular expression parse field message parser bean store parse element database manage database connection pool nearly health system utilize message standard location individual data element message differ health system health system example care provider system record free text chief complaint segment instead segment message resolve mapping problem configuration file write eXtensible Markup Language XML standard protocol define hierarchical datum element define datum element find message listener start read hospital dependent configuration file pass configuration information parser bean use configuration file define database table field parser bean store datum element approach useful allow datum store external database anticipate health department exist NEDSS public health surveillance database wish use component RODS real time collection clinical datum hospital message router approximately experience date RODS accept ED registration datum file secure web base datum upload interface secure file transfer protocol general type datum transfer technically trivial reason group reliability connection undesirable time check integrity datum message receive processing necessary hospital datum flow undesirable characteristic duplicate rods identifie delete duplicate database trigger create composite primary key insert datum RODS filter scheduling message identify fact future admit date time rods monitor datum feed ensure continuous connection health system RODS receive datum hour send alert RODS administrator send health system administrator commercial message router hospital use queue message encounter networking system problem datum integrity preserve RODS use database store ED registration Redwood Shores ensure fast response online query daily count respiratory syndrome county past month develop cache connectivity message router utilize hardware base router vpn router Cisco PIX lease line router pair Cisco Cisco Systems San Jose RODS process run single computer current implementation serve Pennsylvania Sample admission discharge transfer ADT message emergency department circle field age gender home zip code admit date time free text chief complaint Utah application service provider use dedicated server firewall database web server geographic information system GIS server computation process write Java code run platform describe specific platform use indicate approximate sizing processing requirement develop RODS application Java Enterprise Edition Software Toolkit SDK Sun Microsystems cross platform Java application development deployment follow contemporary application programming practice multitiered application consist client tier custom application listener detection algorithm business logic tier database tier web tier business logic parser bean implement Enterprise JavaBeans EJBs NEDSS specifie ejb standard application logic RODS use Jboss opensource application server run ejb web tier comprise graphical user interface RODS use Java Server Pages JSP Java Servlets arcims database tier implement Oracle RODS use naive bayesian classifier call Complaint Coder CoCo classify free text chief complaint follow syndromic category constitutional respiratory gastrointestinal neurological botulinic rash hemorrhagic CoCo compute probability category condition word free text chief complaint assign patient category high probability probability distribution CoCo learn manually create training set CoCo retrain local datum train detect different set syndrome currently use CoCo run local process RODS database server CoCo develop University Pittsburgh available free download course project RODS detection algorithm algorithm formally field test emphasis project date develop datum collection infrastructure field testing algorithm Recursive Square RLS adaptive filter currently run hour alert send public health official Utah Pennsylvania RLS dynamic autoregressive linear model compute expected count syndrome category seven county Utah county Pennsylvania combine count state use RLS minimal reliance historical datum set model parameter high sensitivity rapid increase time series sudden increase daily count RLS trigger alert current actual count exceed confidence interval predict count Olympics strange recent event wsare algorithm WSARE perform heuristic search combination temporal spatial feature detect anomalous density case space time feature include aspect recent patient record include syndromal category age gender geographical information patient criterion past send wsare alert increase number patient specific characteristic relative count day week recent week value careful adjustment multiple testing increase version WSARE incorporate bayesian model compute expect count unadjusted historical count currently development algorithm trigger alert base criterion RODS send mail page alert user RODS use xml base configuration file define user mail pager address mail version alert include url link graph time series trigger alarm comparison time series total visit time period normalize count RODS password protect encrypt web site user review health care registration sale OTC health care product epidemic plot map user log RODS check user profile display datum health department jurisdiction interface comprise screen Main epiplot Mapplot main screen alternate view automatically available datum source currently health care registration OTC product Pennsylvania Utah OTC sale state view alternate minute show Figure clinic visit view show daily total visit seven daily syndrome past week OTC data view show daily sale product category total past week user set view specific county state normalize control box check count time series display divide normalize total daily sale OTC health care product ED visit region Epiplot screen provide general epidemic plotting capability user simultaneously view mixture different syndrome OTC product category geographic region state county zip code time interval user retrieve case detail show Figure case button query database admission date age zip code chief complaint verbatim classify syndrome category patient time interval typically examine anomalous density spike case Download Data button download datum compressed comma separate file analysis Mapplot screen interface ArcIMS Internetenabled GIS product develop Environmental Systems Research Institute mapplot color zip code region indicate proportion patient present particular syndrome GIS server overlay state boundary county boundary water body hospital location landmark street highway public health datum show Figure similar Epiplot Mapplot display case detail user select zip code RODS operation year like production system acquire fault tolerant feature example software level listener continue receive message temporarily store message database line data manager program run minute find cache load unstored message database database line addition datum manager program monitor restart listener necessary database use archive log mode log transaction ensure database recover system failure hardware architecture fault tolerant server dual power supply dual network card hard drive use Redundant Arrays Inexpensive Disk configuration addition dual power supply machine connect uninterrupted power supply capable send mail alert RODS administrator main power important component RODS currently UPMC Health System Pittsburgh Health System Resident component HSRC HSRC locate firewall health system connect directly message router HSRC currently receive diverse set clinical datum message router include culture result radiology report dictate health care registration view main screen RODS Main screen alternate view minute datum type available public health jurisdiction figure show plot health care registration data total visit botulinic constitutional gastrointestinal GI hemorrhagic neurological rash respiratory minute counter datum display main screen situation room display emergency room note purpose provide additional public health surveillance function possible locate outside firewall restriction release identifiable clinical datum HSRC use patient identifier link laboratory radiology information perform case detection past HSRC monitor patient gram positive rod preliminary microbiology culture report mediastinal widening radiology report HSRC case detector distribute outbreak detection system capable achieve high specificity patient diagnostic categorization access information HSRC remove identifiable information transmit datum RODS system function provide health system message router hospital connect HSRC UPMC Health System function electronic laboratory reporting system state local health department ready receive real time messaging system currently send email alert director laboratory hospital infection control group positive culture organism require report public health state Pennsylvania send message hospital infection control detect organism cause nosocomial infection organism include Clostridium difficile methicillin resistant Staphylococcus aureus vancomycin resistant Enterococcus able HSRC prototype additional feature look function facilitate rapid outbreak investigation provide access electronic medical record public health investigator show Figure feature require token pass hospital information system uniquely identify patient reason prototype feature HSRC RODS simply HSRC run firewall unencrypted token lookback accomplish follow public health user identify anonymous patient record interest patient diarrhea today zip code HSRC call UPMC Health System web base electronic medical record system pass patient identifier UPMC Health System request user log upmc issue password provide access record directly secure web site approach intend implement HSRC RODS system outside firewall health system intend use encrypt identifier health system decrypt retrieve correct record HSRC provide encryption decryption service provide data system hospital estimate prevalence health system web base result review United States grow approach quickly improve efficiency outbreak investigation reason move application service provider model dissemination encourage state local health department form coalition support share service fortunate sufficient grant funding Commonwealth Pennsylvania able support service interim basis sustainable funding model evolve original design objective RODS real time collection datum sufficient geographic coverage sample density provide early syndromic warning large scale aerosol release anthrax achieve initial design objective progress substantial research identify type datum freetext chief complaint sale OTC health care uct obtain real time near real time sampling level high United States result obtain large scale deployment RODS Pennsylvania Utah build National Retail Data Monitor describe accompany article issue JAMIA deployment provide insight organizational technical success factor inform effort scale project nationally project establish importance message router know integration engine public health surveillance message router mature highly prevalent technology health care demonstrate free text triage chief complaint obtain real time hospital message router datum represent early syndromal information disease clinical datum value public health transmit standard order diagnostic test especially microbiology test report chest radiograph medication test result integrate RODS surveillance system capable receive message result effort disseminate technology give away learn health department technical resource build maintain real time electronic disease surveillance system application service provider model success ful recommend state form coalition share cost service project early identify need computing component reside firewall health system connect hospital message router component function case detector distribute public health surveillance scheme link laboratory radiology datum increase specificity case detection prove difficult disseminate technology complexity idea threat bioterrorism create need technology approach equivalent function deploy adherence NEDSS architectural standard early design objective meet RODS closely follow NEDSS architectural software message data specification success strong validation standard gain understanding standard attempt use RODS component include listener natural language parser message parser database user interface notification subsystem detection algorithm NEDSS compliant system ongoing project use RODS collect chief complaint integrate Utah Department Health planned NEDSS system demonstrate ability rapidly deploy RODS special event add advantage system look function RODS user select patient investigate screen background partly hide overlap RODS log user result review function electronic medical record request patient chart show screen event experience suggest strongly RODS similar system consider alternative drop surveillance future plan meet initial design objective develop early warning capability large outdoor release anthrax especially ensure datum analysis produce RODS review public health goal require improvement interface detection algorithm reduce false alarm vastly improve efficiency anomaly evaluate use multiple type datum well interface implementation look function like enlarge quickly possible application service provider include state type clinical datum state position prospectively evaluate detection performance different type datum naturally occur outbreak long term goal add additional disease scenario design objective detection building anthrax release vector bear disease food bear disease communicable disease severe acute respiratory syndrome NEDSS compliant public health surveillance system focus real time collection analysis datum routinely collect purpose RODS deploy state instal quickly seven week Olympics experience demonstrate feasibility surveillance system challenge involve outbreak emerge infection bioterrorism threat hope line public health worker astute citizen health care worker detect outbreak early system RODS need timely outbreak detection important leave human detection notion public health operate optimally timely electronic information unwise have commercial airline pilot take weather forecast radar','2003-09-01 00:00:00','http://dx.doi.org/10.1197/jamia.m1345',2982,1022),('5f48792a5fa08bed9f56016f4981ae2ca6031b32.json','Gene expression in epithelial cells in response to pneumovirus infection','Respiratory syncytial virus (RSV) and pneumonia virus of mice (PVM) are viruses of the family Paramyxoviridae, subfamily pneumovirus, which cause clinically important respiratory infections in humans and rodents, respectively. The respiratory epithelial target cells respond to viral infection with specific alterations in gene expression, including production of chemoattractant cytokines, adhesion molecules, elements that are related to the apoptosis response, and others that remain incompletely understood. Here we review our current understanding of these mucosal responses and discuss several genomic approaches, including differential display reverse transcription-polymerase chain reaction (PCR) and gene array strategies, that will permit us to unravel the nature of these responses in a more complete and systematic manner.',' RSV PVM virus family Paramyxoviridae subfamily pneumoviru envelop singlestrande nonsegmented RNA virus cause intense viral bronchiolitis human mouse respectively severe form low respiratory tract infection cause pneumoviruse associate development peribronchiolar infiltrate accompany submucosal edema bronchorrhea ultimately lead bronchiolar obstruction compromise oxygen transfer infection confine respiratory epithelium response cell clearly primary importance determine nature extent result inflammatory process understanding response pneumoviru infection emerge study RSV infection human epithelial target cell vitro list gene gene product produce epithelial cell response RSV infection vitro provide Table cellular level epithelial cell initially respond RSV infection reduce ciliary beat frequency production release chemoattractant cytokine chemokine observe early infection lead recruitment specific leukocyte subset lung tissue RSV infect epithelial cell resistant tumor necrosis factor induce apoptosis later fuse form giant cell syncytia die cellular necrosis review molecular basis extent theyare understand specific response discuss novel strategy permit study response RSV PVM infection coherent systematic manner Tristram et al observe explante respiratory epithelial cell slow ciliary beat frequency immediately exposure RSV complete ciliostasis see early initial infection molecular basis ciliostasis remain completely unknown chemokine cytokine production release associate RSV infection human epithelial cell list Table work recently review focus chemokine molecular mechanism physiologic implication well understand early report subject describe production neutrophil chemoattractant tissue culture supernatant RSV infect cell nasal secretion patient viral rhinitis detect low airway secretion patient severe RSV bronchiolitis neutrophil influx observe response infection probably activity chemokine cellular level production observe response inactivate RSV virion production response active infection inhibit ribavarin amiloride antioxidant group demonstrate activation transcription factor nuclear factor κB NF κB response RSV infection NF κB recognize central role elicit production transcription factor NF produce response RSV infection participate co operative manner NF κB regulation gene expression later study suggest activator function preferentially role interestingly NF κB regulator IκBα function inhibit NF κb activation response TNF produce different kinetic promote reversal NF κb activation response RSV infection response TNF recently Casola et al demonstrate promoter contain independent response element nucleotide represent unique RSV response element distinct element necessary production response TNF concept stimulus specific response probably important contribution understanding pneumoviruse promote transcription unique specific set independent gene product pleiotropic chemokine regulate activation normal cell express secrete rante detect supernatant RSV infect epithelial cell culture upper low airway secretion patient infect virus rante act chemoattractant eosinophil monocyte vitro role vivo somewhat clear similar RANTES produce vitro response inactivated virion involve NF κb activation binding nuclear translocation Koga et al demonstrate stabilization rante mRNA response RSV infection mediate nucleotide rante gene probably primary mechanism underlie increase production secretion rante protein study determine similar mechanism place RSV mediate response group recently show macrophage inflammatory protein release rsvinfected cell culture detect upper low airway secretion RSV infect patient interestingly aforementioned chemokine closely correlate presence eosinophil degranulation product datum pvm model pneumoviru infection suggest play pivotal role eosinophil recruitment response primary pneumovirus infection interestingly production cell culture require active viral replication suggest response proceed mechanism completely distinct elicit production rante report date address molecular mechanism underlie RSV mediate response list cell surface molecule report express response RSV infection show Table focus expression intercellular adhesion molecule leukocyte integrin expression cell surface adhesion protein observe respiratory epithelial cell line human nasal epithelial cell response infection RSV vitro Chini et al demonstrate expression mrna similar rante dependent intact NF κb site gene promoter demonstrate role consensus bind site factor enhancer bind protein Stark et al demonstrate express response RSV serve enhance neutrophil eosinophil binding epithelial cell polypeptide integrin family function mediate cell cell interaction group observe expression epithelial cell response RSV infection show enhance degranulation eosinophil specific setting particular interest recent finding relate expression early literature bacterial superinfection setting viral infection early study report enhanced binding bacteria respiratory epithelial cell infect RSV finding clinical implication relate acute bacterial otitis medium infant recent study address question bind site saadi et al determine strain pathogen Bordetella pertussis bind efficiently RSV infect cell binding reduce pretreatment cell anti anti antibody similarly Raza et al report RSV infect epithelial cell contribute binding nonpilate Neisseria meningitidis vivo testing require clinical significance intriguing finding appreciate RSV infect epithelial cell culture feature suggestive apoptosis ie evidence membrane blebbing fragmentation chromosomal DNA characteristic change nuclear morphology Takeuchi et al show RSV infect epithelial cell express number apoptosis associate gene include interferon regulatory convert enzyme caspase undergo formal apoptosis attempt understand mucosal response systematic manner discover RSV infect epithelial cell express recently describe antiapoptosis gene study find expression response active virus gene expression observe response irradiate replication incompetent virus expression observe response adenoviral infection suggest expression gene universal response cellular perturbation viral infection functionally demonstrate RSV infection protect epithelial cell TNF induce apoptosis effect temporally associate expression generally consider highly efficient self defense mechanism employ host target cell permit infected host dispose viral protein nucleic acid single cell basis induce inflammatory response surprising virus evolve strategy circumvent response interest Krilov et al demonstrate monocyte cord blood mononuclear cell similarly protect apoptosis infect virus induce protection apoptosis appear advantageous virus interpretation consider respiratory epithelial cell recognize major source leukocyte chemoattractant leukocyte recruitment lung associate enhanced viral clearance prolonged survival pneumoviru infection ability maintain chemoattractant production viable cell ultimately benefit host organism available online tissue culture RSV infection characterize formation giant cell syncytia mechanism formation fuse masse cell depend expression RSV specific fusion protein surface infected host cell virusmediated change cytoskeletal architecture important note RSV induce change cytoskeletal architecture restrict cell line grow vitro giant cell syncytia find pathologic lung specimen obtain human animal infect systematic study gene expression response pneumoviru infection find human respiratory epithelial cell respond RSV infection increase expression cytoskeletal protein kda cytoskeletal protein belong class acidic cytokeratin family lung expression normally restrict basal epithelial cell larynx trachea bronchi RSV infect cell find expression predominantly site syncytia formation provide description unique component pathognomonic structure molecular level similar report production expression dependent activity transcription factor NF κB future study determine role NF κb consensus site promoter mediate response date effort study pneumovirus induce alteration gene expression rely heavily vitro model virus infect cell cell line intrinsic value characterize gene identify artificial system definition limit clinical physiologic sig nificance finding ultimately test vivo extent study gene product clinical specimen possible approach limited cumbersome dictate sample availability clear appropriate animal model inflammatory bronchiolitis require characterize alteration gene expression discover available vitro model RSV study specific allergic reaction viral antigen natural pathogen mouse intranasal inoculation virus high titer result good minimal primary infection correspondingly minimal inflammatory response order study gene expression response primary pneumovirus infection vivo develop novel mouse model inflammatory bronchiolitis natural rodent pneumoviru pathogen close phylogenetic relative RSV present initial finding pvm infection mouse recent publication summary finding present Table Fig begin describe cellular biochemical pathology observe response pvm infection mouse find infection establish plaque form unit pfu pvm inoculum infection result significant morbidity mortality viral recovery order pfu lung tissue note inflammatory bronchiolitis immediate response infection bronchoalveolar lavage fluid contain virtually neutrophil eosinophil obtain early day inoculation furthermore find infection accompany production proinflammatory chemokine previously show Cook et al important component inflammatory response orthomyxovirus influenza virus describe role play pathogenesis pvm induce bronchiolitis cally explore response gene delete mouse infection PVM find inflammatory response mouse deficient expression minimal virus induce inflammation mouse lack major receptor granulocyte chemokine receptor inflammatory response consider unnecessary detrimental demonstrate absence granulocytic inflammation associate enhanced recovery infectious virion accelerated mortality result suggest mediate acute inflammatory response protect mouse delay lethal sequelae viral infection recent report subject present direct comparison response mouse challenge PVM RSV RSV natural pathogen mouse extensively mouse model human infection limit semipermissive infection establish intranasal inocula tion virus high titer regard find RSV infection result measurable degree morbidity viral recovery significantly low find inoculum result suggest significant viral replication mouse lung tissue demonstrate inflammatory response RSV challenge minimal leukocyte recruit lung Fig result suggest infection mouse PVM provide superior model study acute inflammatory bronchiolitis response pneumovirus infection vivo advantage model include follow clinical parameter mortalitythat measure clearly specifically clear evidence viral replication lung tissue incremental recovery peak excess pfu response pfu inoculum dramatic granulocytic response modulate proinflammatory chemokine receptor traditionally analysis gene expression measurement steady state level individual mrnas conduct gene time northern blotting dot blot quantitative reverse transcription PCR differential display serial analysis gene expression total gene analysis offer great promise multiplex technology provide simultaneous analysis multiple mrnas isolate condition interest PCR amplification technique dna hybridization array theoretically efficient gene expression analysis technique skeptic describe genome base approach expensive nonhypothesis drive fishing expedition view broad base screening technique enable identify pattern gene expression subject careful characterization analysis differential display semiquantitative reverse transcription PCR base technique compare mrnas condition interest increase decrease expression specific amplicon evident obvious advantage approach total RNA isolate virus infect versus uninfected cell mouse lung infection differential display perform degenerate anchor primer random upstream oligomer describe result PCR product separate electrophoresis gel dry expose film example result compare cdna amplicon RNA extract RSV infect epithelial cell daily day show Fig differentially express band cut gel elute reamplifie primer generate original signal northern blot generate RNA extract pneumoviru infect cell tissue time probe differentially express amplicon serve confirm differential expression identify sequence DNA sequence newly identify differentially express amplicon compare sequence present GenBank database viral sequence expect upregulate time identify immediately entire genome PVM RSV present GenBank case amplicon represent newly discover gene potential openreading frame compare sequence present swiss protein database motif share homology know protein represent important clue identity differentially express gene help differential display identify characterize gene upregulate RSV infect respiratory epithelial cell specific example gene find induce RSV infection later characterize play independent role pathophysiology RSV infection antiapoptosis gene gene encode cytoskeletal protein dna virus know encode virus specific antiapoptosis gene RSV RNA virus small approximately kb viral genome show alter host cell expression apoptosis inhibitor demonstrate mRNA present sevenfold high concentration RSV infect respiratory epithelial cell compare uninfected cell conclude cellular response protect TNF induce program cell death viral infection effort determine RSV protein participate tran activation gene directly indirectly ongoing second example gene specifically upregulate RSV infect respiratory epithelium identify differential display encode characterize molecular event important induction demonstrate link NF κb signal pathway discuss importance transcription factor regulation proinflammatory cytokine gene expression involvement surprised discover role virus induce gene regulation interesting observation experiment situ localization protein area cytopathic syncytia formation suggest role cytoskeletal protein formation note observe dramatic decrease RSV replication syncytia formation block expression suggest block syncytia formation impair direct cell cell spread productive replication virus company market system component cytokine gene macroarray system recently develop Systems Sigma Genosys Minneapolis MN USA Clontech Atlas Palo Alto USA represent new opportunity available focus gene product know involve inflammation array consist different cdnas print PCR product charge nylon membrane example experience Sigma Genosys array show Fig example total RNA extract RSV infect cell uninfected control day infection microgram total RNA cdna synthesis reaction proprietary mixture primer pair trace amount radiolabele result radiolabele product hybridize macroarray overnight wash expose film arrow highlight obviously upregulated sequence experiment identify gene encode human physiologic importance upregulation human RSV infection rodent PVM bronchiolitis describe microarray differentiate macroarray way difference microarray matrix glass plastic slide probe label fluorescent dye radioisotope significantly microarray generally include large number high density imbedded sequence macroarray highly appealing massive amount datum generate microarray technology pose new challenge respect datum normalization management development mathematical model assist datum interpretation pneumoviruse RSV PVM enter respiratory epithelial cell receptor mediate event hostcell attachment internalization target cell begin alter gene expression event involve transcriptional upregulation cytokine chemokine gene RSV replication progress additional change cellular gene expression observe include induction potent antiapoptosis gene increase expression quiescent gene encode know physiologic importance gene gene product describe learn available technology evolve continue capitalize use Display amplicon generate RNA extract RSV infect cell daily interval follow infection day single anchor primer downstream primer random differentially express sequence highlight arrow black arrow show upregulated amplicon white arrow highlight downregulate amplicon differentially express signal approach large scale screening tool identify gene play important role pathophysiology pneumoviru infection elegant simple tool provide mean thorough systematic exploration gene expression cytokine macroarray probe radiolabelle cDNA generate total RNA extract epithelial cell RSV infection upper panel exposure condition medium low panel Signal intensity gene condition compare arrow highlight signal human present fold high concentration infected epithelial cell compare uninfected control','2001-05-11 00:00:00','http://dx.doi.org/10.1186/rr61',2051,741),('61d9a0fe39f4e845c44a06787de6f5f033b998a3.json','An object simulation model for modeling hypothetical disease epidemics – EpiFlex','BACKGROUND: EpiFlex is a flexible, easy to use computer model for a single computer, intended to be operated by one user who need not be an expert. Its purpose is to study in-silico the epidemic behavior of a wide variety of diseases, both known and theoretical, by simulating their spread at the level of individuals contracting and infecting others. To understand the system fully, this paper must be read together in conjunction with study of the software and its results. EpiFlex is evaluated using results from modeling influenza A epidemics and comparing them with a variety of field data sources and other types of modeling. EpiFlex is an object-oriented Monte Carlo system, allocating entities to correspond to individuals, disease vectors, diseases, and the locations that hosts may inhabit. EpiFlex defines eight different contact types available for a disease. Contacts occur inside locations within the model. Populations are composed of demographic groups, each of which has a cycle of movement between locations. Within locations, superspreading is defined by skewing of contact distributions. RESULTS: EpiFlex indicates three phenomena of interest for public health: (1) R(0 )is variable, and the smaller the population, the larger the infected fraction within that population will be; (2) significant compression/synchronization between cities by a factor of roughly 2 occurs between the early incubation phase of a multi-city epidemic and the major manifestation phase; (3) if better true morbidity data were available, more asymptomatic hosts would be seen to spread disease than we currently believe is the case for influenza. These results suggest that field research to study such phenomena, while expensive, should be worthwhile. CONCLUSION: Since EpiFlex shows all stages of disease progression, detailed insight into the progress of epidemics is possible. EpiFlex shows the characteristic multimodality and apparently random variation characteristic of real world data, but does so as an emergent property of a carefully constructed model of disease dynamics and is not simply a stochastic system. EpiFlex can provide a better understanding of infectious diseases and strategies for response.',' commonly measure public health estimate historical datum derive SIS SIR type model descendent forward projection basic reproductive ratio individual infected person go infect public health indicator epidemic probability epidemic generally consider possible large value likely epidemic occur composite value describe behavior infectious agent decompose classically example probability infection occur contact duration infectiousness number contact classical decomposition good tool account age segregation response exist immunity population network topology infectious contact factor observation significant motivation develop EpiFlex EpiFlex model design create system incorporate realism possible epidemic model enable emerge disease event simulate limitation describe separate section model effective stand case limitation EpiFlex share modeling system variety method mathematical modeling disease common SIR susceptible infected recover Kermack McKendrick SIS susceptible infected susceptible SEIR susceptible expose infected recover SIRP susceptible infected recover partially immune develop Hyman et al develop Hyman LaForce SIRP model starting point development object model Epi Flex SIRP SIR model extend include partial immunity denote progressive decline partial immunity allow influenza model accurately need experimentation realistic discrete modeling lattice type discrete modeling understand skew favor propagation discuss Rhodes Anderson Haraguchi Sasaki Eames Keeling edmund et al explore use network model interaction infectable entity Ferguson et al call balance realism epidemiology model EpiFlex complete Lloyd Smith et al show importance superspreade disease transmission SARS epidemic EpiFlex design issue account know weakness SIS descend model discuss Hyman LaForce suggest model deal demographic subgroup useful describe start Theoretical Biology Medical Modelling conceive model create matrix SIRP flow demographic group city model contact group possibility build entirely discrete model object orient approach essentially set granularity Hyman LaForce concept level individual Monte Carlo method attractive object method design good fit object orient programming invent discrete simulation Object orient OO design define primitive element black box subunit define way interact oo language concept originally conceive Simula language purpose verifiable simulation enforcement explicit connection object fundamental OO design procedural language FORTRAN COBOL datum area freely access program OO language wrap datum method access datum black box object set specify behavior possibility invisible unnoticed interaction simulation potentially validate logical proof addition testing entire course introduce OO language concept space interested reader suggest start implementation Smalltalk excellent free version downloadable Smalltalk enthusiastic friendly user community design EpiFlex describe completely appendix design proceed establish definition disease organism cornerstone define practical structure object simulate movement disease population disease object assign set definition draw literature allow wide spectrum disease produce organism specify aim minimize number configuration parameter require understanding mathematical model host infect second primary object host live work area host member demographic group determine type contact spread infectious disease host area live location interact EpiFlex area contain configure number location location container temporary group host people travel metro area model support linkage area people randomly draw configurable set demographic group remainder section present disease model adopt overview component overview program flow description core method follow discussion result EpiFlex software system model stage infection cycle Incubation Prodromal Manifestation chronic stage add fatality phase name extended SIRP fig show diagram model model Fig allow track different phase disease process separately define variable infectiousness symptom fatality recovery transition chronic disease stage appropriate allow model progress disease individual realistically disease identifiable occurrence particular stage stage set length zero bypass entirely contact type design EpiFlex draw literature attempt model spread infection accurately contact type blood contact needle stick blood mucosal contact sexual intercourse skin contact close airborne casual airborne surface hand mucosa food contact probability infection contact type input user estimate literature base hypothetical organism characteristic duration disease stage choose uniformly random user specify interval low high random number denote seed determination infected disease stage period denote Incubation Prodromal Manifestation Chronic low high take medical literature describe range day stage illness calculation simply high low low day particular stage extend future include ability define graph determine flatness distribution normative peak significant difference modeling disease rabie unusual circumstance long follow equation describe immunity decay choose current level partial immunity level partial immunity specify exist random value decide infection occur partial immunity phase show chart decision use output immunity level algorithm number random value EpiFlex use dynamic network model interaction host particular location base skew provide demographic segment movement cycle network contact generate versionextende SIRP disease model Epiflex Figure Extended SIRP disease model Epiflex susceptible infected recover partially immune fatality Extended SIRP break infected stage Incubation Prodroma Manifestation Chronic add fatality terminate EpiFlex visible externally observe effect limitation EpiFlex modeling algorithm carefully design test small scale observe element location describe place activity occur demographic group draw automatically location certain number cell specify identically behave location concurrently act location repetition count area location define user set average number host inhabit cell maximum cell exchange fraction specifiable model host move cell cell algorithm allocate host cell semi random randomly put host cell location cell hit average random draw cell location maximum overload cell interaction cell host exchange cell order infective appendix location component open model look hospital define household model time cell configuration EpiFlex implement Monte Carlo algorithm host location assign certain number interaction accord cauchy distribution parameter set location distribution describe curve axis specify fraction maximum interaction location axis specify fractional ordinal list host location distribution nearly flat severely skew actor provide nearly contact desire user EpiFlex note structure network form depend location define demographic group define population demographic group move location location maximum number interaction specify person base input initially gaussian equation discard favor Cauchy function well fit need skew function compute fast algorithm iterate infectious host select host expose infected party location Monte Carlo function result dynamically allocate network interaction location exposure cycle make use Monte Carlo input location list contact type place particular location maximum frequency interaction interaction frequency determine time contact spread disease contact specification define fractional efficacy infection specific route model effect different type contact discuss literature Song et al epiflex attempt generalized version host infection source target host draw random location queue contact connection establish target long contact allocation target contact connection target keep track location prevent allocation contact target randomly establish connection value set end maximum number connection support maximum end link reach algorithm search different connection location algorithm describe detail user specify maximum number connection location output cauchy distribution function determine connection individual allow variation degree skewness superspreade population model show critical importance Lloyd Smith et al position queue number host queue location denote proportional fraction queue position constant choose location express skew distribution cauchy distribution function number contact particular host max maximum number contact give host location host location model tend maintain rough order ordinal position consequently high location high connection host location tend high connection host reflect real world situation perfectly correspond well persistently maintain high connection individual location location host behavior change place place cauchy distribution function fairly fast execution function approximate radical variation see epidemiology study extreme example active super spreader individual infect large number zero typical type scale free network interaction explore Chowell Chavez Cauchy function allow network generate dynamically type location flexible manner correspond super spreader dynamic addition specification skew location network contact define location present movement cycle define demographic group model process time increase population slowing expect characteristic object modeling system price pay discrete detail epiflex model primary source increase processing time sum series possible infectious event model iteration scale series sum log base contagiousness disease number potential host location infected host minimize process infectious host contact increase stem characteristic network node connection node iteration location contain infectable host number infected host create element series infected host list interact randomly host include infected one location consider network node node host temporary connection host node value determine randomized input return number contact infected host location consequently series consist temporary connection contact modeling cycle interest completeness limitation Epi flex model describe plan address element implementation future version infectious disease occur time competitive inhibition synergistic effect see host reproduce Hosts reproduce model removal addition rate define population basis Census datum meet specification removal addition model host remove randomly choose location similarly add randomly choose location demographic group randomly assign long term modeling modeling alternative short live host reproduction cycle desirable EpiFlex practical way model period year explicit definition age distribution host population significant degree age take account demographic segmentation population demographic define fraction multiple baseline susceptibility host age demographic age provision define previous exposure profile host real population previous exposure significant effect spread disease dramatic effect mortality infection occur proper implementation previous exposure profile intertwine age definition implementation mutation rate disease mutation rate vary considerably type particularly virus Decay immunity model immunity decay act fair surrogate antigenic change pass event define surface contact efficiency EpiFlex eliminate pass infection event model example infected shake hand non infected shake hand non infected wash hand infect model definition account Surface hand mucosa contact person surface model time record contact network dynamically create log file time log potential infectious contact datum require look log file write extract program make network visible item future currently EpiFlex way account seasonal damping similarity result setting rate immunity decline addition seasonal damp function expect cause epiflex result synchronize yearly cycle seasonal damping result loss interesting epidemic behavior overriding function virtually guarantee drown behavior public health response epidemic optimally model public health response definition component present EpiFlex test component thorough review literature indicate method optimal current public health response center contact tracing ring vaccination quarantine mass vaccination backup available closure school daycare travel restriction method model EpiFlex response component importance recently underscore Lloyd Smith et al consequence result current system define theboard cut probability infection consider light clear object system model current technique properly disease range time stage draw literature probability specific disease stage time period infected host choose range equal reasonably adequate disease time measure day rabie unequal distribution long tail make difference modeling discussion present part brief set example native EpiFlex display develop well feel system comparison epiflex result real world datum set example observation EpiFlex purpose example serve guide want experiment analyze result different view epidemic datum simulate influenza different population show figure figure graph second epidemic population graph kind commonly see deviation smooth curve occur real world datum EpiFlex model attribute synchronization immunity combine formation small world network demographic group location location figure alternate view simple influenza epidemic occur naïve population figure EpiFlex NREVSS surveillance compare EpiFlex surveillance datum NREVSS surveillance datum qualitatively similar graph form EpiFlex influenza show figure width primary curve season EpiFlex month NREVSS datum approximately month explain NREVSS datum collect nationally surveillance center EpiFlex datum show single area epiflex run execute multiple city connect transport million population city model combine graph city show self similarity graph individual area wide match NREVSS data graph formation NREVSS datum consist diagnostic sample send physician comparison absolute number term quantity applicable percentage population comparison Table EpiFlex indicate roughly population infect herd immunity stop epidemic depend population size total morbidity obtainable EpiFlex add maximum immune level death death contribute small influenza morbidity practical purpose immune level proxy morbidity true morbidity relatively prone inaccuracy well measure immune fraction influenza available California state average infect range depend county EpiFlex high end state California estimate morbidity range Dowdle give serological influenza datum categorize age influenza seroprevalence range roughly influenza Hong range epiflex figure fall pair range epiflex immune fraction properly comparable notable figure relationship rough sine wave form classically derive SIRP initial startup period influenza repeat wave develop similar overall shape variability real world datum Milwaukee roughly similar scale graph refer population differ size order magnitude Milwaukee time size model run show similar number peak owe need compare graph natively figure optimum essential feature total morbidity rate link population size small population range million high total morbidity rate give identical organism figure intuitively expect population size affect morbidity give network contact connect individual population chance epidemic spread window prior development immunity part population increase population size decrease simulation import comparison figure simulation import comparison vertical scale demarcation horizontal scale year demarcation Upper white line total population standard removal rate effect striking small population order examine literature datum real world population sparse indication historical account small population new world link population size morbidity observable real world population recent account Heyerdahl Pacific mid th century graph figure immune fraction completion proxy total morbidity log scale population long epidemic take progress enclose population great number potentially infectious contact hit dead end host immune small population function window host immunity infection spread large fraction effect public health implication clearly structure network highly significant determine likelihood infected host contact naïve host essentially EpiFlex result indi cate period prior development immune subpopulation disease functionally high variable course figure graph EpiFlex suggest asymptomatic infect spreader influenza population surveillance datum estimate suggest discussion comparison seroprevalence minor experimental result repeat illness influenza continuously active initiate disease vector try infect people day develop high peak initial event vector try infect people day randomly distribute population make intuitive sense low probability subpopulation susceptible host large attempt infect similarly system city interconnect transport linkage later peak tend small variable early one thing degree lowgrade infection link system provide high total level infection event system formally define initiate disease vector second time pass mix immune versus susceptible unsynchronized population host escape infection epidemic infect immunity decline infected expect development complex non repeating waveform similarity type waveform EpiFlex show long simulation large population illustrate figure variety result obtain index case provide seed single city susceptible population index case repeat expect owe random interaction break chain contact percentage case effect expect increase high skew super spreader significance modeled epidemic particularly light recent work case proportion expect vary rough accordance infection die owe random chance Monte Carlo model EpiFlex multiple trial clearly reveal potential range variation epidemic give apparently identical condition epiflex show wave propagation epidemic transport network similar real world epidemic study Viboud et al Viboud et al state mean duration week spread United States range week Epi Flex result show simplified city configuration major airline hub city total million population city show propagation wave week short real world datum factor account difference current EpiFlex vanilla configuration transport network flat term number person move city city second city contain population practical limitation propagation histogram Figure manifesting case data point note flatness transport population purely matter configuration specific model Epi Flex system allow separate specification parameter city kind transport level city desire figure show histogram city manifesting case occur figure show histogram city occurrence incubate case influenza appear note Figure Upper graph show graph point population versus total morbidity estimate immunity figure Upper graph show graph point population versus total morbidity estimate immunity low graph sample graph low left low right green line low graph show immunity vertical demarcation axis month low graph EpiFlex useful silico experiment epidemic behavior easy configure run ordinary computer special configuration datum import tool work effective show factor invisible difficult access normal condition visibility incubation prodromal stage true morbidity estimate immunity EpiFlex capability discuss owe time space constraint system effective duplicate kind multimodality apparent stochastic variation see real population epiflex result provoke think nature epidemic infectious disease spread interesting way provide experimental test environment abstracted subject mathematical model item bold entity object Epi Flex system noun class program write item italic action verb represent interaction object host infectious stage member demographic group locationslocation contain host example hospital home contain location city typically contain special location host location area Disease Vectors introduce disease host population location area limited way regular cycle epidemic response modify spread infection represent action take respond epidemic damp spread SIRP simple flow chart elaborate multi city system graphic reproduce figure conclude output model SIRP paper specialty publication work important conception EpiFlex see SIRP diagram figure host begin susceptible upper left infected recover remove removal show recovery immunity level start near complete SIR immunity decline case influenza period roughly year steep drop beginning year term partial actually case immune system response influenza antigen degrade rapidly virus mutate rapidly host immunity decline net result similar decline immunity model real world antigenic mutation influ enza virus perfect model influenza virus change multiple virus model point immunity decline effectively zero model correspond large antigenic change person return fully population person reinfected diagonal arrow probability reinfection low period decline immunity Multi city SIRP flow figure center denote movement city city example move right figure look actual versus model graph sinusoid curve general correlation period result conventional model file generate EpiFlex generate file configuration year city simulation link transport corridor people city Comparison distribution day manifesting case appear city upper graph versus distri bution day incubate case appear city low graph figure Comparison distribution day manifesting case appear city upper graph versus distribution day incubate case appear city low graph Model file end file xml format file edit Epiflex panel available menu epdm file edit directly WordPad good xml editor sophisticated user read selfexplanatory user familiarize software run record file end file text format comma semicolon delimit format intend easy import possible rpx file import Excel read SAS SPSS etcetera file descriptor field common problem import space delimiter space area name define new column log file end happen model log file fairly large potentially parse secondary piece software match RPX file exercise user time viewer log file View menu snapshot file end run complete SNAP file write read file duplicate model run model exist memory completion run lab notebook aid recognize early remembering exactly contain model nigh impossible lot work document note model complete run rpx file SNAP file case experimenter record state model SNAP file copy outside Epiflex manually change read extension rename result model run like software engineering detail internal class definition structure component present UI well job educating compact disease component look screen shot current UI Figure functional parametric element disease specification particular example select influenza parameter set reproduce know characteristic disease disease stage figure correspond model diagram show Figure definition superior way available SIRP modeling show previous section Figure addition multiple stage define level infectiousness type contact infected host low right partial immunity stage specify able define wide variety immune response asymptotic decline fit available datum theory immune system left SIRP Diagram Multi city SIRP actual versus model figure left SIRP Diagram Multi city SIRP actual versus good disease use slider allow eyeball asymptotic curve approximate reasonable case immunity set begin drop approximately half day period pass make sense datum area fairly sparse want perform multiple run slightly different immune drop characteristic mind width graph number day enter panel host implicitly number population area EpiFlex system object define disease host infect disease receive pointer archetypal disease object store disease stage internally disease progress functionality host consist implementation disease stage transition figure screen shot UI set demographic definition select Census datum nation result model ride default fraction later population specific area fraction represent derive recent census datum demographic group set specify variation overall susceptibility disease modify specific type contact desire example fractional modifier create demographic group necessity stick demographic group define add new one desire location describe place activity occur demographic group draw automatically location certain number cell specify identically behave location concurrently act location repetition count area location define technical point home cell model set create number home base population size distribution household size figure Disease definition panel consequently present define number home cell appropriate hold population area distribution function available specify household size household definable population size practical provide can profile household size base census datum type contact available host location define average number contact host cycle model Figure right contact definition skew function enable specify degree contact vary host subpopulation high contact individual low contact individual example show set difference high describe way service worker interact strongly customer run simulation determination host infected cycle position host queue contact frequency decide contact individual particular cycle think adjustable skew graph histogram contact count demographic group cycle movement location area define illustrate figure model example day divide cycle sequence movement define day week cycle complete restart host model move location area list location contain show Figure area specify particular population level population fraction demographic group make area allocate runtime demographic population removal addition overridden area level area linkage area special location eligible link example Group demographic population definition panel Figure Group demographic population definition Anchorage airport go send population Chicago Illinois cycle cycle population Anchorage airport location go ship Vector define EpiFlex infection event attempt specific location vector repeat cycle limited input vector represent arthropod snail represent people arrive aircraft Hong Kong vector show Figure startup vector influenza modeling run initiate disease vector operate cycle stop attempt produce case location Chicago initiation disease vector delay parameter run continuously run force infection specific number people infect particular rate disease occur medical system respond way illness influenza result little intervention SARS smallpox result great deal intervention rapidly consequently reasonable modeling system allow response epidemic occurrence model example show Figure response specification emergent illness response alert trigger case appearance fatality result period heighten awareness disease symptom set example day probability notice trigger event set unity example overly optimistic specification say fatality alert trigger alert trigger fatality symptom instrument detection fraction disease stage applicable alert trigger detection method illness switch primarily symptom specify fraction occurrence diagnose stage detect event mitigate implementation specify degree infectiousness remain intervention process method disease stop disease spread completely true disease specifiable figure Location definition panel note component author comfortable example paper response consequence primary reason way response model simplify conform response happen real world limitation EpiFlex modeling introduction EpiFlex compose major class core internal model functionality rest infrastructure UI functional walkthrough system operate present model start run verify model datum look reference thing delete definition impossible write error log file information audit run error find inform severity give option cancel error non fatal modify warning change result run EpiFlex system iterate area model allocate host put initial location movement definition demographic group group step location movement list determine area attach location group need EpiFlex increment pointer come location leave host previous real location good find ease practical use allow demographic location exist area step stage pass infected host update stage illness host system check disease response condition meet step pass EpiFlex iterate vector apply disease location area vector accord rule define step pass iterate location area figure contact infected uninfected host contact Group movement cycle definition panel Figure Group movement cycle definition host host infect disease probabilistic determination decide illness communicate step area look location group draw factor randomly pull host group specify group draw location feature simulate thing generally airport type specification step area look area link area link push population location area location specify step iterate location area iterate random exchange factor host assign cell location cell location step addition removal population group apply model allow user model normal birth death rate plus immigration emigration outer boundary model step area iterate location area location iterate host attach location host location pointer group indicate movement cycle increment pointer location different find location area location cell cell semi randomized fashion accord parameter define note currently host remember cell iteration return cell time come movement cycle limitation present require greatly increase space allocation change step audit verify host occur location step iterate location iterate total result result write log file display main window graph form depend parameter specify figure Area definition panel area linkage panel Figure Area linkage panel initiate disease vector definition panel Figure initiate disease vector definition panel','2006-08-23 00:00:00','http://dx.doi.org/10.1186/1742-4682-3-32',3872,1097),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json','Herpes simplex virus type 1 and normal protein permeability in the lungs of critically ill patients: a case for low pathogenicity?','INTRODUCTION: The pathogenicity of late respiratory infections with herpes simplex virus type 1 (HSV-1) in the critically ill is unclear. METHODS: In four critically ill patients with persistent pulmonary infiltrates of unknown origin and isolation of HSV-1 from tracheal aspirate or bronchoalveolar lavage fluid, at 7 (1–11) days after start of mechanical ventilatory support, a pulmonary leak index (PLI) for (67)Gallium ((67)Ga)-transferrin (upper limit of normal 14.1 × 10(-3)/min) was measured. RESULTS: The PLI ranged between 7.5 and 14.0 × 10(-3)/min in the study patients. Two patients received a course of acyclovir and all survived. CONCLUSIONS: The normal capillary permeability observed in the lungs argues against pathogenicity of HSV-1 in the critically ill, and favors that isolation of the virus reflects reactivation in the course of serious illness and immunodepresssion, rather than primary or superimposed infection in the lungs.',' critically ill patient herpe simplex virus isolate upper low respiratory tract immunodepressed patient susceptible transmission acquisition viral disease alternatively viral reactivation occur contribute relatively little morbidity mortality reactivation human common critically ill patient worsen outcome immunocompetent patient isolation associate viral pneumonia reactivation primary infection responsible associate acute respiratory distress syndrome ARDS ventilator associate pneumonia critically ill primary superimpose infection report virus elicit infectious host response demonstrate rise serum antibody bronchoscopic airway disease typical finding compute tomography lung presence giant cell nuclear inclusion body cytology biopsy low respiratory tract Tuxen coworker observe prophylactic antiviral therapy ARDS prevent respiratory emergence impact duration mechanical ventilation patient outcome pathogenicity virus remain unknown rare association critically ill isolation mortality represent reactivation virus immunodepressed patient multiple organ failure poor outcome symptomatic primary infection superinfection contribute death assess pulmonary capillary protein permeability noninvasively bedside yield pulmonary leak index PLI help determine extent tissue injury previously describe radionuclide technique involve label transferrin Gatransferrin label red blood cell tc RBCs bacterial pneumonia instance PLI elevate increase normal directly relate severity pneumonia express lung injury score LIS patient acute lung injury ALI ards course bacterial pneumonia PLI uniformly greatly elevate normal LIS great PLI elevate patient mild injury LIS technique direct measure permeability indirect measure capillary injury lung PLI elevate interstitial lung disease order help differentiate symptomatic asymptomatic viral shedding spread inform decision institute antiviral therapy help determine pathogenicity virus measure PLI consecutive critically ill patient persistent pulmonary infiltrate unknown origin ventilatory support isolate study small series consecutive patient respiratory secretion send viral culture persistent pulmonary infiltrate unknown origin find positive table tracheal aspirate bronchoalveolar lavage fluid transport directly microbiology laboratory place viral transport medium Copan Diagnostics Corona USA isolation specimen inoculate standard procedure triplicate flat tube human embryonal lung fibroblast incubate culture study time weekly day identify presence cytopathic effect cytopathic effect indicate presence apparent day cell fix methanol acetone type immunofluorescence label specific antibodie Syva Mikrotac typing kit Palo Alto USA patient study result available day sample inoculate culture medium day speciman collection viral culture demographic chest radiographic respiratory datum record clinical feature patient mechanical ventilation intubation total respiratory compliance calculate ventilator setting follow ml cmh tidal expiratory pressure radiographic score range depend number quadrant radiographic opacity ratio arterial oxygen tension fractional inspire oxygen level positive end expiratory pressure compliance LIS calculate LIS range value denote ALI ARDS patient visible oropharyngeal vesicle characterize persistent pulmonary infiltrate PLI measure modification method describe previously routine procedure informed consent waive autologous rbc label Tc MBq physical half life hour Mallincrodt Diagnostica Petten Netherlands modify vitro method minute injection label rbc transferrin label vivo follow intravenous injection Ga citrate MBq physical half life hour Mallincrodt Diagnostica patient supine position scintillation detection probe position right leave lung apex probe system manufacture Eurorad Strasbourg France consist small cesium iodide scintillator mm mm tungsten mm aluminium housing cover mm diameter mm height end probe aluminium flange attach mm thickness mm diameter facilitate easy fixation patient chest tape probe weigh approximately probe signal lead dual amplifier output feed multichannel analyzer system connect personal computer probe separate channel electronic crossover start time intravenous injection Ga radioactivity measure minute hour measurement interval entire spectrum photon energy store disk processing Tc Ga plot time PLI calculate linear regression analysis slope increase radioactivity ratio divide intercept order correct physical factor radioactivity detection take pulmonary blood volume presumably surface area account radioactivity ratio represent ratio extravascular intravascular Ga radioactivity PLI represent transport rate Ga transferrin intravascular extravascular space lung measure pulmonary capillary permeability transferrin mean PLI lung take upper limit normal PLI appropriate number summarize median datum present Table patient stay time hospital intensive care unit isolate admit primarily respiratory insufficiency course pneumonia patient admit coronary care unit day intensive care unit admission cardiogenic pulmonary oedema patient dependent mechanical ventilatory support time sample receive adequate antibiotic therapy pneumonia ALI time sampling unknown origin table show patient radiographic abnormality increase PLI central venous pressure elevate suggest persistent pulmonary infiltrate cause overhydration patient high resolution compute tomography scan lung contrast obtain finding nonspecific alveolar consolidation pleural fluid presence interstitial abnormality ground glass appearance patient patient bronchoscopy perform mucosal lesion normal distribution lymphocyte subtype lavage fluid transbronchial biopsy reveal interstitial inflammation macrophage deposit immunohistochemical staining negative multinucleated cell cell inclusion observe bronchoalveolar lavage fluid patient tracheal aspirate patient patient concomitant isolation bacteria culture regard bacterial colonization antibody testing patient find positive anti IgG patient indicative prior infection antiviral agent aciclovir mg kg time daily start culture positive patient discretion treat physician Aciclovir withhold patient presume pulmonary infiltrate cause basis normal PLI finding patient normal PLI course steroid initiate day PLI measure continue despite positivity report day later patient survive discharge intensive care unit Ga transferrin PLI sensitive specific measure pulmonary capillary permeability utilize noninvasive assessment severity broad range pulmonary condition PLI roughly parallel clinical severity LIS involve use relatively routine equipment diagnostic method gain broad application partly laborious nature advantage bedside measurement possible mechanically ventilate critically ill patient easily transport pulmonary inflammation cause increase PLI time normal value severe form lung injury include ARDS severe injury impending ards interstitial lung disease PLI elevate albeit less extent report group patient common prior infectious episode follow relatively prolong period respiratory insufficiency persistent nonspecific pulmonary infiltrate unknown origin treatment primary disease prompt viral culture normal PLI observe suggest involvement relatively harmless reactivation presence primary damaging infection critically ill patient sepsis late immunodepression lymphocytic apoptosis lymphocytopenia cell anergy promote viral reactivation apparently virus latent nerve ending mucous membrane upper respiratory tract patient herpesviruse frequently isolate vivo respiratory secretion patient ARDS detect surveillance culture respiratory tract patient follow burn trauma transplantation major surgery virus detect lung biopsy patient prolonged unresolve ARDS literature widely divergent precise role virus pulmonary disease critically ill contribution patient morbidity mortality believe tracheal aspirate representative low respiratory tract secretion absence herpe orolabiali oral epithelial cell smear Gram stain secretion concurrent colonization pathogen previously describe overlap duration stay patient transmission virus patient exclude suggest respiratory infection critically ill result relatively harmless endogenous reactivation normal PLI argue pulmonary parenchymal pathogenicity tracheobronchitis cause virus rule absence orolabial lesion bronchoscopy perform patient unremarkable patient persistent pulmonary infiltrate patient relate slow radiographic resolution prior bacterial aspiration pneumonia superimpose infection compute tomographic image lung largely nonspecific precise diagnostic criterion pneumonia remain unclear properly standardize instance respect cell number bronchoalveolar fluid tracheal aspirate quantitative culture viral RNA DNA polymerase chain reaction helpful PLI study quantitate viral load ratio replication shed pathogenicity virus low respiratory tract conclusion anecdotal datum present suggest isolation respiratory secretion critically ill patient persistent pulmonary infiltrate warrant evaluation tissue injury potentially cause virus judge pathogenicity radionuclide PLI measurement help inform decision antiviral therapy adverse effect patient normal PLI argue viral pathogenicity withholding aciclovir patient safe','2004-03-31 00:00:00','http://dx.doi.org/10.1186/cc2850',1116,560),('6af02873a430f5f75dc56a917f26feaea89b8e79.json','The Effectiveness of Contact Tracing in Emerging Epidemics','BACKGROUND: Contact tracing plays an important role in the control of emerging infectious diseases, but little is known yet about its effectiveness. Here we deduce from a generic mathematical model how effectiveness of tracing relates to various aspects of time, such as the course of individual infectivity, the (variability in) time between infection and symptom-based detection, and delays in the tracing process. In addition, the possibility of iteratively tracing of yet asymptomatic infecteds is considered. With these insights we explain why contact tracing was and will be effective for control of smallpox and SARS, only partially effective for foot-and-mouth disease, and likely not effective for influenza. METHODS AND FINDINGS: We investigate contact tracing in a model of an emerging epidemic that is flexible enough to use for most infections. We consider isolation of symptomatic infecteds as the basic scenario, and express effectiveness as the proportion of contacts that need to be traced for a reproduction ratio smaller than 1. We obtain general results for special cases, which are interpreted with respect to the likely success of tracing for influenza, smallpox, SARS, and foot-and-mouth disease epidemics. CONCLUSIONS: We conclude that (1) there is no general predictive formula for the proportion to be traced as there is for the proportion to be vaccinated; (2) variability in time to detection is favourable for effective tracing; (3) tracing effectiveness need not be sensitive to the duration of the latent period and tracing delays; (4) iterative tracing primarily improves effectiveness when single-step tracing is on the brink of being effective.',' control epidemic emerge infectious disease SARS pandemic influenza foot mouth disease face difficulty infective observe concentrate control measure observed case treatment isolation culling resource efficiently control effective hand direct control population mass vaccination prophylactic treatment preventive culling epidemic likely contain high cost contact tracing symptomatic infected work intermediate level treatment quarantine contactee contact mean possible transmission event contactee individual contact effective unidentified infected likely find contactee efficient resource direct individual risk tracing quarantine successfully apply smallpox control term Leicester method refer exactly strategy establishment specific smallpox hospital successful recent SARS epidemic british foot mouth epidemic wish investigate difference account tracing process distinguish tracing quarantine have separate effect explain follow symptom detection divide reproduction ratio average number secondary infected primary infect susceptible population detection isolation reduction factor isolation basic reproduction ratio effort isolate pre occur prior detection contact tracing lead early prevention transmission quarantine trace infected reduce uncontrolled transmission pre pre reproduction ratio effect decrease pre pre distinct effect isolation quarantine reduce trace effective small pre small threshold value determine comparison contact tracing different situation choose easy arbitrary value value valid foot mouth disease trace herd cull trace study publish focussing specific infection objective obtain general insight general Markov type SIR model model incorporate symptom development simulation model specific network contact structure different assumption regard possibility trace contact symptomatic infected single step tracing iteratively trace contact trace infected iterative tracing general insight obtain reduce pre relation time relate characteristic infection tracing process latent infectious period time symptom base detection delay tracing process paper consider detection isolation autonomous process likely govern detection clinical symptom allow concentrate contact tracing quarantine initiate autonomous process regard transmission prior control consider trace effective pre study epidemic initial phase use branching process description mean epidemic regard grow tree branch contact connect node contactee hierarchical relation infector having infect infectee contactee notify infect symptomatic trace secondary contact contact trace probability parameter function model give Table contact trace infector infectee forwards trace infectee infector backwards trace traceability determine time transmission infection tree traceable untraceable contact arise figure infection tree type contact tracing distinguish type single step contact tracing traceable contactee symptomatic case quarantine trace continue undiscovered infect quarantine way detect figure second type iterative tracing tracing traceable contactee continue cluster infected link traceable contact quarantine figure continuation possible test available determine infection status trace contactee underlying model infection dynamic base framework model measure time infection individual start latent period lat transmission pathogen infectious period last lat inf infected rise new infected unit time long detect interested effectiveness trace model possible transmission occur isolate transmission cease detection infected leave basic reproduction ratio prior detection define expect number secondary infection infect susceptible population detection pre adjust interpretation model divide asymptomatic symptomatic add detection delay symptomatic pre determine lat inf distribution detection time Gammadistribution mean time measure relative mean detection time shape parameter analysis scale accordingly achieve desire value pre model construction allow flexible way explore different assumption time detection infectious period overlap enable evaluate tracing effectiveness infection understanding contact tracing model analyze special case infectious period detection time distribution infectious period assume short transmission occur instantaneously inf lat long infinite duration inf model infinite duration assume infect stop spread infection detection infected detect large inf take detection time assume fix highly variable exponentially distribute control analyze intermediate case result show example real infection influenza SARS smallpox foot mouth disease parameter value list Table parameter value obtain literature assume time detection consist incubation period time symptom onset plus symptom detection delay assume distribute observe SARS epidemic average day analysis trace effectiveness express critical tracing probability define proportion contact need trace achieve pre contact trace epidemic certainly die transmission isolation quarantine prevent limit small number health care worker introduce infection general community reintroduction exclude low pre aim threshold pre value trace assume value large tracing need publish value specific case table study function pre lat tracing delay second function lat pre epidemic single step tracing apply asymptomatic infected spread infection detect isolate detect infected ask contactee proportion contactee report quarantine quarantine infected detect tracing continue figure determine critical tracing probability function latent period value pre result show figure detection time fix large latent period large detection time result situation infected detect transmit infection tracing need prevent transmission maximum lat panel locate approximate position real infection parameter value table Figure observe long infectious period good approximation infection regard pre detection transmission infected infectious time detection entirely hold influenza fit good short long infectious period detection period distribution smallpox FMD well describe panel SARS influenza fit well panel influenza special case panel fix incubation period short infectious period proportion contact trace pre low limit pre forward trace infected traceable infector quarantine end latent period effective reproduction ratio equal untraceable proportion pre pre result pre likely case smallpox panel latent period short see influenza possibly FMD quarantine occur late prevent infection contact need trace fourth panel variable detection time long infectious period effective contact tracing require proportion contact small pre trace latent period large like SARS explain follow detection time variable latent infectious period large infected detect infectious infected infectious detect infectious superspreader attain give pre average number secondary infection detection infectee superspreader detect early variable detection time superspreader quarantine backwards trace add effect forwards trace prevent infectee reach infectious period effectiveness sensitive latent period especially little variation detection time apparent sharp transition panel tracing effective infector detect lat infectious period lat lat high sensitivity problem assess tracing effectiveness specific infection conclusion largely depend correct available estimate latent incubation period incubation period determine time detection figure indicate problem previous section tracing instantaneous process detection isolation give case follow quarantine traceable contactee case delay effect delay duration contactee detect infected continue transmit infection extra time unit detect interval effect delay tracing step study determine critical tracing probability pre function latent period tracing delay figure show result contour plot special case singlestep tracing indicate approximate position real infection lat axis lie pre curve figure appear iso contour figure linear special case panel approximately linear fourth case long infectious period variable incubation time panel inf slope approximate line mean effect tracing delay comparable sensitivity latent period observe figure plot critical tracing probability function delay resemble plot figure mirror figure contour plot figure trace ineffective dark grey area small latent period large delay smallpox SARS control predict able cope delay problem FMD influenza forwards tracing maximally effective affect delay pre indicate light grey area smallpox detection time variable tracing effective proportion pre trace SARS panel backwards tracing need attain result possible tracing delay short infectious period show contour plot combination latent period delay sensitivity small change delay large contour lie close small change delay major impact time quarantine shift end latent period similar sensitivity lat observe figure effect dramatic little variation detection time smallpox FMD figure show effectiveness single step tracing SARS smallpox influenza FMD parameter value list figure effectiveness single step contact tracing trace delay effectiveness express minimum proportion contact need trace effective control critical tracing probability plot function latent period relative mean time detection lat special case short infectious period variable time detection Short infectious period fix detection time Long infectious period variable time detection Long infectious period fix detection time curve denote different value pre isolation reproduction ratio pre indicate dash line average lat infection panel close correspondence actual parameter value table Influenza appear panel long short infectious period correspond parameter set equally Figure effectiveness single step contact trace tracing delay pre detection reproduction ratio pre effectiveness express minimum proportion contact need trace effective control critical tracing probability contour plot function tracing delay latent period lat measure relative mean detection time special case short infectious period variable incubation period Short infectious period fix incubation period Long infectious period variable incubation period Long infectious period fix incubation period dark grey shadow indicate area tracing ineffective light grey shadow indicate area indicate dash line average lat infection panel close correspondence actual parameter value table Influenza appear panel long short infectious period correspond parameter set equally Figure effectiveness single step iterative contact tracing control influenza smallpox SARS foot mouth disease effectiveness express minimum proportion contact need trace effective control critical tracing probability plot function relative delay proportion incubation period absolute delay day Table Panel show relation relative tracing delay appear general case show figure good predictor assess tracing effectiveness influenza place long short infectious period panel hardly effective delay ineffective SARS control require tracing probability pre relatively insensitive delay Smallpox require pre insensitive delay point tracing quickly ineffective finally FMD trace effectively small sensitive delay small require tracing probability pre measure real time panel effectiveness tracing appear highly dependent actual generation time infection influenza control hardly possible FMD control difficult tracing likely successful smallpox test available determine infection status trace contactee continue trace iteratively infected contactee detect symptom infected find figure evaluate iterative tracing delay case single step tracing result figure similar figure support information expect universal effect iterative tracing lower low limit pre observe special case remain intact large difference single step iterative tracing observe single step tracing brink effective situation iterative tracing difference figure delay difference intuitive single step tracing effective latent period small iterative tracing effective contact traceable detection immediately follow quarantine infected case influenza improvement probably sufficient make trace effective influenza control hand latent period large single step tracing effective iterative tracing improve significant delay provide day reach require smallpox SARS study effectiveness contact tracing model start epidemic flexible use infection effectiveness contact tracing express minimum proportion contact need trace obtain reproduction ratio prior control threshold value choose know transmission infection general community infectious individual isolate quarantine instance isolate infected cause average new case general community isolation quarantine redefine pre conclusion model give pre critical tracing probability value depend infectious disease characteristic model latent period infectious period detection time distribution contrast early publication contact tracing exist general expression proportion vaccinate effective control mixed population smallpox relation pre hold reasonably SARS small influenza FMD large second conclusion variable detection time improve trace effectiveness possibly result pre mean course aim late detection infected increase pre apparent variability argument use trace reason infected detect late essentially discover backwards trace additional effect forwards trace result pre early find small pre different mechanism presence share contact network conclusion sensitivity trace effectiveness latent period tracing delay large especially case single step tracing case small delay influenza less extent FMD reliability parameter estimate crucial establish tracing advantageous case large delay smallpox SARS tracing effective long quick fourth conclusion situation single step iterative tracing equally effective considerable difference expect single step tracing hardly effective sensitivity latent period tracing delay large influenza control benefit iterative tracing delay infection day gain figure have time achieve require real world choice single step iterative tracing base possible capacity problem reduce effectiveness iterative tracing effort direct secondary contactee prior primary contactee hand quarantine vaccination apply worthwhile trace contactee contactee diagnostic test determine cost effectiveness contact tracing critical tracing probability require easily achieve key aspect possibility distinguish relevant contact control sexually transmit disease relevant contact easily identify people willing co operate case respiratory pathogen like SARS influenza notion relevant contact diffuse good insight obtain sensitivity tracing positive predictive value proportion trace individual actually infect effort increase decrease positive predictive relatively easy way increase proportion trace contact respiratory pathogen like influenza quarantine household effective household prime location spread infection specific model take account contact structure household well suited study strategy model regard transmission infection tracing isolation allow focus characteristic contact trace control animal disease epidemic foot mouth disease avian influenza unreasonable assume transmission infected suspect farm cull stop infectious human infection likely isolation quarantine effective result complicated situation transmission reduce detection contact structure change limited number people health care worker have contact multiple isolated patient case SARS detailed model exactly aspect study Lloyd Smith et al conclude minimize hospital transmission crucial especially tracing isolation occur slowly study require complex problem present paper provide good understanding principle contact tracing infectious disease characteristic determine effectiveness effect single step tracing measure effective reproduction ratio average number new infection infected rate trace backwards time infection detection infectee equal infected rate trace forwards detection infector depend generation time distribution turn depend number traceable generation backwards transmission tree traceable ancestor infective need type accord number traceable ancestor large eigenvalue generation matrix entry ij expect number type infected traceable ancestor produce type infect type infected produce type infected untraceable traceable contact respectively entry equal matrix infinitely large conjecture observed numerical calculation positive element converge sense jr numerical evaluation closed matrix matrix nn special case calculate numerically MathematicaH case consider entry matrix determine numerical simulation analysis single step tracing delay calculate generation matrix model delay support Information iterative tracing isolation single infected result quarantine cluster infected mutually link traceable contact figure recognize cluster infective epidemic infected individual regard epidemic traceable cluster untraceable contact transmission tree result infection new cluster index case average number untraceable contact infection cause single cluster cluster reproduction ratio determine effectiveness iterative contact tracing denote expect cumulative infectiousness cluster time cluster quarantine thenas single step tracing effectiveness iterative tracing determine consider critical tracing probability achieve infective separate cluster pre definition cluster size large cluster infectiousness need large backwards trace reduce infectious period superspreader large detection time single step tracing special case case consider Müller et al inf lat calculate numerically case include real infection stochastic simulation need incorporate delay iterative tracing generally change concept make calculation complicated long single time cluster quarantine complication arise delay cause contactee chain emanating symptomatic infect symptomatic tracing process reach initiate new tracing process cluster second infection process fast tracing process cluster size grow infinitely large iterative tracing ineffective determine stochastic simulation cluster quarantine final infected support Information information support information paper Effectiveness Contact Tracing emerge epidemic find mb pdf','2006-12-20 00:00:00','http://dx.doi.org/10.1371/journal.pone.000001',2279,533),('6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json','Nitric oxide: a pro-inflammatory mediator in lung disease?','Inflammatory diseases of the respiratory tract are commonly associated with elevated production of nitric oxide (NO•) and increased indices of NO• -dependent oxidative stress. Although NO• is known to have anti-microbial, anti-inflammatory and anti-oxidant properties, various lines of evidence support the contribution of NO• to lung injury in several disease models. On the basis of biochemical evidence, it is often presumed that such NO• -dependent oxidations are due to the formation of the oxidant peroxynitrite, although alternative mechanisms involving the phagocyte-derived heme proteins myeloperoxidase and eosinophil peroxidase might be operative during conditions of inflammation. Because of the overwhelming literature on NO• generation and activities in the respiratory tract, it would be beyond the scope of this commentary to review this area comprehensively. Instead, it focuses on recent evidence and concepts of the presumed contribution of NO• to inflammatory diseases of the lung.',' discovery biological messenger molecule year ago gaseous molecule nitric oxide recognize involvement diverse biological process include vasodilation bronchodilation neurotransmission tumor surveillance antimicrobial defense regulation inflammatory immune process respiratory tract generate enzymically distinct isoform synthase present different extent numerous cell type include airway alveolar epithelial cell neuronal cell macrophage neutrophil mast cell endothelial smoothmuscle cell contrast NOS isoform express constitutively activate mediator induce stress induce cell activation activity primarily regulate transcriptionally commonly induce bacterial product pro inflammatory cytokine inflammatory disease respiratory tract asthma acute respiratory distress syndrome ARDS bronchiectasis commonly characterize increase expression respiratory epithelial inflammatory immune cell markedly elevated local production presumably additional host defense mechanism bacterial viral infection drawback excessive production accelerated metabolism family potentially harmful reactive nitrogen specie RNS include peroxynitrite ONOO nitrogen dioxide especially presence phagocyte generate oxidant formation RNS think prime reason case contribute etiology inflammatory lung disease despite extensive research pro inflammatory anti inflammatory action overall contribution inflammatory condition lung easily predict depend factor site time degree production relation local redox status acute chronic nature immune response addition current understanding pro inflammatory pro injurious mechanism relate RNS incomplete commentary focus primarily aspect explore role NOS infectious inflammatory disease general research approach take use pharmacological inhibitor NOS isoenzyme targeted deletion individual NOS enzyme mouse approach suffer shortcoming animal model respiratory tract disease generally faithfully reflect human disease use nos inhibitor determine contribution individual NOS isoenzyme hinder problem relate specificity pharmacokinetic concern unconditional gene disruption NOS isoform lead lifelong deficiency markedly different outcome pharmacological inhibition certain stage disease involvement individual NOS isoenzyme different depend disease stage severity despite inherent limitation study targeted deletion nos isoform lead insight indicate role etiology inflammatory lung disease instance mouse deficient susceptible lethality intranasal inoculation influenza virus suffer lung injury administration endotoxin display reduce allergic eosinophilia airway lung injury model asthma wild type counterpart contribution expect inflammatory condition recent study determine primarily involve development airway hyper reactivity similar asthma model linkage etiology asthma recently support asthmatic human association gene polymorphism disease physiological basis association remain unclear potential contribution derive lung injury endotoxemia sequestration neutrophil lung adhesion postcapillary postsinusoidal venule administration endotoxin find markedly increase deficient mouse deficiency alleviate endotoxininduce mortality harmful protective effect contend model make assessment potential role NOS human disease difficult context interesting note human animal cystic fibrosi subnormal level respiratory epithelium relate gene mutation cystic fibrosis transmembrane conductance regulator relative absence epithelial contribute factor excessively exuberant respiratory tract inflammatory response patient cystic fibrosis absence detectable respiratory infection overall apparently contrast finding associate NOS deficiency concern animal disease model interpretation conclusion regard human lung disease difficult pharmacological inhibitor NOS find reduce oxidative injury animal model lung injury ischemia reperfusion radiation paraquat toxicity endotoxemia example result consistent case NOS inhibition find worsen lung injury indicate anti inflammatory protective role despite inconsistency ample evidence study suggest contribute role respiratory disease condition continue stimulate research mechanistic aspect underlie pro inflammatory role modulation generation potential therapeutic target pro inflammatory injurious effect mediate number diverse mechanism commonly assume action largely generation reactive product generate oxidative metabolism collectively term RNS prime suspect commonly implicate adverse injurious property ONOO potent oxidative specie form diffusion limit reaction superoxide product activate phagocyte endothelial epithelial cell formation onooseem highly feasible condition elevated production vivo oxidative cytotoxic potential document direct detection ONOOunder inflammatory condition virtually impossible instability high reactivity formation ONOOin vivo demonstrate indirect method investigator rely analysis characteristic oxidation product biological molecule protein dna notably free protein associate nitrotyrosine product tyrosine oxidation form ONOO RNS commentary review report primary research example elevated level nitrotyrosine observe different inflammatory condition respiratory tract illustrate endogenous formation ONOOor relate RNS case know evidence functional consequence protein tyrosine nitration detection nitrotyrosine regard direct proof pro inflammatory role detection nitrotyrosine case interpret conclusive evidence formation ONOOin vivo example realize RNS form alternative mechanism contribute endogenous tyrosine nitration recently clear presence inflammatory immune cell specifically heme peroxidase myeloperoxidase MPO eosinophil peroxidase EPO catalyze oxidization metabolite reactive RNS contribute protein nitration notion support fact nitrotyrosine commonly detect tissue affect active inflammation phagocytic cell macrophage contain active peroxidase originate apoptotic neutrophil eosinophil detection nitrotyrosine vivo direct proof formation ONOO merely indicate formation RNS multiple oxidative pathway possibly include onoobut probably involve activity phagocyte peroxidase regard preliminary study EPO deficient mouse recently demonstrate critical importance EPO formation nitrotyrosine mouse model asthma future study animal deficient MPO EPO undoubtedly help clarify issue give considerable interest nitrotyrosine collective marker endogenous formation RNS crucial question remain detection nitrotyrosine adequately reflect toxic injurious property formation ONOO RNS induce tyrosine nitration fact represent mechanism decrease excessive level exert pro inflammatory action mechanism instance promote expression pro inflammatory cytokine cyclo oxygenase responsible formation inflammatory prostanoid mechanism independent ONOO removal minimize response furthermore ONOOor relate oxidant cytotoxic induce apoptosis effect necessarily relate ability cause protein nitration example instance bactericidal cytotoxic property onooare minimize presence CO aromatic nitration radical induce modification enhance similarly presence incubation medium decrease cytotoxicity MPO derive hypochlorous acid HOCl epithelial cell bacteria despite increase tyrosine nitration cellular protein van der Vliet Syvanen unpublished datum cytotoxic property metabolite instead mediate preferred reaction biological target necessarily correlate degree tyrosine nitration extent nitrotyrosine immunoreactivity bronchial biopsy asthmatic patient correlate directly measured level exhale inversely provocation concentration methacholine pc force expiratory volume immunohistochemical analysis nitrotyrosine apoptosis pulmonary tissue sample lung transplant recipient identify patient imminent risk develop obliterative bronchiolitis unclear degree tyrosine nitration relate disease progression study purified enzyme suggest nitration critical tyrosine residue adversely affect enzyme activity conclusive evidence vivo biological cellular change direct result tyrosine nitration instance tyrosine nitration suggest effect cellular pathway affect cytoskeletal protein tyrosine phosphorylation affect process involve example cell proliferation differentiation recent study provide support selective tyrosine nitration certain protein selective cellular target nitration RNS example specificity indicate potential physiological role protein modification case tyrosine nitration link directly change enzyme function chemical study indicate tyrosine nitration RNS account minor fraction oxidant involve reaction biological target thiol selenoprotein transition metal ion prominent extent tyrosine nitration vivo low tyrosine residue accord good estimate different analytical method detect nitrotyrosine biological system give inconsistent result important note recent rigorous study unveil substantial source artifact sample preparation frequently lead overestimation tyrosine nitration vivo previous study basis current knowledge formation nitrotyrosine merely marker oxidant questionable pathophysiological significance view low efficiency tyrosine nitration biological RNS endogenous presence variable factor influence protein nitration antioxidant RNS scavenger unlikely tyrosine nitration reliable mechanism example enzyme regulation recent discovery enzymic denitration mechanism reverse tyrosine nitration merit investigation possibility tyrosine nitration reflect signal pathway example analogous tyrosine phosphorylation sulfation biological effect mediate action secondary RNS overall biochemistry deceptively complex metabolism chemistry depend importantly local concentration pH recently describe acidification airway surface asthmatic significantly affect metabolism patient know interaction ion center iron transition metal responsible signal property activation heme enzyme guanylyl cyclase consequent formation cgmp involve smooth muscle relaxation activation certain transcription factor expression pro inflammatory anti inflammatory gene include cytokine cyclo oxygenase production respiratory mucus addition direct signal property action largely secondary RNS react multiple additional target case form nitroso nitro adduct potentially unique signal mechanism discuss formation protein nitrotyrosine postulate potential RNSspecific signal pathway interest give reversible protein cysteine residue propose affect number redox sensitive signal pathway example activation ra inhibition protein tyrosine phosphatase similar modification reactive cysteine residue transcription factor nuclear factor κb caspase contribute regulation gene expression apoptosis precise mechanism lead protein nitrosylation vivo clarify involve dinitrogen trioxide form autoxidation iron nitrosyl complex onoo change metabolism inflammatory lung disease undoubtedly affect signal pathway addition nitrosylation reverse enzymic thioredoxin glutaredoxin chemical metal oxidant mechanism evidence increase reversible modification complementary widely accept oxidant dependent redox signal pathway reported alteration nitrosothiol level tracheal secretion patient asthma cystic fibrosi point alter metabolism case provide new clue role nitrosylation control disease process unfortunately technical limitation detect nitrosylation specific protein target vivo limit understanding potential signal pathway research area expect establish clearly significance pathophysiological property overwhelming evidence increase formation oxidant different lung disease exact contribution metabolite inflammatory lung disease unclear distinctly different role different stage respiratory tract inflammatory disease pro inflammatory pro injurious acute severe stage protective antiinflammatory stable condition uncertain NOS suitable therapeutic target management inflammatory lung disease caution clearly need interpret observation tyrosine nitration animal model disease human tissue automatically implicate ONOO think indicate formation RNS mechanism furthermore animal model chronic lung disease usually reflect short term acute inflammation applicable chronic airway disease human instance phagocyte degranulation common feature observe association human airway inflammatory disease asthma occur mouse model asthma importance granule protein heme peroxidase pathology human airway disease adequately reflect animal model work animal model characteristic human disease biopsy material human subject require unravel precise role inflammatory lung disease establish clearly pharmacological inhibition NOS isoenzyme beneficial bring interesting paradox despite presume adverse role inflammatory lung disease septic shock ARDS inhalation suggest potential therapeutic strategy improve overall gas exchange intriguingly rat model endotoxemia inhalation find reduce neutrophilic inflammation protein nitration support crucial involvement inflammatory immune cell protein modification well assessment role respiratory tract disease human production RNS characteristic marker need carefully monitor disease stage care give analytical technique quantitative capacity possibility artifact monitoring exhale convenient non invasive reflect actual production fate respiratory tract correlated NOS activity lung need continue research local biochemistry lung take account presence secrete phagocyte peroxidase possible change local pH asthmatic airway modulate activity metabolism result well understanding relationship metabolic endproduct nitroso nitro adduct pro inflammatory injurious property','2000-08-15 00:00:00','http://dx.doi.org/10.1186/rr14',1546,647),('6bbbd25c11b320bdd509a9c76417ed4045349708.json','Sentinel surveillance for human enterovirus 71 in Sarawak, Malaysia: lessons from the first 7 years','BACKGROUND: A major outbreak of human enterovirus 71-associated hand, foot and mouth disease in Sarawak in 1997 marked the beginning of a series of outbreaks in the Asia Pacific region. Some of these outbreaks had unusually high numbers of fatalities and this generated much fear and anxiety in the region. METHODS: We established a sentinel surveillance programme for hand, foot and mouth disease in Sarawak, Malaysia, in March 1998, and the observations of the first 7 years are described here. Virus isolation, serotyping and genotyping were performed on throat, rectal, vesicle and other swabs. RESULTS: During this period Sarawak had two outbreaks of human enterovirus 71, in 2000 and 2003. The predominant strains circulating in the outbreaks of 1997, 2000 and 2003 were all from genogroup B, but the strains isolated during each outbreak were genetically distinct from each other. Human enterovirus 71 outbreaks occurred in a cyclical pattern every three years and Coxsackievirus A16 co-circulated with human enterovirus 71. Although vesicles were most likely to yield an isolate, this sample was not generally available from most cases and obtaining throat swabs was thus found to be the most efficient way to obtain virological information. CONCLUSION: Knowledge of the epidemiology of human enterovirus 71 transmission will allow public health personnel to predict when outbreaks might occur and to plan interventions in an effective manner in order to reduce the burden of disease.',' hand foot mouth disease HFMD common acute viral illness primarily affect infant young child occur cluster outbreak characterize rapid onset fever sore throat accompany vesicle ulcer gum tongue buccal mucosa palate punctate usually transient skin lesion appear palm sol occasionally buttock knee area fever rash subside rapidly mouth lesion week virus continue shed week temperate country HFMD occur summer tropic HFMD occur time year major causative agent HFMD coxsackievirus human enterovirus coxsackievirus genus Enterovirus family Picornaviridae enterovirus isolate hfmd case specie human enterovirus coxsackievirus CVA coxsackievirus CVB unlike aetiological agent HFMD normally cause mild disease infection report cause neurological disease manifest aseptic meningitis encephalitis poliomyelitis like acute flaccid paralysis isolate child suffering encephalitis California isolate case severe neurological disease California year historically associate outbreak report Australia Japan Bulgaria Hungary past decade country Asia Pacific region experience increase occurrence associate hfmd outbreak outbreak report Sarawak Taiwan Perth Singapore Korea Malaysia Taiwan outbreak Sarawak state Malaysia island Borneo cluster unusual paediatric death encephalitis cardiac failure observe raise lot fear anxiety heightened concern Sarawak implement sentinel surveillance programme HFMD beginning March programme set operational function Sarawak Health Department approve Director Health principle Helsinki Declaration follow surveillance operation aim investigate epidemiology common childhood disease Sarawak determine difference pattern transmission aetiological agent HFMD aim programme provide datum practical value doctor public health personnel view efficient effective virological surveillance particular provide early warning system outbreak paper describe preliminary observation surveillance programme March June early discussion number community paediatrician team set protocol sentinel surveillance programme HFMD doctor consent actively participate provide standard reporting speciman collection form sterile swab virus transport medium telephone number obtain assistance transport specimen laboratory facsimile number report case Health Department sentinel clinic doctor obtain parental consent swab take Sentinel clinic doctor provide feedback virus isolate patient contact maintain outbreak inter outbreak period assure doctor surveillance programme active ongoing datum obtain manner expect provide accurate information disease trend molecular epidemiology relevant virus specialist paediatric clinic locate town Kuching Sibu state Sarawak actively participate study March include fourth specialist clinic Sibu government polyclinic Kuching Sibu participate sentinel clinic child present sentinel clinic history oral skin lesion typical HFMD enrol surveillance study throat rectal swab obtain child enrol month possible swab obtain mouth ulcer vesicle skin lesion preliminary analysis datum month protocol modify require throat swab sentinel clinic rectal swab optional doctor request provide vesicle swab possible specimen transport ice laboratory ml viral transport medium VTM vortexe freeze thawed aliquote primary objective programme sentinel surveillance system HFMD inoculate specimen human rhabdomyosarcoma RD cell susceptible particular objective exercise identify minor causative agent know associate HFMD include multiple cell line virus isolation protocol RD cell culture normally show characteristic enterovirus CPE day harvest monolayer show extensive CPE blind passage culture show CPE day RNA extract culture harvest Tri Reagent LS Molecular Research Centre Cincinnati OH USA accord manufacturer instruction dry RNA pellet dissolve μl sterile ultra high quality RNase free water store use presence enterovirus RNA culture fluid determine previously describe pan EV RT PCR method modification duration step PCR reduce minute final extension reduce minute specimen positive pan EV primer test presence genome RT PCR primer anneal gene Mark Pallansch Centers Disease Control Atlanta generously primer sequence available prior publication PCR product sequence confirm identification change protocol identification problem misidentification local strain primer set currently specific primer design house specific identification primer list Table reaction perform Big Dye Terminator cycle Sequencing Kit version Applied Biosystems Foster City serotype non enterovirus isolate carry method sequence publish Oberste colleague Chu Ishiko colleague prior serotype select non enterovirus perform exclusively accord Oberste method Ishiko method publish comparison method serotype new isolate method determine human specie enterovirus circulate region Ishiko primer give identical serotype identification obtain Oberste method discrepancy method modify Ishiko primer convert method semi nested non nested method ease use modification verify describe Cardosa colleague retrospectively reteste non enterovirus isolate prior modify method subset isolate identify sequence subject confirmation specific primer dna sequence PCR product describe previously sequence gene product generate RT PCR isolate analysis essentially previously describe software package ClustalX alignment generate bootstrappe phylogenetic tree neighbour joining method accord Saitou Nei DNA sequence TTCCAATACCACCCCTTGGATGA Antisense gene ACYATGAAAYTGTGCAAGG sense gene CCRGTAGGKGTRCACGCRAC antisense gene CTGGGACATAGAYATAACWGG sense gene GCICCICAYTGITGICCRAA antisense gene ATAATAGCAYTRGCGGCAGCCCA sense partial gene AGAGGGAGRTCTATCTCYCC Antisense partial gene cctccggcccctgaatgcggctaat sense partial ATTGTCACCATAAGCAGCCA Antisense partial relative genome strain GenBank accession number primer method describe list Table new DNA sequence phylogenetic analysis hitherto unpublished deposit GenBank accession number sequence previous publication group detailed protocol sample collection method practical information place public domain APNET Asia Pacific Enterovirus Surveillance Network website analysis perform software package JMP Statistics version Institute USA Prism Macintosh Graphpad Software provisional protocol provide sentinel clinic collection specimen require throat rectal swab vesicle ulcer swab possible result virus isolation study specimen obtain sentinel clinic hospital period review protocol originally implement total specimen child clinical diagnosis HFMD receive month period March August age child range month year male female specimen receive subject virus isolation specimen child total specimen heavily contaminate bacteria contaminate specimen rectal swab throat swab skin lesion remain uncontaminated cell culture harvest test enterovirus pan ev set primer specimen test yield enterovirus enterovirus specimen child enterovirus isolate child child early set specimen able isolate enterovirus throat swab rectal swab mouth ulcer vesicle swab clearly vesicle swab useful specimen child vesicle swab take child present skin lesion fill abundant fluid throat swab provide reasonably high yield enterovirus isolate decision require throat swab primary speciman sentinel clinic vesicle swab possible rectal swab require serve reduce laboratory workload outbreak laboratory receive specimen child March June male female ratio histogram panel figure show distribution hfmd case see sentinel clinic period clearly large outbreak HFMD panel figure sporadic activity peak dominant enterovirus serotype isolate outbreak show middle panel figure isolate outbreak isolate interoutbreak period specie human enterovirus isolate inter outbreak period phylogenetic analysis strain isolate Sarawak genogroup genogroup strain circulate Sarawak period figure gene phylogenetic analysis order certain major discrepancy genotype associate gene region furthermore wish provide option group wish compare datum know group use sequence step molecular identification human enterovirus genogroup genogroup strain co circulate Sarawak predominant genogroup outbreak genogroup co circulate genogroup strain outbreak genogroup virus appear sporadically outbreak specie human enterovirus isolate genogroup non outbreak period distribution genogroup genogroup strain surveillance period show panel figure phylogenetic tree Figures hfmd epidemiological curve outbreak year plot accord epidemiological week figure clearly hfmd case begin see early year week clear rise number case see early rise case differ summer outbreak see temperate country suggest outbreak Sarawak preceed summer outbreak country northern hemisphere year HFMD outbreak stretch end year peak phylogenetic tree generate gene show relationship isolate different year Figure number case decline sharply end April week long detect end June coincide hfmd case see year interestingly outbreak coincide SARS outbreak region public health measure place time evidently serve control transmission enterovirus detailed analysis datum collect sentinel clinic code send sample laboratory consistently reliably seven year study period total specimen collect case year case specimen subject virus isolation total specimen subject virus isolation ensure majority specimen majority case test specimen analysis proportion different type specimen virus yield obtain show Table specimen outbreak nonoutbreak period test enterovirus grow test throat swab comprise total number specimen test yield enterovirus detailed information enterovirus serotype isolate surveillance programme provide additional file virus isolation success rate low anticipate result month remain case throat swab useful rectal swab yield non polio enterovirus sample test note month study coincide inter epidemic period non specie human enterovirus cause HFMD compare virus isolation yield outbreak year yield hfmd outbreak cause non enterovirus find successfully isolate virus specimen collect virus outbreak year suggest difficult isolate specie enterovirus virus isolation rate month comparable obtain later circulate child non polio enterovirus isolate exclude analysis miss information age presentation child range age day month mean month median month dominant serotype enterovirus isolate child ask question different serotype enterovirus cause infection child different age table show mean age child infection Comparison mean pair unpaired test alpha show significant difference mean age child different group versus versus versus outbreak associate HFMD Sarawak Malaysia Health Department instal sentinel surveillance programme expectation able study epidemiological trend begin predict expect outbreak sufficient accuracy order implement public health intervention reduce burden disease surveillance programme ongoing Sarawak seek glean preliminary information datum generate year programme outbreak recent report similar surveillance programme Yamagata Prefecture Japan suggest Yamagata frequent importation surround country seed cluster case see annually community strain study isolate small cluster case tend see summer month situation observe outbreak year case see early year northern summer Sarawak Yamagata experience large outbreak situation genogroup shift note interesting Sarawak genogroup virus genogroup strain observe genogroup virus appear change outbreak outbreak suggest likely genogroup virus evolve Borneo outbreak experience seed import virus outbreak Sarawak typically begin early year possible genogroup strain generate Sarawak seed outbreak region typically occur later Sarawak outbreak temporal sequence regional outbreak true occur Singapore Peninsula Malaysia datum obtain year sentinel surveillance programme HFMD Sarawak provide useful clue understand epidemiology state clear appearance associate HFMD sentinel clinic signal start outbreak rise number case rapid approach suitable early warning system week time case see peak outbreak clearly explain rapid effective response public health team way know alternatively year cycle outbreak observe verify come year provide public health official relevant information plan implement intervention programme reduce disease burden year outbreak expect expect prevent outbreak entirely effective public health measure place early limit spread reduce mortality reduce burden community health system important note epidemiological curve show HFMD distinguish infecting agent case stretch broadly month non enterovirus continue isolate cessation activity especially evident associate fatal case report neighbour Singapore September October medium attention surround event generate high index suspicion Sarawak isolate Sarawak August year numerous continue isolate end sociological factor affect shape hfmd epidemiological curve Sarawak epidemiological curve specifically show genogroup strain consistently sharp define mean median age child HFMD month month respectively mean age differ group infect different serotype intrigue cause large sharp outbreak suggest capacity spread rapidly susceptible population quiescent community year outbreak cohort child year age expose child susceptible provide condition sweeping transmission community annual birth cohort Sarawak thousand year susceptible child state accord trend report expect outbreak Sarawak time writing begin pick case sentinel programme past experience outbreak Sarawak follow outbreak country region decide datum public domain order public health practitioner Asia Pacific region benefit experience prepare spread region month come main conclusion arise preliminary report describe outbreak occur year Sarawak start outbreak cause genogroup virus furthermore outbreak associate genogroup virus genetically distinct subgenogroup isolate year surveillance programme closely related genogroup virus isolate sarawak far experience large hfmd outbreak cause genogroup subgenogroup behave like specie enterovirus occur sporadically surveillance Sarawak occurrence genogroup infection tightly cluster case rise fall rapidly','2006-07-07 00:00:00','http://dx.doi.org/10.1186/1471-2458-6-180',1783,660),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json','Reliability of case definitions for public health surveillance assessed by Round-Robin test methodology','BACKGROUND: Case definitions have been recognized to be important elements of public health surveillance systems. They are to assure comparability and consistency of surveillance data and have crucial impact on the sensitivity and the positive predictive value of a surveillance system. The reliability of case definitions has rarely been investigated systematically. METHODS: We conducted a Round-Robin test by asking all 425 local health departments (LHD) and the 16 state health departments (SHD) in Germany to classify a selection of 68 case examples using case definitions. By multivariate analysis we investigated factors linked to classification agreement with a gold standard, which was defined by an expert panel. RESULTS: A total of 7870 classifications were done by 396 LHD (93%) and all SHD. Reporting sensitivity was 90.0%, positive predictive value 76.6%. Polio case examples had the lowest reporting precision, salmonellosis case examples the highest (OR = 0.008; CI: 0.005–0.013). Case definitions with a check-list format of clinical criteria resulted in higher reporting precision than case definitions with a narrative description (OR = 3.08; CI: 2.47–3.83). Reporting precision was higher among SHD compared to LHD (OR = 1.52; CI: 1.14–2.02). CONCLUSION: Our findings led to a systematic revision of the German case definitions and build the basis for general recommendations for the creation of case definitions. These include, among others, that testable yes/no criteria in a check-list format is likely to improve reliability, and that software used for data transmission should be designed in strict accordance with the case definitions. The findings of this study are largely applicable to case definitions in many other countries or international networks as they share the same structural and editorial characteristics of the case definitions evaluated in this study before their revision.',' country case definition national disease reporting case definition AIDS publish Centers Disease Control Prevention CDC sack publish survey state Puerto Rico Washington DC reveal important variation case definition different state conclude necessity unify case definition surveillance datum state compare CDC collaboration Council State Territorial Epidemiologists publish edition case definition public health surveillance case definition important tool national surveillance system international surveillance network Koo colleague analyze surveillance datum Cholera Latin America describe importance uniform case definition datum comparable country European Union EU case definition european network reach obligatory status member state report EU SARS epidemic case definition major impact country consider affected result severe political economic consequence number country colleague demonstrate difficulty determine optimal case definition satisfactory diagnostic gold standard lack sharp contrast importance case definition hardly research publish performance surveillance case definition study rare local health department health professional able understand case definition extent case definition unambiguous assure reliability knowledge publication investigate issue focus case definition nosocomial infection Gastmeier colleague investigate uniform case definition nosocomial infection surveillance system Germany apply different investigator set case study general importance case definition public health surveillance current need harmonization international surveillance system conduct systematic evaluation national case definition objective identify general specific criterion recommendation improvement case definition Germany federal republic state subdivide county country local county health department total number primary recipient infectious disease notification physician laboratory local health department verify incoming notification assess need public health action local health department use software product market classify case report accord national edition case definition report case electronically state health department report forward Robert Koch Institute RKI federal institution charge national infectious disease surveillance Germany edition national case definition notifiable infectious disease introduce Germany follow implementation new law control infectious disease Infektionsschutzgesetz IfSG IfSG determine set disease pathogen notify physician laboratory Federal Republic Germany eastern state belong Democratic Republic Germany East Germany State Berlin enact complementary rule certain disease additionally notifiable state jurisdiction notifiable Germany case definition develop RKI delphi method include expertise state epidemiologist national reference laboratory medical scientific association specific disease case definition infectious condition public health surveillance publish CDC take account having publish IfSG case definition fall implement beginning RKI publish additional case definition January disease exclusively notifiable eastern state jurisdiction June September conduct systematic evaluation case definition purpose revise end german case definition divide type evidence clinical picture laboratory detection epidemiological confirmation type evidence specifically define disease table base requirement type evidence fulfil case classify category revised edition case definition category name clinically diagnose illness epidemiologically laboratory confirm clinically epidemiologically confirm illness laboratory confirm clinically laboratory confirm illness laboratory detect infection fulfil clinical criterion laboratory detect infection unknown clinical picture edition case definition category name slightly differently notifiable disease category reportable local health department level require laboratory detection pathogen epidemiological confirmation exception tuberculosis polio measle Creutzfeldt Jakob disease case report local health department level category clinical picture fulfil June conduct Round Robin test analogy establish quality control procedure laboratory testing unit Round Robin test mainly proficiency test order determine laboratory performance mean compare test identical item laboratory accordance predetermine condition ask local state health department classify selection write case example basis case definition implement respectively disease notifiable east German States proficiency test generally intent assess ability laboratory find identical result apply method assess extend case definition unambiguous assure identical classification health department apply different outcome variable analysis disease identification disease define correctly identify participant Round Robin test able identify correct disease case case categorization case example consider correctly categorize participant classify case example correct disease correct case definition category define gold reporting decision reportability consider correct case report clinical picture clinical picture compatible salmonellosis characterize diarrhea abdominal pain malaise vomiting fever Salmonella cause infection outside intestinal tract example arthritis endocarditis pyelonephritis septicaemia laboratory diagnosis isolation culture pathogen stool clinical material blood urine identification serogroup attempt clinical picture clinical picture acute salmonellosis define following symptom diarrhea cramp like abdominal pain vomiting fever additional information Samonella cause generalize septicemia localize infection outside intestinal tract example arthritis endocarditis pyelonephritis case acute infection report reactive arthritis cause Salmonella infection report laboratory diagnose positive finding follow method direct detection pathogen isolation pathogen culture additional information result identify serogroup lysotype report epidemiological confirmation Epidemiological confirmation define follow constellation take account incubation period Epidemiological link laboratory diagnose human infection person person transmission source exposure animal contact food consumption food include drink water Salmonella spp laboratory detect non consumed Contact animal poultry laboratory detect infection contact secretion consumption product egg term mark asterix define detail glossary case definition level forward accord case definition category give correct disease identify inversely decision reporting see correct case report level fact classify way case hold case forward wrong disease identification priori consider incorrect reporting base question case need forward state level direct result disease identification case definition category sensitivity reporting define number case correctly forward divide number case forward accord gold standard positive predictive value reporting define number case forward forward precision reporting define number case correctly forward state level correctly hold local health department level divide total number case example state report precision outcome parameter follow analysis specifically assess effect different style formulating case definition fourth outcome variable clinical classification consider correct clinical picture classify accord gold standard regardless part case definition correctly classify analysis compare case definition narrative description clinical pic ture ifsg case definition case definition explicit check list format clinical criterion implement disease additionally notifiable specific state new IfSG case definition case example consist facsimile excerpt follow source laboratory report form physician form protocol patient interview additional file case example create base real case come attention RKI quality control process information service hotline RKI offer health department case example pre test epidemiologist RKI epidemiologist public health nurse state local health department datum respondent analyze classification originally intend create case definition challenge result respondent epidemiologist reassess individual case example examine classification originally intend justify base process gold standard define case example compare response establish gold standard stratify follow variable health department east german versus west german state disease case example physician participate routine quality control case report versus exclusively public health nurse institutional level local health depart ment state health department RKI acceptance style case definition check list text software local health department selection distribution case example describe conduct individual analysis group univariate analysis conduct multivariate analysis spss Windows version distribution classification compare gold standard order identify common discrepancy base discrepancy identify case definition affect identify specific aspect case definition repeatedly interpret differently participant indicate failure case definition unambiguous reliable aspect summarize order deduct commonality lead specific recommendation improve particular case definition improve formulation case definition general Round Robin test conduct write survey address local health department Germany question structure equipment local health department experience new IfSG ask profession person actually fill questionnaire attitude experience case definition multivariate analysis limit datum local health department additional case example east german state statistically significant association mention following disease case example group significantly associate report group group health department RKI software show high chance identify disease disease identification case example accord gold standard compare health department commercially available software program group administrative level respondent work significantly associate outcome report analysis case set set additionally disease east German States adjust disease chance correct reporting level time high case state level staff compare local health department staff ci follow observation qualitative analysis concept epidemiological confirmation understand example travel endemic country equivocally see epidemiological confirmation haemorrhagic fever travel Egypt evaluation case definition show fact vague definition epidemiological participant appear difficulty decide clinical sign symptom mention case definition existent case list descriptive case example diarrheal disease evidence specific pathogen frequently classify Laboratory finding elevated antibody value serum repeatedly classify laboratory detection case definition require rise antibody case definition detection pathogen accept detection specific material normally sterile material blood detect meningitidis limitation frequently information case definition intend serve additional background information mistakenly selection criterion statement clinician describe rash typical measle fever ask availability case definition local health department respond case definition accessible work place case definition see useful health department answer question useful clarity individual section case definition rate differently section clinical picture case definition see unambiguous case definition respondent majority case definition minority case definition respondent respondent section laboratory confirmation case definition see unambiguous case definition respondent majority case definition minority health department state case classification exclusively primarily public health nurse respect case example present participant respondent local health department state case example realistic result evaluation show case definition appear clearly define interpret differently user result severe misclassification reduce sensitivity positive predictive value study believe systematically assess effect complexity case definition likely affect report precision unfortunately complexity case definition result methodological limitation available laboratory test influence case definition system different type evidence lead different category appear complex intuitive classical category suspect probable confirm detailed differentiation german case definition enable apply computer algorithm order translate EU case definition datum compatible standard european surveillance network report reassessment gold standard receipt response result modification case example procedure take place initial review process gold standard actual analysis believe legitimate necessary order correct bias cause unforeseen ambiguity case example software local health department significantly associate quality datum subgroup outcome apparently software strong determinant give study design experience implement electronic surveillance system Germany show commercially available software product fully implement standard publish RKI data transmission software delay year interesting finding administrative level participant significantly associate outcome participant state health department significantly high rate agreement reporting gold standard participant local health department explain fact staff state level generally high train epidemiology infectious disease local health department staff routinely involve quality control incoming case report training supervision local health department staff observed quantitative effect propagated explanation merge main conclusion case definition carefully formulate order assure unambiguous interpretation local health department personnel detailed evaluation study result substantially revise edition german case definition rephrase case definition check list format indicate clearly symptom sign fulfil disease previously jointly describe case definition define separately Dengue separate haemorrhagic fever hemolytic uraemic syn drome create new separately EHEC rephrase definition way serologic confirmation necessity sample clearly apparent beginning material pathogen detect highlight list relevant case glossary define expression repeatedly case case definition limited criterion relevant decision process additional explanatory information clearly indicate separate section case definition evidence type epidemiological confirmation completely redesign replace previously term epidemiological link accept type epidemiological link specify individually case definition practical implication support analysis software local health department design strict accordance case definition identical terminology structuring easily archive local health department equip identical software system develop supervision institution possibly country process develop implement new electronic surveillance system want learn form experience case example book result study constitute detailed feed participant study training material public health nurse demonstrate rigorous reduction case definition testable yes criterion check list format likely improve reliability reduce differential diagnostic complexity disease limited number yes criterion major challenge carry benefit facilitate computerized testing algorithm quality control case classification reliability epidemiologic surveillance largely depend reliability case definition essential create revise case definition base systematic evaluation basic principle revision german case definition edition deduct analysis applicable case definition country United States Ireland Sweden Mexico international network EU share structural editorial characteristic identify problematic edition german case definition believe finding highly relevant national international surveillance system','2006-05-10 00:00:00','http://dx.doi.org/10.1186/1471-2458-6-129',1859,645),('6ea2532aacbdcea4609067523e9e72fa8cd73200.json','Mutational analysis of human CEACAM1: the potential of receptor polymorphism in increasing host susceptibility to bacterial infection','A common overlapping site on the N-terminal IgV-like domain of human carcinoembryonic antigen (CEA)-related cell adhesion molecules (CEACAMs) is targeted by several important human respiratory pathogens. These include Neisseria meningitidis (Nm) and Haemophilus influenzae (Hi) that can cause disseminated or persistent localized infections. To define the precise structural features that determine the binding of distinct pathogens with CEACAMs, we have undertaken molecular modelling and mutation of the receptor molecules at previously implicated key target residues required for bacterial binding. These include Ser-32, Tyr-34, Val-39, Gln-44 and Gln-89, in addition to Ile-91, the primary docking site for the pathogens. Most, but not all, of these residues located adjacent to each other in a previous N-domain model of human CEACAM1, which was based on REI, CD2 and CD4. In the current studies, we have refined this model based on the mouse CEACAM1 crystal structure, and observe that all of the above residues form an exposed continuous binding region on the N-domain. Examination of the model also suggested that substitution of two of these residues 34 and 89 could affect the accessibility of Ile-91 for ligand binding. By introducing selected mutations at the positions 91, 34 and 89, we confirmed the primary importance of Ile-91 in all bacterial binding to CEACAM1 despite the inter- and intraspecies structural differences between the bacterial CEACAM-binding ligands. The studies further indicated that the efficiency of binding was significantly enhanced for specific strains by mutations such as Y34F and Q89N, which also altered the hierarchy of Nm versus Hi strain binding. These studies imply that distinct polymorphisms in human epithelial CEACAMs have the potential to decrease or increase the risk of infection by the receptor-targeting pathogens.',' bacterial pathogen Neisseria meningitidis Nm Haemophilus influenzae hi frequently find nasopharynx substantial proportion healthy population capable cause infection susceptible individual Turk Foxwell et al nm typable hi THi invade nasopharyngeal epithelial barrier cause septicaemia meningitis case Nm rapidly life threaten van Deuren et al non typable hi NTHi lack polysaccharide capsule associate localize respiratory tract conjunctival infection Foxwell et al strain belong Hi biogroup aegyptius hi aeg associate brazilian purpuric fever Foxwell et al factor determine susceptibility infection frequent colonizer entirely clear colonization pathogenesis essential step adherence mucosal epithelial cell investigation show bacterial targeting specific human signal molecule integrin sialic acid binding Ig like lectin Siglecs carcinoembryonic antigen cell adhesion molecule CEACAMs lead cellular invasion Virji et al Hauck Meyer Jones et al ceacam emerge common target respiratory mucosal pathogen include Nm hi Moraxella catarrhali urogenital pathogen Neisseria gonorrhoeae enteric pathogen Escherichia coli Salmonella Leusch et al Virji et al Chen et al Gray Owen et al Hill Virji antigen relate cell adhesion molecule belong immunoglobulin Ig superfamily member ceacam subgroup express human epithelial cell include widely express transmembrane GPI anchor CEA ceacam terminal igv like domain variable number like domain comprise extracellular domain long short cytoplasmic tail Tsutsumi et al Prall et al Hammarstrom function attribute include cell cell adhesion insulin regulation angiogenesis Obrink Hammarstrom Wagener Ergun Najjar target ceacam gonorrhoeae Nm hi lead cellular invasion passage polarized monolayer Virji et al Gray Owen Soriani Setchfield Hill Virji unpublished datum wide range bacterial adhesin involve target ceacam terminal domain include Opa protein major adhesin family pathogenic Neisseria protein hi Chen Gotschlich Virji et al Hill et al Nm gonorrhoeae contain multiple copy Opa gene encode conserve domain form barrel structure bacterial membrane variable domain form surface expose loop spite surface diversity afford hyper variable domain loop majority Opa protein capable target ceacam Virji et al Hauck Meyer protein Hi similar barrel form protein surface variable loop Webb Cripps Vandeputte Rutten et al interaction bacterial ligand ceacam complex bind domain appear involve variable loop bacterial adhesin Virji et al Bos et al de Jonge et al interestingly antibody inhibition study show diverse ligand neisseria haemophilus bind overlap site domain addition mutational analysis domain identify critical residue particularly alanine substitution site abrogate binding Opa express bacteria additional residue locate vicinity appear determine efficiency interaction Opa molecule Virji et al bacterial bind surface domain protein face compose beta strand CFG brevity despite extensive investigation number laboratory particularly neisserial Opa protein remain unclear precisely CEACAM bind ligand engage receptor dimensional structural model human domain previously generate base Ig family molecule Virji et al current investigation refine previous model base murine crystal structure Tan et al examination model show well confluence residue continuous bind site suggest substitution position core bind region affect bacterial access implicate key bind residue introduce conservative non conservative substitution position examine binding variety bacterial strain receptor construct datum suggest single nucleotide polymorphism SNPs individual population introduce substitution ceacam sequence particularly bacterial bind site decrease significantly increase functional affinity pathogen interaction increase bind affinity result increase cellular invasion lead increase host susceptibility infection CEACAM target bacteria dimensional model domain previously produce base Ig family molecule REI Fig Virji et al model whilst mutation affect binding Nm Hi locate centrally CFG face protein involve Opa binding locate distance CFG face Fig subsequently crystal structure murine soluble domain report Tan et al base remodelle human Ndomain loop contain fold CFG face model lie close proximity critical residue involve bacterial adhesion Fig examination model suggest bacterial bind pocket form flat surface centre CFG face appear substitute neighbouring residue Phe Asn respectively result flattening bacterial bind surface Fig provide increase access key residue change increase size hydrophobic patch centre Fig effect facilitate binding bacterial ligand target receptor order assess importance residue interaction mucosal pathogen substitution introduce site direct mutagenesis site domain substitution introduce show Fig chimeric receptor protein native sequence Virji et al new model base murine domain crystal structure Tan et al model present Van der Waals surface representation case locate centrally critical residue bacterial binding left Stereo pair present ribbon diagram domain native form substitution flattening bacterial bind region improve accessibility primary bind residue chain atom colour green red blue increase hydrophobic nature bind site mutant structure right surface presentation bind site view correspond rotation stereo image horizontal axis colour accord hydrophobicity stitution produce transient transfection COS cell functional study describe domain specific monoclonal antibody mAb show require receptor recognition substitution amino acid effect binding polyclonal anti ceacam antibody Virji et al novel receptor construct produce current study examine ability bind mAb recognize domain receptor Jones et al bind receptor construct native fc molecule equal efficiency Leu Thr position support binding extent Ile native molecule similarly position Phe effectively substitute tyr finally completely abrogate antibody binding opa express phenotype Nm strain investigate binding modify Fc receptor derivative express ing distinct Opa protein opaa OpaB opad derivative express Opa protein designate opax Virji et al assess receptor overlay experiment Fig Nm isolate show reduced binding soluble receptor substitution whilst affect opad binding significantly confirm previous study Virji et al introduction Leu Thr residue position generally disruptive Opa interaction substitution affect bind opab opax reduce binding opad opaa substitution effect opad opab whilst have opposite effect OpaA reduction opax twofold increase binding substitution position Ala support binding Opa protein substitution unsuitable Opa protein strain tolerate opax interestingly Phe prove favoured residue case opax OpaB bind construct threefold high level native receptor substitution Asn reveal distinct pattern interaction binding reduce order opaa OpaX opab Fig datum suggest Opa bind receptor require extended aliphatic chain position certain arrangement chain facilitate binding Opa protein Ile leu Thr position Virji et al chain residue introduce position study show comparison removal hydroxyl residue tolerate preferred absence aromatic ring overall deleterious effect opa binding finally reduction chain extension reduce binding Opa protein Comparison Opa difference exist surface expose loop structure show Fig suggest combination loop structure involve present appropriate binding partner distinct residue receptor non typable THi strain Rd Eagan receptor overlay experiment Fig alanine substitution confirm previous result Virji et al take individually substitution Ala Thr abrogate binding Fig relative binding Fc construct meningitidis isolate express distinct Opa protein bacterial lysate dot nitrocellulose overlaid fc construct indicate bind relative native receptor determine densitometric analysis immno blot NIH Scion Image programme cent bind level indicate horizontal line allow comparison native receptor binding mean value SE replicate show case OpaA alanine substitution position confirm previous observation Virji et al black insert level binding receptor construct Opaisolate strain experiment conduct simultaneously presence receptor show strain substitution reduce bind THi strain Eagan suggest requirement extended hydrophobic arm Ile Eagan binding alanine substitution abrogate binding Rd Eagan effect NTHi observe contrast substitution lead increase binding strain vary native receptor binding level differential effect strain test range binding Rd slight increase binding datum broadly reflect result neisserial opa binding finally substitution reduction abrogation chain deleterious effect effect enhanced binding Fig summary hi strain primarily require enable receptor target influence binding THi Nm removal OH provide favourable environment chain limit bacterial interaction substitution short chain favourable especially Rd CEACAM binding previous study interaction Hi aeg strain show analogous Nm Virji et al current study accordingly demonstrate requirement Hi aeg Fig addition Nm strain requirement hydrophobic aliphatic chain partly fulfil Leu less extent Thr substitution residue Phe Ser produce complex profile phe generally create suitable well environment loss aromatic ring tolerate position substitution enhance bind dramatically strain test exhibit similar binding receptor construct substitution alanine substitution residue Fig strain exhibit difference bind receptor substitution demonstrate structural diversity bind ligand distinct Hi aeg strain datum clear receptor construct substitution capacity increase binding hi Nm previous investigation show certain isolate bacteria capable compete native receptor Nm derivative displace Eagan receptor Virji et al investigate change receptor structure position influence phenomenon receptor dot blot overlay presence vary amount soluble receptor construct estimate ability selectively adsorb receptor consistent low affinity compare Virji et al proportion native interact compare low decline limit concentration receptor Fig contrast similar affinity receptor construct present excess mg ml Fig limit concentration great affinity apparent bind construct threefold Nm derivative Fig datum demonstrate potential influence structural modulation bacterial bind site change colonization profile target tissue assess CEACAM mutation significantly increase decrease bacterial binding soluble construct affect bacterial binding cellexpressed receptor construct analyse COS cell transiently transfecte contain substitution initially bind anti ceacam antibody transfecte CHO cell examine binding observe soluble receptor Fig bacterially express ligand overcome decrease affinity specific mutant protein multiple ligand receptor engagement target cell surface examination adherent bacteria level receptor expression microscopy reveal bacteria bind transfectant sham transfecte cell Fig receptor expression level considerably high case significant bacterial number bind transfectant bacteria bind cell barely detectable level receptor Fig CHO cell express length ability soluble native Fc inhibit bacterial binding assess opab bacteria Fig datum follow bind CHO cell report previously Virji et al observe Opa bacteria binding see untransfected cell show ii despite low level binding isolate soluble construct carry mutation dot immunoblot analysis Fig bacterial binding receptor express cell clearly visible Fig iii case THi Rd describe association significantly high cell express high level receptor assess parallel experiment show iv Fc compete bacterial binding homologous native structure express CHO cell Fig confirm previous result Virji et al comparison Fc efficiently compete OpaB bind receptor express CHO cell Fig consistent high affinity isolate Fig vi construct largely ineffective compete cell express receptor Fig ent bacterial number low cf fig lack low level binding compare datum emphasize importance cell presentation receptor addition receptor structure bacterial interaction final outcome depend balance effect exert residue substitution receptor density affect functional affinity bacteria host interaction assess relative importance degree influence substitution bacterial interaction degree receptor recognition assign category show Fig datum substitution profound effect interaction strain whilst chain well suited Leu effectively substitute Ile strain whilst polar Thr tolerate Nm Opa protein tolerate substitution require case interestingly prefer general position native tyr substitution native receptor expression level bacterial binding correlate frequently mutate molecule case substitution position bacterial binding see low level receptor expression arrow panel reverse case high receptor level arrowhead leave panel bacterial binding arrowhead middle panel relatively low sampling cell exhibit phenomenon note receptor recognition affect mutation domain determine immunoblotte immunofluorescence microscopy Fig effect binding Nm Opa derivative small apparently well suit overall bind hi especially Hi aeg strain datum emphasize extensive inter intraspecie ligand variation receptor target great interstrain difference Nm structural aspect bacterial ligand affect receptor recognition consider strain Opa protein structure know Fig Hobbs et al hypervariable loop Opa protein implicate binding ceacam Virji et al Bos et al de Jonge et al give combination HV region OpaA strain provide distinct combination Fig consistent distinct pattern targeting variant receptor molecule observe study protein contain sufficient similarity bind hydrophobic region terminal domain cell express CEA family molecule highly homologous majority ceacam target pathogenic neisserial adhesin belong Opa family protein Virji et al Chen et al Gray Owen et al despite structural variability Opa protein target common site receptor centre bind appear CFG face domain conserve CEA member important residue identify alanine scan mutagenesis locate close proximity Virji et al precisely variant Opa protein bind common receptor site entirely clear complementary set sequence variable domain Opa protein involve variability ligand determine bind preference distinct member CEA family represent mechanism determine tissue tropism Virji et al Bos et al Gray Owen Hauck Meyer de Jonge et al strain typable NTHi lineage include biogroup aegyptius bind CFG face domain Virji et al previous analysis receptor binding THi isolate show behave similarly strain test have primary requirement addition affect substitution current study THi strain Rd Eagan demonstrate similar overall bind pattern case intraspecie Fig bind meningitidis isolate cell express receptor competition cell express soluble receptor bacterial binding cell express construct detect anti nm antisera TRITC conjugate secondary antibody interaction bacteria cell express receptor investigate absence presence compete soluble receptor bind soluble receptor low native dot blot Fig relatively high certain cell presumably express high level receptor competitive experiment native Fc preincubate ug ml bacteria min prior infection target cell Fc inhibit bacterial binding homologous receptor significantly abrogate binding cell express soluble inefficient inhibit bacterial binding CHO level homologous inhibition apparent examine adhesion bacteria cell express low level receptor pepper area show evident response different amino acid substitution apparent consistency specie strain test dramatic loss bind follow substitution observe previously appear central overlap bacterial bind footprint Virji et al similar situation occur mouse terminal domain appear engage mouse hepatitis virus spike protein replacement Thr allele reduce virus bind significantly Tan et al cell surface receptor interaction ligand involve hydrophobic contact point provide major bind energy hydrophobic residue surround contribute specificity bind Clackson Wells Kwong et al Kim et al murine protrude surround number surface expose charge residue form bind area Tan et al accordingly current study mutation introduce position human receptor support requirement hydrophobic pocket site addition extended aliphatic chain prefer Ala disrupt binding bacteria polar Thr reduce binding hi Nm strain frequently tolerate bind pocket flank polar residue Fig contribution bacterial ligand binding apparent variable Virji et al importance surround residue human bacterial binding support observation mAb direct domain require residue common bacteria block binding bind bacteria investigate Virji et al Hill Virji murine domain strand arrangement derive form crystal structure depict loop assume convoluted conformation previous model human contain flat loop like Ig fold Fig bind bacterial ligand fc construct determined current study receptor recognition category colour code accord percent bind relative native molecule Diagrams show CEACAM bind meningococcal ligand structure Opa protein structure analyse detail far respect ceacam binding de Jonge et al clarity ease discussion Opa protein structure relationship variable domain strain Opa protein show SV semivariable domain strain Opa identical hypervariable structure OpaA OpaB OpaB base result locate distance bind focus implicate bacterial binding alanine scanning homologue scan mutagenesis Bos et al Virji et al remodelling show produce convoluted structure loop relocate close aromatic ring suggest require maintain convoluted structure loop Tan et al substitute Phe support bacterial binding provide well environment bacterial strain test hand frequently abrogate receptor recognition surprisingly tolerate substantial number bacterial ligand flexible variable loop domain bacterial ligand produce induce fit receptor need consideration interestingly conserve majority human ceacam exception contain site importance human maintain dimensional structure require human receptor crystallographic datum substitution produce interesting datum point view pathogenesis whilst appear affect Opa protein reduce receptor recognition suggest potential contribution determine tissue tropism Virji et al substitution occasionally cause dramatic increase bacterial adhesion especially Hi isolate observe increase binding strain specie examine possible multiple receptor present target cell overcome reduced bind affinity mutate receptor examine strain Rd binding substitute receptor express transiently transfecte COS cell whilst receptor target cell low level receptor expression proportion cell high level receptor significant number bacteria attach point mutation ceacam decrease increase bacterial load additional factor dictate bacterial binding include receptor level target cell ligand Neisseria Opa protein hi far identify protein share similar beta barrel structure surface expose variable loop region engage receptor identify Opa protein study mutagenesis strain de Jonge et al strain relate strain strain current study study de Jonge et al implicate key motif region Fig meningococcal Opa protein receptor target ELK motif Opa region involve dimensional presentation receptor engage residue bacterial ligand de Jonge et al strain opab opad protein contain motif GxLxS position respectively Opa contain PxIxN motif region OpaA contain DLK opad contain EDK study laboratory progress assess precise variant Opa receptor residue pair involve mutual recognition polymorphism affect host susceptibility find site gene encode host receptor target pathogen result loss gain receptor associate function snp lead multiple diverse downstream effect alter transcriptional response manifestation disease Sakuntabhai et al extracellular domain polymorphism direct effect alter binding pathogen ligand receptor snp innate immune system especially affect pathogen associate pattern recognition receptor cytokine study extensively change extracellular domain LPS bind toll like receptor implicate increase risk bacterial infection schroder Schumann snp associate severe respiratory syncytial virus RSV bronchiolitis infant case alter interaction viral fusion protein implicate ligand regard primary mechanism Tal et al Schroder Schumann SNPs correlate meningococcal disease Read et al contrast rare snp find commonly gene patient meningococcal disease Smirnova et al support notion rare common variant associate infectious disease susceptibility study present suggest possible polymorphism increase bacterial load snp ceacam identify list NCBI SNP database identify domain include case substitution observe residue difference occur member ceacam family example CEA contain position Fig major difference CEA affect bacterial binding Fig mutation expect produce CEA like bind pattern affect tropism bacteria observe CEA Virji et al de Jonge et al study require assess ceacam polymorphism example substitution occur human molecular analysis bacterial ligand interaction population frequency susceptible population SNP substitution change colonization profile nasopharynx competition study demonstrate increase binding afford Hi aeg isolate increase binding compete Nm isolate vitro competition assay receptor situation native receptor reverse show Figs additional factor affect bacterial binding cell express receptor include receptor density whilst certain residue substitution reduce functional affinity bacterial interaction high receptor density increase affinity final outcome depend interplay parameter recent study role receptor density enhancement bacterial attachment invasion evaluate detail Bradley et al Rowe et al interesting analyse bacterial invasion cell line express variant carry mutation employ cell line receptor expression level control development receptor polymorphism scenario lead increase bacterial ligand binding ceacam Nm hi ceacam bind ligand Opa know undergo antigenic variation population antigenic structural variant present select course host colonization subsequent pathogenesis Virji et al Duim et al Meyers et al receptor repertoire subtype select bacteria capable bind high affinity mention upregulation receptor expression target cell increase bacterial bind affinity case high affinity interaction result cellular invasion low affinity load bacteria proceed surface adhesion Tran Van Nhieu Isberg Bradley et al expression ceacam normal epithelia respiratory tract report allow bacterial attachment possible subsequent penetration tissue Tsutsumi et al Virji follow exposure cytokine IFN TNF ceacam expression colonic carcinoma cell show increase Fahlgren et al addition certain viral infection show upregulate expression epithelial cell line Avadhanula et al increase cytokine level follow viral infection lead increase ceacam expression bacterial association respiratory epithelia subsequent invasion deep tissue organism situation explain epidemiological association increase incidence Nm hi infection follow certain viral infection Cartwright et al Takala et al summary little know certain people susceptible infection frequent colonizer human nasopharynx interestingly opportunistic pathogen Nm hi catarrhali investigate target specific substitution produce favourable target distinct mucosal isolate possible occurrence receptor polymorphism human population lead great bacterial binding increase chance cellular invasion give colonization rate organism generally population frequency invasive infection population combination event require increase host susceptibility inflammatory condition increase receptor density population carry specific polymorphism provide bad scenario sequence mature human domain align correspond domain murine give gapless alignment residue residue identity crystal structure co ordinate mouse residue take structure file contain domain pdb code homology model human domain build standard method final refinement model perform soak thick layer water energy minimizing constrain backbone atom original position template structure final round minimization conjugate gradient step constrain backbone heavy atom force constant kcal stereochemical analysis structure perform Procheck find similar quality template crystal structure production mutant carry site direct mutagenesis pIG construct contain dna encode domain describe previously Watt et al oligonucleotide primer create amino acid substitution position domain show Table primer sequence publish previously Watt et al introduce mutation amplify polymerase chain reaction pIG construct common forward primer reverse primer contain desire mutation complementary forward primer contain mutation common reverse primer appropriate mutation amplify common forward common reverse primer gene clone pIG restriction site hindiii soluble receptor protein contain appropriate amino acid substitution prepare previously describe transient transfection COS cell Teixeira et al Virji et al Fc overlay experiment kindly donate Dr Watt Virji et al Watt et al strain study describe previously Virji et al Nm strain serogroup variant strain serogroup variant express Opa previously designate opax encode opab locus opa isolate show bind CHO Virji derivative THi Rd know bind CHO Virji unpublished control THi strain Rd acapsulate serotype isolate Eagan serotype isolate NTHi strain hi aeg strain conjunctiva isolate Nm grow brainheart infusion BHI agar supplement heated horse blood HBHI hi strain grow hbhi agar supplement Levinthal base mg ml NAD haemin strain culture cell african green monkey kidney cell transient transfection culture Dulbecco modify Eagle medium DMEM contain heat inactivate Foetal Calf Serum FCS Gibco mm glutamine mg ml penicillin mg ml streptomycin humidified atmosphere bind construct fc protein dot mg ml nitrocellulose non specific bind site block BSA Dulbecco PBS contain PBST room temperature Receptor detect follow antibody rabbit polyclonal rat monoclonal direct domain mouse monoclonal recognize domain Jones et al bind antibody subsequently detect appropriate secondary antibody conjugate alkaline phosphatase develop nitroblue tetrazolium indolylphosphate bacterial interaction receptor construct bacterial lysate bacteria apply nitrocellulose strip air dry non specific bind site block bsa PBST room temperature Strips overlaid native mutate soluble fc dilute bsa PBST require concentration room temperature experiment excess mg ml receptor competition study range concentration mg ml employ follow washing remove unbound fc receptor binding detect anti human fc alkaline phosphatase conjugate substrate describe quantification densitometric analysis develop immunoblot carry NIH Scion Image software case multiple estimation carry mean SE determination report site direct mutagenesis receptor gene perform QuickChange Site Directed Mutagenesis Kit Stratagene La Jolla USA accord manufacturer instruction primer table introduce desire mutation pRc CMV construct kindly provide Professor Wolfgang Zimmermann follow sequence ensure desire substitution obtain pRc CMV construct transiently transfecte cell functional analysis DEAE dextran method describe previously Teixeira et al Virji et al','2007-02-01 00:00:00','http://dx.doi.org/10.1111/j.1462-5822.2006.00',3584,970),('6eeaffdb22b4e310867262d1891a1442769c7803.json','A universal BMV-based RNA recombination system—how to search for general rules in RNA recombination','At present, there is no doubt that RNA recombination is one of the major factors responsible for the generation of new RNA viruses and retroviruses. Numerous experimental systems have been created to investigate this complex phenomenon. Consequently, specific RNA structural motifs mediating recombination have been identified in several viruses. Unfortunately, up till now a unified model of genetic RNA recombination has not been formulated, mainly due to difficulties with the direct comparison of data obtained for different RNA-based viruses. To solve this problem, we have attempted to construct a universal system in which the recombination activity of various RNA sequences could be tested. To this end, we have used brome mosaic virus, a model (+)RNA virus of plants, for which the structural requirements of RNA recombination are well defined. The effectiveness of the new homomolecular system has been proven in an experiment involving two RNA sequences derived from the hepatitis C virus genome. In addition, comparison of the data obtained with the homomolecular system with those generated earlier using the heteromolecular one has provided new evidence that the mechanisms of homologous and non-homologous recombination are different and depend on the virus\' mode of replication.',' RNA recombination common phenomenon observe type virus RNA carrier genetic information positive sense single strand RNA virus negative sense single strand RNA virus double strand RNA virus retrovirus show RNA recombination enable exchange genetic material similar virus distinctly different virus permit crossover viral host RNA take account structure viral genomic molecule location crossover site basic type RNA recombination distinguish homologous aberrant homologous non homologous occur identical similar RNAs molecule display local homology involve different molecule collect datum suggest RNA recombinant form accord copy choice model viral replication complex start nascent RNA strand synthesis template call RNA donor switch template call RNA acceptor accordingly main factor think affect RNA recombination structure recombine molecule ability viral replicase switch template gain knowledge mechanism RNA recombination model experimental system create provide specific datum describe homologous non homologous recombination particular virus poliovirus mouse hepatitis virus brome mosaic virus BMV turnip crinkle virus tomato bushy stunt virus result involvement viral replicase protein recombination demonstrate wide spectrum RNA motif support recombination identify general collect datum suggest exist major type RNA structural element induce recombination event universal one mediate template switching different viral replicase region correspondence address tel fax email online version article publish open access model user entitle use reproduce disseminate display open access version article non commercial purpose provide original authorship properly fully attribute Journal Oxford University Press attribute original place publication correct citation detail give article subsequently reproduce disseminate entirety derivative work clearly indicate commercial use contact local homology complementarity ii virus specific one promoter like structure unfortunately till vivo recombination system test recombination activity give RNA sequence consequently verify hypothesis find general law govern studied process study genetic RNA recombination characterize vivo system develop BMV BMV model RNA virus plant genome compose segment call encode BMV replicase protein respectively encode movement coat protein CP BMV RNAs possess identical region BMV base recombination system create Nagy Bujarski construct recombinationally active BMV mutant genome compose modify call recombination vector detail Figure modify intergenic code region unchanged despite introduce change recombination vector stable replicate infect plant start recombine recombinationally active sequence RAS introduce CP code sequence modify region RAS cloning site non homologous recombination observe nt sequence complementary position insert sequence introduce antisense orientation interestingly fragment insert sense figure BMV base recombination system white black gray box represent code noncoding recombinationally active sequence respectively location primer specific RT PCR amplification BMV parental recombinant indicate arrow BMV genome BMV genome consist RNA segment BMV RNAs share identical region trna like structure end recombination vector vector derivative modify end detail ref include RAS cloning site ii nt sequence derive region cowpea chlorotic mottle virus mark CCMV iii sequence nt count end mark region iv nt end BMV mark non homologous recombination non homologous recombination observe nt sequence call show complementary position insert vector presence sequence derivative call allow local hybridization mediate frequent non homologous crossover think polymerase start nascent strand synthesis end switch local double strand region non homologous recombinant non homologous recombination repair replace modified end fragment come orientation support homologous crossover non homologous recombination repair vector replace highly modify end derive resultant recombinant replicate accumulate well parental molecule compete infected cell system extremely efficient employ selection pressure support accumulation recombinant RAS place different segment BMV genome propose system heteromolecular unfortunately Nagy Bujarski BMV base recombination system limitation design manner viable recombinant easily form sequence derive RAS consequently heteromolecular system apply test recombination capacity RNA motif olsthoorn et al attempt solve problem insert examine sequence region BMV system develop change encode BMV polymerase strongly affect studied process describe new BMV base recombination system construct way test ras place segment BMV genome modify molecule call system homomolecular prove usefulness homomolecular system employ examine recombination activity sequence derive hepatitis virus HCV genome examine sequence insert direct inverted repeat demonstrate nt hypervariable region HCV efficiently support homologous non homologous crossover conservative nt portion HCV induce non homologous recombination event direct comparison hetero homomolecular system reveal crucial difference mechanism homologous non homologous recombination involve preferentially different segment BMV genome occur easily genomic RNAs plasmid contain fulllength cdna BMV modify recombination vector respectively generous gift Bujarski Northern Illinois University DeKalb enzyme ecori SpeI xbai RNA polymerase RNasine RQ DNase RNase free mmlv reverse transcriptase taq polymerase cloning vector Promega follow primer construction pMatNH pMatNH pMatH pMatNH pmath plasmid pMagH contain length cdna vector carry recombinationally active sequence insert antisense sense orientation respectively plasmid construct way linearize spei endonuclease ligate spei cut cdna plasmid carry antisense sense pMagH orientation identify prepare pMatNH pMatH plasmid contain length cdna MatNH MatH pMagH digest KpnI EcoRI endonuclease deleted fragment replace KpnI EcoRI cut nt cdna fragment correspond BMV end contain entire obtain PCR involve primer template construct plasmid contain cdna universal recombination vector follow modification introduce pMatNH digest spei endonuclease religate way remove RAS cloning site include restriction site spei create plasmid cut KpnI ecori remove instead nt fragment end position follow RAS cloning site include KpnI MluI BamHI ecorv restriction site ligate pMatNH insert sequence obtain PCR primer template digest KpnI EcoRI prior ligation test recombination activity HCV derive sequence hypervariable region abbreviate HVR sequence abbreviate cdna correspond fragment virus genome obtain RT PCR method clone vector test sequence amplify PCR primer introduce spei restriction site primer primer obtain HVR cDNA respectively PCR product digest spei ligate derivative bear HVR sense antisense orientation identify HVR amplify PCR involve primer introduce MluI ecorv restriction site primer amplify HVR respectively PCR product cut mlui ecorv ligate RAS cloning site previously identify derivative carry HVR sense antisense orientation result plasmid obtain pMatH HVR contain cdna MatH HVR hvr insert sense orientation ii pMatNH HVR contain cdna MatNH HVR possess hvrs HVR antisense HVR sense orientation iii pmath cdna MatH xes sense orientation iv pMatNH contain cdna MatNH carry Xes sense antisense orientation structure confirm sequence test recombination activity BMV mutant previously describe procedure apply infectious BMV genomic RNAs obtain vitro transcription ecori linearize plasmid pMagH pMatNH pMatH pMatH HVR pMatH HVR pMatHN pMatH leaf plant local lesion host BMV mechanically inoculate mixture contain BMV derivative week post inoculation number lesion develop inoculate leaf count establish infectivity test BMV mutant individual local lesion excise total RNA extract separately lesion isolated RNA subject RT PCR involve primer strand primer primer second strand primer specific fragment amplification region recombination crossover occur control identical reaction involve parental transcript positive control water negative control carry RT PCR product analyze electrophoresis agarose gel formation nt short nt product indicate parental recombinant accumulate analyze lesion respectively RT PCR product clone vector sequence determine location recombinant junction site finally presence recombinant select local lesion additionally confirm northern blot analysis main question answer study design vector examine recombination activity RNA sequence vivo result idea arise construct BMV base homomolecular recombination system system test sequence suppose present segment BMV genome new vector possess separately locate RAS cloning site replicable stable infection capable generate viable recombinant selective advantage parental RNA molecule consequently recombinant ought able compete vector insert RASes assume RNA recombination occur accord copy choice mechanism decide dispensable BMV replication good candidate new vector change encode BMV replicase protein strongly affect studied process important question location RASes homomolecular system different segment BMV genome heteromolecular system influence recombination activity examine RNA sequence address issue decide construct call mixed system homo heteromolecular time end molecule prototype new vector carry ras prepare obtain describe early characterize recombinationally active sequence BMV figure nt correspond position insert RAS cloning site antisense sense orientation figure result obtain magh derivative nt portion magh end replace nt sequence represent end fragment encompass entire sequence figure addition marker mutation call DXho introduce derive fragment distinguishable analogous region present end xhoi restriction site disrupt nt insertion GATC Resultant derivative call MatNH MatH unchanged intergenic code region highly modify include come sequence separate nt spacer sequence CCMV MatNH locate sense antisense orientation math sequence sense orientation figure derivative able construct variant mixed system homologous MatH BMV mutant non homologous MatNH BMV mutant recombination study MatH BMV genome compose MatH MatNH BMV genome MatNH table genome BMV mutant copy present derivative MatH MatNH respectively mixed system identical RASes RAS complementary counterpart capable support respectively homologous non homologous heteroduplex mediate recombination different BMV genomic RNAs result directly compare homo heteromolecular recombination system vivo experiment examine presumption concern new recombination vector correct homologous recombination mixed homo heteromolecular system early Nagy Bujarski demonstrate nt portion support homologous recombination heteromolecular system repeat experiment MH BMV mutant recombinant form entire sequence present magh molecule table figure test activity figure white black gray box represent code noncoding recombinationally active sequence respectively sense antisense orientation dash line square encompass replace part modify molecule magh create insert sequence RAS cloning site antisense sense magh orientation construct MatNH MatH nt end magh KpnI ecori site replace nt portion end fragment contain entire construct contain copy sequence MatNH include MatH comprise furthermore marker mutation DXho mark white dot remove XhoI restriction site introduce end MatNH MatH distinguishable analogous region present mixed homologous recombination system previously establish procedure apply plant local lesion host BMV inoculate mixture contain vitro transcribe MatH week infection symptom developed number lesion form inoculated leaf count determine infectivity MatH BMV mutant individual local lesion excise total RNA extract separately progeny accumulate examine lesion selectively amplify RT PCR involve specific primer location Figure reaction product separate agarose gel length determine formation nt dna fragment indicate lesion contain parental recombinant respectively way able determine number lesion viable recombinant generate presence recombinant analyze lesion confirm standard northern blot hybridization dna fragment obtain selective RT PCR amplification clone sequence finally result obtain new mixed homologous recombination system compare analogous datum previously got heteromolecular system Table figure datum present Table indicate exchange magh carry single MatH bear sequence affect infectivity BMV mutant average number lesion appear leave inoculate MH math BMV similar respectively interestingly support homologous crossover heteromolecular system represent MH BMV active mixed system local lesion develop MatH BMV infection accumulate recombinant instead parental math spacer delete indicate crossover occur end present MatH inter intramolecular crossover locate MatH respectively recombinant junction site place identical region location precisely establish datum present till answer molecule exclusively MatH MatH participate recombination heteromolecular system mediate crossover magh permit situation complicated mixed homologous system copy present sense orientation MatH consequently mediate homologous recombination happen accord different scenario figure engage math occur intra intermolecular process involve MatH case recombination mediate present locate math learn accord scenario homologous recombination occur check mutation DXho introduce MatH present recombinant way able determine derive math molecule undertaken analysis reveal mutation present recombinant result suggest homologous crossover preferentially occur math explanation DXho absent large fraction recombinant possible mutation remove MatH homologous recombination recombinant carry single copy recombine examine possibility progeny extract local lesion accumulate MatH lesion recombinant generate analyze nt RT PCR product obtain selective amplification clone sequence analyze clone DXho present test second possibility length cdna clone recombinant contain DXho DXhoR obtain insert vector polymerase promoter resultant plasmid name DXhoR linearization produce infectious DXhoR molecule vitro transcription DXhoR inoculate plant week total RNA extract individual lesion progeny amplify RT PCR obtain product clone sequence describe previously DXho present analyze clone experiment prove homologous recombination MatH present recombinant occur frequently explain recombinant lack dxho altogether result support initial thesis DXho remove homologous recombinant crossover occur MatH show effectively support nonhomologous recombination insert antisense orientation heteromolecular system experiment compose BMV mutant crossover occur local double strand region local heteroduplex capable form order test activity mixed non homologous recombination system plant inoculate MatNH BMV mutant genome compose MatNH week later progeny analyze describe number lesion develop leaf count total RNA extract individual local lesion RT PCR amplification BMV accumulate lesion analyze agarose gel clone sequence presence recombinant confirm standard northern blot result obtain compare analogous datum get heteromolecular system table figure describe previously observe exchange single sequence MatNH sequence influence infectivity BMV mutant average number lesion develop leaf infection BMV MatNH BMV respectively difference recombination activity BMV MatNH BMV fact support non homologous recombination non homologous recombination heteromolecular mixed homo heteromolecular system figure light thick line represent viral genomic rna nascent recombinant RNA respectively red recombinationally active sequence contain additionally box black modify blue insert derive sequence show red box complementary sequence show green box portion recombinant synthesize red portion synthesize sequence green fragment synthesize blue black dot symbolize DXho mutation present MatNH rf recombination frequency dash line square encompass region identical system region crossover occur show detail Heteromolecular system BMV detailed description BMV genome present Figure nascent molecule accumulate BMV infected plant recombinant rf mixed system MatNH BMV copy locate MatNH BMV genome MatNH recombination frequency observe infection MatNH BMV identify recombinant DXho marker DXho marker putative scenario mediate non homologous recombination heteromolecular system owe presence sequence respectively capable form local double strand structure support non homologous crossover detail Figure putative scenario mediate non homologous recombination mixed system presence sequence MatH create opportunity heteroduplex formation MatNH intermolecular pair sequence different MatNH molecule intermolecular MatNH molecule intramolecular recombinant generate BMV replicase initiate nascent strand synthesis end MatNH switch MatNH local double strand region recombinant identify MatNH BMV infection boxed fragment recombine MatNH MatNH MatNH molecule practically identical system DXho mutation present MatNH location junction site mark arrow letter number indicate recombinant type isolate upper case letter refer BMV low case letter refer MatNH BMV equally system recombination event occur similar frequency MatNH BMV respectively recombinant junction site locate region heteroduplexe recombine molecule capable form heteromolecular system type heteroduplex support non homologous crossover possibly form mixed system recombine molecule capable form type heteroduplexe intermolecular MatNH ii intermolecular locate MatHN molecule iii intramolecular locate MatNH molecule figure determine molecule participate non homologous recombination RT PCR amplify check DXho present recombinant result clearly show non homologous crossover involve intramolecular recombination intermolecular recombination MatNH molecule result present indicate homomolecular system provide new interesting datum concern mechanism RNA recombination especially test recombination activity RNA sequence derive RNA base virus consequently attempt construct universal BMV base recombination vector call detail Materials Methods Figure vector modify compose nt end follow RAS cloning site nt spacer RAS cloning site determine infectivity stability new vector plant inoculate mixture contain BMV mutant week number lesion develop inoculate leave count standard analysis progeny RNA carry separate lesion excise total RNA isolate selective RT PCR amplification progeny length RT PCR product establish electrophoresis agarose gel addition reaction product clone sequence demonstrate BMV mutant infectious usually lesion develop leaf Table stable period infection recombination vector order demonstrate effective tool recombination study apply examine recombination activity specific sequence derive HCV genome nt sequence place HCV genome fragment encoding protein name HVR second call sequence constitute nt end HCV genomic RNA show represent conservative fragment HCV genome sequence obtain standard RT PCR method involve viral RNA isolate blood infected patient template amplified fragment insert site different orientation sense antisense sense orientation site detail Materials Methods result different derivative generate MatH HVR possess copy HVR sense orientation HVRs ii MatH copy sense orientation Xs iii MatNH HVR copy HVR sense antisense orientation HVRs hvras iv MatNH copy locate different orientation Xs Xas Figure apply test HVR ability support homologous crossover examine Xs Xas HVRs HVRas capacity induce nonhomologous heteroduplex mediate recombination unlike previously test mutant MatNH MatH BMV MatH MatNH MatH MatNH BMV examine sequence present recombination vector absent genomic RNAs recombination crossover involve molecule determine recombination activity HCV derive sequence plant inoculate BMV mutant MatH HVR BMV MatNH HVR BMV MatH BMV MatNH BMV genome compose newly generate derivative math MatNH MatH MatNH table week standard procedure BMV progeny analysis apply number lesion develop infection count progeny amplify RT PCR method length RT PCR product establish electrophoresis agarose gel clone sequence find BMV mutant carry HVRs HVRs HVRs HVRas sequence infectious BMV usually develop lesion inoculated leaf math MatNH bmv mutant visibly infectious develop lesion leaf respectively despite difference infectivity BMV mutant carry HVRs hvra Xs Xas support non homologous heteroduplex mediate crossover efficiently recombinant generate lesion develop infection MatNH HVR MatNH BMV respectively recombinant junction site locate left portion local double strand region potentially form HVRs HVRas Xs Xas result sequence support non homologous crossover entirely delete spacer figure homologous recombinant generate infection involve MatH HVR BMV fiftyfive percent analyze lesion contain recombinant sequence recombinant HVR spacer delete Figure compose nt derive sequence HVRs follow test recombination activity HCV derive sequence HVR follow derivative prepare MatH copy sense orientation Xs MatNH copy locate different orientation Xs Xas MatH HVR contain copy HVR sense orientation HVRs MatNH HVR copy HVR sense antisense orientation HVRs hvras math MatH HVR apply test HVR ability support homologous crossover MatNH MatNH HVR apply examine HVR competence induce non homologous heteroduplex mediate recombination rf recombination frequency observe infection involve derivative homologous recombinant generate infection involve MatH HVR BMV recombinant form MatH HVR BMV infection recombinationally active sequence HVRs spacer delete compose derive sequence HVRs follow CP code region crossover occur identical region location recombinant junction site precisely presence homologous non homologous recombinant examine lesion confirm RT PCR northern blot analysis figure reveal tendency observe early heteromolecular system BMV accumulate low level lesion contain parental original molecule duplicate figure lane Figure lane change lesion recombinant generate figure lane Figure lane show BMV base heteromolecular system effective tool investigate mechanism homologous non homologous recombination suitable test recombination activity sequence derive BMV overcome problem attempt create new universal recombination vivo system collect datum suggest BMV base homomolecular system well fulfill expectation confirm correctness presumption determine efficacy homomolecular system compare heteromolecular mixed homoheteromolecular system construct study homologous MatH BMV non homologous MatNH BMV recombination mixed system prepare way identical complementary sequence capable support homologous non homologous crossover respectively molecule represent segment BMV genome modify molecule represent different segment BMV genome modify experiment involve MatH MatNH BMV show recombination occur homo heteromolecular system prove effective previously utilize heteromolecular interestingly sequence support homologous recombination infection MH BMV heteromolecular system Infectivity define average number lesion leaf recombination frequency define ratio number lesion develop recombinant total number analyze lesion active mixed system clearly demonstrate primary secondary structure location RAS viral genome affect ability mediate homologous crossover undertaken experiment reveal homologous recombination occur different RNAs non homologous recombination usually involve molecule represent segment BMV genome obtain result constitute piece evidence mechanism homologous nonhomologous recombination different conclusion reach early study influence specific mutation BMV encode protein identify mutation protein inhibit non homologous crossover affect frequency homologous one present difficult judge stage recombination process observe difference occur suppose structural requirement transfer replicase nascent strand complex donor acceptor molecule different homologous nonhomologous recombination case basic factor facilitate process complementarity acceptor nascent strand consequently necessity replication complex stable homologous crossover replicase leave donor template nascent strand end newly synthesize RNA molecule function guide find complementary sequence acceptor RNA hybridize serve primer allow viral replicase reinitiate RNA synthesis non homologous heteroduplexmediate recombination factor enhance crossover interaction donor acceptor formation local double strand region consider BMV genomic RNAs copy spherule intramolecular hybridization locate MatNH likely result present indicate way virus replicate strongly affect recombination process detailed study necessary order explain phenomenon base result obtain heteromolecular mixed system construct new homomolecular crucial element vector test sequence introduce order usefulness system employ test recombination activity distinctly different sequence derive genome RNA virus relate BMV choice nt sequence HVR HCV genome reason sequence deem drastically different important sequence place noncoding region HVR code ii sequence variable HVR variable fragment HCV genome iii unlike HVR sequence possess stable define secondary tertiary structure ascertain introduction HVR vector sense sense antisense sense orientation influence BMV infectivity reduce HVR replace sequence find HVR support homologous recombination HVRs complementary counterpart HVRas mediate non homologous crossover frequency homologous recombination amount non homologous Sequence Xs support homologous crossover Xs complementary sequence Xas capable induce non homologous one frequency reach obtain result testify local double strand structure induce non homologous recombination crossover efficiently reflect capacity RNA virus remove inverted repeat genome virus lack ability easy target double strand RNA induce RNA silence know plant antiviral mechanism datum present suggest sequence adopt compact stable structure able mediate homologous recombination occur efficiently AU rich HVR sequence structure labile dynamic early research homologous recombination BMV lead similar conclusion show homologous recombination occur effectively AU rich region observe highly structure observation concur propose mechanism homologous RNA recombination assume AU rich region facilitate detachment polymerase nascent strand complex donor RNA hand think stability RNA structure make hybridization nascent strand replicase acceptor difficult currently increasingly clear RNA recombination play complex role virus life cycle permit exchange genetic material virus frequent homologous crossover molecule represent segment virus genome stabilize genetic information show homologous non homologous recombination control organization virus genome remove direct inverted repeat affect virus ability replicate accumulate infected cell interestingly observe complementary sequence effectively delete homologous one prevail viral genome probably compact stable structure prevent recombination event altogether datum present prove newly create bmv base homomolecular recombination system examine vivo recombination activity RNA sequence derive genome related unrelated virus factor addition RNA structure affect course studied process specific property viral replicase host protein necessary recombination event equally great importance need create similar universal recombination system virus believe system helpful find general rule RNA recombination provide knowledge indispensable understand new RNA virus retrovirus generate','2005-07-07 00:00:00','http://dx.doi.org/10.1093/nar/gni106',3517,688),('7d624acf891472c3452b1deb89e0dc44a40d0c94.json','GIDEON: a comprehensive Web-based resource for geographic medicine','GIDEON (Global Infectious Diseases and Epidemiology Network) is a web-based computer program designed for decision support and informatics in the field of Geographic Medicine. The first of four interactive modules generates a ranked differential diagnosis based on patient signs, symptoms, exposure history and country of disease acquisition. Additional options include syndromic disease surveillance capability and simulation of bioterrorism scenarios. The second module accesses detailed and current information regarding the status of 338 individual diseases in each of 220 countries. Over 50,000 disease images, maps and user-designed graphs may be downloaded for use in teaching and preparation of written materials. The third module is a comprehensive source on the use of 328 anti-infective drugs and vaccines, including a listing of over 9,500 international trade names. The fourth module can be used to characterize or identify any bacterium or yeast, based on laboratory phenotype. GIDEON is an up-to-date and comprehensive resource for Geographic Medicine.',' world confront generic infectious disease scatter complex fashion country region new day confront health care worker unexpected outbreak epidemic heretofore unknown pathogen name bacteria virus fungus parasite know cause human disease confront anti infective agent vaccine expert work Health Geographics share obvious immediate need comprehensive timely datum status infection globe recent outline GIDEON address use Infectious Diseases clinician review focus Global Health aspect program initiate project design computer system follow disease outbreak pathogen drug initial DOS base program write Paradox floppy disk later evolve compact disk base program Windows commercial web base program eventually release GIDEON Global Infectious Diseases Epidemiology line Gideon Informatics Inc Los Angeles California current version available CD update month web subscription update program consist module diagnosis Epidemiology Therapy Microbiology program module peripheral interest Health Geographics Therapy Microbiology discuss briefly Diagnosis module design generate rank differential diagnosis base sign symptom laboratory test incubation period nature exposure country disease origin figure depict datum entry screen patient suffering fever joint pain follow trip Indonesia low personal note box record additional case datum write user language differential diagnosis list case figure indicate patient suffer Chikungunya appearance disease list indicate user fail enter positive negative finding example fact cough absent reduce likelihood second disease list Mycoplasma infection increase statistical probability Chikungunya point user generate hard copy mail report access table compare clinical feature disease list examine ranking omission specific disease user click specific disease clinical epidemiological datum disease question depict figure differential diagnosis list generate bayesian formula compare product disease incidence symptom incidence compatible infectious disease example number disease know occur Indonesia capable produce fever joint pain statistical likelihood Chikungunya case compute simple bayesian formula follow spreadsheet GIDEON database respectively follow incidence symptom disease incidence disease country clinical case enter GIDEON program identify compatible disease rank relative likelihood determine formula ie blind study case conduct author find correct diagnosis list differential list case rank second study hospitalize patient Boston find correct diagnosis list rank likely inclusion differential diagnosis list important disease rank system bioterrorism option generate differential diagnosis disease associate suspect bioterror scenario figure bioterrorism simulator substitute Indonesia give constellation fever joint pain etc result differential diagnosis list Ebola probability follow Crimean Data entry screen bioterrorism scenario figure Data entry screen bioterrorism scenario Congo hemorrhagic fever probability similar Worldwide option explore world disease consistent give clinical feature access text global status individual disease theory datum entry user monitor server level purpose surveillance example user China enter case fatal pneumonia red flag monitoring agency World Health Organization indicate possible appearance SARS submission specimen reporting case local authority similarly appearance multiple case dysentery user give community indicate possible outbreak shigellosis Epidemiology module present detailed country specific information status disease globally relevant country current version contain million word note datum derive Health Ministry publication peer review journal standard textbook CDC website datum present conference user access graph follow disease incidence rate numerical datum main Epidemiology screen show Figure note user append custom personal note national language font status disease institution note accessible colleague GIDEON local network map depict global distribution disease access distribution tab Figure module main screen Figure Epidemiology module main screen image tab press access thumbnail image Plague maximize copy PowerPoint etc note addition personal note user low right text outline country specific datum disease figure available list country display module click relevant red dot map text box include datum set automatically generate incidence rate graph Figure chronological account disease outbreak numbered reference link relevant journal publication report ongoing outbreak ProMed separate Graphs option allow user generate custom graph compare multiple disease rate rate multiple country figure tab access descriptive epidemiology clinical background disease synonym tab generate list alternative term disease country spanish german norwegian etc historical datum record incidence individual disease significant outbreak span decade additional Fingerprint option generate list disease compatible set epidemiological parameter example Figure parasitic disease transmit fish Japan Therapy module follow pharmacology application drug vaccine Infectious Diseases current version contain generic drug vaccine sub module present mechanism action pharmacology dosage drug drug interaction contraindication spectrum susceptibility testing standard international synonym list contain trade name module user add Epidemiology module Figure Epidemiology module Map depict global distribution plague specific map area expand element copy reproduction necessary country specific note plague appear correspond red dot note language drug vaccine price resistance pattern local trade name etc Microbiology option similar Diagnosis module user enter combination phenotypic test obtain ranked probability list compatible bacteria current version incorporate taxa Microbiology module design list phenotype prior name ecology disease association organism compare phenotype combination organism select user graphic mapping function GIDEON treat individual country unit datum presentation lack certain degree granularity dif ferential diagnosis fever Venezuela include malaria patient live outside endemic southern region problem correct large extent text associated country specific note general knowledge base treat physician theory manufacturer follow incidence disease state district province oblast variability exist accord occupation rural urban setting season etc additional problem relate availability quality valid epidemiological datum disease reporting vary widely country country example AIDS report statistic sub saharan Africa generally inadequate necessary spreadsheet GIDEON record publish true estimate questionable report instance Health Ministry Figure Plague Tanzania click relevant datum set generate incidence rate graph note numbered link journal publication Plague incidence rate Figure Plague incidence rate conflict report World Health Organisation fact record relevant GIDEON country note occasionally major disease report example recent case cholera Japan originate Thailand Thailand officially report single case year possible GIDEON data base rely publish good estimate time educate guess datum entirely lacking publish datum disease incidence Togo program force rely publication neighbor Ghana reader refer GIDEON website extensive listing datum source publish review technical background pricing information graph contrast AIDS rate user select country figure Graph contrast AIDS rate user select country publish Bio Med Central scientist read work free charge','2005-04-22 00:00:00','http://dx.doi.org/10.1186/1476-072x-4-10',962,466),('8309dc172cf91ff580779bbc1394a9692b798a86.json','Screen for ISG15-crossreactive Deubiquitinases','BACKGROUND: The family of ubiquitin-like molecules (UbLs) comprises several members, each of which has sequence, structural, or functional similarity to ubiquitin. ISG15 is a homolog of ubiquitin in vertebrates and is strongly upregulated following induction by type I interferon. ISG15 can be covalently attached to proteins, analogous to ubiquitination and with actual support of ubiquitin conjugating factors. Specific proteases are able to reverse modification with ubiquitin or UbLs by hydrolyzing the covalent bond between their C-termini and substrate proteins. The tail regions of ubiquitin and ISG15 are identical and we therefore hypothesized that promiscuous deubiquitinating proteases (DUBs) might exist, capable of recognizing both ubiquitin and ISG15. RESULTS: We have cloned and expressed 22 human DUBs, representing the major clades of the USP protease family. Utilizing suicide inhibitors based on ubiquitin and ISG15, we have identified USP2, USP5 (IsoT1), USP13 (IsoT3), and USP14 as ISG15-reactive proteases, in addition to the bona fide ISG15-specific protease USP18 (UBP43). USP14 is a proteasome-associated DUB, and its ISG15 isopeptidase activity increases when complexed with the proteasome. CONCLUSIONS: By evolutionary standards, ISG15 is a newcomer among the UbLs and it apparently not only utilizes the conjugating but also the deconjugating machinery of its more established relative ubiquitin. Functional overlap between these two posttranslational modifiers might therefore be more extensive than previously appreciated and explain the rather innocuous phenotype of ISG15 null mice.',' posttranslational modification ubiquitin regulate process proteasomal degradation intracellular trafficking transcription Ubiquitin attach substrate covalent isopeptide linkage terminal fusion ubiquitination reversible ubiquitin moiety release substrate action deubiquitinate protease rescue ubiquitinated substrate degradative fate contrast proteasome associate DUBs enhance rate proteasomal degradation remove bulky poly ubiquitin chain substrate protein prior proteolysis DUBs enhance processivity proteasome target protein recycle ubiquitin modifier turn slowly dub furthermore require hydrolyze ubiquitin precursor generate active ubiquitin monomer inspection mammalian genome show presence gene encode putative dubs consistent specific diverse regulatory function ubiquitin specific protease USPs dominant family DUBs like molecule sequence structural similarity ubiquitin unlike ubiquitination modification UbLs usually target protein destruction proteasome notable exception modifier serve ubiquitin independent signal proteasomal degradation conjugation UbLs target protein follow reaction pathway similar involve ubiquitination enzyme attach cleave UbLs generally distinct ligase protease ubiquitin pathway closely relate homolog ubiquitin vertebrate UbL polypeptide interferon inducible gene product strongly upregulated follow viral bacterial infection molecular regulatory consequence ISGylation remain unknown consist ubiquitin domain tandem arrangement similar unlike member UbL family co opt ubiquitin conjugate enzyme ubiquitin ligase constitute presently appreciate isopeptidase specific absence profound effect innate immunity lead increase resistance certain viral infection notably effect appear contingent proteolytic activity apart additional protease exist precursor protein cleave properly knockout mouse terminal amino acid ubiquitin identical tail region require specific recognition ubiquitin conjugate enzyme recognition ubiquitin adduct isopeptidase overlap conjugation ubiquitin cterminal similarity imply existence promiscuous dubs capable remove ubiquitin substrate protein report identification new protease measure reactivity active site direct probe isopeptide link substrate base profiling DUBs Figure show consensus phylogram base alignment catalytic core sequence DUBs include majority know human USP homolog tree protease cluster close isoform previous work identify protease affinity ubiquitin show reaction electrophilic derivative vinyl sulfone probe additional reactive protease clone express total human DUB homolog different clade phylogram indicate arrow react ubiquitin base probe probe screen base vitro transcription translation IVT clone cdnas afford rapid method generate radiochemically pure protein technique allow generation DUBs readily express bacterial system sequester subcellular compartment multimolecular complex express cell line profile DUB specificity recombinantly express ubiquitin instal electrophilic trap terminus obtain active site probe ubiquitin vinylmethyl ester UbVME su respectively DUBs generate IVT incubate probe figure follow direct analysis reaction mixture SDS PAGE determine ray crystallography probe type form covalent adduct catalytic cysteine residue active dub yield thioether link adduct enzyme probe unmodified IVT product run adjacent sample incubate activity base probe adduct readily detect shift apparent molecular mass react figure follow observation confirm validity assay bona fide isopeptidase display reactivity Figure DUBs react UbVME example show figure previously uncharacterized otubain homolog second negative control SUMO protease form adduct exclusively figure find evidence DUBs evaluate display reactivity probe show presence reactive group clearly sufficient bind active site cysteine protease specificity DUB depend peptide moiety probe contain ubiquitin lastly covalent modification DUBs active site direct probe block pretreatment translate polypeptide sulfhydryl alkylate agent ethylmaleimide NEM figure confirm cysteinedependency adduct formation accordance previous result observe non linear decrease electrophoretic mobility adduct probe mass kda size increase observe DUBs investigate order kda bind correlation abnormal shift initial mass unmodified DUB suggest decrease gel mobility base steric property branched adduct cause covalent modification single dub multiple molecule fact sample set observed size increase dub adduct base SDS PAGE closely match logarithmic equation figure method verify dub modify single probe molecule replace catalytic cysteine position serine residue expect mutation abolish labeling figure assay conduct IVT lysate size increase adduct potentially reflect modification additional factor recombinantly express bacteria pure show abnormal electrophoretic mobility adduct figure Mass spectrometry confirm monovalent modification purify exclude covalent binding additional factor complex Figure collectively experiment establish observed shift apparent molecular mass dub adduct solely consequence unusual electrophoretic behavior underscore uncertainty estimate degree modification target protein UbLs SDS PAGE react efficiently proteasome associate form yeast counterpart significantly high activity bind proteasome activity fact strictly dependent association proteasome show evidence physiological role interaction investigate allosteric activation influence reactivity examine labeling ubiquitin base probe function concentration add purified proteasome negative control evaluate DUB know interaction partner complex vivo anticipate inclusion purify proteasome effect ubvme reactivity Figure contrast observe dose dependent increase adduct formation increase proteasome concentration indicate enhanced activity DUB effect similar magnitude see ubiquitin probe figure recombinant purified form bind electrophilic probe figure detect robust hydrolytic activity ubiquitin amc ubiquitinor link isopeptide fusion protein datum show sequential ultracentrifugation obtain cytosolic fraction enrich proteasome protease fraction efficiently specifically cleave isopeptide link substrate figure absence proteolytic intermediate suggest specific cleavage isopeptide bond addition bait peptide link stable hydrolyzed prolong incubation hour proteasome fraction Proteolysis link peptide substrate inhibit inclusion NEM indicate cleavage cysteine protease analysis reaction support active red arrow depict DUBs bind probe UbVME black arrow indicate DUBs form covalent adduct indicate probe screen represent biochemical proof protease activity Otubainhomolog DUB homolog publication record biochemical function mark asterisk exception bind alkylhalide aldehyde base probe Michael acceptor employ study datum show specific protease proteasome enrich fraction Figure formally exclude possibility undefined enzyme bind consider unlikely protease display mass highly similar furthermore sediment centrifugation hour deubiquitinate enzyme sedimentable level comparable wealth usp find human proteome likely reflect substrate specificity potentially complementation term expression profile subcellular distribution seek analyze intracellular distribution pattern subset crossreactive dubs confocal microscopy analysis genome wide set terminal GFP fusion protein yeast show remarkably altered function subcellular distribution validate choice Cterminal modification anti YFP antibodie utilize tag assay activity DUBs cell lysate clone transiently express USP EYFP construct cell figure lysate EYFP transfecte cell incubate ubiquitin probe assay anti yfp immunoblot analysis figure EYFP construct react probe respective mutant Mutation catalytic cysteine residue serine abolish reactivity UbVME store long period room temperature probe polymerize covalently presumably formation secondary amine bond internal lysine residue reactive Michael acceptor terminus resemble isopeptide link polyubiquitin polymeric probe UbVME likely cause additional high molecular mass adduct observe note small version adduct UbVME dimer maximum electrophoretic mobility similar diubiquitin like complexe absence adduct mutant exclude possibility multiple bind site probe purify recombinant label UbVME result abnormal mobility shift see IVT labeling experiment MALDI TOF mass spectroscopic analysis incubation describe UbVME engage internal polymerization molecular masse consistent tri tetrameric mark spectrogram number superscript monovalently modify result adduct predict size indicate red arrow complex unique mixture contain absent mass spectra component datum show agreement result IVT screen respect subcellular distribution particularly interested expression pattern give isoform display different specificity ubvme labeling experiment EYFP find cell figure upper left panel similar contrast close relative EYFP express mainly nucleus speckled pattern figure upper right panel consistent vivo interaction proteasome observe EYFP predominantly cytoplasm notice fluorescence nucleus figure low left panel demonstrate presence terminal YFP fusion interfere endogenous distribution enzyme analyze EYFP figure low right panel EYFP figure low right panel predict nuclear protein reactivity augment proteasomal association generate IVT activity UbVME analyze presence increase concentration purified human proteasome activity ubvme increase function concentration add purify proteasome mg ml activity remain unaffected Quantification radioactive signal covalently modify bind affinity depict axis percent labeling intensity determine ratio label versus unlabeled ratio absence exogenous proteasome define Figure Proteasome associate specific isopeptidase activity scheme depict UbL peptide conjugate assay isopeptidase activity biotinylated peptide heptamer attach isopeptide linkage hydrolysis isopeptide bond specific dub heptamer release biotin signal lose Incubation proteasomeenriche fraction hr pellet UbL peptide conjugate overnight incubation peptide conjugate completely cleaved result loss biotin signal significant proteolysis occur hour datum show activity sensitive NEM Hydrolysis observe peptide conjugate anti ha immunoblot ha treat subcellular fraction base previous identification electrophoretic mobility dominant reactive DUB hour supernatant enrich uncomplexed protein light moderate size red asterisk hour pellet represent heavy cytosolic complex particular proteasome contain DUB blue asterisk nucleolar protein protein detect expect subcellular compartment combine result indicate specific protease express cell feature propose DUBs observation support notion unlike SUMOylation believe occur nucleus modification affect cellular compartment datum existence multiple reactive DUBs finding strengthen similarity ubiquitin highly active protease represent mammalian DUBs know target exhibit oncogenic potential prostate cancer stabilize substrate Fatty Acid Synthase FAS FAS identify target modification furthermore implicate regulation pathway recently solve structure protease accommodate ubiquitin molecule shallow pan like protrusion base orientation ubiquitin protein structure easily fit catalytic domain datum show apparent steric clash figure interferon alter composition proteolytic proteasome core tailor activity generation peptide MHC complex inspection immune system Interferon treatment result enhanced modification protein factor evolve vertebrate lineage origin coincide adaptive immune system interestingly inhibition proteasome lead rampant accumulation modify substrate know molecular function structural relative modifier destine protein degradation proteasome demonstrate exhibit proteasome associate isopeptidase activity protein destine proteasomal degradation find evidence markedly change modify substrate cell datum show vital protease low catalytic turnover affect overall ubiquitin conjugation recent study suggest complex role inhibit proteasome addition act deubiquitinase close sequence relationship identity similarity contrast functional difference localization protease enzyme provide unique opportunity investigate structural feature contribute ubiquitin versus ubiquitin like specificity characteristic tandem occurrence UBA domain implicate binding ubiquitin result raise possibility UBA domain general interact ubiquitin alternatively multiple lysine act ubiquitination anchor protease responsible depolymerization chain cterminal hydrolase process precursor identify novel specific protease describe potential candidate ubiquitin gene prone duplication insertion lead formation new fusion protein likely emerge approximately million year ago nucleotide stretch polyubiquitin precursor gene encode ubiquitin dimer accidentally insert area genome come control interferon promoter agnostic point view argue simply relevant function moderate absent phenotype knockout mouse fact establish family conjugate deconjugate enzyme relatively low degree conservation specie support view massive expression interferon challenge likely reflect role anti microbial anti viral defense adaptation specific need host immunity demand polymorphism result gene critical immune response paradoxically conserve example cytokine cytokine receptor substantially differ specie interesting note ubiquitin initially report cytokine similar observation ubiquitin like modifier FUBI know Fau MNSFb true factor gain access secretory pathway pay approach conventional perspective vantage point uncover new function ubiquitin cdnas clone human kidney cdna library BioChain Institute cdnas encode human DUBs obtain ATCC cdnas encode fulllength DUBs subclone protein sequence human DUBs obtain National Library Medicine core domain align CLUSTALW algorithm EBI server manually edit Genedoc Nicholas HB Nicholas putative active site cysteine alignment anchor phylogram represent consensus tree base bootstrap iteration calculate Minimum Evolution method default parameter perform TNT Quick Reticulocyte Lysate System kit Promega min mg dna reaction aliquot reaction mix treat RNase mg ml Sigma min incubate probe describe SDS PAGE follow fluorography perform describe synthesis human ubiquitin UbL probe describe IVT product incubate saturate amount individual probe ml IVT lysate preincubation NEM perform min room temperature final concentration mm RNase treatment autoradiogram subject quantification optical density NIH Image software version ratio label versus unlabele IVT product purify human proteasome experiment figure purchase Biomol International inhibit mm coliexpresse human recombinant experiment show figure mL conjugation buffer mM Tris pH mM MgCl mm DTT mM ATP add Boston Biochem mm final GST Boston Biochem mm final biotinylate peptide mer biotin VKAKIQD OH mm final solution mix thoroughly add activate enzyme Boston Biochem nM final Boston Biochem nM final sumo activate enzyme heterodimer nM final nM final solution mixed incubate hour reaction mixture transfer microcentrifuge membrane filter vivascience Da MWCO dilute mL total volume mM Tris pH concentrate dilution concentration procedure repeat time product transfer clean tube dilute mM Tris pH final volume link biotinylated isopeptide detect transfer pvdf membrane Perkin Elmer Streptavidin HRP cell lyse glassbead proteasome enrich fraction retrieve consecutive ultracentrifugation step previously describe proteasome activity inhibit mm mg fraction protein incubate mg terminally ha tag total volume ml hour room temperature detect protease anti ha immunoblotting cleavage assay branch peptide perform mg total protein proteasome enrich fraction ml branched peptide total volume ml ha treat subcellular fraction analyze monoclonal anti ha antibody Immunoblotting perform publish cell maintain DME medium describe construct express transient transfection liposome mediate transfection protocol mg ml cm dish Invitrogen describe cell analyze transfection terminal EYFP fusion protein DUBs generate subclone pegfp Clontech lysate EYFP EYFP transfecte cell prepare incubate activesite probe describe high similarity GFP EYFP fusion protein detect polyclonal anti gfp rabbit serum immunoblotting perform publish experiment perform describe minor modification cell allow attach slide overnight fixation paraformaldehyde min room temperature subcellular localization EYFP fusion protein analyze Perkin Elmer spinning disk confocal microscope Ultraview RS system microscope Nikon inverted unit Nikon DIC lens imaging medium Nikon type immersion oil','2007-07-25 00:00:00','http://dx.doi.org/10.1371/journal.pone.000067',2033,873),('83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json','Bioinformatic mapping of AlkB homology domains in viruses','BACKGROUND: AlkB-like proteins are members of the 2-oxoglutarate- and Fe(II)-dependent oxygenase superfamily. In Escherichia coli the protein protects RNA and DNA against damage from methylating agents. 1-methyladenine and 3-methylcytosine are repaired by oxidative demethylation and direct reversal of the methylated base back to its unmethylated form. Genes for AlkB homologues are widespread in nature, and Eukaryotes often have several genes coding for AlkB-like proteins. Similar domains have also been observed in certain plant viruses. The function of the viral domain is unknown, but it has been suggested that it may be involved in protecting the virus against the post-transcriptional gene silencing (PTGS) system found in plants. We wanted to do a phylogenomic mapping of viral AlkB-like domains as a basis for analysing functional aspects of these domains, because this could have some relevance for understanding possible alternative roles of AlkB homologues e.g. in Eukaryotes. RESULTS: Profile-based searches of protein sequence libraries showed that AlkB-like domains are found in at least 22 different single-stranded RNA positive-strand plant viruses, but mainly in a subgroup of the Flexiviridae family. Sequence analysis indicated that the AlkB domains probably are functionally conserved, and that they most likely have been integrated relatively recently into several viral genomes at geographically distinct locations. This pattern seems to be more consistent with increased environmental pressure, e.g. from methylating pesticides, than with interaction with the PTGS system. CONCLUSIONS: The AlkB domain found in viral genomes is most likely a conventional DNA/RNA repair domain that protects the viral RNA genome against methylating compounds from the environment.',' purpose study identify domain homology alkb viral genome order well understanding distribution possible function domain alkb protein coli probably homologue involve repair alkylation damage DNA RNA repair methyl adenine methylcytosine oxidative demethylation direct reversal methylate base unmethylated form recently protein identify member oxoglutarate oxygenase superfamily catalytic reaction require molecular oxygen Fe oxoglutar ate subsequently convert succinate CO formaldehyde FeII oxygenase superfamily widespread Eukaryotes bacteria currently large know family oxidise enzyme heme group structure oxygenase know share common fold structurally conserve jelly roll sheet core flank helix residue totally conserve structure basically residue involve coordination ion oxoglutarate alkb like gene widespread type organism Archaea bacteria normally alkb homologue multicellular Eukaryotes tend homologue human genome different AlkB homologue abh identify homologue slightly different property respect substrate preference subcellular localisation reason proliferation abh human detailed functional mapping abh carry sequence alignment know abh identify residue totally conserve basically HxD motif RxxxxxR motif residue conserve general FeII oxygenase superfamily describe final residue HxD involve involve coordination final likely involve alkb specific substrate binding addition dna repair show coli AlkB human alkb homologue involve RNA repair express coli alkb reactivate methylate RNA bacteriophage vivo illustrate direct repair important mechanism maintenance RNA living cell RNA repair proceed mechanism dna repair repair damage RNA previously consider unlikely natural redundancy RNAs cell RNA essential cell function unrepaire RNA lead miscoded truncate protein alkylate RNA signal cell cycle checkpointing apoptosis consequently occurrence RNA repair come great surprise mechanism direct reversal methylation alkb homologue particularly important RNA repair mean single strand region repair introduce strand break repair alkylation damage DNA RNA recently review homologue find plant virus suggest methylation host mediate inactivation viral RNAs alkb homologue plant virus counteract defence mechanism detailed study publish research project report focus well understanding distribution potential function putative AlkB homology domain silico mapping virus domain find relate virus general mapping strategy project identify viral genome alkb homology domain identify common feature genome subsequently find additional genome similar feature alkb homology domain datum set analyse property distribution AlkBlike domain virus basis generate hypothesis evolution function domain PSI blast search virus NCBI nr protein sequence database initiate NCBI restrict residue default inclusion threshold value virus alkb domain replicase polyprotein normally consist viral Pfam domain OTU identify subset sequence member processing enzyme family domain find number unrelated protein include non structural protein type ssrna virus OTU mem ber family cysteine protease homologous ovarian tumour otu gene Drosophila member family find Eukaryotes virus pathogenic bacteria MT HEL RdRp domain identify Pfam describe extract library sequence align ClustalX combine new alignment consist domain region turn necessary order robust alignment intervene region conserve domain extremely variable sequence tend confuse alignment program sense conserve region correctly align combine sequence alignment domain Closteroviridae Flexiviridae Tymoviridae input build phylogenetic tree final tree show Figure polyprotein contain alkb like domain indicate second alignment generate sequence alkb like domain region correspond MT AlkB HEL RdRp Pfam domain domain align individually combine alignment input datum set reliable phylogeny datum show separate domain alignment analyse individually compare analysis summarise Tymoviridae measure include SJA comparison random tree SJA value show Table comparison MT HEL RdRp NJ tree standard deviation expectation value tree node correspond value AlkB NJ tree standard deviation expectation value similar range observe ML tree alternative distance measure Symmetric Difference sd measure datum show mean SJA value compare alkb tree MT HEL RdRp tree significantly well random tree show MT HEL RdRp tree clearly similar AlkB tree alignment AlkB domain comparable quality alignment fact AlkB domain high average pairwise sequence identity see table figure actual alignment word AlkB domain similar domain respect sequence identity represent consistent evolutionary history compare domain polyprotein indicate AlkB domain evolve separately domain possibly independent instance degree co evolution analyse compute pairwise distance sequence region alignment MT AlkB HEL RdRp domain describe Figure select result show scatter plot Blosum score value MT domain pair sequence plot score value alkb domain pair sequence plot MT HEL RdRp domain strongly correlate MT RdRp MT HEL HEL RdRp plot AlkB domain domain set sequence show low degree correlation AlkB RdRp AlkB MT AlkB HEL mention genome organisation replicase polyprotein sequence flexible order analyse domain organisation location identify Pfam domain plot number sequence show Figure result describe indicate AlkB domain integrate replicase polyprotein relatively recently Discussion order test potential source select alkb domain compare non viral sequence PSI Blast search NCBI nr database remove viral hit final search report remain scoring hit bacteria include different strain Xanthomonas axonopodi pv citri campestris pv campestris Xanthomonas attack plant citrus bean grapevine rice cotton search return high scoring hit plant pathogen Xylella fastidiosa bacterium infect great variety plant include grapevine citrus periwinkle almond oleander coffee search obviously identify know domain type protein sequence order identify potential similarity region recognise Pfam systematic PSI blast search perform polyprotein region MT HEL domain search NCBI database reference sequence exclude viral entry maximum PSI blast iteration allow inclusion threshold expect homologue AlkBdomain find high confidence value homologue typical viral domain like viral peptidase obviously find viral database entry exclude multiple alignment sequence region correspond alkb domain figure multiple alignment sequence region correspond AlkB domain alignment generate residue involve coordination essential Fe ion completely conserve Vitivirus sequence residue HxD motif single RxxxxxR motif function remain conserve residue unclear involve coordination substrate pairwise distance sequence region correspond methyltransferase MT RdRp alkb domain data point correspond rp rp MT MT distance pair sequence sequence show similar evolutionary distance region fall diagonal pairwise distance estimate multiple alignment score matrix trend line estimate Excel trend line AlkB RdRp heavily influence point represent Foveavirus sequence NCBI identical polyprotein sequence alignment score alkb alignment score RdRp new similarity find search pepper ringspot virus Tobravirus NCBI show significant similarity site specific dna methyltransferase Nostoc sp cytosine dna methylase Bamboo mosaic virus Potexvirus NCBI show similarity aggregation substance Enterococcus faecalis small number additional similarity cause biased sequence property proline rich region probably significant include match mucin cadherin like protein homo sapien multidomain presynaptic cytomatrix protein piccolo Rattus norvegicus general variable region truly variable little similarity protein Pfam domain identify see figure closely relate sequence lack specific domain sense HMMER find significant similarity Pfam entry domain order understand degree sequence variation associate domain loss general sequence variation conserve non conserved region typical polyprotein dot plot generate dot plot Carlavirus sequence Potato virus NCBI Aconitum latent virus NCBI show Figure dot plot confirm sequence closely relate MT HEL RdRp domain significant difference region MT HEL Potato virus lack AlkB domain Aconitum latent virus lack OTU domain see dot plot short region similarity close diagonal show domain present ancestral sequence region show high degree sequence variation indicate dot plot exclusively mutation non essential non functional domain probably rapidly lose particular case typical AlkB motif conserve Potato virus indicate non functional alkb domain terminal domain Flexiviridae Tymoviridae methyltransferase describe Pfam methyltransferase motif Vmethyltransf match putative methyltransferase domain Flexiviridae Tymoviridae despite fact identify PSI blast search start know methyltransferase additional Pfam type profile generate obviously possibility domain Flexiviridae Tymoviridae methyltransferase false positive PSI Blast essential residue typical viral methyltransferase motif conserve alignment domain datum show Bamboo mosaic virus belong Flexiviridae residue identify putative active site residue similarity Sindbis virus like methyltransferase demonstrate region Bamboo mosaic virus methyltransferase activity catalyse transfer methyl group adenosylmethionine AdoMet GTP guanylylimidodiphosphate GIDP correspond sequence position completely conserve alignment Flexiviridae Tymoviridae terminal domain likely significant position total completely conserve alignment mean majority conserve position know essential methyltransferase activity work Hataya et al support assumption sequence region methyltransferase domain likely sequence alkb domain contain functional MT HEL RdRp domain MT Location Pfam domain variable region Flexiviridae sequence Figure Location Pfam domain variable region Flexiviridae sequence region extract directly Pfam output sequence region draw scale black bar end motif indicate fulllength motif find partial motif bar truncated end miss domain probably involve capping genomic subgenomic RNA bioinformatic evidence generate reasonable assume viral alkb domain identify Pfam functional essential residue find oxoglutarate oxygenase conserve particular putative Fe coordinate residue alignment position figure oxoglutarate coordinate position conserve position characteristic alkb homologue fact alkb like domain identify viral genome length compare Pfam profile support hypothesis domain functional Pfam search AlkB domain find subset viral genome subset phylogenetically consistent figure mainly restrict Flexiviridae particular subset Flexiviridae consist Viti Capillo Tricho Fovea Carlavirus subset separated remain Flexiviridae phylogenetic analysis split robust bootstrap analysis family discuss subfamily Flexiviridae split observe Adams et al recent analysis Flexiviridae family AlkB domain find Flexiviridae remain alkb domain find Flexiviridae Closteroviridae general Flexiviridae sequence extra domain addition MT HEL RdRp alkb OTU like cysteine protease peptidase plant virus include survey additional domain Tymoviridae peptidase domain common remain plant virus family include exclude Tymoviridae Flexiviridae additional domain observed distribution alkb domain easily explain assume ancestral AlkB domain integrate genome common ancestor Flexiviridae subfamily subsequent figure dot plot Potato virus NCBI Aconitum latent virus NCBI left sequence show program default similarity threshold right region AlkB OTU peptidase integration slightly low sensitive threshold sequence similarity Pfam region correspond MT magenta alkb red OTU green peptidase blue HEL yellow RdRp cyan domain indicate virus generation derive common ancestor contain alkb domain case domain lose scenario include subsequent transfer small number virus family recombination scenario correct expect different domain polyprotein similar evolutionary history phylogenetic analysis table confirm MT HEL RdRp domain alkb domain indicate AlkB domain co evolve domain relatively recently confirm look degree co evolution analyse compute pairwise distance alignment region represent relevant domain figure case perfect co evolution point fall diagonal case MT HEL RdRp domain plot AlkB domain domain set sequence similar correlation closely relate sequence pair upper right quadrant plot figure degree correlation AlkB RdRp likely explanation alkb domain coevolve domain significant period time rule possibility ancient integration AlkB domain assume ancient viral alkb domain frequently recombine AlkB domain difficult distinguish scenario frequent recombination alkb domain de novo integration net effect property observe see figure range score value generally small AlkB domain RdRp domain particularly exclude couple high scoring case figure caption hand degree sequence variation collection alkb domain significant average sequence identity pairwise alignment position totally conserve consistent recent integration assume different alkb type vector integration detail increase mutation rate integration contribute sequence diversity region move alkb domain novel structural functional context remove original evolutionarily constraint introduce new one create punctuated equilibrium type situation potentially lead rapid evolution introduce significant difference AlkB domain independent evolution domain high mutation rate case region general indi cat Figure MT HEL RdRp domain conserve dot plot large sequence variation intervene region sequence figure conserve AlkB domain OTU domain fact weak sequence similarity domain dot plot indicate sequence originally domain fact similarity weak typical alkb active site motif indicate high mutation rate non essential domain rapidly lose conservation alkb domain strong indication functional mention assume AlkB domain integrate relatively recently de novo integration recombination horizontal gene transfer main driving force spread AlkB domain new genome case large number individual integration lead present situation horizontal gene transfer main driving force initial number integration small easy differentiate situation map Pfam motif variable region MT HEL domain Flexiviridae polyprotein figure show similar domain organisation basically AlkB domain follow OTU domain peptidase domain locate terminal sub sequence relatively constant domain organisation consistent small number initial integration subsequently diffused relate genome homologous recombination fully consistent fact virus alkb domain collect host different location Canada USA Russia Italy Germany France India Taiwan China Japan import virus infect specie transmission insect transport virus significant distance obvious explain observed distribution alkb like domain independent integration mainly closely relate host consider alternative explanation explanation support apparent lack consistent evolutionary relationship alkb domain see Table easy model consistent observe similarity domain organisation Flexiviridae assume region high degree variability expect variability affect localisation integrate domain possible conserve region polyprotein play significant role integration novel domain relevant context preliminary simulation indicate alkb domain tend form independent folding domain folded RNA structure polyprotein RNA Drabløs unpublished datum property possibly facilitate insertion domain viral genome group organism act vector spread virus include bacteria fungus nematode arthropod arachnid plant virus acquire AlkB domain vector host mention search viral alkb domain protein sequence database result mainly bacterial sequence include plant pathogen fastidiosa campestris reasonable possibility alkb domain plant virus originate bacterial mrna possible mrna originate vector host present time easily verify disprove limited number insect plant genome sequence previously suggest viral alkb domain involve protect virus post transcriptional gene silence PTGS system host PTGS know plant intrinsic defence mechanism virus gene silence occur repression transcription transcriptional gene silence mrna degradation PTGS PTGS mechanism plant show similarity RNA interference RNAi animal mechanism result specific degradation RNA degradation activate introduction transgene RNA virus DNA sequence homologous express gene virus develop mechanism counteract PTGS order successfully infect plant suppressor PTGS identify Hc Protease protein Cucumber mosaic virus protein suppress PTGS likely different mechanism alkb like domain find plant virus suppressor PTGS previously report research indicate methylation transcribed sequence connect PTGS methylation mediate direct RNA dna interaction RNA direct dna methylation describe plant lead de novo methylation nearly cytosine residue region sequence identity RNA DNA RNA methylation methylation host protein essential viral replication detrimental virus mention alkb repair methylcytosine oxidative demethylation possible alkb demethylate nucleotide methylate PTGS mechanism help virus overcome major defence mechanism plant show subset plant virus alkb domain virus utilise naturally occur alkb protein host virus rely number host protein order replicate case probably beneficial virus integrate gene genome order ensure accessible trade advantage increase cost maintain large genome alternative hypothesis respect alkb integration consider discuss AlkB domain integrate relatively recently virus find different geographical location obvious connection virus belong subset Flexiviridae source virus point common feature see table give additional file alkb domain find virus associate grapevine apple cherry citrus blueberry usage pesticide common know common pesticide methyl bromide organophosphorus compound cause methylation DNA RNA integrate repair domain methylation damage viral replication complex virus competitive advantage highly methylate environment application pesticide probably stimulate alkb production co infecting bacteria give virus easy access alkb mrna integration RNA genome argue active PTGS system plant similar effect case expect ancient integration alkb domain argue presence alkb domain artefact cause promiscuous viral domain pick available mrna sequence cultivation virus laboratory give large number different laboratory involve number different host datum show unlikely explanation hypothesis environmental compound particular pesticide provoke integration alkb domain viral genome depend high mutation rate frequent integration non viral domain integration recent relative term compare domain genome absolute term compare progress modern agriculture integration frequent sense likely integration happen time different biotope difficult estimate mutation rate RNA virus evolve rapidly difficult assign reliable phylogeny recent study indicate ssrna virus mutation rate close substitution site year SARS virus non synonymous substitution site year consider moderate ssrna mutation rate assume ssrna virus effective mutation rate order magnitude realistic mutation rate virus include like case MT HEL RdRp tree show additional file represent approximately year evolution general NJ tree estimate slightly short evolutionary history year compare ML tree year estimate Ampelovirus sequence include diverge remain alkb contain virus early stage believe alkb integration happen divergence sequence include indicate lack co evolution figure unrealistic assume integration happen year estimate course approximate particular know true mutation rate virus show likely time span alkb integration compatible evolution modern agriculture unfortunately lack robust phylogeny viral alkb sequence sense similar estimate domain generally accept virus frequently use recombination acquire functionality know include nonviral sequence document example particular property ssrna positive strand Pestivirus relevant context biotope pestiviruse cytopathogenic cp noncytopatogenic noncp host infect noncp form convert cp form integration fragment cellular gene viral genome introduce protease cleavage site polyprotein important point happen normal infection process suggest integration facilitated viral polymerase undergo subsequent template switch minus strand synthesis nonreplicative RNA recombination possibility Inte gration cellular sequence observe virus influenza virus show virus efficient mechanism recruitment host gene viral genome recent rapid integration alkb domain select plant virus genome unlikely scenario study focus AlkB domain mainly attempt well understanding potential function associate domain likely additional information integration pattern relative importance de novo integration recombination achieve close investigation variable domain look correlate evolution AlkB domain believe viral alkb like domain conventional repair domain target viral RNA integration alkb domain viral genome provoke environmental methylating agent introduction DNA RNA methylate pesticide farming hypothesis domain interfere PTGS system plant exclude consistent observe feature AlkB Tymoviridae generate ClustalX alignment hmmbuild hmmcalibrate HMMER package visualisation motif position viral sequence generate directly HMMER output file local tool interface GNU groff software systematic large scale search polyprotein subsequence locally PSI blast NCBI reference sequence database dot plot comparison viral protein sequence generate Dotter version','2005-01-03 00:00:00','http://dx.doi.org/10.1186/1471-2164-6-1',2749,814),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json','Comparisons of substitution, insertion and deletion probes for resequencing and mutational analysis using oligonucleotide microarrays','Although oligonucleotide probes complementary to single nucleotide substitutions are commonly used in microarray-based screens for genetic variation, little is known about the hybridization properties of probes complementary to small insertions and deletions. It is necessary to define the hybridization properties of these latter probes in order to improve the specificity and sensitivity of oligonucleotide microarray-based mutational analysis of disease-related genes. Here, we compare and contrast the hybridization properties of oligonucleotide microarrays consisting of 25mer probes complementary to all possible single nucleotide substitutions and insertions, and one and two base deletions in the 9168 bp coding region of the ATM (ataxia telangiectasia mutated) gene. Over 68 different dye-labeled single-stranded nucleic acid targets representing all ATM coding exons were applied to these microarrays. We assess hybridization specificity by comparing the relative hybridization signals from probes perfectly matched to ATM sequences to those containing mismatches. Probes complementary to two base substitutions displayed the highest average specificity followed by those complementary to single base substitutions, single base deletions and single base insertions. In all the cases, hybridization specificity was strongly influenced by sequence context and possible intra- and intermolecular probe and/or target structure. Furthermore, single nucleotide substitution probes displayed the most consistent hybridization specificity data followed by single base deletions, two base deletions and single nucleotide insertions. Overall, these studies provide valuable empirical data that can be used to more accurately model the hybridization properties of insertion and deletion probes and improve the design and interpretation of oligonucleotide microarray-based resequencing and mutational analysis.',' Oligonucleotide microarray powerful technological platform large scale screen common genetic variation disease cause mutation publish study oligonucleotide microarray design screen specific sequence tract megabase length possible single nucleotide substitution exception emphasis place identify possible small insertion deletion heterozygous state crucial detect small insertion deletion play major role inactivate alter gene function disrupt functional element splice junction cis act element open reading frame represent class common genetic variation fundamental approach commonly analyze data set oligonucleotide microarray tailor identify genetic variation specific dna segment purely hybridization approach involve identify statistically significant gain target hybridization signal oligonucleotide probe complementary specific sequence variant theory gain signal approach advantage detect presence genetic variation identify nature sequence change target feasible screen virtually possible insertion deletion overwhelming online version article publish open access model user entitle use reproduce disseminate display open access version article non commercial purpose provide original authorship properly fully attribute Journal Oxford University Press attribute original place publication correct citation detail give article subsequently reproduce disseminate entirety derivative work clearly indicate commercial use contact number mutation specific probe need analysis furthermore little effort systematically access hybridization property probe complementary small insertion deletion second approach involve identify loss hybridization signal perfect match pm probe fully complementary DNA segment interest theory loss signal approach allow screen possible sequence change include insertion deletion cause give target nucleic acid sequence contain mismatch specific PM probe necessitate sequencing specific dna region identify nature sequence change combination gain loss hybridization signal analysis provide robust mean identify characterize mutation non enzymatic oligonucleotide microarray assay analyze specificity reproducibility nucleic acid hybridization oligonucleotide microarray large scale mutational analysis ATM ataxia telangiectasia mutate gene responsible autosomal recessive disorder involve cerebellar degeneration immunodeficiency radiation sensitivity cancer predisposition commonly mutate certain lymphoid malignancy microarray include oligonucleotide probe complementary possible single base substitution insertion base deletion strand ATM code region provide comparative analysis hybridization property substitution insertion deletion probe oligonucleotide microarray base mutational analysis large gene series dna sample derive biopsy lymphoma patient previously screen possible ATM mutation oligonucleotide microarray select total sample show robust amplification signal code exon analysis total unique mutation mixture wild type sequence occur sample impact give mutation single sample minimal give sample wild type sequence region encompass give mutation include analysis single nucleotide polymorphism SNPs present multiple time sample sample sample likewise snp minimal effect global analysis give large number sample basis interrogate study previously describe individual ATM code exon amplify genomic DNA primer contain RNA polymerase tail pool vitro transcribe RNA polymerase create biotinlabele sense antisense strand target respectively fluorescein label reference target genomic DNA unaffected individual reference test sample target fragment dilute hybridization buffer TMA Cl tetramethylammonium chloride TE pH Triton hybridize ATM microarray describe previously microarray stain phycoerythrinstreptavidin conjugate digitize hybridization image reference test target acquire Gene Array Scanner Hewlett Packard Palo Alto equip appropriate emission filter custom software quantify hybridization signal probe subtract background hybridization signal exclusively focus raw datum biotin label test target provide approximately seven time hybridization signal fluorescein label wild type reference target system enhanced signal provide great sensitivity detect weak hybridization sample base potential type mutation substitution base deletion base insertion specificity calculate ratio PM probe hybridization signal wild type target cognate insertion deletion single base substitution probe strand logarithm ratio plot function position gene illustrate special pattern smooth random variation run average datum basis capture variability base sample sample standard deviation calculate datum derive running average basis sample estimate mean hybridization specificity type mutation geometric mean antilog average logged ratio basis specimen calculate table examine variability specificity ratio coefficient variation cv calculate way cv ratio standard deviation divide mean useful understand variability relative magnitude mean typical value intra sample cv cv calculate sample running average ATM base average coefficient variation take inter sample cv basis cv Hybridization specificity ratio define ratio PM probe hybridization signal bright mismatch probe give category global average hybridization specificity ratio base sample give probe type provide determined hybridization specificity ratio average window basis intra inter calculate sample average coefficient variation take calculation move average instead original value goal understand specificity vary basis sample estimate experimental measurement error order determine relative specificity hybridization complex nucleic acid target oligonucleotide probe complementary single base substitution insertion deletion analyze datum generate oligonucleotide microarray base mutational analysis bp ATM code region study pair oligonucleotide microarray Affymetrix Santa Clara contain probe nt length specifically design screen sense antisense strand ATM code region genetic variation collectively ATM sense antisense microarray contain probe complementary possible single base substitution probe complementary possible base insertion probe complementary possible base deletion probe complementary possible base deletion ATM code sequence figure microarray screen sequence variation ATM gene dna sample snp gene inactivate mutation uncover screen localize loss hybridization signal PM probe complementary nt segment ATM code region hybridization datum deletion insertion probe rely analysis datum set provide unique opportunity examine relative hybridization specificity nucleic acid target class mismatch probe order gain global overview hybridization specificity determine average ratio PM probe hybridization signal wild type target Materials Methods cognate insertion deletion single base substitution probe strand Table calculation consider datum interrogate basis dna sample Materials Methods example report ratio pm probe signal signal cognate bp deletion probe single base substitution report ratio pm probe signal cognate substitution probe high hybridization signal provide rigorous assessment cross hybridization single base substitution probe likewise single base insertion probe report ratio pm probe signal cognate insertion probe high hybridization signal sense antisense strand find base deletion probe high average pm cognate MM hybridization specificity ratio fold sense Table provide fine scale analysis hybridization specificity determine relative frequency hybridization specificity ratio define bin similar distribution specificity ratio single base substitution base deletion probe strand figure overall low hybridization specificity single base deletion insertion probe reflect increase frequency probe low specificity bin fold ratio decrease frequency probe high specificity bin fold ratio strand seek uncover underlying trend hybridization specificity different class mismatch probe entire ATM code region give sample intra sample variation provide insight sequence context effect influence hybridization specificity class mismatch probe approach problem plot average hybridization specificity ratio substitution deletion insertion probe basis sample figure Supplementary Figure analyze datum determine run average basis order maximize ability detect trend minimize effect randomly disperse confound factor intra intermolecular secondary structure skew datum give base expect Table figure base deletion probe consistently show high average hybridization specificity ratio follow single base substitution single base deletion single base insertion probe strand exon figure hybridization specificity ratio class mismatch probe fluctuate exon sequence Figure example base deletion probe show peak value unlogged center base trough value unlogged center base sense strand find similar fluctuation specificity ratio mismatch probe type remain ATM code exon Supplementary Figure assess intra sample variability hybridization specificity different mean determine average cv substitution deletion insertion probe give experiment table analyze datum run average basis order maximize ability detect trend maintain consistency datum analysis substitution probe low average intra sample cv sense antisense strand respectively base deletion base deletion insertion probe show comparable intra sample coefficient variation sense strand respectively insertion probe show relatively high variability deletion probe antisense strand couple plot show Supplementary Figure evident mismatch probe type hybridization specificity base substitution probe affect target sequence context intrigue observation search specific target sequence tract produce low hybridization specificity different class mismatch probe approach problem determine mismatch probe run window basis give poor hybridization specificity previously define hybridization specificity ratio table report nucleotide tract probe give base window show poor hybridization specificity ratio comprehensive listing probe poor hybridization specificity provide Supplementary Table sequence tract include homopolymer homopurine homopyrimidine highly represent table close inspection apparent cross hybridization strong probe homopolymeric region sequence context substitution deletion probe form duplex wild type target long bp length example probe design detect single base deletion position design form bp bp duplex wild type target probe form duplex range bp length wild type sense strand target slippage Figure type ambiguity lead increase stability DNA RNA heteroduplexe principle homopurine homopyrimidine tract uncover capacity form high order structure triple helix tract know alter conformation stability RNA DNA heteroduplexe form RNA target dna probe system finally expect ATM target especially rich sequence tract give strand acceptor sequence typically contain homopyrimidine tract code exon include ATM target increase likelihood highly relate sequence tract ATM target cross hybridize probe interrogate particular homopurine homopyrimidine sequence tract reduce overall hybridization specificity region screen potential structure form PM probe list Table target explain poor hybridization specificity Mfold calculate Gibbs free energy intramolecular structure form PM probe target base Gibbs free energy value classify probe target have strong DG kcal mmol medium kcal mmol DG kcal mmol weak kcal mmol potential secondary structure find target probe sequence form substantial secondary structure display figure artificially lower affinity target PM probe low hybridization specificity difficult model intermolecular structure solution phase complex target solidphase oligonucleotide probe appear likely structure similar negative impact hybridization specificity relative variability hybridization specificity ratio sample inter sample variability represent important issue consider resequence analysis uncover general trend inter sample variability type mismatch probe calculate average cv mismatch probe hybridization specificity ratio determine run window basis table interestingly strand single base substitution probe show low inter sample cv base deletion probe show fold high coefficient variation strand relative substitution probe surprisingly base insertion probe show significantly high coefficient variation class mismatch probe sample fact fold high correspond substitution probe strand relative level inter sample variation mismatch probe exon display graphically figure error bar represent standard deviation mean hybridization specificity ratio determine run window basis sample note substitution probe low inter sample variability base deletion base deletion hybridization specificity mismatch probe base run window log hybridization specificity ratio substitution red base deletion green base deletion blue base insertion black plot sense antisense strand ATM exon light red light green light blue gray shaded area represent sd log hybridization specificity ratio substitution base deletion base deletion base insertion probe respectively base insertion probe agreement Table variability hybridization specificity measurement consistent ATM code exon Supplementary Figure analysis indicate average single base insertion probe substantially low reproducibility experiment base substitution base deletion base deletion probe increase inter intrasample variability hybridization specificity single base insertion deletion probe relative single base substitution base deletion probe consider design interpret microarray base screen genetic variation give microarray design substantially control hybridization experiment need determine baseline fluctuation hybridization specificity insertion deletion probe relative substitution probe contrast single nucleotide mismatch detailed thermodynamic analysis double helical nucleic acid bulge nucleotide recently conduct case bulge nucleotide unpaire nucleic acid strand study relevant understand property deletion insertion probe form duplex contain bulge target nucleic acid deletion probe bulge nucleotide locate target strand figure conversely insertion probe contain bulge nucleotide duplex wild type target figure subject sequence context effect duplex contain single base bulge predict stable contain single nucleotide mismatch reflect low average hybridization specificity single base deletion insertion probe relative substitution probe Table figure conversely duplex contain base bulge predict generally stable contain single base mismatch assumption helical stacking interrupt bulge great basis length preserve base bulge high average hybridization specificity ratio base predict intramolecular structure low Gibbs free energy DG base stretch encompass list sequence tract target PM probe complementary sequence tract use DG value predict stability structure DG kcal mmol weak kcal mmol DG kcal mmol medium DG kcal mmol strong type mismatch probe provide poor hybridization specificity ratio low hybridization specificity find sense antisense strand immediately follow end segment sequence probe relative substitution probe agreement predict property probe Table considerably low average inter sample variability substitution probe relative deletion insertion probe unexpected give target hybridize mismatch probe simultaneously experiment source inter sample variation include sample preparation hybridization condition microarray reasonable assume microarray major source variability combinatorial manufacturing process lead roughly equivalent synthesis quality arrayed probe likely insertion deletion probe sensitive subtle change target preparation fragmentation dye incorporation hybridization condition target concentration temperature wash condition substitution probe definitive explanation observation require investigation addition potential value important note caveat rely mismatch probe mutation detection example important screen possible sequence change include multiple base insertion deletion mutational analysis disease relate loci ATM gene give probe base strand need screen insertion length mixed sequence unlikely oligonucleotide complementary insertion base pair represent microarray screen large sequence tract mutation near future deletion represent tenable situation probe base strand need screen deletion give length mixed sequence limitation number deletion probe realistically represent give microarray finally critical precisely determine nature sequence change give sample order properly assess functional significance important consider error rate assign identity mutation base mismatch probe datum deal clinical sample especially important confirm identity','2005-02-18 00:00:00','http://dx.doi.org/10.1093/nar/gni034',2078,575);
INSERT INTO `Paper` VALUES ('8d7400a2b387820cd391d7df8194642e50402a0c.json','Protein secretion in Lactococcus lactis : an efficient way to increase the overall heterologous protein production','Lactococcus lactis, the model lactic acid bacterium (LAB), is a food grade and well-characterized Gram positive bacterium. It is a good candidate for heterologous protein delivery in foodstuff or in the digestive tract. L. lactis can also be used as a protein producer in fermentor. Many heterologous proteins have already been produced in L. lactis but only few reports allow comparing production yields for a given protein either produced intracellularly or secreted in the medium. Here, we review several works evaluating the influence of the localization on the production yields of several heterologous proteins produced in L. lactis. The questions of size limits, conformation, and proteolysis are addressed and discussed with regard to protein yields. These data show that i) secretion is preferable to cytoplasmic production; ii) secretion enhancement (by signal peptide and propeptide optimization) results in increased production yield; iii) protein conformation rather than protein size can impair secretion and thus alter production yields; and iv) fusion of a stable protein can stabilize labile proteins. The role of intracellular proteolysis on heterologous cytoplasmic proteins and precursors is discussed. The new challenges now are the development of food grade systems and the identification and optimization of host factors affecting heterologous protein production not only in L. lactis, but also in other LAB species.',' Lactic Acid Bacteria LAB anaerobic Gram positive bacteria GRAS generally regard Safe status food grade bacteria delivery protein interest foodstuff digestive tract advantage compare know protein producer lactis produce LPS protease Escherichia coli bacillus subtili respectively decade genetic tool model lab Lactococcus lactis develop transformation protocol cloning screening vector mutagenesis system available lactis genome entirely sequence protein expression targeting system design lactis system engineer lactis intra extra cellular production numerous protein viral bacterial eukaryotic origin table produce protein interest fermentor secretion generally prefer cytoplasmic production allow continuous culture simplifie purification use lactis protein delivery vehicle digestive tract human animal secretion preferable facilitate interaction protein enzyme antigen target substrate immune lab like Gram positive bacteria secrete protein synthesize precursor contain terminal extension call signal peptide SP mature moiety protein precursor recognize host secretion machinery translocate cytoplasmic membrane early step SP cleave degrade mature protein release culture supernatant late step secrete protein require subsequent folding maturation step acquire active conformation work describe heterologous protein production recombinant lactococci cellularlocation cytoplasm external medium surface anchor describe work report production give protein different location backbone vector induction level promoter strength allow rigorous comparison production yield cytoplasmic secrete form example different heterologous protein produce lactis secrete cytoplasmic form review discuss major conclusion good production yield observe case secretion fold high cytoplasmic production engineer expre sion cassette enhance secretion efficiency SE proportion total protein detect mature form supernatant result increase overall amount protein lactis able secrete protein range kda kda molecular mass sec dependant pathway altogether observation suggest heterologous protein produce lactis prone intracellular degradation secretion allow precursor escape proteolysis ii conformation protein size predominant feature impair SE new perspective open study heterologous protein production lactis need food grade system well understanding host factor influence heterologous protein secretion lactis example htra mediate proteolysis htra unique housekeeping protease cell surface characterize lactis overcome use htra lactis strain design stable heterologous protein secretion intracellular proteolysis involve clp complex major cytoplasmic housekeeping probably cellular component remain poorly understand discuss genetic tool target give protein different cellular compartment develop reporter protein table staphylococcal nuclease Nuc characterized secrete protein activity readily detectable petri plate assay reporter protein secretion study Gram positive host lactis Nuc develop protein SP screening system Nuc choose develop pCYT psec vector control production lactis cytoplasmic secrete form protein interest respectively Fig pCYT psec plasmid expression control nisin inducible promoter lactis refer NZ strain bear nisr chromosomal cassette require nisin signal transduction case describe protein sample concentration adjust cell density produce culture detail similar induction level lactococcal strain contain pCYT Nuc pSEC Nuc vector high production yield observe secrete Nuc form table similar result obtain constitutive nuc expression cassette cytoplasmic secrete form Nuc heterologous protein high protein yield obtain secrete form similar result obtain production Brucella abortus ribosomal protein abortus facultative intracellular Gram negative bacterial pathogen infect unpublished result Bacteriocins human animal entry digestive tract immunogenic abortus ribosomal protein promising candidate development oral live vaccine brucellosis lactis delivery vector produce lactis pCYT psec vector similarly Nuc production production yield secrete reproducibly significantly high cytoplasmic form table example high protein yield secrete vs cytoplasmic form production human papillomavirus type antigen good candidate development therapeutic vaccine induce cervical cancer protein constitutively produce cervical carcinoma interact cell compound produce cytoplasmic secrete form lactis similar induction level exponential phase culture production protein sample adjust cell density protein quantification perform describe reference western blot ELISA quantify ratio calculate scan western blot signal compare intensity describe correspond reference nd determine high secrete form cytoplasmic form table difference high induction occur late exponential phase intracellular detect trace secrete accumulate culture supernatant production clearly illustrate fact secretion result high yield lactis production ovine interferon omega IFN illustrate observation case poorly immunogenic antigen co delivery immuno stimulator protein enhance immune response host order optimize use lactococci live vaccine production cytokine investigate lactis IFN cytokine able confer resistance enteric virus digestive tract reduction viral penetration inhibition intracellular multiplication virus delivery ovine IFN digestive tract recombinant lactis strain induce anti viral resistance protect enterocyte Ovine IFN cDNA clone pCYT psec plasmid intracellular pCYT IFN secrete psec IFN production respectively induction recombinant IFN IFN strain perform equal level IFN production measure level IFN activity show active form IFN produce strain ii activity IFN find supernatant cell fraction IFN strain fold high observe cytoplasmic form table similarly observe Nuc secretion lead high heterologous protein yield lactis engineer secrete wide variety heterologous protein bacterial viral eukaryotic origin table report secretion bottleneck biotechnological tool heterologous secretion model bacteria Escherichia coli bacillus subtili datum available concern aspect lactis protein size nature SP presence propeptide parameter interfere protein secretion datum available feature compile optimize secretion production yield nature SP parameter modify heterologous precursor previously show Nuc reporter protein replacement native staphylococcal SP nuc homologous lactococcal SP direct secretion Nuc lactis lead increase SE table hand replacement SP Nuc SP enhance SE NucT truncate mature moiety nuc devoid terminal propeptide suggest importance propeptide SE Nuc table case use homologous SP especially SP allow well SE compare heterologous screen vector develop search new homologous secretion signal lactis screening work offer panel sp suitable heterologous secretion compare SP newly describe sp efficient direct secretion Nuc direct mutagenesis new sp identify screening strategy enhance SE low measure SP recent study Lindholm et al show Lactobacillus brevis SP originate Slayer protein drive secretion coli FedF schematic representation Nuc cassette control target production lactis adhesin efficiently SP high SE result good adequacy mature protein SP direct secretion fusion short synthetic propeptide SP mature moiety innovative biotechnological tool enhance protein secretion propeptide compose amino acid residue LEISSTCDA develop show enhance SE heterologous protein lactis nucb nuct table abortus antigen table amylase Geobacillus stearothermophilus table direct mutagenesis experiment demonstrate positive effect LEISSTCDA protein secretion insertion negatively charge residue terminus mature moiety furthermore enhancement effect depend nature SP secretion NucB fuse SP Nuc SP enhance LEISSTCDA insertion strikingly enhancement SE reproducibly accompany overall increase protein yield determine western blot experiment observation suggest heterologous precursor degrade intracellular protease efficiently secrete high secretion way escape proteolysis protein molecular mass range kDa size dsrd Leuconostoc mesenteroide dextransucrase kda size Streptomyces tendae antifungal protein Freitas et al submit successfully secrete lactis suggest protein size bottleneck heterologous protein secretion lactis contrast protein size conformation major problem heterologous secretion illustrate recent example example production non structural protein bovine rotavirus major etiologic agent severe diarrhea young cattle order develop live vaccine virus antigen successfully produce lactis derivative pCYT psec plasmid construct target cytoplasmic extracellular location high level production obtain secrete form secrete detect supernatant SP precursor mature protein detect cell fraction degradation product detect addition precursor mature protein result suggest cytoplasmic form probably totally degrade inside cell fusion SP protect protein intracellular proteolysis similar result obtain pCYT psec vector produce abortus GroEL chaperone protein psec GroEL plasmid obtain subsequently fusion SP GroEL detect western blot experiment Azevedo unpublished datum case abortus GroEL likely interact lactococcal cytoplasmic protein lead severe cellular defect lethal phenotype hand fusion SP Usp GroEL chimeric protein unfolded inactive state allow heterologous production example production bovine lactoglobulin BLG lactis BLG amino acid residue globular protein dominant allergen cow milk produce lactis test immunomodulation allergenic response mouse BLG deliver bacterial vector western blot ELISA show BLG production significantly high BLG fuse SP SE low detectable BLG supernatant psec BLG strain study reveal fusion leiss propeptide BLG enhance SE BLG determine ELISA rotavirus abortus GroEL BLG medium sized compare dsrd low secretion yield absence secretion observe lactis case fusion SP stabilize heterologous protein production efficiently secrete result SP reportedly act intramolecular chaperone protection chimeric precursor intracellular proteolysis cytoplasmic chaperone Sec machinery GroEL cytoplasmic chaperone structural protein BLG globular protein dramatically different primary sequence high affinity intracellular housekeeping protease particular sequence hypothesize fusion SP lead stabilization protein change conformation predominant criterion involve stabilization precursor high yield observe hand protein undergo rapid folding right synthesis interfere hamper secretion process interference protein conformation SE previously show coli subtilis altogether result suggest protein conformation protein size major problem heterologous protein secretion lactis clearly demonstrate secrete form reportedly labile protein stabilize fusion Nuc Nuc reportedly stable protein use fusion partner affect enzymatic activity production result chimerical protein easy follow cytoplasmic form stabilize fusion nuc production induce stationary phase Fig cytoplasmic degrade Fig fusion stable Nuc rescue production lactis allow high protein yield compare stabilization fusion Nuc observe secrete protein Nuc fusion psec backbone result high production yield SE alter Fig fusion synthetic propeptide LEISSTCDA psec LEISS nuc construction restore efficient secretion yield second attempt increase protein yield secrete fusion Nuc psec Nuc result fold increase production yield Fig recent result concern production BLG provide example yield enhancement fusion Nuc psec Nuc BLG construction allow fold increase BLG yield compare psec BLG result Nuc stable carrier protein protective effect labile heterologous chimerical protein reduce sensitivity intracellular proteolysis knowledge Nuc fusion partner commonly test far stabilization lactis bernasconi et al fuse Lactobacillus bulgaricus proteinase prtb BLG subsequently stabilize prtb carrier difficult postulate rule concern stabilization effect different result stabilization observe different partner help determine mechanism stabilization effect biotechnological use recombinant lactis strain protein production fusion facilitate purification tag strategy protein fusion successfully optimize production subunit heterodimeric complex demonstrate murine lactis heterodimeric enzyme coli case result fusion expect property case fusion dramatically interfere conformation protein deleterious expect activity lactis antigen delivery vector fusion envision demonstrate moiety chimerical protein recognize corresponding antiserum immunogenic result mention suggest secretion efficient way escape intracellular proteolysis hypothesis particularly test production degrade intracellular production induce late exponential early stationary growth phase Fig production test clpp deficient strain ClpP reportedly major house keep protease lactis dnak deficient strain dnak intracellular chaperone promote proteolysis maintain protein unfolded state exponential stationary phase culture significant difference pattern observe wild type clpp Fig dnak show strain equally degrade cytoplasm remain unchanged supernatant sample altogether result indicate intracellular proteolysis ClpP dnak independent recently cytoplasmic protease ClpP FtsH identify lactis existence unidentified protease postulate study clpp mutant suppressor useful screening target identify putative lactis protease suggest datum activate stationary phase feature precursor result rise host factor involve protein stability SE Fig research effort focus analysis host factor involve protein production secretion direct random mutagenesis lactis lactis possess wide range enzyme peptidase housekeep protease dedicate intracellular proteolysis possess extracellular housekeeping protease htra major extracel lular scavenger protease prtp plasmid encode plasmidless strain present protease activity medium well production yield expect secretion versus cytoplasmic production result clue provide research worker target protein study intracellular proteolysis protein stability inside outside host strain study lead development htra deficient lactis strain heterologous protein secretion anchor htra deficient strain allow Fusion Nuc rescue intracellular production increase protein yield secrete form Native production wt lactis depend growth phase figure native production wt lactis depend growth phase production secretion analyze western blot culture induce different time hour nisin induction sample harvest exponential od upper panel stationary phase od low panel wt pCYT strain encode native cytoplasmic form wt psec strain encode precursor position mature precursor form give arrow cell lysate supernatant fraction ClpP involve intracellular degradation lactis analysis western blot show strain lactis deficient intracellular protease ClpP rescue cytoplasmic production induced culture sample wt lactis lactis clpp mutant strain contain pCYT clpp pCYT psec clpp psec take panel panel phase stationary phase high protein stability cell surface heterologous protein research work focus host factor affect protein production secretion lactis lactis complete genome sequence analysis reveal Sec machinery comprise few component characterize subtilis Sec machinery notably lactis possess SecDF equivalent complementation lactococcal Sec machinery subtilis SecDF result well secretion yield determined Nuc reporter protein Nouaille et al submit random mutagenesis approach reveal feature cell compartment cell wall play important role secretion process similar approach allow identification characterization gene unknown function specifically involve production yield secrete protein lactis Nouaille et al preparation molecular tool available direct heterologous protein secretion lactis list heterologous protein produce bacterium regularly increase report cytoplasmic secretion production compare secretion allow well protein yield compare intracellular schematic presentation molecular tool cellular event affect production yield heterologous pro tein lactis Figure Schematic presentation molecular tool cellular event affect production yield heterologous protein lactis thickness arrow proportional final production yield host factor involve cellular event identify characterize sp signal peptide encode psec construction nuc fusion protein interest stable Nuc protein production allow well understanding protein production secretion process lactis future work investigate lactis capacity protein modification example show protein require disulfide bond DSB acquire native conformation efficiently produce secrete lactis equivalent coli dsb subtilis bdb gene involve DSB formation find sequence comparison lactis similarly fold element ppiase call maturase identify lactis capacity post translational modification investigate altogether work contribute development improvement new food grade system lactis lead near future construction lactococcal strain dedicate high level production protein interest GRAS status lactis lab general clear advantage use production secretion therapeutic vaccinal protein','2005-01-04 00:00:00','http://dx.doi.org/10.1186/1475-2859-4-2',2120,658),('8f5770d87614bb85a6c7b64c26f1757226137d23.json','Efficient replication of pneumonia virus of mice (PVM) in a mouse macrophage cell line','Pneumonia virus of mice (PVM; family Paramyxoviridae, subfamily Pneumovirinae) is a natural respiratory pathogen of rodent species and an important new model for the study of severe viral bronchiolitis and pneumonia. However, despite high virus titers typically detected in infected mouse lung tissue in vivo, cell lines used routinely for virus propagation in vitro are not highly susceptible to PVM infection. We have evaluated several rodent and primate cell lines for susceptibility to PVM infection, and detected highest virus titers from infection of the mouse monocyte-macrophage RAW 264.7 cell line. Additionally, virus replication in RAW 264.7 cells induces the synthesis and secretion of proinflammatory cytokines relevant to respiratory virus disease, including tumor necrosis factor-α (TNF-α), interferon-β (IFN-β), macrophage inflammatory proteins 1α and 1β (MIP-1α and MIP-1β) and the functional homolog of human IL-8, mouse macrophage inflammatory peptide-2 (MIP-2). Identification and characterization of a rodent cell line that supports the replication of PVM and induces the synthesis of disease-related proinflammatory mediators will facilitate studies of molecular mechanisms of viral pathogenesis that will complement and expand on findings from mouse model systems.',' pneumonia virus mouse pvm infection mouse originally describe Horsfall colleague relatively recently sole interest virus pathogen laboratory rodent colony past year build Horsfall early study develop characterize vivo model severe respiratory virus infection PVM review finding show minimal physiologically relevant inoculum PVM typically pfu result robust virus replication lung tissue accompany influx granulocyte response local production specific proinflammatory chemokine pathophysiology pvm bronchiolitis lead pneumonia acute respiratory distress syndrome ARDS similar observe response severe respiratory syncytial virus hRSV infection human infant pvm clearly replicate efficiently mouse lung tissue vitro propagation pathogen significantly straightforward primate bs epithelial cell line support minimal rate PVM replication vitro bs cell line traditional plaque assay pvm induce plaque develop slowly relatively indistinct border difficult eval uate quantitatively figure furthermore evolutionary perspective prefer perform molecular study virus pathogenesis cell relevant specie rodent cell type cell line demonstrate PVM replicate mouse respiratory epithelial cell line virus growth similarly slow temperature permissive virus propagation vitro work explore PVM replication independent cell line identify mouse macrophage rodent RAW RLE primate bs cell line obtain American Type Culture Collection Manassas VA maintain Iscove Modified Dulbecco medium heat inactivate fetal calf serum mm glutamine penicillin streptomycin co permissive virus growth culture indicate Mouse passage pvm prepare describe store liquid nitrogen pfu ml Virus replication RAW cell determine RT PCR detection virus SH gene reference complete method western blot probe dilution polyclonal anti pvm peptide antibody prepare sequence SQQLN IVDDTPDDDI encode amino acid PVM protein proinflammatory cytokine culture evaluate ELISA Systems Minneapolis MN RT PCR detection interferon standard method primer set ABI Columbia MD normalize describe RNA prepare infected control uninfected cell culture RNazol Friendship cell line evaluate ability support virus replication include rat epithelial RLE mouse epithelial mouse macrophage RAW primate epithelial bs inoculate PVM day MOI pfu cell day virus titer culture supernatant determine standard plaque assay pneumoviruse maintain strict host pathogen specificity vivo determine PVM replicate limited extent vitro pfu ml supernatant aforementioned cell line mouse macrophage raw cell line establish tumor induce Abelson murine leukemia virus generate high virus titer pfu ml culture condition describe cell RAW line support replication unrelated virus pathogen include murine hepatitis virus japanese encephalitis virus evaluate kinetic virus replication production proinflammatory mediator RAW cell line cell confluence inoculate PVM MOI day harvest day raw semi adherent cell line suit plaque assay virus replication examine qualitatively western blot cellular homogenate probe pvm specific antisera figure Virus detect infected culture day post inoculation increase amount day immunoreactive PVM protein detect uninfected control culture Virus replication examine quantitatively RT PCR virus SH gene target sequence figure pvm replication readily detect inoculate raw cell reach copy microgram total RNA day infection copy virus SH gene detect uninfected cell raw cell respond infection pvm produce variety proinflammatory mediator transcription interferon response virus infection detect RT PCR Figure Cytokines TNF detect culture supernatant ELISA figure interestingly prominent mediator detect BAL fluid infected mouse level correlate directly severity pneumoviru disease PVM hrsv infection parallel finding hRSV replicate human monocytic cell line group provide evidence consistent hRSV bovine RSV bRSV replication alveolar macrophage point remain controversial furthermore hrsv infection human monocytic cell line associate production proinflammatory mediator platelet activate factor PAF summary PVM recently emerge useful novel model study respiratory disease mouse provide significant incentive identify tissue culture system virus propagation mouse RAW cell line support efficient replication pvm vitro respond infection augment production cytokine implicate pathogenesis respiratory disease use ex vivo model pvm infection permit study biological response associate virus infection cellular molecular level','2007-06-04 00:00:00','http://dx.doi.org/10.1186/1743-422x-4-48',583,291),('93c6eef32a1a511ee989a259eab0e12174dc6859.json','Correcting errors in synthetic DNA through consensus shuffling','Although efficient methods exist to assemble synthetic oligonucleotides into genes and genomes, these suffer from the presence of 1–3 random errors/kb of DNA. Here, we introduce a new method termed consensus shuffling and demonstrate its use to significantly reduce random errors in synthetic DNA. In this method, errors are revealed as mismatches by re-hybridization of the population. The DNA is fragmented, and mismatched fragments are removed upon binding to an immobilized mismatch binding protein (MutS). PCR assembly of the remaining fragments yields a new population of full-length sequences enriched for the consensus sequence of the input population. We show that two iterations of consensus shuffling improved a population of synthetic green fluorescent protein (GFPuv) clones from ∼60 to >90% fluorescent, and decreased errors 3.5- to 4.3-fold to final values of ∼1 error per 3500 bp. In addition, two iterations of consensus shuffling corrected a population of GFPuv clones where all members were non-functional, to a population where 82% of clones were fluorescent. Consensus shuffling should facilitate the rapid and accurate synthesis of long DNA sequences.',' method automate chemical synthesis oligonucleotide assembly long double strand DNA dsdna sequence PCR LCR enable chemical synthesis gene entire viral genome technological advance help spur formation new field synthetic biology aim define functional unit live organism modular engineering synthetic organism addition demand fully synthetic gene length dna fragment define sequence dramatically increase recent year use application codon optimization construction dna vaccine de novo synthesis novel biopolymer simply gain access know DNA sequence original template unavailable future demand long synthetic dna likely dramatically increase cheap fast synthesize desire sequence obtain mean assembly DNA currently limit presence random sequence error synthetic oligonucleotide arise reaction synthesis incomplete coupling misincorporation etc result error kb deleterious impact error significant desire length synthetic dna increase remarkable assembly PhiX bacteriophage genome bp gel purify synthetic oligonucleotide product contain average lethal error kb result plaque form genome clone functional selection plaque formation require study identify clone correct sequence error reduction correction requirement efficient production long synthetic dna define sequence process sequence multiple clone manual correction error costly time consume method report removal error contain sequence population DNA method rely selective destruction physical separation mismatch contain heteroduplexe Smith Modrich report selective destruction error contain sequence PCR product generate dsdna break overdigestion Escherichia coli mismatch specific endonuclease muthls Gel purification cloning remain length dna result apparent fold reduction error rate PCR product exist approach suited error removal long synthetic dna sequence virtually member population contain multiple error online version article publish open access model user entitle use reproduce disseminate display open access version article non commercial purpose provide original authorship properly fully attribute Journal Oxford University Press attribute original place publication correct citation detail give article subsequently reproduce disseminate entirety derivative work clearly indicate commercial use contact Error correction MutS outline figure population dna molecule contain random error hybridize expose synthesis error mismatch figure duplex contain mismatch remove population affinity capture immobilized muts figure process term coincidence filtering strand duplex match pass filtering step long synthetic dna sequence sequence high error rate coincidence filtering ineffective likelihood strand perfectly match hybridization low generalize MutS error filtering application synthetic DNA synthetic DNA cleave small overlap fragment muts filtering fragment contain mismatch selectively remove absorption immobilized maltose bind protein aquaticus taq MutS fusion protein MBP MutS remain mixture fragment enrich fragment correct sequence serve template assembly PCR produce length product figure process iterate consensus sequence emerge dominant specie population approach equivalent dna shuffle additional mismatch exposure removal step report assemble GFPuv synthetic oligonucleotide apply coincidence filtering consensus shuffle protocol reduce error resultant dna population error rate characterize gene function fluorescence dna sequence provide mathematical model describe error reduction protocol aid prediction parameter influence effectiveness chemical Sigma restriction enzyme Promega New England Biolabs KOD Hot Start DNA Polymerase Novagen Amylose resin NEB catalog Ni NTA resin Novagen catalog ultrafiltration device Millipore catalog Slide Lyzer dialysis membrane Pierce catalog length Taq MutS amplify template pETMutS primer AAA CAT ATG GAA GGC ATG CTG AAG AAT AAG CTT CCC CTT CAT GGT ATC CAA clone hindiii site vector plasmid strain transform pmbp MutS grow od induce mM isopropylb thiogalactopyranoside Cells culture pellete resuspend ml buffer mm Tris HCl pH mM NaCl mM EDTA mm DTT mm phenylmethlysulfonyl fluoride cell suspension sonicate ice insoluble material remove centrifugation min Supernatant apply ml amylose resin pre equilibrate buffer bind MBP MutS wash time ml buffer mm Tris HCl pH mM NaCl store hybridized gene synthesis product fragment error contain fragment precipitate MBP MutS immobilize amylose support error reduce fragment orange blue red reassemble length gene follow PCR amplification generate error reduce product primer black MBP MutS elute ml buffer mm maltose Eluate apply ml Ni NTA resin pre equilibrate buffer Bound MBP MutS wash time ml buffer mm imidazole Bound MBP MutS elute buffer imidazole Eluate concentrate ultrafiltration Amicon Ultra kDa MWCO Concentrated sample dialyze extensively storage buffer mM Tris HCl pH mM NaCl mM EDTA mm DTT Slide Lyzer kDa MWCO cassette Protein concentration determine calculated extinction coefficient cm dialyzed sample dilute equal volume glycerol store Oligonucleotides purchase Qiagen salt free purification sequence pGFPuv GenBank accession base substitution assemble oligonucleotide bp overlap Supplementary Table assembly reaction contain follow component nM oligonucleotide mm dntps mM MgSO buffer ml KOD Hot Start DNA Polymerase Assembly carry cycle min PCR amplification assembly product contain follow component fold dilution assembly reaction mm bp outside primer mm dntps mM MgSO buffer ml KOD Hot Start DNA Polymerase PCR carry cycle min follow final extension min PCR product purify Qiagen QIAquick PCR purification kit elution dh follow speed vac concentration assume error rate duplication KOD DNA polymerase cycle PCR expect introduce mutation assemble GFPuv molecule assemble GFPuv dilute ng ml mM Tris HCl pH mM NaCl heat min follow cool Heteroduplex consensus filtering split pool digest completion nlaiii NEB TaqI NEB NcoI plus XhoI Promega follow manufacturer protocol digest purify Qiagen QIAquick PCR purification kit elution dh Samples pool concentration determine measure MutS binding reaction contain ng ml DNA nM MBP MutS dimer bind buffer mm Tris HCl pH mM NaCl mM MgCl mm DTT glycerol reaction allow incubate room temperature min incubation min equal volume amylose resin pre equilibrate bind buffer protein DNA complex remove low speed centrifugation aliquot supernatant remove subsequent processing Supernatant ml consensus filtering experiment desalt Centri Sep spin column Princeton Separations concentrate purified concentrated dna fragment reassemble aliquot remove vary cycle aliquot assembly reaction resolve agarose gel monitor reassembly process aliquot show predominantly reassemble fulllength GFPuv PCR amplify aliquot supernatant coincidence filtering experiment dilute fold PCR amplify PCR product digest BamHI ecori ligate bp BamHI EcoRI fragment pGFPuv ligation transform fluorescent colony score handheld nm ultraviolet uv lamp preparation substrate consensus shuffle non fluorescent GFPuv clone non fluorescent GFPuv clone pool equal amount nature location mutation clone show Figure GFP code region PCR amplify mixture submit consensus shuffle protocol application MBP MutS error filter create error filter construct fusion protein MBP mismatch bind protein terminal tag MBP MutS MBP MutS overexpresse purify purity Supplementary Figure MBP MutS immobilize amylose resin show selectively retain heteroduplex contain deletion mutation wild type homoduplex Supplementary Figure demonstrate error correction unpurifie oligonucleotide assemble PCR produce bp gene encode green fluorescent protein GFPuv independent preparation GFPuv contain typical gene synthesis error figure Table hybridize subject iteration coincidence filtering consensus shuffling consensus shuffling GFPuv assembly product split pool digest set overlap fragment distinct Type II restriction enzyme figure digest pool subject error filtering add MBP MutS unbound fragment reassemble length product PCR amplify coincidence filtering unbound fulllength GFPuv PCR amplify follow treatment error filter clone error rate estimate score colony fluorescence handheld uv lamp figure actual error rate input consensus shuffle population determine sequence plasmid dna randomly select colony figure result round consensus shuffling increase percentage fluorescent colony table sequence error input consensus shuffle DNA Table dna shuffling traditionally create diversity combinatorial shuffling mutation population dna shuffling create sub population sequence reduction diversity correct fragment recombine produce error free sequence consensus shuffling possible start population dna molecule individual population contain error create new population dominant sequence error free demonstrate nonfluorescent GFPuv clone contain deletion mutation figure pool subject dna shuffle iteration consensus shuffling product clone percentage fluorescent colony monitor indication progress consensus sequence dna shuffle MBP MutS increase percentage fluorescent colony colony total similar previous report round consensus shuffling give new population fluorescent colony total indicate dominant specie likely consensus sequence input population consensus shuffling coincidence filtering protocol effective reduce error synthetic GFPuv population figure case iteration consensus shuffling coincidence filtering increase fluorescent colony average value sequence datum independent experiment show fold reduction error rate consensus shuffle population compare input population give final error rate error kb respectively result demonstrate usefulness MBP MutS error filter consensus shuffling coincidence filtering protocol taq MutS previously show bind deletion mutation high affinity mutation common synthetic dna important note taq MutS low affinity specific point mutation bind weakly homoduplex DNA factor limit stepwise efficiency error filter specific point mutation refractory removal multiple round consensus shuffling round consensus shuffling MBP MutS error filter prove effective reduce deletion transition consistent previous report selectivity taq MutS emphasize synthetic oligonucleotide point mutation generate heteroduplex dna molecule contain unique mismatch PCR amplification hybridization figure Table example transition mutation synthetic oligonucleotide generate heteroduplexes mismatch PCR amplification hybridization consensus shuffling mismatch contain heteroduplexe evade precipitation MBP MutS error filter participate reassembly length GFPuv Table list pair mismatch rise observed transition transversion mutation result MBP MutS error filter effective remove insertion deletion loop mismatch population possible generalize consensus shuffle protocol large number synthetic DNA construct GFPuv choose synthetic construct study advantage fluorescent reporter gene allow easy optimization protocol need sequence thousand base pair DNA expect result report consensus shuffle readily translate synthetic dna construct varied sequence great overall length high initial error kb synthetic DNA construct varied sequence digest define set fragment Type II restriction enzyme fragment desire size range control DNase digestion digestion reassembly large number different gene expect robust protocol dna shuffle broadly apply variety gene sequence synthetic DNA construct large GFPuv expect amenable error correction consensus shuffling error filtering conduct gene fragment reassembly length gene error kb datum present study expect translate large gene similar initial error kb except mutation introduce PCR amplification follow final application error filter synthetic DNA construct high initial error kb expect amenable error correction consensus shuffling construct require digestion small sized gene fragment affect efficiency error correction contrast consensus shuffling increase size synthetic dna product increase error kb preclude use coincidence filtering protocol molecule population contain error proof utility consensus shuffle protocol non fluorescent GFPuv clone contain error figure convert population clone fluorescent important note dna shuffle show improvement percent fluorescent colony example synthetic dna population dna shuffle show improvement percent fluorescent colony figure MutS treatment dna shuffle improve overall number correct sequence small dna population low error rate example shuffle clone unique mutation clone expect fraction correct product close value observe mathematical model describe error rate shuffle error filtering synthetic dna population present estimate parameter consensus shuffling coincidence filtering simple mathematical model Equations construct input population dsdna molecule length contain error base hybridize fragment short dsdna fragment average length error filter reassemble probability fragment length correct sequence determine probability hybridized duplex zero strand error equation estimate probability fragment correct cycle muts filtering apply MutS selectivity factor adjust relative amount mismatch contain duplex account total fraction correct strand hybridized duplex probability obtain error free assembly product give equation consensus shuffle error rate datum figure estimate MutS selectivity factor figure show prediction emerge model assume typical length kb fragment size bp error rate error kb Consensus shuffling predict effective small fragment size figure mention small fragment size obtain control digestion DNase addition multiple iteration muts filtering dramatic result population correct sequence figure model account differ specificity muts type mismatch model predict modest improvement MutS selectivity factor optimization MutS dna bind condition use combination MutS homolog vary mismatch specificity dramatically improve consensus shuffle protocol Figure coincidence filtering predict effective population low error rate clone Figure ineffective majority hybridized duplex contain mismatch demonstrate consensus shuffling coincidence filtering experimental method significantly reduce error synthetic dna consensus shuffling generally applicable error correction synthetic gene typical length error rate iteration consensus shuffling iteration generate population bp reduction error rate allow identification correct clone sequence dna reduced number colony coincidence filtering simple effective procedure reduce error synthetic dna population low error rate clone method significantly increase speed decrease cost production synthetic gene note manuscript review Carr et al independently report application Taq MutS protocol error reduction synthetic dna','2005-03-30 00:00:00','http://dx.doi.org/10.1093/nar/gni053',1911,638),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json','Identification of new participants in the rainbow trout (Oncorhynchus mykiss) oocyte maturation and ovulation processes using cDNA microarrays','BACKGROUND: The hormonal control of oocyte maturation and ovulation as well as the molecular mechanisms of nuclear maturation have been thoroughly studied in fish. In contrast, the other molecular events occurring in the ovary during post-vitellogenesis have received far less attention. METHODS: Nylon microarrays displaying 9152 rainbow trout cDNAs were hybridized using RNA samples originating from ovarian tissue collected during late vitellogenesis, post-vitellogenesis and oocyte maturation. Differentially expressed genes were identified using a statistical analysis. A supervised clustering analysis was performed using only differentially expressed genes in order to identify gene clusters exhibiting similar expression profiles. In addition, specific genes were selected and their preovulatory ovarian expression was analyzed using real-time PCR. RESULTS: From the statistical analysis, 310 differentially expressed genes were identified. Among those genes, 90 were up-regulated at the time of oocyte maturation while 220 exhibited an opposite pattern. After clustering analysis, 90 clones belonging to 3 gene clusters exhibiting the most remarkable expression patterns were kept for further analysis. Using real-time PCR analysis, we observed a strong up-regulation of ion and water transport genes such as aquaporin 4 (aqp4) and pendrin (slc26). In addition, a dramatic up-regulation of vasotocin (avt) gene was observed. Furthermore, angiotensin-converting-enzyme 2 (ace2), coagulation factor V (cf5), adam 22, and the chemokine cxcl14 genes exhibited a sharp up-regulation at the time of oocyte maturation. Finally, ovarian aromatase (cyp19a1) exhibited a dramatic down-regulation over the post-vitellogenic period while a down-regulation of Cytidine monophosphate-N-acetylneuraminic acid hydroxylase (cmah) was observed at the time of oocyte maturation. CONCLUSION: We showed the over or under expression of more that 300 genes, most of them being previously unstudied or unknown in the fish preovulatory ovary. Our data confirmed the down-regulation of estrogen synthesis genes during the preovulatory period. In addition, the strong up-regulation of aqp4 and slc26 genes prior to ovulation suggests their participation in the oocyte hydration process occurring at that time. Furthermore, among the most up-regulated clones, several genes such as cxcl14, ace2, adam22, cf5 have pro-inflammatory, vasodilatory, proteolytics and coagulatory functions. The identity and expression patterns of those genes support the theory comparing ovulation to an inflammatory-like reaction.',' fish low vertebrate post vitellogenic period important completion oogenetic process step follicle enclose postvitellogenic oocyte undergo key event final acquisition ability resume meiosis response maturation induce steroid MIS resumption meiotic process finally release surround follicular layer addition follicle oocyte surround follicular cell undergo progressive differentiation ultimately lead release metaphase oocyte key hormonal molecular event involve control meiosis resumption thoroughly study study dedicate action gonadotropin regulation steroidogenenic event action MIS review associated follicular extra follicular event involve concomitant process oocyte follicular cell cross talk ovulationmechanism receive far attention researchgroup study periovulatory ovarian physiology classical biochemical histological tool later molecular approach study deal ovarian protease participation ovulatory process differential display PCR suppressive subtractive hybridization SSH approach develop order identify new differentially regulate gene fish periovulatory ovary addition numerous candidate gene study perform fish periovulatory ovary apart gene relate hormonal control study dedicate specific gene family TGF beta family connexin finally few study simultaneously analyze expression profile gene belong different family contrast biological process immune response post vitellogenic period benefit genome wide transcriptomic study provide global view molecular event occur post vitellogenic ovary undergo oocyte maturation context present study aim perform transcriptomic analysis postvitellogenic rainbow Oncorhynchus mykiss trout ovary order gene rainbow trout cDNA microarray hybridize RNA sample originate rainbow trout ovarian tissue collect late vitellogenesis post vitellogenesis oocyte maturation statistical analysis perform order identify gene exhibit differential expression period addition supervised clustering analysis perform differentially express gene order identify group cluster gene exhibit similar expression profile furthermore step long term transcriptomic analysis rainbow trout post vitellogenic ovary deliberately choose focus gene cluster exhibit remarkable expression pattern finally specific gene select cluster base novelty putative identity function gene real time PCR analysis ovarian expression profile perform additional ovarian RNA sample investigation conduct accord guide principle use care laboratory animal compliance french european regulation animal welfare year old female rainbow trout Oncorhynchus mykiss obtain reproductive season experimental fish farm Sizun France hold natural photoperiod circulate water system INRA experimental facility Rennes France water temperature keep constant ovary sample individual female late vitellogenesis post vitellogenesis oocyte maturation Oocyte developmental stage assess binocular microscope accord previously describe criterion late vitellogenic sample collect end vitellogenic process approximately week expect ovulation stage germinal vesicle visible polarized cytoplasm area observe post vitellogenic sample collect week later noticeable morphological change yolk structure process meiosis resumption stage oocyte display subperipheral peripheral germinal vesicle germinal vesicle visible dark mass polarize cytoplasm observe Oocyte maturation sample collect meiosis resumption sample collect yolk clarification time germinal vesicle breakdown GVBD tissue collection trout deeply anesthetize phenoxyethanol kill blow head bleed gill arch section ovary dissect body cavity sterile condition ovarian aliquot freeze liquid nitrogen store RNA extraction low oligonucleotide signal low time background level exclude analysis filtering step signal processing perform vector oligonucleotide datum correct spot signal actual dna present spot correction signal normalize divide gene expression value median value array statistical analysis perform order identify differentially express gene late vitellogenic post vitellogenic mature group SAM software statistical analysis perform order compare group one addition comparison perform sample take prior meiosis resumption late post vitellogenic female oocyte maturation comparison low false discovery rate FDR identify differentially abundant gene gene identify comparison keep cluster analysis order characterize expression profile statistically relevant gene supervised clustering analysis datum log transform median center average linkage clustering perform cluster software cluster visualize TREEVIEW software trout sequence originate INRA Agenae USDA est sequence program generate publicly available contig version release January BlastX comparison swiss Prot database January score alignment retrieve perform BlastX comparison addition est spot membrane accession number correspond rainbow trout cluster Unigene Trout January retrieve UniGene database time PCR perform RNA sample include microarray analysis express clone belong select remarkable cluster select accord putative identity function analysis reverse transcription real time PCR perform previously describe briefly μg total RNA reverse transcribe unit Moloney murine Leukemia virus MMLV reverse transcriptase Promega Madison WI μg random hexamer Promega μg total RNA accord manufacturer instruction RNA dNTPs denature min chill ice min reverse transcription master mix add reverse transcription perform hour min follow min incubation step control reaction run MMLV reverse transcriptase negative control real time PCR study real time PCR experiment conduct Cycler IQ Biorad Hercules reverse transcription product dilute μl real time PCR reaction triplicate run RT product real time PCR perform real time PCR kit provide SYBR Green fluorophore euro gentec Belgium accord manufacturer instruction nM primer min incubation step min incubation step amplification perform follow cycle sec min time primer pair relative abundance target cdna sample set calculate serially diluted ovarian cdna pool Cycler IQ software dilution curve ensure PCR efficiency range amplification linear sample set amplification fusion curve obtain follow protocol sec holding follow increase repeat time start level RNA sample measure target gene abundance normalization sample set addition gene identify transcriptomic analysis widely standard gene elongation factor alpha monitor sample set validate normalization procedure GenBank accession number primer sequence show table statistical analysis perform Statistica software Statsoft Tulsa ok difference ovarian development stage analyze non parametric test signal processing clone keep analysis statistical analysis clone find exhibit differential abundance study ovarian stage late vitellogenesis post vitellogenesis oocyte maturation SAM analysis perform false discovery rate FDR low identify clone regulate oocyte maturation exhibit opposite pattern clustering analysis perform expression datum identify clone order characterize expression profile gene clustering analysis clearly separate express gene figure number clone clustering analysis figure keep subsequent table text regulate gene cluster gene cluster figure characterize high expression level late vitellogenesis low level oocyte maturation intermediate variable level post vitellogenesis figure regulated clone cluster gene cluster figure characterize strong expression time meiosis resumption cluster gene cluster figure exhibit low expression late post vitellogenesis regulation meiosis resumption figure rainbow trout Oncorhynchus mykiss genome sequence number characterize rainbow trout protein mrnas limited identity study transcript base significant hit obtain perform blastx search SwissProt database clone belong cluster result blast search present table clone identity good hit SwissProt score value BlastX comparison give similarity search perform EST sequence available public database fully characterized cdnas display code sequence transcript clone spot trout array correspond mrna previously characterize available public database identity clone unambiguous contrast clone good hit SwissProt give significant incomplete information especially true protein family member phylogenetic analysis allow relevant identification gene good hit text clarity reason large cluster clone characterize clear expression time oocyte maturation clone belong UniGene cluster significant hit Swiss Prot table addition clone exhibit significant hit SwissProt clone significant sequence similarity know gene Table cluster gene exhibit inflammation ovulation relate function clone exhibit sequence similarity human chemokine clone claw frog clone coagulation factor clone addition clone exhibit strong sequence similarity human angiotensin convert enzyme precursor clone exhibit strong sequence similarity salmon Oncorhynchus keta vasotocin neurophysin avt isotocin neurophysin respectively finally cluster contain clone exhibit sequence similarity human Forkhead box protein human pendrin know solute carrier family member clone cluster avt gene keep real time PCR analysis small cluster clone characterize expression occur early gene belong cluster clone belong UniGene cluster significant hit SwissProt Table clone similar rat human aquaporin respectively clone belong UniGene cluster addition clone similar mouse serine protease cluster gene keep real time PCR analysis gene select real time PCR analysis similar regulation observe microarray real time PCR experiment observe dramatic expression aromatase clone ovary preovulatory period figure mrna abundance gene oocyte maturation time low late vitellogenesis addition successive decrease gene expression level observe post vitellogenesis oocyte maturation figure mrna abundance vitamin dependent protein precursor gene clone low oocyte maturation late post vitellogenesis contrast significant difference observe late post vitellogenesis figure similar expression profile observe Cytidine monophosphate acetylneuraminic acid hydroxylase cmah gene figure observe strong expression aquaporin gene post vitellogenesis time oocyte maturation figure mrna abundance gene exhibit fold increase post vitellogenesis fold increase oocyte maturation addition mrna abundance pendrin gene exhibit fold increase oocyte maturation significant difference observe late post vitellogenesis similarly vasotocin avt mrna abundance exhibit fold increase time oocyte maturation figure angiotensinconverte enzyme gene expression level exhibit fold increase late vitellogenesis oocyte maturation figure similar profile observe chemokine gene mrna abundance gene exhibit fold increase late vitellogenesis oocyte maturation figure mrna abundance coagulation factor gene exhibit fold increase late postvitellogenesis oocyte maturation mrna abundance exhibit fold increase late post vitellogenesis oocyte maturation figure finally mrna abundance serine protease gene monitor oocyte maturation high late vitellogenic ovary difference significantly different mrna abundance elongation factor alpha translation regulatory protein commonly stable reference exhibit significant difference preovulatory period figure hybridization radiolabele cdnas cdnas deposit nylon membrane decade use nylon cDNA microarray common comparison glass slide microarray technology technology successfully year present study similar cDNA manufacturing hybridization protocol clone generate microarray putatively correspond distinct gene small proportion gene represent distinct clone clone belong UniGene cluster datum noteworthy clone usually find gene cluster clone position clone clustering analysis base correlation profile indicate display similar expression profile addition gene select real time PCR analysis expression observe consistent microarray datum furthermore expression widely reference gene stable preovulatory period observation suggest overall microarray analysis extremely robust reliable present study identify gene exhibit differential expression preovulatory period regulate oocyte maturation exhibit opposite pattern decide step focus anal ysis gene exhibit differential regulation periovulatory period present identity gene belong specific cluster exhibit remarkable pattern transcript choose discuss informative novel gene base identity putative involvement rainbow trout preovulatory ovarian function clone belong cluster clone correspond rainbow trout ovarian aromatase real time PCR study confirm dramatically express preovulatory period observation total agreement exist datum aromatase expression period addition clone putatively encode NADPH cytochrome reductase EC locate cluster aromatase enzyme complex form principal protein component contain catalytic domain bind steroid substrate proximity heme prosthetic group critical activation molecular oxygen subsequent substrate hydroxylation essential component redox partner flavoprotein NADPH cytochrome reductase interestingly present datum transcript exhibit expression rainbow trout preovulatory period confirm identify clone code oxydoreductase protein involve aromatase complex cytochrome gene cytochrome gene exhibit similar expression profile find cluster clone similar rat cytochrome putatively correspond rainbow trout cytochrome cytochrome protein ubiquitous protein associate detoxification organic compound PCB polychlorinated biphenyl PAH polyaromatic hydrocarbon dioxin fish compound able induce gene expression variety tissue rainbow trout immature ovary constitutive expression protein previously report previous present observation suggest relate detoxification activity rainbow trout ovary expression gene observe ovary immediately prior ovulation speculate decrease detoxification activity ovary require beginning ovulation process addition previously show rat glioma cell epoxygenase inhibit prostaglandin production interestingly cell express epoxygenase mrnas convert arachidonic acid epoxyeicosatrienoic acid epoxyeicosatrienoic acid able inhibit activity cyclooxygenase role prostaglandin ovulatory process thoroughly study review rainbow trout prostaglandin able induce vitro ovulation observe regulation gene ovary prior ovulation totally consistent available datum participation prostaglandin ovulatory process present transcriptomic analysis aquaporin clone find cluster real time PCR datum confirm rainbow trout gene exhibit strong expression preovulatory ovary mammal know mercurial insensitive water channel MIWC previously show water permeability strongly increase African claw frog oocyte express MIWC marine fish strong oocyte hydration occur oocyte maturation addition recently show oocyte hydration involve like protein seabream freshwater specie datum oocyte hydration controversial limited butovarian expression profile aromatase vitamin dependent protein proteinS cytidine monophosphate acetylneuraminic acid hydroxylase cmah gene rainbow trout late oogenesis mean SEM figure ovarian expression profile aromatase vitamin dependent protein proteinS cytidine monophosphate nacetylneuraminic acid hydroxylase cmah gene rainbow trout late oogenesis mean SEM ovary sample separate female late vitellogenesis LV post vitellogenesis PV oocyte maturation MAT mrna abundance gene determine real time PCR normalize abundance Abundance arbitrarily set LV stage datum express percentage transcript abundance stage bar share significantly different mrna abundance ovarian expression profile angiotensin convert enzyme coagulation factor CXC chemokine aquaporin pendrin vasotocin avt serine protease elongation fac tor alpha gene rainbow trout late oogenesis mean SEM figure ovarian expression profile angiotensin convert enzyme coagulation factor CXC chemokine aquaporin pendrin vasotocin avt serine protease elongation factor alpha gene rainbow trout late oogenesis mean SEM ovary sample separate female late vitellogenesis LV post vitellogenesis PV oocyte maturation MAT mrna abundance gene determine real time PCR normalize abundance Abundance arbitrarily set LV stage datum express percentage transcript abundance stage bar share significantly different significant hydration observe freshwater specie include rainbow trout datum suggest similarly marine specie oocyte hydration occur oocyte maturation aquaporin mediate freshwater specie rainbow trout addition gene observe dramatic expression gene time meiosis resumption solute carrier family member know sodium independent iodide transporter pendrin expression gene time oocyte maturation dramatic demonstrate real time PCR strong upregulation gene time meiosis resumption stress importance water ion transport rainbow trout preovulatory ovarian function marine specie major oocyte hydration occur ovulation probably important adjust egg buoyancy contrast freshwater specie lay demersal egg rainbow trout hypothesize limited oocyte hydration occur ovulation necessary completion ovulation process increase oocyte volume facilitate rupture follicular wall subsequently release oocyte follicular layer neurophysial hormone arginine vasotocin AVT isotocin fish counterpart argininevasopressin oxytocin respectively Vasotocin precursor isotocin precursor cdnas previously clone fish specie include chum salmon fish AVT involve physiological process include water conservation excretion electrolyte exist datum fish correspond local effect tissue circulate AVT surprisingly observe AVT precursor avt mrna express rainbow trout preovulatory ovary good knowledge evidence non neural expression avt mrna fish addition noteworthy observe similar expression isotocin mrna precursor ovary time oocyte maturation investigation need elucidate role AVT trout preovulatory ovarian function ovulation complex process result release oocyte surround follicular layer early eighty similarity ovulatory inflammatory process thoroughly discuss accept mammalian ovulation inflammatory like reaction fish despite numerous study hormonal control spawning ovulatory process far document mammal ovulation accompany broad spectrum proteolysis implication class protease document review salmonid fish protease identify periovulatory ovary mammal evidence mature ovarian follicle contain proteolytic enzyme include serine protease serine protease implicate ovulatory inflammatory reaction review present study serine protease gene appear progressively regulated preovulatory period knowledge gene expression report periovulatory ovary vertebrate specie speculate protease participate rainbow trout ovulatory process interestingly datum show metalloprotease disintegrin gene sharply regulated time oocyte maturation metalloprotease disintegrin protein family know adam Disintegrin Metalloproteinases think function cell cell interaction proteolysis luminal extracellular protein domain mammal adam family member involve ovulatory process brook trout Salvelinus fontinali metalloprotease activity increase ovary prior ovulation observation suggest participate rainbow trout ovulatory process mammalian CXC chemokine name conserve pattern conserve cysteine residue initially identify potent mediator neutrophil chemotaxis involve chemotaxis monocyte lymphocyte implicate angiogenesis later large variety function mammal CXC describe fish CXC identify exhibit unambiguous orthologue present study show gene expression strongly increase preovulatory period catfish RT PCR datum show gene express wide variety tissue include ovary carp quantitative PCR datum show predominantly express brain despite good conservation vertebrate evolution number study address vivo limited consequence lot information unavailable fish murine model study Crohn disease expression induce inflammation observation suggest gene expression induction contribute inflammatory like event occur rainbow trout time ovulation date participation gene preovulatory ovarian function unsuspected mammal coagulation factor participate coagulation process zebrafish coagulation factor cDNA previously characterize accord author line evidence include biochemical phylogenetic analysis suggest modern coagulation pathway find mammal functional fish furthermore previously show cultured rabbit macrophage able generate factor procoagulant activity present study observe dramatic increase gene expression ovary oocyte maturation significant difference observe late postvitellogenesis observation speculate immediately prior ovulation trout ovary secrete coagulation factor order prevent bleeding ruptured ovarian follicle time ovulation interestingly transcriptomic analysis show transcript exhibit sequence similarity clot factor Clone table express immediately prior ovulation angiotensin convert enzyme ACE cleave Angiotensin Ang form Angiotensin II Ang II angiotensin convert enzyme recently describe ACE homolog ACE zinc dependent peptidase metalloprotease family renin angiotensin system RAS compete ACE capable hydrolyze Ang nonapeptide human gene expression predominantly detect northern blot analysis kidney heart testis addition moderate expression observe tissue include ovary semiquantitative RT PCR wide distribution observe rat tissue mammal previous observation suggest renin angiotensin system functional ovary cattle great expression Ang II observe large follicle addition line evidence support idea Ang II block inhibitory effect theca cell meiosis resumption bovine oocyte brook trout Salvelinus fontinali salmon Ang human Ang II able increase level vitro spontaneous ovulation present transcriptomic study observe dramatic increase gene expression preovulatory period observation confirm real time PCR datum observation suggest dramatic upregulation gene immediately prior ovulation important ovulatory process mammal little know role ovary know mammal function Ang II degrading enzyme form vasodilatator peptide interestingly local vasodilatation key characteristic inflammatory response observe mammalian ovulatory process review hypothesize observed increase gene expression trout preovulatory participate vascular dynamic change putatively occur ovulatory process gene involve synthesis egg component Cytidine monophosphate acetylneuraminic acid hydroxylase CMAH key enzyme synthesis glycolylneuraminic acid salmonid egg cortical alveolus contain polysialoglycoprotein PSGP rainbow trout previously show PSGP contain glycolylneuraminic acid residue present study observe significant decrease cmah gene expression time oocyte maturation presence cmah gene expression ovary totally consistent presence glycolylneuraminic acid rainbow trout cortical alveolus content difficult speculate dynamic psgp accumulation oocyte observation confirm progressive shut estrogen synthesis gene expression occur ovary prior meiosis resumption addition study gene aromatase present work show gene exhibit similar regulation suggest participation preovulatory decrease circulate estrogen level addition observe strong regulation water transport gene preovulatory ovary identity gene consistent recent identification aquaporin mediate mechanism fish oocyte hydration process support recent description limited significant oocyte hydration occur rainbow trout preovulatory ovary finally addition oocyte hydration relate gene observe strong expression gene proinflammatory factor coagulation clot factor vasodilatation factor protease ovary immediately prior ovulation observation suggest similarly theory develop mammal fish ovulation compare ton inflammatory like reaction addition identification gene allow specific study lead well understanding ovulatory process fish future global analysis differentially regulate gene base ontology need satisfyingly describe preovulatory ovarian mechanism addition cellular localization gene expression contribute understanding respective role preovulaory ovarian physiology present study clearly demonstrate distinct steroidogenic proteolytic proinflammatory concomitant event occur preovulatory ovary event concur achieve goal release time ovulation fully competent oocyte ready fertilize','2006-07-27 00:00:00','http://dx.doi.org/10.1186/1477-7827-4-39',2936,861),('95aec306660c1b7519e9f019d92cc3a8206481e4.json','The influence of locked nucleic acid residues on the thermodynamic properties of 2′-O-methyl RNA/RNA heteroduplexes','The influence of locked nucleic acid (LNA) residues on the thermodynamic properties of 2′-O-methyl RNA/RNA heteroduplexes is reported. Optical melting studies indicate that LNA incorporated into an otherwise 2′-O-methyl RNA oligonucleotide usually, but not always, enhances the stabilities of complementary duplexes formed with RNA. Several trends are apparent, including: (i) a 3′ terminal U LNA and 5′ terminal LNAs are less stabilizing than interior and other 3′ terminal LNAs; (ii) most of the stability enhancement is achieved when LNA nucleotides are separated by at least one 2′-O-methyl nucleotide; and (iii) the effects of LNA substitutions are approximately additive when the LNA nucleotides are separated by at least one 2′-O-methyl nucleotide. An equation is proposed to approximate the stabilities of complementary duplexes formed with RNA when at least one 2′-O-methyl nucleotide separates LNA nucleotides. The sequence dependence of 2′-O-methyl RNA/RNA duplexes appears to be similar to that of RNA/RNA duplexes, and preliminary nearest-neighbor free energy increments at 37°C are presented for 2′-O-methyl RNA/RNA duplexes. Internal mismatches with LNA nucleotides significantly destabilize duplexes with RNA.',' understand thermodynamic nucleic acid duplex important reason example knowledge facilitate design ribozyme antisense RNAi oligonucleotide diagnostic probe include employ microarray structure useful nanotechnology modify residue develop application example include propynylated basis peptide nucleic acid phosphoramidate alkyl RNA modification particularly stabilize DNA RNA duplex methyl bridge oxygen carbon ribose form locked nucleic acid lna show Figure McTigue et al show enhanced stability single lna residue dna duplex predict nearestneighbor model hybridization oligonucleotide RNA important application antisense therapeutic diagnostic profile gene expression microarray identify band northern blot gel probe RNA structure oligonucleotide alkyl modification particularly useful application easily synthesize chemically stable bind relatively tightly RNA application desirable modulate bind affinity example sequence independent duplex stability benefit application involve multiplex detection microarray introduction lna methyl RNA oligonucleotide increase stability methyl RNA RNA hybrid duplex enhancement stability usually predict simple model high performance liquid chromatography HPLC perform Hewlett Packard series HPLC reverse phase Supelco column mm Mass spectra obtain LC MS Hewlett Packard series MSD API ES detector AMD thin layer chromatography TLC carry Merck TLC plate mixture aqueous ammonia water synthesize Applied Biosystems DNA RNA synthesizer cyanoethyl phosphoramidite chemistry synthesis standard RNA oligonucleotide commercially available phosphoramidite tertbutyldimethylsilyl group Glen Research synthesis methyl RNA oligonucleotide phosphoramidite Glen Research Proligo phosphoramidite lna nucleotide synthesize accord publish procedure minor modification detail deprotection purification oligoribonucleotide describe previously melt buffer contain mM NaCl mm sodium cacodylate mm Na EDTA pH relatively low NaCl concentration keep melt temperature reasonable range multiple lna substitution Oligonucleotide single strand concentration calculate absorbency single strand extinction coefficient approximate near neighbor model assume RNA RNA strand identical sequence identical extinction coefficient absorbancy versus temperature melting curve measure nm heating rate min Beckman DU spectrophotometer water cool thermoprogrammer melt curve analyze thermodynamic parameter calculate state model program MeltWin sequence dh derive versus ln plot derive average fit individual melting curve expect state model reasonable free energy parameter predict stability methyl RNA RNA methyl RNA LNA RNA duplex Individual Nearest Neighbor Hydrogen Bonding INN HB model obtain multiple linear regression program Analyse Analyse Software Ltd Leeds England com expand Microsoft Excel Analyse obtain parameter enhancement stability methyl RNA RNA duplex substitution lna nucleotide internally end LNAs separate methyl nucleotide result versus ln plot datum calculation typical datum optical melting curve Table list thermodynamic parameter helix coil transition lna nucleotide primarily methyl strand hybrid Watson Crick complementary RNA strand single LNA substitution end heptamer duplex little effect stabilitythe effect single lna substitution end methyl strand study duplex form superscript denote methyl sugar methyl lna sugar denote ribose sugar Watson Crick complement summarize Table terminal lna substitution duplex stability favorable kcal mol average enhancement kcal mol terminal lna substitution increase bind constant duplex formation fold effect single lna substitution end heptamer duplex idiosyncraticthe effect single lna substitution end methyl strand study duplex form Table lna substitution similar effect average lna substitution make duplex stability favorable kcal mol sequence terminal lna methyl strand duplex stability affect little average destabilization mol case terminal precede GC pair orientation GC pair similar destabilization lna substitution terminal LNA substitution interior enhance stability duplex form complementary RNA kcal molthe effect interior position free energy increment single lna substitution methyl RNA study duplex summarize Table single interior lna substitution make duplex stability favorable kcal mol average kcal mol correspond roughly fold increase bind constant interior terminal lna substitution usually improve bind terminal lna substitution table sequence lna substitution make duplex stability favorable kcal mol average enhancement kcal mol enhancement sequence average kcal mol dependence nearest neighbor nucleotide effect substitute study duplex form neighbor precede respectively case lna substitution enhance duplex stability kcal mol seven duplex enhanced stability lna substitution relatively independent near neighbor nucleotide lna exception near neighbor interestingly near neighbor combination destabilize lna substitution terminal table evidently lna substitution middle methyl strand usually affect heteroduplex stability RNA strand lna substitution terminus effect LNA substitution approximately additive lna nucleotide space methyl nucleotide Table contain thermodynamic parameter measure duplex have lna substitution Table compare stability predict simple model model label additivity predict DG duplex formation UGAUGGU series add free energy increment measure single lna substitution context DG duplex formation absence lna nucleotide second model predict DG kcal mol duplex formation following equation deduce fit datum Tables DG MeRNA RNA free energy change duplex formation absence lna nucleotide tL number terminal LNAs iAL UL iGL CL number internal LNAs AU GC pair respectively tU tAL CL GL number inverse melt temperature kelvin gas constant cal mol total oligonucleotide strand concentration strand concentration table thermodynamic parameter duplex formation RNA methyl oligoribonucleotide single lna substitution Oligonucleotides RNA Average curve fit terminal LNAs respectively method use experimental datum DG MeRNA RNA provide reasonable prediction kcal mol measured value Table method use near neighbor model approximate DG MeRNA RNA provide somewhat accurate reasonable prediction describe duplex bad prediction caagcc near neighbor consistent motif unusually unstable kcal mol likely DG equation favorable kcal mol internal near neighbor duplex evidently effect multiple lna substitution approximately additive LNAs space nt datum fit near neighbor model contain LNA enhancement parameter associate duplex RNA strand bind methyl lna chimera parameter list Supplementary Material number occurrence near neighbor limited value roughly determined prediction RNA RNA duplex NaCl approximate stability methyl RNA RNA duplex NaClThe stability RNA RNA duplex NaCl predict Independent Nearest Neighbor Hydrogen Bonding INN HB model model stability RNA RNA duplex approximate DG init free energy change initiate helix DG NN free energy increment jth type near neighbor Table occurrence sequence term AU number terminal AU pair DG termÀAU free energy increment terminal AU pair DG sym kcal mol self complementary duplex non self complementary duplex similar sequence dependent parameter applicable methyl RNA RNA heteroduplexe expect form conformation similar RNA RNA homoduplexe test compare predict stability RNA RNA duplex NaCl measure methyl RNA RNA duplex NaCl predict thermodynamic list parenthesis Tables average RNA RNA duplex NaCl kcal mol phosphate pair stable RNA RNA duplex NaCl presumably difference sequence independent effect salt concentration primarily affect DS duplex formation reasonable approximation term right hand equation note DG sym RNA RNA calculation subtract methyl RNA RNA duplex selfcomplementary backbone differ duplex study number phosphate pair number base pair effect LNA substitution likely dependent salt concentration probable NaCl presence Mg DG RNA approximate DG RNA RNA NaCl table compare measure value duplex lna prediction combine equation measure DG value average kcal mol root mean square difference measure predict DG value kcal mol large difference kcal mol sequence large difference contain near neighbor prediction improve equation correct apparent instability motif result methyl RNA RNA duplex provide preliminary nearest neighbor free energy increment predict stability duplexesThe comparison predict RNA RNA stabilitie measure methyl RNA RNA duplex suggest INN HB model applicable methyl RNA RNA duplex result Tables Table near neighbor represent twice database parameter parenthesis parameter methyl RNA RNA RNA RNA duplex similar especially RNA RNA Watson Crick nearest neighbor parameter favorable kcal mol largely account difference salt concentration suggest evidently term right hand equation approximate table compare prediction combine Equations measured value duplex lna root mean square difference measure predict DG value kcal mol large difference kcal mol associate duplex contain near neighbor undoubtedly model expand refine measurement appear sufficient aid sequence design application complete lna substitution stabilizing substitution nucleotide start second nucleotide endthe effect complete lna substitution methyl RNA backbone study sequence summarize Table stability duplex experimental error measure respectively evidently effective use lna nucleotide space nucleotide lna place second nucleotide end internal mismatch duplex formation favorable table contain thermodynamic parameter measure formation duplex contain single mismatch difference stability relative completely Watson Crick complementary duplex Tables internal mismatch duplex formation favorable kcal mol correspond fold favorable equilibrium constant duplex formation general terminal mismatch destabilize internal mismatch fact terminal make GU pair duplex stabilize kcal mol relative terminal AU pair case effect mismatch lna nucleotide compare equivalent methyl nucleotide case mismatch penalty lna methyl RNA mismatch flank LNAs context LNAs enhance mismatch penalty kcal mol relative completely methyl RNA strand oligonucleotide contain lna discriminate well mismatch flank LNAs oligonucleotide hybridization RNA application range quantify gene expression design therapeutic lna nucleotide characteristic useful purpose example LNA usually stabilize duplex resistant RNA DNA nuclease digestion result present provide insight useful design methyl RNA LNA chimeric oligonucleotide purpose trend general RNA form helix relevant chimera nucleotide favor form conformation result suggest principle design methyl RNA LNA chimera hybridization RNAThe database Tables small magnitude sequence dependence stabilization LNAs surprising Ribose probably methyl ribose sugar single strand typically find roughly equal fraction conformation methylene bridge lna lock sugar conformation expect stabilization preorganization DDG ÀRT ln kcal mol stabilization observe terminal lna roughly mol average stabilization internal LNAs terminal favorable kcal mol respectively stabilization preorganization lna sugar effect saturate alternate sugar LNA evidently lna substitution affect neighboring base pair way enhance stabilization expect preorganization single sugar interestingly NMR structure DNA LNA chimera bind RNA DNA sugar lna drive conformation sequence CTGAT GACUAUACG non terminal dna sugar ts LNAs free energy increment LNA substitution range kcal mol average kcal mol compare range kcal mol average kcal mol single internal lna substitution Table comparision suggest single lna substitution average stabilize methyl RNA RNA duplex DNA dna duplex reflect expectation lna substitution large effect conformation methyl RNA RNA duplex alter conformation DNA dna duplex lna substitution useful probe RNA short methyl RNA oligonucleotide RNA structure probe short oligonucleotide microarray optimize method necessary tight binding sequence independent discriminate mismatch appear lna nucleotide achieve example free energy increment methyl RNA RNA near neighbor range kcal mol correspond UA CG respectively Table average increment kcal mol internal terminal lna help compensate favorable stability AU relative GC pair stability enhancement LNA allow use short oligonucleotide potential disadvantage lna substitution RNA oligonucleotide discrimination mismatch contain lna complete methyl RNA backbone clearly true case direct comparison internal mismatch LNA nucleotide considerably destabilizing average penalty kcal mol Table translate fold weak binding single mismatch LNAs flank mismatch mismatch penalty kcal mol compare kcal mol absence LNAs effect reflect enhanced rigidity LNA prevent mismatch adopt favorable conformation advantageous use LNAs flank nucleotide likely small mismatch penalty','2005-09-09 00:00:00','http://dx.doi.org/10.1093/nar/gki789',1669,503),('9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json','The immunoregulatory and allergy-associated cytokines in the aetiology of the otitis media with effusion.','Inflammation in the middle ear mucosa, which can be provoked by different primary factors such as bacterial and viral infection, local allergic reactions and reflux, is the crucial event in the pathogenesis of otitis media with effusion (OME). Unresolved acute inflammatory responses or defective immunoregulation of middle inflammation can promote chronic inflammatory processes and stimulate the chronic condition of OME. Cytokines are the central molecular regulators of middle ear inflammation and can switch the acute phase of inflammation in the chronic stage and induce molecular-pathological processes leading to the histopathological changes accompanying OME. In this review we present cytokines identified in otitis media, immunoregulatory [interleukin (IL)-2, IL-10, transforming growth factor-beta]) and allergy associated (IL-4, IL-5, granulocyte-macrophage colony-stimulating factor), as crucial molecular regulators, responsible for chronic inflammation in the middle ear and the chronic condition of OME.',' immunoregulatory cytokine interleukin transform growth factor allergy associate cytokine granulocyte macrophage colony stimulate factor GM CSF mediator immune system actively involve regulation molecular cellular process accompany different type inflammation regulate cytokine stimulate primarily cell mediate inflammatory response promote growth proliferation differentiation cell cell natural killer NK cell monocyte macrophage secrete mainly activate cell induce activation rapid clonal expansion mature cell cytokine production cell include interferon growth cell immunoglobulin Ig chain switching secretion IgM cell proliferation production IFN cytolytic activity NK cell proliferation differentiation macrophage precursor cytolytic activity monocyte contrast know cytokine synthesis inhibitory macrophage deactivating factor regulate immune reaction accompany acute inflammation limit duration inflammatory response promote regulate chronic inflammatory process produce variety cell type include cell activate cell activate cell main anti inflammatory activity inhibition cytokine production macrophage neutrophil cell nk cell inhibition macrophage monocyte activation antigen presentation ability cell resolution inflammation acute inflammatory process resolve induce humoral inflammatory reaction immunoglobulin isotype switching cell differentiation cell plasma cell promote switching inflammation chronic stage TGF produce cell platelet monocyte play important role regulation inflammatory process tissue reparation effect TGF depend differentiation state responsive cell TGF participate resolution acute inflammation inhibit antigen presentation cytokine production macrophage suppress proliferation cell nk cell cytokine production activate cell simultaneously TGF recruit monocyte site inflammation upregulate pro inflammatory activity secretion cytokine growth factor particular contradictory effect TGF monocyte macrophage lineage regulation monocyte function deactivation macrophage demonstrate highly coordinated role TGF regulation inflammatory response immunoregulatory allergy associate cytokine produce cell mast cell basophil actively involve regulation different type inflammation regulate humoral inflammatory response induce growth cell isotype switching activate cell differentiation cell antibody produce plasma cell stimulate cell mediate inflammatory process promote growth activate cell enhance development virus specific cytotoxic cell simultaneously possess powerful anti inflammatory effect particular control numerous molecular process monocyte macrophage neutrophil regulate production secretion pro inflammatory cytokine tumour necrosis factor deactivate inflammatory macrophage lead suppression acute inflammation induce switching acute inflammation chronic stage owe ability regulate expression mannose receptor activate macrophage mannose receptor promote fusion activate macrophage formation giant multinucleated cell zone inflammation create cellular background manifestation chronic inflammation important regulator humoral immune response produce cell nk cell late regulating factor differentiation cell play essential role cytokine induce production secretion immunoglobulin support humoral inflammatory reaction production secretion allergic cytokine stimulate production activation differentiation migration eosinophil create strong molecular background eosinophilic inflammation GM CSF regulate proliferation maturation granulocyte macrophage precursor activate mature neutrophil eosinophil macrophage participate different type inflammation Monocytes cell fibroblast endothelial cell activate macrophage cytokine TNF produce GM CSF GM CSF enhance neutrophil phagocytosis enhance release chemotactic factor neutrophil induce production pro inflammatory cytokine macrophage increase function antigen present cell way promote acute inflammatory reaction presence GM CSF initiate specific differentiation monocyte osteoclast like multinucleate giant cell create background chronic inflammation GM CSF contribute allergic inflammation promote growth prolonged survival activation eosinophil basophil group cytokine immunoregulatory TGF allergy associate GM CSF identify otitis medium presence evidence participation regulation local inflammatory process cytokine switch acute phase middle ear inflammation chronic stage lead chronic condition otitis medium effusion OME ome brief description OME common cause childhood deafness developed world chronic inflammatory condition middle ear cleft repetitive recurrence acute middle ear inflammation disease characterize middle ear effusion persist month year clear normal mucociliary transport mechanism usually remove surgical operation surgical procedure include myringotomy aspiration fluid insertion ventilation tube grommet anterior tympanic membrane Fig effective option treatment OME main problem ventilation tube secondary infection post operative otorrhea manage antibiotic eardrop increase antibiotic resistance bacterial pathogen possible ototoxicity topical antibiotic complicate antimicrobial therapy post surgical management otitis medium non surgical approach treatment prophylaxis OME include traditional method combine steroid antimicrobial therapy polyvalent pneumococcal vaccination relatively new method base herbal medicine homeopathy chronic condition otitis media associate proliferative change middle ear tissue especially surface middle ear mucosa present OME modify pseudostratifie epithelium basal cell differentiate goblet cell goblet cell proliferate enhanced secretory activity formation mucus gland occur Fig cell produce secrete mucin important glycoprotein mucociliary transport system middle ear main component middle ear effusion responsible viscous property effusion disease condition alteration occur middle ear eustachian tube mucin metabolism structure mucin glycoprotein glycoconjugate expression cilia goblet cell promote dysfunction normal mucociliary transport system formation effusion middle ear cleft expression mucin detect OME overproduction mucin observe otitis medium membrane bind secrete secrete mucin present effusion vary amount level maybe link level pro inflammatory cytokine tnf promote different level secretion initial step pathogenesis OME inflammatory process middle ear mucosa Bacteria virus allergic reaction gastroesophageal reflux tandem predisposing factor eustachian tube dysfunction cleft palate obstructive adenoid stimulate middle ear inflammation different group inflammatory mediator identify human middle ear mucosa fluid effusion lot different mediator participate initiation early stage middle ear inflammation include arachidonic acid metabolite prostaglandin leukotriene LT LT histamine platelet activate factor surface cell adhesion molecule intercellular adhesion vascular cell adhesion endothelial leukocyte adhesion platelet endothelial cell adhesion molecule soluble cell adhesion molecule soluble intercellular adhesion soluble vascular cell adhesion chemokine rante complement anaphylatoxin interferon cytokine key mediator middle ear inflammation cytokine regulate different stage inflammation responsible resolution inflammation initiate local molecular process lead histopathological change middle ear mucosa submucosa chronic condition otitis medium different group cytokine identify middle ear effusion mucosa pro inflammatory tnf immunoregulatory TGF allergy associate GM pro inflammatory cytokine participation pathogenesis otitis medium discuss related acute phase middle ear inflammation present review analyse immunoregulatory allergy associate cytokine otitis medium provide cellular molecular background chronic inflammation middle ear promote chronic condition immunoregulatory cytokine TGF cellular molecular network OME otitis medium detect middle ear effusion child chronic ome high concentration great pg ml analysis peripheral blood immunologic pattern patient persistent middle ear effusion show following number lymphocyte decrease cell subset ratio reduce proliferative response blood cell diminish generation peripheral blood lymphocyte decrease immunologic investigation child recurrent otitis medium reveal consistent inability adenoidal cell turn cell mature immunoglobulin secrete plasma cell explain defective production adenoid assumption confirm cytokine assay simultaneously nasopharyngeal lymphoid tissue peripheral blood child recurrent otitis adenoidal lymphocyte produce significantly compare patient peripheral blood lymphocyte assume poor production adenoidal tissue probably link deficiency production middle ear tissue associate persistence OME general local deficiency production induce defective immunoregulation middle ear inflammation promote persistence investigation experimental model otitis medium show involvement regulation acute inflammation produce cell predominant acute ome compare chronic OME recombinant trans tympanically inject middle ear normal guinea pig cause inflammatory middle ear effusion clear inoculation differ degree responsiveness effusion production observe follow instillation middle ear vary pronounced middle ear effusion cause severe mixed hearing loss complete lack effusion detect middle ear effusion experimental OME sign sensorineural hearing loss analysis clinical experimental datum propose follow conclusion role involve regulation middle ear inflammation main cell growth differentiation factor support acute chronic inflammatory process middle ear regulate local acute inflammatory response induce activation differentiation cell production proinflammatory anti inflammatory cytokine TGF activate cell Fig deficiency local general production suppress reaction acute inflammation promote persistence OME develop time chronic OME Fig excessive production provoke chronic cell mediate humoral inflammatory process proliferation activate cell increase production cytokine GM CSF proliferate cell immunoglobulin switching igm secretion cell induce switching middle ear inflammation chronic stage subsequent development chronic OME Fig clinical investigation demonstrate involvement chronic otitis medium identify middle ear effusion child undergo tympanostomy tube placement mean concentration pg ml assume presence middle ear effusion cause lack clinical feature acute inflammation lead chronic inflammatory state chronic otitis medium associate dysregulate local production patient chronic otitis medium show high level pg ml produce sinus lavage cell culture simultaneously low production peripheral blood mononuclear cell investigation cytokine profile different experimental model OME show rapid appearance early stage acute otitis medium mucosa produce predominantly cell Streptococcus pneumoniae induce expression mrna middle ear mucosa day inoculation Lipoteichoic acid component present cell wall layer grampositive bacteria non typeable Haemophilus influenzae NTHi induce expression mrna middle ear mucosa fast inoculation interesting note expression mrna follow expression TNF mrnas occur simultaneously pro inflammatory cytokine clinical experimental datum OME lead follow involve regulation middle ear inflammatory response induce bacterial infection expression acute otitis media regulate shortly taneously pro inflammatory cytokine TNF evidence initiation corrective immunoregulation middle ear mucosa main anti inflammatory cytokine regulate local population inflammatory cell macrophage neutrophil production proinflammatory cytokine tnf promote resolution inflammation Fig acute inflammatory response resolve correctly prolong presence zone inflammation provoke amplification chronic humoral inflammatory process increase secretion immunoglobulin especially IgA activate cell possible generation plasma cell presence Fig way contribute switching disease chronic stage deficiency acute otitis medium lead chronic condition OME shortage promote overproduction inflammatory cytokine especially TNF induce tissue damage irre versible change middle ear mucosa Fig conclusion role aetiology OME cytokine participate balanced immunoregulation middle ear inflammatory response regulate cellular molecular event accompany acute inflammation regulate acute inflammatory reaction promote resolution inflammation imbalance production cytokine induce chronic inflammatory process cellmediate humoral stimulate uncontrolle inflammatory relate damage middle ear tissue Figs consider key cytokine mediator regulate switching acute phase middle ear inflammation chronic stage lead chronic condition investigation TGF OME show relation chronic otitis medium identification TGF serous mucoid middle ear effusion associate history previous tympanostomy tube placement investigation growth factor tympanic membrane perforation show TGF promote development fibrotic scar perforation margin explain deficient healing pattern tympanic membrane chronic otitis medium study cytokine experimental OME demonstrate presence TGF acute otitis medium expression TGF mrna regulate acute otitis medium cause pneumoniae NTHi infection detect early course acute inflammation inoculation cytokine TNF high mrna level TGF record considerably later day nthiinfecte pneumococcus infect animal respectively expression continue resolution acute otitis medium TGF regulate host response infectious disease TGF activate kinase TGF smad signal pathway show NTHi major human bacterial pathogen otitis medium strongly induce regulation mucin human epithelial cell induction occur activation toll like receptor dependent mitogen activate protein kinase MAPK pathway activation TGF smad signaling TGF result downregulation MAPK induce mapk subsequent regulation transcription TGF act negative regulator mucin gene expression significant fact understand role TGF otitis medium varying level secrete mucin present middle ear mucosa effusion patient OME analysis datum follow conclusion role TGF TGF participate regulation middle ear inflammation TGF negative regulator acute inflammation middle ear early activation TGF antiinflammatory cytokine course acute otitis medium demonstrate involvement TGF correct resolution acute inflammatory response TGF inactivate macrophage inhibit pro inflammatory cytokine production site inflammation Fig TGF involve regulation bacterial induced mucin production additional TGF contribute pathogenesis persistence chronic ome way allergy associate cytokine GM CSF cellular molecular network identify middle ear effusion child persistent OME atopic child OME undergo myringotomy ventilation tube placement cytokine analysis effusion show high mean level allergy positive group compare allergynegative group high percentage cell express atopic patient OME compare see non atopic patient high level effusion correlate predominance lymphocyte sign chronic inflammation relate atopic background patient OME effusion non atopic patient level low predominance neutrophil demonstrate sign acute inflammatory response investigation adenoid patient recurrent otitis medium result follow important finding expression secretion detect adenoid adenoidal lymphocyte produce level slightly compare patient peripheral blood lymphocyte epsilon germline transcript IgE detect adenoid level IgE epsilon germline transcript expression dependent level mrna expression local induce immunoglobulin class switching IgE adenoid consider essential molecular process contribute chronic inflammation middle ear promote pathogenesis persistence present group patient OME allergic non allergic local overproduction associate predominantly allergic background patient study experimental OME demonstrate involvement regulation different type otitis medium tran tympanically challenge animal expression detect acute chronic otitis medium predominance cell acute form trans tympanical injection soluble receptor prevent allergic eustachian tube dysfunction formation effusion middle ear cleft involvement regulation mucin gene expression secretion show human airway secrete mucin present middle ear mucosa effusion patient chronic ome relationship production mucin course otitis medium investigate ability soluble receptor prevent eustachian tube dysfunction formation effusion middle ear cleft demonstrate potential involvement regulation middle ear mucin metabolism datum allow follow conclusion role involve regulation different type otitis medium cytokine responsible allergic type inflammation owe ability induce local immunoglobulin class switching IgE Fig support local humoral inflammatory response increase production secretion immunoglobulin activate cell Figs support cell mediate chronic inflammation middle ear promote generation multinucleated giant cell Figs necessary note giant cell OME investigate participate regulation mucin metabolism mucociliary transport system middle ear inflammation important regulator cellular molecular process accompany different type different stage middle ear inflammation important factor contribute chronic condition persistence detect middle ear mucosa effusion atopic patient persistent OME expression mRNA mucosa regulate correlate increase expression major basic protein cell marker eosinophil cell marker activate lymphocyte examination cytokine profile middle ear specimen show increase number cell express correlation percentage eosinophil lymphocyte correlation percentage eosinophil level effusion particularly strike OME patient asthma suggest middle ear eosinophilia dependent local general level production percentage cell express effusion atopic patient slightly high compare cell express evidence cytokine contribute development allergic inflammation middle ear overproduction OME associate atopic background patient responsible allergic type inflammation middle ear Fig assumption confirm experimental model OME late phase allergic inflammatory response prevent pretreatment antibodie soluble receptor involvement regulation non allergic inflammatory response middle ear ignore production detect effusion non atopic patient OME significantly compare atopic patient analysis immunoregulatory cytokine experimental model OME demonstrate absence acute otitis medium significant regulation production chronic otitis medium cell numerous presence OME nonallergic patient show involvement regulation non allergic chronic inflammatory reaction middle ear support humoral process increase production secretion immunoglobulin IgA particular Fig consider late mediator middle ear inflammatory response regulate allergic inflammation increase humoral inflammatory process contribute chronic condition knowledge GM CSF OME insufficient involvement GM CSF regulation middle ear inflammation show GM CSF identify effusion acute purulent chronic otitis medium level GM CSF acute otitis medium high compare chronictype otitis medium despite lack experimental datum GM CSF OME cytokine participate regulation different stage middle ear inflammation GM CSF identify acute otitis medium activate matured neutrophil macrophage support acute inflammatory response Fig csf presence induce monocyte differentiation generation mul tinucleate giant cell way create cellular background chronic middle ear inflammation Fig GM CSF eosinophil survivalpromoting cytokine regulate allergic eosinophil mediate inflammation middle ear Fig past year investigation GM CSF show potential regulator allergic inflammation GM CSF induce apoptosis human eosinophil eosinophil receptor sialic acid bind immunoglobulin like lectin GM CSF important regulator middle ear inflammation additional experimental work need clarify exact role cytokine pathogenesis chronic type immunoglobulin IgM IgG IgA secretory IgA IgE identify effusion middle ear fluid chronic ome effusion level igm patient chronic otitis medium markedly elevate mucoid type effusion contain high level IgG IgA Comparison immunoglobulin level measure effusion sera show case effusion level secretory IgA IgE significantly high correspond serum level evidence local overproduction immunoglobulin middle ear immunologic investigation effusion detect immune complex IgG IgG ICs FIG Cellular molecular network allergy associate cytokine lead chronic condition IgA ICs acute chronic otitis medium high level IgG ic find subacute case IgA ic predominant chronic OME immunoglobulin immune complex provide prolongation inflammatory process middle ear presence immunoglobulin chronic ome associate mainly bacterial infection IgG IgA antibodie specific Hemophilus influenzae pneumoniae IgG IgM IgA secretory IgA antibodie specific outer membrane antigen Moraxella catarrhali Staphylococcus aureus bacteria intensely coat secretory IgA IgG antibodie identify chronic effusion type immunoglobulin secretory IgA identify effusion infect respiratory virus presence IgE chronic ome associate local allergic inflammation local overproduction IgE usually accompany local allergic reaction degranulation mast cell find middle ear biopsy specimen expression IgE mast cell detect nasal mucosa specimen patient OME activity immunoglobulin chronic OME evidence chronic humoral inflammatory process middle ear obviously control cytokine correlation level type immunoglobulin immunoregulatory allergy associate cytokine chronic OME investigate presence main type immunoglobulin IgM IgG IgA secretory IgA IgE effusion indirect evidence cytokine TGF involve regulation immunoglobulin production secretion general participate local production secretion immunoglobulin regulate humoral immune reaction course middle ear inflammation fig immunoregulatory cytokine allergyassociate cytokine consider key regulator middle ear inflammation responsible molecular cellular background chronic immunoregulatory cytokine TGF initiate support molecular switching acute phase inflammation chronic stage allergy associate cytokine GM CSF probably provide molecular cellular background chronic humoral cell mediate allergic inflammatory process middle ear lead chronic condition necessary note additional experimental work need order elucidate molecular mechanism cytokine regulation middle ear inflammation analyse possibility anti cytokine therapy clinical treatment OME','2004-04-30 00:00:00','http://dx.doi.org/10.1080/0962935041000168847',2675,632),('9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json','The Waiting Time for Inter-Country Spread of Pandemic Influenza','BACKGROUND: The time delay between the start of an influenza pandemic and its subsequent initiation in other countries is highly relevant to preparedness planning. We quantify the distribution of this random time in terms of the separate components of this delay, and assess how the delay may be extended by non-pharmaceutical interventions. METHODS AND FINDINGS: The model constructed for this time delay accounts for: (i) epidemic growth in the source region, (ii) the delay until an infected individual from the source region seeks to travel to an at-risk country, (iii) the chance that infected travelers are detected by screening at exit and entry borders, (iv) the possibility of in-flight transmission, (v) the chance that an infected arrival might not initiate an epidemic, and (vi) the delay until infection in the at-risk country gathers momentum. Efforts that reduce the disease reproduction number in the source region below two and severe travel restrictions are most effective for delaying a local epidemic, and under favourable circumstances, could add several months to the delay. On the other hand, the model predicts that border screening for symptomatic infection, wearing a protective mask during travel, promoting early presentation of cases arising among arriving passengers and moderate reduction in travel volumes increase the delay only by a matter of days or weeks. Elevated in-flight transmission reduces the delay only minimally. CONCLUSIONS: The delay until an epidemic of pandemic strain influenza is imported into an at-risk country is largely determined by the course of the epidemic in the source region and the number of travelers attempting to enter the at-risk country, and is little affected by non-pharmaceutical interventions targeting these travelers. Short of preventing international travel altogether, eradicating a nascent pandemic in the source region appears to be the only reliable method of preventing country-to-country spread of a pandemic strain of influenza.',' emergence pandemic strain influenza consider inevitable provide emerge strain virulent possible eliminate nascent influenza pandemic source region combination target antiviral prophylaxis pre vaccination social distancing quarantine early elimination source region achieve delay local epidemic country effect highly value end country consider introduce non pharmaceutical intervention border screening promote early presentation case arrive passenger require use personal protective equipment travel wearing mask reduce traveler number case believe measure stop importation epidemic overseas argue strongly SARS influenza extent intervention delay local epidemic currently quantify considerable interest paper demonstrate delay importation epidemic pandemic strain influenza quantify term grow infection incidence source region traveler volume border screening measure travel duration inflight transmission delay infected arrival initiate chain transmission gather momentum investigate delay affect reproduction number emerge strain early presentation case arrive passenger reduce traveler number note previous simulation modeling aspect delay significant chance component make delay random variable way quantify delay specify probability distribution delay distribution issue delay distribution natural delay arise absence intervention effect reduce traveler number delay study previously specifically originate source specify homogeneous mixing world population assume likely time initial case arise United States day assume additional delay arise travel restriction appear minimal reduction traveler number paper add previous work simultaneously include wide range epidemiological factor possible intervention elevate flight transmission flight duration effect wear mask flight early presentation case traveler quarantine passenger flight detect case arrival consider region new pandemic strain influenza emerge region currently free infection refer source region risk country respectively travel country predominantly commercial air travel rapid transport potentially subject border screening intervention restrict discussion air travel aim assess effect variety non pharmaceutical border control measure individually combination time take epidemic take risk country epidemic say take reach current infectious case growth highly predictable nearly deterministic probability fade chance low intervention enhance source country origin undoubtedly large impact natural delay importation epidemic difficult quantify alternative fix originate city example highly connected city Hong Kong obvious effect result highly dependent choice adopt specific source region assume number international traveler originate reasonably small Methods suggestive rural semirural source region assume current heighten surveillance pandemic influenza continue nascent pandemic human human transmission identify pandemic declare concurrent case source region epidemic risk country series event need occur epidemic need underway source region second intend traveler need infect shortly departure infect traveler actually travel successfully disembark risk country fourth infected traveler fellow traveler infect flight initiate epidemic risk country infectiousness remain arrival finally epidemic need reach sufficient number case begin predictable exponential growth international spread emerge pandemic strain influenza occur recently infect person travel recently infect mean travel schedule occur day infect assume number individual travel source region risk country day know probability randomly select traveler recently infect person take equal prevalence recently infect people source region day incidence infection source region assume grow exponentially initially rate exponential growth determine disease reproduction number mean number case single infective generate direct contact serial interval average interval infection individual contact infect figure time infection recently infect traveler key component calculation affect chance positive border screening chance flight transmission infectivity remain arrival risk country time infection time schedule departure random dependence probability distribution exponential growth rate infection illustrate figure support information high epidemic growth rate source region great probability infected traveler infect recently assume individual detect departure screening prevent travel detect screen infect traveler symptomatic positively screen individual assume symptomatic hour infect cf use day probability symptomatic present departure screening compute curve figure distribution time infection immediately departure screening give infected traveler detect give curve figure contain adjustment probability detect departure instantaneous rate susceptible contact infect depend time infection describe infectiousness function page use peaked infectiousness function motivate viral shedding household transmission datum serial interval day basic reproduction number reproduction number intervention place contacted individual susceptible give area infectiousness function concern effective reproduction number hold intervention place obtain simply multiply infectiousness function appropriate constant area curve equal keep serial interval absence suitable datum assume scenario aircraft ventilation filtration system function properly infected traveler transmit infection rate flight mix community examine sensitivity assumption increase inflight transmission fold potentially happen air circulation filtration system malfunction flight transmission rate set zero optimistic scenario traveler wear effective mask transit term sensitivity analysis illustrate achievable good case scenario number offspring infected traveler infect flight random variable take Poisson distribution mean equal area infectiousness function flight duration traveler infect flight hour duration asymptomatic arrival detect screen probability arrive traveler infect source region detect arrival compute distribution time infection arrival distribution obtain curve figure shift right equal duration flight distribution time infection individual infect source region pass arrival screen undetected adjustment chance detect arrival Figure curve show infected traveler escape detection departure arrival highly likely enter risk country infectious period remain authority assume implement control option detect infected traveler arrival screening option individual base removal passenger test negative release immediately passenger test positive isolate second option flight base quarantining authority prevent passenger disperse community person screen flight passenger detect infect passenger quarantine previously recommend chain initiate risk country infected traveler mix community arrival suppose flight arrive infect passenger mix community classify infect arrival pre symptomatic symptomatic entry assume symptomatic infected arrival recognize symptom pandemic influenza present medical authority word spend remainder infectious period mix community hand presymptomatic infected arrival include individual infect flight assume mix freely community entry present medical authority delay follow onset symptom infected traveler enter community initiate major epidemic reproduction number exceed generally distribution size epidemic initiate infected arrival bimodal distinct peak correspond major epidemic minor outbreak figure event outbreak simply fade chance despite ample susceptible population ongoing step illustrate probabilistic removal traveler complete incubation period distribution time infection shift right equal flight duration case incubate flight detect symptomatic screening symptomatic case detect previously screen sensitivity illustration departure arrival enter community undetected infected traveler initiate minor inconsequential major epidemic depend characteristic disease public health policy mission number case outbreak fade typically small compare epidemic probability typical infective generate local epidemic compute branching process approximation initial stage epidemic equate epidemic event branching process extinct calculation known page modify allow fact process initiate random number infected arrival spend random infectious period arrive risk country distribution random number individual infect infected individual contact susceptible individual need calculation lack datum prevent definitive conclusion appropriate offspring distribution influenza transmission use Poisson distribution mean equal discount individual spend infectious period mix risk country Poisson offspring distribution appropriate area infectiousness function non random individual infection potential assume source region risk country undetected infect traveler flight offspring fail initiate epidemic arrival chain transmission initiate fail large epidemic support calculate probability distribution total delay epidemic gather momentum note give time epidemic initiate time initiation local epidemic gather momentum epidemic initiate risk country day initiate previous day probability distribution time delay epidemic initiate risk country follow identification source region describe Pr denote probability epidemic initiate day denote probability epidemic initiate day support Information calculation successfully initiate epidemic initially hover handful case reach sufficient number case growth essentially predictable mention concurrent case criterion epidemic gather momentum determine distribution time occurrence stochastic simulation approximate empirical distribution shifted gamma distribution criterion concurrent case conservatively high result theory branch process show probability minor epidemic start concurrent case small high value finally distribution total delay pandemic identify source region case risk country calculate convolution distribution illustrative purpose choose value encompass estimate propose previous pandemic number people infected source region assume reasonably small million flight day travel source region risk country carry passenger high number traveler affect delay marginally assume epidemic take source region result assume typical travel duration attempt departure possible arrival hour examine effect vary hour time presentation follow symptom onset varied immediately present time hour consider likely presence education campaign sensitivity symptomatic screening varied result present sensitivity probability recently infect traveler evade screening substantial screen reliably detect symptomatic traveler Figure typical travel duration short day incubation period addition early stage epidemic high source region act increase probability infected traveler infect recently escape detection asymptomatic travel Figure example assume sensitivity detect symptomatic infection calculate early stage epidemic proportion infected traveler evade departure arrival screening hour travel disease reproduction number respectively duration travel approach disease incubation period effective symptomatic screening substantially reduce likelihood traveler evade screening initiate epidemic figure reduce time onset symptom presentation subsequent isolation infected arrival reduce probability major epidemic initiate good case scenario infected traveler flight offspring present immediately follow onset symptom pose substantial risk epidemic initiation arise pre symptomatic transmission Figure delay contain fairly substantial natural component primarily time take increase number infective source region sufficiently chance recently infect traveler appreciable figure time take local epidemic risk country gather momentum follow successful seeding Figure absence intervention number infected individual successfully enter community risk country initially increase exponentially Figure individual base removal infected traveler number individual enter risk country undetecte screen proportionately reduce course epidemic Figure flightbased quarantining number infected individual enter risk country undetected dramatically reduce course epidemic relatively insensitive screening Figure flight base quarantining number infected passenger slip undetected bimodal peak occur number infected traveler attempt travel single figure screen daily probability epidemic initiate increase near certain number infected traveler arrive undetected exceed figure solid line screening individual base removal infected individual follow similar pattern reduce somewhat screen combination flightbase quarantining probability change dramatically initial rise dip essentially zero height epidemic source region figure dotted line arise flight infect traveler probability detect approach screening imperfect passenger flight quarantine epidemic start wane source region assume unlikely event pandemic strain restrict source region probability initiation rise corresponding distribution delay effect time symptom onset presentation probability infected traveler having enter wide community follow arrival initiate epidemic screening epidemic initiate risk country bi modal presence screening figure flight base quarantining effective prevent entry infected traveler height epidemic substantial cumulative risk initiation occur handful infective slip undetected figure whilst effect border screening particularly conjunction flight base quarantining daily probability initiation dramatic effect delay initiation pronounced figure border screening perfect sensitivity detect symptomatic case tend increase time epidemic initiate matter day week time initiation arrival index case epidemic reach concurrent case risk country adequately model shift Gamma distribution figure convolution right skew Gamma distribution left skewed delaydistribution figure yield distribution total delay epidemic reach case risk country figure distribution approximately symmetrical effect border screening total delay modest sensitive screening implement example traveler day sensitive screening individual base removal increase median delay day figure flight base quarantining extend median delay day general add delay arise flight base quarantining fold arise individual base removal natural component delay highly sensitive disease reproduction number figure example passenger day depart source country absence intervention median delay range low day day table delay sensitive number intend traveler little appreciable increase median delay occur traveler number low Figure example border control measure decrease number intend traveler depart source region day increase median total delay day decrease number intend traveler day increase median delay day table delay insensitive rate transmission flight example hour flight traveler day intervention prevent flight transmission altogether increase median delay day conversely double rate flight transmission reduce median delay day fold increase rate transmission flight decrease median delay day encourage early presentation case traveler follow onset symptom limited effect delay distribution figure example intend traveler day intervention reduce time presentation present hour increase median delay day immediate presentation symptom onset increase median delay day scenario general additional delay achieve introduce nonpharmaceutical border control measure generally small comparison natural delay Figure scenario intend traveler day combination flight base quarantining compliance mask wear travel immediate presentation symptom onset extend estimate median delay day Figure add delay diminish absolute term increase example intervention apply median delay extend day Figure exception generalisation travel number reduce dramatically add delay achieve drastic reduction travel number combine border control measure appear great add delay achieve example reduce number intend traveler day implement flight base quarantining implement compulsory mask wear travel presentation hour follow symptom onset substantial probability pandemic strain import assume epidemic confine source country estimate quartile delay median case undefined start major epidemic risk country extend day add delay decrease rapidly increase intervention apply estimate median delay extend day importation epidemic certain Figure formulate model importation infectious disease source region risk country permit comprehensive analysis effect border control measure result relevant early stage pandemic case contain single source region pandemic spread country model great complexity ability realistically model global mixing pattern require model develop pandemic strain influenza mind apply emerge infectious disease transmit person person assume Poisson distribution number secondary infection natural choice infect individual infectivity profile distribution large variance appropriate individual vary substantially infectiousness result conservative sense upper bind probability infected traveler manage initiate epidemic compare offspring distribution great variance reproduction number nature pandemic influenza virus particularly reproduction number uncertain reproduction number low result indicate risk country receive reasonably small number traveler day infected source region expect natural delay import epidemic order month variable favourable condition month natural delay decrease rapidly increase additional delay isolate individual detect border screening merely day plausible scenario departure arrival screening introduce screen detect symptomatic traveler extra delay quadruple flight detect quarantine effect remain modest week questionable extra delay achieve warrant disruption create large number quarantined passenger flight transmission commonly raise concern discussion importation infection inclusion flight transmission attractive feature model event substantial flight transmission influenza document modeling indoor airborne infection risk absence air filtration predict flight transmission risk elevate difficult estimate infectiousness influenza confine cabin space undoubtedly substantial reporting influenza case travel fail generate offspring flight provide aircraft ventilation system include filtration operational consider actual risk flight transmission low perceive risk result indicate delay relatively insensitive rate flight transmission make flight transmission issue commonly believe highly elevated transmission rate inflight hasten importation epidemic marginally consistent eliminate flight transmission wear protective mask increase delay marginally early presentation infected arrival detect border find add day delay extent arise assumption pre symptomatic transmission occur evidence contrast Ferguson et al assume incubation latent period equal mean day model presymptomatic transmission exclude infectiousness estimate spike dramatically immediately follow symptom onset decline rapidly soon model assumption immediate presentation onset symptom reduce transmission effectively presentation occur time onset symptom bulk infectivity occur immediately onset symptom result effect early presentation case likely practical term similar find give variable nature influenza symptom likely difference onset symptom measure clinical trial time person field suspect infect influenza virus fully resolve issue effective early presentation infected traveler delay local epidemic need well knowledge infectiousness individual onset symptom assume pandemic identify declare concurrent case source region attribute human human transmission screening apply departure arrival time screen event assume hour infected traveler isolate follow onset symptom border control measure available reduce traveler number big effect delay necessary number traveler low number equivalent control measure quarantine arrive passenger near perfect compliance result indicate short virtually eliminate international travel border control measure add little avoiding delay local epidemic influenza pandemic take source region form border control eventually overwhelmed cumulative number infected traveler attempt enter country way prevent local epidemic rapidly implement local control measure bring effective reproduction number local area achieve rapid elimination source region agreement recent study prevent exponential growth phase epidemic source region appear method able prevent nascent influenza pandemic reach atrisk country Text estimate daily probability epidemic initiation find mb pdf','2007-01-03 00:00:00','http://dx.doi.org/10.1371/journal.pone.000014',2510,641),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json','Conservation of polyamine regulation by translational frameshifting from yeast to mammals','Regulation of ornithine decarboxylase in vertebrates involves a negative feedback mechanism requiring the protein antizyme. Here we show that a similar mechanism exists in the fission yeast Schizosaccharomyces pombe. The expression of mammalian antizyme genes requires a specific +1 translational frameshift. The efficiency of the frameshift event reflects cellular polyamine levels creating the autoregulatory feedback loop. As shown here, the yeast antizyme gene and several newly identified antizyme genes from different nematodes also require a ribosomal frameshift event for their expression. Twelve nucleotides around the frameshift site are identical between S.pombe and the mammalian counterparts. The core element for this frameshifting is likely to have been present in the last common ancestor of yeast, nematodes and mammals.',' ef ciency ribosomal frameshifting speci codon sensor regulate polyamine level mammalian cell frameshifting occur decode gene antizyme partially overlap open reading frame orf protein sequencing show reading frame shift occur codon cause proportion ribosome enter synthesize transframe protein Matsufuji et al encode main functional domain Matsufuji et al Miyazaki et al antizyme ribosome initiation site antizyme protein bind ornithine decarboxylase ODC Murakami et al Cof inhibit Heller et al target degradation proteosome ubiquitylation Murakami et al Murakami et al ODC catalyze rst usually ratelimite step synthesis polyamine conversion ornithine putrescine putrescine substrate synthesis spermidine spermine inhibition ODC antizyme negative regulator synthesis polyamine addition antizyme negative regulator polyamine transporter Mitchell et al Suzuki et al Sakata et al discover Matsufuji colleague Gesteland et al Rom Kahana increase polyamine level elevate frameshifte decode antizyme mrna increase level antizyme antizyme negatively regulate synthesis uptake polyamine frameshifting sensor autoregulatory circuit second mammalian paralog antizyme antizyme similar property antizyme include regulatory frameshifting stimulate degradation ODC certain condition antizyme active Ivanov et al Zhu et al preparation like antizyme antizyme mrna ubiquitously express body time abundant mrna antizyme Ivanov et al addition antizyme mammal paralog gene antizyme encode orf express spermatogenesis Ivanov et al Zebra sh multiple antizyme gene differ expression pattern activity Saito et al study address regulation fungal ODC response exogenously add polyamine case examine Physarum polycephalum Mitchell Wilson saccharomyce cerevisiae Fonzi Toth Cof Neurospora crassa Barnett et al Williams et al add polyamine especially spermidine result signi cant repression ODC activity mechanism repression vary fungus fungus apparently different mechanism polyamine dependent regulation ODC high eukaryote case existence antizyme like protein suggest disprove case Barnett et al substantiate case expect small cationic nature ability neutralize negative charge locally polyamine play key role process range functioning certain ion channel Williams nucleic acid packaging dna replication apoptosis transcription translation role polyamine complex illustrate transfer butylamine moiety spermidine lysine residue form hypusine mammalian translation initiation factor know substrate reaction Tome et al Lee et al spermine negatively regulate growth prostatic carcinoma cell primary site Smith et al later stage tumor progression fail induce antizyme correlate cell refractory spermine Koike et al lack antizyme function important early deregulation cellular proliferation oral tumor Tsuji Conservation polyamine regulation translational frameshifting yeast mammal EMBO Journal Vol pp European Molecular Biology Organization et al probably level polyamine alter tumor inhibitor polyamine synthesis test antiproliferative cell death effect synthesis ODC vary cell cycle normal cell Linden et al Fredlund et al induce growth stimulus constitutively elevate transform cell Pegg Auvinen et al phosphorylate ODC translocate surface membrane important mitotic cytoskeleton rearrangement event Heiskala et al example certain mrna contain signal elevate speci frameshifting fold background level normal translational error addition antizyme frameshifting involve decoding bacterial yeast gene especially mammalian Retroviruses Coronaviruses plant virus bacterial insertion sequence Atkins et al site frameshifting mammalian antizyme mrnas UCC UGA quadruplet translocation occur uccu underline shift reading frame immediately UGA stop codon initiate frame Matsufuji et al Ivanov et al frameshifting occur ef ciency important base quadruplet rst base stop codon important feature pseudoknot shift site speci sequence shift site Matsufuji et al Ivanov et al pseudoknot shift site common stimulator eukaryotic frameshifting synthesis antizyme know case utilize frameshifting comparative analysis RNA sequence different organism informative important feature different option select evolution know example program frameshifting virus chromosomal mobile element opportunity comparison frameshift cassette divergent organism time divergence approximate limited start frameshifting require bacterial release factor expression Persson Atkins antizyme provide rst opportunity comparison eukaryote antizyme gene genetically tractable low eukaryote helpful understand functionally important interaction responsible autoregulatory program frameshifting identi cation antizyme gene Schizosaccharomyces pombe search dna sequence encode protein sequence homologous Drosophila melanogaster antizyme Ivanov et al homo sapien antizyme identi e anonymous cdna clone DDBJ EMBL GenBank accession similarity limited identity similarity human antizyme antizyme high region highly conserve previously identi ed antizyme figure close examination cdna nucleotide sequence provide evidence encode homolog antizyme initiate AUG codon ORF similar high eukaryotic antizyme gene AUG cdna fact aug close end rst second aug initiate translation ORF overlap long downstream ORF translational frameshifting event overlap generate protein product analogous product antizyme gene high eukaryote furthermore nucleotide ugg UGC UCC UGA identical nucleotide mammalian antizyme include frameshift site nucleotide identical corresponding region previously identi ed antizyme gene figure previous experiment mammalian frameshift sequence test show short nucleotide sequence suf cient stimulate measurable level frameshifting Ivanov et al con rm ORF con guration putative antizyme gene region correspond overlap orf nucleotide UTR nucleotide UTR ampli ed genomic dna cdna library sequence ampli ed dna con rme overlap orf deduced con guration sequence DDBJ EMBL GenBank accession differ previously sequence cDNA clone nucleotide code region UTR change alanine codon proline silent mutation proline codon sequence cdna library genomic DNA identical conclude difference clone likely strain variation gene contain intron ampli ed region protein test antizyme activity gene fusion glutathione transferase GST construct antizyme fuse frame delete nucleotide encode stop codon fusion gene express Escherichia coli protein puri e af nity chromatography ODC inhibitory activity test incubate recombinant antizyme protein crude extract assay mixture ODC activity result figure recombinant protein inhibit ODC GST mg inhibit ODC datum show light result gene call ODC antizyme SPA interestingly ODC inhibit mouse antizyme antizyme express GST fusion yeast fusion protein inhibit mouse ODC datum overexpression SPA effect overexpression antizyme cellular physiology test previously mammalian cell physiological change associate complete absence antizyme activity investigate complication multiple antizyme single antizyme provide chance explore knockout SPA deletion strain Ivanov et al generate replace orf gene orf Materials method complete deletion SPA orf affect viability cell rich ye minimal MM medium temperature differential effect mutant wild type cell growth similarly growth rate mate ef ciencie overall morphology knockout strain apparently indistinguishable wild type cell result wild type cell abundant polyamine spermidine follow putrescine figure spermine cadaverine find small amount distribution polyamine content similar fungus polyamine concentration measure reference review Tabor Tabor effect SPA deletion cellular polyamine content examine exponentially grow stationary phase cell figure cellular concentration putrescine spermidine cadaverine spermine high knockout strain wild type cell great effect see putrescine cadaverine content small effect spermidine presumably eukaryotic ODC activity directly catalyze decarboxylation ornithine lysine produce putrescine cadaverine respectively Pegg McGill subsequent regulatory event affect homeostasis spermidine spermine effect inactivate antizyme polyamine content exponentially grow cell modest fold case effect pronounce cell stationary phase fold increase putrescine cadaverine content respectively knockout strain test overexpression SPA version gene clone expression vector strong thiamine repressible promoter type SPA sequence second fuse frame SPA wild type SPA deletion strain transform overexpression construct derepression promoter gradual process require dilution intracellular pool thiamine repressor cell division day exponential growth derepresse condition yeast strain transform SPA overexpression construct signi cant increase double time Figure growth inhibition great construct express frame version SPA prolonged incubation day cell cease growth accumulate determine bȳ ow cytometry datum show fact inframe overexpression construct differ single nucleotide wild type construct confer severe phenotype consistent hypothesis translational frameshifting require expression SPA growth phenotype associate SPA overexpression partially relieve add mM putrescine medium mm effect datum show slow growth correlate aberrant polyamine level polyamine content deletion strain carry frame SPA overexpression vector measure derepresse repress condition case day exponential growth figure expect overexpression SPA result signi cant reduction intracellular level polyamine long day incubation derepresse condition putrescine cadaverine detect datum frameshifte expression SPA Previously develop assay measure antizyme translational frameshifting Matsufuji et al Ivanov et al nucleotide sequence assay insert GST lacz assayed sequence fuse frame GST fuse frame galactosidase activity provide measure frameshifte ef ciency determine translational frameshifting occur overlap SPA region SPA include rst codon plus nucleotide downstream stop codon test frameshifting occur compare construct fuse frame result consistent frameshifting crucial expression experiment show frameshift cassette mammalian antizyme direct ef cient frameshifting test reverse experiment conduct SPA gene translate vitro rabbit reticulocyte lysate result frameshift ef ciency measure addition polyamine frameshifte ef ciency addition spermidine translation mixture nal concentration mm result fold increase frameshifte level high observe endogenous system vivo autoradiogram show observed ef ciency frameshifte SPA frameshifte cassette vivo signi cantly expect limited nucleotide similarity antizyme frameshift site high eukaryote prompt search additional stimulatory element SPA frameshift cassette follow experiment strain carry deletion SPA high polyamine give high frameshifting high galactosidase activity general obtain similar ratio mutant wild type frameshifting ef ciency strain intact SPA gene delete sequence sense codon little effect frameshifte ef ciency delete sense codon UCC lead fold reduction frameshifte ef ciency figure imply conservation nucleotide UCC UGA frameshift site importance stimulate frameshifting suggest additional sequence SPA stimulate recode event nucleotide region search possible structure computer RNA fold algorithm plus visual inspection algorithm predict minimal structure region deletion construct construct test importance putative structure frameshifte ef ciency result figure deletion lead signi cant fold reduction frameshifting indicate presence major stimulatory element nucleotide region immediately follow frameshift site SPA result indicate putative RNA structure region suf cient activity element additional deletion delineate boundary stimulatory element frameshift site nucleotide downstream construct stimulate fold frameshifting nucleotide downstream probably contain experiment describe characteristic autoregulatory circuit mammalian antizyme con rme SPA inhibition ODC translational frameshifting key question leave recode event responsive polyamine level cell show overexpression SPA lead signi cant reduction polyamine level SPA strain co transform SPA wild type overexpresse plasmid cell overexpresse wild type SPA grow slowly continuously construct monitor frameshifting SPA frameshift sequence frameshifting compare SPA non overexpressing cell case frameshifting measure relative frame control result figure signi cant reduction fold frameshifte ef ciency SPA overproduce cell correlate decrease polyamine content fold putrescine fold spermidine indicate polyamine modulate frameshifte ef ciency SPA alternative likely possibility SPA overexpression reduce frameshifte high level SPA transcript titrate factor limit frameshifting SPA frameshift signal direct fold frameshifting cell SPA cell case measurement stationary phase relatively high standard deviation measurement dif cult draw rm conclusion particular result search Caenorhabditis elegan express sequence tag EST sequence mammalian antizyme sequence identi ed clone sequence deconvolute contiguous cdna sequence primer design basis sequence PCR amplify subclone cdna cdna library sequence subcloned cdna con rme DDBJ EMBL GenBank accession subsequently release genomic sequence gene DDBJ EMBL GenBank accession con rm cdna datum amino acid sequence deduce cdna sequence reveal long ORF similarity previously report antizyme sequence overall identity similarity human antizyme identity similarity Drosophila antizyme similarity high SPA antizyme gene concentrate region highly conserve previously identi ed antizyme figure like mammalian antizyme long ORF lack appropriate frame initiation codon expression provide initiation short upstream overlap ORF lead ribosomal frameshifting overlap putative antizyme frameshift site nucleotide proximal end nucleotide identical consensus sequence antizyme frameshift site figure expression antizyme investigate heterologous system construct contain entire antizyme cdna wildtype sequence single nucleotide deletion fuse frame frame control transcribe vitro RNA translate rabbit reticulocyte lysate product examine figure main product construct apparent kda slightly great predict kDa aberrant slow expect mobility observe antizyme protein specie Ivanov et al ratio wild type frame product estimate ef ciency frameshifting antizyme reticulocyte lysate somewhat low SPA frameshifte system addition spermidine translation reaction double ef ciency frameshifting exact number easy determine dif culty de ning background value frameshifte property antizyme mrna test vivo cell sequence include rst codon plus nucleotide downstream insert GST lacz PIU LAC plasmid Comparison galactosidase activity cell strain transform wild type construct frame control construct indicate frameshifting frameshifting observe heterologous system sequence consideration discuss conclude expression gene require ribosomal frameshifting search EST database newly discover antizyme identi ed antizyme ortholog nematode specie case Necator americanus Haemonchus contortus cdna sequence database suf cient contig complete code region case onchocerca volvulus DDBJ EMBL GenBank accession Pristioncus paci cus DDBJ GenBank accession complete cdna sequence obtain PCR amplify sequence gene cdna library previously identi ed eukaryotic antizyme gene ORF con guration newly find nematode ortholog imply necessity frameshifting synthesis length protein antizyme mrna frameshift site UUU UGA unique differ UCC UGA previously know antizyme mrnas antizyme gene share feature cus antizyme phylogenetic tree nematode antizyme protein sequence match exactly phylogenetic relationship Blaxter nematode express indicate gene sequence result divergent evolution nematode lineage datum show result UUU UGA frameshift site evolve common ancestor cus divergence probably million year ability UUU UGA sequence direct frameshifting test mammalian system context mammalian antizyme mrna presence RNA pseudoknot stimulator BMV coat gene fusion construct TCC TGA TTT TGA substitution transcribe translate rabbit reticulocyte lysate percent frameshift ef ciency see absence exogenously add polyamine time ef ciency see UCC UGA transcript frameshift ef ciency mM spermidine add time wild type Matsufuji et al similar result obtain cultured mammalian cell transfecte TTT TGA mutant construct frameshift high wild type construct high lowpolyamine condition unpublished result result demonstrate putative frameshift site UUU UGA shifty UCC UGA antizyme context subject polyamine stimulation result present yeast homolog mammalian antizyme rst document example antizyme type regulation ODC low eukaryote delete SPA yeast genome detectable effect viability overt phenotypic effect expect result altered accumulation polyamine cell interestingly effect pronounce cell stationary phase knockout cell accumulate time putrescine wild type counterpart compare fold increase putrescine exponentially grow cell likely explanation observation usual rate ornithine decarboxylation exponentially grow cell close capacity concentration substrate enzyme product time newly synthesize polyamine continuously dilute Fig effect polyamine depletion SPA frameshifting polyamine depletion achieve overexpression wild type version SPA culture assay frameshifting polyamine content number column indicate fold reduction frameshifting polyamine content compare cell overexpress SPA antizyme gene cell growth division rate identical rate maximum capacity synthesis cell stationary phase long dilute newly synthesize polyamine importantly lack effective antizymeindependent mechanism shut ODC suggest SPA primary regulator ODC activity cell growth short term regulation non dividing cell long term SPA day derepression lead complete depletion intracellular putrescine result imply ornithine decarboxylation source putrescine synthesis pathway arginine agmatine utilize complete depletion cadaverine SPA overexpresse cell suggest ODC enzyme decarboxylate lysine case rat tissue Pegg McGill somewhat perplexing addition putrescine medium lead partial relief growth phenotype associate SPA overexpression likely explanation import putrescine poorly ii alternatively like mammalian system maybe SPA inhibit ODC polyamine transporter experiment help distinguish model unclear widespread antizyme gene fungal kingdom identi ed clone antizyme homolog ssion yeast schizosaccharomyce octosporu Schizosaccharomyces japonicus distantly relate fungus Botryotinia fuckeliana Emericella nidulan unpublished result antizyme frameshift site fungus evolve unique way different know antizyme distantly relate fungus conserve autoregulatory frameshifting fact yeast antizyme gene suggest possibility high eukaryotic metazoan antizyme gene previously report antizyme activity unicellular organism recent analysis suggest true antizyme Ivanov et al make SPA rst bona de antizyme unicellular organism remarkable similarity core sequence important antizyme frameshifte human convergent divergent evolution near identity sequence worm Drosophila Xenopus zebra sh human argue convergent evolution antizyme frameshifting arise common ancestor billion year ago cis act RNA element know stimulate mammalian antizyme frameshifting nucleotide sequence immediately shift site Matsufuji et al unpublished result second stimulator UGA stop codon RNA pseudoknot start nucleotide UGA stop codon frameshift site previously identi ed antizyme mammal way Drosophila substantial similarity sequence immediately shift site sixteen nucleotide completely conserve gene schizosaccharomyce pombe antizyme nucleotide identical consensus respectively sequence generally distantly relate antizyme similarity con ned end region SPA deletion datum mutation nucleotide consensus sequence lead reduce frameshifting ef ciency indication conservation nucleotide sequence region importance stimulate ef cient frameshifting striking antizyme gene sequence identi ed far include number unpublished one end UGA stop codon particularly surprising stop codon substitute UGA stimulate antizyme frameshifting slightly ef ciently vitro Matsufuji et al vivo unpublished pseudoknot stimulate frameshifte antizyme highly conserve know vertebrate antizyme include mammalian antizyme figure invertebrate antizyme mrnas identi ed far include present sequence equivalent region simply fold comparable RNA structure sequence immediately frameshift site conserve invertebrate vertebrate conservation region Drosophila vertebrate counterpart note Ivanov et al antizyme gene contain sequence YGYCCCYCA pyrimidine region identical consensus antizyme gene nematode similar sequence figure signi cance similarity clear fact sequence region appear play role antizyme vitro frameshifting outside RNA pseudoknot context Matsufuji et al example know RNA element frameshift site stimulate frameshifting RNA pseudoknot mammalian antizyme second short RNA sequence immediately follow frameshift site Farabaugh et al additional example helpful decipher role element play mechanism frameshifting currently know invertebrate antizyme gene contain frameshift stimulator result present stimulator enhance frameshifte fold stimulator appear completely different RNA pseudoknot vertebrate deletion experiment indicate predict RNA structure contain minimally require region nucleotide downstream frameshift site figure suf cient confer stimulatory effect SPA stimulator act directly sequence unusual RNA structure involve non base pair detailed mutagenesis combine phylogenetic analysis require discern nature stimulator nematode antizyme analyze presence possible core frameshift site Computer RNA fold program identify potentially interesting structure importantly phylogenetic analysis ve identi ed nematode antizyme fail identify conservation primary RNA sequence matter potential secondary structure outside core region share member indicate extra cis act stimulator exist nematode antizyme locate different place mrna example untranslated region suggestion support sequence common mechanism frameshifting pair peptidyl tRNA new reading frame alternative mechanism peptidyl tRNA merely occlude rst base codon document yeast frameshifting Farabaugh et al result experiment mutant mammalian antizyme shift site point occlusion mechanism Matsufuji et al mechanism wild type UCC UGA shift site clear antizyme UUU UGA sequence obvious candidate pair Phe tRNA pair perfectly UUU frame UCC UGA Ser trna rst read UCC good pair CCU important problem warrant investigation frameshift ef ciency SPA frameshift site low observe mammalian antizyme test organism frameshift ef ciency antizyme cassette Ivanov et al possible observed ef ciencie antizyme arti cially low construct include cis act stimulatory element hand reason low level frameshifting correctly evolve balance characteristic complex system relative protein stabilitie like core cellular process antizyme polyamine regulatory scheme conserve yeast human obvious special mechanism exquisitely preserve vast evolutionary time aspect system experiment detect viewpoint important exploit genetic system understand thoroughly physiological effect perturb antizyme system SPA gene ampli ed follow primer sense ATCATGAAAAGCCTC antisense cDNA library template ampli cation kindly provide University Utah antizyme gene ampli ed follow primer GTATAATTTTAC sense GTAGCTCATGATG antisense ampli ed DNA construct vitro transcription translation antizyme clone plasmid saci hindiii site incorporate primer frame construct step PCR cDNA sequence cus antizyme gene obtain perform RACE PCR cdna library kindly provide Ralf Sommer cus Susan Haynes SPA overexpression construct amplify gene primer sense TGC antisense insert BamHI digest product msci BamHI site plasmid frame construct step PCR subsequently frame SPA construct describe step PCR plasmid dna template construct frameshift assay dna fragment give nucleotide length describe main text ampli ed SPA antizyme construct describe fragment clone KpnI bsteii site PIU LAC plasmid Ivanov et al PCR primer include spacer cloning site BstEII antizyme sequence order correct reading frame vivo frameshifting assay strain describe Ivanov et al plasmid expression PCR amplify SPA rst codon downstream frame template clone product ecori xhoi restriction site plasmid antizyme frameshift site bmv coatantizyme fusion construct Matsufuji et al mutate step PCR generate knockout strain orf SPA replace exactly ORF accomplish pair primer ampli e nucleotide normallȳ ank orf orf gene ampli ed dna product gel puri ed mg electroporate cell URA LEU transformant select growth URA LEU medium respectively PCR screen partial sequencing region homologous recombination con rme SPA disruption dna clone sequence automate sequencing machine ABI pombe ODC active crude extract prepare follow strain provide grow od ml minimal medium LEU milligram lysing enzyme Sigma add follow continued incubation min cell harvest wash cold homogenization buffer mm pH sucrose mm dithiothreitol DTT mm phosphate mM EDTA resuspend ml homogenization buffer cell break open lysate clari ed centrifugation min extract dialyze overnight dialysis buffer mm HCl pH mm DTT mm phosphate mM EDTA volume ml extract ODC assay ODC activity assay measure release CO Amersham describe Nishiyama et al reaction take pre incubation extract mM ornithine DFMO min lead inhibition CO release cell collect centrifugation wash twice ml phosphate buffer saline PBS pellet freeze use pellet resuspend ml PBS aliquot suspension mix equal volume perchloric acid vortexe min keep ice min centrifuge min microliter supernatant subject polyamine analysis high performance liquid chromatography describe previously Murakami et al protein concentration determine BCA protein assay kit experiment bmv coat antizyme fusion construct perform describe previously Matsufuji et al plasmid dna template prepare QIAGEN Miniprep Kit digest hindiii transcript SPA vitro translation PCR template promoter incorporate PCR primer linearized DNA mg template vitro transcription Ambion MEGAshortscript TM Kit DNasetreated RNAs recover resuspend ml RNase free water microliter speci ed transcript suspension vitro translation reaction ml mM amino acid mix ml reticulocyte lysate Promega ml Amersham total volume ml reaction stop add ml RNase mg ml frameshift ef ciencie quanti ed describe Ivanov et al','2000-04-17 00:00:00','http://dx.doi.org/10.1093/emboj/19.8.1907',3405,994),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json','On pandemics and the duty to care: whose duty? who cares?','BACKGROUND: As a number of commentators have noted, SARS exposed the vulnerabilities of our health care systems and governance structures. Health care professionals (HCPs) and hospital systems that bore the brunt of the SARS outbreak continue to struggle with the aftermath of the crisis. Indeed, HCPs – both in clinical care and in public health – were severely tested by SARS. Unprecedented demands were placed on their skills and expertise, and their personal commitment to their profession was severely tried. Many were exposed to serious risk of morbidity and mortality, as evidenced by the World Health Organization figures showing that approximately 30% of reported cases were among HCPs, some of whom died from the infection. Despite this challenge, professional codes of ethics are silent on the issue of duty to care during communicable disease outbreaks, thus providing no guidance on what is expected of HCPs or how they ought to approach their duty to care in the face of risk. DISCUSSION: In the aftermath of SARS and with the spectre of a pandemic avian influenza, it is imperative that we (re)consider the obligations of HCPs for patients with severe infectious diseases, particularly diseases that pose risks to those providing care. It is of pressing importance that organizations representing HCPs give clear indication of what standard of care is expected of their members in the event of a pandemic. In this paper, we address the issue of special obligations of HCPs during an infectious disease outbreak. We argue that there is a pressing need to clarify the rights and responsibilities of HCPs in the current context of pandemic flu preparedness, and that these rights and responsibilities ought to be codified in professional codes of ethics. Finally, we present a brief historical accounting of the treatment of the duty to care in professional health care codes of ethics. SUMMARY: An honest and critical examination of the role of HCPs during communicable disease outbreaks is needed in order to provide guidelines regarding professional rights and responsibilities, as well as ethical duties and obligations. With this paper, we hope to open the social dialogue and advance the public debate on this increasingly urgent issue.',' world witness spread novel deadly virus SARS health care worker HCWs hospital system bear brunt SARS outbreak continue struggle aftermath crisis hcw clinical care public health severely test SARS unprecedented demand place skill expertise personal commitment profession severely try expose risk morbidity mortality approximately report case hcw die infection number commentator note SARS expose vulnerability current health care system governance structure aftermath SARS spectre pandemic avian influenza imperative need consider obligation hcw patient severe infectious disease particularly disease pose risk provide care press importance organization represent hcw non professional alike clear indication standard care expect member event pandemic expert believe SARS outbreak merely preview flu pandemic soon arrive possibly avian influenza virus clearly avian flu threaten widespread SARS potential truly global pandemic honest critical examination role hcw crisis need order provide guideline professional right responsibility ethical duty obligation paper address issue special obligation health care professional HCPs infectious disease outbreak contend press need clarify right responsibility hcp especially current context pandemic flu preparedness argue right responsibility well codify professional code ethic finally present brief historical account treatment duty care professional health care code ethic intention open social dialogue advance public debate increasingly relevant issue give response hcp SARS crisis generally regard exemplary ask ethical problem truly exist little doubt vast majority HCPs perform job admirably considerable stress significant personal risk hcp provide exemplary care behave truly heroic fashion formally problematize note hcp act supererogatory manner SARS outbreak Carlo Urbani World Health Organization die SARS expose unknown virus course carry professional duty likewise score nurse doctor respiratory technician professional nonprofessional health worker labour extremely long hour personal risk demonstration go duty prove necessary control disease highly morally commendable time concern surface SARS extent hcp tolerate risk infection baulk provide care infect unknown virus circumstance staff issue SARS ward assessment centre failure report duty outbreak result permanent dismissal hospital staff consequence risk face SARS distribute equitably hcp volunteer provide care face great exposure follow outbreak treat SARS patient raise concern protection provide safeguard health family member conflicting obligation significant concern hcp bind ethic care obligation patient primary time hcp compete obligation family friend fear infect addition obligation health particularly special vulnerability co morbid condition SARS hcp question choice career decide leave profession pursue new venture indicate unwillingness inability care patient face risk recent survey datum indicate exist mixed view duty care patient infectious disease outbreak clear issue duty care emerge matter paramount concern health care professional hospital administrator public policy maker bioethicist ethical foundation duty provide care ground longstanding ethical principle foremost principle beneficience recognize define special moral obligation hcp welfare patient advance patient modern health care commonly understand generally accept principle beneficence constitute foundational principle patient provider relationship hcp general physician particular number compelling reason provide care context infectious disease outbreak Clark recently outline ability physician health care professional provide care great public increase obligation provide care self care self protection care protection friend family member acknowledge pandemic plan evident expertise HCPs integral principal component response pandemic sector society legitimately expect fulfil role assume level risk arguably hcp consent great average risk choice profession grant risk contract infectious disease likely concern generation prospective health care worker informed reading medical literature year show infectious disease remain ubiquitous problematic overly optimistic statement future threat infectious disease unreasonable argue hcp aware great average risk pose choice profession publicly fund health care system find western society strong claim social contract HCP society reasonable legitimate expectation public HCPs respond infectious disease emergency society grant permit profession self regulate understanding response occur characteristic self regulate profession development standard practice refer good practice guideline standard articulate professional code ethic develop basis fundamental principle value particular profession case instance respect code ethic develop long ago medicine nursing code ethic long respected tradition health profession today health social care profession code ethic place provide guidance member code ethic refer instrument soft law owe non legislative nature health care profession code ethic interpret guide ethical reasoning framework treatment individual patient substitute reasoning absolute mandate time code vague render ineffectual irrelevant era health care technology evolve rapid pace effort necessary ensure code ethic remain current practical concordant public expectation informative comprehensible code ethic numerous tangible benefit great benefit dispel confusion uncertainty hcp concern professional right responsibility regard duty care course detailed treatment issue professional code ethic serve increase awareness comfort level result increase willingness provide care uncertain risky condition additionally code guide professional conduct effectively serve norm standard recognizable enforceable law act foundation legal obligation decision code ethic serve potent form symbolic communication public serve profession make explicit value health care profession represent professional code ethic reassure public trust invest profession justify legitimate properly note following excerpt College Nurses Ontario Practice Standard commitment nursing profession member profession bring respect trust public continue deserve respect nurse duty uphold standard profession member self regulate profession nurse commitment help regulate nursing protect public right quality nursing service public interest profession continue regulate develop change method self regulation meet change health care society nurse obligation participate effective evolution self regulation self regulation privilege nurse accountable responsibility accompany privilege current code ethic duty care epidemic small concern current professional code ethic fail provide explicit guidance sufficient set policy assure public event infectious disease outbreak Canadian Medical Association instance release revise Code Ethics year SARS pandemic Canada particularly affect despite seemingly fortuitous timing publication revise Code astoundingly altogether silent physician duty care describe equal myriad ethical dilemma emerge global outbreak key revision edition CMA Code addition follow item Fundamental Responsibilities section consider society matter affect health addition little address substantively meaningful way duty care obligation hcp context infectious disease outbreak addition responsibility obligate physician provide treatment health peril wording vague significant guidance clinical practice contrast American Medical Association AMA appear recognize present need address issue duty care wake terrorist attack AMA adopt new ethic policy focus specifically medical profession obligation responsibility context public health emergency following passage AMA policy document Physician obligation Disaster Preparedness Response adopt June national regional local response epidemic terrorist attack disaster require extensive involvement physician commitment care sick injure individual physician obligation provide urgent medical care disaster ethical obligation hold face great usual risk safety health life physician workforce unlimited resource participate disaster response physician balance immediate benefit individual patient ability care patient future AMA take step right direction state obligation member great credit initiate process remain see national medical association health care profession follow suit redress silence code ethic duty provide care extent code ethic see reflection endure professional value time clearly influence product historical circumstance CMA instance previously include Code Ethics strongly word statement explicitly address obligation physician infectious disease outbreak version read follow pestilence prevail physician duty face danger continue labour alleviation suffering jeopardy life appearance CMA Code type strong categorical language professional duty care interestingly specific text cite appear time revision follow influenza pandemic conspicuously disappear revision release AMA include provision Code Ethics likewise excise mark professional retrenchment strong obligation provide care reflect current code ethic attract increase interest late number explanation propose academic commentator instance retrenchment link rise government corporate intrusion medical practice include Clark point increase general belief originate circa infectious disease vanquish likely case factor play significant role observe retrenchment time irrespective reason underlie current silence little doubt infectious disease increase clinical reality developed world long tremendous challenge develop world reason continue silence code ethic greatly problematic clinically normatively current consensus explicitly stringently requirement duty care state survey american physician respondent report decidely mixed view continue care patient event outbreak give narrow majority survey physician report believe professional duty treat patient epidemic author study conclude reinforcement medical profession ethical duty treat event public health crisis reinforcement duty care echo CMA Code Ethics clearly stipulate physician duty provide care jeopardy life statement consider strong categorical today require provision care entail significant risk provider appear demand hcp behave like supreme Samaritans reasonable ethical remain open question doubtful hcp adhere stringent obligation face SARS like crisis Emmanuel instructively note historical record physician decidedly mixed regard response vocal mount opposition treat seropositive patient height HIV AIDS epidemic medical profession reconsider ongoing retrenchment duty care threat new epidemic round open discourse require acceptable standard professional engagement occur level supreme good merely decent Samaritan presently need address professional duty hcp patient risk society include family friend co worker patient addition population large matter balance content current professional code ethic offer little guidance reflection consensus health care community wake SARS current threat avian influenza clearly time medicine nursing self regulate health care profession address issue head number option open profession option argue unacceptable remain silent extreme code ethic revise strong emphasis professional obligation duty provide care infectious disease emergency assume position lean supererogation perform act duty option exist extreme instance code reflect strong limited duty care limit clearly specify alternatively weak emphasis duty care option sympathetic self concern hcp run risk dissolution generally high regard health care profession exist society today maintain respect duty care acceptable code ethic vague ambiguous avoid explicit statement position particularly true light call additional protection hcp infectious disease outbreak include position statement effect issue Ontario Medical Association call danger pay enhance disability insurance justify profession express strong commitment ethical duty provide care public health emergency absence commitment additional measure protect safeguard hcp appear self serve misplace current context pandemic influenza planning public health emergency acute need strategy encourage great discussion dialogue interested party stakeholder step professional college create forum engage membership encourage exchange view issue exchange inform development formal position statement duty care communicable disease outbreak development clear unambiguous guideline professional right responsibility ethical duty obligation hcp outbreak statement ought publicly available prominently post website professional health care college association order encourage sustained dialogue issue raise professional college step foster public debate dialogue position take health care profession promote public debate working group University Toronto Joint Centre Bioethics produce ethical framework guide preparedness planning pandemic influenza base experience study SARS crisis framework present point value base ethical guide pandemic planning include value duty care report publicly available internet use webcaste electronic town hall meeting plan facilitate open exchange utmost importance promote public discourse issue importantly voice directly affect communicable disease outbreak health care professional general public participate discussion determine legitimate hcp eschew duty care face personal risk light recent experience canadian physician nurse hcw frontline SARS outbreak submit canadian health care community lead charge address issue duty care ethical obligation time public health emergency place open honest discussion currently vagueness ambiguity view health care code ethic speak specifically issue order guide professional behaviour infectious disease outbreak time address ethical duty provide care hand arrival public health emergency','2006-04-20 00:00:00','http://dx.doi.org/10.1186/1472-6939-7-5',1758,722),('a77badb297bf4927419481e660ea8f1806ed91b1.json','Torsional restraint: a new twist on frameshifting pseudoknots','mRNA pseudoknots have a stimulatory function in programmed −1 ribosomal frameshifting (−1 PRF). Though we previously presented a model for how mRNA pseudoknots might activate the mechanism for −1 PRF, it did not address the question of the role that they may play in positioning the mRNA relative to the ribosome in this process [E. P. Plant, K. L. M. Jacobs, J. W. Harger, A. Meskauskas, J. L. Jacobs, J. L. Baxter, A. N. Petrov and J. D. Dinman (2003) RNA, 9, 168–174]. A separate ‘torsional restraint’ model suggests that mRNA pseudoknots act to increase the fraction of ribosomes directed to pause with the upstream heptameric slippery site positioned at the ribosome\'s A- and P-decoding sites [J. D. Dinman (1995) Yeast, 11, 1115–1127]. Here, experiments using a series of ‘pseudo-pseudoknots’ having different degrees of rotational freedom were used to test this model. The results of this study support the mechanistic hypothesis that −1 ribosomal frameshifting is enhanced by torsional resistance of the mRNA pseudoknot.',' structure RNA molecule widely recognize play role process include structurally organize complex RNAs assembly ribonucleoprotein complex translational recoding regulation review common RNA fold motif pseudoknot folding single strand RNA form helical structure single strand loop join structure infer RNA sequence frameshifting function demonstrate review theoretical progress understand mrna pseudoknot promote efficient ribosomal frameshifting complete understanding mechanism remain untested program ribosomal frameshift signal typically divide component slippery site form NNW WWH stretch identical nucleotide residue space indicate unshifted zero frame ii spacer region iii mrna structural element pseudoknot general model posit encounter mrna pseudoknot elongating ribosome force pause anticodon site trnas base pair zero frame codon slippery site nature trna mrna interaction relative slip nucleotide allow base pairing non wobble position slippage occur ribosomal pause show change affect ribosome pause time affect frameshift efficiency review important observation mrna pseudoknot energetically equivalent stem loop structure appear promote ribosome pause equal effectiveness mrna pseudoknot efficient promote PRF model provide refinement original simultaneous slippage model frameshifting suggest entire ribosome have slip base direction slippage accomplish move small section mRNA downstream tunnel base direction propose accomplish bulky difficult unwind mrna pseudoknot structure wedge downstream entrance tunnel ribosome prevent downstream region mrna pull ribosome equivalent base accommodation step elongation blockage introduce tension spacer region resolve unpaire mrna trnas allow mrna slip nt backwards result net shift reading frame base model provide partial explanation mrna pseudoknot promote program ribosomal frameshifting PRF efficiently simple correspondence address tel fax email Author publish Oxford University Press right reserve online version article publish open access model user entitle use reproduce disseminate display open access version article non commercial purpose provide original authorship properly fully attribute Journal Oxford University Press attribute original place publication correct citation detail give article subsequently reproduce disseminate entirety derivative work clearly indicate commercial use contact stem loop structure answer question mrna pseudoknot direct ribosome pause correct position mrna complementary torsional restraint model address issue stem loop structure unwound elongating ribosome unwinding stem force loop rotate simple stem loop restrain loop rotate freely base pair stem resist ribosomal movement potential energy unwinding distribute length Stem figure loop anchor restrain pseudoknot Stem intrinsic ribosomal helicase processive Stem fully unwound Stem denature mechanically ribosome begin unwind base Stem Stem force supercoiling remainder Stem provide extra resistance ribosome movement specific point resistance ribosome movement provide supercoiling counteract forward movement ribosome increase likelihood ribosome stop precise point mrna energetically unwinding Stem dependent complete denaturation Stem potential energy unwinding pseudoknot structure similarly direct point view mechanically energetically point ribosome direct specifically pause mrna occur trnas ribosomal site position slippery site frameshifting stimulate summarize figure efficiency PRF view function fraction ribosome pause slippery site ii rate structure denature increase evidence single molecule experiment unfold occur quick rip particular force suggest case unfold pseudoknot frameshifting efficiency relate energy barrier unfold pseudoknot structure resistance structure force ribosome context torsional restraint model resistance dependent ability Stem remain intact Stem unwound experimental datum indirectly support model disruption bp Stem displace ribosome pause site point slippery site show eliminate frameshifting ii destabilizing Stem allow unwound readily show result decrease frameshifte efficiency iii replace bulge Stem base pair increase energy require unwind basis long ribosomal pause slippery site follow yield increase efficiency frameshifting iv destabilize base Stem replace base pair pair decrease frameshifting efficiency model eliminate need pseudoknot recognize factor evidence forthcoming competition assay vitro translation system gel retardation assay Dinman unpublished datum vi elimination potential torsion restrain Stem non torsion restrain Stem result decrease PRF efficiency cite study support torsional restraint model directly address experiment present study series pseudo pseudoknot contain reporter construct test torsional restraint hypothesis vitro frameshifting assay frameshifting significantly stimulate limit rotational freedom loop region stem loop structure degree rotational freedom Stem important determine extent PRF furthermore mrna toeprint analysis reveal pseudo pseudoknot specific strong stop nt slippery site consistent structure able direct ribosome pause site position slippery site synthetic dna oligonucleotide purchase IDT Coralville IA modify viral PRF signal contain GGU UUA slippery site follow simple stem loop amplify primer digest hindiii NcoI restriction site underline clone firefly luciferase reporter plasmid LUC minus region result reporter construct expression firefly luciferase require frameshift sequence Stem able base pair sequence stem loop pseudoknot able form primer amplify dna pJDRC ry construct complementary mutation acceptor sequence pseudoknot form region Stem allow formation mrna pseudoknot previously show promote frameshifte frequency wild type primer CON contain additional nucleotide upstream slippery site zero frame control eliminate possibility internal initiation occur luciferase initiation codon downstream frameshift signal AUG codon change AUA Stratagene Quik Change kit mutate ry ry respectively oligonucleotide mutate codon underline mutate zero frame control place upstream slippery site oligonucleotide TG insert base underline series synthetic dna oligonucleotide design join loop acceptor region mRNA transcribe downstream region form pseudoknot wild type PRF signal oligos sequence base pair loop mrna transcribe region oligos base pair downstream sequence orientation reverse oligos general orientation show figure naming oligonucleotide name refer number additional residue place region complementarity basis complementary sequence underline Plasmid DNAs prepare Qiagen mini prep kit linearize DraI total volume ml protein eliminate addition ml mg ml proteinase SDS final concentration follow digestion min Volumes increase ml extract twice phenol chloroform DNA precipitate ml NH Ac ml ethanol purified DNA resuspend DEPCtreated prepare synthetic mrnas ml purified linear dna vitro transcription Ambion mMachine mMessage kit RNAs precipitate ml depc ml LiAc RNA resuspend ml depc ml OD mg anneal oligonucleotide mrna oligos oligos equivalent volume dilution buffer mM Tris pH mM MgCl mM edta final concentration add synthetic mrna mg mixture incubate heating block min block remove allow cool min briefly spin incubate ice experiment molar ratio oligos synthetic mrnas experiment compete oligonucleotide oligo add molar ratio oligonucleotide Reticulocyte lysate thaw ice ml Àmet ml Àleu master mix plus ml add ml lysate ml add annealed reaction start vitro translation reaction incubate min reaction reach plateau min great difference see zero frame control frameshifte plasmid datum show reaction place ice aliquot ml vitro translation reaction add ml prewarmed luciferase reagent luminescence reading take delay triplicate Turner Luminometer synthetic transcript generate DraI digest kb end label RNAs ml incubate ml anneal buffer ml ml oligo ng heating block allow cool room temperature min ml rnaseh buffer add mm hepe mm KCl mm MgCl mM DTT aliquot ml enzyme add mung bean nuclease rnaseh reaction incubate reaction stop add ml stop solution product separate polyacrylamide urea denature gel visualize toeprinte mRNA mg ml anneal ml end label toeprinting primer complementary sequence bp slippery site describe add ml lysate ml Ambion retic lysate ml master mix Àleu Àmet ml mm KCl ml add ml RT buffer mM Tris HCl mm KCl mm MgCl mm DTT mm dntps ribosome control vitro translation reaction incubate room temperature min empirically determined provide optimum time allow ribosome initiate translation pause frameshift signal subsequently ml RT buffer contain RNasin inhibitor cycloheximide final concentration ng ml add stop translation ml Superscript II Invitrogen add reaction incubate room temperature min reaction terminate phenol chloroform extraction ml stop solution add toeprinting primer conjunction produce sequence ladder standard dideoxynucleotide chain termination method Sequenase usb product separate polyacrylamide urea denature gel visualize storm phosphorimager pseudoknot stimulate frameshifting frameshifte efficiency change degree pseudo pseudoknot rotational freedomWe previously show intact yeast cell pseudoknot contain mrna produce pjdrc able promote efficient PRF stem loop form transcribe step study test ability synthetic mrnas produce RC plasmid derive parental construct promote PRF total luciferase activity produce synthetic mrnas divide luciferase activity produce zero frame control plasmid multiply determine PRF efficiency result trend observe yeast replicate vitro RC mrna promote efficiency PRF compare promote mrna figure torsional restraint model predict condition inhibit rotational freedom loop region derive mrna result enhance PRF efficiency strategy study anneal mrna synthetic oligonucleotide complementary loop region sequence downstream normally involve pseudoknot formation pseudopseudoknot predict restore pseudoknot like structure mrna diagram Figure different class oligonucleotide have different orientation relative mrna end join reverse oligos orientation oligos promote formation structure contain equivalent Loop region oligos promote Loop equivalent model predict pseudo pseudoknot have different degree rotational freedom promote different frequency ribosome pause slippery site result different efficiency PRF order control parameter increase number nucleotide insert mrna hybridize region oligos additional non complementary basis oligos stem loop residue involve Stem effectively increase Loop similarly additional non complementary basis oligos loop acceptor residue correspond increase Loop structure stem loop maximum base pair interaction representative oligos show figure demonstrate oligonucleotide mrna hybrid capable form assay condition oligo incubate mrna subject rnaseh digestion digestion RNA DNA hybrid result label nt fragment demonstrate oligonucleotide bind mrna position pseudoknot figure demonstrate utility vitro frameshifting assay series oligonucleotide able hybridize synthetic mrna produce step monitor frameshifting efficiency promote hybrid specie significant increase frameshifting observe incubation mrna oligonucleotide modest increase see Figure finding consistent notion change degree rotational freedom structure affect distribution paused ribosome vicinity slippery site potential complication oligos possibility interact Loop Stem region oligos additional basis distal possible Loop stem interaction analogous increase Loop oligos stimulate PRF high extent oligos Figure importantly increase length bridging region oligonucleotide predict increase rotational freedom stem loop result decrease frameshifting activity predict torsional resistance model addition residue bind region oligo result unexpected increase frameshifte large variation series control experiment nt oligos complementary loop Stem form region pseudo pseudoknot hybridize sl mrna PRF assay perform able stimulate PRF concentration fold molar excess mrna template datum show supportive central hypothesis possible result thermodynamic instability RNA dna duplex course experimental protocol determine stimulation frameshifting specifically bridging stem loop downstream sequence pseudo pseudoknot oppose nonspecific presence RNA dna hybrid compete oligonucleotide oligo design form bp duplex mrna include Stem form region expect significantly compete oligos bind site disrupt formation pseudo pseudoknot Figure additionally presence oligo oligos predict hybridize Stem form region enable address question interaction able stimulate frameshifting result demonstrate addition oligo severely inhibit ability oligos promote efficient frameshifting figure finding demonstrate frameshifting specifically stimulate bridging Stem form region oligos ii presence RNA dna hybrid Stem form region sufficient stimulate frameshifte torsional restraint model predict pseudoknot direct elongate ribosome pause specific location efficient frameshifting stimulate pseudo pseudoknot vitro translation assay perform retic lysate mrnas derive sl oligos anneal luciferase activity divide obtain mrnas generate result ratio multiply calculate percent frameshifting average independent experiment perform triplicate show error bar denote standard deviation mrna distribute Stem mrna toeprint assay test hypothesis mrna toeprint reaction movement reverse transcriptase block paused ribosomes result strong stop position nt site eukaryotic ribosome synthetic mrnas anneal sequence oligonucleotide second oligo vitro translation reaction period time min empirically determined optimal elongation reaction stop addition cycloheximide reverse transcription reaction initiate sequencing oligonucleotide parallel control reverse transcription reaction carry synthetic mrna oligonucleotide vitro translation result consistent model show oligos specifically promote strong reverse transcriptase stop nt site slippery site vitro translation reaction figure predict model broad distribution stop equal intensity observe region mrna figure lane importantly stop observe toeprint reaction carry absence ribosome additional strong stop interest correspond end base Stem observe sample consistent presence structure oligo specific pause observe reason strong pause oligo unknown oligo specific pause revealing occur end RNA DNA hybrid form oligo mrna structure promote pausing reverse transcriptase result present study provide strong support torsional restraint model program frameshifting specifically demonstrate RNA DNA hybrid mimic mrna pseudoknot significantly stimulate frameshifting predict model change rotational freedom structure alter length oligos mrna hybridizing region result change ability stimulate frameshifting demonstration pseudopseudoknot structure cause elongate ribosome specifically pause site position slippery site provide independent evidence support model case oligo series frameshifting well stimulate suggest structure create rotational freedom allow optimal PRF experimental design assume similar rate unfold oligo predict maximum base pairing note type nucleotide separate separately pair region oligos presentation play role PRF efficiency recent NMR structural solution pseudoknot reveal highly structure Loop stem interface include base triple involve residue end loop additional base Mutagenesis experiment region group show example replace base Loop IBV residue promote significant increase frameshifte efficiency mutation removal residue base Loop BWYV pseudoknot reduce frameshifting level pseudoknot propose important frameshifting model differential transition state energy barrier small difference local structure stability dynamic primary determinant frameshifting efficiency Loop stem triplex interaction see small frameshifte pseudoknot luteoviruse show critical PRF similar pseudoknot lack triplex efficient frameshifte reference extra structural feature limit rate unfold provide extra anchoring Stem ribosome attempt unwind Stem help provide additional torsional restraint possible help form pseudo pseudoknot additional basis interfere stabilization Stem oligos general correlation observe minimization rotational freedom frameshifting efficiency case oligo stability pseudo pseudoknot generate similar oligonucleotide base base pairing result suggest additional consideration uncover regard Frameshifting stimulate figure competition oligo bind site inhibit ability promote efficient frameshifting mrna transcribe sl anneal oligos combination different concentration compete oligo ratio equimolar indicate Sample mark SL mrna luciferase activity generate vitro translation reaction rabbit reticulocyte lysate divide obtain mrnas generate result ratio multiply calculate percent structure influence frameshifting addition residue oligos analogous lengthen Loop typically short frameshifte pseudoknot limited conflicting datum available importance Loop frameshifte pseudoknot study addition basis Loop affect frameshifte efficiency mutation region detrimental frameshifte efficiency give complex interaction occur helix loop region account oligo stimulate frameshifting efficiently variable result examination RNA toeprint datum present reveal pseudo pseudoknot structure form oligos promote strong stop reverse transcriptase nt site codon slippery site consistent hypothesis presence Stem force ribosome pause site position slippery site previous study map lagging edge paused ribosome mrna heelprint study reveal striking difference effect pseudoknot versus stem loop interestingly method ribosomal pause appeared distribute broad stretch mRNA nt observe possible critical level resolution lose requirement additional manipulation substrate mrna heelprint compare toeprint method remain question center role RNA pseudoknot PRF passive active solution frameshift mechanism activate movement site codon anticodon complex base direction accommodation currently describe mrna pseudoknot merely passively block entry downstream message ribosome result stretching segment mrna locate codon anticodon complex pseudoknot model energetic input frameshift derive hydrolysis gtp possible pseudoknot actively contribute frameshift mechanism specifically pull downstream message ribosome accommodation result unwinding Stem pseudoknot additional base pair energetic cost introduce equivalent torsional resistance Stem Stem release resistance pull base pair Stem form turn contribute energy require dissociate site codonanticodon complex zero frame follow slippage mrna base direction relative ribosome follow formation frame codon anticodon complex propose active role mrna pseudoknot reduce energetic barrier PRF sum suggest torsional restraint model combine solution mechanistically explain original simultaneous mRNA Ribos mRNA figure pseudo pseudoknot direct ribosome pause slippery site mrnas generate sl anneal sequence oligonucleotide oligo lane respectively vitro translation reaction reaction stop min addition cycloheximide reverse transcription reaction initiate sequencing oligonucleotide parallel control reverse transcription reaction carry synthetic mrna oligonucleotide vitro translation lane position slippery site loop stem pseudo pseudoknot indicate sequence reaction arrowhead indicate position reverse transcriptase strong stop map representation stem loop structure slippage model PRF word solution torsional restraint simultaneous slippage recent publication show oligonucleotide mrna duplex stimulate efficient ribosomal frameshifting study differ present number way particularly insofar examine effect duplex structure immediately slippery site address mrna pseudoknot related question finding support notion specific location ribosome pause mrna play critical role determine frameshifting come caveat study directly map ribosomal pausing use different slippery site downstream context likely contribute disparate finding optimal distance end slippery site end frameshift stimulate oligonucleotide potentially useful therapeutically known natural example frameshifting stimulate manner result affect hypothesis present study important raise possibility new role micro rnas regulate gene expression therapeutic approach correct inborn error metabolism presence frameshift mutation','2005-03-30 00:00:00','http://dx.doi.org/10.1093/nar/gki329',2556,775),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json','Public awareness of risk factors for cancer among the Japanese general population: A population-based survey','BACKGROUND: The present study aimed to provide information on awareness of the attributable fraction of cancer causes among the Japanese general population. METHODS: A nationwide representative sample of 2,000 Japanese aged 20 or older was asked about their perception and level of concern about various environmental and genetic risk factors in relation to cancer prevention, as a part of an Omnibus Survey. Interviews were conducted with 1,355 subjects (609 men and 746 women). RESULTS: Among 12 risk factor candidates, the attributable fraction of cancer-causing viral and bacterial infection was considered highest (51%), followed by that of tobacco smoking (43%), stress (39%), and endocrine-disrupting chemicals (37%). On the other hand, the attributable fractions of cancer by charred fish and meat (21%) and alcohol drinking (22%) were considered low compared with other risk factor candidates. For most risk factors, attributable fraction responses were higher in women than in men. As a whole, the subjects tended to respond with higher values than those estimated by epidemiologic evidence in the West. The attributable fraction of cancer speculated to be genetically determined was 32%, while 36% of cancer was considered preventable by improving lifestyle. CONCLUSION: Our results suggest that awareness of the attributable fraction of cancer causes in the Japanese general population tends to be dominated by cancer-causing infection, occupational exposure, air pollution and food additives rather than major lifestyle factors such as diet.',' Japan cancer recognize major component overall pattern disease decade importance cancer prevention lifestyle modification strongly acknowledge internationally study estimate proportion total cancer death attributable risk factor base epidemiologic evidence international guideline recommendation derive appear surprisingly domestic guideline recommendation cancer prevention Japan recommendation cancer prevention Healthy People Japan significantly influence report public awareness risk factor relation cancer prevention survey country result demonstrate poor awareness study focus specific cancer appear study quantitatively evaluate public awareness attributable fraction individual risk factor Japan appear people aware major risk factor cancer unaware publish evidence public knowledge information cancer prevention influence largely mass medium source information provide directly health professional result distorted picture causation cancer control policy urgently require clarification discrepancy exist ideal level public concern risk factor current reality particularly public health policy maker formulation cancer control measure address need present study design provide information awareness attributable fraction cancer cause japanese general population interested quantitatively estimate awareness preventability place special emphasis gauge awareness attributable fraction cancer study conduct omnibus survey December commission polling agency omnibus survey monthly multipurpose cross sectional survey include public opinion research social research scientific research market research stratified stage sampling method total people age old randomly select study subject district select area unit represent geographical block Hokkaido Tohoku Kanto Keihin Koshinetu Hokuriku Tokai Kinki Hanshin Chugoku Shikoku Kyushu type city scale metropolis city town village proportion population distribution March initial visit obtain oral informed consent schedule visit interview survey conduct face face interview train interviewer district omnibus survey collect personally identifiable information date birth address detail interview present report obtain electronic datum file relevant interview component personal identifier ethical approval applicable present study japanese ethical guideline epidemiologic study comply declaration Helsinki people select survey man woman interview successfully obtain questionnaire survey comprise question awareness environmental genetic risk factor relation cancer prevention enquire attributable fraction cancer fraction risk factor candidate alcoholic beverage unbalanced diet use food additive pesticide chemical char fish meat tobacco smoking obesity physical inactivity endocrine disrupt chemical air pollution diesel emission occupational exposure cancer cause viral bacterial infection stress genetic factor general preventable fraction cancer occurrence lifestyle modification additional file question ask preventable fraction cancer result Japan factor completely totally eliminate fine category category exhibit pie chart risk factor candidate select reference previous international domestic recommendation guideline second question ask fraction cancer genetically predetermine category ask preventable fraction cancer modification lifestyle estimation actual percent value addition question subject ask smoking drinking practice occupational educational status mean value attributable fraction calculate risk factor cancer compare demographic habitual smoking drinking status analysis mid value category assign categorical variable analysis perform Stata statistical software Version total subject respond survey high response rate woman man response rate low age strata age group trend increase response rate increase age observe overall significant difference area age distribution see sample population survey respondent response rate tend low subject reside Kanto region city metropolis subject table respondent man woman present Table proportion current smoker man woman decrease age gender female subject age currently smoke drink alcohol beverage time week awareness attributable fraction cancer cause japanese general population present Table risk factor candidate attributable fraction consider high cancer cause viral bacterial infection follow tobacco smoking stress endocrine disrupt chemical contrast attributable fraction charred fish meat alcohol drinking consider low compare risk factor candidate attributable fraction risk factor candidate occupational exposure air pollution food additive pesticide unbalanced diet obesity physical activity rank high low fraction attributable fraction response tend high woman man increase inhabitant large city homemaker decrease engage agriculture forestry fishery contrast risk factor candidate ranking similar gender age group city scale educational occupational status man smoke drink tend consider preventive fraction risk factor high smoke drink woman subject consider value low speculate fraction cancer genetically determine average table fraction high current heavy smoker drinker low homemaker student hand average cancer consider preventable lifestyle improvement ratio high homemaker smoker drinker present survey target japanese general population show attributable fraction cancer Japanese tend high cancer cause infection occupational exposure air pollution food additive major lifestyle factor dietary factor addition attributable fraction cancer estimate japanese general population high derive epidemiologic evidence West frequently quote tobacco smoking food major cancer Japan include gastric liver cancer know relate cancer cause viral bacterial infection high level concern infection Japanese western population understandable notwithstanding validity concern high level concern infection endocrine disrupt chemical identify present survey likely severe acute respiratory likewise high level concern tobacco smoking observe spite relatively dull reduction rate male current smoker past decade compare probably recent enactment Health Promotion Law curb passive smoking public space respondent estimate attributable fraction generally high anchor adjustment effect response category tendency people respond near middle scale give response tend generally high concern present result probably focus ranking absolute value se tobacco smoking rank factor risk factor candidate actual contribution consider low endocrine disrupt chemical occupational exposure air pollution diesel emission use food additive pesticide chemical rank high previous estimate attributable fraction cancer cause contrast compare result unbalanced diet rank risk factor candidate despite actual ranking estimate high tobacco smoking particularly light finding long term exposure common lifestyle factor diet cause cancer result suggest public awareness cancer prevention insufficient unaware previous study aim determine public awareness attributable fraction cancer gauge level awareness cancer prevention attributable fraction accordingly knowledge attempt discover level awareness risk factor candidate questionnaire fully validate addition indicate response type cross sectional survey subject social condition information mass medium source disease epidemic putative risk factor result necessarily reflect actual public awareness study subject recruit nationally representative random sample response rate similar recent omnibus survey country exclusion non respondent distort result conclusion awareness attributable fraction cancer cause japanese general population tend dominate infection occupational exposure air pollution food additive dietary factor result present survey provide valuable clue perspective formulation relevant cancer prevention strategy Japan','2006-01-10 00:00:00','http://dx.doi.org/10.1186/1471-2458-6-2',960,395),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json','Molecular dynamics simulations of human [Formula: see text]: the role of modified bases in mRNA recognition','Accuracy in translation of the genetic code into proteins depends upon correct tRNA–mRNA recognition in the context of the ribosome. In human [Formula: see text] three modified bases are present in the anticodon stem–loop—2-methylthio-N6-threonylcarbamoyladenosine at position 37 (ms(2)t(6)A37), 5-methoxycarbonylmethyl-2-thiouridine at position 34 (mcm(5)s(2)U34) and pseudouridine (ψ) at position 39—two of which, ms(2)t(6)A37 and mcm(5)s(2)U34, are required to achieve wild-type binding activity of wild-type human [Formula: see text] [C. Yarian, M. Marszalek, E. Sochacka, A. Malkiewicz, R. Guenther, A. Miskiewicz and P. F. Agris (2000) Biochemistry, 39, 13390–13395]. Molecular dynamics simulations of nine tRNA anticodon stem–loops with different combinations of nonstandard bases were performed. The wild-type simulation exhibited a canonical anticodon stair-stepped conformation. The ms(2)t(6) modification at position 37 is required for maintenance of this structure and reduces solvent accessibility of U36. Ms(2)t(6)A37 generally hydrogen bonds across the loop and may prevent U36 from rotating into solution. A water molecule does coordinate to ψ39 most of the simulation time but weakly, as most of the residence lifetimes are <40 ps.',' Nonstandard basis prevalently find trna molecule involve variety modification simple methylation addition peptide moiety trna molecule require posttranscriptionally modify nucleic acid basis proper translation recognition protein initiation human immunodeficiency virus reverse transcription specifically ribosomal binding proper translocation aminoacyl site peptidyl site site translational speed dependent nonstandard basis lack modified basis result frameshifte error trnas contain modify base modification occur position modified basis generally think increase stability trna structure provide additional hydrogen bond contact well base base stacking interaction metal ion bind site specifically base modification position trna important facilitate cognate codon recognition role specific modification play determine RNA structure stability function understand functional group modify nucleoside dramatically affect ability trna anticodon recognize cognate mrna codon participate accurate translation genetic code human trna coding lysine tRNA contain posttranscriptionally modify basis anticodon stemloop ASL Figure biochemical assay establish thiouridine position mcm methylthio threonylcarbamoyl adenosine position ms require mimic wild type human trna binding programmed ribosome Pseudouridine presence position critical anticodon codon recognition show contribute helical stability Escherichia coli tRNA Lys analogous human counterpart contain thiouridine position mnm threonylcarbamoyl adenosine position important ribosomal site binding translocation site discrimination AAA AAG codon structure human trna tRNA Lys variant variously modify basis variety context determine NMR spectroscopy ray crystallography expensive complex synthesis nonstandard basis complicate incorporation fully modify basis structural study indicate preservation turn motif anticodon loop important recognition cognate mrna systematic detailed NMR study fully modify partially modify human trna indicate flexibility position context ribosomal subunit anticodon loop contain find recognize cognate mrna codon canonical Fuller Hodgson trna form contribution modify base free trna ribosomally bind trna structure remain elusive molecular dynamic MD study consider specific contribution modify basis mcm ms structure function human ASL report specifically role modify basis preserve canonical trna anticodon structure examine fully refine NMR structure currently available fully modify tRNA simulation begin ray crystal structure md simulation trna UUU anticodon stem loop basis ASL wild type derive fully modify trna ray crystal structure solve resolution pdb code perform different combination mcm ms unmodified basis table variation ASL derive Bénas et al structure study conduct crystal structure ASL Lys contain ribosomal subunit mrna solve resolution pdb code ASL Lys ureido ring stack stabilize anticodon codon interaction ensure adequate sampling relevant structure additional system wild type derive graft ASL Lys anticodon loop contain ureido ring Bénas ASL stem apply appropriate human ASL modification MD simulation perform accord protocol trajectory propagate Cornell et field implement SANDER module program AMBER ASLs solvate rectangular box water Na ion add neutralize system simulation perform periodic boundary condition employ particle mesh Ewald formalism account long range electrostatic md trajectory propagate fs time step employ shake algorithm hydrogen atom nonbonded cutoff pairlist update step constant pressure atm temperature maintain accord Berendsen coupling algorithm equilibration protocol Cheatham follow similar previous simulation nucleic acid simulation allow equilibrate production run condition ns collection statistic ns parameter nonstandard basis derive consistent Cornell et al force field specific parameter available Supplementary Data atom type force constant equilibrium value assign analogy Cornell et al nucleic acid threonine chain methionine ms force field parameter General Amber Force Field sulfur relate parameter partial atomic charge derive Hartree Fock electrostatic potential charge fit step restrain electrostatic potential procedure electronic structure calculation perform Gaussian revision mechanical calculation employ modify Perdew Wang parameter model kinetic basis set determine rotational barrier amide bond involve ureido ring formation trajectory analyze CARNAL PTRAJ module amber statistic collect percentage simulation anticodon exhibit stair step conformation left SASA base right modified basis include simulation indicate absence modification indicate unmodified base substitute position ps snapshot ns datum helical parameter measure program trajectory structure visualize VMD UCSF chimera criterion hydrogen bonding distance heavy atom heavy atom heavy atom angle Solvent accessible surface area SASA calculate Connolly surface implement MM pbsa module amber determine basis parallel angle normal vector plane respective basis plane define point pyrimidine purine normal vector define point direction overall helical portion ASL structure exhibit classic form geometry displacement value range base pair rise parameter majority structural analysis occur loop region ASLs wild type simulation unusually stable average starting structure ensure wild type simulation naturally stable periodic effect lack sample time influence result datum additional simulation collect McCrate unpublished datum simulation water box surround RNA increase solute simulation water add solute datum collect n additional simulation retain structure observe report wild type simulation presence mcm ms widen loop region simulation contain hydrogen bond exocyclic amine vary extent simulation average occupancy simulation contain modified basis exhibit additional hydrogen bond exocyclic amine simulation time respectively addition mcm ms decrease interaction simulation time chain mcm form hydrogen bond part ASL detectable hydrogen bond observe occur mcm carbonyl oxygen hydroxyl ms wildtype simulation simulation contain mcm ms simulation time model present correlation presence hydrogen bond conformation simulation contain mcm lose hydrogen bond simulation lack modification position close form hydrogen bond simulation time exception trend wild type simulation contain hydrogen bond wild type wild type simulation respectively fully modify ASL structure include mcm ms wild type ASL structure retain stair step conformation uridine figure offset nearly parallel observe previously ray crystal structure free human trna ASL Lys pair cognate mrna codon context ribosome characterize stair step conformation relative position anticodon basis determine analyze position anticodon basis relative distance center mass COM anticodon basis aromatic ring measure distance vector transform relative coordinate system utilize rotation matrix decompose dx dy dz component describe relative position basis analogy dx dy dz component describe height width length stair visual inspection wild type simulation determined exhibit prototypical stair step conformation determine geometry anticodon model match wild type dx dy dz set ring classify stair step value pair ring fall sd average value calculate wild type model angle normal vector plane basis table aid datum analysis snapshot simulation assess presence stair step conformation base criterion outline percent occupancy stair step interaction position determine table simulation contain ms modification position exhibit stair step anticodon configuration simulation time asls lack base modification position stair step simulation wild type model anticodon appear stair step base position close ms observe model result difference position wild type simulation classify have stair step codon model stair step slightly different conformation ms substitute rotate glycosidic bond expose solvent observe simulation lack ms modification position Quantification solvent exposure SASA base indicate presence ms modification position reduce SASA table relatively small SASA value correspond stair step conformation large SASA value correspond unstacking addition mcm position appear slightly low SASA model lack mcm modification slightly high SASA modify counterpart table visual inspection wild type simulation basis interact basebase stacking interaction simulation examine determine extent base base stacking anticodon loop stacking interaction define occur distance measure COM aromatic ring angle normal vector plane form basis determined appropriate previous study stack criterion outline percent occupancy stack interaction position determine table simulation contain ms portion simulation close proximity membered ring stacking datum indicate presence ms slightly enhance base base stacking position encourage base remain intercalate general modify unmodified hydrogen bond functional group anticodon loop simulation contain ms tend interact loop base sugar atom residue relatively short lifetime instance wildtype trajectory hydroxyl group hydrogen bond simulation time group trna prefer interact sugar oxygen residue hydrogen bond base oxygen carbonyls ms modification remove disrupt interaction form new bond basis close proximity residue absence ms ASL rearrange provide new hydrogen bond partner truncate position context ribosome contain ureido ring stack membered ring crystal structure contain ureido ring fully refined trna ray structure pdb code ms adopt conformation ureido ring initial attempt try clear electron density show ureido ringlike conformation Bénas personal communication figure Ref Quantum mechanical calculation methylate ms level theory indicate barrier rotation bond form ureido ring kcal mol room temperature present water bridge form water bridge present production run detailed table Supplementary Data component hydrogen bond water present large percentage production run typically water involve water bridge short live bulk bridging interaction attributable water molecule have lifetime ps nearly residence lifetime attributable water lifetime ps observe tRNA Phe ASL study involve trna Lys variant modification act sterically anticodon loop widen loop prevent closed structure consistent previously observe closed loop conformation observe experimentally NMR spectroscopy addition mcm position ms position introduce steric hindrance loop widen interaction long observe md simulation Cornell et al force field water show identify long live structural water molecule system water molecule coordinate present time long live pseudouridine show stabilize trna structure water mediate hydrogen bond network crystal structure yeast tRNA Phe contain water molecule bridging position neighbor phosphate oxygen additionally long live water molecule coordinate find trna Asp percent simulation time base stacking present interaction basis show COM distance basis determine aromatic ring indicate indicate membered ring indicate membered ring indicate ureido ring ms modify basis include simulation indicate absence modification indicate unmodified base substitute md simulation water mediate hydrogen bond believe restrict motion stabilize base pair prevalently find position wild type simulation retain stair step conformation dx dy dz value similar find ray crystal structure human trna asl bind ribosome modification position stabilize anticodon conformation datum appear presence ms encourage retention highly order stair step conformation base modification position stabilize participation stair step conformation base anticodon require proper reading cognate codon base ms base require proper positioning recognize cognate codon base uridine hydrophilic carbonyls general predispose extra helical position result increase solvent exposure note systematic study RNAs contain single base bulge highly order stair step conformation anticodon basis important codon recognition originally hypothesize observe crystal structure trnas interact trnas crystal packing effect context ribosome base stacking datum indicate modification position base well interact stack interaction long live simulation order favorable base base stacking interaction interaction hydrophobic basis enthalpically favorable case negative electron density aromatic basis repel electrostatic interaction positive portion base interact negative electron density base predominate favorable electrostatic interaction observe simple system benzene water benzene water pair partially positive hydrogen water molecule interact negative cloud benzene ring favorable canonical anticodon stairsteppe positive portion basis align interact negative cloud neighboring low base modify partake hydrogen bond anticodon loop effectively hold modification remove base long enthalpically favorable hydrogen bond interaction rotate solution leave unmodified favorably interact electrostatic interaction hydrogen bond close basis truncated amino group bulky ms modification contain hydrogen bond donor acceptor act sterically restrict movement discourage displacement weak stacking interaction stabilize position hydrogen bond contact loop wild type simulation bulk substituent hydrogen bond donor acceptor solvent expose exhibit markedly increase conformational freedom behavior suggest delicate balance force cross strand hydrogen bonding portion prevent displacement figure study indicate SCH ms weaken ureido hydrogen bond interaction compare take quantum calculation indicate facile interconversion room temperature ureido ring conformation non ureido ring structure conformation change base context ribosome ribosomal subunit primarily responsible decode find recognize base pair anticodon codon complex minor groove trigger closure small subunit trna system ureido ring allow great stack interaction possibly give flexibility adjust codon prokaryotic eukaryotic ribosome contain conserve region rrna decode site mrna crosslink correspond residue human subunit human trna rotate moiety bond allow chain hydrogen bond loop stabilize correct positioning ribosome canonical fashion accuracy trna mrna recognition dependent kinetic proofreading mechanism stability cognate codon anticodon complex role modify basis stablize codon anticodon complex oppose destabilize loop conformation induce fit ribosome base system dependent datum available NAR Online National Science Foundation Truman State University fund pay Open Access publication charge article provide Petroleum Research Fund conflict interest statement declare','2006-09-29 00:00:00','http://dx.doi.org/10.1093/nar/gkl580',1880,628),('ada57a9f084a016deb20b6f83c442072e6e37955.json','The \'polysemous\' codon--a codon with multiple amino acid assignment caused by dual specificity of tRNA identity.','In some Candida species, the universal CUG leucine codon is translated as serine. However, in most cases, the serine tRNAs responsible for this non-universal decoding (tRNA(Ser)CAG) accept in vitro not only serine, but also, to some extent, leucine. Nucleotide replacement experiments indicated that m1G37 is critical for leucylation activity. This finding was supported by the fact that the tRNA(Ser)CAGs possessing the leucylation activity always have m1G37, whereas that of Candida cylindracea, which possesses no leucylation activity, has A37. Quantification of defined aminoacetylated tRNAs in cells demonstrated that 3% of the tRNA(Ser)CAGs possessing m1G37 were, in fact, charged with leucine in vivo. A genetic approach using an auxotroph mutant of C.maltosa possessing this type of tRNA(Ser)CAG also suggested that the URA3 gene inactivated due to the translation of CUG as serine was rescued by a slight incorporation of leucine into the polypeptide, which demonstrated that the tRNA charged with multiple amino acids could participate in the translation. These findings provide the first evidence that two distinct amino acids are assigned by a single codon, which occurs naturally in the translation process of certain Candida species. We term this novel type of codon a \'polysemous codon\'.',' term trna Ser CAG reveal decode mechan Bioscience Biotechnology Tokyo Institute Technology ism mean vitro translational assay system Nagatsuta Midori ku Yokohama Japan Yokogawa et al Suzuki et al furthermore correspond author investigate distribution non universal genetic code fungus Candida specie universal CUG leucineCandida codon translate serine case serine trnas responsible non universal dii find utilize codon CUG decode trna Ser CAG accept vitro serine serine instead leucine have trna Ser CAG extent leucine nucleotide replacethe mediator unusual decoding Ohama et al ment experiment indicate critical Ueda et al investigator leucylation activity finding support show codon CUG actually translate serine fact trna Ser cag possess leucylation vivo Santos Tuite activity Candida Sugiyama et al Zimmer Schunck remarkable structural feature observe Quantification define aminoacetylate trnas trna Ser cag nucleotide cell demonstrate trna Ser cag adjacent anticodon position occupy possess fact charge leucine conserve residue residue speculate necessary form possess type trna Ser CAG turn structure anticodon loop trnas suggest gene inactivate reported far Quigley Rich Sprinzl et al translation CUG serine rescue slight nucleotide position adjacent incorporation leucine polypeptide anticodon CAG methyl guanosine demonstrate trna charge multiple trna Ser cag amino acid participate translation serine trnas fungus correspond finding provide evidence distinct Introduction Normanly Abelson Shimizu et al McClain Schimmel et al line study universality genetic code consider begin artificial conversion leucine trna essential characteristic life lead Escherichia coli serine tRNA Abelson group conception frozen accident theory year ago Normanly et al recently trna identity theory propose extant live organism use element Saccharomyces cerevisiae leucine trna universal genetic code bear accident elucidated unmodified variant synthesize frozen originate single closely RNA polymerase Soma et al indicate interbreed population Crick recent addition discriminator base second letter year number non universal genetic code anticodon nucleotide adjacent report non plant mitochondrial system anticodon important recognition nuclear system review Osawa leucyl trna synthetase LeuRS majority Candida et al Osawa contradict frozen trna Ser cag discriminaccident theory deviation universal code ator occupy nucleoside adenosine respect trna ser CAG potentially chimeric trna molecule capable recognize seryl leucyl trna synthetase previously show trna Ser cag originate serine trna correspond codon UCG suggest evolutionary pathway conversion take place position emergence trna Ser CAG bring change universal code mutation position potentially result leucylation trna Ser CAG attempt elucidate charge property trna Ser cag vitro vivo base result vitro aminoacylation reaction trna variant construct microsurgery method direct analysis aminoacylate trnas cell genetic approach demonstrate serine trnas actually leucylate vitro vivo furthermore position find indispensable leucylation trna Ser cag fact trna Ser CAG position exhibit leucylation activity high GϩC content utilize CUG major serine codon Candida specie high GϩC content utilize CUG minor serine codon Kawaguchi et al Lloyd Sharp unpublished observation consider relationship usage codon CUG serine leucylation property trna Ser CAG Candida specie genome incidence CUG serine codon low possess serine trna Ser CAG leucylate furthermore trna Ser cag charge heterogeneous amino acid utilize equally translation process demonstration single trna specie assign different amino acid cell propose designate type codon have multiple amino acid assignment polysemous codon correlation dual assignment state pathway genetic code diversification discuss Yokogawa et al Ohama et al number system abbreviation modify nucleotide conform Candida zeylanoide trna Ser CAG leucylate Sprinzl et al Crain McCloskey respectively vitro time dependent aminoacylation SerRS LeuRS leucylation trna Ser cag cell aminoacylation reaction carry examine LeuRS partially µm trnas amount enzyme activity calculate cognate trnas serylation leucylation show dotted purify know leucine solid line respectively right hand frame show solid trnas yeast identity determinant curve left hand frame plot enlarged ordinate position Soma et al trna Ser cag aminoacylation trna Ser CAG different trna Ser CAG compare trna Ser GCU have leucylation activity show tide position respectively figure control TLC analysis acetylleucyl trna fragment derive trnas show serylation activity leucylated trna Ser cag leucylation pmol unit show figure leucyl trnas acetylate acetic anhydride Acetyl trna Ser CAG evidently leucylate leucylate case specie trna Ser CAG gel electrophoresis acidic observe leurs datum show serine trna specific codon AGY responsible recognition trna Ser GCU employ control substrate leucyl trna synthetase figure right hand graph value trna Ser cag Candida specie oide LeuRS trna Ser CAG µm unique possess order magnitude large serylation leucylation capacity sequence comparison tRNA µm leucylation trnas figure prompt speculate cognate leucine trnas µm Soma nucleotide position strongly associate et al trna Ser cag possess leucyl order verify leucylation activity observe ation activity common trna Ser CAG actually come trna Ser CAG possess leucylthe trna Ser CAG trace ation activity position leucine trna contaminate trna sample examine validity speculation series leucylated terminal RNA fragment derive leucyl tRNA ser CAG variant construct vitro trna Ser CAG analyze follow manner Ctranscription method RNA polymerase leucylate trna Ser CAG microsurgery method leucylation activity acetylate acetic anhydride prevent deacylation variant measure trna Ser CAG digest RNase result synthesize vitro transcription terminal fragment label acetylleucine employ substrate leucylation activity analyze cellulose TLC result show detect trna transcript have Figure leucylate trna Ser CAG digest Figure hand show Figure RNase label acetylleucyl CCA serylation activity exceed pmol unit release label fragment figure lane result strongly suggest nucleoside modifica locate position trna Ser CAG figure tion necessary trna Ser CAG recognition left hand structure contaminate leucine trnas LeuRS attempt replace exist label fragment large trna Ser CAG variant symbolthan tetramer figure lane ized microsurgery leucine trnas yeast far analyze Sprinzl et al method figure detail Materials include Suzuki unpuband method examine contribution lishe result know end leucylation contribution resistant RNase mobility trna Ser CAG prevention leucylation acetylleucyl oligonucleotide derive trna Ser CAG aminoacylation figure lane identical examine Figure result indicate acelylleucyl CCA prepare RNase substitution lead complete loss leucylation figure digest leucyl trna Leu lane right hand graph apparent influence observation clearly demonstrate leucine observe serylation figure left hand graph definitely attach trna possess trna finding strongly indicate methyl group trna Ser CAG trna Leu play crucial role enhance leucylation activity conclude trna incorporate leucine trna Ser CAG vitro fact trna Ser CAG deduction supportedthe slight reduction leucylation activity observe result additional experiment incorporation control variant Figure compare trna ser CAG sample LeuRS native tRNA figure right hand graph find reduce addition SerRS non label result partial deacetylation acetyl serine reaction mixture datum show cytidine ac acid treatment half clearly indicate trna molecule comfragment trna Ser CAG Materials method petitively aminoacylate enzyme consider discussion conclude trna Ser CAG aminoacylate leucine carry experiment act modulator leucylation trna Ser CAG charge serine serylate addition unique feature serine trna Ser cag Candida specie presence trna Ser CAG separate non aminoacylate effect mutation position variant confirm replace expect show trnas find different case tRNA mutation position cause leucylation trna observe native trna Ser CAG reduction position pyrimidine completely conserve usual trnas Sprinzl et al serylation activity figure contrast replacement pyrimidine trna Ser CAG consider possible notable feature way relate unusual considerably enhance leucylation activity figure right hand graph significant difference aminoacylation characteristic describe translation non universal genetic code examine observe serylation activity figure left hand graph kinetic parameter leucylation effect residue aminoacylation spot correspond acetylleucine acetylserine marker respectively analysis acetylamino acid attach trna fragment TLC plate Lane show spot correspond acetylamino acid derive RNase fragment trna Ser CAG lane indicate spot correspond acetylleucine acetylserine respectively microgram trna Ser CAG digest RNase develop cellulose TLC plate condition CCA fragment acid scrape plate fragment elute desalt Sep pak condition describe literature Wang et al acid discharge fragment develop TLC visualize imaging analyzer Fuji Photo Systems variant trna show Table itevidence leucylation trna Ser CAG vivo notable value pyrimidine point establish trna Ser CAG mutant µm µm actually able accept leucine vitro clearly low purine mutant consider fact SerRS LeuRS µM µm max value coexist cell judge value pmol min pmol min affinity trna Ser CAG SerRS order explain show low leucylation magnitude high LeuRS need activity despite have nearly ascertain trna Ser CAG value figure right hand graph judge fact leucylate vivo purpose adopt sequence analysis datum show slight reduction newly develop method quantify individual leucylation µm compare aminoacyl trna cell Suzuki et al native trna Ser CAG µm probably Aminoacyl trnas separately prepare cell partial deacetylation ac position immediately submentione confirm observation jecte acetylation anhydride label slight reduction leucylation activity acidthe amino acid stabilize aminoacylated treat native trna Ser CAG datum show trnas acetylated aminoacyl tRNA conclude replacement pyrimidine purine mixture trna Ser cag position repressive effect leucylation racea fish solid phase attach DNA trna Ser CAG describe previously Tsurui et al Wakita translation efficiency variant muta et al single band aminoacyltion position examine cell free trnas detect stain figure translation system Yokogawa et al radioactivity coincide case figure Suzuki et al evaluate effect amino acid attach trnas change position apparently enhance deacylate alkaline treatment analyze TLC translation activity fold decode show Figure acetylserine observe property change datum show major amino acid derivative trna Ser cag consider serve modulator leucylation acetylleucine detect trna Ser CAG despite slight disadvantage transla trna Ser CAG acetylserine acetylleucine spot identify describe previously Suzuki et al tion activity radioactivity remain origin probably come direct acetylation nucleotide trnas discuss previously Suzuki et al comparison radioactivity acetylserine calculate trna Ser CAG attach acetylleucine result reproducible digestion purified acetyl aminoacyl trna Ser CAG RNase give label CCA fragment show figure acetylate amino acid release fragment purify corresponding spot TLC analyze TLC ratio acetylleucine acetylserine find figure indicate acetylleucine covalently attach trna ser CAG fragment clear trna Ser CAG fact charge leucine serylation trna Ser CAG cell Aminoacylation generally consider final stage determine translational accuracy review Parker Kurland Farabaugh case trna Gln charge glutamate chloroplast Glu tRNA Gln reject elongation factor chloroplast translation machinery employ mischarge aminoacyl tRNA Stanzel et al likely exceptional case lack glutamyl trna synthetase chloroplast order prove leucylate trna Ser cag actually participate translation process Candida cell rejection mechanism utilize gene expression system derive develop Sugiyama et al Candida maltosa utilize codon CUG serine possess relevant trna Ser CAG gene Sugiyama et al Zimmer Schunck et al present study gene CTG codon replace leucine serine codon utilize marker gene figure Ohkuma et al growth monitor minimal medium sd plate plasmid gene insert downstream specific promoter presence absence uracil uracil supply sd plate construct designate pcsu CTG Sugiyama control mutant plasmid pcsu positive control experiment transformant grow normally figure middle row absence CTG codon CTG replace serine codon TCT leucine codon CTC uracil cell harbor pccu pcsu CTC show normal growth growth observe construct name pcsu TCT pCSU CTC respectively addition plasmid pCCU consist harbor pcsu TCT contain gene insertion cell harbor pcsu CTG show gene have CTT leucine codon correspond site combine weak significant growth figure uppermost row result demonstrate codon position specific promoter positive control variant plasmid introduce translate leucine active odcase produce cell able grow translation defective strain codon serine produce inactive ODCase believe trna Ser CAG molecule responsible leucine insertion correspond cell unable grow result cell harbor pcsu CTG clearly demonstrate mutation codon CUG cell base follow obervation purify sequence number chromosome way complement introduce pcsu CTG plasmid suggest leucine serine trnas Candida specie codon CUG translate serine fail CTG codon read partially leucine cell possess trna ser trna anticodon sequence potentially complementary codon CUG trna Ser CAG order quantify growth rate cell harbor pcsu CTG viability cell examine Yokogawa et al Ohama et al Suzuki et al Ueda et al unpublished observation liquid medium uracil show Figure translation CTG codon serine completely futhermore trna gene serine leucine Candida specie sequence follow block cell growth case pcsu TCT complementation observe case pcsu ctc amplification clone PCR method find trna Ser CAG able translate codon CTC codon read leucine intermediate cell growth observe case pcsu CTG CUG Yokogawa et al Ohama et al Suzuki et al Ueda et al unpublished observ indicate ODCase express active form albeit low level slight incorporation ation conclude trna Ser CAG species insert leucine polypeptide leucine CTG codon slow growth cell harbor pcsu CTG spontaneous corresponding codon CUG reversion CTG codon leucine codon mutation cell harvest Discussion colony sd plate growth phenotype result unlikely reflect different observation present clearly demonstrate certain live organism single codon simul expression level gene variant mrna level alter mutation position taneously assign distinct amino acid codon genetic code degenerate finding Ohkuma furthermore possibility gene CTG position translate amino acid able degenerate respect particular codon codon ambiguity efficiently gene TCT site codon preference Ikemura exclude govern trna acceptable amino acid simultaneously describe propose desig fact TCT codon preferred serine codon include CUG codon nate codon correspond multiple amino acid polysemous codon Sugiyama et al activity result translation high degree accuracy trna aminoacylation consider crucial preserve fidelity protein gene examine presence pyrimidine analog fluoroorotic acid inhibitor synthesis establish aminoacyl trna synthetase able discriminate precisely cognate pyrimidine biosynthesis incorporation ODCase result formation fluorouridylate amino acid structurally relate amino acid adenylation reaction step cognate trnas harmful cell propagation Boeke et al defective strain grow normally medium non cognate one review Parker Kurland misacylation error process contain cell possess active gene unable grow medium cell estimate range Lin et al Okamoto et al discrimination cognate harbor respective plasmid cultivate presence addition uracil tRNA non cognate trnas mediate positive negative identity determinant localize trna show row Figure cell harbor pcsu CTG exhibit similar growth molecule Yarus Normanly Abelson exception report far tRNA Gln agar plate pcsu TCT transformant pcsu CTC pCCU unable aminoacylate glutamate Gram positive bacteria organelle Lapointe et al Schön grow result indicate CTG codon position mainly translate serine maltosa et al differ misaminoacylation process indispensable compensate produce inactive ODCase liquid medium supply slight lack glutamyl trna synthetase organism general high fidelity aminoacylation process reduction growth rate observe case pcsu CTG compare pcsu TCT Figure consider indispensable translate gene functionally active protein high degree accuracy slow growth observe case pcsu CTC control order detect lowthe discovery polysemous codon Candida specie contradict established notion aminoacyl level ODCase activity arise slight incorporation leucine CUG codon position ation high fidelity show single trna acceptable different amino acid adjust ratio uracil show Materials method growth rate reduction clearly suggest transfer different amino acid correspond particular codon expression slow growth observe sd medium low expression active ODCase conclude experiment ODCase encode gene contain codon CUG site essential activity CUG codon partially translate leucine Sugiyama et al suggest leucine incorporate gene product correspond experiment artificial mutation reflect experimental observation extant live codon CUG judge complementation test mutation organism basis peptide sequence research leucine incorporate CUG codon quantitatively determined clear incorporation group report codon CUG correspond serine Sugiyama et al mediate leucyl trna Ser CAG conclude codon CUG simultaneously assign Santos Tuite White et al leucine insert peptide detect study serine leucine normal translation process quantitative analysis amino acid consider peptide leucine insert codon CUG attach tRNA indicate trna Ser CAG leucylate cell high level miss purification undetectable peptide sequencing leucine insert leucylation far conventional misacylation rate estimate proofreading peptide low positively identify sequence experiment mechanism exist ribosome incorporation leucine CUG codon site reflect relative ratio ofWe show trna Ser CAG Candida specie chimera trna Ser CAG tRNALeuCAG far trna Ser CAG leucylation order magnitude high conventional substrate SerRS LeuRS value LeuRS µm order date artificial manipulation molecule participate translation process overproduction magnitude large SerRS µm vitro aminoacylation experiment trna Ser CAG aminoacyl trna synthetase Swanson et al mutation trnas etc control growth condi subject reaction convert leucyl trna ser CAG increase LeuRS tion deprivation amino acid medium Edelmann Gallant Parker long incubation time datum show observe presence SerRS non radioactive Precup find increase error rate translation review Parker serine reduce leucylation complete loss leucylation achieve datum show indicate observation base experiment wild type cell grow rich medium suitable high viability affinity LeuRS trna Ser CAG relatively high proliferate cell leucyl respect polysemous codon phenomenon completely different artificial translational trna Ser CAG cell estimate seryl trna Ser CAG low error know example exist alternative decoding universal codon initiation codon obtain vitro experiment consider reduction leucylation competition AUG Gold Kozak leaky stop codon cause nonsense suppresser native trnas Murgola trna Ser CAG SerRS LeuRS cell despite competition distinct detection UGA codon incorporation selenocysteine Leinfelder et al leucylate trna Ser CAG vivo support existence ambiguous aminoacylation reaction single strong dependence context effect possible secondary structure mrnas recode trna Ser CAG specie polysemous codon result coexistence event program mrnas Gesteland et al sequence gene trna identity determinant serine leucine single trna molecule construction trna Ser CAG Candida genome find secondary structure codon CUG gene con variant microsurgery method lead finding single methyl moiety position sidere polysemous codon mediate single trna unlikely polysemous codon occur involve leucylation process contrast trna Ser CAG influence neighboring region mrnas alternative decode polysemous codon CUG position deprive leucine accept activity Himeno co worker note nucleotide possible assume LeuRS overexpresse certain physiological condition depend leucine trnas strongly recognize LeuRS unmodified variant transcribe RNA increase LeuRS cell incorporation leucine correspond codon CUG occur fre polymerase Soma et al discriminator base strong recognition site quently cause production polypeptide new function possibility examine anticodon loop play role determinant trna able compare experiment idea polysemous codon differ activity variant mutate position variant discriminator position near cognate concept propose Schultz Yarus claim ambiguous decoding occur effectively elevate leucylation activity work utilize serine tRNA modify nucleoside consequence irregular codon anticodon interaction induce base pair anticodon discriminator position substrate LeuRS transcript trna Ser CAG show stem trna result genetic code change transition state polysemous codon find study activity leucylation experiment microsurgery method indicate great importance cause trna aminoacylation process trna codon anticodon interaction proceeding precisely leucylation despite fact presence discriminator position unsuitable recognition conventional manner furthermore hypothesis Schultz Yarus base LeuRS modify nucleotide tRNA know involve recoginition synthetase Muramatsu et al pütz et al show position yeast trna Asp negative determinant arginyl trna synthetase demonstrate nucleotide position trna Ser CAG uniquely possess modulate leucine accept activity prevent trna Ser CAG excessive leucylation cell viability cell decrease substantially finding suggest polysemous state tolerate ambiguous recognize cognate leucine tRNA anticodon loop afford uridine turn translation strict constraint consider function negative modulator leucylation structure figure methyl moiety directly recognize LeuRS case trna Ser CAG control relative seryl leucyl trna ser CAG anticodon loop distort decrease affinity LeuRS judge line experiment suggest involve trna function ribosomes observation increase value leucylation approximately fold comparison rigid codon anticodon interaction proper gtp hydrolysis ternary complex efficient translation prymidine basis position figure replace mean trna termination codon Bare et al Dix et al replacement lose major determinant LeuRS discriminator base Figure consequently LeuRS trna Ser CAG increase efficiency vitro translation fold datum show negative effect unable recognize trna Ser CAG concomitantly lose function modulator LeuRS translation indicate involvement mechanism decode polysemous codon course unable recognize serine isoacceptor trnas correspond universal codon possibility need clarify study show role modify position interaction LeuRS suppression leucylation consider nucleotide position directly involve trna Ser CAG evolve Candida specie utilize CUG serine classify distinct group group recognition LeuRS basis observation leucylation detectable contain specie tRNA Ser CAG leucylation activity include trna Ser CAG variant replace pyrimidine base speculate Figure group represent solely contain specie tRNA Ser CAG influence location conformation accompany alteration anticodon loop leucylation activity Figure plausible evolutionary process group arise structure decrease affinity LeuRS trna Ser group genetic code change speculate basis follow observation generally consider reconstruct trna lose activity reaction homology trna Ser cag group isoacceptor trnas codon UCG high step need microsurgery method cleavage trna strand ligation trna fragment trna Ser CAG isoacceptor second Ohyama et al slight reduction leucylation activity observe control variant group possess high copy number trna Ser CAG gene copy diploid genome Suzuki compare native trna figure right hand graph turn result et al low copy number copy observe group trna Ser CAG gene Santos partial deacetylation acetyl cytidine acid treatment half fragment material et al Sugiyama et al personal observation codon CUG utilize major method reasonable deduce effect base replacement aminoacylation activity serine codon gene lipase Kawaguchi et al chitin synthase compare activity reconstruct trnas half fragment unpublished result CUG appear infrequently genome specie belong group Lloyd manipulate trna molecule plausible mechanism LeuRS recog Sharp Sugiyama et al personal observation course change nize cognate leucine serine trnas specific codon CUG illustrate figure leurs contact genetic code genome pass state group Ohama et al fourth phylogenetic dextrose minimal medium sd yeast nitrogen base tree specie relative construct amino acid Difco dextrose supply mg ml uracil cultivation yeast gene support evolutionary pathway sd plate uracil prepare add agar manuscript preparation Boeke et al codon UCG Pesole et al nucleotide position likely haveIn order introduce mutation codon reading frame mutate direction modify alternative splicing generate multiple protein contain autonomously replicate sequence Takagi et al Hikiji et al sequence single gene mrna level codon appear infrequently observe group mutate CTG CTC pCCU Sugiyama et al synthesis cause polysemous codon speculate instruction manual electrified cell spread sd plate ambiguity give rise protein contain uracil incubate multiple amino acid sequence non house keep gene confer multifunctionality onin vitro aminoacylation assay Seryl leucyl trna synthetase partially purify protein strain develop oide cell describe previously industrially production lipase large scale purification trna Ser cag mmol leucine MBq mmol Amersham fluoroorotic acid monohydrate PCR inc biotinylated Candida cylindracea cell kg treat phenol unit unfractionated tRNA extract eighty DNA probe synthesize Sci Media Japan synthetic RNA oligomer chimeric oligonucleotide compose dna thousand unit trna mixture obtain DEAE cellulose chromatography stepwise elution apply methyl RNA synthesize Genset enzyme microsurgery Takara Shuzo Tokyo sephadex column cm elution perform linear gradient NaCl buffer chemical obtain Wako Chemical mM Tris HCl pH mM MgCl fraction possess cleavage site posi rich trna Ser apply column cm elute linear gradient NaCl NaCl buffer tion trna Ser CAG microgram purify trna Ser CAG incubate consist mM Tris HCl pH mm result chromatography unit purify trna Ser CAG min nmol buffer consist mM Tris HCl pH mM NaCl mm DTT BSA finally obtain thousand unit trna glycerol µl anneal room temperature magnesium chloride add mixture final concentration mm cell kg fractionate DEAE Sepharose fast flow column cm linear gradient NaCl reaction carry addition unit RNase Takara Shuzo µg cleave half buffer consist mM Tris HCl pH mM MgCl unit trna Ser CAG finally obtain fragment obtain purification page contain urea synthetic oligo RNAs pcagap pcaggp column chromatography Sepharose reverse gradient ammonium sulfate buffer consist ligate half fragment digest RNase variant mutate position condition describe mM NaOAc pH mm MgCl mm mercaptoethanol mm ligate dephosphorylate half fragment anneal ligate half fragment digest RNase µg variant mutate position construction trna variant mutation position microsurgery procedure basically carry accord obtain phosphorylation purification page contain urea literature Ohyama et al Ohyama et al limited digestion mg purify trna Ser CAG RNase perform min reaction mixture contain mM Tris HCl pH identification amino acid attach trna Ser cag mm MgCl mg ml trna unit ml rnase cell Sigma phenol extraction result fragment treat Identification aminoacyl trna Ser CAG Candida cell carry HCl order cleave cyclic new method develop recently Suzuki et al phosphate end fragment form limited digestion experimental condition report half fragment separate page fish aminoacyl trnas design biotinylate dna contain urea cm thirty microgram probe half µg half fragment recover trna Ser CAG gel purified half fragment dephosphorylate trna Ser CAG probe immobilize strepbacterial alkaline phosphatase Takara Shuzo end tavidin agarose Gibco BRL report previously Wakita et al half fragment remove oxidation sodium periodate describe literature Keith Gilham','1997-03-03 00:00:00','http://dx.doi.org/10.1093/emboj/16.5.1122',3891,1057),('ae19236936b05793ceac08ed5d58269c25486320.json','An ontology for immune epitopes: application to the design of a broad scope database of immune reactivities','BACKGROUND: Epitopes can be defined as the molecular structures bound by specific receptors, which are recognized during immune responses. The Immune Epitope Database and Analysis Resource (IEDB) project will catalog and organize information regarding antibody and T cell epitopes from infectious pathogens, experimental antigens and self-antigens, with a priority on NIAID Category A-C pathogens () and emerging/re-emerging infectious diseases. Both intrinsic structural and phylogenetic features, as well as information relating to the interactions of the epitopes with the host\'s immune system will be catalogued. DESCRIPTION: To effectively represent and communicate the information related to immune epitopes, a formal ontology was developed. The semantics of the epitope domain and related concepts were captured as a hierarchy of classes, which represent the general and specialized relationships between the various concepts. A complete listing of classes and their properties can be found at . CONCLUSION: The IEDB\'s ontology is the first ontology specifically designed to capture both intrinsic chemical and biochemical information relating to immune epitopes with information relating to the interaction of these structures with molecules derived from the host immune system. We anticipate that the development of this type of ontology and associated databases will facilitate rigorous description of data related to immune epitopes, and might ultimately lead to completely new methods for describing and modeling immune responses.',' epitope define molecular structure recognize product immune response accord definition epitope specific molecular entity engage bind antibody molecule specific cell receptor extended definition include specific molecule bind peptide bind site MHC receptor previously describe general design Immune Epitope Database Analysis Resource IEDB broad program recently initiate National Institute Allergy Infectious Diseases NIAID overall goal IEDB catalog organize large body information antibody cell epitope infectious pathogen source priority place NIAID Category pathogen emerge emerge infectious disease epitope human non human primate rodent specie detailed information available include envision new effort catalyze development new method predict model immune response aid discovery development new vaccine diagnostic assist basic immunological investigation IEDB catalog structural phylogenetic information epitope information capacity bind specific receptor MHC TCR BCR Antibodies type immune response observe follow engagement receptor RFP NIH NIAID broad term database contain general category datum information associate immune epitope extrinsic context dependent datum intrinsic feature epitope characteristic unequivocally define specify epitope sequence structure example intrinsic feature epitope sequence structural feature bind interaction immune system molecule describe immune response associate specific epitope context information need take account contextual information include example specie host route dose immunization health status genetic makeup host presence adjuvant respect IEDB project transcend strict boundary database development reach system biology application attempt time integrate structural information epitope comprehensive detail describe complex interaction immune system host infected organism vaccine recipient reason apparent outset project crucial develop rigorous conceptual framework represent knowledge relate epitope framework key share information idea developer scientist potential user allow design effective logical structure database accordingly decide develop formal ontology year term ontology define utilize way knowledge engineering community adopt definition ontology explicit formal specification term domain relationship accord Noy McGuinness ontology define common vocabulary researcher need share information domain help separate domain knowledge operational knowledge availability formal ontology relevant design database catalog information communicate database structure researcher developer user integrate multiple database schema design application exist database catalog epitope relate datum gratefully acknowledge able use previous experience design implementation IEDB MHCPEP SYFPEITHI FIMM HLA Ligand Database HIV Immunology Database JenPep AntiJen MHCBN publicly available epitope relate database general database provide information relate epitope catalog depth information relate interaction host immune system note database publish formal ontology rely informal implicit ontology take account possible ontology infer structure informal communication database developer perusal database website ontology develop IEDB describe complement explicit ontology presently available IMGT Ontology Gene Ontology IMGT Ontology create international ImMunoGeneTics Database IMGT integrate database specialize antigen receptor immunoglobulin Cell receptor MHC molecule vertebrate specie good knowledge ontology domain immunogenetic immunoinformatic project provide structure control vocabulary cover domain molecular cellular biology provide excellent framework gene gene product sequence address specific epitope substructure gene product IMGT provide excellent ontological framework immune receptor lack information relate epitope necessary expand available ontology create ontology specifically design represent information immune receptor interaction immune epitope possible IEDB ontology conform standard vocabulary capture value certain field capture disease name IEDB use International Classification Diseases NCBI Taxonomy database nomenclature capture specie strain name HLA Allele name consistent HLA nomenclature report IEDB develop web accessible database Oracle Enterprise Java industry standard software design follow expect IEDB available public user end design document IEDB ontology Protégé free open source ontology editor knowledge base framework write Java provide environment create ontology term ontology protégé support class slot instance creation allow user specify relationship appropriate entity feature IEDB ontology effort extensively Protégé support create ontology term view term hierarchy definition support central repository ontology browse support key review reuse ontology open source tool available develop ontology select Protégé extensibility variety plug in readily available integration ability export different format include Ontology web Language OWL allow interoperability ontology previously describe general concept relate IEDB design information relate aspect project access report detailed description novel aspect IEDB ontology design application architecture follow common system engineering practice determine scope nature datum involve essential step understand semantic domain capture knowledge agree format arrange domain concept taxonomy initial organizing step ontology design process class hierarchy represent generalization specialization relationship class object domain briefly class describe concept domain subclass represent concept class specific superclass subclass unique property slot represent property class example Figure class name Reference specific subclass Reference Journal Article Patent Application Direct Submission figure show class Epitope number property slot associate Epitope Structure Epitope approach create class hierarchy topdown development process define class domain identify property build hierarchy main class identify IEDB Reference Epitope Structure Epitope Source MHC Binding Naturally Processed Ligand Cell Response Cell Response figure Epitope class main class encompass individual concept identify individual concept relate class primary relationship use sub class relationship use property show figure arc label restriction type value fill slot class encompass information relate data source epitope related information extract IEDB identify broad subclass reference describe epitope information obtain Journal Article Patent Application Direct Submission complete listing slot field encompass Journal Article Patent Application submission class provide figure Journal Article class refer manuscript publish peer review journal Patent Application class capture reference field patent application contain epitope information Submission class capture information source contribute datum IEDB directly datum deposit Large Scale Antibody Cell Epitope Discovery contract transfer website fall class Epitope Structure Epitope Source class capture intrinsic feature epitope Epitope Structure class capture physical chemical feature epitope virtually molecular structure provoke immune response protein carbohydrate dna lipid Epitope Structure class structural information relate linear sequence structure epitope available catalogue Epitope Source class capture phylogenetic source epitope include specie origin gene protein link database detailed information protein gene figure listing property slot field encompass Epitope Structure Epitope Source class experimental datum information specific experiment methodology utilize capture Assay Information class assay type response measure assay readout assay example information capture Assay Information class important class superclass class property inherit class complete list property slot field Assay Information class show figure Assay Information class Immunization Antigen Antigen Presenting Cell multiple class description feature relate induction immune response capture Immunization class Figure relationship class like Immunized Species Immunogen vivo immunization vitro immunization Immunized Species contain information relate host immunize Immunogen class describe molecule induce immune response associated carrier molecule present feature relate immunogen introduce immunize specie capture vivo vitro immunization class similarly antigen define molecule react product immune response oppose epitope specific structure contain antigen engage immune receptor information relate antigen associate carrier molecule capture Antigen class figure immune response antigen present cell process antigen present peptide epitope complexe MHC molecule information capture Antigen Presenting Cells class relationship MHC Molecules Source Species class figure Source Species class describe species information antigen present cell derive MHC bind class capture detail relate interaction epitope specific MHC molecule information relate MHC molecule available Epitope MHC complex structure detail class slot restrict instance Assay Information class figure IEDB Class Hierarchy Figure Overview IEDB Class extrinsic feature context dependent attribute dependent specific experimental condition Naturally Processed Ligand class capture datum relate epitope naturally process present cell surface class property instance class include Antigen Presenting Cell Antigen Assay Information Figure naturally Processed Ligand class differ MHC bind class information relate antigen process cell type processing occur represent MHC bind class capture datum relate vitro MHC bind assay assess epitope bind capacity MHC molecule MHC bind class require Antigen class Antigen Presenting Cells class general naturally process ligand assess absence cell response example identify direct elution MHC molecule extract infected cell antigen present cell Immunization class value restriction Naturally Processed Ligand class Cell Response class capture cell mediate immunity relate information figure property type Immunization Effector Cells Antigen Presenting Cell Antigen Assay Information Epitope MHC TCR Complex Effector Cell class describe cell elicit immunization acquire measurable function result Cell Response class describe antibody response relate epitope figure class property type Immunization Antibody Molecule Antigen Assay Information Antigen Antibody Complex cell response require MHC binding class capture information structure complex Epitope MHC Complex Epitope MHC TCR Complex Antigen Antibody Complex Epitope MHC Complex Epitope MHC TCR Complex Antigen Antibody Complex class restriction property MHC Binding Cell Response Cell Response class respectively figure complex class capture Protein Data Bank PDB Identifier provide detailed information structure Protein Data Bank contain approximately structure immunological interest information available pdb atom pair involve interaction molecule specific residue contact area molecule allosteric effect capture class numerous slot capture detailed information associate epitope mention complete list class property relationship find file provide supplementary material contain example literature reference contain epitope information extract IEDB ontology additional file class hierarchy IEDB datum dictionary additional file provide detailed information field define IEDB datum dictionary contain textual overview description listing field require complete IEDB entry datum dictionary allow database user provide comment suggestion IEDB team enhance formal ontology IEDB comprehensive resource pertain epitope immune system extensively curate intrinsic extrinsic feature associate epitope IEDB expect provide substantially great detail specific epitope database presently available IEDB populate datum derive main source peer review literature patent application direct submission Epitope datum publish literature patent application curate manually IEDB curation team datum exist epitope database author agree share datum import IEDB apart main data source direct submission datum large scale Antibody Cell Epitope Discovery program fund NIAID presently fourteen contract award program submit datum IEDB direct antibody cell epitope submission seek broad research community emphasis antibody epitope NIAID Category pathogen large scale IEDB project formal ontology critical ensure consistency representation datum communication database developer researcher analysis tool developer team member crucial perform harmony common vocabulary establish ontology explicit formal specification term domain relationship effective way share knowledge contain domain accordingly IEDB domain epitope relate datum create ontology capture detailed conceptual structure relate datum development ontology relevance expansion modification epitope knowledge base ontology design define individual concept separate class define relationship class object domain class serve restrict value describe property object database example species separate concept define class depend context refer immunize specie specie antigen present cell derive similarly MHC Molecules define separate class value restriction concept like MHC Binding Antigen Presenting Cells define concept separate class restrict value property class facilitate expansion modification ontology add property slot concept task easily accomplish define class description serve value restriction property provide high level classification immunization Antigen Antigen Presenting Cells class class description general apply instance ensure design concept atomic class development formal ontology valuable database user particular scientist contribute datum download datum IEDB anticipate availability formal ontology ensure common language share understanding concept inspire type communication ensure maximum efficiency accuracy formal ontology develop likely require refinement fine tuning user provide suggestion new technology perform experiment discover IEDB website provide mechanism user provide suggestion participate ment ontology IEDB Data Dictionary separate column user provide comment specific datum field IEDB website host web form guide user conform ontology definition submit datum apart web form xml schema definition XSD available website user inspect use datum submission user able download epitope record website process develop new ontology good practice leverage exist community standard initial analysis confirm explicit ontology efficiently capture epitope detail scope IEDB program mention utilize possible infer ontology exist epitope database ontology analyze IMGT Ontology Gene Ontology formal ontology relate epitope domain IMGT Ontology design immunogenetic database IMGT integrate database specialize antigen receptor immunoglobulin cell receptor major histocompatibility complex vertebrate specie ontology develop database specific immunological content describe classification specification term need immunogenetic IEDB conform IMGT standard receptor MHC molecule chain sense chain name follow IMGT control vocabulary provide structured control vocabulary gene gene product sequence annotate organism IEDB complement term epitope immunological interest incomplete area antigen primary source epitope annotate essence IEDB utilize provide extension antigen contain epitope relate information important element development IEDB ontology good knowledge represent immunological ontology specifically design capture intrinsic biochemical extrinsic context dependent information respect similar spirit different approach knowledge resource relate system biology anticipate development type ontology associate database lead completely new method describe model immune response accordingly new program represent novel tool assist design testing development new way combat infectious disease immune related pathology cancer autoimmune disease complete listing IEDB class hierarchy property available ontology','2005-09-20 00:00:00','http://dx.doi.org/10.1186/1745-7580-1-2',1974,598),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json','A Gene Encoding Sialic-Acid-Specific 9-O-Acetylesterase Found in Human Adult Testis','Using differential display RT-PCR, we identified a gene of 2750 bp from human adult testis, named H-Lse, which encoded a putative protein of 523 amino acids and molecular weight of 58 kd with structural characteristics similar to that of mouse lysosome sialic-acid-specific 9-O-acetylesterase. Northern blot analysis showed a widespread distribution of H-Lse in various human tissues with high expression in the testis, prostate, and colon. In situ hybridization results showed that while H-Lse was not detected in embryonic testis, positive signals were found in spermatocytes but not spermatogonia in adult testis of human. The subcellular localization of H-Lse was visualized by green fluorescent protein (GFP) fused to the amino terminus of H-Lse, showing compartmentalization of H-Lse in large dense-core vesicles, presumably lysosomes, in the cytoplasm. The developmentally regulated and spermatogenic stage-specific expression of H-Lse suggests its possible involvement in the development of the testis and/or differentiation of germ cells.',' sialic acid diverse family acidic carbon sugar frequently find terminal unit oligosaccharide chain different glycoconjugate high invertebrate vertebrate determinant glycoprotein sialic acid play important role intercellular intermolecular recognition acetylation de Oacetylation common modification sialic acid find mammalian cell surface sialoglycoconjugate alter size hydrophobicity net charge antigenicity modification regulate variety biological phenomenon include endogenous lectin recognition tumor antigenicity virus binding complement activation specifically capable remove acetyl ester position sialic acid sialic acidspecific acetylesterase enzyme mammal form cytosolic sialic acid specific acetylesterase Cse cytosolic fraction lysosome sialic acid specific acetylesterase Lse lysosomal endosomal compartment Lse likely participate terminal lysosomal degradation acetylate sialoglycoconjugate Cse likely salvage acetylate molecule escape initial action Lse enzyme process de acetylation sialic acid catalyze sialicacid specific acetylesterase implicate organogenesis cellular differentiation complicated process germ cell differentiation adult testis establish testicular development type germ cell specific developmental stage find seminiferous tubule spermatogonia primary spermatocyte secondary spermatocyte spermatid sperm divide group accord DNA content dna content cell cell DNA content cell cell dna content cell cell separation cell enable researcher investigate molecular mechanism underlie testicular development spermatogenesis present study separate cell seminiferous tubule human adult testis flow cytometry identify human Lse differential display RT PCR expression pattern Lse find developmentally regulated stage specific indicate possible role testicular development germ cell differentiation human testis obtain Donation Center Nanjing Medical University consent relative seminiferous tubule collect DMEM contain collagenase wash remove Leydig cell interstitial cell trypsin treatment brief treatment DNase release spermatogenic cell seminiferous tubule suspension cell filter nylon mesh disaggregate spermatogenic cell suspend cell mL sodium citrate solution PH fresh depc overnight room temperature day centrifuge resuspend sodium citrate solution PH fresh depc day day use cell centrifuge resuspend PBS mm hepe PH bsa fresh depc cell spin resuspend PBS µg mL PI propidium iodide fresh depc cell stain overnight flow cytometry FCM research FACSVantage SE Becton Dickinson Calif equip argon laser power mw wavelength nm nm filter set detector Cellquest Becton Dickinson sort sort mode Normal Drops sort drop delay density cell sort cell total RNA cell cell perform Trizol Reagent Gibco BRL Ontario nanogram total RNA differential display RT PCR chain cdna synthesize oligo dT primer template PCR PCR perform follow minute minute minute cycle microlitre PCR product cell run agarose gel fragment highly specifically display cell excise purify DNA reamplifie combination primer subclone Pinpoint vector Promega colony length cDNA screen PCR Human Testis Large Insert cDNA Library Clontech Calif convert plasmid cDNA Library arrayed cDNA library plate accord method Munroe arrayed cDNA library colony screen tissue northern MTN blot Clontech hybridize label probe probe correspond bp Lse hybridization stringent wash blot place storage phosphor screen Packard USA expose hour dark signal detect Cyclone storage phosphor system Stanford TNG Radiation Hybrid Panel Research Genetics Huntsville Ala map chromosomal localization HSE primer hsemapf hsemapr accord manufacturer instruction cycle amplification reaction product separate agarose gel positive amplification label negative label result analyze Stanford genome center web server determine probable chromosomal location RNA DIG label probe vitro transcription promoter sequence incorporate side template bp Lse PCR sense antisense probe DIG RNA labeling mix Roche USA accord manufacturer instruction fixation paraffin embed mount section section human embryonic adult testis prehybridize hybridization buffer DIG Easy Hyb Roche Germany hour hybridization carry hybridization buffer contain appropriate probe hour humidity chamber subcellular localization HSEI HSEII perform method green fluorescent protein pegfp hsei pegfp HSEII construct set primer hsei HSEII EcoRI sali restriction site pegfp recombinant vector transfecte cell cell cell lineage adenocarcinoma pancreas Lipofectin reagent Gibco BRL cell image hour transfection fluorescence microscope stain PI measure FCM group cell seminiferous tubule human adult testis detect figure cell subsequently sort clone identify differential display RT PCR highly express cell figure high homology mouse lysosome sialic acid acetylesterase clone name Lse round screening arrayed cdna library plasmid contain length Lse GenBank accession number find Lse bp length encode putative protein amino acid molecular weight kd isoelectric point terminus aa protein region contain hydrophobic amino acid residue signal peptide comparison protein sequence figure hypothesize Lse human counterpart mouse lysosome sialic acid acetylesterase PCR amplification result show pattern retrieve result Stanford genome center web server show HSE localize human figure distribution Lse human tissue analyze northern blot Figure result show presence distinct mrna specie approximately kb kb kb expect transcript Lse approximately kb consistently express tissue examine high expression find testis prostate colon transcript approximately kb exclusively express colon transcript approximately kb distribute testis colon small intestine prostate thymus high level expression find testis examine possible role Lse testicular development spermatogenesis situ hybridization experiment conduct compare Lse expression human embryonic adult testis spermatogenesis initiate embryo meiosis embryonic seminiferous tubule result show signal detect embryonic testis positive signal detect spermatocyte spermatogonia seminiferous tubule adult testis signal associate germ cell somatic cell testis Sertoli Leydig cell negative control sense probe confirm specificity result figure subcellular localization Lse fusion protein visualize transiently transfecte Lse gene fuse GFP cell show Figure control cell transfecte GFP protein exhibit fluorescence evenly distribute cytoplasm GFP Lse fusion protein compartmentalize numerous large dense core vesicle cytoplasm Spermatogenesis developmental program occur mitotic meiotic postmeiotic phase mitotic phase spermatogonia proliferate expand quantity germ cell meiotic phase spermatocyte accomplish chromosomal synapsis genetic recombination meiotic division postmeiotic phase haploid spermatid remodel spermatozoa process acrosome formation nuclear condensation flagellar development loss majority cytoplasm control intrinsic extrinsic factor spermatogenesis characterize expression spectrum gene celltype specific stage specific think play essential role spermatogenesis particular stage example MutS homologue require chromosome pair CPEB require synaptonemal complex assembly chromosome synapsis primary spermatocyte present study identify gene Lse human adult testis high homology cse similarly inhibit binding sialoadhesin macrophage restricted sialic acid dependent adhesion molecule hand acetylation sialic acid form novel epitope influenza virus haemagglutinin specifically require acetylate sialic acid bind host cell incubation red blood cell sialate oacetylesterase render erythrocyte resistant agglutination influenza virus acetylation disialoganglioside human melanoma cell report create unique antigenic determinant modification sialic acid important mechanism underlie interaction cross talk different type cell essential role sialic acid modification cellular communication explain presently observe wide distribution Lse examine tissue present study suggest expression Lse developmentally regulated spermatogenic stage specific evidence include lack expression embryonic testis association high level mrna detect DD RT PCR cell adult testis detection situ hybridization signal spermatocyte spermatogonia somatic cell absence spermatogenesis embryonic testis contain distinct cell type spermatogonia Sertoli cell seminiferous epithelium adult testis consist germ cell different stage spermatogensis cell find adult testis include primary spermatocyte spermatogonia stage cell include spermatogonia stage secondary spermatocyte Sertoli cell absence Lse mrna embryonic testis high level mrna cell adult testis suggest expression restrict spermatocyte particularly primary spermatocyte situ hybridization result show mRNA Lse restrict spermatocyte spermatogonia Sertoli cell interstitial cell datum suggest Lse likely involve process spermatogenesis role testicular development entirely rule unfortunately deformation available human testis able distinction primary secondary spermatocyte clearly show present data Lse present stage spermatogonia suggest possible role differentiation germ cell Lse Lse Cse lse lse Cse Lse lse DInterestingly process acetylation de acetylation sialic acid implicate organogenesis cellular differentiation alteration process lead interruption cellular development embryogenesis transgenic mouse constitutively overexpresse acetyl sialicacid specific esterase influenza find arrest embryo development cell stage report vitro development embryonic stem cell show expression level Lse low initiation development follow increase later stage transgenic mouse selective expression acetyl sialic acid specific esterase retina adrenal gland organ show abnormality organization tissue appear normal Lse consider play key role differentiation lymphocyte express late early lymphocyte presently observe developmentdependent pattern Lse expression consistent find cell type absence low expression early stage differentiation high later stage take acetyl ester sialic acid appear important development cellular differentiation spermatogenesis multiple stage continuous progress cellular differentiation report cell surface glycoconjugate modify early step spermatogenesis influence differentiation spermatogenic cell ninecarbon sugar commonly find glycoprotein spermatogenic cell sialic acid represent target cell surface modification removal acetyl ester enzyme Lse modification sialic acid result alteration cell cell communication Sertoli cell germ cell interaction influence differentiation spermatogenic cell future study presently identify Lse provide insight molecular mechanism underlie testicular development germ cell differentiation spermatogenesis human','2004-07-29 00:00:00','http://dx.doi.org/10.1155/s1110724304307084',1333,600),('b0a0f45be3069a32ba190502ab88b6df6d520199.json','Positional clustering improves computational binding site detection and identifies novel cis-regulatory sites in mammalian GABA(A) receptor subunit genes','Understanding transcription factor (TF) mediated control of gene expression remains a major challenge at the interface of computational and experimental biology. Computational techniques predicting TF-binding site specificity are frequently unreliable. On the other hand, comprehensive experimental validation is difficult and time consuming. We introduce a simple strategy that dramatically improves robustness and accuracy of computational binding site prediction. First, we evaluate the rate of recurrence of computational TFBS predictions by commonly used sampling procedures. We find that the vast majority of results are biologically meaningless. However clustering results based on nucleotide position improves predictive power. Additionally, we find that positional clustering increases robustness to long or imperfectly selected input sequences. Positional clustering can also be used as a mechanism to integrate results from multiple sampling approaches for improvements in accuracy over each one alone. Finally, we predict and validate regulatory sequences partially responsible for transcriptional control of the mammalian type A γ-aminobutyric acid receptor (GABA(A)R) subunit genes. Positional clustering is useful for improving computational binding site predictions, with potential application to improving our understanding of mammalian gene expression. In particular, predicted regulatory mechanisms in the mammalian GABA(A)R subunit gene family may open new avenues of research towards understanding this pharmacologically important neurotransmitter receptor system.',' understand transcription factor TF mediate control gene expression remain major challenge interface computational experimental biology computational technique predict TF bind site specificity frequently unreliable hand comprehensive experimental validation difficult time consume introduce simple strategy dramatically improve robustness accuracy computational bind site prediction evaluate rate recurrence computational TFBS prediction commonly sampling procedure find vast majority result biologically meaningless clustering result base nucleotide position improve predictive power additionally find positional clustering increase robustness long imperfectly select input sequence positional clustering mechanism integrate result multiple sampling approach improvement accuracy finally predict validate regulatory sequence partially responsible transcriptional control mammalian type aminobutyric acid receptor GABA subunit gene positional clustering useful improve computational bind site prediction potential application improve understanding mammalian gene expression particular predict regulatory mechanism mammalian GABA subunit gene family open new avenue research understand pharmacologically important neurotransmitter receptor system co regulation basic mechanism coordinately control expression gene module biochemical pathway protein complex eukaryote expression mediate transcription factor tfs bind upstream transcription start site TSS recruit polymerase assembly tfs bind vary affinity set similar short nt sequence computational bind site discovery focus find significantly overrepresented sequence upstream region co regulated gene computational tfbs prediction algorithm begin input set promoter gene hypothetically co regulate share TF algorithm aim predict bind position consequently nucleotide specificity TF transcription factor bind site TFBS discovery input set identify computational experimental method experimental technique chromatin immunoprecipitation ChIP successfully generate genome scale mapping approximate tf bind position computational technique phylogenetic profiling artificial neural network identify set co regulated gene experimental computational approach suffer significant false positive FP prediction rate inclusion extraneous promoter input set dilute enrichment share TFBS sequence make computational tfbs discovery significantly challenging term erroneously include promoter decoy sequence receive set upstream region co regulate share TF input computational method aim predict bind position TF give set input promoter motif detection algorithm identify set short oligonucleotide segment hypothesize bind TF interest predict sequence construct position weight matrix PWM represent average nucleotide frequency position site ideally computational detection return sequence bind TF biologically relevant function upstream region source bind specificity tfs understand heuristic approach ad hoc multiple alignment base scoring scheme identify locally optimal solution local optimum exist give set promoter correspond distinctly different motif score differently relative accord different scoring scheme bind site prediction algorithm generally confound factor degeneracy bind site unknown length bind site relatively large length promoter inclusion ds input set result predict position correspond biologically functional bind site low accuracy rate computational bind site identification limited use problem identify bind site exacerbate mammalian genome large promoter region scarcity reliable information co regulation gene demanding test efficacy TFBS identification approach lie application mammalian system subsequent validation prediction computational complexity problem Gibbs sampling identify bind position paper present new strategy cluster Gibbs sample result input nucleotide technique term positional clustering improve accuracy predict tf binding evaluate efficacy approach know example binding regulation yeast experimentally testing predict TF bind site upstream subunit gene code heteromeric mammalian neurotransmitter receptor system type aminobutyric acid receptor GABA GABA major inhibitory neurotransmitter receptor central nervous system CNS important role development disease receptor believe pentamer multiple subunit come different subunit class gene code subunit differentially combine form numerous pharmacologically distinct GABA receptor isoform isoform utilization depend relative abundance subunit change condition understand subunit regulatory mechanism provide insight GABA receptor isoform usage related phenotype current study test ability positional clustering detect know tf bind site series increasingly noisy set yeast promoter find marked improvement percentage correct prediction Gibbs sample present de novo prediction tf bind site promoter region GABA receptor subunit gene GABRs expression alter regulated regulated animal model temporal lobe epilepsy positional clustering identify number putative cis regulatory site correspond know bind element tfs find CNS mobility shift assay show predict GABR bind sequence specifically bind nuclear protein derive primary neocortical neuron keep culture furthermore particular non consensus GABR putative regulatory sequence show functional role cultured cortical neuron demonstrate efficacy positional clustering detect functional regulatory element mammal identify gene predict high confidence regulate TF YPD growth medium accord genome TF location datum identify gene collect upstream intergenic promoter intergenic region truncate kb upstream gene select study set gabr GABRD GABRE promoter extract gene include alternative exon give set seven promoter length promoter bp bp bp exon bp exon bp GABRD bp GABRE bp augment input set orthologous promoter rat exception orthologous gene mouse total promoter upstream gabr select analysis give input set promoter run Gibbs sampler BioProspector time evenly distribute motif width bp order background model derive appropriate genomic promoter collect good result BioProspector run count number time BioProspector identify nucleotide input set promoter identify maximally occur nucleotide extract position identify BioProspector maximum cluster neighboring position putative TFBS dust filter remove putative tfbs bp long figure set promoter result BioProspector run evaluation GABRps consider non subset seven promoter orthologous sequence consider result BioProspector run evenly distribute promoter subset analysis addition dust filtering require putative tfbs occur human orthologous rodent promoter positive predictive value evaluate bind site prediction classify prediction true positive TP false positive FP comparison bind motif TGAAACA determine sequence calculate distance know PWM modified local ungapped sequence alignment similar alignment score sum Pearson correlation coefficient prediction PWM aligned position score range zero position align seven length PWM observe bimodal distribution score Supplementary Figure choose alignment score correspond minima distribution alignment score threshold classify prediction TP complement seed set bind promoter randomly choose yeast promoter perform motif detection procedure input set compare PPV putative TFBS raw BioProspector result figure solid evaluate background rate bind site recovery create seed set randomly choose promoter evaluate percentage like bind site identify random seed set version seed set augment randomly choose yeast promoter figure dash line additional yeast evaluation substitute BioProspector house implementation BioProspector algorithm comparison result implementation implementation approximately equivalent run MotifScanner search GABR promoter vertebrate TF bind motif find TRANSFAC promoter analyze prior probability correspond organism specific order promoter background model Eukaryotic Promoter Database EPD consider positional overlap Motif Scanner prediction putative tfbs indicative know bind motif prediction double strand oligonucleotide EMSA contain follow sequence nuclear extract prepare gel shift analysis concentration Microcon column Amicon MA Quantification perform emsa condition yield standard curve band intensity single strand sense antisense phosphorothioate oligonucleotide predict GGCGGCGTGCACACACACGC CCACCGCGG bind site anneal boil sense antisense oligonucleotide min equal molar ratio dh Oligos cool room temperature place ice transfection DOTAP solution perform oligonucleotide correspond wildtype mutant DOTAP solution lacking oligonucleotide MOCK describe effect oligonucleotide application neuron assess real time RT tfbs predict computationally local optimization strategy evaluate extent strategy gibb sampling identify set segment repeat run input datum identify stably recur motif require clustering related result turn require definition relate sequence similarity base clustering impair combination sequence variation motif short length TF bind site align motif different length instead sequence base clustering choose cluster result position count number time Gibbs sampling identifie nucleotide promoter figure find Gibbs sample prediction generate BioProspector power law distribute nucleotide position Supplementary Figure gibb sample converge majority nucleotide infrequently small number nucleotide frequently frequently recur nucleotide appear result find power law distribution result robust gibbs sample algorithm scoring scheme datum show hypothesize frequently occur position biologically significant discard frequent Gibbs sampling result yield high accuracy robust identification biologically insignificant position preliminary test hypothesis apply repeat run Gibbs sample set promoter enrich binding identify wholegenome chip chip experiment positional clustering result identify frequently recur position Methods incorporation additional result significantly alter distribution result datum show choose well studied tfs known highly conserved experimentally define binding motif frequently recur position compare know motif classify prediction category true positive TP resemble experimentally identify bind motif false positive FP Methods finally calculate positive predictive value ppv ppv find positional clustering subsequent selection frequently recur nucleotide improve PPV bind site Gibbs sample figure validate result specific number input promoter bind motif particular Gibbs sample implementation repeat prediction process promoter predict bind repeat analysis replace original Gibbs sampling procedure implementation MotifSampler case find positional clustering significantly improve result local optimization procedure figure discovery TFBS type FP prediction type identification incorrect motif set upstream region know bind TF interest describe Methods second type FP error background discovery rate correct motif upstream region bind TF simulate rate like bind site recovery repeat analysis describe start randomly choose yeast promoter find positional clustering identifie like site result compare gibb sample positional clustering performance computational motif discovery enhance improve positive predictive value promoter gene co regulate decrease false discovery like site evaluate effect add ds performance gibbs sample positional clustering addition ds dilute enrichment TF bind site input set make motif detection challenging model ds repeat estimate PPV TFBS detection addition random yeast promoter ds original set bind promoter find positional clustering improve PPV Gibbs sample addition noise ds figure Supplementary Figure additionally result Gibbs sample positional clustering decay linearly addition decoy respectively Supplementary Figure extrapolate predict positional clustering maintain improved ppv addition noise ds address issue generality repeat procedure additional set promoter add benefit evaluate effect information content bind motif number promoter improvement positional clustering repeat analysis find independently set sampling procedure positional clustering improve accuracy broad range random ds figure improvement appear limit unreliable sample correctly identify bind site few result result consistent analysis bind promoter figure correspond low limit efficacy positional clustering recently researcher note complementary motif detection approach predict bind site effectively method mind evaluate positional clustering term ability combine result different sampling implementation dataset equal number result approach combine single dataset positional clustering predict bind site describe figure measure average percent change PPV TF dataset find positional clustering improve combined sampling compare improvement BioProspector MotifSampler respectively additionally cluster combined sampling improve dataset evaluate clustering BioProspector MotifSampler result improve dataset respectively positional clustering effective mechanism integrate result multiple sampling procedure identification GABR cis regulatory sequence describe introduction identify functional TFBS mammal difficult inclusion decoy sequence long upstream region lack information co regulation gene positional clustering show robust noisy input gibbs sample well suited identify de novo cis regulatory element mammalian promoter coordinately regulate test possibility choose seven mammalian GABR promoter GABRps activity potentially alter response status epilepticus identify change mrna level gene product set initial promoter analyze Gibbs sample positional clustering solid triangle open triangle gibbs sampling approach apply dataset Gibbs sampler procedure similar BioProspector row MotifSampler row row show combination sample procedure positional clustering combine result axis count addition ds evaluate positive predictive value technique dataset find positional clustering generally improve PPV addition random ds include orthologous rodent promoter input set orthologous promoter include provide instance bind site input set expect random allow easy detection site inclusion orthologous promoter additional effect selectively amplify evolutionarily conserve bind site bind site likely major functional role regulation GABR receptor sensitivity sequence improve expense sensitivity species specific bind site effect mind require GABR bind site prediction exist orthologous promoter mechanism co regulation seven GABRs unknown hypothetical co regulation model evaluate query possible subset seven GABRps cluster result nucleotide position select frequently occur position predict functional tf bind site prediction compare instance know bind motif TRANSFAC prediction resemble known bind site tfs Table tfs identify CNS rodent furthermore previous analysis GABR promoter region agree prediction assign putative regulatory role regulation GABRs choose validate novel motif prediction emsa functional study primary cultured neuron EMSA perform excess cold competitor define specificity protein bind nuclear extract derive primary neocortical neuron fibroblast FIBs keep culture show figure predict bind site find upstream subunit gene GABRD display specific binding addition specific binding neuronal extract novel motif evidence specific binding FIB extract figure especial interest give expression GABRs restrict nervous system repressor silence transcription factor REST express non neuronal cell implicate neural specificity gene expression clearly protein bind DNA necessitate regulatory function begin address functional Table positional clustering base prediction transcriptional regulatory sequence upstream GABRs total predict orthologous pair regulatory sequence represent unique sequence compare know mammalian bind motif prediction similarity previously characterize TFBS indicate known bind motif exist corresponding vitro EMSA functional assay applicable indicate similar prediction group align hand significance predict regulatory motif evaluate effect transfecting neuron double strand oligonucleotide contain novel bind motif describe especially interesting give regulate brain derive neurotrophic factor BDNF status epilepticus Transfection produce significant downregulation gene expression neocortical neuron monitor quantitative real time RT PCR change MOCK transfection transfection dso contain copy camp regulatory element CRE figure reliable bind site prediction return Gibbs sample base tfbs identification algorithm begin evaluate stability bind site prediction repeat run Gibbs sampling quantify robustness prediction count number Gibbs sample result nucleotide position input set figure large number repeat trial find frequently return position well predict tf bind site maximally score motif Gibbs sampling figure score function empirically derive necessarily represent biological reality result altogether unexpected find select frequently recur position allow filtering spurious sampling result cause convergence biologically uninformative local minima positional clustering allow unbiased aggregation result different motif width approximate width bind site free positional clustering improve robustness addition ds figure sequence arise inclusion promoter region input set direct binding TF experimental error computational mis annotation example study linear regression model indicate approach maintain advantage traditional Gibbs sample addition noise original signal Supplementary Figure empirical datum sharp decrease improvement close addition ds roughly double input set figure evaluation promoter co regulate tfs Figure double strand oligonucleotide functional assay regulation primary culture rat neocortical neuron treat DOTAP trimethylammonium methylsulfate Mock DOTAP phosphothioate oligonucleotide camp response element CRE Decoy sequence GABA promoter predict positional clustering GABA Decoy GTGCACACACACGCCCACCGCGGCTCGGG mRNA harvest real time RT PCR perform GABA specific primer error bar refer individual experiment different plating cell different animal Data normalize rrna level express relative mrna level GABA rRNA result show mean SEM asterisk indicate significantly different control confidence interval figure EMSA putative TF bind site form dna protein complex neocortical fibroblast nuclear extract Neocortical NEO fibroblast FIB nuclear extract rat embryo incubate radiolabele probe human receptor subunit cold wild type oligonucleotide define specificity competition cold oligonucleotide add fold excess probe indicate positional clustering likely improve prediction Gibbs sample identify correct site repetition figure possible simplistic linear model overestimate improvement robustness practically achievable multiple motif exist input promoter preliminary evidence suggest positional clustering uniquely identify single dominant motif Supplementary Figure refinement possible recover subordinate motif enable identification cis regulatory module spite limitation positional clustering repeat run researcher successfully apply sample algorithm identification functional bind site dataset significant proportion noise computational prediction TF bind mammalian genome pose challenge increase decoy sequence large upstream region having establish increase robustness ds yeast apply approach identify potentially unknown GABA receptor subunit gene regulatory sequence participate response genome seizure activity reason GABA receptor subunit gene regulated regulate animal model epilepsy share common bind motif positional clustering predict TF bind site upstream GABA receptor subunit gene table prediction verify comparison know bind site experimental verification vitro bind assay initially positive experimental result highlight ability computational technique direct research transcriptional regulation mammalian model approach applicable study protein complex mammalian proteome report prediction enable pharmacologically important downstream research example predict site starting point quantify vivo effect downstream transcription identify tfs bind complex task understand role site determine relative abundance GABA receptor isoform research line dramatically improve understanding GABA receptor regulation role disease development additionally comprehensive evaluation remain GABA receptor subunit gene reveal additional tfbinding site uncover evolutionary significance GABR cluster human genome Supplementary Data available NAR Online Charles DeLisi partially support NIH grant Daniel Roberts support NIH training grant Shelley Russek support NIH NINDS Grant fund pay Open Access publication charge article provide Boston University Bioinformatics Program','2007-01-03 00:00:00','http://dx.doi.org/10.1093/nar/gkl1062',2559,775),('b2897e1277f56641193a6db73825f707eed3e4c9.json','Sequence requirements for RNA strand transfer during nidovirus discontinuous subgenomic RNA synthesis','Nidovirus subgenomic mRNAs contain a leader sequence derived from the 5′ end of the genome fused to different sequences (‘bodies’) derived from the 3′ end. Their generation involves a unique mechanism of discontinuous subgenomic RNA synthesis that resembles copy-choice RNA recombination. During this process, the nascent RNA strand is transferred from one site in the template to another, during either plus or minus strand synthesis, to yield subgenomic RNA molecules. Central to this process are transcription-regulating sequences (TRSs), which are present at both template sites and ensure the fidelity of strand transfer. Here we present results of a comprehensive co-variation mutagenesis study of equine arteritis virus TRSs, demonstrating that discontinuous RNA synthesis depends not only on base pairing between sense leader TRS and antisense body TRS, but also on the primary sequence of the body TRS. While the leader TRS merely plays a targeting role for strand transfer, the body TRS fulfils multiple functions. The sequences of mRNA leader–body junctions of TRS mutants strongly suggested that the discontinuous step occurs during minus strand synthesis.',' genetic information RNA virus organize ef ciently practically nucleotide genome utilize protein code sequence cis act signal translation RNA synthesis RNA encapsidation genome expression strategy group positive strand RNA RNA virus produce subgenomic sg mrnas review Miller Koev replication genomic RNA mrna viral replicase supplement generation sg transcript express structural auxiliary protein encode downstream replicase gene genome sg mrnas RNA virus terminal genomic RNA different mechanism synthesis virus brome mosaic virus initiate sg mrna synthesis internally length minus strand RNA template Miller et al exempli ed red clover necrotic mosaic virus RCNMV rely premature termination minus strand synthesis genomic RNA template follow synthesis sg plus strand truncated minus strand template sit et al member order Nidovirales include coronaviruse arteriviruse evolve unique mechanism employ discontinuous RNA synthesis generation extensive set sg RNAs review Brian Spaan Lai Cavanagh Snijder Meulenberg Nidovirus sg mrnas differ fundamentally viral sg RNAs terminal genome figure common leader sequence nucleotide derive end genomic RNA attach sg RNA thè mrna model forward explain cotranscriptional fusion non contiguous part nidovirus genome sg RNA synthesis figure central model short transcription regulate sequence trs present end leader end sg RNA body region genomic RNA trs copy mrna connect leader body Spaan et al Lai et al Synthesis sg mrnas initially propose prime free leader transcript base pair complementary trs region length minus strand extend subsequently sg plus strand figure Baric et al Baric et al model base report sg minus strand present coronavirus infect cell Lai et al subsequent discovery molecule Sethna et al result reconsideration prime transcription model Sawicki Sawicki propose alternative model figure discontinuous step occur minus instead plus strand RNA synthesis model minus strand synthesis attenuate copy body trs plus strand template nascent minus strand trs complement end transfer leader trs attach mean base pair RNA synthesis reinitiate complete sg minus strand add complement genomic leader sequence subsequently sg minus strand template sg mrna synthesis presence leader complement end allow use RNA signal direct genome synthesis fulllength minus strand sequence requirement RNA strand transfer nidovirus discontinuous subgenomic RNA synthesis EMBO Journal Vol pp site direct mutagenesis trs arterivirus equine arteritis virus EAV show previously base pair sense leader trs antisense body trs crucial sg mrna synthesis van Marle et al base pairing step nascent strand transfer process essential model outline figure EAV genomic RNA contain sequence match leader trs precisely sg RNA synthesis den Boon et al Pasternak et al suggest trs similarity necessary suf cient strand transfer occur gain insight cis act signal regulate sg RNA synthesis perform comprehensive site direct mutagenesis study EAV leader body trs nucleotide trs substitute alternative nucleotide analysis reveal number strike similarity process copy choice RNA recombination occur RNA virus leader trs play target role translocation nascent strand body trs nucleotide appear ful diverse position speci base speci function addition sequence junction sg mrnas produce mutant provide strong evidence discontinuous minus strand extension model eav genome replication signi cantly affect leader trs body trs mutation dissect EAV RNA synthesis routinely use fulllength cdna clone van Dinten et al infectious EAV RNA vitro transcribe follow transfection RNA baby hamster kidney cell intracellular RNA isolate analyse northern blot hybridization van Marle et al difference transfection ef ciency total virus speci RNA genomic RNA sg mrna isolate transfected cell culture somewhat variable accurate quantitation sg mrna synthesis trs mutant require internal standard transfection ef ciency viral genomic RNA standard ampli cation dramatically affect TRS mutation prove case previously describe mutant van Marle et al ve nucleotide trs replace sequence leader trs body trs trs mutant test independent experiment Intracellular RNA isolate posttransfection early prevent spread wildtype control virus non transfected cell rst cycle analysis Transfection ef ciencie determine assay Materials method vary datum show prior RNA analysis isolated intracellular RNA correct transfection ef ciency sample lane figure represent EAV speci RNA approximately equal number EAV positive cell phosphoimager quantitation reveal genomic RNA replication mutant vary table difference example slight RNA secondary structure change trs region genomic RNA synthesis remarkably genomic RNA level trs double mutant affect view result obtain pentanucleotide trs mutant assume genomic RNA internal standard analysis mutant contain single nucleotide replacement leader trs body trs region genome specify leader sequence replicase gene orf structural gene indicate nested set EAV mRNAs genome sg mrnas depict black box genomic RNA indicate position leader major body trs alternative model nidovirus discontinuous sg RNA synthesis discontinuous step occur plus strand minus strand RNA synthesis case sg mrnas synthesize sg minus strand template detail text northern analysis EAV speci RNA isolate cell transfecte RNA transcribe wild type EAV infectious cDNA clone trs pentanucleotide mutant UCAAC AGUUG result independent experiment show interaction leader body trs factor regulate EAV sg RNA synthesis numerous example regulatory interaction eukaryotic prokaryotic cell RNA virus essential process translation replication encapsidation RNA virus genomes frequently depend interaction high order RNA structure regulation sg RNA synthesis RNA virus interaction precedent tomato bushy stunt virus RNA element locate nucleotide upstream sg promoter base pair promoter necessary sg RNA production Zhang et al similarly base pair interaction complementary sequence end potato virus genomic RNA sequence upstream major sg RNA promoter require ef cient sg RNA synthesis Kim Hemenway rcnmv intermolecular interaction require sg RNA synthesis sit et al establish pivotal role interaction sense antisense RNA sequence life cycle EAV van Marle et al study role trs nucleotide test substitute conclude base pair sense leader trs antisense body trs play crucial role nidovirus sg RNA synthesis take systematic approach perform extensive site direct co variation mutagenesis study entire leader trs body trs direct synthesis abundant EAV sg RNA nucleotide trs replace possible nucleotide study van Marle et al mutation introduce leader trs body trs trs result mutant construct mutant give unique bu refer mutant change position body trs LU refer substitution leader trs DU mean substitution combine double mutant construct sg quantitate phosphoimager scanning hybridized gel correct genomic RNA lane outline figure show relative sg level mutant compare level wild type control selection interesting mutant analysis repeat time figure observe signi cant variation sg RNA synthesis comprehensive analysis effect trs mutation considerably expand understanding van Dinten et al take positive control mutant level sg synthesis calculate sg wt correct level genomic RNA internal standard text subsequently relate level sg produce wild type control experiment correct correspond genomic RNA level relative sg level wild type control set Pasternak et al discontinuous sg RNA synthesis remarkably effect single leader body trs mutation base speci different nucleotide substitution position affect sg synthesis different extent example position bu mutant retain wild type synthesis level BU BU mutant lose synthesis completely conversely leader trs change synthesis completely abolish wild type level maintain LU position BC mutant retain wild type synthesis level position single mutant lose wild type synthesis example position bu leave wild type synthesis bu produce high level imply position certain mismatch duplex plus leader trs minus body trs bu bu LU BC allow limited extent contrast mismatch allow position single nucleotide substitution abolish synthesis completely surprisingly body trs substitution position bu BU result low level despite fact mutation allow formation base pair plus leader trs provide nucleotide minus body trs provide hand position base pair show functional mutant LA LA form base pair plus leader trs minus body trs position single mutant produce reasonable level take nding suggest factor body base pair play role sg RNA synthesis primary sequence secondary structure trs dictate strong base preference certain position analysis degree complementation double mutant provide strong support assumption differentiate effect level primary trs sequence level duplex formation trs nucleotide case DU level double mutant clearly high corresponding single mutant mean base pair leader body trs nucleotide involve sg RNA synthesis double mutant reach wild type sg level double mutant position mutant DU clear contradiction prediction pair model synthesis signi cantly restore comparison value mutant figure show mutant position mutant sg produce double mutant appear limit level allow body trs mutation level double mutant leader mutant DU da da DU clearly substitution restoration possibility duplex formation restore sg RNA synthesis apparently effect body trs mutation level primary sequence secondary structure duplex restore effect double mutation body trs mutant fall distinct type determine position chemistry substitution mutant rst type sg RNA synthesis impair mainly disruption body trs duplex effect compensate introduction corresponding mutation leader trs double mutant sg RNA synthesis restore compare corresponding single mutant mutant second type sg RNA synthesis regulate consequence trs duplex disruption disruption primary sequence secondary structure body trs obviously effect compensate mutate leader trs corresponding double mutant restoration sg RNA synthesis contrast nding body trs mutant obtain leader trs mutation appear determine level sg synthesis corresponding double mutant figure effect mutation leader trs duplex restore effect double mutation suggest affect duplex formation indicate leader trs probably additional sequence speci function sg RNA synthesis addition participation base pair fact single leader trs mutation Nidovirus discontinuous subgenomic RNA synthesis position severely repress synthesis indicate base pair trs nucleotide contribute sg RNA production respect signi cant leader trs mutant high level LA LA form base pair maintain duplex observation leader trs mutation bè rescue introduce complementary mutation body trs body trs mutation bè rescue correspond change leader trs clearly illustrate mutant restoration trs base pair possibility synthesis double mutant DU signi cantly increase compare LU level bu restoration duplex DU exert clear effect sg production compare LU effect sg RNA synthesis compare bu exempli e dominant nature mutation primary sequence body trs contrast instance BC mutation probably affect duplex formation synthesis restore wildtype level DC double mutant result indicate strong base preference constraint body trs position interpret base preference accurately necessary limit analysis double mutant mutant regulation sg RNA synthesis sequence change body trs disruption trs duplex strict preference position body trs position substitution allow signi cant level wild type position mutant retain wild type level position sequence constraint stringent substitution allow wild type level DC DC reach position body trs allow double mutant produce word functional EAV body trs base analysis single nucleotide substitution describe wild type nucleotide show upper case nucleotide allow wild type level show low case remarkably trs nucleotide conserve trs arteriviruse snijder Meulenberg fact DC retain synthesis correspond nicely presence position arteriviruse recently Almazan et al Thiel et al infectious cdna clone lack coronaviruse consequently study coronavirus sg RNA synthesis carry defective interfere DI RNAs replicon carry body trs moderate level sg mrnas produce presence helper virus system Joo Makino van der et al perform body trs mutagenesis study murine coronavirus MHV Joo Makino systematically mutagenize core MHV body trs contrast result find body trs mutant sg RNA synthesis DI RNA genome abolish support normal level sg RNA production possible MHV TRS study tolerant single nucleotide mismatch EAV sg similar study van der et al observe substitution position MHV body trs maintain duplex change base pair base pair reduce sg RNA level strongly substitution disrupt duplex van der et al imply case EAV trs duplex formation factor determine coronavirus sg RNA synthesis limitation DI RNA system leader trs mutagenize study body trs speci effect distinguish effect level duplex formation discontinuous step nidovirus sg RNA synthesis occur minus strand RNA synthesis recent study arterivirus coronavirus sg RNA synthesis van Marle et al Baric Yount Sawicki et al discontinuous minus strand extension model figure gain ground model predict trsderive sequence form junction sg mrna copy body trs leader trs leader prime transcription model predict opposite figure determine origin junction sg mrnas help distinguish model wild type situation EAV leader body trs identical consequently determine origin sg mrna junction problem overcome trace mutation introduce leader body trs mutant retain ability produce previous study van Marle et al find nucleotide junction sequence derive exclusively body trs leader trs show direct sequencing product obtain residual produce mutant BC LC BC LC van Marle et al approach analyse mutant BC BC transcript contain mutate nucleotide derive body trs datum show assume crossover event occur join map crossover position sg RNA junction sequence leave intriguing question crossover site map precisely word nucleotide junction sequence derive body trs leader position mutant describe answer question Figure striking result mutant bu bu LU contain exclusively body trs derive nucleotide position mutant crossover site map precisely trs nucleotide position mean complete body junction sequence EAV sg mRNA body trs derive hand sg RNAs mutant LU bu LU contain mixed population leader trs body trs derive nucleotide position figure LU bu LU Remarkably pattern correlate relative amount sg produce mutant figure mutant produce population sg RNAs mix respect origin nucleotide position junction lose synthesis completely hand mutant contain exclusively body nucleotide position retain high level synthesis observation explain follow wild type situation large majority crossover probably occur position lead body trs derive nucleotide position sg RNA low number crossover place nucleotide result leader trs derive nucleotide position mutant sg RNA synthesis block substitution position de cient crossover retain ability crossover detect sequence analysis conversely position mutant retain reasonable sg RNA synthesis crossover occur position obscure minority crossover sequence electropherogram alternatively position trs mutation strongly interfere sg RNA synthesis force shift crossover site remain molecule believe present nding strongly support discontinuous minus strand extension model fact complete body trs copy sg RNA dif cult reconcile alternative model sg RNA synthesis genomic minus strand template prime free plus strand leader transcript contain leader trs end figure explain presence complete copy body trs sg mrna model assume exonuclease activity trim free leader transcript prior extension sg mRNA Baker Lai note single base pair leave hold leader molecule template enzymatic activity unprecedented RNA virus exist yeast retrotransposon Ke et al reverse transcription prime internal region trna system duplex remove single strand tail trna base pair trs end nidovirus leader base pair template energetically unfavourable RdRp instead start elongation intact properly position leader primer disrupt newly form duplex degrade leader RNA reinitiate polymerization base pair primer template show virus transcription require sequence match cellular RNA primer viral template Plotch et al nidovirus system leader RNA xe template solely interaction targeting nascent strand trs base pair extremely puzzling sequence datum sg RNA junction arteriviruse dif cult reconcile leader prime transcription model porcine simian arteriviruse Meulenberg et al Godeny et al junction sg RNAs map nucleotide upstream body trs leave single nucleotide hold putative free leader template hypothetical trim hand nding datum explain readily discontinuous minus strand extension model Figure nucleotide Fig sequence analysis junction position trs mutant sequence determine directly sg speci product mutant sequence show correspond plus strand sg sequence relate technical reason minus strand sequence determine mutant mirror image electropherogram show corresponding plus strand sequence list panel mutant sequence alignment leader red body blue trs surround sequence show trs box junction detect sequence analysis show yellow duplex form body trs complement end leaderless sg minus strand leader trs genomic RNA template suf ce position nascent minus strand properly subsequent elongation add complement leader sequence case nascent minus strand contain entire body trs complement end moment strand transfer lead body trs derive junction sequence sg mrna molecule small number transcript minus strand synthesis appear interrupt nucleotide body trs copy strand transfer resume incorporate complement nucleotide leader trs state postulate detection phenomenon determine level crossover position allow mutation introduce position body trs leader trs formally exclude trim activity degrade nucleotide minus strand strand transfer note discontinuous minus strand extension model Figure activity disturb proper positioning nascent minus strand leader template trs duplex shorten nucleotide Nidovirus discontinuous minus strand extension resemble similarity assist copy choice RNA recombination discontinuous sg RNA synthesis nidoviruse occupy RNA virus world mode sg RNA production clearly different RNA virus resemble welldocumente RNA virus feature RNA recombination recent review Nagy Simon Aaziz Tepfer Worobey Holmes experimental evidence support RdRp template switch Kirkegaard Baltimore main mechanism RNA recombination mechanistically template switch involve transfer nascent strand RNA template donor acceptor nidovirus discontinuous sg RNA synthesis involve transfer nascent RNA strand sg RNA site template base datum currently available refer discontinuous minus strand extension model working model nidovirus sg RNA synthesis apply term model Chang et al Brian Spaan van Marle et al donor strand body genomic RNA template acceptor strand leader genomic RNA template nascent strand discontinuously synthesize minus strand Nagy Simon de ned main class RNA recombination similarity essential similarity non essential similarity assist recombination de ned mechanism strand transfer determine sequence similarity parental rna additional RNA determinant present parental RNAs result present study strongly suggest nidovirus discontinuous sg RNA synthesis consider special case high frequency similarity assist RNA recombination obvious function leader trs ensure delity strand transfer base pair end nascent strand body trs donor template additional sequence speci function function apparently pause terminate nascent strand synthesis provide opportunity strand transfer addition body trs derive nucleotide play role reinitiation nascent strand synthesis acceptor template give compact nature EAV TRS possible nucleotide ful multiple task RNA secondary structure body trs regulate sg RNA synthesis sequence speci function body trs reveal study exert level primary sequence secondary structure number RNA virus RNA secondary structure motif locate proximity sg RNA promoter vital sg RNA synthesis alfalfa mosaic virus Haasnoot et al turnip crinkle virus TCV Wang et al barley yellow dwarf virus Koev et al structure sg RNA promoter region template strand require sg RNA synthesis sg promoter virus especially interesting contain domain secondary structure primary sequence important sg RNA synthesis domain act primary sequence secondary structure Koev et al similarly RNA secondary structure play minor role sequence speci recognition BMV sg RNA promoter RdRp Siegel et al suggest previously RNA secondary structure body trs region contribute attenuate potential determine relative portion nascent minus strand transfer leader trs template pasternak et al present unknown EAV body trs RNA structural motif essential body trs function recognize protein factor sequence speci manner likely Figure ve trs nucleotide substitute produce sg fold wild type control fact sequence EAV genome match leader trs perfectly sg mrna synthesis argue recognition speci sequence pasternak et al probably mutagenesis body trs disturb RNA structure necessary function example explain fact BU substitution reduce fold rescue mutation leader trs wild type body trs contain position protein factor involve sequence speci trs recognition expect recognize trs similarly RNA structure important recognition protein BU substitution probably disturb structural motif TRS present trs hand conservation trs arteriviruse suggest sequence speci recognition study require distinguish possibility TCV satellite RNA recombination system hairpin structure acceptor strand homology region necessary template switch hairpin postulate bind RdRp homology region target nascent strand crossover site TCV RdRp probably recognize secondary tertiary structure hairpin individual nucleotide play important role EAV leader trs acceptor template predict reside loop extensive hairpin base pair interaction body trs complement end nascent minus strand resemble certain antisense RNA regulate control mechanism base interaction single strand tail hairpin loop van Marle et al reference possible EAV RdRp accessory protein bind stem long hairpin present leader trs case leader trs recognize protein sequence speci manner body trs well candidate serve protein recognition site protein mediate pausing nascent strand synthesis nascent strand transfer resemble dna dependent RNA polymerase termination system speci dna bind terminator protein bind termination sequence Reeder Lang function HIV nucleocapsid protein promote minus strand strong stop dna transfer Guo et al EAV replicase component recently show possess sg RNA synthesis speci activity Tijms et al good candidate regulatory role residue predict form zinc nger structure show necessary sg RNA synthesis interestingly zinc nger structure HIV nucleocapsid protein facilitate strand transfer Guo et al finally note RNA structure nascent strand pause strand transfer reinitiation illustrate fact stable hairpin structure nascent strand promote termination transcription Escherichia coli RNA polymerase Wilson von Hippel direct mutagenesis RNA transfection analysis site direct mutagenesis EAV leader body trs carry describe van Marle et al mutant construct sequence follow vitro transcription infectious cdna clone length EAV RNA introduce cell electroporation describe van Dinten et al assays EAV speci antisera perform posttransfection describe van der Meer et al visualize nucleus cell counting nuclear DNA stain mg ml Hoechst Sigma cell count Scion Image software Scion Corporation percentage transfected cell calculate basis number cell positive EAV replicase component Pedersen et al RNA analyse cell lyse post transfection Intracellular RNA isolation perform acidic phenol method describe Pasternak et al total intracellular RNA resolve denature gel hybridization dry gel radioactively label oligonucleotide probe complementary end EAV genome recognize viral mrna molecule genomic subgenomic phosphoimager quantitation individual band perform describe Pasternak et al determine junction sequence sg speci carry describe van Marle et al antisense RT PCR primer body region sense PCR primer match leader sequence product sequence directly describe Pasternak et al leader derive primer ABI prismô sequence kit Perkin Elmer ABI PRISMÔ Genetic Analyser Perkin Elmer','2001-12-17 00:00:00','http://dx.doi.org/10.1093/emboj/20.24.7220',3427,804),('b3ab47699907f78c2ac478d06f682f56fa0cf351.json','Bench-to-bedside review: Critical illness-associated cognitive dysfunction – mechanisms, markers, and emerging therapeutics','Cognitive dysfunction is common in critically ill patients, not only during the acute illness but also long after its resolution. A large number of pathophysiologic mechanisms are thought to underlie critical illness-associated cognitive dysfunction, including neuro-transmitter abnormalities and occult diffuse brain injury. Markers that could be used to evaluate the influence of specific mechanisms in individual patients include serum anticholinergic activity, certain brain proteins, and tissue sodium concentration determination via high-resolution three-dimensional magnetic resonance imaging. Although recent therapeutic advances in this area are exciting, they are still too immature to influence patient care. Additional research is needed if we are to understand better the relative contributions of specific mechanisms to the development of critical illness-associated cognitive dysfunction and to determine whether these mechanisms might be amenable to treatment or prevention.',' advent year ago specialty critical care remarkable advance care severely ill patient mortality rate commonly encounter critical illness severe sepsis acute respiratory distress syndrome ARDS decline sharply past decade great number patient survive intensive care increasingly evident quality life critical illness optimal instance nearly half ards survivor manifest neurocognitive sequelae year illness fall percentile normal distribution cognitive function consider Americans wish keep alive severe irreversible neurologic damage finding concern cognitive dysfunction CD common critically ill patient acute illness long acute illness resolve delirium form acute cd manifest fluctuating change mental status inattention alter level consciousness occur mechanically ventilate intensive care unit ICU patient clinician consider ICU delirium expect iatrogenic consequence recent datum associate delirium increase duration mechanical ventilation ICU stay bad month mortality high cost chronically critical illnessassociate cd manifest difficulty memory attention executive function mental processing speed spatial ability general intelligence interestingly patient develop acute cd develop chronic cd hospital discharge suggest entity share common etiology clearly define risk factor critical illness associate cd little understanding underlying pathophysiology precise mechanism unknown likely multiple mechanism work give patient figure choose focus mechanism appear great merit neurotransmitter abnormality occult diffuse brain injury bench bedside review discuss evidence support mechanism potential marker evaluate mechanism individual patient emerge therapy prevent mitigate critical illness associate cd relative dopamine excess central nervous system CNS antipsychotic haloperidol antagonize central dopamine receptor counteract cognitive effect anticholinergic medication support anticholinergic hypothesis drug potent central anticholinergic effect tricyclic antidepressant antihistamine particularly likely cause delirium medication commonly ICU generally consider anticholinergic blocker opiate furosemide digoxin glucocorticoid benzodiazepine recently show central anticholinergic property volatile anesthetic sevoflurane intravenous anesthetic propofol anticholinergic effect responsible postoperative delirium complex phenomenon postoperative cognitive dysfunction acute illness associate production endogenous anticholinergic substance study elderly medical inpatient detectable anticholinergic activity serum medication individual anticholinergic activity characterization substance improve understanding delirium lead useful intervention strategy consider activation specific cholinergic pathway inhibit proinflammatory cytokine synthesis protect endotoxemia ischemia reperfusion injury tempting speculate inhibition pathway exogenous endogenous contribute CD outcome critical illness assess overall risk develop cd pose medication central anticholinergic activity give patient individual difference drug pharmacokinetic dose receive poor estimate patient overall anticholinergic burden objectively measure anticholinergic burden individual patient assay refer serum anticholinergic activity SAA describe Tune Coyle SAA measure ability individual serum block central muscarinic receptor rat forebrain preparation elevated SAA level associate cognitive impairment study medical ward inpatient community dwell senior single small study assay investigate cd ICU patient Golinger colleague examine SAA level surgical ICU patient find mean SAA level draw hour mental status change significantly great delirious patient delirium ng ml versus ng ml result apply critically ill patient uncertain study examine SAA broad range ICU admit diagnosis medical ICU setting furthermore SAA measurement require fresh rat brain preparation use likely remain limited research setting foreseeable future pathophysiologic mechanism predispose factor think underlie critical illness associate cognitive dysfunction Apo apolipoprotein HIV human immunodeficiiency virus ht serotonin hydroxytryptamine GABA aminobutyric acid NE norepinephrine neurotransmitter system dopamine serotonin aminobutyric acid GABA norepinephrine noradrenaline glutamate think contribute critical illnessassociate cd dopaminergic hyperfunction think underlie cognitive symptom schizophrenia dopamine administration risk factor delirium serotonin syndrome consequence excess serotonergic agonism see selective serotonin reuptake inhibitor variety drug drug combination single therapeutic dose selective serotonin reuptake inhibitor cause syndrome manifest mental status change autonomic hyperactivity neuromuscular abnormality gaba abnormality think contribute hepatic encephalopathy mediate branched chain aromatic amino acid act false neurotransmitter excess gaba activity occur withdrawal chronic ethanol benzodiazepine use known dangerous cause delirium acutely sedative stimulate GABA receptor benzodiazepine probably propofol impair cognitive function deliriogenic raise possibility strategy minimize sedative drug accumulation daily interruption sedative infusion show reduce duration mechanical ventilation ICU hospital length stay reduce incidence duration delirium sedative drug lead neurocognitive deficit long use unknown suggest certain high risk group old year pre existing cognitive impairment hyperfunction fight flight response lead panic attack delusion Glutamate implicate chinese food syndrome food high amount monosodium glutamate interfere normal neurotransmission cause confusion complete review neurotransmitter abnormality underlie delirium reader refer critical illness associate CD solely acute medication effect probably resolve exposure end significant percentage individual develop delirium hospital continue demonstrate symptom cd discharge patient manifest decrease cerebral activity increase cognitive deterioration likely develop dementia patient delirium patient develop delirium great rate decline cognitive test nondelirious patient take observation raise possibility degree occult diffuse brain injury consequence local hypoxia hypoperfusion cytokine mediate inflammation microvascular thrombosis characterize multisystem organ dysfunction critical illness occur patient give organ system damage force implausible brain uniquely spare datum support occult diffuse brain injury cause critical illness associate cd come study sepsis septic encephalopathy form delirium animal model sepsis oxidative damage occur early hippocampus cerebellum cortex significant alteration cerebral vascular hemodynamic tissue acid base balance indicate cerebral ischemia acidosis occur sharshar colleague complete study compare brain pathology small number patient die septic shock patient die cause septic patient demonstrate diffuse severe ischemic hemorrhagic CNS lesion correlate persistent hypotension severe coagulation disorder multiple microscopic focus necrosis involve white matter pon see ischemia apoptosis cerebral autonomic center white matter lesion associate elevated level proinflammatory cytokine suggest possible role inflammation microvascular thrombosis genesis CNS injury study demonstrate ischemic brain injury occur sepsis determine delirium occur study attempt examine relationship ischemic brain injury delirium study patient severe sepsis multiple organ dysfunction severe hypotension factor multivariable analysis associate delirium suggest sepsis relate encephalopathy cause ischemic damage metabolic abnormality study examine cerebral blood flow cerebral oxygen metabolic rate patient septic encephalopathy multiple organ dysfunction find significantly low normal awake individual study support idea occult brain injury cause delirium author use standardized diagnostic Statistical Manual Mental Disorders base tool diagnose delirium Confusion Assessment Method ICU support hypothesis acute inflammation lead brain injury subsequent development delirium recent study find delirium postoperative hipfracture patient significantly associate serum level reactive protein acute phase protein marker acute inflammation importantly patient study diagnose delirium Confusion Assessment Method ward base predecessor Confusion Assessment Method ICU provide DSM IV base evidence acute inflammation causative pathway delirium brain target free radical damage large lipid content high rate metabolism low antioxidant capacity free radical induce oxidative stress play role delirium see cardiopulmonary bypass Karlidag colleague note patient low preoperative level catalase erythrocyte base antioxidant enzyme susceptible delirium postoperatively suggest preoperative catalase level day identify risk patient antioxidant treatment preoperatively reduce incidence delirium remain speculative regional cerebral blood flow appear reduce delirium xenon enhance compute tomography CT Yakota colleague demonstrate significant focal global brain hypoperfusion ICU patient hypoactive delirium recovery delirium cerebral blood flow return normal imply cerebral hypoperfusion contribute development delirium study ards survivor suggest combination acute hypoxia hypoperfusion hyperglycemia play important role long term cognitive sequelae critical illness difficult demonstrate clear relationship give lengthy interval stimulus effect great number additional contributing variable obscure downstream effect ards survivor Hopkins colleague show degree cd year significantly correlate duration hypoxia mean arterial blood pressure mmhg ICU stay animal hyperglycemia markedly enhance hypoxic ischemic brain damage increase brain edema disrupt cerebral metabolism ards survivor duration blood glucose great mg dl show correlate bad visual spatial ability visual memory process speed executive function year give recent interest maintain tight glucose control critical illness means reduce mortality interesting patient manage technique well cognitive outcome clearly approach need balance benefit tight glucose control know risk hypoglycemia pose perceive difficulty look evidence occult brain injury human apparent need cns tissue specimen prove brain injury actually occur study stroke trauma cardiopulmonary bypass associate brain injury serum marker brain injury correlate extent cns damage neuron specific enolase NSE myelin basic protein MBP marker look evidence occult brain injury critical illnessassociate dimeric calcium bind protein consist subunit unit highly brain specific locate mainly astrocyte circulate level elevate patient cerebral ischemia cardiopulmonary bypass associate decline explicit memory function traumatic brain injury TBI mild head injury serum level correlate clinical measure injury severity neuroradiologic finding outcome include postconcussion symptom elevated serum level recently demonstrate critically ill patient respiratory failure porcine model endotoxic shock acute lung injury group elevated level associate hippocampal histopathologic change include basophilic shrunken neuron pyramidal cell layer interestingly beneficial detrimental effect low level protective neurotrophic effect high level lead exacerbation neuroinflammation neuronal dysfunction marker astrocyte damage NSE MPB marker neuron white matter myelin damage respectively NSE protein base enzyme find primarily neuron serum level NSE elevate TBI exhibit close relationship outcome severe head injury volume contusion minor head injury interestingly elevated NSE level recently show predict death small study patient severe sepsis patient acute cns disorder stroke neurotrauma MBP major protein component myelin serum level MBP elevate disease myelin breakdown study patient TBI show MBP level correlate clinical measure severity allow early prediction outcome development neuroimaging functional magnetic resonance imaging MRI positron emission tomography revolutionize understanding abnormal brain function disease state include schizophrenia Parkinson disease post traumatic stress disorder study critical illness associate CD associate occult brain injury human useful imaging test detect subtle evidence brain injury unfortunately traditional CT scan MRI appear sensitive pick microscopic cellular change underlie CD small study assess brain CT finding critically ill patient sepsis study demonstrate CT abnormality brain pathology nonsurvivor consistent previously cite finding Sharshar colleague recent study ards survivor find individual exhibit sign significant brain atrophy ventricular enlargement head ct obtain acute illness significant correlation abnormality subsequent neurocognitive score new mri technique prove useful identify occult brain injury critically ill patient specifically highresolution dimensional mri assess noninvasively difference brain tissue sodium concentration highly sensitive marker tissue viability highlight area traditional MRI miss method base sodium ion homeostasis tightly regulate body major energy consuming process event perturb energy level cell disrupt sodium ion gradient ischemia important impact cell viability tissue sodium concentration MRI successfully evaluate CNS include nonhuman primate study clinical study stroke reversible focal brain ischemia assess patient acute chronic critical illnessassociate recent development preliminary interest potential prevent mitigate critical illness associate year manage agitation mechanically ventilate ICU patient recommend drug treatment ICU delirium Kalisvaart colleague compare effect haloperidol prophylaxis mg day preoperatively day postoperatively placebo elderly hip surgery patient risk delirium difference incidence postoperative delirium treatment control group haloperidol group significantly reduce severity duration delirium day versus day Haloperiodol appear reduce length hospital stay develop delirium day versus day recent retrospective cohort study examine haloperidol use patient mechanically ventilate long hour despite similar baseline characteristic patient treat haloperidol significantly low hospital mortality receive drug versus association persist adjust potential confounder observational nature study potential risk associate haloperidol use finding require confirmation randomized control trial apply routine patient care leung colleague test hypothesis gabapentin add agent treat postoperative pain reduce occurrence postoperative delirium patient age year old undergo spine surgery randomly assign gabapentin mg placebo mouth hour surgery continue day postoperatively postoperative delirium occur gabapentin treat patient placebo patient reduction delirium appear opioid spare effect gabapentin give small size study result require confirmation Donepezil cholinesterase inhibitor increase synaptic availability acetylcholine improve cognitive function Alzheimer disease sampson colleague randomly assign elderly patient undergo elective total hip replacement donepezil mg placebo immediately follow surgery hour day Donepezil tolerate adverse event drug significantly reduce incidence delirium versus length hospital stay mean standard error day versus day outcome show consistent trend suggest possible benefit author project sample size patient require definitive trial Dexmedetomidine sedative effect selective stimulation locus ceruleus CNS anticholinergic gaba stimulate effect potential delirium spare sedative preliminary result present abstract form cardiac surgery patient randomly assign dexmedetomidine postoperative sedation nonsignificantly low incidence postoperative delirium compare sedate propofol combination fentanyl midazolam versus versus author report plan enroll total patient study impressive difference statistically significant great number patient recombinant human erythropoietin rHuEPO receive considerable attention potential transfusion sparing strategy ICU interestingly EPO receptor express nervous system systemically administer rHuEPO reach site brain preclinical study rHuEPO reduce neuronal injury produce focal ischemia TBI spinal cord injury subarachnoid hemorrhage enthusiasm use general neuroprotectant ICU temper potential risk thromboembolism considerable cost drug concern safety partially address recent finding erythropoietin derivative tissue protective hematopoietic property chemically inert gas anesthetic agent contrast enhancement CT scan rat xenon appear protect brain neurologic damage associate use cardiopulmonary bypass effect potentially related methyl aspartate receptor antagonism tendency expand gaseous bubble bypass associate cerebral air embolus abolish beneficial effect worsen cerebral outcome setting ischemic stroke TBI variety compound potential improve neurologic outcome example free radical trap agent reduce disability day give hour stroke onset pilot randomize trial patient simvastatin give hour stroke onset significantly improve neurologic functioning National Institutes Health Stroke Scale score day Ethyl pyruvate pyruvate derivative prevent mortality murine sepsis model reduce motor impairment neurologic deficit infarct volume rat stroke model give late hour middle cerebral artery occlusion rodent model TBI cyclosporin reduce acute motor deficit improve cognitive performance give traumatic insult phase II dose escalation trial currently underway human mount evidence suggest mild moderate hypothermia mitigate neurologic injury Shankaran colleague find body hypothermia hour reduce risk death disability infant moderate severe hypoxic ischemic encephalopathy adult successfully resuscitate cardiac arrest moderate hypothermia hour increase rate favorable neurologic outcome reduce mortality practical limitation therapeutic hypothermia reach target temperature take hour fast currently available cool technique polderman colleague demonstrate hypothermia induce safely quickly min mean ice cold intravenous fluid combine icewater cool blanket cognitive rehabilitation involve teaching skill strategy target specific problem perception memory thinking problem solve goal improve function compensate deficit benefit cognitive rehabilitation known care patient stroke anoxia TBI predict benefit prove challenging severely disabled patient dramatic neurocognitive recovery study evaluate effectiveness cognitive rehabilitation patient recover non neurologic critical illness stand reason patient benefit find cognitively impaired cognitive impairment critically ill patient appear underrecognize ICU physical rehabilitation provider patient refer cognitive rehabilitation therapy education cognitive sequelae critical illness need enhance referral rehabilitation weakness physical debilitation cognitive impairment cognitive function important relatively understudied outcome critical illness evidence suggest neurotransmitter abnormality occult diffuse brain injury important pathophysiologic mechanism underlie critical illness associate cd marker evaluate influence mechanism individual patient include following SAA certain brain protein NSE MPB mri tissue sodium concentration recent advance area exciting immature influence patient care additional research need understand well relative contribution specific mechanism development critical illness associate cognitive dysfunction determine mechanism amenable treatment prevention article thematic series Translational research edit John Kellum article series find online','2006-11-15 00:00:00','http://dx.doi.org/10.1186/cc5078',2335,935),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json','Neutralizing Antibody Fails to Impact the Course of Ebola Virus Infection in Monkeys','Prophylaxis with high doses of neutralizing antibody typically offers protection against challenge with viruses producing acute infections. In this study, we have investigated the ability of the neutralizing human monoclonal antibody, KZ52, to protect against Ebola virus in rhesus macaques. This antibody was previously shown to fully protect guinea pigs from infection. Four rhesus macaques were given 50 mg/kg of neutralizing human monoclonal antibody KZ52 intravenously 1 d before challenge with 1,000 plaque-forming units of Ebola virus, followed by a second dose of 50 mg/kg antibody 4 d after challenge. A control animal was exposed to virus in the absence of antibody treatment. Passive transfer of the neutralizing human monoclonal antibody not only failed to protect macaques against challenge with Ebola virus but also had a minimal effect on the explosive viral replication following infection. We show that the inability of antibody to impact infection was not due to neutralization escape. It appears that Ebola virus has a mechanism of infection propagation in vivo in macaques that is uniquely insensitive even to high concentrations of neutralizing antibody.',' editor note potential efficacy pre post exposure prophylaxis Ebola virus infection fundamentally important question neutralizing body important Ebola virus resistance address relate manuscript issue PLoS Pathogens doi journal Feldmann et al passive transfer relatively high concentration neutralizing antibody protect challenge range virus animal model human protection case form sterilize immunity viral replication observe follow challenge case replication observe protection disease achieve presumably neutralizing antibody sufficiently blunt infection cell innate immunity resolve infection expect passive neutralizing antibody effective challenge acute virus acute viral infection resolve absence neutralizing antibody blunt effect passive antibody provide time development effective cellular immune response contrast chronic virus present great challenge passive antibody absence sterilize immunity window opportunity virus establish chronic infection cellular immunity mobilize Ebola virus EBOV cause severe acute infection human infection Ebola Zaire strain Zaire ebolavirus ZEBOV produce mortality range death generally occur follow appearance symptom single report describe use convalescent sera treat EBOV infection patient report bad stage disease clear serum antibody responsible recovery neutralizing antibody titer survivor EBOV infection tend low isolate neutralizing human monoclonal antibody mAb good potency convalescent individual ability passive antibody protect EBOV investigate number animal model guinea pig mouse model use ebov serially passage adapt replication respective animal highly lethal protection demonstrate guinea pig model neutralizing horse sheep goat immunoglobulin IgG EBOV human anti ebov GP mAb IgG antibody neutralize ZEBOV Kikwit inhibitory concentration IC lg ml IC lg ml Vero cell IC approximately lg ml IC lg ml primary human monocyte macrophage show administer subcutaneously dosage mg kg challenge antibody protect robust ZEBOV challenge plaque form unit pfu guinea pig model provide model EBOV infection likely close human infection human virus directly macaque need adaptation course disease mirror see human cynomolgus macaque Macaca fascicularis ZEBOV infection produce mortality rate death occur follow infection pfu rhesus macaque Macaca mulatta ZEBOV produce mortality death occur infection pfu contrast guinea pig experiment passively transfer polyclonal equine neutralize IgG describe provide minor benefit form slight delay onset viremia day day follow ZEBOV challenge cynomolgus monkey significant reduction mortality observe protection EBOV primate observe low dose challenge model neutralizing equine IgG protect baboon ld lethal dose ZEBOV challenge IgG give infection serum contain high neutralizing antibody titer similarly neutralize ovine serum protect baboon LD ZEBOV challenge study ability passively transfer neutralizing human mAb protect ZEBOV challenge rhesus macaque passive transfer fail protect macaque challenge ZEBOV furthermore minimal effect explosive viral replication follow infection show ELISA antibody present high level serum monkey neutralization escape responsible resistance virus antibody prophylaxis evaluate IgGl protect Ebola virus infection nonhuman primate animal model antibody passively transfer rhesus macaque follow challenge ZEBOV Kikwit isolate later protection virus challenge neutralize antibody naive animal require high dose antibody high dose mg kg close maximum practically achievable addition give second bolus mg kg day follow infection result antibodytreate animal steady increase plasma viremia pfu ml day figure level plasma viremia closely parallel see control animal typically see historical control second bolus antibody give day appear impact rate increase plasma virus figure treat animal euthanize moribund day post infection fourth treat animal show decrease plasma viral load peak survive day moribund euthanize monkey severe symptom animal study conclude animal suffer disease ZEBOV evidence example copious Ebola virus antigen lung antibody load measure virus challenge day challenge antibody boosting day ELISA design detect human antibody bind immobilize ZEBOV glycoprotein serum antibody level day approximate range lg ml table figure Plasma Viremia Macaques challenge ZEBOV show measured viremia log pfu ml antibodytreate monkey untreated control animal EHD day plasma plaque assay describe Materials Methods mg kg human antibody give intravenously rhesus macaque challenge pfu intramusculary ZEBOV Kikwit isolate ab antibody Pathogens January volume issue Ebola Antibody Activity VivoEbola virus fear human pathogen mortality approach extremely rapid disease course lead death day infection Antibodies able inhibit viral infection culture neutralizing antibody typically prevent viral infection animal human present prior infection sufficient concentration neutralizing antibody provide passive administration induce vaccination previously show human neutralizing antibody protect guinea pig Ebola virus antibody protect monkey Ebola virus surprisingly appear little impact rapid course infection despite present high level blood monkey conclude administer antibody prior immediately follow exposure Ebola virus example accident research setting bioterrorist attack unlikely effective prevent disease recent success protect monkey Ebola virus vaccination independent antibody likely critically dependent cooperation antibody cellular control monkey EHD untreated challenge negative monkey treat challenge similar background level reactivity glycoprotein anti human antibody treat monkey treatment mg kg dose typically produce serum mAb concentration animal order lg ml injection neutralization titer IC ZEBOV order lg ml depend target cell presence complement concentration animal time challenge day expect great order ic concentration typically provide sterilize immunity challenge number virus formal possibility neutralizing antibody little effect course infection treat monkey rapid emergence neutralization escape mutant accordingly virus isolate selection plasma day monkey day monkey virus sensitive essentially neutralization observe vitro lg ml plaque assay Materials Methods Vero target cell order gain well understanding difference pathology control antibody treat animal level virus different organ survey postmortem viral level liver spleen kidney adrenal gland lung mesenteric inguinal lymph node high pfu control treat animal table monkey survive long animal day show major histopathological difference infected monkey norm ZEBOV infection relatively low viral level observe organ liver spleen adrenal gland addition large immunoreactive monocyte find blood monkey figure typically small immunoreactive monocyte see ZEBOV infection presence large immunoreactive monocyte simply reflect uptake antibody coat virion Fc receptor subsequent viral clearance interesting note previous study implicate mononuclear phagocyte vehicle transport filovirus particle specific organ liver spleen virus particle remain infectious follow Fc receptormediate uptake subset cell compare DC SIGN mediate uptake dendritic cell course disease monkey represent net result inhibition neutralization enhancement antibody mediate cellular uptake interestingly monkey fare somewhat well lack virus liver spleen presence large immunoreactive monocyte suggestive lower Fc receptor mediate uptake virus reduce activity mononuclear phagocytic system describe case potent neutralizing human monoclonal antibody administer high serum concentration show unable protect macaque challenge lethal dose ZEBOV antibody appear little effect course virus replication disease treat animal neutralization escape appear explain lack protection observe antibody animal limited infection observe macaque eventually succumb effect viral disease challenge dose pfu correspond EBOV contain relatively small quantity fluid order ll infected individual give high titer virus typically find individual order pfu ml blood example challenge dose unreasonable term natural exposure virus negative result passive antibody contrast strongly recent success prevent EBOV infection macaque vaccination mean neutralizing antibody unimportant vaccine answer question await study plausible hypothesis explain datum allow important contribution antibody vaccine protection hypothesis argue passive antibody unable completely block EBOV entry cell cell infect virus replication explosive contain de novo generate cellular immune response vaccination hand provide memory cell rapidly recruit effector cell limit infection certainly mAb able provide protection disease follow ZEBOV challenge guinea pig model immunity sterilize viral replication note pfu EBOV lethal dose primate failure passive antibody achieve sterilize immunity critical immunohistochemistry discuss give intriguing hint uptake antibody coat virion monocyte possibly role play course infection follow antibody treatment note previous vitro study isolate human monocyte macrophage find evidence infectivity enhance antibody detailed vitro vivo investigation require firm conclusion draw note experiment carry single neutralizing monoclonal antibody possible favorable outcome apparent combination neutralize antibody combination neutralize nonneutralize antibody possibility weigh high concentration neutralizing monoclonal antibody experiment efficacy antibody guinea pig model summary inability high concentration neutralizing antibody slow viral replication infected macaque remarkable imply mechanism infection propagation virtually insensitive antibody overall result suggest monoclonal antibody prophylaxis post exposure prophylaxis unlikely effective strategy protect EBOV example follow needle stick accident research setting bioterrorist attack passive transfer experiment mg kg human antibody give intravenously rhesus macaque weight kg challenge day pfu intramuscularly ZEBOV Kikwit isolate later day monkey give antibody treatment animal carefully monitor sign disease Ebola virus plasma viremia determine day serum plaque assay describe investigator adhere Guide Care Use Laboratory Animals conduct research animal United States Army Medical Research Institute Infectious Diseases USAMRIID animal facility animal care use program accredit Association Assessment Accreditation Laboratory Animal Care International infectious material animal handle maximum containment biosafety level facility USAMRIID standard operating condition antibody purification IgGl produce purify describe Parren et al pure determine sodium dodecyl sulfate polyacrylamide gel electrophoresis contain IU endotoxin ml determine quantitative chromagenic Limulus amoebecyte lysate assay BioWhittaker Cambrex determine plaque assay plasma viremia viral load organ determine virus titration conventional plaque assay Vero cell describe assay sample dilute Eagle minimal essential medium EMEM Invitrogen heat inactivate fetal bovine serum FBS presence absence constant dilution mg ml mg ml mg ml plasma titrate dilution viremia determine count pfu Vero cell monolayer cell grow confluence plate give ml plasma additional titrate sample incubate presence co absorption cell overlaid ml EMEM contain FBS mm HEPES buffer lg gentamicin ml agarose plaque visible cell remove humidified incubator visualize plaque inverted phase microscope ml neutral red final concentration Sigma Aldrich add additional incubation plaque count Nunc Immuno Maxisorp enzyme link immunosorbent assay ELISA plate Nunc coat ll lg ml lectin Galanthus Nivalis Sigma Aldrich PBS incubate overnight plate block phosphate buffer saline PBS contain fbs well wash twice wash buffer PBS contain Tween Sigma Aldrich cell transfecte mammalian expression plasmid code transmembrane domaindelete Ebola glycoprotein Supernatant ll cell contain mg ml total protein coat room temperature RT block solution remove ELISA plate plate wash time wash buffer Monkey sera add fold dilution dilution buffer PBS contain BSA Tween incubate RT well wash time secondary antibody alkaline phosphatase conjugate goat anti human immunoglobulin IgG portion antibody Pierce dilute add incubate finally plate wash time develop tablet phosphatase substrate typically small immunoreactive monocyte see ZEBOV infection immunohistochemistry perform describe Materials Methods Aldrich ml alkaline phosphatase stain buffer pH plate assay perform manufacturer direction plate read optical density nm microplate reader Molecular Devices min add substrate panel normal sera run time assay perform immunohistochemistry section pretreate Dako Ready use Proteinase Dako min RT deparaffinization rehydration series grade ethanol blocking perform Dako Serum Free Protein Block min pre antibody exposure tissue section incubate overnight primary antibody equal mixture mouse monoclonal antibodie EBOV GP alkaline phosphatase label polymer Dako Envision System alkaline phosphatase incubate section min color develop expose tissue naphtholic acid HistoMark Red Kikegaard Perry Laboratories substrate min dark counterstaining hematoxylin positive control include archive EBOV infect cynomolgus tissue negative control include replicate section expose anti marburg virus antibodie uninfected cynomolgus macaque tissue','2007-01-19 00:00:00','http://dx.doi.org/10.1371/journal.ppat.003000',1724,678),('b6eed04e43ad653c06247b10ab05d03bdccf04bc.json','Estimating Individual and Household Reproduction Numbers in an Emerging Epidemic','Reproduction numbers, defined as averages of the number of people infected by a typical case, play a central role in tracking infectious disease outbreaks. The aim of this paper is to develop methods for estimating reproduction numbers which are simple enough that they could be applied with limited data or in real time during an outbreak. I present a new estimator for the individual reproduction number, which describes the state of the epidemic at a point in time rather than tracking individuals over time, and discuss some potential benefits. Then, to capture more of the detail that micro-simulations have shown is important in outbreak dynamics, I analyse a model of transmission within and between households, and develop a method to estimate the household reproduction number, defined as the number of households infected by each infected household. This method is validated by numerical simulations of the spread of influenza and measles using historical data, and estimates are obtained for would-be emerging epidemics of these viruses. I argue that the household reproduction number is useful in assessing the impact of measures that target the household for isolation, quarantine, vaccination or prophylactic treatment, and measures such as social distancing and school or workplace closures which limit between-household transmission, all of which play a key role in current thinking on future infectious disease mitigation.',' household fundamental unit transmission directly transmit infection addition household provide laboratory key measure transmission infectiousness generation time effect immunity vaccination study recent year considerable effort go understand dynamic transmission population organise household mathematical model effort go analyse asymptotic behaviour model elucidate threshold level transmission require infection self sustain calculate final epidemic size predict impact generalised target intervention design reduce eliminate transmission parallel method derive estimate parameter govern transmission household detailed case report scant effort appear pay apply household structure model analysis epidemic retrospectively real time concurrently mathematical model play great role interpret respond emerge pathogen model typically simple tractable variety ignore average demographic structure social mixing pattern complex computer simulation variety capture detail demographic structure dynamic behaviour determine intensive numerical analysis aim study develop method slightly simple tractable variety capture detail micro simulation show important rapidly apply daily basis emerge outbreak situation inform policy specifically aim arrive method estimate key transmission control parameter model transmission household detailed observation likely gather heat major outbreak result analysis base major simplification respect spatial social construct govern disease transmission base simple assumption free homogeneous mixing context state good robustly parameterise microsimulation gross approximation describe fabulously complex web human behaviour attempt characterise statistical property system extensive effort continue spend identify condition different type simplification household model static network model spatial metapopulation model justify develop analytical approximation describe disease transmission simplified structure individual base simulation influenza smallpox pandemic spread control incorporate detailed information population density age structure commute pattern workplace size long distance travel highlight particular importance household fundamental unit transmission review pure household model fruitfully explore detailed policy option city wide response influenza pandemic priori household model natural starting point term extend theory previously develop simple assumption homogeneous mixing analysis present focus derive new estimator individual household reproduction number denote respectively individual reproduction number define roughly average number people infect time infect entire infectious lifespan way define precisely household reproduction number define average number household household infect time infect individual reproduction number rightly play privileged role epidemiology meaningful measure contact network possible summary measure epidemic progress necessarily useful example emerge directly transmit pathogen pandemic influenza virus public health intervention target household individual enforce household quarantine offer antiviral household limit transmission household situation household reproduction number directly relate parameter characterize intervention well measure effect intervention quantity share essential property reproduction number increase infectiousness increase decrease infectiousness decrease monotonicity mark threshold separate exponentially grow epidemic equivalently exponentially decline epidemic equivalently structure paper focus derive estimator individual reproduction number household reproduction number finally example pandemic influenza dynamic measle know compartmental counterpart SIR SIS etc time infection model offer intuitive starting point model infectious disease transmission importantly application provide major advantage typically easy identify key parameter second readily adapt describe multi level transmission multi level mean withinhousehold household disadvantage hard include heterogeneity nomenclature confusing type model origin classic paper Kermack McKendrick SIR model simple time infection model know Kermack McKendrick model formalism choose predict change incidence rate function calendar time term transmissibility denote arbitrary function calendar time time infection typically reflect pathogen load precisely pathogen shed commonly single peaked function reflect pathogen growth follow immune suppression host death exotic double peaked profile associate early late transmission HIV repeat peak malaria reflect effective contact rate infectious susceptible individual change course single infection increase example person cough sneeze respiratory disease decrease person take bed illness course epidemic public health measure implement discussion component infectiousness contact find interested outbreak emerge infection describe explicitly reduction susceptible population cause epidemic formally correspond work infinite population limit assumption essential section think incorporate proportion case susceptible assumption important later section household model mathematically transmission define Poisson infection process probability time infect time ago successfully infect small time interval assumption result prediction mean incidence time follow call renewal equationthis equation state number newly infect individual proportional number prevalent case multiply infectiousness convenient realistic truncate function time asymptotic behaviour incidence determine reproduction number intuitively define reproduction number case reproduction number denote instantaneous reproduction number denote case reproduction number property individual infect time average number people infect time expect infect person infect time total infection hazard time onwards case reproduction number widely worth consider quantity instantaneous reproduction number property epidemic time average number people infect time expect infect condition remain unchanged give byTo illustrate distinction consider situation transmission rate abruptly reduce time instantaneous reproduction number estimate people case infect circumstance remain fix abruptly switch high low value time case reproduction number hand estimate people case actually infect account fact infect time spend infectious period reduction transmission occur time smoothly transition high low value derive simple estimate equation consider case function separable correspond say relative progression infectiousness function time infection independent calendar time case write product function Þw counter example reactive patient isolation introduce act reduce infectiousness late stage infection case decompose way type situation reasonable assume decompose separately different stage epidemic pre post implementation isolation measure example product arbitrary normalise function loss generality choose total integral function equal instantaneous reproduction number function distribution infection event distribute function time infection idealised definition generation time distribution denote infectiousness decompose product instantaneous reproduction number generation time distribution relationship idealised generation time distribution distribution observe generation time complex number reason infection rarely observe backcalculate generation time base surrogate appearance symptom second right censoring cause observed generation time short long expect grow decline epidemic respectively apparent reproduction number change depletion susceptible change contact rate public health measure change observed generation time infectious individual period change distribution intend measure infectiousness correspond generation time index case ideal large closed setting contact rate constant infer datum timing case yield novel estimator instantaneous reproduction numberBy substitute decomposition equation relation instantaneous case reproduction number obtain case reproduction number smoothed function instantaneous reproduction number usually incidence report discrete time series form incident case report time time case generation time distribution appropriately discretise form estimator reproduction number becomeandequation propose real time estimator reproduction number equation analyse polio transmission India base work present case reproduction number intuitively appealing quantity instantaneous reproduction number estimate equation consider practical application suffer few problem right censoring incompletely observe epidemic right censoring real problem case reproduction number track epidemic real time estimator time see equation rely know incidence future time point algorithm deal issue propose switch instead instantaneous reproduction number estimate equation simple solution right censoring complication associate estimate reproduction number practice completely absent delay detect infection leave censoring arise know baseline number infect epidemic unfold time observation record finally estimate generation time distribution straightforward strategy possible deal fact observe infection time series case form case definition base symptom hospitalisation seroconversion strategy simply ignore use case surrogate infection estimation generation time reproduction number possible characterise distribution time infection case case define symptom incubation period distribution backcalculate incidence followsa drawback approach estimate incidence time series Iˆi tend smooth relative original time series make clear problem right censoring incompletely observe epidemic estimator equation equation property estimator straightforward previously note point estimator essentially ratio incidence case fraction case observe polio tiny fraction infection lead disease order confidence interval change special case applicable case surveillance poor epidemic growth decline rate know case incidence take form rt estimator reproduction number becomewhere reproduction number express function exponential rate change likely useful formula present study detail link early ecological demographic modelling highlight subsequent analysis concern derive equation equivalent household reproduction number model define assume function describe natural history infection infected individual specialise model household transmission worth consider case different individual experience different natural history define susceptibility infection infectiousness infection denote vector random variable describe factor influence susceptibility infectiousness example standard SEIR model infection random variable duration latent period infectious period let denote probability distribution random variable new infection take account difference susceptibility define thatwhere integral take domain random variable word proportion new infection state let denote infectiousness profile individual state individual mix homogeneously transmission model define early equation generalise towhere incidence infection state define function denote integralwhich clearly depend time state total incidence time define integralBy substitute equation rewrite equation obtain tot thati substitute obtaindivide side equation equation total incidenceif define average infectiousness followsthen equation see standard Kermack McKendrick model equation word model emerge infectious disease epidemic heterogeneity susceptibility infectiousness dynamic mean total incidence infection exactly equivalent basic model infectiousness appropriately average equation expression derive average infectiousness result equation consideration heterogeneity infectiousness susceptibility heterogeneity transmit preserve infection example non random mixing different risk group situation consider lead complex result public health intervention isolation contact tracing induce heritability basic property transmission process useful exercise apply formalism elaborate derivation standard formulae basic reproduction number function exponential growth rate SEIR model approach estimate household reproduction number simply switch perspective individual household directly estimate generation time distribution time take household infect incidence infection household apply result equation estimate reproduction number function time exponential growth rate show linearise Kermack McKendrick model applicable susceptibility infectiousness heterogeneous method acceptable despite fact household heterogeneous size number people infect analogous situation approach estimate farm farm reproduction number UK foot mouth virus epidemic specifically tailor task unlikely datum collect requisite form approach human household situation section explore alternative approach explicitly model transmission household homogeneous transmission model interpret twolevel hierarchical model process guide natural history infection host consider separate drive transmission host link think function translate impact change process host change infectiousness function time infection approach take model household transmission study level hierarchical model transmission level host household household natural history infection describe individual infectiousness function assume section individual homogenous infectiousness susceptibility use predict course epidemic household derive function describe average infectiousness household household function time household infect use star symbol denote property household un star symbol denote property individual basic idea analysis illustrate Fig simplify notation main aim section study case epidemic grow exponentially consider situation infectiousness independent calendar time relax variation time somewhat slow typical duration infection household specifically model assumption individual distribute household mix randomly homogeneously outside small time interval individual infect time ago infect person random population probability time interval infect susceptible individual household probability allow depend household size empirical evidence suggest variation occur population large disease low prevalence probability household repeatedly infect function proportional function time infection result assumption discussion equation infectiousness function decompose average number people infect individual infect random non household contact generation time distribution household transmission parameter describe infection household interpretation clarify start analyse process transmission single infected household size term function consider household size individual infect time give Poisson process describe assumption list probability second individualthe probability second person infect isthe distribution time infection second individual conditional infection thenwhere rate change cumulative probability infect probability density infect time normalising factor total probability infect difference standard generation time distribution saturation effect second case tend infect early infectiousness index case increase infectiousness second individual nonhousehold member population conditional infection describe function time infection household convolution total infectiousness household isgeneralise exact result large household involve complication consider example household size individual infect time probability individual infect individual time exp tÃ Þds directly analogous situation household size somewhat great actual probability infect infect infect probability escape infection somewhat exp progress analyse system propose approximate process assume infection household approximately describe discrete generation Reed Frost model probability infect generation individual infect previous generation exp escape probability infectious susceptible pair individual consider isolation formalism propose Ludwig correspond infectious rank surrogate infectious generation dynamic recover assume time generation describe standard generation time distribution ordering infection event influence final number individual infect approximation produce exact result final number people infect household possibility later generation precede early one note case household size ignore saturation effect present equation term actual generation time household approximation overestimate time take individual infected household general form relation generation time reproduction number see equation result estimate household reproduction number provide counter balance estimate consider alternative approximation obtain assume total number case predict Reed Frost model case assume infect index case formal lower bind limit infinite infectiousness household member household infect simultaneously introduction infection household find example highly infectious measle virus approximation sufficient provide practical lower bind probability different chain infection household easily compute assume Reed Frost model denote pr probability chain infection occur household size index case infect turn infect tertiary case maximum generation infection assumption pr ðþbinomialthe second approximation time take infect infect distribute accord standard generation time distribution time th generation infection infect result draw th auto convolution distribution denote define recursive convolution equationwhich satisfie consider individual th generation infection household consider household time index case infect individual infect early time distribute accord distribution infectiousness outside household give average possible value average infectiousness individual th generation isthus having average possible time infection chain transmission event household infectious household stratify size number case generation notation define early define state vector variable define infectiousness susceptibility infected household household size number infected individual th generation infection household infectiousness household state household mediate random mixing individual household sum infectiousness individual give equation give infection process involve random mixing individual outside household distribution size household infect call size biased household distribution distribution size obtain sample individual random population record size household oppose commonly record household size distribution obtain sample household random denote household size distribution thenis size biased household size distribution give household size gets infect probability chain infection give Reed Frost probability pr distribution random variable infection pr mean infectiousness household let average total number case infected household household reproduction number take intuitive know form derive express term parameter follow household reproduction number product expected number infection household multiply number people individual infect household mean household generation time distribution time household infect isthe mean generation time household express term individual generation time asthe generation time distribution previously define estimator reproduction number household incidence datum exponential growth rate exponential growth rate exponentially grow epidemic measure individual household incidence exponentially grow decline epidemic obtain estimatornow consider integrationwhere equality use definition auto convolution second ordering integral involve change variable fourth factorisation fifth arise induction sixth use definition individual reproduction number obtain ignore household structure equation household reproduction number express term individual reproduction number asExamination equation immediately reveal estimate number people person infect household isi derive simple analytic relation individual household reproduction number approximation ignore effect local saturation generation time tend produce overestimate reproduction number alternative approximate household reproduction number provide underestimate find secondary household case assume arise second generation equation reason consider household structure analyse pandemic influenza situation influenza transmission know concentrate household parameter estimate ignore heterogeneity likely frail second public health policy future pandemic likely organise household net effect social distance measure school workplace closure cancellation social gathering effectively reduce transmission household inadvertently increase transmission furthermore antiviral treatment prophylaxis quarantine measure likely target household individual restrict family suspect case stay support possibly undesirable number study identify parameter need estimate household reproduction number influenza important bear mind parameter different future pandemic robust methodology useful respond new outbreak numerical estimate obtain past outbreak straightforward use demographic datum exponential growth rate early pandemic combine interpandemic datum transmissibility influenza household obtain estimate historical pandemic show household transmission parameter inter pandemic influenza adequately describe pandemic situation focus instead provide illustrative example current demographic datum household size distribution UK recent datum transmissibility influenza modern household household size datum truncate size assume household size great size exactly datum household single person household size mean household average size household household sample random household mean individual size average size household individual belong individual sample french influenza study obtain maximum likelihood estimate household transmission parameter consistent good fit Tecumseh datum study follow seronegative household week winter outbreak seasonal influenza corresponding escape probability probability infect household member household size scale infection place influenza approximately infectious mump lot infectious varicella zoster measle apply Reed Frost model datum distribution household obtain estimate average number infection generation infection index case directly infect average people household estimate total expect number case infected household compare mean size calculation perform Microsoft Excel equation consensus generation time influenza estimate range day day use Gamma distribution mean day standard deviation day report datum compare predicted simulate infectiousness household Fig show average household size compare final analytical approximation give equation alternate approximation consider secondary infection arise second generation infection simulation approximation clearly good agreement individual base stochastic simulation program Berkeley Madonna describe Appendix case exponentially grow epidemic estimate individual household reproduction number respectively show Figure estimate number people person infect outside household estimate approximation show estimate obtain simulate generation time distribution expect low infectiousness scenario simulate value close overestimate approximation range approximation bracket true value narrow indicate method predictive Spanish Flu pandemic median growth rate large city day comparable estimate UK value serve upper estimate spread pandemic virus base growth rate estimate individual reproduction number estimate household reproduction number household reproduction number course household big actual value likely high estimate imply proportion household transmission need block prevent epidemic spread figure provide rough guide likely value new influenza pandemic rate exponential growth reliably determine consider index case household expect infect people household household validate assume proportion transmission household early simulation study sum great reproduction number average different generation infection household value estimate take account local saturation effect determine numerically fig show value numerically estimate value close curve estimate application equation ignore local saturation effect final check method epidemic community household simulate individual base stochastic model Appendix choose infer epidemic growth rate day parameter describe exponential rate growth estimate directly simulated incidence timeserie figure close predict value provide support validity method especially restriction place infection household small simulate community note influenza relatively uninfectious compare common virus contrast application method focus measle infectious pathogen study measle peaked generation time distribution generation infection distinct contrast influenza estimate great use demographic datum household size choose national census household size great analysis little artificial apply measle large proportion population immunity past infection vaccination live MMR vaccine principal motivation test illustrate method case good datum transmission dynamic household available stratification household recent outbreak measle cause decrease uptake MMR vaccine reveal household heterogeneity account estimate change reproduction number measle household size datum truncate size assume household size great size exactly datum household single person household size mean household average size household household sample random household mean individual size average size household individual belong individual sample Simpson report susceptible infectious escape probability mump varicella measle result report independent household size regard unreliable base apply Reed Frost model measle show infectiousness typical infected household denote curve obtain simulate epidemic transmission household start infected case analytical approximation describe text show Approx main approximation describe Approx obtain assume infection occur second generation infection household parameter describe main text curve arbitrarily scale individual infect average person outside household estimate distribution household obtain estimate average number infection generation infection index case directly infect average people household estimate total expect number case infected household compare mean size Hope Simpson report interval link case household different case definition interval regard reliable case definition maximum rash datum describe Gamma distribution show maximum likelihood estimate generation time day standard deviation day base datum repeat simulation previous section influenza parameter measle Figs figure show expect average infectiousness household approximate approximation infectious case influenza case multiple peak infectiousness correspond generation infection household clearly distinguish case second generation infection term predicting household reproduction number method find strongly predictive evidence small gap upper low estimate reliable compare numerical estimate influenza simulation close upper approximation close low approximation expect infectious situation measle transmission simulation transmission community household find figure validate approach difference shape epidemic curve influenza reflect different shape generation time distribution exponential growth rate new method present estimate individual household reproduction number epidemic new method present estimate individual reproduction number relate closely early work provide alternative possibly simple solution problem incomplete observation unfold epidemic provide alternative satisfying solution incidence prevalence ratio method problem long generation time distribution infection HIV epidemiological circumstance change substantially course single infection case reproduction number represent average convey secular change behaviour transmission study challenge central role individual reproduction number epidemiological measure empirical measure reproduction number propose use incident observe case base complication define typical eigen case structured model discuss clearly neatly sidestep study highlight complexity hide effectively define estimate generation time distribution structured population case study generation time individual short household transmission household transmission particularly infectious pathogen result systematic bias associate estimate reproduction number ignore effect substantial case highly infectious measle virus method present estimation household reproduction number affect problem way analytical approximation derive bracket estimate low upper bind numerical simulation show range bracket narrow approximation show robust worth note assumption population mix randomly household result valid scenario emerge pathogen overall prevalence low usefulness method likely find predict understand impact household target infection control measure emerge epidemic actually cover wide class intervention household central living administrative unit population decision isolation quarantine vaccination prophylaxis entire household similarly school workplace closure restriction leisure activity think try reduce household transmission analytical approach invaluable calibrate provide independent check detailed individual base micro simulation control intervention require subtle analysis example show vaccinate household effective strategy give vaccine coverage rate alternative strategy preferentially vaccinate large household consider avenue research include study statistical property estimator different situation assumption individual mix nearly homogeneously household appropriate approximation describe transmission neighbourhood city ultimately consider develop estimator complex demographic situation hierarchy organisation household village region country etc complex overlap household workplace regular social space interest study intervention measure particularly respond presence symptomatic case measure pre symptomatic transmission present clearly generalise household analytical result efficacy isolation quarantine evidently obtainable estimator household reproduction number show robust term address issue model misspecification example inaccurate determination generation time distribution individual heterogeneity infectiousness susceptibility household scenario explore test method different infection address issue model misspecification case desirable quantify household transmission degree natural vaccineinduced immunity present population problem address consider complex situation possible obtain analytic form infectiousness household numerical form usually obtain quickly offer benefit individual base micro simulation easily explore wide range parameter finally likely practical benefit estimate household transmission parameter emerge epidemic need clearly establish communicate effective way enhance datum collection protocol allow rapid estimation need identify Appendix Description simulation find mb pdf','2007-08-22 00:00:00','http://dx.doi.org/10.1371/journal.pone.000075',3625,912),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json','Markers of exacerbation severity in chronic obstructive pulmonary disease','BACKGROUND: Patients with chronic obstructive pulmonary disease (COPD) can experience \'exacerbations\' of their conditions. An exacerbation is an event defined in terms of subjective descriptors or symptoms, namely dyspnoea, cough and sputum that worsen sufficiently to warrant a change in medical management. There is a need for reliable markers that reflect the pathological mechanisms that underlie exacerbation severity and that can be used as a surrogate to assess treatment effects in clinical studies. Little is known as to how existing study variables and suggested markers change in both the stable and exacerbation phases of COPD. In an attempt to find the best surrogates for exacerbations, we have reviewed the literature to identify which of these markers change in a consistent manner with the severity of the exacerbation event. METHODS: We have searched standard databases between 1966 to July 2004 using major keywords and terms. Studies that provided demographics, spirometry, potential markers, and clear eligibility criteria were included in this study. Central tendencies and dispersions for all the variables and markers reported and collected by us were first tabulated according to sample size and ATS/ERS 2004 Exacerbation Severity Levels I to III criteria. Due to the possible similarity of patients in Levels II and III, the data was also redefined into categories of exacerbations, namely out-patient (Level I) and in-patient (Levels II & III combined). For both approaches, we performed a fixed effect meta-analysis on each of the reported variables. RESULTS: We included a total of 268 studies reported between 1979 to July 2004. These studies investigated 142,407 patients with COPD. Arterial carbon dioxide tension and breathing rate were statistically different between all levels of exacerbation severity and between in out- and in-patient settings. Most other measures showed weak relationships with either level or setting, or they had insufficient data to permit meta-analysis. CONCLUSION: Arterial carbon dioxide and breathing rate varied in a consistent manner with exacerbation severity and patient setting. Many other measures showed weak correlations that should be further explored in future longitudinal studies or assessed using suggested mathematical modelling techniques.',' chronic obstructive pulmonary disease COPD respiratory disease characterize airflow limitation inflammation low airway disease worsen patient experience exacerbation principal symptom dyspnoea cough sputum exacerbation frequently result visit general practitioner office local hospital treatment exacerbation occur COPD patient median time year half unreported heterogeneity copd exacerbation difficult define classify manage range symptom varied treatment requirement seasonal occurrence ambiguous aetiology address problem attempt develop consensus definition copd exacerbation recently American Thoracic Society ATS European Respiratory Society ERS adopt follow definition event natural course disease characterise change patient baseline dyspnea cough sputum day day variability sufficient warrant change management severity exacerbation difficult classify despite scheme propose deal issue ATS ERS jointly suggest classification base severity type medical management Exacerbation Level home treatment Level II hospitalization Level III specialised care aim scheme improve exist management exacerbation serve aid assessment treatment efficacy different operational definition copd exacerbation propose past help determine relative importance particular relationship copd progression definition rely primarily symptom absence standard classification degree symptom severity delay development new therapy condition current therapy exacerbation evaluate base ability reduce symptom improve patient forced expiratory volume second FEV strongly correlate copd mortality FEV discriminate stable exacerbative state copd particularly later stage disease development biological marker biomarker sensitive specific severity copd exacerbation provide investigator new insight direction research time clinical variable inflammatory mediator show associate copd exacerbation related morbidity mortality include age FEV force vital capacity peak expire flow body mass index albumin sodium pH eosinophil interleukin fibrinogen reactive protein significant clinical event number exacerbation year number hospital admission year time relapse day hospital regard useful measure clinical study design assess drug efficacy cost effectiveness standardize exist hospital support program COPD know measure change increase severity copd exacerbation survey medical literature identify commonly accept variable suggest marker copd exacerbation change accord ATS ERS level exacerbation severity long term aim work assess sensitivity specificity potential marker use future copd study determine marker study fully integrate development new drug search standard database medical search heading relate term obtain major consensus document relate copd exacerbation major keyword exacerbation unstable acute bronchitis variant term COPD phase search retrieve total citation citation read title abstract citation exclude citation concerned exacerbation coronary artery disease myocardial infarction cystic fibrosis asthma pulmonary embolus community pneumonia citation case study letter review meta analysis animal study exclude initial screening identify citation paper possible interest retrieve original article electronic hard copy form critically read article result step arrive total study final review analysis select study base availability demographic spirometry clear study eligibility criterion potential marker assess exacerbation objective literature review datum analysis determine baseline measure com monly copd exacerbation study change extent exacerbation disease severity determine copd exacerbation model event time event future investigation initially consider exacerbation definition classification scheme particular suggest Rodriguez Roisin describe Pauwels colleague determine ATS ERS operational classification exacerbation severity sensible feasible system systematically assess patient baseline characteristic biomarker information majority publish study classification scheme related clinical history physical finding diagnostic procedure manage exacerbation perform datum abstraction study retrieve report demographic spirometry smoking status clinical cytological biochemical variable suggest marker severity exacerbation baseline condition immediately prior exacerbation event time intervention interest investigate table variable measure stable condition abstract information study note type definition define exacerbation symptom event base research question ask experimental design sponsorship presence absence datum individual study patient Data organize accord sample size smoke status available cytological biochemical datum classify accord collection method include sputum induction bronchial biopsy bronchoalveolar lavage BAL exhale breath sampling blood sampling aware possibility study group severity Levels II iii ATS ERS criterion include review experience similar quality care medical management report adequately original publication attempt correct problem combine exacerbation datum Levels II iii patient category compare Level regard patient category collect calculated study mean median standard error standard deviation confidence interval inter quartile range statistical algorithm Microsoft Excel conduct fix effect meta analysis obtain mean point estimate confidence interval standard deviation exacerbation level Exacerbation Severity Levels II II III III compare twotaile test alpha level adjust multiple testing accord Bonferroni correction procedure event specific exacerbation severity level large number study median datum available datum consider normally distribute median treat mean study publish datum individual patient limit address non normality datum log perform fixed effect meta analysis obtain mean point estimate confidence interval standard deviation patient patient category measure compare category tail test value search strategy yield suitable study meet selection criterion study publish July reference study find LACDR Division Pharmacology website total number study subject include review group fall Exacerbation Severity Level Level II Level III analyse datum accord patient setting patient patient Meta analysis typical study demographic show significant overlap confidence interval study data distribution exacerbation severity level age study patient Level II mean age year confidence interval CI year compare year ci year patient Level III table demographic analyze accord patient setting determine body mass index statistically different patient setting mean point estimate kg ci kg patient setting mean point estimate kg ci kg table spirometry measure force Expired Volume Second FEV Forced Vital Capacity FVC percent predict decrease Exacerbation Levels II remain unchanged Levels II III figure respectively Levels II iii combine create patient category variable statistically signifi respiratory Research nt decrease patient versus patient figure respectively observe trend FEV figure spirometry measure study available Level III meta analysis find smoke pack year increase exacerbation severity Levels II statistically different figure compare pack year patient setting statistically high patient patient figure term hemodynamic measure heart rate show statistically significant difference high Level II Level difference study variable measure time exacerbation variable measure stable condition COPD patient measurement take week month prior exacerbation Levels II iii Figure heart rate high patient patient figure clinical measure dyspnoea breathing rate figure borg dyspnoea score tend increase Levels II decrease Levels II III breathing rate demonstrate clear statistical difference level Levels II iii Borg Dyspnoea Score statistically different statistical comparison level Level possible lack datum patient setting compare breathing rate show clear statistical difference statistically low patient patient figure Levels II III statistically different respect pH bicarbonate pH decrease Level II III bicarbonate increase insufficient Level data variable allow statistical comparison Levels insufficient datum available compare patient patient term blood gas measure study arterial carbon dioxide tension PaCO show statistically significant increase increase exacerbation severity figure versus patient figure case oxygen saturation gradually decrease increase exacerbation severity statistically significant difference Levels II Levels iii figure decrease go patient inpatient setting figure minute walking distance challenge test decrease trend increase exacerbation severity change reach statistical significance case patient compare variable relate spirometry respiratory status exacerbation hospital event category change significantly exacerbation severity patient additional file datum bacteriology virology category permit meta analysis study sample half contain datum biochemical variable conduct review COPD exacerbation literature determine commonly accept baseline variable suggest marker change consistent manner severity copd exacerbation index COPD severity recently publish ATS ERS operational classification exacerbation severity medical management publish literature rarely provide sufficient detail characterise severity patient exacerbation addition analyze datum accord patient setting account possible overlap medical management Levels II iii report original publication long term aim work improve quality applicability exacerbation management identification sensitive specific marker assessment treatment effect review identify potential marker exacerbation severity assess spirometry measure FEV FVC predict FEV observe statistically significant difference exacerbation severity patient figure draw paucity information Level III study confirm clinical situation exacerbation worsen specialised care require spirometry measurement likely baseline condition exacerbation datum rare publish study number smoking relate pack year increase exacerbation severity show clear difference patient setting figure finding consistent idea copd patient smoke long high likelihood COPD exacerbation severe accord mean point estimate obtain patient mmhg study COPD patient pack year smoking experience increase severity copd exacerbation conclusion finding limit datum study Level heart rate varied little Exacerbation Levels II III important note substantially elevate patient figure clear difference patient possibly associate anxiety dyspnea experience exacerbation occur increase heart rate course increase oxygen requirement heart increase heart rate result underlie cardiovascular disease prominent severe COPD patient relationship pH bicarbonate exacerbation severity consistent sign respiratory acidosis evident COPD patient exacerbation shortage datum Level proper statistical conclusion variable difficult relation breathing rate significantly increase Levels II decrease Levels II III figure observation reflect component exacerbation episode anxiety dyspnea physiological need breathe maintain adequate blood gas level reduction Level III possibly reflect result specialized care patient give ventilatory support return breathing rate normal Borg Dyspnea Score show trend breathing rate insufficient datum Level allow comparison inpatient datum compare variable breathing rate demonstrate clear statistical difference Figure Borg Dyspnoea score hand study patient category perform statistical test overall observe trend consistent fact management dyspnoea main factor generate high hospital cost associate copd exacerbation keep direct measure dyspnoea arterial carbon dioxide tension show clear relationship exacerbation severity patient management setting figure consistent conclusion report medical literature arterial oxygen tension contrast change exacerbation severity patient setting possibly lack correlation reflect immediate administration supplemental oxygen give hypoxaemic patient hospital setting decrease trend oxygen saturation increase exacerbation severity clear difference patient setting figure consistent present thinking blood gas change commonly accept measure suggest biomarker poorly reflect exacerbation severity fact sufficient datum undertake meta analysis additional file finding recall Department Health Human Services report exacerbation treatment outcome randomised control trial aim study create new guideline improve management copd exacerbation study conclude current literature limit term number study detail available reliability accuracy clinical assessment discriminate copd exacerbation cause worsen respiratory status observation agree previous observation assessment unstable COPD literature previously discuss study review predominately hospitalize patient Level II copd occur patient setting Level implication study population poorly represent basic categorisation accord ATS ERS operational scheme classify severity copd exacerbation patient category knowledge undertake type literature review face lack consistency definition exacerbation study try overcome difficulty select rank clinical study improve comparability subject study aware clinical study analyse differ respect comorbiditie identifiable cause exacerbation report patient elderly likely suffer co exist disease asthma cardiovascular disease co morbidity make interpretation finding difficult respect true cause exacerbation aetiology determine susceptible patient Level identify new treatment develop help prevent onset relate hospital cost finally compatibility study COPD exacerbation analyse limit substantial variation time location study exacerbation likely summer study fail report time year time period study implementation seasonal effect combine low incidence exacerbation patient represent inherent bias addition different institution probably different standard respect diagnosis management copd exacerbation study perform variation explain observed inconsistency finding attempt overcome possible bias Exacerbation Levels II iii subsequent analysis datum basis patient patient setting observe additional online file scarcity information particularly biomarker different exacerbation level unclear variable change exacerbation severity causally relate longitudinal study restrictive eligibility criterion need address question difficulty tackle problem enormous time expense involve implement study addition current method datum analysis clinical study limitation impose assessment reduction frequency total suppression exacerbation episode rare event non overcome drawback obtain accurate evaluation treatment effect copd exacerbation alternative analytical method base example predictive mathematical model hide Markov chain Bayesian forecasting try model characterise predict rare event undertake scale long term longitudinal study approach predict rare event previously study migraine epilepsy cardiovascular disease size treatment effect measure term reduction frequency repetition event give probability give time period example mathematical model development include use Markov model predict COPD exacerbation rate clinical trial inhaled anticholinergic bronchodilator tiotropium example model develop basis prior knowledge exacerbation rate estimate meta analysis randomised control trial datum give probability copd exacerbation different stage COPD study proportional hazard model identify risk factor COPD patient hospitalise exacerbation current ATS ERS guideline exacerbation consider implication probabilistic model means assess severity copd exacerbation effect treatment modelling approach offer new insight variable relate copd exacerbation investigate research planning perspective study finding generate hypothesis relate consideration evaluate future clinical trial hypothesis combination variable observe change study FEV FVC FEV FVC arterial carbon dioxide breathing rate heart rate pack year oxygen saturation represent new definition severe exacerbation event definition literature include recent ATS ERS definition indicate assessment variable sign exacerbation simply regard exacerbation worsening normal day day symptom adjustment medical management definition encompass clear set objective measure useful medical practitioner predominantly rely clinical judgement past experience diagnose exacerbation severity assess treatment effect important consideration future clinical trial assessment treatment effect base prediction exacerbation frequency intensity word collection datum rate onset resolution exacerbation longitudinal study determine probability second fourth etc exacerbation event individual patient alteration probability experimental treatment sensitive reliable approach assess treatment effect clinical trial record daily change symptom medical management lastly finding obtain COPD patient experience exacerbation study assessment period study patient experience exacerbation indicate fraction COPD patient regard susceptible exacerbation fraction exacerbation free interesting determine variable identify study change patient group accord FEV publish study stratify COPD patient basis exacerbation frequency generally categorise patient have infrequent frequent exacerbation great mean exacerbation year respectively study unable distinction copd patient publish study provide individual patient datum exacerbation frequency currently investigate commercial database clinical trial enable look patient infrequent frequent exacerbation result work help well select patient identify potential marker future longitudinal study current management treatment copd exacerbation primarily dependent evaluation symptom sign relate exacerbation event find arterial carbon dioxide tension breathing rate consistently varied severity copd exacerbation versus patient commonly accept measure suggest biomarker exacerbation fail consistent trend lack sufficient datum permit meta analysis recommend design longitudinal study look frequency exacerbation use advanced modelling technique improve selection potential marker categorization severity copd exacerbation assessment treatment effect future study COPD Obstructive Pulmonary Disease','2006-05-10 00:00:00','http://dx.doi.org/10.1186/1465-9921-7-74',2215,716),('bada3cc0577a226694617d77f95ac443c6431969.json','DNA Vaccines against Protozoan Parasites: Advances and Challenges','Over the past 15 years, DNA vaccines have gone from a scientific curiosity to one of the most dynamic research field and may offer new alternatives for the control of parasitic diseases such as leishmaniasis and Chagas disease. We review here some of the advances and challenges for the development of DNA vaccines against these diseases. Many studies have validated the concept of using DNA vaccines for both protection and therapy against these protozoan parasites in a variety of mouse models. The challenge now is to translate what has been achieved in these models into veterinary or human vaccines of comparable efficacy. Also, genome-mining and new antigen discovery strategies may provide new tools for a more rational search of novel vaccine candidates.',' spite success vaccine public health numerous pathogen particular protozoan parasite Plasmodium falciparum Trypanosoma sp Leishmania sp effective vaccine discovery direct injection plasmid dna encode foreign protein lead endogenous protein biosynthesis specific immune response open new perspective vaccine development year later dna vaccine go scientific curiosity dynamic field research offer new alternative control infectious disease dna vaccine licence recent year protect horse west nile virus salmon infectious hematopoietic necrosis virus confirm usefulness biotechnology review advance challenge development dna vaccine study protozoan parasite Leishmania sp Trypanosoma cruzi belong trypanosomatidae family rank major protozoan parasite affect human Leishmaniasis complex disease cause specie parasite Leishmania genus transmit human hematophagous sandfly estimate million case major public health impact region particular India Sudan Brazil manifestation range self heal cutaneous lesion fatal visceral form variety attribute respective parasite specie present specific relationship host diverse mechanism pathogenesis represent additional difficulty development treatment vaccine hand cruzi agent Chagas disease present southern Argentina southern USA estimate million person infect Americas close million people risk infection short benign acute phase week long year asymtomatic phase infected patient develop chronic chagasic cardiomyopathy eventualy die heart failure current chemotherapy rely nitrofuran Nifurtimox nitroimidazole Benznidazole usefulness drug limit reduce efficacy early stage infection effect emergence drugresistant strain parasite new treatment slow develop vaccine induce complete immune response encode antigen exact mechanism involve proccess poorly understand particularly type effector memory cell activate aspect review detail apart immunogenicity efficacy discuss feature dna vaccine advantageous tropical disease extremely safe contain pathogenic organism revert virulence major concern genomic integration plasmid DNA extensively study safety study find unlikely additional safety issue anti dna antibody autoimmunity address grow number preclinical clinical study confirm high safety vaccine respect manufacturing storage distribution present major benefit production process dna vaccine case type biological vaccine specific protocol develop make production easy cost likely type vaccine mainstream future technological improvement implement plasmid DNA stable molecule specially compare recombinant live attenuate vaccine greatly facilitate storage distribution dna vaccine tropical setting limited health infrastructure huge cost associate cold chain offset administration easy simple IM ID injection sufficient multiple plasmid combine elaboration multivalent vaccine overall dna vaccine represent ideally affordable alternative disease control explain grow interest development control tropical parasitic disease malaria leishmaniasis Chagas disease mention leishmaniasis cause specie parasite diverse relationship host mechanism pathogenesis early study cross protection Leishmania specie clearly show complex problem infection specie protect subsequent infection specie depend specie order infection vaccine study focus homologous protection single vaccine able protect pathogenic specie ideal correlate protection extensively study case major contribute considerably development paradigm general agreement type immune response characterize high IFNγ low production lead control major infection immune response antibody exacerbatory role contribute cell response IFNγ produce cell contribute protective immunity induction production macrophage central parasite elimiation assume long time paradigm apply Leishmania specie clear recent year specie distinct relationship host different mechanism pathogenesis possibly different correlate protection nonetheless IFNγ production general requirement necessarily sufficient protection Leishmania specie early dna vaccine experiment Leishmania major antigen extensively recombinant peptide vaccine immunization plasmid encode able induce type cytokine profile significant reduction lesion size challenge immunize mouse major subsequent study investigate DNA vaccine encode large variety Leishmania protein table show different dna vaccine able induce immune response confer variable degree protection assess reduction skin lesion size parasite burden mouse model give large variety experimental model design difficult compare effectiveness different vaccine induce protective immune response nonetheless clear study compare different dna vaccine nature antigen encode vaccine key parameter efficacy study provide interesting comparison antigen administer recombinant protein dna vaccine show overall effective recombinant protein counterpart dna vaccine able induce strong bias immune response long last immunity well protection disease progression study artificial infectious challenge base injection nonnatural route high parasite dose experimental system criticize author superior efficacy dna vaccine observe low dose intradermal challenge ear propose closely mimick natural infection study dna protein vaccination able induce similar level short term week postvaccination protection infection major dna vaccine able induce long term week postvaccination protection result confirm strong potential dna vaccine Leishmania indicate case partial protection achieve prime boost immunization protocol test antigen increase vaccine potency table base prime immune response dna vaccine boost correspond recombinant vaccine base recombinant virus protein table study immunization protocol result increase immunogenicity vaccine well protection level DNA remain good formulation optimum efficacy nonetheless major drawback vaccine formulation remain complexity limit practical use alternative way broaden vaccine immunogenicity increase efficacy use combination plasmid encode antigen example cysteine proteinase CP dna vaccine protective individually immunization combination plasmid induce long term protective immunity alternatively gene fusion successively achieve expression antigenic fusion protein single plasmid construct overall expression antigen result increase efficacy depend antigen combination author argue successful Leishmania vaccine likely base multiple antigen immunization large number plasmid basis expression library immunization powerful laborintensive strategy vaccine discovery Leishmania immunization mouse major genomic expression library fraction able induce significant protection author pursue library fractionation study identification protective library subset donovani amastigote cdna library successive fractionation small protective library lead identification novel protective antigen interestingly antigen identify predict good vaccine candidate surface secrete protein stage specific intracellular conserve histone ribosomal protein Vaccine discovery logical step follow recent completion major genome sequence approach random screening gene upregulate amastigote test DNA vaccine allow identification novel protective exacerbating antigen function cellular localization poor predictor protective efficacy antigen predict localize surface share similarity ribosomal protein cytoskeleton metabolic enzyme increasingly clear little rationale limit Leishmania vaccine discovery search surface secrete antigen new criterion need consider rational identification vaccine candidate strategy base random screening apply large genome Leishmania annotate gene additional advantage dna vaccine potential therapeutic vaccine aim reinforce redirect immune response infected host control disease progression major advantage strategy addition efficacy rely short treatment regimen attractive alternative chemotherapy particularly case Leishmania chemotherapeutic option administration little dose dna vaccine encode control ongoing infection major mouse therapeutic effect shift immune response immune response similarly dna vaccine encode donovani nucleoside hydrolase therapeutic activity murine visceral leishmaniasis cause chagasi simplicity treatment make advantageous compare chemotherapy addition fact dna vaccine effective prophylaxis therapy Leishmania infection promising provide versatile tool control parasite mention add challenge Leishmania vaccine development large number specie variability specie study polymorphism lead antigen quickly reveal polymorphic polymorphism important implication vaccine development limit efficacy variant strain parasite novel escape mutant restrict vaccine protection single species Antigen polymorphism multiple strain specie major issue vaccine development study case Leishmania dna vaccine test mul tiple specie lack antigen initially identify major find conserve Leishmania specie protect mouse major amazonensis mexicana donovani chagasi hand amazonensis nuclease protein protect amazonensis major cross protection require different formulation adjuvant resp study antigen specie induce protection specie extent crossprotection specie investigate recently single formulation donovani DNA vaccine find induce good protection chagasi mexicana suggest dna vaccine able provide broad protection Leishmania specie importantly dna vaccine test braziliensi spite specie responsible case cutaneous leishmaniasis South America dna vaccine base Leishmania antigen alternative approach antigen derive sand fly saliva show sand fly saliva exacerbate Leishmania infection preexposure mouse saliva component sufficient induce protection infection number salivary antigen test vaccine Leishmania Maxadilan potent vasodilator sand fly saliva find responsible exacerbatory effect saliva Leishmania infection immunization antigen recombinant vaccine protect mouse major infection salivary component Phlebotomus papatasi test dna vaccine find protect mouse major vaccine induce humoral dth response protection account cell deficient mouse remain protect characterization sand fly salivary protein lead identification new vaccine candidate Leishmania antigen salivary protein polymorphism remain important issue limit usefulness antigen vaccine candidate success dna vaccine study mouse vaccine candidate test additional animal model possibly relevant development veterinary human vaccine table antigen test dna vaccine hamster highly susceptible animal model test protein dna dna protein immunization protection level mexicana varied greatly depend vaccine formulation route immunization sex animal contrary mouse study protein vaccination protective DNA vaccination mouse study heterologous prime boost vaccination dna protein well dna dna vaccine encode find immunogenic hamster decrease parasitemia infection infantum assessment disease perform immunization DNA induce mixed response able protect hamster visceral leishmaniasis cause donovani dog protein vaccine test purified protein vaccine licence veterinary use dna vaccine study perform heterologous prime boost strategy cpa CPb DNA protein report immunogenic protective study limited power give reduce number animal study dog immunize mixture DNA vaccine encode different antigen previously test mouse model immunization induce good immune response high production IFNγ evaluation protection limit acute vitro assay study require assess potential vaccine dog spite limitation study clearly show dna vaccine induce potent immune response nonmurine animal model likely good level protection achieve provide correct antigen vaccine formulation vaccine development Chagas disease dramatically limited extensive debate mechanism involve pathology study suggest tissue damage associate presence replication intracellular amastigote propose autoimmunity induce parasite antigen mimic host protein responsible unclear immune response need inhibit reduce autoimmunity stimulate eliminate parasite accept presence parasite cardiac tissue necessary initiate maintain inflammatory response therapeutic treatment vaccine aim eliminate cruzi limit prevent progression chronic chagasic cardiomyopathy grow consensus protection cruzi rely immune response activation cytotoxic cell dna vaccine test cruzi encode antigen characterize trans sialidase family protein member family make large protein family parasite abundant surface protein study different member family TS table immunization TS find induce significant antibody titer able inhibit trans sialidase enzyme activity strong dth lymphoproliferative response immune response protective determine increase survival decrease parasitemia immunization DNA find induce specific CTL response lead low parasitemia increase survival BALB mouse Leishmania vaccine study author address question compare protein dna vaccine encode antigen Sn mouse immunization recombinant TS induce high antibody titer TS DNA comparable decrease parasitemia DNA vaccine unable increase survival author attribute strain mouse DNA vaccine protective BALB mouse hand immunization recombinant CRP CRP DNA induce comparable immune response DNA vaccine protective infection number study show DNA vaccine encode antigen induce significant protection cruzi infection evidence decreased parasitemia improved survival vaccinated mouse table addition study present evidence reduction cardiac tissue damage inflammation histopathologic level furthermore cell analysis confirm protection rely cell recent study show cell rapidly activate follow infection mouse immunize dna vaccine dna vaccine base define cell epitope TS antigen test find Journal Biomedicine Biotechnology IM intramuscular CTL cytolytic activity protection little protection fair protection good cell epitope necessary sufficient induce protective immune response take datum clearly demonstrate vaccination result increase pathology initially fear allow partial control disease progression confirm central role parasite persistence Chagas disease pathogenesis open way assessment dna vaccine cruzi note antigen test belong trans sialidase family protein little diversity term antigen test vaccine cruzi table protection induce single antigen dna vaccine remain partial number study evaluate strategy increase vaccine efficacy include use cytokine chemokine encode plasmid potentiate immune response induce vaccine studied molecule GM CSF generally able potentiate protection table alternatively mixture plasmid encode distinct antigen immunization mention Leishmania vaccine example immunization mouse plasmid encode TS protein result specific immune response antigen increase protection infection hand immunization mixture dna vaccine encode protein mucin family result poorly protective mixture protein TS family protective single antigen vaccine encode ts like antigen similarly mixture dna vaccine encode similar protective activity lack efficacy multivalent vaccine attribute presence share immunodominant epitope significant sequence similarity result broad immune reponse heterologous prime boost approach evaluate immunization combination dna recombinant ts find enhance immune response protection significantly different obtain dna take study suggest additional strategy need investigate potentiate dna vaccine efficacy cruzi therapeutic administration dna vaccine control ongoing infection cruzi represent additional Eric Dumonteil alternative Chagas disease control concept demonstrate mouse acutely chronically infect case administration dose dna vaccine encode antigen sufficient limit disease progression treat mouse present increase survival reduce cardiac tissue damage assess histopathologic analysis comparative study different dna vaccine identify antigen therapeutic vaccine candidate dna vaccine encode antigen TS family previously find protective signifiacnt therapeutic effect find therapeutic vaccination rapidly induce spleen cell proliferation include IFNγ produce cell effect cardiac tissue inflammation parasite burden long detectable importantly study therapeutic vaccination cruzi infected mouse result increase inflammatory reaction heart confirm safe stimulate immune response cruzi infected mouse attack parasite lead reduction pathology study open attractive perspective control cruzi infection study efficacy DNA vaccine encode antigen immune mechanism underlie therapeutic effect provide clue optimization strategy vaccine nature antigen remain key factor vaccine efficacy little variety term antigen evaluate dna vaccine candidate cruzi number study aim identify novel antigen strategy classical approach screening cdna library antibody screen amastigote library allow identification novel antigen previously characterize one FcaBP dna vaccine encode antigen induce variable level protection good LYT alternatively expression library immunization describe Leishmania test cruzi find immunogenic attempt fractionate library identify protective antigen likely reason strategy labor intensive large genome library usefulness limit pathogen small genome availability cruzi genome sequence open new possibility antigen discovery study resource combination bioinformatic analysis identify GPI anchor secrete protein identify clone immunogenic dna vaccine study confirm usefulness new vaccine protect cruzi infection nonetheless discuss Leishmania rationale limit antigen search surface protein totally relevant additional strategy include unbiaised genome wide survey antigen discovery detailed review considerable advance dna vaccine Leishmania cruzi recent year take study clearly validate concept dna vaccine protection therapy protozoan parasite variety mouse model sterile immunity irrealistic goal Leishmania cruzi reduction disease severity development pathology clearly reach dna vaccine nonetheless relevance mouse model development veterinary human vaccine parasite challenge author dna vaccine study nonmurine model leishmaniasis suggest extrapolation feasible certainly completely additional advanced preclinical study dna vaccine candidate nonmurine animal model rat hamster dog monkey warrant year explore immunology efficacy dna vaccine parasite observe study pathogen lead challenge achieve specie immunogenicity comparable level protective efficacy obtain murine model advance adjuvant dna vaccine formulation delivery system likely contribute result major issue antigen discovery number dna vacine test far Leishmania cruzi show promise unsure good possible antigen particularly parasite relatively large genome limited variety antigen test availability genome sequence parasite doubt key resource genome wide screening new protective antigen key lesson initial study review similar antigen discovery study cellular localization protein function poor predictor antigenicity protective efficacy protein alternative criterion potent vaccine candidate miss important development genome mining bioinformatic tool provide new tool rational search vaccine candidate conclude dna vaccine represent promising approach control Leishmania sp cruzi vaccine major impact develop endemic country question DNA vaccine control parasite study clearly show case translate achieve mouse model veterinary human vaccine comparable efficacy work fund Grant Consejo Nacional de Ciencia Tecnología CONACYT','2007-06-24 00:00:00','http://dx.doi.org/10.1155/2007/90520',2359,724),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json','MIMOX: a web tool for phage display based epitope mapping','BACKGROUND: Phage display is widely used in basic research such as the exploration of protein-protein interaction sites and networks, and applied research such as the development of new drugs, vaccines, and diagnostics. It has also become a promising method for epitope mapping. Research on new algorithms that assist and automate phage display based epitope mapping has attracted many groups. Most of the existing tools have not been implemented as an online service until now however, making it less convenient for the community to access, utilize, and evaluate them. RESULTS: We present MIMOX, a free web tool that helps to map the native epitope of an antibody based on one or more user supplied mimotopes and the antigen structure. MIMOX was coded in Perl using modules from the Bioperl project. It has two sections. In the first section, MIMOX provides a simple interface for ClustalW to align a set of mimotopes. It also provides a simple statistical method to derive the consensus sequence and embeds JalView as a Java applet to view and manage the alignment. In the second section, MIMOX can map a single mimotope or a consensus sequence of a set of mimotopes, on to the corresponding antigen structure and search for all of the clusters of residues that could represent the native epitope. NACCESS is used to evaluate the surface accessibility of the candidate clusters; and Jmol is embedded to view them interactively in their 3D context. Initial case studies show that MIMOX can reproduce mappings from existing tools such as FINDMAP and 3DEX, as well as providing novel, rational results. CONCLUSION: A web-based tool called MIMOX has been developed for phage display based epitope mapping. As a publicly available online service in this area, it is convenient for the community to access, utilize, and evaluate, complementing other existing programs. MIMOX is freely available at .',' pioneering work Smith co worker phage display technology widely basic research exploration protein protein interaction site network apply research development new drug diagnostic vaccine Phage display promising epitope mapping method apply field allergology oncology phage display base epitope mapping usually accomplish compare sequence mimotope antibody select phage display peptide anti gen case mimotope sequence identical similar sequence antigen indicate location native epitope case rare usually mimotope sequence little similarity antigen sequence compare traditional epitope mapping method solve crystal structure antigen antibody complex scan overlap peptide antigen phage display base epitope mapping generally cheap arduous epitope mapping base phage display manually tedious time consume compare set mimotope antigen computational support low sequence similarity mimotope antigen make mapping hard solve problem group research algorithm program assist automate phage display base epitope mapping accord dependency antigen structure exist program phage display base epitope mapping classify category program category FINDMAP work sequence datum mimotope antigen second category need sequence datum antigen structure SiteLight Mapitope belong category recently publish work MIMOP make category integrate different approach work antigen structure implement differently exist program succeed give case exist tool implement freely available online service make convenient community access utilize evaluate present study describe web base tool phage display base epitope mapping name MIMOX code Perl CGI program align set mimotope derive consensus sequence consensus sequence single mimotope sequence map antigen structure potential epitope determine spatial clustering map residue result map antigen structure view interactively validate web base tool compare result MIMOX result computational tool experimentally identify native epitope case study overall architecture MIMOX MIMOX code Perl module Bioperl project online service provide MIMOX accomplish set CGI script MIMOX service divide main section section MIMOX provide simple interface ClustalW align set mimotope sequence implement script alignment derive consensus sequence simple statistical method implement script alignment view manage embed Java applet version JalView implement script second section MIMOX try map user supply sequence give antigen structure implement script program NAC CESS wrap calculate surface accessibility mapping result mapping result rank base solvent accessible surface mapping result detailed information accessibility candidate residue parse script display table new window mapping result view interactively antigen structure implement script wrap Java applet version Jmol overall architecture MIMOX show schematically figure describe MIMOX wrap clustalw align set mimotope sequence allow alignment view edit analyze embed version JalView base review Smith et al implement simple statistical method script derive consensus sequence alignment firstly script count appearance kind amino acid position alignment calculate percentage frequency frequency give amino acid position alignment xi define Xi mean time give amino acid appear position alignment number sequence alignment frequency compare threshold value default frequency threshold corresponding residue consider motif residue position sum frequency similar residue position threshold similar residue regard motif residue MIMOX similar residue group residue consider unique classification scheme Mapitope motif residue find give position stand amino acid residue Motif residue position alignment display table overall architecture MIMOX Figure overall architecture MIMOX MIMOX section section perl script script align set mimotope sequence power script wrap JalView view manage alignment script derive consensus sequence alignment second section perl script script map user supply sequence give antigen structure utilize NACCESS calculate accessibility script display detailed accessibility information mapping result script wrap Jmol view mapping result interactively antigen structure consensus sequence suggest program script create bar figure base statistical analysis axis represent amino acid type gap type axis frequency axis stand position aligned sequence show figure mimotope native epitope antigen bind antibody assume mimotope native epitope similar physicochemical property similar spatial organization assumption basis MIMOX algorithm mapping process MIMOX base input sequence consensus sequence upload antigen structure fragment sequence input firstly position input sequence MIMOX search upload pdb structure match residue place array candidate residue position matching mode available present strict mode mean type mimotope residue match antigen residue exactly call conservative mode mean similar residue include candidate residue array group similar residue MIMOX describe previous section web interface MIMOX section figure web interface MIMOX section mimotope select trastuzumab input align wrap Clus frequency give amino acid position alignment calculate display table bar figure create axis represent amino acid type gap axis frequency axis stand position align sequence consensus sequence suggest mapping MIMOX alignment manage embed JalView array candidate residue position add array array MIMOX find residue neighbour pair consecutive candidate residue array array array residue neighbour determine distance residue distance threshold value distance residue threshold residue take neighbour pair MIMOX provide method calculate neighbour residue pair method distance Cα atom amino acid distance residue Cα atom well reflect backbone position second method use distance Cβ atom well reflect chain position Cα atom glycine Cβ atom method describe base distance heavy atom amino acid distance mention Euclidean distance calculate mean distance atom atom coordinate atom atom method base Cα Cβ atom position default distance threshold angstrom approximate upper limit noncovalent interaction macromolecular structure method distance threshold calculate aswhere DT distance threshold df Distance Factor give user vdwatom Van der Waals radius atom default df value residue pair heavy atom near distance threshold calculate Van der Waals radius residue take neighbour pair MIMOX recursively link neighbour pair possible way form input sequence result rank accord sum absolute residue accessibility residue calculate NACCESS result file end result display table hyperlink script parse display accessibility datum detail script view result interactively map antigen structure web interface MIMOX MIMOX successfully implement online service simple web interface input output describe previously MIMOX divide section input output section figure figure respectively test MIMOX apply case take similar research literature compare result MIMOX result computational tool native epitope epitope know CED database point case monoclonal antibody appropriate test order compare previously publish tool appropriate case polyclonal antibody take correspond literature case study additional file find test dataset page case take FINDMAP Jesaitis co worker anti actin polyclonal antibody select phage display random peptide library VPHPTWMR consensus sequence derive select mimotope manually map VPHPTWMR know structure actin pdb suggest correspond residue Mumey et al FINDMAP align VPHPTWMR actin sequence utilize information antigen structure result FIND MAP show VPHPTWMR map residue FINDMAP map input sequence slightly different set residue instead run MIMOX parameter default get result distance threshold change maximum distance allow MIMOX find mapping return candidate cluster candidate cluster chain amino acid arginine span distance great MIMOX take value maximum allowable distance distance restriction Mapitope case need high distance threshold lie distance map residue MIMOX suggest possibility cluster big solvent accessible surface cluster clearly sequential segment VPHPT WM second case take work Enshell Seijffers monoclonal antibody HIV envelope glycoprotein select phage display random peptide library get set figure mapping result suggest MIMOX expose able bind antibody easily case take MIMOP human monoclonal antibody human coagulation factor VIII Villard et al select phage display random peptide library get set mimotope recently Moreau et al apply newly develop tool MIMOP analyze mimotope combine method MimAlign Mim Cons MIMOP epitope predict compose segment yftnmf residue structure human coagulation factor VIII complex withComparison mapping result MIMOX map LLTTNKD HIV different method take initial case study MIMOX fully partially repeat result manual mapping exist tool provide novel suggestion MIMOX design tool interactive automatic acknowledge tune probe sequence parameter require good result interactive process give hint user step step greatly decrease load server prevent loss reasonable result MIMOX list match result prediction threshold allow user find reasonable result base background knowledge give antibody give antigen give phage display experiment accord test dataset page MIMOX true epitope segment fall result entry result entry result real epitope unknown suggest run MIMOX range parameter consensus sequence derive fragment find overlap promise high surface accessibility candidate mention previously group research algorithm program assist automate phage display base epitope mapping base dependency antigen structure exist program classify category FINDMAP belong category independent structural information FINDMAP implement program align probe consensus sequence derive set mimotope sequence native antigen allow permutation probe sequence use scoring system evaluate quality alignment branch andbound algorithm find alignment maximum score program second category include SiteLight Mapitope MIMOX SiteLight implement test Red Hat Linux program divide native protein surface overlap patch base geodesic distance residue align mimotope library patch score sort finally high scoring match select iteratively native protein cover program implement Visual Basic run Windows divide sequence set overlap subsequence user define length maximum length mimotope search match residue position subsequence sequence pdb structure native protein link neighbour iteratively subsequence complete repeat follow subsequence complete mimotope return result Mapitope implement algorithm describe Enshell seijffer briefly Mapitope deconvolute set mimotope sequence set overlap amino acid pair AAP set major statistically significant pair ssp identify base AAP later ssp map cluster antigen structure finally elaborate diverse cluster antigen surface identify regard predict epitope candidate MIMOP work antigen structure publish case study sequence case independent antigen structure continuous subsequence antigen sequence study need prove MimAlign work antigen structure information exist program describe succeed give case systematic evaluation tool absent show table exist tool implement publicly accessible online service make convenient community access utilize evaluate like software bug creep MIMOX programming expect user send feedback help maintain improve MIMOX future new version MIMOX user definable option support multiple chain antigen implement future allow epitope form residue different polypeptide chain predict systematic evaluation comparison study available tool include MIMOX assist phage display base epitope mapping consideration MIMOX web application phage display base epitope mapping code Perl helpful molecular biologist identify native epitope antibody base antigen structure set mimotope sequence phage display technology publicly accessible web tool area MIMOX convenient community access utilize evaluate complement exist program','2006-10-12 00:00:00','http://dx.doi.org/10.1186/1471-2105-7-451',1577,474),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json','Open lung biopsy in early-stage acute respiratory distress syndrome','INTRODUCTION: Acute respiratory distress syndrome (ARDS) has heterogeneous etiologies, rapid progressive change and a high mortality rate. To improve the outcome of ARDS, accurate diagnosis is essential to the application of effective early treatment. The present study investigated the clinical effects and safety of open lung biopsy (OLB) in patients with early-stage ARDS of suspected non-infectious origin. METHODS: We undertook a retrospective study of 41 patients with early-stage ARDS (defined as one week or less after intubation) who underwent OLB in two medical intensive care units of a tertiary care hospital from 1999 to 2005. Data analyzed included baseline characteristics, complication rate, pathological diagnoses, treatment alterations, and hospital survival. RESULTS: The age of patients was 55 ± 17 years (mean ± SD). The average ratio of arterial partial pressure of oxygen (PaO(2)) to fraction of inspired oxygen (FiO(2)) was 116 ± 43 mmHg (mean ± SD) at biopsy. Seventeen patients (41%) were immunocompromised. Postoperative complications occurred in 20% of patients (8/41). All biopsies provided a pathological diagnosis with a diagnostic yield of 100%. Specific pathological diagnoses were made for 44% of patients (18/41). Biopsy findings led to an alteration of treatment modality in 73% of patients (30/41). The treatment alteration rate was higher in patients with nonspecific diagnoses than in patients with specific diagnoses (p = 0.0024). Overall mortality was 50% (21/41) and was not influenced by age, gender, pre-OLB oxygenation, complication rate, pathological results, and alteration of treatment. There was no surgery-related mortality. The survival rate for immunocompromised patients was better than that for immunocompetent patients (71% versus 33%; p = 0.0187) in this study. CONCLUSION: Our retrospective study suggests that OLB was a useful and acceptably safe diagnostic procedure in some selected patients with early-stage ARDS.',' clinical definition acute respiratory distress syndrome ARDS include acute onset bilateral pulmonary infiltrate ratio arterial partial pressure oxygen PaO fraction inspire oxygen fio mmHg evidence leave atrial hypertension risk factor pneumonia sepsis aspiration associate development ARDS disease condition bronchiolitis obliteran organize pneumonia BOOP adverse reaction drug diffuse alveolar hemorrhage DAH hypersensitivity pneumonitis HP cause ARDS despite similar clinical presentation etiological diagnosis difficult especially early stage ARDS mortality rate patient ards ALI acute lung injury ARDS acute respiratory distress syndrome BAL bronchoalveolar lavage DAD diffuse alveolar damage FiO fraction inspire oxygen hrct high resolution compute tomography ICU intensive care unit OLB open lung biopsy PaO arterial partial pressure oxygen PEEP positive end expiratory recently rapid clinical deterioration patient progress multiple organ failure remain significant challenge intensivist intensive care unit ICU halt disease progression early stage ards accurate diagnosis critical difficult differentiate infectious noninfectious etiology cause ARDS early stage current microbiological sampling technique insufficiently sensitive determine cause ARDS patient patient negative microbiological culture separate true infection inflammatory response clinical datum remain problematic empiric broad spectrum antibiotic typically prescribe critically ill patient immediately admission unnecessary antibiotic therapy non infectious patient enhance occurrence antibiotic resistant strain bacteria increase potential subsequent nosocomial infection therapeutic benefit prolonged glucocorticoid therapy fibroproliferative stage ARDS emphasize need elucidation underlie lung pathology additionally specific disease BOOP drug reaction DAH HP cause ards response steroid therapy inappropriate steroid therapy patient ARDS associate complication gastrointestinal bleeding hyperglycemia increase susceptibility infection previous study demonstrate open lung biopsy OLB useful acceptably safe diagnostic technique patient ARDS study Papazian colleague result OLB directly alter therapeutic management patient ARDS OLB complication air leak occur patient OLB result obtain Patel colleague lead change management majority patient ARDS addition specific therapy patient withdrawal unnecessary therapy patient major complication occur patient study duration intubation OLB long Papazian colleague study range day Patel colleague study day retrospective study attempt evaluate utility safety OLB patient clinically suspect non infectious early stage record patient ard receive OLB icu tertiary care referral center year period January April examine chart discharge diagnosis code International Classification Diseases Ninth Revision Clinical Modification suggest ARDS relate surgery trauma review possible inclusion study total patient ARDS identify olb perform patient olb perform early stage ARDS week intubation patient support noninvasive positive pressure ventilation intubate seven day time biopsy exclude patient meet ards criterion define american european consensus conference decision perform OLB senior intensivist charge respective icu OLB indicate ARDS suspect noninfectious origin obvious etiology possible indication corticosteroid treatment base clinical presentation rapid progression relative symmetric distribution chest ray predominant ground glass attenuation high resolution compute tomography HRCT chest informed consent OLB obtain patient family chest HRCT perform bronchoscopic sampling OLB location bronchoalveolar lavage BAL sampling select basis HRCT finding chest ray HRCT unavailable BAL perform introduce ml sterile warm saline solution lung subsegment aspirate ml aliquot aliquot return bronchial fraction discard speciman send bacterial examination Legionella Mycoplasma pneumoniae pneumocystis carinii Mycobacteria fungal virological cytomegalovirus influenza virus parainfluenza virus adenovirus herpe simplex virus respiratory syncytial virus coxsackie virus analyse specimen send cytology iron stain analysis BAL result deem positive minimum microorganism grow concentration colony form unit ml procedure perform hour perform operating room bedside ICU experienced thoracic surgeon Bedside OLB indicate fio reach applied positive end expiratory pressure PEEP cmh regard mechanical ventilator setting prevent air leakage PEEP immediately reduce cmh baseline level surgery pulmonary tissue harvest site consider new progressive lesion identify chest HRCT chest ray tissue speciman cultured examine pulmonary pathologist medical record patient review analyze follow datum age gender Acute Physiology Chronic Health Evaluation APACHE ii score admission ICU acute lung injury ALI score PEEP PaO ratio ards diagnosis date ards onset respiratory failure intubation biopsy underlie disease diagnostic test biopsy medication time biopsy result complication biopsy pathological diagnosis postoperative therapeutic change addition removal drug analyze outcome parameter include ICU hospital survival rate cause death evaluate normally distribute datum value report mean sd student test compare normally distribute continuous variable difference subgroup compare test Fisher exact test expect number event significance level statistical test set consider statistically significant patient undergo OLB ards evaluation study period exclude duration intubation OLB exceed seven day total patient enrol table list baseline characteristic patient study patient immunocompetent patient immunocompromise cause immunocompromise status hematological malignancy patient solid tumor patient bronchogenic cancer breast cancer HIV infection patient renal transplantation duration intubation OLB patient day mean sd range perform hour OLB finding BAL compatible pathological diagnosis patient diagnosis bacterial pneumonia mycobacterial tuberculosis cytomegalovirus pneumonitis pneumocystis carinii pneumonia patient chest HRCT OLB identify appropriate biopsy site remain patient undergo chest HRCT olb perform right middle lobe patient lingular lobe seven patient patient underwent OLB operating room receive bedside OLB ICU videoassiste thoracotomy perform patient remain patient undergo limited anterior thoracotomy intra operative complication occur patient postoperative complication seven day operation patient develop transient hypotension OLB regain normal status fluid resuscitation vasopressor treatment hour patient pneumothorax diagnose chest ray require chest tube low pressure suction cmh drainage hour OLB patient subcutaneous emphysema localize chest area OLB resolve spontaneously day additionally patient bronchopleural fistula persistent air leak operative chest tube day need surgery patient transient hypotension pneumothorax subcutaneous emphysema bronchopleural fistula die surgical complication result directly death incidence postoperative complication patient undergo OLB operating room bedside ICU respectively complication rate significantly different group biopsy provide sufficient datum pathological diagnosis diagnostic yield specimen obtain OLB send tissue culture bacteria virus culture result negative pathological diagnosis subdivide specific nonspecific category eighteen patient specific diagnosis establish OLB nonspecific diagnosis table OLB finding lead alteration therapy patient OLB patient administrate high dose corticosteroid therapy day methylprednisolone divided dose day seven patient treat low dose corticosteroid therapy mg kg day methylprednisolone divided dose patient receive co trimoxazole pneumocystis carinii pneumonia antibiotic change patient discontinue patient basis pathological finding treatment change patient table present comparative result patient characteristic complication rate alteration treatment survival rate patient specific nonspecific pathological diagnosis OLB rate treatment alteration high nonspecific pathological diagnosis group specific diagnosis versus significant difference group note patient die ICU result ICU survival rate hospital survival rate ICU survival rate multiple organ dysfunction syndrome lead cause death patient follow septic shock patient hypovolemic shock patient acute myocardial infarction patient table present comparative result patient characteristic outcome survivor nonsurvivor significant difference observe survivor nonsurvivor baseline datum age gender severity illness complication rate treatment alteration rate group significantly immunocompromise patient survivor group nonsurvivor group vs immunocompromise immunocompetent patient Table show immunocompromise patient young low ALI score furthermore immunocompromise patient well hospital survival rate immunocompetent patient versus study show OLB acceptably safe useful procedure select patient early stage ARDS treatment alteration rate high patient ARDS nonspecific pathological diagnosis specific diagnosis recent study patient ARDS OLB employ relatively late time intubation OLB considerable day study Papazian colleague day study Patel present study OLB perform week intubation day substantially early previous study Patel Specific diagnosis rate base OLB finding vary study patient different disease entity specific diagnostic rate review Cheson colleague immunocompetent patient immunocompromise patient study specific nonspecific diagnostic rate respectively specific diagnostic rate immunocompetent immunocompromise patient respectively statistically significant specific diagnostic rate immunocompetent immunocompromise patient similar previous study indicate OLB obtain high percentage specific pathological diagnosis immunocompromise patient study rate therapy alteration OLB low previous report range group nonspecific specific pathological diagnosis rate change therapy high nonspecific group versus analytical finding result large number patient nonspecific pathological diagnosis undergo corticosteroid treatment rescue anti inflammatory therapy exclude potential active infection fibroproliferative stage DAD interstitial pneumonitis nonspecific interstitial pneumonitis organize pneumonia early OLB achieve diagnosis fibrosi potentially treatable corticosteroid furthermore recent study ARDS Clinical Trials Network support routine use methylprednisolone patient persistent ARDS seven day onset andsuggeste methylprednisolone therapy harmful initiate week onset ARDS duration ARDS corticosteroid treatment interact significantly survival immunocompromise patient study suggest OLB advantageous diagnosis treatment alteration benefit survival remain unclear McKenna colleague find immunocompromise patient early OLB average day admission benefit histological diagnosis interstitial pneumonitis treat steroid OLB improve clinical outcome patient overall mortality rate present study similar obtain previous report range immunocompromise patient survivor group well survival rate immunocompetent patient versus young age low ALI score immunocompromise patient probably account well outcome furthermore enhanced survival rate immunocompromise patient attribute immunocompromise patient immunocompetent patient receive high dose corticosteroid therapy active infection exclude OLB pulmonary condition infection disease progression therapeutic reaction new unrelated pathology combination present immunocompromise patient diagnostic yield adequate treatment early OLB consider reliable diagnostic modality provide early accurate etiological diagnosis immunocompromise patient operative complication rate report OLB patient ARDS range study overall rate OLB postoperative complication late fibrotic stage lung parenchyma stiff early exudative fibroproliferative stage ARDS operative complication multifactorial early OLB non stiff lung fibrosi present study report account low surgical complication rate study patient present study transport operating room administer high peep consequently OLB perform bedside ICU intra operative complication exacerbation oxygenation hemodynamic occur patient ARDS severe hypoxemia patient developed postoperative complication hypotension pneumothorax subcutaneous emphysema bronchopleural fistula respectively death attributable OLB risk complication OLB early stage ARDS acceptable critically ill patient severe hypoxemia limitation study consider retrospective nature study directly address question early OLB survival benefit understanding specific etiology permit initiation specific therapy assume therapy available diagnosis find study metastatic malignancy infectious pneumonia hypersensitivity pneumonitis establish positive therapeutic effect outcome second result study generally apply patient ARDS decision perform OLB random patient refer OLB unlikely representative sample ards population selection bias patient intensivist expect increase possibility alternative intervention limitation specific diagnosis viral pneumonitis underdiagnose identification depend availability laboratory facility standardized comprehensive microbiological examination BAL OLB establish retrospective study demonstrate OLB high diagnostic yield rate acceptable complication rate select patient early stage ARDS rate treatment alteration high patient nonspecific pathological diagnosis specific pathologically diagnose ard prospective randomized control study investigate appropriate indication effect OLB outcome patient Open lung biopsy acceptably safe diagnostic procedure select early stage patient acute respiratory distress patient early stage acute respiratory distress syndrome suspect non infectious origin open lung biopsy high diagnostic yield role open lung biopsy patient acute respiratory distress syndrome need investigate prospective randomized control clinical trial','2006-07-19 00:00:00','http://dx.doi.org/10.1186/cc4981',1661,611),('c1ad13d83e926979dbf2bbe52e4944082f28dfea.json','Antisense-induced ribosomal frameshifting','Programmed ribosomal frameshifting provides a mechanism to decode information located in two overlapping reading frames by diverting a proportion of translating ribosomes into a second open reading frame (ORF). The result is the production of two proteins: the product of standard translation from ORF1 and an ORF1–ORF2 fusion protein. Such programmed frameshifting is commonly utilized as a gene expression mechanism in viruses that infect eukaryotic cells and in a subset of cellular genes. RNA secondary structures, consisting of pseudoknots or stem–loops, located downstream of the shift site often act as cis-stimulators of frameshifting. Here, we demonstrate for the first time that antisense oligonucleotides can functionally mimic these RNA structures to induce +1 ribosomal frameshifting when annealed downstream of the frameshift site, UCC UGA. Antisense-induced shifting of the ribosome into the +1 reading frame is highly efficient in both rabbit reticulocyte lysate translation reactions and in cultured mammalian cells. The efficiency of antisense-induced frameshifting at this site is responsive to the sequence context 5′ of the shift site and to polyamine levels.',' standard triplet readout genetic code reprogramme signal mrna induce ribosomal frameshifting review generally result trans frame protein product functional case express equal amount product standard translation elaboration genetic code demonstrate versatility decode requirement eukaryotic ribosomal frameshifting include shift prone sequence decode site downstream secondary structure mRNA majority program frameshift site consist heptanucleotide sequence XXY YYZ nucleotide configuration site trnas pair nt shift nt end mrna similarly frameshift site identity codon site ribosome critical efficient frameshifting factor affect frameshift efficiency initial stability site trna mrna interaction frame high efficiency frameshifting occur site trna form standard codon anticodon interaction study correlation frameshift efficiency final stability site trna mrna interaction frame show previously system appear little correlation addition competition decode frame frame codon site affect frameshifte efficiency slow decode frame codon stop codon decode low abundance trnas favor frameshifting frame codon high level correspond cognate trnas level frameshifting achieve stimulatory action cis act element locate downstream shift site wide variety structure commonly type pseudoknot identify stimulate frameshifte eukaryote review mutagenic structural datum frameshift stimulator demonstrate pseudoknot key structural feature require frameshift stimulation unify structural feature essential frameshifting identify observation combine recent report simple antisense oligonucleotide functionally mimic cis act stimulator frameshifting demonstrate different structure stimulate frameshifte note structure equal thermodynamic stability stimulate frameshifte Discussion RNA pseudoknot show stimulate program frameshifting eukaryotic antizyme gene Antizyme negative regulator cellular polyamine level ability target ornithine decarboxylase rate limit enzyme polyamine biosynthesis degradation inhibit polyamine import stimulate export antizyme expression induce high intracellular polyamine level decrease lowered level polyamine sensor programmed frameshift event require antizyme synthesis correspondence address tel fax email Open Access article distribute term Creative Commons Attribution non commercial License permit unrestricted non commercial use distribution reproduction medium provide original work properly cite low polyamine level termination end open reading frame efficient high level polyamine substantial proportion ribosome shift reading frame resume standard decode synthesize length active antizyme protein frameshifte mammalian antizyme mrna shift site UCC UGA stimulate cis act signal element encompass basis upstream shift site important polyamine effect cis act element pseudoknot locate shift site mammalian antizyme pseudoknot structurally distinct counterpart subset invertebrate antizyme mrnas pseudoknot know act stimulator frameshifting eukaryote unknown pseudoknot stimulate frameshifting frameshifting different mechanism notable difference find positioning downstream structure relative shift site naturally occur pseudoknot stem loop stimulator frameshifting typically begin nt downstream site codon shift site frameshift pseudoknot locate close nt separation site codon mutagenic study reveal alter size spacer affect frameshifte general reduce efficiency test ability antisense oligonucleotide anneal downstream shift prone site UCC UGA induce shift ribosome reading frame directionality frameshifting reading frame show dependent position duplex region relative shift site efficiency frameshifting responsive polyamine level enhance inclusion stimulatory sequence find upstream human antizyme program frameshift site complementary oligonucleotide construct sequence describe paper synthesize University Utah DNA Peptide Core Facility anneal appropriate end ligate BamHI site dual luciferase vector dual luciferase construct prepare sequence verify describe previously sequence shift site boldface give follow TCGACGGTCTCCCTCCACTGCTGTAG TAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATG CCCCTCACCCACCCCTGAAGATCCCAGGTGGGCGAG TCGACGGTCTCCCTCCACTGCTGTAG TAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGAC CCTCACCCACCCCTGAAGATCCCAGGTGGGCGAGG TCGACGGTCTCCCTCCACTGCTGTAG TAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATG TCGACGGTCTCCCTCCACTGCTG TAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCT TCGACGGTCTCCCTCCACTGCTGT AGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTG TCGACGGTCTCCCTCCACTGCTGTAGT AACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATG TCGACGTGCTCCTGATGCCCCTG TCGACGTGCTCCTGGTGCCCCTG GATC dual luciferase construct mg describe add directly TNT couple reticulocyte lysate reaction Promega label methionine volume ml reaction incubate radiolabele protein separate SDS PAGE gel fix acetic acid methanol min dry vacuum gel visualize storm PhosphorImager Molecular Dynamics radioactive band quantify ImageQuant software percent frameshifting calculate percentage length frameshift product relative termination product length product combine value product correct number methionine codon present code sequence report value average standard deviation obtain independent measurement table show percent frameshifting standard deviation find Supplementary Data Plasmid co transfecte cell vary concentration Methyl antisense oligonucleotide follow condition cell ml DMEM fetal bovine serum add well area tissue culture treat plate contain ng DNA vary amount antisense oligonucleotide ml Lipofectamine Invitrogen ml Optimem cell incubate co medium remove cell transfecte cell lyse ml lysis buffer luciferase activity determine measure light emission follow injection ml luminescence reagent Promega percent frameshifting calculate compare firefly renilla luciferase ratio experimental construct control construct firefly experimental rlu renilla experimental control rlu Renilla control rlu ability cis act RNA structure trans act Methyl antisense oligonucleotide induce ribosomal frameshifting determine vitro transcription translation dual luciferase reporter vector contain Renilla firefly luciferase gene multiple cloning site transcribe promoter locate upstream Renilla luciferase gene sequence contain shift prone site clone reporter gene downstream firefly luciferase gene reading frame result construct transcribe translate vitro complementary cis act stimulator frameshifting antizyme shift site initially dual luciferase reporter vector generate contain human antizyme frameshift cassette stimulator frameshifting pseudoknot delete replace stem loop figure construct subject couple transcription translation reaction presence increase amount spermidine label product separate SDS PAGE table maximal level frameshifting find occur mm antisense oligonucleotide add transcription translation reaction Supplementary Table presence mM exogenous spermidine highly efficient shifting ribosome reading frame high observe wild type antizyme frameshift cassette observe addition Supplementary Table efficient frameshifting observe antisense oligonucleotide anneal space shift site beginning duplex region observe shift site beginning stem natural antizyme pseudoknot structure nt spacer verify antisense oligonucleotide activate ribosomal frameshifting transcription slippage RNA transcribe absence oligonucleotide add reticulocyte lysate translation presence increase amount Methyl oligonucleotide frameshifting level increase level observe couple transcription translation reaction demonstrate oligonucleotide act induce frameshifting translation Supplementary Figure addition spacer induce high level frameshifting reading frame manner modestly inhibit addition spermidine absence presence mM exogenous spermidine figure Supplementary Table frameshift product observe wild type antizyme cassette examine absence antisense oligonucleotide addition figure AZwt antisense oligonucleotide design anneal directly adjacent UGA codon shift site interest determine wild type antizyme pseudoknot induce frameshifting locate equivalent position address new construct figure delete nt spacer pseudoknot shift site case wild type pseudoknot directly adjacent shift site product vitro transcription translation separate SDS PAGE frameshift product observe level frameshift product significantly reduce figure Supplementary Table design complement RNA sequence encode originate vector determine duplex form antisense oligonucleotide adjacent antizyme sequence result efficient frameshift stimulation reporter vector design contain portion antizyme stimulator construct contain sequence half axis form stacking stem stem pseudoknot figure complementary Methyl antisense oligonucleotide design perfect complimentarity region start nt end nt downstream UGA shift site codon second mispaired bulge locate position respectively alteration include closely mimic natural pseudoknot contain mispaired bulge equivalent position extended stem form stacking pseudoknot stem figure compare duplex form antisense oligonucleotide induce frameshifting respectively add couple transcription translation reaction presence spermidine figure Supplementary Table induce frameshifting level see suggest sequence content duplex region affect efficiency frameshift stimulation native antizyme sequence require second construct design contain half stem antizyme pseudoknot downstream UCC UGA shift site figure Methyl antisense oligonucleotide design anneal nt nt downstream UGA codon shift site frameshift efficiency induce antisense oligonucleotide respectively somewhat low observe figure Supplementary Table case frameshift efficiency high long antisense oligonucleotide suggest frameshift efficiency probably correlate stability duplex see frameshifting efficiency stimulate antisense oligonucleotide strongly correlate concentration exogenously add spermidine Supplementary Table importance antizyme sequence context antisense oligonucleotide induce ribosome frameshifting examine test frameshift site UCC UGA stimulatory antizyme sequence end antizyme stimulatory sequence delete antisense oligonucleotide add couple transcription translation reaction presence absence spermidine frameshift efficiency measure presence spermidine absence respectively figure determine stop codon shift site essential frameshifting UGA codon alter ugg shift site UCC UGG UGG frameshift efficiency significant reduce compare shift site UCC UGA show little stimulation addition spermidine induce frameshifting presence spermidine frameshifting absence respectively figure ability antisense oligonucleotide induce frameshifting cultured mammalian cell examine co transfection cell increase amount Methyl antisense oligonucleotide describe Materials Methods absence antisense oligonucleotide frameshifting level determined grade increase frameshift level observe addition figure maximal frameshifting level presence mm transfection medium model attempt explain pseudoknot stimulation program frameshifting propose review model invoke pausing mechanism ribosome pause shift site time allow trnas reposition new reading frame explanation clearly simplistic stem loop pseudoknot similar thermodynamic stability cause ribosome pausing necessarily effective frameshift stimulator addition variation IBV pseudoknot demonstrate lack correlation extent pausing efficiency frameshifting recent publication Brierley co worker present structural datum demonstrate IBV frameshift stimulate pseudoknot block mrna entrance tunnel lead structural deformation site trna result movement trna displace anticodon loop end mrna model present movement result disruption codon anticodon interaction allow trna slippage relative mrna similar trna movement observe non frameshift stimulate stem loop structure model provide feasible mechanistic explanation ability downstream structure induce frameshifting ability antisense oligonucleotide induce highlevel frameshifting demonstrate elaborate tertiary structure require duplex form complementary antisense oligonucleotide variety chemistry include RNA Methyl morpholino sufficient induce high level frameshifting demonstrate time trans act antisense oligonucleotide stimulate ribosome shift reading frame surprisingly high level level great achieve natural cis act mrna pseudoknot structure program frameshifting structural study indicate mrna begin enter ribosome nt downstream site codon direct relevance study result indicate maximal frameshifting induce antisense mrna duplex begin nt downstream UGA shift site agreement distance find UGA shift site beginning stem pseudoknot stimulator find antizyme gene give distance implication stimulatory secondary structure encounter ribosome UCC codon enter site ribosome suggest structural study frameshift induce pseudoknot codon anticodon interaction UCC codon ser trna ser disrupt translocation site give importance UGA codon frameshifte UCC UGA shift site subsequent event follow translocation UCC codon site UGA site influence frameshifting efficiency event probably involve competition termination frame decode UGA codon site discussion present importance site site event ribosomal frameshifting clearly investigation topic warrant observation present antisense oligonucleotide anneal directly adjacent UGA stop codon induce ribosome frameshift reading frame surprising light discussion space naturally occur cis act frameshift stimulator possible frameshifting occur codon upstream known UCC UGA shift site visual examination upstream codon reveal obvious frameshift site ability spermidine stimulate antisense oligonucleotide induce ribosome frameshifting reading frame UCC UGA shift site absence natural stimulator demonstrate cis act element require polyamine responsiveness similarly spermidine stimulation observe absence element virtually eliminate alter UGA codon shift site UGG observation agreement previous study examine importance cis act element polyamine induce frameshifting expression antizyme gene ability direct ribosome reading frame living cell figure suggest potential therapeutic application antisense oligonucleotide direct frameshifte reading frame near disease cause frameshift mutation cause ribosome resume decode wild type ORF restore partial production length protein mutant allele importance stop codon efficient frameshifting suggest stop codon follow frameshift mutation present promising target antisense induce phenotypic suppression modulation intracellular polyamine level essential increase effectiveness approach experiment require determine therapeutic potential approach vivo include generality efficiency frameshift induction non programmed frameshift site','2006-08-18 00:00:00','http://dx.doi.org/10.1093/nar/gkl531',1708,541),('c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json','ProCAT: a data analysis approach for protein microarrays','Protein microarrays provide a versatile method for the analysis of many protein biochemical activities. Existing DNA microarray analytical methods do not translate to protein microarrays due to differences between the technologies. Here we report a new approach, ProCAT, which corrects for background bias and spatial artifacts, identifies significant signals, filters nonspecific spots, and normalizes the resulting signal to protein abundance. ProCAT provides a powerful and flexible new approach for analyzing many types of protein microarrays.',' dna microarray technology prove extremely valuable probe biological process measure mrna expression profile study protein level potential provide direct information gene function protein product traditional investigation focus individual protein system combine individual analysis provide global perspective recently technology analyze protein high throughput unbiased fashion feasible particular powerful technology protein microarray contain high density protein allow systematic probing biochemical activity type protein microarray functional protein microarray contain set protein individually produce position addressable format microarray surface functional protein microarray useful identify bind activity target modification enzyme version proteome microarray report contain yeast protein amino terminal glutathione transferase GST tag print array second version yeast protein microarray generate recently contain protein carboxy terminal ha ZZ domain tag protein collection overexpresse purify spot protein microarray global proteome study perform chip understand biological mechanism example yeast kinase examine substrate yeast protein microarray vitro substrate represent unique protein identify compare approximately know vivo kinase substrate interaction global study serve important step dissect yeast signal network addition search kinase substrate proteome chip probe label protein DNA lipid antibody molecule search interact protein large amount datum generate protein microarray present significant challenge develop robust method process raw datum build reasonable biological hypothesis dataset second type protein microarray analytical protein microarray antibody microarray share similarity immunoassay use antibody detect specific probe study show antibody array recognize specific target generate dose dependent signal intensity indicate quantify level target crude mixture crossreactivity certain antibody variety protein highly specific antibody suitable type study remain limit factor prepare antibody microarray DNA protein microarray prone systematic error usually generate different source surface defect spatial artifact study offer insight noise subtraction dna microarray little investigation protein microarray functional protein microarray differ respect DNA microarray goal microarray different dna microarray measure relative dna level pool probe functional protein array aim discover global interaction single probe molecule second typical dna microarray experiment measure signal ratio color channel test mrna sample reference sample signal second channel serve intrinsic control help decrease effect amount reagent array local array nonuniformity furthermore current scaling method base assumption signal intensity balance color channel despite variation slide location intensity source systematic variation control miss onecolor channel protein microarray scaling approach DNA microarray base set housekeeping gene constant signal intensity different condition protein microarray control group customize accord type activity assay ubiquitous reference group exist fourth unlike dna microarray non specific binding address signal comparison mismatch probe cross reactivity protein microarray directly correct separate slide require probe parallel negative control protein microarray experiment finally protein specific artifact serve common noise source protein microarray kinase assay example signal strongly phosphorylate spot bleed neighboring spot lead incorrect background measurement difference particularly applicable functional protein microarray comparison antibody array normalization technique DNA microarray usually directly applicable functional protein microarray develop new protein chip analysis tool ProCAT deal artifact specific functional protein microarray work start careful survey characterization potential source systematic error protein microarray specific approach design deal type noise correction approach apply reduce measurement error background signal addition spatial variation reduce efficiently novel parameter signal normalization approach call positive spot locally generate list positive negative control slide analyze approach spot subtract list appear control slide slide feature poor signal quality remove finally signal intensity positive normalize accord protein amount module account challenge datum processing specific protein microarray build ProCAT test ProCAT contain flexible modular design individual component adjust accord experimental design stringency level select user sequential module currently implement ProCAT final annotation report assemble figure module carry background correction signal normalization positive spot identification spot cross reactivity filter signal quality inspection protein normalization performance step test type experiment describe fundamental issue microarray experiment background correction aim reduce noise background quantification Signal intensity generally quantify subtract foreground intensity local background intensity measure background signal immediately surround spot interest term adjacent background figure protein microarray local background region easily skew artifact small speckle addition strong positive signal chip kinase assays tend produce signal smear film phosphoimager exceed normal feature size figure case measurement spot inaccurate background intensity arbitrarily high diminish real signal intensity spot second intensity affect alignment grid extent smear variance protein replicate experiment increase method reduce artifact local background user manually adjust grid size fit circle individual spot align process require considerable time effort size smear prevent refit grid adversely affect neighboring spot additionally large spot size diminish signal spot signal density decrease increase spot size second method background correction apply procat replace background intensity central spot background local neighborhood surround window assign protein spot median background spot neighborhood background value central spot Materials method detail additional time need alignment method significantly reduce artifact produce erroneous measurement spot background analysis phosphorylome dataset apply neighborhood background correction observe high sensitivity identify positive target characterize effect neighborhood background correction perform test kinase assay nM protein kinase PKA spot location slide figure block slide contain PKA pair random yeast protein spot approximately spot incubate slide ATP PKA spot autophosphorylate show strong signal case signal go grid circle boundary figure apply neighborhood background correction PKA spot expect median PKA signal intensity enhance furthermore PKA signal different position similar variance decrease value Fig neighborhood method access background provide robust measurement adjacent background method spatial artifact arise uneven signal distribution slide uneven probe condition smear artifact uneven probing occur mean uneven mixing probe exposure probe solution uneven washing dry slide twocolor channel experiment DNA microarray provide intrinsic control account spatial artifact functional protein microarray use color channel especially prone spatial artifact spatial artifact cause inaccurate measurement signal intensity hinder identification significant interaction add control help remove spatial artifact signal spot normalize accord local control variable shape size spatial artifact ideally large number control need space constraint protein chip inability anticipate use array usually prevent necessary number control fully account spatial artifact array scaling method reduce signal variation spot protein different array location decrease spatial artifact develop new normalization method deal spatial artifact specific functional protein microarray assume signal distribution large window consistent slide foreground signal spot normalize accord signal intensity surround neighborhood assumption usually valid protein microarray experiment protein randomly print array figure parameter median median absolute deviation MAD calculate represent signal distribution local window figure perform normalization median MAD slide window average average value correct signal central spot test performance parameter scaling approach signal normalization slide design test microarray contain multiple positive control print different position slide test array organize format commercially available protein microarray Invitrogen protein print duplicate array contain block print protein figure GST fusion protein purify separately dilution protein prepared concentration print random position array spot occupy bovine serum albumin BSA negative control order visualize fusion protein anti gst antibody probe slide probe typical spatial artifact show Figure artifact contain slide show different signal level edge middle portion array produce block variable signal distribution range high low edge slide opposite edge variability occur block simple block normalization method adopt dna microarray normalization approach suitable deal problem apply ProCAT normalize slide different parameter Figure window size test term window number correspond window size function number spot edge block example block spot analyze window window size block edge case spot central spot window size contain area roughly large block observation analysis different window size window size increase computational time normalization increase second obvious spatial artifact leave normalization window size test figure small window size diminish signal inequality exist positive signal background noise small scaling window tend introduce extreme change original signal increase discrepancy duplicate spot protein variance signal protein normalization different window size calculate case dilution protein scale window successfully reduce signal variance range figure decrease signal variation suggest large scaling window help reduce spatial artifact large window size possible default number ProCAT analysis reasonable time minimal improvement achieve window size additional datum file addition provide accurate measurement spot intensity ProCAT develop assign threshold identify positive target experiment traditionally global cutoff calculate spot apply slide variable spatial artifact cutoff assign locally ProCAT spot array signal distribution window calculate cutoff define number standard deviation away mean default ProCAT standard deviation cutoff correspond significance level signal distribution local window normal spot strong signal include window cutoff arbitrarily high decrease sensitivity detect positive spot program avoid loss sensitivity ProCAT build function identify possible outlier remove outli spot extremely strong signal calculate cutoff identify positive spot remain spot receiver operate characteristic ROC curve compare performance local window cutoff versus global cutoff test slide area ROC curve AUC performance indicator range good performing method GST GST positive control BSA negative control sensitivity specificity local global cutoff method estimate window size test compare global cutoff figure prediction performance increase significantly local window spot edge window default ProCAT large representative protein microarray high quality datum figure representative protein microarray high quality datum slide image reconstruct protein microarray experiment minimal noise datum density plot signal local window window size spot computationally combine show high similarity window size result increase computing time minimal improvement sensitivity AUC value large local cutoff compare global cutoff improvement unlikely random chance value conclude local cutoff significantly well identify positive spot global cutoff scheme signal scaling method Figure scheme signal scaling method signal spot array normalize accord distribution local neighborhood spot surround window choose spot window define neighborhood signal center spot normalize compare local median MAD average value norm normalize signal Origin original signal average distribution layer filter implement ProCAT positive spot negative control experiment remove example chip kinase assay kinase dead allele probe separate array experimental condition wild type kinase spot produce signal absence active kinase identify ProCAT remove target list kinase probing probe tag protein detect protein protein interaction test epitope tag absence protein interest essential control proper negative control experiment available ProCAT analyze way regular experiment construct experimental positive spot list void protein produce positive signal control condition second filter check quality positive spot protein spot duplicate protein microarray similar signal intensity procat use difference duplicate signal indicator signal quality difference signal duplicate spot calculate fit normal distribution protein exceptionally large difference duplicate spot likely bias certain artifact remove positive list default threshold duplicate spot difference ProCAT set standard deviation away mean goal protein microarray experiment identify affinity bind interaction protein protein interaction assay extent phosphorylation kinase assay compare relative strength reaction positive protein ideally spot intensity directly correspond strength interaction number factor contribute array signal intensity include systematic noise artifact discuss protein print chip nonetheless semi quantitative estimate obtain background correction signal normalization raw signal standardize relative protein amount estimate interaction strength protein microarray different amount share epitope purpose large scale protein purification probe anti epitope antibody provide estimate relative protein amount spot array protein determine spot row column ProCAT divide raw signal intensity protein signal use quotient approximation strength interaction approximation generally work slide follow situation abundant protein bias value estimate antiepitope probing susceptible background noise slide artifact hand overpower spot biased saturate signal intensity saturate value underestimate real signal reason protein amount minimal cutoff signal intensity low saturation threshold normalize protein amount protein conform requirement record unnormalized signal flag inspection additional caveat relative protein assess antibody include native denature protein give spot estimation interaction strength underestimate functional protein overestimate ProCAT design flexible tool analyze functional protein microarray datum program script Perl version Tomcat version web server module discuss implement independently include exclude depend experimental design input dataset user characterize datum aspect experimental design datum file format normalization parameter experimental design contain parameter number test array negative control array particular assay datum file format describe layout uploaded dataset ProCAT recognize extract useful information normalization parameter allow user try different stringency level level supply sufficient information uniquely characterize experiment allow ample flexibility individual user customize parameter suit different type experimental design inputte description upload dataset ProCAT take minute average complete analysis module array time vary depend select analysis module size protein microarray task assign unique ID result organize database future query process datum include analysis parameter list positive spot protein annotation normalize signal intensity available user download server functional protein microarray serve efficient platform screen protein biochemical function present procat systematic approach process analyze datum specific functional protein microarray calibrate explicit test experiment ProCAT prove able handle type functional protein microarray study unique feature ProCAT include novel scaling method provide robust reproducible measurement quantitative signal crucial protein microarray chip signal intensity indicate strength interaction addition call positive candidate locally ProCAT demonstrate excellent performance identify positive comparison global threshold finally step integrate modular design fit experimental design stringency requirement major challenge design automate datum processing method think anticipate possible situation arise ProCAT use local window correct background contain signal smear dust speckle method assume artifact sparse majority spot local window provide correct measurement background signal median value spot correct background biased spot window severely affect correct background value assumption usually valid percentage spot positive signal contaminate artifact protein microarray experiment generally low extreme case spot likely close large window example large artifact bright speckle incubation bubble affect spot particular region shape artifact variable necessary manually flag spot initially remove future analysis commercially available software package microarray experiment build flagging function ProCAT automatically discard flag spot key aspect ProCAT parameter approach reduce spatial nonuniformity factor affect performance ProCAT normalization ProCAT normalize signal spot accord signal distribution local neighborhood diminish signal intensity spot locate high signal neighborhood compensate intensity low signal neighborhood approach base assumption signal intensity slide share distribution hold true regional variation observe slide technical artifact real biological difference protein print random order current protein microarray unlikely particular region slide gain high intensity result biologically relevant reason second size neighborhood window largely affect performance normalization small window size tend add bias signal diminish local variation large window size increase computational burden tend preserve local variation find optimal window size ProCAT protein protein interaction figure correspond approximately block chip default window size choose fit shape spatial artifact ProCAT apply experiment protein microarray kinase assay protein protein interaction protein dna interaction far twoparameter scaling approach single chip normalization similar strategy extend rescale multiple slide assume signal neighborhood window different slide similarly distribute overall ProCAT provide powerful flexible new approach optimal processing analysis functional protein microarray slide test background correction nm PKA Sigma Louis MO USA spot different place positive control slide incubate μl kinase buffer mm Tris pH mM NaCl mm MgCl mm glutathione glycerol plus ml BSA Triton μl ATP humidified chamber hour slide wash twice mm Tris pH SDS double distil spin dry expose ray film Kodak Rochester NY anti gst probing slide print positive control nM BSA negative control array surface block SuperBlock Pierce Rockford IL USA hour rabbit polyclonal IgG Santa Cruz Biotechnology Santa Cruz USA incubate slide fold dilution array wash PBST Sigma incubate dilution conjugate anti rabbit IgG antibody Jackson Laboratories Bar Harbor USA slide wash PBST time scan AxonTesting experiment signal scaling approach figure previous page testing experiment signal scaling approach design test slide positive spot show red spot test normalization window indicate red square give spot array show blue Comparison signal intensity normalization window size testing experiment image computationally reconstruct signal file normalization GenePix scanner Molecular Devices Sunnyvale USA raw signal extract GenePix Pro software Molecular spot let row column protein microarray represent adjacent background intensity denote foreground intensity raw signal intensity calculate neighborhood background correction use neighborhood background replace adjacent background local window choose neighborhood background define protein slide row column local window spot define signal set spot satisfy size parameter dependent window size factor win block size block block represent number spot edge block win choose user option different window overlap block edge let denote signal intensity spot local window ProCAT use parameter characterize signal distribution median MED median absolute deviation calculate MED MAD spot array average obtain parameter reference distribution spot ProCAT normalize raw signal compare MED MAD average value give spot row column normalize signal compare surround spot window ij signal window fit normal distribution mean standard deviation calculate default threshold set standard deviation signal mean spot call positive signal threshold positive spot likely close pro CAT use box plot examine remove possible outlier surround window let low quartile percentile upper quartile percentile difference term interquartile range δq spot define outlier signal ROC curve compare global cutoff local cutoff call positive spot Figure ROC curve compare global cutoff local cutoff call positive spot test slide unique positive control different titration performance identify positive control increase local cutoff generate relatively large surround window window size test good performance achieve large window obtain robust threshold correct mean standard deviation generate non outli spot follow additional data file available online version paper additional data file figure illustrate variance reduction positive control different normalization window size additional data file table list raw signal generate autophosphorylation experiment test background correction method additional data file table list raw signal generate anti gst probe experiment calibrate signal scaling approach Additional File Variance reduction positive control different normaliza tion window size Variance reduction positive control different normaliza tion window size click file Additional File raw signal generate autophosphorylation experiment test background correction method raw signal generate autophosphorylation experiment test background correction method click file Additional File raw signal generate anti gst probe experiment cali brate signal scaling approach raw signal generate anti gst probe experiment cali brate signal scaling approach click file','2006-11-16 00:00:00','http://dx.doi.org/10.1186/gb-2006-7-11-r110',2843,750),('c6202f96bd4b2aea699d5fbe867f311498dca3aa.json','Bioethical Implications of Globalization: An International Consortium Project of the European Commission','The BIG project looks at some of the ethical concerns surrounding globalization and health.',' term globalization popularize Marshall McLuhan War Peace Global Village book McLuhan describe global medium shape current event surround Vietnam War predict modern information communication technology accelerate world progress trade knowledge development globalization refer broad range issue movement good service trade liberalization movement people migration write global effect environmental degradation population growth economic disparity addition pace scientifi development accelerate negative positive implication global health concern national health transcend border need share human security enhanced role international cooperation development issue signifi nt bioethical implication renew academic focus ethical dimension public health need future development science health policy require fi rm ground bioethical principle core principle include benefi cence nonmalefi cence harm respect person human dignity autonomy attention equity social justice accord World Health Organization global ethical approach monitor update ethical norm research necessary anticipate ethical implication advance science technology health apply internationally accept code ethic ensure agree standard guide future work human genome ensure quality health system service assess promote Bioethical Implications Globalization BIG Project month dialogue fund European Commission involve series expert panel discussion specifi globalization health subject area mobility technological globalization liberalization trade new global health threat bioterrorism addition BIG include multipleround Delphi Process Box solicit input issue broad interdisciplinary audience project purpose raise short term practical consideration globalization health andDelphi group communication technique design obtain opinion panel select expert specifi issue sending questionnaire complete specifi ed time expert contact individually know group participant opinion aim submit group participant condition participant meet personally avoid undue infl uence process foresee following point process repeat number time convergence group member obtain process end analysis answer formulation fi nal org Thomas Novotny Emilio Mordini Ruth Chadwick Martin Pedersen Fabrizio Fabbri Reidar lie Natapong Thanachaiboot provide long term strategic perspective select public health relate issue fi nal conclusion present high level meeting European Union EU policy maker conclusion inform future research direction stimulate additional critical thinking globalization bioethical implication health policy article present preliminary result BIG Project mobility result increase ease domestic international travel instantaneous access information internet electronic resource mobility involve pursuit well quality life development market trade good service return resource home country improvement professional business network migration affect psychological physical health result confl ict famine poverty insuffi cient cultural economic integration migrant new home society contribute spread infectious disease border figure recent epidemic SARS classic example infectious disease propagate movement people border require attention original site control migration quarantine vigilance secondary site protect population figure reason International Organization Migration increasingly concerned migratory pattern health consequence globalize world illustration emerge confl ict idea int iom border health commerce relate mobility Europe commerce likely increase EU enlarge include Eastern central european nation commerce include movement health provider East West medical tourism pursuit costly accessible high quality health care addition international trade illegal health product inconsistent regulatory safety standard export threaten public health especially unregulated pharmaceutical market ethical concern result vast growth international tourist travel travel account twelfth world trade support economy size middle income country tourism provide substantial economic benefi ts develop country improve cultural understanding traveler benefi ts require ethical concern environment person employ tourist industry right nation protect infectious disease unsafe medical practice right human being displace war traffi cking economic cultural disruption critical concern health policy maker poverty social disparity key factor growth global migration timely consider mobility human right migrate right health care new country question consider health policy maker ethical context individual autonomy social justice nonmalefi cence benefi cence technology drive globalization turn drive globalization considerable ambiguity value technological globalization especially health low income country digital divide important improve health income disparity electronic revolution provide scientist health worker developed develop world unprecedented access information reduce information inequity develop world collective international action new global governance mechanism need achieve goal information technology internet structural necessity fi nancial corporate globalization technology nongovernmental organization political group cultural movement support grassroots social justice human right campaign globalize corporation struggle advocate limitation expansion internet technology side need consider bioethical implication increase information access hand ethical issue surround genomic environmental human concern ambiguous signifi nt benefi ts identify genetically benefi cial product genetic determinant disease concern alter natural environment collect routine genetic information general population example expert assert genetically modifi ed GM crop signifi cantly increase crop yield require additional farmland preserve valuable rain forest animal habitat herbicide pesticide farmer allow trade save GM seed harvest terminator technology produce grain genetically modifi ed generate new crop development EnvIssues GEFood information technology ethical consideration distributive justice benefi cence consider debate global applicability GM crop pharmaceutical health care industry genetic testing provide information shape future market possible tailoring specifi pharmacotherapy genomic susceptibility government genetic testing provide predictive information population basis aid future health care planning genetic information similarly insurance industry identifi cation genetically high risk individual likely interfere autonomy able purchase health insurance example Apolipoprotein test indicate individual copy form allele gene lead Alzheimer disease information insurance company possible employer deny insurability despite current adverse health post genomic era potential reduce increase health inequity depend ethical issue address intervention increase life span access high quality health care compete expensive investment genetic research infectious disease affect poor health inequality amplifi ed access access health care research participant patient low income country unequal access information properly inform genetic testing counseling need adverse genetic information find population base genomic research characterize group people way encourage discrimination research lead dispute ownership genetic resource participant population health professional solid grounding bioethical issue clinical decision base genetic information health policy maker global governance structure accountable potential adverse consequence decision engender ask genomic science help develop nation extent benefi ts share pharmaceutical biotechnology company invest poor country money work therapeutic high income country concern bioethical issue social justice benefi cence arise Genomics potential global public good considerable uncertainty bioethical justifi cation culture general globalization help liberalize trade removal import restriction tariff removal restriction trade service linkage trade sanction protection intellectual property right activity impact population health defender trade liberalization claim process effective mean increase country wealth extension population health true specifi policy particularly detrimental effect health open market trade manufacture tobacco product ethical argument base social justice trade liberalization policy example trade liberalization rich poor country produce proportionally wealth rich country compare poor country suggest socially unjust result liberalization poor country economy grow fast rich country economy situation relationship wealth health actually somewhat controversial call Preston curve demonstrate dramatic relationship health economic prosperity Purchasing Power Parity capita year cheap cost effective approach population health vaccination clean water sewage disposal affect increase Purchasing Power Parity approach relatively important economic development se early intervention develop country likely important infl uence health develop country population today simple economic growth hand high intensity technological improvement economic growth important health rich country compare develop country concern intellectual property right trade extraordinarily contentious issue recent year new drug effective disease resourcepoor highly impact country antiretroviral drug HIV prohibitively expensive country patent protection Trade relate Intellectual Property agreement patent protection link trade policy country need cheap essential drug conform patent rule trade retaliation export country ensue restriction poor country response legitimate public health emergency unethical basis distributive justice nonmalefi cence benefi cence exception public health emergency HIV AIDS Trade relate Intellectual Property agreement include right compulsory licensing local company produce patented medicine exchange royalty payment patent holder parallel importing import patented drug sell cheaply essential medicine available highly impact country fear trade retaliation originate country General Agreement Trade Services relatively new treaty cover trade health service agreement severely criticize claim increase privatization health care service undermine public health care system give ambiguity actual impact agreement health sector largely determine way agreement specifi ed multinational commitment social justice equity benefi cence nonmalefi cence come play implementation treaty concern security biological weapon recently arise poor wealthy nation question enormous sum spend address perceive threat bioterrorism strong evidence actual threat evidence global bioethical principle suggest need framework consideration distributive justice arena example nation limited supply vaccine weaponized smallpox offer stockpile neighboring country direct attack case complicate fact infection spread territory case widespread biological attack global govern agency country entity responsible global resource allocation clearly risk bioweapon trans border resource unevenly inequitably distribute require bioethically base policy determination global basis concern respect biomedical research issue dual use technology development health benefi ts possible bioweapon government balance secrecy necessary security need disclosure information essential research development health diffi cult sequester new knowledge apply build biological weapon simultaneously impede research defense bioweapon benefi cial biomedical advance BIG Project scientist agree benefi ts release scientifi information general outweigh risk misuse scientifi community need consider new code conduct necessary exist governance suffi cient support bioethical approach research possible dualuse technology global bioethical challenge require careful theoretical deliberation practical consideration international health policy BIG Project seek guide process select area interest EU project result helpful policy maker local national international level BIG Project find bioethical principle important consideration migration trade information technology genomic bioweapon threat globalization arena right wrong process demand careful consideration bioethical principle include social justice benefi cence nonmalefi cence individual autonomy concern immediately obvious health policy maker BIG Project result help clarify large goal purpose bioethically base health policy development EU information BIG Project find','2006-01-24 00:00:00','http://dx.doi.org/10.1371/journal.pmed.003004',1519,715),('c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json','Association and Host Selectivity in Multi-Host Pathogens','The distribution of multi-host pathogens over their host range conditions their population dynamics and structure. Also, host co-infection by different pathogens may have important consequences for the evolution of hosts and pathogens, and host-pathogen co-evolution. Hence it is of interest to know if the distribution of pathogens over their host range is random, or if there are associations between hosts and pathogens, or between pathogens sharing a host. To analyse these issues we propose indices for the observed patterns of host infection by pathogens, and for the observed patterns of co-infection, and tests to analyse if these patterns conform to randomness or reflect associations. Applying these tests to the prevalence of five plant viruses on 21 wild plant species evidenced host-virus associations: most hosts and viruses were selective for viruses and hosts, respectively. Interestingly, the more host-selective viruses were the more prevalent ones, suggesting that host specialisation is a successful strategy for multi-host pathogens. Analyses also showed that viruses tended to associate positively in co-infected hosts. The developed indices and tests provide the tools to analyse how strong and common are these associations among different groups of pathogens, which will help to understand and model the population biology of multi-host pathogens.',' pathogen highly variable host range natural condition infect related specie specialist pathogen infect wide range host belong different taxonomic group multi host generalist pathogen large fraction describe pathogen human animal plant generalist ability infect different host condition epidemiology pathogenicity generalist pathogen highly relevant pathogen management disease control distribution multihost pathogen host range frequency infection host specie ecosystem vary largely determine population dynamic structure pathogen distribution pathogen specie host range determine important aspect biology host significant anthropocentric viewpoint target host reservoir inoculum source emergence reemergence population threshold disease invasion critical community size disease persistence plant specie host range pathogen host population encounter large number different pathogen specie significant host specie abundant evidence difference infection frequency pathogen specie present ecosystem distribution pathogen host distribution different pathogen host specie affect frequency multiple infection individual host different pathogen multiple infection important consequence infected host pathogen host pathogen coevolution host frequent co infection lead heterozygote superiority multiple pathogen contribute persistence host population allele confer susceptibility disease multiple infected host pathogen cooperate compete host resource affect fitness multiple infection factor pathogen evolution theoretical analysis predict withinhost dynamic microparasite multiple infected host important consequence evolution virulence evidence multiple infection result increase reduced virulence multiple infection host directly affect genetic diversity pathogen population co infection prerequisite genetic exchange different pathogen specie strain infection pathogen result increase host susceptibility second pathogen common phenomenon name facilitation predisposition animal plant pathologist respectively spite potential impact pathogenicity evolution epidemiology control distribution pathogen host range occurrence co infection largely overlook research pathogen ecology epidemiology deal specific pathogen host interaction knowledge analyse distribution pathogen host range random alternatively association pathogen host occur address host co infection different pathogen random association pathogen occur particular host address issue propose index observe pattern host infection different pathogen observe pattern coinfection test analyse conform null hypothesis randomness reflect association second apply test datum prevalence insect bear virus specie wild plant specie agroecosystem Central Spain result analysis uncover pattern general highly relevant understand ecology evolution pathogen TSWV single strand RNA genome negative ambisense polarity virus single strand RNA genome messenger polarity AMV CMV WMV transmit aphid non persistent manner virus retain distal structure aphid mouth part short period time BWYV transmit circulative non propagative manner virus penetrate gut wall haemocoel insect vector circulate haemolymph reach salivary gland inoculate new plant tswv follow similar path thrips body infect multiplie insect cell virus cause important disease vegetable crop worldwide include studied region Central Spain infection analyse wild host asymptomatic table show number sample analyse number infected plant virus specie single multiple infection frequently find plant specie monitored habitat method analyse period datum set test association host pathogen method apply index selectivity pathogen ISP significance show Table virus distribution analyse virus host significantly non random available host preferentially infect fig show relationship prevalence isp virus positive correlation find parameter spearman rank correlation test host selective virus high prevalence analyse ecosystem similarly index selectivity host ISH significance calculate host plant specie Table value show Table half analyse host Amaranthus spp Cirsium arvense Convolvulus arvensis Diplotaxis erucoide Lactuca serriola Medicago sativa Portulaca oleracea Solanum nigrum Taraxacum spp difference prevalence virus depart significantly random fig show relationship virus prevalence ISH host specie positive correlation parameter find spearman rank correlation test virus selective host high prevalence virus infection relationship prevalence selectivity virus host coincidence frequency infection host different virus show contingency analysis count infected host different virus plant specie Table co infection virus occur plant specie plant infect virus analyse plant table describe test association pathogen apply set datum Table show tendency analyse virus associate positively distribution association index AI skew positive value Fig ai compute virus plant specie third positive negative conspicuous tendency positive AI value small probability spearman rank correlation test pooled sample sixteen plant specie consider AI positive significantly different zero virus virus find co infection frequency significantly high expect null hypothesis independence infection datum sixteen plant specie analyze separately association analysis uncover pattern obvious general tendency analyse virus associate positively ii association depend plant virus specie effort understand population biology pathogen focusse specialist pathogen population biologist successfully develop formal understanding dynamic evolution single host pathogen pathogen human animal plant multi host pathogen state Woolhouse et al understand complex population biology multi host pathogen major challenge century evidence ecosystem prevalence multi host pathogen differ largely different specie host range similarly evidence large difference prevalence host specie pathogen able infect attempt knowledge analyse difference distribution multihost pathogen host random association host pathogen uncovering association host pathogen highly relevant understand model population biology multi host pathogen understand phenomenon generalism present index test analyse association multi host pathogen host propose index selectivity pathogen host measure degree association host pathogen test analyse homogeneity distribution pathogen different host specie population different pathogen host analyse significantly value index depart zero association literature pathogen ecology abound datum prevalence pathogen host apply index unpublished datum prevalence insectborne plant virus specie wild plant agroecosystem central Spain year period analysis prevalence different virus host specie homogeneity test propose show half analyse plant specie show index selectivity host ISH significantly different zero distribution host species show virus selectivity relate taxonomy habitat fallow field edge wasteland seasonality vegetative cycle annual perennial show interestingly positive correlation ISH average virus prevalence host plant specie show selective host prone virus infect obviously well infect phenomenon suggest spite host encounter wide array pathogen mechanism escape resistance operate explain selectivity fact contingency analysis count infected host different virus suggest different virus specialise different host analysis homogeneity prevalence virus host specie show analyse virus significant host association value index selectivity pathogen isp significantly depart form zero major unexpected finding analysis positive highly significant correlation value isp prevalence virus value isp condition number host plant specie infect virus correlation spearman rank correlation test isp number plant specie virus infect analyse system selective virus infect small number plant specie host selective virus well analyse ecosystem result highly relevant understand evolution generalism pathogen describe pathogen generalist advantage generalism poorly understand generalist strategy provide pathogen opportunity transmission survival predict evolution favour specialism pathogen host co evolution result functional trade off limit generalist fitness host result compatible hypothesis specialism advantageous pathogen host selectivity rule analyse set generalist virus host selective virus successful strategy result suggest generalist pathogen degree host specialisation host selectivity define successful strategy host specialisation generalist pathogen relevant important issue host pathogen biology host specialisation affect hostpathogen co evolution co speciation reduce opportunity host switch jump constrain evolution host expansion result spatial heterogeneity host favour stable maintenance pathogen host diversity addition host specialisation affect opportunity different pathogen share host consequence multiple infection pathogen host evolution discuss propose simple procedure estimate association pathogen enable compute association index significance test null assumption independence infection follow binomial distribution test apply datum set second major contribution analysis finding co infection non random association analyse virus positive result relevant co infection different pathogen important consequence pathogen infect host hostpathogen co evolution virus co infection host result generation new genotype recombination reassortment genomic segment different viral specie strain dramatic change host range pathogenicity classical example reassortment avian human strain influenza result novel virus pandemic potential example abound animal plant virus individual host coinfection lead aggravate disease result extracellular cooperativity independently replicate virus virus modulate host response infection benefit addition direct interaction different virus co infected cell result complementation highly pathogenic defective genotype increase virus replication modify cell tissue tropism alternatively evidence mixed infection pathogen result reduce pathogenicity severe disease example virus include mixed infection satellite defective interfere nucleic acid datum set association virus depend particular virus host system datum suggest host coinfection advantageous virus underlie mechanism remain analyse analysis report plant virus infection weed uncover major feature relevant understand population biology virus hostselective virus well analyse ecosystem ii virus tend associate positively co infected host high interest know general feature type pathogen occur index test propose general use analysis ecology pathogen hope result prompt research ecology pathogen host pathogen pathogen association analysis uncover pathogen property relevant formal understanding population biology multi host pathogen study factor relative distribution pathogen different host different host population genotype specie etc association pathogen host association pathogen analyse factor propose follow test index association pathogen host let number analyse individual host ik number individual infect pathogen prevalence pathogen host ratio ik ik average prevalence pathogen host beconversely average prevalence different pathogen host define asHomogeneity prevalence pathogen host test mean contingency table element ik ik different proportion lack homogeneity indicate property pathogen selectivity selectivity measure Cramer coefficient contingency contingency table chisquared table index selectivity pathogen index range zero zero mean equal prevalence pathogen host pathogen host selectivity pathogen host association different pathogen let Xs ik Xa ik number analyse individual host infect pathogen single infection pathogen associate infection respectively ik Xs ik Xa ik frequency pathogen host estimate equal define prevalence null hypothesis independence infection different pathogen probability sampled host individual infect pathogen conditional probability non infection pathogen give presence pathogen ps ik jk conditional probability observe multiple infection give presence hypothesis independence infection different pathogen non association pathogen Xs ik distribute binomial ik trial probability ps ik define association index AI pathogen host difference proportion sample infect pathogen infect pathogen Xa ik ik minus expectation proportion null hypothesis pa ik index range minus expect value null hypothesis zero significance observation estimate onetail test binomial test association different pathogen host give pathogen different host follow process expectation sum observation equal sum expectation correspond sum observation binomially distribute give ik single significant test group raw significance probability correct sequential Bonferroni method multiple independent test indicate sample monthly year horticultural area central Spain habitat characterise different degree human intervention fallow field edge field wasteland plant sample systematically fix itinerary consideration symptom expression describe Sacristán et al infection AMV BWYV CMV WMV TSWV sample plant analyse double antibody sandwich enzyme link immunosorbent assay DAS ELISA commercial antisera Bio rad Marnes La Coquette France accord manufacturer instruction distribution host species show virus selectivity accord taxonomy habitat fallow field edge wasteland seasonality vegetative cycle annual perennial analyse chi square test contingency table significance assess rest test work simulation follow model III','2006-12-20 00:00:00','http://dx.doi.org/10.1371/journal.pone.000004',1669,514),('c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json','Transmission Parameters of the 2001 Foot and Mouth Epidemic in Great Britain','Despite intensive ongoing research, key aspects of the spatial-temporal evolution of the 2001 foot and mouth disease (FMD) epidemic in Great Britain (GB) remain unexplained. Here we develop a Markov Chain Monte Carlo (MCMC) method for estimating epidemiological parameters of the 2001 outbreak for a range of simple transmission models. We make the simplifying assumption that infectious farms were completely observed in 2001, equivalent to assuming that farms that were proactively culled but not diagnosed with FMD were not infectious, even if some were infected. We estimate how transmission parameters varied through time, highlighting the impact of the control measures on the progression of the epidemic. We demonstrate statistically significant evidence for assortative contact patterns between animals of the same species. Predictive risk maps of the transmission potential in different geographic areas of GB are presented for the fitted models.',' FMD epidemic UK substantial cost human animal health economic term Alexandersen et al Kao understand risk factor underlie transmission dynamic epidemic evaluate effectiveness control measure essential minimise scale cost future outbreak epidemic modelling prove critical decision make control policy case controversially adopt control epidemic model peace time contingency planning role weakness model study undertake relatively ad hoc nature parameter estimation method employ paper Ferguson et al maximum likelihood method fit observe incidence time series attempt fit spatio temporal pattern spread later work author develop robust method estimate species specific susceptibility infectiousness parameter spatial kernel parameter Supplementary information time statistical basis method develop lack retrospect method develop turn closely relate develop SARS epidemic Wallinga Teunis early work incorporate population denominator datum allow spatial species base heterogeneity disease transmission method employ limitation fully parametric mean extend fit arbitrary transmission model observe datum keel et al maximum likelihood method estimate transmission parameter supplement ad hoc square matching regional incidence time series remain need develop rigorous modern statistical approach parameter estimation non linear model FMD outbreak Bayesian Markov Chain Monte Carlo MCMC technique well establish method successfully employ analysis range spatiotemporal outbreak datum past purely temporal incidence datum develop MCMC base inference model FMD epidemic GB model examine extent transmission spatially localise temporal variation transmission species specific variation susceptibility infectious ness heterogeneity contact rate specie farm unit study ignore possible impact farm epidemic dynamic implicitly assume disease spread farm rapid practically instantaneous animal farm infectious time datum consist information farm UK list agricultural census uk footandmouth case total farm list dataset uniquely identify County Parish Holding CPH number spatial coordinate provide number animal specie farm partition GB farm accord animal type represent show figure geographical distribution represent figure number farm km notice high density area North West Cumbria South West Devon Wales Scotland main epidemic focus develop area high density Shetland Islands correspond small crofter smallholding figure number sheep cattle keep km square FMD outbreak total infected premise ip record FMD diagnose subsequently cull IP dataset contain farm estimate date infection determine clinical evaluation age lesion affected animal date disease reporting confirmation culling total non ip farm cull contiguous premise cp dangerous contact dcs local culling policy Cumbria Scotland instance total sheep farm Cumbria sheep cull local km radial sheep cull policy adopt farm record dc cp multiple record farm find disease control management system dataset unclear datum entry error result sequential specie specific cull farm analysis consider farm cull record date culling frequent specie cattle sheep Figure farm pig farm pig diagnose ip ip indicate priori pig contribute far outbreak FMD outbreak despite high level shed decide discard pig farm current study simplify analysis Sensitivity Analysis section show simplification significantly affect estimate epidemiological parameter discard ip miss information possible mistake location number animal leave total ip analyse dataset model epidemic space time survival process total observation time day th February th October farm location associate infection time infect removal time slaughter integer represent respectively number cattle sheep farm represent capita cattle sheep susceptibility respectively represent capita cattle sheep infectivity susceptibility relative measure animal sensitivity disease infectivity represent infectious risk pose animal use continuous kernel describe probability contact farm scale distance transmission naturally assume decrease distance farm accord power lawwhere ij represent euclidian distance infected farm susceptible farm parameter kernel offset kernel power estimate kernel capture form movement contact farm use simple parameter function inevitably highly simplified representation true complexity inter farm contact examine functional form kernel analysis result model fit poor find power law kernel give susceptibility infectiousness parameter kernel infection hazard infected farm susceptible farm quantify model specify state arbitrarily assume mean represent ratio cattle tosheep susceptibility constant distance independent kernel mass action closed epidemic model heterogeneous susceptibility infectiousness model assume susceptibility infectiousness parameter scale linearly number animal different specie farm relatively strong assumption impose model parsimony reason mixing matrix embed quantify specie specific mixing rate animal different farm cattle cattle sheep cattle cattle sheep sheep sheep model formulation identical Keeling et al functional form kernel force infection susceptible farm time depend history event justwhere farm susceptible farm infectious time otherwiseBy default assume latent period day latency represent function farm infectious day infect test sensitivity estimate assumption examine latent period day probability density function farm infect time give byHence contribution farm observe infect time make log likelihood farm infect contribute overall likelihood probability escape infection observation period time cull duration epidemic whichever short contribution log likelihood thereforethe total log likelihood model write asWe extend simple model introduce additional parameter understand extent transmission specie alter specie transmission parameter quantifie degree mix specie assortative represent assortative mixing disassortative mixing interaction model assume constant parameter respect time observation period mixing matrix define equation rS rS fix avoid model specification force infection model log likelihood equation change accordingly assume transmission parameter constant time epidemic obviously crude simplification allow infectivity vary continuously time result specify model problem parameter identification confound examine set model change transmission parameter restrict significant point time denote cut rd February national ban animal movement introduce st March control measure intensify call hour IP CP culling policy introduce model respectively fit individual case datum start epidemic conditioning infection February conditioning farm infect date detailed history epidemic give Kao separately fit model variant assume discrete change parameter rd February st March confounding mean limited number parameter varied time examine effect vary infectiousness kernel parameter separately fit separate time vary model variant vary cattle infectivity factor keep sheep infectivity constant time Cattle Infectivity model ii vary sheep infectivity factor cattle infectivity Sheep Infectivity model iii vary cattle sheep infectivity ratio Cattle Sheep Infectivity model iv vary kernel parameter Time Varying Kernel model model variant fit version include non assortative mixing specie equation general mathematical expression transmission model wherethe script pre post self explanatory time vary parameter fitting model time vary infectivity parameter actually fit post ratio pre post call infectivity factor species ratio parameter directly fit model unlike specie infectivity ratio additionally calculate explain later text Parameter estimate section note model treat epidemic fully observe infection time assume know fact estimate infection time know Sensitivity Analysis section ip assume infectious adopt bayesian framework statistical inference use MCMC method fit model individual case datum strictly necessary give simplify assumption epidemic completely observe provide consistent robust framework relax assumption future work obtain parameter estimate equal tail credible interval marginal posterior distribution fit parameter basic model instance estimate relative cattle susceptibility infectivity parameter kernel parameter post pre post pre posterior mean deviance bayesian measure fit model adequacy define Spiegelhalter et al posterior density deviance define log likelihood function observe data vector give parameter vector constant need know model comparison purpose function datum small mean posterior deviance well correspond model fit datum posterior deviance distribution different model overlap significantly necessary use additional criterion compare model fit comparison relative complexity model Deviance Information Criterion DIC general method generalisation Akaike information criterion bayesian hierarchical model define complexity model effective number parameter define aswhere represent take expectation posterior DIC define asA low value DIC correspond well model criterion offer flexibility compare non nested model straightforwardly compute MCMC algorithm apply classic random walk Metropolis Hastings algorithm block sampling parameter computationally expensive form likelihood log scale sample parameter positive definite expect potentially vary order magnitude linear scale sampling yield similar result convergence chain improved Robert perfect sampling reparameterization issue compare sample linear scale model code parallelize OpenMP MCMC sampler allow equilibrate convergence evaluate visually likelihood parameter trace simple model iteration sufficient equilibration increase complex model log scale sampling verify chain able converge start initial parameter value far final posterior mean value posterior distribution estimate iteration rate acceptance vary model model baseline model achieve rate acceptance complex model parameter rate approx value compare golden acceptance rate Random Walk Metropolis Hastings Roberts encounter common problem MCMC estimation like slow convergence slow mixing correlation parameter have biological explanation cattle sheep infectivity instance careful parameterization lower verify parameter estimate dependent parameterization choice difference see fit specie infectivity individually fit sheep infectivity ratio cattle sheep infectivity table list parameter estimate obtain set fit model condition infection table present estimate model condition infection occur rd February posterior deviance set model plot Figure figure respectively figure illustrate clear conclusion model time variation parameter interaction model fit significantly well baseline model heterogeneous mixing specie fit interaction model broaden credible interval infectivity parameter estimate table indicate unsurprisingly slight confound infectivity susceptibility parameter model allow infectivity vary rd February allow cattle infectivity variation give slightly well fit vary sheep infectivity model parameter vary rd February model variant allow kernel parameter vary time point fit substantially well deviance dic criterion Table allow species specific variation infectivity encouraging inference procedure main control measure initiate date banning Figure figure parameter estimate precise rd February table relatively small number ip date look complex model interaction model time vary kernel cattle estimate susceptible sheep Figure table mention animal specific infectivity figure Table informative comment cattle sheep infectivity ratio parameter complex fit ratio dot appear table model parameter calculate MCMC algorithm ratio specie infectiousness sample parameter point complex model suggest cattle fold infectious sheep figure parameter quantify assortativity mixing estimate level mix specie random Figure comparison model time vary kernel random mixing specie effect heterogeneous mixing specie modify species transmission give matrix indicate Cattle cattle sheep sheep transmission high respectively model non random mixing sheep cattle cattle sheep transmission drop respectively condition rd February model variant consider table figure examine baseline interaction model change parameter time allow cattle infectivity vary st March cattle sheep infectivity vary factor st March heterogeneity mixing allow kernel parameter vary st March unsurprisingly kernel parameter significantly different allow different st March model prove good fit overall variation mean deviance figure see model variant small model condition infection figure interaction model allow time vary cattle infectivity give adequate fit measure mean deviance DIC Table statistically compare set model Table Table datum different case parameter estimate well fitting model table largely consistent rd February estimate value good fit model Table include correspond st March credible interval good fit model table vice important message second set model model cattle time vary infectivity good fit indicate high value infectivity st March table paradoxical reflect fact cull effect explicitly include input datum dramatically reduce case incidence April September case incidence maintain low level entirely cattle farm increase cattle infectivity reflect impact reduce biosecurity increase non compliance movement control informative examine parameter estimate imply term geographic variation transmission potential give parameter estimate model define relative risk transmission infectious farm pose susceptible farm country Ic zi Sc Ss zn sThis quantity multiply average duration infectiousness farm time end latency culling give reproduction number farm divide UK km square calculate average transmission risk farm square local figure show geographic risk change rd February good fit model condition infection kernel shape major influence average risk distribution country figure show correspond risk map estimate infer good fit model condition rd February slightly high risk predict st March model condition rd February increase cattle infectivity date risk estimate rd February set model appear consistent obtain model condition rd February rigorous statistical comparison appropriate strong assumption study infected farm epidemic report ip farm infect cull clinical diagnosis responsible cause infection interesting calculate proactively cull farm model predict infect definition calculate probability particular proactively cull farm infect need adjust infection hazard probability farm report clinical case culling date outbreak datum calculate probability density time infection report reported ip cumulative probability distribution time infection report denote force infection proactively cull farm time good fit model condition rd February probability farm gets infect escape report potential infection time cull time isWe calculate expect number infection different class dc CPs proactively cull farm cull particular time interval instance expect number cp cull time predict infect formally write asthis simplification reality delay infection report certainly depend size species mix farm result indicative expect level infection proactive culling stage calculation culling non informative censoring process reasonable assumption proactively cull farm dc definition identify veterinarian having high risk exposure method underestimate infection rate calculate infection report delay distribution divide epidemic rd February time period rd st March st st st th October interval total farm slaughter respectively good fit model condition rd February predict different infectivity regime st March Parameter Estimates Table split second period time different delay reporting cull infection report delay day period time respectively infection cull delay drop day respectively apply approach interaction model time vary cattle infectivity condition rd February calculate expect proportion proactively cull farm infect estimate approximately cull non ip farm infect total figure farm cull rd February st March infect farm farm cull st March st estimate infect farm period st th October estimate farm cull infect proportion cp estimate infect equate farm figure epidemic estimate farm designate dc infect farm estimate Figure allow high risk infection imply veterinary judgement lead dc identify mean high proportion fact infected assume dc time likely infect status model predict incidence infection dcs go accordingly farm farm cull dc cp typically cull km local sheep cull policy Cumbria Dumfries Galloway area low estimate rate infectiona mere farm section examine sensitivity result number factor leave pig analysis possible error estimated ip infection date assumed latent period justify simplification analysis discard number pig farm present detailed statistic variable fit simple model condition farm exclusively pig farm denote number pig farm pig susceptibility pig infectivity respectively simple model similar condition infection fit reduce number parameter manner addition estimate pig sheep susceptibility ratio pig infectivity assume parameter constant time find cattle sheep susceptibility ratio table show parameter estimate cattle sheep largely unaffected ignore pig population estimate analysis significantly different conclude include pig change conclusion present Table cattle sheep give small number ip pig decrease power analysis increase model complexity understand extent estimate affect assumption infection date accurately observe randomize estimate infection date add Gaussian noise zero mean standard deviation day motivate substantial proportion observed standard deviation equal day distribution time estimate infection date report date ip fit simple model condition infection rd February randomised dataset average estimate give Table lie confidence interval predict Table average cattle sheep infectivity ratio close value estimate original datum average estimate randomized dataset appropriate model condition rd February cattle infectivity interaction model Table value present Table assess sensitivity analysis estimate proportion infection proactively cull farm previous section respect infection time predict parameter dataset calculate average proportion category proactively cull farm average proportion infection DC farm period time respectively CP farm quantity evaluate respectively overall proactively cull farm obtain average percentage consider period time value predict original datum previous section figure result present assume fix latent period day test sensitivity parameter estimate assumption examine latent period day overall expect infectiousness parameter increase compensate short infectious period slightly increase generation time mean time infection case time infection case case generate interestingly kernel parameter estimate alter latent period varied kernel slightly local increase latent period day latent period pre rd February value drop table paper present statistical analysis spatiotemporal evolution foot mouth outbreak GB qualitatively result agree obtain Keeling et al identify cattle key specie epidemic interaction model condition rd February time vary cattle infectivity estimate ip rd st March infect cattle sheep sheep sheep transmission account ip period st March estimate cattle infectivity increase slightly table allow non random mixing specie indicate contact farm assortative basis specie composition farm like specie mix like agree intuition nature farming practice sharing personnel equipment likely common farm livestock species implication moderate degree assortativity find control measure remain explore use datum collect epidemic trace contact farm fix spatial kernel function analysis final version FMD epidemic data warehouse contact apparently identify early epidemic remain confirm share concern early work distribution contact distance trace contact bias estimate kernel function offset power law functional form high value kernel power parameter estimate rd February figure consistent expect dramatic shortening typical contact distance follow national movement ban localize spread high estimate level infectivity cattle st March explain long tail epidemic see estimate transmission risk farm assume dependence euclidian distance reality metric time require travel farm reasonable examine future work include information landscape height sea level location river tree estimate risk map figure Figure match area country high case incidence rate see notable exception Wales discrepancy high predict risk Wales small number case observe reflect inaccuracy input datum set et al reduce farm level sheep population number Wales obtain well geographic match datum Matt Keeling personal communication discrepancy reflect model inadequacy allow farm level risk factor farm fragmentation index consider Ferguson et al explore complex non linear model dependence susceptibility infectiousness number animal farm relax implicit assumption contact rate farm scale linearly local density farm assumption relax ongoing work important issue revise future work allow proactively cull farm diagnose ip potentially infect infectious farm require modification inference model allow arbitrary number unobserved infection low number proactively cull farm estimate infect suggest effect model refinement limit note infection prevalence estimate result relatively non local kernel estimate simultaneously kernel estimate change refined analysis dc attribute high risk infection estimate status possible estimate infection rate dcs proactively cull farm increase somewhat factor increase estimated infection prevalence proactively cull farm fold unlikely ongoing work mean small proportion dc cp cull infect imply proactive culling effect epidemic large expect effect culling targeted depletion susceptible animal regard proactive culling epidemiological impact vaccination future work revisit past estimate exactly important culling control FMD epidemic','2007-06-06 00:00:00','http://dx.doi.org/10.1371/journal.pone.000050',2874,790),('c6bf372c094f035a514975c35a7f9c094abbe493.json','Sequence specific visual detection of LAMP reactions by addition of cationic polymers','BACKGROUND: Development of a practical gene point-of-care testing device (g-POCT device) requires innovative detection methods for demonstrating the results of the gene amplification reaction without the use of expensive equipment. We have studied a new method for the sequence-specific visual detection of minute amounts of nucleic acids using precipitation reaction by addition of cationic polymers to amplicons of Loop mediated isothermal Amplification (LAMP). RESULTS: Oligo DNA probes labeled with different fluorescent dyes were prepared for multiple nucleic acid templates, and the templates were amplified by the LAMP reactions under the existence of the probes. At completion of the LAMP reaction, an optimal amount of low molecular weight polyethylenimine (PEI) was added, resulting in the precipitation of the insoluble LAMP amplicon-PEI complex. The fluorescently labeled Oligo DNA probes hybridized to the LAMP product were incorporated into the precipitation, and the precipitate emitted fluorescence corresponding to the amplified nucleic acid templates. The color of emitted fluorescence can be detected easily by naked eye on a conventional UV illuminator. CONCLUSION: The presence or absence of minute amount of nucleic acid templates could be detected in a simple manner through visual assessment for the color of the LAMP amplicon-PEI complex precipitate. We conclude that this detection method may facilitate development of small and simple g-POCT device.',' loop mediate isothermal amplification LAMP unique gene amplification method DNA isothermally amplify enzyme advent LAMP method researcher engage basic research variety perspective result currently practical use reagent detect pathogen SARS West Nile virus reagent identify sex fertilize egg cow vitro fertilization furthermore LAMP gene amplification method variety characteristic application wide range field include SNP type quantification template DNA particular LAMP consider effective gene amplification method use gene point care testing POCT device simple genetic testing necessary LAMP amplify gene isothermally amplification reaction carry simple heater need special device Pattern diagram LAMP reaction hybridization fluorescently label oligo DNA probe Figure Pattern diagram LAMP reaction hybridization fluorescently label oligo dna probe LAMP reaction take place step start material production step cycling amplification step elongation recycling step primer depict enclosure starting material production step starting material generate primer forward inner primer FIP backward inner primer BIP complementary strand starting material synthesize starting material reaction use template reaction FIP anneal loop segment make cycle amplification step step probe probe probe respectively design region region region hybridize loop segment cycle reaction progress elongation recycling step take place elongate product etc inverted repeat structure generate number cauliflower structure generate loop structure probe polymerase chain reaction PCR rapidly control temperature large dna µl synthesize short time min maintain high specificity characteristic greatly facilitate detection LAMP reaction LAMP reaction progress generate characteristic stem loop structure LAMP product single strand segment molecule loop segment figure reference oligo DNA probe design recognize sequence single strand segment possible carry hybridization assay perform heat denaturation amplification mean process amplification reaction detection reaction carry completely isothermally characteristic LAMP method effectively believe possible develop simple genetic testing device realize despite strong awareness necessity wide range field include infectious disease testing food inspection environmental testing key develop simple device figure simply clearly present final amplification result objective research establish new technique sequence specific visual detection amplification result mean LAMP method end use reaction know long time cationic polymer like polyamine form insoluble complex DNA known polyamine polyethylenimine PEI strongly interact DNA PEI widely nucleic acid precipitant nucleic acid purification vivo vitro non viral vector discover insoluble PEI LAMP product complex generate certain optimize condition PEI add LAMP reaction solution precipitation titration investigate possible perform free separation fluorescently label probe paper report novel visual detection method presence sequence HBV HCV clone plasmid model experiment method result investigation basic reaction condition require achieve primer LAMP reaction schematically depict enclosure figure Forward Inner Primer FIP consist complementary sequence Backward Inner Primer BIP contain complementary sequence sequence define template sequence show Figure reference Notomi et al Parida et al spacer thymidine insert inner primer FIP BIP thymidine spacer insert FIP BIP HBV respectively spacer study LAMP reaction progress use inner primer spacer show Hong et al Hirayama et al Iwasaki et al LAMP reaction take place isothermally step show Figure start material production step cycling amplification step elongation recycling step polymerase strand displacement activity starting structure structure generate template nucleic acid starting material production step starting structure structure stem loop structure self prime dna synthesis forward inner primer hybridize loop segment strand displacement synthesis take place structure complementary strand structure generate mean auto cycle reaction establish structure structure addition product bind inverted repeat amplified region like structure generate association auto cycle reaction cycling amplification step structure start point product elongate length kbp product complex structure cauliflower like structure ultimately generate LAMP product loop structure oligo dna probe green arc figure reaction solution sequentially hybridize product LAMP reaction proceed cauliflower structure particular contain loop probe hybridize characteristic LAMP reaction play important role detection method describe Plasmid DNA clone HBV HCV sequence add LAMP reaction solution contain HBV primer probe FITC label HCV primer probe ROX label amplify low molecular weight pei Mw µmol monomer add show Figure precipitate emit green fluorescence characteristic FITC obtain LAMP reaction solution contain HBV template precipitate emit red fluorescence characteristic ROX obtain LAMP reaction solution contain HCV template precipitate emit color orange combination FITC green ROX red fluorescence obtain LAMP reaction solution contain template observe ordinary uv illuminator uv LED light emitted diode LAMP product lambda dna relate HBV HCV present precipitate fluore sequence specific visual detection method utilize precipitation titration LAMP product add PEI Figure sequence specific visual detection method utilize precipitation titration LAMP product add PEI result sequence specific visual detection add PEI LAMP reaction solution LAMP reaction presence FITC label HBV probe ROX label HCV probe follow addition prescribed µmol monomer PEI Mw centrifuge second desk low speed centrifuge tube visually observe uv illuminator nm possible differentiate LAMP reaction visualize presence precipitate fluorescence color fluorescence LAMP reaction negative LAMP reaction psa amplification unrelated LAMP reaction occur contain HBV template nucleic acid contain HCV template nucleic acid contain HBV HCV template nucleic acid Diagram principle sequence specific visual detection method utilize precipitation titration LAMP product add PEI LAMP reaction carry LAMP primer set type template nucleic acid fluorescently label probe hybridize loop segment LAMP product LAMP reaction correspond certain fluorescently label probe progress probe sequentially hybridize loop segment generate reaction hand unrelated probe remain free solution optimize PEI add reaction set length time positive charge PEI neutralize negative charge DNA form insoluble LAMP product pei complex stage fluorescently label probe hybridize LAMP product take LAMP product pei complex LAMP product PEI add formation LAMP product pei complex free oligo dna probe form complex PEI generate insoluble complex pelletize centrifugation pellet irradiate excitation light label fluorescent dye hybridize LAMP product produce fluorescence figure Precipitation titration LAMP product addition PEI effect PEI sequence specific incorporation ROX label lambda dna recognition probe DNA PEI complex Mw PEI µmol µmol PEI add monomer label probe hybridize LAMP product lambda DNA take DNA PEI complex hand µmol µmol PEI add monomer reaction solution amplification reaction place small label probe precipitate precipitation nonspecific probe place unrelated LAMP product PSA present effect Mw PEI sequence specific incorporation ROX label lambda dna recognition probe DNA PEI complex PEI Mw fluorescence intensity supernatant decrease LAMP product lambda DNA present Mw PEI increase fluorescence intensity supernatant decrease absence LAMP reaction formation insoluble PEI oligo dna probe complex occur effect KCI sequence specific incorporation ROX label lambda dna recognition probe LAMP product PEI Mw complex normal LAMP reaction solution control contain mm KCI formation PEI LAMP product complex inhibit increase KCI add fluorescence intensity supernatant increase regardless sequence LAMP product effect LAMP product lambda dna sequence specific incorporation ROX label lambda dna recognition probe LAMP product pei complex Mw PEI LAMP product µg µl label probe pmol take PEI DNA obtain visible precipitate generate sample contain template LAMP reaction negative mean possible assess HBV template nucleic acid reaction solution HCV template nucleic acid reaction solution reaction solution visualize fluorescent color precipitate LAMP method nucleic acid amplification method sensitive possible create amplification reaction copy template nucleic acid combination LAMP amplification detection method make possible sequence specific visual presentation presence trace amount nucleic acid copy find sample model represent principle method show figure oligo DNA probe label fluorescent dye hybridize specific LAMP product optimize low molecular weight pei Mw add positive charge PEI neutralize negative charge DNA result formation insoluble dna PEI complex solution leave stand minute centrifuge small desk centrifuge second complex deposit tube precipitate observe illuminator nm fluorescence dye probe take precipitate visualize hand probe unrelated sequence amplified LAMP product take LAMP PEI complex hybridize LAMP product molecular weight PEI small oligo dna probe PEI interact sufficiently form insoluble complex unrelated probe remain supernatant fluorescent probe supernatant disperse fluorescence visualize consequentially bind free separation label oligo dna achieve result insolubilization PEI LAMP product follow experiment conduct confirm principle investigate effect add PEI precipitation titration Figure oligo DNA probe lambda DNA capture precipitate µmol PEI add µl LAMP reaction solution lambda dna PEI add high low relative optimal range PEI DNA complex precipitate form result oligo DNA probe remain solution phenomenon characteristic ionic interaction cationic polymer anionic polymer cationic polymer PEI low PEI DNA complex anionic contrast PEI high PEI DNA complex cationic case PEI DNA complex solubilize result characteristic show Figure indicate precipitation titration base neutralization negative electric charge DNA cationic polymer PEI case LAMP negative small free probe deposit certain condition µmol µmol PEI add monomer precipitate case small impossible confirm visually hand LAMP product PSA unrelated probe sequence present free probe deposit PEI molecule add consume precipitation unrelated LAMP product excess present relative probe word reliable detection achieve confirm LAMP reaction occur check white DNA PEI complex precipitate generate result addition PEI determine nucleic acid template LAMP reaction occur base fluorescent color precipitate result PEI add fix µmol follow experiment order avoid generation free probe precipitate possible investigate effect Mw PEI detection system figure PEI different Mw add LAMP solution monomer find sufficient BF separation occur LAMP amplicon present Mw PEI increase fluorescence intensity supernatant LAMP solution specific amplification lambda dna low unrelated amplification PSA Mw PEI increase probe LAMP reaction negative solution form insoluble pei oligo dna complex result mean LAMP reaction negative solution distinguish LAMP reaction positive solution successfully amplify target sequence PEI high molecular weight observe Figure LAMP amplicon present added PEI react LAMP amplicon little opportunity interaction free oligo dna probe LAMP amplicon present added PEI interact oligo dna probe condition large PEI high molecular weight strongly interact oligo DNA undesirable insoluble PEIoligo dna probe complex form molecular weight PEI high average molecular weight PEI detection system conduct similar experiment spermine polyamine low molecular weight experiment adequate insoluble complex generate compare PEI average molecular weight condition present research datum show known spermine dna insoluble indicate report optimal condition case spermine precipitant exist investigation go scope paper investigation carry insolubilization LAMP product PEI inhibit addition excessive KCl LAMP reaction solution amplification figure excessive amount potassium ion chloride ion inhibit electrostatic interaction DNA PEI finding indication detection method base neutralization negative electric charge dna positive electric charge PEI effect ionic strength LAMP reaction investigate find presence mM KCl markedly delay LAMP reaction datum show say detection method function trouble solution composition optimize LAMP reaction investigate sensitivity detection method Figure probe take precipitate case µg LAMP product able visualize fluorescence precipitate specific LAMP product µg find detection system sensitive detection system visual assessment simple poct device result show figure pmol probe add hybridize µg nmol nucleotide LAMP product word molecule probe bind base pair LAMP product LAMP product mixture product different size average molecular size kbp find molecule probe bind base pair reflect fundamental characteristic LAMP reaction new detection method describe utilize unique nature low molecular weight PEI form insoluble complex single strand anionic polymer low molecular weight oligo dna probe form insoluble complex dna high molecular weight LAMP product attention appear pay complex consist oligo dna PEI molecular weight low study probably interaction low molecular weight dna oligo DNA weak practical utility vector nucleic acid precipitant low Kunath et al report complex form PEI kda plasmid DNA unstable form PEI kDa Osland Kleppe report spemidine structure similar PEI form aggregate double strand oligo dna form aggregate single strand DNA draw result low molecular weight PEI study think interact dna high selectivity difference dna concentration structure number strand molecular weight fact take advantage nature lowmolecular weight PEI nucleic acid precipitant detection lead establishment detection method detection method effectively utilize characteristic LAMP method large amplification product synthesize short time maintain high specificity large amplification product create LAMP reaction precipitate size easily confirm eye generate PEI add LAMP reaction solution fact amplification highly efficient mean label probe detection add large result characteristic LAMP reaction follow addition PEI yield precipitate clear color size identify visually show photograph figure furthermore fact amplification product large mean range optimum PEI necessary generate insoluble complex wide precise system add PEI solution necessary contribute simplification poct device use detection method case PCR widely gene amplification method amplification product usually LAMP consequently order apply method PCR device rapidly cycle temperature perform PCR high luminance fluorogenic reagent system wash unreacted probe system accurately dispense PEI necessary likely obstacle put practical use simple inexpensive poct device genetic testing way LAMP product insoluble addition cationic polymer like PEI try carry BF separation chilled ethanol addition chill ethanol lower temperature LAMP solution strong tendency nonspecific probe weakly hybridize LAMP product precipitate datum show isopropanol PEG believe inferior PEI solution necessary add time LAMP reaction solution easy handle believe low molecular weight PEI efficiently insolubilize amplicon small good option high BF separation show study end inner primer fluorescently label LAMP product visible current study visualization inner primer fluorescently label end preferred possibility false positive self extension label primer exclude risk false positive oligo DNA probe fluorescently label end study highly accurate genetic testing establish necessary add PEI LAMP reaction solution LAMP reaction take place PEI strongly inhibit LAMP reaction open reaction tube amplification generally avoid prevent carry contamination development technique add PEI closed system need method practical use possible solution apply PEI lid reaction tube turn reaction tube upside LAMP amplification use wax respond heat furthermore able use technology like micro total Analysis system lab chip research conduct recent year solve problem establish extremely simple method visually detect LAMP product sequence specific manner simply add small low molecularweight pei LAMP reaction solution big feature technique ability visually present sequence information amplicon expensive source light detector contrast conventional genetic test require expensive reagent complex skillful manipulation large device equip expensive optical system main reason genetic testing keep wall specific institution university laboratory special equipment train engineer combination LAMP method new detection method describe overcome factor prevent true practical application super simple poct device genetic testing simple inexpensive poct device small light hold hand main component disposable develop day parent able genetically test pathogen sit bed child fever far dna hepatitis virus GenBank accession number digest BamHI clone reverse transcriptase PCR conduct HCV viral RNA GenBank accession number patient serum purify QIAmp viral RNA Mini kit Qiagen PCR primer design sequence non code region core PCR product clone accord establish method concentration respective plasmid obtain determine nm spectrophotometer Ultrospec Pharmacia Biotech template dna solution LAMP reaction prepare serial dilution plasmid solution purchase lambda dna solution Takara Bio Tris HCl buffer mm contain µl LAMP reaction carry scale µl base reference briefly forward inner primer FIP backward inner primer BIP final concentration LAMP primer Sigma Genosis Japan HPLC purification grade follow hyphen add LAMP reaction Lambda DNA LAMP reaction basic investigation clarify fundamental characteristic reaction occur LAMP product PEI establish LAMP product mrna prostate specific antigen PSA LAMP reaction product unrelated LAMP reaction product solution prepare serially dilute LAMP reaction solution know concentration µg µl LAMP reaction buffer order systematically investigate reaction LAMP product PEI concentration dna synthesize LAMP reaction determine accord attached protocol PicoGreen dsDNA Quantitation kit Molecular Probes spectrofluorophotometer spectrofluorophotometer Shimadzu Scientific Instruments loop segment LAMP product sequence complementary sequence probe detection show Fig probe design sequence melt temperature Tm low LAMP reaction temperature fluorescently label oligo dna probe HPLC purification grade purchase Sigma Genosis Japan KK sequence probe follow number parenthesis indicate length melt temperature Tm supplier information commercially available PEI Wako Pure Chemical Industries purification average molecular weight PEI study concentration PEI aqueous solution express concentration monomer unit monomer PEI stock solution mol prepare dissolve PEI mL deionized distil water graduated cylinder time use stock solution dilute water desire concentration white precipitate magnesium pyrophosphate form LAMP reaction solution product amplification reaction amplification precipitate magnesium pyrophosphate reaction tube centrifugate LAMP reaction solution second small desk centrifuge rpm µl PEI solution adjust desire concentration add supernatant form insoluble dna cationic polymeric polymer complex room temperature pellet form immediately centrifuge second centrifuge fluorescent image pellet photograph fluorescence microscope VB Keyence Corporation equip Handy uv Lamp wavelength nm Vilber Lourmat uv blocking filter NEO Dynamic MARUMI Optical magnesium pyrophosphate precipitate nonfluorescent effect visualization pellet fluorescence investigate effect ionic strength precipitation titration aqueous KCl add reaction solution LAMP reaction prescribed concentration centrifugation µl supernatant PEI DNA complex solution aliquote assay plate Corning fluorometry fluorescence plate leader polarion Tecan Japan percentage probe capture precipitate PEI LAMP product complex calculate accord following formula base obtain result','2006-01-10 00:00:00','http://dx.doi.org/10.1186/1472-6750-6-3',2665,695),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json','Local public health workers\' perceptions toward responding to an influenza pandemic','BACKGROUND: Current national preparedness plans require local health departments to play an integral role in responding to an influenza pandemic, a major public health threat that the World Health Organization has described as \"inevitable and possibly imminent\". To understand local public health workers\' perceptions toward pandemic influenza response, we surveyed 308 employees at three health departments in Maryland from March – July 2005, on factors that may influence their ability and willingness to report to duty in such an event. RESULTS: The data suggest that nearly half of the local health department workers are likely not to report to duty during a pandemic. The stated likelihood of reporting to duty was significantly greater for clinical (Multivariate OR: 2.5; CI 1.3–4.7) than technical and support staff, and perception of the importance of one\'s role in the agency\'s overall response was the single most influential factor associated with willingness to report (Multivariate OR: 9.5; CI 4.6–19.9). CONCLUSION: The perceived risk among public health workers was shown to be associated with several factors peripheral to the actual hazard of this event. These risk perception modifiers and the knowledge gaps identified serve as barriers to pandemic influenza response and must be specifically addressed to enable effective local public health response to this significant threat.',' local health department consider backbone public health response plan infectious disease outbreak influenza pandemic consider increasingly likely consider significant urgent threat nation public health preparedness infrastructure argue disaster scenario recently assess Department Homeland Security pandemic influenza likely deadly United States pandemic influenza plan release November lay critical role local state public health agency pandemic include provide regular situational update provider provide guidance infection control measure healthcare non healthcare setting conduct facilitate testing investigation pandemic influenza case investigate report special pandemic situation specify activity require extensive prompt response local health department current contingency plan account possible personnel shortage influenza morbidity previous study show extreme scenario vary proportion healthcare worker unable unwilling report duty truer health department unlike traditional responder agency law enforcement fire service emergency medical service capacity willingness respond crisis historically ingrain workforce professional culture training environment recent datum indicate inconsistent slow hour response health department urgent event involve communicable disease perception theory provide revealing framework well understanding response limitation need public health workforce perceive risk accord theory multifactorial phenomenon involve summation actual risk peripheral influence independent actual risk perceive authority trust situational control peripheral influence term outrage dread model previously suggest contribute factor peripheral actual risk considerable practical impact public health employee respond crisis aside physical circumstantial barrier availability transportation dependency family member identify specific risk perception issue impact markedly high unique importance public health workforce response crisis factor modifier stem number feature previously suggest associate elevated risk perception include manageability threat risk future generation direct personal impact sense control event base modifier major barrier effective public health workforce emergency response suggest include uncertainty work environment safety unclear expectation role specific emergency response requirement safety family member inadequate emphasis critical value employee agency response effort insufficient emphasis stress management techniquesall heighten employee sense dread lack personal control light project impact influenza pandemic health department optimize response rate employee crisis scenario base emer gency response principle disaster local set assess local public health employee risk perception likelihood reporting duty local outbreak pandemic influenza uncover variable affect outcome provide need evidence base health department planning training effort conduct study Carroll Dorchester Harford county health department March July health department locate Maryland range size employee employee select health department location community range Maryland Eastern Shore Dorchester County great Baltimore Towson metropolitan area Harford County reflect nation local public health agency serve community population few self administer anonymous survey send health department personnel respective health department complete survey directly mail investigator Johns Hopkins Center Public Health Preparedness survey include question personal characteristic job classification gender age respondent point likert scale question pertain possible flu pandemic probability report work likely likely possibility ask health department respond emergency likely likely knowledgeable think potential public health impact pandemic influenza knowledgeable knowledgeable confident safe work role confident confident likely family prepare function absence likely likely likely feel health department provide timely update likely likely familiar role specific response requirement familiar familiar think address question concerned member public significant role think play agency overall response significant significant important pre event preparation training important important important psychological support available event important important important psychological support available event important job classification variable collapse technical support staff computer entry staff clerical staff receptionist computer specialist health information system datum analyst etc professional staff include public health official clinical staff nurse dentist physician public health communicable disease staff environmental health staff public information staff public health professional staff health educator legal professional financial officer dichotomize response job classification question professional technical support category question likelihood reporting work pandemic influenza relate attitude belief dichotomize response score response logistic regression compute Odds Ratios evaluate association demographic variable attitude belief selfdescribe likelihood report work multivariate logistic regression explore association attitude belief relate pandemic influenza preparedness self describe likelihood reporting work model include adjustment age gender job classification similarly bivariate multivariate adjust age gender job classification logistic regression model evaluate association attitude belief order assess non response bias compare age gender job classification distribution respondent health department personnel TeleForm Version Cardiff Vista Stata Version Stata Corporation College Station TX data capturing analysis respectively receive survey fromcarroll Dorchester Harford county health department respectively result overall response rate find statistically significant difference age gender distribution respondent health department personnel small statistically significant difference proportion technical support staff professional staff detect study group personnel respectively significant difference proportion professional staff subgroup detect respond question likelihood report pandemic influenza relate emergency indicate likely report work emergency age gender association likelihood reporting clinical staff indicate high likelihood report Multivariate CI technical support staff table professional staff technical support staff likely ask health department respond pandemic influenza relate emergency perception Table perception exist knowledge pandemic influenza perception have important role agency overall response significantly high professional staff compare technical support staff figure multivariate analysis increase self describe likelihood reporting work influenza pandemic emergency significantly associate agreement construct notably perception capacity communicate risk effectively perception importance role agency overall response familiarity role specific response requirement pandemic influenza relate emergency table vast majority respondent feel benefit additional training activity low perceive level familiarity role significantly associate high perceive need additional training figure World Health Organization urge country prepare influenza pandemic term federally adopt model hazard emergency readiness present local health department new organizational challenge learn curve hazard approach entail ability willingness respond broad spectrum disaster range intentional chemical biological radiological terror naturally occur weather relate crisis non bioterrorism relate infectious disease national contingency plan account possible personnel shortage healthcare public health setting mainly expect influenza morbidity worker datum suggest regardless expect morbidity personnel influenza pandemic nearly half local health department worker likely report duty extreme public health crisis fact worker nearly technical support worker believe ask report work find willingness report duty pandemic varie considerably accord individual job classification clinical staff state significantly likely report duty compare worker difference correlate single influential construct associate willingness report duty perception importance role agency overall response respondent believe important role agency response local outbreak pandemic influenza subgroup willingness report duty high belief importance role low technical support staff environmental health staff non clinical professional staff respectively group willingness report show low believe effort direct ensure local public health worker notably non clinical professional staff understand advance importance role influenza pandemic fail need finding fit theoretical framework emphasize risk communication need public health worker serve risk communicator factor previously suggest risk perception modifier substantial impact public healthproportion individual agree attitude belief construct staff type figure proportion individual agree attitude belief construct staff type Technical Support Staff Professional Staff workforce response crisis prove important context lack knowledge ambiguity exact task questionable ability perform role risk communicator significantly associate high perceive personal risk fold decrease willingness report duty factor prove influential perceive level family preparedness function absence important recognize public health employee intend serve purveyor risk communication community represent community specific perception address context emergency readiness training threat impending influenza pandemic new take place decade year couple year highly pathogenic strain increasingly endemic southeast Asia lethal infection virus occur alarmingly increase rate human urgency situation openly declare national international health authority rapidity evolve situation serve explain respondent feel adequately knowledgeable pan demic influenza respondent feel capable effectively communicate pandemic risk finding especially noteworthy member public health support staff frontline telephone risk communicator crisis serve point interface concerned caller contact health department technical support staff worker feel incapable effective risk communication willing report duty believe health department ability provide timely information study relevant limitation factor overall analysis sample limit non randomly select health department serve community large resident staff size sample size survey employee limit study power study include Maryland health department account potential jurisdictional regional variation nationwide response capacity risk perception pandemic influenza response furthermore job classification develop CDC adopt emergency preparedness competency necessarily map neatly functional responsibility disaster Odds Ratios reporting work case pandemic influenza relate emergency staff attitude belief construct figure Odds Ratios reporting work case pandemic influenza relate emergency staff attitude belief construct Technical Support Staff Professional Staff response example health educator play frontline role clinical staff term degree interface public disaster job category necessarily reflect relative impact job specific cohort disaster response event report work assess presence direction nonresponse bias compare distribution personal characteristic respondent health department personnel lack significant difference age gender distribution lack significant difference job classification support staff indicate extent bias study probably limited small statistically significant representation technical support staff study group potentially cause slight underestimation overall willingness report internal association variable study separately technical support staff professional staff figure representation impact general conclusion present having account limitation important note finding internally consistent survey health department health department serve large metropolitan area few employee recognize nation local health department serve population local public health agency tend small staff size median time employee finding similar pattern datum willingness urban healthcare worker nonpublic health setting respond emergency survey employee healthcare facility New York City area reveal worker willing report duty untreatable naturally occur infectious disease outbreak affect facility SARS compare disaster scenario study detect similar rate likelihood report duty low rate expect study population New York City survey focus healthcare worker organizational culture historically accustomed local public health worker emergency response city heightened awareness disaster preparedness wake World Trade Center attack subsequent anthrax attack face pandemic influenza threat local health department employee unwillingness report duty pose threat nation emergency response infrastructure address specific factor associate unwillingness necessary help ensure exist local health department preparedness competency translate scope response describe nation pandemic influenza plan intervention suggest enhance willingness healthcare worker non public health department setting report duty disaster include workforce preparedness education provision appropriate personal protective equipment crisis counseling family preparedness social support recommendation fit framework finding recommend education program include specialized training emphasize specific nature guideline role response pandemic influenza relevance worker role effectiveness overall public health response worker ability provide effective risk communication additional research focus good practice model address describe gap local public health response urgent public health threat datum offer current evidence base window need public health worker serve backbone locally drive emergency response influenza pandemic setting find worker feel work significant personal risk scenario adequately knowledgeable perform role sufficiently train believe role significant impact agency overall response specific perception need attend specific intervention program initiate order reduce perceive risk associate worker role influenza pandemic worker well understanding scenario importance personal role setting confidence agency provide adequate protective equipment employee psychological support timely information belief train cope emergency responsibility include ability communicate risk view currently consider impending influenza pandemic wide gap desire target current status exist lead significant hindrance ability local health department function adequately','2006-04-18 00:00:00','http://dx.doi.org/10.1186/1471-2458-6-99',1754,602),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json','Development of a humanized monoclonal antibody with therapeutic potential against West Nile virus','Neutralization of West Nile virus (WNV) in vivo correlates with the development of an antibody response against the viral envelope (E) protein. Using random mutagenesis and yeast surface display, we defined individual contact residues of 14 newly generated monoclonal antibodies against domain III of the WNV E protein. Monoclonal antibodies that strongly neutralized WNV localized to a surface patch on the lateral face of domain III. Convalescent antibodies from individuals who had recovered from WNV infection also detected this epitope. One monoclonal antibody, E16, neutralized 10 different strains in vitro, and showed therapeutic efficacy in mice, even when administered as a single dose 5 d after infection. A humanized version of E16 was generated that retained antigen specificity, avidity and neutralizing activity. In postexposure therapeutic trials in mice, a single dose of humanized E16 protected mice against WNV-induced mortality, and may therefore be a viable treatment option against WNV infection in humans. SUPPLEMENTARY INFORMATION: The online version of this article (doi:10.1038/nm1240) contains supplementary material, which is available to authorized users.',' development humanize monoclonal antibody therapeutic potential West Nile virus protein nickel affinity chromatography datum show immunization screen hybridoma isolate new monoclonal antibody recognize WNV protein Supplementary Table evaluate antibody ability block WNV infection cell standard plaque reduction assay strong neutralizing activity greatly exceed potency immune human globulin plaque reduction neutralization titer PRNT µg immune human globulin PRNT value µg inhibitory activity neutralizing antibody reproduce mouse macrophage human adrenal carcinoma cell Supplementary Fig online specific fibroblast potent neutralizing monoclonal antibody inhibited infection genetically diverse WNV lineage strain isolate mosquito bird horse New York neutralize WNV strain PRNT value ng PRNT value ng Supplementary Table online notably fab fragment inhibit WNV PRNT ng suggest neutralization require bivalent protein binding potently block infection strain original lineage ii strain isolate ref virus specific recognize neutralize flavivirus include distantly relate dengue yellow fever virus datum show closely relate Japanese Louis encephalitis virus Supplementary Table figure Mapping monoclonal antibodie DIII yeast surface display WNV protein yeast fusion protein compose ectodomain DIII WNV protein yeast protein attach protein yeast cell wall Yeast transform vector row entire WNV ectodomain amino acid middle row DIII amino acid row induction yeast stain indicate monoclonal antibody negative control anti SARS process flow cytometry datum representative neutralizing non neutralizing antibody show flow cytometric enrichment DIII express yeast variant lose binding round increase percentage DIII express yeast recognize polyclonal WNV specific antibody final round DIII express variant boxed region harvest figure fine epitope mapping DIII neutralizing non neutralizing monoclonal antibody flow cytometry profile immunoreactivity nonneutralize weakly neutralizing strongly neutralizing monoclonal antibodie yeast express wild type variant DIII red yellow green arrow point mutation abolish yeast surface binding individual monoclonal antibodie correspond distinct region DIII mapping neutralizing non neutralizing monoclonal antibodie surface WNV DIII molecular surface representation depict base nuclear magnetic resonance structure WNV DIII indicate amino acid residue associate binding neutralizing weakly neutralizing non neutralizing monoclonal antibody show red yellow green respectively map strongly neutralizing monoclonal antibody develop strategy yeast surface display ectodomain amino acid DIII amino acid WNV protein express fusion protein yeast cell surface Fig antibodie recognize DIII consider DIII specific monoclonal antibody recognize ectodomain DIII contact residue map domain II cooperative contact DIII rule monoclonal antibody recognize yeast display entire ectodomain Fig Supplementary Table online sixteen antibody recognize yeast display DIII strongly neutralizing antibody localize DIII neutralizing antibody recognize ectodomain DIII error prone PCR mutagenesis DIII WNV protein yeast surface expression map antibody contact residue highthroughput manner perform individual screen identify DIII mutant lose bind selectively strongly neutralizing weakly neutralizing non neutralizing monoclonal antibody eliminate mutant abolish surface expression DIII yeast stain sequentially Alexa Fluor conjugate individual monoclonal antibody Alexa Fluor conjugate oligoclonal antibody derive pool individual monoclonal antibodie cell sorting identify yeast selectively lose expression individual monoclonal antibody epitope retain surface expression DIII Fig multiple independent yeast lose binding individual monoclonal antibody subject plasmid recovery sequencing monoclonal antibodie localize DIII strongly neutralize WNV reduce binding residue alter Fig Table locate adjacent loop form contiguous patch solvent expose surface lateral tip DIII Fig mutation cause considerable loss binding neutralize monoclonal antibody test reduce bind neutralizing monoclonal antibody identical loss bind pattern example reduce binding abolish binding affect strongly contrast decrease binding neutralize monoclonal antibody affect non neutralizing poorly neutralizing monoclonal antibody change residue abolish binding neutralizing non neutralizing monoclonal antibody strongly reduce binding neutralize monoclonal antibodie exception weaken binding monoclonal antibodie recognize DIII poorly neutralizing non neutralizing engage dominant neutralizing epitope define abolish reduce mutation binding weaken change proximal physically distinct residue affect binding neutralizing monoclonal antibody Fig mutation identify loss bind sort effect non neutralizing monoclonal individual WNV specific monoclonal antibody µg ml mix yeast display wild type mutant DIII surface washing incubation goat anti mouse IgG secondary antibody yeast analyze antibody bind flow cytometry value represent percentage yeast positive DIII expression give monoclonal antibody parenthesis mean linear fluorescence intensity positive cell Yeast analyze induction galactose give baseline surface expression DIII positive cell Fig bold value indicate complete loss binding underline value mark reduction percentage mean fluorescence intensity positive cell result representative independent experiment antibody DIII monoclonal antibody weak neutralizing activity map site non neutralizing neutralizing monoclonal antibody mutation strongly inhibit binding determine human antibodie specific WNV recognize neutralizing epitope DIII infection plasma obtain WNV positive individual sample convalescent individual negative WNV RNA positive neutralize WNV specific antibody individual report mild systemic illness progress severe disease determine sample contain antibody localize neutralizing epitope DIII test Fab IgG compete bind recombinant wildtype mutant form DIII Supplementary Fig online retain wild type binding markedly reduce binding equivalently inhibit binding patient WNV specific antibody wild type Fab IgG DIII Fab IgG IgG WNV specific monoclonal antibody recognize epitope outside DIII compete bind wild type DIII expect weakly recognize poorly compete fab IgG bind DIII datum suggest human clear WNV infection develop antibody recognize epitope close proximity define evaluate correlation neutralization epitope localization vivo protection assess therapeutic activity different neutralizing monoclonal antibodie established mouse model study perform week old wild type mouse survival rate mouse inoculate subcutaneously plaque form unit PFU WNV administer single dose monoclonal antibody day infection notably µg non neutralizing monoclonal antibody provide protection datum show contrast µg different neutralizing monoclonal antibodie map protect great mouse lethal infection Fig single µg treatment day infection prevent mortality give human present WNV infection CNS evaluate therapeutic efficacy monoclonal antibodie later monoclonal antibodie control mouse independently administer saline PBS negative control monoclonal antibody anti sars µg survival curve construct datum independent experiment number animal antibody dose range difference survival curve statistically significant WNV specific monoclonal antibody dose show WNV burden brain week old wild type mouse day WNV infection brain harvest viral burden determine plaque assay following percentage mouse viral burden detection PFU day day day Efficacy WNV specific monoclonal antibody therapy day infection single dose mg day mg day monoclonal antibody anti SARS administer WNV infection datum reflect approximately mouse condition difference survival curve statistically significant WNV specific monoclonal antibody dose show day day effect therapy viral burden mouse treat mg PBS day WNV infection day brain recover homogenize subject plaque assay subset receive PBS treatment brain harvest day moribund mouse data express PFU mouse treat undetectable viral load brain day mouse treat PBS detectable virus time harvest dotted line represent limit sensitivity assay dark bar represent mean log value difference statistically different point day WNV infection detect virus brain mouse respectively Fig single µg dose day result survival rate Fig single mg dose day result survival Fig complete clearance WNV brain mouse day Fig administration neutralizing monoclonal antibodie mouse active cns infection improve survival induce virologic cure expect lengthen interval treatment associate decreased benefit administration day enhance survival datum show average survival time increase consider humanize possible therapeutic measure humanize monoclonal antibody substantially long half life human mouse counterpart sequence study indicate great homology human framework region make simple construct humanize version amplify cDNA encode heavy VH light VL variable domain hybridoma cellular RNA rapid amplification cdna end race procedure VH belong mouse heavy chain subgroup II family VL belong mouse chain subgroup complementarity determine region graft human backbone Fig human Vκ backbone Fig create Hm VL framework mutation VH VL introduce create variant Hm Hm respectively result humanize VH VL combine human constant region fuse IgG signal sequence insert expression plasmid construct chimerized antibody mouse VH VL sequence combine human constant region express humanize Hm chimerize Ch cell purify supernatant affinity size exclusion chromatography datum show affinity analyze surface plasmon resonance purified antibody solid phase Mouse bind DIII affinity nM half life min affinity Ch Hm similar range nM Fig Hm Ch parent similar PRNT value Fig hypothesize control infection mouse effector function include antibody dependent complement fixation cytotoxicity test generate Ch aglycosyl variant neutralize WNV Fig efficiently fix complement bind Fc receptor mouse administer Ch Ch day WNV infection high dose Ch Ch provide virtually complete protection low dose aglycosyl variant afford protection Fig administration µg Ch result survival µg Ch provide protection test effector function enhance activity perform study week old deficient mouse Fig mouse increase susceptibility lethal WNV infection compare wild type control deficient mouse activate complement antibody dependent classical pathway similar potency compare wild type mouse contrast deficient mouse high dose afford complete protection low dose result high mortality rate dose µg day strongly protect wild type deficient mouse improve survival rate deficient mouse Fc region enhance potency mouse virtue ability bind Fc receptor confirm efficacy humanize wild type mouse administer different version purify Hm day infection Hm variant protect mouse lethal infection Fig dose µg variant show high affinity DIII Fig protective versus survival mean survival time versus generate panel monoclonal antibodie WNV protein apply new high throughput epitope mapping strategy identify dominant epitope recognize majority neutralizing monoclonal antibodie DIII epitope detect convalescent antibody individual recover WNV infection clinical consequence neutralize monoclonal antibody protect WNV mortality postexposure therapy model humanize confirm therapeutically effective mouse previous study map amino acid contact residue neutralizing monoclonal antibodie sequence vitro neutralization escape variant site specific substitution specific charge polar residue perform bind assay overlap peptide library error prone PCR mutagenesis yeast surface expression identify contact residue high throughput manner technique previously highthroughput epitope mapping application have large panel DIII monoclonal antibodie select variant abolish markedly reduce binding monoclonal antibody minimize possibility mutation alter protein folding recently confirm recognition site DIII validity yeast display strategy solve crystal structure Fab DIII complex unpublishe neutralize monoclonal antibody localize distal lateral surface DIII result consistent prior study map neutralize monoclonal antibodie WNV vitro escape variant weakly neutralizing non neutralizing monoclonal antibody recognize epitope localize distinct region recognize dominant epitope neutralize strain test sequence analysis WNV strain public database show complete conservation contact residue clinically attenuate lineage II isolate mutation residue structural homology flavivirus analogous amino acid map distal lateral surface DIII readily identify region highly variable flavivirus mutation abolish binding virus specific neutralizing antibody map suggest existence analogous dominant neutralizing epitope flavivirus speculate successful vaccine WNV flavivirus induce potent humoral response neutralizing epitope passive administration immune human globulin WNV infection improve survival mice limit low titer neutralizing activity variability risk transmission infectious agent prior study show postexposure therapy neutralizing monoclonal antibodie flavivirus reduce mortality infection Louis encephalitis virus reduced mortality result infection japanese encephalitis virus different neutralizing monoclonal antibody improve survival administer WNV infection therapy day completely clear WNV brain day mouse inhibitory WNV monoclonal antibody improve clinical virologic outcome viral spread CNS result agree study show antibody mediate viral clearance infected neuron experiment consistent model therapeutic efficacy monoclonal antibody determine property addition neutralization monoclonal antibody strong neutralizing activity vitro great efficacy vivo ii aglycosyl version lack ability fix complement bind Fc receptor equivalent neutralizing reduced therapeutic activity iii potent mouse lack Fc receptor humanize possible therapeutic measure human Hm bind DIII similar affinity show efficacy postexposure therapy possible improve Hm introduce mutation enhance affinity create form readily cross blood brain barrier combine monoclonal antibodie neutralize WNV infection independent mechanism result successful demonstration humanize monoclonal antibody postexposure therapy viral disease suggest antibody base therapeutic broad utility previously appreciate especially treatment cns infection effective antibody response important limit virus dissemination injury neuron culture Vero Aedes albopictus cell previously describe perform neutralization experiment prototype strain Louis Parton Japanese encephalitis Nakayama viruse vivo experiment virus dilute inject mouse describe WNV protein expression WNV protein ectodomain generate baculovirus expression system accord previously describe method related flavivirus nucleotide prM endogenous signal sequence nucleotide WNV protein New York strain fuse downstream polyhedrin promoter upstream histidine repeat baculovirus shuttle vector pFastBac Invitrogen PCR high fidelity Taq polymerase Platinum Taq Invitrogen day baculovirus infection insect cell multiplicity infection MOI supernatant harvest filter buffer exchange purify nickelaffinity chromatography accord manufacturer instruction purified WNV ectodomain lack terminal amino acid associate membrane proximal transmembrane cytoplasmic domain purify WNV DIII construction expression purification refolding DIII WNV protein describe great detail unpublished datum briefly wild type DIII generate infectious cdna clone New York strain WNV gift Kinney Centers Disease Control Prevention Fort Collins CO PCR Quik change mutagenesis Stratagene clone vector Novagen sequence confirmation mutation transform plasmid Codon plus coli cell Stratagene Bacteria grow Luria broth induce mm isopropyl thiogalactoside IPTG pellete subsequently lyse bacteria addition lysozyme sonicate recover DIII insoluble aggregate inclusion body DIII denature presence guanidine hydrochloride mercaptoethanol refolde slowly dilute denature reagent presence arginine EDTA PMSF reduce glutathione oxidize glutathione separate refolde DIII aggregate Superdex size exclusion column Amersham Bioscience concentrate spin column mm Hepes pH mM NaCl NaN refolde wild type DIII react domain iii specific monoclonal antibody include recognize conformationally sensitive epitope generation purification monoclonal antibody BALB mouse prime boost week interval insect cell generate purify recombinant WNV protein µg complexe adjuvant RIBI Immunochemical approximately month boost harvest serum test immunoreactivity solid phase purify Mice high titer boost intravenously purified protein µg PBS harvest splenocyte later fuse myeloma cell generate hybridoma accord publish procedure purify monoclonal antibodie WNV control antigen stan dard protein protein chromatography accord manufacturer instruction WNV infection experiment wild type mouse purchase commercial source Jackson Laboratories obtain congenic deficient deficient mouse Stahl Beth Israel Deaconess Medical Center Boston MA Carroll CBR Institute Biomedical Research Harvard Medical School Boston MA respectively obtain congenic deficient mouse commercially taconic mouse week age depend particular experiment inoculate subcutaneously WNV footpad injection anesthetization xylazine ketamine mouse experiment approve perform accord guideline Washington University School Medicine Animal Safety Committee passive transfer experiment administer mouse single dose purify monoclonal antibody intraperitoneal injection give time point day infection analyze virus production brain infected mouse kill give day inoculation cardiac perfusion PBS remove brain weigh homogenize perform plaque assay previously describe WNV protein yeast ectodomain DIII WNV protein express surface yeast modification previously describe protocol surface expression cell receptor amino acid residue ectodomain DIII WNV protein amplify BamHI XhoI site end respectively PCR New York infectious cdna clone Kinney Centers Disease Control Prevention Fort Collins CO result product digest BamHI xhoi clone downstream fusion yeast Xpress epitope tag gene yeast surface display vector Invitrogen upstream promoter control fusion protein expression construct transfecte cerevesiae yeast strain ref result yeast express WNV ectodomain DIII yeast express Xpress epitope tag link prepare parallel transfecte cell parent vector individual yeast colony grow log phase overnight tryptophan free medium contain glucose harvest log phase fusion protein expression induce yeast surface grow yeast additional tryptophan free medium contain galactose harvest yeast wash PBS supplement mg ml BSA immunostaine µl monoclonal antibody µg ml Xpress tag WNV protein min wash yeast time stain goat anti mouse secondary antibody conjugate Alexa Fluor Molecular Probes subsequently yeast cell analyze Becton Dickinson FACSCaliber flow cytometer library construction screening mutate DIII WNV protein error prone PCR protocol include Mn Mg concentration mm respectively subsequently cdna library ligate transform blue ultracompetent cell Strategene colony pool plasmid DNA recover Qiagen HiSpeed Maxi kit individual antibody screen yeast library DIII mutant accord follow protocol identify yeast selectively lose binding give monoclonal antibody epitope library initially stain Alexa Fluor conjugate WNV specific monoclonal antibody min control surface expression DIII washing yeast subsequently stain min Alexa Fluor conjugate oligoclonal antibody derive pool individual monoclonal antibody immunostaine subject yeast flow cytometry identify population single monoclonal antibody negative pool oligoclonal antibody positive yeast cell sort event rate cell population monoclonal antibody negative oligoclonal antibody positive enrich round sort final enrichment sort plate yeast select individual colony test bind individual monoclonal antibodie individual clone lose desire monoclonal antibody epitope DIII plasmid recover Zymoprep Yeast Miniprep kit Zymo Research plasmid transform cell purify Qiaprep Spin Miniprep kit Qiagen sequence case DIII variant independent mutation isolate determine mutation confer loss bind phenotype single independent mutation engineer site direct mutagenesis DIII mutant oligonucleotide Quik Change II mutagenesis kit Strategene mutation confirm sequence determine titer neutralize antibody standard plaque reduction neutralization titer PRNT assay cell result plot titer PRNT inhibition PRNT calculate inhibition assay mouse macrophage perform follow mix medium µg monoclonal antibody PFU WNV incubate mixture add mouse macrophage individual well plate cell wash time PBS remove free virus monoclonal antibody DMEM FBS add cell incubate additional subsequently harvest supernatant viral plaque assay Vero cell competition ELISA human anti WNV antibodie purification refolde wild type DIII dilute µg ml Na carbonate buffer pH adsorb plate overnight block PBS BSA Tween PBS BT well preincubate PBS BT contain antibody IgG µg ml Fab µg ml IgG µg ml serve negative control recognize epitope domain ii WNV protein subsequently human plasma dilution PBS BT heat inactivated directly add additional obtain human sample informed consent seven different WNV infect patient gift Busch Tobler Blood Systems Research Institute San Francisco sample sequentially number linkable original subject satisfy criterion exemption approval Human Studies Committee Washington University wash PBS BT plate serially incubate biotin conjugate goat anti human IgG µg ml streptavidin horseradish peroxidase µg ml tetramethylbenzidine develop substrate DAKO determine optical density nm automatic ELISA plate reader Tecan adjust subtraction value obtain nonimmune human plasma surface plasmon resonance antibody affinity analysis DIII WNV perform surface plasmon resonance biacore Biacore Inc bind curve kinetic parameter obtain follow capture antibody flow nm rate µl min min immobilized fragment specific goat anti human mouse IgG Fc region specificity subsequently DIII New York strain WNV protein amino acid generate coli inject nm flow rate µl min min allow dissociate min surface regenerate pulse injection mm glycine pH mM NaOH injection analyze curve global fit bind algorithm drift baseline cloning humanization heavy light chain RNA isolate hybridoma cell guanidinium thiocyanate phenol chloroform extraction convert cdna reverse transcription VH VL segment amplify PCR rapid amplification cdna end race system Invitrogen gene specific primer GSP VH VL follow VH VH VL VL cggatccgatggatac PCR product insert plasmid topo topota kit Invitrogen subject result plasmid dna sequence determine VH VL sequence cDNA sequence translate predict amino acid sequence determine sequence framework CDR region identify previously define join mouse VH human constant region Ig leader sequence insert pCI neo mammalian expression join mouse VL human Cκ segment Ig leader sequence clone pCI neo mammalian expression chimeric Ch Ch site direct mutagenesis perform change residue asparagine glutamine heavy chain eliminate single glycosylation site Fc Humanized VH consist framework segment human germline VH segment segment CDR region VH respectively humanize VL consist framework segment human germline vk VL segment ref segment CDR region VL humanize VH segment assemble de novo oligonucleotide amplify PCR humanize VL segment assemble PCR overlap PCR result VH VL segment subsequently combine overlap PCR leader sequence appropriate constant region segment clone expression vector pci neo nhei ecori fragment confirm DNA sequence result plasmid sequence analysis site direct mutagenesis perform substitute mouse human residue key framework position result plasmid cotransfecte human cell humanize antibody recover result condition medium purify protein size exclusion chromatography statistical analysis datum analyze prism software GraphPad Software survival analysis Kaplan Meier survival curve analyze log rank Mantel Haenszel test viral burden experiment determine statistical significance Mann Whitney test note supplementary information available Nature Medicine website','2005-04-24 00:00:00','http://dx.doi.org/10.1038/nm1240',3137,938),('d09b79026117ec9faebba46a8d13aa9b23ec751e.json','A Method to Identify p62\'s UBA Domain Interacting Proteins','The UBA domain is a conserved sequence motif among polyubiquitin binding proteins. For the first time, we demonstrate a systematic, high throughput approach to identification of UBA domain-interacting proteins from a proteome-wide perspective. Using the rabbit reticulocyte lysate in vitro expression cloning system, we have successfully identified eleven proteins that interact with p62’s UBA domain, and the majority of the eleven proteins are associated with neurodegenerative disorders, such as Alzheimer’s disease. Therefore, p62 may play a novel regulatory role through its UBA domain. Our approach provides an easy route to the characterization of UBA domain interacting proteins and its application will unfold the important roles that the UBA domain plays.',' novel cellular protein initially identify human phosphotyrosine independent ligand src homology domain lck lck member src family cytoplasmic tyrosine kinase find predominantly cell lymphoid origin addition interaction lck associate Ser Thr kinase atypical protein kinase ubiquitin addition domain possess structural motif include ubiquitin associate UBA domain capable bind ubiquitin nonconvalently Ubiquitin Ub small polypeptide amino acid convalently attach protein specific lysine residue chain compose mono ubiquitin moiety poly addition classical role protein degradation ubiquitin emerge signal protein transport processing conjugation ubiquitin substrate protein require enzyme ubiquitin activating enzyme ubiquitin conjugate enzyme ubiquitin ligase initially activate ubiquitin form high energy thioester intermediate terminal glycine ATP activate ubiquitin sequentially transfer catalyze isopeptide bond formation activate terminal glycine ubiquitin amino group lysine residue substrate follow linkage ubiquitin chain additional molecule ubiquitin attach lysine chain previously conjugate moiety form branched polyubiquitin chain fate ubiquitinated substrate depend number ubiquitin moiety conjugate lysine linkage Ub Ub conjugation conjugation ubiquitin eukaryotic intracellular protein way protein target proteasome subsequent rapid degradation mechanism particularly important short live regulatory protein cyclin cyclin dependent protein kinase inhibitor nuclear factor kappa precursor IκB ubiquitinproteasome system consist step target protein conjugate polyubiquitin molecule mark substrate degradation target protein transfer proteasome unfold degraded UBA domain conserve sequence motif protein bind polyubiquitin comprise amino acid amino acid bind polyubiquitin show possess homology recently describe UBA domain interestingly protein UBA domain likely bind polyubiquitin chain monoubiquitin yeast UBA protein highly conserve protein involve nucleotide excision repair recently show yeast cell lack UBA protein deficient protein degradation UBA motif essential function proteolysis addition important role recycling amino acid damaged misfolde protein ubiquitin protein conjugation function unrelated proteasomal targeting example polyubiquitination require internalization yeast mammalian cell surface protein endocytic pathway interestingly appear sequester ubiquitinated substrate cytoplasmic structure refer sequestosome excess ubiquitinated protein segregate addition immediate early response gene product variety signal appear play novel regulatory role polyubiquitinate protein essential function cell proliferation differentiation develop method enable identification interact UBA domain human adult brain library plate cdnas Gold TNT search novel protein bind UBA domain perform vitro expression cloning ivec ProteoLink IVEC system human adult brain library plate cdnas transcribe translate employ Gold TNT Express plate methionine TNT Quick couple transcription translation system contain rabbit reticulocyte lysate pre mixed reaction component necessary carry transcription translation lysate include amino acid methionine Methionine label newly synthesize protein reaction set accord manufacturer instruction rabbit reticulocyte lysate show capable carry ubiquitination protein translate vitro translation system reaction mixture contain ubiquitin newly synthesize protein ubiquitinated reaction incubate hour result protein assay determine bind ability UBA domain potential positive hit subdivide reassaye link individual clone protein interest Fig translate pool resuspend bind buffer mM Tris pH mM NaCl source protein UBA pull assays protein specifically interact UBA domain isolate interaction agarose immobilised UBA peptide amino acid µg hour wash time washing buffer mM Tris pH mM NaCl bind protein release addition SDS sample buffer separate SDS PAGE SDS PAGE gel fix methanol acetic acid min stain Commassie Brilliant Blue methanol acetic acid min destain acetic acid methanol overnight enhance autoradiography enhancer en HANCE hr expose ray film combine pool mixed pool protein pool divide mixed protein pool use UBA pull assays positive mixed protein pool select individual pool reteste ability bind UBA domain individual cdna pool positive protein pool generate transform competent cell plate LB ampicillin plate individual colony choose grow overnight ml LB medium plus ampicillin Plasmid DNA purify cell culture TNT Quick couple vitro translation individual protein synthesize plasmid dna choose screen ability bind UBA domain confirm interaction UBA domain final result individual protein couple TNT UBA pull assays cDNA insert sequence Genomics Core Facility Auburn University sequence compare know sequence NCBI database blast analysis human embryonic kidney HEK cell culture high glucose Dulbecco modify Eagle medium DMEM contain heat inactivate fetal calf serum transfecte myc tag plasmid Mammalian Cell Transfection Kit cell harvest lyse ml SDS lysis buffer mM Tris HCl pH mM NaCl mm NaF mM Na VO µg ml aprotinin µg ml leupeptin mM PMSF SDS min ice follow centrifugation rpm min remove insoluble fraction protein concentration supernatant determine bio rad DC protein assay reagent bovine serum albumin BSA standard equal protein µg immunoprecipitate anti myc collect agarose couple secondary antibody agarose bead contain immunoprecipitate µl reaction buffer mM Tris HCl pH mm MgCl mm DTT mM ATP add contain ng ng µg flag tag µg GST WT Ub GST Ub GST Ub GST Ub Ub control sample GST WT Ub include reaction carry continuous shake hour wash time reaction buffer protein release boil min SDS PAGE sample buffer separate SDS PAGE Western blot anti ubiquitin search novel protein bind UBA domain perform vitro expression cloning ivec ProteoLink ivec system Promega Cat human adult brain library plate cdnas transcribe translate employ Gold TNT Express plate presence methionine ubiquitin Sigma combine protein pool mixed pool protein pool divide mixed pool Fig lane contain protein theoretically different molecular weight lane appear smear indicate vitro transcription translation system Promega work successfully order examine protein synthesize IVEC system ubiquitinated western blot analysis perform blot newly synthesize protein presence cold methionine instead methionine ubiquitin monoclonal antibody mixed protein pool lane appear smear indicate protein synthesize IVEC system ubiquitinated Fig furthermore rabbit reticulocyte lysate IVEC system utilize different lysine linkage ubiquitin Ub Ub Ub ubiquitination Fig order investigate agarose immobilised UBA peptide bind specificity mixed protein pool synthesize IVEC system test pull assay presence agarose bead presence UBA agarose bead Fig result reveal protein bind UBA domain pull agarose bead indicate agarose immobilised UBA peptide bind specificity order identify protein bind UBA domain UBA pull assay perform mixed protein pool pool contain methionine label band primary UBA pull assay Fig choose pool pool strong signal specifically identify individual protein pool mixed pool ability bind UBA domain secondary screen conduct positive individual mixed pool represent individual protein pool bind UBA domain Fig mixed protein pool generate positive protein molecular weight KDa Fig individual protein pool protein pool comprise protein pool protein molecular weight Fig depend size protein pull secondary screen compare primary screen Fig individual protein pool identify Fig specifically identify protein individual protein pool ability bind UBA domain cdnas positive individual protein pool transform competent cell plate LB ampicillin plate individual colony choose grow overnight ml LB medium plus ampicillin Plasmid DNAs purify cell culture TNT Quick couple vitro transcription translation individual protein synthesize plasmid DNA reteste ability bind UBA domain synthesize individual protein individual plasmid Gold TNT Quick couple vitro transcription translation system subject UBA pull assay positive clone isolate positive individual pool surprising clone show bind ability UBA domain cdnas positive individual pool low bind ability show weak signal mixed protein pool possible efficiently synthesize mixed TNT reaction individual TNT reaction cdna template positive plasmid sequence compare know cdna sequence NCBI database blast analysis result show Table protein identify screen fall distinct category set protein associate Alzheimer disease include myelin basic protein protein syntaxin bind protein transketolase heat shock protein reelin calcium calmodulin kinase II table significant decrease myelin basic protein report white matter Alzheimer disease patient accompany increase quantity βamyloid peptide presence amyloid peptide contain senile plaque neurofibrillary tangle major pathological feature brain patient Alzheimer disease interestingly protein demonstrate component neurofibrillary tangle Alzheimer disease brain syntaxin bind protein powerfully regulate amyloid precursor protein metabolism amyloid secretion direct indirect interaction protein activity transketolase report reduce dementia Alzheimer type brain heat shock protein expression significantly increase temporal cortex patient Alzheimer disease heat shock protein link Alzheimer disease example increase synthesis suggest play role prevent neuronal injury ad alpha crystallin heat shock protein close relationship neurofibrillary tangle ad brain Reelin large secrete protein control cortical layering signal low density lipoprotein receptor apolipoprotein receptor induce tyrosine phosphorylation adaptor protein suppress tau phosphorylation neurofibrillary tangle comprise highly phosphorylate tau protein key component Alzheimer disease enhanced activity calcium calmodulin kinase II suggest contribute phosphorylation tau protein lead neurofibrillary tangle deposition neuronal death Alzheimer disease relationship neurofibrillary tangle neuritic plaque unclear neurofibrillary tangle dystrophic neuritis neuritic plaque associate ubiquitin suggest dysfunction ubiquitin mediate proteolysis result accumulation ubiquitinconjugated protein contribute origination dystrophic neuritis neurofibrillary tangle furthermore recently report accumulate early neurofibrillary tangle Alzheimer disease suggest play important role Alzheimer disease interact protein UBA domain second set protein identify screen bind UBA domain associate brain development include homeobox protein like kinase II table Meis protein extensively study human protein show require hindbrain development zebrafish like kinase II demonstrate play role axonal elongation need formation complicated neuronal network set protein exhibit ability bind UBA domain protein link neurodegenerative disease include bind protein nuclear receptor corepressor table tacrolimus potent immunosuppressive drug treatment patient organ transplantation select autoimmune disorder activate bind member immunophilin family protein designate bind protein immunophilin chaperone protein bind protein suggest therapeutic neurological disorder nuclear receptor corepressor suggest play role Huntington disease able interact huntingtin protein identify suggest UBA domain ability interact multiple protein play important role neurodegenerative disease screen genome wide perspective necessary define important role UBA domain play report polyubiquitin chain assemble lysine ubiquitin act signal substrate proteolysis proteasome order understand protein identify screen bind UBA domain lysine polyubiquitin chain add UBA pull assay Fig inclusion polyubiquitin chain assay compete binding substrate UBA domain reduce interaction protein UBA domain protein assemble chain alternative interpretation polyubiquitin chain competition ubiquitin chain compete bind site bind partner ubiquitinated non ubiquitinated randomly choose protein bind partner competition pull Fig protein protein show reduced bind ability UBA domain polyubiquitin chain include Fig chain fail compete suggest UBA domain bind ubiquitin lysine linkage interestingly report heat shock protein cognate ubiquitinate CHIP carboxyl terminus interact protein ubiquitin chain synthesis use order examine lysine linkage utilize bind UBA domain vitro ubiquitination assay perform incubate lysate hek cell express reaction buffer mM Tris HCl pH mm MgCl mm DTT mM ATP control ubiquitination utilize rabbit reticulocyte lysate investigate western blot analysis result reveal ubiquitinated IVEC system Fig rabbit reticulocyte lysate contain enzyme vitro ubiquitination Fig choose vitro ubiquitination assay ring domain common feature ligase observation scaffold interaction vitro ubiquitination assay system perform presence ubiquitin wild type ubiquitin mutant lysine mutant block ubiquitination suggest ubiquitination utilize specific lysine linkage result reveal utilize linkage assemble polyubiquitin chain bind UBA domain ubiquitin mutant block ubiquitination Fig similar result observe reaction conduct wild type ubiquitin mutant ubiquitin lysine mutate arginine ubiquitination occur reaction intact ubiquitin wild type ubiquitin Fig finding consistent previous report demonstrate polyubiquitinate furthermore vivo interaction confirm transfecte myc tag hek cell presence proteasome inhibitor subject cell lysate immunoprecipitation western blot anti myc antibody Fig interaction vivo take place include suggest interaction vivo dependent ubiquitination specific type polyubiquitin chain recognize UBA domain know study underway lab determine interaction specific polyubiquitin chain preliminary study suggest UBA domain recognize link polyubiquitin chain protein synthesize employ TNT Quick couple vitro transcription translation system presence ubiquitin resolve SDS PAGE gel transfer nitrocellulose membrane western blot ubiquitin monoclonal antibody Protein synthesize employ TNT Quick couple vitro transcription translation system presence ubiquitin methionine resolve SDS PAGE expose ray film western blot rabbit reticulocyte lysate summary time demonstrate systematic approach identify UBA domain bind protein proteome wide perspective approach readily adapt high throughput screening rabbit reticulocyte lysate vitro expression cloning system successfully identify protein human adult brain interact UBA domain majority protein associate neurodegenerative disorder Alzheimer disease interesting finding cdnas screen show bind specificity UBA domain study underway unfold functional role ubiquitin system approach provide easy route characterization UBA domain bind protein level proteome application unfold important role UBA domain play method easily adapt identify protein interact UBA domain','2003-12-12 00:00:00','http://dx.doi.org/10.1251/bpo66',1911,625),('d1aafb70c066a2068b02786f8929fd9c900897fb.json','Clinical features of culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia','OBJECTIVE: This retrospective chart review describes the epidemiology and clinical features of 40 patients with culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia. METHODS: Patients with positive M. pneumoniae cultures from respiratory specimens from January 1997 through December 1998 were identified through the Microbiology records. Charts of patients were reviewed. RESULTS: 40 patients were identified, 33 (82.5%) of whom required admission. Most infections (92.5%) were community-acquired. The infection affected all age groups but was most common in infants (32.5%) and pre-school children (22.5%). It occurred year-round but was most common in the fall (35%) and spring (30%). More than three-quarters of patients (77.5%) had comorbidities. Twenty-four isolates (60%) were associated with pneumonia, 14 (35%) with upper respiratory tract infections, and 2 (5%) with bronchiolitis. Cough (82.5%), fever (75%), and malaise (58.8%) were the most common symptoms, and crepitations (60%), and wheezes (40%) were the most common signs. Most patients with pneumonia had crepitations (79.2%) but only 25% had bronchial breathing. Immunocompromised patients were more likely than non-immunocompromised patients to present with pneumonia (8/9 versus 16/31, P = 0.05). Of the 24 patients with pneumonia, 14 (58.3%) had uneventful recovery, 4 (16.7%) recovered following some complications, 3 (12.5%) died because of M pneumoniae infection, and 3 (12.5%) died due to underlying comorbidities. The 3 patients who died of M pneumoniae pneumonia had other comorbidities. CONCLUSION: our results were similar to published data except for the finding that infections were more common in infants and preschool children and that the mortality rate of pneumonia in patients with comorbidities was high.',' Mycoplasma pneumoniae common cause upper low respiratory tract infection remain frequent cause atypical pneumonia particu larly young adult highly transmissible infection cause organism relatively minor include pharyngitis tracheobronchitis bronchiolitis croup fifth fection asymptomatic infected subject develop symptom consistent bronchopneumonia mortality infection rare organism fastidious difficult grow culture diagnosis infection cause organism usually confirm serological test polymerase chain reaction gene amplification technique King Abdulaziz University Hospital KAUH Jeddah Saudi Arabia facility perform Mycoplasma culture available January publish information concern pneumoniae infection Saudi Arabia scarce wish study epidemiology clinical feature cultureproven infection cause organism hospital KAUH tertiary care teach hospital bed capacity bed annual admission patient patient pneumoniae positive culture respiratory specimen identify month period January December review study period respiratory specimen sputum nasopharyngeal aspiration endotracheal secretion bronchoalveolar lavage pneumoniae culture obtain patient upper low respiratory tract infection see inpatient outpatient emergency department respiratory specimen aslo Gram stain culture bacteria virus pneumoniae serological test IgG IgM available KAUH study period positive culture result obtain Microbiology laboratory record chart patient review standardized data collection information collect include patient demographic comorbiditie clinical manifestation complication outcome pneumoniae culture classic pneumoniae agar medium agar Pneumofast tray Pneumofast International Microbio Signes France Specimens process accord instruction manufacturer agar Pneumofast tray incubate anaerobically inspect daily week organism identify base typical colonial morphology granular colony rarely fry egg like diameter agar medium change Pneumofast broth color red orange yellow glucose fermentation absence turbidity broth antibiotic sensitivity profile Pneumofast tray identification accord instruction manufacturer bacterial viral culture perform standard method pneumoniae isolate consider community acquire recover unhospitalized patient hour admission hospital nosocomial recover period Pneumonia diagnose base clinical symptom sign radiographic evidence pneumonia possible severe pneumonia define pneumonia associate tachycardia tachypnoea minute hypotension systolic blood pressure mmHg hypoxemia arterial oxygen partial pressure kPa oxygen saturation area consolidation outcome patient pneumoniae infection classify category uneventful recovery recovery follow complication death pneumoniae infection death unrelated pneumoniae infection Statistical Package Social Sciences spss program datum analysis Comparison categorical data Chi square statistic Fisher exact test small expect value total respiratory specimen patient positive pneumoniae month study period demographic epidemiological characteristic patient summarize Table isolate community acquire nosocomial thirty patient require admission hospital remain treat outpatient isolate associate pneumonia upper respiratory tract infection bronchiolitis case pneumonia confirm radiologically remain diagnose clinically case bronchiolitis occur child year old thirty patient comorbiditie patient cardiopulmonary comorbiditie asthma lung fibrosi congestive heart failure congenial heart disease patient immunocompromise malignancy steroid therapy human immunodeficiency virus infection patient comorbiditie premature newborn myelodysplastic syndrome myelopro liferative disorder sickle cell anemia Evan syndrome syndrome sarcoidosis demyelinate disease cerebral palsy spinal muscle atrophy organism concomitantly isolate pneumoniae respiratory tract include herpe simplex virus type occasion adenovirus occasion cytomegalo virus occasion respiratory syncytial virus occasion bacterial isolate occasion acinetobacter specie enter obacter cloacae manifestation associate pneumoniae infection summarize Table Pneumonia common upper respiratory tract infection versus respectively immunocompromise patient likely present pneumonia oppose upper respiratory tract infection bronchiolitis non immunocompromise patient versus similarly tendency patient year age old present pneumonia frequently versus patient clinically radiologically confirm pneumonia crepitation bronchial breath sound physical examination patient wheeze detect know asthma obstructive airway disease table patient pneumonia admit hospital comorbiditie patient upper respiratory tract infection patient bronchiolitis patient uneventful recovery patient pneumonia uneventful recovery recover follow complication acute respiratory distress syndrome respiratory failure septic shock die pneumoniae infection die underlie comorbiditie patient die pneumoniae pneumonia comorbiditie congestive heart failure second congenital heart disease old infant bear prematurely week gestation previously episode pneumonia pathogen Mycoplasma pneumoniae common cause atypical pneumonia accounting community acquire pneumonia study child acute respiratory tract infection Riyadh Saudi Arabia Mycoplasma pneumoniae find second common causative agent respiratory syncytial virus RSV accounting case study adult patient community acquire pneumonia admit military hospital Riyadh Saudi Arabia organism account case retrospective study pneumonic episode adult patient Al Qassim area organism account episode organism cause relatively minor infection pharyngitis tracheobronchitis bronchiolitis croup transmit person person infected respiratory droplet close contact common school aged child military recruit college student case occur singly family outbreak large outbreak occur closed population military recruit camp boarding school infection occur frequently fall winter temperate climate develop year round average incubation period week follow exposure rare complication protean involve virtually organ system respiratory system pleurisy pneumothorax acute respiratory distress syndrome lung abscess hematologic system hemolytic anemia intravascular coagulation thrombocytopenia dermatologic system maculopapular urticarial rash erythema multiforme erythema nodosum musculoskeletal system myalgia arthralgia arthritis cardiovascular system pericarditis myocarditis nervous system meningoencephalitis Guillain Barre syndrome neuropathy acute psychosis eye optic disc edema optic nerve atrophy retinal exudation hemorrhage immunity follow infection long lasting study infection affect age group common infant preschool child common adult age year elderly year age contrast datum temperate country infection common school aged child young adult possible explanation difference infant preschool child severe infection school aged child young adult prompt presentation group hospital infection occur year round common fall spring common summer infection community acquire half patient present pneumonia present upper respiratory tract infection immunocompromise patient patient year age old likely present pneumonia oppose upper respiratory tract infection non immunocompromise patient year age cough fever malaise common present symptom Cough usually dry slightly productive white sputum mild moderate severity febrile patient mild erate fever high grade fever rare crepitation wheeze common sign Wheezes common patient history obstructive airway disease patient history patient bronchial breathing sign consolidation detect fourth patient pneumonia consistent know disparity clinical radiological sign pneumoniae pneumonia crepitation detect majority patient pleuritic chest pain pleural effusion rare half patient pneumonia uneventful recovery mortality pneumoniae pneumonia high occur patient underlying comorbiditie patient underlie comorbiditie die pneumoniae pneumonia relatively high complication rate mortality relate pneumoniae pneumonia likely selection bias patient pneumonia sick require admission hospital comorbiditie conclusion datum shed light epidemiology clinical feature pneumoniae infection saudi tertiary care center datum comparable country finding infection common infant preschool child school child young adult additionally mortality attributable pneumoniae pneumonia relatively high patient comorbiditie hope information assist clinician approach management respiratory tract infection','2001-07-04 00:00:00','http://dx.doi.org/10.1186/1471-2334-1-6',987,472),('d230ca9dcc54e1d5c0ed4503752774eb44711dff.json','Role of RNA helicases in HIV-1 replication','Viruses are replication competent genomes which are relatively gene-poor. Even the largest viruses (i.e. Herpesviruses) encode only slightly >200 open reading frames (ORFs). However, because viruses replicate obligatorily inside cells, and considering that evolution may be driven by a principle of economy of scale, it is reasonable to surmise that many viruses have evolved the ability to co-opt cell-encoded proteins to provide needed surrogate functions. An in silico survey of viral sequence databases reveals that most positive-strand and double-stranded RNA viruses have ORFs for RNA helicases. On the other hand, the genomes of retroviruses are devoid of virally-encoded helicase. Here, we review in brief the notion that the human immunodeficiency virus (HIV-1) has adopted the ability to use one or more cellular RNA helicases for its replicative life cycle.',' helicase enzyme separate energy dependent manner stretch duplexed DNA RNA singlestranded component currently base characteristic motif sequence comparison superfamily small family helicase identify superfamily contain helicase share seven recognize signature amino acid motif helicase characterize generally conserved motif protein largely bacterial bacteriophage protein currently caution helicase bona fide helicase function RNA translocase fulfill function remodeling ribonucleoprotein complex box related DEAH DExH DExD helicase numerous member ubiquitously present eukaryotic genome helicase share conserve motif commonly refer DExH family helicase human Arabidopsis Saccharomyces entity respectively differ dna helicase dexh protein dead helicase poor unwind long nucleic acid duplex well suited separate short RNA hybrid dead box protein bind high affinity RNAprotein complex exhibit little RNA sequence preference suggest specificity determinant dead helicase recognition protein factor regard well understanding role dead protein depend clear characterization respective interact protein precise substrate helicase await definition dead helicase generally think participate pleiotropically aspect RNA metabolism include transcription mrna splicing mrna export translation RNA stability mitochondrial gene expression example helicase attribute function include following play role RNA splicing chaperone RNAs nucleus cytoplasm serve translation mrnas helicase contribute mrna stability dead helicase act ribosome biogenesis regulation small nucleolar rna ribosomal RNAs rrnas interaction finally Neurospora Trypanosoma DEAD protein contribute mitochondrial gene expression Cryptococcus DEAD helicase require cryptococcosis pathogenesis dipteran Chironomus tentan use dead helicase regulate mrna transport nucleus cytoplasm polyribosome helicase significantly contribute normal cellular metabolism similarly essential operational answer appear qualified yes DEAD DEAH box helicase motif InterPro search EMBL ebi database match individual viral sequence entry find suggest virus evolve encode directly helicase helicase like protein strong biological evidence support importance helicase virus life cycle come virus RNA genome positive strand RNA viruse encode helicase helicase like open reading frame ORF aside RNA dependant RNA polymerase highly conserve viral sequence ubiquitous helicase find type virus example list Table direct mutagenesis study confirm helicase function biologically require replication virus include vaccinia virus poliovirus alphaviruse brome mosaic virus nidoviruse flavivirus case acquire immunodeficiency syndrome AIDS describe american homosexual man short year french american scientist confirm human immunodeficiency virus HIV causative agent AIDS ensue year million individual die AIDS currently million people worldwide infect million incremental AIDS death million new infection occur annually magnitude burden cast urgency medical research HIV retrovirus lentivirus genus RNA genome kilobase encode polypeptide major structural protein encode gene gag group specific antigen pol polymerase env envelope accessory protein Vif Vpu Vpr Nef regulatory protein Tat Rev primary translation product multiply splice mrna infect human cell macrophage integrate provirus host cell dna gene expression govern transcriptionally viral protein Tat binding nascent viral TAR RNA post transcriptionally second viral protein Rev association viral RRE RNA Tat Rev interact host cell protein transcriptional post transcriptional function encode RNA helicase finding suggest host cell RNA helicase involve reverse transcription RNA mrna transcription nucleus cytoplasm transport viral recent unexpected finding reveal possibility RNA helicase potentially contribute role particle assembly reverse transcription proteomic analysis Roy et al report DEAH protein RNA helicase RHA find associate Gag package virion rnadependent manner RHA knock cell particle produce cell significantly infectious appear compatible possible explanation conceivable RHA participate formation infectious virus particle shape Gag RNA interaction viral particle assembly bud failure RHA properly restructure viral RNP explain observed reduce infectivity second Roy et al report evidence particle contain RHA show reduce virionendogenous reverse transcriptase activity respect RHA assist reverse transcriptase efficiently copy RNA unwind RNA secondary structure promote interaction viral RNA nucleocapsid protein order assemble well reverse transcription complex separate reverse transcription unwinding highly structured RNAs reason important transcription direct evidence RNA helicase role somewhat elusive example seemingly support activity RNA helicase transcription vaccinia virus postulate NPH II helicase assist transcription strand separate duplexed RNA structure prevent loop formation elongate RNA polymerase second RHA invoke provide factorrecruitment role bridge promoter creb bind protein RNA polymerase II dead box helicase show recently novel transcriptional co activator transcriptional function interestingly instance ATPase helicase activity RHA apparently require attribute transcriptional role recent study provide clue RNA helicase serve co factor function transcription viral long terminal repeat LTR Fujii et al observe RHA conserve terminus double strand RNA bind dsRBD domain characterize previously TAR RNA bind protein TRBP investigator find reporter virus replication assay RHA activate TAR RNA bind dependant manner LTR direct transcription role transcription indirectly influence milieu polymerase II initiation elongation LTR Downstream transcription fate encode RNA regulate step export unspliced partially splice moiety nucleus cytoplasm unspliced partially splice viral RNAs code genomic RNAs package progeny virion structural protein egress RNAs nucleus cytoplasm critical life cycle virus exit HIV RNAs nucleus significant issue unspliced partially splice cellular mrnas routinely retain permit export nucleus large body work suggest elegant solution conundrum establish encode Rev protein bind highly secondary structured element Rev responsive element RRE present unspliced partially splice HIV transcript bind specifically distinguishe purpose nuclear export viral transcript cellular RNAs new evidence suggest RNA helicase co factor Rev direct export mrnas role transport unspliced incompletely splice viral RNAs nucleus Rev directly interact nuclear export receptor require Rev mediate export HIV RNAs recent report provide datum RNA helicase additional player Rev rre complex show expression enhance Revdependent export pathway nucleo cytoplasmic shuttle protein bind necessity Rev RRE function demonstrate knock cell endogenous finally locate nuclear pore complex NPC Yedavalli et al propose human helicase like analogous yeast function Rev remodel thread large unspliced RNAs nuclear pore facilitate final release cytoplasmic result consistent additional paper describe similar finding related RNA helicase Pomerantz co worker show bind directly terminus Rev RRE RNA motif participate export unspliced RNA nucleus cytoplasm additionally illustrate reduced expression astrocyte explain previously observe tissue restrict function Rev fully splice viral mrnas encoding viral Tat Rev Nef protein show previously exit nucleus cellular mrna export pathway export mrna require RNA helicase involvement export splice viral RNA fully clarify story RNA helicase likely complex unlikely conclude simply RHA figure rna extensively regulate splicing splicing multiple step process require recognition splice site spliceosome generally believe remodeling RNA RNA RNA protein interaction spliceosome catalyze family DEAD DExH box RNA helicase date seven mammalian protein RNA helicase implicate mrna splicing specific preference subclass RNA helicase viral mrna splicing remain clarify cellular RNA helicase contribute translation viral mrnas require investigation recently wout et al Krishnan Zeichner provide evidence expression cellular RNA helicase include modulate infection precise cellular role significance helicase pathogenesis characterize interestingly Krishnan Zeichner report microarray datum examine transition infection latency productive replication find cellular RNA helicase upregulate future understanding function important design experiment segregate helicase serve direct overlap redundant role participate indirectly viral life cycle convergence evidence support discrete cellular RNA helicase contribute importantly efficient execution step replicative cycle give AIDS disease burden reach pandemic global proportion new antiviral strategy target molecularly delineate mechanism virus urgently need possibility host cell helicase therapeutic target anti chemotherapy Implicit question concept attack host cell protein order treat infecting pathogen target cellular protein involve viral pathway risk obvious cytotoxicity approach avoid inherent problem pose rapid mutation currently utilize chemotherapeutic target virus encode protein note inhibition cell encode enzyme medical therapy unprecedented strategy suppression angiotensin convert enzyme ACE widely treat hypertension congestive heart failure myocardial infarction endothelial dysfunction renal disease aromatase inhibitor treat hormone dependent breast cancer inhibitor cellular secretory protease contemplate Alzheimer disease recently inhibit cellular polyprotein convertase furin minimal toxicity cell order block replication priori exclusion cellular helicase antiviral target warrant guard optimism small molecule helicase inhibitor develop virus arise encourage progress non retroviral system unlike human herpesviruse physically encode helicase herpe simplex virus gene helicase superfamily respectively HSV form heterotrimeric helicase primase complex responsible unwind duplex viral dna replication fork recent study provide proof concept HSV helicase primase target low host cell toxicity new class drug amino thiazolyphenylmolecule thiazole amide derivative addition study suggest protein RNA helicase encode Hepatitis virus relate West Nile virus Japanese Encephalitis virus target inhibit viral replication conceptual break drug development important indicate target discrimination different helicase small molecule inhibitor possible relevance synthetic immunomodulator Murabutide show recently suppress replication macrophage cell Murabutide show inhibit activity RNA helicase block positive transcriptional activity gene expression look signature motif conserve helicase clear different protein widely divergent code sequence principle suggest individual helicase abrogate specificity knowledge direct manner theory helicase attack inhibition NTPase activity direct competition NTP binding ii inhibition substrate binding direct competition active site iii allosteric mechanism affect NTPbinding NTP hydrolysis polynucleotide binding iv inhibition unwind activity steric hinderance helicase translocation polynucleotide substrate NTP bind substrate bind pocket sufficiently similar helicase specificity inhibition site likely extremely difficult impossible hand tremendous variation sequence size helicase oligomerization state discrete domain responsible protein protein interaction target specific nucleic acid differential localization cell offer interventional possibility outside NTP polynucleotidebinde site preliminary stage screen ringexpande nucleoside analog find previously successful NTPase helicase inhibitor West Nile virus Hepatitis virus japanese encephalitis virus observe candidate inhibitor substantial anti activity dose incur cytotoxicity cell treat tissue culture week study need conclude compound exert specific inhibition cellular helicase target altogether area cellular helicase activity likely intersect emerge research focus role small interfere RNAs micrornas miRNA innate cell defense virus include human cell precursor miRNA pre mirna process DICER Figure ribonuclease bona fide RNA helicase domain surprising recent finding reveal human TRBP show potent binder TAR RNA RNA indispensable dsRNA bind partner DICER allow associate pre mirna TRBP DICER miRNA processing activity lose intriguing scenario potentially unfold accordingly ribonuclease helicase protein DICER require TRBP process duplex structure mirnas order cell use mature mirnas antiviral defense speculate evolve restrict defense ability transcribe viral TAR RNA squelch trbp away DICER Bennasser Yeung Jeang manuscript submit figure thinking correct develop mechanism co opt active function virus propitious cellular helicase inactivate role second virus pernicious helicase DICER purpose selfish gain separate perspective virus infection trigger double strand viral RNAs innate antiviral immune response viral dsrnas recognize cellular protein pattern recognition receptor PRRs initiate antiviral response induce production variety cytokine include type interferon IFN IFN initiate additional inflammatory adaptive immune response recently DExD RNA helicase retinoic acid inducible melanoma differentiation associate gene identify suppressor viral replication bind virus associate dsRNA activate type interferon dependent antiviral immunity expression find enhance dsrna induced type interferon antiviral response currently remain speculative helicase like recognize dsRNA trigger innate immune response intriguingly report exist literature infection induce activation type interferon conclusion study helicase protein gain insight normal cellular metabolic process abnormal inherit human disease Bloom syndrome Werner syndrome Cockayne syndrome xeroderma pigmentosum disease mutation cellular helicase remarkably biology virus','2006-08-25 00:00:00','http://dx.doi.org/10.1093/nar/gkl398',1730,770),('d450fc8885843d48772df9a898552302f8c80b98.json','Draft versus finished sequence data for DNA and protein diagnostic signature development','Sequencing pathogen genomes is costly, demanding careful allocation of limited sequencing resources. We built a computational Sequencing Analysis Pipeline (SAP) to guide decisions regarding the amount of genomic sequencing necessary to develop high-quality diagnostic DNA and protein signatures. SAP uses simulations to estimate the number of target genomes and close phylogenetic relatives (near neighbors or NNs) to sequence. We use SAP to assess whether draft data are sufficient or finished sequencing is required using Marburg and variola virus sequences. Simulations indicate that intermediate to high-quality draft with error rates of 10(−3)–10(−5) (∼8× coverage) of target organisms is suitable for DNA signature prediction. Low-quality draft with error rates of ∼1% (3× to 6× coverage) of target isolates is inadequate for DNA signature prediction, although low-quality draft of NNs is sufficient, as long as the target genomes are of high quality. For protein signature prediction, sequencing errors in target genomes substantially reduce the detection of amino acid sequence conservation, even if the draft is of high quality. In summary, high-quality draft of target and low-quality draft of NNs appears to be a cost-effective investment for DNA signature prediction, but may lead to underestimation of predicted protein signatures.',' Draft sequencing require order base pair clone fragment genome determine usually time depth coverage position minimum degree draft accuracy information assemble contig fragment genome join lack sequence information gap contig generate high quality draft usually coverage optimal finish sequence gap ambiguous base call usually require coverage additional analysis manual orient contig relative close gap process call finish fact state define distinction draft sequencing avoidance significant human intervention computational tool capable automate finishing circumstance tabulate cost differential highquality draft versus finished sequence fold speed differential fold state cost differential modest fold case draft sequencing cheap fast expert debate finish sequencing necessary consider high cost set determine draft sequence datum adequate computational prediction dna protein diagnostic signature signature mean short region sequence sufficient uniquely identify organism species level false negative strain variation false positive cross reaction close phylogenetic relative addition dna signature require signature suitable TaqMan reaction compose primer probe desire limited fund facility sequence biothreat pathogen mean decision maker choose wisely organism sequence money time save result draft finish sequence enable target organism isolate target near neighbor NNs target sequence draft datum facilitate generation highquality signature detection tradeoff quantity quality worth Sequencing Analysis Pipeline SAP compare value finished sequence real draft sequence simulate draft sequence different quality computational prediction dna protein signature pathogen detection diagnostic marburg variola virus model organism analysis availability multiple genome organism hope variola serve guide make prediction online version article publish open access model user entitle use reproduce disseminate display open access version article non commercial purpose provide original authorship properly fully attribute Journal Oxford University Press attribute original place publication correct citation detail give article subsequently reproduce disseminate entirety derivative work clearly indicate commercial use contact bacteria genome substantially large cost sequence high virus Variola select good available surrogate bacteria time begin analysis double strand dna ii relatively low mutation rate like bacteria like RNA short dna virus high mutation rate high level variation iii long virus albeit short bacterial genome iv access genome sequence collaborator Centers Disease Control Prevention Atlanta GA finish genome available compare actual finished datum simulated draft datum recently fairly large number Bacillus anthracis genome available finish currently compare finish draft result bacterial genome sequence analysis pipeline use DNA protein signature pipelinesthe draft SAP simulation nearly identical finished genome describe previously SAP Figure perform stochastic Monte Carlo simulation include DNA Protein Signature Pipelines component summarize briefly necessary describe signature pipeline SAP clearly describe signature pipeline discuss step DNA Protein Signature Pipelines dna sequence alignment multiple draft genome require use WGASA software summarize component SAP present sap describe DNA Signature Prediction Pipeline describe detail find sequence region conserve target genome create consensus base multiple sequence alignment WGASA software analysis create alignment discuss DNA Signature Pipeline identify region unique target sequence consensus relative non target bacterial viral sequence Gb database frequently update NCBI GenBank sequence database source collaborator CDC USDA public source TIGR Sanger Institute Joint Genome Institute conserve unique region signature select base requirement particular technology case TaqMan PCR signature candidate proceed silico screening blast analysis look undesired inexact match undergo laboratory screening SAP run pool target genome pool NN genome collect random subsample target NN genome select pool subsample run DNA signature pipeline protein signature pipeline identify region conserve target genome unique relative non target genome unique region evaluate compare large sequence database currently available bacterial viral complete genome non redundant protein database exclude nn NN pool random subsample run SAP require run DNA protein signature pipeline different random sample generate range outcome plot range plot protein signature prediction SAP method previously describe detail follow briefly describe procedure target genome align WGASA set gene start end pair plus minus strand relative reference genome require imply code frame translation nucleic acid codon amino acid protein target organism genome correctly determine align genome nucleotide codon translate amino acid sequence base gene location conserve string amino acid target genome record conserve fragment compare NCBI GenBank non redundant nr database amino acid sequence unveil peptide unique target specie computation require peptide signature long amino acid sub string length amino acid conserve unique conserve unique peptide signature different protein result conserve unique peptide amino acid long open reading frame consider protein signature candidate signature peptide target antibody ligand binding develop use detection therapeutic vaccine signature region highly conserve specie likely functionally important organism survival reproduction signature land near protein active site develop therapeutic antibody ligand binding interfere protein function signature region consider vaccine target unique peptide evoke specific response host draft genome WGASA Genome Analysis Scalable Algorithms align multiple sequence available tool enable multiple sequence alignment draft genome capable align large genome WGASA require finished reference genome draft recently possible use DNA Signature Pipeline predict signature candidate draft genome capability invention software multiple sequence alignment draft genome complete genome WGASA develop David Hysom Chuck Baldwin Scott Kohn Computations directorate Lawrence Livermore National Laboratory design software close communication member bioinformatic team tailor need generate diagnostic forensic pathogen signature WGASA efficiently align large bacterial genome addition developer create parallel version run minute allow SAP simulation involve thousand call WGASA complete feasible time frame addition SAP analysis tool enable revisit signature prediction important organism food bear pathogen Listeria previously problematic sequence available draft format tool require complete finished genome number draft sequence base suffix tree algorithm require anchor identical sequence fragment user specify length find genome align level sequence conservation genome order discover anchor sufficient length bp present genome region anchor align tool clustalw HMMer algorithm function efficiently anchor frequent disperse genome provide coverage substitution deletion insertion gap sequence information contig result anchor absence genome region align clustalw slow memory intensive large amount sequence datum similar anchor base alignment algorithm WGASA dependent high degree co linearity input genome SAP DNA signature analyse operate follow available complete genome target gather pool total genome count call second pool create available NN complete genome total count sequence call select random sample size target nn range range run DNA Signature Prediction Pipeline sample signature prediction base conservation target strain uniqueness relative Gb database minus nn NN pool choose sample sample signature candidate predict target NN sequence rest closely relate organism database consider nn addition number TaqMan signature candidate fraction genome conserve target sequence calculate base combine result signature pipeline run random target sample size assess sequence datum value require approximate number signature candidate predict datum set target NN sequence analyze signature pipeline datum set yield few signature lack conservation uniqueness winnow away unsuitable candidate SAP perform Monte Carlo sample target NN genome run sample signature pipeline summarize result hundred signature pipeline run single plot cpu Sun server seven signature pipeline simulation run parallel require min viral genome SAP analysis dozen bacteria virus date total run time year operate parallel average pipeline run time process time span year span prediction generate different random sample genome illustrate range plot figure axis number represent number target strain incremental value integer represent number NN genome figure incremental value plot target sequence conservation number nn relevant protein analyse NN comparison describe outcome number signature candidate fraction target genome conserve plot axis horizontal line span range predict number signature fraction conserve random sample size median quantile range indicate colored short vertical line random sample target strain NN strain sequence chance number signature candidate sample equal quantile mark expect outcome reduction number signature candidate fraction genome conserve number target NN sequence simulation increase reduction conservation additional target reduction uniqueness additional nn SAP analyse protein proceed like dna signature random sample size target sequence generate range total number target sequence pool finish datum actual draft datum draft datum simulate describe align WGASA protein signature prediction pipeline run random sample range median quantile number protein signature candidate sample give target size plot range plot describe DNA SAP analyse examine effect number target number NN sequence discriminate sample zero nn range draw horizontal gray line range draw black line good estimate true value quality measure determine entire target NN pool represent vertical black line good estimate plus constant location vertical dash line select indicate reasonable distance true answer quantile range show black vertical tick mark protein SAP analysis investigate effect number target sequence compose list NN protein random temporary exclusion protein nr database estimate value NN sequence datum difficult automate rapid high throughput computation compare target protein protein nr regardless phylogenetic relationship target comparable DNA SAP result available NN datum sequence datum strain Marburg virus actual draft finished version isolate provide analysis colleague work Lawrence Livermore National Laboratory draft sequence coverage enable compare SAP result strain finished form identity sequence provide Appendix draft Marburg analysis select finished strain reference strain GenBank virus complete genome WGASA reference genome random sub sample draft genome Marburg organism obtain sufficient number draft genome SAP Monte Carlo simulation total simulation dna signature individual run DNA signature pipeline simulation protein signature perform Marburg finish draft datum require average min simulation finish sequence datum generously provide collaborator CDC variola major genome NN genome orthopox family sequence identity provide Appendix real draft datum available develop program simulate draft sequence finished sequence base guidance colleague involve sequence effort finish process Biology Biotechnology Research Program Lawrence Livermore National Laboratory outline draft simulator program randomly cut genome contig size randomly select exponential distribution stochastic simulation determine gap overlap contig size gap overlap sequence error simulate follow paragraph describe draft simulation process great detail end sequence simulate missing present accord random Bernoulli trial base probability gap sequence datum simulation randomly determine sequence missing size miss segment select randomly uniform distribution range minimum gap size maximum gap size length contig select randomly exponential distribution non zero minimum contig size maximum contig size fraction mean genome length specie mean exponential distribution specify fraction mean genome length random Bernoulli trial determine gap overlap second contig size gap overlap choose appropriate uniform distribution range gap size basis range overlap size basis size contig select exponential distribution describe additional contig simulate similar manner contig sequence error simulate base size contig base position end basis middle contig long double strand dna virus variola bacteria sequence error rate large beginning end contig middle small contig likely contain sequence error large contig contrast difference generate product Sanger sequencing employ small RNA DNA virus sequence error middle contig small viral draft genome specifically simulate draft RNA short dna virus simulator work minor modification parameter parameter specify simulate sequence error size cutoff small versus large contig ii probability error middle portion small versus large contig iii length contig end sequence accurate bacteria long doublestrande dna virus accurate small virus RNA viruse iv probability sequence error contig end sequence error particular base assume base randomly change basis equal probability additional feature add draft simulation tool stochastic feature incorporate capture main feature draft sequence produce datum suitable SAP analysis perform set analysis simulate variola draft set simulate variola draft run SAP follow parameter probability gap contig probability overlap contig minimum gap size gap uniform distribution bp maximum gap size bp minimum overlap overlap uniform distribution bp maximum overlap bp minimum contig size exponential distribution bp maximum contig size mean genome length bp mean contig size mean genome length bp cutoff size small versus large contig bp probability sequence error inside large contig probability sequence error inside small contig refer set simulation high probability sequence error low quality draft simulated variola draft run parameter sequence error rate dramatically low line error rate base Centers Disease Control Prevention CDC indicate draft variola genome probability sequence error inside large contig probability sequence error inside small contig probability sequence error contig end run refer low error rate highquality draft finally perform SAP run high error rate low quality simulated draft NN sequence intermediate quality simulate draft target genome follow probability sequence error probability sequence error inside large contig probability sequence error inside small contig probability sequence error contig end intermediate quality simulate draft consistent error rate draft sequencing cite literature parameter value specify SAP experiment simulate target sequence simulate draft NN sequence remain finish sequence second NN sequence convert simulate draft target sequence remain finish target NN sequence simulate draft second case nn run draft simulator time choose draft sequence location extent gap sequence error differ genome sample case target sequence align WGASA require sequence finished genome reference random sample pool target genome genome randomly select finished genome leave finished sequence genome sample replace simulated draft sequence run draft simulator alignment nn target draft sequence differ genome sample randomness draft simulation time run addition target genome choose finished reference genome differ sample target genome sample simulation replace simulated draft version actual finished sequence sequence align WGASA SAP process run describe total stochastic simulation experiment perform require min simulation simulation involve randomly select subset target NN sequence include simulate draft datum base finish genome align target sample finally run DNA Signature Pipeline combination examine finish variola finish NN ii draft variola finish NN iii finish variola draft NN iv draft variola draft NN draft run repeat low high sequence error rate combination iv run intermediate quality simulate draft variola low quality simulate draft nn total computational experiment finished simulate draft variola datum following function estimate viral sequence cost base discussion laboratory colleague involve sequence finish merely rough estimate actual cost sequence give organism differ substantially rule thumb calculation equation assume cost sequence virus decline sequence second subsequent isolate false assumption case isolate similar case new sequence divergent isolate different outbreak virus rapid mutation rate cost especially unlikely decline addition bp figure draft coverage range bp shotgun sequencing low bp primer walking work know primer site find new isolate finish time cost draft factor time draft bp finish bp equation reasonable estimate rapidly evolve sequence technology cost rough guide quickly outdated substantially expensive order fold generate draft compare finished sequence datum organism like Marburg virus accord estimate equation example sequence finish genome finish draft draft sequencing low quality Marburg cause dramatic decline ability computationally eliminate region poor conservation exclude poor signature region figure occur gap draft datum sequence mask sequence variation strain good available datum finish genome sequence conservation deficiency draft datum false impression sequence conservation table additional finished genome reduce conserve fraction compare reduction genome draft datum overestimation conservation draft Marburg datum result overestimation number signature candidate figure sample draft target plus finished reference yield signature candidate small sample size draft target finished reference generate upwards candidate result differ finished genome lack sequence conservation evident zero TaqMan signature conserve strain combination finished genome sufficient eliminate nonconserved signature figure prediction signature candidate share finished strain argue TaqMan method fact information provide important guidance development TaqMan signature degenerate basis set signature combination pick sequence strain analysis indicate TaqMan signature conserve strain signature form minimal set detect divergent strain estimate sequence cost draft variola draft nn indicate draft require quarter half cost finish sequence simulation high quality draft datum indicate good finish datum diagnostic signature prediction conservation range plot figure virtually identical finished high quality draft indicate genome conserve sequenced isolate intermediate quality draft figure conservation range plot similar finished sequence show genome appear conserve range plot number TaqMan signature candidate similar finished sequence datum high intermediate quality draft target high low quality draft nn figure contrast result high quality simulate draft actual Marburg draft simulation low quality variola draft target illustrate sequence conservation underestimate compare result finish sequence datum sequence error table Figure lowquality draft target appear genome conserve isolate low quality high error rate draft NN datum yield result similar high quality draft finish NN datum long target sequence information intermediate high quality figure andfigure NN sequence necessary ensure signature region unique NN datum low highquality draft finish simulation indicate low quality draft NN datum adequate predict dna signature long good quality target sequence datum result error NN sequence occur random location differ NN sequence long nn correct sequence eliminate non unique target region unique region target determine result illustrate figure emphasize datum table table show fraction target genome conserve number region contiguous base pair long number base pair large region section sufficient length possibly primer locate number region similar finished datum draft low error rate low quality draft high rate sequence error target datum give false impression few short region conserve suitable signature region actually case artifact result consequence order calculate conservation uniqueness affect signature predict conservation gestalt generate sequence alignment non conserved basis replace dot uniqueness calculate base perfect match bp long conservation gestalt large sequence database non target percent target genome conserve vary slightly run finish target sequence different genome randomly select reference strain multiple sequence non conserve basis conservation gestalt break region conserve fragment basis long result short fragment test uniqueness consequently low level conservation overestimate fraction genome unique example Table fraction finish variola target datum overestimate low quality draft artifact affect TaqMan signature prediction region suitable primer probe contiguous conserve basis test uniqueness underestimation uniqueness conserve fragment bp length underestimate uniqueness predict signature work eliminate issue future version software protein result large disparity finish draft datum protein signature candidate finish Marburg datum compare protein signature candidate Marburg draft figure variola available target datum protein signature predict finish low error intermediate error high error draft target datum respectively Figure sequence error substantially reduce detection amino acid sequence conservation sequence error occur low rate genome pattern additional sequence reduce number protein signature candidate differ draft compare finished sequence datum finished datum large range number peptide signature candidate predict few variola genome range narrow lower bind genome genome quantile mark approach final predict number signature Figure pattern indicate set peptide highly conserve currently sequence variola unlikely erode sequence datum obtain word additional sequence datum probably need time order computationally predict good peptide signature target finished target sequence likely adequate generate list peptide signature draft datum contrast low high quality mask pattern figure range quantile number peptide signature gradually decline additional target sequence sudden sharp drop see finished datum suggest additional target sequence continue erode number peptide signature occur sequence error occur random different location draft target genome obscure truly conserve peptide falsely infer peptide SAP result base draft datum additional sequencing variola major genome useful generate peptide signature candidate actuality SAP analyse finished sequence datum indicate ample sequence datum peptide signature prediction failure draft sequencing Marburg coverage simulate variola draft high error rate facilitate prediction detection signature highlight need finish viral sequence draft high quality large number signature candidate fail screening incorrectly designate conserve strain observe Marburg result region classify conserve observe variola consider signature variola simulation intermediate high quality draft low error rate approximate observe coverage target NN genome deliver virtually result finish genome consider cost approximately time generate finished sequence draft analysis indicate invest high quality draft target genome well invest few finished genome analysis target strain finish remain target sequence nn provide draft result indicate NN sequencing low coverage low quality detriment signature prediction long NN draft genome sequence high quality draft sequence appear little sequence conservation target strain relax specification conservation strain diagnostic signature prediction calculation indicate possible generate signature allow base consider conserve present fraction genome standard requirement conservation finish sequence datum ratio win option generate signature candidate highly divergent RNA virus finish sequence usually preference include degenerate basis especially basis heterogeneity strain give signature candidate ratio win approach particularly important generation protein signature candidate draft target datum severely compromise ability detect conserve string amino acid summary intermediate high quality draft sequencing target genome combine low quality draft sequencing close phylogenetic relative sufficient prediction dna diagnostic signature prediction protein signature candidate contrast require finish sequence avoid substantial underestimation conserve peptide region','2005-10-20 00:00:00','http://dx.doi.org/10.1093/nar/gki896',3234,718),('d47eb01f2085b582008bb1ff7c604355c5f4c582.json','FluGenome: a web tool for genotyping influenza A virus','Influenza A viruses are hosted by numerous avian and mammalian species, which have shaped their evolution into distinct lineages worldwide. The viral genome consists of eight RNA segments that are frequently exchanged between different viruses via a process known as genetic reassortment. A complete genotype nomenclature is essential to describe gene segment reassortment. Specialized bioinformatic tools to analyze reassortment are not available, which hampers progress in understanding its role in host range, virulence and transmissibility of influenza viruses. To meet this need, we have developed a nomenclature to name influenza A genotypes and implemented a web server, FluGenome (http://www.flugenome.org/), for the assignment of lineages and genotypes. FluGenome provides functions for the user to interrogate the database in different modalities and get detailed reports on lineages and genotypes. These features make FluGenome unique in its ability to automatically detect genotype differences attributable to reassortment events in influenza A virus evolution.',' infection influenza virus continue public health problem cause seasonal epidemic sporadic devastating pandemic year influenza epidemic cause hospitalization result influenza relate death influenza pandemic infrequent result high mortality estimate million people kill worldwide influenza pandemic current level pandemic alert high level phase recent pandemic virus belong family Orthomyxoviridae classify type base identity major internal protein antigen Influenza virus infect multiple mammalian specie influenza virus exclusively human pathogen influenza virus cause great morbidity mortality human interestingly large pool influenza virus maintain horizontal spread wild aquatic bird virus normally cause disease food companion animal population poultry swine horse dog support sustained replication certain lineage influenza minimal lethal disease depend virulence strain influenza virus evolve association host different continent extended period time co evolution result extensive genetic divergence extant virus currently available analysis influenza virus classify subtype basis antigenic analysis hemagglutinin ha neuraminidase NA glycoprotein far ha subtype NA subtype find recent year gene sequence available large number viral strain create diverse pool influenza virus historical current isolate collect multiple geographic region Comparison deduce amino acid sequence ha NA reveal excellent agreement result cluster virus antigenic reactivity sequence similarity molecular genetic analysis allow comprehensive analysis entire viral genome gain popularity practical laboratory method classification importantly study influenza genomic structure genotype reveal mechanism virus evolution spread disease pathogenesis influenza genome consist negativestranded RNA segment encode viral protein viral genome evolve accumulation mutation viral RNA dependent RNA polymerase lack proofread ability reassortment entire gene segment force select viral variant neutralizing antibody response vertebrate host specie relate structural variation promote rapid evolution segment evolve different rate subject differential selective pressure functional constraint segmented nature viral genome allow segment exchange term reassortment distinct virus co infect cell generate progeny mixed genome reassortment theoretically yield different combination gene segment parent virus comprehensive influenza genotype database search web tool genotype virus available unlike HIV HCV influenza virus segmented genome separate phylogeny analyze establish genotype approach problem genotype influenza virus analyze gene segment independently segregate gene segment subtype subsequently lineage genotype influenza viral strain sequential aggregate assign gene segment lineage nomenclature influenza viral genotype allow researcher unequivocally describe influenza viral genotype analyze compare communicate molecular epidemiology virus report define nomenclature influenza viral genotype describe web tool develop genotype influenza virus genome sequence tool facilitate identification reassortment event divergent lineage nomenclature convention routinely influenza research segment influenza genome number PA HA NP NA NS respectively ii currently allele ha gene term subtype likewise allele NA allele non structural NS protein influenza virus unusual genomic structure approach genotype problem analyze gene segment separately accord convention consider evolutionary rate vary segment segment define genotype sequential combination lineage segment genome letter assign lineage PA NP number follow letter assign lineage ha NA NS number represent subtype allele example genotype human seasonal subtype virus lineage lineage PA lineage ha subtype lineage follow convention numbering influenza genome segment nomenclature identify genotype reassortment easy task accomplish compare predict genotype genome classify previously genomic sequence influenza virus segment length download NCBI Influenza Virus Resource genome FLU alignment perform individual gene segment clustalw program MEGA software construct phylogenetic tree neighbor join method model select goal genotype method determine reassortment event gene segment non traditional host location occur lineage viral gene carefully determine detailed subsequently phylogenetic tree construct significant cluster segregate nucleotide difference distance assign lineage ii bootstrap analysis small set sequence value consider significant iii initial lineage evaluate nucleotide difference lineage strength bootstrap support iv approximately sequence lineage randomly select maximum likelihood ML analysis gene segment serotype ha NA allele NS MultiPhyl server lineage assignment influenza gene available public database upload Segment Table database describe subsequently FluGenome database contain table segment Genome Genotype Segment table contain information relate sequence include assign lineage strain segment serotype host country year GenBank accession number nucleotide sequence sequence length Genome table contain information complete genome include assign genotype accession number gene segment sequence available gene segment long sequence keep genome accession unique genotype store Genotype table total number genome genotype Genotype table create query Genome table distinct genotype host category create separate genome genotype include Human Hu Avian Av Swine Sw Equine Eq Canine FluGenome database update automatically night new sequence download NCBI Influenza Virus Resource genome add FluGenome database lineage information predict new sequence update Segment Genome Genotype table necessary sequence database script check information need update sequence entry flag validation web interface database implement LAMP strategy server Linux operating system Apache web server genotype database build mysql database management system PHP PERL code part web tool end program end interface JavaScript HTML sparingly end interface domain acquire provide access database web tool blast algorithm sequence comparison advantage fast computation accurate result detect local highly similar sequence region overcome inherent disadvantage global alignment algorithm parameter call coverage detect gene wide sequence similarity default threshold identify lineage set identity coverage user reset threshold allowable value blast result user submit query sequence sort identity coverage good result assign lineage query sequence result blast fall threshold lineage flag asterisk determine genotype complete partial influenza virus genome script execute establish lineage viral gene segment genotype create sequential incorporation lineage segment arrange convention show Table lineage meet threshold specify default identity coverage lineage assign asterisk indicate query sequence meet criterion new lineage blast result find blank lineage display segment belong know genotype genotype query genomic sequence provide output result genotype compare previously identify genotype Genotype database analysis reveal reassortment event host switching genotype determine FluGenome find Genotype database genome flag virus potentially new genotype viral genotype report new FluGenome simply result identification gene novel phylogenetically define lineage presence gene know lineage novel combination online tool present query option user enter gene segment genotype figure segment query determine Individual Gene Segment Lineage identify lineage viral gene segment interest example case input fasta file contain sequence correspond gene segment analyze data set gene simultaneously user enter number different gene segment provide sequence datum set separate fasta file genotype query Determine Genotype analyze incomplete complete genome sequence genome separate fasta file alternatively user cut paste sequence genome time multiple genome analyze simultaneously nearly sequence collect public database lineage analyse result lineage viral gene segment show wide range diversity ha partition lineage mp seven table mine aforementioned sequence result complete genome consist unique genotype serotype Serotypes comprise different genotype Step enter number different gene segment analyzestep select gene analyzestep enter FASTA formatstep result page virus gene enter number influenza genome comparestep input genome FASTA formatstep result page propose nomenclature system name influenza viral genotype nomenclature exploit analyze complete viral genome nearly length length segment sequence reveal unique genotype FluGenome web server implementation include facility analysis sorting lineage genotype allow user explore evolutionary history viral strain particular FluGenome web server provide genotype information greatly facilitate inference genetic reassortment influenza virus','2007-05-30 00:00:00','http://dx.doi.org/10.1093/nar/gkm365',1118,463),('d4f0247db5e916c20eae3f6d772e8572eb828236.json','Automated identification of multiple micro-organisms from resequencing DNA microarrays','There is an increasing recognition that detailed nucleic acid sequence information will be useful and even required in the diagnosis, treatment and surveillance of many significant pathogens. Because generating detailed information about pathogens leads to significantly larger amounts of data, it is necessary to develop automated analysis methods to reduce analysis time and to standardize identification criteria. This is especially important for multiple pathogen assays designed to reduce assay time and costs. In this paper, we present a successful algorithm for detecting pathogens and reporting the maximum level of detail possible using multi-pathogen resequencing microarrays. The algorithm filters the sequence of base calls from the microarray and finds entries in genetic databases that most closely match. Taxonomic databases are then used to relate these entries to each other so that the microorganism can be identified. Although developed using a resequencing microarray, the approach is applicable to any assay method that produces base call sequence information. The success and continued development of this approach means that a non-expert can now perform unassisted analysis of the results obtained from partial sequence data.',' Task Task II Task III','2006-09-29 00:00:00','http://dx.doi.org/10.1093/nar/gkl565',5,3),('d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json','Globalization and Health','This debut editorial of Globalization and Health introduces the journal, briefly delineating its goals and objectives and outlines its scope of subject matter. \'Open Access\' publishing is expected to become an increasingly important format for peer reviewed academic journals and that Globalization and Health is \'Open Access\' is appropriate. The rationale behind starting a journal dedicated to globalization and health is three fold: Firstly: Globalization is reshaping the social geography within which we might strive to create health or prevent disease. The determinants of health – be they a SARS virus or a predilection for fatty foods – have joined us in our global mobility. Driven by economic liberalization and changing technologies, the phenomenon of \'access\' is likely to dominate to an increasing extent the unfolding experience of human disease and wellbeing. Secondly: Understanding globalization as a subject matter itself needs certain benchmarks and barometers of its successes and failings. Health is one such barometer. It is a marker of social infrastructure and social welfare and as such can be used to either sound an alarm or give a victory cheer as our interconnectedness hurts and heals the populations we serve. And lastly: In as much as globalization can have an effect on health, it is also true that health and disease has an effect on globalization as exemplified by the existence of quarantine laws and the devastating economic effects of the AIDS pandemic. A balanced view would propose that the effects of globalization on health (and health systems) are neither universally good nor bad, but rather context specific. If the dialogue pertaining to globalization is to be directed or biased in any direction, then it must be this: that we consider the poor first.',' secondly understanding globalization subject matter need certain benchmark barometer success failing health barometer marker social infrastructure social welfare sound alarm victory cheer interconnectedness hurt heal population serve lastly globalization effect health true health disease effect globalization exemplify existence quarantine law devastating economic effect AIDS pandemic balanced view propose effect globalization health health system universally good bad context specific dialogue pertain globalization direct bias direction consider poor pleased introduce Globalization Health peer review open access free end user journal début editorial briefly outline purpose scope journal highlight intention publish balanced mixture opinion subject journal Open Access entirely appropriate Knowledge good utility public good nonrival non excludable journal deal subject matter create global public good health virtue existence contribute process Globalization Health Open Access policy change way article pub lishe article freely universally accessible online author work read cost second author hold copyright work grant right reproduce disseminate article provide correctly cite error introduce copy text Open Access article permanently archive online repository separate journal Globalization Health article archive PubMed Central National Library Medicine text repository life science literature repository University Potsdam Germany INIST France Depot National Library Netherlands digital archive electronic publication importantly result publicly fund research accessible taxpayer access library subscription Open Access help increase public interest support research note public accessibility legal requirement USA propose Public Access Science Act law add country economy influence scientist ability access article resource poor country institution able read material wealthy one create access internet matter rationale start journal dedicate globalization health fold firstly globalization reshape social geography strive create health prevent disease determinant health SARS virus predilection fatty food join global mobility drive economic liberalization change technology phenomenon access likely dominate increase extent unfold experience human disease wellbeing secondly understanding globalization subject matter need certain benchmark barometer success failing health barometer marker social infrastructure social welfare sound alarm victory cheer interconnectedness hurt heal population serve lastly globalization effect health true health disease effect globalization exemplify existence quarantine law devastating economic effect AIDS pandemic balanced view propose effect globalization health health system univer sally good bad context specific extent individual state able engage process globalization term differ widely country child mortality example change quickly response subtle change purchase power impoverished community affluent community small change income little effect utility direction consider effect globalization wellbee apparent need consider long term scenario population immediate effect vulnerable population dependent fragile local economy dialogue pertain globalization direct bias direction consider poor','2005-04-22 00:00:00','http://dx.doi.org/10.1186/1744-8603-1-1',403,232),('d77233dd9085c9c76787c84ee9a1e249424b283a.json','Australian public health policy in 2003 – 2004','In Australia, compared with other developed countries the many and varied programs which comprise public health have continued to be funded poorly and unsystematically, particularly given the amount of publicly voiced political support. In 2003, the major public health policy developments in communicable disease control were in the fields of SARS, and vaccine funding, whilst the TGA was focused on the Pan Pharmaceutical crisis. Programs directed to health maintenance and healthy ageing were approved. The tertiary education sector was involved in the development of programs for training the public health workforce and new professional qualifications and competencies. The Abelson Report received support from overseas experts, providing a potential platform for calls to improve national funding for future Australian preventive programs; however, inconsistencies continued across all jurisdictions in their approaches to tackling national health priorities. Despite 2004 being an election year, public health policy was not visible, with the bulk of the public health funding available in the 2004/05 federal budget allocated to managing such emerging risks as avian flu. We conclude by suggesting several implications for the future.',' public health small component health system term budgetary allocation state national level term number practitioner incorporate myriad activity legislation regulation health protection preventive service direct specific disease population health promotion program gear particular risk factor vulnerable group community look like disparate collection program investment Australia confusion terminology public health despite extensive history global understanding Australia term variously refer publicly fund health service intervention regardless funding source aim primary prevention promotion protection public health rat drain lead increase number jurisdiction adopt label population renovation public health system international agenda year Institute Medicine release report public health workforce require st century challenge visit update landmark report Future Public Health Century UK follow path break review NHS Derek Wanless Treasury commission undertake review government effort public health arise challenge confront Canada outbreak sudden acute respiratory syndrome SARS new public health agency arm length government create public health Australia remain fragmented program jurisdiction particularly state territory systematic approach funding organisation conceptualisation gap rhetoric funding continue noticeable tension frame priority popular appeal versus technical language evidence base article examine indicative development public health Australia key development identify number select depth analysis article use traditional meaning term public health focus activity usually design promote protect health population driver development short term implication signpost future suggest early global anxiety SARS occupy headline February persistent popular headline focus obesity summit hold NSW Victoria National Obesity Taskforce convene auspice Australian Health Ministers Council Kay Patterson Federal Health Minister declare prevention fourth pillar Medicare want Minister Prevention federal budget limit contain bundle initiative entitle Prevention Health Agenda particular number immunisation health promotion program include significant funding initiative public health announce government support meningococcal vaccine culmination month careful planning perception exist occur considerable public interest anxiety death outbreak disease change recommend schedule Australian Technical Advisory Group Immunisation ATAGI particular inclusion pneumococcal varicella vaccine result similar prescribed vaccine program similar funding development review great detail section National Public Health Partnership NPHP AHMC adopt influenza pandemic plan October advent newly identify disease SARS outbreak meningococcal disease management prevention communicable disease prominent follow significant funding boost bioterrorism preparedness public health preparedness generic theme arrival SARS occupy national popular political imagination test infrastructure capacity public health Australia fare outbreak apart escape australian case provide opportunity establish coordinated approach Commonwealth state territory contribute global epidemiological investigation prevention effort SARS prompt amendment Quarantine Act recall follow Pan Pharmaceutical crisis Therapeutic Goods Administration TGA spotlight manage conclude negotiation train year Trans tasman regulatory regime authority regulatory Australian New Zealand Food Regulation Ministerial Council endorse nutrition health relate claim policy guideline establish review genetically modify GM labeling food development point global nature public health intersection public health activity economy policy development public health confine set health program lead take outside health sector significant adoption National Agenda Early Childhood push public health advocate child health mid National Public Health Partnership respond coordinate scoping child health strategy Australia Government promote Maintaining Good Health adopt National Research Priorities healthy ageing emerge policy theme Ageing Research public health workforce development pursue outside mainstream education training arrangement public health university Community Services Health Training Board commission consultative process develop population health competency Vocational Education Training VET sector new population health qualification competency propose incorporation Health Training Package certificate population health environmental health diploma population health indigenous environmental health release report return Investments Public Health epidemiological economic analysis refer Abelson report significant impact subsequent year commission year early Population Health Division Department Health Ageing DoHA report experience relatively low profile Derek Wanless visit UK having chair review contribute significant budgetary increase NHS Wanless commission British Treasury examine prevention government September meeting Canberra senior official key agency Wanless marvel value Abelson report describe detail election year public health policy visible campaign policy development generally Federal Government initiative wind National Occupational Health Safety Commission receive little publicity comment indicate Commonwealth increase tendency pursue pathway separate state territory bring function statutory body department jurisdictional annual report state territory multiple plan draft guideline meeting episodic training program broad range area health issue take jurisdiction tobacco control sexually transmit infection Aboriginal health vaccination innovative activity report jurisdiction new Health Impact Assessment Branch new public health training program Western Australia apparent consistency health priority nation apparent divergence interest state territory federal government prevention management overweight obesity agenda appear observer new issue arrival precede year intensive work NHMRC release Acting Australia Weight strategic plan prevention overweight obesity year abs publish finding National Nutrition Survey reveal man woman Australia overweight additional man woman classify obese furthermore overweight obesity common low socio economic group rural population immigrant group Aboriginal Torres Strait Islander ATSI people despite longstanding national cooperation nutrition day National Better Health Program late recent national cooperation physical activity public political imagination capture issue recast obesity focus particular childhood obesity follow NSW Childhood Obesity Summit late Australian Health Ministers agree national approach require establish National Obesity Taskforce NSW Health release response Summit recommendation support vast majority resolution Victorian Department Human Services hold summit Healthy Weight Future release Commonwealth NHMRC join release late clinical practice guideline general practitioner health professional specific vary major theme strategy capture Healthy Weight summarise Table Commonwealth strategy relatively weak intersectoral policy regulatory measure illustrative example contrast state level implementation NSW range school physical activity nutrition survey school canteen strategy negotiate Commercial Television Australia code practice advertising peak child view hour Commonwealth apparently choose consider exercise relevant taxation legislative power despite history health promotion point importance public policy measure health system examination manner obesity issue frame detail contain national strategy raise number issue frame issue obesity successful focus nutrition physical activity obesity gain traction term Commonwealth opt soft programmatic approach tackle obesity strong public policy measure taxation regulation demonstrate national leadership capacity absence strong public policy measure obesity regard largely health issue government issue Government wait oppose Global Strategy account strength industry number year public concern eat disorder arise promotion certain type body image obesity label backward step mental health return traditional image risk people include child label overweight obese stigmatise extent voice affected community incorporate development national strategy correlation obesity socioeconomic disadvantage propose strategy exacerbate inequality child target captive audience easy target evidence suggest good return investment term health gain manage demand health care system come focus appeal populist agenda simultaneously progress long term agenda tackle health inequality multi sectoral partnership triumph public health complex thread interwoven moment publicly enunciate agenda represent confluence number rationale new vaccine add schedule recommend vaccine Australians additional change schedule recommend polio immunisation change oral inject IPD vaccine discuss vaccine protect serogroup meningococcal disease strain pneumococcal disease chicken pox time recommend vaccine fund Government prior introduction vaccine quality information epidemiology burden disease cause infection extremely variable meningococcal disease notifiable year Australia cause serogroup Whilst serogroup predominantly occur young child new strain serogroup cause increase anxiety public health professional microbiologist staff accident emergency department intensive care unit course public medium cause anxiety health professional base fact new strain carry high fatality rate severe effect high proportion survivor attack rate small increase exponentially year reach important trigger point majority case healthy teenager young adult initial accelerated catch programme introduce teenager major risk group new conjugate vaccine introduce childhood schedule age age dose cost consider necessary protection serogroup disease pneumococcal disease notifiable short surveillance history certain locally epidemiologically speak risk group effect reason suppose different epidemiological pattern develop country pneumococcal disease think occur time meningococcal disease know carry major sequelae high case fatality rate time know common indigenous australian population attack rate year knowledge underpin decision target aboriginal people free vaccination soon new vaccine available unfortunately dose conjugate pneumococcal vaccine expensive protection young child bear brunt disease license give dose course make provision vaccine australian child prohibitively expensive Varicella predominantly childhood disease cause Herpes virus know herpe virus varicella zoster virus VZV notifiable Australia epidemiological population datum available reliable varicella vaccine available mid USA american routine immunisation schedule vaccine available Australia cost dose dose require protection Commonwealth provide periodic update Australian Standard Vaccination Schedule list vaccine provide appropriate cost Australians time differ National Immunisation Program recommendation meningococcal serogroup conjugate vaccine pneumococcal vaccine varicella vaccine inactivated polio inject vaccine recommend funding secure meningococcal conjugate vaccine continuation provision pneumococcal vaccine indigenous child result recommend pneumococcal varicella vaccine fund parent decide pay funding decision important implication vaccine protect recipient unpleasant life threaten disease view subscribe UK ethically child deny access parent inability pay vaccine subject cost benefit study generally favourable extremely favourable pro vaccination result table summarise model frame policy policy fund meningococcal serogroup vaccine build sustained program epidemiological evidence ethical decision making public support arguably hone public pressure pneumococcal disease varicella vaccination program support good local epidemiological evidence respectable level public awareness disease similar program sustained policy building support drive decision fund vaccine funding policy noteworthy mark departure previous policy recommend vaccine fully fund government national vaccination policy design advise vaccination policy maker practitioner date thinking optimal vaccination schedule australian child proscriptive unlike United Kingdom UK change add vaccine recommend schedule advisory matter question fund vaccination program decide separately cost benefit study indicate pneumococcal polysaccharide conjugate vaccine cost effective vaccine cost clearly affect ratio cost benefit greatly Varicella vaccine contentious disease severe old case possible result vaccination program increase old case severe disease whilst vaccine undoubtedly work consensus precisely vaccinate maximum population health cost benefit potential financial saving highly dependant vaccine cost cost preventive vaccine program curative medicine fund different source vaccine currently fund Commonwealth subsidise state accord local vaccination policy whilst cost cure case disease broadly fund Medicare private health insurance system saving Medicare health insurance fund result successful vaccination program automatically transfer Commonwealth fund vaccine program saving cost area little interest importance program area Government revise funding policy provide funding conjugate pneumococcal vaccine population immunisation program child seven year age specific people risk group commence January Australian Technical Advisory Group Immunisation ATAGI complete ministerial report varicella polio inject oral vaccination late possible program vaccine fund future Federal Budget paper state Government committed make disease prevention health promotion fundamental pillar health system evident subsequent budget Government Focus Prevention Package aim incorporate disease prevention core business primary health care system reflective public health agenda evolve national level package comprise largely range measure direct specific disease plus bundle initiative general practitioner refer primary health care health condition affect Australians breast cancer receive attention National Breast Cancer Centre fund develop partnership approach review dissemination new information information support management initiative rural woman diagnose breast cancer Hepatitis receive attention funding national education prevention project financial support offer SARS effort undertake state territory particular provide medical personnel international airport clear process assess priority broad band National Public Health Program flag purpose budget primary health care define general practitioner measure fund Lifestyle prescription help gps raise community awareness understanding benefit preventive collaborative approach learning train education support coordinated care plan people chronic terminal condition Involvement multidisciplinary case conference budget adopt comprehensive approach primary health care system community health service represent important arm delivery public health service responsibility state timetable renew Public Health Outcome Funding Agreements PHOFAs Commonwealth state territory raise mind stakeholder possibility Commonwealth adopt comprehensive strategic approach link public health primary health care funding stream judge actual quantum fund available budget element package actually receive additional funding show Table GP initiative previously cast improve primary health care subsequently package combination measure reflect tight fiscal climate little growth overall health budget portfolio package demonstrate relatively limited imagination support establish issue breast cancer repackage general practice measure train Medicare spending uncapped target public health program cap attain prevention dollar GP sector appear way grow dollar prevention consider consistent Ottawa Charter reorient health service GPs train population base approach practice simply provide new payment represent undifferentiated uncoordinated untargeted approach prevention limited support gps little monitoring measure unlikely translate improved health outcome fund Tough Drugs strategy announce outside Focus Prevention package source fact Tough Drugs responsibility Parliamentary Secretary require separate communication strategy Prime Minister strong personal interest illicit drug strategy range measure fund include introduction retractable needle syringe technology address problem relate increase availability use psycho stimulant establish research fund support alcohol drug workforce development need promote access drug treatment rural area tackle problem face drug user concurrent mental health problem certainly suggest government interest commitment illicit drug course Howard Government gradual process cast landscape interest group policy constituency strong support breast cancer zero tolerance illicit drug contrast sharply delay experience renewal National HIV Hepatitis Strategy new prominence give meningococcal vaccine child health obesity create space interest group framing shape nutrition physical activity lobby clinical interest bring picture development illustrate political consideration important determine public health interesting observe interest prevention outside health portfolio particularly Treasury motivate Intergenerational Report concern sustainability Medicare social economic cost burden arise age society help ensure interest Abelson Report country conduct research return investment prevention effort Australia praise Derek Wanless high level consultation complete analysis visit Canberra conduct review UK Treasury secure Good Health Population final report point Australia Netherlands country increasingly economic evaluation public health program interesting public health policy analyst Treasury official draw report future year future interesting focus high visibility program demonstrate short term economic return give election year political economy prevention program arguably focus future public health policy agenda provide Government opportunity gauge public reaction new positioning design election campaign appropriately case american emphasis preparedness appear resonate australian public way perspective public health policy advocate lesson draw Government response public health proposal shape understanding popular interest desire communicate directly general long term public health issue struggle gain support progress cleverly shape fit Government develop nurture new advocate particularly seek engage broad health system work medium partner adversariesthese lesson need learn quickly assist move forum public health policy debate public domain essentially house discourse politician researcher public health advocate engaged informed community take public health issue government likely respond','2005-04-06 00:00:00','http://dx.doi.org/10.1186/1743-8462-2-7',2279,975),('d882f79283751d6c7f2e48cb991ee8abbcd913ae.json','The Transmissibility of Highly Pathogenic Avian Influenza in Commercial Poultry in Industrialised Countries','BACKGROUND: With the increased occurrence of outbreaks of H5N1 worldwide there is concern that the virus could enter commercial poultry farms with severe economic consequences. METHODOLOGY/PRINCIPAL FINDINGS: We analyse data from four recent outbreaks of highly pathogenic avian influenza (HPAI) in commercial poultry to estimate the farm-to-farm reproductive number for HPAI. The reproductive number is a key measure of the transmissibility of HPAI at the farm level because it can be used to evaluate the effectiveness of the control measures. In these outbreaks the mean farm-to-farm reproductive number prior to controls ranged from 1.1 to 2.4, with the maximum farm-based reproductive number in the range 2.2 to 3.2. Enhanced bio-security, movement restrictions and prompt isolation of the infected farms in all four outbreaks substantially reduced the reproductive number, but it remained close to the threshold value 1 necessary to ensure the disease will be eradicated. CONCLUSIONS/SIGNIFICANCE: Our results show that depending on the particular situation in which an outbreak of avian influenza occurs, current controls might not be enough to eradicate the disease, and therefore a close monitoring of the outbreak is required. The method we used for estimating the reproductive number is straightforward to implement and can be used in real-time. It therefore can be a useful tool to inform policy decisions.',' new highly pathogenic strain avian influenza emerge poultry market Hong Kong subsequently emerge Vietnam time onwards rapidly spread globe likely endemic poultry part world onward transmission human present remain limited high case fatality rate people infect raise concern impact potential human pandemic whilst research planning currently underway contain outbreak human relatively little know extent infection poultry particular transmissibility highly pathogenic avian influenza poultry farm understanding vital limit potential human pandemic reduce extent infection poultry movement restriction culling vaccination avian influenza occur naturally wild water fowl usually low pathogenic version LPAI cause symptom mild disease poultry strain occur highlypathogenic form HPAI result devastating disease kill infected bird hour highly transmissible individual bird transmission flock keep different farm think occur movement infected bird equipment staff current evidence suggest air bear transmission long distance rare increase HPAI outbreak past year addition implication human health outbreak severe economic consequence affected country typical control measure HPAI poultry comprise swift isolation culling flock infected farm restriction movement farm increase bio security culling flock vicinity infected farm deplete susceptible poultry population vaccination couple strict surveillance programme demonstrate effective reduce risk outbreak reproductive number infected poultry farm define average number farm original infected farm infect start outbreak farm susceptible important measure overall transmissibility virus population determine self sustain epidemic occur importantly yield tool assess effectiveness control measure average point time infected farm infect farm epidemic continue average infected farm infect farm epidemic decline intervention measure apply point interpret sufficient control outbreak paper analyse publish datum outbreak HPAI commercial poultry industrialise country estimate farm farm reproductive number HPAI explore extent different intervention measure implement outbreak reduce reproductive number result analysis inform current planning outbreak HPAI similar commercial poultry sector analyse datum different outbreak HPAI occur past year industrialised country outbreak Italy outbreak Netherlands treat distinct outbreak geographic separation outbreak Canada figure show time course outbreak brief detail outbreak give Outbreak Italy Northern Italy experience number avian influenza outbreak onwards occur extremely dense poultry production area bird km involve significant number farm keep turkey specie know experimental study highly susceptible avian influenza furthermore region wetland rest site migratory waterfowl close proximity poultry industry likely lead multiple introduction wild bird host March LPAI detect farm keeping turkey outbreak control rigorously AI continue circulate December case HPAI find strict control measure implement include cull affect flock movement restriction pre emptive slaughter flock deem high risk LPAI circulate time confirmation HPAI delay disease infect number farm time control measure enforce result HPAI epidemic affect total flock LPAI HPAI epidemic last April involve total million bird epidemic Netherlands affect total commercial flock distinct geographical temporal cluster outbreak situate Gelderse Vallei dense poultry production area Netherlands million bird keep flock density km month outbreak infection pass Limburg dense poultry production area continue spread Gelderse Vallei HPAI confirm th February day clinical sign appear infected farm March early April total farm infect Limburg farm infect April early number control policy enforce stage st March movement poultry poultry product ban tracing dangerous contact initiate reinforcement strict bio security measure implement day later rd March culling infected farm initiate th March additional pre emptive culling farm km radius infected farm place extend km radius turkey flock km radius flock th April control measure insufficient hypothesize epidemic area finally come halt depletion susceptible flock culling million bird commercial hobby flock epidemic human infection report present conjunctivitis mild influenza like illness person die infection evidence limited human human transmission follow detection index case broiler breeder farm surveillance program initiate lead detection second case th March Fraser Valley south River Fraser declare Control Area restrict movement bird bird product equipment furthermore active surveillance undertake high Risk Region HRR km index case flock deem dangerous contact Surveillance Region SR km index identification infected farm bird HRR slaughter th March onwards fail stop transmission th April decide depopulate Control Area contain approximately million bird infected farm locate mainly distinct local cluster Control Area hypothesize long distance spread cluster bird equipment people movement farm densely populated area infect shed metre virus spread dust feather debris assume homogeneous mixing farm equally infectious time dependence infectiousness point infection identical estimate distribution generation time interval reproductive number individual farm time course epidemic follow method infected farm label order farm contract infection outside source infection time farm simple model neglect difference farm spatial location etc probability farm infect farm iswhere generation time distribution density define index unknown parameter vector assumption number farm infect farm reproductive number farm outbreak represent outcome random variablethat sum Bernoulli random variable expect valuenow denote infection tree define farm infect farm likelihood observe isbut unobserved sum possible infection tree obtain integrated set index integrate likelihood genuine likelihood multiplicative constant permit valid inference conditional outbreak size maximum likelihood ML estimateĥ obtain minimize twice negative log likelihoodmore detail ML estimation perform give Appendix assume generation time Weibull distribute densityand Minimization perform Downhill Simplex method code calculation give Code Code ensure global minimum reach procedure run different starting point investigate generation time distribution change control measure introduce extend model allow distinct parameter generation time control pre post Code Code improvement fit compare original model assess likelihood ratio estimation reproductive number Givenĥ estimate mean variance generation time distribution estimate reproductive number infected farm equation mean reproduction number subset infected farm calculate confidence interval reproductive number use approximation parametric bootstrap percentile interval method obtain proper parametric bootstrap interval involve generate infection time tree accord underlie epidemic model wish specify completely instead follow step approximation propose good approximation large step approximate generating realisation underlie epidemic process bootstrap sample parameter value conventional approximation sampling distribution ML estimator bivariate normal distribution mean andvariance covariance matrix base inverse observed information matrix Appendix detail code generate bootstrap sample give Code Code stage loosely think sample mean behaviour subgroup possible outbreak allow variability subgroup stage involve fix independently generate reproductive number farm accord model step approximate bootstrap sample reproductive number farm sample draw set reproductive number generate finally approximate ci give th th percentile bootstrap distribution second stage calculation approximate ci Code Code generation time define time infection farm time farm pass infection farm assume generation time distribution Weibull biologically plausible choice verify empirically assess robustness choice plausible choice gamma distribution result show following result alternative differ substantively show figure show estimate generation time distribution different outbreak parameter estimate detail Table estimate distribution differ substantially outbreak hypothesise generation time shorten measure place isolate infected farm allow different generation time distribution pre post control time period significantly improve model fit table upper bound range impact control measure effective reproductive number clearly see outbreak outbreak Italy Figure introduction rapidly reduce reproductive number hover threshold month finally die British Columbia figure control place detection ip estimate reproductive number remain high th March decision take cull high risk region estimate control activity follow decision effective reduce reproductive number result situation Gelderse Vallei Netherlands Figure differ initial control measure fail bring reproductive number reliably epidemic die end March depletion susceptible flock affected area control apply Limburg epidemic estimate case reproductive number reduce figure potentially effective control outbreak end epidemic late April coincide depletion susceptible flock possible epidemic take substantially long control large pool susceptible flock area estimate farm farm reproductive number prior intervention HPAI range remarkably consistent dataset estimate substantially low previously report dutch epidemic prior implementation control measure obtain estimate ci Gelderse Vallei ci Limburg significantly low previously report outbreak prior notification ci epidemic Gelderse Vallei demonstrate Figure substantial variation estimate individual reproductive number prior intervention addition previous study generation time estimate directly datum base observational experimental datum course infection farm estimate generation time region order day value previously assume infectious period define flock time detection culling plus additional day cover time infection detect bird infectious previously publish estimate assume long mean generation time lead high estimated reproductive number result show substantial difference estimate generation time distribution different outbreak whilst know experimental study course infection individual bird estimate generation time farm level difficult obtain surprising generation time differ outbreak plausible difference arise variation farming practice contact pattern farm addition latent infectious period determine generation time differ strain HPAI alternatively estimate bias assumption method particular assume dataset complete infected farm detect farm outbreak infect outside source undetected farm play role transmission substantially alter estimate generation time reproductive number particularly infection occur beginning end epidemic overall case sparse outbreak investigate occur dense poultry farming area difficult control control policy implement different outbreak similar comprise strict bio security measure movement poultry poultry product swift culling infected flock fail control epidemic additional pre emptive culling flock neighbourhood infected farm result demonstrate bio security measure movement restriction culling infected farm initiate early outbreak effect outbreak reduce reproductive number close threshold value additional pre emptive culling flock de population area need fully control outbreak current contingency plan HPAI outbreak Europe focus set control measure contain outbreak whilst difference farming practice country mean difficult predict measure sufficient new outbreak analysis suggest additional intervention require close monitoring outbreak couple quantitative estimation reproductive number need ensure additional measure require promptly implement method estimate reproductive number generation time parameter extension develop Wallinga Teunis SARS epidemic method require time series datum outbreak easily apply real time technically appropriate censoring term add likelihood account infection time occur straightforward application method describe estimate unbiased asymptotic sense datum spatial location infected farm available information easily incorporate estimate spatial transmission kernel improve estimation reproductive number approach successfully apply Foot Mouth epidemic UK work require explicitly incorporate miss datum likely strong influence estimate generation time reproductive number method particular importance estimate reproductive number outbreak HPAI Asia high general level poultry mortality case likely document Appendix Code Source code maximum likelihood estimation parameter generation time distribution show Figure Table Code Source code estimate confidence interval base uncertainty inherent estimation procedure uncertainty generation time distribution show Figure','2007-04-04 00:00:00','http://dx.doi.org/10.1371/journal.pone.000034',1620,594),('d90d787828b6f96b4a93eb6948d613293bc51bf9.json','Endogenous Cell Repair of Chronic Demyelination','In multiple sclerosis lesions, remyelination typically fails with repeated or chronic demyelinating episodes and results in neurologic disability. Acute demyelination models in rodents typically exhibit robust spontaneous remyelination that prevents appropriate evaluation of strategies for improving conditions of insufficient remyelination. In the current study, we used a mouse model of chronic demyelination induced by continuous ingestion of 0.2% cuprizone for 12 weeks. This chronic process depleted the oligodendrocyte progenitor population and impaired oligodendrocyte regeneration. Remyelination remained limited after removal of cuprizone from the diet. Fibroblast growth factor 2 (FGF2) expression was persistently increased in the corpus callosum of chronically demyelinated mice as compared with nonlesioned mice. We used FGF2(−/−)mice to determine whether removal of endogenous FGF2 promoted remyelination of chronically demyelinated areas. Wild-type and FGF2(−/−)mice exhibited similar demyelination during chronic cuprizone treatment. Importantly, in contrast to wild-type mice, the FGF2(−/−)mice spontaneously remyelinated completely during the recovery period after chronic demyelination. Increased remyelination in FGF2(−/−)mice correlated with enhanced oligodendroglial regeneration. FGF2 genotype did not alter the density of oligodendrocyte progenitor cells or proliferating cells after chronic demyelination. These findings indicate that attenuating FGF2 created a sufficiently permissive lesion environment for endogenous cells to effectively remyelinate viable axons even after chronic demyelination.',' central nervous system CNS demyelinate disease multiple sclerosis MS myelin damage impairs impulse conduction denude axon limited remyelination occur MS lesion typically insufficient prevent long term neurologic disability demyelinate event improve remyelination maximize functional recovery viable axon prevent associate axonal damage degeneration rodent model extensive spontaneous remyelination functional recovery possible episode acute transient demyelination proliferation immature cell differentiation myelinate oligodendrocyte require extensive remyelination similarly immature oligodendrocyte lineage cell persist adult human CNS proliferate increase number near MS lesion immature cell fail differentiate sufficiently remyelinate extent MS lesion repair capacity endogenous cell limit nonpermissive signal chronic MS lesion growth factor cytokine cell adhesion molecule regulate oligodendrocyte progenitor op differentiation remyelinate oligodendrocyte environment demyelinated lesion potential signal molecule examine vivo role endogenous fibroblast growth factor upregulate postnatal development acute demyelination remyelination acute demyelination null mouse exhibit enhance oligodendrocyte repopulation demyelinated lesion improve oligodendrocyte regeneration study vivo retroviral lineage analysis wild type versus null mouse demonstrate predominant effect vivo remyelination inhibition op differentiation current study examine effect op differentiation oligodendrocyte regeneration significant impact remyelination acute demyelination model rodent typically exhibit spontaneous remyelination robust prevent appropriate evaluation strategy improve remyelination current study challenge capacity endogenous cell regenerate oligodendrocyte remyelinate chronically demyelinate lesion induce active demyelination mouse add cuprizone diet return normal chow allow analysis spontaneous remyelination ongoing disease pathogenesis cuprizone reproducibly demyelinate corpus callosum mouse spontaneous remyelination greatly reduce chronic cuprizone demyelination contrast acute demyelination importantly chronic cuprizone demyelination axon remain viable remyelinate transplanted op cell chronic demyelination model current study demonstrate dramatically improve remyelination endogenous cell mouse genetic deletion compare wild type mouse expression chronic lesion limit remyelination attenuation generate sufficiently permissive environment spontaneous remyelination endogenous cell mouse breed maintain Uniformed Services University Health Sciences usuhs animal housing facility procedure perform accordance guideline National Institutes Health Society Neuroscience USUHS Institutional Animal Care Use Committee knockout mouse Sv Ev Black swiss genetic background obtain breed heterozygous pair provide Doetschman University Cincinnati knockout generate target deletion replace kb portion gene include bp promoter entire exon Hprt minigene ingestion result reproducible pattern extensive corpus callosum demyelination cuprizone treatment start week age male mouse cuprizone finely powdered oxalic Sigma Aldrich Louis MO thoroughly mix chow diet Harlan Teklad Madison WI available ad libitum mouse maintain cuprizone diet perfuse analysis return normal chow week week cuprizone ingestion mouse perfuse paraformaldehyde brain dissect overnight postfixation paraformaldehyde brain tissue cryoprotecte overnight sucrose freeze OCT compound immunostaine situ hybridization situ hybridization preparation digoxigeninlabele riboprobe perform previously detail antisense riboprobe detect mrna transcript proteolipid protein PLP gift Lynn Hudson National Institutes Health PDGF gift Bill Richardson University College London digoxigenin label riboprobe hybridize km thick coronal brain section Digoxigenin detect alkaline phosphatase conjugate sheep antidigoxigenin antibody Boehringer Mannheim Indianapolis follow reaction NBT BCIP substrate DAKO Carpinteria identify op situ Km coronal brain section immunostaine PDGF primary antibody rabbit polyclonal antibody gift William Stallcup La Jolla rat monoclonal antiPDGF antibody Pharmingen San Diego Donkey antirabbit IgG conjugate Jackson Immunoresearch West Grove PA detect PDGF detect biotinylate donkey antirat IgG Jackson Immunoresearch follow coumarin tyramide amplification New England Nuclear Boston immunostaine monoclonal antibody recognize myelin oligodendrocyte glycoprotein MOG hybridoma cell provide Minetta Gardinier University Iowa MOG immunolabeling detect donkey antimouse IgG conjugate Jackson proliferation estimate immunostaine antigen express nucleus actively divide cell absent recognize rat monoclonal antibody mouse antigen DAKO follow detection ABC elite kit diaminobenzidine DAB substrate Vector Labs Burlingame situ hybridization immunostaine result capture Spot ccd digital camera Spot advanced image acquisition software Diagnostic Instruments Sterling Heights MI Olympus microscope image prepare panel Adobe Photoshop San Jose compare cell density quantification perform investigator blind treatment condition cell express PLP mRNA quantify unbiased stereologic morphometric analysis Stereologer System Systems Planning Analysis Alexandria VA Analysis restrict corpus callosum region midline extend laterally cingulum Km thick coronal section Stereologer System speciman thickness contribute sample volume measurement reflect cell mm unbiased stereologic method appropriately condition relatively cell interest choose category quantification cell corpus callosum express PDGF require counting label cell Spot ccd camera software measure area sample result density unit cell mm corpus callosum myelination estimate MOG immunofluorescence detect Spot ccd camera Metamorph software pixel intensity value normalize section thresholde exclude value level immunoreactivity dorsal fornix select adjacent white matter tract demyelinate cuprizone percent area corpus callosum midline bilaterally point cingulum apex MOG immunoreactivity threshold level estimate myelinated area category analyze include tissue section mouse mouse condition large sample size note text figure legend way analysis variance ANOVA post hoc Tukey multiple comparison test determine significant difference stage disease progression treatment Unpaired Student test compare genotype nonlesione mouse significance genotype effect multiple treat ment condition calculate way ANOVA statistical comparison mouse different genetic background mouse cuprizone Demyelination provide Relevant Model Insufficient Remyelination characterize Repair Capacity Endogenous Cells mouse establish parameter analyze later manipulating capacity endogenous figure spontaneous remyelination corpus callosum compromise chronic demyelination mouse Corpus callosum myelination estimate immunofluorescence myelin oligodendrocyte glycoprotein MOG pixel intensity value normalize tissue section thresholde exclude value immunoreactivity dorsal fornix demyelinate cuprizone df percent area corpus callosum MOG immunoreactivity threshold level estimate myelinated area bar color indicate treatment condition cuprizone white acute cuprizone week gray chronic cuprizone week black section quantify mouse mouse condition acute cuprizone myelination return near nonlesione value cuprizone week cuprizone week recovery nonlesione level occur chronic cuprizone week cuprizone week value week cuprizone significantly different week cuprizone week cuprizone significant difference note asterisk comparison cuprizone oligodendrocyte lineage cell repopulate remyelinate chronically demyelinate lesion cuprizone model advantage ability stop active demyelination simply return mouse normal diet characterize mouse continuous cuprizone ingestion result persistent demyelination variable degree partial remyelination indicate regenerative potential endogenous cell mouse feed cuprizone week continuous cuprizone ingestion mouse spontaneous remyelination remain limited week recovery period Fig poor remyelination chronic demyelination contrast complete remyelination see recovery period acute demyelination week cuprizone follow week cuprizone Fig well evaluate underlie cause limited remyelination chronic demyelination characterize oligodendrocyte lineage cell response relative disease progression focus repair potential Oligodendrocyte OP population characterize multiple stage disease progression nonlesione cuprizone acute demyelination week continuous cuprizone chronic demyelination week continuous cuprizone recovery chronic demyelination week continuous cuprizone follow normal chow week situ hybridization identify oligodendrocyte express PLP mrna transcript Fig op cell express figure oligodendroglial repopulation corpus callosum compromise chronic demyelination mouse Quantification density oligodendrocyte corpus callosum mouse situ hybridization PLP mRNA identify premyelinate myelinate oligodendrocyte quantify unbiased stereology section mouse mouse condition bar color indicate cuprizone white acute cuprizone gray chronic cuprizone black oligodendrocyte density week cuprizone follow week recovery significantly nonlesione value condition significant difference note asterisk comparison cuprizone carrot comparison recovery stage week cuprizone platelet derive growth factor receptor PDGF mrna transcript Fig analysis oligodendrocyte population reveal ongoing partial regeneration chronic demyelination Fig oligodendrocyte density low initial week cuprizone acute phase week continuous cuprizone oligodendrocyte density continue severely reduce relative value nontreated mouse ongoing partial regeneration chronic demyelination indicate fact oligodendrocyte density week high initial week end chronic cuprizone treatment oligodendrocyte regeneration improve recovery period interestingly show Figure remyelination progress similarly week recovery period PLP mrna situ hybridization identify oligodendrocyte detect premyelinating myelinate oligodendrocyte PLP transcription precede myelin formation result indicate chronic demyelination proportion premyelinate oligodendrocyte generate recovery period fail differentiate myelinate oligodendrocyte op population dynamic dramatically different response acute versus chronic demyelination Fig response initial episode acute demyelination op population amplify fourfold Fig end chronic demyelination period op density greatly reduce remain low recovery period chronic lesion similar result observe op identification PDGF Fig week cuprizone cell mm week cuprizone plus week recovery figure corpus callosum myelination estimate immunostaine myelin oligodendrocyte glycoprotein MOG cuprizone treat mouse mouse corpus callosum show persistent demyelination cuprizone ingestion week week week cuprizone treatment return normal chow remyelination mouse significantly improve week cuprizone week cuprizone week recover nonlesione level cuprizone week cuprizone week contrast value mouse increase chronic demyelination week cuprizone week cuprizone week remain significantly nonlesione level cuprizone week cup cell mm week cuprizone plus week recovery cell mm week cuprizone treatment cell corpus callosum identify actively undergo mitosis DAPI counterstaining reveal mitotic chromatin figure actively divide cell immunostaine inset Fig immunostaine nuclear antigen broadly identify actively cycle cell corpus callosum Fig label cycling cell present half density op cell cycling population relatively small significantly increase corpus callosum mouse feed cuprizone week compare age match control mouse recover week chronic demyelination finding demonstrate compromise continue OP cycling regeneration oligodendrocyte chronic demyelination subsequent recovery period Figs expression increase acute cuprizone demyelination inhibit op differentiation oligodendrocyte regeneration subsequent remyelination situ hybridization mrna transcript show elevate expression persist chronic cuprizone lesion compare nonlesione corpus callosum Fig mrna transcript abundance relate change protein detection develop white matter acute demyelination predict chronic lesion inhibition differentiation contribute limited capacity generate remyelinate oligodendrocyte Figs especially context reduce op pool Fig endogenous population efficiently remyelinate chronic cuprizone demyelination mouse Fig transplantation op cell show axon remain viable remyelination chronic demyelination model FIGURE oligodendroglial repopulation corpus callosum chronic demyelination dramatically improve mouse compare mouse Quantification density oligodendrocyte corpus callosum mouse situ hybridization PLP mRNA identify premyelinate myelinate oligodendrocyte quantify unbiased stereology section mouse mouse condition oligodendrocyte density week continuous cuprizone significantly nonlesione value note asterisk mouse condition week cuprizone subsequent week period recovery normal chow oligodendrocyte density mouse long significantly different compare nonlesion value week week condition Comparison value matching condition mouse demonstrate significant effect genotype knockout mouse test repair endogenous op population improve remove endogenous lesion environment extent demyelination week cuprizone ingestion similar mouse mouse wild type null genotype Figs percentage corpus callosum area myelin immunolabeling mouse mouse mouse chronic demyelination mouse myelin immunostaine corpus callosum significantly increase recovery period Fig surprisingly myelin immunostaining recover approximately nonlesion level week removal cuprizone diet mouse contrast mouse Fig mouse Fig myelinated proportion corpus callosum area significantly increase recovery period striking improvement remyelination correspond dramatic increase oligodendrocyte repopulation chronic lesion mouse find mouse Fig cuprizone treatment nonlesione mouse similar density oligodendrocyte corpus callosum week age correspond long cuprizone treatment recovery protocol addition cuprizone induce loss oligodendrocyte mouse similar end chronic cuprizone treatment conjunction quantitative myelin immunostaine analysis finding indicate mouse experience similar chronic disease severity importantly week recovery period chronic demyelination oligodendrocyte density return nonlesione value mouse mouse analysis additional intermediate recovery period mouse confirm increase oligodendrocyte regeneration demonstrate repopulation occur week removal cuprizone diet previously show absence significantly alter op amplification response acute cuprizone demyelination robust op proliferative response occur week cuprizone null wild type mouse similar previously mention datum mouse Fig op population deplete chronic demyelination contrast amplify op response acute demyelination Fig examine effect genotype density op cell cycling cell corpus callosum chronic demyelination genotype appear alter op population dynamic chronically demyelinate mouse age match nonlesione mouse Fig density op cell identify situ hybridization PDGF mrna similar mouse mouse Fig somewhat unexpectedly mouse null wild type genotype Sv Ev Black swiss genetic background exhibit increase op cell density week cuprizone observe mouse Fig strain difference op accumulation chronic demyelination correspond few oligodendrocyte observe recovery period mouse Fig compare mouse Fig background appear favorable background op differentiation oligodendrocyte regeneration detrimental effect chronic lesion environment overcome mouse immunostaine Fig indicate density cycling cell corpus callosum density op cell indicate relatively low level ongoing proliferation cell nuclear immunoreactivity observe doublet appropriate confirm detection cycling population density cell immunolabele nuclear antigen increase chronically lesione corpus callosum return nonlesione level recovery phase density endogenous cell proliferate corpus callosum mouse similar value mouse Fig mouse Fig similarity mouse mouse analysis op cell cycling cell indicate dramatic difference observe oligodendrocyte remyelination result permissive effect removal op differentiation remyelinate oligodendrocyte MS lesion initial episode transient demyelination follow spontaneous remyelination remyelination typically fail recur chronic myelin damage current study advantage cuprizone model chronic demyelination focus compromise repair response importantly analysis mouse demonstrate possible overcome chronic lesion effect endogenous oligodendrocyte lineage cell increase remyelination specifically take previous work finding mouse indicate remove inhibition op differentiation promote spontaneous remyelination deplete pool endogenous progenitor persist chronic demyelination current finding mouse consistent previous report cuprizone treatment cause extensive demyelination corpus callosum mouse strain advantage model examine mechanism promote remyelination endogenous cell characterize endogenous oligodendrocyte lineage population response recovery period mouse study facilitate comparison study difference mouse strain influence response cuprizone overall cellular response CNS injury transient episode demyelination week cuprizone ingestion spontaneous remyelination effective corpus callosum contrast prolong period demyelination week continuous cuprizone remyelination remain limited progress recovery interval examine week limited remyelination correspond depletion op pool current study number oligodendrocyte increase somewhat recovery period corresponding increase remyelination corpus callosum take finding indicate deplete op pool limited capacity generate oligodendrocyte available op cell fail fully differentiate remyelinate oligodendrocyte environment chronic lesion lesion MS case immature oligodendrocyte progenitor premyelinate oligodendrocyte present relatively normal density efficiently remyelinate study important well understand regulation op pool size transition op cell premyelinate myelinate oligodendrocyte robust op proliferation response acute demyelination appear require spontaneous remyelination acute demyelination op pool dramatically amplify efficient differentiation mature oligodendrocyte require extensive remyelination PDGF AA ligand act PDGF activation important mitogen op amplification response acute cuprizone demyelination PDGF heterozygous mouse amplification op pool response acute cuprizone demyelination compromise correspond generation oligodendrocyte reduce cross PDGF heterozygous mouse mouse actually increase oligodendroglial repopulation lesion acute cuprizone demyelination acute chronic lesion environment removal promote generation oligodendrocyte op cell remyelination endogenous predict contribute proliferation neonatal adult op especially present combination PDGF AA base vitro study current analysis density op cell immunolabele cell similar wild type null genotype result consistent previous BrdU incorporation study indicate endogenous predominant mitogen op acute cuprizone model subtle change proliferation rate asymmetrically divide op pool relatively difficult detect small population cell prolong disease recovery course predict stimulate neural stem cell subventricular zone SVZ contribute repopulation corpus callosum lesion prolong demyelination period current study contribution cell derive SVZ evident dynamic op pool corpus callosum lack detrimental effect absence op pool indicate effect SVZ cell significant contribution remyelination overlying corpus callosum direct analysis response cell SVZ require determination addition endogenous elevation lesion elicit specific effect replicate vivo method elevate exogenous level effective neuroprotective growth factor diverse model CNS injury include stroke trauma excitotoxicity axotomy contrast expect protective effect neuron removal endogenous actually improve regenerative parameter lesion involve remyelination improve regeneration oligodendrocyte observe mouse acute cuprizone demyelination corpus callosum murine hepatitis virus strain demyelination spinal cord current study example peripheral nervous system knockout mouse prevent normal upregulation FGF receptor FGFR activation associate Schwann cell macrophage site sciatic nerve crush injury compare wild type mouse knockout mouse exhibit improved myelination increase axon diameter regeneration sciatic nerve crush result indicate detrimental effect endogenous remyelination outweigh protection acute axon damage lesion model improved remyelination nerve knockout mouse provide significant protection axon damage subsequent initial crush injury correlative support similar role chronic autoimmune model MS find study neural stem cell transplantation improve mouse experimental allergic encephalomye litis EAE transplanted mouse exhibit improved remyelination major contribution endogenous cell axonal loss lesion area neural stem cell transplantation associate significant decrease expression reduce astroglial scar formation EAE lesion level modulate scar formation demyelinated lesion expression correspond scarring nonscarre astroglial response form CNS injury clear FGFR type type mediate effect endogenous context demyelination remyelination differential effect different stage oligodendrocyte lineage occur differential expression activation FGFR isoform Oligodendrocyte lineage cell expression high affinity fgfr coreceptor vary developmental stage exposure furthermore expression multiple FGFR type significantly increase response demyelination differential effect base change interaction signal pathway regulate oligodendrocyte lineage differentiation myelination neuregulin interpret direct effect complicated neuron glial cell express multiple FGFR type near demyelinated lesion future study require identify specific signal component mediate endogenous signal vivo context demyelination remyelination MS lesion pathology heterogeneous effect oligodendrocyte lineage population vary dramatically Oligodendrocyte density severely compromise demyelinated lesion patient MS study imperative optimize regenerative response immature oligodendrocyte lineage cell population observe MS lesion target inhibitory signal upregulate lesion viable strategy therapeutic promote differentiation need near lesion expression report reactive astrocyte acute chronic MS plaque addition culture adult human brain white matter inhibit differentiation preoligodendrocyte oligodendrocyte strategy promote remyelination attenuate inhibition op differentiation advantage reagent modulate signaling currently develop angiogenesis cancer treatment vivo analysis absence detrimental normal adult disease process inhibition signaling acceptable demyelinate disease stage improve treatment feasibility disease MS unstable disease course treatment promote remyelination valuable complement strategy abrogate ongoing cause demyelination immunomodulatory therapy','2006-03-01 00:00:00','http://dx.doi.org/10.1097/01.jnen.0000205142.',2675,769),('dff6195f5e24ebfa2fb4060142611b48021f673c.json','Neutrophil elastase, an acid-independent serine protease, facilitates reovirus uncoating and infection in U937 promonocyte cells','BACKGROUND: Mammalian reoviruses naturally infect their hosts through the enteric and respiratory tracts. During enteric infections, proteolysis of the reovirus outer capsid protein σ3 is mediated by pancreatic serine proteases. In contrast, the proteases critical for reovirus replication in the lung are unknown. Neutrophil elastase (NE) is an acid-independent, inflammatory serine protease predominantly expressed by neutrophils. In addition to its normal role in microbial defense, aberrant expression of NE has been implicated in the pathology of acute respiratory distress syndrome (ARDS). Because reovirus replication in rodent lungs causes ARDS-like symptoms and induces an infiltration of neutrophils, we investigated the capacity of NE to promote reovirus virion uncoating. RESULTS: The human promonocyte cell line U937 expresses NE. Treatment of U937 cells with the broad-spectrum cysteine-protease inhibitor E64 [trans-epoxysuccinyl-L-leucylamido-(4-guanidino)butane] and with agents that increase vesicular pH did not inhibit reovirus replication. Even when these inhibitors were used in combination, reovirus replicated to significant yields, indicating that an acid-independent non-cysteine protease was capable of mediating reovirus uncoating in U937 cell cultures. To identify the protease(s) responsible, U937 cells were treated with phorbol 12-myristate 13-acetate (PMA), an agent that induces cellular differentiation and results in decreased expression of acid-independent serine proteases, including NE and cathepsin (Cat) G. In the presence of E64, reovirus did not replicate efficiently in PMA-treated cells. To directly assess the role of NE in reovirus infection of U937 cells, we examined viral growth in the presence of N-Ala-Ala-Pro-Val chloromethylketone, a NE-specific inhibitor. Reovirus replication in the presence of E64 was significantly reduced by treatment of cells with the NE inhibitor. Incubation of virions with purified NE resulted in the generation of infectious subviron particles that did not require additional intracellular proteolysis. CONCLUSION: Our findings reveal that NE can facilitate reovirus infection. The fact that it does so in the presence of agents that raise vesicular pH supports a model in which the requirement for acidic pH during infection reflects the conditions required for optimal protease activity. The capacity of reovirus to exploit NE may impact viral replication in the lung and other tissues during natural infections.',' Mammalian reoviruse prototypic member Reoviridae family include pathogenic rotavirus coltiviruse seadornaviruse orbiviruse virus share element replication cycle structural feature include non envelop multi layered capsid surround segmented dsrna genome human mammalian reovirus typically associate mild self limit enteric respiratory infection study neonatal mouse reveal reovirus spread distant tissue site immunocompromise host review factor determine reovirus cellular host range poorly understand reoviru attach cell interaction broadly express receptor subsequent step viral life cycle help regulate host range pathogenesis recent study suggest step proteolysis capsid protein cell culture step infection attachment cellular receptor interaction viral protein interact know receptor sialic acid junctional adhesion molecule follow bind virion internalize receptor mediate endocytosis Endocytosis essential step viral life cycle standard infection condition endosomal lysosomal compartment protease convert virion particle resemble vitro generate intermediate subvirion particle isvp uncoating intermediate typically prepare chymotrypsin trypsin lack cleave form study isvp isvp recoate recombinant outer capsid protein reveal play key role regulate reovirus cell entry interact protect control conformational status underlying penetration protein cell efficiently mediate degradation uncoating reovirus infection slow block cell productively infect particle lack vitro ISVP like particle generate variety protease addition chymotrypsin trypsin include proteinase thermolysin endoproteinase lys Cat Cat Cat work provide insight cellular determinant reoviru uncoating murine fibroblast reoviru entry well study cysteine protease Cat less extent Cat require removal aspartyl protease Cat virion disassembly murine fibroblast require acidic pH recently demonstrate reovirus uncoating macrophage like cell line mediate acid independent lysosomal cysteine protease Cat finding reveal different cell type distinct protease facilitate reovirus uncoating result suggest model infection cell acid dependent protease mediate removal cell require acidic pH maximal activity fibroblast cell acid dependent protease Cat Cat mediate removal infection acid dependent Cat express cell Cat maintain activity neutral pH insight analysis reovirus cell entry facilitate recent discovery activation Ebola virus glycoprotein depend activity acid dependent endosomal protease Cat Cat role specific intracellular extracellular protease play regulate reovirus tropism spread disease animal largely unknown murine intestinal tract pancreatic serine protease show mediate removal Reovirus naturally infect host respiratory tract protease describe effect respiratory tract elastase GenBank inflammatory serine protease chymotrypsin family predominantly express neutrophil NE play prominent role wound repair control microbial infection ne expression promote pathogenesis implicate smokeinduce emphysema respiratory syncytial viral bronchiolitis respiratory syndrome ARDS fact reovirus replication rodent lung cause influx neutrophil reovirus infection recapitulate ARDS lead ask NE mediate productive reovirus uncoating investigate reovirus infection monocyte like cell line know express NE experiment describe report demonstrate reovirus infection cell require cysteine protease activity block presence agent raise vesicular Studies protease inhibitor suggest absence cysteine protease activity NE largely responsible productive infection cell NE directly mediate removal reovirus virion resultant particle infectious require additional intracellular proteolysis datum raise possibility NE involve reovirus replication respiratory tract analysis viral replication cell treat Figure Analysis viral replication cell treat cell untreated black treat grey µM Cysteine protease activity assess fluorogenic substrate Phe Arg MCA Sigma plot arbitrary unit activity level treat cell low cell unit unit cell unit unit day visualize graph black bar grey bar cell treat µM prior infection cell infect reoviru strain Lang virion isvp MOI infectious virus present determine plaque assay cell monolayer time point represent mean derive independent sample virion ISVP establish condition lysosomal cysteine protease activity inhibit cell treat µM broad spectrum cysteine protease inhibitor protease activity assess Cat Cat specific fluorogenic substrate Phe Arg MCA analyze enzyme activity time point treatment typically pre treat cell inhibitor prior infection second time point viral yield quantify show Fig treatment µM completely abolish cysteine protease activity cell consistent previous finding completely block cysteine protease activity cell raw value provide illustrate relative difference Cat enzyme activity level cell fibroblast absence inhibitor Cat activity significantly low cell cell consequence high expression cell cystatin intracellular cysteine protease inhibitor specificity Cat papain compare reovirus replication treat cell cell pre treat infect Lang virion isvp multiplicity infection MOI result representative experiment show Fig absence cell support reovirus replication consistent fact cell express Cat expect block virion infection cell viral yield treat cell slightly reduce relative untreated cell isvp lack capsid protein replicate efficiently treat cell indicate µM toxic cell type result demonstrate productive infection cell Lang virion require activity sensitive papain like cysteine protease Acidic pH require productive reovirus infection murine fibroblast aciddependent protease Cat Cat mediate uncoating serine protease include NE metalloprotease function broad ph range gain insight nature promote reovirus uncoating cell investigate requirement acidic cell leave untreated pre treat presence absence bafilomycin Baf NH Cl agent raise vesicular pH block vacuolar pump act weak base respectively pre treatment cell infect Lang virion MOI viral yield determine day post infection representative experiment show Fig treatment Baf NH Cl inhibit viral replication cell yield reach log respectively furthermore agent little effect viral replication cell cell treat inhibit cysteine protease activity contrast Baf NH Cl completely block reovirus replication cell consistent requirement Cat mediate removal cell give reoviru uncoating essential step viral life cycle finding reveal non cysteine protease function neutral pH facilitate step cell treatment promonocytic cell phorbol ester derivative result differentiation macrophage like cell differentiation characterize major phenotypic change include increase expression urokinase plasminogen activator receptor upregulation collagenase activity significant decrease expression NE Cat predict PMA treatment decrease capacity reovirus virion replicate cell cysteine protease inhibit confirm significant decrease NE expression cell differentiate PMA cell treat nM PMA expression NE analyze immunoblotte show Fig NE express untreated cell expression dramatically reduce follow pmainduce differentiation examine effect cell differentiation reovirus infection PMA treat untreated cell leave untreated treat infect Lang virion isvp MOI yield measure result typical experiment show Fig absence PMAtreated cell permissive infection virion pma treatment decrease yield log relative untreated cell contrast PMA differentiate cell treat inhibit cysteine protease activity long support productive infection Lang virion result explain toxic PMA treat cell examine replication isvp presence isvp replicate high yield undifferentiate differentiate cell PMA induce differentiation cell cause substantial decrease NE expression result consistent hypothesis NE similarly regulate neutral protease facilitate productive reovirus infection promonocytic pre differentiate cell effect agent raise vesicular pH reovirus replication cell reovirus replication cell differentiate PMA Figure Analysis reovirus replication cell differentiate PMA Lysates generate cell untreated treat nM PMA resolve polyacrylamide gel electrophoretically transfer nitrocellulose filter filter subsequently incubate polyclonal goat antibody human NE Santa Cruz Biotechnology filter wash incubate secondary anti goat antibody conjugate horseradish peroxidase Santa Cruz Biotechnology protein band detect reagent generate chemiluminescent signal Amersham cell undifferentiated black bar differentiate PMA grey bar nM PMA leave untreated treat µM follow pre treatment protease inhibitor cell infect Lang virion isvp MOI viral yield quantify describe legend Fig directly examine capacity NE facilitate reovirus infection irreversible elastase inhibitor Ala pro val chloromethyl ketone inhibitor highly specific NE inhibit activity related serine protease Cat establish efficacy specificity inhibitor treatment experimental condition cell treat NE inhibitor Baf NH Cl activity NE cell lysate examine colorimetric substrate show Table NE inhibitor active time point cell treat specific inhibitor NE activity untreated cell contrast cell treat Baf NH Cl NE activity modestly reduce remain result consistent capacity NE function neutral verify specificity NE inhibitor examine effect Cat activity fluorogenic substrate Phe Arg MCA expect Cat activity completely inhibit largely unaffected NE inhibitor examine effect NE inhibitor reovirus replication cell pre treat presence absence NE inhibitor infect Lang virion isvp MOI quantify viral yield representative experiment show Fig consistent result show Fig virion replication block cell presence NE inhibitor yield significantly reduce isvp replicate high yield treat cell indicate combination inhibitor toxic cell result demonstrate NE play critical role reovirus infection cell cysteine protease inhibit NE like cellular protease express proenzyme activate pro region remove envision model NE facilitate reovirus infection cell NE directly mediate degradation lead generation ISVP like particle second NE act indirectly activate protease try distinguish model examine capacity purify NE directly mediate removal Lang virion vitro Purified Lang virion treat NE treat virus particle analyze SDS PAGE show Fig NE efficiently remove Lang virion little intact remain viral particle NE treatment completely remove underlie cleave fragment retain gel assay infectivity resultant particle plaque assay find NE treatment negatively affect titer Lang particle datum determine NE generate svp require proteolytic processing cell pre treat block cysteine protease activity infect MOI Lang virion isvp NE generate subviral particle NE SVPs viral yield determine expect block infection cell virion contrast isvp NE SVPs replicate efficiently presence cysteine protease inhibitor Fig virion disassembly cell require acidic pH examine capacity NE SVPs infect cell treat Baf NH Cl monensin agent raise vesicular pH distinct mechanism cell treat agent infect virion isvp NE SVPs MOI cell treat indicate inhibitor NE activity assess colorimetric substrate MeOSuc Ala Ala pro Val ρna percent activity relative untreated cell calculate Cathepsin activity assess fluorogenic substrate Phe Arg MCA percent activity relative untreated cell hour post infection cell lysate harvest expression reovirus non structural protein µns analyze immunoblotte Fig expect treat cell infect virion viral protein expression block contrast µns expression evident presence agent raise pH infection initiate isvp NE SVPs Fig result demonstrate NE directly mediate removal virion generate infectious particle require proteolytic processing acid dependent cysteine protease cell serine protease involve reovirus infection mammalian intestinal tract report pro vide evidence mediate uncoating promote infection cell expand range protease promote reovirus infection cell culture include NE cysteine protease Cat Cat Cat line evidence support notion protease expression cell specific host factor impact reovirus infection example reovirus strain inefficiently uncoated Cat replicate high yield macrophage report demonstrate PMA induce differentiation influence type protease mediate reovirus uncoating cell cell PMA treatment report increase Cat expression decrease expression serine protease NE Cat accordingly PMA induce cell culture differentiate reovirus infection sensitive cysteine protease inhibitor suspect Cat largely responsible uncoating PMA differentiate cell acid independent protease Cat play role currently address question analyze infection pmadifferentiated cell treat Baf NH Cl datum completely resolve question Cat express cell like NE regulate PMA treatment furthermore find vitro treatment reovirus virion purify Cat generate svp behave like NE SVPs infectious absence proteolytic processing datum show result experiment NE specific inhibitor suggest NE largely responsible resistant infection cell inhibitor report inhibit Cat independently confirm approach assess role Cat reovirus infection cell examine effect Cat specific inhibitor infection try inhibitor Cathepsin inhibitor Calbiochem find cytotoxic cell culture give NE Cat generate infectious reovirus svp work need order understand role protease play infection cell previously report virion uncoating mediate Cat require acidic pH result consistent acid independence Cat activity result Fig Fig reveal like Cat NE mediate infection acid independent manner finding provide support model requirement acidic pH reovirus infection cell type reflect requirement acid dependent protease activity cell requisite acid dependent aspect cell entry small effect Baf NH Cl resistant reovirus growth Fig reflect participation acid dependent protease Cat activation store azurophilic granule major source acid dependent hydrolase neutrophil granule contain contain lysosomal marker accessible endocytose fluid phase marker condition cellular stimulation NE release neutrophil degranulation cell surface expression induce PMA treatment study cell show NE predominantly retain intracellularly little activity present extracellular medium consistent unable generate ISVP like particle treatment virion culture supernatant datum show observation finding PMA treatment decrease capacity treat cell support reovirus infection lead favor model NE mediate virion uncoating cell culture occur intracellularly vivo number virus include dengue respiratory syncytial virus induce release cytokine serve chemoattractant neutrophil promote degranulation Reovirus replication rat lung result neutrophilic invasion study cell culture indicate reovirus infection induce expression capacity reoviru induce secretion vivo facilitate release neutrophilic lysosomal hydrolase include NE extracellular milieu report show mammalian reoviru utilize acid independent serine protease uncoating datum suggest vivo consequence reovirus induce expression generation infectious NE SVPs like isvp particle predict expand cellular host range infect cell restrict intracellular uncoating inflammation predict exacerbate reovirus infection promote viral spread future study mouse deletion NE gene require elucidate role protease play reovirus infection respiratory tract tissue finally give recent finding endosomal proteolysis Ebola virus glycoprotein necessary infection result raise interesting possibility NE neutrophil protease play role cell entry virus Furlong isvp prepare treat purified virion chymotrypsin describe Nibert protease activity measure describe previously sample freeze thaw time titrate plaque assay cell describe viral yield calculate accord follow formula log PFU ml hrlog PFU ml deviation sd analyze NE expression cell lysate generate cell treat untreated nM PMA describe analysis viral protein expression lysate equivalent cell run polyacrylamide gel transfer nitrocellulose membrane block overnight TBST contain nonfat dry milk ne expression analyze polyclonal antibody NE TBST Santa Cruz Biotechnology Inc Santa Cruz membrane wash TBST incubate horseradish peroxidase conjugate anti goat IgG TBST bind antibody detect treat nitrocellulose filter enhanced chemiluminescence ECL detection reagent Amersham expose Speed Blue ray film Henry Schein Melville plate plate prior infection Virus allow adsorb cell temperature virus bind cell internalize adsorption culture incubate fresh medium prior infection cell pre treat µM nM Baf µm monensin Sigma mM NH Cl instance inhibitor include post adsorption culture medium indicate time cell collect centrifugation wash twice chill PBS lyse TLB centrifugation remove cellular debris sample resuspend sample buffer protein sample represent cell analyze electrophoresis polyacrylamide gel transfer nitrocellulose membrane mM mM methanol Nitrocellulose membrane Bio rad Laboratories Hercules block overnight TBST mm Tris pH mM NaCl Tween contain nonfat dry milk rinse TBST incubate rabbit polyclonal antiserum TBST Membranes subsequently wash TBST incubate horseradish peroxidase conjugate anti rabbit immunogloblin IgG TBST Amersham Arlington Heights bind antibody detect treat nitrocellulose filter enhanced chemilumescence ECL detection reagent Amersham expose filter Speed Blue ray film Eastman Kodak Rochester virion incubate ml purified neutrophil elastase Calbiochem µl VDB reaction terminate add mM PMSF µm NE inhibitor reaction mixture particle run polyacrylamide gel stain Coomassie Brilliant Blue confirm removal viral infectivity determine plaque assay cell monlayer Purified Lang virion treat ml NE µl VDB time indicate reaction terminate describe verify removal protein particle separate polyacrylamide gel visualize Coomassie Brilliant Blue stain viral infectivity time point determine plaque assay cell monolayer','2005-05-31 00:00:00','http://dx.doi.org/10.1186/1743-422x-2-48',2311,640),('e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json','Immune reconstitution inflammatory syndrome (IRIS): review of common infectious manifestations and treatment options','The immune reconstitution inflammatory syndrome (IRIS) in HIV-infected patients initiating antiretroviral therapy (ART) results from restored immunity to specific infectious or non-infectious antigens. A paradoxical clinical worsening of a known condition or the appearance of a new condition after initiating therapy characterizes the syndrome. Potential mechanisms for the syndrome include a partial recovery of the immune system or exuberant host immunological responses to antigenic stimuli. The overall incidence of IRIS is unknown, but is dependent on the population studied and its underlying opportunistic infectious burden. The infectious pathogens most frequently implicated in the syndrome are mycobacteria, varicella zoster, herpesviruses, and cytomegalovirus (CMV). No single treatment option exists and depends on the underlying infectious agent and its clinical presentation. Prospective cohort studies addressing the optimal screening and treatment of opportunistic infections in patients eligible for ART are currently being conducted. These studies will provide evidence for the development of treatment guidelines in order to reduce the burden of IRIS. We review the available literature on the pathogenesis and epidemiology of IRIS, and present treatment options for the more common infectious manifestations of this diverse syndrome and for manifestations associated with a high morbidity.',' introduction ART lead significant decline aids associate morbidity mortality benefit result partial recovery immune system manifest increase lymphocyte count decrease plasma viral load initiation ART opportunistic infection OI HIV relate event occur secondary delayed recovery adequate immunity patient initiate ART experience unique symptom immune system recovery patient clinical deterioration occur despite increase Tlymphocyte count decrease plasma viral load clinical deterioration result inflammatory response dysregulation immune system intact subclinical pathogen residual antigen result clinical manifestation syndrome diverse depend infectious noninfectious agent involve manifestation include mycobacterial induce lymphadenitis paradoxical tuberculosis reaction worsen progressive multifocal leukoencephalopathy PML recurrence cryptococcosis pneumocystis jirovecii pneumonia PCP Cytomegalovirus CMV retinitis shingle viral hepatitis noninfectious phenomenon clinical deterioration occur immune recovery phenomenon describe immune restoration disease IRD immune reconstitution syndrome IRS paradoxical reaction give role host inflammatory response syndrome term immune reconstitution inflammatory syndrome IRIS propose widely accept term describe clinical entity possible infectious noninfectious etiology IRIS summarize Table date prospective therapeutic trial concern management IRIS conduct evidence management IRIS literature relate case report small case series report management practice provide reliable evidence safety efficacy approach merely guidance practice manage difficult condition severe case discontinuation ART possibility potential disadvantage therapy cessation development viral resistance AIDS progression consider despite numerous description manifestation IRIS pathogenesis remain largely speculative current theory concern pathogenesis syndrome involve combination underlie antigenic burden degree immune restoration follow HAART host genetic susceptibility pathogenic mechanism interact likely depend underlying burden infectious noninfectious agent elicit infectious noninfectious agent presence antigenic stimulus development syndrome appear necessary antigenic stimulus intact clinically silent organism dead die organism residual antigen iris occur result unmask clinically silent infection characterize atypical exuberant inflammation accelerated clinical presentation suggest restoration antigen specific immunity characteristic differentiate IRIS incident opportunistic infection occur ART result delay adequate immunity example IRIS response intact organism include limit unmasking latent cryptococcal infection infection Mycobacterium avium complex MAC frequently report IRIS symptom response previously treat partially treat infection include report clinical worsening recurrence clinical manifestation Mycobacterium tuberculosis TB cryptococcal meningitis follow initiation ART noninfectious cause IRIS autoimmunity innate antigen play likely role syndrome example include exacerbation rheumatoid arthritis autoimmune disease give role antigenic stimulus frequency manifestation IRIS give population determine prevalence opportunistic non opportunistic infection initiation mechanism receive attention involve theory syndrome precipitate degree immune restoration follow ART assess theory investigator examine association cell count viral load risk IRIS study suggest difference baseline profile quantitative viral load ART initiation rate change HAART IRIS non iris patient study demonstrate trend significant difference IRIS non iris patient immunological difference group difficult verify small number IRIS case lack control group alternative immunological mechanism involve qualitative change lymphocyte function lymphocyte phenotypic expression instance follow ART increase memory cell type observe possibly result redistribution peripheral lymphoid tissue phenotype prime recognize previous antigenic stimulus responsible manifestation IRIS see soon art initiation redistribution naïve cell increase think responsible later quantitative increase cell count datum suggest IRIS combination quantitative restoration immunity qualitative function phenotypic expression observe soon initiation purport pathogenic mechanism IRIS involve host genetic susceptibility exuberant immune response infectious noninfectious anti genic stimulus immune restoration evidence limited carriage specific HLA allele suggest association development IRIS specific pathogen increase level IRIS patient explain exuberant response mycobacterial antigen subject clinical IRIS genetic predisposition partially explain manifestation IRIS differ patient similar antigenic burden immunological response numerous description infectious noninfectious cause IRIS overall incidence syndrome remain largely unknown study date retrospective focus specific manifestation IRIS tuberculosis associate IRIS TB IRIS large retrospective analysis examine form IRIS patient exhibit disease episode initiation ART cohort analyse examine manifestation IRIS estimate patient initiate ART develop syndrome large retrospective study report patient tuberculosis avium complex Cryptococcus neoforman coinfection develop IRIS initiate factor identify development IRIS cohort include male sex short interval initiate treatment OI start ART rapid fall RNA ART ART naïve time OI diagnosis significant predictor include young age low baseline cell percentage low cell count ART initiation low cell ratio baseline note cohort differ substantially study population type IRIS TB IRIS examine make conclusion risk factor IRIS difficult clinical factor associate development IRIS present Table report describe different clinical manifestation IRIS continue appear expand clinical spectrum syndrome definition IRIS clinical suspicion disease specific criterion develop determine true incidence difficult take study suggest IRIS affect substantial proportion HIV patient initiate ART future epidemiologic genetic study conduct diverse cohort important determine importance host susceptibility underlie opportunistic infection risk develop order aid clinician management IRIS review epidemiology clinical feature treatment option common infectious manifestation IRIS additionally manifestation associate significant morbidity mortality CMV associate immune recovery vitritis IRV immune recovery uveitis IRU review treatment option evidence present disease specific guideline develop IRIS therapy base Rheumatoid arthritis systemic lupus erythematosus SLE Graves disease Autoimmune thyroid disease Mycobacterium avium complex Sarcoidosis granulomatous reaction mycobacteria tattoo ink Cytomegalovirus AIDS relate lymphoma Molluscum contagiosum genital wart Sinusitis Folliculitis ing evidence individualized accord severity presentation Mycobacterium tuberculosis TB frequently report pathogen associate IRIS Narita et al perform prospective study evaluate incidence paradoxical response patient TB therapy subsequently initiate ART HIV TB coinfecte patient undergo dual therapy develop paradoxical symptom frequency symptom group great observe HIV infect control receive TB therapy support role exaggerated immune system response pathogenesis syndrome retrospective study corroborate finding significant proportion HIV TB coinfecte patient undergo HAART symptom consistent IRIS estimate range association short delay TB treatment initiation art initiation area debate investigator find difference time TB therapy initiation ART IRIS non iris subject report significant difference group general IRIS occur subject initiate ART month TB therapy initiation base datum decision analysis ART initiation timing TB patient find high rate IRIS occur patient initiate ART month TB therapy initiation withholding defer ART month TB therapy associate high mortality scenario IRIS relate mortality future report large prospective observational cohort aid resolve difficult issue consist primarily case report TB IRIS affect central nervous system CNS pose unique problem availability ART increase endemic country incidence CNS TB IRIS increase clinician vigilant diagnosis common clinical manifestation TB IRIS fever lymphadenopathy worsen respiratory symptom pulmonary disorder new pulmonary infiltrate mediastinal lymphadenopathy pleural effusion common extrapulmonary presentation possible include disseminate tuberculosis associate acute renal failure systemic inflammatory response SIRS intracranial tuberculoma Pulmonary TB IRIS diagnose transient worsening chest radiograph especially old radiograph available comparison symptom nonspecific include persistent fever weight loss worsen respiratory symptom abdominal TB IRIS present nonspecific abdominal pain obstructive jaundice study TB IRIS occur month ART initiation case MTB associate IRIS median onset IRIS day range day case occur week initiation antiretroviral therapy study suggest onset mycobacterial associate IRIS relatively soon initiation ART clinician maintain high level vigilance period paradoxical CNS TB reaction describe hivnegative patient include expand intracranial tuberculoma tuberculous meningitis spinal cord lesion TB associate CNS IRIS report HIV positive patient compare non cns TB IRIS symptom tend occur later usually month ART initiation crump et al describe HIV seropositive patient develop cervical lymphadenopathy week ART month later CNS symptom associate expand intracranial tuberculoma appear initiation antituberculous therapy significant morbidity case illustrate importance maintain high clinical suspicion disease particularly endemic area treatment mycobacterial associated IRIS depend presentation disease severity patient present non life threaten presentation respond institution appropriate antituberculous therapy range life threaten presentation acute renal failure acute respiratory distress syndrome ARDS describe significant morbidity mortality morbidity mortality great resource limit setting limited management option exist pathogenesis syndrome inflammatory systemic corticosteroid nonsteroidal anti inflammatory drug NSAIDS alleviate symptom study therapy IRIS mention use corticosteroid variable anecdotally effective therapy range intravenous methylprednisolone mg hour prednisone mg day week practice reflect lack evidence control trial use anti inflammatory agent IRIS randomized placebo control trial examine dose prednisone mg kg day week follow mg kg day week mild moderate TB IRIS currently underway South Africa datum available reasonable administer corticosteroid severe case IRIS tracheal compression lymphadenopathy refractory debilitate lymphadenitis severe respiratory symptom stridor ARDS interruption ART rarely necessary consider life threaten situation HIV negative patient adjuvant corticosteroid use tuberculous meningitis provide evidence improved survival decrease neurologic sequelae standard therapy infectious etiology exclude standard antituberculous therapy initiate continue clinical situation dictate course corticosteroid therapy consider CNS TB IRIS continuation ART desirable discontinuation necessary unresponsive case present advanced neurological symptom addition TB atypical mycobacteria frequently report causative pathogen IRIS early observation involve atypical presentation Mycobacterium avium intracellulare MAC note zidovudine therapy report atypical presentation Mycobacterium tuberculosis MTB MAC increase frequency introduction protease inhibitor ART large cohort MAC remain frequently report atypical mycobacterium atypical mycobacteria rarely associate IRIS reference Table general MAC associate IRIS typically present lymphadenitis abscess formation suppuration common presentation include respiratory failure secondary acute respiratory distress syndrome ARDS leprosy pyomyositis cutaneous abscess intra abdominal disease involvement joint skin soft tissue spine study characterize time onset Mycobacterium associate IRIS study MAC lymphadenitis onset febrile illness sign IRIS occur day initiation antiretroviral therapy study median time interval start antiretroviral therapy development mycobacterial lymphadenitis day range day TB IRIS evidence treatment IRIS atypical mycobacterium scarce occasionally surgical excision profoundly enlarge node debridement necrotic area anecdotally report healing poor leave large persistent sinus needle aspiration option enlarge fluctuant symptomatic node treatment similar TB IRIS Mycobacterium tuberculosis IRIS pre art era CMV retinitis vision threaten disease carry high annual incidence significant AIDS associate morbidity introduction HAART Jacobson et al describe patient diagnose CMV retinitis week ART initiation speculate HAART induce inflammatory response responsible unmask subclinical infection addition classical CMV retinitis ART lead new clinical manifestation infection term immune recovery vitritis IRV immune recovery uveitis IRU patient previously diagnose inactive AIDS relate CMV retinitis distinct minimal intraocular inflammation classic CMV retinitis manifestation exhibit significant posterior segment ocular inflammation think presence residual CMV antigen protein serve antigenic stimulus syndrome clinical manifestation include vision impairment floater retrospective cohort CMV relate IRIS common IRIS case prospective cohort symptomatic vitritis occur incidence rate yr ART responder carry previous diagnosis CMV retinitis inactive disease onset antiretroviral therapy median time ART initiation IRV week large prospective surveillance study identify patient history CMV retinitis involve eye thirtyone ART responder diagnose IRU male gender use ART high cell count involvement posterior retinal pole factor associate reduce risk develop IRU prior use intravitreous injection cidofovir large retinal lesion adequate immune recovery ART associate increase risk diagnosis ocular manifestation IRIS require high level suspicion addition sign retinitis inflammatory symptom include vitritis papillitis macular edema result symptom loss visual acuity floater affect eye treatment IRIS associate CMV retinitis IRV involve anti cmv therapy gancyclovir valgancyclovir occurrence IRU patient receive anti cmv therapy draw use question use systemic corticosteroid successful IRV require periocular corticosteroid injection significant morbidity vary temporal presentation clinician maintain high level vigilance ocular manifestation CMV associate introduction protease inhibitor increase rate herpe zoster note HIV infect patient study compare ART non art patient report increase incident case zoster rate estimate case person year time high rate observe pre haart era study report difference overall incidence HAART era case person year use HAART associate increase odd develop incident zoster outbreak confidence interval study suggest ART play role increase risk zoster reflect large observational IRIS cohort dermatomal varicella zoster comprise IRIS case mean onset disease ART initiation week range week case occur week therapy study identify significant increase cell risk factor develop dermatomal zoster complication encephalitis myelitis cranial peripheral nerve palsy acute retinal necrosis occur immunocompromise HIV patient vast majority patient exhibit typical atypical dermatomal involvement dissemination systemic symptom randomize control trial demonstrate oral acyclovir effective dermatomal zoster HIV infect patient facilitate healing shorten time zoster associate pain use case varicella zoster IRIS appear clinical benefit benefit corticosteroid combination acyclovir acute varicella zoster demonstrate large randomized control trial combination corticosteroid acyclovir decrease healing time improve acute pain quality life affect incidence duration postherpetic neuralgia incidence postherpetic neuralgia immunocompetent individual differ significantly HIV infect patient increase increase patient age successful symptomatic management involve opioid tricyclic antidepressant gabapentin topical lidocaine patch individually combination show beneficial attempt HIV patient postherpetic neuralgia complication herpe zoster incidence neoforman associate IRIS unknown infrequently report overall IRIS cohort case appear single case report recent study evaluate antifungal combination therapy treatment neoforman meningitis HIV patient significant log reduction colony form unit observe combination substantial number patient remain culture positive week therapy important delay ART CSF sterility achieve effective antifungal combination amphotericin flucytosine exact timing ART attain csf culture sterility important avoid IRIS unknown illustrate case reactivation cryptococcal meningitis describe patient receive week antifungal therapy prior ART reasonable administer systemic corticosteroid alleviate unresponsive inflammatory effect anecdotal benefit observe patient furthermore serial lumbar puncture require manage persistent csf pressure elevation patient continuation ART perform safely interruption antiviral therapy necessary severe unresponsive case common infectious etiology noninfectious etiology list Table infectious non infectious etiology rare recommendation exist management exact estimate incidence available IRIS patient initiate ART firmly establish significant problem high low income country wide variation clinical presentation increase spectrum symptom etiology report diagnosis remain problematic Fur thermore test currently available establish IRIS diagnosis standardized disease specific clinical criterion common infectious manifestation disease develop identify risk factor develop syndrome optimize prevention management opportunistic infection result trial address optimal timing duration treatment opportunistic infection assist develop guideline prevention management IRIS treatment IRIS remain clinical challenge variety clinical presentation presence multiple pathogen capable cause syndrome great understanding syndrome achieve different region world clinician need remain vigilant initiate ART individualize therapy accord know treatment option specific infectious agent','2007-05-08 00:00:00','http://dx.doi.org/10.1186/1742-6405-4-9',2191,736),('e7f311d3c752063f75f28de85abd5f82eacab413.json','Epicatechins Purified from Green Tea (Camellia sinensis) Differentially Suppress Growth of Gender-Dependent Human Cancer Cell Lines','The anticancer potential of catechins derived from green tea is not well understood, in part because catechin-related growth suppression and/or apoptosis appears to vary with the type and stage of malignancy as well as with the type of catechin. This in vitro study examined the biological effects of epicatechin (EC), epigallocatechin (EGC), EC 3-gallate (ECG) and EGC 3-gallate (EGCG) in cell lines from human gender-specific cancers. Cell lines developed from organ-confined (HH870) and metastatic (DU145) prostate cancer, and from moderately (HH450) and poorly differentiated (HH639) epithelial ovarian cancer were grown with or without EC, EGC, ECG or EGCG. When untreated cells reached confluency, viability and doubling time were measured for treated and untreated cells. Whereas EC treatment reduced proliferation of HH639 cells by 50%, EGCG suppressed proliferation of all cell lines by 50%. ECG was even more potent: it inhibited DU145, HH870, HH450 and HH639 cells at concentrations of 24, 27, 29 and 30 µM, whereas EGCG inhibited DU145, HH870, HH450 and HH639 cells at concentrations 89, 45, 62 and 42 µM. When compared with EGCG, ECG more effectively suppresses the growth of prostate cancer and epithelial ovarian cancer cell lines derived from tumors of patients with different stages of disease.',' accrue evidence green tea anticancer activity mechanism action poorly understand green tea produce shrub Camellia sinensis Fig leave dry ferment green coloration attribute polyphenol retain commercially prepare green tea extract contain polyphenol polyphenol source bioflavonoid strong antioxidant activity major bioflavonoid green tea epicatechin like bioflavonoid tea catechin hydrocarbon ring hydroxyl molecule find position Fig major tea catechin epicatechin EC EC gallate ECG epigallocatechin EGC EGC gallate EGCG relative proportion EC ECG EGC EGCG non decaffeinated green tea mg dry wt respectively corresponding proportion non decaffeinated black tea mg dry wt apparent activity human cancer reportedly promote apoptosis arrest metastasis inhibit metalloproteinase impair angiogenesis reverse multidrug resistance epicatechin EC potentially suppress cell proliferation EGCG appear promising clinical investigation chemoprevention trial give wide range physiologic potency different catechin exclusive focus EGCG probably short sighted EGCG reportedly effective egc decrease intestinal absorption cholesterol potent catechin inhibitor reverse transcriptase ECG strong collagenase inhibitory effect high antioxidant potential contrast egc potent mediator oxidative modification inhibitor xanthine oxidase hepatic catabolism purine hypothesize vitro anticancer action catechin vary type stage malignancy test hypothesis examine proliferation catechin treat cell line derive organ confine metastatic prostate cancer CaP moderately poorly differentiate epithelial ovarian cancer EOC goal obtain datum useful develop chemopreventive therapeutic clinical trial patient gender specific non specific solid tumor gender specific human cancer cell line androgen receptor negative CaP cell line develop Hoag Cancer Center Newport Beach organ confine primary tumor resect year old previously untreated Caucasian tumor Gleason Grade evidence vascular perineural invasion extracapsular extension stage metastatic CaP cell line American Type Culture Collection line derive brain lesion year old male Caucasian androgen insensitive express prostate specific antigen EOC cell line develop Hoag Cancer Center poorly differentiate clear cell Grade carcinoma omentum leave ovary old caucasian female moderately differentiate metastatic cell recover abdominal fluid year old asian female cell line cryopreserve liquid nitrogen freezer recovery cryopreserve cell vial transfer water bath thaw room temperature transfer ml polypropylene tube Pasteur pipette aliquot ml fetal bovine serum FBS add drop cell allow settle min centrifuge min Supernatant remove cell suspend fresh RPMI gently tap vortexe cell viability monitor trypan blue dye exclusion cell count determine hemocytometer cell recover cryovial grow glutamine Invitrogen Carlsbad supplement FBS HEPES buffer gentamycin fungizone humidified atmosphere co confluency cell detach sterile EDTA dextrose mm sodium chloride mm potassium chloride mm dextrose mm ethylene diamine tetra acetate EDTA mm sodium bicarbonate min min recover cold FBS resuspend medium use trypsin avoid harvest cell cell viability cell count reassess cell seed culture flask epicatechin study Fig mm epicatechin epicatechin control glutamine Invitrogen FBS hepes buffer gentamycin fungizone experiment ml sterile polystyrene tissue culture flask vent blue plug seal cap Beckton Dickinson Franklin Lakes NJ Cat flask contain stock solution epicatechin concentration mm flask epicatechin flask control mm flask epicatechin flask control cell suspend ml fbs solution describe transfer flask allow grow control cell reach confluency cell detach sterile EDTA dextrose min recover cold FBS medium resuspend medium cell count hemocytometer trypan blue dye exclusion determine number viable versus dead cell interval seeding confluent growth control cell calculate double time number cell cycle inhibitory concentration catechin cell line calculate software program Microcal Origin Corp OriginLab Corporation Northampton MA cell photograph directly flask light microscopy Olympus variance Fisher significant difference LSD method pairwise comparison value significant level Organ confine prostate cancer cell line primary metastatic epithelial ovarian cancer cell line seed cell flask concentration mM ECG EGCG photograph light microscope untreated control cell reach confluency Fig ECG EGCG significantly affect density cell line decrease cell density high concentration pronounce ECG EGCG find significant consider recommendation clinical trial EGCG mean density viable cell number million treatment different cell line examine catechin mm Fig cell density measure growth untreated cell reach confluency statistical analysis ANOVA pairwise comparison show ECG EGCG significantly affect cell density ECG decrease cell density prostate cancer cell ovarian cancer cell line potently EGCG EGCG inhibit growth ovarian cancer cell line well ECG suggest need determine relative efficacy ECG EGCG clinical trial different cancer Tumor cell double time ECG versus EGCG Figure show influence epicatechin cell doubling time ECG EGCG prolong doubling EGC ECG inhibit growth human lung cancer cell line cell potently EGCG EC significant growth inhibition mechanism growth inhibition EGCG study relation cell cycle regulation EGCG mm increase percentage cell relative cytotoxicity CTX ECG carcinoma cell normal fibroblast cell human oral cavity compare polyphenol tea evaluate carcinoma cell ECG CG EGCG group highly toxic EGC moderately toxic EC toxic fibroblast ECG CG group highly toxic EGCG moderately toxic EGC EC toxic CTX effect polyphenol pronounced carcinoma normal cell table summarize effect EC ECG EGC EGCG viability double time cycling cell line untreated cell line reach confluency cell cycle EC affect proliferation cell reduce proliferation cell half prevent confluent growth table EGC affect proliferation cell line table EGCG arrest proliferation line ECG follow EGCG potent inhibitor cell growth cycling proliferation cell line treatment monitor ECG EGCG concentration dosimetric result plot Fig show concentration dependent suppression cell growth ECG EGCG suppressive effect cell density strike high concentration ECG EGCG ECG potent inhibitor cell growth EGCG mm EGCG cell number significantly high control value base result plot Fig value calculate value mm ECG versus mm EGCG table ECG suppress growth high concentration test Fig EGCG significantly enhanced proliferation CaP cell mm find relevant chemoprevention trial EGCG green tea widely consume Japan China polyphenolic component chemopreventive effect cancer vitro vivo cup green tea contain mg catechin EC EGC ECG EGCG numerous investigation show role EGCG cancer chemoprevention study attempt compare relative antitumor efficacy catechin Table systematic approach assess effect catechin cell line derive gender base cancer find catechin antitumor activity depend type tumor EGCG potent chemopreventive agent early literature Table indicate EGCG potent growth inhibitor cell line glioblastoma melanoma cancer breast colon lung prostate androgen receptor positive pancreas liver mouth EGCG prevent proliferation cell arrest cell cycle Gupta document arrest independent mutation EGCG treatment induce cyclin kinase inhibitor observation suggest EGCG impose cell cycle checkpoint result show ECG potent EGCG inhibition primary metastatic CaP EOC cell Fig Tables ECG significantly reduce cell proliferation table Figs increase mean double time table Fig vitro effect chemopreventive agent study tumor cell matrix suspension suspension method expose entire cell surface chemotherapeutic agent finding confirm early report matrix method ECG potent EGCG suppress proliferation CaP cell report difference relative efficacy different catechin difference methodology tumor cell kill catechin study ECG mm induce death cell double ECG concentration increase tumor kill rate ECG epimerize CG vitro dose mm equivalent mg EC EGC mg ECG far mg EGCG cup green tea Lee et al report plasma level EGCG EGC healthy volunteer increase ng ml respectively min drink brew green tea tea solid ml hot water suggest drink cup green tea necessary maintain plasma concentration EGCG equivalent vitro dose mm mg Kaegi suggest daily intake cup green tea chemopreventive measure level tea consumption impractically high chemoprevention cancer catechin require administration appropriate catechin purified form conclusion state green black tea polyphenol important component antitumor aspect complementary alternative medicine CAM play significant role american health care system patient suffer chronic problem green tea catechin gallate EGCG ECG possess potent antitumor activity epimer commonly find black tea act potent inhibitor protease involve replication virus include coronoviruse need understand preventive therapeutic potential catechin gallate green black tea currently design phase chemopreventive study examine effect purify EGCG ECG patient choose observational management organ confine prostate cancer','2006-04-25 00:00:00','http://dx.doi.org/10.1093/ecam/nel003',1235,508),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json','The role of mast cells in the pathogenesis of pain in chronic pancreatitis','BACKGROUND: The biological basis of pain in chronic pancreatitis is poorly understood. Mast cells have been implicated in the pathogenesis of pain in other conditions. We hypothesized that mast cells play a role in the pain of chronic pancreatitis. We examined the association of pain with mast cells in autopsy specimens of patients with painful chronic pancreatitis. We explored our hypothesis further using an experimental model of trinitrobenzene sulfonic acid (TNBS) -induced chronic pancreatitis in both wild type (WT) and mast cell deficient mice (MCDM). METHODS: Archival tissues with histological diagnoses of chronic pancreatitis were identified and clinical records reviewed for presence or absence of reported pain in humans. Mast cells were counted. The presence of pain was assessed using von Frey Filaments (VFF) to measure abdominal withdrawal responses in both WT and MCDM mice with and without chronic pancreatitis. RESULTS: Humans with painful chronic pancreatitis demonstrated a 3.5-fold increase in pancreatic mast cells as compared with those with painless chronic pancreatitis. WT mice with chronic pancreatitis were significantly more sensitive as assessed by VFF pain testing of the abdomen when compared with MCDM. CONCLUSION: Humans with painful chronic pancreatitis have an increased number of pancreatic mast cells as compared with those with painless chronic pancreatitis. MCDM are less sensitive to mechanical stimulation of the abdomen after induction of chronic pancreatitis as compared with WT. Mast cells may play an important role in the pathogenesis of pain in chronic pancreatitis.',' pain present symptom patient chronic pancreatitis neurobiological basis remains poorly understand past investigator focus role anatomical abnormality stricture pancreatic duct narrow intraparenchymal duct mechanical decompression technique endoscopic stent placement surgical pancreatojejunostomy frequently provide permanent solution pain recently investigator begin focus role neurotransmitter neurotrophin substance nerve growth factor known suspect role nociceptive signaling sensitization report increase expression pancrea patient painful chronic pancreatitis mast cell increase acute chronic pancreatitis role generation pain pancreatitis investigate hypothesize mast cell involve pathogenesis pain chronic pancreatitis hypothesis base follow observation mast cell associate human condition pain predominant symptom interstitial cystitis irritable bowel syndrome condition pain proportion objective pathological finding condition increase number mast cell describe bladder colon respectively mast cell frequently find close proximity nerve intestinal mucosa bladder observe pancrea total number mast cell significantly high pancreatic tissue patient chronic pancreatitis normal pancreatic control preferential location mast cell perineurium nerve fiber tissue sample patient chronic pancreatitis suggest potential interaction mast cell nervous system lastly evidence bi directional functional communication mast cell nerve mast cell release mediator increase excitability neuron turn neurotransmitter substance trigger mast cell degranulation Mast cell contribute pathogenesis pain pancreatitis degranulation product sensitize pancreatic afferent neuron ongoing vicious circle neuronally mediate mast cell degranulation aim analyze presence distribution mast cell autopsy specimen chronic pancreatitis study correlation historical documentation pain explore hypothesis experimental model trinitrobenzene sulfonic acid chronic pancreatitis wild type Kit mouse strain deficient mast cell record University Texas Medical Branch year search electronically term pancreatitis sixtysix patient identify patient carry autopsy diagnosis chronic pancreatitis patient carry diagnosis acute pancreatitis medical chart patient autopsy diagnosis chronic pancreatitis review documentation medical history chronic pancreatitis documentation present chart patient exclude study patient document history autopsy base diagnosis chronic pancreatitis include study patient categorize painful chronic pancreatitis fulfil follow criterion document history chronic abdominal pain clinically attribute chronic pancreatitis require use narcotic frequent admission recurrent abdominal pain clinically attribute chronic pancreatitis surgical endoscopic procedure refractory abdominal pain clinically attribute chronic pancreatitis patient categorize non painful chronic pancreatitis patient fit criterion list painful chronic pancreatitis addition follow datum collect demographic factor age race cause death comorbiditie clinical history pancreatitis etiology pancreatitis diagnostic study support diagnosis pancreatitis amylase lipase calcification abdominal plain film ct scan ultrasound ERCP human pancreatic control tissue obtain arbitrarily choose patient autopsy record record acute myocardial infarction cause death medical record review ensure clinical history pancreatitis category patient painful chronic pancreatitis non painful chronic pancreatitis non pancreatitis control pathologist blind group assignment verify histological diagnosis count mast cell Giemsa stain tissue section average high power randomly choose field speciman protocol approve Institutional Review Board University Texas Medical Branch mouse purchase Jackson Laboratory Bar Harbor male mouse follow strain kit MCDM respective littermate control mouse strain Kit WT mouse month age onset experiment body weight gram experimental protocol involve mouse approve Institutional Animal Care Use Committee IACUC accordance guideline provide National Institutes Health mouse anesthetize sodium Nembutal mg kg body weigh follow midabdominal laparotomy canula introduce common pancreato biliary duct duct ligate proximally distally ensure perfusion pancrea prevent entry inject substance liver duodenum ml TNBS phosphate buffer saline ethanol pH infuse pancrea modify Puig Divi canula remove abdoman close control mouse treat exact fashion perfuse saline instead figure mouse sacrifice week surgery VFF hair consist series filament increase diameter produce increase sensation touch apply skin tip fiber give length diameter press skin force application increase fiber bend fiber bend continue advance create bend force application principle make possible apply reproducible force skin surface vff testing established behavioral pain assay determine mechanical pain threshold somatic pain recently VFF testing surrogate marker visceral pain place cage mesh floor habituate environment minute measurement take abdoman plantar surface hindpaw period week prior surgery total week surgery figure VFF filament caliber apply mid abdomen ascend order time second second interval response define sharp retraction abdoman immediate licking scratching site application hair jumping response frequency define total number response application express percentage skin filament investigator blind different treatment group perform behavioral testing fresh specimen mouse pancrea fix formaldehyde PBS pH contain mM MgCl overnight section paraffin embed specimen stain hematoxyline eosin observe light microscope pathological change score base scale describe Tito et al pathologist blind different treatment group number mast cell autopsy speciman analyze Mann Whitney test behavioral experiment figure average response frequency calculate mean mean response frequency mouse measure post pre response frequency calculate subtract pre surgical average response frequency post surgical average response frequency assess independent effect TNBS VFF response ie control effect surgery postpre response frequency TNBS infusion compare post pre response frequency saline infusion comparison perform analysis variance factor experiment repeat measure time level force type mouse WT mcdm factor induction pancreatitis tnb saline respectively time pre surgical post surgical TNBS infusion consider independent effect VFF response postpre response frequency great TNBS saline infusion Fisher significant difference procedure multiple comparison square mean experimental design Figure Experimental design mouse undergo pre post surgical VFF testing VFF testing measure take mouse WT mcdm randomize saline TNBS perfusion pancreatic duct patient demographic summarize Table alcohol abuse common cause pancreatitis group analysis result Mann Whitney test reveal significantly mast cell patient history painful chronic pancreatitis compare patient non painful chronic pancreatitis vs average mast cell high power field control vs average mast cell high power field figure increase number mast cell patient painful pancreatitis note predominantly interstitial area less degree periacinar space figure show post pre surgical response frequency WT mcdm TNBS significant independent effect abdominal VFF response WT mouse force level mN respectively figure trend significant effect force level mN contrast mcdm TNBS significant effect abdominal VFF response force level figure significant TNBS effect VFF response left pancreatic histology confirm presence chronic pancreatitis WT mcdm marked fibrosis inflammatory infiltrate ductular proliferation mimicking change see human chronic pancreatitis figure pancrea saline treat control normal significant difference overall inflammatory score WT MCDM figure increase number mast cell count WT mouse chronic pancreatitis compare saline Histology Giemsa mouse chronic pancreatitis figure expect mast cell present pancrea pancreatitis define continue inflammatory disease pancrea characterize irreversible morphologic change typically cause pain permanent loss function pathogenesis pain condition remain satisfactorily establish examine association pain mast cell quantify autopsy specimen patient history painful non painful chronic pancreatitis normal control significantly mast cell present pancreatic tissue patient history painful chronic pancreatitis indicate association condition potential role cell pathogenesis pain painful chronic pancreatitis clearly limitation retrospective autopsybased study report instance know pain present time death incomplete information different pattern pain finding pertain mainly patient history alcoholic pancreatitis finding suggest association painful chronic pancreatitis increase number mast cell observation provide rationale experimental testing perform mouse develop model chronic pancreatitis mouse follow modify protocol describe Puig Divi et al histological change consist periductal lobular fibrosi duct stenosis chronic inflammatory cell infiltrate gland atrophy mimic feature chronic pancreatitis human significantly mast cell present WT mouse chronic pancreatitis add validity model use study role mast cell pancreatitis WT mcdm develop histological change consistent chronic pancreatitis indicate elimination mast cell modulate animal ability mount inflammatory response change observe pain behavior unlikely stem difference underlie inflammation determine mouse model study behavioral difference associate chronic pancreatitis assessment spontaneous pain visceral organ present significant difficulty behavioral method assess rely association visceral pain sensitization somatic region body share segmental innervation level spinal cord referred pain somatic sensitization quantify VFF stim ulate somatotopically appropriate abdominal region measure abdominal withdrawal response VFF testing anterior abdominal wall surrogate marker visceral pain time technique measurement refer visceral hyperalgesia mouse model chronic pancreatitis method previously describe validate assess severity refer visceral pain model colonic hypersensitivity rat model acute necrotize pancreatitis chronic pancreatitis abdominal VFF response compare hind paw response assess specificity intervention pancrea tnb treat mouse saline control develop increase abdominal wall withdrawal response VFF testing compare baseline suggest development force dependent referred hyperalgesia abdominal wall WT mouse evidence refer hyperalgesia hindpaw suggest measured effect abdominal withdrawal specific intra abdominal origin pain Vera Portocarrero et al previously describe similar finding increase withdrawal frequency VFF stimulation abdominal area rat model chronic pancreatitis behavioral change abrogate morphine rat demonstrate behavioral change express increase substance expression nociceptive layer spinal cord suggestive central nociceptive change Mast cell produce variety degranulation product setting inflammation activate sensitize primary nociceptive neuron neurotrophin growth factor NGF product NGF release setting inflammation function chemoattractant mast cell trigger mast cell degranulation speculate NGF production inflame pancreas responsible plastic change sensory neuron activate proalgesic receptor channel NGF receptor tyrosine kinase TrkA Transient Receptor Potential Family receptor previously know contribute generation pain similarly mast cell degranulation product tryptase histamine capable modulate neuronal function tryptase directly activate proteinase activate protein couple receptor express pancreatic nerve important pathogenesis pain pancreatitis role mast cell mediation visceral nociceptive signal need explore speculate mast cell product release pancreatitis contribute development pain direct effect nociceptor locate pancreatic afferent neuron figure conclude definite role mast cell experimental datum note MCDM carry spontaneous mutation tyrosine kinase receptor kit produce deficiency mast cell independent effect function sensory neuron know express remain determine detect difference nociceptive response absence mast cell se unknown change responsiveness sensory neuron congenital lack kit receptor reconstitution mast cell mcdm mouse restore nociceptive response close wild type phenotype datum increase awareness importance mast cell pathogenesis painful inflammatoryproposed involvement mast cell nociceptive signaling pancreatitis Figure propose involvement mast cell nociceptive signaling pancreatitis pancreatitis mast cell migrate site inflammation response release mast cell chemoattractant Mast cell degranulation product modulate neurotransmission directly activate proalgesic receptor channel trka NGF NGF tryptase trypsin pre publication history paper access prepub','2005-03-03 00:00:00','http://dx.doi.org/10.1186/1471-230x-5-8',1572,632),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json','Detection and characterization of horizontal transfers in prokaryotes using genomic signature','Horizontal DNA transfer is an important factor of evolution and participates in biological diversity. Unfortunately, the location and length of horizontal transfers (HTs) are known for very few species. The usage of short oligonucleotides in a sequence (the so-called genomic signature) has been shown to be species-specific even in DNA fragments as short as 1 kb. The genomic signature is therefore proposed as a tool to detect HTs. Since DNA transfers originate from species with a signature different from those of the recipient species, the analysis of local variations of signature along recipient genome may allow for detecting exogenous DNA. The strategy consists in (i) scanning the genome with a sliding window, and calculating the corresponding local signature (ii) evaluating its deviation from the signature of the whole genome and (iii) looking for similar signatures in a database of genomic signatures. A total of 22 prokaryote genomes are analyzed in this way. It has been observed that atypical regions make up ∼6% of each genome on the average. Most of the claimed HTs as well as new ones are detected. The origin of putative DNA transfers is looked for among ∼12 000 species. Donor species are proposed and sometimes strongly suggested, considering similarity of signatures. Among the species studied, Bacillus subtilis, Haemophilus Influenzae and Escherichia coli are investigated by many authors and give the opportunity to perform a thorough comparison of most of the bioinformatics methods used to detect HTs.',' widely admit actual genome common ancestor LUCA Universal Common Ancestor current diversity result event modify genome evolution event happen nucleotide level point mutation indel nucleotide strand inversion duplication repetition transposition horizontal transfer ht concern significant part genome postulate ht exchange genetic material different specie frequent stage evolution essentially subsist nowadays prokaryote consequence detection ht appear crucial understanding evolutionary process qualitative quantitative evaluation exchange rate specie recent complete sequencing genome allow systematically search presence dna transfer specie especially prokaryote probability occurrence high report particular ht bacteria account genome ii archaebacteria non pathogenic bacteria prone transfer pathogenic bacteria iii operational gene likely transfer gene deal information management HT concept originally coin explain dramatic homology gene unrelated specie unusual match subsequently criterion detection ht approach allow detection gene transfer partial knowledge genome require sequencing homologous gene number specie consequently HT screening gene give specie similar respect base composition codon bias short oligonucleotide composition general rule usage oligonucleotide vary genome genome addition observe transfer dna retain time characteristic specie origin particularity conjunction detect dna transfer specie transfer DNA consequently detect basis singularity respect sequence characteristic recipient specie technique suffer drawback weakness lead consider generalize approach screening atypical region sequence fact genomic signature account possible bias DNA sequence show speciesspecific signature approximately invariant genome way specie origin dna segment small kb identify surprisingly high efficiency mean signature consequence sequence signature bacteria consider valuable estimation genomic signature assume transfer DNA fragment exhibit signature specie come ii recipient donor signature different screening local variation signature genome expect reveal region interest ht locate addition status HT strongly suggest signature region interest find close signature specie sequence signature define frequency set short oligonucleotide observe sequence easily obtain thank fast algorithm derive Chaos Game Representation CGR allow cope mb sequence second laptop computer signature visualize square image color gray level pixel represent frequency give oligonucleotide call word example signature Supplementary Materials sequence gather GenBank genome prokaryote scan ht genome give special attention illustrate approach particular provide valuable benchmark thank set previous work address issue signature specie obtain genomic sequence long kb sequence derive specie concatenate accuracy purpose specie domain life archaea specie bacteria specie eukarya specie virus specie represent total Gb detection atypical region base observation deviation local signature signature small fragment dna genomic signature recipient specie genome consequently sample mean slide window appropriate size fact interesting window small possible high sampling accuracy intra genomic variability signature increase small window addition variability depend specie word length base composition letter word letter word poorly speciesspecific allow good discrimination specie general rule long word letter long high specificity signature count long word small window low allow reliable estimation parameter hand analysis letter word slide window kb kb step offer good trade reliability count file size computational charge specie addition double strand signature call local signature compute window rid variation induce strand asymmetry illustration purpose local signature develop vertical vector stack genome order overall picture word usage variation genome plot horizontal line variation frequency word genome local change word usage appear vertical break figure figure signature letter word kb window genome Clostridium acetobutylicum Deinococcus radioduran Mycobacterium tuberculosis kind display line represent frequency word genome column represent signature window consider great genome speciestypical signature recipient specie estimate analysis sequence vast majority local signature look believe instance recipient specie signature greatly differ order avoid potential bias link outlier subsequently decide select typical local signature basis similarity observe cluster underlying idea typical local signature aggregate large group outlier find small complementary group great distance recipient genome signature group consequently determine mean cluster tool scheme cluster specie finally good scheme cluster obtain decision tree base partition cart algorithm purpose CART algorithm predict value categorical dependent variable cluster local signature work signature characterize distance estimate genomic signature continuous categorical predictor variable different clustering scheme cluster work CART algorithm provide optimal split group collect signature close estimate recipient genome signature group specie clustering scheme select group clustering partition offer continue example group recipient specie signature subsequently calculate mean signature group belong partition small distance estimate genomic signature Comparison signature possible thank euclidian metric account difference word usage point distance signature calculate high dimensional datum dimension correspond different letter word consequently subject call concentration measure phenomenon distance high dimension space comparable increase square root dimension space variance distribution remain unchanged fact radius hyper sphere hold signature database seven time near neighbor distance small distance specie small difference distance consequently consider highly significant specie set recipient specific distance obtain local signature belong large cluster give distance host signature order select outlying signature cut distance choose basis distribution distance observe specie appear percentile offer good trade sensibility specificity outli detection impact threshold detection atypical region result signature minority cluster detect way isolated signature detect signature recipient specie cluster select outlier flank region genome later reanalyze small window step th original size typically order accurately determine limit signal noise ratio allow finally gene content detect region analyze help specie dedicate database Genome Information Broker blastn search GenBank default setting carry atypical region order identify origin potential ht homology high search origin atypical region specie include chromosomal plasmidic mitochondrial chloroplastic dna GenBank find eligible genomic signature give signature atypical dna fragment specie close signature consider potential donor screening perform atypical region specie consideration nearby specie retain distance outlier donor compatible total genome screen atypical region table Supplementary Material average cluster scheme offer good partition single case aeropyrum pernix cluster require general single cluster devoted rrna mean distance window host varie specie mean coefficient variation tightly correlate value Pearson correlation coefficient cut distance vary mean coefficient variation large variation hardly explain mere basis statistical fluctuation observe variation oligonucleotide usage genome depend specie consequently consider species property segmentation quality atypical region test rrna gene rRNA detect atypical table Borders rrna gene accurate nt kb window bp step threshold adjacent trnas identify general rule conclude rRNA specific signature consistently variance host signature context worth notice rRNA remain outlier lie comparable distance specie belong clearly different rrnas consistently find cluster percentage RNA free outlier nucleotide level vary function specie threshold Table show high percentage atypical region pyrococcus abyssi low percentage specie find correlate cut distance high cut distance low percentage outlier fact high cutoff distance take place specie display high intragenomic variability express high mean distance host table actual percentage atypical DNA intrinsic property specie mere consequence resolution power nucleotide bias base method remain consequently open question addition observe percentage outlier significantly high long genome cut distance relate length genome mean cut distance species table value choose select credible donor atypical region subsequently give credible donor Supplementary Material species set table plasmid virus find agreement know molecular mechanism horizontal transfer Table Supplementary Material cluster class allow assess signature populated class collect segment choose represent subpopulation mean distance arbitrary unit recipient centroid class cut distance respectively table run contiguous outlying window share cluster consider single transfer event consequence region figure Supplementary Material fall cut distance potential candidate host foreign dna segmentation genome term gene Supplementary Material figure illustrate accuracy segmentation atypical region obtain slide window kb bp step rrna gene genome table rrna gene find outli population addition window contain rRNA assign specific cluster fact know rRNA signature variance host signature rrna gene account outlier trnas consider study size small generate significant deviation host signature total genome consider typical table look origin segment signature database appear potential donor set kb sequence derive genome result confirm segment have signature belong predominant cluster good representative recipient specie signature rrna free atypical region vary size kb total genome table atypical region kb long distance outlier potential donor fall intra genomic range instance outli donor distance great consider close specie potential donor contrast unusual small value deserve specific attention particular small distance bacteriophage atypical region allow spot genome bacteriophage incorporate region genome find similar term signature bacteriophage correspond bacteriophage imbed genome free form sequence observe similarity expect signature phage usually share characteristic specie infect sequence foreign sequence identify homology criterion BlastN parameter set default fact blast analysis outlier lead contrasted result prophage imbed genome atypical region identify contain rrna gene code genome gene homologous part atypical region find specie belong Bacillus genus interesting note house keep gene rrna detect atypical region fact great number gene atypical region bacteriophage gene rrna know function clustering class require determine recipient specie signature populated class collect segment choose calculate signature mean distance host cut distance subsequently find equal respectively table similarly cluster genome devoted rrna gene copy table total rRNA label atypical account outlier analysis Table show genome consider typical fact potential donor kb sequence derive genome observe concordance percentage corroborate partition procedure selection typical atypical fragment rrna free atypical region vary size kb genome table Annex figure Annex segmentation genome term gene atypical region kb long number clearly variance small percentage genome qualifie atypical average size atypical region small result examine context intra species signature variability cluster class require determine recipient specie signature main feature summarize Table potential donor RNAfree atypical region give Annex segmentation genome term gene Annex worth notice potential donor belong Enterobacteriales family segmentation term gene display Annex analysis genome particularly useful comparison literature approach detect horizontal gene transfer propose decade phylogenetic tree protein dna sequence unusual distribution gene nucleotide composition include codon bias HT feature consider framework model Hidden Markov Models hmm Factorial Correspondence Analysis FCA criterion currently employ result model advantage caveat recently point Ragan Lawrence Ochman approach deal particular subset ht example efficient detect recent transfer effective detection ancient ht approach clearly base oligonucleotide composition assume different specie different signature rely assumption surprising genomic signature approach provide result term dna transfer reasonable agreement propose Garcia Vallve Nakamura et al specie analyze common correlation percentage ht find method highly significant specie extensively study HT content method include method include analyze Garcia Vallve Nakamura comparison method present Tables detail Supplementary Materials voting procedure majority rule implement determine status gene respect atypicality task initial analysis convert term gene Supplementary Materials degree agreement method subsequently observe statistical Kappa coefficient kappa measure degree agreement scale minus infinity Kappa indicate agreement Kappa zero indicate agreement expect chance negative value observe agreement weak expect chance rare situation respectively number detect gene method close range Nakamura work median detailed vote give Table gene genome gene detect method gene number single vote gene range Garcia Vallve Nicolas total gene majority consensus set detect good score method good agreement majority consensus term Kappas reach Nicolas follow method Moszer table method get good agreement Nicolas bad HMM method Nakamura pairwise kappa comparison table Supplementary Material fact Nakamura approach variance approach get low Kappa Garcia Vallve Hayes Lawrence Nakamura Medigue work majority consensus method table probable number HT gene range reasonable estimation correspond majority consensus note method unable find gene detect method Supplementary Material gene nt long respectively compare nt median size atypical region detect method Table clearly method appropriate detect short isolated atypical Garcia Vallve Nakamura et al voter concern analysis genome Supplementary Material Table originality result obtain Nakamura salient feature comparison number detect HT gene twice high Nakamura et al belong majority consensus small table gene detect Garcia Vallve Nakamura method small number voter preclude specific comment respect probable number HT gene range reasonable estimation majority consensus table result obtain Hayes Borodovsky clearly variance table proportion claim outlier range publish number method specific agreement method weak table Hayes Borodovsky obviously develop approach base HMM deal specific outlier Lawrence Ochman poor rating especially detect twice gene author table worth note cut distance method lower instead instance single vote gene dig detail impact cut distance Supplementary Material percentage outlier report approach rise percentage single vote gene reach expect high cut distance provide single vote gene risk miss potentially transfer gene lower cut increase proportion single vote gene advantage detect potential transfer Supplementary Material obviously continuous grading gene atypicality suggest consider consensual gene potential ht apply amelioration model explain grading difficult assess relevancy propose donor gene detect potential HT generally undergo amelioration comparison recently diverge genome specie strain provide opportunity find recent ht correspond homologous gene donor specie detect study perform strain strain uropathogenic strain enterohaemorrhagic strain RIMD Shigella flexneri strain seven specie recently diverge genome size different proportion horizontally transfer gene vary strain specie instance non redundant set protein common strain EDL strain specie cluster group respect phylogeny Table criterion search recent horizontally transfer gene atypical region window size kb step kb signature differ greatly host distance host time intrinsic mean distance Table ii present limited number strain specie ascertain recentness fact outlier meet criterion generally aggregate heterogeneous cluster mean clustering usually include sample strain specie instance strain specie absent cluster subsequently consider correspond region recently acquire relevant specie table show selection potential recently transfer gene cluster atypical region contain gene present specific set strain atypical gene strainspecific absent non pathogenic strain intermediate situation encounter FASTA blast search confirm gene absent test strain observe analysis complete genome large number case able find conserve homologous gene specie Table interesting note suggest donor signature database agreement specie find alignment method homologous gene find propose donor credit know mechanism gene transfer bacteriophage plasmid Table worth notice select gene absent strain involve pathogenicity strain strain exhibit great number gene absent strain great number gene homolog find Table unable propose donor great gene Table select gene lie Ter region genome position agreement publish result observe genomic region typical genome belong signature endpoint consider genomic signature species specific atypicality region term oligonucleotide usage promote criterion detection ht atypicality base method suffer caveat reduce effectiveness way ht detect fact transfer specie close signature detect significant difference characteristic transfer dna recipient specie dna require similar reason ht drastically ameliorate follow introduction detect stringent constraint result size screening window hand ideally good signal noise ratio obtain window ht comparable size hand window size large provide significant word count requirement strengthen size word consideration intrinsic variability genomic signature genome trade implement paper allow detect atypical region small kb fact rrna region share characteristic consistently detect point small fragment eventually detect signature radically atypical atypicality consider criterion detect ht approach suffer drawback note signature base method detect region lie standard deviation mean specie instance region Supplementary Materials observe Nicolas et al rRNA definitely atypical signature systematically classify outlier specie table transfer rrna specie unlikely firmly rule clear atypical signature rRNA imply horizontally transfer signature approach interesting property share HMM detection bind specific function genome contrast method signature approach detect gene transfer region agreement describe mechanism dna exchange specie notice method allow detect atypical non code region Supplementary Materials major difference HMM signature method lie time require learning process resource HMM mobilize deal short kind HT hand ht short kb hardly detect signature base approach innovative HT detector likely result adequate fusion method factor contribute efficiency search donor course distance putative HT donor signature essential Accuracy signature link length available sequence density signature vicinity HT amelioration sustain HT presence host importance Deschavanne Lespinats Fertil unpublished result distance signature putative HT close specie vary large extent usually short one fall intra genomic range table Supplementary Materials case distance close donor signature atypical segment signature great potential donor propose Supplementary Materials strong similarity give dna sequence foreign specie observe hypothesis underlie transfer highly strengthen true donor previously sequence include bank signature allow situation occur account intrinsic variability short dna segment signature function size species specific compare signature complete genome large species sample present state signature database way representative diversity richness life notice obvious structure term distance signature express taxonomy relationship specie signature database related specie find close cluster potential donor consequently provide pertinent information origin ht diversity signature putative ht observe specie analyze paper reveal multiplicity transfer event donor Supplementary Materials outlier necessarily neighbor genome give set potential donor table Supplementary Materials general potential donor belong set taxonomically close specie table share biotope host Supplementary Materials instance live distinct biotope potential donor particularly encouraging find potential donor approach point opportunity exchange dna material recipient specie numerous virus plasmid qualify potential donor Tables Supplementary Materials surprising know HT vector totally partially insert transfer gene host genome atypical dna segment particularly peculiar isolated specific signature distance neighbor great give credible set donor Supplementary Materials lack datum search domain shift signature feature substantial amelioration process structural constraint serve special function role rrna code region track remain explore circumstance interesting localize region transfer come complete genome donor available homology dna level pertinent criterion comparison sequence soon amelioration take place fact homology weak gene Escherichia Salmonella specie recently diverge clear powerful search origin putative ht embody model amelioration design Lawrence Ochman search recent horizontally transfer gene different strain species instance possible find great homology detect gene gene specie Table numerous case selection donor consistent FASTA result Table confirm pertinence similarity signature putative ht donor propose method retrieve specie origin transfer region search origin ht basis genomic signature powerful approach understand mechanism evolution usage know species specific suffer minor variation genome consider property allow search atypical local signature point dna transfer result obtain genome analyze paper find good agreement literature Tables Supplementary Materials specie specificity signature allow search donor specie set donor specie common taxonomic feature obtain help environmental consideration subsequently possible identify collect clue potential donor search donor make use non homologous sequence partially sequence specie consequently eligible inasmuch kb genome available thank exponentially grow rate nucleotide databank search donor specie mean sequence signature turn pertinent fruitful future context worth notice computational power clearly issue CGR algorithm describe paper fast order calculation time proportional number method propose look ht signature method base different hypothesis complementary describe method detect preferentially certain type ht agreement author appear conjunction method require obtain overview HT extent genome signature method describe paper generalize approach ground detection outlier basis bias oligonucleodide strong specie specificity signature allow detect kind outlier provide clue possible origin obviously detection ht remain open question consensus emerge additional material experimentation genomic signature available GENSTYLE site','2005-01-13 00:00:00','http://dx.doi.org/10.1093/nar/gni004',2962,803),('f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json','The influenza pandemic preparedness planning tool InfluSim','BACKGROUND: Planning public health responses against pandemic influenza relies on predictive models by which the impact of different intervention strategies can be evaluated. Research has to date rather focused on producing predictions for certain localities or under specific conditions, than on designing a publicly available planning tool which can be applied by public health administrations. Here, we provide such a tool which is reproducible by an explicitly formulated structure and designed to operate with an optimal combination of the competing requirements of precision, realism and generality. RESULTS: InfluSim is a deterministic compartment model based on a system of over 1,000 differential equations which extend the classic SEIR model by clinical and demographic parameters relevant for pandemic preparedness planning. It allows for producing time courses and cumulative numbers of influenza cases, outpatient visits, applied antiviral treatment doses, hospitalizations, deaths and work days lost due to sickness, all of which may be associated with economic aspects. The software is programmed in Java, operates platform independent and can be executed on regular desktop computers. CONCLUSION: InfluSim is an online available software which efficiently assists public health planners in designing optimal interventions against pandemic influenza. It can reproduce the infection dynamics of pandemic influenza like complex computer simulations while offering at the same time reproducibility, higher computational performance and better operability.',' preparedness pandemic influenza high priority public health issue country pandemic preparedness plan design plan mathematical model computer simulation play essential role allow predict compare effect different intervention strategy outstanding significance tool purpose intervention optimization limit fact maximize realism generality precision time public health planner hand wish optimal combination property need formulate intervention strategy generalize recommendation sufficiently realistic precise satisfy public health requirement publish influenza model come application represent extreme generalized oversimplify model dynamic structure publicly available complex computer simulation specifically adjust real condition publicly available complexity simulation necessary reliable description infection dynamic large population minimum requirement pandemic influenza planning tool dynamic modelling structure allow investigation time dependent variable like incidence height epidemic peak antiviral availability etc tool hand adjustable local condition adequately support pandemic preparedness plan different country involve considerably different assumption table describe publicly available influenza pandemic preparedness planning tool design meet requirement preparedness planning base explicitly formulate dynamic system allow address time dependent factor sufficiently flexible evaluate impact candidate intervention consider local condition like demographic economic factor contact pattern constraint public health system subsequent paper provide example application model intervention like antiviral treatment social distancing measure model base system differential equation extend classic seir model demographic parameter reflect situation Germany adjust country epidemiologic clinic value take literature Tables source quote pre set value vary slider input field different assumption transmissibility clinical severity new pandemic strain change cost connect medical treatment work loss simply apply simulation different demographic setting model property summarize follow mathematical formulation model present detail online support material corresponding source code program Java information download German National Pandemic Preparedness Plan total population divide age class subdivide individual low high risk table transmission age class base contact matrix table scale model standard parameter value yield give basic reproduction number value associate influenza strain pandemic potential suggest lie value high effective reproduction number estimate slightly low standard parameter use mean case infect average individual everybody susceptible intervention perform susceptible individual infected incubate infection fully contagious finally develop protective immunity table fraction case remain asymptomatic moderately sick clinically ill need medical help depend combination age risk group fraction clinically ill case need hospitalize agedependent fraction hospitalize case die disease Table partitioning case category allow combine realistic description transmission dynamic easy calculation resource consume outbreak degree duration contagiousness patient depend course disease furthermore depend age patient Table pass incubation contagious period model stage allow realistic distribution sojourn time Table stage incubation period early infectious period patient spread disease infectiousness high onset symptom decline geometrically Table clinically ill patient seek medical help average day onset symptom sick patient advise withdraw home disease extremely sick patient need hospitalize die disease table end contagious period clinically ill patient convalescent period resume ordinary life work table provide example model output InfluSim version mean sensitivity analysis investigation present figure show graphical user interface software divide input output window user set new value input field slider simultaneously obtain new result course epidemic give population figure pandemic wave result vary basic reproduction number standard parameter value give Tables omit intervention town inhabitant result pandemic wave last week figure peak pandemic wave reach seven week daily incidence influenza patient seek medical help hospital bed occupy influenza case work adult unable work illness convalescence result depend assumption concern unknown contagiousness pathogenicity virus figure shape curve depend course contagiousness pandemic wave proceed relative slowly contagiousness change infectious period proceed quickly contagiousness high onset symptom decrease influenza pandemic preparedness planning tool InfluSim stand simple spreadsheet model sophisticated stochastic computer simulation describe pandemic wave homogeneously mix population like town city surprisingly produce dynamic individual base simulation explicitly consider geographic spread cf figure similar observation simple deterministic compartmental model stochastic model know behave quasi deterministically simulated population large reason congruence complex stochastic simple deterministic model lie incredi bly quick way pandemic influenza spread geographically control place origin pandemic starting far country lead multiple introduction large industrialize nation expect quickly spread neighbouring town rural area large population consider susceptible pandemic virus quick geographic spread tend diminish difference result sophisticated individual base simple deterministic model deterministic model like InfluSim reliably represent effect originate stochasticity effect small population heterogeneity example geographically limited spread fairly effective control measure imply epidemic affect small population strongly influence stochastic event ii transmission predominantly occur household hospital drive substantial feature contact network agreement assumption homogeneous mixing deterministic model reliably predict spread infection particular iii super spread event substantially change course epidemic compare deterministic prediction apart factor predictability intervention success generally subject uncertainty choice parameter value assume scenario outcome pandemic preparedness plan gross attack rate clinically ill moderately ill case population inhabitant Germany subdivide accord age risk category assume age group fully susceptible begin outbreak fraction child age year regard high risk influenza infection remain low risk high risk fraction work adult age elderly age respectively source demand additional effort like Bayesian approach evaluate reliability prediction preparedness plan consider constraint capacity locally operate public health system time dependent solution InfluSim allow assess peak value relevant variable outpatient hospitalization death intervention combine find optimal way reduce total number case lower peak value delay peak hope population benefit newly develop vaccine special care take implement variety pharmaceutical non pharmaceutical intervention discuss subsequent paper despite comprehensible structure model suffer simplification common usual compartment model instead implicitly exponentially distribute sojourn time implement realistically distribute delay example model consider individual transmit infection onset symptom case remain asymptomatic infect feature implication success targeted control measure InfluSim freely accessible run regular desktop computer produce result second change parameter value user friendly interface ease result generate program useful public health planning tool take care provide bug free program include source code user encourage treat result caution test participate bug report discussion open source platform provide regular update InfluSim declare compete interest develop model MS design software HPD write manuscript SOB formulate public acquire infection matrix show frequency contact week person different age class source distribution sojourn time stage latent period early infectious period average duration day source assume health requirement software author read approve final manuscript susceptible individual infect rate depend age time infected individual incubate infection mean duration obtain realistic distribution duration incubation period model stage progression stage occur rate incubation stage regard early infectious period patient spread infection account average time lD early infectious period half day standard set parameter pass incubation stage infected individual fully contagious fraction develop clinical symptom course disease depend age infected individual risk category belong fraction asymptomatic fraction moderately sick fraction sick remain fraction extremely sick need hospitalization combination fraction severe fraction extremely severe case visit doctor day onset symptom offer antiviral treatment give supply exhaust model explicitly consider age disease demand partial differential equation use contagious stage measure time onset allow treatment stage source contagiousness asymptomatic case degree contagiousness early infectious period equality contagiousness moderately severely sick case assume independent age risk group fraction infection result asymptomatic case fraction moderately sick remain fraction develop severe disease age risk dependent fraction untreated patient severe disease need hospitalization age dependent fraction hospitalize case die source fraction asymptomatic case symptomatic case doctor hospitalization severe case case fatality hospitalize untreated patient calculate detail impose variability maximum time treatment give improve realism model respect real life scenario antiviral treatment reduce patient contagiousness percent reduce hospitalization death percent extremely sick patient hospitalization prevent treatment send home join group treat sick remain duration disease contagiousness treat case reduce percent rate progressing stage obtain realistic distribution sojourn time convalescence model stage progression stage occur rate fully recover patient pass stage convalescence join group healthy immune work adult work intervention describe reduction contact discuss presentation differential equation InfluSim user interface Figure InfluSim user interface mean cumulative contagiousness concentrate half contagious period Table cumulative number death value parameter list Tables hospitalize untreated casescontact matrix mixing age class employ whoacquire infection matrix give relative frequency contact infective individual age people age paper assume bi directional contact child total number contact adult adult child multiplication matrix appropriate constant scaling factor matrix crude contact rate absence intervention multiply contact rate contagiousness factor obtain effective contact rate early infectious period asymptomatic case moderately sick case untreated sick case assess effect day care centre school closing transmission infectious disease assumption fraction sch contact healthy child age class occur day care centre school contact rate sick hospitalize child attend day care centre school child need reduce contact rate healthy sick child age class sick child take care adult home hospital contact rate adult increase factor HC contact rate sick child age adult age sick child child high low age class remain unchanged contact rate healthy child age sick child different age day care centre school time necessarily prevent contact happen child closing school day care centre contact rate susceptible child age infected child age late incubation period asymptomatic moderately sick reduce factor sch child age class sch function indicate school day care centre open close day care centre school close child age need adult supervision home contact susceptible adult age increase child care factor cc child care home increase exposure healthy child age contagious adult age cancel mass gathering event effect contact adult healthy attend event assume intervention time reduce contact fraction mass contact susceptible adult age infectious adult age following contact rate mass function indicate mass gathering event possible closed contact adult sick attend mass gathering event prevent measure time epidemic general population effectively reduce contact result wear facial mask increase social distance adopt improved measure respiratory hygiene simply general change behaviour implement program reduce contact susceptible individual time factor gen mass gathering event forbid mass gathering event allow population reduce contact population behave usual contact rate case late incubation period asymptomatic case remain unchanged infected individual late incubation period asymptomatic case allow contagiousness change course disease multiply contact rate weighting factor stage contagiousness lead following contact rate asymptomatic case contagiousness equally high stage stage contagious contagiousness decrease geometric procession simplify assumption contagiousness change late incubation period case stage incubation period time absence intervention generation matrix follow element fraction untreated extremely severe case die disease detail dominant eigenvalue matrix call basic reproduction number determine value contact rate give eigenvector matrix numerically calculate user specify value determine numerically scaling factor let eigenvector large eigenvalue user specify number people age class fraction people high risk age class table obtain initial population size accord age risk class initial value set differential equation solve numerically Runge kutta method step size control treatment window','2007-03-13 00:00:00','http://dx.doi.org/10.1186/1471-2334-7-17',1722,600),('f4d18f2beafa875cd4e30a9a237a64b9d3110856.json','Delivery Systems for the Direct Application of siRNAs to Induce RNA Interference (RNAi) In Vivo','RNA interference (RNAi) is a powerful method for specific gene silencing which may also lead to promising novel therapeutic strategies. It is mediated through small interfering RNAs (siRNAs) which sequence-specifically trigger the cleavage and subsequent degradation of their target mRNA. One critical factor is the ability to deliver intact siRNAs into target cells/organs in vivo. This review highlights the mechanism of RNAi and the guidelines for the design of optimal siRNAs. It gives an overview of studies based on the systemic or local application of naked siRNAs or the use of various nonviral siRNA delivery systems. One promising avenue is the the complexation of siRNAs with the polyethylenimine (PEI), which efficiently stabilizes siRNAs and, upon systemic administration, leads to the delivery of the intact siRNAs into different organs. The antitumorigenic effects of PEI/siRNA-mediated in vivo gene-targeting of tumor-relevant proteins like in mouse tumor xenograft models are described.',' altered expression level certain gene play pivotal role pathological condition example cancer upregulation certain growth factor growth factor receptor deregulation intracellular signal transduction pathway represent key element process malignant transformation progression normal cell tumor cell lead uncontrolled proliferation decreased apoptosis process result direct autocrine stimulation tumor cell paracrine stimulation cell include stimulation tumor angiogenesis novel therapeutic strategy focus reversal effect inhibition protein downregulation expression likewise disease firmly link endogenous wildtype mutate gene take addition strategy base inhibition target protein example low molecular weight inhibitor inhibitory antibody open avenue gene target approach aim decrease expression respective gene method introduce specific inhibition gene expression use antisense oligonucleotide late introduction cell antisense odn able hybridize target RNA lead degradation RNA DNA hybrid double strand RNAase inhibition translation target mrna steric conformational obstacle protein translation inhibition correct splicing early discovery ribozyme catalytically active rna able sequence specifically cleave target mrna expand gene target strategy subsequently method extensively study develop regard optimization target efficacy antisense ODN ribozyme delivery strategy vitro vivo recently naturally occur biological strategy gene silencing discover term RNA interference RNAi RNAi represent particularly powerful method specific gene silence able provide relatively easy ablation expression give target gene commonly tool biological biomedical research include rnaimediate targeting vitro vivo functional study gene expression know upregulate development novel therapeutic approach base gene target double strand RNA molecule describe elegan Fire et al introduce RNA interference finding explain early observation petunia turn white purple introduction purple gene form dsRNA gene silence antisense oligonucleotide sense oligonucleotide elegan subsequent study demonstrate RNAi describe different name posttranscriptional gene silence PTGS cosuppression quell present eukaryotic organism response dsRNA complicated high organism RNAi rely multistep intracellular pathway roughly divide phase initiation phase effector phase initiation phase double strand RNA molecule endogenous exogenous origin present cell process cleavage activity ribonuclease iii type protein short nucleotide fragment term sirnas effector sirnas contain symmetric nt overhang group effector phase incorporate nuclease contain multiprotein complex call RISC RNA induce silence complex structural biochemical study shed light processing double strand RNA formation RISC complex eg recent review unwinding siRNA duplex RNA helicase activity complex activate single strand siRNA guide RISC complex complementary target RNA binding siRNA hybridization target mrna RISC complex catalysis endonucleolytical cleavage mrna strand target site generation unprotected RNA end result rapid degradation mrna molecule RISC complex recover bind cleavage cycle process translate net reduction specific mrna level decreased expression correspond gene overview RNAi pathway figure mechanism obvious sirna molecule complementary target mrna able serve guide sequence RISC complex play pivotal role process need derive long double strand precursor molecule omit initiation phase deliver directly target cell figure upper right study lead development guideline generation sirnas optimal term efficacy specificity include initial definition preferable length bp combine low content range requirement symmetric nt overhang late study synthetic siRNA molecule reveal fold higher target efficacy case long duplex nucleotide act substrate Dicer allow direct incorporation newly produce sirnas RISC complex expect intramolecular foldback structure result internal repeat palindrome sequence decrease number functional siRNA molecule silence capability additional silencing enhance criterion include position position sense strand absence position importantly position sense strand nucleotide represent site RISC mediate cleavage target mrna indicate RISC comparable endonuclease preferentially cleave nucleotide furthermore show generally thermodynamic flexibility position sense strand correlate silence efficacy presence base pair region improve sirna mediate silence efficacy decrease internal stability different siRNA sequence display differ efficacy suggest additional unknown criterion optimal siRNA selection emphasize influence target mrna accessibility fact study correlate siRNA efficacy mrna secondary structure conclusion apart selection criterion define individual screening different sirnas highly efficient specific duplex pooling multiple sirnas effective approach increase sirnamediate target efficacy design effective sirnas algorithm publicly accessible web site available review reduce risk nonspecific target effect sirnas homology search target sequence gene database necessary incorporate web site show sirnas cross react target limited sequence similarity region partial sequence identity target mrna siRNA exist fact case region comprise contiguous nucleotide sequence identity sufficient induce gene silence prediction target activity difficult far additional mechanism lead nonspecific effect vivo relie interferon system induce double strand RNA molecule enter cell activate multi component signal complex effect particularly true long dsrna molecule essentially prevent inducer RNA interference mammalian system development synthetic sirnas largely circumvent problem small synthetic sir NAs induce component interferon system dependent sequence case vitro transcribed siRNAs initiating triphosphate strategy avoid far possible unwanted interferon response application sirnas vivo include design sirnas know interferon stimulate sequence use low possible siRNA dose achieve desire effect optimize siRNA delivery method base know mechanism antisense technology ribozyme target RNAi small oligonucleotide plasmid base expression vector specifically downregulate expression give gene interest pathological relevance vitro principle apply vivo situation lead novel potentially relevant therapeutic approach delivery therapeutic nucleic acid viral vector advantage high transfection efficacy inherent ability virus transport genetic material cell hand viral system limited loading capacity genetic material difficult produce large scale importantly pose severe safety risk oncogenic potential inflammatory immunogenic effect prevent repeat administration light problem concern limitation nonviral system emerge promising alternative gene delivery main requirement protection nucleic acid load efficient uptake target cell subsequent release DNA RNA molecule necessary transfer nucleus strategy distinguish mainly lipofection polyfection rely cationic lipid polymer respectively eg efficient protection enzymatic nonenzymatic degradation particularly important RNA molecule include sirnas fact therapeutic potential sirnas treatment disease principle promising limitation transfer vector turn rate limiting development RNAi base therapeutic strategy approach solve problem use dna expression plasmid encode palindromic hairpin loop desire sequence transcription folding RNA doublestranded short hairpin RNAs shRNAs recognize Dicer cleave desire sirnas additionally vitro method describe recently base expression shrnas coli delivery bacterial invasion different DNAbased system offer advantage siRNA expression long duration probably high level gene silence rely viral nonviral delivery dna molecule raise safety issue vivo direct delivery sirnas molecule derive vitro transcription chemically synthesize offer advantage dna base strategy preferable vivo therapeutic use year large body study publish describe different strategy systemic local application sirnas vivo table overview probably large number paper focus use unmodified sirnas table administration perform IV hydrodynamic transfection high pressure tail vein injection method widely case lead efficient target gene inhibition liver less extent lung spleen pancrea kidney suffer certain technical practical limitation therapeutical setting rely rapid IV injection comparably large volume ml mouse injection theory equivalent IV bolus injection man alternative strategy application naked sirnas include delivery route provide local administration rely administration close target tissue target organ restrict number target organ relevant certain disease note study describe use large amount sirnas intravenous injection sirnas liver primary site siRNA uptake alternative approach application siR NAs vivo delivery liposome cationic lipid describe liposome base siRNA formulation wide variety mode application allow local systemic delivery table finally strategy local systemic siRNA administration explore include chemical modification siRNA molecule electropulsation polyamine basic complex atelocollagen virosome certain protein preparation table alternative approach rely complexation unmodified siRNA molecule cationic polymer polyethylenimine pei synthetic polymer available branched linear form figure upper panel broad range molecular weight Da kd commercial PEI preparation label define molecular weight consist PEI molecule broad molecular weight distribution pei possess high cationic charge density protonable amino group position quarternary amino group present cationic charge generate protonation amino group dependent pH environment eg pH review ability condense compact dna complex form small colloidal particle allow efficient cellular uptake endocytosis PEI introduce potent dna transfection reagent variety cell line animal dna delivery review reference fact study PEI show able deliver large dna molecule Mb yeast artificial chromosome YACs plasmid small oligonucleotide mammalian cell vitro vivo ratio indicate ratio nitrogen atom PEI DNA phosphate complex describe PEI complex formation independent molecular weight influence efficiency dna delivery positive net charge complex result high ratio inhibit electrostatic repulsion aggregation improve solubility aqueous solution interaction negatively charge extracellular matrix component cellular uptake additionally strong buffer capacity describe proton sponge hypothesis postulate enhance transgene delivery cationic polymer dna complex polyplexe contain buffer polyamine enhanced endosomal Cl accumulation osmotic swelling lysis responsible fact PEI base delivery require endosome disruptive agent lysosomal escape tight condensation dna molecule buffering capacity PEI certain cellular compartment like endosome lysosome protect dna degradation pei successfully nonviral gene delivery vitro vivo initial publication show increase transfection efficacy high molecular weight pei recent study demonstrate advantage certain low molecular weight pei high transfection efficacy low molecular weight pei efficient uptake result PEI dna complex well intracellular release dna low vitro cytotoxicity compare high molecular weight PEI fact decrease molecular weight PEI lead increase complex size favourable vitro use hand pei low molecular weight kd display little transfection efficacy high ratio attribute fact decrease molecular weight PEI show translate increasingly low ability form small complex low molecular weight pei require high ratio optimal transfection efficacy compare high molecular weight pei high ratio lead increase compaction reduce complex size reduce tendency complex aggregate hydrophobic interaction parameter extensively study precise determinant transfection efficacy remain elucidate review mechanism cytotoxic effect PEI complex poorly understand rely formation large aggregate range μm form cell surface impairs membrane function finally lead cell necrosis clearly trend low molecular weight pei nontoxic delivery reagent vitro vivo combine high biocompatibility reduce effect allow employ large PEI dna complex amount significant cytotoxicity recently use polyethylenimine extend complexation delivery RNA molecule especially small RNA molecule like nt RNA ribozyme sirnas figure chemically unmodified RNA molecule instable prone rapid degradation PEI complexation show lead complete protection enzymatic nonenzymatic degradation fact PEI complexe siR NAs well detection remain intact vitro hour presence RNase fetal calf serum non complexed sirnas rapidly degrade figure indicate siRNA molecule efficiently condense fully cover protect PEI analysis PEI siRNA complex atomic force microscopy show absence free siRNAs siRNA molecule end confirm finding efficient complexation Grzelinski et al submit complex stability sufficient siRNA protection pei test Werth et al press Aigner et al unpublished datum complex target efficacy fact certain polyethylenimine PEI siRNA complex efficiently deliver target cell vitro sirnas release display bioactivity figure general see PEI dna complex transfection efficacy dependent PEI indicate sirna target efficiency mainly depend endocytotic uptake complex intracellular decomposition vitro complex stability good result obtain commercially available jetpei vivo jetpei supplier show poor sirna delivery efficacy likewise novel low molecular weight PEI base fractionation commercially available polyethylenimine demonstrate high siRNA protection delivery efficacy vitro Werth et al press certain condition PEI RNA siRNA ribozyme complex retain physical stability biological activity lyophilization Werth et al press PEI transfection transient datum lab PEI siRNA effect stable day Urban Klein Aigner unpublished result finally study explore use siRNA nanoplexe comprise PEI pegylate RGD peptide ligand attach distal end PEI siRNA nanoplexe protect sirnas serum degradation vitro activity ultimate goal application sirnas vivo explore study different mouse model ge et al show PEI complexe sirnas targeting conserve region influenza virus gene able prevent treat influenza virus infection mouse IV injection PEI promote delivery sirnas lung give virus infection siRNA reduce influenza virus production lung biological effect systemic application peicomplexed sirnas determine different mouse tumor model target different protein show previously tumor relevant include epidermal growth factor receptor neu growth factor pleiotrophin PTN vascular endothelial growth factor VEGF receptor VEGF fibroblast growth factor bind protein fgf vivo administration PEI complexe naked sirnas ip subcutaneous injection result detection intact sirnas hour injection figure radiolabele siRNA molecule find organ include subcutaneous tumor muscle liver kidney small extent lung brain important note sirnas actually internalize tissue indicate fact blood negative sirnas Figure receptor observe wide variety human cancer cancer cell line display strong cell growth stimulate antiapoptotic effect especially heterodimer formation member EGFR family overexpression establish negative prognostic factor link aggressive malignant behaviour tumor eg consequently qualify attractive target molecule antitumoral treatment strategy include anti antibody low molecular weight inhibitor specific gene target approach fact relevance tumor growth establish vitro target study include use ribozyme sirnas propose mechanism PEI mediate siRNA transfer electrostatic interaction PEI able complex negatively charge sirnas lead compaction formation small colloidal particle endocytose proton sponge effect exhibit PEI complex lead osmotic swelling ultimately disruption endosome sirnas protect degradation tight condensation complex buffering capacity PEI release PEI base complex intact sirnas incorporate RISC complex induce RNAi figure demonstrate reduction vitro lead inhibition cell proliferation increase apoptosis systemic treatment athymic nude mouse bear subcutaneous ovarian carcinoma tumor xenograft ip injection PEI complexe specific sirnas lead mark antitumoral effect see significant reduction tumor growth figure peicomplexe nonspecific sirnas specific naked sir NAs effect parallel detection intact specific sirnas tumor specific treatment group min administration downregulation mrna protein level receptor VEGF target study employ self assemble nanoparticle base sirnas complexe PEI pegylate RGD peptide ligand attach distal end PEG pegylation allow steric stabilization reduce nonspecific interaction complex RGD motif provide tumor selectivity ability target integrin express activate endothelial cell tumor vasculature IV administration mouse bear subcutaneous neuroblastoma tumor xenograft selective tumor uptake vegf downregulation observe result decrease tumor growth tumor angiogenesis receptor ligand VEGF mitogenic angiogenic growth factor stimulate tumor growth angiogenesis tumor include prostate carcinoma represent attractive target molecule RNAi base genetargete strategy bear mind double antitumoral effect reduction tumor cell proliferation tumor angiogenesis subcutaneous intraperitoneal injection vegf specific sirnas complexe novel pei obtain fractionation commercially available PEI Werth et al press result reduction tumor growth decrease vegf expression level Höbel Aigner unpublished result true PEI siRNA mediate targeting FGF BP Dai Aigner unpublished result establish figure protection vivo delivery sirnas PEI complexation vitro protection sirnas nucleolytic degradation end label sirnas complexe upper panel complexed low panel PEI subject treatment fetal calf serum time point indicate sample analyse agarose gel electrophoresis blot autoradiography band represent length siRNA molecule indicate PEI complexation lead efficient protection sirnas noncomplexed sirnas rapidly degrade vivo delivery intact sirnas PEI complexation sirnas complexe complexed PEI inject IP mouse bear subcutaneous ovarian carcinoma cell tumor xenograft min total RNA organ tissue homogenate prepare subject agarose gel electrophoresis prior blotting autoradiography band represent intact siRNA molecule hour mainly find tumor muscle liver time dependently kidney little sirna amount detect lung trace rate limit tumor growth angiogenesis tumor PEI siRNA mediate targeting pleiotrophin ptn exert strong antitumoral effect ptn secrete growth factor show mitogenic chemotactic angiogenic transform activity markedly upregulated human tumor include cancer breast testis prostate pancrea lung melanomas meningioma neuroblastoma glioblastoma vivo treatment nude mouse systemic subcutaneous IP application PEI complexe PTN sirnas lead delivery intact sirnas subcutaneous tumor xenograft significant inhibition tumor growth likewise clinically relevant orthotopic mouse glioblastoma model cell grow intracranially injection PEI complexe ptn sirnas CNS exert antitumoral effect establishe complex relevant orthotopic tumor model potential PEI siRNA mediate ptn gene target novel therapeutic option GBM extend mode delivery PEI siRNA complex intrathecal strategy employ therapy glioblastoma antisense oligonucleotide year discovery sirnas catch ribozyme antisense oligonucleotide efficient tool gene target vitro recently vivo include exploration potential therapeutic lead development siRNA base therapeutic strategy ultimate success figure systemic treatment mouse PEI complexe specific sirnas lead reduced growth subcutaneous tumor xenograft decrease expression athymic nude mouse bear subcutaneous tumor xenograft inject IP nmole specific naked open circle PEI complexe closed circle sirnas time week tumor size evaluate daily product perpendicular diameter tumor mean error mean SEM depict Student unpaired test comparison datum set difference tumor growth reach significance day indicate antitumoral effect PEI complexe specific sirnas strongly depend development powerful feasible sirna delivery strategy need address issue include stability stabilization siRNA molecule preserve efficacy maintain genesilencing activity efficient delivery target sufficiently long siRNA half life organism particularly target organ sirna delivery strategy provide siRNA protection transfection efficacy absence toxic nonspecific effect efficacious small amount sirnas applicable treatment regimen disease require overcome biological barrier administration reach target tissue target organ research dna base gene delivery ribozyme target antisense technology facilitate process provide basis establish technology true complexation sirnas polyethylenimine represent promising avenue siRNA application vivo eventually lead novel therapeutic strategy work Aigner support Deutsche Forschungsgemeinschaft AI Deutsche Krebshilfe author like apologize author primary work cite length consideration','2006-05-18 00:00:00','http://dx.doi.org/10.1155/jbb/2006/71659',2625,911),('faaf1022ccfe93b032c5608097a53543ba24aedb.json','Heme oxygenase-1 and carbon monoxide in pulmonary medicine','Heme oxygenase-1 (HO-1), an inducible stress protein, confers cytoprotection against oxidative stress in vitro and in vivo. In addition to its physiological role in heme degradation, HO-1 may influence a number of cellular processes, including growth, inflammation, and apoptosis. By virtue of anti-inflammatory effects, HO-1 limits tissue damage in response to proinflammatory stimuli and prevents allograft rejection after transplantation. The transcriptional upregulation of HO-1 responds to many agents, such as hypoxia, bacterial lipopolysaccharide, and reactive oxygen/nitrogen species. HO-1 and its constitutively expressed isozyme, heme oxygenase-2, catalyze the rate-limiting step in the conversion of heme to its metabolites, bilirubin IXα, ferrous iron, and carbon monoxide (CO). The mechanisms by which HO-1 provides protection most likely involve its enzymatic reaction products. Remarkably, administration of CO at low concentrations can substitute for HO-1 with respect to anti-inflammatory and anti-apoptotic effects, suggesting a role for CO as a key mediator of HO-1 function. Chronic, low-level, exogenous exposure to CO from cigarette smoking contributes to the importance of CO in pulmonary medicine. The implications of the HO-1/CO system in pulmonary diseases will be discussed in this review, with an emphasis on inflammatory states.',' heme carbon monoxide CO system recently see explosion research interest newly discover physiological effect metabolic pathway characterize Tenhunen et al recently reveal surprising cytoprotective property research CO embrace entire field medicine reactive oxygen nitrogen specie inflammation growth control apoptosis represent important pathophysiological mechanism number publication recent year concern increase exponentially list disease physiological response associate change continue expand relatively study address role CO pulmonary medicine investigator focus diagnostic application CO system measure exhaled CO CO pathological pulmonary condition asthma chronic obstructive pulmonary disease COPD experimental approach investigator examine expression lung tissue healthy diseased subject review highlight action CO context heme degradation antioxidant property liberated heme iron undergoe detoxification extracellular efflux sequestration ferritin intracellular iron storage molecule potential cytoprotective function know isoform HO respond xenobiotic induction constitutively express tissue occur high level nervous vascular tissue respond regulation glucocorticoid differ genetic origin primary structure molecular weight substrate kinetic parameter display high sequence homology little enzymatic activity review focus inducible form addition physiological substrate heme respond induction wide variety stimulus associate oxidative stress induce agent include hypoxia hyperoxia cytokine nitric oxide heavy metal ultraviolet nm radiation heat shock shear stress hydrogen peroxide thiol substance multiplicity toxic inducer suggest function critical cytoprotective molecule study suggest act inducible defense oxidative stress model inflammation ischemia reperfusion hypoxia hyperoxia mediate injury review mechanism mediate cytoprotection poorly understand product HO reaction potentially participate cellular defense gaseous molecule CO recently receive attention administration CO low concentration compensate protective effect presence competitive inhibitor activity gene transfer confer protection oxidative stress number system clearly study support beneficial role expression cell culture study suggest protective effect overexpression fall critical range excess production counterprotective transient excess reactive iron generate active heme metabolism important caveat comparative study therapeutic effect CO administration versus gene delivery arise fact approach addition produce CO profound effect intracellular iron metabolism expression primarily regulate transcriptional level genetic analysis reveal enhancer sequence murine gene locate kb kbp transcriptional start site enhancer mediate induction agent include heavy metal phorbol ester endotoxin oxidant heme contain repeat stress responsive element consist overlap bind site transcription factor include activator maf oncoprotein basic leucine zipper family protein CNC bZIP NF relate factor play critical role transcription promoter region contain potential bind site nuclear factor κB NF κB functional significance clear NF κB identify regulatory element responsive oxidative cellular stress response hyperoxic stress factor mediate induction cooperation signal transducer activator transcription STAT protein furthermore distinct hypoxia response element HRE mediate response hypoxia represent bind site hypoxia inducible toxic property CO known field pulmonary medicine invisible odorless gas claim victim year accidental exposure CO evolve combustion organic material present smoke automobile exhaust toxic action CO relate high affinity hemoglobin fold great CO replace rapidly hemoglobin cause tissue hypoxia high concentration mechanism CO induce toxicity include apoptosis lipid peroxidation inhibition drug metabolism respiratory enzyme function recently know low concentration CO participate physiological reaction CO exposure part million ppm volume CO air toxic ppm hundredth fortieth stimulate physiological effect apparent toxicity majority endogenous CO production originate active heme metabolism portion produce lipid peroxidation drug metabolism reaction cigarette smoking practice lung patient represent major source chronic lowlevel exposure CO Inhaled CO initially target alveolar macrophage respiratory epithelial cell exact mechanism CO act molecular level remain incompletely understand CO potentially exert physiological effect influence know pathway Fig complexation heme moiety enzyme CO activate soluble guanylate cyclase sGC stimulate production cyclic monophosphate cgmp sgc cgmp pathway mediate effect CO vascular relaxation smooth muscle cell relaxation bronchodilation neurotransmission inhibition platelet aggregation coagulation smooth muscle proliferation furthermore CO cause vascular relaxation directly activate calcium dependent potassium channel CO potentially influence intracellular signal transduction pathway mitogen activate protein kinase MAPK pathway transduce oxidative stress inflammatory signaling response lipopolysaccharide represent important target possible carbon monoxide action figure possible carbon monoxide action endogenous carbon monoxide CO arise principally product heme metabolism action heme oxygenase enzyme portion arise environmental source pharmacological administration accidental exposure endogenous process drug lipid metabolism vasoregulatory property CO include effect cellular proliferation platelet aggregation vasodilation largely ascribe stimulation guanylate cyclase direct heme binding lead generation cyclic GMP anti inflammatory property CO associate downregulation proinflammatory cytokine production dependent selective modulation mitogen activate protein kinase MAPK kilodalton protein addition mechanism CO potentially interact hemoprotein target functional consequence interaction respect cellular signal remain poorly understand Anti Platelet Aggregation Anti Proliferation inhibition pro inflammatory cytokine productionmodulation hemoprotein function CO action anti apoptotic effect CO relation MAPK recently describe overexpression exogenous administration CO prevent tumor necrosis factor TNF apoptosis murine fibroblast endothelial cell anti apoptotic effect CO depend modulation kilodalton protein mapk pathway role remain heme metabolite fe biliverdin IXα modulation apoptosis currently investigate scope review recent study report potent anti inflammatory effect CO involve inhibition proinflammatory cytokine production endotoxin stimulation dependent modulation MAPK clinical relevance MAPK lie possibility modulate pathway clinical condition downregulate inflammatory response stress arise imbalance oxidant antioxidant play central role pathogenesis airway disease lung tissue expression occur respiratory epithelial cell fibroblast endothelial cell large extent alveolar macrophage induction tissue vitro vivo respond common cause oxidative stress airway include hyperoxia hypoxia endotoxemia heavy metal exposure bleomycin diesel exhaust particle allergen exposure induction administration CO protect cell stressful stimulus experiment well illustrate protective role CO vivo rat expose hyperoxia absence presence CO low concentration ppm CO treat rat show increase survival diminished inflammatory response hyperoxia demonstrate model endotoxin induce inflammation protection afford CO likely result downregulate synthesis proinflammatory cytokine TNF upregulation anti inflammatory cytokine furthermore increase exhaled CO CO report number pathological pulmonary condition unstable asthma copd infectious lung disease increase reflect increase endogenous activity elevated carboxyhemoglobin Hb CO level report disease nonsmoke subject CO Hb CO level decrease normal level response therapy CO human originate primarily systemic heme metabolism produce CO tissue localize lung heme metabolism result combined action inducible constitutive enzymatic activity endogenously produce inspire CO eliminate exclusively respiration elevation CO reflect increase exogenous source smoking air pollution addition change environmental factor elevation CO lung disease reflect increase blood Hb CO level response systemic inflammation increase pulmonary expression response local inflammation diagnostic value measure CO remain controversial conflicting report report indicate difference CO measurement disease activity control report possible explanation discrepancy include large difference patient population method measure CO undefined correction background level CO furthermore remarkable difference arise study magnitude CO level control group treat untreated asthma patient active passive smoking occur presence high background level CO measurement CO particularly useful monitor airway inflammation patient smoke CO confirm smoking habit comparable begin era measurement exhaled standardization technique agreement background correction reach CO measurement allow proper conclusion draw area investigation Asthma form allergic lung disease feature accumulation inflammatory cell mucus airway associate bronchoconstriction generalize airflow limitation Inflammation key component asthma involve multiple cell mediator imbalance oxidant antioxidant contribute cell damage pathway associate oxidative stress participate asthma example redox sensitive transcription factor NF κB control expression proinflammatory mediator light potential protective effect CO inflammatory process study asthma gain popularity mouse model asthma expression increase lung tissue response ovalbumin aerosol challenge indicate role asthma similar model aeroallergen induce asthma ovalbumin sensitize mouse exposure CO atmosphere result marked attenuation eosinophil content bronchoalveolar lavage fluid BALF downregulation proinflammatory cytokine experiment show exogenous CO inhibit asthmatic response allergen mouse recent human study reveal high expression alveolar macrophage high CO untreated asthmatic patient healthy nonsmoking control patient exacerbation asthma patient withdraw inhaled steroid show high CO level steroid treat asthmatic healthy control high level CO occur child persistent asthma healthy control CO level correlate functional parameter peak expiratory flow rate low rate asthma exacerbation correlate high CO normalization rate oral glucocorticoid treatment result reduction CO furthermore increase CO associate great expression airway alveolar macrophage obtain induce sputum untreated asthmatic patient control asthma patient show high bilirubin level induce sputum indicate high HO activity furthermore patient asthma increase Hb CO level time exacerbation value decrease control level oral glucocorticoid treatment human asthmatic CO airway eosinophil count decrease response month treatment inhaled corticosteroid direct contrast study promote CO useful noninvasive tool monitor airway inflammation study report difference CO level asthma patient versus healthy control patient stable unstable asthma report change CO occur asthma patient month treatment inhaled corticosteroid despite observed decrease airway eosinophil content bronchial responsiveness metacholine recent study accentuate finding asthma excerbation decrease CO child asthma detect oral prednisolone treatment human allergic response result elevation CO inconclusive clear elevation CO allergen exposure occur patient asthma late response early response immediately inhalation report show elevation CO occur allergen induce asthma hour allergen challenge finally increase CO measure allergic rhinitis correlate seasonal change exposure allergen pollen inflammation play important role development copd characterize presence macrophage neutrophil inflammatory mediator proteinase oxidant cytokine inflammatory consequence chronic microbiological infection contribute progression disease current paradigm pathogenesis COPD involve imbalance protease antiprotease activity antioxidant pro oxidant status protease tissue degrade capacity elastase matrix metalloproteinase insufficiently inhibit antiprotease induce tissue damage lead emphysema oxidant supersede cellular antioxidant defense furthermore inactivate antiprotease cause direct injury lung tissue interfere repair extracellular matrix smoking play important role hypothesis cigarette smoke act primarily alveolar macrophage epithelial cell react oxidative stress produce proinflammatory cytokine chemokine release growth factor smoking factor development copd smoker develop disease reactive oxygen specie cigarette smoke chronic infection imbalance oxidant antioxidant status main risk factor development COPD defend oxidative stress cell tissue contain endogenous antioxidant defense system include millimolar concentration tripeptide glutathione GSH close relation exist GSH concentration depletion GSH augment transcriptional regulation oxidant suggest CO system act secondary defense oxidative stress accumulate clinical evidence suggest CO play important COPD Alveolar macrophage produce strong response stimulus represent main source CO production airway patient COPD display high CO healthy nonsmoking control furthermore high level observe airway smoker nonsmoker subject smoke patient COPD low expression alveolar macrophage healthy subject microsatellite polymorphism link development COPD occur promoter region result low production people polymorphism genetically dependent downregulation expression arise subpopulation possibly link increase susceptibility oxidative stress future study genetic predisposition possible therapeutic modality reveal involvement CO system fibrosi CF involve deposition hyperviscous mucus airway associate pulmonary dysfunc tion pancreatic insufficiency accompany chronic microbiological infection CO reading high untreated versus oral steroidtreate CF patient furthermore CO increase patient exacerbation CF correlate deterioration forced expiratory volume second FEV normalization CO level treatment CO level correlate exhaled ethane product lipid peroxidation serve indirect marker oxidative stress CO exhaled ethane high steroid treat untreated CF patient healthy control CO high child CF control patient addition inflammatory oxidative stress response continuous infectious pressure patient CO possibly respond hypoxia CO increase CF child follow exercise test correlate degree oxyhemoglobin desaturation finding suggestive increase expression CF patient hypoxic state induce exercise patient pneumonia high Hb CO level measure onset illness value decrease control level antibiotic treatment CO level report high low respiratory tract infection bronchiectasis normalization antibiotic treatment furthermore CO level upper respiratory tract infection high healthy control relationship high measure CO infectious state high Hb CO level conclude study role development interstitial lung disease remain undetermined comparative immunohistochemical analysis reveal lung tissue control subject patient sarcoidosis usual interstitial pneumonia desquamative interstitial pneumonia show high expression alveolar macrophage weak expression fibrotic area antiproliferative property suggest possible beneficial role limit fibrosi hypothesis complicate newly discover relation produce bronchial epithelial cell promote growth proliferation lung fibroblast expression CO treatment show increase production macrophage follow proinflammatory stimulus conversely induce production apparently require anti inflammatory action recent report clearly show suppression bleomycin induce pulmonary fibrosi adenovirus mediate gene transfer overexpression mouse involve inhibition apoptotic cell death research need elucidate mechanism interstitial lung disease possible therapeutic implication action great importance solid tumor environment foster hypoxia oxidative stress neovascularization pro antagonistic effect tumor growth survival CO cause growth arrest cell culture system represent potential therapeutic modality modulate tumor growth overexpression administration CO mesothelioma adenocarcinoma mouse model result improved survival reduction tumor size furthermore expression oral squamous cell carcinoma useful identify patient low risk lymph node metastasis high expression detect group lymph node metastasis report contrast growth arrest protect solid tumor oxidative stress hypoxia possibly promote neovascularization study zinc protoporphyrin competitive inhibitor enzyme activity suppress tumor growth represent critical mediator body adaptive response hypoxia common feature pulmonary vascular disease CO modulate vascular tone induce cgmp large calcium dependent potassium channel CO probably play important role pulmonary vascular disease NOmediated induction occur hepatopulmonary syndrome cirrhosis associate enhancement vascular relaxation portopulmonary hypertension elevated level cgmp inducible nitric oxide synthase inos expression vascular endothelium expression macrophage bronchial epithelium describe transgenic mouse model mouse differ development pulmonary hypertension follow chronic hypoxia treatment despite development right ventricular dilation right myocardial infarction preinduction protein chemical inducer prevent development pulmonary hypertension rat lung consequence chronic hypoxia treatment transgenic mouse overexpresse lung resistant hypoxia induce inflammation hypertension research need elucidate potential role CO primary human lung vascular disease primary pulmonary hypertension supplemental oxygen therapy clinically treatment respiratory failure exposure high oxygen tension hyperoxia cause acute chronic lung injury induce extensive inflammatory response lung degrade alveolar capillary barrier lead impaired gas exchange pulmonary edema Hyperoxia induce lung injury cause symptom rodent resemble human acute respiratory distress syndrome induce expression adult rat apparently neonatal rat expression activity developmentally upregulated prenatal early postnatal period potentially influence pulmonary adaptation high level example adenoviral mediate gene transfer rat lung protect development lung apoptosis inflammation hyperoxia vitro study show overexpression lung epithelial cell rat fetal lung cell cause growth arrest conferred resistance hyperoxia induce cell death oxygen tolerant variant hamster fibroblast moderately overexpresse comparison parent line resist oxygen toxicity vitro treatment oxygen tolerant strain antisense oligonucleotide reduce resistance hyperoxia contrast additional vector mediate expression increase oxygen tolerance model vivo study gene delete mouse strain provide information role oxygen tolerance Dennery et al demonstrate heme knockout mouse sensitive lethal effect hyperoxia wild type mouse addition absence expression mouse display compensatory increase protein expression high total lung HO activity model combination deletion overexpression result hyperoxiasensitive phenotype recent study Dennery et al show delete mouse resistant lethal effect hyperoxia correspond wild type hyperoxia resistance observe reverse reintroduction adenoviral mediate gene transfer contrast mouse embryo fibroblast derive show increase sensitivity toxic effect hemin generate intracellular reactive oxygen specie response agent anemic display abnormal accumulation tissue iron specifically nonheme iron kidney liver decrease total iron content lung accumulate total lung iron absence compensatory increase ferritin level deletion result accumulation tissue iron remain unclear study take indicate animal deficient display alter sensitivity oxidative stress condition aberration distribution intra extra cellular iron underlie differential sensitivity observe et al show exogenous CO anti inflammatory action protect lung rat model hyperoxia induce lung injury presence CO ppm prolong survival rat hyperoxic environment inhibit appearance marker hyperoxia induce lung injury hemorrhage fibrin deposition edema airway protein accumulation BALF neutrophil influx furthermore mouse model CO inhibit expression proinflammatory cytokine TNF mouse induce hyperoxia treatment genedelete mouse Otterbein colleague observe protection afford CO model similar lipopolysaccharide induce model lung injury depend mapk pathway Otterbein et al unpublished observation review direct contrast study group Piantadosi colleague report significant difference hyperoxia tolerance rat CO dose ppm model CO alter accumulation fluid airway furthermore CO apply combination hyperoxia increase activity myeloperoxidase marker airway neutrophil influx study suggest inhalation CO ppm alter expression antioxidant enzyme manganese superoxide dismutase mnsod vivo furthermore Piantadosi colleague able induce oxygen tolerance rat expression hemoglobin treatment tolerance occur presence HO inhibitor support role HO activity oxygen tolerance consensus reach protective role CO inhalation induction hyperoxic lung injury human study require CO supersede provide significant therapeutic benefit context severe lung disease antioxidant therapy examine human study exist role CO acute respiratory distress syndrome ARDS bronchopulmonary dysplasia transplantation ultimate therapeutic option end stage lung disease lung transplantation remain ongoing hazardous situation acute chronic graft failure complication toxic immunosuppressive regimen severe bacterial fungal viral infection renal failure Epstein Barr virus relate lymphoma determine outcome development chronic graft failure obliterative bronchiolitis OB determine overall outcome lung transplantation ob develop month transplantation main cause morbidity death follow half year transplantation despite therapeutic intervention OB develop retransplantation remain therapeutic option available little know pathophysiological background OB possible determinant develop OB include ongoing immunological allograft response HLADR mismatch cytomegalovirus infection acute rejection episode organ ischemia time recipient age ob patient display elevated neutrophil count BALF evidence increase oxidant activity increase methionine oxidation BALF protein decrease ratio GSH oxidize glutathione GSSG epithelial lining fluid far limited research datum available possible role allograft rejection lung transplantation high expression detect alveolar macrophage lung tissue lung transplant recipient acute chronic graft failure stable recipient protective role allograft rejection show transplantation model solid organ transplantation typically benefit modulation high expression protective gene observe episode acute renal allograft rejection furthermore induction alleviate graft versus host disease Adenoviral gene therapy result remarkable protection rejection rat liver transplant upregulation protect pancreatic islet cell Fas mediate apoptosis dose dependent fashion support anti apoptotic function confer protection early phase transplantation induce dependent cytokine suppress interferon production demonstrate rat liver allograft model effect modulation describe xenotransplantation model gene expression appears functionally associate xenograft survival mouse rat heart trans plant model effect upregulation mimic CO administration suggest HOderived CO suppress graft rejection author propose CO suppress graft rejection inhibition platelet aggregation process facilitate vascular thrombosis myocardial infarction contribute ischemic preconditioning process acquire cellular protection reperfusion injury observe guinea pig transplant lung overexpression provide potent protection cold ischemia reperfusion injury rat model anti apoptotic pathway induction rat undergo liver transplantation cobalt protoporphyrin adenoviral gene therapy result protection reperfusion injury improved survival transplantation possibly suppression cytokine production decrease apoptosis reperfusion report address CO measurement lung transplantation possible difference CO find patient acute chronic allograft rejection evolution CO exhaled breath serve general marker diagnostic indicator inflammatory disease state lung research require verify reliability increase exhaled CO presumably reflect change systemic airway heme metabolic activity action HO enzyme evidence numerous vitro animal study indicate provide protective function disease involve inflammation oxidative stress exploitation therapeutic gain achieve modulation enzyme activity downstream regulatory factor gene transfer pharmacological inducer direct application CO gas administration chemical delivery CO release molecule transition metal carbonyls develop Motterlini et al promise pharmacological delivery CO therapeutic application vascular immune regulation CO release molecule show limit hypertension vivo promote vasorelaxation isolated heart aortic ring challenge remain apply therapeutic potential treatment human disease vivo model transplantation show gene therapy protect allograft rejection give toxic therapy transplant patient receive especially lung transplantation field transplantation medicine bring frontier human application gene therapy exogenous CO administration potential use inhalation CO clinical therapeutic inflammatory lung disease appear horizon promising study inhalation dose ppm CO rate time day week produce cardiovascular effect cigarette smoking CO inhalation identical interval produce comparable Hb CO level approximately question CO inhalation therapy soon replace question long fear administer CO weigh severe toxicity immunosuppressive agent current use negative outcome solid organ transplantation','2003-08-07 00:00:00','http://dx.doi.org/10.1186/1465-9921-4-7',3083,1002),('fb3c146b1766b4aa33df1637742bf93159967289.json','Natural Killer Cells Promote Early CD8 T Cell Responses against Cytomegalovirus','Understanding the mechanisms that help promote protective immune responses to pathogens is a major challenge in biomedical research and an important goal for the design of innovative therapeutic or vaccination strategies. While natural killer (NK) cells can directly contribute to the control of viral replication, whether, and how, they may help orchestrate global antiviral defense is largely unknown. To address this question, we took advantage of the well-defined molecular interactions involved in the recognition of mouse cytomegalovirus (MCMV) by NK cells. By using congenic or mutant mice and wild-type versus genetically engineered viruses, we examined the consequences on antiviral CD8 T cell responses of specific defects in the ability of the NK cells to control MCMV. This system allowed us to demonstrate, to our knowledge for the first time, that NK cells accelerate CD8 T cell responses against a viral infection in vivo. Moreover, we identify the underlying mechanism as the ability of NK cells to limit IFN-α/β production to levels not immunosuppressive to the host. This is achieved through the early control of cytomegalovirus, which dramatically reduces the activation of plasmacytoid dendritic cells (pDCs) for cytokine production, preserves the conventional dendritic cell (cDC) compartment, and accelerates antiviral CD8 T cell responses. Conversely, exogenous IFN-α administration in resistant animals ablates cDCs and delays CD8 T cell activation in the face of NK cell control of viral replication. Collectively, our data demonstrate that the ability of NK cells to respond very early to cytomegalovirus infection critically contributes to balance the intensity of other innate immune responses, which dampens early immunopathology and promotes optimal initiation of antiviral CD8 T cell responses. Thus, the extent to which NK cell responses benefit the host goes beyond their direct antiviral effects and extends to the prevention of innate cytokine shock and to the promotion of adaptive immunity.',' development antiviral immune response involve orchestration complex network innate adaptive immune cell promote health disease natural killer NK cell plasmacytoid dendritic cell pDCs conventional dendritic cell cdcs cell cell demonstrate important generation protective immunity viral infection antiviral defense coordinate particular function different type immune cell impact shaping global immune response virus vivo thoroughly understand importance efficient nk cell response promotion favorable outcome viral infection demonstrate mouse human rapid activation nk cell infection hallmark potency innate immune system effector increase evidence support idea optimal coordination immune response involve intricate relationship NK cell innate leukocyte example report document importance NK DC crosstalk reciprocal activation cell type promotion antitumor immunity recently review show involvement macrophage intermediary activation NK cell NK cell nk cell show capacity interact neighbor nk cell cell stimulate cellular proliferation initially identify human mouse base unique ability cell secrete enormous amount IFN early response viral challenge pdcs respond way consequence ability recognize molecular signature virus manner independent pdc infection early course murine cytomegalovirus MCMV infection pDCs major producer interferon IFN critical host survival IFN direct antiviral effect inhibit viral replication infected cell convey noninfecte cell defense mechanism protect viral infection IFN mediate variety immunoregulatory function activate inhibitory contribute proliferation NK cell induce express functional cytotoxic granule promote cdc phenotypic functional maturation help activate antiviral cell contrast depend context IFN compromise immune response host inhibit DC differentiation directly lead attrition cell different virus recently suggest induce profound immunosuppression host induce overwhelming level IFN importance host genotype efficiency IFN antiviral function context infection specific virus document contrast know naturally occur difference interaction virus host exist shape IFN response specifically dampen immunosuppressive function cytokine detrimental host beneficial pathogen identification condition help well understand physiopathology viral infection lead development innovative treatment fight infection early phase MCMV infection tight temporal relationship activation exertion effector function NK cell pdcs production IFN pdcs promote nk cell proliferation cytotoxicity impact NK cell function pdc response MCMV remain unexplored goal study evaluate effect efficient NK cell response pDC IFN production development adaptive immune response host mouse able mount NK cell response control MCMV replication efficiently expression activate receptor endow nk cell ability recognize kill viral infect cell express viral ligand review NK cell BALB strain mouse lack gene strongly activate acquisition cytotoxic machinery production IFN early MCMV infection fail recognize kill infected cell consider inefficient inbred mouse strain resistant sensitive MCMV infection nk cell dependent manner differ immune parameter include major histocompatibility MHC haplotype DC subset frequency function rigorous evaluation impact NK cell response pdc adaptive response require comparison mouse genetic background differ ability nk cell control infection take advantage fact activation axis sufficient NK cell control MCMV introduce BALB genome response pDCs cell MCMV compare BALB mouse animal BALB background congenic natural killer complex particular mouse name system provide unique model study antiviral immune response vivo context efficient inefficient NK cell activation absence broad defect NK cell development activity affect homeostasis function immune cell system allow demonstrate knowledge time NK cell accelerate cell response viral infection vivo identify underlie mechanismthe ability NK cell limit pDC IFN production level immunosuppressive host result early control viral replication trigger require proliferation accumulation NK cell day MCMV infection NK cell expansion indicator BALB mouse generate nk cell response different quality MCMV infection Figure compare ability BALB mouse control early viral load consistent report difference viral load see mouse lack possess gene substantially high viral burden spleen BALB mouse compare congenic animal Figure addition nk cell depletion mouse increase viral replication level similar observe BALB animal Figure datum confirm previous report demonstrate mouse mount strong nk cell response able efficiently control MCMV replication contrast BALB animal evaluate impact efficient antiviral nk cell activity induction IFN examine level cytokine present serum BALB mouse serum level IFN dramatically low peak response day decrease fight viral infection vertebrate develop battery innate adaptive immune response aim inhibit viral replication kill infected cell response include early production innate antiviral cytokine especially interferon IFN activation cytotoxic lymphocyte innate natural killer NK cell adaptive cell critical antiviral defense cytokine cell response detrimental fatal host deregulate need well understand different arm antiviral immunity regulate particular nk cell propose play protective role variety viral infection human underlie mechanism remain poorly understand mouse model cytomegalovirus infection demonstrate NK cell prevent excessive production IFN promote efficient antiviral cell response NK cell help promote health disease viral infection regulate innate adaptive immune response important examine human nk cell control innate cytokine production prevent immunopathology promote adaptive immunity herpesviruse influenza time point examine mouse compare BALB mouse figure serum level innate cytokine significantly low mouse include figure TNF unpublished similar result observe compare serum cytokine level control treat NK cell deplete mouse figure unpublished datum systemic production IFN innate cytokine response MCMV infection high susceptible animal greatly reduce presence nk cell response control viral replication early efficiently MCMV infect immunocompetent animal systemic production IFN significant proportion come pDCs productively infect come infected cell unpublished datum knowledge contribution pDCs IFN innate cytokine production assess animal BALB genetic background susceptible MCMV infection cdcs infect MCMV vitro report produce high level IFN generally virus infect cell theoretically produce cytokine increase systemic level IFN observe BALB animal result high activation pdcs significant contribution function cell type high number infected cell animal compare activation pDCs IFN production BALB animal intracellular stain cytokine ex vivo isolated splenocyte figure pDCs find produce IFN compare BALB mouse consistent previous report animal pDCs sole producer IFN time point examine BALB mouse unpublished datum pDCs cDCs contribute production mouse high serum level observe BALB mouse solely attribute pDCs figure unpublished datum high activation pDCs innate cytokine production observe BALB mouse drastically reduce congenic animal endow efficient antiviral NK cell function link early viral load level IFN production pDCs perform infection BALB mouse fold serial dilution viral inoculum lead challenge dose pfu mouse systemic IFN titer animal measure splenic leukocyte isolate BALB mouse analyze frequency IFN pdcs pdcs directly ex vivo number dot plot represent percent IFN percent cell total pdc population representative animal group mouse day post mcmv infection show graph represent total number IFN pdcs pdcs present spleen BALB mouse day post mcmv infection result express mean sd mouse group thousand cell experiment representative show detect ELISA serum sample number pdcs produce IFN flow cytometry figure result clearly show IFN secrete pDCs increase dramatically high viral dose inoculum mouse level similar observe BALB animal fold difference virus inoculum pfu mouse sufficient switch IFN production pDCs mouse low level nearly maximal level similar observe BALB animal low dose viral infection pfu mouse BALB mouse activate nearly maximal pdc cytokine production BALB animal confirm exhibit dramatically different level pdc IFN production fold range viral inoculum datum pdcs mouse able produce high level IFN condition stimulation high viral inoculum nk cell depletion mouse infect low virus inoculum show lead dramatic increase pdc IFN production level observe BALB animal figure altogether datum ability host control MCMV replication early NK cell response limit activation pDCs prevent induction high systemic level IFN datum demonstrate pdcs mouse intrinsically deficient IFN production response MCMV infection high systemic level IFN production show lead ablation cdcs attrition antigen specific bystander cell population infection lymphocytic choriomeningitis virus LCMV hypothesize high systemic level IFN induce MCMV infection context inefficient nk cell activity lead ablation cdcs impinge development antiviral cell response previously report MCMV infection lead severe ablation cdcs BALB mouse prevent efficient antiviral NK cell function animal cause ablation cdcs BALB animal identify effect ablation development antiviral cell evaluate compare number DC subset activation antiviral cell different time point infection BALB mouse confirm pDC number remain homeostatic level BALB mouse early phase MCMV infection severe ablation cdcs occur subset figure contrast presence efficient nk cell response mouse cdc pdc compartment preserve consistent previous observation report animal compare kinetic cell expansion BALB mouse figure observe significant increase total cell spleen day post infection mouse later BALB animal investigate accelerate expansion cell result fast activation antiviral effector cell analyze total splenic cell expression glycosylate activation associate isoform monoclonal antibody mAb specifically recognize isoform lymphocyte identify subset effector cell capable IFN production potent cytolytic activity ex vivo acute phase variety viral infection observe appearance hi effector cell start day post infection mouse later BALB animal figure MHC class tetramer load peptide derive protein MCMV order quantify subset antiviral cell specific epitope figure observe fast expansion anti cell mouse similar observe total effector lymphocyte similar result obtain analyze cell response viral peptide derive protein unpublished datum conversely splenic leukocyte isolate BALB mouse analyze frequency pDCs int cdcs hi cdcs hi TCRb population number dot plot represent percent pdcs percent cdcs percent cdcs total splenocyte population representative animal group mouse day post mcmv infection graph represent total number million pDCs cdcs cdcs present spleen BALB mouse day post mcmv infection result express mean sd mouse group experiment representative show accordance previously report mouse deplete NK cell antibody treatment observe accumulation effector cell BALB mouse later time point challenge order investigate effector potential MCMVspecific cell observe day post infection mouse measure different parameter associate protective function antiviral cell proliferation monitor expression marker Figure capacity produce IFN response antigen specific restimulation vitro intracellular staining figure ELISPOT Figure cytotoxic potential measure antigen specific target cell killing vivo Figure parameter observe accelerated acquisition effector function cell mouse compare BALB animal BALB mouse differ NK cell locus include gene express leukocyte subset seek ensure difference observe activation kinetic cell mouse strain intrinsic difference reactivity cell ii dependent NK cell activation achieve show accelerated expansion cell occur animal antibacterial response BALB mouse compare Listeria monocytogene infection Figure ii response MCMV infection mouse deplete nk cell figure cell response BALB mouse compare infection virus oppose infection wild type revertant virus figure impact efficient nk cell activity development antiviral cell response confirm mouse genetic background delay antiviral cell activation observe animal deficient mediate NK cell activation inactivation associate adaptor molecule compare wild type control figure altogether datum efficient NK cell response viral infection accelerate development effector antiviral cell response determine dramatic reduction pdc IFN production NK cell function mouse account ability mouse preserve intact cdc compartment mount early cell response examine exogenous administration cytokine animal directly impact initiation cdc cell response despite efficient NK cell mediate control viral load MCMV infect mouse treat recombinant mouse IFN describe Materials Methods DC cell compartment compare controltreate BALB mouse figure treatment mouse IFN lead drastic reduction splenic cdcs pDCs compare control treat counterpart likewise IFN treatment induce delay expansion total hi Tet cell number figure note IFN treatment administer mouse significantly change level viral replication animal low detectable level infectious viral particle observe control treat IFN inject mouse versus log pfu spleen contrast sharply high viral replication observe BALB animal log spleen result demonstrate exogenous injection IFN mouse sufficient decrease number cdcs ablate early antiviral cell response excessive level IFN direct negative impact antiviral immune cell response manner independent level viral replication host possibility remain innate cytokine TNF produce high level BALB compare mouse bear contribution function synergistic redundant manner IFN case datum strongly suggest ability mouse preserve intact cdc compartment mount early cell response ability control viral replication early need host produce high systemic level IFN altogether datum identify naturally occur difference interaction virus host tilt balance function IFN eventually innate cytokine condition promote induction early adaptive immunity development state transient immunosuppression aim understand link NK cell function IFN production downstream effect DCs cell BALB mouse high viral burden infection MCMV plausible difference viral load BALB mouse impact intensity DC cell response order test impact extent viral replication vivo pdc cytokine production antiviral cell activation independently function NK cell seek strategy control viral replication efficiently BALB mouse tightly control timing magnitude achieve utilize MCMV DN SCP MCMV genetically engineer vivo replication effectively arrest administration doxycycline system able design doxycycline treatment protocol BALB mouse closely mimic extent kinetic efficient nk cell mediate control viral replication observe experimental condition select viral burden mouse DN SCP MCMV infect doxycycline treat BALB mouse comparable different time point examine low untreated DN SCP MCMV infect BALB animal figure reduction viral burden doxycycline treat DN SCP mcmvinfecte BALB animal lead significant strong decrease serum level pdc derive cytokine IFN figure allow maintenance cdcs Figure observe accelerated activation figure treatment DN SCP MCMV infect mouse doxycycline effect parameter test unpublished datum datum indicate mouse ability NK cell dampen innate cytokine production pDCs promote optimal condition initiation antiviral cdc cell response result exquisite capacity control viral replication early efficiently infection result study demonstrate efficient NK cell response promote accelerated generation effector antiviral cell infection vivo prevent generation high immunosuppressive level antiviral cytokine study demonstrate NK cell serve effector lymphocyte regulator immune system function defense viral infection datum bring important original advance understanding contribution nk cell immunity infectious disease demonstrate control function critical player innate antiviral response pdc modulate cytokine milieu induce early challenge ii accelerate generation effector antiviral cell mechanism nk cell activity modulate immune response pathogen summarize recent review include vivo regulation homeostasis maturation dc ii prevention detrimental persistence activation cell later time point immune response recently show perforin mediate nk cell kill modulate activation macrophage prevent development hemophagocytic lymphohistiocytosis like syndrome MCMV infection result demonstrate NK cell dramatically decrease intensity duration pdc activation control viral burden prevent production high systemic level IFN eventually innate cytokine detrimental effect host mechanism protect MCMV mediate loss splenic cdcs report measle virus LCMV exploit host IFN response inhibit cdc development drive cdc loss vivo result suggest MCMV induce high production IFN promote survival ablate cdcs delay activation antiviral effector cell light observation interesting note MCMV develop strategy actively counteract antiviral response IFN IFNc infected cell oppose mechanism employ negative strand RNA virus act shut production cytokine infected cell pDCs complete deficiency IFN response associate dramatic increase susceptibility mouse MCMV infection clearly appear benefit high level IFN production host bring efficient nk cell response mouse viral titer low see BALB mouse mouse produce fold IFN tempting speculate efficient NK cell activity drive activate receptor ability dampen pDC IFN production promote adaptive immunity direct host countermeasure subversion IFN response MCMV altogether result suggest NK cell response govern balance positive negative effect IFN eventually innate cytokine optimal orchestration immune response datum indicate efficient nk cell activity contribute adaptive arm immune response MCMV promote accelerated expansion antigen specific cell like contribution nk cell maintenance cdc compartment datum support role NK cell control pdc IFN production promotion antigen specific cell expansion IFN positive negative effect cell response example optimal expansion cell response LCMV infection demonstrate dependent ability cell compartment receive IFN mediate signal viral system IFN drive early attrition antigen specific bystander cell population delay expansion antiviral effector cell absence efficient nk cell response MCMV occur context excessive production IFN pDCs demonstrate exogenous administration IFN override ability efficient nk cell activity promote accelerated expansion functional antiviral cell IFN mediate effect cell expansion direct downstream consequence loss cdcs result suggest direct activity IFN cell administration IFN lead complete disappearance early antiviral cell incomplete decrease cdc number focus study datum delay cell response reach high level sustain long period time absence efficient antiviral NK cell activity consistent previous observation extended activation cell compartment require MCMV control condition likely result poor ability innate immune system control virus early hypothesize lead chronic inflammation long term detrimental effect host include increase susceptibility BALB mouse MCMV induce cell dependent autoimmune disease myocarditis efficient NK cell response beneficial host early direct antiviral effect reduce degree antiviral cell activation require later control viral replication indirect cost bear health knowledge study demonstrate naturally occur genetically determined difference interaction virus host tilt balance function IFN condition promote induction early adaptive immunity development state transient immunosuppression datum confirm NK cell antiviral function prevent generation chronic state cell activation later infection lead immunopathology key question future study determine general function NK cell antiviral defense mechanism place HIV infection genetic functional evidence suggest role variation NK cell function resistance susceptibility development AIDS detrimental effect excessive level IFN pro inflammatory cytokine recently suggest initiation antiviral cell response appear delay case infection highly pathogenic strain influenza excessive production pro inflammatory cytokine show occur propose play crucial role immunopathology nk cell confer resistance recognition infected cell receptor recently researcher demonstrate human nk cell able recognize dc infect influenza Ebola virus correlation unregulate IFN response malfunction switch innate adaptive immunity report fatal SARS highlight potential relevance observation variety viral infection important determine impact NK cell function modulation innate adaptive immunity development immunopathology model viral infection result imply individual susceptible viral infection pharmacological control viral replication early benefit directly limit viral cytopathogenicity establish condition well suit development balanced protective immunity case mouse susceptible MCMV infection viral replication control drug treatment role nk cell generation antitumor cell response link ability increase inflammation secrete IFN promote production DCs collectively study reveal role nk cell mediator innate cytokine balance optimal generation immune response nk cell require produce cytokine increase inflammation intrinsically low tumor development prevent excessive production innate cytokine intrinsically high context pathogenic encounter note apparent requirement instance NK cell engage cognate receptor mediate interaction naturally develop immunoregulatory function example response MCMV systemic activation NK cell produce IFN environment sufficient promote control viral replication associated immunoregulatory function cognate interaction enable NK cell act direct regulator antiviral immunity illustrate generation antitumor cell response response MCMV expression NK cell dramatically increase efficiency recognition killing infected cell enable NK cell deliver IFNc proper place time help priming antiviral cell infected mouse tripartite interaction MCMV infect express dcs importance cognate interaction promotion efficient NK cell function differ depend tissue target cell type exemplify heighten requirement dependent mechanism MCMV control lung conclusion demonstrate efficient nk cell activity contribute optimal orchestration innate adaptive immunity course viral infection vivo implication finding extend design therapeutic strategy fight human disease way emphasize benefit innate cytokine production immune cell recruitment activation outweigh detrimental effect directly reduce capacity host fight infection ii need well understand account complex interaction relatively rare cell type occur physiological context purpose exploit human immune system promote health disease mouse BALB mouse Charles River Laboratories purchase use study refer Jackson Laboratory deficient animal breed pathogen free breeding facility Centre de Marseille Luminy Marseille France experiment conduct accordance institutional guideline animal care use protocol approve French Provence ethical committee number Office Laboratory Animal Welfare assurance vivo treatment protocol stock wild type Smith Strain MCMV salivary gland extract bacterial artificial chromosome wild type DN SCP MCMV MCMV revertant prepare previously describe infection initiate day delivery PFU vivo derive virus PFU vitro derive virus modest dose choose induce lymphopenia infected animal harbor leukocyte number equal great uninfected control show addition dose likely close physiologic dose reach natural exposure virus contact infected animal vivo arrest DN SCP MCMV replication lg doxycycline deliver injection postinfection follow immediately addition mg ml doxycycline plus sucrose drinking water unit recombinant mouse IFN HyCult Biotech administer injection post mcmv infection dose titrate ELISA show correspond ng ELISA IFN standard cytokine titer measure ELISA serum BALB mouse post infection range ng ml volume lymph blood adult mouse roughly estimate ml dose rmIFN inject mouse similar physiologic level cytokine naturally induce infected BALB mouse nk cell deplete delivery lg purified anti mAb day pre mcmv infection day post mcmv infection control mouse treat lg mouse IgG Jackson ImmunoResearch Laboratories uninfected mouse deplete NK cell schedule day infected mouse assess nk cell depletion combination anti anti TCRb staining order avoid risk underevaluate number remain nk cell occur potential problem epitope masking antibody immunophenotype depletion nk cell depletion great figure lymphocyte analysis cell population spleen mince pass nylon mesh wash PBS mM EDTA FCS PBS EDTA FCS analysis DC population spleen digest collagenase liberase CI Boehringer Mannheim tease apart repeat pipete PBS EDTA FCS protocol erythrocyte osmotically lyse ammonium chloride treatment cell suspension keep EDTA FCS specify total live splenocyte count trypan blue exclusion hemocytometer total number specific leukocyte subset calculate individual mouse percent cell live gate total splenocyte total live splenocyte cell stimulation IFN production intracellular ifnc detection splenic lymphocyte isolate incubate YPHFMPTNL peptide AGPPRYSRI pulse cell ratio brefeldin Sigma Aldrich add culture cell harvest analyze intracellular IFN protein color staining follow flow cytometry vivo cytotoxicity assay Antigen specific cell mediate cytotoxicity assay describe briefly splenocyte naive mouse costaine Sigma Aldrich lM nm nM CFSE Molecular Probes label cell pulse indicate peptide mix equal ratio transfer total cell indicate group mouse lymphocyte isolate spleen recipient mice post transfer analyze transfer cell level CFSE expression unpulsed peptide pulse cell percent kill gate calculate peptide pulse unpulsed peptide pulse unpulsed uninfected viral titer serum cytokine level spleen homogenize viral titer determine plaque assay mouse embryonic fibroblast centrifugal enhancement cell Serum collect indicate time point cytokine level determine ELISA IFN PBL Biomedical Laboratories Systems manufacturer instruction abs reagent PE PE clone PE clone PerCp clone TCR allophycocyanin clone IFN allophycocyanin clone clone streptavidin PE purchase BD Pharmingen FITC clone ct purchase Caltag FITC clone purchase Novocastra purify rat anti mouse IFN clone anti IFN clone purchase TEBU bio mAb provide Schering Plough conjugate Alexa kit Molecular Probes Isotype control mAb purchase appropriate manufacturer follow tetramer conjugate PE obtain NIH Tetramer Facility Atlanta Georgia United States YPHFMPTNL AGPPRYSRI mention reagent FACS analysis study flow cytometric analysis cell incubate mAb min cell stain mAbs specific cell surface marker isotype control min Cells wash fix paraformaldehyde PBS Intracellular stain IFN perform Cytoperm kit BD Pharmingen Intracellular stain IFN perform previously describe depend experiment event collect FACSCalibur datum acquire analyze CellQuest software BD Biosciences Isotype control set gate present FACS analysis statistical analysis statistical analysis perform Microsoft Excel Microsoft Corporation Student tail test mean standard deviation sd calculate graph visually absent error bar small depict base scale axis','2007-08-24 00:00:00','http://dx.doi.org/10.1371/journal.ppat.003012',3594,900);
/*!40000 ALTER TABLE `Paper` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Word`
--

DROP TABLE IF EXISTS `Word`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `Word` (
  `idWord` int NOT NULL AUTO_INCREMENT,
  `word` varchar(45) DEFAULT NULL,
  PRIMARY KEY (`idWord`),
  UNIQUE KEY `word_UNIQUE` (`word`)
) ENGINE=InnoDB AUTO_INCREMENT=380827 DEFAULT CHARSET=utf8mb3;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Word`
--

LOCK TABLES `Word` WRITE;
/*!40000 ALTER TABLE `Word` DISABLE KEYS */;
INSERT INTO `Word` VALUES (28720,'aa'),(64526,'AAA'),(348253,'AAAAAA'),(69631,'aac'),(143274,'AACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATG'),(347165,'AADFAA'),(347170,'AADFBB'),(64533,'AAG'),(347169,'AAHFAA'),(150664,'AAP'),(352599,'AAR'),(64534,'AAT'),(13470,'Aaziz'),(69783,'AB'),(337490,'abaecin'),(17472,'abandon'),(309628,'abandonment'),(305424,'abasic'),(80404,'abbreviate'),(71634,'abbreviation'),(301220,'ABC'),(365834,'abcdefg'),(59806,'abcþ'),(58116,'abdoman'),(58217,'abdomen'),(57891,'abdominal'),(338757,'abdominally'),(63,'Abdulaziz'),(72577,'Abelson'),(24760,'aberrant'),(19880,'aberration'),(45339,'abh'),(14287,'ABI'),(21640,'abide'),(2760,'ability'),(379022,'ablate'),(131103,'ablation'),(16088,'able'),(69784,'ABm'),(5636,'abnormal'),(4228,'abnormality'),(118734,'aboard'),(11975,'abolish'),(340019,'abolishe'),(72693,'Aboriginal'),(33018,'abortive'),(49559,'abortus'),(313150,'abound'),(60761,'abrasive'),(58924,'abrogate'),(341787,'abrogation'),(374318,'abruptly'),(72749,'abs'),(708,'abscess'),(233,'absence'),(4508,'absent'),(20902,'absenteeism'),(15486,'absolute'),(147171,'absolutely'),(124159,'absorb'),(43795,'Absorbance'),(85653,'absorbancy'),(85635,'absorbency'),(64406,'absorption'),(101373,'abstinence'),(101359,'abstinencebase'),(110518,'abstract'),(138970,'abstracted'),(110658,'abstraction'),(118453,'abstractor'),(20801,'absurd'),(6393,'abundance'),(5074,'abundant'),(32964,'abuse'),(100479,'abuser'),(52231,'abyssi'),(59804,'abþd'),(59809,'abþþ'),(62352,'Ac'),(95044,'academic'),(339358,'Acanthamoeba'),(343968,'acapsulate'),(14676,'accelerate'),(2087,'accelerated'),(18870,'accentuate'),(3223,'accept'),(101543,'acceptability'),(66026,'acceptable'),(310940,'acceptably'),(121184,'acceptance'),(13496,'acceptor'),(17494,'access'),(4593,'accessibility'),(36001,'accessible'),(24204,'accession'),(13989,'accessory'),(73149,'accident'),(17939,'accidental'),(362210,'accidentally'),(101094,'accommodate'),(61689,'accommodation'),(8852,'accompany'),(20692,'accomplish'),(198,'accord'),(15714,'accordance'),(79660,'accordingly'),(622,'account'),(21602,'Accountability'),(95873,'accountable'),(583,'accounting'),(320140,'accredit'),(320143,'Accreditation'),(108931,'accrue'),(5455,'accumulate'),(4122,'accumulation'),(51567,'accuracy'),(11419,'accurate'),(12468,'accurately'),(106929,'accustomed'),(136073,'ACE'),(76600,'acelylleucyl'),(109323,'acetate'),(30875,'acetic'),(51731,'acetobutylicum'),(30114,'acetone'),(145252,'acetonitrile'),(56326,'acetyl'),(76845,'acetylamino'),(34858,'acetylate'),(33579,'acetylated'),(35906,'acetylation'),(326281,'acetylcholine'),(56333,'acetylesterase'),(76476,'acetylleucine'),(76287,'acetylleucyl'),(134795,'acetylneuraminic'),(76841,'acetylserine'),(28324,'acetyltransferase'),(141639,'achievable'),(16129,'achieve'),(353810,'achievement'),(36763,'aciclovir'),(2788,'acid'),(83785,'aciddependent'),(124132,'acidemia'),(5427,'acidic'),(3069,'acidification'),(124098,'acidify'),(111496,'acidosis'),(56332,'acidspecific'),(77035,'acidthe'),(5626,'acinar'),(455,'acinetobacter'),(48104,'acknowledge'),(69787,'ACm'),(46273,'Aconitum'),(255,'acquire'),(114102,'acquiredquantification'),(36043,'acquisition'),(57133,'acrosome'),(145528,'acrylamide'),(3542,'act'),(4868,'actin'),(27236,'acting'),(28342,'actinomycin'),(2124,'action'),(353214,'actionable'),(321082,'actionset'),(2034,'activate'),(16825,'Activated'),(30474,'activating'),(2040,'activation'),(6133,'activator'),(2636,'active'),(63792,'actively'),(362655,'activesite'),(2041,'activity'),(137334,'actor'),(363423,'ACTTTTTTTTTTTTTTTTV'),(3478,'actual'),(90052,'actuality'),(15279,'actually'),(345921,'acuity'),(541,'acute'),(325070,'acutely'),(125141,'ACYATGAAAYTGTGCAAGG'),(346096,'acyclovir'),(31639,'ad'),(123945,'adair'),(135838,'adam'),(46612,'Adams'),(32230,'adapt'),(5955,'adaptation'),(19506,'adaptive'),(31668,'adaptor'),(366340,'ADCI'),(11176,'add'),(67137,'ÀDDCt'),(98951,'added'),(100284,'addict'),(67209,'Addinsoft'),(2147,'addition'),(2077,'additional'),(970,'additionally'),(86027,'additive'),(86050,'additivity'),(6560,'address'),(115353,'addressable'),(3132,'adduct'),(43206,'adenine'),(6222,'adenocarcinoma'),(322574,'adenoid'),(322783,'adenoidal'),(306835,'adenomatous'),(16462,'adenosine'),(46428,'adenosylmethionine'),(9574,'adenoviral'),(443,'adenovirus'),(6455,'adenylate'),(77798,'adenylation'),(50001,'adequacy'),(36661,'adequate'),(2710,'adequately'),(363490,'ÀdF'),(21003,'adhere'),(5417,'adherence'),(342062,'adherent'),(49994,'adhesin'),(2318,'adhesion'),(122352,'adhesive'),(8152,'adjacent'),(15255,'adjust'),(139744,'adjustable'),(15095,'adjusted'),(15675,'adjustment'),(90484,'adjuvant'),(20585,'administer'),(311613,'administrate'),(2263,'administration'),(48999,'administrative'),(21915,'administrator'),(107387,'admirably'),(103,'admission'),(521,'admit'),(301253,'Adobe'),(46429,'AdoMet'),(14426,'adopt'),(15511,'adoption'),(57474,'adrenal'),(6900,'adrenergic'),(85187,'adsorb'),(17319,'adsorbent'),(29888,'adsorption'),(21406,'ADT'),(18,'adult'),(16206,'advance'),(21065,'Advanced'),(118028,'advancement'),(5236,'advantage'),(9649,'advantageous'),(72348,'advent'),(74270,'adversariesthese'),(2483,'adverse'),(2871,'adversely'),(94323,'advertisement'),(72899,'advertising'),(21194,'advice'),(17399,'advise'),(72316,'Advisory'),(101330,'advocacy'),(72485,'advocate'),(94165,'AE'),(104132,'Aedes'),(340707,'aeg'),(340705,'aegyptius'),(94020,'AEOJ'),(18673,'aeroallergen'),(124178,'aerobic'),(52114,'aeropyrum'),(18666,'aerosol'),(118739,'aerosolize'),(119907,'aerosolized'),(5770,'aeruginosa'),(124541,'aetiological'),(110223,'aetiology'),(24410,'af'),(60763,'AFA'),(48402,'affair'),(760,'affect'),(73016,'affected'),(57785,'afferent'),(307944,'affiliated'),(3671,'affinity'),(366236,'affinitypurified'),(355018,'affirm'),(35492,'affix'),(100163,'afflict'),(75631,'affluent'),(18399,'afford'),(101527,'affordable'),(354983,'Affy'),(54896,'Affymetrix'),(100340,'Afghanistan'),(9274,'aforementioned'),(75135,'Africa'),(135331,'African'),(107192,'aftermath'),(125157,'AGAGGGAGRTCTATCTCYCC'),(336577,'Agaisse'),(186,'agar'),(30019,'agarose'),(493,'age'),(655,'aged'),(334206,'agedependent'),(72510,'ageing'),(134096,'Agenae'),(48750,'agency'),(72110,'agenda'),(602,'agent'),(4511,'agglutinate'),(6365,'agglutinated'),(4442,'agglutination'),(313539,'aggravate'),(4447,'aggregate'),(4440,'aggregation'),(42035,'aggressive'),(66107,'Agilent'),(42874,'aging'),(326116,'agitation'),(103564,'aglycosyl'),(25855,'agmatine'),(362218,'agnostic'),(1980,'ago'),(325015,'agonism'),(6671,'agonist'),(359550,'agonistic'),(306829,'agouti'),(380550,'AGPPRYSRI'),(21581,'agree'),(16929,'agreement'),(73843,'Agreements'),(126597,'agricultural'),(47618,'agriculture'),(312668,'agroecosystem'),(143272,'AGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTG'),(11639,'AGUUG'),(76332,'AGY'),(146779,'ahead'),(365851,'ahelical'),(72342,'AHMC'),(113673,'AHSG'),(133526,'AI'),(41599,'aid'),(35474,'AIDS'),(102107,'AIDSrelated'),(132529,'Aigner'),(14396,'aim'),(17997,'air'),(137033,'airborne'),(139855,'aircraft'),(6846,'airflow'),(23575,'airline'),(73768,'airport'),(3594,'airspace'),(516,'airway'),(349535,'Akaike'),(630,'Al'),(56718,'Ala'),(24359,'alanine'),(22374,'alarm'),(106578,'alarmingly'),(33931,'albeit'),(338240,'Albert'),(320643,'Alberta'),(320645,'Albertans'),(104133,'albopictus'),(367589,'albumax'),(17324,'albumin'),(305848,'AlCl'),(58428,'alcohol'),(58670,'alcoholic'),(361488,'aldehyde'),(301024,'Aldrich'),(20636,'alert'),(62442,'Àleu'),(102715,'Alexa'),(348408,'Alexandersen'),(301280,'Alexandria'),(13701,'alfalfa'),(61362,'algebraic'),(61438,'algebraically'),(21276,'algorithm'),(87804,'Algorithms'),(36209,'ALI'),(338216,'alien'),(45640,'align'),(149816,'aligned'),(13297,'alignment'),(107262,'alike'),(341567,'aliphatic'),(26864,'aliquot'),(100079,'aliquote'),(16968,'ALIVE'),(77118,'alkaline'),(124071,'alkalinization'),(45228,'alkb'),(46041,'AlkBdomain'),(45555,'AlkBlike'),(85421,'alkyl'),(45447,'alkylate'),(45245,'alkylation'),(361487,'alkylhalide'),(309085,'allay'),(14755,'allcause'),(29446,'allele'),(305954,'Allelic'),(5914,'allergen'),(50286,'allergenic'),(2267,'allergic'),(149285,'allergology'),(90337,'Allergy'),(324425,'allergyassociate'),(323572,'allergynegative'),(2330,'alleviate'),(352851,'alleviated'),(108283,'alleviation'),(14644,'allocate'),(72031,'allocation'),(14435,'allogeneic'),(7006,'allograft'),(380679,'allophycocyanin'),(91644,'allosteric'),(7801,'allotransplantation'),(4205,'allow'),(150296,'allowable'),(12567,'Almazan'),(45996,'almond'),(48197,'alongside'),(31640,'alpha'),(116432,'alphabet'),(34096,'alphaherpe'),(147839,'alphaviruse'),(3240,'alter'),(3230,'alteration'),(6308,'altered'),(126979,'altering'),(29255,'altern'),(22422,'alternate'),(2600,'alternative'),(13031,'alternatively'),(10585,'Alto'),(49350,'altogether'),(36498,'aluminium'),(15729,'Alvarez'),(2017,'alveolar'),(5559,'alveolus'),(31530,'Alzheimer'),(108123,'AMA'),(307951,'amalgamate'),(312850,'Amaranthus'),(372062,'amastigote'),(372318,'amazonensis'),(303758,'AMBER'),(305664,'ambient'),(55490,'ambiguity'),(78353,'ambiguous'),(26936,'Ambion'),(312688,'ambisense'),(17293,'ambitious'),(38009,'ambulant'),(37556,'ambulatory'),(339753,'AMC'),(85550,'AMD'),(5833,'ameliorate'),(53185,'amelioration'),(65542,'amenable'),(49023,'amend'),(49027,'amendment'),(118289,'America'),(16728,'american'),(324554,'Americans'),(25485,'americanus'),(371408,'Americas'),(26829,'Amersham'),(62440,'Àmet'),(64674,'Amicon'),(148801,'amide'),(9068,'amiloride'),(140957,'amine'),(4009,'amino'),(75789,'aminoacetylate'),(77002,'aminoacyl'),(76063,'aminoacylate'),(77039,'aminoacylated'),(76054,'aminoacylation'),(77086,'aminoacyltion'),(305889,'aminoalkyl'),(324979,'aminobutyric'),(306236,'aminocaproic'),(124164,'aminoglycoside'),(305724,'aminohexanoate'),(305726,'aminohexanoyl'),(305684,'aminohexanoylextended'),(85561,'ammonia'),(79373,'ammonium'),(3739,'amnionic'),(3749,'amniotic'),(338127,'amoebae'),(320186,'amoebecyte'),(6677,'amount'),(47703,'Ampelovirus'),(100382,'amphetamine'),(3806,'amphipathic'),(124165,'amphotericin'),(43932,'amphotropic'),(30938,'ampicillin'),(2435,'ample'),(11436,'ampli'),(10263,'amplicon'),(112482,'amplicone'),(95813,'amplifi'),(363766,'amplifiable'),(60,'amplification'),(3706,'amplified'),(36527,'amplifier'),(25240,'amplify'),(347231,'amplitude'),(16386,'Amsterdam'),(312697,'AMV'),(50039,'amylase'),(31567,'amyloid'),(64500,'Amylose'),(92876,'anadromous'),(14299,'anaemia'),(14423,'anaemic'),(49106,'anaerobic'),(205,'anaerobically'),(135048,'anal'),(77772,'analog'),(3046,'analogous'),(120644,'analogy'),(11393,'analyse'),(14296,'Analyser'),(323,'analysis'),(74171,'analyst'),(348113,'analyte'),(375928,'analytic'),(2965,'analytical'),(35252,'Analytik'),(21022,'analyze'),(36531,'analyzer'),(368900,'analyzestep'),(14495,'anaphylactic'),(322643,'anaphylatoxin'),(5596,'anatomical'),(25589,'ancestor'),(46316,'ancestral'),(10283,'anchor'),(139820,'Anchorage'),(63526,'anchoring'),(364132,'anchorun'),(46833,'ancient'),(150563,'andbound'),(95171,'andDelphi'),(136651,'Anderson'),(89625,'andfigure'),(109149,'androgen'),(311960,'andsuggeste'),(333162,'andvariance'),(37102,'anecdotal'),(345447,'anecdotally'),(419,'anemia'),(19837,'anemic'),(116538,'anergize'),(36927,'anergy'),(324758,'anesthetic'),(104481,'anesthetization'),(58073,'anesthetize'),(136076,'Ang'),(74407,'Angeles'),(357969,'Angelini'),(16017,'angina'),(8198,'angiogenesis'),(133041,'angiogenic'),(7176,'angiotensin'),(134852,'angiotensinconverte'),(303965,'angle'),(150049,'angstrom'),(16260,'Angus'),(76308,'anhydride'),(2186,'animal'),(320121,'Animals'),(42921,'anion'),(98751,'anionic'),(26711,'ank'),(62385,'anneal'),(62452,'annealed'),(113961,'annealing'),(52661,'Annex'),(92166,'annotate'),(337275,'annotated'),(327395,'annotation'),(72292,'announce'),(102,'annual'),(21623,'annually'),(147240,'annul'),(22288,'anomalous'),(23446,'anomaly'),(22966,'anonymous'),(336066,'anophele'),(335937,'Anopheles'),(109556,'ANOVA'),(326688,'anoxia'),(12878,'answer'),(338341,'ant'),(124401,'antacid'),(7309,'antagonism'),(6483,'antagonist'),(5562,'antagonistic'),(324716,'antagonize'),(58817,'anterior'),(87406,'anthracis'),(20810,'anthrax'),(312430,'anthropocentric'),(2122,'anti'),(9555,'antiapoptosis'),(132812,'antiapoptotic'),(378813,'antibacterial'),(236,'antibiotic'),(34827,'antibodie'),(90410,'Antibodies'),(3653,'antibody'),(319279,'antibodytreate'),(108935,'anticancer'),(112025,'anticholinergic'),(21812,'anticipate'),(364198,'anticipated'),(16568,'anticoagulant'),(61586,'anticodon'),(324734,'antidepressant'),(301114,'antidigoxigenin'),(100290,'antidrug'),(328602,'antiepitope'),(35081,'antifade'),(50115,'antifungal'),(3549,'antigen'),(5243,'antigenic'),(346559,'antigenically'),(56307,'antigenicity'),(126927,'antigenome'),(129613,'antigenomic'),(365993,'antigens'),(367425,'antigensepharose'),(117449,'antigenspecific'),(324735,'antihistamine'),(3305,'antiinflammatory'),(90629,'AntiJen'),(54757,'antilog'),(1996,'antimicrobial'),(301197,'antimouse'),(3018,'antioxidant'),(301147,'antiPDGF'),(19344,'antiproliferative'),(18956,'antiprotease'),(324714,'antipsychotic'),(301153,'antirabbit'),(301166,'antirat'),(96096,'antiretroviral'),(11233,'antisense'),(123404,'antiseptic'),(14164,'antisera'),(50653,'antiserum'),(16569,'antithrombin'),(345368,'antituberculous'),(109854,'antitumor'),(132836,'antitumoral'),(36120,'antiviral'),(23597,'antizyme'),(25818,'antizymeindependent'),(72234,'anxiety'),(27019,'Äol'),(20500,'aortic'),(43013,'AP'),(15760,'APACHE'),(72387,'apart'),(16579,'APC'),(340305,'APEC'),(43141,'APendonuclease'),(36482,'apex'),(34549,'aphid'),(85547,'API'),(337491,'apidaecin'),(337405,'Apis'),(337498,'apisimin'),(125203,'APNET'),(324969,'Apo'),(31663,'apolipoprotein'),(133531,'apologize'),(2756,'apoptosis'),(2645,'apoptotic'),(32436,'apparatus'),(4757,'apparent'),(2384,'apparently'),(72189,'appeal'),(10694,'appealing'),(7835,'appear'),(17166,'appearance'),(63721,'appeared'),(20301,'appears'),(74788,'append'),(37788,'Appendix'),(47494,'apple'),(92635,'applet'),(95707,'applicability'),(3349,'applicable'),(17584,'application'),(67013,'Applied'),(13527,'apply'),(317713,'appreciable'),(9499,'appreciate'),(983,'approach'),(4182,'appropriate'),(20648,'appropriately'),(42197,'appropriateness'),(21650,'approval'),(58020,'approve'),(349672,'approx'),(22035,'approximate'),(3949,'approximately'),(20922,'approximation'),(38418,'April'),(31047,'aprotinin'),(43012,'apurinic'),(43014,'apyrimidinic'),(93533,'aquacultural'),(134704,'aquaporin'),(367939,'aquatic'),(64411,'aquaticus'),(85560,'aqueous'),(69,'Arabia'),(147602,'Arabidopsis'),(146107,'arabinoside'),(135260,'arachidonic'),(47215,'arachnid'),(42815,'aration'),(57968,'arbitrarily'),(15160,'arbitrary'),(91053,'arc'),(133964,'arch'),(45325,'Archaea'),(144740,'archaeal'),(51310,'archaebacteria'),(146460,'archaeon'),(139541,'archetypal'),(23307,'architectural'),(20987,'architecturally'),(6996,'architecture'),(22654,'archive'),(22114,'arcims'),(311016,'ard'),(2061,'ARDS'),(149346,'arduous'),(296,'area'),(96268,'arena'),(83556,'Arg'),(371401,'Argentina'),(25854,'arginine'),(135653,'argininevasopressin'),(78616,'arginyl'),(56571,'argon'),(73438,'arguably'),(13839,'argue'),(60288,'argument'),(17066,'arise'),(85293,'Arlington'),(5259,'arm'),(338331,'Armstrong'),(320127,'Army'),(113105,'aRNA'),(359741,'Arner'),(134740,'aromatase'),(2773,'aromatic'),(337388,'Aronstein'),(90936,'arrange'),(48698,'arrangement'),(4413,'array'),(56126,'arrayed'),(22679,'Arrays'),(19434,'arrest'),(123181,'arrhythmia'),(72372,'arrival'),(21379,'arrive'),(10647,'arrow'),(35015,'arrowhead'),(337940,'arsenal'),(17484,'art'),(47561,'artefact'),(289,'arterial'),(7466,'arterio'),(7441,'arteriole'),(11223,'arteritis'),(11221,'arterivirus'),(10990,'arteriviruse'),(7141,'artery'),(728,'arthralgia'),(729,'arthritis'),(15313,'arthroplasty'),(47214,'arthropod'),(26463,'arti'),(3557,'article'),(107770,'articulate'),(307373,'articulation'),(2982,'artifact'),(353507,'artifice'),(9832,'artificial'),(55625,'artificially'),(312853,'arvense'),(312855,'arvensis'),(349555,'asA'),(44856,'asbestos'),(58218,'ascend'),(44203,'ascertain'),(43066,'ascorbic'),(338169,'Ascosphaera'),(18177,'ascribe'),(123728,'aseptic'),(375915,'asExamination'),(366904,'asexual'),(313732,'asHomogeneity'),(48223,'Asia'),(100328,'asian'),(105624,'aside'),(83464,'ask'),(303483,'ASL'),(144,'aslo'),(303759,'ASLs'),(71718,'asn'),(78613,'Asp'),(105273,'asparagine'),(326483,'aspartate'),(83293,'aspartyl'),(2161,'aspect'),(36291,'aspirate'),(123,'aspiration'),(15571,'aspirin'),(3681,'assay'),(24822,'assayed'),(14161,'assays'),(354226,'asse'),(338286,'assemblage'),(3846,'assemble'),(4050,'assembly'),(95654,'assert'),(3669,'assess'),(2339,'assessment'),(15315,'assign'),(58001,'assignment'),(981,'assist'),(101996,'assistance'),(118474,'assistant'),(129518,'asso'),(279,'associate'),(4213,'associated'),(2300,'association'),(102118,'Associations'),(349157,'assortative'),(349904,'assortativity'),(2467,'assume'),(41254,'assumed'),(12116,'assumption'),(69147,'assurance'),(108034,'assure'),(32732,'aster'),(34070,'asterisk'),(120905,'asterix'),(375847,'asthe'),(350343,'asthis'),(390,'asthma'),(2298,'asthmatic'),(17003,'astonishingly'),(108061,'astoundingly'),(148447,'astrocyte'),(29661,'astrocytoma'),(303130,'astroglial'),(17358,'astronaut'),(23546,'astute'),(349138,'asWe'),(150061,'aswhere'),(306782,'asymmetric'),(302898,'asymmetrically'),(3978,'asymmetry'),(33,'asymptomatic'),(61338,'asymptotic'),(371421,'asymtomatic'),(367698,'asynchronous'),(125153,'ATAATAGCAYTRGCGGCAGCCCA'),(72319,'ATAGI'),(54450,'ataxia'),(64541,'ATC'),(26528,'ATCATGAAAAGCCTC'),(34723,'ATCC'),(45275,'ate'),(122757,'atelectasis'),(132036,'atelocollagen'),(64528,'ATG'),(132927,'athymic'),(76398,'ation'),(29256,'ative'),(24044,'Atkins'),(87389,'Atlanta'),(92548,'Atlantic'),(10583,'Atlas'),(54449,'ATM'),(17479,'atmosphere'),(91634,'atom'),(91970,'atomic'),(323553,'atopic'),(75990,'ator'),(7185,'ATP'),(148169,'ATPase'),(310720,'atrial'),(101844,'atrisk'),(430,'atrophy'),(100384,'ATS'),(72781,'ATSI'),(11029,'attach'),(4348,'attached'),(5674,'attachment'),(6838,'attack'),(39076,'attain'),(9541,'attempt'),(15815,'attend'),(17748,'attention'),(7310,'attenuate'),(18684,'attenuation'),(125164,'ATTGTCACCATAAGCAGCCA'),(101912,'attitude'),(108329,'attract'),(117722,'attraction'),(132833,'attractive'),(117707,'attractor'),(972,'attributable'),(3571,'attribute'),(321268,'Attributes'),(37235,'attribution'),(377461,'attrition'),(13,'atypical'),(52860,'atypicality'),(27090,'AU'),(62237,'AUA'),(30990,'Auburn'),(328212,'AUC'),(73033,'audience'),(140058,'audit'),(66723,'Auer'),(24232,'AUG'),(6668,'augment'),(125262,'August'),(22913,'aureus'),(114668,'auspex'),(72252,'auspice'),(145567,'Austin'),(47990,'Australia'),(48009,'australian'),(48805,'Australians'),(358390,'authentic'),(4769,'author'),(42624,'authority'),(54326,'authorship'),(97963,'auto'),(115739,'autoantibody'),(6523,'autocrine'),(31820,'autoimmune'),(344558,'autoimmunity'),(15317,'autologous'),(26748,'automate'),(105071,'automatic'),(3329,'automatically'),(140421,'automation'),(17948,'automobile'),(325038,'autonomic'),(100696,'autonomous'),(79096,'autonomously'),(95074,'autonomy'),(327639,'autophosphorylate'),(329579,'autophosphorylation'),(57801,'autopsy'),(58651,'autopsybased'),(24912,'autoradiogram'),(30214,'autoradiography'),(22227,'autoregressive'),(16372,'autoregulation'),(23725,'autoregulatory'),(54455,'autosomal'),(3193,'autoxidation'),(23997,'Auvinen'),(10922,'auxiliary'),(69910,'auxotrophic'),(368531,'Av'),(343637,'Avadhanula'),(130019,'avail'),(9856,'availability'),(74,'available'),(340243,'avenae'),(131000,'avenue'),(686,'average'),(357712,'averaging'),(29803,'Aves'),(48219,'avian'),(34344,'Avitabile'),(344527,'avium'),(20917,'avoid'),(17383,'avoidance'),(318743,'avoiding'),(134651,'avt'),(16219,'await'),(124445,'awake'),(91794,'award'),(42122,'aware'),(37581,'awareness'),(23232,'away'),(40006,'axis'),(114207,'Axon'),(31786,'axonal'),(45964,'axonopodi'),(329262,'AxonTesting'),(302988,'axotomy'),(50236,'Azevedo'),(367431,'azide'),(84865,'azurophilic'),(143670,'AZwt'),(59802,'aþcd'),(59812,'aþcþ'),(59810,'aþþd'),(59801,'aþþþ'),(69789,'BA'),(34868,'BAbCO'),(319163,'baboon'),(11386,'baby'),(49135,'bacillus'),(37646,'backache'),(43926,'backbone'),(374468,'backcalculate'),(18541,'background'),(21016,'BackgroundThe'),(138024,'backup'),(72999,'backward'),(61705,'backwards'),(148,'bacteria'),(245,'bacterial'),(342045,'bacterially'),(2766,'bactericidal'),(49572,'Bacteriocins'),(111263,'bacteriology'),(45410,'bacteriophage'),(45987,'bacterium'),(71640,'baculovirus'),(21137,'bad'),(59454,'badnaviruse'),(336696,'Baeg'),(27365,'Baer'),(83818,'Baf'),(83817,'bafilomycin'),(123472,'bag'),(27048,'bait'),(13106,'Baker'),(5144,'BAL'),(8079,'balance'),(75309,'balanced'),(104367,'BALB'),(87852,'Baldwin'),(18690,'BALF'),(48774,'Bali'),(127914,'Ball'),(359992,'Balmer'),(13481,'Baltimore'),(69790,'BAm'),(46180,'Bamboo'),(26596,'BamHI'),(332724,'ban'),(59455,'banana'),(10315,'band'),(123376,'bangle'),(53745,'bank'),(349836,'banning'),(34528,'bar'),(17033,'Barcelona'),(78724,'Bare'),(146169,'barely'),(11100,'Baric'),(13714,'barley'),(23813,'Barnett'),(75272,'barometer'),(20105,'Barr'),(738,'Barre'),(340920,'barrel'),(116611,'Barret'),(19643,'barrier'),(6160,'basal'),(211,'base'),(304435,'basebase'),(15399,'baseline'),(93765,'basepair'),(17862,'basic'),(45313,'basically'),(117721,'basin'),(2305,'basis'),(7098,'basolateral'),(322066,'basophil'),(325815,'basophilic'),(21103,'batch'),(109245,'bath'),(377866,'battery'),(308144,'battle'),(107459,'baulk'),(43387,'Baum'),(122630,'baumannii'),(21489,'bayesian'),(11982,'BC'),(26895,'BCA'),(301122,'BCIP'),(69713,'bcli'),(90409,'BCR'),(347164,'BCtype'),(128882,'BD'),(51165,'bdb'),(331677,'BDNF'),(12381,'bè'),(109160,'Beach'),(28796,'bead'),(21705,'bean'),(569,'bear'),(6627,'beat'),(85663,'Beckman'),(109375,'Beckton'),(374574,'becomeandequation'),(374786,'becomewhere'),(313725,'beconversely'),(56567,'Becton'),(99,'bed'),(36164,'bedside'),(335658,'bee'),(338549,'BeeBase'),(9932,'begin'),(33601,'beginning'),(84731,'behave'),(370608,'behavio'),(4858,'behavior'),(58175,'behavioral'),(108916,'behaviour'),(335641,'behavioural'),(363719,'Behn'),(100367,'Beijing'),(95485,'being'),(17473,'belgian'),(16348,'Belgium'),(337638,'belie'),(106008,'belief'),(3578,'believe'),(306559,'bell'),(9757,'belong'),(29449,'Ben'),(303691,'Bénas'),(324686,'bench'),(42606,'benchmark'),(347769,'benchtop'),(58155,'bend'),(34356,'bene'),(95065,'benefi'),(107613,'beneficence'),(3406,'beneficial'),(107594,'beneficience'),(9679,'benefit'),(371415,'benign'),(149102,'Bennasser'),(304972,'benzene'),(371439,'Benznidazole'),(324751,'benzodiazepine'),(305849,'benzophenone'),(43102,'BER'),(303805,'Berendsen'),(93918,'Bergen'),(376272,'Berkeley'),(120470,'Berlin'),(50571,'bernasconi'),(88610,'Bernoulli'),(357944,'Bertini'),(92642,'BestKeeper'),(6899,'beta'),(32420,'Betakova'),(104446,'Beth'),(309622,'betrayal'),(72790,'Better'),(48811,'betterment'),(335829,'Beutler'),(96789,'beverage'),(98891,'BF'),(69740,'bglii'),(343985,'BHI'),(126869,'BHK'),(57754,'bi'),(105095,'biacore'),(44740,'biallelic'),(4296,'biantennary'),(936,'bias'),(46196,'biased'),(109327,'bicarbonate'),(8144,'big'),(93906,'BigDye'),(337338,'Bilak'),(310704,'bilateral'),(301374,'bilaterally'),(17401,'bilevel'),(58087,'biliary'),(18788,'bilirubin'),(18274,'biliverdin'),(48952,'bill'),(26009,'billion'),(317203,'bimodal'),(55110,'bin'),(117532,'binary'),(3538,'bind'),(114320,'binder'),(3880,'binding'),(133884,'binocular'),(313464,'binomial'),(313913,'binomially'),(31067,'bio'),(44661,'bioactivity'),(66108,'bioanalyzer'),(69189,'bioassay'),(20646,'bioattack'),(362313,'BioChain'),(362468,'Biochem'),(6553,'biochemical'),(339524,'biochemically'),(3057,'biochemistry'),(132418,'biocompatibility'),(307264,'bioethic'),(95041,'bioethical'),(96311,'bioethically'),(107582,'bioethicist'),(108801,'Bioethics'),(122354,'biofilm'),(108965,'bioflavonoid'),(147709,'biogenesis'),(340704,'biogroup'),(70300,'Biohit'),(46524,'bioinformatic'),(332230,'Bioinformatics'),(64493,'Biolabs'),(7695,'biologic'),(1976,'biological'),(147831,'biologically'),(43582,'Biologicals'),(150802,'biologist'),(23959,'Biology'),(110365,'biomarker'),(68242,'biomedical'),(35739,'biomedicine'),(362444,'Biomol'),(149514,'Bioperl'),(69288,'biophysical'),(141646,'BioPlex'),(64110,'biopolymer'),(330406,'BioProspector'),(2823,'biopsy'),(134231,'Biorad'),(29822,'bioreagent'),(347864,'BioRobot'),(320154,'biosafety'),(75674,'Bioscience'),(43645,'Biosciences'),(119349,'biosecurity'),(20703,'biosensor'),(367220,'BioSepraH'),(118381,'Biosis'),(66804,'biosize'),(360547,'Biosource'),(8147,'biosynthesis'),(126723,'biosynthetic'),(145283,'Biosystem'),(67014,'Biosystems'),(99763,'Biotech'),(49840,'biotechnological'),(29832,'Biotechnology'),(74642,'bioterror'),(20873,'bioterrorism'),(42708,'bioterrorist'),(87208,'biothreat'),(54666,'biotin'),(54596,'biotinlabele'),(79507,'biotinylate'),(79187,'biotinylated'),(359163,'biotinylation'),(47627,'biotope'),(363534,'BioTrove'),(96303,'bioweapon'),(320189,'BioWhittaker'),(97582,'BIP'),(352233,'bipartisan'),(135196,'biphenyl'),(48240,'bird'),(5599,'birth'),(43553,'Bis'),(117877,'Bittner'),(102396,'bivalent'),(106052,'bivariate'),(10797,'black'),(57704,'bladder'),(363671,'Blake'),(309154,'blame'),(34554,'Blanc'),(337769,'Blandin'),(368724,'blank'),(326661,'blanket'),(32477,'Blasco'),(30998,'blast'),(52043,'blastn'),(338519,'BLASTP'),(134108,'BlastX'),(338644,'BLAT'),(25569,'Blaxter'),(9511,'blebbing'),(16658,'bleed'),(124131,'bleeding'),(310564,'blending'),(6085,'bleomycin'),(50270,'BLG'),(20657,'blind'),(10552,'block'),(8043,'blockade'),(61692,'blockage'),(7717,'blocker'),(43568,'blocking'),(285,'blood'),(149238,'Bloom'),(45859,'Blosum'),(10184,'blot'),(4485,'blotting'),(133960,'blow'),(13301,'blue'),(47497,'blueberry'),(318878,'blunt'),(334476,'bly'),(43632,'BM'),(13766,'BMV'),(6982,'BO'),(58023,'Board'),(674,'boarding'),(94330,'Bochum'),(48153,'bodily'),(358982,'Bodmer'),(5202,'body'),(29760,'Boehringer'),(77806,'Boeke'),(30256,'boil'),(102937,'bold'),(143258,'boldface'),(131953,'bolus'),(336888,'Bombyx'),(25976,'bona'),(4180,'bond'),(85712,'Bonding'),(110828,'Bonferroni'),(141735,'Bonnet'),(94970,'book'),(117531,'Boolean'),(115237,'boom'),(338376,'Boomsma'),(11268,'Boon'),(310736,'BOOP'),(72364,'boost'),(126290,'boosting'),(46604,'bootstrap'),(125127,'bootstrappe'),(48343,'border'),(52174,'Borders'),(9464,'Bordetella'),(124037,'bore'),(111110,'borg'),(124608,'Borneo'),(53083,'Borodovsky'),(340980,'Bos'),(28783,'Bost'),(21100,'Boston'),(25926,'Botryotinia'),(49839,'bottleneck'),(367655,'bottomed'),(22140,'botulinic'),(64656,'Bound'),(22584,'boundary'),(335858,'Boutros'),(4028,'bovine'),(57690,'bowel'),(359545,'Bowen'),(20865,'Bowl'),(6219,'box'),(81731,'boxed'),(37428,'Bozzette'),(30175,'bp'),(42857,'BPD'),(336238,'brachyceran'),(376315,'bracket'),(70265,'Bradford'),(343491,'Bradley'),(3731,'brain'),(343983,'brainheart'),(57831,'Branch'),(30523,'branched'),(314307,'branching'),(59446,'Brassica'),(329655,'brate'),(102074,'Brazil'),(340709,'brazilian'),(372371,'braziliensi'),(302883,'BrdU'),(115796,'breadth'),(26797,'break'),(133951,'breakdown'),(16898,'breakthrough'),(8029,'breast'),(506,'breath'),(48118,'breathe'),(882,'breathing'),(300947,'breed'),(332806,'breeder'),(380234,'breeding'),(32852,'brefeldin'),(49977,'brevis'),(341039,'brevity'),(110043,'brew'),(11004,'Brian'),(85430,'bridge'),(4167,'bridging'),(7299,'brief'),(16846,'briefcase'),(37621,'briefly'),(144079,'Brierley'),(54816,'bright'),(30880,'Brilliant'),(3407,'bring'),(315523,'brink'),(72612,'British'),(29870,'BRL'),(10234,'broad'),(338078,'broaden'),(33151,'broadly'),(338259,'Brodsgaard'),(332805,'broiler'),(52042,'Broker'),(10940,'brome'),(47505,'bromide'),(336704,'bromo'),(145026,'bromophenol'),(3645,'bronchi'),(505,'bronchial'),(2062,'bronchiectasis'),(3613,'bronchiolar'),(29,'bronchiolitis'),(110506,'bronchitis'),(126,'bronchoalveolar'),(6423,'bronchoconstriction'),(7996,'bronchoconstrictor'),(1992,'bronchodilation'),(112026,'bronchodilator'),(311402,'bronchogenic'),(122777,'bronchogram'),(311530,'bronchopleural'),(39,'bronchopneumonia'),(20073,'bronchopulmonary'),(8855,'bronchorrhea'),(36097,'bronchoscopic'),(36710,'bronchoscopy'),(7992,'bronchospasm'),(335670,'brood'),(135858,'brook'),(226,'broth'),(335580,'Brown'),(90862,'browse'),(351779,'bRSV'),(49558,'Brucella'),(49588,'brucellosis'),(35246,'Bruker'),(73281,'brunt'),(123051,'brush'),(16347,'Brussels'),(351365,'bs'),(31076,'BSA'),(60645,'BsiWI'),(145372,'bsshii'),(26637,'bsteii'),(104974,'BT'),(11824,'bu'),(123543,'bubble'),(124481,'buccal'),(366958,'Buchs'),(43646,'Buckinghamshire'),(130411,'bud'),(128244,'budding'),(72271,'budget'),(72030,'budgetary'),(367082,'Buenaventura'),(5527,'buffer'),(132242,'buffering'),(150742,'bug'),(23142,'build'),(21292,'building'),(79869,'Bujarski'),(124580,'Bulgaria'),(50576,'bulgaricus'),(55910,'bulge'),(145511,'bulk'),(61670,'bulky'),(16196,'bullet'),(33177,'bundle'),(135614,'buoyancy'),(29503,'Burd'),(15250,'burden'),(359111,'Burgess'),(335688,'Burgett'),(32818,'Burkhardt'),(366006,'Burkina'),(358800,'Burkitts'),(30149,'Burlingame'),(100331,'Burma'),(8301,'burn'),(366668,'burst'),(48705,'bus'),(105021,'Busch'),(11687,'bushy'),(21429,'business'),(92590,'Bustin'),(21159,'busy'),(139206,'bution'),(135361,'butovarian'),(124493,'buttock'),(22524,'button'),(23889,'butylamine'),(29500,'Buvoli'),(100562,'buy'),(63474,'BWYV'),(24717,'bȳ'),(69530,'BYDV'),(349104,'byHence'),(14584,'bypass'),(42914,'byproduct'),(378417,'bystander'),(374311,'byTo'),(17868,'bZIP'),(64542,'CAA'),(86183,'caagcc'),(60603,'Cabb'),(318509,'cabin'),(144810,'CAC'),(141209,'CACAGC'),(29259,'Caceres'),(21955,'cache'),(360606,'CaCl'),(85613,'cacodylate'),(336269,'Cactin'),(336204,'Cactus'),(24549,'cadaverine'),(46207,'cadherin'),(44885,'cadmium'),(25217,'Caenorhabditis'),(75670,'CAG'),(58192,'cage'),(86123,'cal'),(84776,'Calbiochem'),(57955,'calcification'),(3527,'calcium'),(11898,'calculate'),(44241,'calculated'),(54112,'calculation'),(365501,'calculational'),(374097,'calendar'),(29675,'calf'),(321693,'Calgary'),(329654,'cali'),(58214,'caliber'),(348004,'calibrant'),(38746,'calibrate'),(113309,'calibration'),(67131,'calibrator'),(56569,'Calif'),(74408,'California'),(17385,'call'),(310199,'Callahan'),(106626,'caller'),(142087,'calling'),(300891,'callosum'),(10005,'cally'),(31546,'calmodulin'),(27352,'calreticulin'),(380694,'Caltag'),(69792,'CAm'),(320190,'Cambrex'),(35242,'Cambridge'),(108945,'Camellia'),(34999,'camera'),(673,'camp'),(37226,'campaign'),(45967,'campestris'),(59440,'CaMV'),(139677,'can'),(15128,'Canada'),(14531,'Canadian'),(320646,'Canadians'),(72619,'Canberra'),(26196,'cance'),(140066,'cancel'),(375998,'cancellation'),(369378,'cancelling'),(8051,'cancer'),(113824,'cancerous'),(75706,'Candida'),(14009,'candidate'),(368536,'Canine'),(303602,'canonical'),(11874,'cant'),(72891,'canteen'),(11358,'cantly'),(58083,'canula'),(27111,'cap'),(22499,'capability'),(22692,'capable'),(100,'capacity'),(14338,'capillary'),(46592,'Capillo'),(96024,'capita'),(321700,'Capital'),(10771,'capitalize'),(145318,'CapLC'),(46520,'capping'),(34114,'capsid'),(4633,'capsular'),(340691,'capsule'),(46878,'caption'),(73032,'captive'),(20866,'capture'),(106093,'capturing'),(3816,'carbohydrate'),(17517,'carbon'),(104963,'carbonate'),(304122,'carbonyl'),(20475,'carbonyls'),(326939,'carboxy'),(305857,'carboxyethyl'),(114311,'carboxyfluorescein'),(18434,'carboxyhemoglobin'),(31973,'carboxyl'),(140942,'carboxylated'),(340747,'carcinoembryonic'),(44887,'carcinogen'),(112708,'carcinogenesis'),(8105,'carcinoid'),(19463,'carcinoma'),(22674,'card'),(14333,'cardiac'),(106083,'Cardiff'),(60621,'cardinal'),(36650,'cardiogenic'),(371428,'cardiomyopathy'),(388,'cardiopulmonary'),(122196,'cardiothoracic'),(730,'cardiovascular'),(125087,'Cardosa'),(96,'care'),(101328,'career'),(10244,'careful'),(3460,'carefully'),(123514,'caregiver'),(367033,'caretaker'),(5769,'carinii'),(46268,'Carlavirus'),(87439,'Carlo'),(29717,'Carlsbad'),(303912,'CARNAL'),(135930,'carp'),(301125,'Carpinteria'),(65943,'Carr'),(94935,'carriage'),(37929,'carrier'),(104453,'Carroll'),(301750,'carrot'),(12584,'carry'),(92481,'carryover'),(51821,'cart'),(343672,'Cartwright'),(15078,'Carven'),(358265,'Casagrande'),(145548,'casamino'),(146412,'casaminoacid'),(337162,'cascade'),(371,'case'),(335132,'casescontact'),(370200,'casesonset'),(100273,'cash'),(9139,'Casola'),(3176,'caspase'),(24137,'cassette'),(73882,'cast'),(337437,'castaneum'),(337586,'casteel'),(335724,'Casteels'),(37468,'casual'),(48925,'casualty'),(28325,'CAT'),(109094,'catabolism'),(35085,'catalase'),(64503,'catalog'),(67434,'cataloging'),(91178,'catalogue'),(46424,'catalyse'),(131278,'catalysis'),(357793,'catalyst'),(45268,'catalytic'),(131050,'catalytically'),(2620,'catalyze'),(357758,'catalyzed'),(324307,'catarrhali'),(16847,'catch'),(138547,'cate'),(108977,'catechin'),(7107,'catecholamine'),(325,'categorical'),(111751,'categorisation'),(112263,'categorise'),(22839,'categorization'),(14481,'categorize'),(303,'category'),(135918,'catfish'),(84523,'Cathepsin'),(122987,'catheter'),(11437,'cation'),(23858,'cationic'),(50156,'cattle'),(43339,'caucasian'),(137308,'cauchy'),(59437,'cauliflower'),(59458,'caulimoviruse'),(8799,'causal'),(111914,'causally'),(96587,'causation'),(601,'causative'),(4,'cause'),(3317,'caution'),(61075,'cautionary'),(11007,'Cavanagh'),(17796,'caveat'),(109658,'cavity'),(104454,'CBR'),(335338,'cc'),(76503,'CCA'),(358281,'CCAAA'),(6149,'CCAAT'),(330702,'CCACCGCGG'),(64537,'CCC'),(358280,'CCCCC'),(143263,'CCCCTCACCCACCCCTGAAGATCCCAGGTGGGCGAG'),(301235,'ccd'),(362901,'CCM'),(79994,'CCMV'),(358811,'CCRF'),(125144,'CCRGTAGGKGTRCACGCRAC'),(143266,'CCTCACCCACCCCTGAAGATCCCAGGTGGGCGAGG'),(125161,'cctccggcccctgaatgcggctaat'),(26433,'CCU'),(74412,'CD'),(21015,'CDC'),(362902,'CDCE'),(377259,'cdcs'),(4034,'cdna'),(5133,'cdnas'),(105231,'CDR'),(13330,'ce'),(340822,'CEA'),(340764,'ceacam'),(340752,'CEACAMs'),(24713,'cease'),(337661,'cecropin'),(150177,'CED'),(124208,'cefotaxime'),(356399,'CEL'),(362951,'CELI'),(60762,'Celite'),(418,'cell'),(342021,'cellexpressed'),(71074,'cellfree'),(44997,'cellline'),(126286,'cellmediate'),(366338,'Cellmediated'),(56581,'Cellquest'),(29910,'Cells'),(57151,'celltype'),(2797,'cellular'),(49267,'cellularlocation'),(76482,'cellulose'),(358812,'CEM'),(95066,'cence'),(333750,'censoring'),(137784,'Census'),(341691,'cent'),(956,'center'),(21011,'Centers'),(9086,'central'),(101748,'centralized'),(341178,'centrally'),(14626,'centre'),(64921,'Centri'),(357177,'centric'),(380650,'centrifugal'),(99993,'centrifugate'),(26802,'centrifugation'),(26876,'centrifuge'),(33757,'centrin'),(33752,'centriolar'),(33760,'centriole'),(52344,'centroid'),(33751,'centrosomal'),(32673,'centrosome'),(72121,'century'),(122430,'cephalosporin'),(363393,'Cepheid'),(367230,'Ceramic'),(54459,'cerebellar'),(325263,'cerebellum'),(426,'cerebral'),(104607,'cerevesiae'),(23807,'cerevisiae'),(2223,'certain'),(45062,'certainly'),(48833,'certainty'),(72556,'certificate'),(326346,'ceruleus'),(43074,'ceruloplasmin'),(49628,'cervical'),(36491,'cesium'),(149575,'CESS'),(94320,'cessation'),(32521,'CEV'),(19157,'CF'),(44520,'CFA'),(341038,'CFG'),(69593,'cfmv'),(380590,'CFSE'),(140729,'CFTR'),(122995,'cfu'),(86963,'CG'),(105203,'cggatccgatggatac'),(149462,'CGI'),(3094,'cgmp'),(51507,'CGR'),(103508,'Ch'),(95233,'Chadwick'),(371397,'Chagas'),(372230,'chagasi'),(371427,'chagasic'),(57,'chain'),(72603,'chair'),(5785,'challenge'),(127925,'challenging'),(43688,'chamber'),(16439,'championed'),(24487,'chance'),(4695,'Chang'),(224,'change'),(6467,'channel'),(51504,'Chaos'),(31832,'chaperone'),(96797,'char'),(118069,'character'),(110248,'characterise'),(347,'characteristic'),(10245,'characterization'),(2064,'characterize'),(34588,'characterized'),(4276,'charge'),(332202,'Charles'),(97014,'charred'),(166,'chart'),(73937,'Charter'),(137620,'Chavez'),(64128,'cheap'),(96174,'cheaply'),(303810,'Cheatham'),(21864,'check'),(66588,'checking'),(354086,'checklist'),(109927,'checkpoint'),(45452,'checkpointing'),(75285,'cheer'),(5045,'chelation'),(2934,'chemical'),(85483,'chemically'),(145127,'ChemiDoc'),(85302,'chemilumescence'),(43633,'chemiluminescence'),(70012,'chemiluminescent'),(3061,'chemistry'),(8918,'chemoattractant'),(8920,'chemokine'),(109048,'chemoprevention'),(109132,'chemopreventive'),(4860,'chemotactic'),(4977,'chemotaxis'),(109969,'chemotherapeutic'),(148648,'chemotherapy'),(366917,'ChemTech'),(102205,'Chen'),(338681,'Chenna'),(47495,'cherry'),(311844,'Cheson'),(902,'chest'),(327,'Chi'),(43600,'Chicago'),(27611,'chicken'),(20910,'chief'),(74486,'Chikungunya'),(381,'child'),(72481,'Childhood'),(85227,'chill'),(99449,'chilled'),(78164,'chimera'),(50258,'chimeric'),(50451,'chimerical'),(103507,'chimerize'),(103494,'chimerized'),(47109,'China'),(5308,'chinese'),(9359,'Chini'),(31972,'CHIP'),(114099,'ChipReader'),(112926,'chipsTo'),(147734,'Chironomus'),(313758,'chisquared'),(78980,'chitin'),(339186,'chladubs'),(339705,'Chlamydia'),(339243,'Chlamydiae'),(339262,'Chlamydiaelike'),(28323,'chloramphenicol'),(124325,'chlorhexidine'),(54621,'chloride'),(306305,'chloro'),(30005,'chloroform'),(84185,'chloromethyl'),(77260,'chloroplast'),(52067,'chloroplastic'),(79990,'chlorotic'),(342037,'CHO'),(11314,'choice'),(75191,'cholera'),(109070,'cholesterol'),(324804,'cholinergic'),(326276,'cholinesterase'),(66457,'Chomczynski'),(30942,'choose'),(116204,'chord'),(378422,'choriomeningitis'),(300879,'chow'),(137619,'Chowell'),(335681,'Christe'),(61387,'Christiansen'),(337015,'christophide'),(336797,'Christophides'),(320184,'chromagenic'),(35673,'chromatin'),(364047,'chromatogram'),(364781,'chromatograph'),(145320,'chromatographic'),(24412,'chromatography'),(9513,'chromosomal'),(69301,'chromosomally'),(57166,'chromosome'),(2143,'chronic'),(7199,'chronically'),(324073,'chronictype'),(74867,'chronological'),(348293,'chronologically'),(125033,'Chu'),(87851,'Chuck'),(96694,'Chugoku'),(135666,'chum'),(83211,'chymotrypsin'),(106167,'CI'),(29302,'cial'),(26464,'cially'),(119617,'Cidofovir'),(373605,'Ciencia'),(11483,'ciencie'),(11404,'ciency'),(11279,'cient'),(10878,'ciently'),(18022,'cigarette'),(322506,'cilia'),(6626,'ciliary'),(3617,'ciliated'),(8986,'ciliostasis'),(37387,'cinate'),(124918,'Cincinnati'),(301290,'cingulum'),(305810,'cinnamic'),(108354,'circa'),(21993,'circle'),(23726,'circuit'),(60624,'circular'),(61280,'circularization'),(59650,'circularize'),(6923,'circulate'),(6832,'circulating'),(6403,'circulation'),(312717,'circulative'),(14861,'circulatory'),(5572,'circumstance'),(105626,'circumstantial'),(9628,'circumvent'),(19538,'cirrhosis'),(312852,'Cirsium'),(10886,'cis'),(21965,'Cisco'),(32394,'cisterna'),(54339,'citation'),(59655,'cite'),(23547,'citizen'),(36464,'citrate'),(45966,'citri'),(45973,'citrus'),(43719,'City'),(48923,'civilian'),(35658,'CKI'),(95489,'cking'),(54619,'Cl'),(342753,'Clackson'),(337378,'clade'),(347402,'cladelevel'),(17936,'claim'),(363203,'clamp'),(363928,'clamps'),(339143,'clan'),(3618,'Clara'),(26800,'clari'),(96593,'clarification'),(2694,'clarify'),(48766,'clarity'),(107633,'Clark'),(337169,'Clarke'),(362038,'clash'),(4416,'class'),(33152,'classi'),(184,'classic'),(30457,'classical'),(136518,'classically'),(17155,'classification'),(21491,'classifier'),(302,'classify'),(129009,'clathrin'),(134624,'claw'),(60833,'clean'),(123384,'cleaning'),(93349,'cleanup'),(2607,'clear'),(5162,'clearance'),(3269,'clearly'),(306426,'cleavability'),(4278,'cleavage'),(43140,'cleave'),(306803,'cleaved'),(322341,'cleft'),(105948,'clerical'),(21384,'clerk'),(74253,'cleverly'),(74532,'click'),(22064,'client'),(140813,'Clifton'),(682,'climate'),(20860,'clinic'),(87,'clinical'),(377,'clinically'),(982,'clinician'),(337179,'clip'),(459,'cloacae'),(20879,'clock'),(124033,'clog'),(60259,'clonal'),(60299,'clonally'),(11373,'clone'),(126362,'cloned'),(26649,'cloning'),(116422,'clonotype'),(10582,'Clontech'),(651,'close'),(669,'closed'),(9276,'closely'),(335200,'closing'),(45673,'Closteroviridae'),(22908,'Clostridium'),(339621,'Clostripain'),(138026,'closure'),(136062,'clot'),(37921,'clothing'),(304984,'cloud'),(10959,'clover'),(49419,'clp'),(50681,'clpp'),(124352,'CLRT'),(100455,'club'),(123249,'clude'),(3245,'clue'),(149916,'Clus'),(87958,'clustalw'),(45641,'ClustalX'),(3914,'cluster'),(38174,'clustered'),(51846,'clustering'),(29857,'cm'),(108078,'CMA'),(134798,'cmah'),(36385,'cmh'),(34147,'CMV'),(367378,'CNBr'),(17867,'CNC'),(103163,'CNS'),(2772,'CO'),(714,'coagulation'),(17512,'coal'),(23064,'coalition'),(367087,'Coast'),(353271,'coastal'),(5953,'coat'),(26687,'coatantizyme'),(129010,'coated'),(115455,'coating'),(20370,'cobalt'),(136787,'COBOL'),(147498,'Cobucci'),(118388,'Cochrane'),(149242,'Cockayne'),(69590,'cocksfoot'),(363366,'CoCL'),(22127,'CoCo'),(6234,'code'),(22126,'Coder'),(141969,'Codes'),(107340,'codify'),(303474,'coding'),(4037,'codon'),(63830,'codonanticodon'),(38819,'coefficient'),(309201,'coercive'),(312498,'coevolution'),(46826,'coevolve'),(5836,'coexist'),(78373,'coexistence'),(23645,'Cof'),(45998,'coffee'),(31970,'cognate'),(324551,'cognitive'),(326716,'cognitively'),(357499,'coherence'),(355852,'coherency'),(8967,'coherent'),(321785,'cohesive'),(15178,'cohort'),(3809,'coil'),(366848,'COILS'),(51330,'coin'),(32729,'coincide'),(64361,'coincidence'),(60515,'coinfecte'),(59280,'coinfected'),(60047,'coinfection'),(367102,'COLCIENCIAS'),(20356,'cold'),(33443,'coldtreate'),(337425,'Coleoptera'),(4490,'coli'),(362448,'coliexpresse'),(101041,'collaborate'),(101211,'collaboration'),(73800,'collaborative'),(308888,'collaboratively'),(87384,'collaborator'),(3997,'collagen'),(56488,'collagenase'),(3525,'collagenous'),(105941,'collapse'),(307683,'collateral'),(19947,'colleague'),(173,'collect'),(3530,'collectin'),(171,'collection'),(2700,'collective'),(2476,'collectively'),(659,'college'),(104268,'Collins'),(145326,'collision'),(132115,'colloidal'),(33035,'colocalization'),(359488,'colocalize'),(366009,'Colombia'),(367079,'colombian'),(8032,'colon'),(213,'colonial'),(58844,'colonic'),(124168,'colonisation'),(100264,'colonist'),(36749,'colonization'),(122221,'colonize'),(123872,'colonized'),(340720,'colonizer'),(216,'colony'),(227,'color'),(108952,'coloration'),(8052,'colorectal'),(41748,'colored'),(43747,'colorimetric'),(48377,'colossus'),(70289,'colour'),(83084,'coltiviruse'),(333333,'Columbia'),(25785,'column'),(85727,'com'),(122438,'coma'),(43027,'combat'),(123757,'Combes'),(3702,'combination'),(56118,'combinatorial'),(6283,'combine'),(18466,'combined'),(17943,'combustion'),(45458,'come'),(32540,'comet'),(107890,'comfort'),(140005,'comfortable'),(76713,'comfragment'),(22553,'comma'),(141697,'command'),(352996,'commander'),(30879,'Commassie'),(73640,'commence'),(107449,'commendable'),(339452,'commensal'),(53068,'comment'),(2158,'commentary'),(107223,'commentator'),(95386,'commerce'),(21920,'commercial'),(66504,'commercially'),(72140,'commission'),(353240,'commit'),(74041,'commitment'),(8145,'committed'),(49018,'Committee'),(3,'common'),(121246,'commonality'),(2045,'commonly'),(143043,'Commons'),(23072,'Commonwealth'),(307217,'commu'),(23500,'communicable'),(74241,'communicate'),(5792,'communication'),(106467,'communicator'),(309156,'communitiesThe'),(254,'community'),(370036,'communitybase'),(373849,'commute'),(15992,'comorbid'),(177,'comorbiditie'),(15253,'comorbidity'),(13654,'compact'),(132362,'compaction'),(367946,'companion'),(10566,'company'),(111782,'comparability'),(958,'comparable'),(131944,'comparably'),(17797,'comparative'),(5211,'comparatively'),(6240,'compare'),(368915,'comparestep'),(86891,'comparision'),(324,'Comparison'),(4943,'compartment'),(334456,'compartmental'),(57093,'compartmentalize'),(111832,'compatibility'),(47745,'compatible'),(8797,'compelling'),(12246,'compensate'),(309008,'compensation'),(14326,'compensatory'),(4687,'compete'),(68525,'competence'),(72540,'competency'),(30934,'competent'),(31924,'competition'),(17757,'competitive'),(331534,'competitor'),(49868,'compile'),(20911,'complaint'),(11160,'complement'),(62311,'complementarity'),(3221,'complementary'),(12110,'complementation'),(4127,'complete'),(7876,'completely'),(20701,'completeness'),(64837,'completion'),(3059,'complex'),(18061,'complexation'),(91345,'complexe'),(132482,'complexed'),(23294,'complexity'),(36379,'compliance'),(23348,'compliant'),(5839,'complicate'),(31790,'complicated'),(180,'complication'),(336293,'compliment'),(143771,'complimentarity'),(96757,'comply'),(352080,'compo'),(4482,'component'),(30451,'compose'),(15430,'composite'),(51366,'composition'),(47507,'compound'),(305842,'Compounds'),(107861,'comprehensible'),(11293,'comprehensive'),(307764,'comprehensively'),(22552,'compressed'),(345503,'compression'),(22102,'comprise'),(8860,'compromise'),(28694,'Compton'),(96156,'compulsory'),(22023,'computation'),(51675,'computational'),(89354,'computationally'),(87855,'Computations'),(17158,'compute'),(20997,'computer'),(348273,'computergenerate'),(121960,'computerized'),(23263,'computing'),(123453,'comtamination'),(17487,'con'),(373607,'CONACYT'),(51566,'concatenate'),(348230,'concatenation'),(40217,'conceal'),(29466,'conceivable'),(5506,'conceivably'),(102266,'conceive'),(16497,'concentrate'),(64678,'Concentrated'),(2833,'concentration'),(9160,'concept'),(75899,'conception'),(60287,'conceptual'),(72179,'conceptualisation'),(35560,'conceptually'),(78,'concern'),(95370,'concerned'),(10466,'conclude'),(369860,'concluding'),(944,'conclusion'),(2591,'conclusive'),(354183,'conclusively'),(33126,'concominantly'),(16653,'concomitant'),(432,'concomitantly'),(52643,'concordance'),(107857,'concordant'),(353213,'concrete'),(82914,'concur'),(37002,'concurrent'),(137230,'concurrently'),(127264,'cond'),(123548,'condens'),(123494,'condensate'),(57136,'condensation'),(129591,'condense'),(130421,'condensed'),(78214,'condi'),(2128,'condition'),(38124,'conditional'),(349250,'conditioning'),(5425,'Conditions'),(145486,'Condò'),(102239,'condom'),(310519,'conducive'),(7241,'conduct'),(2363,'conductance'),(6986,'conducting'),(300685,'conduction'),(17762,'confer'),(16493,'conference'),(19711,'conferred'),(14757,'confidence'),(105874,'confident'),(136928,'configurable'),(21763,'configuration'),(136909,'configure'),(8864,'confine'),(53,'confirm'),(104279,'confirmation'),(95322,'confl'),(75173,'conflict'),(18517,'conflicting'),(73076,'confluence'),(43371,'confluency'),(109438,'confluent'),(114097,'confocal'),(76193,'conform'),(4235,'conformation'),(3990,'conformational'),(104360,'conformationally'),(358092,'conformationsensitive'),(55211,'confound'),(326202,'confounder'),(349274,'confounding'),(72149,'confront'),(45661,'confuse'),(374072,'confusing'),(48549,'confusion'),(396,'congenial'),(104441,'congenic'),(564,'congenital'),(393,'congestive'),(74674,'Congo'),(16837,'congress'),(334468,'congruence'),(316083,'conjecture'),(321688,'Conjoint'),(28795,'conjugate'),(339367,'conjugating'),(30467,'conjugation'),(45101,'conjunction'),(343979,'conjunctiva'),(340696,'conjunctival'),(332788,'conjunctivitis'),(3807,'connect'),(316616,'connected'),(21468,'connection'),(7420,'connective'),(21956,'connectivity'),(133684,'connexin'),(303972,'Connolly'),(150390,'Cons'),(7802,'conscious'),(324589,'consciousness'),(15086,'consecutive'),(373602,'Consejo'),(53180,'consensual'),(9376,'consensus'),(36435,'consent'),(2575,'consequence'),(3092,'consequent'),(98686,'consequentially'),(12578,'consequently'),(4128,'conservation'),(6249,'conservative'),(38537,'conservatively'),(4002,'conserve'),(46259,'conserved'),(253,'consider'),(2697,'considerable'),(4874,'considerably'),(5520,'consideration'),(3794,'consist'),(44102,'consistency'),(38,'consistent'),(14981,'consistently'),(117790,'Consisting'),(297,'consolidation'),(335738,'Consortium'),(312961,'conspicuous'),(108307,'conspicuously'),(28078,'constant'),(307526,'constantly'),(74650,'constellation'),(74054,'constituency'),(4383,'constituent'),(4524,'constitute'),(48270,'Constitution'),(22136,'constitutional'),(18469,'constitutive'),(2033,'constitutively'),(306416,'constrain'),(12461,'constraint'),(6645,'constrict'),(11820,'construct'),(29693,'construction'),(310121,'constructively'),(48985,'consultation'),(72535,'consultative'),(347647,'consumable'),(64215,'consume'),(120891,'consumed'),(326064,'consuming'),(14992,'consumption'),(652,'contact'),(316921,'contacted'),(314075,'contactee'),(41283,'Contacts'),(308663,'contagion'),(345014,'contagiosum'),(40804,'contagious'),(137683,'contagiousness'),(2641,'contain'),(40866,'containable'),(136913,'container'),(38274,'containment'),(66015,'contaminant'),(76420,'contaminate'),(20729,'contaminated'),(4896,'contamination'),(148720,'contemplate'),(22057,'contemporary'),(2336,'contend'),(8705,'content'),(73533,'contentious'),(2346,'context'),(90469,'contextual'),(320748,'contextually'),(25493,'contig'),(11040,'contiguous'),(100164,'continent'),(105510,'contingency'),(360877,'contingent'),(123910,'continous'),(356616,'continual'),(73370,'continuation'),(2444,'continue'),(8202,'continued'),(3724,'continuity'),(7498,'continuous'),(20945,'continuously'),(25487,'contortus'),(314993,'contour'),(91135,'contract'),(8533,'contractile'),(42330,'contracting'),(8522,'contraction'),(75980,'contradict'),(12162,'contradiction'),(322039,'contradictory'),(74968,'contraindication'),(372525,'contrary'),(774,'contrast'),(52548,'contrasted'),(2112,'contribute'),(325604,'contributing'),(2126,'contribution'),(100682,'contributor'),(3248,'control'),(118389,'Controlled'),(366975,'controller'),(378946,'controltreate'),(17149,'controversial'),(348438,'controversially'),(17020,'controversy'),(122759,'contusion'),(148297,'conundrum'),(30444,'convalently'),(334386,'convalescence'),(102991,'convalescent'),(72251,'convene'),(347294,'convenience'),(3474,'convenient'),(356874,'conveniently'),(368236,'convention'),(20600,'conventional'),(21039,'conventionally'),(316089,'converge'),(95216,'convergence'),(25989,'convergent'),(308669,'conversant'),(353830,'conversation'),(7224,'conversely'),(8531,'conversion'),(8143,'convert'),(148727,'convertase'),(306995,'convey'),(14803,'convince'),(16118,'convincing'),(17478,'convivial'),(342857,'convoluted'),(317502,'convolution'),(312854,'Convolvulus'),(9988,'Cook'),(62415,'cool'),(69990,'Coomassie'),(366499,'Cooper'),(5158,'cooperate'),(17905,'cooperation'),(3934,'cooperative'),(313543,'cooperativity'),(46544,'coordinate'),(48687,'coordinated'),(329814,'coordinately'),(45316,'coordination'),(353258,'coordinator'),(36303,'Copan'),(17599,'COPD'),(48187,'cope'),(319345,'copious'),(11066,'copy'),(75404,'copyright'),(313986,'Coquette'),(7775,'cor'),(62137,'Coralville'),(9636,'cord'),(4494,'core'),(303179,'coreceptor'),(31809,'corepressor'),(305603,'Coriell'),(303751,'Cornell'),(20741,'corner'),(136850,'cornerstone'),(43376,'Corning'),(36305,'Corona'),(301105,'coronal'),(14582,'coronary'),(27125,'Coronaviridae'),(11113,'coronavirus'),(10989,'coronaviruse'),(24038,'Coronaviruses'),(48583,'coroner'),(48576,'coronial'),(110137,'coronoviruse'),(109458,'Corp'),(95589,'corporate'),(14190,'Corporation'),(300890,'corpus'),(11497,'correct'),(18540,'correction'),(44631,'corrective'),(26477,'correctly'),(82452,'correctness'),(2815,'correlate'),(3483,'correlated'),(14922,'correlation'),(303087,'correlative'),(4606,'correspond'),(52763,'Correspondence'),(12135,'corresponding'),(9892,'correspondingly'),(139192,'corridor'),(52645,'corroborate'),(31620,'cortex'),(31656,'cortical'),(18823,'corticosteroid'),(341364,'COS'),(32323,'Cossart'),(16186,'cost'),(380584,'costaine'),(115863,'costimulatory'),(42801,'costly'),(131174,'cosuppression'),(11037,'cotranscriptional'),(105369,'cotransfecte'),(45977,'cotton'),(842,'cough'),(48164,'coughing'),(301171,'coumarin'),(21626,'council'),(95832,'counseling'),(14185,'count'),(20905,'counter'),(45501,'counteract'),(119983,'counterbalance'),(16421,'counterintuitively'),(379564,'countermeasure'),(2276,'counterpart'),(17786,'counterprotective'),(301989,'counterstaining'),(14176,'counting'),(777,'country'),(21951,'county'),(30748,'couple'),(6449,'coupling'),(22191,'course'),(48360,'Court'),(4206,'covalent'),(77225,'covalently'),(333163,'covariance'),(36500,'cover'),(20746,'coverage'),(360602,'coverslip'),(4249,'cow'),(4987,'coworker'),(79989,'cowpea'),(119626,'cowpox'),(338151,'Cox'),(311191,'coxsackie'),(124521,'coxsackievirus'),(324864,'Coyle'),(47785,'cp'),(372586,'cpa'),(372587,'CPb'),(124895,'CPE'),(57168,'CPEB'),(140894,'CPG'),(348660,'CPH'),(122847,'CPIS'),(350325,'CPs'),(88145,'cpu'),(359623,'Cr'),(100212,'crack'),(100955,'crackdown'),(76203,'Crain'),(313753,'Cramer'),(120806,'cramp'),(360123,'Crane'),(346069,'cranial'),(23812,'crassa'),(100922,'CRC'),(3819,'CRD'),(358977,'CRDs'),(331704,'CRE'),(21432,'create'),(90837,'creation'),(143042,'Creative'),(117917,'creativity'),(148150,'creb'),(353473,'credibility'),(52302,'credible'),(53383,'credit'),(150743,'creep'),(144762,'crenarchaeon'),(504,'crepitation'),(100335,'Crescent'),(16923,'Créteil'),(120626,'Creutzfeldt'),(75943,'Crick'),(100128,'crime'),(74662,'Crimean'),(101560,'criminal'),(13708,'crinkle'),(339107,'cripple'),(340959,'Cripps'),(48797,'crisis'),(16282,'criterion'),(2678,'critical'),(14302,'critically'),(96199,'criticize'),(44855,'crocidolite'),(360331,'Croft'),(348708,'crofter'),(135956,'Crohn'),(95659,'crop'),(22050,'cross'),(49096,'crosse'),(59953,'crossing'),(3892,'crosslink'),(4207,'crosslinke'),(4211,'crosslinked'),(4183,'crosslinking'),(12849,'crossover'),(372343,'crossprotection'),(3691,'crossreaction'),(361547,'crossreactive'),(327061,'crossreactivity'),(367039,'crosssectional'),(29542,'crosstalk'),(4651,'Crouch'),(30,'croup'),(335602,'crowded'),(372837,'CRP'),(2705,'crucial'),(24421,'crude'),(345346,'crump'),(303042,'crush'),(29831,'Cruz'),(371333,'cruzi'),(93611,'cryo'),(109235,'cryopreserve'),(301061,'cryoprotecte'),(109291,'cryovial'),(5281,'cryptic'),(344523,'cryptococcal'),(147731,'cryptococcosis'),(147727,'Cryptococcus'),(3922,'crystal'),(31641,'crystallin'),(3873,'crystallographic'),(303555,'crystallography'),(358283,'CSAAA'),(358282,'CSCCC'),(56342,'Cse'),(321835,'CSF'),(113690,'CSNKIE'),(57959,'ct'),(77390,'CTC'),(306539,'CTCGGCCCTT'),(339031,'CTcontrol'),(360910,'cterminal'),(366303,'Cterminus'),(64532,'CTG'),(86860,'CTGAT'),(125147,'CTGGGACATAGAYATAACWGG'),(116047,'CTL'),(76445,'Ctranscription'),(64536,'CTT'),(339032,'CTtarget'),(109649,'CTX'),(3814,'Ctype'),(47338,'Cucumber'),(32319,'Cudmore'),(122327,'cuff'),(75709,'CUG'),(314209,'cull'),(314046,'culling'),(72297,'culmination'),(13064,'cult'),(77845,'cultivate'),(47570,'cultivation'),(95328,'cultural'),(102220,'culturally'),(47,'culture'),(6672,'cultured'),(89,'cultureproven'),(25419,'culty'),(9853,'cumbersome'),(348691,'Cumbria'),(35826,'cumulative'),(100664,'cumulatively'),(109833,'cup'),(300875,'cuprizone'),(91724,'curate'),(91763,'curation'),(73570,'curative'),(97230,'curb'),(73587,'cure'),(371293,'curiosity'),(2148,'current'),(3559,'currently'),(101071,'curricula'),(117006,'curse'),(15461,'curve'),(25506,'cus'),(34945,'cushion'),(22066,'custom'),(139726,'customer'),(116875,'customization'),(327203,'customize'),(10316,'cut'),(345639,'cutaneous'),(39616,'cutoff'),(66532,'cuvette'),(54771,'cv'),(124537,'CVA'),(124539,'CVB'),(135451,'CXC'),(358147,'CXXC'),(112873,'cy'),(46724,'cyan'),(85569,'cyanoethyl'),(3091,'cyclase'),(11479,'cycle'),(134229,'Cycler'),(18073,'cyclic'),(117817,'cyclically'),(30558,'cyclin'),(94444,'cycling'),(2738,'cyclo'),(33778,'cycloheximide'),(56706,'Cyclone'),(135269,'cyclooxygenase'),(326572,'cyclosporin'),(99967,'cylinder'),(79189,'cylindracea'),(319096,'cynomolgus'),(83698,'cystatin'),(3150,'cysteine'),(361167,'cysteinedependency'),(2351,'cystic'),(57688,'cystitis'),(367614,'cytapheresis'),(76704,'cytidine'),(35150,'cytochalasin'),(135111,'cytochrome'),(9760,'cytokeratin'),(2051,'cytokine'),(351740,'Cytokines'),(110667,'cytological'),(36111,'cytology'),(321897,'cytolytic'),(46214,'cytomatrix'),(445,'cytomegalo'),(20161,'cytomegalovirus'),(104699,'cytometer'),(102492,'cytometric'),(24719,'cytometry'),(10536,'cytopathic'),(47784,'cytopathogenic'),(379979,'cytopathogenicity'),(380771,'Cytoperm'),(27017,'cytoplasm'),(3650,'cytoplasmic'),(17733,'cytoprotection'),(17539,'cytoprotective'),(43203,'cytosine'),(2893,'cytoskeletal'),(24007,'cytoskeleton'),(129517,'cytosol'),(56337,'cytosolic'),(2510,'cytotoxic'),(2784,'cytotoxicity'),(149993,'Cα'),(150009,'Cβ'),(105254,'Cκ'),(12195,'da'),(301224,'DAB'),(310776,'DAD'),(310743,'DAH'),(133108,'Dai'),(207,'daily'),(105066,'DAKO'),(360538,'DakoCytomation'),(34542,'Dales'),(144680,'dalgarno'),(144818,'Dalgarnolike'),(18628,'damage'),(30649,'damaged'),(36915,'damaging'),(137969,'damp'),(377500,'dampen'),(137960,'damping'),(363185,'Dana'),(48916,'danger'),(116532,'dangerous'),(332208,'Daniel'),(308337,'Daniels'),(34493,'DAPI'),(104422,'dard'),(56703,'dark'),(43594,'Darmstadt'),(313978,'DAS'),(16048,'dash'),(170,'data'),(54088,'databank'),(10355,'database'),(60999,'dataset'),(9315,'date'),(43242,'datp'),(322,'datum'),(145330,'daughter'),(29450,'David'),(144758,'dawn'),(9977,'day'),(138028,'daycare'),(27461,'DBT'),(12453,'DC'),(348757,'dcs'),(114036,'dctp'),(57287,'DD'),(24201,'DDBJ'),(67138,'DDCt'),(86803,'DDG'),(7690,'de'),(76697,'deacetylation'),(104448,'Deaconess'),(322128,'deactivate'),(321914,'deactivating'),(322048,'deactivation'),(77117,'deacylate'),(76461,'deacylation'),(28085,'dead'),(105456,'deadly'),(79217,'DEAE'),(322333,'deafness'),(147582,'DEAH'),(44324,'deal'),(309,'death'),(14425,'debate'),(345506,'debilitate'),(326745,'debilitation'),(345697,'debridement'),(352658,'debriefing'),(85234,'debris'),(75342,'début'),(127524,'dec'),(49138,'decade'),(109005,'decaffeinated'),(23640,'decarboxylase'),(25866,'decarboxylate'),(24605,'decarboxylation'),(37865,'decay'),(115,'December'),(108532,'decent'),(3058,'deceptively'),(356741,'deci'),(15137,'decide'),(108495,'decidedly'),(108408,'decidely'),(61484,'decimal'),(26241,'decipher'),(15245,'decision'),(308236,'decisionmake'),(40381,'decisive'),(141744,'deck'),(96758,'declaration'),(72262,'declare'),(7572,'decline'),(23595,'decode'),(24031,'decoding'),(136517,'decompose'),(132572,'decomposition'),(57619,'decompression'),(340157,'deconjugate'),(39019,'decontainment'),(124147,'decontamination'),(25231,'deconvolute'),(348127,'deconvolve'),(329952,'decoy'),(2724,'decrease'),(103383,'decreased'),(23021,'decrypt'),(23028,'decryption'),(50793,'dedicate'),(22012,'dedicated'),(25270,'deduce'),(24341,'deduced'),(121245,'deduct'),(76663,'deduction'),(117973,'deductive'),(82758,'deem'),(343656,'deep'),(133956,'deeply'),(45577,'default'),(335782,'defeat'),(50250,'defect'),(12585,'defective'),(45502,'defence'),(19040,'defend'),(95944,'defender'),(1997,'defense'),(308751,'defensible'),(336833,'defensin'),(147400,'defensive'),(352837,'defensively'),(308971,'defer'),(2217,'deficiency'),(2253,'deficient'),(325110,'deficit'),(139672,'definable'),(277,'define'),(59054,'definite'),(53571,'definitely'),(9834,'definition'),(56149,'definitive'),(14920,'definitively'),(14394,'deformable'),(57383,'deformation'),(67630,'Deg'),(330053,'degeneracy'),(10282,'degenerate'),(54460,'degeneration'),(66713,'DegFact'),(127361,'degradable'),(5439,'degradation'),(360713,'degradative'),(5199,'degrade'),(30586,'degraded'),(136216,'degrading'),(66323,'degradometer'),(3356,'degranulation'),(2136,'degree'),(92414,'dehydrogenase'),(51732,'Deinococcus'),(99963,'deionized'),(80244,'DeKalb'),(364936,'del'),(10054,'delay'),(317974,'delaydistribution'),(14490,'delayed'),(363672,'Delcourt'),(10009,'delete'),(80318,'deleted'),(16093,'deleterious'),(2178,'deletion'),(38751,'deliberate'),(133793,'deliberately'),(96399,'deliberation'),(308376,'deliberative'),(305074,'delicate'),(139269,'delimit'),(139290,'delimiter'),(25066,'delineate'),(325081,'deliriogenic'),(324917,'delirious'),(324577,'delirium'),(332203,'DeLisi'),(13613,'delity'),(5870,'deliver'),(14939,'delivery'),(95153,'Delphi'),(325132,'delusion'),(16329,'demand'),(330101,'demanding'),(138443,'demarcation'),(31609,'dementia'),(135619,'demersal'),(47406,'demethylate'),(45254,'demethylation'),(106600,'demic'),(120464,'Democratic'),(176,'demographic'),(16317,'demographically'),(142640,'demon'),(16779,'demonstrable'),(2526,'demonstrate'),(145867,'demonstrating'),(63378,'demonstration'),(356891,'demonstrative'),(424,'demyelinate'),(300783,'demyelinated'),(300725,'demyelination'),(11267,'den'),(146667,'denaturate'),(61890,'denaturation'),(14226,'denature'),(319748,'dendritic'),(339577,'deneddylating'),(84946,'dengue'),(3032,'denitration'),(19760,'Dennery'),(348516,'denominator'),(36415,'denote'),(57096,'dense'),(332892,'densely'),(341683,'densitometric'),(60895,'densitometry'),(4572,'density'),(333034,'densityand'),(105968,'dentist'),(6724,'denudation'),(6643,'denude'),(17254,'deny'),(140483,'deoxy'),(141108,'deoxynucleotide'),(305484,'deoxyribose'),(128885,'deoxyuridine'),(320458,'deparaffinization'),(121178,'depart'),(141,'department'),(73471,'departure'),(56521,'depc'),(62358,'DEPCtreated'),(2132,'depend'),(49348,'dependant'),(3984,'dependence'),(105630,'dependency'),(3225,'dependent'),(133212,'dependently'),(79396,'dephosphorylate'),(11582,'depict'),(43083,'deplete'),(19061,'depletion'),(20705,'deploy'),(22055,'deployment'),(33325,'depolymerization'),(32743,'depolymerize'),(332866,'depopulate'),(36726,'deposit'),(5244,'deposition'),(75447,'Depot'),(4798,'deprivation'),(78439,'deprive'),(140916,'deprotecte'),(305740,'deprotected'),(85601,'deprotection'),(72207,'depth'),(366893,'deq'),(2800,'der'),(377907,'deregulate'),(23940,'deregulation'),(72137,'Derek'),(24687,'derepresse'),(24672,'derepression'),(307744,'derivate'),(375010,'derivation'),(34877,'derivative'),(2311,'derive'),(360362,'dermatitis'),(716,'dermatologic'),(346038,'dermatomal'),(64920,'desalt'),(348068,'desalting'),(19252,'desaturation'),(136694,'descend'),(136486,'descendent'),(53689,'Deschavanne'),(3068,'describe'),(9782,'description'),(74899,'descriptive'),(139284,'descriptor'),(6723,'desensitization'),(52495,'deserve'),(77734,'desig'),(14719,'design'),(29549,'designate'),(4365,'designation'),(354158,'designer'),(21085,'desirable'),(30358,'desire'),(98183,'desk'),(334754,'desktop'),(93193,'desmolt'),(93194,'desmoltification'),(92931,'desmoltifie'),(2117,'despite'),(19332,'desquamative'),(85875,'destabilization'),(62023,'destabilize'),(61991,'destabilizing'),(30886,'destain'),(362084,'destine'),(42983,'destruction'),(43750,'destructive'),(27815,'detach'),(82925,'detachment'),(11617,'detail'),(20991,'detailed'),(512,'detect'),(2380,'detectable'),(4686,'detectably'),(2514,'detection'),(22828,'detector'),(101670,'detention'),(4994,'detergent'),(19189,'deterioration'),(3878,'determinant'),(66827,'determination'),(2199,'determine'),(28262,'determined'),(316591,'deterministic'),(334463,'deterministically'),(17623,'detoxification'),(90167,'detriment'),(10034,'detrimental'),(339121,'deubiquitinase'),(339126,'deubiquitinate'),(340383,'deubiquitinating'),(340682,'Deuren'),(133524,'Deutsche'),(122714,'devascularized'),(75304,'devastating'),(124320,'develoment'),(36,'develop'),(81069,'developed'),(87882,'developer'),(2286,'development'),(5985,'developmental'),(19672,'developmentally'),(57496,'developmentdependent'),(349470,'deviance'),(92853,'deviate'),(3976,'deviation'),(64510,'device'),(43801,'Devices'),(357713,'devise'),(49912,'devoid'),(348694,'Devon'),(355647,'devote'),(52121,'devoted'),(6121,'dexamethasone'),(147584,'DExD'),(147583,'DExH'),(326340,'Dexmedetomidine'),(344215,'dextran'),(50110,'dextransucrase'),(79036,'dextrose'),(150065,'df'),(26841,'DFMO'),(55600,'DG'),(362899,'DGGE'),(43238,'dgtp'),(64796,'dh'),(362900,'dHPLC'),(4640,'di'),(5463,'diabete'),(116606,'diabetes'),(5704,'diabetic'),(266,'diagnose'),(48,'diagnosis'),(17583,'diagnostic'),(36447,'Diagnostica'),(36304,'Diagnostics'),(46127,'diagonal'),(62792,'diagram'),(342888,'Diagrams'),(75322,'dialogue'),(14621,'dialysis'),(17339,'dialytic'),(26805,'dialyze'),(64706,'dialyzed'),(221,'diameter'),(109321,'diamine'),(301223,'diaminobenzidine'),(336451,'diaminopimelic'),(22971,'diarrhea'),(121419,'diarrheal'),(349531,'DIC'),(131389,'Dicer'),(105991,'dichotomize'),(366074,'dichroism'),(44867,'dichromate'),(359994,'Dick'),(56568,'Dickinson'),(9854,'dictate'),(91682,'dictionary'),(62663,'dideoxynucleotide'),(549,'die'),(301151,'Diego'),(18355,'diesel'),(96791,'diet'),(97147,'dietary'),(119823,'Dietz'),(27426,'Diez'),(13063,'dif'),(79049,'Difco'),(4551,'differ'),(787,'difference'),(2010,'different'),(10189,'differential'),(10313,'differentially'),(10668,'differentiate'),(2903,'differentiation'),(48779,'differently'),(96343,'diffi'),(22909,'difficile'),(45,'difficult'),(37174,'difficulty'),(7929,'diffuse'),(47086,'diffused'),(2491,'diffusion'),(53130,'dig'),(26597,'digest'),(365454,'digested'),(30205,'digestion'),(49122,'digestive'),(43649,'Digital'),(54634,'digitize'),(358650,'digitonin'),(358300,'digitoninpermeabilize'),(95073,'dignity'),(301099,'digoxigenin'),(301073,'digoxigeninlabele'),(324749,'digoxin'),(140582,'diGTP'),(305803,'dihydroxyphenyl'),(75720,'dii'),(102435,'DIII'),(19577,'dilation'),(101701,'dilemma'),(25764,'dilute'),(93335,'diluted'),(24677,'dilution'),(51907,'dimension'),(51905,'dimensional'),(117007,'dimensionality'),(4125,'dimer'),(325744,'dimeric'),(34754,'dimethyl'),(306050,'diminish'),(369655,'diminishe'),(18389,'diminished'),(3190,'dinitrogen'),(62049,'Dinman'),(11375,'Dinten'),(57414,'DInterestingly'),(98117,'diode'),(2100,'dioxide'),(135200,'dioxin'),(317830,'dip'),(16463,'diphosphate'),(78920,'diploid'),(72561,'diploma'),(312856,'Diplotaxis'),(337423,'Diptera'),(147733,'dipteran'),(4111,'dipyridyl'),(2513,'direct'),(344177,'Directed'),(16517,'direction'),(4867,'directional'),(143197,'directionality'),(351997,'directive'),(2827,'directly'),(22881,'director'),(87856,'directorate'),(108685,'disability'),(326695,'disabled'),(20875,'disadvantage'),(56510,'disaggregate'),(67981,'disagreement'),(17288,'disappear'),(379713,'disappearance'),(83297,'disassembly'),(349161,'disassortative'),(105446,'disaster'),(745,'disc'),(137367,'discard'),(26306,'discern'),(347469,'discernable'),(15967,'discernible'),(17275,'discharge'),(117990,'discipline'),(306950,'disclose'),(96337,'disclosure'),(123516,'disconnection'),(37419,'discontinuation'),(122916,'discontinue'),(10995,'discontinuous'),(13553,'discontinuously'),(317343,'discount'),(101030,'discourage'),(74286,'discourse'),(9547,'discover'),(1975,'discovery'),(15810,'discrepancy'),(3795,'discrete'),(36772,'discretion'),(374569,'discretise'),(75985,'discriminaccident'),(77782,'discriminate'),(51638,'discrimination'),(75946,'discriminator'),(117186,'discriminatory'),(307776,'discursive'),(3137,'discuss'),(21607,'discussion'),(398,'disease'),(15544,'diseased'),(74358,'Diseases'),(371202,'diseasespecific'),(316686,'disembark'),(43375,'dish'),(16226,'dishearten'),(57230,'disialoganglioside'),(123399,'disinfect'),(123333,'disinfection'),(135826,'disintegrin'),(22681,'Disk'),(16595,'dismal'),(107478,'dismissal'),(20012,'dismutase'),(416,'disorder'),(325418,'Disorders'),(33156,'disorganized'),(64001,'disparate'),(310283,'disparitie'),(891,'disparity'),(42639,'dispatch'),(107869,'dispel'),(80726,'dispensable'),(99422,'dispense'),(32834,'disperse'),(32633,'dispersion'),(61980,'displace'),(97923,'displacement'),(2265,'display'),(99699,'disposable'),(96036,'disposal'),(9613,'dispose'),(309338,'disproportionate'),(23849,'disprove'),(95850,'dispute'),(5433,'disrupt'),(2212,'disruption'),(132234,'disruptive'),(28475,'DIssE'),(11365,'dissect'),(8302,'disseminate'),(21034,'dissemination'),(44404,'dissimilar'),(115429,'dissociable'),(63828,'dissociate'),(129154,'dissociation'),(108640,'dissolution'),(34753,'dissolve'),(5557,'distal'),(58092,'distally'),(45763,'distance'),(309074,'distancing'),(5277,'distant'),(25923,'distantly'),(17182,'distension'),(16498,'distil'),(2006,'distinct'),(57388,'distinction'),(320860,'distinctive'),(3288,'distinctly'),(12708,'distinguish'),(80824,'distinguishable'),(148319,'distinguishe'),(78685,'distort'),(96585,'distorted'),(353539,'distract'),(353504,'distraction'),(543,'distress'),(139205,'distri'),(5119,'distribute'),(3928,'distribution'),(95700,'distributive'),(75103,'district'),(309793,'distrust'),(13418,'disturb'),(14848,'disturbance'),(3891,'disulfide'),(26786,'dithiothreitol'),(361660,'diubiquitin'),(306126,'divalent'),(47706,'diverge'),(24141,'divergence'),(15519,'divergent'),(1987,'diverse'),(76180,'diversification'),(340483,'diversified'),(336906,'diversify'),(39402,'diversion'),(46911,'diversity'),(42083,'divert'),(58113,'Divi'),(22481,'divide'),(311620,'divided'),(25834,'dividing'),(24683,'division'),(78727,'Dix'),(37797,'Dixon'),(15328,'dl'),(343203,'DLK'),(362619,'DME'),(31012,'DMEM'),(367581,'DMIRB'),(34756,'DMSO'),(305867,'dmt'),(379175,'DN'),(2538,'dna'),(131843,'DNAbased'),(29190,'DNAdependent'),(348262,'dnadist'),(50691,'dnak'),(31433,'DNAs'),(56499,'DNase'),(145455,'DNAseI'),(26940,'DNasetreated'),(305311,'DNAzyme'),(363197,'DNeasy'),(338934,'dNTP'),(62600,'dntps'),(367855,'DOC'),(141733,'dock'),(107427,'doctor'),(2512,'document'),(321474,'documentary'),(57808,'documentation'),(3832,'dodecamer'),(141005,'dodecyl'),(300986,'Doetschman'),(102015,'dofor'),(7704,'dog'),(144490,'dogma'),(72592,'DoHA'),(59766,'DOI'),(73926,'dollar'),(336620,'Dom'),(3797,'domain'),(320340,'domaindelete'),(336100,'domeless'),(48410,'domestic'),(337960,'domesticate'),(127050,'dominance'),(12434,'dominant'),(28701,'dominantnegative'),(16189,'dominate'),(343919,'donate'),(56476,'Donation'),(326275,'Donepezil'),(5006,'Dong'),(301152,'Donkey'),(7896,'donor'),(372061,'donovani'),(100310,'door'),(324708,'dopamine'),(324998,'dopaminergic'),(105768,'Dorchester'),(123892,'dorsal'),(74377,'DOS'),(74964,'dosage'),(7272,'dose'),(109753,'dosimetric'),(336604,'Dostert'),(10183,'dot'),(29756,'DOTAP'),(39355,'dotted'),(47970,'Dotter'),(11540,'double'),(88792,'doublestrande'),(131820,'doublestranded'),(28130,'doublet'),(39811,'doubling'),(48539,'doubt'),(16433,'doubtful'),(138314,'Dowdle'),(333037,'Downhill'),(22190,'download'),(136833,'downloadable'),(357387,'downloaded'),(8177,'downregulate'),(8062,'downregulation'),(6211,'downstream'),(337985,'downward'),(379185,'doxycycline'),(88160,'dozen'),(127309,'dp'),(129761,'dpol'),(29793,'Dr'),(47194,'Drabløs'),(72675,'draft'),(141591,'drag'),(62324,'DraI'),(72097,'drain'),(123509,'drainage'),(10161,'dramatic'),(11438,'dramatically'),(32324,'Dramsi'),(318229,'drastic'),(53479,'drastically'),(17416,'draw'),(2084,'drawback'),(35650,'drawing'),(105607,'dread'),(27009,'Dreyfuss'),(38028,'drier'),(105160,'drift'),(96977,'drink'),(97106,'drinker'),(96881,'drinking'),(10230,'drive'),(72224,'driver'),(47021,'driving'),(121814,'drome'),(20850,'drop'),(21130,'dropin'),(650,'droplet'),(101582,'dropout'),(56588,'Drops'),(24187,'Drosophila'),(137989,'drown'),(43144,'drp'),(43147,'dRPase'),(17977,'drug'),(42870,'druginduce'),(371450,'drugresistant'),(73970,'Drugs'),(850,'dry'),(86375,'DS'),(51153,'DSB'),(337030,'DSCAM'),(64055,'dsdna'),(325472,'DSM'),(331698,'dso'),(148205,'dsRBD'),(50107,'dsrd'),(83102,'dsrna'),(149144,'dsrnas'),(43856,'DT'),(372448,'dth'),(26787,'DTT'),(11836,'DU'),(22669,'dual'),(306961,'dualdye'),(96391,'dualuse'),(360743,'dub'),(360718,'DUBs'),(3717,'duct'),(58543,'ductular'),(343548,'Duim'),(31008,'Dulbecco'),(97215,'dull'),(350554,'Dumfries'),(373125,'Dumonteil'),(32450,'Duncan'),(58101,'duodenum'),(12011,'duplex'),(147530,'duplexed'),(86480,'duplexesThe'),(21876,'duplicate'),(51249,'duplication'),(15006,'duration'),(5912,'dust'),(94072,'dutch'),(105530,'duty'),(13716,'dwarf'),(324888,'dwell'),(38017,'dwelling'),(304240,'dx'),(80820,'DXho'),(81366,'DXhoR'),(304241,'dy'),(10680,'dye'),(306540,'DyeProbe'),(112874,'dyes'),(34128,'dynactin'),(22226,'dynamic'),(117377,'dynamical'),(127145,'dynamically'),(143315,'Dynamics'),(32775,'dynamitin'),(145621,'dynamo'),(100262,'Dynasty'),(34087,'dynein'),(16371,'dys'),(74731,'dysentery'),(19165,'dysfunc'),(8419,'dysfunction'),(20074,'dysplasia'),(94234,'dyspnea'),(110186,'dyspnoea'),(323046,'dysregulate'),(344288,'dysregulation'),(31711,'dystrophic'),(304242,'dz'),(130333,'dðàþnc'),(130249,'dðþÞnc'),(12433,'e'),(303107,'EAE'),(340297,'EAEC'),(341639,'Eagan'),(16218,'eagerly'),(29668,'Eagle'),(136654,'Eames'),(322315,'ear'),(322394,'eardrop'),(4877,'early'),(37705,'ease'),(2130,'easily'),(48222,'East'),(95396,'Eastern'),(85313,'Eastman'),(25416,'easy'),(48131,'eat'),(337031,'Eater'),(11225,'EAV'),(147766,'ebi'),(74659,'Ebola'),(318944,'ebolavirus'),(318933,'EBOV'),(6153,'EBP'),(6171,'EBPβ'),(35593,'EBV'),(4650,'EC'),(7866,'ECE'),(108992,'ECG'),(32779,'echeverri'),(108444,'echo'),(70011,'ECL'),(305632,'Eclipse'),(374802,'ecological'),(75046,'ecology'),(72574,'economic'),(116299,'economical'),(126632,'economically'),(72462,'economy'),(26678,'ecori'),(80380,'ecorv'),(312411,'ecosystem'),(100392,'ecstasy'),(102445,'ectodomain'),(358968,'Ectodomains'),(359969,'ectopic'),(359046,'ectopically'),(10957,'ed'),(140964,'EDC'),(78229,'Edelmann'),(746,'edema'),(63703,'edge'),(139240,'edit'),(354442,'editing'),(21331,'Edition'),(90820,'editor'),(75343,'editorial'),(343206,'EDK'),(53251,'EDL'),(136656,'edmund'),(4708,'EDTA'),(75210,'educate'),(139393,'educating'),(17467,'education'),(96884,'educational'),(105986,'educator'),(32512,'EEV'),(10877,'ef'),(2335,'effect'),(14860,'effective'),(21248,'effectively'),(16484,'effectiveness'),(3553,'effector'),(370243,'Effects'),(40281,'efficacious'),(14984,'efficacy'),(3006,'efficiency'),(9602,'efficient'),(4604,'efficiently'),(17625,'efflux'),(9811,'effort'),(905,'effusion'),(131251,'eg'),(108994,'EGC'),(108997,'EGCG'),(337105,'EGF'),(43404,'egfp'),(132818,'EGFR'),(219,'egg'),(148267,'egress'),(34889,'EGTA'),(121394,'Egypt'),(319395,'EHD'),(121818,'EHEC'),(37451,'Eichner'),(340299,'EIEC'),(376899,'eigen'),(316055,'eigenvalue'),(335523,'eigenvector'),(114252,'eighteen'),(21500,'eighth'),(79198,'eighty'),(339043,'Eisen'),(21708,'EJB'),(22085,'EJBs'),(139042,'elaborate'),(142799,'elaboration'),(339435,'elad'),(129263,'elapse'),(8492,'elastase'),(7161,'elastic'),(771,'elderly'),(35891,'ele'),(142527,'elect'),(72631,'election'),(15293,'elective'),(98783,'electric'),(79129,'electrified'),(14506,'electrolyte'),(32903,'electron'),(21005,'Electronic'),(57834,'electronically'),(13030,'electropherogram'),(339694,'electrophile'),(339664,'electrophilic'),(30031,'electrophorese'),(10293,'electrophoresis'),(28624,'electrophoretic'),(84105,'electrophoretically'),(26723,'electroporate'),(14155,'electroporation'),(132032,'electropulsation'),(346645,'electrospray'),(99060,'electrostatic'),(25218,'elegan'),(10823,'elegant'),(6118,'element'),(34964,'elemental'),(6841,'elevate'),(2073,'elevated'),(18479,'elevation'),(4866,'elicit'),(336456,'elicitor'),(110579,'eligibility'),(52071,'eligible'),(371701,'elimiation'),(4996,'eliminate'),(4214,'elimination'),(49662,'ELISA'),(114646,'ELISPOT'),(301221,'elite'),(343171,'ELK'),(34824,'Èllekrug'),(34474,'Elliott'),(14292,'Elmer'),(63154,'elongate'),(61582,'elongating'),(13155,'elongation'),(64647,'Eluate'),(4676,'elucidate'),(75927,'elucidated'),(310902,'elucidation'),(148115,'elusive'),(10318,'elute'),(64795,'elution'),(22879,'email'),(118310,'emanate'),(316256,'emanating'),(353470,'embarrassing'),(118380,'Embase'),(56780,'embed'),(338420,'embellish'),(24202,'EMBL'),(11213,'EMBO'),(53956,'embody'),(110533,'embolus'),(17543,'embrace'),(19821,'embryo'),(57431,'embryogenesis'),(36316,'embryonal'),(31001,'embryonic'),(320215,'EMEM'),(105731,'emer'),(8882,'emerge'),(35850,'emergence'),(140,'emergency'),(48918,'emergent'),(25928,'Emericella'),(140243,'emigration'),(95230,'Emilio'),(54650,'emission'),(98068,'emit'),(98116,'emitted'),(108489,'Emmanuel'),(352274,'Emory'),(309374,'emotional'),(3568,'Emphasis'),(310414,'emphasise'),(16949,'emphasize'),(5627,'emphysema'),(310864,'empiric'),(123286,'empirical'),(62610,'empirically'),(9606,'employ'),(105621,'employee'),(42620,'employer'),(332619,'emptive'),(123527,'empty'),(352878,'EMS'),(330670,'EMSA'),(30894,'en'),(10238,'enable'),(100879,'enact'),(97226,'enactment'),(129737,'encap'),(59521,'encapsidate'),(129021,'encapsidated'),(10893,'encapsidation'),(4602,'encapsulate'),(337128,'encapsulation'),(102427,'encephalitis'),(303105,'encephalomye'),(325047,'encephalopathy'),(133550,'enclose'),(97566,'enclosure'),(10426,'encode'),(81976,'encoding'),(54548,'encompass'),(5462,'encounter'),(16912,'encourage'),(16622,'encouraging'),(22387,'encrypt'),(23027,'encryption'),(17239,'encumber'),(3911,'end'),(33947,'enddirected'),(49032,'endeavor'),(75080,'endemic'),(36935,'ending'),(305444,'Endo'),(120780,'endocarditis'),(96805,'endocrine'),(5430,'endocytic'),(84879,'endocytose'),(83183,'endocytosis'),(132568,'endocytotic'),(2566,'endogenous'),(18472,'endogenously'),(43182,'endonuclease'),(131279,'endonucleolytical'),(337129,'endoparasite'),(4203,'endoplasmic'),(83267,'endoproteinase'),(72438,'endorse'),(101400,'endorsement'),(57621,'endoscopic'),(56352,'endosomal'),(129015,'endosome'),(2026,'endothelial'),(6637,'endothelin'),(6401,'endothelium'),(2314,'endotoxemia'),(325802,'endotoxic'),(2264,'endotoxin'),(2331,'endotoxininduce'),(124,'endotracheal'),(146785,'endow'),(43748,'endpoint'),(3513,'endproduct'),(86589,'endthe'),(108247,'endure'),(63784,'energetic'),(13150,'energetically'),(30486,'energy'),(332638,'enforce'),(107910,'enforceable'),(105539,'enforcement'),(101810,'enforcer'),(74263,'engage'),(74292,'engaged'),(90416,'engagement'),(338139,'Engels'),(32457,'Engelstad'),(95878,'engender'),(21418,'engine'),(49165,'engineer'),(64082,'engineering'),(16842,'England'),(100226,'English'),(2778,'enhance'),(4837,'enhanced'),(4454,'enhancement'),(6150,'enhancer'),(23458,'enlarge'),(5613,'enlarged'),(326009,'enlargement'),(325730,'enolase'),(96247,'enormous'),(96780,'enquire'),(4014,'enrich'),(112451,'enriched'),(102493,'enrichment'),(124806,'enrol'),(326385,'enroll'),(17227,'enrollment'),(356583,'Ensembl'),(150328,'Enshell'),(309018,'enshrine'),(96126,'ensue'),(13612,'ensure'),(342210,'ent'),(106293,'entail'),(457,'enter'),(122477,'enteral'),(49741,'enteric'),(122569,'enterobacter'),(52720,'Enterobacteriales'),(22916,'Enterococcus'),(49765,'enterocyte'),(53224,'enterohaemorrhagic'),(124523,'enterovirus'),(21330,'Enterprise'),(304946,'enthalpically'),(17507,'enthusiasm'),(136835,'enthusiastic'),(4101,'entire'),(3689,'entirely'),(54346,'entirety'),(356090,'Entities'),(54312,'entitle'),(309206,'entitlement'),(90302,'entity'),(61678,'entrance'),(357164,'Entrez'),(309511,'entrust'),(21315,'entry'),(73073,'enunciate'),(147913,'env'),(8828,'envelop'),(126628,'envelope'),(3725,'environment'),(18155,'environmental'),(33945,'envisage'),(50646,'envision'),(95694,'EnvIssues'),(32430,'enwrap'),(139120,'enza'),(13117,'enzymatic'),(142671,'enzymatically'),(2181,'enzyme'),(3031,'enzymic'),(2005,'enzymically'),(109126,'EOC'),(58273,'eosin'),(2617,'eosinophil'),(2268,'eosinophilia'),(6604,'eosinophilic'),(330653,'EPD'),(139245,'epdm'),(340300,'EPEC'),(136618,'Epi'),(122580,'EPIC'),(108973,'epicatechin'),(101906,'epicenter'),(21061,'epidemic'),(96507,'epidemiologic'),(346,'epidemiological'),(38065,'epidemiologically'),(120491,'epidemiologist'),(120207,'Epidemiologists'),(86,'epidemiology'),(6513,'epidermal'),(136548,'EpiFlex'),(108993,'epigallocatechin'),(122246,'epiglottic'),(111993,'epilepsy'),(331338,'epilepticus'),(110124,'epimer'),(110010,'epimerize'),(22418,'epiplot'),(574,'episode'),(72678,'episodic'),(117892,'epistemological'),(343597,'epithelia'),(2018,'epithelial'),(2356,'epithelium'),(57205,'epitope'),(117791,'Epitopes'),(114484,'epitopespecific'),(337836,'epizootic'),(2619,'EPO'),(135262,'epoxyeicosatrienoic'),(135250,'epoxygenase'),(323637,'epsilon'),(20104,'Epstein'),(145128,'EQ'),(6676,'equal'),(334972,'equality'),(141140,'equalized'),(6646,'equally'),(317291,'equate'),(43851,'equation'),(65708,'Equations'),(374229,'equationthis'),(375642,'equationwhich'),(64592,'equilibrate'),(303808,'equilibration'),(46929,'equilibrium'),(63596,'equimolar'),(11222,'equine'),(20614,'equip'),(17336,'equipment'),(307550,'equitable'),(107486,'equitably'),(95076,'equity'),(6678,'equivalent'),(103037,'equivalently'),(121387,'equivocally'),(16239,'era'),(140883,'EraCode'),(141028,'EraCoded'),(141061,'EraCodes'),(100958,'eradicate'),(37225,'eradication'),(140872,'EraGen'),(94437,'Erasmus'),(861,'erate'),(57962,'ERCP'),(340858,'Ergun'),(373124,'Eric'),(89959,'erode'),(38784,'err'),(327589,'erroneous'),(329949,'erroneously'),(24026,'error'),(110240,'ERS'),(312857,'erucoide'),(37650,'eruptive'),(721,'erythema'),(344993,'erythematosus'),(57224,'erythrocyte'),(365908,'erythrocytic'),(29435,'erythroleukemia'),(326398,'erythropoietin'),(85548,'ES'),(369922,'escalate'),(326586,'escalation'),(39619,'escape'),(4489,'Escherichia'),(108865,'eschew'),(346650,'ESI'),(331575,'especial'),(2101,'especially'),(145629,'Espoo'),(308331,'esse'),(21062,'ESSENCE'),(11251,'essential'),(5645,'essentially'),(357765,'Essex'),(25222,'EST'),(3268,'establish'),(133303,'establishe'),(58174,'established'),(29694,'establishment'),(17843,'ester'),(57439,'esterase'),(2963,'estimate'),(60518,'estimated'),(333016,'estimateĥ'),(350423,'Estimates'),(51450,'estimation'),(61399,'estimator'),(375877,'estimatornow'),(136316,'estrogen'),(5057,'et'),(6387,'ETA'),(6398,'etb'),(66070,'EtBr'),(6502,'etc'),(139282,'etcetera'),(340301,'ETEC'),(19203,'ethane'),(30007,'ethanol'),(367412,'ethanolamine'),(16251,'ethic'),(73433,'ethical'),(73399,'ethically'),(307388,'ethicist'),(108047,'Ethics'),(66068,'ethidium'),(60861,'ethydium'),(326543,'Ethyl'),(109320,'ethylene'),(361163,'ethylmaleimide'),(50151,'etiologic'),(310753,'etiological'),(2113,'etiology'),(32494,'Èttger'),(305597,'Etude'),(95261,'EU'),(61459,'euclidean'),(51894,'euclidian'),(51576,'eukarya'),(23842,'eukaryote'),(45285,'Eukaryotes'),(11658,'eukaryotic'),(94435,'EUR'),(134258,'euro'),(140871,'Eurogentec'),(38048,'Europe'),(16966,'European'),(36486,'Eurorad'),(322497,'eustachian'),(93588,'euthanize'),(124947,'EV'),(5959,'evade'),(351390,'eval'),(15079,'evaluate'),(15470,'Evaluation'),(420,'Evan'),(16925,'Evans'),(141776,'evaporation'),(305714,'evaporator'),(57086,'evenly'),(5132,'event'),(8553,'eventual'),(5623,'eventually'),(371429,'eventualy'),(334168,'everybody'),(59203,'everyday'),(272,'evidence'),(10264,'evident'),(76313,'evidently'),(100950,'evil'),(44423,'evoke'),(16446,'evolution'),(46922,'evolutionarily'),(26503,'evolutionary'),(9626,'evolve'),(303777,'Ewald'),(116734,'ex'),(73028,'exacerbate'),(372115,'exacerbating'),(6897,'exacerbation'),(110355,'exacerbative'),(371685,'exacerbatory'),(331,'exact'),(25566,'exactly'),(7408,'exaggerated'),(509,'examination'),(14322,'examine'),(92392,'examined'),(2418,'example'),(16337,'exceed'),(43890,'Excel'),(34304,'excellent'),(65568,'except'),(3727,'exception'),(59675,'exceptional'),(328468,'exceptionally'),(18873,'excerbation'),(107947,'excerpt'),(10156,'excess'),(2085,'excessive'),(2370,'excessively'),(3428,'exchange'),(123553,'exchanger'),(43121,'excise'),(30626,'excision'),(57766,'excitability'),(98338,'excitation'),(16897,'exciting'),(302987,'excitotoxicity'),(75368,'excludable'),(3690,'exclude'),(88410,'exclusion'),(109057,'exclusive'),(3845,'exclusively'),(135676,'excretion'),(21611,'execute'),(137594,'execution'),(324628,'executive'),(355255,'exemplar'),(107374,'exemplary'),(10956,'exempli'),(75300,'exemplify'),(105037,'exemption'),(19247,'exercise'),(2727,'exert'),(377530,'exertion'),(2830,'exhale'),(17588,'exhaled'),(17949,'exhaust'),(369905,'exhausted'),(117091,'exhaustive'),(27077,'exhibit'),(6484,'exist'),(6230,'existence'),(121413,'existent'),(325122,'existing'),(32317,'exit'),(3632,'exocy'),(304082,'exocyclic'),(5831,'exogenous'),(23797,'exogenously'),(4054,'exon'),(364097,'exonspcr'),(13096,'exonuclease'),(4669,'exopolysaccharide'),(370594,'exorbitant'),(338189,'exotic'),(375407,'exp'),(11885,'expand'),(5617,'expansion'),(340303,'ExPEC'),(334,'expect'),(14575,'expectancy'),(45744,'expectation'),(22231,'expected'),(94236,'expectoration'),(10232,'expedition'),(100135,'expenditure'),(20913,'expense'),(10228,'expensive'),(10608,'experience'),(66653,'experienced'),(5521,'experiment'),(8286,'experimental'),(27445,'experimentally'),(54178,'experimentation'),(139363,'experimenter'),(66338,'expert'),(107206,'expertise'),(2837,'expiratory'),(110396,'expire'),(110989,'Expired'),(11036,'explain'),(786,'explanation'),(121849,'explanatory'),(6001,'explant'),(8974,'explante'),(37403,'explicit'),(37441,'explicitly'),(21165,'exploit'),(20449,'exploitation'),(10830,'exploration'),(2162,'explore'),(17523,'explosion'),(42808,'explosive'),(24685,'exponential'),(17565,'exponentially'),(66679,'export'),(5280,'expose'),(691,'exposure'),(49319,'expre'),(2032,'express'),(2066,'expression'),(354521,'expressional'),(48325,'expressly'),(306732,'exquisite'),(26500,'exquisitely'),(69771,'exsite'),(75913,'extant'),(5008,'extend'),(90310,'extended'),(140589,'extender'),(90876,'extensibility'),(21766,'eXtensible'),(11352,'extension'),(2118,'extensive'),(5774,'extensively'),(2011,'extent'),(335770,'exterior'),(21810,'external'),(137199,'externally'),(317296,'extinct'),(38331,'extinction'),(19883,'extra'),(109178,'extracapsular'),(50802,'extracel'),(8698,'extracellular'),(16240,'extracorporeal'),(3773,'extract'),(14342,'extraction'),(329933,'extraneous'),(96083,'extraordinarily'),(308139,'extraordinary'),(37175,'extrapolate'),(373437,'extrapolation'),(3666,'extrapulmonary'),(36592,'extravascular'),(16318,'extreme'),(13202,'extremely'),(59647,'extremity'),(57144,'extrinsic'),(2371,'exuberant'),(751,'exudation'),(312113,'exudative'),(122347,'exude'),(743,'eye'),(139489,'eyeball'),(361584,'EYFP'),(102387,'fab'),(95237,'Fabbri'),(115448,'fabricate'),(95236,'Fabrizio'),(373798,'fabulously'),(135460,'fac'),(16423,'face'),(335403,'facial'),(305099,'facile'),(14090,'facilitate'),(47814,'facilitated'),(312580,'facilitation'),(352009,'facilitator'),(70,'facility'),(380742,'FACS'),(104697,'FACSCaliber'),(360561,'FACSCalibur'),(121067,'facsimile'),(56565,'FACSVantage'),(2630,'fact'),(2133,'factor'),(52762,'Factorial'),(375892,'factorisation'),(148146,'factorrecruitment'),(66712,'Factors'),(33567,'factory'),(49563,'facultative'),(321698,'Faculty'),(316593,'fade'),(46190,'faecalis'),(343624,'Fahlgren'),(7720,'fail'),(75274,'failing'),(395,'failure'),(71213,'faint'),(137890,'fair'),(68688,'fairly'),(307534,'fairness'),(2192,'faithfully'),(365654,'falciparum'),(679,'fall'),(313202,'fallow'),(23441,'false'),(90034,'falsely'),(122725,'falsenegative'),(113677,'FALZ'),(114313,'FAM'),(44728,'familial'),(17337,'familiar'),(17241,'familiarity'),(139258,'familiarize'),(664,'family'),(95324,'famine'),(20196,'far'),(26235,'Farabaugh'),(363186,'Farber'),(72385,'fare'),(133852,'farm'),(95676,'farmer'),(47914,'farming'),(95667,'farmland'),(122921,'Fartoukh'),(8088,'Fas'),(319099,'fascicularis'),(6524,'fashion'),(359596,'FasL'),(366007,'Faso'),(21943,'fast'),(53345,'FASTA'),(45986,'fastidiosa'),(44,'fastidious'),(363566,'FastStart'),(35354,'fatal'),(73168,'fatality'),(3480,'fate'),(114394,'FatiGo'),(75528,'fatty'),(362289,'Fau'),(21470,'fault'),(5354,'favor'),(29194,'favorable'),(305022,'favorably'),(355689,'favorite'),(15488,'favour'),(73412,'favourable'),(341535,'favoured'),(7369,'fawn'),(61728,'fax'),(43355,'FBS'),(103573,'Fc'),(52765,'FCA'),(373288,'FcaBP'),(56563,'FCM'),(344026,'FCS'),(134056,'FDR'),(18273,'fe'),(20584,'fear'),(23531,'feasibility'),(2504,'feasible'),(332901,'feather'),(88,'feature'),(357210,'featured'),(858,'febrile'),(72238,'February'),(32,'fection'),(47985,'Federal'),(106276,'federally'),(48810,'federation'),(49985,'FedF'),(21902,'feed'),(68924,'feedback'),(17252,'feel'),(352954,'feeling'),(45281,'FeII'),(318841,'Feldmann'),(316692,'fellow'),(351929,'FEMA'),(96972,'female'),(326378,'fentanyl'),(75071,'ferential'),(136666,'Ferguson'),(108950,'ferment'),(232,'fermentation'),(49181,'fermentor'),(17627,'ferritin'),(53691,'Fertil'),(97480,'fertilization'),(97476,'fertilize'),(5999,'fetal'),(6864,'FEV'),(843,'fever'),(37990,'few'),(364972,'FFPE'),(132754,'fgf'),(303033,'FGFR'),(113692,'FGR'),(95057,'fi'),(331572,'FIB'),(57739,'fiber'),(19922,'fibrin'),(110406,'fibrinogen'),(8519,'fibroblast'),(8536,'fibronectin'),(310897,'fibroproliferative'),(392,'fibrosi'),(2361,'fibrosis'),(8554,'fibrotic'),(331546,'FIBs'),(367634,'Ficoll'),(147570,'fide'),(69488,'fidelity'),(16538,'field'),(31,'fifth'),(82246,'fiftyfive'),(3827,'Fig'),(16915,'fight'),(109947,'Figs'),(11020,'figure'),(125498,'Figures'),(58135,'filament'),(17454,'file'),(43155,'fill'),(10297,'film'),(319727,'filovirus'),(21890,'filter'),(64362,'filtering'),(122241,'filtration'),(90621,'FIMM'),(8452,'final'),(8541,'finally'),(16907,'finance'),(73560,'financial'),(598,'find'),(960,'finding'),(149398,'FINDMAP'),(55098,'fine'),(301020,'finely'),(367065,'finger'),(74927,'Fingerprint'),(61276,'finish'),(87134,'finished'),(87126,'finishing'),(38325,'finite'),(16448,'Fink'),(145630,'Finland'),(32322,'Finlay'),(145627,'Finnzymes'),(310716,'fio'),(97578,'FIP'),(105540,'fire'),(62161,'firefly'),(22014,'firewall'),(319998,'firm'),(16064,'firmly'),(75513,'firstly'),(73893,'fiscal'),(34347,'fish'),(330,'Fisher'),(97062,'fishery'),(10231,'fishing'),(311531,'fistula'),(48676,'fit'),(30134,'fitc'),(60836,'Fitchburg'),(60019,'fitness'),(349358,'fitting'),(14975,'Fitzgerald'),(4991,'fix'),(36516,'fixation'),(128912,'fixative'),(34893,'fixed'),(363175,'fixedparaffin'),(124560,'flaccid'),(27547,'Flag'),(57137,'flagellar'),(328977,'flagging'),(27604,'Flagtagged'),(128360,'Flanagan'),(36508,'flange'),(43488,'flank'),(109348,'flask'),(36313,'flat'),(66619,'flatness'),(341239,'flattening'),(102413,'flavivirus'),(135144,'flavoprotein'),(17295,'flaw'),(353426,'flawed'),(29841,'fleme'),(136619,'Flex'),(3980,'flexibility'),(35836,'flexible'),(45674,'Flexiviridae'),(53228,'flexneri'),(316511,'flight'),(317823,'flightbase'),(317758,'flightbased'),(367606,'floatation'),(345816,'floater'),(332344,'flock'),(48207,'flooding'),(58194,'floor'),(74382,'floppy'),(124185,'flora'),(145355,'Florence'),(6255,'Floros'),(337963,'Flottum'),(306742,'Flourescence'),(14937,'flow'),(112821,'flowchart'),(105849,'flu'),(69841,'Fluc'),(345712,'fluctuant'),(55254,'fluctuate'),(324582,'fluctuating'),(52151,'fluctuation'),(346235,'flucytosine'),(368453,'FluGenome'),(3750,'fluid'),(60764,'Fluka'),(102716,'Fluor'),(98127,'fluore'),(54602,'fluorescein'),(56847,'fluorescence'),(10679,'fluorescent'),(97533,'fluorescently'),(64576,'fluoride'),(83553,'fluorogenic'),(340031,'fluorography'),(100083,'fluorometry'),(77773,'fluoroorotic'),(134257,'fluorophore'),(77791,'fluorouridylate'),(129260,'flux'),(335868,'fly'),(314768,'FMD'),(305725,'FMOC'),(5614,'foamy'),(37662,'focal'),(303874,'Fock'),(2159,'focus'),(21316,'focused'),(313024,'focusse'),(314213,'focussing'),(344023,'Foetal'),(3950,'fold'),(131400,'foldback'),(47189,'folded'),(66682,'folder'),(47187,'folding'),(133549,'follicle'),(133572,'follicular'),(345018,'Folliculitis'),(307,'follow'),(34798,'following'),(374718,'followsa'),(374937,'followsthen'),(354305,'followup'),(74794,'font'),(135861,'fontinali'),(23808,'Fonzi'),(23497,'food'),(49121,'foodstuff'),(124454,'foot'),(348649,'footandmouth'),(118804,'footnote'),(104479,'footpad'),(342707,'footprint'),(335429,'forbid'),(2836,'force'),(6861,'forced'),(100265,'forcefully'),(123987,'fore'),(324875,'forebrain'),(23579,'forecast'),(111973,'forecasting'),(356597,'foregoing'),(327439,'foreground'),(52371,'foreign'),(48907,'foremost'),(87872,'forensic'),(95209,'foresee'),(324956,'foreseeable'),(95671,'forest'),(97061,'forestry'),(308109,'forethought'),(148781,'fork'),(6207,'forkhead'),(2490,'form'),(9539,'formal'),(4990,'formaldehyde'),(363174,'formalin'),(303778,'formalism'),(310174,'formalize'),(13400,'formally'),(66683,'format'),(2106,'formation'),(368906,'formatstep'),(60732,'formula'),(375012,'formulae'),(101224,'formulate'),(121012,'formulating'),(16723,'formulation'),(301361,'fornix'),(133525,'Forschungsgemeinschaft'),(104267,'Fort'),(62040,'forthcoming'),(18001,'fortieth'),(136786,'FORTRAN'),(108056,'fortuitous'),(23068,'fortunate'),(116511,'fortunately'),(74277,'forum'),(11035,'forward'),(112618,'forwards'),(6137,'fos'),(19420,'foster'),(107584,'foundation'),(107615,'foundational'),(347688,'founder'),(100277,'founding'),(301874,'fourfold'),(61424,'fourmarker'),(91792,'fourteen'),(886,'fourth'),(46594,'Fovea'),(46147,'Foveavirus'),(332316,'fowl'),(340662,'Foxwell'),(329929,'FP'),(366117,'fplc'),(2942,'fraction'),(17010,'fractional'),(79323,'fractionate'),(4190,'fractionation'),(15191,'fracture'),(75656,'fragile'),(16770,'fragment'),(9512,'fragmentation'),(72169,'fragmented'),(364136,'fragmentsTdT'),(94146,'frail'),(10377,'frame'),(144979,'FrameFuc'),(24136,'frameshift'),(23710,'frameshifte'),(145143,'frameshifted'),(23584,'frameshifting'),(52756,'framework'),(74084,'framing'),(195,'France'),(38396,'Francisco'),(109377,'Franklin'),(332817,'Fraser'),(308325,'fraught'),(78474,'fre'),(23986,'Fredlund'),(2540,'free'),(62095,'freedom'),(62688,'freedomWe'),(61823,'freely'),(23115,'freetext'),(26858,'freeze'),(109238,'freezer'),(50117,'Freitas'),(133835,'french'),(6628,'frequency'),(11,'frequent'),(497,'frequently'),(14994,'fresh'),(140740,'freshly'),(92932,'freshwater'),(27061,'Fridell'),(107523,'friend'),(136836,'friendly'),(351543,'Friendship'),(134625,'frog'),(106098,'fromcarroll'),(20532,'frontier'),(106617,'frontline'),(93064,'Frost'),(75900,'frozen'),(335867,'fruit'),(54097,'fruitful'),(373867,'fruitfully'),(353612,'frustrated'),(218,'fry'),(303786,'fs'),(140810,'FTA'),(50737,'FtsH'),(34823,'Fu'),(362287,'FUBI'),(144852,'fuc'),(25927,'fuckeliana'),(147406,'fucose'),(144760,'fucosidase'),(146093,'fucoside'),(147415,'fucosylate'),(369683,'fuel'),(76906,'Fuji'),(148194,'Fujii'),(11332,'ful'),(57885,'fulfil'),(82449,'fulfill'),(309736,'full'),(303603,'Fuller'),(11200,'fulllength'),(47091,'fully'),(339876,'fumigatus'),(2933,'function'),(2574,'functional'),(47767,'functionality'),(9585,'functionally'),(23871,'functioning'),(72082,'fund'),(54251,'fundamental'),(11014,'fundamentally'),(23071,'funding'),(4771,'fungal'),(336462,'fungi'),(109301,'fungizone'),(4787,'fungus'),(367586,'FUP'),(101745,'Fur'),(148728,'furin'),(85068,'Furlong'),(324748,'furosemide'),(2750,'furthermore'),(27270,'Furuya'),(8942,'fuse'),(9702,'fusion'),(357834,'fusogenic'),(77566,'futhermore'),(2686,'future'),(110996,'FVC'),(93561,'FW'),(145167,'fwd'),(348098,'fwhm'),(36463,'Ga'),(64529,'GAA'),(324978,'GABA'),(326225,'gabapentin'),(330291,'GABR'),(330358,'GABRD'),(330359,'GABRE'),(330462,'GABRps'),(330261,'GABRs'),(363422,'GACCACGCGTATCGATGTCG'),(86861,'GACUAUACG'),(34185,'Gag'),(11283,'gain'),(305915,'Gaithersburg'),(69968,'galactose'),(24829,'galactosidase'),(32859,'galactosyltransferase'),(320308,'Galanthus'),(336991,'galectin'),(78230,'Gallant'),(108991,'gallate'),(350555,'Galloway'),(43446,'Galveston'),(336067,'gambiae'),(51505,'Game'),(317469,'gamma'),(314485,'Gammadistribution'),(339350,'Gammaproteobacteria'),(345935,'gancyclovir'),(6596,'ganglia'),(119812,'Gani'),(100363,'Gansu'),(43154,'gap'),(92415,'GAPDH'),(343756,'gapless'),(52816,'Garcia'),(301190,'Gardinier'),(3427,'gas'),(1981,'gaseous'),(120321,'Gastmeier'),(3780,'gastric'),(124129,'gastro'),(122464,'gastroesophageal'),(22138,'gastrointestinal'),(124077,'gastrooesophageal'),(80835,'GATC'),(6464,'gate'),(51534,'gather'),(335352,'gathering'),(36182,'Gatransferrin'),(66314,'gauge'),(137365,'gaussian'),(51582,'Gb'),(114120,'GBA'),(133319,'GBM'),(85880,'GC'),(125150,'GCICCICAYTGITGICCRAA'),(76275,'GCU'),(357090,'GDS'),(132678,'ge'),(72054,'gear'),(95695,'GEFood'),(10294,'gel'),(367609,'gelatin'),(332666,'Gelderse'),(141682,'Gemini'),(149303,'gen'),(10354,'GenBank'),(105732,'gency'),(21394,'gender'),(59,'gene'),(94458,'GeneAmp'),(357157,'GeneBank'),(19944,'genedelete'),(340576,'Genedoc'),(129271,'geneencoding'),(355480,'GeneID'),(145354,'Genenco'),(127794,'geneorder'),(114204,'GenePix'),(2168,'general'),(318221,'generalisation'),(374902,'generalise'),(373662,'generalised'),(313108,'generalism'),(312377,'generalist'),(144473,'generality'),(354252,'generalizable'),(3728,'generalization'),(6376,'generalize'),(137441,'generalized'),(2191,'generally'),(2004,'generate'),(333143,'generating'),(2454,'generation'),(72370,'generic'),(69721,'generous'),(88528,'generously'),(133447,'genesilencing'),(141658,'Genesis'),(133058,'genetargete'),(6231,'genetic'),(19133,'genetically'),(29772,'Geneticin'),(56716,'Genetics'),(344726,'genic'),(114026,'Genisphere'),(345015,'genital'),(125424,'genogroup'),(10225,'genome'),(11670,'genomes'),(127222,'genomesize'),(6226,'genomic'),(30987,'Genomics'),(92633,'genorm'),(99807,'Genosis'),(10578,'Genosys'),(35509,'genotype'),(61414,'genotypic'),(307068,'genotyping'),(79226,'Genset'),(54182,'GENSTYLE'),(320267,'gentamicin'),(109300,'gentamycin'),(134259,'gentec'),(128936,'gentian'),(109275,'gently'),(140791,'Gentra'),(333003,'genuine'),(52570,'genus'),(355081,'GEO'),(50040,'Geobacillus'),(150590,'geodesic'),(22018,'geographic'),(22304,'geographical'),(334482,'geographically'),(74341,'Geographics'),(75517,'geography'),(54755,'geometric'),(334265,'geometrically'),(304006,'geometry'),(366920,'George'),(352222,'Georgia'),(56387,'germ'),(74913,'german'),(35253,'Germany'),(133902,'germinal'),(105288,'germline'),(89749,'gestalt'),(572,'gestation'),(23701,'Gesteland'),(52952,'get'),(350308,'gets'),(102147,'getting'),(57075,'GFP'),(64452,'GFPuv'),(64530,'GGC'),(330701,'GGCGGCGTGCACACACACGC'),(355890,'GGI'),(356173,'GGIs'),(64540,'GGT'),(62144,'GGU'),(75225,'Ghana'),(35520,'Ghedin'),(22767,'GI'),(8944,'giant'),(330752,'gibb'),(357749,'Gibbins'),(55581,'Gibbs'),(29869,'Gibco'),(367590,'GibcoBRL'),(28782,'Gibson'),(74354,'GIDEON'),(46432,'GIDP'),(58008,'Giemsa'),(29792,'gift'),(79557,'Gilham'),(92787,'gill'),(338290,'Gilliam'),(22021,'GIS'),(2696,'give'),(333085,'Givenĥ'),(86115,'GL'),(3643,'gland'),(3719,'glandular'),(10674,'glass'),(362555,'glassbead'),(60706,'glasshouse'),(125825,'glean'),(109172,'Gleason'),(85581,'Glen'),(303211,'glial'),(109890,'glioblastoma'),(135248,'glioma'),(77257,'Gln'),(16407,'global'),(47981,'globalization'),(95375,'globalize'),(74748,'globally'),(74351,'globe'),(50276,'globular'),(102328,'globulin'),(120908,'glossary'),(43564,'Gloucester'),(123337,'glove'),(77261,'Glu'),(6004,'glucocorticoid'),(231,'glucose'),(146109,'glucoside'),(4397,'glucosyl'),(4388,'glucosylceramide'),(77259,'glutamate'),(366221,'glutamic'),(105274,'glutamine'),(77281,'glutamyl'),(3212,'glutaredoxin'),(19054,'glutathione'),(4006,'Gly'),(30313,'glycerol'),(26996,'glycine'),(29374,'glycinerich'),(3936,'glycoconjugate'),(3904,'glycolipid'),(136262,'glycolylneuraminic'),(92444,'glycolysis'),(4345,'glycoprotein'),(146930,'glycosidase'),(4241,'glycoside'),(304375,'glycosidic'),(43129,'glycosylase'),(4335,'glycosylate'),(336581,'glycosylated'),(4252,'glycosylation'),(305390,'glycosylic'),(28762,'glyoxal'),(28500,'glyoxalate'),(72449,'GM'),(145609,'GmbH'),(18187,'GMP'),(113980,'GMRCL'),(47947,'GNU'),(42853,'go'),(14438,'goal'),(29825,'goat'),(322458,'goblet'),(13223,'Godeny'),(30716,'Gold'),(29823,'Golden'),(32435,'Golgi'),(324898,'Golinger'),(133601,'gonadotropin'),(102114,'GONGOs'),(338676,'GONNET'),(340779,'gonorrhoeae'),(2962,'good'),(72416,'Goods'),(81157,'got'),(340902,'Gotschlich'),(366992,'Goundry'),(48835,'govern'),(48289,'governance'),(47988,'government'),(100211,'governmental'),(123394,'gown'),(73879,'GP'),(146986,'gpG'),(146987,'gpgt'),(340820,'GPI'),(128390,'GPM'),(73793,'gps'),(864,'grade'),(30320,'gradient'),(53175,'grading'),(24674,'gradual'),(5601,'gradually'),(99966,'graduated'),(66756,'graduation'),(7620,'graft'),(15092,'grafting'),(95686,'grain'),(145,'Gram'),(4582,'gramnegative'),(323106,'grampositive'),(29871,'Grand'),(23070,'grant'),(215,'granular'),(75069,'granularity'),(3371,'granule'),(10027,'granulocyte'),(10037,'granulocytic'),(345004,'granulomatous'),(45975,'grapevine'),(16031,'graph'),(75056,'graphic'),(22103,'graphical'),(55742,'graphically'),(93976,'GraphPad'),(74882,'Graphs'),(49110,'GRAS'),(95599,'grassroots'),(308228,'grateful'),(90610,'gratefully'),(47829,'gration'),(337058,'Graveley'),(344995,'Graves'),(51520,'gray'),(5349,'great'),(101551,'greater'),(33734,'greatly'),(30139,'green'),(357789,'Greene'),(338147,'Gregory'),(83547,'grey'),(327500,'grid'),(141766,'grip'),(339878,'grisea'),(100564,'grocery'),(50221,'GroEL'),(47948,'groff'),(359719,'Gromer'),(322374,'grommet'),(335638,'Grooming'),(114319,'groove'),(123490,'gross'),(12747,'ground'),(95859,'grounding'),(762,'group'),(59449,'grouping'),(301159,'Grove'),(46,'grow'),(4430,'growth'),(336990,'grp'),(132514,'Grzelinski'),(19055,'GSH'),(105195,'GSP'),(20192,'GSSG'),(24392,'GST'),(26547,'GTAGCTCATGATG'),(26545,'GTATAATTTTAC'),(331891,'GTGCACACACACGCCCACCGCGGCTCGGG'),(43240,'GTP'),(86666,'GU'),(100365,'Guangdong'),(100355,'Guangxi'),(104313,'guanidine'),(105171,'guanidinium'),(75848,'guanosine'),(18068,'guanylate'),(3090,'guanylyl'),(46431,'guanylylimidodiphosphate'),(39715,'guarantee'),(148742,'guard'),(75211,'guess'),(88555,'guidance'),(37269,'guide'),(58067,'guideline'),(737,'Guillain'),(116610,'Guillian'),(20348,'guinea'),(366161,'guinean'),(100362,'Guizhou'),(124479,'gum'),(113832,'Gung'),(14051,'Guo'),(109910,'Gupta'),(24315,'guration'),(67001,'GUSB'),(124010,'gut'),(140762,'Guthrie'),(133952,'GVBD'),(343193,'GxLxS'),(140707,'gynecologist'),(76092,'GϩC'),(5675,'ha'),(344447,'HAART'),(13704,'Haasnoot'),(18579,'habit'),(95673,'habitat'),(96896,'habitual'),(58195,'habituate'),(57208,'haemagglutinin'),(122203,'haematogenous'),(344003,'haemin'),(312725,'haemocoel'),(336567,'haemocyte'),(15850,'haemodynamically'),(14403,'haemoglobin'),(312729,'haemolymph'),(14491,'haemolytic'),(25486,'Haemonchus'),(5807,'Haemophilus'),(121391,'haemorrhagic'),(105401,'Haenszel'),(58132,'hair'),(13914,'hairpin'),(814,'half'),(108836,'hall'),(7425,'hallmark'),(37394,'Halloran'),(20742,'hallway'),(324715,'haloperidol'),(326161,'Haloperiodol'),(39330,'halt'),(59923,'halving'),(93048,'Hamalainen'),(43389,'Hamburg'),(27097,'Hamilton'),(340845,'Hammarstrom'),(50405,'hamper'),(5309,'hamster'),(30895,'HANCE'),(12061,'hand'),(317443,'handful'),(64979,'handheld'),(48114,'handle'),(66494,'handling'),(123407,'handrub'),(123356,'handwashing'),(100035,'Handy'),(43690,'Haneline'),(48809,'hangover'),(360345,'Hansen'),(96693,'Hanshin'),(57128,'haploid'),(337574,'haplotype'),(47624,'happen'),(4861,'haptotactic'),(136652,'Haraguchi'),(355349,'Harbig'),(58033,'Harbor'),(71265,'harbour'),(22675,'hard'),(52146,'hardly'),(21960,'hardware'),(105769,'Harford'),(301031,'Harlan'),(14602,'harm'),(2091,'harmful'),(36911,'harmless'),(120347,'harmonization'),(91836,'harmony'),(357730,'harsh'),(303873,'Hartree'),(104458,'Harvard'),(26777,'harvest'),(318555,'hasten'),(349578,'Hastings'),(150580,'Hat'),(46459,'Hataya'),(366918,'Hatley'),(340759,'Hauck'),(29877,'Hausser'),(23573,'have'),(79080,'haveIn'),(17224,'having'),(52976,'Hayes'),(26585,'Haynes'),(39514,'hazard'),(20087,'hazardous'),(18435,'Hb'),(338525,'HBGP'),(343991,'HBHI'),(305824,'HBr'),(128814,'HBSS'),(97822,'HBV'),(47335,'Hc'),(113093,'HCC'),(26810,'HCl'),(67617,'HClevels'),(107329,'hcp'),(107319,'HCPs'),(35589,'HCV'),(107194,'hcw'),(107183,'HCWs'),(48311,'head'),(37645,'headache'),(110491,'heading'),(72237,'headline'),(363624,'headquarters'),(75288,'heal'),(323342,'healing'),(8819,'health'),(105484,'healthcare'),(106477,'healthproportion'),(17607,'healthy'),(17342,'hear'),(322897,'hearing'),(394,'heart'),(17699,'heat'),(343988,'heated'),(97524,'heater'),(21171,'Heath'),(360456,'heatinactivated'),(62409,'heating'),(9819,'heavily'),(17694,'heavy'),(15368,'Hebert'),(63707,'heelprint'),(36504,'height'),(47983,'heighten'),(106937,'heightened'),(85294,'Heights'),(34546,'Heinlein'),(24010,'Heiskala'),(31003,'HEK'),(45631,'HEL'),(35128,'HeLa'),(55907,'helical'),(61844,'helicase'),(147382,'Helicobacter'),(4119,'helix'),(27361,'Hellen'),(23647,'Heller'),(17350,'helmet'),(2693,'help'),(12599,'helper'),(24165,'helpful'),(321324,'helping'),(96759,'Helsinki'),(29876,'hemacytometer'),(7027,'hemagglutinate'),(4805,'hemagglutination'),(367993,'hemagglutinin'),(367702,'hematocrit'),(709,'hematologic'),(311396,'hematological'),(371353,'hematophagous'),(326457,'hematopoietic'),(320514,'hematoxylin'),(58272,'hematoxyline'),(2613,'heme'),(11731,'Hemenway'),(19829,'hemin'),(125539,'hemisphere'),(109288,'hemocytometer'),(111064,'hemodynamic'),(15861,'hemoglobin'),(711,'hemolytic'),(379412,'hemophagocytic'),(324293,'Hemophilus'),(18212,'hemoprotein'),(752,'hemorrhage'),(22142,'hemorrhagic'),(335624,'Hempel'),(27103,'Henics'),(85178,'Henry'),(59451,'hepadnavirus'),(16628,'heparin'),(367034,'heparinized'),(6215,'hepatic'),(14528,'hepatitis'),(113797,'hepatocarcinogenesis'),(112570,'hepatocellular'),(112589,'hepatoma'),(19536,'hepatopulmonary'),(56542,'hepe'),(104343,'Hepes'),(366536,'heptad'),(69315,'heptamer'),(144928,'heptameric'),(142824,'heptanucleotide'),(4499,'heptose'),(352513,'herald'),(322432,'herbal'),(95674,'herbicide'),(43546,'Hercules'),(138261,'herd'),(44723,'hereditary'),(74321,'heretofore'),(375000,'heritability'),(140657,'heritage'),(107399,'heroic'),(100344,'heroin'),(438,'herpe'),(73299,'Herpes'),(35595,'herpesvirus'),(36942,'herpesviruse'),(141025,'herre'),(16754,'hetereogeneity'),(80208,'hetero'),(132815,'heterodimer'),(50619,'heterodimeric'),(64831,'Heteroduplex'),(55496,'heteroduplexe'),(65411,'heteroduplexes'),(82679,'heteroduplexmediate'),(4277,'heterogeneity'),(4536,'heterogeneous'),(25343,'heterologous'),(330159,'heteromeric'),(80099,'heteromolecular'),(101141,'heterosexual'),(148771,'heterotrimeric'),(312504,'heterozygote'),(6737,'heterozygous'),(22281,'heuristic'),(54644,'Hewlett'),(358234,'hexahistidine'),(94367,'hexamer'),(138485,'Heyerdahl'),(123080,'Heyland'),(124457,'HFMD'),(34166,'HFV'),(354646,'HG'),(352302,'HHS'),(340648,'hi'),(52758,'Hidden'),(23384,'hide'),(21773,'hierarchical'),(90856,'hierarchy'),(863,'high'),(67055,'highcapacity'),(131380,'higher'),(123465,'highlevel'),(10648,'highlight'),(19,'highly'),(332329,'highlypathogenic'),(347319,'highlyspecific'),(87131,'highquality'),(326034,'highresolution'),(42642,'highrisk'),(102687,'highthroughput'),(22593,'highway'),(33870,'hijack'),(79102,'Hikiji'),(94474,'Hilden'),(340798,'Hill'),(78443,'Himeno'),(58857,'hind'),(31777,'hindbrain'),(2204,'hinder'),(148912,'hinderance'),(26560,'hindiii'),(58203,'hindpaw'),(107166,'hindrance'),(66046,'hint'),(15190,'hip'),(21604,'HIPAA'),(325442,'hipfracture'),(14122,'Hippel'),(325811,'hippocampal'),(325262,'hippocampus'),(29664,'Hirano'),(97897,'Hirayama'),(32463,'hirt'),(104732,'HiSpeed'),(6694,'histamine'),(70812,'histidine'),(113857,'histo'),(92127,'histocompatibility'),(67852,'histogram'),(58003,'histological'),(58532,'histology'),(320505,'HistoMark'),(35691,'histone'),(325812,'histopathologic'),(319671,'histopathological'),(38694,'historic'),(22247,'historical'),(15163,'historically'),(874,'history'),(30812,'hit'),(125183,'hitherto'),(14041,'HIV'),(345314,'hivnegative'),(67120,'HKG'),(90622,'HLA'),(20159,'HLADR'),(103466,'Hm'),(123565,'HME'),(52761,'hmm'),(47930,'hmmbuild'),(47931,'hmmcalibrate'),(46239,'HMMER'),(94066,'hmpv'),(43181,'hMTH'),(43177,'hMYH'),(26987,'hnRNP'),(29497,'hnrnps'),(43184,'hNTH'),(17637,'HO'),(305825,'HOAc'),(109156,'Hoag'),(342391,'Hobbs'),(332781,'hobby'),(133097,'Höbel'),(38258,'hoc'),(2789,'HOCl'),(20318,'HOderived'),(366503,'Hodges'),(359444,'Hodgkin'),(303604,'Hodgson'),(14182,'Hoechst'),(305989,'Hoff'),(335814,'Hoffmann'),(359681,'Hogg'),(96685,'Hokkaido'),(96690,'Hokuriku'),(13113,'hold'),(96167,'holder'),(94436,'Holding'),(363583,'hole'),(13473,'Holmes'),(147297,'holoenzyme'),(21395,'home'),(105451,'Homeland'),(97057,'homemaker'),(31763,'homeobox'),(322434,'homeopathy'),(24618,'homeostasis'),(335610,'homeostatic'),(24193,'homo'),(65055,'homoduplex'),(86332,'homoduplexe'),(133191,'homogenate'),(37509,'homogeneity'),(67046,'homogeneous'),(334434,'homogeneously'),(26780,'homogenization'),(103288,'homogenize'),(375162,'homogenous'),(82461,'homoheteromolecular'),(24229,'homolog'),(24186,'homologous'),(43176,'homologue'),(4065,'homology'),(80165,'homomolecular'),(55432,'homopolymer'),(55445,'homopolymeric'),(55433,'homopurine'),(55434,'homopyrimidine'),(147852,'homosexual'),(141562,'homozygous'),(73439,'hone'),(107295,'honest'),(335657,'honey'),(97894,'Hong'),(309781,'honour'),(7370,'hood'),(100579,'hook'),(352231,'Hoover'),(144572,'hop'),(979,'hope'),(105828,'Hopkins'),(69226,'hopping'),(336624,'Hopscotch'),(360363,'Horie'),(16911,'horizon'),(38309,'horizontal'),(53238,'horizontally'),(133589,'hormonal'),(135646,'hormone'),(102371,'horse'),(70004,'horseradish'),(351280,'Horsfall'),(29879,'Horsham'),(313935,'horticultural'),(65,'Hospital'),(118662,'hospitalacquire'),(122072,'hospitalisation'),(112059,'hospitalise'),(94125,'hospitalization'),(16278,'hospitalize'),(2078,'host'),(10724,'hostcell'),(335773,'hostile'),(313407,'hostpathogen'),(137775,'Hosts'),(313627,'hostselective'),(16237,'hot'),(121095,'hotline'),(259,'hour'),(29938,'hourly'),(42645,'house'),(38015,'household'),(50791,'housekeep'),(49400,'housekeeping'),(36499,'housing'),(35237,'Houthaeve'),(317442,'hover'),(74045,'Howard'),(310746,'HP'),(332331,'HPAI'),(334808,'HPD'),(85527,'HPLC'),(355873,'HPRD'),(301002,'Hprt'),(30896,'hr'),(310784,'hrct'),(17917,'HRE'),(85100,'hrlog'),(145077,'HRP'),(332838,'HRR'),(44071,'hrs'),(351882,'HRSA'),(351348,'hRSV'),(32408,'Hruby'),(145622,'HS'),(56722,'HSE'),(56809,'HSEI'),(56810,'HSEII'),(56724,'hsemapf'),(56725,'hsemapr'),(22712,'HSRC'),(148769,'HSV'),(51254,'ht'),(102272,'HTC'),(368606,'HTML'),(49395,'htra'),(356765,'Hu'),(29147,'Huang'),(138778,'hub'),(360310,'Hubinger'),(301088,'Hudson'),(71442,'huge'),(92496,'Huggett'),(60734,'Hull'),(335815,'Hultmark'),(305599,'Humaine'),(404,'human'),(105163,'humanization'),(102283,'humanize'),(105282,'Humanized'),(367536,'humid'),(122242,'humidification'),(43382,'humidified'),(122372,'humidifier'),(56805,'humidity'),(5886,'humoral'),(100752,'Hunan'),(35346,'hundred'),(18000,'hundredth'),(124581,'Hungary'),(31850,'huntingtin'),(31846,'Huntington'),(56717,'Huntsville'),(353275,'hurricane'),(75287,'hurt'),(342413,'HV'),(80405,'HVR'),(82192,'hvra'),(82050,'hvras'),(80498,'hvrs'),(45368,'HxD'),(347919,'HxNy'),(56793,'Hyb'),(145043,'Hybond'),(27046,'hybrid'),(3680,'hybridization'),(10641,'hybridize'),(11847,'hybridized'),(63371,'hybridizing'),(102299,'hybridoma'),(43356,'HyClone'),(380352,'HyCult'),(43261,'hydration'),(108978,'hydrocarbon'),(104314,'hydrochloride'),(131911,'hydrodynamic'),(17703,'hydrogen'),(305815,'hydrogenate'),(84871,'hydrolase'),(63788,'hydrolysis'),(361393,'hydrolytic'),(43232,'hydrolyze'),(361433,'hydrolyzed'),(4060,'hydrophilic'),(114842,'hydrophilicity'),(56916,'hydrophobic'),(56304,'hydrophobicity'),(140925,'hydroxide'),(43243,'hydroxy'),(145289,'hydroxycynnamic'),(42924,'hydroxyl'),(134797,'hydroxylase'),(4114,'hydroxylation'),(306239,'hydroxylprolinol'),(4023,'hydroxylysine'),(4239,'hydroxylysyl'),(4018,'hydroxyproline'),(305685,'hydroxyprolinol'),(324977,'hydroxytryptamine'),(123327,'hygiene'),(335654,'hygienic'),(123547,'hygroscopic'),(35051,'Hyman'),(337499,'hymenoptaecin'),(2288,'hyper'),(325039,'hyperactivity'),(58829,'hyperalgesia'),(38639,'Hypercube'),(367231,'HyperDH'),(324999,'hyperfunction'),(310928,'hyperglycemia'),(150121,'hyperlink'),(92918,'hyperosmotic'),(6264,'Hyperoxia'),(43298,'hyperoxiainduce'),(19793,'hyperoxiasensitive'),(16415,'hyperoxic'),(6082,'hyperplasia'),(7054,'hyperplastic'),(7190,'hyperpolarization'),(6761,'hyperreactivity'),(6743,'hyperresponsiveness'),(5938,'hypersensitivity'),(6931,'hypertension'),(7198,'hypertensive'),(146931,'hyperthermophile'),(146469,'hyperthermophilic'),(5611,'hypertrophy'),(80188,'hypervariable'),(19160,'hyperviscous'),(99813,'hyphen'),(325559,'hypoactive'),(124046,'hypoalbuminemia'),(2787,'hypochlorous'),(363635,'hypochromicity'),(325692,'hypoglycemia'),(325214,'hypoperfusion'),(283,'hypotension'),(14505,'hypothermia'),(18998,'hypothesis'),(333283,'hypothesise'),(29534,'hypothesize'),(5369,'hypothetical'),(329870,'hypothetically'),(311712,'hypovolemic'),(111641,'hypoxaemic'),(288,'hypoxemia'),(42879,'hypoxemic'),(6496,'hypoxia'),(6942,'hypoxic'),(23895,'hypusine'),(87850,'Hysom'),(141271,'Hyther'),(102264,'HZQ'),(62138,'IA'),(358828,'IAA'),(58065,'IACUC'),(86101,'iAL'),(324603,'iatrogenic'),(4839,'IAV'),(348057,'Ibis'),(63462,'IBV'),(67190,'IC'),(26874,'ice'),(326659,'icewater'),(33611,'Ichihashi'),(140567,'ics'),(95323,'ict'),(15011,'ICU'),(338870,'Icycler'),(328775,'ID'),(363414,'Idaho'),(23295,'idea'),(92299,'ideal'),(374438,'idealised'),(53491,'ideally'),(24174,'identi'),(7037,'identical'),(137227,'identically'),(95745,'identifi'),(21452,'identifiable'),(241,'identification'),(21878,'identifie'),(22803,'identifier'),(111,'identify'),(10392,'identity'),(8570,'idiopathic'),(85840,'idiosyncraticthe'),(62136,'IDT'),(100144,'IDU'),(7423,'ie'),(90330,'IEDB'),(32385,'IEV'),(366190,'IFAT'),(6501,'IFN'),(379493,'IFNc'),(371675,'IFNγ'),(120449,'IfSG'),(27205,'IG'),(323205,'IgA'),(5918,'IgE'),(153,'IgG'),(319230,'IgGl'),(86103,'iGL'),(154,'IgM'),(37184,'ignorance'),(14776,'ignore'),(340825,'igv'),(94030,'IH'),(3606,'II'),(16570,'III'),(316059,'ij'),(313710,'ik'),(339119,'Ikb'),(77724,'Ikemura'),(29797,'IL'),(356261,'ilar'),(341420,'Ile'),(14303,'ill'),(59634,'illegal'),(20675,'illequippe'),(73993,'illicit'),(80242,'Illinois'),(15249,'illness'),(324622,'illnessassociate'),(352105,'illuminate'),(98112,'illuminator'),(2565,'illustrate'),(51692,'illustration'),(38738,'illustrative'),(371585,'IM'),(13286,'image'),(143319,'ImageQuant'),(145536,'Imager'),(101181,'imaginable'),(72378,'imagination'),(117207,'imagine'),(76904,'imaging'),(18314,'imbalance'),(52518,'imbed'),(10690,'imbedded'),(335854,'Imd'),(360471,'IMDM'),(90681,'IMGT'),(64664,'imidazole'),(4015,'imino'),(135211,'immature'),(6774,'immediate'),(6321,'immediately'),(362714,'immersion'),(72775,'immigrant'),(140242,'immigration'),(2851,'imminent'),(341685,'immno'),(30844,'immobilised'),(64619,'immobilize'),(64356,'immobilized'),(2000,'immune'),(338363,'immunepathway'),(72282,'immunisation'),(753,'immunity'),(90477,'immunization'),(29806,'immunize'),(91272,'Immunized'),(49719,'immuno'),(114645,'immunoassay'),(27571,'immunoblot'),(83966,'immunoblotte'),(358763,'immunoblotting'),(104384,'Immunochemical'),(36063,'immunocompetent'),(71616,'immunocomplexe'),(400,'immunocompromise'),(34213,'immunode'),(405,'immunodeficiency'),(324973,'immunodeficiiency'),(36039,'immunodepressed'),(36922,'immunodepression'),(373074,'immunodominant'),(4545,'immunofluorescence'),(91274,'Immunogen'),(90708,'immunogenetic'),(90689,'ImMunoGeneTics'),(49578,'immunogenic'),(126308,'immunogenicity'),(85289,'immunogloblin'),(90697,'immunoglobulin'),(2839,'immunohistochemical'),(3742,'immunohistochemistry'),(90709,'immunoinformatic'),(302525,'immunolabele'),(301194,'immunolabeling'),(5115,'immunolocalize'),(322748,'immunologic'),(5933,'immunological'),(90626,'Immunology'),(114497,'immunome'),(114507,'immunomic'),(15590,'immunomodulation'),(148839,'immunomodulator'),(303345,'immunomodulatory'),(377971,'immunopathology'),(380458,'immunophenotype'),(31825,'immunophilin'),(28791,'immunoprecipitate'),(28819,'immunoprecipitation'),(145069,'ImmunoPure'),(367762,'immunopurified'),(140693,'immunoreactive'),(2821,'immunoreactivity'),(322839,'immunoregulation'),(321820,'immunoregulatory'),(301157,'Immunoresearch'),(313976,'immunosorbent'),(8352,'immunostaine'),(6907,'immunostaining'),(122739,'immunosuppresion'),(377469,'immunosuppression'),(7945,'immunosuppressive'),(33543,'immunouorescence'),(144489,'immutable'),(14819,'impact'),(6946,'impair'),(14366,'impaired'),(14935,'impairment'),(123852,'impairs'),(96353,'impede'),(36870,'impending'),(20836,'imperative'),(317848,'imperfect'),(355014,'imperfection'),(329745,'imperfectly'),(378441,'impinge'),(325233,'implausible'),(22082,'implement'),(21982,'implementation'),(2482,'implicate'),(7166,'implication'),(95120,'Implications'),(90657,'implicit'),(139527,'implicitly'),(12007,'imply'),(25888,'import'),(2679,'importance'),(2974,'important'),(3063,'importantly'),(38047,'importation'),(96169,'importing'),(60045,'impose'),(2519,'impossible'),(75629,'impoverished'),(114957,'impractical'),(110078,'impractically'),(354519,'imprecise'),(14439,'impress'),(60284,'impression'),(5762,'impressive'),(3425,'improve'),(7857,'improved'),(7503,'improvement'),(300684,'impulse'),(32381,'IMV'),(90879,'in'),(29405,'inability'),(138292,'inaccuracy'),(69582,'inaccurate'),(9059,'inactivate'),(9215,'inactivated'),(42966,'inactivation'),(38672,'inactive'),(96804,'inactivity'),(351115,'inadequacy'),(39284,'inadequate'),(353512,'inadvertent'),(376005,'inadvertently'),(16748,'inappropriate'),(54079,'inasmuch'),(324586,'inattention'),(39318,'inaugurate'),(64039,'inborn'),(377619,'inbred'),(43563,'Inc'),(106635,'incapable'),(16385,'Ince'),(351814,'incentive'),(16657,'incidence'),(374933,'incidenceif'),(37273,'incident'),(26,'include'),(31903,'inclusion'),(307850,'inclusive'),(310297,'inclusiveness'),(75635,'income'),(33597,'incoming'),(9569,'incompetent'),(2157,'incomplete'),(18050,'incompletely'),(18888,'inconclusive'),(317260,'inconsequential'),(2434,'inconsistency'),(2972,'inconsistent'),(101954,'inconvenience'),(114298,'incoporated'),(13379,'incorporate'),(43377,'Incorporated'),(28155,'incorporation'),(61247,'incorrect'),(90094,'incorrectly'),(2065,'increase'),(6257,'Increased'),(47975,'increasingly'),(334475,'incredi'),(85906,'increment'),(10153,'incremental'),(204,'incubate'),(687,'incubation'),(43383,'incubator'),(148987,'incur'),(43751,'indefinitely'),(51245,'indel'),(14721,'independence'),(2745,'independent'),(65946,'independently'),(20638,'index'),(354605,'Indexes'),(46950,'indi'),(47107,'India'),(100267,'indian'),(29762,'Indianapolis'),(2245,'indicate'),(26092,'indication'),(20872,'indicative'),(20412,'indicator'),(102105,'indifference'),(72564,'indigenous'),(2527,'indirect'),(4813,'indirectly'),(76078,'indispensable'),(351387,'indistinct'),(24534,'indistinguishable'),(2179,'individual'),(346407,'individualize'),(345021,'individualized'),(29781,'individually'),(375327,'individualthe'),(344088,'indolylphosphate'),(74466,'Indonesia'),(318492,'indoor'),(2036,'induce'),(50942,'induced'),(17709,'inducer'),(6031,'inducible'),(8229,'induction'),(332483,'industrialise'),(332520,'industrialised'),(120066,'industrialize'),(79168,'industrially'),(99939,'Industries'),(72984,'industry'),(33815,'inef'),(41304,'ineffective'),(107840,'ineffectual'),(342664,'inefficient'),(4598,'inefficiently'),(73029,'inequality'),(96308,'inequitably'),(95569,'inequity'),(326460,'inert'),(316308,'inevitable'),(308980,'inevitably'),(87560,'inexact'),(22680,'Inexpensive'),(119970,'inexperienced'),(34048,'inf'),(116296,'infancy'),(568,'infant'),(372552,'infantum'),(326557,'infarct'),(15208,'infarction'),(353639,'infeasible'),(8932,'infect'),(136664,'infectable'),(34,'infected'),(314325,'infectee'),(47533,'infecting'),(9,'infection'),(350560,'infectiona'),(122145,'Infections'),(2165,'infectious'),(37369,'infectiousness'),(39094,'infective'),(4804,'infectivity'),(314322,'infector'),(120448,'Infektionsschutzgesetz'),(59367,'infer'),(14716,'inference'),(99472,'inferior'),(355239,'inferiority'),(59865,'inferred'),(7737,'infiltrate'),(122742,'infiltration'),(314542,'infinite'),(316081,'infinitely'),(52883,'infinity'),(114563,'infirmity'),(95206,'infl'),(58975,'inflame'),(2637,'inflammation'),(1999,'inflammatory'),(59119,'inflammatoryproposed'),(316409,'inflight'),(139119,'influ'),(3015,'influence'),(106386,'influential'),(2258,'influenza'),(5808,'influenzae'),(334308,'InfluSim'),(6762,'influx'),(23159,'inform'),(90656,'informal'),(321270,'informant'),(74404,'Informatics'),(77,'information'),(142617,'informational'),(24119,'informative'),(36434,'informed'),(24723,'inframe'),(22206,'infrastructure'),(112266,'infrequent'),(78992,'infrequently'),(43699,'infuse'),(7499,'infusion'),(60052,'ing'),(301004,'ingestion'),(105550,'ingrain'),(319655,'inguinal'),(137243,'inhabit'),(97054,'inhabitant'),(3420,'inhalation'),(20809,'inhalational'),(6734,'inhaled'),(2236,'inherent'),(71789,'inherit'),(4773,'inhibit'),(102361,'inhibited'),(2222,'inhibition'),(2174,'inhibitor'),(5512,'inhibitory'),(140844,'inhouse'),(75445,'INIST'),(86274,'init'),(7298,'initial'),(141685,'initialization'),(4196,'initially'),(10943,'initiate'),(131628,'initiating'),(23634,'initiation'),(72275,'initiative'),(58098,'inject'),(36454,'injection'),(108177,'injure'),(2153,'injurious'),(2262,'injury'),(345008,'ink'),(100358,'inland'),(85713,'INN'),(149020,'innate'),(97576,'inner'),(58795,'innervation'),(37356,'innovative'),(10084,'inocula'),(36309,'inoculate'),(81073,'inoculated'),(59700,'inoculating'),(2257,'inoculation'),(9950,'inoculum'),(19552,'inos'),(4379,'inositol'),(138,'inpatient'),(21128,'input'),(328751,'inputte'),(48591,'inquest'),(133861,'INRA'),(47116,'insect'),(313160,'insectborne'),(109198,'insensitive'),(21887,'insert'),(24042,'insertion'),(115981,'inset'),(50205,'inside'),(2244,'insight'),(330866,'insignificant'),(63960,'insofar'),(98695,'insolubilization'),(99489,'insolubilize'),(5577,'insoluble'),(206,'inspect'),(25017,'inspection'),(123438,'inspiratory'),(17011,'inspire'),(123533,'inspired'),(2520,'instability'),(132452,'instable'),(21381,'InstaCare'),(23524,'instal'),(2251,'instance'),(145535,'Instant'),(95291,'instantaneous'),(129065,'instantaneously'),(307715,'instantiate'),(367481,'instantly'),(336651,'instar'),(2808,'instead'),(5804,'instill'),(5487,'instillation'),(22568,'Institute'),(17136,'Institutes'),(42286,'institution'),(58021,'Institutional'),(199,'instruction'),(108490,'instructively'),(107812,'instrument'),(16896,'instrumentation'),(99886,'Instruments'),(95326,'insuffi'),(7936,'insufficiency'),(14919,'insufficient'),(18969,'insufficiently'),(5717,'insulin'),(326582,'insult'),(95776,'insurability'),(21600,'Insurance'),(95382,'int'),(5904,'intact'),(110069,'intake'),(47828,'Inte'),(66915,'integer'),(107664,'integral'),(374919,'integralBy'),(320876,'integrally'),(374910,'integralwhich'),(23220,'integrate'),(117384,'integrated'),(21417,'integration'),(375879,'integrationwhere'),(9340,'integrin'),(21865,'integrity'),(95937,'intellectual'),(324636,'intelligence'),(23006,'intend'),(307800,'intended'),(8834,'intense'),(324311,'intensely'),(349234,'intensify'),(7143,'intensity'),(14837,'intensive'),(122104,'intensivist'),(120694,'intent'),(75350,'intention'),(37145,'intentional'),(21531,'intentionally'),(3541,'inter'),(4381,'interact'),(3078,'interaction'),(150432,'interactive'),(149490,'interactively'),(75941,'interbreed'),(304499,'intercalate'),(9336,'intercellular'),(36573,'intercept'),(4152,'interchain'),(141057,'interchange'),(129435,'interchangeable'),(70807,'intercistronic'),(100840,'interclade'),(138616,'interconnect'),(75286,'interconnectedness'),(338732,'intercontig'),(305100,'interconversion'),(137029,'intercourse'),(309441,'interdependence'),(95160,'interdisciplinary'),(115466,'interdistance'),(2698,'interest'),(96628,'interested'),(2347,'interesting'),(3751,'interestingly'),(21347,'interface'),(4115,'interfere'),(47304,'interference'),(9533,'interferon'),(74112,'Intergenerational'),(27204,'intergenic'),(49003,'intergovernmental'),(23077,'interim'),(85892,'interior'),(352400,'interject'),(48562,'interjurisdictional'),(110405,'interleukin'),(377345,'intermediary'),(30488,'intermediate'),(123878,'intermittent'),(11733,'intermolecular'),(11427,'internal'),(4735,'internalization'),(4949,'internalize'),(5197,'internalized'),(10947,'internally'),(192,'International'),(48886,'internationally'),(75504,'internet'),(22561,'Internetenabled'),(21304,'interoperability'),(125410,'interoutbreak'),(369049,'interpandemic'),(5941,'interplay'),(2590,'interpret'),(356686,'interpretable'),(2394,'interpretation'),(147763,'InterPro'),(329484,'interquartile'),(321056,'interrelated'),(54574,'interrogate'),(115564,'interrogative'),(13371,'interrupt'),(145868,'interrupted'),(4004,'interruption'),(149010,'intersect'),(72458,'intersection'),(72873,'intersectoral'),(16735,'INTERSEPT'),(338578,'interspecific'),(48702,'interstate'),(19314,'interstitial'),(342374,'interstrain'),(137870,'intertwine'),(10782,'interval'),(67188,'Intervals'),(45653,'intervene'),(16955,'intervention'),(148954,'interventional'),(96716,'interview'),(321488,'interviewee'),(96723,'interviewer'),(20925,'interviewing'),(73070,'interwoven'),(57714,'intestinal'),(56998,'intestine'),(7156,'intimal'),(340339,'intimate'),(19882,'intra'),(359824,'intraand'),(4121,'intracellular'),(345583,'intracellulare'),(84902,'intracellularly'),(3890,'intrachain'),(345236,'intracranial'),(133293,'intracranially'),(371874,'intradermal'),(52254,'intragenomic'),(50351,'intramolecular'),(14949,'intramucosal'),(372912,'intramuscular'),(320090,'intramuscularly'),(319413,'intramusculary'),(2256,'intranasal'),(345789,'intraocular'),(57616,'intraparenchymal'),(104507,'intraperitoneal'),(320976,'intraprovincial'),(55856,'intrasample'),(342368,'intraspecie'),(133326,'intrathecal'),(6059,'intratracheal'),(5803,'intratracheally'),(713,'intravascular'),(36461,'intravenous'),(104399,'intravenously'),(345886,'intravitreous'),(377320,'intricate'),(55375,'intrigue'),(9497,'intriguing'),(3429,'intriguingly'),(9827,'intrinsic'),(119556,'intrinsically'),(11812,'introduce'),(12247,'introduction'),(24380,'intron'),(108344,'intrusion'),(48475,'intrusive'),(122279,'intubate'),(36376,'intubation'),(350939,'intuition'),(121612,'intuitive'),(41163,'intuitively'),(60182,'invade'),(377034,'invaluable'),(51429,'invariant'),(59850,'invasion'),(3476,'invasive'),(136751,'invent'),(87838,'invention'),(86117,'inverse'),(2831,'inversely'),(51248,'inversion'),(26148,'invertebrate'),(80184,'inverted'),(90138,'invest'),(15171,'investigate'),(354965,'investigated'),(3038,'investigation'),(2529,'investigator'),(72066,'investment'),(72570,'Investments'),(17933,'invisible'),(307873,'invite'),(29716,'Invitrogen'),(144036,'invoke'),(695,'involve'),(118772,'involved'),(1986,'involvement'),(36492,'iodide'),(145259,'iodoacetamide'),(95383,'iom'),(2952,'ion'),(92914,'ionic'),(145756,'ionization'),(42935,'ionize'),(301192,'Iowa'),(132761,'ip'),(73095,'IPD'),(354702,'IPI'),(354606,'IPIs'),(104296,'IPTG'),(134230,'IQ'),(42937,'IR'),(100341,'Iran'),(21649,'IRB'),(344343,'IRD'),(121998,'Ireland'),(43396,'IRES'),(344361,'IRIS'),(3081,'iron'),(9567,'irradiate'),(117915,'irrational'),(323238,'irre'),(373406,'irrealistic'),(78533,'irregular'),(107841,'irrelevant'),(70492,'irrespective'),(58594,'irreversible'),(57689,'irritable'),(344347,'IRS'),(344976,'IRU'),(344972,'IRV'),(367249,'ISA'),(32460,'Isaacs'),(343590,'Isberg'),(332991,'isbut'),(15236,'ischaemic'),(2412,'ischemia'),(16021,'ischemic'),(351453,'Iscove'),(375388,'isgeneralise'),(360838,'ISGylation'),(312837,'ISH'),(125034,'Ishiko'),(375925,'isi'),(29872,'Island'),(72780,'Islander'),(348705,'Islands'),(3733,'islet'),(75679,'ism'),(140486,'iso'),(78773,'isoacceptor'),(140913,'isoc'),(56909,'isoelectric'),(2176,'isoenzyme'),(2007,'isoform'),(140912,'isog'),(140484,'isoguanosine'),(252,'isolate'),(4889,'isolated'),(14213,'isolation'),(357815,'isomerase'),(339785,'Isopeptidase'),(30497,'isopeptide'),(99469,'isopropanol'),(104294,'isopropyl'),(64556,'isopropylb'),(97442,'isothermal'),(97450,'isothermally'),(134652,'isotocin'),(116113,'isotype'),(312796,'ISP'),(104447,'Israel'),(2695,'issue'),(375332,'isthe'),(375675,'isthus'),(83206,'isvp'),(350317,'isWe'),(332953,'iswhere'),(17210,'italian'),(138986,'italic'),(16221,'Italy'),(21153,'item'),(64434,'iterate'),(46035,'iteration'),(307849,'iterative'),(38677,'iteratively'),(76913,'itevidence'),(129940,'ith'),(313954,'itinerary'),(113768,'ity'),(320179,'IU'),(60667,'IV'),(23745,'Ivanov'),(351508,'IVDDTPDDDI'),(30728,'ivec'),(360995,'IVT'),(97900,'Iwasaki'),(18275,'IXα'),(35746,'Iyer'),(76552,'ized'),(374725,'Iˆi'),(30568,'IκB'),(9117,'IκBα'),(58030,'Jackson'),(345740,'Jacobson'),(101238,'jail'),(336092,'JAK'),(120627,'Jakob'),(149557,'JalView'),(94432,'James'),(21527,'JAMIA'),(75,'January'),(335855,'Janus'),(7029,'Japan'),(96625,'japanese'),(25922,'japonicus'),(366967,'jasco'),(345269,'jaundice'),(21329,'Java'),(21707,'JavaBean'),(22084,'JavaBeans'),(368605,'JavaScript'),(22094,'Jboss'),(308477,'JCB'),(149104,'Jeang'),(77027,'jecte'),(67,'Jeddah'),(15682,'Jehovah'),(45303,'jelly'),(310200,'Jennings'),(90628,'JenPep'),(32409,'Jensen'),(27213,'Jeong'),(108285,'jeopardy'),(102265,'JES'),(150208,'Jesaitis'),(132581,'jetpei'),(123377,'jewellery'),(28478,'JHM'),(337190,'Jiang'),(100753,'Jiangxi'),(338645,'Jim'),(344322,'jirovecii'),(313831,'jk'),(310062,'JL'),(149614,'Jmol'),(125214,'JMP'),(335857,'JNK'),(100486,'job'),(326824,'John'),(105827,'Johns'),(32361,'Johnson'),(12852,'join'),(125131,'joining'),(74462,'joint'),(110273,'jointly'),(32969,'Joklik'),(301144,'Jolla'),(340761,'Jones'),(340984,'Jonge'),(12602,'Joo'),(357748,'Jordan'),(21975,'Jose'),(335725,'Josson'),(11214,'Journal'),(340580,'JPred'),(316091,'jr'),(22111,'JSP'),(86285,'jth'),(37122,'judge'),(15136,'judgement'),(105774,'July'),(117838,'jump'),(58235,'jumping'),(363240,'JumpStart'),(6138,'jun'),(11341,'junction'),(83174,'junctional'),(15506,'juncture'),(94557,'June'),(22413,'jurisdiction'),(72668,'jurisdictional'),(309507,'jurisdictionsIn'),(358652,'Jurkat'),(95078,'justice'),(95917,'justifi'),(309895,'justification'),(39123,'justified'),(14399,'justify'),(349070,'justwhere'),(92875,'juvenile'),(116951,'juxtapose'),(110066,'Kaegi'),(34105,'Kaelin'),(23705,'Kahana'),(34541,'Kalicharran'),(4693,'Kalina'),(326126,'Kalisvaart'),(337469,'Kambris'),(338138,'Kanbar'),(337168,'Kanost'),(96687,'Kanto'),(337407,'Kanzok'),(348411,'Kao'),(15458,'Kaplan'),(35596,'Kaposi'),(30566,'kappa'),(52944,'Kappas'),(117903,'Karl'),(325506,'Karlidag'),(34721,'Karsenti'),(354242,'Katrina'),(66,'KAUH'),(76109,'Kawaguchi'),(72257,'Kay'),(10442,'kb'),(17831,'kbp'),(55601,'kcal'),(98456,'KCI'),(30308,'KCl'),(56908,'kd'),(3792,'kda'),(13125,'Ke'),(348538,'keel'),(136655,'Keeling'),(21529,'keep'),(332596,'keeping'),(96688,'Keihin'),(4694,'Keisari'),(79556,'Keith'),(43450,'Kelley'),(326825,'Kellum'),(86120,'kelvin'),(338646,'Kent'),(6071,'Keratinocyte'),(136587,'Kermack'),(333778,'kernel'),(4910,'Kesari'),(134648,'keta'),(104483,'ketamine'),(84186,'ketone'),(15491,'key'),(100032,'Keyence'),(110502,'keyword'),(36765,'kg'),(6074,'KGF'),(33748,'Khodjakov'),(3791,'kidney'),(320507,'Kikegaard'),(367054,'Kikwalila'),(319036,'Kikwit'),(4456,'kill'),(321864,'killer'),(4818,'killing'),(69519,'kilobase'),(18205,'kilodalton'),(367056,'Kilombero'),(11730,'Kim'),(18118,'kinase'),(35440,'kind'),(26535,'kindly'),(34231,'kinesin'),(9127,'kinetic'),(62,'King'),(25911,'kingdom'),(96692,'Kinki'),(104262,'Kinney'),(13480,'Kirkegaard'),(33729,'Kirschner'),(14290,'kit'),(99920,'KK'),(94010,'KKL'),(337623,'Klaudiny'),(4603,'Klebsiella'),(132640,'Klein'),(33490,'Klemperer'),(99252,'Kleppe'),(301103,'km'),(15312,'knee'),(148024,'knock'),(338433,'knockdown'),(6738,'knockout'),(305812,'Knoevenagel'),(513,'know'),(2995,'knowledge'),(105864,'knowledgeable'),(17929,'known'),(120421,'Koch'),(337109,'Kocks'),(64494,'KOD'),(43648,'Kodak'),(10909,'Koev'),(9224,'Koga'),(87854,'Kohn'),(23932,'Koike'),(122188,'Kollef'),(94511,'Kolmogorov'),(139857,'Kong'),(120225,'Koo'),(124600,'Korea'),(96689,'Koshinetu'),(119077,'Kosovo'),(307298,'Kotalik'),(43419,'kozak'),(293,'kPa'),(26636,'KpnI'),(39565,'Kr'),(360649,'Krammer'),(133528,'Krebshilfe'),(124432,'Kress'),(9631,'Krilov'),(148556,'Krishnan'),(59902,'Kruskal'),(35600,'KSHV'),(75687,'ku'),(4986,'Kuan'),(124776,'Kuching'),(99237,'Kunath'),(77253,'Kurland'),(335567,'kutta'),(342755,'Kwong'),(16600,'KyberSept'),(96696,'Kyushu'),(12068,'LA'),(32152,'lab'),(66553,'LabChip'),(4483,'label'),(3652,'labeling'),(32719,'labelling'),(50434,'labile'),(141584,'LabMap'),(21135,'labor'),(30148,'Laboratories'),(164,'laboratory'),(372036,'laborintensive'),(36841,'laborious'),(107434,'labour'),(29804,'Labs'),(25442,'LAC'),(110896,'LACDR'),(4003,'lack'),(75214,'lacking'),(69809,'lactamase'),(49102,'Lactic'),(49129,'lactis'),(49976,'Lactobacillus'),(49519,'lactococcal'),(49266,'lactococci'),(49143,'Lactococcus'),(50269,'lactoglobulin'),(305865,'lactone'),(312858,'Lactuca'),(24821,'lacz'),(62661,'ladder'),(30257,'laemmli'),(6386,'laevis'),(136610,'LaForce'),(63702,'lagging'),(336744,'Lagueux'),(357843,'Lahav'),(11006,'Lai'),(43718,'Lake'),(109378,'Lakes'),(126988,'Lam'),(98120,'lambda'),(5201,'lamellar'),(337139,'lamellocyte'),(35158,'lament'),(64983,'lamp'),(87774,'land'),(22591,'landmark'),(74050,'landscape'),(11502,'lane'),(14039,'Lang'),(21768,'Language'),(100332,'Laos'),(58082,'laparotomy'),(20739,'lapel'),(77894,'Lapointe'),(51513,'laptop'),(666,'large'),(2469,'largely'),(16,'larly'),(335668,'larvae'),(338794,'larval'),(9769,'larynx'),(56572,'laser'),(61218,'last'),(757,'lasting'),(5510,'lastly'),(314424,'lat'),(5993,'late'),(14554,'latency'),(35455,'latent'),(35638,'latently'),(8941,'later'),(102781,'lateral'),(301289,'laterally'),(38638,'Latin'),(136641,'lattice'),(366961,'Laufelfinger'),(141686,'launch'),(127,'lavage'),(47991,'law'),(52776,'Lawrence'),(17441,'lawsuit'),(348933,'lawwhere'),(105464,'lay'),(7140,'layer'),(83098,'layered'),(31657,'layering'),(328716,'layout'),(359022,'Lazar'),(336970,'Lazzaro'),(30937,'LB'),(12827,'LC'),(305891,'LCAA'),(363413,'LCGreen'),(30400,'lck'),(145305,'LCMSMS'),(378424,'LCMV'),(64058,'LCR'),(319164,'ld'),(27082,'le'),(119813,'Leach'),(2215,'lead'),(11022,'leader'),(13320,'leaderless'),(72963,'leadership'),(60204,'leaf'),(102124,'League'),(36167,'leak'),(122334,'leakage'),(78320,'leaky'),(108610,'lean'),(10766,'learn'),(53635,'learning'),(21462,'lease'),(11999,'leave'),(129218,'lec'),(367623,'Lecourbe'),(3529,'lectin'),(17485,'lecture'),(130114,'lecwhere'),(98114,'LED'),(23906,'Lee'),(85725,'Leeds'),(7804,'left'),(309787,'legacy'),(59627,'legal'),(101476,'legally'),(41545,'legend'),(311171,'Legionella'),(48293,'legislate'),(48083,'legislation'),(48006,'legislative'),(48638,'legislature'),(307777,'legitimacy'),(37250,'legitimate'),(107671,'legitimately'),(367580,'Leica'),(314102,'Leicester'),(78343,'Leinfelder'),(371263,'Leishmania'),(371343,'Leishmaniasis'),(50315,'leiss'),(50025,'LEISSTCDA'),(377025,'leisure'),(16922,'Lemaire'),(337321,'Lemaitre'),(309922,'lend'),(3999,'length'),(63627,'lengthen'),(325597,'lengthy'),(362709,'lens'),(147895,'lentivirus'),(34025,'Leopold'),(337424,'Lepidoptera'),(337365,'lepidopteran'),(345637,'leprosy'),(367639,'Les'),(7016,'lesion'),(302530,'lesione'),(53690,'Lespinats'),(7089,'less'),(118950,'lessdetaile'),(338105,'lessen'),(74229,'lesson'),(129823,'let'),(10055,'lethal'),(2255,'lethality'),(51630,'letter'),(26726,'LEU'),(17863,'leucine'),(75710,'leucineCandida'),(123227,'leucocytosis'),(50108,'Leuconostoc'),(75971,'leucyl'),(76072,'leucylate'),(76295,'leucylated'),(75757,'leucylation'),(76410,'leucylthe'),(34222,'leukaemia'),(29219,'leukemia'),(6340,'leukocyte'),(122702,'leukocytosis'),(344317,'leukoencephalopathy'),(123231,'leukopenia'),(14791,'leukoreduction'),(6695,'leukotriene'),(326221,'leung'),(31050,'leupeptin'),(75974,'LeuRS'),(340785,'Leusch'),(337770,'Levashina'),(2354,'level'),(110745,'Levels'),(92077,'leverage'),(5790,'Levine'),(343998,'Levinthal'),(17426,'Levy'),(3735,'Leydig'),(128888,'lg'),(27272,'Li'),(308882,'liability'),(62377,'LiAc'),(27215,'Liao'),(14887,'liberal'),(75535,'liberalization'),(95923,'liberalize'),(380488,'liberase'),(17619,'liberated'),(5366,'liberation'),(308950,'liberty'),(301037,'libitum'),(24332,'library'),(371305,'licence'),(73283,'license'),(96157,'licensing'),(58230,'licking'),(99577,'lid'),(346169,'lidocaine'),(8036,'lie'),(348284,'lieu'),(11750,'life'),(2216,'lifelong'),(415,'liferative'),(373917,'lifespan'),(73790,'Lifestyle'),(100411,'lifetime'),(3556,'ligand'),(30480,'ligase'),(43411,'ligate'),(64974,'ligation'),(947,'light'),(145620,'LightCycler'),(21063,'Lightweight'),(338753,'ligustica'),(220,'like'),(42172,'likelihood'),(333022,'likelihoodmore'),(476,'likely'),(105844,'likert'),(42542,'likewise'),(126986,'Lim'),(123447,'limb'),(332683,'Limburg'),(2492,'limit'),(2237,'limitation'),(9852,'limited'),(131798,'limiting'),(320185,'Limulus'),(27249,'Lin'),(23983,'Linden'),(49972,'Lindholm'),(3714,'line'),(25579,'lineage'),(20824,'linear'),(375056,'linearise'),(88000,'linearity'),(81376,'linearization'),(30325,'linearize'),(26930,'linearized'),(71111,'linearly'),(311457,'lingular'),(5580,'lining'),(2929,'link'),(105032,'linkable'),(2293,'linkage'),(305686,'linker'),(305627,'linkerbase'),(363031,'linkermediate'),(141634,'Linux'),(57954,'lipase'),(3546,'lipid'),(5606,'lipidosis'),(143392,'Lipofectamine'),(56839,'Lipofectin'),(131766,'lipofection'),(4470,'lipopolysaccharide'),(31661,'lipoprotein'),(132003,'liposome'),(4677,'lipoteichoic'),(26885,'liquid'),(36204,'LIS'),(75391,'lishe'),(8891,'list'),(21473,'listener'),(87912,'Listeria'),(55421,'listing'),(365211,'liter'),(9426,'literature'),(303106,'litis'),(58041,'littermate'),(4064,'little'),(43583,'Littleton'),(30555,'live'),(359803,'lived'),(6410,'liver'),(87858,'Livermore'),(126640,'livestock'),(45419,'living'),(60758,'ll'),(76112,'Lloyd'),(150409,'LLTTNKD'),(380587,'lM'),(113676,'LMNA'),(85683,'ln'),(85438,'lna'),(85745,'LNAs'),(22638,'load'),(131713,'loading'),(74089,'lobby'),(17168,'lobe'),(21078,'Lober'),(58706,'lobular'),(2074,'local'),(45349,'localisation'),(348605,'localise'),(351902,'locality'),(3695,'localization'),(3756,'localize'),(3745,'locally'),(4253,'locate'),(21541,'location'),(138998,'locationslocation'),(56180,'loci'),(86797,'lock'),(85435,'locked'),(60225,'locus'),(22404,'log'),(43357,'Logan'),(54719,'logarithm'),(70046,'logarithmic'),(54759,'logged'),(21711,'logic'),(37227,'logical'),(37336,'logistic'),(307460,'logistical'),(353482,'logistically'),(16926,'London'),(20655,'lone'),(756,'long'),(17209,'longawaite'),(111916,'longitudinal'),(72784,'longstanding'),(42883,'longterm'),(22922,'look'),(22959,'lookback'),(8163,'loop'),(333180,'loosely'),(367261,'Lopez'),(74406,'Los'),(8080,'lose'),(28084,'loss'),(124618,'lot'),(30302,'Louis'),(337705,'Lourenço'),(100041,'Lourmat'),(93067,'louse'),(6,'low'),(363181,'Lowe'),(40769,'lower'),(143022,'lowered'),(113018,'lowexpression'),(138628,'lowgrade'),(18031,'lowlevel'),(99293,'lowmolecular'),(25672,'lowpolyamine'),(89593,'lowquality'),(77962,'lowthe'),(29795,'Loyola'),(332321,'LPAI'),(4471,'LPS'),(94078,'LRTI'),(124914,'LS'),(109474,'LSD'),(56350,'Lse'),(322606,'LT'),(85724,'Ltd'),(148193,'LTR'),(11831,'LU'),(62165,'LUC'),(51228,'LUCA'),(62162,'luciferase'),(375461,'Ludwig'),(50803,'lular'),(123893,'luman'),(346274,'lumbar'),(123952,'luminal'),(99413,'luminance'),(62487,'luminescence'),(140628,'Luminex'),(62493,'Luminometer'),(380239,'Luminy'),(129035,'lump'),(123208,'luna'),(391,'lung'),(344992,'lupus'),(104290,'Luria'),(63507,'luteoviruse'),(135410,'LV'),(43274,'lyase'),(130629,'lyle'),(19469,'lymph'),(344310,'lymphadenitis'),(345208,'lymphadenopathy'),(358736,'lymphoblastoid'),(5897,'lymphocyte'),(5062,'lymphocytic'),(36925,'lymphocytopenia'),(379413,'lymphohistiocytosis'),(5078,'lymphoid'),(20108,'lymphoma'),(380303,'lymphopenia'),(372770,'lymphoproliferative'),(301087,'Lynn'),(132624,'lyophilization'),(83268,'lys'),(24884,'lysate'),(84095,'Lysates'),(14208,'lyse'),(23892,'lysine'),(26768,'lysing'),(31032,'lysis'),(56351,'lysosomal'),(56345,'lysosome'),(120854,'lysotype'),(104302,'lysozyme'),(373298,'LYT'),(64514,'Lyzer'),(29935,'MA'),(318987,'mAb'),(380754,'mAbs'),(141632,'Mac'),(319098,'Macaca'),(319088,'macaque'),(319381,'Macaques'),(22687,'machine'),(28305,'machinery'),(125220,'Macintosh'),(118708,'Mack'),(10572,'macroarray'),(127504,'macromolecular'),(130493,'macromolecule'),(2022,'macrophage'),(16405,'macroscopic'),(345915,'macular'),(718,'maculopapular'),(327836,'MAD'),(134179,'Madison'),(376273,'Madonna'),(37886,'Madras'),(94378,'Maertzdorf'),(17860,'maf'),(46714,'magenta'),(80800,'magh'),(16195,'magic'),(79314,'magnesium'),(102578,'magnetic'),(18549,'magnitude'),(22349,'mail'),(13482,'main'),(100296,'mainland'),(12240,'mainly'),(72522,'mainstream'),(9673,'maintain'),(72502,'Maintaining'),(45417,'maintenance'),(3660,'major'),(899,'majority'),(2338,'make'),(21038,'maker'),(90481,'makeup'),(17462,'making'),(12603,'Makino'),(844,'malaise'),(75076,'malaria'),(366228,'malarial'),(124601,'Malaysia'),(33466,'MALDI'),(145281,'MALDIMS'),(43340,'male'),(123493,'malfunction'),(16530,'malign'),(401,'malignancy'),(130941,'malignant'),(36446,'Mallincrodt'),(124031,'malposition'),(122632,'maltophilia'),(77309,'maltosa'),(4707,'maltose'),(3685,'mammal'),(23591,'mammalian'),(14413,'man'),(21727,'manage'),(105660,'manageability'),(116520,'manageable'),(984,'management'),(321195,'managementrelate'),(22632,'manager'),(351450,'Manassas'),(107835,'mandate'),(321337,'mandated'),(20672,'mandatory'),(337223,'Manduca'),(20010,'manganese'),(7928,'manifest'),(179,'manifestation'),(138817,'manifesting'),(78998,'manipulate'),(301458,'manipulating'),(63735,'manipulation'),(141737,'manipulator'),(58297,'Mann'),(5684,'mannan'),(5109,'manner'),(29761,'Mannheim'),(3537,'mannose'),(17145,'manoeuvre'),(352814,'manpower'),(105400,'Mantel'),(20609,'manual'),(22162,'manually'),(36485,'manufacture'),(200,'manufacturer'),(56119,'manufacturing'),(4699,'manuscript'),(12853,'map'),(149412,'Mapitope'),(18119,'MAPK'),(21761,'mapping'),(22419,'Mapplot'),(367146,'Maprik'),(87269,'marburg'),(16349,'March'),(323339,'margin'),(349448,'marginal'),(102021,'marginalize'),(15441,'marginally'),(14942,'Marik'),(135337,'marine'),(147332,'maritima'),(17504,'mark'),(7850,'marked'),(2072,'markedly'),(2701,'marker'),(10567,'market'),(52759,'Markov'),(357836,'Markovic'),(21767,'Markup'),(11241,'Marle'),(313984,'Marnes'),(101936,'marry'),(119065,'Marseille'),(16524,'Marshall'),(15013,'Martin'),(357044,'martview'),(100047,'MARUMI'),(72625,'marvel'),(105778,'Maryland'),(17371,'mask'),(380456,'masking'),(33467,'mass'),(363158,'Massachusetts'),(9696,'masse'),(10695,'massive'),(117395,'massively'),(32364,'Massung'),(2024,'mast'),(62443,'master'),(335582,'Masuda'),(135416,'MAT'),(69818,'MATa'),(11260,'match'),(149870,'matching'),(24526,'mate'),(3390,'material'),(11486,'Materials'),(80312,'MatH'),(61367,'Mathematica'),(316102,'MathematicaH'),(10709,'mathematical'),(38058,'mathematically'),(81810,'MatHN'),(114216,'MathWorks'),(61312,'mating'),(114214,'MATLAB'),(80311,'MatNH'),(8699,'matrix'),(23619,'Matsufuji'),(351108,'Matt'),(26339,'matter'),(358020,'Matthias'),(51179,'maturase'),(4564,'maturation'),(4143,'mature'),(324098,'matured'),(76945,'max'),(372413,'Maxadilan'),(104733,'Maxi'),(122468,'maxillary'),(4085,'maximal'),(127532,'maximally'),(307593,'maximisation'),(307569,'maximise'),(55203,'maximize'),(25804,'maximum'),(320295,'Maxisorp'),(43714,'MAXO'),(43716,'Maxtec'),(25896,'maybe'),(51510,'mb'),(64416,'MBP'),(36441,'MBq'),(141180,'MC'),(83557,'MCA'),(33398,'McAuslan'),(27372,'McBride'),(75874,'McClain'),(76204,'McCloskey'),(304046,'McCrate'),(43441,'McCullough'),(58039,'MCDM'),(24613,'McGill'),(4696,'McGregor'),(90568,'McGuinness'),(28343,'mci'),(136588,'McKendrick'),(311988,'McKenna'),(94965,'McLuhan'),(303490,'mcm'),(348575,'MCMC'),(377396,'MCMV'),(379241,'mcmvinfecte'),(378723,'MCMVspecific'),(85441,'McTigue'),(303620,'MD'),(100449,'MDMA'),(347870,'MDx'),(10827,'mean'),(72212,'meaning'),(61197,'meaningful'),(353597,'meaningfully'),(329729,'meaningless'),(34404,'means'),(118607,'meas'),(120625,'measle'),(10094,'measurable'),(5724,'measure'),(2828,'measured'),(6847,'measurement'),(48133,'meat'),(75673,'mechan'),(36128,'mechanical'),(36847,'mechanically'),(2080,'mechanism'),(377734,'mechanismthe'),(2447,'mechanistic'),(13486,'mechanistically'),(75257,'Med'),(79203,'Media'),(7139,'medial'),(15041,'median'),(22822,'mediastinal'),(2462,'mediate'),(59030,'mediation'),(2035,'mediator'),(312860,'Medicago'),(17453,'medical'),(72266,'Medicare'),(23217,'medication'),(16274,'Medicine'),(52979,'Medigue'),(187,'medium'),(100230,'Medline'),(14170,'Meer'),(14345,'meet'),(72618,'meeting'),(340583,'MEGA'),(54207,'megabase'),(26937,'MEGAshortscript'),(15459,'Meier'),(57027,'meiosis'),(57105,'meiotic'),(31769,'Meis'),(335810,'melanization'),(336296,'melanize'),(24188,'melanogaster'),(57232,'melanoma'),(133257,'melanomas'),(34574,'Melki'),(337406,'mellifera'),(115694,'mellitus'),(336487,'Mellroth'),(85606,'melt'),(338957,'meltcurve'),(85656,'melting'),(85678,'MeltWin'),(119828,'Meltzer'),(85180,'Melville'),(29672,'MEM'),(3524,'member'),(304483,'membered'),(108734,'membership'),(3651,'membrane'),(4635,'membraneassociate'),(85279,'Membranes'),(113833,'Memorial'),(87984,'memory'),(133258,'meningioma'),(9491,'meningitidis'),(124555,'meningitis'),(38403,'meningococcal'),(736,'meningoencephalitis'),(35892,'ment'),(73001,'mental'),(45902,'mention'),(139244,'menu'),(84511,'MeOSuc'),(362478,'mer'),(79393,'mercaptoethanol'),(85556,'Merck'),(135320,'mercurial'),(52148,'mere'),(2655,'merely'),(101737,'merge'),(3037,'merit'),(86087,'MeRNA'),(339145,'MEROPS'),(366630,'merozoite'),(140953,'MES'),(319654,'mesenteric'),(50109,'mesenteroide'),(3782,'mesentery'),(56509,'mesh'),(4692,'Mesika'),(19448,'mesothelioma'),(3658,'message'),(22875,'messaging'),(1977,'messenger'),(113670,'MEST'),(14728,'meta'),(3512,'metabolic'),(339661,'metabolically'),(2088,'metabolism'),(2622,'metabolite'),(18867,'metacholine'),(2951,'metal'),(83795,'metalloprotease'),(4923,'metalloproteinase'),(135840,'Metalloproteinases'),(301348,'Metamorph'),(133586,'metaphase'),(94065,'metapneumovirus'),(373823,'metapopulation'),(8021,'metastasis'),(8212,'metastatic'),(25956,'metazoan'),(101501,'metha'),(2834,'methacholine'),(100894,'methadone'),(100391,'methamphetamine'),(305854,'methanesulfonic'),(30874,'methanol'),(22910,'methicillin'),(122616,'methicillinsensitive'),(20183,'methionine'),(250,'method'),(5519,'methodological'),(91213,'methodology'),(55010,'Methods'),(45250,'methyl'),(46179,'methylase'),(45257,'methylate'),(47907,'methylating'),(35905,'methylation'),(339752,'methylcoumarin'),(45252,'methylcytosine'),(86794,'methylene'),(100450,'methylenedioxymethamphetamine'),(311619,'methylprednisolone'),(331873,'methylsulfate'),(303491,'methylthio'),(46105,'methyltransferase'),(332897,'metre'),(51895,'metric'),(136919,'metro'),(96700,'metropolis'),(105797,'metropolitan'),(11009,'Meulenberg'),(372319,'mexicana'),(122000,'Mexico'),(340760,'Meyer'),(343551,'Meyers'),(141547,'MFI'),(55579,'Mfold'),(14180,'mg'),(93736,'MGB'),(30310,'MgCl'),(35171,'MgGTP'),(128387,'MGP'),(64741,'MgSO'),(80956,'MH'),(90319,'MHC'),(90630,'MHCBN'),(90619,'MHCPEP'),(12615,'MHV'),(301247,'MI'),(69174,'MIAME'),(103951,'mice'),(27037,'Michael'),(64031,'micro'),(5568,'microanatomic'),(10667,'microarray'),(365569,'microarraybase'),(124019,'microaspiration'),(338287,'microbe'),(4429,'microbial'),(193,'Microbio'),(18944,'microbiological'),(73147,'microbiologist'),(163,'Microbiology'),(109456,'Microcal'),(66353,'microcapillary'),(362513,'microcentrifuge'),(15594,'microcirculation'),(14391,'microcirculatory'),(330681,'Microcon'),(359874,'microenvironment'),(66104,'microfluidic'),(10625,'microgram'),(363230,'microlit'),(26878,'microliter'),(56628,'microlitre'),(145312,'Micromass'),(360245,'micromolar'),(347886,'micron'),(3545,'microorganism'),(312536,'microparasite'),(140394,'microparticle'),(43798,'microplate'),(149018,'micrornas'),(19120,'microsatellite'),(33001,'microscope'),(16501,'microscopic'),(3754,'microscopically'),(109468,'microscopy'),(373794,'microsimulation'),(43889,'Microsoft'),(140611,'microsphere'),(338126,'microsporidia'),(76059,'microsurgery'),(22049,'Microsystems'),(32605,'microtubule'),(34192,'microtubuledependent'),(325218,'microvascular'),(21147,'mid'),(58081,'midabdominal'),(326379,'midazolam'),(77378,'middle'),(301287,'midline'),(75686,'Midori'),(111992,'migraine'),(95331,'migrant'),(59137,'migrate'),(8413,'migration'),(95371,'migratory'),(36345,'Mikrotac'),(17205,'Milan'),(855,'mild'),(115773,'milder'),(5426,'mildly'),(145314,'Milford'),(84986,'milieu'),(616,'military'),(50281,'milk'),(10908,'Miller'),(26767,'milligram'),(19051,'millimolar'),(17993,'million'),(64511,'Millipore'),(93658,'milliq'),(138362,'Milwaukee'),(150389,'Mim'),(150388,'MimAlign'),(20314,'mimic'),(371879,'mimick'),(58545,'mimicking'),(116604,'mimicry'),(339360,'Mimivirus'),(149418,'MIMOP'),(149296,'mimotope'),(149459,'MIMOX'),(26775,'min'),(380470,'mince'),(66300,'mind'),(368880,'mine'),(93722,'MinElute'),(141772,'mineral'),(301189,'Minetta'),(62320,'mini'),(59475,'minichromosome'),(301003,'minigene'),(330549,'minima'),(9889,'minimal'),(117738,'minimalist'),(26275,'minimally'),(123969,'minimise'),(63551,'minimization'),(2748,'minimize'),(343790,'minimizing'),(38161,'minimum'),(373555,'mining'),(26916,'Miniprep'),(72261,'Minister'),(49012,'Ministerial'),(72255,'Ministers'),(74760,'Ministry'),(10579,'Minneapolis'),(25,'minor'),(13027,'minority'),(10949,'minus'),(282,'minute'),(149019,'miRNA'),(149081,'mirnas'),(13285,'mirror'),(119650,'mis'),(77820,'misacylation'),(144662,'misalignment'),(77911,'misaminoacylation'),(309153,'misattribution'),(77271,'mischarge'),(121572,'misclassification'),(117298,'misclassifie'),(45444,'miscoded'),(309149,'misconception'),(30650,'misfolde'),(66240,'mishandling'),(124995,'misidentification'),(43221,'misincorporate'),(64150,'misincorporation'),(307513,'misinformation'),(355030,'misleading'),(12010,'mismatch'),(145170,'mismatched'),(117995,'misnomer'),(43281,'mispair'),(143783,'mispaired'),(108709,'misplace'),(42283,'miss'),(21234,'missile'),(88606,'missing'),(317269,'mission'),(377142,'misspecification'),(348854,'mistake'),(60138,'mistaken'),(121465,'mistakenly'),(96373,'misuse'),(23689,'Mitchell'),(5913,'mite'),(139981,'mitigate'),(369480,'mitigating'),(369465,'mitigation'),(42917,'mitochondria'),(16441,'mitochondrial'),(18115,'mitogen'),(6436,'mitogenic'),(301987,'mitosis'),(24006,'mitotic'),(43468,'Mitra'),(135324,'MIWC'),(12948,'mix'),(7841,'mixed'),(316475,'mixing'),(10632,'mixture'),(23629,'Miyazaki'),(141727,'MJ'),(14181,'ml'),(60647,'MluI'),(24513,'MM'),(62369,'mMachine'),(62370,'mMessage'),(287,'mmHg'),(80256,'mmlv'),(55602,'mmol'),(376531,'MMR'),(101577,'MMT'),(10580,'MN'),(303526,'mnm'),(367831,'MNN'),(362290,'MNSFb'),(20013,'mnsod'),(30303,'MO'),(24132,'mobile'),(25381,'mobility'),(53639,'mobilize'),(28859,'mock'),(15671,'MOD'),(317893,'modal'),(19151,'modality'),(13451,'mode'),(2187,'model'),(138709,'modeled'),(37701,'modeling'),(112077,'modelling'),(52760,'Models'),(6262,'modelssilicosis'),(856,'moderate'),(19722,'moderately'),(37265,'moderatesized'),(14640,'modern'),(24629,'modest'),(7399,'modestly'),(28144,'modi'),(95656,'modifi'),(76518,'modifica'),(2777,'modification'),(37720,'modified'),(105649,'modifier'),(4427,'modify'),(353581,'modifying'),(64234,'Modrich'),(64081,'modular'),(3505,'modulate'),(2453,'modulation'),(6438,'modulator'),(74420,'module'),(301185,'MOG'),(307891,'MOHLTC'),(83592,'MOI'),(4398,'moiety'),(123552,'moisture'),(337771,'Moita'),(85819,'mol'),(62422,'molar'),(8951,'molecular'),(148635,'molecularly'),(99622,'molecularweight'),(1978,'molecule'),(91356,'Molecules'),(345013,'Molluscum'),(134171,'Moloney'),(85900,'molthe'),(13352,'moment'),(316418,'momentum'),(16764,'MONARCS'),(84486,'monensin'),(87217,'money'),(3461,'monitor'),(39779,'monitored'),(3472,'monitoring'),(319146,'monkey'),(119625,'monkeypox'),(85351,'monlayer'),(110594,'monly'),(30452,'mono'),(71086,'monocistronic'),(3670,'monoclonal'),(69597,'monocotyledonous'),(7380,'monocrotaline'),(8504,'monocyte'),(322226,'Monocytes'),(4979,'monocytic'),(378819,'monocytogene'),(79184,'monohydrate'),(348128,'monoisotopic'),(83600,'monolayer'),(4124,'monomer'),(3947,'monomeric'),(126608,'Mononegavirales'),(4864,'mononuclear'),(69603,'monopartite'),(18074,'monophosphate'),(147405,'monosaccharide'),(325141,'monosodium'),(373999,'monotonicity'),(30617,'monoubiquitin'),(306146,'monovalent'),(361704,'monovalently'),(17518,'monoxide'),(367248,'Montanide'),(87438,'Monte'),(357879,'Montfort'),(112,'month'),(96651,'monthly'),(141014,'MOPS'),(147501,'Moracci'),(107598,'moral'),(48185,'morale'),(107448,'morally'),(324306,'Moraxella'),(107533,'morbid'),(7961,'morbidity'),(95231,'Mordini'),(150376,'Moreau'),(32513,'Morgan'),(336889,'mori'),(103299,'moribund'),(367058,'Morogoro'),(58925,'morphine'),(32369,'morphogenesis'),(144161,'morpholino'),(58595,'morphologic'),(27754,'morphological'),(32913,'morphologically'),(214,'morphology'),(301273,'morphometric'),(337962,'Morse'),(40,'mortality'),(10143,'mortalitythat'),(122860,'mortem'),(42137,'mortuary'),(10941,'mosaic'),(102369,'mosquito'),(32478,'Moss'),(366995,'Mossi'),(52949,'Moszer'),(335870,'moth'),(10384,'motif'),(330982,'MotifSampler'),(330629,'MotifScanner'),(32580,'motility'),(304811,'motion'),(41309,'motivate'),(101296,'motivated'),(136546,'motivation'),(101586,'motivational'),(358183,'Motohashi'),(32764,'motor'),(34285,'motordriven'),(20477,'Motterlini'),(69591,'mottle'),(30145,'mount'),(2182,'mouse'),(109902,'mouth'),(123694,'mouthpiece'),(23052,'move'),(33859,'movement'),(48617,'moving'),(368877,'mp'),(325841,'MPB'),(128386,'MPG'),(2616,'MPO'),(325924,'MRI'),(6055,'mrna'),(10177,'mrnas'),(122627,'MRSA'),(85542,'MS'),(26599,'msci'),(85546,'MSD'),(45630,'MT'),(345278,'MTB'),(56681,'MTN'),(32714,'MTOC'),(34446,'MTOCorchestrated'),(46206,'mucin'),(322358,'mucociliary'),(323316,'mucoid'),(3781,'mucosa'),(9543,'mucosal'),(6578,'mucous'),(3115,'mucus'),(324112,'mul'),(319113,'mulatta'),(316221,'Müller'),(35416,'multi'),(45330,'multicellular'),(14633,'multicentre'),(36530,'multichannel'),(140458,'MultiCode'),(116181,'multicolor'),(21337,'multidata'),(346788,'multidimensional'),(73814,'multidisciplinary'),(46212,'multidomain'),(109035,'multidrug'),(105587,'multifactorial'),(344316,'multifocal'),(722,'multiforme'),(79150,'multifunctionality'),(312403,'multihost'),(3857,'multimer'),(5303,'multimeric'),(4738,'multimerization'),(5147,'multimerize'),(138948,'multimodality'),(361016,'multimolecular'),(142054,'multimutation'),(96224,'multinational'),(322273,'multinucleate'),(36730,'multinucleated'),(15434,'multiorgan'),(368438,'MultiPhyl'),(2659,'multiple'),(95152,'multipleround'),(14697,'multiplesystem'),(10201,'multiplex'),(140465,'multiplexe'),(140346,'multiplexed'),(40938,'multipli'),(49750,'multiplication'),(333005,'multiplicative'),(17707,'multiplicity'),(40650,'multiplie'),(60334,'multiply'),(131231,'multiprotein'),(96652,'multipurpose'),(116251,'multispectral'),(321118,'multistage'),(48740,'multistate'),(131186,'multistep'),(325221,'multisystem'),(22061,'multitiered'),(357866,'multitude'),(3954,'multivalent'),(325369,'multivariable'),(106028,'multivariate'),(35531,'multivi'),(35549,'multivirus'),(150237,'Mumey'),(376177,'mump'),(62535,'mung'),(320923,'municipal'),(100698,'municipality'),(56673,'Munroe'),(148840,'Murabutide'),(23642,'Murakami'),(78603,'Muramatsu'),(78328,'Murgola'),(5766,'murine'),(324872,'muscarinic'),(429,'muscle'),(7741,'muscular'),(725,'musculoskeletal'),(43175,'Mut'),(77081,'muta'),(305387,'mutagen'),(3874,'mutagenesis'),(142939,'mutagenic'),(12619,'mutagenize'),(4218,'mutant'),(363835,'mutantonly'),(363120,'mutantto'),(12283,'mutate'),(2359,'mutation'),(4351,'mutational'),(363035,'mutationcontaine'),(363857,'mutationcontaining'),(363112,'mutationpositive'),(140385,'mutationspecific'),(64252,'muthls'),(57163,'MutS'),(307697,'mutual'),(316131,'mutually'),(362923,'MutY'),(122052,'MV'),(367161,'MVEEP'),(56574,'mw'),(64677,'MWCO'),(337398,'mya'),(727,'myalgia'),(100330,'Myanmar'),(31020,'myc'),(311176,'Mycobacteria'),(311431,'mycobacterial'),(51734,'Mycobacterium'),(1,'Mycoplasma'),(31533,'myelin'),(300730,'myelinate'),(301383,'myelinated'),(301340,'myelination'),(346068,'myelitis'),(412,'myelodysplastic'),(8534,'myelofibroblast'),(104408,'myeloma'),(2615,'myeloperoxidase'),(414,'myelopro'),(44551,'myh'),(133731,'mykiss'),(15207,'myocardial'),(733,'myocarditis'),(113672,'MYOG'),(72039,'myriad'),(322368,'myringotomy'),(368591,'mysql'),(304062,'n'),(31043,'Na'),(149574,'NAC'),(149774,'NACCESS'),(135393,'nacetylneuraminic'),(373603,'Nacional'),(29959,'NaCl'),(305769,'NaClO'),(86252,'NaClThe'),(344002,'NAD'),(42901,'NADH'),(135110,'NADPH'),(31041,'NaF'),(75685,'Nagatsuta'),(13468,'Nagy'),(367374,'NaHCO'),(22121,'naive'),(340859,'Najjar'),(52818,'Nakamura'),(104146,'Nakayama'),(43754,'naked'),(24899,'nal'),(100354,'Nam'),(56883,'name'),(62302,'naming'),(104347,'NaN'),(95588,'nancial'),(56478,'Nanjing'),(66550,'Nano'),(93134,'NanoDrop'),(56606,'nanogram'),(358946,'nanomolar'),(132982,'nanoparticle'),(132649,'nanoplexe'),(85408,'nanotechnology'),(79388,'NaOAc'),(105152,'NaOH'),(33755,'Nap'),(320503,'naphtholic'),(367678,'naphtyl'),(305209,'NAR'),(57899,'narcotic'),(100878,'Narcotics'),(345033,'Narita'),(121038,'narrative'),(42018,'narrow'),(131617,'NAs'),(17357,'NASA'),(9033,'nasal'),(11156,'nascent'),(122480,'nasogastric'),(122,'nasopharyngeal'),(122302,'nasopharynx'),(122303,'nasotracheal'),(95240,'Natapong'),(77744,'nate'),(48904,'nation'),(17135,'National'),(23163,'nationally'),(100289,'nationwide'),(7616,'native'),(138386,'natively'),(4846,'natural'),(23478,'naturally'),(2144,'nature'),(301121,'NBT'),(127336,'nc'),(130335,'ncap'),(30996,'NCBI'),(62156,'NcoI'),(28339,'NCS'),(113688,'NCSTN'),(33033,'nd'),(27518,'nde'),(12086,'nding'),(341203,'Ndomain'),(83428,'NE'),(4254,'near'),(20660,'nearby'),(85709,'Nearest'),(85453,'nearestneighbor'),(16614,'nearly'),(106704,'neatly'),(64502,'NEB'),(123676,'nebulise'),(123955,'nebuliser'),(122371,'nebulizer'),(25484,'Necator'),(2758,'necessarily'),(3916,'necessary'),(54411,'necessitate'),(25534,'necessity'),(3811,'neck'),(4908,'necrosis'),(10960,'necrotic'),(58849,'necrotize'),(13560,'ned'),(21009,'NEDSS'),(3319,'need'),(74002,'needle'),(101687,'needleexchange'),(147920,'Nef'),(4475,'negative'),(23718,'negatively'),(30172,'negativestrand'),(368073,'negativestranded'),(38847,'neglect'),(130450,'negligible'),(72893,'negotiate'),(72422,'negotiation'),(125135,'Nei'),(51942,'neighbor'),(327563,'neighborhood'),(78430,'neighboring'),(125130,'neighbour'),(333645,'neighbourhood'),(32569,'neighbouring'),(9490,'Neisseria'),(341046,'neisserial'),(34566,'Nejmeddine'),(33756,'Nek'),(361164,'NEM'),(25481,'nematode'),(58075,'Nembutal'),(29933,'NEN'),(352081,'nent'),(8268,'neo'),(330300,'neocortical'),(344823,'neoforman'),(19668,'neonatal'),(8278,'neoplastic'),(19424,'neovascularization'),(8752,'nephrosclerosis'),(748,'nerve'),(734,'nervous'),(348614,'ness'),(27174,'nest'),(11575,'nested'),(46856,'net'),(355139,'NetAffx'),(16387,'Netherlands'),(21460,'network'),(21928,'networking'),(5160,'neu'),(113678,'NEUGRIN'),(66824,'neural'),(346145,'neuralgia'),(4281,'neuraminidase'),(303205,'neuregulin'),(32601,'neurite'),(31706,'neuritic'),(31712,'neuritis'),(57601,'neurobiological'),(133022,'neuroblastoma'),(324543,'neurocognitive'),(31802,'neurodegenerative'),(8115,'neuroendocrine'),(31572,'neurofibrillary'),(325919,'neuroimaging'),(325834,'neuroinflammation'),(300698,'neurologic'),(22139,'neurological'),(325040,'neuromuscular'),(57767,'neuron'),(2020,'neuronal'),(57790,'neuronally'),(740,'neuropathy'),(135645,'neurophysial'),(134650,'neurophysin'),(326438,'neuroprotectant'),(302977,'neuroprotective'),(325784,'neuroradiologic'),(300966,'Neuroscience'),(23811,'Neurospora'),(1993,'neurotransmission'),(57636,'neurotransmitter'),(325891,'neurotrauma'),(325828,'neurotrophic'),(57637,'neurotrophin'),(5075,'neutral'),(59389,'neutrality'),(5962,'neutralization'),(5677,'neutralize'),(102321,'neutralizing'),(2023,'neutrophil'),(3436,'neutrophilic'),(3244,'new'),(411,'newborn'),(10346,'newly'),(109159,'Newport'),(17348,'newspaper'),(9078,'NF'),(30183,'ng'),(14076,'nger'),(35262,'ngerprinting'),(58958,'NGF'),(359083,'NGFR'),(101988,'NGO'),(102001,'NGOs'),(62351,'NH'),(69712,'NheI'),(343589,'Nhieu'),(72738,'NHMRC'),(72136,'NHS'),(64504,'Ni'),(43069,'niacin'),(35856,'NIAID'),(85076,'Nibert'),(71155,'nice'),(12562,'nicely'),(362344,'Nicholas'),(43160,'nick'),(102292,'nickel'),(104216,'nickelaffinity'),(52929,'Nicolas'),(10987,'Nidovirales'),(11010,'Nidovirus'),(13446,'nidoviruse'),(25929,'nidulan'),(9839,'nificance'),(371437,'Nifurtimox'),(37465,'Nigeria'),(34720,'Nigg'),(139349,'nigh'),(100454,'night'),(312865,'nigrum'),(35857,'NIH'),(360674,'Nikon'),(97471,'Nile'),(93065,'Nilsen'),(352971,'NIMS'),(332218,'NINDS'),(57542,'ninecarbon'),(365677,'ninety'),(25421,'ning'),(311036,'Ninth'),(113602,'nique'),(26831,'Nishiyama'),(49490,'nisin'),(49498,'nisr'),(2578,'nitration'),(1983,'nitric'),(3131,'nitro'),(344086,'nitroblue'),(30022,'nitrocellulose'),(371436,'nitrofuran'),(2093,'nitrogen'),(371438,'nitroimidazole'),(146092,'nitrophenyl'),(3130,'nitroso'),(3231,'nitrosothiol'),(3195,'nitrosyl'),(3186,'nitrosylation'),(2543,'nitrotyrosine'),(24411,'nity'),(17354,'NIV'),(320309,'Nivalis'),(79013,'nize'),(45740,'NJ'),(321865,'NK'),(94386,'NL'),(363745,'Nla'),(64838,'nlaiii'),(17697,'nm'),(340306,'NMEC'),(79292,'nmol'),(133384,'nmole'),(63434,'NMR'),(86281,'NN'),(100442,'NNCC'),(122148,'NNIS'),(367830,'NNM'),(87231,'NNs'),(61557,'NNW'),(128880,'Nobel'),(129753,'nocap'),(57645,'nociceptive'),(59047,'nociceptor'),(32742,'nocodazole'),(6889,'nocturnal'),(19470,'node'),(724,'nodosum'),(52027,'noise'),(20921,'noisy'),(92591,'Nolan'),(19533,'NOmediated'),(90780,'nomenclature'),(485,'non'),(324004,'nonallergic'),(144856,'nonameric'),(136104,'nonapeptide'),(303794,'nonbonded'),(16112,'noncardiac'),(122749,'noncardiogenic'),(79940,'noncoding'),(133162,'noncomplexed'),(89447,'nonconserved'),(30437,'nonconvalently'),(150053,'noncovalent'),(47787,'noncp'),(6974,'noncystic'),(47786,'noncytopatogenic'),(325200,'nondelirious'),(5706,'nondiabetic'),(131777,'nonenzymatic'),(68343,'nonetheless'),(85134,'nonfat'),(65218,'nonfluorescent'),(95592,'nongovernmental'),(19844,'nonheme'),(113913,'nonhepatoma'),(81434,'nonhomologous'),(375374,'nonhousehold'),(115805,'nonhuman'),(10229,'nonhypothesis'),(105080,'nonimmune'),(14501,'nonimmunerelate'),(377416,'noninfecte'),(310835,'noninfectious'),(17352,'noninvasive'),(36163,'noninvasively'),(302192,'nonlesion'),(301422,'nonlesione'),(14666,'nonleukocyte'),(115115,'nonlinear'),(95067,'nonmalefi'),(128316,'nonmultiplicative'),(372644,'nonmurine'),(140601,'nonnatural'),(102533,'nonneutralize'),(125639,'nonoutbreak'),(339247,'nonpathogenic'),(300764,'nonpermissive'),(318174,'nonpharmaceutical'),(9489,'nonpilate'),(107431,'nonprofessional'),(106877,'nonpublic'),(358663,'nonreduce'),(358317,'nonreducing'),(348203,'nonredundant'),(38328,'nonrenewe'),(47824,'nonreplicative'),(106771,'nonresponse'),(75366,'nonrival'),(303144,'nonscarre'),(8830,'nonsegmented'),(78324,'nonsense'),(15974,'nonsignificant'),(326369,'nonsignificantly'),(18440,'nonsmoke'),(19109,'nonsmoker'),(18717,'nonsmoking'),(7043,'nonspecific'),(303352,'Nonstandard'),(345433,'nonsteroidal'),(28024,'nonstructural'),(15664,'nonsurvivor'),(44707,'nontoxic'),(7637,'nontransplante'),(301793,'nontreated'),(128288,'nontrivial'),(327160,'nonuniformity'),(47770,'nonviral'),(38990,'nonzero'),(324991,'noradrenaline'),(16731,'NORASEPT'),(359740,'Nordberg'),(140656,'nordic'),(324982,'norepinephrine'),(20731,'norm'),(4095,'normal'),(374418,'normalise'),(375350,'normalising'),(61060,'normality'),(329608,'normaliza'),(10706,'normalization'),(22382,'normalize'),(92674,'normalizing'),(7065,'normally'),(75869,'Normanly'),(66306,'normative'),(108394,'normatively'),(7406,'normoxic'),(32455,'Norrby'),(16727,'North'),(109461,'Northampton'),(321171,'northcentral'),(352201,'northeast'),(321169,'northeastern'),(10181,'northern'),(140848,'Northwest'),(321167,'northwestern'),(46218,'norvegicus'),(93579,'Norway'),(74914,'norwegian'),(2030,'NOS'),(123550,'nose'),(262,'nosocomial'),(46175,'Nostoc'),(3726,'notable'),(2539,'notably'),(375199,'notation'),(2348,'note'),(139341,'notebook'),(66562,'Notes'),(73469,'noteworthy'),(52199,'notice'),(72184,'noticeable'),(73127,'notifiable'),(21058,'Notification'),(120455,'notify'),(2628,'notion'),(97864,'Notomi'),(97189,'notwithstanding'),(51015,'Nouaille'),(138981,'noun'),(366960,'Novabiochem'),(64499,'Novagen'),(8959,'novel'),(133808,'novelty'),(66995,'November'),(7691,'novo'),(380698,'Novocastra'),(95229,'Novotny'),(43581,'Novus'),(51271,'nowadays'),(90567,'Noy'),(122024,'NP'),(148393,'NPC'),(148126,'NPH'),(72341,'NPHP'),(45569,'nr'),(138174,'NREVSS'),(337692,'NRFG'),(127344,'nrib'),(303823,'ns'),(345437,'NSAIDS'),(325731,'NSE'),(72246,'NSW'),(48437,'NT'),(64505,'NTA'),(306688,'NTC'),(29956,'NTE'),(323112,'NTHi'),(323386,'nthiinfecte'),(148888,'NTP'),(148884,'NTPase'),(148902,'NTPbinding'),(49445,'Nuc'),(50033,'nucb'),(114044,'nucle'),(3173,'nuclear'),(49444,'nuclease'),(33051,'nucleate'),(33813,'nucleation'),(9616,'nucleic'),(148375,'nucleo'),(140478,'nucleobase'),(14042,'nucleocapsid'),(147712,'nucleolar'),(27379,'nucleolin'),(133123,'nucleolytic'),(94384,'nucleoprotein'),(43235,'nucleoside'),(35985,'nucleosomal'),(9148,'nucleotide'),(14174,'nucleus'),(49907,'NucT'),(132928,'nude'),(300803,'null'),(2463,'number'),(374534,'numberBy'),(74871,'numbered'),(66743,'numbering'),(66279,'numeric'),(61363,'numerical'),(60428,'numerically'),(2012,'numerous'),(320293,'Nunc'),(43534,'NuPage'),(20962,'nurse'),(335687,'nursery'),(107949,'Nurses'),(107790,'nursing'),(74258,'nurture'),(4797,'nutrient'),(72439,'nutrition'),(124044,'nutritional'),(29873,'NY'),(10605,'nylon'),(49495,'NZ'),(56292,'Oacetylation'),(57222,'oacetylesterase'),(69601,'oats'),(20117,'OB'),(458,'obacter'),(125031,'Oberste'),(77490,'obervation'),(72764,'obese'),(72243,'obesity'),(144528,'obey'),(90953,'object'),(21228,'objective'),(324850,'objectively'),(21225,'ObjectivesThe'),(75105,'oblast'),(108107,'obligate'),(21628,'obligation'),(120249,'obligatory'),(6981,'obliteran'),(6994,'obliterate'),(2854,'obliterative'),(340855,'Obrink'),(13026,'obscure'),(77622,'observ'),(138479,'observable'),(3321,'observation'),(15169,'observational'),(2559,'observe'),(6188,'observed'),(72730,'observer'),(99426,'obstacle'),(140706,'Obstetricians'),(8859,'obstruction'),(515,'obstructive'),(130,'obtain'),(138270,'obtainable'),(374928,'obtaindivide'),(10265,'obvious'),(10649,'obviously'),(40535,'occa'),(442,'occasion'),(38054,'occasional'),(7662,'occasionally'),(15543,'occlude'),(26398,'occlusion'),(324682,'occult'),(5495,'occupancy'),(75109,'occupation'),(72646,'Occupational'),(13447,'occupy'),(380,'occur'),(45455,'occurrence'),(145965,'OCH'),(52777,'Ochman'),(301065,'OCT'),(72347,'October'),(25920,'octosporu'),(345799,'ocular'),(26762,'od'),(23641,'ODC'),(77452,'odcase'),(15805,'odd'),(106016,'Odds'),(131022,'odn'),(17934,'odorless'),(14382,'oedema'),(124005,'oesophagus'),(4648,'ofek'),(313359,'off'),(100864,'offender'),(7521,'offer'),(21184,'office'),(101697,'officer'),(37243,'Officers'),(20633,'official'),(75196,'officially'),(141624,'offline'),(5848,'offset'),(48541,'offshore'),(317002,'offspring'),(340117,'offtarget'),(78157,'ofWe'),(114122,'OGDHL'),(42926,'oh'),(75746,'Ohama'),(77340,'Ohkuma'),(34716,'Ohta'),(78900,'Ohyama'),(344252,'OI'),(76354,'oide'),(141773,'oil'),(134360,'ok'),(77837,'Okamoto'),(310793,'OLB'),(383,'old'),(45997,'oleander'),(312863,'oleracea'),(92896,'olfactory'),(56616,'oligo'),(102724,'oligoclonal'),(300731,'oligodendrocyte'),(301693,'oligodendroglial'),(363419,'oligodT'),(3836,'oligomer'),(4049,'oligomeric'),(4162,'oligomerization'),(54155,'oligonucleodide'),(14233,'oligonucleotide'),(64713,'Oligonucleotides'),(85603,'oligoribonucleotide'),(62282,'oligos'),(4244,'oligosaccharide'),(80126,'olsthoorn'),(20973,'Olympics'),(109469,'Olympus'),(322325,'OME'),(29773,'omega'),(109216,'omentum'),(357077,'Ominibus'),(74528,'omission'),(37402,'omit'),(96643,'omnibus'),(25498,'onchocerca'),(43591,'Oncogene'),(131726,'oncogenic'),(16539,'oncology'),(126964,'oncolytic'),(17861,'oncoprotein'),(133730,'Oncorhynchus'),(26112,'one'),(327182,'onecolor'),(313890,'onetail'),(356819,'oneto'),(10487,'ongoing'),(79151,'onin'),(8223,'online'),(2098,'ONOO'),(2769,'onooare'),(2664,'onoobut'),(2524,'ONOOin'),(2568,'ONOOor'),(2502,'onooseem'),(2515,'ONOOunder'),(19270,'onset'),(56605,'Ontario'),(90736,'ontological'),(90547,'ontology'),(332256,'onward'),(332247,'onwards'),(136756,'OO'),(133552,'oocyte'),(135379,'oogenesis'),(133546,'oogenetic'),(337335,'Ooi'),(300778,'op'),(340892,'Opa'),(341450,'opaa'),(341451,'OpaB'),(36396,'opacity'),(341452,'opad'),(341724,'Opaisolate'),(341458,'opax'),(14629,'opelz'),(6462,'open'),(363524,'OpenArray'),(7184,'opening'),(106584,'openly'),(349619,'OpenMP'),(10376,'openreading'),(22095,'opensource'),(21538,'operate'),(68892,'operating'),(21045,'operation'),(51319,'operational'),(21260,'operationalize'),(38272,'operationally'),(353005,'Operations'),(7761,'operative'),(66591,'operator'),(100321,'opiate'),(75354,'opinion'),(326264,'opioid'),(100268,'opium'),(35572,'opportunistic'),(10589,'opportunity'),(479,'oppose'),(12766,'opposite'),(102106,'opposition'),(4453,'opsonic'),(122254,'opsonin'),(72949,'opt'),(744,'optic'),(85757,'optical'),(14859,'optimal'),(23568,'optimally'),(143395,'Optimem'),(17313,'optimism'),(107710,'optimistic'),(64102,'optimization'),(49727,'optimize'),(62613,'optimum'),(16202,'option'),(124837,'optional'),(68883,'optionally'),(22118,'Oracle'),(18762,'oral'),(101499,'orally'),(229,'orange'),(83086,'orbiviruse'),(377238,'orchestration'),(3856,'order'),(375478,'ordering'),(137325,'ordinal'),(48439,'ordinance'),(48790,'ordinarily'),(37657,'ordinary'),(76256,'ordinate'),(11571,'orf'),(146846,'orfs'),(95227,'org'),(697,'organ'),(77893,'organelle'),(3547,'organic'),(45904,'organisation'),(307119,'organisational'),(373638,'organise'),(23,'organism'),(359716,'organismal'),(3964,'organization'),(23155,'organizational'),(7982,'organize'),(90941,'organizing'),(56382,'organogenesis'),(47506,'organophosphorus'),(33161,'orient'),(5258,'orientation'),(12769,'origin'),(10322,'original'),(53036,'originality'),(3882,'originally'),(2644,'originate'),(31727,'origination'),(109459,'OriginLab'),(23639,'ornithine'),(37036,'orolabial'),(36994,'orolabiali'),(123998,'orophanryngeal'),(36420,'oropharyngeal'),(122222,'oropharynx'),(140517,'orthogonal'),(25480,'ortholog'),(330386,'orthologous'),(135908,'orthologue'),(336060,'orthology'),(367903,'Orthomyxoviridae'),(9995,'orthomyxovirus'),(88537,'orthopox'),(133287,'orthotopic'),(141633,'OS'),(75970,'Osawa'),(126942,'oscillate'),(127154,'oscillatory'),(99251,'Osland'),(43562,'Osmonics'),(92915,'osmotic'),(380501,'osmotically'),(322271,'osteoclast'),(20906,'OTC'),(349076,'otherwiseBy'),(114045,'otide'),(9449,'otitis'),(322391,'otorrhea'),(322401,'ototoxicity'),(73936,'Ottawa'),(19946,'Otterbein'),(45591,'OTU'),(340381,'otubain'),(361476,'Otubainhomolog'),(367000,'Ouagadougou'),(367002,'Oubritenga'),(48639,'ought'),(665,'outbreak'),(181,'outcome'),(89321,'outdated'),(21290,'outdoor'),(4486,'outer'),(59754,'outermost'),(129796,'outgoing'),(339622,'outgroup'),(4783,'outgrowth'),(51988,'outli'),(51774,'outlier'),(11253,'outline'),(51971,'outlying'),(130578,'outnumber'),(139,'outpatient'),(14334,'output'),(105606,'outrage'),(101352,'outreach'),(39415,'outrun'),(17237,'outset'),(21291,'outside'),(333880,'outstanding'),(32566,'outward'),(96371,'outweigh'),(18665,'ovalbumin'),(45618,'ovarian'),(5310,'ovary'),(2125,'overall'),(12791,'overcome'),(64246,'overdigestion'),(89810,'overestimate'),(2987,'overestimation'),(8746,'overexpress'),(5827,'overexpresse'),(27930,'overexpressed'),(25141,'overexpressing'),(5812,'overexpression'),(124251,'overgrowth'),(131224,'overhang'),(36686,'overhydration'),(141770,'overlaid'),(5537,'overlap'),(22582,'overlay'),(14384,'overload'),(307517,'overlook'),(107709,'overly'),(302953,'overlying'),(10643,'overnight'),(328610,'overpower'),(25159,'overproduce'),(42836,'overproduction'),(336566,'overproliferation'),(44743,'overrepresentation'),(329854,'overrepresented'),(139799,'overridden'),(48367,'override'),(137985,'overriding'),(48175,'overrun'),(48569,'overseas'),(333922,'oversimplify'),(25713,'overt'),(370730,'overtax'),(21366,'overview'),(72726,'overweight'),(307418,'overwhelm'),(318756,'overwhelmed'),(3273,'overwhelming'),(49707,'ovine'),(133900,'ovulation'),(133621,'ovulationmechanism'),(133642,'ovulatory'),(24718,'ow'),(37756,'owe'),(340795,'Owen'),(90891,'OWL'),(95851,'ownership'),(301022,'oxalic'),(54331,'Oxford'),(2105,'oxidant'),(35089,'oxidase'),(2533,'oxidation'),(2406,'oxidative'),(1984,'oxide'),(45291,'oxidise'),(2621,'oxidization'),(20190,'oxidize'),(360101,'oxidizing'),(357805,'oxidoreductase'),(43237,'oxo'),(43223,'oxog'),(45274,'oxoglutar'),(45265,'oxoglutarate'),(43171,'oxoguanine'),(339971,'oxyanion'),(339318,'oxyanionstabilize'),(135162,'oxydoreductase'),(290,'oxygen'),(2739,'oxygenase'),(6940,'oxygenation'),(19251,'oxyhemoglobin'),(135654,'oxytocin'),(145628,'Oy'),(16262,'PA'),(69758,'pac'),(95015,'pace'),(25505,'paci'),(124586,'Pacific'),(35984,'pack'),(317,'Package'),(23878,'packaging'),(54645,'Packard'),(304919,'packing'),(111195,'PaCO'),(70086,'pacrf'),(124611,'paediatric'),(124694,'paediatrician'),(336642,'Paenibacillus'),(351800,'PAF'),(22350,'page'),(22362,'pager'),(22110,'Pages'),(135197,'PAH'),(41914,'paign'),(903,'pain'),(57653,'painful'),(10634,'pair'),(11245,'pairing'),(303796,'pairlist'),(45805,'pairwise'),(357770,'Paiva'),(76891,'pak'),(100342,'Pakistan'),(124483,'palate'),(369745,'pale'),(131405,'palindrome'),(131812,'palindromic'),(367219,'pall'),(124975,'Pallansch'),(124490,'palm'),(10584,'Palo'),(427,'palsy'),(338530,'PAM'),(72412,'Pan'),(57651,'pancrea'),(56838,'pancreas'),(3732,'pancreatic'),(57600,'pancreatitis'),(58086,'pancreato'),(57625,'pancreatojejunostomy'),(72345,'pandemic'),(348385,'pane'),(10847,'panel'),(48206,'panic'),(60305,'panmictically'),(94003,'PAO'),(34810,'Paoletti'),(83705,'papain'),(372437,'papatasi'),(122862,'Papazian'),(29980,'paper'),(345914,'papillitis'),(49619,'papillomavirus'),(366159,'Papua'),(113088,'para'),(6617,'paracrine'),(18950,'paradigm'),(3409,'paradox'),(344311,'paradoxical'),(362267,'paradoxically'),(56779,'paraffin'),(364971,'paraffinembedde'),(34882,'paraformaldehyde'),(88596,'paragraph'),(94063,'Parainfluenza'),(35306,'parallel'),(115641,'parallelize'),(23729,'paralog'),(336786,'paralogous'),(336260,'paralogue'),(124561,'paralysis'),(10142,'parameter'),(373793,'parameterise'),(349703,'parameterization'),(129957,'parameterize'),(61068,'parametric'),(94516,'parametrical'),(107572,'paramount'),(8825,'Paramyxoviridae'),(2415,'paraquat'),(59447,'pararetrovirus'),(59460,'pararetroviruse'),(74328,'parasite'),(366346,'parasitemia'),(74938,'parasitic'),(338131,'parasitize'),(6595,'parasympathetic'),(27370,'Pardigon'),(7732,'parenchyma'),(37029,'parenchymal'),(19725,'parent'),(13581,'parental'),(41544,'parenthesis'),(97867,'Parida'),(17151,'Paris'),(348658,'Parish'),(96023,'Parity'),(77252,'Parker'),(32452,'Parkinson'),(73982,'Parliamentary'),(49035,'Parliaments'),(92879,'parr'),(320167,'Parren'),(21480,'parse'),(21704,'parser'),(60237,'parsimonious'),(338689,'parsimony'),(27376,'Parsley'),(6562,'part'),(305002,'partake'),(291,'partial'),(15542,'partially'),(17247,'participant'),(3898,'participate'),(3909,'participation'),(4175,'particle'),(15,'particu'),(9419,'particular'),(51388,'particularity'),(18569,'particularly'),(3548,'particulate'),(51820,'partition'),(334213,'partitioning'),(4706,'partly'),(31931,'partner'),(72340,'Partnership'),(104143,'Parton'),(108726,'party'),(21484,'pass'),(28469,'passage'),(48707,'passenger'),(18559,'passive'),(21321,'passively'),(22385,'password'),(21952,'past'),(368850,'paste'),(11271,'Pasternak'),(109253,'Pasteur'),(102777,'patch'),(48517,'patchwork'),(5603,'patchy'),(310966,'Patel'),(90983,'Patent'),(96161,'patented'),(38177,'path'),(576,'pathogen'),(5936,'pathogenesis'),(35381,'pathogenic'),(36132,'pathogenicity'),(318838,'Pathogens'),(9785,'pathognomonic'),(6556,'pathologic'),(5456,'pathological'),(312294,'pathologically'),(57998,'pathologist'),(3375,'pathology'),(324958,'pathophysiologic'),(3002,'pathophysiological'),(10420,'pathophysiology'),(2661,'pathway'),(104,'patient'),(4046,'pattern'),(72258,'Patterson'),(111372,'paucity'),(338694,'PAUP'),(13631,'pause'),(62955,'paused'),(14016,'pausing'),(110623,'Pauwels'),(58858,'paw'),(73384,'pay'),(73950,'payment'),(32454,'Payne'),(356473,'PBK'),(380663,'PBL'),(145401,'pBlu'),(145387,'pBluescript'),(116755,'pbmc'),(26856,'PBS'),(303976,'pbsa'),(329243,'PBST'),(2835,'pc'),(79366,'pcagap'),(79367,'pcaggp'),(135194,'PCB'),(355165,'pcc'),(355086,'PCCs'),(77394,'pccu'),(105246,'pCI'),(356472,'PCNA'),(344324,'PCP'),(3677,'PCR'),(77364,'pcsu'),(49472,'pCYT'),(305817,'Pd'),(91617,'PDB'),(377390,'pdc'),(377255,'pDCs'),(316302,'pdf'),(301092,'PDGF'),(357816,'PDI'),(360541,'PE'),(48286,'peace'),(10155,'peak'),(316902,'peaked'),(115782,'peanut'),(52135,'Pearson'),(53885,'peculiar'),(14202,'Pedersen'),(17177,'PEEP'),(74762,'peer'),(99470,'PEG'),(56816,'pegfp'),(23996,'Pegg'),(132652,'pegylate'),(132995,'pegylation'),(97772,'PEI'),(132952,'peicomplexe'),(132718,'peicomplexed'),(98984,'PEIoligo'),(26857,'pellet'),(33538,'pellete'),(98334,'pelletize'),(336268,'Pellino'),(358504,'Peltoniemi'),(86687,'penalty'),(134667,'pendrin'),(100377,'penetrate'),(49747,'penetration'),(43359,'penicillin'),(125932,'Peninsula'),(20975,'Pennsylvania'),(305723,'pentafluorophenyl'),(330184,'pentamer'),(11546,'pentanucleotide'),(366561,'pentapeptide'),(19131,'people'),(46163,'pepper'),(46052,'peptidase'),(3808,'peptide'),(366089,'Peptides'),(35267,'PeptideSearch'),(366383,'peptidespecific'),(336416,'peptidoglycan'),(26370,'peptidyl'),(96251,'perceive'),(25627,'percent'),(14191,'percentage'),(51982,'percentile'),(72301,'perception'),(26869,'perchloric'),(380676,'PerCp'),(303894,'Perdew'),(313210,'perennial'),(42299,'perfect'),(13835,'perfectly'),(323331,'perforation'),(379402,'perforin'),(71,'perform'),(23474,'performance'),(328217,'performing'),(58123,'perfuse'),(7561,'perfusion'),(34294,'peri'),(58483,'periacinar'),(5618,'peribronchial'),(8850,'peribronchiolar'),(732,'pericarditis'),(33754,'pericentrin'),(58705,'periductal'),(108112,'peril'),(109176,'perineural'),(57737,'perineurium'),(345953,'periocular'),(113,'period'),(79553,'periodate'),(73334,'periodic'),(352399,'periodically'),(21111,'periodicity'),(15299,'perioperative'),(133627,'periovulatory'),(6802,'peripheral'),(33987,'periphery'),(7736,'perivascular'),(45995,'periwinkle'),(14291,'Perkin'),(149461,'Perl'),(29743,'permanent'),(75421,'permanently'),(5549,'permeability'),(34892,'permeabilization'),(358651,'permeabilize'),(84007,'permissive'),(3932,'permit'),(127795,'permutation'),(127766,'permute'),(149124,'pernicious'),(52115,'pernix'),(2614,'peroxidase'),(17975,'peroxidation'),(17704,'peroxide'),(358510,'peroxiredoxin'),(2097,'peroxynitrite'),(366591,'peroxysome'),(133401,'perpendicular'),(6797,'perpetuate'),(25874,'perplexing'),(336578,'Perrimon'),(320508,'Perry'),(300740,'persist'),(14358,'persistence'),(18740,'persistent'),(67565,'persistently'),(646,'person'),(5791,'personal'),(95203,'personally'),(73766,'personnel'),(31868,'perspective'),(24151,'Persson'),(307317,'persuasively'),(58666,'pertain'),(124598,'Perth'),(53989,'pertinence'),(53798,'pertinent'),(26518,'perturb'),(9582,'perturbation'),(9465,'pertussis'),(90669,'perusal'),(310558,'pervade'),(335956,'pervasive'),(79074,'Pesole'),(38787,'pessimism'),(40262,'pessimistic'),(337959,'pest'),(47499,'pesticide'),(108275,'pestilence'),(339192,'pestis'),(47779,'Pestivirus'),(47783,'pestiviruse'),(118550,'petechiae'),(76719,'petitively'),(64524,'pETMutS'),(49452,'petri'),(305225,'Petroleum'),(36448,'Petten'),(338268,'Pettis'),(131147,'petunia'),(366563,'PEXEL'),(339872,'Pezizomycotina'),(365655,'pf'),(92644,'Pfaffl'),(45589,'Pfam'),(104192,'pFastBac'),(305882,'PFP'),(366733,'pftool'),(9948,'pfu'),(66123,'pg'),(146678,'pGEX'),(64720,'pGFPuv'),(128388,'PGM'),(336477,'PGRP'),(3066,'pH'),(52527,'phage'),(2103,'phagocyte'),(2638,'phagocytic'),(4455,'phagocytosis'),(34876,'phalloidin'),(370351,'pharmaceu'),(72413,'Pharmaceutical'),(99762,'Pharmacia'),(2208,'pharmacokinetic'),(2173,'pharmacological'),(329805,'pharmacologically'),(74946,'pharmacology'),(95724,'pharmacotherapy'),(101479,'pharmacy'),(301149,'Pharmingen'),(27,'pharyngitis'),(3943,'phase'),(26418,'Phe'),(14216,'phenol'),(336298,'phenoloxidase'),(8272,'phenomenon'),(19794,'phenotype'),(25714,'phenotypic'),(133958,'phenoxyethanol'),(305820,'phenyl'),(64575,'phenylmethlysulfonyl'),(356742,'phere'),(339295,'PHI'),(101360,'philosophy'),(64166,'PhiX'),(372436,'Phlebotomus'),(94301,'phlegm'),(43931,'phoenix'),(73844,'PHOFAs'),(17842,'phorbol'),(6461,'phosholipase'),(3165,'phosphatase'),(26789,'phosphate'),(6458,'phosphatidyl'),(3906,'phosphatidylinositol'),(35702,'phospho'),(305495,'phosphodiester'),(6902,'phosphodiesterase'),(11514,'phosphoimager'),(5196,'phospholipase'),(56697,'phosphor'),(85420,'phosphoramidate'),(85570,'phosphoramidite'),(62677,'phosphorimager'),(330698,'phosphorothioate'),(24000,'phosphorylate'),(2896,'phosphorylation'),(327594,'phosphorylome'),(331876,'phosphothioate'),(30394,'phosphotyrosine'),(76907,'Photo'),(99362,'photograph'),(36554,'photon'),(60710,'photoperiod'),(301254,'Photoshop'),(368595,'PHP'),(146179,'phs'),(305845,'phthalic'),(141164,'phycoerythrin'),(54632,'phycoerythrinstreptavidin'),(338698,'PHYLIP'),(9916,'phylogenetic'),(46581,'phylogenetically'),(45714,'phylogeny'),(339416,'phylogram'),(340581,'Phylograms'),(23802,'Physarum'),(508,'physical'),(102861,'physically'),(15133,'physician'),(365706,'physico'),(149831,'physicochemical'),(117331,'physics'),(9016,'physiologic'),(2304,'physiological'),(351310,'physiologically'),(15467,'Physiology'),(377513,'physiopathology'),(124374,'physiotherapy'),(340401,'phytopathogen'),(3907,'PI'),(367190,'piag'),(19972,'Piantadosi'),(336686,'PIAS'),(121041,'pic'),(46216,'piccolo'),(47566,'pick'),(66552,'Pico'),(99876,'PicoGreen'),(124528,'Picornaviridae'),(94720,'Picornaviruses'),(17356,'picture'),(41751,'pie'),(29978,'piece'),(30231,'Pierce'),(4041,'pig'),(92905,'pigment'),(149245,'pigmentosum'),(309584,'pile'),(72265,'pillar'),(23576,'pilot'),(27018,'Pin'),(56647,'Pinpoint'),(8328,'pio'),(115670,'pioneer'),(149253,'pioneering'),(87247,'Pipeline'),(87445,'Pipelines'),(87424,'pipelinesthe'),(34884,'PIPES'),(380495,'pipete'),(109254,'pipette'),(129011,'pit'),(16261,'Pittsburgh'),(25441,'PIU'),(9289,'pivotal'),(21966,'PIX'),(51522,'pixel'),(62190,'pJDRC'),(327618,'PKA'),(3569,'place'),(16620,'placebo'),(57623,'placement'),(74827,'Plague'),(57957,'plain'),(17230,'plan'),(115356,'planar'),(117229,'plane'),(23361,'planned'),(309889,'planner'),(42601,'planning'),(20310,'plant'),(59492,'planta'),(58201,'plantar'),(101057,'plantation'),(60752,'plantlet'),(9945,'plaque'),(6934,'plasma'),(377252,'plasmacytoid'),(25122,'plasmid'),(52065,'plasmidic'),(50809,'plasmidless'),(83928,'plasminogen'),(366527,'plasmodb'),(337793,'plasmodium'),(103521,'plasmon'),(10675,'plastic'),(27816,'plate'),(62462,'plateau'),(306624,'plateaue'),(6509,'platelet'),(22029,'platform'),(331908,'plating'),(104199,'Platinum'),(38393,'plausible'),(8016,'play'),(148365,'player'),(75330,'pleased'),(132740,'pleiotrophin'),(9177,'pleiotropic'),(147668,'pleiotropically'),(904,'pleural'),(701,'pleurisy'),(901,'pleuritic'),(7444,'plexiform'),(36169,'PLI'),(318837,'PLoS'),(22401,'plot'),(13185,'Plotch'),(22498,'plotting'),(380714,'Plough'),(301085,'PLP'),(90878,'plug'),(310325,'pluralistic'),(7526,'pluripotent'),(10976,'plus'),(140459,'PLx'),(54381,'pm'),(83940,'PMA'),(84703,'pmadifferentiated'),(80273,'pMagH'),(83977,'pmainduce'),(71637,'pmal'),(60651,'pMark'),(80269,'pMatH'),(80550,'pMatHN'),(80267,'pMatNH'),(84005,'PMAtreated'),(64550,'pmbp'),(128438,'PMG'),(344318,'PML'),(76300,'pmol'),(31052,'PMSF'),(72322,'pneumococcal'),(323387,'pneumococcus'),(5418,'Pneumocystis'),(5180,'pneumocyte'),(189,'Pneumofast'),(14,'pneumonia'),(2,'pneumoniae'),(626,'pneumonic'),(310745,'pneumonitis'),(339444,'pneumophila'),(702,'pneumothorax'),(8827,'pneumoviru'),(9295,'pneumovirus'),(8847,'pneumoviruse'),(367141,'PNG'),(113691,'PNOC'),(148921,'pocket'),(97510,'POCT'),(3239,'point'),(139540,'pointer'),(93309,'poisoning'),(317010,'Poisson'),(71480,'Pol'),(118736,'Poland'),(103781,'polar'),(100087,'polarion'),(312689,'polarity'),(133935,'polarize'),(133905,'polarized'),(326645,'polderman'),(345877,'pole'),(101696,'police'),(42803,'policy'),(73090,'polio'),(124557,'poliomyelitis'),(79694,'poliovirus'),(72377,'political'),(74287,'politician'),(27058,'Pollard'),(18925,'pollen'),(96647,'polling'),(18487,'pollution'),(16461,'poly'),(363339,'polyA'),(30093,'polyacrylamide'),(27187,'polyadenylate'),(23589,'polyamine'),(135198,'polyaromatic'),(43508,'polybrene'),(23803,'polycephalum'),(135195,'polychlorinated'),(124790,'polyclinic'),(27612,'polyclonal'),(97771,'polyethylenimine'),(131767,'polyfection'),(104184,'polyhedrin'),(360588,'polyhis'),(97762,'polymer'),(56,'polymerase'),(100014,'polymeric'),(4869,'polymerization'),(35162,'polymerize'),(122595,'polymicrobial'),(114885,'polymorphic'),(2302,'polymorphism'),(124163,'polymyxin'),(148905,'polynucleotide'),(148958,'polynucleotidebinde'),(9397,'polypeptide'),(339359,'Polyphaga'),(108954,'polyphenol'),(109826,'polyphenolic'),(132216,'polyplexe'),(306836,'polyposis'),(109251,'polypropylene'),(45585,'polyprotein'),(70416,'polyproteinencoding'),(27356,'polypyrimidine'),(147744,'polyribosome'),(4414,'polysaccharide'),(76171,'polysemous'),(136269,'polysialoglycoprotein'),(109366,'polystyrene'),(30524,'polyubiquitin'),(30696,'polyubiquitinate'),(30660,'polyubiquitination'),(322425,'polyvalent'),(145044,'polyvinylidenfluorid'),(24179,'pombe'),(148426,'Pomerantz'),(325317,'pon'),(147499,'Ponzano'),(21730,'pool'),(312972,'pooled'),(131506,'pooling'),(8315,'poor'),(14393,'poorly'),(353147,'pop'),(117904,'Popper'),(101053,'poppy'),(346572,'popula'),(72188,'popular'),(18656,'popularity'),(94963,'popularize'),(91742,'populate'),(52332,'populated'),(670,'population'),(73052,'populist'),(100165,'populous'),(13217,'porcine'),(148391,'pore'),(101001,'porous'),(338638,'port'),(21601,'Portability'),(73900,'portfolio'),(13787,'portion'),(58904,'Portocarrero'),(19543,'portopulmonary'),(312862,'Portulaca'),(10701,'pose'),(79255,'posi'),(61578,'posit'),(4149,'position'),(329740,'positional'),(363581,'positioned'),(13420,'positioning'),(107,'positive'),(5710,'positively'),(27127,'positivesense'),(36797,'positivity'),(325925,'positron'),(14059,'possess'),(3039,'possibility'),(274,'possible'),(2662,'possibly'),(7585,'post'),(2319,'postcapillary'),(325788,'postconcussion'),(123420,'poster'),(93606,'posterior'),(103753,'postexposure'),(301057,'postfixation'),(335910,'postgenomic'),(346144,'postherpetic'),(34793,'postinfection'),(128827,'postinoculation'),(57106,'postmeiotic'),(66241,'postmortem'),(19676,'postnatal'),(14710,'postoperative'),(325520,'postoperatively'),(308827,'postpone'),(58339,'postpre'),(124084,'postpyloric'),(2320,'postsinusoidal'),(69346,'postslippage'),(140921,'postsynthesis'),(131170,'posttranscriptional'),(303369,'posttranscriptionally'),(11468,'posttransfection'),(339135,'posttranslational'),(3141,'postulate'),(371893,'postvaccination'),(134890,'postvitellogenesis'),(133551,'postvitellogenic'),(6466,'potassium'),(11714,'potato'),(320828,'poten'),(4744,'potency'),(2487,'potent'),(2309,'potential'),(2090,'potentially'),(372996,'potentiate'),(102399,'potently'),(46183,'Potexvirus'),(100369,'potion'),(75443,'Potsdam'),(120894,'poultry'),(95325,'poverty'),(48214,'powder'),(301021,'powdered'),(17235,'power'),(35479,'powerful'),(31593,'powerfully'),(74830,'PowerPoint'),(48892,'Powers'),(357878,'Powis'),(73107,'pox'),(33494,'poxvirus'),(11216,'pp'),(355653,'PPI'),(51177,'ppiase'),(356187,'ppisignificant'),(17994,'ppm'),(330573,'PPV'),(317424,'Pr'),(95167,'practical'),(10879,'practically'),(15150,'practice'),(309417,'practiceThe'),(308194,'practise'),(37255,'practitioner'),(101011,'pragmatic'),(48086,'praise'),(340842,'Prall'),(344192,'pRc'),(38820,'prcc'),(7273,'pre'),(48895,'preamble'),(21618,'precaution'),(16494,'precede'),(11685,'precedent'),(125534,'preceed'),(28355,'precipitable'),(97780,'precipitant'),(5395,'precipitate'),(28810,'precipitation'),(3182,'precise'),(11263,'precisely'),(120980,'precision'),(15507,'preclinical'),(53066,'preclude'),(6011,'precocious'),(20340,'preconditioning'),(7256,'preconstriction'),(78242,'Precup'),(30567,'precursor'),(340148,'predate'),(325466,'predecessor'),(66596,'predefine'),(96864,'predetermine'),(2131,'predict'),(334587,'predictability'),(17201,'predictable'),(67482,'predictably'),(376238,'predicted'),(376762,'predicting'),(12163,'prediction'),(15044,'predictive'),(15037,'predictor'),(75527,'predilection'),(5507,'predispose'),(322566,'predisposing'),(19148,'predisposition'),(18880,'prednisolone'),(345455,'prednisone'),(3831,'predominance'),(49369,'predominant'),(3865,'predominantly'),(7251,'predominate'),(111735,'predominately'),(147453,'preexist'),(372399,'preexposure'),(125839,'Prefecture'),(49184,'prefer'),(49202,'preferable'),(370197,'preferably'),(12105,'preference'),(29107,'preferential'),(4371,'preferentially'),(2810,'preferred'),(101900,'pregnant'),(56788,'prehybridize'),(4720,'preincubate'),(362412,'preincubation'),(19581,'preinduction'),(2671,'preliminary'),(410,'premature'),(570,'prematurely'),(42852,'prematurity'),(320642,'Premier'),(48468,'premise'),(301712,'premyelinate'),(301833,'premyelinating'),(19674,'prenatal'),(303295,'preoligodendrocyte'),(325511,'preoperative'),(325531,'preoperatively'),(86802,'preorganization'),(136446,'preovulaory'),(134743,'preovulatory'),(42814,'prep'),(2984,'preparation'),(366941,'preparative'),(26757,'prepare'),(60751,'prepared'),(72366,'preparedness'),(59166,'prepub'),(312562,'prerequisite'),(765,'preschool'),(365142,'prescreene'),(117971,'prescribe'),(72327,'prescribed'),(21204,'prescription'),(2102,'presence'),(477,'present'),(799,'presentation'),(91255,'Presenting'),(57259,'presently'),(303569,'preservation'),(21933,'preserve'),(92973,'presmolt'),(16834,'press'),(286,'pressure'),(96013,'Preston'),(14790,'prestorage'),(2076,'presumably'),(3411,'presume'),(80931,'presumption'),(307477,'presuppose'),(307486,'presupposition'),(317167,'presymptomatic'),(46213,'presynaptic'),(320450,'pretreate'),(9473,'pretreatment'),(360653,'Preuss'),(20829,'prevail'),(23035,'prevalence'),(23179,'prevalent'),(303354,'prevalently'),(5822,'prevent'),(96632,'preventability'),(96823,'preventable'),(370576,'preventative'),(21014,'Prevention'),(72045,'preventive'),(107276,'preview'),(2991,'previous'),(573,'previously'),(62484,'prewarmed'),(61642,'PRF'),(75012,'price'),(75238,'pricing'),(367066,'prick'),(2042,'primarily'),(2553,'primary'),(148773,'primase'),(90368,'primate'),(2109,'prime'),(10284,'primer'),(380118,'priming'),(136759,'primitive'),(145056,'PRIMM'),(64925,'Princeton'),(69200,'principal'),(18145,'principally'),(39113,'principle'),(10601,'print'),(14549,'prion'),(5229,'prior'),(118318,'priori'),(72508,'Priorities'),(308801,'prioritise'),(307251,'prioritize'),(72187,'priority'),(67088,'prism'),(14288,'prismô'),(100866,'prison'),(25504,'Pristioncus'),(48072,'privacy'),(21459,'private'),(96202,'privatization'),(108009,'privilege'),(373937,'privileged'),(355484,'PRL'),(70082,'prlnull'),(355478,'Prlr'),(104172,'prM'),(102333,'PRNT'),(2049,'pro'),(350360,'proactive'),(350252,'proactively'),(58983,'proalgesic'),(112068,'probabilistic'),(16247,'probability'),(52985,'probable'),(2665,'probably'),(10334,'probe'),(34879,'Probes'),(354462,'probeset'),(357006,'probesetprotein'),(326886,'probing'),(2205,'problem'),(87914,'problematic'),(107402,'problematize'),(308330,'proc'),(327289,'ProCAT'),(371467,'proccess'),(136784,'procedural'),(307542,'procedurally'),(14512,'procedure'),(8679,'proceed'),(78578,'proceeding'),(197,'process'),(91024,'Processed'),(309723,'Processes'),(21528,'processing'),(335484,'procession'),(61845,'processive'),(360734,'processivity'),(343815,'Procheck'),(136019,'procoagulant'),(307935,'procure'),(141183,'prod'),(137191,'Prodroma'),(38993,'prodromal'),(5307,'produce'),(49128,'producer'),(2048,'product'),(2075,'production'),(18232,'productionmodulation'),(852,'productive'),(83250,'productively'),(70608,'productivity'),(43593,'Products'),(84339,'proenzyme'),(107209,'profession'),(20624,'professional'),(34807,'Professor'),(8517,'profibrotic'),(120656,'proficiency'),(238,'profile'),(65969,'profiling'),(100994,'profitable'),(17812,'profound'),(345694,'profoundly'),(300777,'progenitor'),(59217,'Progeny'),(114557,'prognosis'),(115712,'prognostic'),(321,'program'),(72951,'programmatic'),(14821,'programme'),(143026,'programmed'),(22059,'programming'),(10742,'progress'),(335073,'progressing'),(2862,'progression'),(7061,'progressive'),(33122,'progressively'),(101151,'prohibit'),(100217,'prohibition'),(73292,'prohibitively'),(4905,'proinflammatory'),(21074,'Project'),(32603,'projection'),(51272,'prokaryote'),(11659,'prokaryotic'),(355494,'prolactin'),(5668,'proliferate'),(2902,'proliferation'),(6990,'proliferative'),(85590,'Proligo'),(24361,'proline'),(337493,'prolinerich'),(306753,'prolinol'),(14960,'prolong'),(324276,'prolongation'),(9668,'prolonged'),(4113,'prolyl'),(127129,'prom'),(74880,'ProMed'),(26964,'Promega'),(74073,'prominence'),(2953,'prominent'),(108766,'prominently'),(47563,'promiscuous'),(10200,'promise'),(20551,'promising'),(83909,'promonocytic'),(2733,'promote'),(6105,'promoter'),(72052,'promotion'),(798,'prompt'),(42461,'promptly'),(358595,'promyelocytic'),(16222,'prone'),(24633,'pronounce'),(109691,'pronounced'),(2583,'proof'),(368090,'proofread'),(78139,'proofreading'),(44868,'prooxidant'),(29685,'propagate'),(121761,'propagated'),(60260,'propagation'),(312719,'propagative'),(305804,'propanoate'),(140933,'propanol'),(32559,'propel'),(366471,'propensity'),(6238,'propeptide'),(13419,'proper'),(13157,'properly'),(2485,'property'),(52550,'prophage'),(337675,'prophenoloxidase'),(36119,'prophylactic'),(122431,'prophylaxis'),(56554,'propidium'),(305821,'propionic'),(149117,'propitious'),(337676,'proPO'),(324762,'propofol'),(3835,'proportion'),(48078,'proportional'),(308656,'proportionality'),(71422,'proportionally'),(309083,'proportionate'),(317753,'proportionately'),(49022,'proposal'),(3152,'propose'),(355858,'proposition'),(10631,'proprietary'),(85415,'propynylated'),(73495,'proscriptive'),(66299,'prospect'),(16288,'prospective'),(23471,'prospectively'),(96020,'prosperity'),(7500,'prostacyclin'),(6697,'prostaglandin'),(2743,'prostanoid'),(8028,'prostate'),(23913,'prostatic'),(135131,'prosthetic'),(100947,'prostitution'),(134111,'Prot'),(694,'protean'),(5193,'protease'),(30657,'proteasomal'),(30547,'proteasome'),(361882,'proteasomeenriche'),(8087,'protect'),(9646,'protection'),(2334,'protective'),(90815,'Protégé'),(2537,'protein'),(18938,'proteinase'),(3863,'proteinosis'),(135368,'proteinS'),(43597,'Proteintech'),(339560,'Proteobacteria'),(30729,'ProteoLink'),(301083,'proteolipid'),(30642,'proteolysis'),(5438,'proteolytic'),(32225,'proteome'),(115482,'proteomic'),(23652,'proteosome'),(17257,'protocol'),(132205,'proton'),(132088,'protonable'),(132099,'protonation'),(60117,'protoplast'),(19494,'protoporphyrin'),(22919,'prototype'),(83077,'prototypic'),(304265,'prototypical'),(371257,'protozoan'),(369428,'protract'),(370715,'protraction'),(342762,'protrude'),(362024,'protrusion'),(11442,'prove'),(380253,'Provence'),(2906,'provide'),(17427,'Providence'),(21209,'provider'),(75104,'province'),(101265,'provincial'),(321124,'provinciallevel'),(147936,'provirus'),(16234,'provision'),(125223,'provisional'),(2832,'provocation'),(47591,'provoke'),(16831,'PROWESS'),(25330,'proximal'),(3608,'proximally'),(13693,'proximity'),(138286,'proxy'),(141256,'Prozyme'),(149151,'PRRs'),(50578,'prtb'),(50806,'prtp'),(78715,'prymidine'),(303941,'ps'),(98208,'psa'),(123056,'PSB'),(49473,'psec'),(6972,'Pseduomonas'),(62078,'pseudo'),(340275,'pseudogene'),(4784,'pseudohyphae'),(24087,'pseudoknot'),(122567,'Pseudomonas'),(62785,'pseudopseudoknot'),(34094,'pseudorabie'),(322453,'pseudostratifie'),(303505,'Pseudouridine'),(136270,'PSGP'),(45564,'PSI'),(60646,'PstI'),(74011,'psycho'),(95318,'psychological'),(742,'psychosis'),(140891,'PT'),(366883,'ptarget'),(27360,'PTB'),(122988,'PTC'),(362791,'PTEN'),(47277,'PTGS'),(132741,'PTN'),(113687,'PTPRF'),(303913,'PTRAJ'),(75390,'pub'),(20601,'public'),(118687,'publica'),(9926,'publication'),(72652,'publicity'),(72081,'publicly'),(76,'publish'),(68244,'PubMed'),(120182,'Puerto'),(58112,'Puig'),(28710,'pull'),(7776,'pulmonale'),(2843,'pulmonary'),(105146,'pulse'),(83827,'pump'),(124484,'punctate'),(46928,'punctuated'),(346275,'puncture'),(337704,'pupae'),(29819,'purchase'),(96021,'Purchasing'),(15047,'purdy'),(37494,'pure'),(140803,'Puregene'),(54266,'purely'),(24408,'puri'),(49191,'purification'),(2864,'purified'),(3672,'purify'),(43009,'purine'),(65038,'purity'),(131150,'purple'),(344712,'purport'),(20898,'purpose'),(321089,'purposefully'),(340710,'purpuric'),(359235,'pursuant'),(72520,'pursue'),(95299,'pursuit'),(122829,'purulence'),(324061,'purulent'),(106543,'purveyor'),(307921,'purview'),(72482,'push'),(99427,'put'),(3760,'putative'),(134968,'putatively'),(23670,'putrescine'),(78606,'pütz'),(147514,'puzzle'),(13203,'puzzling'),(45965,'pv'),(71614,'pvdf'),(8822,'PVM'),(329992,'PWM'),(343198,'PxIxN'),(120781,'pyelonephritis'),(69889,'pyes'),(147383,'pylori'),(345638,'pyomyositis'),(325818,'pyramidal'),(26185,'pyrimidine'),(52230,'pyrococcus'),(99979,'pyrophosphate'),(362814,'pyrosequence'),(144738,'pyrrolysine'),(326544,'pyruvate'),(631,'Qassim'),(126462,'Qb'),(347856,'QiaAmp'),(26915,'QIAGEN'),(99734,'QIAmp'),(104848,'Qiaprep'),(64791,'QIAquick'),(100261,'Qing'),(93776,'qpcr'),(92546,'qrt'),(36394,'quadrant'),(41330,'quadruple'),(24054,'quadruplet'),(68750,'qualification'),(52678,'qualifie'),(147756,'qualified'),(21673,'qualify'),(51281,'qualitative'),(61256,'qualitatively'),(4549,'quality'),(26980,'quanti'),(347056,'quantifiable'),(44093,'quantification'),(349153,'quantifie'),(43655,'quantify'),(127941,'quantifying'),(88240,'quantile'),(11844,'quantitate'),(11420,'quantitation'),(3468,'quantitative'),(60468,'quantitatively'),(145606,'Quantitect'),(4548,'quantity'),(117518,'quantize'),(73867,'quantum'),(38437,'quarantine'),(318469,'quarantined'),(317109,'quarantining'),(89498,'quarter'),(132092,'quarternary'),(67224,'quartile'),(146823,'quasi'),(347176,'quasispecie'),(94789,'Quebec'),(337851,'queen'),(131175,'quell'),(305277,'quench'),(114323,'quencher'),(78484,'quently'),(21946,'query'),(2706,'question'),(3001,'questionable'),(95184,'questionnaire'),(21925,'queue'),(30747,'Quick'),(344175,'QuickChange'),(23046,'quickly'),(10756,'quiescent'),(75829,'Quigley'),(62239,'Quik'),(97163,'quote'),(328587,'quotient'),(3161,'ra'),(24882,'rabbit'),(126614,'rabie'),(26575,'RACE'),(77063,'racea'),(31068,'rad'),(23580,'radar'),(348772,'radial'),(2414,'radiation'),(2775,'radical'),(53541,'radically'),(78241,'radioactive'),(14231,'radioactively'),(28162,'radioactivity'),(361002,'radiochemically'),(51733,'radioduran'),(23216,'radiograph'),(271,'radiographic'),(123247,'radiographical'),(10681,'radioisotope'),(10637,'radiolabele'),(10836,'radiolabelle'),(893,'radiological'),(374,'radiologically'),(94245,'radiologist'),(22734,'radiology'),(36177,'radionuclide'),(51933,'radius'),(92844,'Radonic'),(43827,'Rads'),(69969,'raffinose'),(52775,'Ragan'),(17474,'rail'),(95670,'rain'),(133729,'rainbow'),(367011,'rainy'),(4709,'raise'),(335684,'raising'),(359625,'Raji'),(370609,'ral'),(26581,'Ralf'),(320640,'Ralph'),(362074,'rampant'),(357652,'ran'),(352161,'RAND'),(10285,'random'),(111690,'randomised'),(15267,'randomize'),(15260,'randomized'),(15314,'randomly'),(89111,'randomness'),(14527,'range'),(38817,'rank'),(75030,'ranked'),(74527,'ranking'),(9185,'rante'),(9211,'RANTES'),(37726,'Rao'),(60702,'rapa'),(16876,'rapid'),(106590,'rapidity'),(5992,'rapidly'),(42,'rare'),(217,'rarely'),(351903,'rarity'),(79902,'RAS'),(80716,'RASes'),(720,'rash'),(3430,'rat'),(337526,'ratchet'),(929,'rate'),(129034,'ratedetermine'),(23664,'ratelimite'),(53112,'rating'),(7468,'ratio'),(115220,'rational'),(48897,'rationale'),(355222,'rationality'),(106017,'Ratios'),(46217,'Rattus'),(359128,'Rauscher'),(54664,'raw'),(17046,'ray'),(9479,'Raza'),(118454,'RB'),(14305,'RBC'),(14436,'RBCs'),(28645,'rbd'),(26995,'RBDs'),(62714,'RC'),(10963,'RCNMV'),(14733,'rct'),(124868,'RD'),(13152,'RdRp'),(12151,'reach'),(3124,'react'),(58,'reaction'),(45407,'reactivate'),(35454,'reactivation'),(2092,'reactive'),(2289,'reactivity'),(17347,'read'),(348288,'readability'),(3588,'reader'),(13248,'readily'),(106281,'readiness'),(19174,'reading'),(35451,'readout'),(69229,'readthrough'),(22871,'ready'),(31072,'reagent'),(17508,'real'),(333144,'realisation'),(136555,'realism'),(39524,'realistic'),(56226,'realistically'),(38772,'reality'),(39421,'realization'),(2597,'realize'),(117338,'realm'),(20887,'Realtime'),(10319,'reamplifie'),(52014,'reanalyze'),(356546,'reannotation'),(126484,'rearrange'),(4201,'rearrangement'),(2110,'reason'),(5498,'reasonable'),(308334,'reasonableness'),(125345,'reasonably'),(20671,'reasoning'),(30814,'reassaye'),(64628,'reassemble'),(64944,'reassembly'),(109344,'reassess'),(121639,'reassessment'),(144502,'reassign'),(346547,'reassortant'),(59221,'reassortment'),(107937,'reassure'),(48465,'recall'),(83461,'recapitulate'),(72804,'recast'),(21447,'receipt'),(14317,'receive'),(328194,'receiver'),(2281,'recent'),(2296,'recently'),(53290,'recentness'),(123679,'receptacle'),(105950,'receptionist'),(309765,'receptive'),(3762,'receptor'),(319739,'receptormediate'),(54456,'recessive'),(2848,'recipient'),(113737,'reciprocal'),(308155,'reciprocity'),(17320,'recirculating'),(83219,'recoate'),(25004,'recode'),(61506,'recoding'),(79004,'recog'),(78601,'recoginition'),(46012,'recognise'),(3817,'recognition'),(107909,'recognizable'),(1985,'recognize'),(3842,'recombinant'),(361024,'recombinantly'),(11316,'recombination'),(79871,'recombinationally'),(46843,'recombine'),(23252,'recommend'),(17131,'recommendation'),(13065,'reconcile'),(108511,'reconsider'),(11126,'reconsideration'),(305779,'reconstitute'),(59099,'reconstitution'),(78870,'reconstruct'),(145334,'reconstruction'),(165,'record'),(256,'recover'),(305,'recovery'),(658,'recruit'),(5866,'recruitment'),(123224,'rectal'),(117751,'rectangle'),(303761,'rectangular'),(66448,'rectum'),(302585,'recur'),(14709,'recurrence'),(57902,'recurrent'),(22210,'Recursive'),(150096,'recursively'),(5200,'recycle'),(30646,'recycling'),(228,'red'),(27430,'reddy'),(315626,'redefine'),(309412,'redeploy'),(121862,'redesign'),(128752,'redirect'),(305717,'redissolve'),(344670,'redistribution'),(93626,'redo'),(2140,'redox'),(108235,'redress'),(2266,'reduce'),(17380,'reduced'),(358548,'reducing'),(360130,'reductant'),(135112,'reductase'),(6740,'reduction'),(114470,'reductionist'),(360105,'reductive'),(16751,'redundancy'),(22678,'Redundant'),(21940,'Redwood'),(375440,'Reed'),(14038,'Reeder'),(100935,'reeducation'),(31544,'reelin'),(100305,'reemerge'),(100312,'reemergence'),(79979,'ref'),(3589,'refer'),(13985,'reference'),(48861,'referral'),(58799,'referred'),(123674,'refill'),(86572,'refine'),(304580,'refined'),(61645,'refinement'),(327531,'refit'),(2193,'reflect'),(45032,'reflection'),(45015,'reflective'),(35247,'REFLEX'),(122472,'reflux'),(104316,'refolde'),(104240,'refolding'),(23930,'refractory'),(356582,'RefSeq'),(15685,'refuse'),(101395,'refute'),(311495,'regain'),(2396,'regard'),(29271,'regardless'),(105145,'regenerate'),(300813,'regeneration'),(301593,'regenerative'),(72428,'regime'),(14642,'regimen'),(4071,'region'),(94949,'regional'),(357086,'register'),(20968,'registration'),(36566,'regression'),(356779,'regret'),(21716,'regular'),(51065,'regularly'),(2043,'regulate'),(3565,'regulated'),(322177,'regulating'),(1998,'regulation'),(100876,'Regulations'),(2364,'regulator'),(6099,'regulatory'),(100920,'rehabilitation'),(320459,'rehydration'),(341165,'REI'),(337249,'Reichhart'),(5080,'reid'),(95238,'Reidar'),(139145,'reinfected'),(139149,'reinfection'),(119540,'reinforce'),(108431,'reinforcement'),(11171,'reinitiate'),(13647,'reinitiation'),(69750,'reintroduce'),(19814,'reintroduction'),(17384,'reintubation'),(365288,'reiterate'),(7673,'reject'),(7539,'rejection'),(100930,'relapse'),(931,'relate'),(3693,'related'),(337824,'relatedness'),(2138,'relation'),(3511,'relationship'),(2365,'relative'),(24,'relatively'),(59905,'relax'),(3098,'relaxation'),(100938,'RELC'),(5474,'release'),(338213,'relentlessly'),(18301,'relevance'),(53191,'relevancy'),(37623,'relevant'),(20420,'reliability'),(3024,'reliable'),(67533,'reliably'),(22246,'reliance'),(336040,'reliant'),(131580,'relie'),(25880,'relief'),(24748,'relieve'),(80351,'religate'),(15688,'religious'),(336431,'Relish'),(27881,'relocalization'),(27301,'relocate'),(127989,'relocation'),(48854,'reluctant'),(2530,'rely'),(10,'remain'),(4066,'remainder'),(7600,'remains'),(61076,'remark'),(18545,'remarkable'),(11535,'remarkably'),(140283,'remember'),(139345,'remembering'),(16327,'reminder'),(33303,'reminiscent'),(6744,'remodel'),(7072,'remodeling'),(341201,'remodelle'),(342952,'remodelling'),(116911,'remote'),(141704,'remotely'),(2747,'removal'),(13136,'remove'),(300755,'remyelinate'),(300689,'remyelination'),(14615,'renal'),(139376,'rename'),(35569,'render'),(17506,'renew'),(74067,'renewal'),(70078,'Renilla'),(136095,'renin'),(133864,'Rennes'),(72105,'renovation'),(126483,'reorder'),(126540,'reordering'),(33253,'reorganization'),(32537,'reorganize'),(73938,'reorient'),(83079,'Reoviridae'),(83135,'reoviru'),(83106,'reovirus'),(83076,'reoviruse'),(68588,'rep'),(73911,'repackage'),(15338,'repair'),(349610,'reparameterization'),(321996,'reparation'),(8475,'reparative'),(69903,'repdaug'),(4005,'repeat'),(14624,'repeatedly'),(138665,'repeating'),(304954,'repel'),(2413,'reperfusion'),(115645,'repertoire'),(51250,'repetition'),(7293,'repetitive'),(121788,'rephrase'),(11454,'replace'),(11559,'replacement'),(75742,'replacethe'),(80697,'replicable'),(10915,'replicase'),(5700,'replicate'),(5687,'replication'),(71916,'replicationassociate'),(126331,'replicative'),(12589,'replicon'),(33842,'repolymerize'),(373082,'reponse'),(301563,'repopulate'),(300808,'repopulation'),(2552,'report'),(120754,'reportability'),(20618,'reportable'),(3229,'reported'),(50349,'reportedly'),(28326,'reporter'),(17405,'reporting'),(144046,'reposition'),(69181,'repository'),(2722,'represent'),(49987,'representation'),(16295,'representative'),(12351,'repress'),(24655,'repressible'),(23826,'repression'),(77060,'repressive'),(24681,'repressor'),(54314,'reproduce'),(54439,'reproducibility'),(58168,'reproducible'),(49603,'reproducibly'),(38248,'reproduction'),(118226,'reproductive'),(142778,'reprogramme'),(120368,'republic'),(118727,'Republics'),(132186,'repulsion'),(22992,'request'),(357,'require'),(11204,'requirement'),(14472,'Requirements'),(84842,'requisite'),(67652,'rescale'),(12382,'rescue'),(2119,'research'),(48036,'researcher'),(133625,'researchgroup'),(109165,'resect'),(8252,'resection'),(13435,'resemble'),(55666,'resequence'),(34731,'Reserve'),(122309,'reservoir'),(368654,'reset'),(75515,'reshape'),(13953,'reside'),(304663,'residence'),(22710,'Resident'),(12823,'residual'),(2870,'residue'),(64501,'resin'),(19727,'resist'),(7245,'resistance'),(5445,'resistant'),(8474,'resolution'),(14225,'resolve'),(102579,'resonance'),(74220,'resonate'),(305851,'resorcinol'),(16332,'resource'),(96092,'resourcepoor'),(351879,'Resources'),(372335,'resp'),(113208,'respec'),(5256,'respect'),(73452,'respectable'),(107794,'respected'),(49034,'respective'),(473,'respectively'),(18478,'respiration'),(7,'respiratory'),(8082,'respond'),(96946,'respondent'),(42204,'responder'),(2146,'response'),(108084,'Responsibilities'),(48908,'responsibility'),(2740,'responsible'),(17850,'responsive'),(18866,'responsiveness'),(7809,'rest'),(22649,'restart'),(378749,'restimulation'),(7827,'resting'),(12200,'restoration'),(12169,'restore'),(61820,'restrain'),(61802,'restraint'),(3712,'restrict'),(57193,'restricted'),(22795,'restriction'),(14883,'restrictive'),(148053,'restructure'),(161,'result'),(44301,'resultant'),(61185,'Results'),(13378,'resume'),(71802,'resumption'),(48919,'resurgent'),(326618,'resuscitate'),(123471,'resuscitation'),(26792,'resuspend'),(140955,'resuspension'),(21520,'Retail'),(11958,'retain'),(96123,'retaliation'),(62047,'retardation'),(101552,'retention'),(30921,'reteste'),(48141,'rethink'),(62575,'retic'),(24883,'reticulocyte'),(4204,'reticulum'),(57473,'retina'),(750,'retinal'),(344327,'retinitis'),(149174,'retinoic'),(353183,'retire'),(74001,'retractable'),(58227,'retraction'),(22167,'retrain'),(20140,'retransplantation'),(108320,'retrenchment'),(22518,'retrieve'),(17291,'retrospect'),(624,'retrospective'),(125089,'retrospectively'),(13124,'retrotransposon'),(43384,'retroviral'),(35414,'Retrovirology'),(29216,'retrovirus'),(24037,'Retroviruses'),(45979,'return'),(325019,'reuptake'),(123468,'reusable'),(90866,'reuse'),(335672,'Reuter'),(145166,'rev'),(148372,'Revdependent'),(7656,'reveal'),(63325,'revealing'),(9129,'reversal'),(3034,'reverse'),(128718,'reversegenetic'),(112633,'reversely'),(305703,'reversephase'),(3148,'reversible'),(77655,'reversion'),(371494,'revert'),(378842,'revertant'),(116,'review'),(118440,'reviewer'),(73621,'revise'),(120567,'revised'),(101812,'revision'),(87904,'revisit'),(95556,'revolution'),(325928,'revolutionize'),(114548,'reward'),(21328,'rewrite'),(81606,'rf'),(364489,'RFLP'),(90418,'RFP'),(132653,'RGD'),(148015,'RHA'),(124867,'rhabdomyosarcoma'),(126612,'Rhabdoviridae'),(319110,'rhesus'),(72181,'rhetoric'),(115742,'rheumatoid'),(9037,'rhinitis'),(94062,'Rhinovirus'),(34496,'rhodamine'),(136650,'Rhodes'),(326399,'rHuEPO'),(17428,'RI'),(127348,'rib'),(9067,'ribavarin'),(341317,'ribbon'),(104383,'RIBI'),(43071,'riboflavin'),(70100,'RiboMax'),(131210,'ribonuclease'),(94352,'ribonucleic'),(27316,'ribonucleoprotein'),(301074,'riboprobe'),(63868,'Ribos'),(16464,'ribose'),(23583,'ribosomal'),(303612,'ribosomally'),(23614,'ribosome'),(63214,'ribosomes'),(85397,'ribozyme'),(45976,'rice'),(6992,'rich'),(301095,'Richardson'),(53775,'richness'),(339594,'Rickettsiae'),(120183,'Rico'),(51687,'rid'),(119901,'ride'),(33749,'Rieder'),(33383,'rifampicin'),(7074,'right'),(373935,'rightly'),(78704,'rigid'),(87039,'rigidity'),(71466,'rigidly'),(2977,'rigorous'),(332600,'rigorously'),(53226,'RIMD'),(66335,'RIN'),(7714,'ring'),(148963,'ringexpande'),(304599,'ringlike'),(46164,'ringspot'),(35311,'rinse'),(61941,'rip'),(131234,'RISC'),(32437,'rise'),(2852,'risk'),(107898,'risky'),(93576,'river'),(593,'Riyadh'),(33277,'RK'),(120423,'RKI'),(351440,'RLE'),(22212,'RLS'),(143434,'rlu'),(70080,'rluc'),(70298,'RLUs'),(24312,'rm'),(359136,'RMA'),(24338,'rme'),(380389,'rmIFN'),(3772,'RNA'),(131034,'RNAase'),(148021,'rnadependent'),(52705,'RNAfree'),(47305,'RNAi'),(131114,'rnaimediate'),(147631,'RNAprotein'),(338833,'rnaqueous'),(11002,'RNAs'),(26945,'RNase'),(62520,'rnaseh'),(29970,'RNasin'),(80251,'RNasine'),(86771,'RNAThe'),(351542,'RNazol'),(66472,'RNeasy'),(148055,'RNP'),(2095,'RNS'),(3143,'RNSspecific'),(32493,'Ro'),(100748,'road'),(29683,'robb'),(120420,'Robert'),(332209,'Roberts'),(120642,'Robin'),(141660,'robotic'),(37761,'robust'),(373792,'robustly'),(66649,'robustness'),(328197,'ROC'),(43570,'Roche'),(85315,'Rochester'),(128799,'rock'),(30232,'Rockford'),(22817,'rod'),(9912,'rodent'),(34839,'Rodriguez'),(20891,'RODS'),(94358,'Rohde'),(43658,'ROI'),(110621,'Roisin'),(2163,'role'),(45304,'roll'),(352270,'Rollins'),(23704,'Rom'),(27020,'Roma'),(60765,'Ronkonkoma'),(35105,'roo'),(17501,'room'),(51925,'root'),(32447,'Roper'),(42931,'ROS'),(35659,'Roscovitine'),(337208,'Ross'),(147500,'Rossi'),(61816,'rotate'),(304237,'rotation'),(62094,'rotational'),(34565,'rotavirus'),(335659,'Rothenbuhler'),(94439,'Rotterdam'),(17150,'Rouby'),(4202,'rough'),(36825,'roughly'),(685,'round'),(69724,'Rousset'),(21267,'route'),(21415,'router'),(21427,'routine'),(11369,'routinely'),(77379,'row'),(343494,'Rowe'),(98054,'ROX'),(148007,'Roy'),(338238,'royal'),(96164,'royalty'),(336218,'Royet'),(46113,'rp'),(31058,'rpm'),(109274,'RPMI'),(338590,'RPS'),(139275,'rpx'),(80252,'RQ'),(147960,'RRE'),(52122,'rrna'),(52210,'rrnas'),(349176,'rS'),(11478,'rst'),(606,'RSV'),(9262,'rsvinfected'),(21075,'RSVP'),(3678,'RT'),(357888,'Rubartelli'),(70277,'RUC'),(118723,'rug'),(94328,'Ruhr'),(100689,'Ruili'),(37034,'rule'),(21978,'run'),(117426,'runaway'),(335566,'Runge'),(54745,'running'),(362827,'runningcost'),(139794,'runtime'),(135637,'rupture'),(136049,'ruptured'),(72773,'rural'),(35532,'rus'),(332215,'Russek'),(47103,'Russia'),(95232,'Ruth'),(340961,'Rutten'),(45370,'RxxxxxR'),(62191,'ry'),(324861,'SAA'),(9458,'saadi'),(308338,'Sabin'),(3879,'saccharide'),(23806,'saccharomyce'),(69816,'Saccharomyces'),(66458,'Sacchi'),(26559,'saci'),(120178,'sack'),(58128,'sacrifice'),(313959,'Sacristán'),(335702,'Sadd'),(17471,'sadly'),(15628,'safe'),(69127,'safeguard'),(326650,'safely'),(14517,'safety'),(75134,'saharan'),(4649,'Sahly'),(34178,'Saib'),(23788,'Saito'),(125134,'Saitou'),(23695,'Sakata'),(343249,'Sakuntabhai'),(337389,'Saldivar'),(20904,'sale'),(43425,'sali'),(53040,'salient'),(26855,'saline'),(34809,'Salisbury'),(101977,'saliva'),(312731,'salivary'),(75603,'sally'),(92549,'salmon'),(53946,'Salmonella'),(340273,'Salmonellae'),(120764,'salmonellosis'),(92560,'salmonid'),(43717,'Salt'),(56364,'salvage'),(135860,'Salvelinus'),(134028,'SAM'),(108533,'Samaritan'),(108474,'Samaritans'),(120814,'Samonella'),(2845,'sample'),(313816,'sampled'),(330405,'sampler'),(64858,'Samples'),(20733,'sampling'),(326287,'sampson'),(21974,'San'),(95935,'sanction'),(372389,'sand'),(303755,'SANDER'),(34808,'Sanders'),(32490,'Sanderson'),(371354,'sandfly'),(313973,'sandwich'),(87534,'Sanger'),(29830,'Santa'),(75774,'Santos'),(87248,'SAP'),(24194,'sapien'),(338276,'saprophyte'),(124596,'Sarawak'),(423,'sarcoidosis'),(35597,'sarcoma'),(27381,'Sarnow'),(35853,'SARS'),(139280,'SAS'),(303926,'SASA'),(136653,'Sasaki'),(13910,'satellite'),(58607,'satisfactorily'),(120278,'satisfactory'),(375643,'satisfie'),(127075,'satisfied'),(105035,'satisfy'),(376847,'satisfying'),(136432,'satisfyingly'),(312861,'sativa'),(86828,'saturate'),(295,'saturation'),(68,'Saudi'),(358655,'sav'),(367007,'savannah'),(16180,'save'),(73561,'saving'),(11130,'Sawicki'),(99094,'say'),(118432,'SB'),(358233,'SBP'),(69938,'SC'),(37679,'scab'),(37983,'scabbing'),(32044,'scaffold'),(21310,'scalability'),(87803,'Scalable'),(10812,'scale'),(327163,'scaling'),(36692,'scan'),(54643,'Scanner'),(11846,'scanning'),(373680,'scant'),(126864,'scanty'),(303131,'scar'),(83,'scarce'),(111902,'scarcity'),(303143,'scarring'),(45857,'scatter'),(16485,'scavenge'),(3020,'scavenger'),(357871,'scavenging'),(23487,'scenario'),(17242,'scepticism'),(305087,'SCH'),(72313,'schedule'),(21891,'scheduling'),(85179,'Schein'),(90601,'schema'),(35649,'schematic'),(97839,'schematically'),(20694,'scheme'),(305801,'Schemes'),(380713,'Schering'),(75875,'Schimmel'),(366637,'schizont'),(325004,'schizophrenia'),(25919,'schizosaccharomyce'),(24178,'Schizosaccharomyces'),(32440,'Schmelz'),(335623,'Schmid'),(360309,'Schneider'),(101414,'scholar'),(307769,'scholarship'),(77897,'Schön'),(654,'school'),(343289,'schroder'),(78516,'Schultz'),(343290,'Schumann'),(75782,'Schunck'),(303036,'Schwann'),(79202,'Sci'),(303040,'sciatic'),(6276,'SCID'),(43650,'Science'),(319,'Sciences'),(29774,'Scienti'),(95016,'scientifi'),(17446,'scientific'),(35843,'scientist'),(28358,'scintillation'),(36493,'scintillator'),(14186,'Scion'),(7424,'scleroderma'),(300679,'sclerosis'),(18280,'scope'),(72495,'scoping'),(15472,'score'),(354090,'scorecard'),(16532,'scoring'),(348696,'Scotland'),(87853,'Scott'),(379176,'SCP'),(29947,'scrape'),(58231,'scratching'),(22416,'screen'),(10236,'screening'),(141698,'script'),(92422,'scrutinize'),(308756,'scrutiny'),(45767,'sd'),(124150,'SDD'),(22047,'SDK'),(30864,'SDS'),(43901,'SE'),(351059,'sea'),(135353,'seabream'),(83085,'seadornaviruse'),(43161,'seal'),(320920,'seamless'),(22282,'search'),(75113,'season'),(18921,'seasonal'),(313206,'seasonality'),(92873,'seawater'),(49347,'sec'),(50999,'SecDF'),(563,'second'),(3054,'secondary'),(32839,'secondly'),(96333,'secrecy'),(73983,'Secretary'),(3496,'secrete'),(125,'secretion'),(366868,'secretomep'),(3622,'secretory'),(44688,'section'),(96654,'sectional'),(72475,'sector'),(73063,'sectoral'),(376873,'secular'),(21457,'secure'),(95032,'security'),(326375,'sedate'),(122276,'sedation'),(124430,'sedative'),(33456,'sediment'),(361522,'sedimentable'),(33350,'sedimentation'),(137,'see'),(43676,'seed'),(109437,'seeding'),(15829,'seek'),(41829,'seemingly'),(21756,'segment'),(58794,'segmental'),(52163,'segmentation'),(59463,'segmented'),(30682,'segregate'),(136534,'segregation'),(150652,'seijffer'),(150329,'Seijffers'),(136593,'SEIR'),(48469,'seize'),(332076,'seizure'),(354408,'seldom'),(22606,'select'),(935,'selection'),(2908,'selective'),(64404,'selectively'),(6486,'selectivity'),(78342,'selenocysteine'),(2949,'selenoprotein'),(5218,'self'),(86395,'selfcomplementary'),(106024,'selfdescribe'),(139256,'selfexplanatory'),(149128,'selfish'),(96173,'sell'),(133407,'SEM'),(90930,'semantic'),(48156,'semen'),(114153,'semi'),(139268,'semicolon'),(308495,'seminal'),(17489,'seminar'),(56402,'seminiferous'),(10080,'semipermissive'),(10249,'semiquantitative'),(316640,'semirural'),(352617,'semistructured'),(342922,'semivariable'),(21655,'send'),(95183,'sending'),(31570,'senile'),(72620,'senior'),(339161,'SENP'),(339709,'SENPs'),(58140,'sensation'),(11230,'sense'),(51986,'sensibility'),(110632,'sensible'),(116912,'sensing'),(113767,'sensitiv'),(3156,'sensitive'),(237,'sensitivity'),(57647,'sensitization'),(18677,'sensitize'),(23587,'sensor'),(322909,'sensorineural'),(58980,'sensory'),(115264,'Sente'),(100865,'sentence'),(100247,'sentinel'),(64922,'Sep'),(374369,'separable'),(4170,'separate'),(46597,'separated'),(21359,'separately'),(4087,'separation'),(64926,'Separations'),(79232,'sephadex'),(79325,'Sepharose'),(367149,'Sepik'),(8300,'sepsis'),(6592,'septae'),(21561,'September'),(547,'septic'),(120782,'septicaemia'),(120817,'septicemia'),(10056,'sequelae'),(62667,'Sequenase'),(3694,'sequence'),(89528,'sequenced'),(363204,'Sequences'),(4035,'sequencing'),(114577,'sequential'),(30494,'sequentially'),(30672,'sequester'),(30678,'sequestosome'),(2315,'sequestration'),(26426,'Ser'),(318960,'sera'),(146927,'serendipitous'),(115443,'SEREX'),(10191,'serial'),(99848,'serially'),(21342,'series'),(35993,'serine'),(16904,'seriously'),(374691,'seroconversion'),(73100,'serogroup'),(121822,'serologic'),(54,'serological'),(346867,'serology'),(376155,'seronegative'),(108503,'seropositive'),(126647,'seropositivity'),(138322,'seroprevalence'),(16417,'serosa'),(16399,'serosal'),(325014,'serotonergic'),(324976,'serotonin'),(4647,'serotype'),(368890,'Serotypes'),(6580,'serous'),(337242,'serpin'),(312859,'serriola'),(76210,'SerRS'),(57053,'Sertoli'),(3536,'serum'),(5032,'serve'),(21533,'server'),(22009,'service'),(14533,'Services'),(22113,'Servlets'),(76004,'seryl'),(76740,'serylate'),(76231,'serylation'),(17302,'session'),(9173,'set'),(340877,'Setchfield'),(11122,'Sethna'),(145666,'setpoint'),(5589,'setting'),(109265,'settle'),(14732,'seven'),(10455,'sevenfold'),(275,'severe'),(12350,'severely'),(857,'severity'),(324759,'sevoflurane'),(96035,'sewage'),(93512,'sex'),(337224,'sexta'),(137028,'sexual'),(72690,'sexually'),(10905,'sg'),(18070,'sGC'),(367771,'SGI'),(23780,'sh'),(41882,'shaded'),(38311,'shading'),(144950,'shadow'),(31133,'shake'),(307538,'shall'),(362021,'shallow'),(342072,'sham'),(100366,'Shanghai'),(326600,'Shankaran'),(59679,'shape'),(377279,'shaping'),(10385,'share'),(100146,'sharing'),(58226,'sharp'),(74064,'sharply'),(325279,'sharshar'),(6494,'shear'),(141258,'sheath'),(946,'shed'),(36247,'shedding'),(301113,'sheep'),(45305,'sheet'),(332214,'Shelley'),(337856,'sheltered'),(348704,'Shetland'),(32456,'Shida'),(7260,'shift'),(317468,'shifted'),(127168,'shifting'),(25683,'shifty'),(53227,'Shigella'),(340276,'Shigellae'),(74738,'shigellosis'),(96695,'Shikoku'),(99884,'Shimadzu'),(75871,'Shimizu'),(336243,'Shin'),(144679,'Shine'),(344328,'shingle'),(118735,'ship'),(119542,'shipboard'),(115803,'SHIV'),(548,'shock'),(105791,'Shore'),(21941,'Shores'),(3345,'short'),(105515,'shortage'),(2185,'shortcoming'),(13428,'shorten'),(351015,'shortening'),(119364,'shortly'),(139399,'shot'),(89287,'shotgun'),(3946,'show'),(131824,'shRNAs'),(108944,'shrub'),(325816,'shrunken'),(64444,'shuffle'),(130865,'Shuffled'),(65089,'shuffling'),(101126,'shun'),(16395,'shunting'),(25820,'shut'),(27015,'shuttle'),(79230,'Shuzo'),(57221,'sialate'),(4298,'sialic'),(56378,'sialicacid'),(372736,'sialidase'),(57191,'sialoadhesin'),(56301,'sialoglycoconjugate'),(4274,'sialylate'),(124777,'Sibu'),(100361,'Sichuan'),(939,'sick'),(38777,'sickbe'),(16701,'sicker'),(417,'sickle'),(369231,'sickness'),(43487,'side'),(307463,'sideline'),(78412,'sidere'),(353521,'sidestep'),(13771,'Siegel'),(9838,'sig'),(34354,'sight'),(109062,'sighted'),(340746,'Siglecs'),(10577,'Sigma'),(270,'sign'),(3043,'signal'),(18125,'signaling'),(336442,'signalling'),(366869,'SignalP'),(20793,'signature'),(194,'Signes'),(11357,'signi'),(95039,'signifi'),(373178,'signifiacnt'),(3003,'significance'),(4925,'significant'),(3073,'significantly'),(28901,'signify'),(72229,'signpost'),(47276,'silence'),(131085,'silencing'),(24362,'silent'),(66467,'silica'),(45520,'silico'),(4882,'silicotic'),(100271,'silk'),(358312,'silver'),(13218,'simian'),(2290,'similar'),(11276,'similarity'),(489,'similarly'),(13469,'Simon'),(4373,'simple'),(439,'simplex'),(66386,'simplicity'),(99385,'simplification'),(49190,'simplifie'),(16607,'Simplified'),(32945,'simplify'),(144052,'simplistic'),(22953,'simply'),(376600,'Simpson'),(77682,'simul'),(136772,'Simula'),(38168,'simulate'),(38715,'simulated'),(37309,'simulation'),(74646,'simulator'),(3933,'simultaneous'),(22501,'simultaneously'),(326527,'simvastatin'),(338695,'Sinauer'),(46413,'Sindbis'),(138342,'sine'),(108946,'sinensis'),(124599,'Singapore'),(308342,'Singer'),(27353,'Singh'),(4208,'single'),(314997,'singlestep'),(8829,'singlestrande'),(147533,'singlestranded'),(663,'singly'),(51399,'singularity'),(123411,'sink'),(123986,'sinus'),(122300,'sinusitis'),(139173,'sinusoid'),(27040,'Siomi'),(49320,'sion'),(131616,'sir'),(131255,'siRNA'),(131512,'sirnamediate'),(131218,'sirnas'),(92440,'Sirover'),(136598,'SIRP'),(345235,'SIRS'),(122424,'Sirvent'),(136482,'SIS'),(338368,'sister'),(10982,'sit'),(2134,'site'),(149411,'SiteLight'),(6908,'situ'),(306869,'situate'),(5217,'situation'),(105476,'situational'),(115821,'SIV'),(26058,'sixteen'),(37675,'sixteenth'),(375896,'sixth'),(57837,'sixtysix'),(4752,'size'),(27172,'sized'),(365603,'sizedependent'),(22036,'sizing'),(133853,'Sizun'),(45728,'SJA'),(10223,'skeptic'),(92824,'skepticism'),(55217,'skew'),(317973,'skewed'),(137508,'skewness'),(100487,'skill'),(99648,'skillful'),(3774,'skin'),(130180,'skip'),(141760,'skirt'),(63030,'sl'),(332620,'slaughter'),(117860,'slave'),(49980,'Slayer'),(344994,'SLE'),(10676,'slide'),(139487,'slider'),(4722,'slight'),(851,'slightly'),(122350,'slime'),(61601,'slip'),(55487,'slippage'),(61554,'slippery'),(36568,'slope'),(90835,'slot'),(8978,'slow'),(44254,'slowing'),(25129,'slowly'),(146664,'slurry'),(144980,'sm'),(323407,'smad'),(333,'small'),(348709,'smallholding'),(32360,'smallpox'),(136829,'Smalltalk'),(38171,'smallworld'),(363391,'Smart'),(363472,'SmartCycler'),(31207,'smear'),(66635,'smearing'),(94512,'Smirnov'),(343332,'Smirnova'),(23918,'Smith'),(17947,'smoke'),(83442,'smokeinduce'),(6945,'smoker'),(6920,'smoking'),(92880,'smolt'),(93636,'smoltifi'),(92770,'smoltification'),(92929,'smoltifie'),(93261,'smoltified'),(93225,'smoltifye'),(3096,'smooth'),(365246,'smoothed'),(321674,'smoothly'),(2027,'smoothmuscle'),(100867,'smuggle'),(100214,'smuggling'),(372806,'Sn'),(139851,'snail'),(139327,'SNAP'),(139322,'snapshot'),(374154,'sneeze'),(100517,'sniff'),(11008,'Snijder'),(100518,'snort'),(44759,'SNP'),(44755,'SNPs'),(43631,'soak'),(334811,'SOB'),(69595,'Sobemovirus'),(354982,'socalled'),(318,'Social'),(118726,'Socialist'),(335586,'sociality'),(338500,'socialize'),(95993,'socially'),(308575,'societal'),(16974,'Society'),(72770,'socio'),(73024,'socioeconomic'),(126071,'sociological'),(336682,'SOCS'),(32398,'Sodeik'),(360156,'Soderberg'),(44866,'sodium'),(72950,'soft'),(14188,'software'),(123491,'soilage'),(334248,'sojourn'),(124491,'sol'),(312864,'Solanum'),(351284,'sole'),(13194,'solely'),(144764,'solfataricus'),(95156,'solicit'),(3942,'solid'),(307699,'solidarity'),(55642,'solidphase'),(338352,'solitary'),(70513,'solubility'),(35227,'solubilize'),(5142,'soluble'),(134669,'solute'),(21150,'solution'),(35241,'Solutions'),(303760,'solvate'),(35403,'solve'),(102778,'solvent'),(75937,'Soma'),(44749,'somatic'),(58807,'somatotopically'),(9208,'somewhat'),(26582,'Sommer'),(137436,'Song'),(64579,'sonicate'),(70247,'sonication'),(16546,'soon'),(139253,'sophisticated'),(367602,'sorbitol'),(124474,'sore'),(340876,'Soriani'),(43526,'sort'),(43531,'sorted'),(102732,'sorting'),(507,'sound'),(16208,'soup'),(2981,'source'),(48221,'South'),(100327,'southeast'),(35727,'southern'),(100337,'southwest'),(93578,'Southwestern'),(123139,'Souweine'),(118725,'Soviet'),(3523,'SP'),(24445,'SPA'),(11005,'Spaan'),(8773,'space'),(26648,'spacer'),(127549,'spacing'),(336144,'Spaetzle'),(312670,'Spain'),(5138,'span'),(74912,'spanish'),(325236,'spare'),(326405,'sparing'),(368607,'sparingly'),(138467,'sparse'),(3927,'spatial'),(4169,'spatially'),(348474,'spatio'),(348584,'spatiotemporal'),(135757,'spawning'),(73224,'speak'),(16956,'speaker'),(312822,'spearman'),(11335,'speci'),(13600,'special'),(313388,'specialisation'),(313256,'specialise'),(110290,'specialised'),(313351,'specialism'),(68528,'specialist'),(90950,'specialization'),(90694,'specialize'),(107025,'specialized'),(129227,'specially'),(139054,'specialty'),(313411,'speciation'),(456,'specie'),(42930,'Species'),(51426,'speciesspecific'),(51750,'speciestypical'),(95134,'specifi'),(137249,'specifiable'),(3256,'specific'),(2612,'specifically'),(21010,'specification'),(2207,'specificity'),(22087,'specifie'),(11566,'specify'),(36361,'speciman'),(110,'specimen'),(196,'Specimens'),(327464,'speckle'),(361757,'speckled'),(66534,'spectra'),(116919,'spectral'),(107236,'spectre'),(99882,'spectrofluorophotometer'),(361701,'spectrogram'),(35250,'spectrometer'),(33468,'spectrometry'),(66019,'spectrophotometer'),(70112,'spectrophotometrically'),(361687,'spectroscopic'),(303553,'spectroscopy'),(36553,'spectrum'),(5499,'speculate'),(7013,'speculation'),(149206,'speculative'),(38889,'speed'),(305713,'SpeedVac'),(80247,'SpeI'),(99254,'spemidine'),(96249,'spend'),(73917,'spending'),(56410,'sperm'),(56409,'spermatid'),(56406,'spermatocyte'),(23775,'spermatogenesis'),(56501,'spermatogenic'),(57321,'spermatogensis'),(56404,'spermatogonia'),(57131,'spermatozoa'),(23674,'spermidine'),(23675,'spermine'),(337199,'sph'),(51935,'sphere'),(82697,'spherule'),(122465,'sphincter'),(337175,'SPHs'),(349477,'Spiegelhalter'),(22545,'spike'),(56548,'spin'),(428,'spinal'),(326240,'spine'),(362696,'spinning'),(92212,'spirit'),(123473,'spirometer'),(94343,'spirometric'),(110576,'spirometry'),(65993,'spite'),(335671,'Spivak'),(92790,'spleen'),(378204,'splenic'),(104405,'splenocyte'),(54238,'splice'),(148507,'spliceosome'),(27004,'splicing'),(17163,'split'),(355920,'SPLOSH'),(132206,'sponge'),(16972,'sponsor'),(110705,'sponsorship'),(58775,'spontaneous'),(311526,'spontaneously'),(125395,'sporadic'),(125477,'sporadically'),(48120,'spore'),(366199,'sporozoite'),(29977,'spot'),(72419,'spotlight'),(120887,'spp'),(10562,'spread'),(137606,'spreader'),(74577,'spreadsheet'),(808,'spring'),(75831,'Sprinzl'),(8235,'sprouting'),(337158,'SPs'),(320,'spss'),(64069,'spur'),(27115,'spurious'),(121,'sputum'),(351507,'SQQLN'),(8098,'squamous'),(328,'square'),(149098,'squelch'),(332848,'SR'),(43731,'SRB'),(30397,'src'),(127355,'Srib'),(364327,'sRT'),(350142,'Ss'),(144851,'Ssa'),(358284,'SSAAA'),(362898,'SSCP'),(339959,'ssel'),(112424,'SSH'),(25917,'ssion'),(150670,'ssp'),(45609,'ssrna'),(30301,'St'),(26485,'stabilitie'),(6533,'stability'),(85786,'stabilitythe'),(9228,'stabilization'),(3893,'stabilize'),(86583,'stabilizing'),(3306,'stable'),(305193,'stablize'),(27464,'stably'),(51699,'stack'),(55998,'stacking'),(48183,'staff'),(20880,'staffing'),(2224,'stage'),(127167,'staggered'),(16535,'staging'),(69665,'Stahl'),(146,'stain'),(28103,'staining'),(303922,'stair'),(304990,'stairsteppe'),(73850,'stakeholder'),(71844,'stall'),(301142,'Stallcup'),(104421,'stan'),(14605,'stand'),(249,'standard'),(18585,'standardization'),(37070,'standardize'),(169,'standardized'),(49017,'Standing'),(37698,'standpoint'),(56710,'Stanford'),(77274,'Stanzel'),(49443,'staphylococcal'),(22912,'Staphylococcus'),(375182,'star'),(9383,'Stark'),(335693,'Starks'),(13154,'start'),(48954,'starting'),(138349,'startup'),(17910,'STAT'),(96909,'Stata'),(5929,'state'),(14400,'statement'),(20974,'States'),(310500,'static'),(106090,'Station'),(24574,'stationary'),(20852,'stationing'),(329,'statistic'),(134356,'Statistica'),(316,'Statistical'),(15017,'statistically'),(117985,'statistician'),(125215,'Statistics'),(134358,'Statsoft'),(2141,'status'),(72665,'statutory'),(15007,'stay'),(101232,'std'),(10173,'steady'),(33728,'Stearns'),(50041,'stearothermophilus'),(139088,'steep'),(40636,'steeply'),(358550,'steer'),(336874,'Steiner'),(13992,'stem'),(69569,'stemloop'),(336467,'Stenbak'),(58709,'stenosis'),(122631,'Stenotrophomonas'),(57622,'stent'),(8146,'step'),(65357,'stepwise'),(341314,'Stereo'),(343811,'stereochemical'),(301275,'Stereologer'),(301272,'stereologic'),(301717,'stereology'),(131039,'steric'),(304680,'sterically'),(101398,'sterile'),(346229,'sterility'),(318861,'sterilize'),(301245,'Sterling'),(402,'steroid'),(133603,'steroidogenenic'),(136454,'steroidogenic'),(19179,'steroidtreate'),(337810,'Stevenson'),(307621,'stewardship'),(350144,'sThis'),(137024,'stick'),(312111,'stiff'),(101884,'stigma'),(73013,'stigmatise'),(307510,'stigmatization'),(102025,'stigmatize'),(58805,'stim'),(74012,'stimulant'),(2445,'stimulate'),(69540,'stimulating'),(8656,'stimulation'),(24104,'stimulator'),(24936,'stimulatory'),(6492,'stimulus'),(108449,'stipulate'),(367397,'stir'),(341360,'stitution'),(37397,'stochastic'),(42399,'stochastically'),(334530,'stochasticity'),(32940,'stock'),(96277,'stockpile'),(309594,'stockpiling'),(28140,'Stohlman'),(128617,'stoichiometric'),(69249,'stoichiometry'),(3789,'stomach'),(34571,'stomatitis'),(120788,'stool'),(14048,'stop'),(14931,'storage'),(3621,'store'),(62676,'storm'),(100963,'story'),(69091,'straight'),(68701,'straightforward'),(349569,'straightforwardly'),(4513,'strain'),(53337,'strainspecific'),(72779,'Strait'),(10899,'strand'),(22274,'strange'),(36487,'Strasbourg'),(96926,'strata'),(30202,'Stratagene'),(30201,'stratalinker'),(66477,'Strataprep'),(142643,'strate'),(104725,'Strategene'),(72743,'strategic'),(3424,'strategy'),(376545,'stratification'),(16776,'stratified'),(112256,'stratify'),(321157,'stratum'),(27371,'Strauss'),(20825,'streak'),(73864,'stream'),(20740,'street'),(49286,'strength'),(53510,'strengthen'),(130845,'Strengths'),(79533,'strepbacterial'),(105058,'streptavidin'),(5806,'streptococcus'),(50113,'Streptomyces'),(43360,'streptomycin'),(2037,'stress'),(18365,'stressful'),(44470,'stressor'),(16905,'stretch'),(63775,'stretching'),(12486,'strict'),(41602,'strictly'),(57612,'stricture'),(345511,'stridor'),(11310,'strike'),(12881,'striking'),(50074,'strikingly'),(87694,'string'),(327385,'stringency'),(12508,'stringent'),(108398,'stringently'),(140366,'strip'),(344110,'Strips'),(14592,'strive'),(302985,'stroke'),(8193,'stroma'),(3649,'strong'),(12676,'strongly'),(337809,'Stroschein'),(3796,'structural'),(45301,'structurally'),(3563,'structure'),(92159,'structured'),(121891,'structuring'),(74249,'struggle'),(660,'student'),(70503,'studied'),(83496,'Studies'),(85,'study'),(100149,'stunning'),(11688,'stunt'),(101256,'style'),(361039,'su'),(47074,'sub'),(324265,'subacute'),(326433,'subarachnoid'),(4189,'subcellular'),(90960,'subclass'),(344292,'subclinical'),(25241,'subclone'),(25246,'subcloned'),(132762,'subcutaneous'),(103115,'subcutaneously'),(30813,'subdivide'),(340632,'subdivision'),(3526,'subfamily'),(126200,'subgenogroup'),(10904,'subgenomic'),(123873,'subglottic'),(15464,'subgroup'),(35,'subject'),(67476,'subjective'),(77024,'submentione'),(356087,'submerge'),(74721,'submission'),(4700,'submit'),(16416,'submucosa'),(3642,'submucosal'),(2353,'subnormal'),(354877,'suboptimal'),(332016,'subordinate'),(133926,'subperipheral'),(5604,'subpleural'),(4568,'subpopulation'),(87575,'subsample'),(73397,'subscribe'),(74416,'subscription'),(114642,'subsection'),(311158,'subsegment'),(47955,'subsequence'),(4200,'subsequent'),(11095,'subsequently'),(3612,'subset'),(124498,'subside'),(73579,'subsidise'),(51270,'subsist'),(346657,'subspecie'),(17706,'substance'),(2980,'substantial'),(15437,'substantially'),(23854,'substantiate'),(307130,'substantive'),(108094,'substantively'),(305060,'substituent'),(11304,'substitute'),(305809,'substituted'),(6248,'substitution'),(17660,'substrate'),(321158,'substratifie'),(90730,'substructure'),(23344,'subsystem'),(49136,'subtili'),(50415,'subtilis'),(5977,'subtle'),(15652,'subtract'),(105077,'subtraction'),(112422,'subtractive'),(6488,'subtype'),(346979,'subtyping'),(3823,'subunit'),(302917,'subventricular'),(379565,'subversion'),(84447,'subviral'),(83204,'subvirion'),(38116,'succeed'),(23157,'success'),(16200,'successful'),(31215,'successfully'),(60156,'successive'),(372000,'successively'),(45278,'succinate'),(305877,'succinic'),(141383,'succinimidyl'),(8347,'succumb'),(124406,'sucralfate'),(26784,'sucrose'),(122338,'suction'),(123734,'suctioning'),(69727,'Sud'),(371365,'Sudan'),(367006,'sudanese'),(22258,'sudden'),(11278,'suf'),(2184,'suffer'),(74460,'suffering'),(96384,'suffi'),(5727,'sufficient'),(20687,'sufficiently'),(87928,'suffix'),(4322,'sugar'),(2438,'suggest'),(17443,'suggestion'),(9506,'suggestive'),(75778,'Sugiyama'),(339995,'suicide'),(108234,'suit'),(69681,'suitability'),(3310,'suitable'),(64269,'suited'),(79374,'sulfate'),(3049,'sulfation'),(361160,'sulfhydryl'),(144763,'Sulfolobus'),(360969,'sulfone'),(57815,'sulfonic'),(43729,'sulforhodamine'),(34755,'sulfoxide'),(303866,'sulfur'),(63856,'sum'),(45724,'summarise'),(349,'summarize'),(9927,'summary'),(105590,'summation'),(810,'summer'),(72244,'summit'),(339053,'SUMO'),(361970,'SUMOylation'),(22048,'Sun'),(338696,'Sunderland'),(307866,'Sunnybrook'),(43802,'Sunnyvale'),(34022,'Suomalainen'),(85535,'Supelco'),(20864,'Super'),(329222,'SuperBlock'),(90965,'superclass'),(61859,'supercoiling'),(104333,'Superdex'),(108611,'supererogation'),(107406,'supererogatory'),(5103,'superfamily'),(353123,'superficial'),(36085,'superimpose'),(9428,'superinfection'),(10127,'superior'),(312505,'superiority'),(9029,'supernatant'),(2494,'superoxide'),(62638,'Superscript'),(18978,'supersede'),(136682,'superspreade'),(314870,'superspreader'),(114941,'supertype'),(114450,'supervene'),(133766,'supervised'),(121753,'supervision'),(94032,'supervisor'),(36473,'supine'),(10916,'supplement'),(19618,'supplemental'),(51531,'Supplementary'),(99931,'supplier'),(14519,'supply'),(2297,'support'),(76664,'supportedthe'),(14806,'supportive'),(59787,'suppose'),(19499,'suppress'),(78325,'suppresser'),(14386,'suppression'),(109765,'suppressive'),(47332,'suppressor'),(345625,'suppuration'),(108473,'supreme'),(352713,'sure'),(142606,'surely'),(3071,'surface'),(3521,'surfactant'),(352561,'surge'),(311222,'surgeon'),(14585,'surgery'),(15337,'surgical'),(45460,'surprise'),(10522,'surprised'),(9624,'surprising'),(12035,'surprisingly'),(58187,'surrogate'),(8195,'surround'),(1995,'surveillance'),(16986,'survey'),(362949,'Surveyor'),(45089,'survivability'),(7538,'survival'),(324128,'survivalpromoting'),(36802,'survive'),(7967,'survivor'),(26584,'Susan'),(19142,'susceptibility'),(2254,'susceptible'),(2480,'suspect'),(43503,'suspend'),(4464,'suspension'),(7651,'suspicion'),(48212,'suspicious'),(6536,'sustain'),(74115,'sustainability'),(23079,'sustainable'),(73429,'sustained'),(23692,'Suzuki'),(66481,'SV'),(27121,'Svitkin'),(84429,'svp'),(84450,'SVPs'),(302919,'SVZ'),(93567,'SW'),(124713,'swab'),(122306,'swallowing'),(78206,'Swanson'),(112879,'swap'),(128234,'swapping'),(121999,'Sweden'),(126127,'sweeping'),(367386,'swell'),(132225,'swelling'),(42034,'swift'),(100184,'swiftly'),(339362,'Swine'),(309654,'swing'),(10381,'swiss'),(134505,'SwissProt'),(13479,'switch'),(59607,'switching'),(366959,'Switzerland'),(145623,'Syber'),(66072,'SYBR'),(90620,'SYFPEITHI'),(113675,'SYK'),(86307,'sym'),(340343,'symbiont'),(339377,'symbiotic'),(41753,'symbol'),(107922,'symbolic'),(81601,'symbolize'),(76548,'symbolthan'),(45765,'Symmetric'),(317988,'symmetrical'),(108634,'sympathetic'),(16188,'symposium'),(37,'symptom'),(15346,'symptomatic'),(121813,'syn'),(57121,'synapsis'),(326279,'synaptic'),(57170,'synaptonemal'),(138152,'synchronization'),(137975,'synchronize'),(8946,'syncytia'),(449,'syncytial'),(27745,'syncytium'),(22300,'syndromal'),(413,'syndrome'),(20762,'syndromic'),(137770,'synergistic'),(8446,'synergistically'),(74904,'synonym'),(47654,'synonymous'),(355160,'synonymy'),(31537,'syntaxin'),(2008,'synthase'),(6042,'synthesis'),(3592,'synthesize'),(85568,'synthesizer'),(75973,'synthetase'),(50009,'synthetic'),(74003,'syringe'),(100573,'syrinx'),(698,'system'),(8968,'systematic'),(12618,'systematically'),(8754,'systemic'),(326414,'systemically'),(10576,'Systems'),(321776,'systemslevel'),(142215,'systemthe'),(284,'systolic'),(36344,'Syva'),(2802,'Syvanen'),(122956,'TA'),(143265,'TAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGAC'),(143262,'TAACCCGGGTCCGGGGCCTCGGTGGTGCTCCTGATG'),(74811,'tab'),(350,'Table'),(6040,'Tables'),(320402,'tablet'),(351952,'tabletop'),(24563,'Tabor'),(87128,'tabulate'),(354171,'tabulation'),(357910,'Tac'),(145358,'TACGACTCACTATAGGTCAGTTAAATGTCACAAAA'),(280,'tachycardia'),(281,'tachypnoea'),(72953,'tackle'),(335884,'tackled'),(104469,'taconic'),(31811,'tacrolimus'),(41505,'tact'),(25221,'tag'),(357907,'Tagaya'),(143270,'TAGTAACCCGGGTCCGGGGCCTCGGTGGTGCTCCT'),(13139,'tail'),(363818,'tailing'),(35844,'tailor'),(95722,'tailoring'),(47108,'Taiwan'),(79099,'Takagi'),(343675,'Takala'),(79229,'Takara'),(2171,'take'),(9519,'Takeuchi'),(335707,'taking'),(86113,'tAL'),(57247,'talk'),(339103,'tamper'),(33397,'Tan'),(145302,'tandem'),(77692,'taneously'),(34238,'Tang'),(107865,'tangible'),(31573,'tangle'),(16213,'tantalizing'),(75152,'Tanzania'),(367045,'Tanzanian'),(109276,'tap'),(36519,'tape'),(64412,'taq'),(64840,'TaqI'),(35755,'TaqMan'),(147950,'TAR'),(312866,'Taraxacum'),(2457,'target'),(306257,'Targetdependent'),(2177,'targeted'),(13196,'targeting'),(141058,'targetspecific'),(95928,'tariff'),(337852,'Tarpy'),(29775,'Tarzana'),(13662,'task'),(72250,'Taskforce'),(72426,'tasman'),(147923,'Tat'),(345007,'tattoo'),(31671,'tau'),(337306,'Tauszig'),(79539,'tavidin'),(370546,'tax'),(75038,'taxa'),(72910,'taxation'),(35064,'taxol'),(54054,'taxonomic'),(53833,'taxonomically'),(53784,'taxonomy'),(75461,'taxpayer'),(344549,'TB'),(325773,'TBI'),(338546,'TBLASTN'),(43607,'TBST'),(36187,'tc'),(28353,'TCA'),(25617,'TCC'),(143269,'TCGACGGTCTCCCTCCACTGCTG'),(143271,'TCGACGGTCTCCCTCCACTGCTGT'),(143261,'TCGACGGTCTCCCTCCACTGCTGTAG'),(143273,'TCGACGGTCTCCCTCCACTGCTGTAGT'),(143275,'TCGACGTGCTCCTGATGCCCCTG'),(143276,'TCGACGTGCTCCTGGTGCCCCTG'),(90408,'TCR'),(378652,'TCRb'),(77387,'TCT'),(13710,'TCV'),(362924,'TDG'),(363814,'TdT'),(54622,'TE'),(100272,'tea'),(97,'teach'),(326665,'teaching'),(42636,'team'),(353885,'teambuilding'),(380492,'tease'),(380709,'TEBU'),(100088,'Tecan'),(367638,'TechGen'),(3252,'technical'),(21856,'technically'),(107429,'technician'),(61,'technique'),(105709,'techniquesall'),(54187,'technological'),(66110,'Technologies'),(10202,'technology'),(373606,'Tecnología'),(376151,'Tecumseh'),(112465,'tedious'),(73188,'teenager'),(34461,'tegument'),(51090,'tein'),(343909,'Teixeira'),(301032,'Teklad'),(61727,'tel'),(54451,'telangiectasia'),(106081,'TeleForm'),(21205,'telephone'),(122986,'telescope'),(72895,'Television'),(307575,'tell'),(326440,'temper'),(681,'temperate'),(5189,'temperature'),(10952,'template'),(347990,'templated'),(22284,'temporal'),(9596,'temporally'),(22626,'temporarily'),(88409,'temporary'),(147447,'tempting'),(56210,'tenable'),(123630,'tenacious'),(45332,'tend'),(50114,'tendae'),(490,'tendency'),(17533,'Tenhunen'),(16383,'tension'),(353148,'tent'),(147735,'tentan'),(338611,'tentative'),(37674,'tenth'),(337737,'tep'),(13471,'Tepfer'),(336743,'TEPs'),(53422,'Ter'),(2477,'term'),(86299,'termÀAU'),(93551,'termed'),(3799,'terminal'),(29567,'terminaldeletion'),(45023,'terminally'),(13632,'terminate'),(10966,'termination'),(14033,'terminator'),(3826,'termini'),(72069,'terminology'),(27033,'terminus'),(338344,'termite'),(78711,'ternary'),(120206,'Territorial'),(309490,'Territoriality'),(48307,'Territories'),(47987,'Territory'),(106305,'terror'),(47982,'terrorism'),(21575,'terrorist'),(85579,'tertbutyldimethylsilyl'),(95,'tertiary'),(55,'test'),(121937,'testable'),(141723,'TeStack'),(16487,'testament'),(112626,'tester'),(3734,'testicular'),(82839,'testify'),(9493,'testing'),(56391,'testis'),(378971,'Tet'),(109322,'tetra'),(76549,'tetramer'),(361699,'tetrameric'),(54620,'tetramethylammonium'),(105063,'tetramethylbenzidine'),(92557,'tetraploidization'),(344087,'tetrazolium'),(333735,'Teunis'),(43443,'Texas'),(11618,'text'),(74766,'textbook'),(91694,'textual'),(329676,'TF'),(332188,'tfbinding'),(329719,'TFBS'),(16576,'TFPI'),(67002,'TFRC'),(329828,'tfs'),(62259,'TG'),(25618,'TGA'),(330508,'TGAAACA'),(26593,'TGC'),(133681,'TGF'),(94419,'TGYAATGATGAGGGTGTCACTGCGGTTG'),(52018,'th'),(75194,'Thailand'),(95241,'Thanachaiboot'),(51500,'thank'),(374926,'thati'),(374877,'thatwhere'),(366240,'thatwith'),(30376,'thaw'),(124855,'thawed'),(11032,'thè'),(138048,'theboard'),(136158,'theca'),(21141,'theclock'),(21200,'thecounter'),(321437,'thematic'),(72371,'theme'),(316180,'thenas'),(375769,'thenis'),(375340,'thenwhere'),(61533,'theoretical'),(10217,'theoretically'),(16370,'theory'),(2456,'therapeutic'),(131938,'therapeutical'),(64014,'therapeutically'),(403,'therapy'),(349132,'thereforethe'),(60484,'thereof'),(141213,'thermal'),(363437,'thermocycle'),(141729,'thermocycler'),(55904,'thermodynamic'),(70301,'ThermoLabsystems'),(83266,'thermolysin'),(144853,'thermophilic'),(85668,'thermoprogrammer'),(101746,'thermore'),(144854,'thermostable'),(147331,'thermotoga'),(81423,'thesis'),(8954,'theyare'),(340668,'THi'),(24654,'thiamine'),(148800,'thiazole'),(148799,'thiazolyphenylmolecule'),(38572,'thick'),(6769,'thickening'),(36511,'thickness'),(12570,'Thiel'),(38580,'thin'),(138626,'thing'),(2108,'think'),(73489,'thinking'),(105172,'thiocyanate'),(30487,'thioester'),(361068,'thioether'),(64557,'thiogalactopyranoside'),(104295,'thiogalactoside'),(2948,'thiol'),(357795,'thioldisulfide'),(337734,'thiolester'),(336741,'thiolestercontaine'),(3211,'thioredoxin'),(303488,'thiouridine'),(118860,'third'),(365864,'thirdly'),(119448,'thirteen'),(355,'thirty'),(345862,'thirtyone'),(95228,'Thomas'),(110234,'Thoracic'),(311471,'thoracotomy'),(92536,'Thornton'),(10828,'thorough'),(26515,'thoroughly'),(48005,'thought'),(308051,'thoughtfully'),(39699,'thousand'),(30418,'Thr'),(73069,'thread'),(14537,'threat'),(73394,'threaten'),(3977,'threefold'),(303855,'threonine'),(4314,'threonyl'),(303492,'threonylcarbamoyl'),(14428,'threshold'),(301355,'thresholde'),(312740,'thrips'),(124475,'throat'),(715,'thrombocytopenia'),(326443,'thromboembolism'),(20335,'thrombosis'),(7886,'thromboxane'),(32232,'throughput'),(89248,'thumb'),(74825,'thumbnail'),(97871,'thymidine'),(347984,'thymine'),(57000,'thymus'),(6200,'thyroid'),(320829,'tial'),(357478,'Tian'),(370352,'tical'),(88394,'tick'),(36386,'tidal'),(76280,'tide'),(321022,'tie'),(21712,'tier'),(367799,'TIF'),(29805,'tigard'),(73892,'tight'),(35997,'tightly'),(87533,'TIGR'),(14065,'Tijms'),(356657,'tile'),(79811,'till'),(362980,'tilling'),(379096,'tilt'),(2135,'time'),(20615,'timely'),(376463,'timeserie'),(73837,'timetable'),(16749,'timing'),(324113,'tinucleate'),(146803,'tiny'),(10085,'tion'),(112027,'tiotropium'),(58144,'tip'),(372304,'tiple'),(2634,'tissue'),(141192,'Titanium'),(141110,'TiTaq'),(5702,'titer'),(110517,'title'),(58283,'Tito'),(25187,'titrate'),(97805,'titration'),(113209,'tively'),(86097,'tL'),(85230,'TLB'),(76285,'TLC'),(336132,'tlr'),(336120,'TLRs'),(344277,'Tlymphocyte'),(26938,'TM'),(54618,'TMA'),(366874,'TMHMM'),(363085,'tmmediate'),(366873,'TMPRED'),(58369,'tnb'),(58103,'TNBS'),(9124,'TNF'),(358081,'tnfr'),(56711,'TNG'),(30717,'TNT'),(72688,'tobacco'),(34545,'tobamoviru'),(105022,'Tobler'),(46166,'Tobravirus'),(43064,'tocopheral'),(22972,'today'),(62111,'toeprint'),(62554,'toeprinte'),(62562,'toeprinting'),(145308,'TOF'),(33312,'Togel'),(75219,'Togo'),(96686,'Tohoku'),(96691,'Tokai'),(22940,'token'),(75676,'Tokyo'),(17370,'tolerance'),(12647,'tolerant'),(16089,'tolerate'),(323428,'toll'),(336267,'Tollip'),(337348,'Tollo'),(354920,'tomany'),(11686,'tomato'),(328674,'Tomcat'),(23903,'Tome'),(37058,'tomographic'),(17159,'tomography'),(365345,'tomolecule'),(94025,'TON'),(7069,'tone'),(124480,'tongue'),(10814,'tool'),(114684,'toolbox'),(22046,'Toolkit'),(91002,'topdown'),(16238,'topic'),(124157,'topical'),(93844,'topo'),(136540,'topology'),(105209,'topota'),(135461,'tor'),(16525,'Toronto'),(72778,'Torres'),(62055,'torsion'),(61801,'torsional'),(349001,'tosheep'),(3633,'tosis'),(336757,'tot'),(336,'total'),(45310,'totally'),(23809,'Toth'),(58141,'touch'),(73969,'Tough'),(95408,'tourism'),(95440,'tourist'),(374903,'towhere'),(364313,'towild'),(96702,'town'),(105796,'Towson'),(2712,'toxic'),(2416,'toxicity'),(339468,'toxin'),(330501,'TP'),(367658,'TPP'),(129210,'Tr'),(10635,'trace'),(314351,'traceability'),(38374,'traceable'),(3646,'trachea'),(3233,'tracheal'),(28,'tracheobronchitis'),(339948,'trachomatis'),(37478,'tracing'),(8796,'track'),(38711,'tracking'),(8,'tract'),(24162,'tractable'),(72946,'traction'),(47448,'trade'),(87241,'tradeoff'),(107795,'tradition'),(72211,'traditional'),(10168,'traditionally'),(95488,'traffi'),(100372,'traffic'),(100192,'trafficking'),(48775,'tragedy'),(48901,'tragic'),(129208,'trailer'),(22170,'train'),(22164,'training'),(335642,'trait'),(303749,'trajectory'),(5161,'tralization'),(10482,'tran'),(335699,'Traniello'),(20309,'trans'),(22658,'transaction'),(7645,'transbronchial'),(90488,'transcend'),(11382,'transcribe'),(47360,'transcribed'),(356751,'transcribing'),(10919,'transcript'),(3675,'transcriptase'),(3101,'transcription'),(6140,'transcriptional'),(2044,'transcriptionally'),(354575,'transcriptlevel'),(68996,'transcriptome'),(113480,'transcriptomewide'),(112586,'transcriptomic'),(18121,'transduce'),(17907,'transducer'),(336123,'transducing'),(5131,'transduction'),(6623,'transepithelial'),(330638,'TRANSFAC'),(342071,'transfectant'),(5312,'transfecte'),(11414,'transfected'),(331659,'transfecting'),(6170,'transfection'),(7024,'transfer'),(142444,'transferable'),(24391,'transferase'),(335696,'transference'),(36181,'transferrin'),(6506,'transform'),(26727,'transformant'),(49146,'transformation'),(23617,'transframe'),(14437,'transfuse'),(14401,'transfusing'),(14306,'transfusion'),(8756,'transgene'),(7394,'transgenic'),(363294,'Transgenomic'),(7282,'transient'),(57070,'transiently'),(119903,'transit'),(2950,'transition'),(31540,'transketolase'),(77153,'transla'),(70587,'translatability'),(71988,'translatable'),(4053,'translate'),(27181,'translated'),(144718,'translating'),(10889,'translation'),(23949,'translational'),(28143,'translationally'),(147574,'translocase'),(24002,'translocate'),(9223,'translocation'),(2362,'transmembrane'),(40768,'transmissibility'),(20,'transmissible'),(14548,'transmission'),(645,'transmit'),(307625,'transparency'),(17349,'transparent'),(2847,'transplant'),(7003,'transplantation'),(7613,'transplanted'),(360115,'transplasma'),(8443,'transport'),(105629,'transportation'),(23688,'transporter'),(27052,'transportin'),(147211,'transposase'),(51251,'transposition'),(6125,'transregulatory'),(100357,'transship'),(43211,'transversion'),(8414,'trap'),(358047,'trapping'),(367659,'Trasadingen'),(149912,'trastuzumab'),(14564,'trauma'),(325770,'traumatic'),(122321,'traumatize'),(47980,'travel'),(95464,'traveler'),(190,'tray'),(148213,'TRBP'),(129214,'Trc'),(72139,'Treasury'),(361,'treat'),(311941,'treatable'),(5194,'treatment'),(48405,'treaty'),(25560,'tree'),(134090,'TREEVIEW'),(108382,'tremendous'),(113403,'tremendously'),(15484,'trend'),(305914,'Trevigen'),(337347,'Trex'),(124912,'Tri'),(336842,'triad'),(20961,'triage'),(14476,'trial'),(118391,'Trials'),(100325,'Triangle'),(337436,'Tribolium'),(14475,'TRICC'),(29982,'trichloroacetic'),(46593,'Tricho'),(324733,'tricyclic'),(305760,'triethylamine'),(305707,'triethylammonium'),(305880,'trifluoroacetate'),(305853,'trifluoroacetic'),(15360,'trigger'),(13098,'trim'),(3848,'trimer'),(3822,'trimeric'),(305870,'trimethylacetic'),(331872,'trimethylammonium'),(354952,'trimming'),(311639,'trimoxazole'),(57814,'trinitrobenzene'),(3191,'trioxide'),(74465,'trip'),(380123,'tripartite'),(19053,'tripeptide'),(43236,'triphosphate'),(55507,'triple'),(4131,'triplet'),(63501,'triplex'),(36312,'triplicate'),(29961,'Tris'),(367218,'trisacrylh'),(8970,'Tristram'),(342607,'TRITC'),(30135,'tritcconjugated'),(34891,'Triton'),(73065,'triumph'),(21857,'trivial'),(367462,'trizma'),(56601,'Trizol'),(58990,'TrkA'),(13133,'trna'),(78168,'tRNALeuCAG'),(52185,'trnas'),(124512,'tropic'),(38037,'tropical'),(83392,'tropism'),(99098,'trouble'),(93463,'troublesome'),(55268,'trough'),(133732,'trout'),(141662,'trp'),(11054,'trs'),(12750,'trsderive'),(25968,'true'),(105531,'truer'),(46222,'truly'),(305214,'Truman'),(27470,'truncate'),(10978,'truncated'),(28712,'truncation'),(105600,'trust'),(307628,'trustworthiness'),(357819,'trx'),(359360,'TrxR'),(61198,'try'),(29867,'Trypan'),(147720,'Trypanosoma'),(371335,'trypanosomatidae'),(35095,'trypsin'),(29865,'trypsinize'),(140694,'trypsinogen'),(59006,'tryptase'),(146260,'tryptic'),(29678,'tryptone'),(104636,'tryptophan'),(86866,'ts'),(140575,'TSE'),(141232,'TSEs'),(329834,'TSS'),(23945,'Tsuji'),(77074,'Tsurui'),(340839,'Tsutsumi'),(312682,'TSWV'),(146043,'TTC'),(125138,'TTCCAATACCACCCCTTGGATGA'),(113674,'ttid'),(113669,'TTN'),(25619,'TTT'),(30223,'Tu'),(36314,'tube'),(345237,'tuberculoma'),(51735,'tuberculosis'),(345320,'tuberculous'),(123557,'tubing'),(56403,'tubule'),(33724,'tubulin'),(75775,'Tuite'),(301406,'Tukey'),(134359,'Tulsa'),(1994,'tumor'),(67261,'tumoral'),(16581,'tumour'),(43797,'tunable'),(150435,'tune'),(36496,'tungsten'),(92009,'tuning'),(61665,'tunnel'),(336756,'Turandot'),(234,'turbidity'),(121042,'ture'),(340661,'Turk'),(332565,'turkey'),(8400,'turn'),(62492,'Turner'),(128705,'turning'),(13707,'turnip'),(27095,'turnover'),(101371,'turpitude'),(17490,'tutorial'),(36116,'Tuxen'),(85267,'Tween'),(95444,'twelfth'),(26851,'twice'),(327708,'twocolor'),(116088,'twodye'),(341515,'twofold'),(375111,'twolevel'),(329119,'twoparameter'),(110820,'twotaile'),(43447,'TX'),(45675,'Tymoviridae'),(322376,'tympanic'),(323711,'tympanical'),(322865,'tympanically'),(323020,'tympanostomy'),(340666,'typable'),(441,'type'),(323109,'typeable'),(141544,'typeplus'),(100383,'typestimulant'),(339958,'typhi'),(212,'typical'),(20233,'typically'),(337828,'typify'),(36346,'typing'),(145802,'Tyr'),(301172,'tyramide'),(2545,'tyrosine'),(86962,'UA'),(71571,'UAC'),(351391,'uate'),(30439,'Ub'),(30432,'UBA'),(30423,'ubiquitin'),(31971,'ubiquitinate'),(30527,'ubiquitinated'),(30785,'ubiquitination'),(31724,'ubiquitinconjugated'),(361397,'ubiquitinor'),(339789,'ubiquitinprotease'),(30569,'ubiquitinproteasome'),(107707,'ubiquitous'),(23756,'ubiquitously'),(23653,'ubiquitylation'),(339157,'Ubl'),(339052,'Ubls'),(339606,'ublspecific'),(361038,'UbVME'),(11638,'UCAAC'),(24052,'UCC'),(24057,'uccu'),(76018,'UCG'),(303953,'UCSF'),(23122,'uct'),(75753,'Ueda'),(95207,'uence'),(43547,'ug'),(24053,'UGA'),(367582,'Uganda'),(86055,'UGAUGGU'),(24272,'UGC'),(24271,'ugg'),(139390,'UI'),(16927,'UK'),(86102,'UL'),(33908,'Ulaeto'),(58806,'ulate'),(124389,'ulcer'),(367640,'Ulis'),(339160,'ULP'),(20076,'ultimate'),(8856,'ultimately'),(64675,'Ultra'),(66455,'ultracentrifugation'),(104723,'ultracompetent'),(358411,'ultrafiltrate'),(64509,'ultrafiltration'),(57961,'ultrasound'),(362700,'Ultraview'),(17696,'ultraviolet'),(99761,'Ultrospec'),(112508,'un'),(14734,'unable'),(108591,'unacceptable'),(22337,'unadjusted'),(54608,'unaffected'),(67613,'unambiguous'),(68784,'unambiguously'),(338565,'unassembled'),(64119,'unavailable'),(42748,'unavoidable'),(96566,'unaware'),(117500,'unbalance'),(96790,'unbalanced'),(373374,'unbiaised'),(301271,'unbiased'),(65115,'unbound'),(73918,'uncapped'),(3308,'uncertain'),(39639,'uncertainty'),(336658,'unchallenged'),(50722,'unchanged'),(339889,'uncharacterized'),(355661,'unclassified'),(2308,'unclear'),(305540,'uncleaved'),(84641,'uncoated'),(83207,'uncoating'),(20667,'uncommon'),(361930,'uncomplexed'),(2210,'unconditional'),(39965,'uncontained'),(125291,'uncontaminated'),(42739,'uncontrollable'),(323281,'uncontrolle'),(130949,'uncontrolled'),(73953,'uncoordinated'),(128315,'uncoupled'),(54957,'uncover'),(313093,'uncovering'),(310384,'uncritically'),(369512,'und'),(18539,'undefined'),(48132,'undercooked'),(338417,'undercut'),(308563,'underdetermine'),(312256,'underdiagnose'),(44710,'underestimate'),(60219,'underestimation'),(380447,'underevaluate'),(7556,'undergo'),(17622,'undergoe'),(553,'underlie'),(24058,'underline'),(336352,'Underlining'),(918,'underlying'),(96206,'undermine'),(73260,'underpin'),(326724,'underrecognize'),(352861,'underrepresented'),(115839,'underscore'),(7167,'understand'),(97188,'understandable'),(2149,'understanding'),(326752,'understudied'),(48075,'undertake'),(81302,'undertaken'),(7520,'underway'),(311462,'underwent'),(340057,'undescribe'),(338076,'undescribed'),(21862,'undesirable'),(87559,'undesired'),(60262,'undetectable'),(317751,'undetecte'),(317060,'undetected'),(19318,'undetermined'),(84047,'undifferentiate'),(45016,'undifferentiated'),(364814,'undigested'),(314380,'undiscovered'),(2692,'undoubtedly'),(95205,'undue'),(101562,'unemployment'),(4540,'unencapsulated'),(22957,'unencrypted'),(95825,'unequal'),(126810,'unequally'),(90440,'unequivocally'),(17253,'unethical'),(327685,'uneven'),(96307,'unevenly'),(304,'uneventful'),(34585,'unexpected'),(302442,'unexpectedly'),(7416,'unexplained'),(377552,'unexplored'),(141231,'unextended'),(101813,'unfavorable'),(13151,'unfavourable'),(30585,'unfold'),(50260,'unfolded'),(121665,'unforeseen'),(3251,'unfortunately'),(79195,'unfractionated'),(330517,'ungapped'),(257,'unhospitalized'),(305265,'unhybridized'),(25963,'unicellular'),(50741,'unidentified'),(38130,'uniform'),(300949,'Uniformed'),(49008,'uniformity'),(36215,'uniformly'),(120195,'unify'),(134130,'Unigene'),(17068,'unilateral'),(101723,'unimaginable'),(319884,'unimportant'),(43787,'unincorporated'),(34050,'uninf'),(10274,'uninfected'),(376483,'uninfectious'),(331783,'uninformative'),(338777,'uninjected'),(307681,'unintended'),(22689,'uninterrupted'),(95260,'Union'),(3134,'unique'),(22945,'uniquely'),(88050,'uniqueness'),(4621,'unit'),(23042,'United'),(48178,'Units'),(139947,'unity'),(121204,'univariate'),(75602,'univer'),(9579,'universal'),(75880,'universality'),(68230,'universally'),(69725,'Universite'),(64,'University'),(95994,'unjust'),(4072,'unknown'),(362435,'unlabele'),(361845,'unlabeled'),(4019,'unlike'),(3021,'unlikely'),(40843,'unlimited'),(55265,'unlogged'),(344486,'unmask'),(344521,'unmasking'),(17411,'unmatched'),(307406,'unmet'),(45259,'unmethylated'),(75928,'unmodified'),(10033,'unnecessary'),(328638,'unnormalized'),(136808,'unnoticed'),(332992,'unobserved'),(118693,'unobtainable'),(16840,'unopened'),(55917,'unpaire'),(4221,'unpaired'),(73392,'unpleasant'),(13119,'unprecedented'),(310125,'unpredictable'),(100599,'unprotected'),(76568,'unpuband'),(103843,'unpublishe'),(2803,'unpublished'),(380620,'unpulsed'),(65061,'unpurifie'),(3394,'unravel'),(99418,'unreacted'),(47725,'unrealistic'),(107716,'unreasonable'),(119488,'unrecognize'),(119028,'unrecognized'),(379931,'unregulate'),(95431,'unregulated'),(313,'unrelated'),(329696,'unreliable'),(37041,'unremarkable'),(45124,'unrepaire'),(110207,'unreported'),(36968,'unresolve'),(17042,'unresolved'),(345561,'unresponsive'),(143049,'unrestricted'),(95480,'unsafe'),(356708,'unsatisfying'),(60208,'unsequestered'),(61565,'unshifted'),(20725,'unsolved'),(66235,'unspecific'),(148248,'unspliced'),(16016,'unstable'),(42644,'unstably'),(304410,'unstacking'),(22639,'unstored'),(351976,'unstructured'),(68547,'unsuitable'),(352744,'unsure'),(41999,'unsurprisingly'),(135981,'unsuspected'),(138651,'unsynchronized'),(73954,'untargeted'),(61546,'untested'),(39219,'untraceable'),(342125,'untransfected'),(26361,'untranslated'),(106895,'untreatable'),(18555,'untreated'),(309826,'unused'),(26293,'unusual'),(4059,'unusually'),(37809,'unvaccinated'),(2979,'unveil'),(131634,'unwanted'),(17248,'unwilling'),(106957,'unwillingness'),(61672,'unwind'),(61812,'unwinding'),(23572,'unwise'),(61809,'unwound'),(30141,'uorescence'),(369999,'upcoming'),(336583,'Upd'),(20946,'update'),(340307,'UPEC'),(142506,'upgrade'),(107967,'uphold'),(21846,'upload'),(328717,'uploaded'),(21567,'UPMC'),(5,'upper'),(77436,'uppermost'),(145048,'Uppsala'),(10359,'upregulate'),(10650,'upregulated'),(10659,'upregulation'),(99583,'upside'),(6098,'upstream'),(4724,'uptake'),(89427,'upwards'),(340497,'ur'),(26725,'URA'),(69912,'uracil'),(121812,'uraemic'),(20826,'urban'),(107411,'Urbani'),(62550,'urea'),(118608,'ured'),(303711,'ureido'),(106270,'urge'),(21279,'urgency'),(101449,'urgent'),(96591,'urgently'),(78655,'uridine'),(6936,'urinary'),(120792,'urine'),(22368,'url'),(340776,'urogenital'),(83927,'urokinase'),(53222,'uropathogenic'),(719,'urticarial'),(100768,'Urumqi'),(10581,'USA'),(47498,'usage'),(320133,'USAMRIID'),(62668,'usb'),(87530,'USDA'),(2172,'use'),(8275,'useful'),(16547,'usefully'),(65319,'usefulness'),(69129,'useless'),(21346,'user'),(66357,'userindependent'),(50256,'Usp'),(360770,'USPs'),(17477,'usual'),(52,'usually'),(300954,'usuhs'),(43358,'UT'),(20976,'Utah'),(47425,'utilise'),(307590,'utilitarian'),(17057,'utility'),(17299,'utilization'),(10882,'utilize'),(108842,'utmost'),(24324,'UTR'),(62145,'UUA'),(25543,'UUU'),(28649,'uv'),(344975,'uveitis'),(301281,'VA'),(37386,'vac'),(51220,'vaccinal'),(37211,'vaccinate'),(37714,'vaccinated'),(37203,'vaccination'),(40717,'vaccinator'),(37173,'vaccine'),(377171,'vaccineinduced'),(32334,'vaccinia'),(339550,'Vacciniavirus'),(373494,'vacine'),(83826,'vacuolar'),(143309,'vacuum'),(107838,'vague'),(108903,'vagueness'),(84184,'val'),(347859,'Valencia'),(345936,'valgancyclovir'),(75120,'valid'),(58837,'validate'),(21073,'Validation'),(15241,'validity'),(367124,'Valle'),(332667,'Vallei'),(332818,'Valley'),(52817,'Vallve'),(42777,'valuable'),(335,'value'),(332981,'valuenow'),(123523,'valve'),(15077,'Vamvakas'),(2799,'van'),(22914,'vancomycin'),(340960,'Vandeputte'),(33612,'Vanderpasschen'),(92599,'Vandesompele'),(138797,'vanilla'),(108357,'vanquish'),(32407,'Vanslyke'),(362301,'vantage'),(122059,'VAP'),(39486,'variability'),(3013,'variable'),(332975,'variablethat'),(51928,'variance'),(4300,'variant'),(5978,'variation'),(34149,'varicella'),(52127,'varie'),(6468,'varied'),(3544,'variety'),(32356,'variola'),(72079,'variously'),(3837,'vary'),(66666,'varying'),(5548,'vascular'),(8269,'vascularization'),(7084,'vasculature'),(7883,'vasoactive'),(6418,'vasoconstriction'),(7180,'vasoconstrictor'),(136223,'vasodilatation'),(136219,'vasodilatator'),(1991,'vasodilation'),(7179,'vasodilator'),(311500,'vasopressor'),(7752,'vasoregulation'),(18166,'vasoregulatory'),(20497,'vasorelaxation'),(134649,'vasotocin'),(6653,'vast'),(23443,'vastly'),(100031,'VB'),(92634,'VBA'),(101820,'VCT'),(85324,'VDB'),(150070,'vdwatom'),(11451,'ve'),(27602,'VEC'),(30146,'Vectashield'),(19742,'vector'),(139015,'Vectors'),(312750,'vegetable'),(313207,'vegetative'),(132746,'VEGF'),(49196,'vehicle'),(131916,'vein'),(75074,'Venezuela'),(367127,'venipuncture'),(7467,'venous'),(109370,'vent'),(36848,'ventilate'),(16941,'ventilated'),(7214,'ventilation'),(36079,'ventilator'),(36269,'ventilatory'),(15554,'ventricle'),(19576,'ventricular'),(107545,'venture'),(17496,'venue'),(2321,'venule'),(58903,'Vera'),(138988,'verb'),(22534,'verbatim'),(136775,'verifiable'),(16965,'Verification'),(20419,'verify'),(104131,'Vero'),(113919,'versa'),(36013,'versatile'),(142803,'versatility'),(323239,'versible'),(20979,'version'),(137168,'versionextende'),(472,'versus'),(26142,'vertebrate'),(40005,'vertical'),(36421,'vesicle'),(34570,'vesicular'),(3776,'vessel'),(72544,'VET'),(350377,'veterinarian'),(350518,'veterinary'),(307781,'vetting'),(58131,'VFF'),(103427,'VH'),(60668,'VI'),(24508,'viability'),(9676,'viable'),(109242,'vial'),(138754,'Viboud'),(305950,'VIC'),(113918,'vice'),(94682,'Vicente'),(32713,'vicinity'),(57788,'vicious'),(14565,'victim'),(48708,'Victoria'),(48065,'Victorian'),(75284,'victory'),(84602,'vide'),(311470,'videoassiste'),(100353,'Viet'),(94979,'Vietnam'),(3004,'view'),(139317,'viewer'),(26509,'viewpoint'),(147917,'Vif'),(95359,'vigilance'),(345200,'vigilant'),(42568,'vigorous'),(362926,'VII'),(150362,'VIII'),(100040,'Vilber'),(94969,'Village'),(150363,'Villard'),(360968,'vinyl'),(361036,'vinylmethyl'),(340000,'vinylmethylester'),(340002,'vinylsulfone'),(128937,'violet'),(246,'viral'),(357897,'virally'),(37952,'viremia'),(9061,'virion'),(148071,'virionendogenous'),(340756,'Virji'),(103376,'virologic'),(124674,'virological'),(111264,'virology'),(94434,'VIRONOVATIVE'),(132037,'virosome'),(114100,'Virtek'),(21458,'virtual'),(696,'virtually'),(75378,'virtue'),(59187,'virulence'),(120010,'virulent'),(149,'virus'),(29657,'viruse'),(130866,'Viruses'),(9708,'virusmediated'),(93688,'Vis'),(58189,'visceral'),(14337,'viscosity'),(322488,'viscous'),(74182,'visibility'),(36419,'visible'),(82179,'visibly'),(118042,'vision'),(21185,'visit'),(307262,'visitation'),(309077,'visitor'),(106084,'Vista'),(25016,'visual'),(47934,'visualisation'),(35010,'visualization'),(14173,'visualize'),(43664,'visualized'),(98188,'visually'),(13697,'vital'),(43065,'vitamin'),(133750,'vitellogenesis'),(133542,'vitellogenic'),(46591,'Viti'),(46084,'Vitivirus'),(344971,'vitritis'),(4971,'vitro'),(362516,'vivascience'),(367107,'vivax'),(335589,'vivid'),(2508,'vivo'),(319426,'VivoEbola'),(116635,'VJ'),(105303,'vk'),(362480,'VKAKIQD'),(103429,'VL'),(113671,'VLDLR'),(2801,'Vliet'),(303952,'VMD'),(340001,'VME'),(46359,'Vmethyltransf'),(31044,'VO'),(90572,'vocabulary'),(108499,'vocal'),(72541,'Vocational'),(73015,'voice'),(328430,'void'),(11215,'Vol'),(324757,'volatile'),(6463,'voltage'),(2838,'volume'),(62341,'Volumes'),(42246,'voluntary'),(102136,'volunteer'),(25499,'volvulus'),(120770,'vomiting'),(14121,'von'),(26871,'vortexe'),(70249,'vortexing'),(93575,'Vosso'),(52913,'vote'),(53029,'voter'),(52851,'voting'),(145284,'Voyager'),(69999,'VPg'),(150322,'VPHPT'),(150221,'VPHPTWMR'),(21461,'VPN'),(147919,'Vpr'),(147918,'Vpu'),(124263,'VRE'),(49614,'vs'),(34573,'VSV'),(124852,'VTM'),(107227,'vulnerability'),(72058,'vulnerable'),(128918,'VWR'),(73307,'VZV'),(103462,'Vκ'),(48433,'WA'),(150073,'Waals'),(340857,'Wagener'),(27380,'Waggoner'),(371174,'WAIFW'),(38126,'wait'),(38162,'waiting'),(36436,'waive'),(357920,'Wakasugi'),(48900,'wake'),(77077,'Wakita'),(79243,'Wako'),(348695,'Wales'),(21665,'walk'),(89292,'walking'),(140037,'walkthrough'),(4481,'wall'),(333734,'Wallinga'),(59903,'Wallis'),(37843,'wane'),(4911,'Wang'),(72138,'Wanless'),(44825,'want'),(94966,'War'),(107469,'ward'),(350977,'warehouse'),(35284,'warm'),(66767,'warn'),(21232,'warning'),(26436,'warrant'),(345016,'wart'),(4729,'wash'),(35288,'washed'),(30853,'washing'),(21102,'Washington'),(33839,'washout'),(338342,'wasp'),(42999,'waste'),(313205,'wasteland'),(360364,'Watanabe'),(123375,'watch'),(8426,'water'),(332580,'waterfowl'),(14748,'Watering'),(145313,'Waters'),(85773,'Watson'),(343836,'Watt'),(138343,'wave'),(138666,'waveform'),(43803,'wavelength'),(68971,'wavy'),(99587,'wax'),(3748,'way'),(14714,'weak'),(102839,'weaken'),(27997,'weakly'),(51408,'weakness'),(95953,'wealth'),(75500,'wealthy'),(17376,'weaning'),(48138,'weapon'),(96274,'weaponized'),(316516,'wear'),(316362,'wearing'),(23578,'weather'),(21843,'web'),(340958,'Webb'),(108833,'webcaste'),(74768,'website'),(17060,'wedge'),(313618,'weed'),(208,'week'),(36323,'weekly'),(333032,'Weibull'),(20587,'weigh'),(27042,'Weighardt'),(17659,'weight'),(335465,'weighting'),(33309,'weisshaar'),(17442,'welcome'),(75281,'welfare'),(3448,'well'),(75643,'wellbee'),(48912,'wellbeing'),(13460,'welldocumente'),(43691,'Wells'),(17309,'Wendon'),(149240,'Werner'),(132525,'Werth'),(128266,'Wertz'),(95406,'West'),(28107,'western'),(332576,'wetland'),(87472,'WGASA'),(70213,'WGE'),(140809,'Whatman'),(69602,'wheat'),(367451,'wheel'),(339024,'Wheeler'),(511,'wheeze'),(871,'Wheezes'),(349349,'wherethe'),(349127,'whichever'),(48097,'whilst'),(853,'white'),(58298,'Whitney'),(335138,'whoacquire'),(330881,'wholegenome'),(134180,'WI'),(123544,'wick'),(3543,'wide'),(3222,'widely'),(304075,'widen'),(22823,'widening'),(337954,'wideranging'),(25907,'widespread'),(93520,'width'),(61064,'Wilcoxon'),(2274,'wild'),(11472,'wildtype'),(301141,'William'),(23816,'Williams'),(101939,'willing'),(105546,'willingness'),(93692,'Wilmington'),(14120,'Wilson'),(90211,'win'),(20830,'wind'),(38318,'window'),(74390,'Windows'),(88121,'winnow'),(680,'winter'),(117681,'wiring'),(60837,'Wisconsin'),(42745,'wise'),(87214,'wisely'),(84,'wish'),(150404,'withComparison'),(17456,'withdraw'),(58813,'withdrawal'),(17457,'withhold'),(37140,'withholding'),(312534,'withinhost'),(374065,'withinhousehold'),(15683,'witness'),(60831,'Wizard'),(150323,'WM'),(312699,'WMV'),(102305,'WNV'),(61607,'wobble'),(336541,'Wojda'),(32481,'Wolffe'),(344198,'Wolfgang'),(14411,'woman'),(102119,'Women'),(32662,'wonder'),(48125,'wool'),(313046,'Woolhouse'),(12524,'word'),(108113,'wording'),(139249,'WordPad'),(3383,'work'),(115453,'workable'),(5082,'worker'),(68837,'workflow'),(72118,'workforce'),(13521,'working'),(123371,'workload'),(38193,'workplace'),(141661,'workstation'),(13450,'world'),(16827,'Worldwide'),(25995,'worm'),(13472,'Worobey'),(354563,'worry'),(2425,'worsen'),(112142,'worsening'),(15498,'worst'),(52198,'worth'),(315820,'worthwhile'),(83432,'wound'),(336314,'wounding'),(148553,'wout'),(34732,'WR'),(32802,'wrap'),(33059,'wrapping'),(5007,'Wright'),(123374,'wrist'),(21765,'write'),(94036,'writing'),(96440,'wrong'),(22277,'wsare'),(11900,'wt'),(27364,'Wu'),(122766,'Wunderink'),(61558,'WWH'),(305639,'www'),(313783,'Xa'),(109091,'xanthine'),(45963,'Xanthomonas'),(82058,'Xas'),(33563,'xation'),(30326,'xbai'),(13192,'xe'),(34792,'xed'),(17639,'xenobiotic'),(20304,'xenograft'),(325544,'xenon'),(6385,'Xenopus'),(20297,'xenotransplantation'),(149244,'xeroderma'),(80508,'xes'),(26679,'xhoi'),(149669,'xi'),(100364,'Xinjiang'),(67206,'XLSTAT'),(21769,'XML'),(94550,'XP'),(104590,'Xpress'),(82040,'Xs'),(92057,'XSD'),(71749,'XXA'),(69341,'XXX'),(71767,'XXXAAAC'),(69316,'XXY'),(104482,'xylazine'),(45985,'Xylella'),(132149,'YACs'),(325549,'Yakota'),(125838,'Yamagata'),(338150,'Yang'),(77871,'Yarus'),(6173,'YC'),(24511,'ye'),(382,'year'),(137976,'yearly'),(13123,'yeast'),(148394,'Yedavalli'),(230,'yellow'),(339120,'Yersinia'),(117945,'yes'),(149103,'Yeung'),(361571,'YFP'),(150396,'yftnmf'),(145552,'YGM'),(26184,'YGYCCCYCA'),(102273,'YHR'),(28519,'yield'),(126989,'Yin'),(100770,'Yining'),(75690,'Yokogawa'),(75688,'Yokohama'),(133918,'yolk'),(339185,'yopj'),(60767,'York'),(360171,'Yoshida'),(17,'young'),(12736,'Yount'),(102067,'youth'),(69952,'YPD'),(380548,'YPHFMPTNL'),(345832,'yr'),(135049,'ysis'),(145554,'YSM'),(358815,'YT'),(100903,'yuan'),(119078,'Yugoslavia'),(100351,'Yunnan'),(69342,'YYY'),(69317,'YYZ'),(318941,'Zaire'),(339880,'zeae'),(72433,'Zealand'),(318945,'ZEBOV'),(23779,'Zebra'),(31779,'zebrafish'),(148557,'Zeichner'),(38831,'zero'),(76195,'zeylanoide'),(11704,'Zhang'),(23748,'Zhu'),(350140,'zi'),(345586,'zidovudine'),(75781,'Zimmer'),(344199,'Zimmermann'),(14075,'zinc'),(21396,'zip'),(17864,'zipper'),(350143,'zn'),(38041,'zone'),(34150,'zoster'),(337236,'Zou'),(104842,'Zymo'),(104838,'Zymoprep'),(326942,'ZZ'),(130339,'ðÀÞnc'),(130108,'ðencap'),(129896,'ðpol'),(375614,'ðþbinomialthe'),(130255,'ðþþnc'),(59800,'þbcd'),(59811,'þbcþ'),(59813,'þbþd'),(59803,'þbþþ'),(375409,'Þds'),(374385,'Þw'),(59808,'þþcd'),(59805,'þþcþ'),(59807,'þþþd'),(31564,'βamyloid'),(329486,'δq'),(3175,'κb'),(30849,'µg'),(31093,'µl'),(76223,'µm'),(98062,'µmol'),(84547,'µns'),(84516,'ρna'),(39151,'φProb');
/*!40000 ALTER TABLE `Word` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `paper_to_author`
--

DROP TABLE IF EXISTS `paper_to_author`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `paper_to_author` (
  `fk_paper_id` varchar(200) NOT NULL,
  `fk_author_id` int NOT NULL,
  PRIMARY KEY (`fk_paper_id`,`fk_author_id`),
  KEY `fk_author_paper_to_author_idx` (`fk_author_id`),
  CONSTRAINT `fk_author_paper_to_author` FOREIGN KEY (`fk_author_id`) REFERENCES `Author` (`idAuthor`),
  CONSTRAINT `fk_paper_paper_to_author` FOREIGN KEY (`fk_paper_id`) REFERENCES `Paper` (`idPaper`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `paper_to_author`
--

LOCK TABLES `paper_to_author` WRITE;
/*!40000 ALTER TABLE `paper_to_author` DISABLE KEYS */;
INSERT INTO `paper_to_author` VALUES ('d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),('d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),('d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),('d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),('6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),('6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),('6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',7),('06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),('5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',9),('5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',10),('5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',11),('b2897e1277f56641193a6db73825f707eed3e4c9.json',12),('b2897e1277f56641193a6db73825f707eed3e4c9.json',13),('b2897e1277f56641193a6db73825f707eed3e4c9.json',14),('b2897e1277f56641193a6db73825f707eed3e4c9.json',15),('3bb07ea10432f7738413dff9816809cc90f03f99.json',16),('3bb07ea10432f7738413dff9816809cc90f03f99.json',17),('3bb07ea10432f7738413dff9816809cc90f03f99.json',18),('5806726a24dc91de3954001effbdffd7a82d54e2.json',19),('5806726a24dc91de3954001effbdffd7a82d54e2.json',20),('faaf1022ccfe93b032c5608097a53543ba24aedb.json',21),('faaf1022ccfe93b032c5608097a53543ba24aedb.json',22),('faaf1022ccfe93b032c5608097a53543ba24aedb.json',23),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',24),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',25),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',26),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',27),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',28),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',29),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',30),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',31),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',32),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',33),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',34),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',35),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',36),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',37),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',38),('d09b79026117ec9faebba46a8d13aa9b23ec751e.json',39),('d09b79026117ec9faebba46a8d13aa9b23ec751e.json',40),('d09b79026117ec9faebba46a8d13aa9b23ec751e.json',41),('44102e3e69e70ad2a73e753133283334ce1f8736.json',42),('44102e3e69e70ad2a73e753133283334ce1f8736.json',43),('44102e3e69e70ad2a73e753133283334ce1f8736.json',44),('44102e3e69e70ad2a73e753133283334ce1f8736.json',45),('44102e3e69e70ad2a73e753133283334ce1f8736.json',46),('44102e3e69e70ad2a73e753133283334ce1f8736.json',47),('44102e3e69e70ad2a73e753133283334ce1f8736.json',48),('30a4842a2e257f725cc041e32fd682c495111a50.json',49),('30a4842a2e257f725cc041e32fd682c495111a50.json',50),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',51),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',52),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',53),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',54),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',55),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',56),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',57),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',58),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',59),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',60),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',61),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',62),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',63),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',64),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',65),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',66),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',67),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',68),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',69),('83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',70),('83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',71),('025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',72),('8d7400a2b387820cd391d7df8194642e50402a0c.json',73),('8d7400a2b387820cd391d7df8194642e50402a0c.json',74),('8d7400a2b387820cd391d7df8194642e50402a0c.json',75),('8d7400a2b387820cd391d7df8194642e50402a0c.json',76),('8d7400a2b387820cd391d7df8194642e50402a0c.json',77),('8d7400a2b387820cd391d7df8194642e50402a0c.json',78),('8d7400a2b387820cd391d7df8194642e50402a0c.json',79),('8d7400a2b387820cd391d7df8194642e50402a0c.json',80),('8d7400a2b387820cd391d7df8194642e50402a0c.json',81),('8d7400a2b387820cd391d7df8194642e50402a0c.json',82),('8d7400a2b387820cd391d7df8194642e50402a0c.json',83),('8d7400a2b387820cd391d7df8194642e50402a0c.json',84),('8d7400a2b387820cd391d7df8194642e50402a0c.json',85),('8d7400a2b387820cd391d7df8194642e50402a0c.json',86),('8d7400a2b387820cd391d7df8194642e50402a0c.json',87),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',88),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',89),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',90),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',91),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',92),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',93),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',94),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',95),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',96),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',97),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',98),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',99),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',100),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',101),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',102),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',103),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',104),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',105),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',106),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',107),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',108),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',109),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',110),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',111),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',112),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',113),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',114),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',115),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',116),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',117),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',118),('a77badb297bf4927419481e660ea8f1806ed91b1.json',119),('a77badb297bf4927419481e660ea8f1806ed91b1.json',120),('a77badb297bf4927419481e660ea8f1806ed91b1.json',121),('a77badb297bf4927419481e660ea8f1806ed91b1.json',122),('a77badb297bf4927419481e660ea8f1806ed91b1.json',123),('a77badb297bf4927419481e660ea8f1806ed91b1.json',124),('a77badb297bf4927419481e660ea8f1806ed91b1.json',125),('a77badb297bf4927419481e660ea8f1806ed91b1.json',126),('a77badb297bf4927419481e660ea8f1806ed91b1.json',127),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',128),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',129),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',130),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',131),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',132),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',133),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',134),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',135),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',136),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',137),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',138),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',139),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',140),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',141),('40e41f0c52b39669dee24e37875d7a9fabc38636.json',142),('40e41f0c52b39669dee24e37875d7a9fabc38636.json',143),('d77233dd9085c9c76787c84ee9a1e249424b283a.json',144),('d77233dd9085c9c76787c84ee9a1e249424b283a.json',145),('7d624acf891472c3452b1deb89e0dc44a40d0c94.json',146),('7d624acf891472c3452b1deb89e0dc44a40d0c94.json',147),('d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',148),('ada57a9f084a016deb20b6f83c442072e6e37955.json',149),('ada57a9f084a016deb20b6f83c442072e6e37955.json',150),('ada57a9f084a016deb20b6f83c442072e6e37955.json',151),('6eeaffdb22b4e310867262d1891a1442769c7803.json',152),('6eeaffdb22b4e310867262d1891a1442769c7803.json',153),('6eeaffdb22b4e310867262d1891a1442769c7803.json',154),('6eeaffdb22b4e310867262d1891a1442769c7803.json',155),('6eeaffdb22b4e310867262d1891a1442769c7803.json',156),('dff6195f5e24ebfa2fb4060142611b48021f673c.json',157),('dff6195f5e24ebfa2fb4060142611b48021f673c.json',158),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',159),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',160),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',161),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',162),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',163),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',164),('d450fc8885843d48772df9a898552302f8c80b98.json',165),('d450fc8885843d48772df9a898552302f8c80b98.json',166),('d450fc8885843d48772df9a898552302f8c80b98.json',167),('d450fc8885843d48772df9a898552302f8c80b98.json',168),('d450fc8885843d48772df9a898552302f8c80b98.json',169),('ae19236936b05793ceac08ed5d58269c25486320.json',170),('ae19236936b05793ceac08ed5d58269c25486320.json',171),('ae19236936b05793ceac08ed5d58269c25486320.json',172),('ae19236936b05793ceac08ed5d58269c25486320.json',173),('ae19236936b05793ceac08ed5d58269c25486320.json',174),('ae19236936b05793ceac08ed5d58269c25486320.json',175),('ae19236936b05793ceac08ed5d58269c25486320.json',176),('ae19236936b05793ceac08ed5d58269c25486320.json',177),('ae19236936b05793ceac08ed5d58269c25486320.json',178),('ae19236936b05793ceac08ed5d58269c25486320.json',179),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',180),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',181),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',182),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',183),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',184),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',185),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',186),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',187),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',188),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',189),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',190),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',191),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',192),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',193),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',194),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',195),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',196),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',197),('c6bf372c094f035a514975c35a7f9c094abbe493.json',198),('c6bf372c094f035a514975c35a7f9c094abbe493.json',199),('c6bf372c094f035a514975c35a7f9c094abbe493.json',200),('c6bf372c094f035a514975c35a7f9c094abbe493.json',201),('c6bf372c094f035a514975c35a7f9c094abbe493.json',202),('c6bf372c094f035a514975c35a7f9c094abbe493.json',203),('c6bf372c094f035a514975c35a7f9c094abbe493.json',204),('c6bf372c094f035a514975c35a7f9c094abbe493.json',205),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',206),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',207),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',208),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',209),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',210),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',211),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',212),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',213),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',214),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',215),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',216),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',217),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',218),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',219),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',220),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',221),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',222),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',223),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',224),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',225),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',226),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',227),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',228),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',229),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',230),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',231),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',232),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',233),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',234),('e7f311d3c752063f75f28de85abd5f82eacab413.json',235),('e7f311d3c752063f75f28de85abd5f82eacab413.json',236),('e7f311d3c752063f75f28de85abd5f82eacab413.json',237),('e7f311d3c752063f75f28de85abd5f82eacab413.json',238),('e7f311d3c752063f75f28de85abd5f82eacab413.json',239),('e7f311d3c752063f75f28de85abd5f82eacab413.json',240),('e7f311d3c752063f75f28de85abd5f82eacab413.json',241),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',242),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',243),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',244),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',245),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',246),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',247),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',248),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',249),('4db3931c4b24ae432d68b7e24f593b1272445523.json',250),('4db3931c4b24ae432d68b7e24f593b1272445523.json',251),('4db3931c4b24ae432d68b7e24f593b1272445523.json',252),('4db3931c4b24ae432d68b7e24f593b1272445523.json',253),('4db3931c4b24ae432d68b7e24f593b1272445523.json',254),('40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',255),('40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',256),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',257),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',258),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',259),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',260),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',261),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',262),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',263),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',264),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',265),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',266),('575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',267),('575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',268),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',269),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',270),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',271),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',272),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',273),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',274),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',275),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',276),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',277),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',278),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',279),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',280),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',281),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',282),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',283),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',284),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',285),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',286),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',287),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',288),('f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',289),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',290),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',291),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',292),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',293),('61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',294),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',295),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',296),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',297),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',298),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',299),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',300),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',301),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',302),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',303),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',304),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',305),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',306),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',307),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',308),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',309),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',310),('c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',311),('c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',312),('c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',313),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',314),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',315),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',316),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',317),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',318),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',319),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',320),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',321),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',322),('d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',323),('d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',324),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',325),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',326),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',327),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',328),('d1aafb70c066a2068b02786f8929fd9c900897fb.json',329),('d1aafb70c066a2068b02786f8929fd9c900897fb.json',330),('6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',331),('6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',332),('6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',333),('06ced00a5fc04215949aa72528f2eeaae1d58927.json',334),('5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',335),('5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',336),('5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',337),('b2897e1277f56641193a6db73825f707eed3e4c9.json',338),('b2897e1277f56641193a6db73825f707eed3e4c9.json',339),('b2897e1277f56641193a6db73825f707eed3e4c9.json',340),('b2897e1277f56641193a6db73825f707eed3e4c9.json',341),('3bb07ea10432f7738413dff9816809cc90f03f99.json',342),('3bb07ea10432f7738413dff9816809cc90f03f99.json',343),('3bb07ea10432f7738413dff9816809cc90f03f99.json',344),('5806726a24dc91de3954001effbdffd7a82d54e2.json',345),('5806726a24dc91de3954001effbdffd7a82d54e2.json',346),('faaf1022ccfe93b032c5608097a53543ba24aedb.json',347),('faaf1022ccfe93b032c5608097a53543ba24aedb.json',348),('faaf1022ccfe93b032c5608097a53543ba24aedb.json',349),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',350),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',351),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',352),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',353),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',354),('5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',355),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',356),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',357),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',358),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',359),('9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',360),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',361),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',362),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',363),('14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',364),('d09b79026117ec9faebba46a8d13aa9b23ec751e.json',365),('d09b79026117ec9faebba46a8d13aa9b23ec751e.json',366),('d09b79026117ec9faebba46a8d13aa9b23ec751e.json',367),('44102e3e69e70ad2a73e753133283334ce1f8736.json',368),('44102e3e69e70ad2a73e753133283334ce1f8736.json',369),('44102e3e69e70ad2a73e753133283334ce1f8736.json',370),('44102e3e69e70ad2a73e753133283334ce1f8736.json',371),('44102e3e69e70ad2a73e753133283334ce1f8736.json',372),('44102e3e69e70ad2a73e753133283334ce1f8736.json',373),('44102e3e69e70ad2a73e753133283334ce1f8736.json',374),('30a4842a2e257f725cc041e32fd682c495111a50.json',375),('30a4842a2e257f725cc041e32fd682c495111a50.json',376),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',377),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',378),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',379),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',380),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',381),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',382),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',383),('6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',384),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',385),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',386),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',387),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',388),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',389),('367af6bb9a8bbda02206506a819656bb6f14ce4c.json',390),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',391),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',392),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',393),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',394),('4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',395),('83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',396),('83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',397),('025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',398),('8d7400a2b387820cd391d7df8194642e50402a0c.json',399),('8d7400a2b387820cd391d7df8194642e50402a0c.json',400),('8d7400a2b387820cd391d7df8194642e50402a0c.json',401),('8d7400a2b387820cd391d7df8194642e50402a0c.json',402),('8d7400a2b387820cd391d7df8194642e50402a0c.json',403),('8d7400a2b387820cd391d7df8194642e50402a0c.json',404),('8d7400a2b387820cd391d7df8194642e50402a0c.json',405),('8d7400a2b387820cd391d7df8194642e50402a0c.json',406),('8d7400a2b387820cd391d7df8194642e50402a0c.json',407),('8d7400a2b387820cd391d7df8194642e50402a0c.json',408),('8d7400a2b387820cd391d7df8194642e50402a0c.json',409),('8d7400a2b387820cd391d7df8194642e50402a0c.json',410),('8d7400a2b387820cd391d7df8194642e50402a0c.json',411),('8d7400a2b387820cd391d7df8194642e50402a0c.json',412),('8d7400a2b387820cd391d7df8194642e50402a0c.json',413),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',414),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',415),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',416),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',417),('f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',418),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',419),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',420),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',421),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',422),('867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',423),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',424),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',425),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',426),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',427),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',428),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',429),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',430),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',431),('b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',432),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',433),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',434),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',435),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',436),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',437),('f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',438),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',439),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',440),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',441),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',442),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',443),('03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',444),('a77badb297bf4927419481e660ea8f1806ed91b1.json',445),('a77badb297bf4927419481e660ea8f1806ed91b1.json',446),('a77badb297bf4927419481e660ea8f1806ed91b1.json',447),('a77badb297bf4927419481e660ea8f1806ed91b1.json',448),('a77badb297bf4927419481e660ea8f1806ed91b1.json',449),('a77badb297bf4927419481e660ea8f1806ed91b1.json',450),('a77badb297bf4927419481e660ea8f1806ed91b1.json',451),('a77badb297bf4927419481e660ea8f1806ed91b1.json',452),('a77badb297bf4927419481e660ea8f1806ed91b1.json',453),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',454),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',455),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',456),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',457),('93c6eef32a1a511ee989a259eab0e12174dc6859.json',458),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',459),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',460),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',461),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',462),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',463),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',464),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',465),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',466),('184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',467),('40e41f0c52b39669dee24e37875d7a9fabc38636.json',468),('40e41f0c52b39669dee24e37875d7a9fabc38636.json',469),('d77233dd9085c9c76787c84ee9a1e249424b283a.json',470),('d77233dd9085c9c76787c84ee9a1e249424b283a.json',471),('7d624acf891472c3452b1deb89e0dc44a40d0c94.json',472),('7d624acf891472c3452b1deb89e0dc44a40d0c94.json',473),('d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',474),('ada57a9f084a016deb20b6f83c442072e6e37955.json',475),('ada57a9f084a016deb20b6f83c442072e6e37955.json',476),('ada57a9f084a016deb20b6f83c442072e6e37955.json',477),('6eeaffdb22b4e310867262d1891a1442769c7803.json',478),('6eeaffdb22b4e310867262d1891a1442769c7803.json',479),('6eeaffdb22b4e310867262d1891a1442769c7803.json',480),('6eeaffdb22b4e310867262d1891a1442769c7803.json',481),('6eeaffdb22b4e310867262d1891a1442769c7803.json',482),('dff6195f5e24ebfa2fb4060142611b48021f673c.json',483),('dff6195f5e24ebfa2fb4060142611b48021f673c.json',484),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',485),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',486),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',487),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',488),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',489),('95aec306660c1b7519e9f019d92cc3a8206481e4.json',490),('d450fc8885843d48772df9a898552302f8c80b98.json',491),('d450fc8885843d48772df9a898552302f8c80b98.json',492),('d450fc8885843d48772df9a898552302f8c80b98.json',493),('d450fc8885843d48772df9a898552302f8c80b98.json',494),('d450fc8885843d48772df9a898552302f8c80b98.json',495),('ae19236936b05793ceac08ed5d58269c25486320.json',496),('ae19236936b05793ceac08ed5d58269c25486320.json',497),('ae19236936b05793ceac08ed5d58269c25486320.json',498),('ae19236936b05793ceac08ed5d58269c25486320.json',499),('ae19236936b05793ceac08ed5d58269c25486320.json',500),('ae19236936b05793ceac08ed5d58269c25486320.json',501),('ae19236936b05793ceac08ed5d58269c25486320.json',502),('ae19236936b05793ceac08ed5d58269c25486320.json',503),('ae19236936b05793ceac08ed5d58269c25486320.json',504),('ae19236936b05793ceac08ed5d58269c25486320.json',505),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',506),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',507),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',508),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',509),('0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',510),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',511),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',512),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',513),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',514),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',515),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',516),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',517),('4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',518),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',519),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',520),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',521),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',522),('a78fd1b34372e1e54bf2a192d04aa36670cea307.json',523),('c6bf372c094f035a514975c35a7f9c094abbe493.json',524),('c6bf372c094f035a514975c35a7f9c094abbe493.json',525),('c6bf372c094f035a514975c35a7f9c094abbe493.json',526),('c6bf372c094f035a514975c35a7f9c094abbe493.json',527),('c6bf372c094f035a514975c35a7f9c094abbe493.json',528),('c6bf372c094f035a514975c35a7f9c094abbe493.json',529),('c6bf372c094f035a514975c35a7f9c094abbe493.json',530),('c6bf372c094f035a514975c35a7f9c094abbe493.json',531),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',532),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',533),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',534),('1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',535),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',536),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',537),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',538),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',539),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',540),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',541),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',542),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',543),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',544),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',545),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',546),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',547),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',548),('ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',549),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',550),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',551),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',552),('c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',553),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',554),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',555),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',556),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',557),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',558),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',559),('a1b6c2b3b808e995697f94d9676d5d5c85180177.json',560),('e7f311d3c752063f75f28de85abd5f82eacab413.json',561),('e7f311d3c752063f75f28de85abd5f82eacab413.json',562),('e7f311d3c752063f75f28de85abd5f82eacab413.json',563),('e7f311d3c752063f75f28de85abd5f82eacab413.json',564),('e7f311d3c752063f75f28de85abd5f82eacab413.json',565),('e7f311d3c752063f75f28de85abd5f82eacab413.json',566),('e7f311d3c752063f75f28de85abd5f82eacab413.json',567),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',568),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',569),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',570),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',571),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',572),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',573),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',574),('babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',575),('4db3931c4b24ae432d68b7e24f593b1272445523.json',576),('4db3931c4b24ae432d68b7e24f593b1272445523.json',577),('4db3931c4b24ae432d68b7e24f593b1272445523.json',578),('4db3931c4b24ae432d68b7e24f593b1272445523.json',579),('4db3931c4b24ae432d68b7e24f593b1272445523.json',580),('40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',581),('40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',582),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',583),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',584),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',585),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',586),('0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',587),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',588),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',589),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',590),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',591),('6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',592),('575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',593),('575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',594),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',595),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',596),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',597),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',598),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',599),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',600),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',601),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',602),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',603),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',604),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',605),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',606),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',607),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',608),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',609),('6bbbd25c11b320bdd509a9c76417ed4045349708.json',610),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',611),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',612),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',613),('4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',614),('f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',615),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',616),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',617),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',618),('94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',619),('61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',620),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',621),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',622),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',623),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',624),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',625),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',626),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',627),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',628),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',629),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',630),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',631),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',632),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',633),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',634),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',635),('315e5980d6fdce4553a9df8c0e36e55ffed32365.json',636),('c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',637),('c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',638),('c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',639),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',640),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',641),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',642),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',643),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',644),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',645),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',646),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',647),('25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',648),('d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',649),('d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',650),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',651),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',652),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',653),('bdcfff55eaabeaa26d71810113d717ef85e417d1.json',654),('d90d787828b6f96b4a93eb6948d613293bc51bf9.json',655),('d90d787828b6f96b4a93eb6948d613293bc51bf9.json',656),('d90d787828b6f96b4a93eb6948d613293bc51bf9.json',657),('d90d787828b6f96b4a93eb6948d613293bc51bf9.json',658),('d90d787828b6f96b4a93eb6948d613293bc51bf9.json',659),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',660),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',661),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',662),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',663),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',664),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',665),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',666),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',667),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',668),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',669),('ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',670),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',671),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',672),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',673),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',674),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',675),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',676),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',677),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',678),('52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',679),('12de3563e18c3c6c5943fa8216e0eea970a590cd.json',680),('12de3563e18c3c6c5943fa8216e0eea970a590cd.json',681),('12de3563e18c3c6c5943fa8216e0eea970a590cd.json',682),('12de3563e18c3c6c5943fa8216e0eea970a590cd.json',683),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',684),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',685),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',686),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',687),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',688),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',689),('c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',690),('c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',691),('c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',692),('c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',693),('c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',694),('6af02873a430f5f75dc56a917f26feaea89b8e79.json',695),('6af02873a430f5f75dc56a917f26feaea89b8e79.json',696),('6af02873a430f5f75dc56a917f26feaea89b8e79.json',697),('9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',698),('9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',699),('9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',700),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',701),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',702),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',703),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',704),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',705),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',706),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',707),('b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',708),('53dbd21da3e3a026ab15e946639b45970d6304de.json',709),('53dbd21da3e3a026ab15e946639b45970d6304de.json',710),('53dbd21da3e3a026ab15e946639b45970d6304de.json',711),('53dbd21da3e3a026ab15e946639b45970d6304de.json',712),('53dbd21da3e3a026ab15e946639b45970d6304de.json',713),('9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',714),('9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',715),('9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',716),('b3ab47699907f78c2ac478d06f682f56fa0cf351.json',717),('b3ab47699907f78c2ac478d06f682f56fa0cf351.json',718),('c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',719),('c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',720),('c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',721),('c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',722),('b0a0f45be3069a32ba190502ab88b6df6d520199.json',723),('b0a0f45be3069a32ba190502ab88b6df6d520199.json',724),('b0a0f45be3069a32ba190502ab88b6df6d520199.json',725),('b0a0f45be3069a32ba190502ab88b6df6d520199.json',726),('b0a0f45be3069a32ba190502ab88b6df6d520199.json',727),('d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',728),('d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',729),('d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',730),('f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',731),('f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',732),('f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',733),('f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',734),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',735),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',736),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',737),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',738),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',739),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',740),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',741),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',742),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',743),('323ece9056ef7a1d81faf3422e98def1efd38f70.json',744),('305cf1019ca9c81126b2e527fedf79969c2c522f.json',745),('305cf1019ca9c81126b2e527fedf79969c2c522f.json',746),('305cf1019ca9c81126b2e527fedf79969c2c522f.json',747),('305cf1019ca9c81126b2e527fedf79969c2c522f.json',748),('6ea2532aacbdcea4609067523e9e72fa8cd73200.json',749),('6ea2532aacbdcea4609067523e9e72fa8cd73200.json',750),('6ea2532aacbdcea4609067523e9e72fa8cd73200.json',751),('6ea2532aacbdcea4609067523e9e72fa8cd73200.json',752),('6ea2532aacbdcea4609067523e9e72fa8cd73200.json',753),('e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',754),('e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',755),('e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',756),('e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',757),('435baf47b4f78fc06a8376f451993059407d2972.json',758),('435baf47b4f78fc06a8376f451993059407d2972.json',759),('435baf47b4f78fc06a8376f451993059407d2972.json',760),('435baf47b4f78fc06a8376f451993059407d2972.json',761),('435baf47b4f78fc06a8376f451993059407d2972.json',762),('c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',763),('c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',764),('8f5770d87614bb85a6c7b64c26f1757226137d23.json',765),('8f5770d87614bb85a6c7b64c26f1757226137d23.json',766),('8f5770d87614bb85a6c7b64c26f1757226137d23.json',767),('8f5770d87614bb85a6c7b64c26f1757226137d23.json',768),('8f5770d87614bb85a6c7b64c26f1757226137d23.json',769),('8f5770d87614bb85a6c7b64c26f1757226137d23.json',770),('418f1b85e589f3ac37ffc37075b6e558e510fa74.json',771),('418f1b85e589f3ac37ffc37075b6e558e510fa74.json',772),('418f1b85e589f3ac37ffc37075b6e558e510fa74.json',773),('2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',774),('2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',775),('2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',776),('2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',777),('2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',778),('2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',779),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',780),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',781),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',782),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',783),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',784),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',785),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',786),('4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',787),('8309dc172cf91ff580779bbc1394a9692b798a86.json',788),('8309dc172cf91ff580779bbc1394a9692b798a86.json',789),('8309dc172cf91ff580779bbc1394a9692b798a86.json',790),('8309dc172cf91ff580779bbc1394a9692b798a86.json',791),('8309dc172cf91ff580779bbc1394a9692b798a86.json',792),('8309dc172cf91ff580779bbc1394a9692b798a86.json',793),('281b4272005dbc998e36b034b7fb528c9ba038cd.json',794),('281b4272005dbc998e36b034b7fb528c9ba038cd.json',795),('281b4272005dbc998e36b034b7fb528c9ba038cd.json',796),('281b4272005dbc998e36b034b7fb528c9ba038cd.json',797),('281b4272005dbc998e36b034b7fb528c9ba038cd.json',798),('281b4272005dbc998e36b034b7fb528c9ba038cd.json',799),('4994fa72322bbf19120592304d92629226948d8e.json',800),('4994fa72322bbf19120592304d92629226948d8e.json',801),('4994fa72322bbf19120592304d92629226948d8e.json',802),('4994fa72322bbf19120592304d92629226948d8e.json',803),('4994fa72322bbf19120592304d92629226948d8e.json',804),('d47eb01f2085b582008bb1ff7c604355c5f4c582.json',805),('d47eb01f2085b582008bb1ff7c604355c5f4c582.json',806),('d47eb01f2085b582008bb1ff7c604355c5f4c582.json',807),('d47eb01f2085b582008bb1ff7c604355c5f4c582.json',808),('555bbbbd23da02c97c6a4238937e31b5781438a9.json',809),('555bbbbd23da02c97c6a4238937e31b5781438a9.json',810),('555bbbbd23da02c97c6a4238937e31b5781438a9.json',811),('555bbbbd23da02c97c6a4238937e31b5781438a9.json',812),('555bbbbd23da02c97c6a4238937e31b5781438a9.json',813),('bada3cc0577a226694617d77f95ac443c6431969.json',814),('b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',815),('fb3c146b1766b4aa33df1637742bf93159967289.json',816),('fb3c146b1766b4aa33df1637742bf93159967289.json',817),('fb3c146b1766b4aa33df1637742bf93159967289.json',818),('fb3c146b1766b4aa33df1637742bf93159967289.json',819),('fb3c146b1766b4aa33df1637742bf93159967289.json',820),('fb3c146b1766b4aa33df1637742bf93159967289.json',821),('fb3c146b1766b4aa33df1637742bf93159967289.json',822),('fb3c146b1766b4aa33df1637742bf93159967289.json',823),('fb3c146b1766b4aa33df1637742bf93159967289.json',824),('fb3c146b1766b4aa33df1637742bf93159967289.json',825),('fb3c146b1766b4aa33df1637742bf93159967289.json',826);
/*!40000 ALTER TABLE `paper_to_author` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `word_to_paper`
--

DROP TABLE IF EXISTS `word_to_paper`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `word_to_paper` (
  `fk_word_id` int NOT NULL,
  `fk_paper_id` varchar(200) NOT NULL,
  `counter` int DEFAULT NULL,
  PRIMARY KEY (`fk_word_id`,`fk_paper_id`),
  KEY `fk_paper_word_to_paper_idx` (`fk_paper_id`),
  CONSTRAINT `fk_paper_word_to_paper` FOREIGN KEY (`fk_paper_id`) REFERENCES `Paper` (`idPaper`),
  CONSTRAINT `fk_word_word_to_paper` FOREIGN KEY (`fk_word_id`) REFERENCES `Word` (`idWord`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `word_to_paper`
--

LOCK TABLES `word_to_paper` WRITE;
/*!40000 ALTER TABLE `word_to_paper` DISABLE KEYS */;
INSERT INTO `word_to_paper` VALUES (1,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(2,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',24),(3,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',14),(4,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',7),(5,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',8),(6,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',7),(7,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',22),(8,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',11),(9,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',31),(10,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(11,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(13,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(14,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',51),(15,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(16,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(17,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(18,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',7),(19,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(20,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(23,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',9),(24,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(25,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(26,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(27,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(28,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(29,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',6),(30,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(31,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(32,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',32),(33,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(34,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(35,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(36,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(37,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(38,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(39,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(40,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(42,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(44,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(45,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(46,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(47,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',9),(48,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(52,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(53,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(54,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(55,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(56,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(57,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(58,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(59,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(60,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(61,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(62,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(63,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(64,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(65,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(66,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(67,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(68,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(69,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(70,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(71,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(74,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(75,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(76,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(77,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(78,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(83,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(84,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(85,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',8),(86,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(87,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',7),(88,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(89,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(95,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(96,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(97,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(99,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(100,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(102,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(103,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(104,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',45),(107,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(110,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(111,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(112,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(113,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',6),(115,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(116,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(121,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(122,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(123,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(124,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(125,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(126,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(127,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(130,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(137,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(138,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(139,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(140,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(141,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(144,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(145,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(146,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(148,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(149,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',7),(153,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(154,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(161,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(163,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(164,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(165,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(166,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(169,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(170,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(171,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(173,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(176,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(177,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',11),(179,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(180,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(181,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(184,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(186,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(187,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(189,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(190,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(192,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(193,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(194,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(195,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(196,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(197,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(198,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(199,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(200,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(204,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(205,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(206,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(207,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(208,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(211,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(212,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(213,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(214,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(215,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(216,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(217,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(218,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(219,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(220,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(221,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(224,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(226,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(227,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(228,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(229,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(230,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(231,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(232,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(233,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(234,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(236,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(237,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(238,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(241,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(245,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(246,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(249,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(250,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(252,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(253,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(254,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(255,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(256,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',8),(257,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(259,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(262,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(266,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(270,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(271,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(272,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(274,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(275,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(277,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(279,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(280,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(281,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(282,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(283,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(284,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(285,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(286,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(287,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(288,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(289,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(290,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(291,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(293,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(295,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(296,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(297,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(302,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(303,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(304,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(305,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(307,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(309,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(313,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(316,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(317,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(318,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(319,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(320,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(321,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(322,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(323,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(324,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(325,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(327,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(328,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(329,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(330,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(331,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(333,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(334,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(335,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(336,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(346,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(347,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(349,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(350,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(355,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(357,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(361,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(371,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(374,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(377,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(380,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',6),(381,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',9),(382,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',8),(383,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(388,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(390,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(391,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(392,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(393,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(394,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(395,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(396,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(398,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(400,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(401,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(402,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(403,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(404,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(405,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(410,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(411,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(412,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(413,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',6),(414,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(415,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(416,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(417,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(418,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(419,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(420,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(423,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(424,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(426,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(427,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(428,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(429,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(430,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(432,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(438,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(439,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(441,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(442,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(443,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(445,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(449,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(455,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(456,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(457,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(458,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(459,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(472,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(473,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(476,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(477,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',7),(479,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(485,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(489,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(490,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(493,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',15),(497,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(504,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(505,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(506,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(507,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(508,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(509,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(511,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(512,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(513,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(515,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(516,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(521,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(541,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(543,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(547,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(548,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(549,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(553,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(563,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(564,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(568,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(569,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(570,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(572,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(573,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(574,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(576,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(583,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(593,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(598,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(601,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(602,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(606,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(616,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(622,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(624,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(626,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(630,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(631,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(645,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(646,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(650,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(651,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(652,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(654,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',8),(655,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(658,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(659,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(660,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(663,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(664,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(665,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(666,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(669,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(670,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(673,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(674,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(679,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(680,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(681,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(682,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(685,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(686,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(687,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(691,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(694,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(695,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(696,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(697,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',10),(698,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',7),(701,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(702,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(708,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(709,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(711,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(713,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(714,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(715,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(716,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(718,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(719,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(720,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(721,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(722,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(724,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(725,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(727,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(728,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(729,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(730,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(732,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(733,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(734,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(736,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(737,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(738,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(740,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(742,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(743,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(744,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(745,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(746,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(748,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(750,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(751,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(752,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(753,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(756,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(757,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(760,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(762,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(765,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(771,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(774,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(777,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(786,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(787,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(798,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(799,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(808,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(810,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(814,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(842,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(843,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(844,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(850,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(851,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(852,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(853,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(855,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(856,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(857,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(858,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(861,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',4),(863,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(864,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(871,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(874,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(882,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(886,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(891,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(893,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',3),(899,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(901,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(902,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(903,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(904,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(905,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(918,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(929,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',5),(931,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(935,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(936,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(939,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(944,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(946,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(947,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',2),(956,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(958,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(960,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(970,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(972,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(979,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(981,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(982,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(983,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(984,'d1aafb70c066a2068b02786f8929fd9c900897fb.json',1),(1975,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(1976,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(1977,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1978,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(1980,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',7),(1981,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1983,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1984,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(1985,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1986,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(1987,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(1991,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1992,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1993,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1994,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1995,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1996,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(1997,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(1998,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(1999,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',36),(2000,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2004,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2005,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2006,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2007,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2008,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2010,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(2011,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2012,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2017,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2018,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2020,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2022,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2023,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2024,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2026,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2027,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2030,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',14),(2032,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2033,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2034,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2035,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2036,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',7),(2037,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2040,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2041,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2042,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2043,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2044,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2045,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2048,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',13),(2049,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',76),(2051,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2061,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2062,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2064,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2065,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2066,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2072,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2073,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2074,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2075,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',7),(2076,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2077,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2078,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2080,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',13),(2084,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2085,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2087,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2088,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',7),(2090,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2091,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2092,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2093,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2095,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',20),(2097,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2098,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',12),(2100,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2101,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2102,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2103,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2105,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2106,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',16),(2108,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2109,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2110,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2112,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2113,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2117,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2118,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2119,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2122,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2124,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',10),(2125,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2126,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2128,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2130,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2131,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2132,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2133,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2134,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2135,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2136,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2138,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2140,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2141,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2143,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2144,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2146,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2147,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2148,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2149,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2153,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2157,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2158,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2159,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2161,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2162,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2163,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',13),(2165,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2168,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2171,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2172,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(2173,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2174,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2176,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2177,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2178,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2179,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2181,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',10),(2182,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2184,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2185,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2186,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(2187,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',13),(2191,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2192,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2193,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2199,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2204,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2205,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2207,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2208,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2210,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2212,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2215,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2216,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2217,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2222,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2223,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2224,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2236,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2237,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2244,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2245,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2251,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2253,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2254,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2255,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2256,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2257,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2258,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2262,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2263,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2264,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2265,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2266,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2267,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2268,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2274,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2276,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2281,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2286,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2288,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2289,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2290,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2293,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2296,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2297,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2298,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2300,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2302,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2304,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2305,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2308,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2309,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',10),(2311,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2314,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2315,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2318,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2319,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2320,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2321,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2330,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2331,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2334,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2335,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2336,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2338,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2339,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2346,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2347,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2348,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2351,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2353,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2354,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2356,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2359,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2361,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2362,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2363,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2364,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2365,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2370,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2371,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2380,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2384,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2394,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2396,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2406,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2412,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2413,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2414,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2415,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2416,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2418,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(2425,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2434,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2435,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',12),(2438,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2444,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2445,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2447,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2453,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2454,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2456,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2457,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(2462,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2463,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2467,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2469,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2476,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2477,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2480,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2482,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2483,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2485,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2487,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',11),(2490,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',25),(2491,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2492,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2494,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2502,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2504,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2508,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(2510,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2512,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2513,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2514,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2515,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2519,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2520,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2524,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2526,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2527,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2529,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2530,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2533,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2537,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',19),(2538,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2539,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2540,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2543,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',14),(2545,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',38),(2552,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2553,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2559,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2565,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2566,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2568,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2574,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2575,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2578,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',26),(2583,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2590,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2591,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2597,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2600,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2607,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2612,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2613,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2614,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',7),(2615,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2616,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2617,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2619,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2620,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2621,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2622,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2628,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2630,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',10),(2634,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2636,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',6),(2637,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2638,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2641,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2644,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2645,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2655,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2659,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2661,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',8),(2662,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2664,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2665,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2671,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2678,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2679,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2686,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2692,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2693,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2694,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2695,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2696,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2697,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2698,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2700,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2701,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2705,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2706,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2710,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2712,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',7),(2722,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2724,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2727,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2733,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2738,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2739,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2740,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2743,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2745,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2747,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2748,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2750,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2756,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2758,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2760,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2766,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2769,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2772,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2773,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2775,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2777,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(2778,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2784,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2787,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2788,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2789,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2797,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2799,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2800,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',10),(2801,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2802,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2803,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2808,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2810,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2815,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2821,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2823,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2827,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2828,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2830,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2831,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2832,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2833,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2834,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2835,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2836,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2837,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2838,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2839,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2843,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2845,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2847,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2848,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2851,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2852,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2854,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2862,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2864,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2870,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(2871,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2893,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2896,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2902,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2903,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2906,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2908,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2929,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2933,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2934,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2942,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2948,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2949,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2950,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2951,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(2952,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',176),(2953,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2962,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2963,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2965,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2972,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2974,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2977,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2979,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2980,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2981,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2982,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(2984,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2987,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2991,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(2995,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3001,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3002,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3003,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3004,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3006,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3013,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3015,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3018,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3020,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3021,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3024,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3031,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(3032,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3034,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3037,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3038,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3039,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3043,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',9),(3046,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3049,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3054,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3057,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3058,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3059,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3061,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(3063,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3066,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3068,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3069,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3071,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3073,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3078,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3081,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3090,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3091,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3092,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3094,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3096,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3098,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3101,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(3115,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3124,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',11),(3130,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(3131,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',27),(3132,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3134,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3137,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3141,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3143,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3148,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3150,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3152,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3156,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3161,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',109),(3165,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3173,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3175,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3176,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3182,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3186,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',4),(3190,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3191,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3193,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3195,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(3211,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3212,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3221,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3222,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3223,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3225,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3229,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3230,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3231,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3233,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3239,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3240,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(3244,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3245,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3248,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3251,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3252,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3256,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',5),(3268,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3269,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(3273,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3288,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3305,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3306,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3308,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3310,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3317,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3319,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',3),(3321,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3329,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3345,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3349,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3356,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3371,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3375,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3383,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3390,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3394,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3406,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3407,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3409,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3411,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3420,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3424,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3425,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3427,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3428,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3429,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3430,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',61),(3436,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3448,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3460,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3461,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',2),(3468,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3472,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3474,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3476,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3478,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3480,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3483,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3496,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3505,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3511,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3512,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3513,'6b0567729c2143a66d737eb0a2f63f2dce2e5a7d.json',1),(3521,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3523,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',122),(3524,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3525,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3526,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3527,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(3529,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',31),(3530,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(3536,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3537,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',12),(3538,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',65),(3541,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',66),(3542,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',102),(3543,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3544,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(3545,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',9),(3546,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',10),(3547,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3548,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3549,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',12),(3553,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3556,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',20),(3557,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3559,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3563,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(3565,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3568,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3569,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3571,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3578,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3588,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3589,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3592,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3594,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3606,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3608,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3612,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3613,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3617,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3618,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3621,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3622,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3632,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3633,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(3642,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3643,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3645,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3646,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3649,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3650,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3651,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(3652,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3653,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3658,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3660,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3666,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3669,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3670,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3671,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3672,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(3675,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3677,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3678,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3680,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3681,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3685,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3689,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3690,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3691,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3693,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3694,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',13),(3695,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3702,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3706,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3712,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3714,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3717,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',12),(3719,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3724,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3725,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3726,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3727,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3728,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3731,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3732,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3733,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3734,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3735,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3739,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3742,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3745,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3748,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3749,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3750,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3751,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(3754,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3756,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3760,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3762,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',17),(3772,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3773,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3774,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3776,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3780,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3781,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3782,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3789,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3791,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3792,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3794,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',7),(3795,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3796,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3797,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',25),(3799,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',18),(3806,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3807,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3808,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3809,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3811,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3814,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3816,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3817,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3819,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',13),(3822,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',17),(3823,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',12),(3826,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3827,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3831,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3832,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',9),(3835,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3836,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(3837,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3842,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3845,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3846,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3848,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',24),(3856,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3857,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',10),(3863,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3865,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3873,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3874,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3878,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3879,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',18),(3880,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',30),(3882,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3890,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3891,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3892,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(3893,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3898,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3904,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(3906,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3907,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(3909,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3911,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',34),(3914,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3916,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3922,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3927,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',7),(3928,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3932,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3933,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3934,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3936,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(3942,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3943,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(3946,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',11),(3947,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3949,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3950,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3954,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3964,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3976,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3977,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3978,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3980,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3984,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(3990,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(3997,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',14),(3999,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4002,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4003,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4004,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4005,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4006,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4009,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4014,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4015,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4018,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4019,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4023,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4028,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4034,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4035,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4037,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4041,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4046,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4049,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4050,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4053,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4054,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4059,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4060,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4064,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4065,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4066,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4071,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4072,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4085,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4087,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4095,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',10),(4101,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4111,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4113,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4114,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4115,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4119,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4121,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4122,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',7),(4124,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4125,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4127,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4128,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4131,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4143,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4149,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4152,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4162,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4167,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4169,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4170,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4175,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4180,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4182,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4183,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4189,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4190,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4196,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4200,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4201,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4202,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4203,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4204,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4205,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4206,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4207,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4208,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4211,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4213,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4214,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4218,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4221,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4228,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4235,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4239,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4241,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4244,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',7),(4249,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4252,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(4253,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4254,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4274,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4276,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4277,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4278,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4281,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4296,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4298,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4300,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4314,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4322,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4335,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4345,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4348,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4351,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4365,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4371,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4373,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4379,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4381,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',42),(4383,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4388,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4397,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4398,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4413,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4414,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(4416,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4427,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4429,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',10),(4430,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4440,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(4442,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(4447,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4453,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4454,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4455,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4456,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4464,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4470,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4471,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',17),(4475,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',7),(4481,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4482,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4483,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4485,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4486,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4489,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4490,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4494,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4499,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4508,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4511,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4513,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',10),(4524,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4536,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4540,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4545,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4548,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4549,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4551,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',9),(4564,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4568,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4572,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4582,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4593,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4598,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4602,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4603,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4604,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4606,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4621,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',16),(4633,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4635,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4640,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',122),(4647,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4648,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4649,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4650,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4651,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4669,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4676,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4677,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4686,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4687,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(4692,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4693,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4694,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4695,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4696,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4699,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4700,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4706,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4707,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4708,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4709,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4720,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4722,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4724,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4729,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4735,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4738,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4744,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4752,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4757,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(4769,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4771,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4773,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',11),(4783,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4784,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4787,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4797,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4798,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4804,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4805,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4813,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4818,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4837,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',5),(4839,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(4846,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4858,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4860,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4861,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4864,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4866,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4867,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4868,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4869,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4874,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4877,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4882,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4889,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4896,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4905,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4908,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4910,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4911,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4923,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4925,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(4943,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4949,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4971,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',8),(4977,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(4979,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4986,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4987,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4990,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4991,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4994,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(4996,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5006,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5007,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5008,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5032,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',12),(5045,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5057,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',67),(5062,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5074,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5075,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(5078,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5080,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5082,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(5103,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5109,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5115,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5119,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5131,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5132,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5133,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5138,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5142,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5144,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5147,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5158,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5160,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',19),(5161,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5162,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5180,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5189,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5193,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5194,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5196,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5197,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5199,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5200,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5201,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5202,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(5211,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5217,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5218,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5229,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5236,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5243,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5244,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5256,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5258,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5259,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5277,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5280,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5281,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5303,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5307,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5308,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5309,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5310,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5312,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5349,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5354,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5366,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5369,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5395,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5417,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5418,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5425,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5426,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5427,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5430,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5433,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5438,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5439,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5445,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5455,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5456,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5462,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5463,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5474,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5487,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5495,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5498,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5499,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5506,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5507,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5510,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5512,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5519,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5520,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5521,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5527,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5537,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5548,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5549,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5557,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5559,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5562,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5568,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5572,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5577,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5580,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5589,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5596,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5599,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5601,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5603,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5604,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5606,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5611,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5613,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5614,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5617,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5618,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5623,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5626,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5627,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5636,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',6),(5645,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5668,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5674,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5675,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',95),(5677,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5684,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5687,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5700,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5702,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5704,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(5706,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5710,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5717,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5724,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5727,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5762,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5766,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5769,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5770,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5774,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5785,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5790,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5791,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5792,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5803,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5804,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(5806,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5807,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5808,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5812,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(5822,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5827,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5831,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5833,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5836,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5839,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5848,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5866,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5870,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5886,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5897,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5904,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5912,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5913,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5914,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5918,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5929,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(5933,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5936,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5938,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5941,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5953,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5955,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5959,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5962,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5977,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5978,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5985,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5992,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(5993,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',33),(5999,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(6001,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6004,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(6011,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6031,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6040,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6042,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6055,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6059,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(6071,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6074,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6082,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6085,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6098,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(6099,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(6105,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',7),(6118,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',4),(6121,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6125,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6133,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6137,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6138,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6140,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6149,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6150,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6153,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',3),(6160,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6170,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6171,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6173,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6188,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6200,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6207,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(6211,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6215,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6219,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6222,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6226,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6230,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(6231,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6234,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6238,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6240,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6248,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6249,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6255,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6257,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6262,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6264,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6276,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',2),(6283,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6308,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6321,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6340,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6365,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6376,'06ced00a5fc04215949aa72528f2eeaae1d58927.json',1),(6385,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6386,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6387,'348055649b6b8cf2b9a376498df9bf41f7123605.json',30),(6393,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6398,'348055649b6b8cf2b9a376498df9bf41f7123605.json',24),(6401,'348055649b6b8cf2b9a376498df9bf41f7123605.json',7),(6403,'348055649b6b8cf2b9a376498df9bf41f7123605.json',7),(6410,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6418,'348055649b6b8cf2b9a376498df9bf41f7123605.json',13),(6423,'348055649b6b8cf2b9a376498df9bf41f7123605.json',13),(6436,'348055649b6b8cf2b9a376498df9bf41f7123605.json',9),(6438,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6449,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6455,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6458,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6461,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6462,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6463,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6464,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6466,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6467,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6468,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6483,'348055649b6b8cf2b9a376498df9bf41f7123605.json',12),(6484,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6486,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6488,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6492,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6494,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6496,'348055649b6b8cf2b9a376498df9bf41f7123605.json',9),(6501,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6502,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6506,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6509,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6513,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6523,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(6524,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6533,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6536,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6553,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6556,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6560,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6562,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6578,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6580,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6592,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6595,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6596,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6604,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6617,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6623,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6626,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6627,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6628,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6637,'348055649b6b8cf2b9a376498df9bf41f7123605.json',5),(6643,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6645,'348055649b6b8cf2b9a376498df9bf41f7123605.json',31),(6646,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6653,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6668,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(6671,'348055649b6b8cf2b9a376498df9bf41f7123605.json',14),(6672,'348055649b6b8cf2b9a376498df9bf41f7123605.json',5),(6676,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6677,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6678,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6694,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6695,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6697,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6723,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6724,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6734,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6737,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6738,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6740,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6743,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6744,'348055649b6b8cf2b9a376498df9bf41f7123605.json',7),(6761,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6762,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6769,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6774,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6797,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6802,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6832,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6838,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6841,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6846,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6847,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6861,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6864,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6889,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6897,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6899,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6900,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6902,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6907,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6908,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6920,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6923,'348055649b6b8cf2b9a376498df9bf41f7123605.json',7),(6931,'348055649b6b8cf2b9a376498df9bf41f7123605.json',19),(6934,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(6936,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6940,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6942,'348055649b6b8cf2b9a376498df9bf41f7123605.json',11),(6945,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6946,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6972,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6974,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6981,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6982,'348055649b6b8cf2b9a376498df9bf41f7123605.json',6),(6986,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6990,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(6992,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6994,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(6996,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7003,'348055649b6b8cf2b9a376498df9bf41f7123605.json',8),(7006,'348055649b6b8cf2b9a376498df9bf41f7123605.json',14),(7013,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7016,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(7024,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7027,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7029,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7037,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7043,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7054,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7061,'348055649b6b8cf2b9a376498df9bf41f7123605.json',5),(7065,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7069,'348055649b6b8cf2b9a376498df9bf41f7123605.json',7),(7072,'348055649b6b8cf2b9a376498df9bf41f7123605.json',5),(7074,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7084,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7089,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7098,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7107,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7139,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7140,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7141,'348055649b6b8cf2b9a376498df9bf41f7123605.json',6),(7143,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7156,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7161,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7166,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7167,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7176,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7179,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7180,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7184,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7185,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7190,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7198,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7199,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7214,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7224,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7241,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7245,'348055649b6b8cf2b9a376498df9bf41f7123605.json',8),(7251,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7256,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7260,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7272,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(7273,'348055649b6b8cf2b9a376498df9bf41f7123605.json',34),(7282,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7293,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7298,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7299,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7309,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7310,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(7369,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7370,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7380,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7394,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7399,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7406,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7408,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7416,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7420,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7423,'348055649b6b8cf2b9a376498df9bf41f7123605.json',41),(7424,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(7425,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7441,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7444,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7466,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(7467,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7468,'348055649b6b8cf2b9a376498df9bf41f7123605.json',12),(7498,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7499,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7500,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7503,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7520,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7521,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7526,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7538,'348055649b6b8cf2b9a376498df9bf41f7123605.json',5),(7539,'348055649b6b8cf2b9a376498df9bf41f7123605.json',11),(7556,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7561,'348055649b6b8cf2b9a376498df9bf41f7123605.json',7),(7572,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7585,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7600,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7613,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(7616,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7620,'348055649b6b8cf2b9a376498df9bf41f7123605.json',17),(7637,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7645,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7651,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7656,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7662,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7673,'348055649b6b8cf2b9a376498df9bf41f7123605.json',12),(7690,'348055649b6b8cf2b9a376498df9bf41f7123605.json',129),(7691,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7695,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7704,'348055649b6b8cf2b9a376498df9bf41f7123605.json',6),(7714,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7717,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7720,'348055649b6b8cf2b9a376498df9bf41f7123605.json',6),(7732,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7736,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7737,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7741,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7752,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7761,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7775,'348055649b6b8cf2b9a376498df9bf41f7123605.json',14),(7776,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7801,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7802,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7804,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7809,'348055649b6b8cf2b9a376498df9bf41f7123605.json',8),(7827,'348055649b6b8cf2b9a376498df9bf41f7123605.json',6),(7835,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(7841,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7850,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7857,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7866,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7876,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7883,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7886,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7896,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7928,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(7929,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7936,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7945,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7961,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7967,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7982,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7992,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(7996,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8016,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(8021,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8028,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8029,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8032,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8036,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8043,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8051,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8052,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8062,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8079,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8080,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8082,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8087,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8088,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8098,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8105,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8115,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8143,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8144,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8145,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8146,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8147,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8152,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8163,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8177,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8193,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8195,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(8198,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(8202,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8212,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8223,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8229,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8235,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8252,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8268,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8269,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8272,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8275,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8278,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8286,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8300,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8301,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8302,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8315,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8328,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8347,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8352,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(8400,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8413,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8414,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8419,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8426,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8443,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8446,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8452,'348055649b6b8cf2b9a376498df9bf41f7123605.json',3),(8474,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8475,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8492,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8504,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8517,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8519,'348055649b6b8cf2b9a376498df9bf41f7123605.json',5),(8522,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8531,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8533,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8534,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8536,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8541,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8553,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8554,'348055649b6b8cf2b9a376498df9bf41f7123605.json',4),(8570,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8656,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8679,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8698,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8699,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8705,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8746,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8752,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8754,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8756,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8773,'348055649b6b8cf2b9a376498df9bf41f7123605.json',2),(8796,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8797,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8799,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8819,'348055649b6b8cf2b9a376498df9bf41f7123605.json',1),(8822,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',8),(8825,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8827,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',15),(8828,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8829,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8830,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8834,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8847,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(8850,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8852,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(8855,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8856,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(8859,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8860,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8864,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8882,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8891,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(8918,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',5),(8920,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',10),(8932,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',86),(8941,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(8942,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(8944,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(8946,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',7),(8951,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',6),(8954,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8959,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(8967,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8968,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(8970,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8974,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8978,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(8986,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9016,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9029,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9033,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9037,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9059,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9061,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(9067,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9068,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9078,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',17),(9086,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9117,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9124,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',5),(9127,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9129,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9139,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9148,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9160,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9173,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9177,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9185,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',7),(9208,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9211,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9215,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9223,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9224,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9228,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9262,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9274,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9276,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9289,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9295,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',7),(9315,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9336,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9340,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9359,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9376,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9383,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9397,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9419,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9426,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9428,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9449,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9458,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9464,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9465,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9473,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9479,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9489,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9490,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9491,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9493,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9497,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9499,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9506,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9511,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9512,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9513,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9519,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9533,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9539,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9541,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9543,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9547,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9555,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9567,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9569,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9574,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9579,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9582,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9585,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9596,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9602,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(9606,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9613,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9616,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9624,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9626,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9628,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9631,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9636,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9646,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9649,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9668,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9673,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9676,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9679,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9696,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9702,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9708,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9757,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9760,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9769,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9782,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9785,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9811,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9819,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9827,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9832,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9834,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9838,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',10),(9839,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9852,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9853,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9854,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9856,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9889,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9892,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9912,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9916,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9926,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9927,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9932,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(9945,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9948,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9950,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(9977,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(9988,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(9995,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10005,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(10009,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10027,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10033,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10034,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10037,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10054,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10055,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10056,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10080,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10084,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(10085,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',157),(10094,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10127,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10142,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10143,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10153,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10155,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10156,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10161,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10168,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10173,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10177,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(10181,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10183,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10184,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(10189,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',12),(10191,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10200,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10201,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10202,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(10217,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10223,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10225,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(10228,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10229,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10230,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10231,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10232,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10234,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10236,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10238,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10244,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10245,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10249,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10263,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',8),(10264,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10265,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10274,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(10282,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10283,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10284,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',5),(10285,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10293,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10294,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10297,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10313,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',6),(10315,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10316,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(10318,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10319,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10322,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10334,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(10346,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10354,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10355,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10359,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',5),(10376,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10377,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10381,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10384,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10385,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10392,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10420,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10426,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',5),(10442,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10455,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10466,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10482,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',14),(10487,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10522,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10536,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10552,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10562,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10566,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',3),(10567,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10572,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',5),(10576,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10577,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10578,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10579,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10580,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10581,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10582,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10583,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10584,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10585,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10589,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10601,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10605,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10608,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10625,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10631,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10632,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10634,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10635,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10637,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10641,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10643,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10647,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',5),(10648,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(10649,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10650,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10659,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10667,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(10668,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10674,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10675,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10676,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10679,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10680,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10681,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10690,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10694,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10695,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10701,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',4),(10706,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10709,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10724,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10742,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10756,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10766,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10771,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10782,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10797,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10812,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10814,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10823,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10827,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10828,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10830,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10836,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',1),(10847,'5f48792a5fa08bed9f56016f4981ae2ca6031b32.json',2),(10877,'b2897e1277f56641193a6db73825f707eed3e4c9.json',28),(10878,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10879,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10882,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10886,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(10889,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(10893,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(10899,'b2897e1277f56641193a6db73825f707eed3e4c9.json',71),(10904,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(10905,'b2897e1277f56641193a6db73825f707eed3e4c9.json',110),(10908,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(10909,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(10915,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(10916,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10919,'b2897e1277f56641193a6db73825f707eed3e4c9.json',15),(10922,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10940,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10941,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(10943,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(10947,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10949,'b2897e1277f56641193a6db73825f707eed3e4c9.json',36),(10952,'b2897e1277f56641193a6db73825f707eed3e4c9.json',27),(10956,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(10957,'b2897e1277f56641193a6db73825f707eed3e4c9.json',27),(10959,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10960,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10963,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10966,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(10976,'b2897e1277f56641193a6db73825f707eed3e4c9.json',11),(10978,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10982,'b2897e1277f56641193a6db73825f707eed3e4c9.json',66),(10987,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(10989,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(10990,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(10995,'b2897e1277f56641193a6db73825f707eed3e4c9.json',19),(11002,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11004,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11005,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11006,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(11007,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11008,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11009,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11010,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11014,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11020,'b2897e1277f56641193a6db73825f707eed3e4c9.json',15),(11022,'b2897e1277f56641193a6db73825f707eed3e4c9.json',74),(11029,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11032,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11035,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11036,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(11037,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11040,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11054,'b2897e1277f56641193a6db73825f707eed3e4c9.json',151),(11066,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11095,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11100,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11113,'b2897e1277f56641193a6db73825f707eed3e4c9.json',7),(11122,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11126,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11130,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11156,'b2897e1277f56641193a6db73825f707eed3e4c9.json',20),(11160,'b2897e1277f56641193a6db73825f707eed3e4c9.json',13),(11171,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11176,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11200,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11204,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11213,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11214,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11215,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11216,'b2897e1277f56641193a6db73825f707eed3e4c9.json',20),(11221,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11222,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11223,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11225,'b2897e1277f56641193a6db73825f707eed3e4c9.json',29),(11230,'b2897e1277f56641193a6db73825f707eed3e4c9.json',9),(11233,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(11241,'b2897e1277f56641193a6db73825f707eed3e4c9.json',12),(11245,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11251,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(11253,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11260,'b2897e1277f56641193a6db73825f707eed3e4c9.json',7),(11263,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11267,'b2897e1277f56641193a6db73825f707eed3e4c9.json',11),(11268,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11271,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(11276,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11278,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11279,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(11283,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11293,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11304,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11310,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11314,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11316,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11332,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(11335,'b2897e1277f56641193a6db73825f707eed3e4c9.json',24),(11341,'b2897e1277f56641193a6db73825f707eed3e4c9.json',15),(11352,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11357,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(11358,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11365,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11369,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11373,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(11375,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11382,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11386,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11393,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11404,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11414,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11419,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11420,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11427,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(11436,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11437,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(11438,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11442,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11451,'b2897e1277f56641193a6db73825f707eed3e4c9.json',120),(11454,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11468,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11472,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11478,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11479,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11483,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11486,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11497,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(11502,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11514,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11535,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11540,'b2897e1277f56641193a6db73825f707eed3e4c9.json',17),(11546,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11559,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11566,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11571,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11575,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11582,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11617,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11618,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11638,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11639,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11658,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11659,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11670,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11685,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11686,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11687,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11688,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11704,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11714,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11730,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11731,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11733,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11750,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11812,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(11820,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11824,'b2897e1277f56641193a6db73825f707eed3e4c9.json',17),(11831,'b2897e1277f56641193a6db73825f707eed3e4c9.json',10),(11836,'b2897e1277f56641193a6db73825f707eed3e4c9.json',7),(11844,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11846,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11847,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11874,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(11885,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11898,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(11900,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(11958,'b2897e1277f56641193a6db73825f707eed3e4c9.json',8),(11975,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(11982,'b2897e1277f56641193a6db73825f707eed3e4c9.json',7),(11999,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12007,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12010,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(12011,'b2897e1277f56641193a6db73825f707eed3e4c9.json',20),(12035,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12061,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(12068,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12086,'b2897e1277f56641193a6db73825f707eed3e4c9.json',11),(12105,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12110,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12116,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12135,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(12151,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12162,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12163,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12169,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(12195,'b2897e1277f56641193a6db73825f707eed3e4c9.json',29),(12200,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12240,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12246,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12247,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12283,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12350,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12351,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12381,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12382,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(12433,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2498),(12434,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12453,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12461,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12468,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12486,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12508,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12524,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12562,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12567,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12570,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12578,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12584,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12585,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12589,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12599,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12602,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12603,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12615,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12618,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12619,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12647,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12676,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12708,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(12736,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12747,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12750,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12766,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12769,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(12791,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12823,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12827,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(12849,'b2897e1277f56641193a6db73825f707eed3e4c9.json',13),(12852,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12853,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(12878,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12881,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(12948,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13026,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13027,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13030,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13031,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13063,'b2897e1277f56641193a6db73825f707eed3e4c9.json',10),(13064,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(13065,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13096,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13098,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(13106,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13113,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13117,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13119,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13123,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13124,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13125,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13133,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13136,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13139,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(13150,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13151,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13152,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(13154,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13155,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13157,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13185,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13192,'b2897e1277f56641193a6db73825f707eed3e4c9.json',6),(13194,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13196,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13202,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13203,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13217,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13218,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13223,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13248,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13285,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13286,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(13297,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13301,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13320,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13330,'b2897e1277f56641193a6db73825f707eed3e4c9.json',169),(13352,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13371,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13378,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13379,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13400,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13418,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(13419,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(13420,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13428,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13435,'b2897e1277f56641193a6db73825f707eed3e4c9.json',4),(13446,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13447,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13450,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13451,'b2897e1277f56641193a6db73825f707eed3e4c9.json',25),(13460,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13468,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13469,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13470,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13471,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13472,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13473,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13479,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(13480,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13481,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13482,'b2897e1277f56641193a6db73825f707eed3e4c9.json',9),(13486,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13496,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(13521,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13527,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13553,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13560,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13581,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13600,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13612,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13613,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13631,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13632,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13647,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(13654,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13662,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13693,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13697,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13701,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13704,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13707,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13708,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13710,'b2897e1277f56641193a6db73825f707eed3e4c9.json',3),(13714,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13716,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13766,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13771,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13787,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13835,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13839,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13910,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13914,'b2897e1277f56641193a6db73825f707eed3e4c9.json',7),(13953,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13985,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(13989,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(13992,'b2897e1277f56641193a6db73825f707eed3e4c9.json',9),(14009,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14016,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14033,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14038,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14039,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14041,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14042,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14048,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14051,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14059,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14065,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14075,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14076,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14090,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14120,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14121,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14122,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14155,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14161,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14164,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14170,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14173,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14174,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14176,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14180,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14181,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14182,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14185,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14186,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14188,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14190,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14191,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14202,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14208,'b2897e1277f56641193a6db73825f707eed3e4c9.json',5),(14213,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14216,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14225,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14226,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14231,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14233,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14287,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14288,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14290,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14291,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14292,'b2897e1277f56641193a6db73825f707eed3e4c9.json',2),(14296,'b2897e1277f56641193a6db73825f707eed3e4c9.json',1),(14299,'3bb07ea10432f7738413dff9816809cc90f03f99.json',11),(14302,'3bb07ea10432f7738413dff9816809cc90f03f99.json',19),(14303,'3bb07ea10432f7738413dff9816809cc90f03f99.json',24),(14305,'3bb07ea10432f7738413dff9816809cc90f03f99.json',25),(14306,'3bb07ea10432f7738413dff9816809cc90f03f99.json',60),(14317,'3bb07ea10432f7738413dff9816809cc90f03f99.json',8),(14322,'3bb07ea10432f7738413dff9816809cc90f03f99.json',6),(14326,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14333,'3bb07ea10432f7738413dff9816809cc90f03f99.json',13),(14334,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14337,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14338,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14342,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14345,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14358,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14366,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14382,'3bb07ea10432f7738413dff9816809cc90f03f99.json',5),(14384,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14386,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14391,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14393,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14394,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14396,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14399,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14400,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14401,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14403,'3bb07ea10432f7738413dff9816809cc90f03f99.json',19),(14411,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14413,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14423,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(14425,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14426,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14428,'3bb07ea10432f7738413dff9816809cc90f03f99.json',5),(14435,'3bb07ea10432f7738413dff9816809cc90f03f99.json',15),(14436,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14437,'3bb07ea10432f7738413dff9816809cc90f03f99.json',15),(14438,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14439,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14472,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14475,'3bb07ea10432f7738413dff9816809cc90f03f99.json',10),(14476,'3bb07ea10432f7738413dff9816809cc90f03f99.json',19),(14481,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14490,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14491,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14495,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14501,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14505,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14506,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14512,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14517,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14519,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14527,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14528,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14531,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14533,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14537,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14548,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14549,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14554,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14564,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14565,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14575,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14582,'3bb07ea10432f7738413dff9816809cc90f03f99.json',8),(14584,'3bb07ea10432f7738413dff9816809cc90f03f99.json',6),(14585,'3bb07ea10432f7738413dff9816809cc90f03f99.json',7),(14592,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14602,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14605,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14615,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14621,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14624,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14626,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14629,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14633,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14640,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14642,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14644,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14666,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14676,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14697,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14709,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14710,'3bb07ea10432f7738413dff9816809cc90f03f99.json',6),(14714,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14716,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14719,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14721,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14728,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14732,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14733,'3bb07ea10432f7738413dff9816809cc90f03f99.json',9),(14734,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14748,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14755,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14757,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(14776,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14790,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14791,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14803,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14806,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14819,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(14821,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14837,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14848,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14859,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14860,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(14861,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(14883,'3bb07ea10432f7738413dff9816809cc90f03f99.json',24),(14887,'3bb07ea10432f7738413dff9816809cc90f03f99.json',22),(14919,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14920,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14922,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14931,'3bb07ea10432f7738413dff9816809cc90f03f99.json',5),(14935,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14937,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14939,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(14942,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14949,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14960,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14975,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14981,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14984,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14992,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(14994,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15006,'3bb07ea10432f7738413dff9816809cc90f03f99.json',6),(15007,'3bb07ea10432f7738413dff9816809cc90f03f99.json',5),(15011,'3bb07ea10432f7738413dff9816809cc90f03f99.json',9),(15013,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15017,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15037,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15041,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15044,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15047,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15077,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15078,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15079,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(15086,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15092,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15095,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15128,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15133,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15136,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15137,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15150,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(15160,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15163,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15169,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15171,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15178,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15190,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(15191,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15207,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15208,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15236,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15241,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15245,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15249,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15250,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15253,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15255,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15260,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15267,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15279,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15293,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15299,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15312,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15313,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15314,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15315,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15317,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15328,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(15337,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15338,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15346,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15360,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15368,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15399,'3bb07ea10432f7738413dff9816809cc90f03f99.json',4),(15430,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15434,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15437,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15441,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15458,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15459,'3bb07ea10432f7738413dff9816809cc90f03f99.json',5),(15461,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15464,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15467,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15470,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15472,'3bb07ea10432f7738413dff9816809cc90f03f99.json',8),(15484,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15486,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(15488,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15491,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15498,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15506,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15507,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15511,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15519,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15542,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15543,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15544,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15554,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15571,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15590,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15594,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15628,'3bb07ea10432f7738413dff9816809cc90f03f99.json',3),(15652,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15664,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15671,'3bb07ea10432f7738413dff9816809cc90f03f99.json',5),(15675,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15682,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15683,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15685,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15688,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15714,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15729,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15760,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15805,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15810,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15815,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15829,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15850,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15861,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15967,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15974,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(15992,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(16016,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16017,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16021,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(16031,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16048,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16064,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16088,'3bb07ea10432f7738413dff9816809cc90f03f99.json',12),(16089,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16093,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(16112,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16118,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16129,'3bb07ea10432f7738413dff9816809cc90f03f99.json',2),(16180,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16186,'3bb07ea10432f7738413dff9816809cc90f03f99.json',1),(16188,'5806726a24dc91de3954001effbdffd7a82d54e2.json',4),(16189,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16195,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16196,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16200,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16202,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16206,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16208,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16213,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16218,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16219,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16221,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16222,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(16226,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16234,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(16237,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16238,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16239,'5806726a24dc91de3954001effbdffd7a82d54e2.json',22),(16240,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(16247,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16251,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16260,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16261,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16262,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16274,'5806726a24dc91de3954001effbdffd7a82d54e2.json',5),(16278,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16282,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(16288,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16295,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16317,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16318,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16327,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16329,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16332,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16337,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16347,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16348,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16349,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16370,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16371,'5806726a24dc91de3954001effbdffd7a82d54e2.json',4),(16372,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16383,'5806726a24dc91de3954001effbdffd7a82d54e2.json',5),(16385,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16386,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16387,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16395,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16399,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16405,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16407,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16415,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16416,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16417,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16421,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16423,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16433,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16439,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16441,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16446,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16448,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16461,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16462,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16463,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16464,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16484,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16485,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16487,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16493,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16494,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16497,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16498,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16501,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16517,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16524,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16525,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16530,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16532,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16535,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16538,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16539,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16546,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16547,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16568,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16569,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16570,'5806726a24dc91de3954001effbdffd7a82d54e2.json',9),(16576,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16579,'5806726a24dc91de3954001effbdffd7a82d54e2.json',6),(16581,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16595,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16600,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16607,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16614,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16620,'5806726a24dc91de3954001effbdffd7a82d54e2.json',8),(16622,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16628,'5806726a24dc91de3954001effbdffd7a82d54e2.json',4),(16653,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16657,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(16658,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16701,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16723,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(16727,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16728,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16731,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16735,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16748,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16749,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16751,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16754,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16764,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16770,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16776,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16779,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16825,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16827,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16831,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16834,'5806726a24dc91de3954001effbdffd7a82d54e2.json',6),(16837,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16840,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16842,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16846,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16847,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16876,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16896,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16897,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16898,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16904,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16905,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16907,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16911,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16912,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16915,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16922,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16923,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16925,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16926,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16927,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16929,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16941,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16949,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16955,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16956,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16965,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16966,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(16968,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16972,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16974,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(16986,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17003,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17010,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17011,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17020,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17033,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17042,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17046,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17057,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17060,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17066,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17068,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17131,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17135,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17136,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17145,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17149,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17150,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17151,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17155,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17158,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17159,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17163,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17166,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17168,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(17177,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(17182,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17201,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17205,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17209,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17210,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17224,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17227,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17230,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(17235,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17237,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17239,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17241,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17242,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17247,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17248,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17252,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17253,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17254,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17257,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17275,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17288,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17291,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17293,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17295,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17299,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17302,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(17309,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17313,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17319,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17320,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17324,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17336,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17337,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17339,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17342,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17347,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17348,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17349,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17350,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17352,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17354,'5806726a24dc91de3954001effbdffd7a82d54e2.json',3),(17356,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17357,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17358,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17370,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17371,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17376,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17380,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17383,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17384,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17385,'5806726a24dc91de3954001effbdffd7a82d54e2.json',4),(17399,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17401,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17405,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17411,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17416,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17426,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17427,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17428,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17441,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17442,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17443,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17446,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17453,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17454,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17456,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17457,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17462,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17467,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17471,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17472,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17473,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17474,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17477,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17478,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17479,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17484,'5806726a24dc91de3954001effbdffd7a82d54e2.json',8),(17485,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17487,'5806726a24dc91de3954001effbdffd7a82d54e2.json',36),(17489,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17490,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17494,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17496,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17501,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17504,'5806726a24dc91de3954001effbdffd7a82d54e2.json',2),(17506,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17507,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17508,'5806726a24dc91de3954001effbdffd7a82d54e2.json',4),(17512,'5806726a24dc91de3954001effbdffd7a82d54e2.json',1),(17517,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',5),(17518,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(17523,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17533,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17539,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17543,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17565,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17583,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17584,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(17588,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',7),(17599,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',7),(17607,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',9),(17619,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17622,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17623,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17625,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17627,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(17637,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',8),(17639,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17659,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17660,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(17694,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17696,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17697,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',6),(17699,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17703,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17704,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(17706,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17707,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17709,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17733,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17748,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17757,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(17762,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17786,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17796,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17797,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(17812,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17831,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17842,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17843,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17850,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17860,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17861,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17862,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17863,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17864,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17867,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17868,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17905,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17907,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17910,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17917,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17929,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17933,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17934,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17936,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17939,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(17943,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17947,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',9),(17948,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17949,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(17975,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17977,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(17993,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(17994,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',7),(17997,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',20),(18000,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18001,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18022,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(18031,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18050,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18061,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18068,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18070,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18073,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18074,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18115,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18118,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18119,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',5),(18121,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18125,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18145,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18155,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18166,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18177,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18187,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18205,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18212,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18232,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18273,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',67),(18274,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18275,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18280,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18301,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18314,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(18355,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18365,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18389,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18399,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18434,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18435,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',7),(18440,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18466,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18469,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18472,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18478,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18479,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',5),(18487,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18517,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18539,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18540,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18541,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(18545,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18549,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18555,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',5),(18559,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18569,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18579,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18585,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18628,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18656,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18665,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18666,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18673,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18677,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18684,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18690,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(18717,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18740,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18762,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',5),(18788,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18823,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18866,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18867,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18870,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18873,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18880,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18888,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18921,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18925,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18938,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18944,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18950,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18956,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(18969,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(18978,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(18998,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19040,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19051,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19053,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19054,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19055,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(19061,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19109,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19120,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19131,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19133,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19142,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19148,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19151,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19157,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',7),(19160,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19165,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19174,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19179,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19189,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19203,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19247,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19251,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19252,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19270,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19314,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(19318,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19332,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19344,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19420,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19424,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19434,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(19448,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19463,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19469,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(19470,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19494,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19499,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',8),(19506,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19533,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19536,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19538,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19543,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19552,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19576,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19577,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(19581,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19618,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19643,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19668,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19672,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19674,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19676,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19711,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19722,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19725,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(19727,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',6),(19742,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19760,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19793,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19794,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19814,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19821,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19829,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19837,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19844,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19880,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19882,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',5),(19883,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(19922,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19944,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(19946,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(19947,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(19972,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(20010,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20012,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20013,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20073,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20074,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20076,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20087,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20104,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20105,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20108,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20117,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',4),(20140,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20159,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20161,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20183,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20190,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20192,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20196,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',3),(20233,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20297,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20301,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20304,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20309,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',43),(20310,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',23),(20314,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20318,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20335,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20340,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20348,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20356,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20370,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20412,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20419,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20420,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20449,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20475,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20477,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20497,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20500,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20532,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20551,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20584,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20585,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',1),(20587,'faaf1022ccfe93b032c5608097a53543ba24aedb.json',2),(20600,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20601,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',28),(20609,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(20614,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20615,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(20618,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20624,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20633,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(20636,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',10),(20638,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20646,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20648,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20655,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20657,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20660,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20667,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20671,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20672,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20675,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20687,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20692,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20694,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20701,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20703,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20705,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',9),(20725,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20729,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20731,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(20733,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20739,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20740,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20741,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20742,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20746,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20762,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',10),(20793,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20801,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20809,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20810,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',7),(20824,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20825,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20826,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20829,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20830,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20836,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20850,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(20852,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20860,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',11),(20864,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20865,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20866,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20872,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20873,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(20875,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20879,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20880,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20887,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20891,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',55),(20898,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(20902,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(20904,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',10),(20905,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(20906,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',11),(20910,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',15),(20911,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',15),(20913,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20917,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20921,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20922,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20925,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20945,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20946,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20961,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(20962,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(20968,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',13),(20973,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(20974,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(20975,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(20976,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(20979,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(20987,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20991,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(20997,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(21003,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21005,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21009,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',9),(21010,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21011,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21014,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21015,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21016,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21022,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21034,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21038,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21039,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21045,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21058,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21061,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21062,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21063,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21065,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21073,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21074,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21075,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21078,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21085,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21100,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21102,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21103,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21111,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21128,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21130,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21135,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',7),(21137,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21141,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21147,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21150,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21153,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21159,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21165,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21171,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21184,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21185,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',9),(21194,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21200,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21204,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21205,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21209,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',7),(21225,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21228,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21232,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21234,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21248,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21260,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21267,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',19),(21276,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',10),(21279,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21290,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21291,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21292,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21304,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21310,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21315,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21316,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21321,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21328,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21329,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21330,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21331,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21337,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21342,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(21346,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',19),(21347,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',11),(21359,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21366,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21379,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21381,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21384,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21394,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21395,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21396,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',9),(21406,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21415,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',18),(21417,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21418,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21427,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21429,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21432,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21447,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21452,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21457,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21458,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21459,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21460,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21461,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21462,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21468,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(21470,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21473,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21480,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',11),(21484,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21489,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21491,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21500,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21520,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21527,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21528,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21529,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21531,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21533,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21538,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21541,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21561,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21567,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(21575,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21581,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(21600,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21601,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21602,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21604,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21607,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21611,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21618,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21623,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21626,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21628,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21640,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21649,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21650,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21655,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',10),(21665,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21673,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21704,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21705,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',6),(21707,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21708,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21711,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21712,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21716,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21727,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21730,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21756,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21761,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21763,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',6),(21765,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21766,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21767,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21768,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21769,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21773,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21810,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21812,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21843,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(21846,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21856,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21857,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21862,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21864,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21865,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21876,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21878,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(21887,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21890,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21891,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21902,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21915,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21920,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21925,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21928,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21933,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21940,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21941,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21943,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21946,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21951,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',6),(21952,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(21955,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21956,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21960,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21965,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(21966,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21974,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21975,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(21978,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',7),(21982,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(21993,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22009,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',8),(22012,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22014,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',6),(22018,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(22021,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22023,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22029,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22035,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22036,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22046,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22047,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22048,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22049,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22050,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22055,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22057,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22059,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22061,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22064,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22066,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22082,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(22084,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22085,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22087,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22094,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22095,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22102,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22103,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22110,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22111,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22113,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22114,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22118,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22121,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22126,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22127,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',6),(22136,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22138,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22139,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22140,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22142,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22162,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22164,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22167,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22170,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22190,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22191,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22206,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22210,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22212,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(22226,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22227,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22231,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22246,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22247,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22258,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22274,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22277,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22281,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22282,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',6),(22284,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22288,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22300,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22304,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22337,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22349,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(22350,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22362,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22368,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22374,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22382,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22385,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22387,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(22401,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',10),(22404,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',24),(22413,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22416,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',9),(22418,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22419,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22422,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22481,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22498,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22499,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22501,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22518,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22524,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22534,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22545,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22552,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22553,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22561,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22568,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22582,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22584,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22591,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22593,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22606,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22626,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22632,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22638,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',5),(22639,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22649,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22654,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22658,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22669,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',9),(22674,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22675,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22678,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22679,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22680,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22681,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22687,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22689,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22692,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22710,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22712,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',14),(22734,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(22767,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(22795,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22803,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(22817,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',17),(22822,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22823,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22828,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22839,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22871,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22875,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22879,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22881,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22908,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22909,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22910,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22912,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22913,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22914,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22916,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22919,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22922,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(22940,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(22945,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22953,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22957,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22959,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22966,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22971,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22972,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(22992,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23006,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23021,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23027,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23028,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23035,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23042,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23046,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(23052,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23064,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23068,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23070,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23071,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23072,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23077,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23079,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23115,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23122,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',15),(23142,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',4),(23155,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23157,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(23159,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',15),(23163,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23179,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23216,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23217,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23220,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23232,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23252,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23263,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23294,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23295,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23307,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',3),(23344,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23348,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',2),(23361,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23384,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23441,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23443,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23446,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23458,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23471,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23474,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23478,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23487,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23497,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23500,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23524,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23531,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23546,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23547,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23568,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23572,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23573,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23575,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23576,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23578,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23579,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23580,'5b44feca5d6ffaaeb66501fa84cc6dd44d06660a.json',1),(23583,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(23584,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',57),(23587,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23589,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',41),(23591,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',22),(23595,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23597,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',125),(23614,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23617,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23619,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',13),(23629,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23634,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',4),(23639,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(23640,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23641,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',26),(23642,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',4),(23645,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23647,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23652,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23653,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23664,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23670,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',16),(23674,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',11),(23675,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(23688,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23689,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23692,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23695,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23701,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23704,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23705,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23710,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',18),(23718,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23725,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',4),(23726,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23729,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23745,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',16),(23748,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23756,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23775,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23779,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23780,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',152),(23788,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23797,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23802,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23803,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23806,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(23807,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23808,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23809,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23811,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23812,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23813,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23816,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(23826,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',4),(23842,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(23849,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23854,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23858,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23871,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23878,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23889,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23892,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(23895,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23903,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23906,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23913,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23918,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23930,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23932,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23940,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23945,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23949,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',7),(23959,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23983,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23986,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(23996,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(23997,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24000,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24002,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24006,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24007,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24010,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24026,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24031,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24037,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24038,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24042,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24044,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24052,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',10),(24053,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',18),(24054,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24057,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24058,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24087,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',8),(24104,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',14),(24119,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24132,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24136,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',109),(24137,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(24141,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24151,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24162,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24165,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24174,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',26),(24178,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24179,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24186,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24187,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(24188,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24193,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',8),(24194,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24201,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(24202,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(24204,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(24229,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(24232,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24271,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',8),(24272,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24312,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',47),(24315,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24324,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24332,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(24338,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',4),(24341,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24359,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24361,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24362,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24380,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24391,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24392,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(24408,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24410,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24411,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24412,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24421,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24445,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',61),(24487,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24508,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24511,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',14),(24513,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24526,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',4),(24534,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24549,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',7),(24563,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24574,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(24605,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24613,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24618,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24629,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24633,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24654,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24655,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24672,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24674,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24677,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24681,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24683,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24685,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(24687,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24713,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24717,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24718,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',64),(24719,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24723,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24748,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24760,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24821,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(24822,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24829,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24882,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(24883,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(24884,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(24899,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',24),(24912,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(24936,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(25004,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25016,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25017,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25066,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25122,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',9),(25129,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25141,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25159,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25187,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25217,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25218,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25221,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(25222,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25231,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25240,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',4),(25241,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25246,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25270,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25330,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25343,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25381,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25416,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25419,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25421,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(25441,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25442,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25480,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25481,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',9),(25484,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25485,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25486,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25487,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25493,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25498,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25499,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25504,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25505,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(25506,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',8),(25534,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25543,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',6),(25560,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25566,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25569,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25579,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25589,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25617,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25618,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(25619,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(25627,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25672,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25683,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25713,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25714,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25764,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25785,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25804,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25818,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25820,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25834,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25854,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25855,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25866,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25874,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25880,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25888,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',15),(25896,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25907,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25911,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25917,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',36),(25919,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25920,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25922,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25923,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(25926,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25927,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25928,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25929,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25956,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25963,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25968,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25976,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(25989,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(25995,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26009,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26058,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26092,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26112,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',20),(26142,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',7),(26148,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(26184,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26185,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26196,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26235,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26241,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26275,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26293,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26306,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26339,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26361,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26370,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26398,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26418,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26426,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26433,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26436,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26463,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',7),(26464,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(26477,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26485,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26500,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26503,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26509,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26515,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26518,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26528,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26535,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26545,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26547,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26559,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26560,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26575,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26581,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26582,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26584,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26585,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26593,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26596,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26597,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26599,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26636,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26637,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26648,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26649,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26678,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26679,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26687,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26711,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',7),(26723,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26725,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26726,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(26727,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26748,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26757,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26762,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',59),(26767,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26768,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26775,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',70),(26777,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26780,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26784,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26786,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26787,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26789,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(26792,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',3),(26797,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26800,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26802,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26805,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26810,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26829,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26831,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26841,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26851,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26855,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26856,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26857,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26858,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26864,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26869,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26871,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26874,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',5),(26876,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26878,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26885,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26895,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26915,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26916,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26930,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26936,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26937,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26938,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26940,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26945,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',2),(26964,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26980,'9d4e3e8eb092d5ed282d0aa4aadcaa8b7165b5e9.json',1),(26987,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',158),(26995,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(26996,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(27004,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',5),(27009,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',5),(27015,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(27017,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',38),(27018,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(27019,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(27020,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(27033,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',6),(27037,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27040,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27042,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27046,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27048,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27052,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27058,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27061,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27077,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(27082,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',136),(27090,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27095,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27097,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27103,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27111,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',5),(27115,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27121,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27125,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27127,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27172,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27174,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27181,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27187,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27204,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27205,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',8),(27213,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27215,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27236,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27249,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27270,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27272,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',12),(27301,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27316,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27352,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27353,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27356,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27360,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27361,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27364,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27365,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27370,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27371,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27372,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27376,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27379,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27380,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27381,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27426,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27430,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27445,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27461,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',92),(27464,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(27470,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(27518,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',14),(27547,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',9),(27571,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27602,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',17),(27604,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27611,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(27612,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',6),(27745,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(27754,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27815,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27816,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',12),(27881,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(27930,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(27997,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28024,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28078,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28084,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',5),(28085,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(28103,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28107,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(28130,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28140,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28143,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28144,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28155,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28162,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28262,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28305,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28323,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28324,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28325,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',9),(28326,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28339,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28342,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28343,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28353,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(28355,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28358,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28469,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(28475,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',8),(28478,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(28500,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28519,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28624,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28645,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28649,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',6),(28694,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28701,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28710,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(28712,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28720,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28762,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(28782,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28783,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(28791,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(28795,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(28796,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',5),(28810,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(28819,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(28859,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(28901,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29107,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29147,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',3),(29190,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29194,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29216,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29219,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29255,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29256,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',24),(29259,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29271,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29302,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29374,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29405,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29435,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29446,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29449,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29450,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29466,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29497,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29500,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29503,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29534,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29542,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29549,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29567,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29657,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29661,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29664,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29668,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29672,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29675,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29678,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29683,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29685,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29693,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29694,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29716,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29717,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29743,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29756,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29760,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29761,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29762,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29772,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29773,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29774,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29775,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29781,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29792,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29793,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',6),(29795,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29797,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29803,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29804,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29805,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29806,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29819,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29822,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29823,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29825,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29830,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29831,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29832,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29841,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29857,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29865,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29867,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29869,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29870,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29871,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29872,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29873,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29876,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29877,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29879,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29888,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29910,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29933,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29935,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29938,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29947,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29956,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29959,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29961,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29970,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29977,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29978,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(29980,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(29982,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30005,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30007,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30019,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(30022,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(30031,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(30093,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(30114,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30134,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30135,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30139,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30141,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30145,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',5),(30146,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30148,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30149,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30172,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30175,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30183,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',65),(30201,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30202,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30205,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30214,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',2),(30223,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30231,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30232,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30256,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30257,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30301,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',4),(30302,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30303,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30308,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30310,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30313,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30320,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30325,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30326,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30358,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30376,'14e0cac6e86d62859e6c9f1351ab67466d89e5b3.json',1),(30394,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30397,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30400,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30418,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30423,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',81),(30432,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',53),(30437,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30439,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',13),(30444,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30451,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30452,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',4),(30457,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30467,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',4),(30474,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30480,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30486,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30487,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30488,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30494,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30497,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30523,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30524,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',17),(30527,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',9),(30547,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',5),(30555,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30558,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30566,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30567,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30568,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30569,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30585,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30586,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30617,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30626,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30642,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30646,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30649,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30650,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30657,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30660,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30672,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30678,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30682,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30696,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30716,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',4),(30717,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',12),(30728,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30729,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30747,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',6),(30748,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',8),(30785,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',13),(30812,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30813,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30814,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30844,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30849,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',6),(30853,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30864,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',9),(30874,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30875,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30879,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30880,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30886,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30894,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',104),(30895,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30896,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(30921,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30934,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30937,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',4),(30938,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',4),(30942,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',6),(30987,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30990,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(30996,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(30998,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31001,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31003,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31008,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31012,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31020,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',4),(31032,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',8),(31041,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31043,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',5),(31044,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31047,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31050,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31052,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31058,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31067,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31068,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',7),(31072,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31076,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31093,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31133,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31207,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31215,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31433,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31530,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',12),(31533,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31537,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31540,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31544,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31546,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31564,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31567,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',4),(31570,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31572,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',9),(31573,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',9),(31593,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31609,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31620,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31639,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',32),(31640,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31641,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31656,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31657,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31661,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31663,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31668,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31671,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(31706,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31711,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31712,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31724,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31727,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31763,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31769,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31777,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31779,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31786,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31790,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31802,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(31809,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31811,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31820,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31825,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31832,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31846,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31850,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31868,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31903,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31924,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31931,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(31970,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31971,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',12),(31972,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(31973,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(32044,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(32152,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(32225,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',2),(32230,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',3),(32232,'d09b79026117ec9faebba46a8d13aa9b23ec751e.json',1),(32317,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32319,'44102e3e69e70ad2a73e753133283334ce1f8736.json',6),(32322,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32323,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32324,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32334,'44102e3e69e70ad2a73e753133283334ce1f8736.json',50),(32356,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32360,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32361,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32364,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32369,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(32381,'44102e3e69e70ad2a73e753133283334ce1f8736.json',19),(32385,'44102e3e69e70ad2a73e753133283334ce1f8736.json',18),(32394,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32398,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32407,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(32408,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(32409,'44102e3e69e70ad2a73e753133283334ce1f8736.json',6),(32420,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32430,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32435,'44102e3e69e70ad2a73e753133283334ce1f8736.json',13),(32436,'44102e3e69e70ad2a73e753133283334ce1f8736.json',7),(32437,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32440,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(32447,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32450,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32452,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32454,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32455,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32456,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32457,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32460,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32463,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32477,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(32478,'44102e3e69e70ad2a73e753133283334ce1f8736.json',6),(32481,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(32490,'44102e3e69e70ad2a73e753133283334ce1f8736.json',14),(32493,'44102e3e69e70ad2a73e753133283334ce1f8736.json',12),(32494,'44102e3e69e70ad2a73e753133283334ce1f8736.json',9),(32512,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32513,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32521,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32537,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32540,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32559,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32566,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32569,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32580,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(32601,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32603,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(32605,'44102e3e69e70ad2a73e753133283334ce1f8736.json',156),(32633,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32662,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(32673,'44102e3e69e70ad2a73e753133283334ce1f8736.json',24),(32713,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(32714,'44102e3e69e70ad2a73e753133283334ce1f8736.json',17),(32719,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(32729,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32732,'44102e3e69e70ad2a73e753133283334ce1f8736.json',7),(32742,'44102e3e69e70ad2a73e753133283334ce1f8736.json',17),(32743,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(32764,'44102e3e69e70ad2a73e753133283334ce1f8736.json',7),(32775,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(32779,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(32802,'44102e3e69e70ad2a73e753133283334ce1f8736.json',7),(32818,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32834,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32839,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32852,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(32859,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32903,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32913,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(32940,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32945,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(32964,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(32969,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33001,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33018,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33033,'44102e3e69e70ad2a73e753133283334ce1f8736.json',106),(33035,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33051,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(33059,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33122,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33126,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33151,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33152,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33156,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33161,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33177,'44102e3e69e70ad2a73e753133283334ce1f8736.json',6),(33253,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(33277,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33303,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33309,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33312,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33325,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(33350,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(33383,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(33397,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33398,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33443,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33456,'44102e3e69e70ad2a73e753133283334ce1f8736.json',9),(33466,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(33467,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(33468,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33490,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33494,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33538,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33543,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33563,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(33567,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(33579,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33597,'44102e3e69e70ad2a73e753133283334ce1f8736.json',6),(33601,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33611,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33612,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33724,'44102e3e69e70ad2a73e753133283334ce1f8736.json',11),(33728,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33729,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33734,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33748,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33749,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33751,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(33752,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33754,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33755,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33756,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33757,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(33760,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33778,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33813,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(33815,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33839,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33842,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33859,'44102e3e69e70ad2a73e753133283334ce1f8736.json',9),(33870,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33908,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33931,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33945,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(33947,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(33987,'44102e3e69e70ad2a73e753133283334ce1f8736.json',5),(34022,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34025,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34048,'44102e3e69e70ad2a73e753133283334ce1f8736.json',92),(34050,'44102e3e69e70ad2a73e753133283334ce1f8736.json',10),(34070,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34087,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34094,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(34096,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34105,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(34114,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(34128,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34147,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34149,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34150,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34166,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(34178,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34185,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(34192,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34213,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34222,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34231,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34238,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34285,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34294,'44102e3e69e70ad2a73e753133283334ce1f8736.json',18),(34304,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34344,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34347,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34354,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34356,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34404,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34446,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34461,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34474,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34493,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34496,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(34528,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34541,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34542,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34545,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34546,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34549,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34554,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34565,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34566,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34570,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34571,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34573,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34574,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34585,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34588,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34716,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34720,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34721,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34723,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34731,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34732,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34753,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34754,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34755,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34756,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(34792,'44102e3e69e70ad2a73e753133283334ce1f8736.json',3),(34793,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34798,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34807,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(34808,'44102e3e69e70ad2a73e753133283334ce1f8736.json',15),(34809,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34810,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34823,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34824,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34827,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34839,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34858,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34868,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34876,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34877,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34879,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34882,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34884,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34889,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34891,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34892,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34893,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34945,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(34964,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(34999,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(35010,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35015,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35051,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35064,'44102e3e69e70ad2a73e753133283334ce1f8736.json',6),(35081,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35085,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35089,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35095,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35105,'44102e3e69e70ad2a73e753133283334ce1f8736.json',4),(35128,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(35150,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35158,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35162,'44102e3e69e70ad2a73e753133283334ce1f8736.json',6),(35171,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35227,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35237,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35241,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35242,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35246,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(35247,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35250,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35252,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35253,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35262,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35267,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(35284,'44102e3e69e70ad2a73e753133283334ce1f8736.json',2),(35288,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35306,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35311,'44102e3e69e70ad2a73e753133283334ce1f8736.json',1),(35346,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35354,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35381,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35403,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35414,'30a4842a2e257f725cc041e32fd682c495111a50.json',4),(35416,'30a4842a2e257f725cc041e32fd682c495111a50.json',16),(35440,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35451,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35454,'30a4842a2e257f725cc041e32fd682c495111a50.json',2),(35455,'30a4842a2e257f725cc041e32fd682c495111a50.json',5),(35474,'30a4842a2e257f725cc041e32fd682c495111a50.json',2),(35479,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35492,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35509,'30a4842a2e257f725cc041e32fd682c495111a50.json',2),(35520,'30a4842a2e257f725cc041e32fd682c495111a50.json',7),(35531,'30a4842a2e257f725cc041e32fd682c495111a50.json',10),(35532,'30a4842a2e257f725cc041e32fd682c495111a50.json',27),(35549,'30a4842a2e257f725cc041e32fd682c495111a50.json',9),(35560,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35569,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35572,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35589,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35593,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35595,'30a4842a2e257f725cc041e32fd682c495111a50.json',2),(35596,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35597,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35600,'30a4842a2e257f725cc041e32fd682c495111a50.json',6),(35638,'30a4842a2e257f725cc041e32fd682c495111a50.json',2),(35649,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35650,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35658,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35659,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35673,'30a4842a2e257f725cc041e32fd682c495111a50.json',7),(35691,'30a4842a2e257f725cc041e32fd682c495111a50.json',6),(35702,'30a4842a2e257f725cc041e32fd682c495111a50.json',5),(35727,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35739,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35746,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35755,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35826,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35836,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35843,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35844,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35850,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35853,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35856,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35857,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35891,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35892,'30a4842a2e257f725cc041e32fd682c495111a50.json',11),(35905,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35906,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35984,'30a4842a2e257f725cc041e32fd682c495111a50.json',2),(35985,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35993,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(35997,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(36001,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(36013,'30a4842a2e257f725cc041e32fd682c495111a50.json',1),(36039,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36043,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36063,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36079,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',4),(36085,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36097,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36111,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36116,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36119,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36120,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',4),(36128,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',4),(36132,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',6),(36163,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36164,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36167,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36169,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',24),(36177,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36181,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',6),(36182,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36187,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',4),(36204,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',6),(36209,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',3),(36215,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36247,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36269,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36291,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',4),(36303,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36304,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36305,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36309,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36312,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36313,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36314,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36316,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36323,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36344,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36345,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36346,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36361,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36376,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36379,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36385,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36386,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36394,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36396,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36415,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36419,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36420,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36421,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36434,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36435,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36436,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36441,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36446,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36447,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36448,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36454,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',3),(36461,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36463,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',7),(36464,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36473,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36482,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36485,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36486,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36487,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36491,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36492,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36493,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36496,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36498,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36499,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36500,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36504,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36508,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36511,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36516,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36519,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36527,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36530,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36531,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36553,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36554,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36566,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36568,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36573,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36592,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36650,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36661,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36686,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36692,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36710,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36726,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36730,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36749,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36763,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36765,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36772,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36797,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36802,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36825,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36841,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36847,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36848,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36870,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36911,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36915,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36922,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36925,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36927,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36935,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(36942,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36968,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(36994,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37002,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37029,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37034,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37036,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',2),(37041,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37058,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37070,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37102,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37122,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37140,'6a8ac55ea2a1fbd99deb683e24dd986e55e707b3.json',1),(37145,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37173,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',16),(37174,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37175,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37184,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37203,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',185),(37211,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',32),(37225,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37226,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',9),(37227,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',5),(37235,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37243,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37250,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37255,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37265,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37269,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37273,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37309,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',21),(37336,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',8),(37356,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37369,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',6),(37386,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',237),(37387,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',33),(37394,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37397,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(37402,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37403,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',5),(37419,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',6),(37428,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37441,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37451,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37465,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37468,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',8),(37478,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',48),(37494,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37509,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37556,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37581,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',17),(37621,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37623,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(37645,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37646,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37650,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37657,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',10),(37662,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37674,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37675,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37679,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(37698,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37701,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37705,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',51),(37714,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',11),(37720,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(37726,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37756,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(37761,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37788,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',10),(37797,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37809,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(37843,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37865,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37886,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37921,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37929,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37952,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37983,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(37990,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',11),(38009,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38015,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',50),(38017,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38028,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38037,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38041,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38047,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38048,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(38054,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(38058,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38065,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38116,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(38124,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38126,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(38130,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38161,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38162,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38168,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',22),(38171,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38174,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38177,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38193,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',45),(38248,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38258,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38272,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(38274,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',65),(38309,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38311,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38318,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(38325,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38328,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38331,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38374,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',15),(38393,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38396,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38403,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38418,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38437,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38537,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38572,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38580,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(38638,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(38639,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(38672,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38677,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38694,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(38711,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(38715,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',6),(38738,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38746,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',12),(38751,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38772,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38777,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38784,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',6),(38787,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38817,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38819,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38820,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38831,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(38847,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38889,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(38990,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(38993,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',16),(39019,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',7),(39076,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39094,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',5),(39113,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39123,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39151,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39219,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39284,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39318,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39330,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(39355,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39402,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(39415,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(39421,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',14),(39486,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(39514,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',8),(39524,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39565,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39616,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39619,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(39639,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',8),(39699,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(39715,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(39779,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',6),(39811,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(39965,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40005,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(40006,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(40217,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40262,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40281,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40381,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40535,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',6),(40636,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40650,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40717,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(40768,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40769,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(40804,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(40843,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40866,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(40938,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(41163,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(41254,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41283,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41304,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(41309,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41330,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41505,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',278),(41544,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41545,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41599,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41602,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(41748,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41751,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',3),(41753,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41829,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41882,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(41914,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',10),(41999,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42018,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42034,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42035,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(42083,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42122,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',18),(42137,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42172,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42197,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42204,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42246,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42283,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',53),(42286,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42299,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(42330,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(42399,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42461,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(42542,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42568,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42601,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(42606,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42620,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42624,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42636,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42639,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42642,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42644,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42645,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',51),(42708,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42739,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42745,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(42748,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42777,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42801,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42803,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',1),(42808,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',2),(42814,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',5),(42815,'367af6bb9a8bbda02206506a819656bb6f14ce4c.json',4),(42836,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42852,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42853,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(42857,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42870,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42874,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',4),(42879,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42883,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42901,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42914,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42917,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42921,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(42924,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(42926,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42930,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42931,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',12),(42935,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(42937,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',10),(42966,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(42983,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(42999,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43009,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43012,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43013,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',4),(43014,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43027,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43064,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43065,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',5),(43066,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43069,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43071,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43074,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43083,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43102,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43121,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43129,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',12),(43140,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43141,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43144,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43147,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43154,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43155,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43160,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43161,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43171,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',4),(43175,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43176,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43177,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',20),(43181,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',5),(43182,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43184,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',7),(43203,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43206,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43211,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43221,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43223,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',14),(43232,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43235,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43236,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43237,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',17),(43238,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43240,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43242,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43243,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43261,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43274,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43281,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43298,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43339,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43340,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43355,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43356,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43357,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43358,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43359,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43360,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43371,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43375,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43376,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43377,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43382,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43383,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',4),(43384,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43387,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43389,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43396,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43404,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',4),(43411,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',5),(43419,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43425,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43441,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43443,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43446,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43447,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43450,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43468,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43487,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',4),(43488,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43503,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43508,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43526,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',5),(43531,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43534,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43546,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43547,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',8),(43553,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43562,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43563,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43564,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43568,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43570,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43581,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43582,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43583,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43591,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43593,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43594,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43597,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43600,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43607,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43631,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43632,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43633,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43645,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43646,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43648,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43649,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43650,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43655,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43658,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43664,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43676,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',4),(43688,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43690,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43691,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43699,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43714,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43716,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43717,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43718,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43719,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43729,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43731,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',3),(43747,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43748,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43750,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43751,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43754,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43787,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43795,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43797,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43798,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43801,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43802,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43803,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43827,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43851,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43856,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(43889,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43890,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43901,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43926,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43931,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(43932,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44071,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(44093,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44102,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44203,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44241,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44254,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(44301,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44324,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44404,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44423,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44470,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(44520,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44551,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',19),(44631,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44661,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44688,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44707,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44710,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44723,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44728,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44740,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44743,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44749,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(44755,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44759,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',2),(44825,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44855,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44856,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44866,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44867,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44868,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44885,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44887,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(44997,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45015,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45016,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45023,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45032,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45062,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45089,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45101,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45124,'4df2c6eecb985fcb20da3c5ed46e3b910c7077ab.json',1),(45228,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',65),(45245,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45250,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',39),(45252,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45254,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45257,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',7),(45259,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45265,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',4),(45268,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45274,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',5),(45275,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',111),(45278,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45281,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45285,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45291,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45301,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45303,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45304,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45305,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45310,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45313,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45316,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',4),(45325,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45330,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45332,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45339,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',4),(45349,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45368,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45370,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45407,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45410,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45417,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45419,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45444,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45447,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45452,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45455,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45458,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45460,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45501,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45502,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45520,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45555,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45564,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',7),(45569,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',6),(45577,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45585,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',14),(45589,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',17),(45591,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',8),(45609,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',5),(45618,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45630,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',26),(45631,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',21),(45640,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',28),(45641,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45653,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45661,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45673,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45674,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',21),(45675,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',9),(45714,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45724,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45728,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45740,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45744,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45763,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',8),(45765,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45767,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45805,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',6),(45857,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45859,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45902,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',4),(45904,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',5),(45963,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45964,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45965,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45966,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45967,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45973,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45975,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(45976,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45977,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45979,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45985,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45986,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(45987,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45995,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45996,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45997,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(45998,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46012,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46035,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46041,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46052,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',6),(46084,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46105,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',12),(46113,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',4),(46127,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(46147,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46163,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46164,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46166,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46175,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46179,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46180,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(46183,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46190,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46196,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46206,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46207,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46212,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46213,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46214,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46216,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46217,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46218,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46222,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46239,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(46259,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46268,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(46273,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(46316,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(46359,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46413,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46424,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46428,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46429,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46431,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46432,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46459,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46520,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46524,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46544,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(46581,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46591,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(46592,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46593,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46594,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(46597,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46604,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46612,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46714,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46724,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46826,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46833,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(46843,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46856,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',5),(46878,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46911,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46922,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46928,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46929,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(46950,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',20),(47021,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47074,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',25),(47086,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47091,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47103,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47107,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47108,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47109,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47116,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47187,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47189,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47194,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47214,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47215,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47276,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',3),(47277,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',12),(47304,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47305,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47332,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47335,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47338,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47360,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47406,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47425,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47448,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47494,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47495,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47497,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47498,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47499,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',5),(47505,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47506,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47507,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47533,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47561,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47563,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47566,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47570,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47591,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47618,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47624,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',4),(47627,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47654,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47703,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47706,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47725,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47745,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47767,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47770,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47779,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47783,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47784,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47785,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',4),(47786,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47787,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',2),(47814,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47824,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47828,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47829,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',30),(47907,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47914,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47930,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47931,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47934,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47947,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47948,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47955,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47970,'83b05e8afa6cbe7a68170ee155f7eca37c91d51f.json',1),(47975,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(47980,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',5),(47981,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(47982,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',5),(47983,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(47985,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(47987,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',7),(47988,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(47990,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',14),(47991,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',28),(48005,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48006,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',9),(48009,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',5),(48036,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48065,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48072,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48075,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(48078,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48083,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',14),(48086,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48097,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48104,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48114,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48118,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48120,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48125,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48131,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',18),(48132,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48133,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48138,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48141,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48153,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48156,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48164,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48175,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48178,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48183,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48185,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48187,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(48197,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48206,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48207,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48212,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48214,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48219,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48221,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48222,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48223,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48240,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48270,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(48286,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48289,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48293,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48307,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',5),(48311,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48325,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48343,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',4),(48360,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48367,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48377,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48402,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48405,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48410,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48433,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48437,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48439,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48465,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48468,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48469,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48475,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48517,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48539,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(48541,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48549,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48562,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48569,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48576,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48583,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(48591,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48617,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48638,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48639,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',4),(48676,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48687,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48698,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48702,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48705,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48707,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48708,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48740,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48750,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48766,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48774,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48775,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48779,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48790,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48797,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48805,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(48809,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48810,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48811,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48833,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48835,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',6),(48854,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48861,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48886,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48892,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48895,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48897,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48900,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48901,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48904,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',7),(48907,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48908,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(48912,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48916,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48918,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48919,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48923,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48925,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48952,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48954,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48985,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(48999,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49003,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49008,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49012,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49017,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49018,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49022,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49023,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',3),(49027,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',2),(49032,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49034,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49035,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49096,'025339bfce1cb8efa81c5accdabefe04dcdac9d2.json',1),(49102,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49106,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49110,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49121,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49122,'8d7400a2b387820cd391d7df8194642e50402a0c.json',5),(49128,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49129,'8d7400a2b387820cd391d7df8194642e50402a0c.json',66),(49135,'8d7400a2b387820cd391d7df8194642e50402a0c.json',5),(49136,'8d7400a2b387820cd391d7df8194642e50402a0c.json',6),(49138,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49143,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49146,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49165,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(49181,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49184,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49190,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49191,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49196,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49202,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49266,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49267,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49286,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49319,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(49320,'8d7400a2b387820cd391d7df8194642e50402a0c.json',31),(49347,'8d7400a2b387820cd391d7df8194642e50402a0c.json',93),(49348,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49350,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(49369,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49395,'8d7400a2b387820cd391d7df8194642e50402a0c.json',6),(49400,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(49419,'8d7400a2b387820cd391d7df8194642e50402a0c.json',7),(49443,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49444,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49445,'8d7400a2b387820cd391d7df8194642e50402a0c.json',32),(49452,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49472,'8d7400a2b387820cd391d7df8194642e50402a0c.json',11),(49473,'8d7400a2b387820cd391d7df8194642e50402a0c.json',18),(49490,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(49495,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49498,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49519,'8d7400a2b387820cd391d7df8194642e50402a0c.json',5),(49558,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49559,'8d7400a2b387820cd391d7df8194642e50402a0c.json',7),(49563,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49572,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49578,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(49588,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49603,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49614,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49619,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49628,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49662,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(49707,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(49719,'8d7400a2b387820cd391d7df8194642e50402a0c.json',5),(49727,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(49741,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49747,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49750,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49765,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49839,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49840,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(49868,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49907,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49912,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49972,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49976,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(49977,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49980,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49985,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49987,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(49994,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50001,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50009,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50025,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(50033,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50039,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50040,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50041,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50074,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50107,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50108,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50109,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50110,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50113,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50114,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50115,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50117,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50151,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50156,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50205,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50221,'8d7400a2b387820cd391d7df8194642e50402a0c.json',7),(50236,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50250,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50256,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50258,'8d7400a2b387820cd391d7df8194642e50402a0c.json',5),(50260,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50269,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50270,'8d7400a2b387820cd391d7df8194642e50402a0c.json',16),(50276,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50281,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50286,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50315,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50349,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(50351,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50405,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50415,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(50434,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50451,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(50571,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50576,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50578,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50619,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50646,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50653,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50681,'8d7400a2b387820cd391d7df8194642e50402a0c.json',6),(50691,'8d7400a2b387820cd391d7df8194642e50402a0c.json',4),(50722,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50737,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50741,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50791,'8d7400a2b387820cd391d7df8194642e50402a0c.json',5),(50793,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(50802,'8d7400a2b387820cd391d7df8194642e50402a0c.json',3),(50803,'8d7400a2b387820cd391d7df8194642e50402a0c.json',32),(50806,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50809,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50942,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(50999,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(51015,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(51065,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(51090,'8d7400a2b387820cd391d7df8194642e50402a0c.json',108),(51153,'8d7400a2b387820cd391d7df8194642e50402a0c.json',2),(51165,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(51177,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(51179,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(51220,'8d7400a2b387820cd391d7df8194642e50402a0c.json',1),(51228,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51245,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51248,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51249,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51250,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51251,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51254,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',28),(51270,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51271,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51272,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(51281,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51310,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51319,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51330,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51366,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',5),(51388,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51399,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51408,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51426,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(51429,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51450,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',4),(51500,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',4),(51504,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51505,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51507,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(51510,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',19),(51513,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51520,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51522,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51531,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',23),(51534,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51566,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51567,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(51576,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51582,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51630,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',6),(51638,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51675,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(51687,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(51692,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51699,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51731,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51732,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51733,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51734,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51735,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51750,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51774,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',20),(51820,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',5),(51821,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51846,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',5),(51894,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51895,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51905,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51907,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',4),(51925,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51928,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',6),(51933,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51935,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51942,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(51971,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(51982,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51986,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(51988,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',23),(52014,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52018,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',67),(52027,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52042,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52043,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52065,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52067,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52071,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52114,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52115,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52121,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52122,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',16),(52127,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52135,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52146,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52148,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52151,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52163,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',6),(52174,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52185,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52198,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',5),(52199,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',6),(52210,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52230,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52231,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52254,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52302,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(52332,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52344,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52371,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(52495,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52518,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52527,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',8),(52548,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52550,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52570,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52643,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52645,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52661,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',5),(52678,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52705,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52720,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52756,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52758,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52759,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52760,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52761,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52762,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52763,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52765,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52775,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52776,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',4),(52777,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(52816,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',6),(52817,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',6),(52818,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',10),(52851,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52860,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',5),(52883,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52913,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',10),(52929,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',4),(52944,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52949,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52952,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(52976,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(52979,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(52985,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53029,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53036,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53040,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53066,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53068,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53083,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53112,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53130,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53175,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53180,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53185,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',6),(53191,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53222,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53224,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53226,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53227,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53228,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53238,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',4),(53251,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53290,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53337,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53345,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53383,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53422,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53479,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53491,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53510,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53541,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53571,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53635,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53639,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53689,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53690,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53691,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53745,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(53775,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53784,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53798,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',3),(53833,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53885,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53946,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53956,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(53989,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54054,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',2),(54079,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54088,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54097,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54112,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54155,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54178,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54182,'f1d1b9694aa43c837d9b758cb2d45d8a24d293e3.json',1),(54187,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54207,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54238,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54251,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54266,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54312,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54314,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(54326,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54331,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54339,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54346,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54381,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',5),(54411,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54439,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(54449,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',20),(54450,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54451,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54455,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54456,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54459,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54460,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54548,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(54574,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',3),(54596,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54602,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(54608,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54618,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54619,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(54620,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54621,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54622,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54632,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54634,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54643,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54644,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54645,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54650,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54664,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54666,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(54719,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54745,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(54755,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54757,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54759,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',3),(54771,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',10),(54816,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54896,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(54957,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',4),(55010,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(55098,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',4),(55110,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',5),(55203,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(55211,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55217,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55254,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55265,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(55268,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55375,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55421,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55432,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(55433,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',3),(55434,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',4),(55445,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55487,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55490,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55496,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(55507,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55579,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55581,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',3),(55600,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',7),(55601,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',8),(55602,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',8),(55625,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55642,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55666,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55742,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55856,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55904,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55907,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',2),(55910,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',9),(55917,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(55998,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56118,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56119,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56126,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56149,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56180,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56210,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56226,'867e1b0f6ca8757f2a32a625d99b23888ab40d49.json',1),(56292,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56301,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56304,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56307,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56326,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',22),(56332,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56333,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',7),(56337,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56342,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',4),(56345,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',3),(56350,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',33),(56351,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56352,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56364,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56378,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56382,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56387,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',9),(56391,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',21),(56402,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',8),(56403,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',7),(56404,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',9),(56406,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',12),(56409,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56410,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',42),(56476,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56478,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56488,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56499,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56501,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',6),(56509,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56510,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56521,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',4),(56542,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56548,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56554,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56563,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56565,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56567,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56568,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56569,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56571,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56572,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56574,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56581,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56588,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56601,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56605,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56606,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56616,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56628,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56647,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56673,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56681,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56697,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56703,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56706,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56710,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',3),(56711,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56716,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56717,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56718,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56722,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',6),(56724,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56725,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56779,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56780,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56788,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56793,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56805,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56809,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',4),(56810,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',3),(56816,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',3),(56838,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56839,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56847,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56883,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56908,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56909,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(56916,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(56998,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57000,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57027,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57053,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',5),(57070,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57075,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',3),(57086,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57093,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57096,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57105,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',5),(57106,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(57121,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(57128,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57131,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57133,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57136,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57137,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57144,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57151,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57163,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57166,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',2),(57168,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57170,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57191,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57193,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57205,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57208,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57221,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57222,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57224,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57230,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57232,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57247,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57259,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',3),(57287,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57321,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57383,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57388,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57414,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57431,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57439,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',9),(57473,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57474,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57496,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57542,'b022e6ee1e7b9a0a822b1255c2fa12f81ddc52fd.json',1),(57600,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',59),(57601,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57612,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57616,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57619,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57621,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(57622,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(57623,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57625,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57636,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(57637,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(57645,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',9),(57647,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(57651,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',81),(57653,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',15),(57688,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57689,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57690,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57704,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(57714,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57737,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57739,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',4),(57754,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',8),(57766,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57767,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',9),(57785,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(57788,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57790,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57801,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',8),(57808,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(57814,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57815,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57831,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(57834,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57837,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57885,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57891,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',16),(57899,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57902,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57954,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57955,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57957,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57959,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',71),(57961,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57962,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57968,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(57998,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58001,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58003,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58008,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58020,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58021,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58023,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58030,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58033,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58039,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58041,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58065,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58067,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58073,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58075,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58081,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58082,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58083,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58086,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58087,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58092,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58098,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58101,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58103,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',9),(58112,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58113,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58116,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58123,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58128,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58131,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',15),(58132,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58135,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58140,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58141,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58144,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58155,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58168,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58174,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58175,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',7),(58187,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58189,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',7),(58192,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58194,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58195,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58201,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58203,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58214,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58217,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58218,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58226,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58227,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58230,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58231,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58235,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58272,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58273,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58283,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58297,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58298,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58339,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58369,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58428,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58483,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58532,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58543,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58545,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58594,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58595,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58607,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58651,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58666,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58670,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58705,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58706,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58709,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58775,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58794,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58795,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58799,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58805,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58806,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',8),(58807,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58813,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',4),(58817,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58829,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58837,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58844,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58849,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58857,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58858,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58903,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58904,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58924,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58925,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58958,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',6),(58975,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(58980,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(58983,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',2),(58990,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59006,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',3),(59030,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59047,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59054,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59099,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59119,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59137,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59166,'f1bbb1b2e330c440acd6ff3d8a0f0d2e5adedd70.json',1),(59187,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59203,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(59217,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59221,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59280,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59367,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',5),(59389,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',4),(59437,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59440,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',37),(59446,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59447,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',6),(59449,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59451,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',3),(59454,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(59455,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59458,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59460,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',5),(59463,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59475,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',3),(59492,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59521,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59607,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',6),(59627,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(59634,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59647,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(59650,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59655,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59675,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59679,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59700,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59754,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59766,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(59787,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59800,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59801,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59802,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59803,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59804,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59805,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59806,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59807,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59808,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59809,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59810,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59811,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59812,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59813,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59850,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(59865,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59902,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59903,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59905,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59923,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(59953,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60019,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60045,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60047,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60052,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',37),(60117,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60138,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60156,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60182,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60204,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60208,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60219,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60225,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',9),(60237,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60259,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60260,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60262,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60284,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60287,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60288,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60299,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60305,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60334,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60428,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60468,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60484,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60515,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60518,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60603,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',9),(60621,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60624,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',4),(60645,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60646,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60647,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60651,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60667,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60668,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60702,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60706,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60710,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60732,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60734,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60751,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60752,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60758,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',66),(60761,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60762,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60763,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60764,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60765,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60767,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60831,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60833,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60836,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60837,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60861,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(60895,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(60999,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',8),(61060,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61064,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61068,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61075,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61076,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(61185,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61197,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61198,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',4),(61218,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',3),(61247,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61256,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61276,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61280,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(61312,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61338,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61362,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',3),(61363,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(61367,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61387,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61399,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61414,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',4),(61424,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61438,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(61459,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',2),(61484,'03203ab50eb64271a9e825f94a1b1a6c46ea14b3.json',1),(61506,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61533,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61546,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61554,'a77badb297bf4927419481e660ea8f1806ed91b1.json',23),(61557,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61558,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61565,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61578,'a77badb297bf4927419481e660ea8f1806ed91b1.json',11),(61582,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(61586,'a77badb297bf4927419481e660ea8f1806ed91b1.json',5),(61601,'a77badb297bf4927419481e660ea8f1806ed91b1.json',32),(61607,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61642,'a77badb297bf4927419481e660ea8f1806ed91b1.json',22),(61645,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61665,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(61670,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61672,'a77badb297bf4927419481e660ea8f1806ed91b1.json',9),(61678,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61689,'a77badb297bf4927419481e660ea8f1806ed91b1.json',3),(61692,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61705,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61727,'a77badb297bf4927419481e660ea8f1806ed91b1.json',3),(61728,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61801,'a77badb297bf4927419481e660ea8f1806ed91b1.json',12),(61802,'a77badb297bf4927419481e660ea8f1806ed91b1.json',10),(61809,'a77badb297bf4927419481e660ea8f1806ed91b1.json',4),(61812,'a77badb297bf4927419481e660ea8f1806ed91b1.json',5),(61816,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(61820,'a77badb297bf4927419481e660ea8f1806ed91b1.json',14),(61823,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61844,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61845,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61859,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(61890,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61941,'a77badb297bf4927419481e660ea8f1806ed91b1.json',18),(61980,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(61991,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62023,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62040,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62047,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62049,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62055,'a77badb297bf4927419481e660ea8f1806ed91b1.json',14),(62078,'a77badb297bf4927419481e660ea8f1806ed91b1.json',70),(62094,'a77badb297bf4927419481e660ea8f1806ed91b1.json',10),(62095,'a77badb297bf4927419481e660ea8f1806ed91b1.json',10),(62111,'a77badb297bf4927419481e660ea8f1806ed91b1.json',9),(62136,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62137,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62138,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62144,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62145,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62156,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62161,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62162,'a77badb297bf4927419481e660ea8f1806ed91b1.json',8),(62165,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62190,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62191,'a77badb297bf4927419481e660ea8f1806ed91b1.json',42),(62237,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62239,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62259,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62282,'a77badb297bf4927419481e660ea8f1806ed91b1.json',30),(62302,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62311,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62320,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62324,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62341,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62351,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62352,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62358,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62369,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62370,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62377,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62385,'a77badb297bf4927419481e660ea8f1806ed91b1.json',9),(62409,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62415,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62422,'a77badb297bf4927419481e660ea8f1806ed91b1.json',4),(62440,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62442,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62443,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62452,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62462,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62484,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62487,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62492,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62493,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62520,'a77badb297bf4927419481e660ea8f1806ed91b1.json',3),(62535,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62550,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62554,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62562,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62575,'a77badb297bf4927419481e660ea8f1806ed91b1.json',4),(62600,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62610,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62613,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62638,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62661,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62663,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62667,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62668,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62676,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62677,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62688,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62714,'a77badb297bf4927419481e660ea8f1806ed91b1.json',3),(62785,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(62792,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(62955,'a77badb297bf4927419481e660ea8f1806ed91b1.json',3),(63030,'a77badb297bf4927419481e660ea8f1806ed91b1.json',52),(63154,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(63214,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63325,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63371,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63378,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63434,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63462,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63474,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63501,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(63507,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63526,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63551,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63596,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63627,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63702,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63703,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(63707,'a77badb297bf4927419481e660ea8f1806ed91b1.json',2),(63721,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63735,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63775,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63784,'a77badb297bf4927419481e660ea8f1806ed91b1.json',6),(63788,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63792,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63828,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63830,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63856,'a77badb297bf4927419481e660ea8f1806ed91b1.json',3),(63868,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(63960,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(64001,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(64014,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(64031,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(64039,'a77badb297bf4927419481e660ea8f1806ed91b1.json',1),(64055,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',4),(64058,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64069,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64081,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64082,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64102,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',3),(64110,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64119,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64128,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64150,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64166,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64215,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64234,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64246,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64252,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64269,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64356,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64361,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',14),(64362,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',24),(64404,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64406,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64411,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64412,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',4),(64416,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',18),(64434,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64444,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',20),(64452,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',18),(64493,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64494,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',4),(64499,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64500,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64501,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',6),(64502,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',3),(64503,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',4),(64504,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64505,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64509,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64510,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64511,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64514,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64524,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64526,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64528,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64529,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64530,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64532,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64533,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64534,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64536,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64537,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64540,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64541,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64542,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64550,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64556,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64557,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64575,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64576,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64579,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64592,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',3),(64619,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',4),(64628,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',5),(64647,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64656,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64664,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64674,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64675,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64677,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64678,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64706,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64713,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64720,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64741,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64791,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64795,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64796,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',4),(64831,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64837,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64838,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64840,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64858,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64920,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64921,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64922,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64925,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64926,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64944,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',4),(64974,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(64979,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(64983,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(65038,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65055,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(65061,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65089,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',23),(65115,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(65218,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65319,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65357,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65411,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65542,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',2),(65568,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65708,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65943,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65946,'93c6eef32a1a511ee989a259eab0e12174dc6859.json',1),(65969,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',3),(65993,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66015,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',3),(66019,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66026,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66046,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66068,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66070,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66072,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66104,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66107,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',9),(66108,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',5),(66110,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66123,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66235,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66240,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66241,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66279,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',3),(66299,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66300,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66306,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66314,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66323,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',13),(66335,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',47),(66338,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',10),(66353,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(66357,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66386,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66448,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66455,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66457,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66458,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66467,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66472,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66477,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66481,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66494,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66504,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66532,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66534,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66550,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66552,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66553,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66562,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66588,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66591,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',11),(66596,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',7),(66619,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66635,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66649,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66653,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66666,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66679,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66682,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66683,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',15),(66712,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66713,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',32),(66723,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',3),(66743,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66756,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66767,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66804,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(66824,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',5),(66827,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66915,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(66995,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(67001,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(67002,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(67013,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(67014,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(67046,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(67055,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67088,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67120,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',3),(67131,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(67137,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(67138,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',5),(67188,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67190,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',7),(67206,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67209,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67224,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(67261,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(67434,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67476,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67482,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67533,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(67565,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67613,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',4),(67617,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67630,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',36),(67652,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67852,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(67981,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68230,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68242,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68244,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68343,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68525,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68528,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68547,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68588,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',24),(68688,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68701,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68750,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68784,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68837,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68883,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68892,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',3),(68924,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68971,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(68996,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(69091,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',2),(69127,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(69129,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(69147,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(69174,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(69181,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(69189,'184aded923f0ac3cbdbcf74d2a5b42cda0f414c2.json',1),(69200,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69226,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69229,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69249,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(69288,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69301,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69315,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',16),(69316,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69317,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69341,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69342,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69346,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69488,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69519,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69530,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(69540,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69569,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69582,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69590,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69591,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69593,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',40),(69595,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69597,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69601,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69602,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69603,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69631,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69665,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69681,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69712,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69713,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69721,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69724,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69725,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69727,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69740,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69750,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69758,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',28),(69771,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69783,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',5),(69784,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69787,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(69789,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',5),(69790,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69792,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69809,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69816,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69818,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69841,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69889,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',15),(69903,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',4),(69910,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(69912,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69938,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',4),(69952,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69968,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69969,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(69990,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(69999,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70004,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70011,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(70012,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(70046,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70078,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',5),(70080,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(70082,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70086,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70100,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70112,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70213,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(70247,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70249,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70265,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70277,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70289,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70298,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70300,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70301,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70416,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70492,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(70503,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(70513,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70587,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(70608,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70807,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(70812,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71074,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71086,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71111,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71155,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(71213,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(71265,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',3),(71422,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71442,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71466,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71480,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71571,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71614,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71616,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71634,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71637,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71640,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71718,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71749,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(71767,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71789,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71802,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71844,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',2),(71916,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(71988,'40e41f0c52b39669dee24e37875d7a9fabc38636.json',1),(72030,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72031,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72039,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72045,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72052,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',6),(72054,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72058,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72066,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72069,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72079,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72081,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72082,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',34),(72097,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72105,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72110,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',7),(72118,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72121,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72136,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72137,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72138,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',5),(72139,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',5),(72140,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',4),(72149,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72169,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72179,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72181,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72184,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72187,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72188,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',4),(72189,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72207,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72211,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72212,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72224,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72229,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72234,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',4),(72237,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72238,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72243,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',14),(72244,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72246,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',4),(72250,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72251,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72252,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72255,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72257,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72258,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72261,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',6),(72262,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72265,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72266,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',5),(72271,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',8),(72275,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',6),(72282,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',4),(72292,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72297,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72301,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72313,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',7),(72316,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72319,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72322,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',11),(72327,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72340,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72341,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72342,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72345,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72347,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72348,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72364,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72366,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72370,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72371,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72372,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72377,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',4),(72378,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72385,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72387,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72412,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72413,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72416,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72419,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72422,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72426,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72428,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72433,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72438,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72439,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',5),(72449,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72458,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72462,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72475,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',5),(72481,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72482,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72485,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',4),(72495,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72502,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72508,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72510,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72520,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72522,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72535,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72540,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72541,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72544,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72556,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72561,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72564,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72570,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72574,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',6),(72577,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72592,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72603,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72612,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72618,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72619,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72620,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72625,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72631,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72646,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72652,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72665,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72668,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72675,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72678,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72688,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72690,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72693,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72726,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',5),(72730,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72738,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72743,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72749,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72764,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72770,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72773,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72775,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72778,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72779,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72780,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72781,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72784,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72790,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72804,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72873,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72891,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72893,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72895,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72899,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72910,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(72946,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72949,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',8),(72950,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72951,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72953,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(72963,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72984,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(72999,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73001,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',5),(73013,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73015,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73016,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73024,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73028,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73029,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73032,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73033,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73052,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73063,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73065,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73069,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73070,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73073,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73076,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73090,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(73095,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73100,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',7),(73107,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73127,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(73147,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73149,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73168,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73188,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73224,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73260,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73281,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73283,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73292,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73299,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73307,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73334,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73370,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73384,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(73392,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73394,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73397,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73399,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73412,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73429,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73433,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73438,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73439,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73452,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73469,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73471,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73489,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73495,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73533,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73560,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73561,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(73570,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73579,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73587,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(73621,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73640,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73766,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73768,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73790,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73793,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73800,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73814,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73837,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73843,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73844,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73850,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73864,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73867,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73879,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(73882,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(73892,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73893,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73900,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73911,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73917,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73918,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73926,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73936,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73937,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73938,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73950,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73953,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73954,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73969,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73970,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(73982,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73983,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(73993,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',3),(74001,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74002,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74003,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74011,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74012,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74041,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74045,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74050,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74054,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74064,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74067,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74073,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74084,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74089,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74112,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74115,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74171,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74182,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74220,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74229,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(74241,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74249,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74253,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74258,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74263,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',2),(74270,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74277,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74286,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74287,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74292,'d77233dd9085c9c76787c84ee9a1e249424b283a.json',1),(74321,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74328,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74341,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74351,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74354,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',10),(74358,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',3),(74377,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74382,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74390,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74404,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74406,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74407,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74408,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74412,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74416,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74420,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',14),(74460,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74462,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',3),(74465,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74466,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',3),(74486,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',3),(74527,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74528,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74532,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',3),(74577,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74642,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',5),(74646,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74650,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74659,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74662,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74674,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74721,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74731,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74738,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74748,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74760,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74762,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74766,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74768,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74788,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74794,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74811,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',6),(74825,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74827,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',4),(74830,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74867,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74871,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74880,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74882,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74899,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74904,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74912,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74913,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74914,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74927,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74938,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74946,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(74964,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(74968,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75012,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75030,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75038,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75046,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75056,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',3),(75069,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75071,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',8),(75074,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75076,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75080,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75103,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75104,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75105,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75109,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75113,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75120,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75134,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75135,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75152,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75173,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75191,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75194,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(75196,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75210,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75211,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75214,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75219,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75225,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75238,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',1),(75257,'7d624acf891472c3452b1deb89e0dc44a40d0c94.json',2),(75272,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',4),(75274,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75281,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75284,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75285,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75286,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75287,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75288,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',16),(75300,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75304,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75309,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',3),(75322,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',2),(75330,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75342,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75343,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75350,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75354,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75366,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75368,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75378,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75390,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',8),(75391,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75404,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75421,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75443,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75445,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75447,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75461,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75500,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75504,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75513,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75515,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75517,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75527,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75528,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75535,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75602,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',3),(75603,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',3),(75629,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75631,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75635,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75643,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75656,'d706e82e2ebc8571e70654d1acf0ffd3969ab2d3.json',1),(75670,'ada57a9f084a016deb20b6f83c442072e6e37955.json',99),(75673,'ada57a9f084a016deb20b6f83c442072e6e37955.json',5),(75674,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75676,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75679,'ada57a9f084a016deb20b6f83c442072e6e37955.json',9),(75685,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75686,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75687,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(75688,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75690,'ada57a9f084a016deb20b6f83c442072e6e37955.json',5),(75706,'ada57a9f084a016deb20b6f83c442072e6e37955.json',19),(75709,'ada57a9f084a016deb20b6f83c442072e6e37955.json',31),(75710,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75720,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75742,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75746,'ada57a9f084a016deb20b6f83c442072e6e37955.json',5),(75753,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(75757,'ada57a9f084a016deb20b6f83c442072e6e37955.json',47),(75774,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(75775,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75778,'ada57a9f084a016deb20b6f83c442072e6e37955.json',10),(75781,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75782,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75789,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75829,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75831,'ada57a9f084a016deb20b6f83c442072e6e37955.json',4),(75848,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75869,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(75871,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75874,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75875,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75880,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75899,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75900,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(75913,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75927,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75928,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75937,'ada57a9f084a016deb20b6f83c442072e6e37955.json',4),(75941,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75943,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75946,'ada57a9f084a016deb20b6f83c442072e6e37955.json',6),(75970,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75971,'ada57a9f084a016deb20b6f83c442072e6e37955.json',77),(75973,'ada57a9f084a016deb20b6f83c442072e6e37955.json',10),(75974,'ada57a9f084a016deb20b6f83c442072e6e37955.json',27),(75980,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(75985,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(75990,'ada57a9f084a016deb20b6f83c442072e6e37955.json',13),(76004,'ada57a9f084a016deb20b6f83c442072e6e37955.json',12),(76018,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(76054,'ada57a9f084a016deb20b6f83c442072e6e37955.json',15),(76059,'ada57a9f084a016deb20b6f83c442072e6e37955.json',8),(76063,'ada57a9f084a016deb20b6f83c442072e6e37955.json',6),(76072,'ada57a9f084a016deb20b6f83c442072e6e37955.json',13),(76078,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(76092,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76109,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76112,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76171,'ada57a9f084a016deb20b6f83c442072e6e37955.json',13),(76180,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76193,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76195,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76203,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76204,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76210,'ada57a9f084a016deb20b6f83c442072e6e37955.json',8),(76223,'ada57a9f084a016deb20b6f83c442072e6e37955.json',11),(76231,'ada57a9f084a016deb20b6f83c442072e6e37955.json',8),(76256,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76275,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76280,'ada57a9f084a016deb20b6f83c442072e6e37955.json',23),(76285,'ada57a9f084a016deb20b6f83c442072e6e37955.json',8),(76287,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(76295,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76300,'ada57a9f084a016deb20b6f83c442072e6e37955.json',4),(76308,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(76313,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76332,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76354,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(76398,'ada57a9f084a016deb20b6f83c442072e6e37955.json',175),(76410,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76420,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76445,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76461,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76476,'ada57a9f084a016deb20b6f83c442072e6e37955.json',8),(76482,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(76503,'ada57a9f084a016deb20b6f83c442072e6e37955.json',4),(76518,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76548,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76549,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76552,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76568,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76600,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76663,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76664,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76697,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(76704,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76713,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76719,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76740,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76841,'ada57a9f084a016deb20b6f83c442072e6e37955.json',6),(76845,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(76891,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76904,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76906,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76907,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76913,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(76945,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77002,'ada57a9f084a016deb20b6f83c442072e6e37955.json',31),(77024,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77027,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77035,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77039,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77060,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77063,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77074,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77077,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77081,'ada57a9f084a016deb20b6f83c442072e6e37955.json',20),(77086,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77117,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77118,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77153,'ada57a9f084a016deb20b6f83c442072e6e37955.json',32),(77225,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77252,'ada57a9f084a016deb20b6f83c442072e6e37955.json',4),(77253,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77257,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(77259,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77260,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(77261,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77271,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77274,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77281,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77309,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77340,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77364,'ada57a9f084a016deb20b6f83c442072e6e37955.json',19),(77378,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77379,'ada57a9f084a016deb20b6f83c442072e6e37955.json',25),(77387,'ada57a9f084a016deb20b6f83c442072e6e37955.json',8),(77390,'ada57a9f084a016deb20b6f83c442072e6e37955.json',7),(77394,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77436,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77452,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77490,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77566,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77622,'ada57a9f084a016deb20b6f83c442072e6e37955.json',31),(77655,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77682,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(77692,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(77724,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77734,'ada57a9f084a016deb20b6f83c442072e6e37955.json',4),(77744,'ada57a9f084a016deb20b6f83c442072e6e37955.json',21),(77772,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77773,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77782,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77791,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77798,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77806,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77820,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(77837,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77845,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77871,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(77893,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77894,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77897,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77911,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(77962,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78139,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78157,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78164,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78168,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78206,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78214,'ada57a9f084a016deb20b6f83c442072e6e37955.json',6),(78229,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78230,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78241,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78242,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78320,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78324,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78325,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78328,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78342,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78343,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78353,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(78373,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78412,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78430,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78439,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78443,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78474,'ada57a9f084a016deb20b6f83c442072e6e37955.json',5),(78484,'ada57a9f084a016deb20b6f83c442072e6e37955.json',4),(78516,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(78533,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78578,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78601,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78603,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78606,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78613,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78616,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78655,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78685,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78704,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78711,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78715,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78724,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78727,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78773,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(78870,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(78900,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(78920,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78980,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(78992,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(78998,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79004,'ada57a9f084a016deb20b6f83c442072e6e37955.json',14),(79013,'ada57a9f084a016deb20b6f83c442072e6e37955.json',7),(79036,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(79049,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79074,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79080,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79096,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79099,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79102,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79129,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79150,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79151,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79168,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79184,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79187,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79189,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79195,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79198,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79202,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79203,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79217,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(79226,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79229,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(79230,'ada57a9f084a016deb20b6f83c442072e6e37955.json',3),(79232,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79243,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79255,'ada57a9f084a016deb20b6f83c442072e6e37955.json',44),(79292,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79314,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79323,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(79325,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(79366,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79367,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79373,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79374,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79388,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79393,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79396,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(79507,'ada57a9f084a016deb20b6f83c442072e6e37955.json',2),(79533,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79539,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79553,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79556,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79557,'ada57a9f084a016deb20b6f83c442072e6e37955.json',1),(79660,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(79694,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(79811,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(79869,'6eeaffdb22b4e310867262d1891a1442769c7803.json',4),(79871,'6eeaffdb22b4e310867262d1891a1442769c7803.json',8),(79902,'6eeaffdb22b4e310867262d1891a1442769c7803.json',18),(79940,'6eeaffdb22b4e310867262d1891a1442769c7803.json',3),(79979,'6eeaffdb22b4e310867262d1891a1442769c7803.json',6),(79989,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(79990,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(79994,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(80099,'6eeaffdb22b4e310867262d1891a1442769c7803.json',24),(80126,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80165,'6eeaffdb22b4e310867262d1891a1442769c7803.json',10),(80184,'6eeaffdb22b4e310867262d1891a1442769c7803.json',3),(80188,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(80208,'6eeaffdb22b4e310867262d1891a1442769c7803.json',35),(80242,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80244,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80247,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80251,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80252,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80256,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80267,'6eeaffdb22b4e310867262d1891a1442769c7803.json',9),(80269,'6eeaffdb22b4e310867262d1891a1442769c7803.json',8),(80273,'6eeaffdb22b4e310867262d1891a1442769c7803.json',4),(80311,'6eeaffdb22b4e310867262d1891a1442769c7803.json',65),(80312,'6eeaffdb22b4e310867262d1891a1442769c7803.json',52),(80318,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80351,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80380,'6eeaffdb22b4e310867262d1891a1442769c7803.json',3),(80404,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(80405,'6eeaffdb22b4e310867262d1891a1442769c7803.json',62),(80498,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80508,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80550,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80697,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80716,'6eeaffdb22b4e310867262d1891a1442769c7803.json',3),(80726,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(80800,'6eeaffdb22b4e310867262d1891a1442769c7803.json',8),(80820,'6eeaffdb22b4e310867262d1891a1442769c7803.json',17),(80824,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(80835,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(80931,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(80956,'6eeaffdb22b4e310867262d1891a1442769c7803.json',4),(81069,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(81073,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(81157,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(81302,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(81366,'6eeaffdb22b4e310867262d1891a1442769c7803.json',4),(81376,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(81423,'6eeaffdb22b4e310867262d1891a1442769c7803.json',6),(81434,'6eeaffdb22b4e310867262d1891a1442769c7803.json',4),(81601,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(81606,'6eeaffdb22b4e310867262d1891a1442769c7803.json',3),(81731,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(81810,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(81976,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82040,'6eeaffdb22b4e310867262d1891a1442769c7803.json',9),(82050,'6eeaffdb22b4e310867262d1891a1442769c7803.json',2),(82058,'6eeaffdb22b4e310867262d1891a1442769c7803.json',6),(82179,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82192,'6eeaffdb22b4e310867262d1891a1442769c7803.json',3),(82246,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82449,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82452,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82461,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82679,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82697,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82758,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82839,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82914,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(82925,'6eeaffdb22b4e310867262d1891a1442769c7803.json',1),(83076,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83077,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83079,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83084,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83085,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83086,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83098,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83102,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83106,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',49),(83135,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',56),(83174,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83183,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83204,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83206,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',18),(83207,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',16),(83211,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',4),(83219,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83250,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83266,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83267,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83268,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',20),(83293,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83297,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(83392,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83428,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',65),(83432,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83442,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83461,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83464,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83496,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83547,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',3),(83553,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',4),(83556,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',4),(83557,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',4),(83592,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',8),(83600,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(83698,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83705,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(83785,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83795,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83817,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83818,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',9),(83826,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83827,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83909,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(83927,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83928,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(83940,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',21),(83966,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(83977,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84005,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84007,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84047,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(84095,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84105,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84184,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',13),(84185,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84186,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84339,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84429,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',21),(84447,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84450,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',7),(84486,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(84511,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84516,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84523,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(84547,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(84602,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',7),(84641,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84703,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84731,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84776,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(84842,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84865,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84871,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(84879,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84902,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(84946,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(84986,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85068,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85076,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85100,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85134,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(85178,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85179,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85180,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85187,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85227,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85230,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85234,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85267,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85279,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85289,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85293,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85294,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85302,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85313,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85315,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85324,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',2),(85351,'dff6195f5e24ebfa2fb4060142611b48021f673c.json',1),(85397,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85408,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85415,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85420,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85421,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85430,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85435,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85438,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',59),(85441,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85453,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85483,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85527,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85535,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85542,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85546,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85547,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85548,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85550,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85556,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85560,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85561,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85568,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85569,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85570,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',4),(85579,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85581,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85590,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85601,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85603,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85606,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',7),(85613,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85635,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85653,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85656,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',3),(85663,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85668,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85678,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85683,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',62),(85709,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85712,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85713,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',3),(85724,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85725,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85727,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',28),(85745,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',14),(85757,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85773,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',4),(85786,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85819,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',39),(85840,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85875,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(85880,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',4),(85892,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',4),(85900,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(85906,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',8),(86027,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(86050,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86055,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86087,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',3),(86097,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86101,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86102,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86103,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86113,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86115,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(86117,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86120,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86123,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',47),(86183,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86252,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86274,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',3),(86281,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',4),(86285,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86299,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86307,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(86332,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86375,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86395,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86480,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86572,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86583,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(86589,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86666,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(86687,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',5),(86771,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86794,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86797,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(86802,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',3),(86803,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86828,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86860,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86861,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86866,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',5),(86891,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(86962,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',3),(86963,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',2),(87039,'95aec306660c1b7519e9f019d92cc3a8206481e4.json',1),(87126,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87128,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87131,'d450fc8885843d48772df9a898552302f8c80b98.json',4),(87134,'d450fc8885843d48772df9a898552302f8c80b98.json',34),(87208,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87214,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87217,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87231,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87241,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87247,'d450fc8885843d48772df9a898552302f8c80b98.json',8),(87248,'d450fc8885843d48772df9a898552302f8c80b98.json',26),(87269,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87384,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(87389,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87406,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87424,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87438,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(87439,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(87445,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(87472,'d450fc8885843d48772df9a898552302f8c80b98.json',13),(87530,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87533,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87534,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(87559,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87560,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87575,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(87694,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(87774,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87803,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87804,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87838,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87850,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87851,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87852,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87853,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87854,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87855,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87856,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87858,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(87872,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87882,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87904,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87912,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87914,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87928,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(87958,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(87984,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88000,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88050,'d450fc8885843d48772df9a898552302f8c80b98.json',9),(88121,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88145,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88160,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88240,'d450fc8885843d48772df9a898552302f8c80b98.json',6),(88394,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88409,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88410,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88528,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88537,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88555,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(88596,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(88606,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(88610,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(88792,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89111,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89248,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89287,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89292,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89321,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89354,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(89427,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89447,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89498,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89528,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89593,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89625,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(89749,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(89810,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(89959,'d450fc8885843d48772df9a898552302f8c80b98.json',2),(90034,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(90052,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(90094,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(90138,'d450fc8885843d48772df9a898552302f8c80b98.json',3),(90167,'d450fc8885843d48772df9a898552302f8c80b98.json',1),(90211,'d450fc8885843d48772df9a898552302f8c80b98.json',5),(90302,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(90310,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90319,'ae19236936b05793ceac08ed5d58269c25486320.json',28),(90330,'ae19236936b05793ceac08ed5d58269c25486320.json',42),(90337,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90368,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90408,'ae19236936b05793ceac08ed5d58269c25486320.json',4),(90409,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90410,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90416,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90418,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90440,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90469,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90477,'ae19236936b05793ceac08ed5d58269c25486320.json',6),(90481,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90484,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90488,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90547,'ae19236936b05793ceac08ed5d58269c25486320.json',49),(90567,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90568,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90572,'ae19236936b05793ceac08ed5d58269c25486320.json',6),(90601,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(90610,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90619,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90620,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90621,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90622,'ae19236936b05793ceac08ed5d58269c25486320.json',3),(90626,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90628,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90629,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90630,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90656,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(90657,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90669,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90681,'ae19236936b05793ceac08ed5d58269c25486320.json',9),(90689,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90694,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(90697,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(90708,'ae19236936b05793ceac08ed5d58269c25486320.json',3),(90709,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90730,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90736,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90780,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(90815,'ae19236936b05793ceac08ed5d58269c25486320.json',3),(90820,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90835,'ae19236936b05793ceac08ed5d58269c25486320.json',10),(90837,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90856,'ae19236936b05793ceac08ed5d58269c25486320.json',6),(90862,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90866,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90876,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90878,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90879,'ae19236936b05793ceac08ed5d58269c25486320.json',220),(90891,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90930,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90936,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90941,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90950,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(90953,'ae19236936b05793ceac08ed5d58269c25486320.json',3),(90960,'ae19236936b05793ceac08ed5d58269c25486320.json',4),(90965,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(90983,'ae19236936b05793ceac08ed5d58269c25486320.json',4),(91002,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91024,'ae19236936b05793ceac08ed5d58269c25486320.json',4),(91053,'ae19236936b05793ceac08ed5d58269c25486320.json',13),(91135,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(91178,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91213,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91255,'ae19236936b05793ceac08ed5d58269c25486320.json',7),(91272,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(91274,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(91345,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91356,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(91617,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91634,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(91644,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91682,'ae19236936b05793ceac08ed5d58269c25486320.json',3),(91694,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(91724,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(91742,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91763,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91792,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91794,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91836,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(91970,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(92009,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(92057,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(92077,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(92127,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(92159,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(92166,'ae19236936b05793ceac08ed5d58269c25486320.json',2),(92212,'ae19236936b05793ceac08ed5d58269c25486320.json',1),(92299,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',3),(92392,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92414,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92415,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',13),(92422,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92440,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92444,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92481,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92496,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92536,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92546,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',5),(92548,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',5),(92549,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',25),(92557,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92560,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(92590,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92591,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92599,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92633,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',18),(92634,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(92635,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',4),(92642,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92644,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92674,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92770,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',19),(92787,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',12),(92790,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',7),(92824,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92844,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92853,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92873,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',8),(92875,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92876,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(92879,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92880,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',54),(92896,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92905,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92914,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92915,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(92918,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(92929,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',10),(92931,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(92932,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',4),(92973,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',8),(93048,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93064,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93065,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(93067,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93134,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',3),(93193,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',10),(93194,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(93225,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(93261,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(93309,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93335,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93349,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93463,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(93512,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93520,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93533,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93551,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93561,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93567,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93575,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(93576,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93578,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93579,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93588,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93606,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93611,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93626,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93636,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',30),(93658,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93688,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93692,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93722,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93736,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93765,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93776,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(93844,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93906,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93918,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(93976,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',2),(94003,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(94010,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(94020,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(94025,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(94030,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(94032,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(94036,'0a1533470817bc5ef0d0d0af56386a96b505dc0d.json',1),(94062,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94063,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94065,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94066,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',32),(94072,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94078,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94125,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94146,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',2),(94165,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',20),(94234,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',2),(94236,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94245,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94301,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94320,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94323,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94328,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94330,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94343,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94352,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94358,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94367,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94378,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94384,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94386,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',4),(94419,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94432,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94434,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94435,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94436,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94437,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94439,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94444,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94458,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94474,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94511,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94512,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94516,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94550,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94557,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94682,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94720,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94789,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94935,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94949,'4ba79e54ecf81b30b56461a6aec2094eaf7b7f06.json',1),(94963,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(94965,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(94966,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(94969,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(94970,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(94979,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95015,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95016,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',3),(95032,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',3),(95039,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',3),(95041,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',14),(95044,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95057,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',50),(95065,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',15),(95066,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',12),(95067,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',5),(95073,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95074,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(95076,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(95078,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',10),(95120,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95134,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',6),(95152,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95153,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95156,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95160,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95167,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95171,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95183,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95184,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95203,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95205,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95206,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95207,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(95209,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95216,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95227,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95228,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95229,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95230,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95231,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95232,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95233,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95236,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95237,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95238,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95240,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95241,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95260,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95261,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(95291,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95299,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95318,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95322,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95323,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',7),(95324,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95325,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95326,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95328,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(95331,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95359,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95370,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95371,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95375,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95382,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',23),(95383,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',3),(95386,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',3),(95396,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95406,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95408,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95431,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95440,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95444,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95464,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95480,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95485,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95488,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95489,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95556,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95569,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95588,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95589,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95592,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95599,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95654,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95656,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95659,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(95667,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95670,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95671,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95673,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95674,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95676,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95686,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95694,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95695,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95700,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',3),(95707,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95722,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95724,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95745,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95776,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95813,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95825,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95832,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95850,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95851,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95859,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95873,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95878,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95917,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95923,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95928,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95935,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95937,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(95944,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95953,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(95993,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(95994,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96013,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96020,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96021,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(96023,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(96024,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96035,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96036,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96083,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96092,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96096,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96123,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(96126,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96156,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96157,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96161,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(96164,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96167,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96169,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96173,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96174,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96199,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96202,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96206,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96224,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96247,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96249,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96251,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96268,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(96274,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96277,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96303,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',4),(96307,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96308,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96311,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(96333,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96337,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96343,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96353,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96371,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96373,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96384,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',2),(96391,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96399,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96440,'c6202f96bd4b2aea699d5fbe867f311498dca3aa.json',1),(96507,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',3),(96566,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(96585,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96587,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96591,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96593,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96625,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',6),(96628,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96632,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96643,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',4),(96647,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96651,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96652,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96654,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(96685,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96686,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96687,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(96688,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96689,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96690,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96691,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96692,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96693,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96694,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96695,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96696,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96700,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(96702,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96716,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',6),(96723,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96757,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96758,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96759,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96780,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96789,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(96790,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',3),(96791,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',6),(96797,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',3),(96804,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96805,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',4),(96823,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',4),(96864,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96881,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',3),(96884,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(96896,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96909,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96926,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96946,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',4),(96972,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(96977,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',8),(97014,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97054,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97057,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',3),(97061,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97062,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97106,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(97147,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',2),(97163,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97188,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97189,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97215,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97226,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97230,'a78fd1b34372e1e54bf2a192d04aa36670cea307.json',1),(97442,'c6bf372c094f035a514975c35a7f9c094abbe493.json',5),(97450,'c6bf372c094f035a514975c35a7f9c094abbe493.json',4),(97471,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97476,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97480,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97510,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97524,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97533,'c6bf372c094f035a514975c35a7f9c094abbe493.json',10),(97566,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(97576,'c6bf372c094f035a514975c35a7f9c094abbe493.json',9),(97578,'c6bf372c094f035a514975c35a7f9c094abbe493.json',6),(97582,'c6bf372c094f035a514975c35a7f9c094abbe493.json',5),(97762,'c6bf372c094f035a514975c35a7f9c094abbe493.json',11),(97771,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97772,'c6bf372c094f035a514975c35a7f9c094abbe493.json',94),(97780,'c6bf372c094f035a514975c35a7f9c094abbe493.json',4),(97805,'c6bf372c094f035a514975c35a7f9c094abbe493.json',8),(97822,'c6bf372c094f035a514975c35a7f9c094abbe493.json',10),(97839,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97864,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97867,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97871,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(97894,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97897,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97900,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(97923,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(97963,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(98054,'c6bf372c094f035a514975c35a7f9c094abbe493.json',8),(98062,'c6bf372c094f035a514975c35a7f9c094abbe493.json',10),(98068,'c6bf372c094f035a514975c35a7f9c094abbe493.json',4),(98112,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(98114,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(98116,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(98117,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(98120,'c6bf372c094f035a514975c35a7f9c094abbe493.json',12),(98127,'c6bf372c094f035a514975c35a7f9c094abbe493.json',34),(98183,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(98188,'c6bf372c094f035a514975c35a7f9c094abbe493.json',5),(98208,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(98334,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(98338,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(98456,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(98686,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(98695,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(98751,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(98783,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(98891,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(98951,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(98984,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99060,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99094,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(99098,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99237,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99251,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99252,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99254,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99293,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99362,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(99385,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99413,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99418,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99422,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99426,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99427,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99449,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99469,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99470,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99472,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99489,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99577,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99583,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99587,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99622,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99648,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99699,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99734,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99761,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99762,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99763,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99807,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(99813,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99848,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99876,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99882,'c6bf372c094f035a514975c35a7f9c094abbe493.json',2),(99884,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99886,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99920,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99931,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99939,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99963,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99966,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99967,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(99979,'c6bf372c094f035a514975c35a7f9c094abbe493.json',3),(99993,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100014,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100031,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100032,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100035,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100040,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100041,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100047,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100079,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100083,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100087,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100088,'c6bf372c094f035a514975c35a7f9c094abbe493.json',1),(100128,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100135,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100144,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',13),(100146,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',5),(100149,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100163,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100164,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100165,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100184,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100192,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',5),(100211,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',10),(100212,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',5),(100214,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(100217,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100226,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100230,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100247,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',5),(100261,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100262,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100264,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100265,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100267,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100268,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',5),(100271,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100272,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100273,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100277,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100284,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100289,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100290,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100296,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100305,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100310,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100312,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100321,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100325,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100327,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100328,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100330,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(100331,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100332,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100335,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100337,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100340,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100341,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100342,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100344,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',6),(100351,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',7),(100353,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100354,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100355,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',6),(100357,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100358,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(100361,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(100362,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(100363,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100364,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',5),(100365,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100366,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100367,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100369,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100372,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',6),(100377,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100382,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(100383,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100384,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100391,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100392,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100411,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100442,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100449,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100450,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100454,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100455,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100479,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100486,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100487,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100517,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100518,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100562,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',6),(100564,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100573,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(100579,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100599,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100664,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100682,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100689,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100696,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100698,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100748,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100752,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(100753,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100768,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100770,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100840,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100864,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100865,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100866,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100867,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100876,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100878,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100879,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100894,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',9),(100903,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100920,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100922,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100930,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100935,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100938,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100947,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(100950,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(100955,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100958,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(100963,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(100994,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101001,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101011,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101030,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101041,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101053,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101057,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101071,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101094,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101126,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101141,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101151,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(101181,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101211,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101224,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101232,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101238,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101256,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101265,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101296,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101328,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101330,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101352,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(101359,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101360,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101371,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101373,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(101395,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101398,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101400,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101414,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101449,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101476,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101479,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101499,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101501,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',12),(101527,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101543,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101551,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101552,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101560,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101562,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101577,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',5),(101582,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101586,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101670,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101687,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101696,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101697,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101701,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101723,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101737,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(101745,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101746,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101748,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101810,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101812,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101813,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101820,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',6),(101844,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101884,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(101900,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(101906,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101912,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101936,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101939,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(101954,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101977,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(101988,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',16),(101996,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102001,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',12),(102015,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102021,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102025,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102067,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102074,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102105,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102106,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102107,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102114,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',4),(102118,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102119,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',3),(102124,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102136,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102147,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102205,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102220,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102239,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102264,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',2),(102265,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102266,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102272,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102273,'1cafb8587a42557c4f96ed28b2dc5717ff54d20d.json',1),(102283,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',14),(102292,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(102299,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',4),(102305,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',71),(102321,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',51),(102328,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',3),(102333,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',9),(102361,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102369,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102371,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(102387,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(102396,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102399,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102413,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',7),(102427,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',4),(102435,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',65),(102445,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',11),(102492,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102493,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(102533,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102578,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102579,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',4),(102687,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(102715,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',5),(102716,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',5),(102724,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',4),(102732,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102777,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102778,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102781,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',3),(102839,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(102861,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102937,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(102991,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(103037,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(103115,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(103163,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(103288,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(103299,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(103376,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(103383,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(103427,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',16),(103429,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',17),(103462,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(103466,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',10),(103494,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(103507,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(103508,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',13),(103521,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',3),(103564,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',3),(103573,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',7),(103753,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',4),(103781,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(103843,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(103951,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104131,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104132,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104133,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104143,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104146,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104172,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104184,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104192,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104199,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104216,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104240,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104262,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104267,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104268,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104279,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104290,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104294,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104295,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104296,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104302,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104313,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104314,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104316,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',4),(104333,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104343,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104347,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104360,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104367,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104383,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104384,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104399,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104405,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104408,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104421,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',9),(104422,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',3),(104441,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104446,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104447,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104448,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104453,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104454,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104458,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104469,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104479,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104481,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104482,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104483,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104507,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104590,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',3),(104607,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104636,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104697,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104699,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104723,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104725,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104732,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104733,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104838,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104842,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(104848,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104963,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(104974,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',4),(105021,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105022,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105032,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105035,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105037,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105058,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105063,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105066,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105071,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105077,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105080,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105095,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105145,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105146,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105152,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105160,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105163,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105171,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105172,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105195,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105203,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105209,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105231,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',3),(105246,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(105254,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105273,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105274,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105282,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105288,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',2),(105303,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105369,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105400,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105401,'ccc36b04ad5c71de61967624f7f739e868d7c0a5.json',1),(105446,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',9),(105451,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105456,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105464,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(105476,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(105484,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',8),(105510,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(105515,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(105530,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',13),(105531,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105539,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105540,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105546,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',12),(105550,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105587,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105590,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105600,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105606,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105607,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(105621,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',14),(105624,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105626,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105629,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105630,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105649,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(105660,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105709,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105731,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',20),(105732,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',35),(105768,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(105769,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(105774,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105778,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(105791,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105796,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105797,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(105827,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105828,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105844,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105849,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',36),(105864,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',5),(105874,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(105941,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105948,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105950,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105968,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(105986,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(105991,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(106008,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',10),(106016,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(106017,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(106024,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106028,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(106052,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106081,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106083,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106084,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106090,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106093,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106098,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106167,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106270,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',5),(106276,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106281,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(106293,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106305,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(106386,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(106467,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',3),(106477,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106543,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106578,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106584,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106590,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106600,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',35),(106617,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(106626,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106635,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106704,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106771,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106877,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106895,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106929,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106937,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(106957,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',2),(107025,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(107166,'c8213638a9602d46f2f7ff6ba62b0df69beb0001.json',1),(107183,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107192,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107194,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',6),(107206,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107209,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',57),(107223,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107227,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107236,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107262,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107276,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107295,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107319,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',4),(107329,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',23),(107340,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107374,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107387,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107399,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107402,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107406,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107411,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107427,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107429,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107431,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107434,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107448,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107449,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107459,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107469,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107478,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107486,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107523,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',3),(107533,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107545,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107572,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107582,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107584,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',3),(107594,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107598,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107613,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107615,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107633,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(107664,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107671,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107707,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107709,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107710,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107716,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107770,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107790,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',5),(107794,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107795,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107812,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107835,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107838,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',4),(107840,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107841,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107857,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107861,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107865,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107869,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107890,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107898,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107909,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107910,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107922,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107937,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107947,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107949,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(107967,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108009,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(108034,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(108047,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',4),(108056,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108061,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108078,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',4),(108084,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108094,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108107,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108112,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108113,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108123,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',5),(108177,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108234,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108235,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108247,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108275,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108283,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108285,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(108307,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108320,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',4),(108329,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108344,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108354,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108357,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108382,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108394,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108398,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108408,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108431,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(108444,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108449,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108473,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(108474,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108489,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108490,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108495,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108499,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108503,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108511,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108532,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108533,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(108591,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108610,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108611,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108634,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108640,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108685,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108709,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108726,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108734,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108766,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108801,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108833,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108836,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',2),(108842,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108865,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108903,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108916,'a1b6c2b3b808e995697f94d9676d5d5c85180177.json',1),(108931,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108935,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(108944,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108945,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108946,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108950,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108952,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108954,'e7f311d3c752063f75f28de85abd5f82eacab413.json',7),(108965,'e7f311d3c752063f75f28de85abd5f82eacab413.json',3),(108973,'e7f311d3c752063f75f28de85abd5f82eacab413.json',10),(108977,'e7f311d3c752063f75f28de85abd5f82eacab413.json',29),(108978,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108991,'e7f311d3c752063f75f28de85abd5f82eacab413.json',4),(108992,'e7f311d3c752063f75f28de85abd5f82eacab413.json',34),(108993,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(108994,'e7f311d3c752063f75f28de85abd5f82eacab413.json',56),(108997,'e7f311d3c752063f75f28de85abd5f82eacab413.json',45),(109005,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109035,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109048,'e7f311d3c752063f75f28de85abd5f82eacab413.json',4),(109057,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109062,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109070,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109091,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109094,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109126,'e7f311d3c752063f75f28de85abd5f82eacab413.json',3),(109132,'e7f311d3c752063f75f28de85abd5f82eacab413.json',6),(109149,'e7f311d3c752063f75f28de85abd5f82eacab413.json',3),(109156,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109159,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109160,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109165,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109172,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109176,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109178,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109198,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109216,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109235,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109238,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109242,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109245,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109251,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109253,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109254,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109265,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109274,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109275,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109276,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109288,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109291,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109300,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109301,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109320,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109321,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109322,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109323,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109327,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109344,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109348,'e7f311d3c752063f75f28de85abd5f82eacab413.json',10),(109366,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109370,'e7f311d3c752063f75f28de85abd5f82eacab413.json',14),(109375,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109377,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109378,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109437,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109438,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109456,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109458,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109459,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109461,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109468,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109469,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109474,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109556,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109649,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(109658,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109691,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109753,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109765,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109826,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109833,'e7f311d3c752063f75f28de85abd5f82eacab413.json',4),(109854,'e7f311d3c752063f75f28de85abd5f82eacab413.json',4),(109890,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109902,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109910,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109927,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109947,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(109969,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(110010,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(110043,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(110066,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(110069,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(110078,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(110124,'e7f311d3c752063f75f28de85abd5f82eacab413.json',2),(110137,'e7f311d3c752063f75f28de85abd5f82eacab413.json',1),(110186,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',5),(110207,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110223,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',2),(110234,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110240,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',9),(110248,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',3),(110273,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110290,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',2),(110355,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110365,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',5),(110396,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110405,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110406,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110491,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110502,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110506,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110517,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110518,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',3),(110533,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110576,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',7),(110579,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',2),(110594,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',5),(110621,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110623,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110632,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110658,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110667,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',2),(110705,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110745,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',20),(110820,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110828,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110896,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110989,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(110996,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',4),(111064,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111110,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111195,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111263,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111264,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111372,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111496,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111641,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111690,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',2),(111735,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111751,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111782,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111832,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111902,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111914,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111916,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',5),(111973,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111992,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(111993,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112025,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112026,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112027,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112059,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112068,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112077,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',2),(112142,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112256,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112263,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',1),(112266,'babad3896fca8ed7c89d9b785c08e7c42a48a6a2.json',2),(112422,'4db3931c4b24ae432d68b7e24f593b1272445523.json',7),(112424,'4db3931c4b24ae432d68b7e24f593b1272445523.json',69),(112451,'4db3931c4b24ae432d68b7e24f593b1272445523.json',6),(112465,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(112482,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(112508,'4db3931c4b24ae432d68b7e24f593b1272445523.json',27),(112570,'4db3931c4b24ae432d68b7e24f593b1272445523.json',4),(112586,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(112589,'4db3931c4b24ae432d68b7e24f593b1272445523.json',45),(112618,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(112626,'4db3931c4b24ae432d68b7e24f593b1272445523.json',5),(112633,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(112708,'4db3931c4b24ae432d68b7e24f593b1272445523.json',3),(112821,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(112873,'4db3931c4b24ae432d68b7e24f593b1272445523.json',12),(112874,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(112879,'4db3931c4b24ae432d68b7e24f593b1272445523.json',3),(112926,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113018,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(113088,'4db3931c4b24ae432d68b7e24f593b1272445523.json',5),(113093,'4db3931c4b24ae432d68b7e24f593b1272445523.json',11),(113105,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113208,'4db3931c4b24ae432d68b7e24f593b1272445523.json',14),(113209,'4db3931c4b24ae432d68b7e24f593b1272445523.json',19),(113309,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113403,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113480,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113602,'4db3931c4b24ae432d68b7e24f593b1272445523.json',10),(113669,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113670,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113671,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113672,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113673,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(113674,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113675,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(113676,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113677,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113678,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113687,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(113688,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113690,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113691,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113692,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113737,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113767,'4db3931c4b24ae432d68b7e24f593b1272445523.json',8),(113768,'4db3931c4b24ae432d68b7e24f593b1272445523.json',34),(113797,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113824,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113832,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(113833,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(113857,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(113913,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113918,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113919,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113961,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(113980,'4db3931c4b24ae432d68b7e24f593b1272445523.json',2),(114026,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114036,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114044,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114045,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114097,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114099,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114100,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114102,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114120,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114122,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114153,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114204,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114207,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114214,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114216,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114252,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114298,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114311,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114313,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114319,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114320,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114323,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114394,'4db3931c4b24ae432d68b7e24f593b1272445523.json',1),(114450,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(114470,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114484,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114497,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',4),(114507,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',59),(114548,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(114557,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(114563,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(114577,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',4),(114642,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114645,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114646,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',10),(114668,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114684,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114842,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114885,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(114941,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(114957,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115115,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115220,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',7),(115237,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',4),(115264,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115353,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(115356,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(115429,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115443,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115448,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115453,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115455,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115466,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115482,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(115564,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',5),(115641,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(115645,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(115670,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115694,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115712,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(115739,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115742,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115773,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115782,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(115796,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115803,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115805,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115821,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115839,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115863,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(115981,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116047,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116088,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116113,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(116181,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',4),(116204,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116251,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116296,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116299,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116422,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116432,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(116511,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116520,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116532,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116538,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116604,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116606,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116610,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116611,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116635,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116734,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',77),(116755,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116875,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116911,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116912,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(116919,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(116951,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117006,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117007,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117091,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117186,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117207,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117229,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117298,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117331,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117338,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117377,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(117384,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117395,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117426,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117449,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117500,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117518,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',4),(117531,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117532,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117681,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117707,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',8),(117721,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(117722,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(117738,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117751,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117790,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117791,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117817,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117838,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117860,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(117877,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117892,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(117903,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117904,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',2),(117915,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',3),(117917,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',4),(117945,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117971,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117973,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117985,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117990,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(117995,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(118028,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(118042,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',1),(118069,'40500cd7ae5b4e116e8b13e5408e7dfd96d43ab4.json',6),(118226,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',7),(118289,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',3),(118310,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118318,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118380,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118381,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118388,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118389,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118391,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118432,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118440,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',2),(118453,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118454,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',2),(118474,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118550,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118607,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',15),(118608,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118662,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118687,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',12),(118693,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118708,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118723,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118725,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118726,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118727,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118734,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118735,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',3),(118736,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',2),(118739,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',3),(118772,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(118804,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',2),(118860,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',3),(118950,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119028,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',3),(119065,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119077,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',3),(119078,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119349,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119364,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119448,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119488,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',4),(119540,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119542,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119556,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119617,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119625,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119626,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119650,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',19),(119812,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119813,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119823,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119828,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',2),(119901,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119903,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119907,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',2),(119970,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(119983,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(120010,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(120066,'0dcf7336a3569d4103dc5be47ddd6b583dea98f0.json',1),(120178,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120182,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120183,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120195,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120206,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120207,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120225,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120249,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120278,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120321,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120347,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120368,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120420,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120421,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120423,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',9),(120448,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120449,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',5),(120455,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120464,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120470,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120491,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',4),(120567,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120625,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(120626,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120627,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120642,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',4),(120644,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120656,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(120694,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120754,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120764,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(120770,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(120780,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(120781,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(120782,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120788,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120792,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120806,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120814,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120817,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120854,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120887,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120891,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120894,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120905,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(120908,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(120980,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',3),(121012,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121038,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121041,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',11),(121042,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',13),(121067,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121095,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121178,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',34),(121184,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121204,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121245,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(121246,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121387,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121391,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(121394,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121413,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121419,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121465,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121572,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121612,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121639,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121665,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121753,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(121761,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121788,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',2),(121812,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121813,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121814,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121818,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121822,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121849,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121862,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121891,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121937,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121960,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121998,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(121999,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(122000,'6cfcbd2a102a3666d0d168f9bdaba0b2107fc8bd.json',1),(122024,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',4),(122052,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',7),(122059,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',58),(122072,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(122104,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122145,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122148,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122188,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122196,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122203,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122221,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122222,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(122241,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122242,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(122246,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122254,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122276,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122279,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122300,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',4),(122302,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122303,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122306,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122309,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',4),(122321,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122327,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(122334,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(122338,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',12),(122347,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122350,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122352,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122354,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122371,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122372,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122424,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122430,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122431,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(122438,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122464,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(122465,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(122468,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122472,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',5),(122477,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(122480,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(122567,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122569,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122580,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122595,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122616,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122627,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(122630,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(122631,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122632,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122702,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122714,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122725,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122739,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122742,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122749,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122757,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122759,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122766,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122777,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122829,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122847,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',7),(122860,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122862,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122916,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(122921,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122956,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',7),(122986,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(122987,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',7),(122988,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',4),(122995,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(123051,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123056,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(123080,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123139,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123181,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123208,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123224,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123227,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123231,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123247,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123249,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(123286,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123327,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123333,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(123337,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123356,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123371,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(123374,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123375,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123376,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123377,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123384,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(123394,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123399,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(123404,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123407,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123411,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123420,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123438,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123447,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123453,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123465,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123468,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123471,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123472,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123473,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123490,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123491,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123493,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123494,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',8),(123509,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123514,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123516,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123523,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123527,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123533,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123543,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123544,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123547,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123548,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',9),(123550,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123552,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123553,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123557,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123565,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(123630,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123674,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123676,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123679,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123694,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123728,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123734,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',4),(123757,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123852,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123872,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123873,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(123878,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123892,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123893,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123910,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123945,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123952,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123955,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123969,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(123986,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',5),(123987,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(123998,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124005,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124010,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124019,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124031,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124033,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124037,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124044,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124046,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124071,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(124077,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124084,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124098,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(124129,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',7),(124131,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124132,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124147,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124150,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',6),(124157,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(124159,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124163,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124164,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124165,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124168,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(124178,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',4),(124185,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(124208,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124251,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124263,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124320,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124325,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(124352,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124374,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124389,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(124401,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124406,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',2),(124430,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',3),(124432,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124445,'575e5af469e1a43638c3dac9fdafda882fbe9d7a.json',1),(124454,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124457,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',20),(124474,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124475,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',10),(124479,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124480,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124481,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124483,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124484,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124490,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124491,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',40),(124493,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124498,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124512,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124521,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',4),(124523,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',32),(124528,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124537,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124539,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124541,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(124555,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124557,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124560,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124561,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124580,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124581,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124586,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(124596,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',26),(124598,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124599,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(124600,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124601,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',4),(124608,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(124611,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(124618,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(124674,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124694,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124713,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',23),(124776,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(124777,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(124790,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124806,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(124837,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124852,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124855,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124867,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124868,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(124895,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(124912,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124914,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124918,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124947,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(124975,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(124995,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125031,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(125033,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125034,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',4),(125087,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125089,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125127,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125130,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(125131,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125134,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125135,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125138,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125141,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125144,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125147,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125150,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125153,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125157,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125161,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125164,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125183,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125203,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125214,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125215,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125220,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125223,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125262,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(125291,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125345,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125395,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(125410,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125424,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',25),(125477,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(125498,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125534,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125539,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125639,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125825,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125838,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',3),(125839,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(125932,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(126071,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(126127,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',1),(126200,'6bbbd25c11b320bdd509a9c76417ed4045349708.json',2),(126286,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126290,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126308,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',5),(126331,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(126362,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126462,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126483,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(126484,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',9),(126540,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126597,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126608,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126612,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126614,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126628,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126632,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126640,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126647,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126723,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126810,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126864,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126869,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',16),(126927,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(126942,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126964,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126979,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126986,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(126988,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(126989,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127050,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(127075,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(127129,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',57),(127145,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127154,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127167,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127168,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127222,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127264,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',24),(127309,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',18),(127336,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',163),(127344,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(127348,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',71),(127355,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127361,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127504,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127524,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',28),(127532,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127549,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',7),(127766,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127794,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(127795,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(127914,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(127925,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127941,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(127989,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128234,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128244,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128266,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128288,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128315,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128316,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128360,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',5),(128386,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128387,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128388,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128390,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(128438,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128617,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(128705,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128718,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128752,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128799,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128814,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128827,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128880,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128882,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128885,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128888,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128912,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(128918,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128936,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(128937,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129009,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129010,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129011,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(129015,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129021,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',5),(129034,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129035,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129065,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(129154,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129208,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',5),(129210,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',4),(129214,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(129218,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',22),(129227,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129260,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',4),(129263,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129271,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129435,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129517,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129518,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',20),(129591,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(129613,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(129737,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',22),(129753,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(129761,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',9),(129796,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129823,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',5),(129896,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(129940,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(129957,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',2),(130019,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',16),(130108,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',4),(130114,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130180,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130249,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130255,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130333,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130335,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',23),(130339,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130411,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',3),(130421,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130450,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130493,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130578,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130629,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130845,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130865,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130866,'4c84dbfd01f7b2009ebed54376da8afcbcf1ec64.json',1),(130941,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(130949,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131000,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(131022,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131034,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131039,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(131050,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131085,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',3),(131103,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131114,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131147,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131150,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(131170,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131174,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131175,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131186,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131210,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131218,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',58),(131224,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(131231,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131234,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',10),(131251,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',64),(131255,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',43),(131278,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131279,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131380,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131389,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(131400,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131405,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131506,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131512,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131580,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131616,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',71),(131617,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',8),(131628,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131634,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131713,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131726,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131766,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131767,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131777,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(131798,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131812,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131820,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131824,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131843,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131911,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131916,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131938,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131944,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(131953,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132003,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132032,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132036,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132037,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132088,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132092,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132099,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132115,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132149,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132186,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132205,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',4),(132206,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132216,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132225,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132234,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132242,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132362,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132418,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132452,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132482,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',5),(132514,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132525,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',4),(132529,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',5),(132568,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132572,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132581,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132624,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132640,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132649,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132652,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132653,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',3),(132678,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',141),(132718,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132740,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132741,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132746,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',4),(132754,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132761,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',15),(132762,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',10),(132812,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132815,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132818,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132833,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132836,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',6),(132927,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132928,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',3),(132952,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(132982,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(132995,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133022,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(133041,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(133058,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133097,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133108,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133123,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(133162,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133191,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133212,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133257,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133258,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133287,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(133293,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133303,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133319,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133326,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133384,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133401,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133407,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133447,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133524,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',2),(133525,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133526,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133528,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133531,'f4d18f2beafa875cd4e30a9a237a64b9d3110856.json',1),(133542,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',13),(133546,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133549,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(133550,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133551,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(133552,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',51),(133572,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',8),(133586,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133589,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',3),(133601,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133603,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133621,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133625,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133627,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',6),(133642,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',40),(133681,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133684,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133729,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',31),(133730,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(133731,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',3),(133732,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',39),(133750,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',25),(133766,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(133793,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133808,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133835,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133852,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133853,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133861,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(133864,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133884,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133900,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',23),(133902,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(133905,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133918,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(133926,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133935,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(133951,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133952,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133956,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133958,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133960,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(133964,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(134028,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134056,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134090,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134096,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134108,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',3),(134111,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',7),(134130,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134171,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134179,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134180,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134229,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134230,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134231,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134257,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134258,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(134259,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134356,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134358,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134359,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134360,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',8),(134505,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(134624,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134625,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134648,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134649,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(134650,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134651,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',7),(134652,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(134667,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(134669,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134704,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',7),(134740,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',8),(134743,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',21),(134795,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(134797,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(134798,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(134852,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(134890,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(134968,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(135048,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',35),(135049,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',35),(135110,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135111,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',7),(135112,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',3),(135131,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135144,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135162,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135194,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135195,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135196,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135197,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135198,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135200,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135211,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135248,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135250,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135260,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135262,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135269,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135320,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135324,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135331,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135337,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',3),(135353,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135361,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135368,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135379,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(135393,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135410,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',6),(135416,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135451,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',5),(135460,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',21),(135461,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',73),(135614,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135619,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135637,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135645,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135646,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135653,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135654,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135666,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135676,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135757,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135826,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135838,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135840,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135858,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135860,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135861,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(135908,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135918,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135930,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135956,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(135981,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136019,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136049,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136062,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(136073,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',4),(136076,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',10),(136095,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(136104,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136158,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136216,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136219,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136223,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(136262,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',3),(136269,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136270,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(136316,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',2),(136432,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136446,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136454,'94be6e5f47cbef1a762c2cb3c1f37d4862b53403.json',1),(136482,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(136486,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136517,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136518,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(136534,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136540,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136546,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136548,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',46),(136555,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(136587,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136588,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136593,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136598,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',18),(136610,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(136618,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',55),(136619,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',50),(136641,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136650,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136651,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136652,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136653,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136654,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136655,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136656,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136664,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(136666,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136682,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(136694,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(136751,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136756,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(136759,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136772,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136775,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136784,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136786,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136787,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136808,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136829,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(136833,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136835,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136836,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136850,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136909,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(136913,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136919,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(136928,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137024,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137028,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137029,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137033,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137168,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137191,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(137199,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137227,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137230,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137243,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137249,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137308,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(137325,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137334,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',6),(137365,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137367,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137436,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137441,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137508,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137594,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137606,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(137619,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137620,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137683,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137770,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137775,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137784,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137870,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137890,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(137960,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137969,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(137975,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(137976,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137985,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(137989,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138024,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138026,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138028,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138048,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138152,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138174,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',6),(138261,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138270,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138286,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(138292,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138314,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138322,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(138342,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138343,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',6),(138349,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(138362,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(138386,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138443,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(138467,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(138479,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138485,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138547,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',13),(138616,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138626,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(138628,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138651,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138665,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138666,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(138709,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138754,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(138778,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138797,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138817,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(138948,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138970,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138981,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138986,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138988,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(138998,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139015,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139042,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139054,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139088,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139119,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',20),(139120,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',20),(139145,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139149,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139173,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139192,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139205,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',17),(139206,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',16),(139240,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(139244,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(139245,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139249,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139253,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139256,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139258,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139268,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139269,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(139275,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(139280,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139282,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139284,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139290,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139317,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139322,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139327,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(139341,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139345,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139349,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139363,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139376,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139390,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(139393,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139399,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',3),(139487,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139489,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139527,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139540,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',4),(139541,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139672,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139677,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',8),(139726,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139744,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139794,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139799,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139820,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(139851,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139855,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139857,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(139947,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',30),(139981,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(140005,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(140037,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(140058,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(140066,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(140242,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(140243,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',1),(140283,'61d9a0fe39f4e845c44a06787de6f5f033b998a3.json',2),(140346,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140366,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140385,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140394,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140421,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',4),(140458,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',8),(140459,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',12),(140465,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(140478,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',6),(140483,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(140484,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140486,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',5),(140517,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140567,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',4),(140575,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',21),(140582,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',8),(140589,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',11),(140601,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(140611,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',14),(140628,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',18),(140656,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140657,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140693,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140694,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140706,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140707,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140729,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',12),(140740,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(140762,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140791,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140803,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140809,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140810,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140813,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140844,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140848,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140871,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140872,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',12),(140883,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',28),(140891,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140894,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140912,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(140913,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140916,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140921,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140925,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140933,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140942,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140953,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140955,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140957,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(140964,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141005,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141014,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141025,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141028,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141057,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141058,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141061,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',7),(141108,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141110,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(141140,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141164,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',4),(141180,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',4),(141183,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',10),(141192,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141209,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141213,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141231,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141232,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',4),(141256,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141258,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141271,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141383,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141544,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141547,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',9),(141562,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(141584,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141591,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141624,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141632,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141633,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141634,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141639,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141646,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141658,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141660,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',5),(141661,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',4),(141662,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141682,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141685,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141686,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141697,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141698,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(141704,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141723,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141727,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141729,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141733,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141735,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141737,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141744,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(141760,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141766,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141770,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141772,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141773,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141776,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(141969,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',8),(142054,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(142087,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',2),(142215,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(142444,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(142506,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(142527,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',3),(142606,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(142617,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(142640,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',7),(142643,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',6),(142671,'315e5980d6fdce4553a9df8c0e36e55ffed32365.json',1),(142778,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(142799,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(142803,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(142824,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(142939,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',2),(143022,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143026,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',2),(143042,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143043,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143049,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143197,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143258,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143261,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',4),(143262,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',2),(143263,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143265,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143266,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143269,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',6),(143270,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143271,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',5),(143272,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143273,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143274,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',3),(143275,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143276,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143309,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143315,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143319,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143392,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143395,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143434,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',3),(143670,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143771,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(143783,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',2),(144036,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(144046,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(144052,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(144079,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(144161,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(144473,'c1ad13d83e926979dbf2bbe52e4944082f28dfea.json',1),(144489,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144490,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144502,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144528,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144572,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',3),(144662,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144679,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(144680,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144718,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144738,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144740,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',3),(144758,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144760,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',23),(144762,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144763,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144764,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144810,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',3),(144818,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144851,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',33),(144852,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',66),(144853,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(144854,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144856,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(144928,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(144950,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144979,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(144980,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',44),(145026,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145043,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145044,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145048,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145056,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',3),(145069,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145077,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145127,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145128,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145143,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145166,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',31),(145167,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',4),(145170,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145252,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145259,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145281,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',6),(145283,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145284,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145289,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145302,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145305,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',4),(145308,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145312,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145313,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145314,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145318,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145320,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145326,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145330,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145334,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145354,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145355,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145358,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145372,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',3),(145387,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145401,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145455,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145486,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145511,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145528,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145535,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145536,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145548,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145552,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',8),(145554,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',10),(145567,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145606,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145609,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145620,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145621,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145622,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145623,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145627,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145628,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145629,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145630,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145666,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145756,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(145802,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',3),(145867,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145868,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(145965,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146043,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(146092,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',3),(146093,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146107,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146109,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146169,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146179,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146260,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(146412,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146460,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',5),(146469,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146664,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146667,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146678,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146779,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146785,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146803,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146823,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146846,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146927,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146930,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146931,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',2),(146986,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(146987,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147171,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147211,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147240,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147297,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147331,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147332,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147382,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147383,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147400,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147405,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147406,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147415,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147447,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147453,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147498,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147499,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147500,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147501,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147514,'25fd0362dfe0a62ca9ee6f9f634e24ef9a527dfc.json',1),(147530,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(147533,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147570,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(147574,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147582,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',3),(147583,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',3),(147584,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(147602,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147631,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147668,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147709,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147712,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147720,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147727,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147731,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147733,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147734,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147735,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147744,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147756,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147763,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147766,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(147831,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147839,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147852,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147895,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147913,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(147917,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147918,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147919,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147920,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(147923,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',4),(147936,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(147950,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',5),(147960,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',4),(148007,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148015,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',11),(148021,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148024,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148053,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148055,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148071,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148115,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148126,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148146,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148150,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148169,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148193,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',3),(148194,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148205,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148213,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',4),(148248,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',7),(148267,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148297,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148319,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148365,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148372,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148375,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',8),(148391,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148393,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148394,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148426,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148447,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148507,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148553,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148556,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148557,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148635,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148648,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148720,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148727,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148728,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148742,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148769,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148771,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148773,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148781,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148799,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148800,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148801,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148839,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148840,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148884,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(148888,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',7),(148902,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148905,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',3),(148912,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148921,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148954,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148958,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148963,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(148987,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149010,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149018,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149019,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',3),(149020,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',3),(149081,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',2),(149098,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149102,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149103,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149104,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149117,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149124,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149128,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149144,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149151,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149174,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149206,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149238,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149240,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149242,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149244,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149245,'d230ca9dcc54e1d5c0ed4503752774eb44711dff.json',1),(149253,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',1),(149285,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',1),(149296,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',24),(149303,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',34),(149346,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',1),(149398,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',6),(149411,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',3),(149412,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',6),(149418,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',5),(149459,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',47),(149461,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',3),(149462,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',2),(149490,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',4),(149514,'bdcfff55eaabeaa26d71810113d717ef85e417d1.json',1),(300679,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300684,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300685,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300689,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',47),(300698,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300725,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',61),(300730,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',38),(300731,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',60),(300740,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',5),(300755,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',15),(300764,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300777,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(300778,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',90),(300783,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',5),(300803,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',6),(300808,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',7),(300813,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',11),(300875,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',82),(300879,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',6),(300890,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',33),(300891,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',34),(300947,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(300949,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300954,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300966,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(300986,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301002,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301003,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301004,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',7),(301020,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301021,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301022,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301024,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301031,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301032,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301037,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301057,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301061,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301065,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301073,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301074,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(301083,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301085,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',7),(301087,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301088,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301092,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',13),(301095,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301099,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(301103,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301105,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(301113,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301114,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301121,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301122,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301125,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301141,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301142,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301144,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301147,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301149,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301151,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301152,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301153,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301157,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301159,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301166,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301171,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301172,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301185,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',7),(301189,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301190,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301192,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301194,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301197,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301220,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301221,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301223,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301224,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301235,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(301245,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301247,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301253,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301254,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301271,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',4),(301272,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301273,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301275,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301280,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301281,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(301287,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301289,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301290,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301340,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',116),(301348,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301355,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301361,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301374,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301383,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',8),(301406,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301422,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',13),(301458,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301563,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301593,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(301693,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(301712,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',5),(301717,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(301750,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301793,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301833,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301874,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301987,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(301989,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302192,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',15),(302442,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302525,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(302530,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',14),(302585,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302883,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302898,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302917,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302919,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',4),(302953,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302977,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302985,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302987,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(302988,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303033,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',5),(303036,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303040,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(303042,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',3),(303087,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303105,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303106,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303107,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(303130,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',2),(303131,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',4),(303143,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303144,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303179,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303205,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303211,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',6),(303295,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303345,'d90d787828b6f96b4a93eb6948d613293bc51bf9.json',1),(303352,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303354,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(303369,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(303474,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303483,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',18),(303488,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(303490,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',13),(303491,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303492,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(303505,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303526,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303553,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(303555,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303569,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303602,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',4),(303603,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303604,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303612,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303620,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',3),(303691,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',3),(303711,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',13),(303749,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',5),(303751,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',4),(303755,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303758,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303759,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(303760,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303761,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303777,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303778,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303786,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(303794,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303796,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303805,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303808,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303810,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303823,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',33),(303855,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303866,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303873,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303874,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303894,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303912,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303913,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303922,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',19),(303926,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',8),(303941,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',6),(303952,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303953,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303965,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',4),(303972,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(303976,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304006,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(304046,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304062,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1003),(304075,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',3),(304082,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(304122,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',3),(304237,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',3),(304240,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',4),(304241,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',14),(304242,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',4),(304265,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304375,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304410,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304435,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304483,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',4),(304499,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304580,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304599,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304663,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304680,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(304811,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304919,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304946,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(304954,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(304972,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',3),(304984,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',2),(304990,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305002,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305022,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305060,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305074,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305087,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305099,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305100,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305193,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305209,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305214,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305225,'ad9ac0ac5e7da097253fd545b56e2b15ee9de34f.json',1),(305265,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305277,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',5),(305311,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305387,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(305390,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305424,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',8),(305444,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',38),(305484,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305495,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',6),(305540,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305597,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305599,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305603,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(305627,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305632,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',5),(305639,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305664,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305684,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305685,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',9),(305686,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',17),(305703,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305707,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(305713,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305714,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305717,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305723,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',3),(305724,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305725,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305726,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(305740,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305760,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305769,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305779,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305801,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305803,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305804,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305809,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305810,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305812,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305815,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305817,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305820,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',5),(305821,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305824,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305825,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305842,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305845,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305848,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305849,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305851,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305853,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(305854,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305857,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305865,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305867,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305870,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305877,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305880,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305882,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(305889,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305891,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305914,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305915,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305950,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305954,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(305989,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306050,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306126,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306146,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(306236,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306239,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306257,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306305,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306416,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(306426,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306539,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306540,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306559,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306624,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306688,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306732,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',3),(306742,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306753,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',11),(306782,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306803,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(306829,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306835,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306836,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306869,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306950,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(306961,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(306995,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',1),(307068,'52566dccb4bd8044edc87b1a0aa268320a6ea3d4.json',2),(307119,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',5),(307130,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',7),(307217,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',20),(307251,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307262,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307264,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307298,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307317,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307373,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307388,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307406,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307418,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307460,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307463,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307477,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307486,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307510,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307513,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307517,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307526,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307534,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',4),(307538,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307542,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307550,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307569,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307575,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307590,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307593,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307621,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',5),(307625,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307628,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307681,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307683,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',5),(307697,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307699,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',7),(307715,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307744,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307764,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307769,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307776,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307777,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',7),(307781,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',7),(307800,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307849,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307850,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',5),(307866,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(307873,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',5),(307891,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',8),(307921,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307935,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(307944,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(307951,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308051,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308109,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308139,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308144,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308155,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308194,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308228,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308236,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(308325,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308330,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',57),(308331,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',9),(308334,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(308337,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',4),(308338,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',4),(308342,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308376,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308477,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308495,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308563,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308575,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308656,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308663,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308669,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308751,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308756,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308801,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308827,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308882,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308888,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(308950,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',7),(308971,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(308980,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309008,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309018,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309074,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309077,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309083,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',4),(309085,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309149,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309153,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309154,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309156,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309201,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(309206,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309338,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(309374,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(309412,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309417,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309441,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309490,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309507,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309511,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(309584,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(309594,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309622,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309628,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309654,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309723,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(309736,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(309765,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(309781,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309787,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309793,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309826,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309889,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309895,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(309922,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(310062,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310121,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310125,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310174,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310199,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(310200,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',3),(310283,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310297,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',2),(310325,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310384,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310414,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310500,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310519,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310558,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310564,'12de3563e18c3c6c5943fa8216e0eea970a590cd.json',1),(310704,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(310716,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(310720,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(310736,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(310743,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(310745,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',7),(310746,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(310753,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(310776,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(310784,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(310793,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',54),(310835,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(310864,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(310897,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',3),(310902,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(310928,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(310940,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',3),(310966,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',4),(311016,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',14),(311036,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311158,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311171,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311176,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311191,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311222,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311396,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311402,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311431,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311457,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311462,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311470,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311471,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(311495,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311500,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311526,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311530,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',3),(311531,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',3),(311613,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311619,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',4),(311620,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(311639,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311712,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311844,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311941,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311960,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(311988,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(312111,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',2),(312113,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(312256,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(312294,'c0f8a4eef924cdeeeb5b17ae685760bf7cce9223.json',1),(312377,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',9),(312403,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(312411,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',8),(312430,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312498,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312504,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312505,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312534,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312536,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312562,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312580,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312668,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(312670,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',4),(312682,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(312688,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312689,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(312697,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(312699,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(312717,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312719,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312725,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312729,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312731,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312740,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312750,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312796,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312822,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',4),(312837,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',4),(312850,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312852,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312853,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312854,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312855,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312856,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312857,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312858,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312859,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312860,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312861,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312862,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312863,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312864,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312865,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312866,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312961,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(312972,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313024,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313046,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313093,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313108,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',3),(313150,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(313160,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313202,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',3),(313205,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',3),(313206,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(313207,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(313210,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(313256,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313351,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(313359,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313388,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',4),(313407,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',2),(313411,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313464,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',4),(313539,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313543,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313618,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313627,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313710,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',18),(313725,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313732,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313753,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313758,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313783,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',3),(313816,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313831,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313890,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313913,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313935,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313954,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313959,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313973,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313976,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313978,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313984,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(313986,'c64f525d77bf8b2ae4439a875f1bd8eb197a0519.json',1),(314046,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',2),(314075,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',19),(314102,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(314209,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',4),(314213,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(314307,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(314322,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',6),(314325,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',6),(314351,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(314380,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(314424,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',87),(314485,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(314542,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',4),(314768,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',9),(314870,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',4),(314993,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',6),(314997,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(315523,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(315626,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(315820,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316055,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316059,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316081,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',2),(316083,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316089,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316091,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316102,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316131,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316180,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316221,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316256,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316302,'6af02873a430f5f75dc56a917f26feaea89b8e79.json',1),(316308,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316362,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316409,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',3),(316418,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',5),(316475,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(316511,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',50),(316516,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',6),(316591,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316593,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',3),(316616,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316640,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316686,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316692,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316902,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(316921,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317002,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',7),(317010,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',4),(317060,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',7),(317109,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',10),(317167,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(317203,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(317260,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317269,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',29),(317291,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(317296,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317343,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317424,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317442,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317443,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(317468,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317469,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317502,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(317713,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(317751,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',8),(317753,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317758,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317823,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',2),(317830,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317848,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317893,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',3),(317973,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317974,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(317988,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318174,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318221,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318229,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318469,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318492,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318509,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318555,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318743,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318756,'9b7a0ad7b6c7f59e7a6cf1dc9d07912a273d19b5.json',1),(318837,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(318838,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(318841,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(318861,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',5),(318878,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(318933,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',36),(318941,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(318944,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(318945,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',22),(318960,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',3),(318987,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',5),(319036,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',4),(319088,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',13),(319096,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',4),(319098,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(319099,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319110,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',5),(319113,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319146,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',19),(319163,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(319164,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',7),(319230,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(319279,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(319345,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319381,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319395,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(319413,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319426,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319654,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319655,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319671,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319727,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319739,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319748,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319884,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(319998,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320090,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320121,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320127,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320133,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(320140,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320143,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320154,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320167,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320179,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320184,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320185,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320186,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320189,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320190,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320215,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(320267,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320293,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',2),(320295,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320308,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320309,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320340,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320402,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320450,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320458,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320459,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320503,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320505,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320507,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320508,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320514,'b6353f8b0fcd86c2fd1e6f27c9b18a3ccc07980b.json',1),(320640,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(320642,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(320643,'53dbd21da3e3a026ab15e946639b45970d6304de.json',18),(320645,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(320646,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(320748,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(320828,'53dbd21da3e3a026ab15e946639b45970d6304de.json',6),(320829,'53dbd21da3e3a026ab15e946639b45970d6304de.json',9),(320860,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(320876,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(320920,'53dbd21da3e3a026ab15e946639b45970d6304de.json',2),(320923,'53dbd21da3e3a026ab15e946639b45970d6304de.json',2),(320976,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321022,'53dbd21da3e3a026ab15e946639b45970d6304de.json',4),(321056,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321082,'53dbd21da3e3a026ab15e946639b45970d6304de.json',2),(321089,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321118,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321124,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321157,'53dbd21da3e3a026ab15e946639b45970d6304de.json',5),(321158,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321167,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321169,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321171,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321195,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321268,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321270,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321324,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321337,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321437,'53dbd21da3e3a026ab15e946639b45970d6304de.json',2),(321474,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321488,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321674,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321688,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321693,'53dbd21da3e3a026ab15e946639b45970d6304de.json',2),(321698,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321700,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321776,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321785,'53dbd21da3e3a026ab15e946639b45970d6304de.json',1),(321820,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',10),(321835,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',20),(321864,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(321865,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(321897,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(321914,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(321996,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322039,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322048,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322066,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(322128,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322177,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322226,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322271,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322273,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',3),(322315,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',89),(322325,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',42),(322333,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322341,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',5),(322358,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',4),(322368,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(322374,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322376,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',6),(322391,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322394,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322401,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322425,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322432,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322434,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322453,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322458,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',3),(322488,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322497,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',4),(322506,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',5),(322566,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322574,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',10),(322606,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(322643,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(322748,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',3),(322783,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',4),(322839,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',3),(322865,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(322897,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(322909,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323020,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(323046,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323106,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323109,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323112,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',3),(323205,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',15),(323238,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323239,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323281,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323316,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(323331,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(323339,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323342,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323386,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323387,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323407,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(323428,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323553,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',10),(323572,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(323637,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',2),(323711,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',3),(324004,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324061,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324073,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324098,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324112,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',15),(324113,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',4),(324128,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324265,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324276,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324293,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324306,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324307,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324311,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324425,'9795fe49f827ff0654ce919cd8b7450e8a2b40d8.json',1),(324543,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(324551,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',29),(324554,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324577,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',42),(324582,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324586,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324589,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324603,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324622,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(324628,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(324636,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324682,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',9),(324686,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324708,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',5),(324714,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324715,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',6),(324716,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324733,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324734,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324735,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324748,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324749,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324751,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(324757,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324758,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(324759,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324762,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(324804,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',15),(324850,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324861,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',8),(324864,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324872,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324875,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324888,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324898,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324917,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(324956,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324958,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(324969,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324973,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324976,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',5),(324977,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324978,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(324979,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(324982,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(324991,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324998,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(324999,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325004,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325014,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325015,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325019,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325038,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325039,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325040,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325047,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',5),(325070,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325081,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325110,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(325122,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325132,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325141,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325200,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325214,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(325218,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325221,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325233,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325236,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(325262,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325263,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325279,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325317,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(325369,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325418,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325442,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325466,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325472,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325506,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325511,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(325520,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(325531,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325544,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325549,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325559,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325597,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325604,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325692,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325730,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325731,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',6),(325744,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325770,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(325773,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',7),(325784,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325788,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325802,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325811,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325812,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325815,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325816,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325818,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325828,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325834,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325841,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(325891,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325919,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325924,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(325925,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(325928,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326009,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326034,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326064,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326116,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326126,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326161,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326202,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326221,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326225,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',4),(326240,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326264,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326275,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(326276,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326279,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326281,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326287,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326340,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326346,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326369,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326375,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326378,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326379,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326385,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326398,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(326399,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',3),(326405,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326414,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326433,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326438,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326440,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(326443,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326457,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326460,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326483,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326527,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326543,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326544,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',2),(326557,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326572,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326582,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326586,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326600,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326618,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326645,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326650,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326659,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326661,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326665,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326688,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326695,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326716,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326724,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326745,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326752,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326824,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326825,'b3ab47699907f78c2ac478d06f682f56fa0cf351.json',1),(326886,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',5),(326939,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(326942,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327061,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327160,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',2),(327163,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',15),(327182,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327203,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',2),(327289,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',32),(327385,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',3),(327395,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',2),(327439,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',3),(327464,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',3),(327500,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',4),(327531,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327563,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',17),(327589,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327594,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327618,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',7),(327639,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327685,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',5),(327708,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(327836,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',5),(328194,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328197,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',4),(328212,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',2),(328217,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328430,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',2),(328468,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328587,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328602,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328610,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328638,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328674,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328716,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328717,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328751,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328775,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(328977,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(329119,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(329222,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(329243,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',2),(329262,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(329484,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(329486,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',1),(329579,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',3),(329608,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',24),(329654,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',19),(329655,'c34dcfec2a5e68f31f6a0401d2cd636021d984bc.json',4),(329676,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',31),(329696,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(329719,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',10),(329729,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(329740,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',36),(329745,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(329805,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',3),(329814,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(329828,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',9),(329834,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(329854,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(329870,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(329929,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',5),(329933,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(329949,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(329952,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',4),(329992,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',4),(330053,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330101,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330159,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330184,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330261,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',5),(330291,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',22),(330300,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',5),(330358,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',3),(330359,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(330386,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',9),(330405,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(330406,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',13),(330462,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',3),(330501,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',3),(330508,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330517,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330549,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(330573,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',6),(330629,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330638,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(330653,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330670,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',4),(330681,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330698,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330701,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330702,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(330752,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',7),(330866,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330881,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(330982,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',4),(331338,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(331534,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331546,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331572,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',3),(331575,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(331659,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331677,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331698,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331704,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',2),(331783,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331872,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331873,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331876,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331891,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(331908,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332016,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332076,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332188,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332202,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332203,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332208,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332209,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332214,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332215,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332218,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332230,'b0a0f45be3069a32ba190502ab88b6df6d520199.json',1),(332247,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',3),(332256,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',4),(332316,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',2),(332321,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',4),(332329,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332331,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',16),(332344,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',20),(332483,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',2),(332520,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332565,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',3),(332576,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332580,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332596,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332600,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332619,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',4),(332620,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',2),(332638,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',2),(332666,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',5),(332667,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',5),(332683,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',4),(332724,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332781,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332788,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332805,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332806,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332817,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',2),(332818,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332838,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',2),(332848,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332866,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332892,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332897,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332901,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332953,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332975,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332981,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332991,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(332992,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333003,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333005,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333016,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333022,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333032,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',2),(333034,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333037,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333085,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333143,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333144,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333162,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333163,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333180,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333283,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333333,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333645,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333734,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333735,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333750,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333778,'d882f79283751d6c7f2e48cb991ee8abbcd913ae.json',1),(333880,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(333922,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334168,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334206,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334213,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334248,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',4),(334265,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334308,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',8),(334386,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',3),(334434,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334456,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334463,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334468,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334475,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334476,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',4),(334482,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',2),(334530,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334587,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334754,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334808,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334811,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(334972,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335073,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335132,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335138,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335200,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',2),(335338,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',13),(335352,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',5),(335403,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335429,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335465,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335484,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335523,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',2),(335566,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335567,'f3f471d10a36a7a28e9050c10bd4dfd680cba17b.json',1),(335580,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335582,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335586,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335589,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335602,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335610,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335623,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335624,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335638,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335641,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335642,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(335654,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335657,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',74),(335658,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',133),(335659,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335668,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',18),(335670,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335671,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335672,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335681,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335684,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335687,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335688,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335693,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335696,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335699,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335702,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335707,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335724,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(335725,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(335738,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',5),(335770,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335773,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335782,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335810,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335814,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335815,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',5),(335829,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335854,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',7),(335855,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335857,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(335858,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(335867,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(335868,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',22),(335870,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(335884,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335910,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(335937,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',12),(335956,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336040,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336060,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336066,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',7),(336067,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(336092,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',12),(336100,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336120,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336123,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336132,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336144,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336204,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336218,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336238,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336243,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336260,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',9),(336267,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336268,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336269,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336293,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336296,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336298,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336314,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336352,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336416,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',6),(336431,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336442,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(336451,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336456,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336462,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336467,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336477,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',8),(336487,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336541,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336566,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336567,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(336577,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(336578,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336581,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336583,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336604,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336620,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336624,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336642,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(336651,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336658,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336682,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336686,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336696,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336704,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336741,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336743,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336744,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336756,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336757,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336786,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336797,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',5),(336833,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336842,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336874,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336888,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(336889,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(336906,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(336970,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(336990,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(336991,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337015,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337030,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337031,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337058,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337105,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337109,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337128,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337129,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337139,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337158,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337162,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337168,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337169,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337175,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337179,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337190,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337199,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',6),(337208,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337223,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337224,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337236,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337242,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',5),(337249,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337275,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337306,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337321,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337335,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337338,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337347,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337348,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337365,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337378,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337388,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337389,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337398,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337405,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337406,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',5),(337407,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337423,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337424,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337425,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337436,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337437,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337469,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337490,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337491,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',5),(337493,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337498,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337499,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337526,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337574,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(337586,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337623,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337638,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337661,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337675,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337676,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337692,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337704,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337705,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337734,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337737,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',4),(337769,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337770,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337771,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337793,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337809,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337810,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337824,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337828,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337836,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337851,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337852,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337856,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337940,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337954,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337959,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(337960,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(337962,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337963,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(337985,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338076,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338078,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338105,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338126,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338127,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338131,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338138,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338139,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338147,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338150,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338151,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338169,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338189,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338213,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338216,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338238,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338240,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338259,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338268,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338276,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338286,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338287,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(338290,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338331,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338341,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',34),(338342,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338344,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338352,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',3),(338363,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338368,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338376,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338417,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338420,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338433,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338500,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338519,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(338525,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338530,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338546,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338549,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338565,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338578,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338590,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338611,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338638,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',9),(338644,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338645,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338646,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338676,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338681,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338689,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338694,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338695,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338696,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338698,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338732,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338753,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338757,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338777,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338794,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338833,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338870,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338934,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(338957,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(339024,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(339031,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(339032,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',1),(339043,'323ece9056ef7a1d81faf3422e98def1efd38f70.json',2),(339052,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',8),(339053,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',5),(339103,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339107,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339119,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339120,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339121,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',7),(339126,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',10),(339135,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339143,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',12),(339145,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339157,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',9),(339160,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',9),(339161,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',7),(339185,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',6),(339186,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339192,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339243,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',7),(339247,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339262,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339295,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339318,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339350,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339358,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339359,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339360,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339362,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339367,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',3),(339377,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339416,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',3),(339435,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',31),(339444,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',3),(339452,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',3),(339468,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339524,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339550,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339560,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339577,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339594,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339606,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339621,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339622,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339661,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339664,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339694,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339705,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',9),(339709,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339752,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339753,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',7),(339785,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339789,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339872,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',5),(339876,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339878,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339880,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',6),(339889,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339948,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339958,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(339959,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',3),(339971,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(339995,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340000,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340001,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340002,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(340019,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340031,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340057,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340117,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340148,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340157,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(340243,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340273,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340275,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340276,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340297,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340299,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340300,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340301,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340303,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340305,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340306,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340307,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340339,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340343,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',2),(340381,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340383,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340401,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340483,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340497,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',36),(340576,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340580,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340581,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340583,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340632,'305cf1019ca9c81126b2e527fedf79969c2c522f.json',1),(340648,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',66),(340661,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340662,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(340666,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(340668,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',13),(340682,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340691,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340696,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340704,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(340705,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(340707,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',9),(340709,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340710,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340720,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(340746,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340747,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340752,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340756,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',43),(340759,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(340760,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(340761,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(340764,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',23),(340776,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340779,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(340785,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340795,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(340798,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(340820,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340822,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',14),(340825,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340839,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(340842,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340845,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(340855,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340857,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340858,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340859,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340876,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340877,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340892,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',51),(340902,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340920,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(340958,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340959,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340960,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340961,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(340980,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(340984,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',8),(341038,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',7),(341039,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341046,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(341165,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341178,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(341201,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341203,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341239,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(341314,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341317,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341360,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',49),(341364,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(341420,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(341450,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(341451,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',5),(341452,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',6),(341458,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',6),(341515,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341535,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341567,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(341639,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',7),(341683,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(341685,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341691,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',13),(341724,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(341787,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342021,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342037,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',8),(342045,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342062,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342071,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(342072,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342125,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342210,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',114),(342368,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(342374,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342391,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342413,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342607,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342664,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342707,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342753,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342755,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342762,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342857,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',3),(342888,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342922,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(342952,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343171,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343193,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343198,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343203,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343206,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343249,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343289,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343290,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(343332,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343491,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(343494,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343548,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343551,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343589,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343590,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343597,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',6),(343624,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343637,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343656,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343672,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343675,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343756,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343790,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343811,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343815,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343836,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',4),(343909,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(343919,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343968,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343979,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(343983,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343985,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(343988,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343991,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(343998,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344002,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344003,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344023,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344026,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344086,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344087,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344088,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344110,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344175,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344177,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344192,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',2),(344198,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344199,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344215,'6ea2532aacbdcea4609067523e9e72fa8cd73200.json',1),(344252,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(344277,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344288,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344292,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(344310,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',5),(344311,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',5),(344316,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344317,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344318,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344322,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344324,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344327,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',10),(344328,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344343,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344347,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(344361,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',85),(344447,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',7),(344486,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(344521,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344523,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(344527,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',4),(344549,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',30),(344558,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344670,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(344712,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344726,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',10),(344823,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(344971,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',4),(344972,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',5),(344975,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(344976,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',5),(344992,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344993,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344994,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(344995,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345004,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345007,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345008,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(345013,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345014,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345015,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345016,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345018,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345021,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345033,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345200,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(345208,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',4),(345235,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345236,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(345237,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(345269,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345278,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(345314,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345320,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',5),(345346,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345368,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(345433,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345437,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345447,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(345455,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(345503,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345506,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345511,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345561,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(345583,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345586,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345625,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345637,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345638,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345639,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345694,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345697,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345712,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345740,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345789,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345799,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',5),(345816,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(345832,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(345862,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345877,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345886,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345914,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345915,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345921,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345935,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(345936,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(345953,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(346038,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',4),(346068,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(346069,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',4),(346096,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(346144,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(346145,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',3),(346169,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(346229,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(346235,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(346274,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(346275,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',1),(346407,'e148597ac3bd53fe2dd4fa4fc6677502fc8da423.json',2),(346547,'435baf47b4f78fc06a8376f451993059407d2972.json',3),(346559,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(346572,'435baf47b4f78fc06a8376f451993059407d2972.json',13),(346645,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(346650,'435baf47b4f78fc06a8376f451993059407d2972.json',21),(346657,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(346788,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(346867,'435baf47b4f78fc06a8376f451993059407d2972.json',2),(346979,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347056,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347164,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347165,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347169,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347170,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347176,'435baf47b4f78fc06a8376f451993059407d2972.json',2),(347231,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347294,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347319,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347402,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347469,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347647,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347688,'435baf47b4f78fc06a8376f451993059407d2972.json',2),(347769,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347856,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347859,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347864,'435baf47b4f78fc06a8376f451993059407d2972.json',2),(347870,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347886,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347919,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347984,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(347990,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348004,'435baf47b4f78fc06a8376f451993059407d2972.json',3),(348057,'435baf47b4f78fc06a8376f451993059407d2972.json',2),(348068,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348098,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348113,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348127,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348128,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348203,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348230,'435baf47b4f78fc06a8376f451993059407d2972.json',2),(348253,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348262,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348273,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348284,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348288,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348293,'435baf47b4f78fc06a8376f451993059407d2972.json',1),(348385,'435baf47b4f78fc06a8376f451993059407d2972.json',3),(348408,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348411,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(348438,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348474,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',3),(348516,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348538,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348575,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',7),(348584,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(348605,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348614,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',12),(348649,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348658,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348660,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348691,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',4),(348694,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348695,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',4),(348696,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(348704,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348705,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348708,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348709,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348757,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',3),(348772,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348854,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(348933,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349001,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349070,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349076,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349104,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349127,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349132,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349138,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349153,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349157,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',5),(349161,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349176,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(349234,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349250,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(349274,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349349,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349358,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(349448,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349470,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',8),(349477,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349531,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',4),(349535,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349555,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349569,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349578,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(349610,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349619,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349672,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(349703,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',3),(349836,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(349904,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',2),(350140,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350142,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350143,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350144,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350252,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',13),(350308,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350317,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350325,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350343,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350360,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',16),(350377,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350423,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350518,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350554,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350555,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350560,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350939,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(350977,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(351015,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(351059,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',6),(351108,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(351115,'c65f0939cf35a0f04bf93bd6e8f771b8521563a5.json',1),(351280,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',2),(351284,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351310,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351348,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',3),(351365,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',5),(351387,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351390,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',4),(351391,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',4),(351440,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',2),(351450,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351453,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351507,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351508,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351542,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351543,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351740,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351779,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351800,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351814,'8f5770d87614bb85a6c7b64c26f1757226137d23.json',1),(351879,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(351882,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(351902,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(351903,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(351929,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(351952,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',24),(351976,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(351997,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',3),(352009,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',16),(352080,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(352081,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(352105,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352161,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',3),(352201,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352222,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',4),(352231,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352233,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352270,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352274,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352302,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352399,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352400,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(352513,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352561,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',7),(352599,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352617,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352658,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352713,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',13),(352744,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352814,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352837,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352851,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352861,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352878,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(352954,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352971,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(352996,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353005,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353123,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353147,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',3),(353148,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',15),(353183,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353213,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(353214,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353240,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353258,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353271,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353275,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(353426,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353470,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353473,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(353482,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(353504,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353507,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353512,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353521,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353539,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(353581,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353597,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353612,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353639,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353810,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(353830,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(353885,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354086,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',2),(354090,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354158,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354171,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354183,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354226,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',9),(354242,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354252,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354305,'418f1b85e589f3ac37ffc37075b6e558e510fa74.json',1),(354408,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354442,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354462,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',169),(354519,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354521,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(354563,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354575,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(354605,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354606,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354646,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',16),(354702,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',21),(354877,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',3),(354920,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354952,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354965,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354982,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(354983,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',42),(355014,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',4),(355018,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355030,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355081,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',10),(355086,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(355139,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',8),(355160,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355165,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',22),(355222,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355239,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355255,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355349,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',8),(355478,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',3),(355480,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(355484,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',11),(355494,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355647,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355653,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',43),(355661,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355689,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355852,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355858,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(355873,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',3),(355890,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',26),(355920,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(356087,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356090,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356173,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356187,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356261,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',13),(356399,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(356472,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356473,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356546,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356582,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',3),(356583,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',9),(356597,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(356616,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356657,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(356686,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356708,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356741,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356742,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356751,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356765,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(356779,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356819,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356874,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(356891,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357006,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357044,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357077,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357086,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(357090,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',3),(357157,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357164,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357177,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357210,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357387,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357478,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357499,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',2),(357652,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',108),(357712,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357713,'2bf62ecb50fd95973de0bf6c1285ac3475fc4cc8.json',1),(357730,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357748,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(357749,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(357758,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357765,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357770,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357789,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357793,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',4),(357795,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357805,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',6),(357815,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357816,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357819,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(357834,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357836,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357843,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357866,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357871,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357878,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357879,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357888,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(357897,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(357907,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(357910,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357920,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(357944,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(357969,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358020,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358047,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',11),(358081,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358092,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358147,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',4),(358183,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358233,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358234,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358265,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358280,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358281,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358282,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358283,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358284,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358300,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358312,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358317,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358390,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358411,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358504,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358510,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358548,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358550,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358595,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358650,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358651,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358652,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358655,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358663,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358736,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(358763,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358800,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358811,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358812,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(358815,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',7),(358828,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358946,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',5),(358968,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358977,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(358982,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359022,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359046,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359083,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359111,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359128,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359136,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(359163,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359235,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359360,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(359444,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(359488,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359545,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(359550,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(359596,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(359623,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',5),(359625,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359681,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359716,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359719,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(359740,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359741,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359803,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359824,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359874,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(359969,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(359992,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(359994,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(360101,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360105,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360115,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360123,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360130,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360156,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360171,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360245,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360309,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360310,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360331,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360345,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360362,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360363,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360364,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360456,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360471,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360538,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',2),(360541,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',3),(360547,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360561,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360588,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360602,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360606,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360649,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360653,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360674,'4b938c2f60ce863541ac9ca19fe0a98cd9a8730e.json',1),(360713,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(360718,'8309dc172cf91ff580779bbc1394a9692b798a86.json',21),(360734,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(360743,'8309dc172cf91ff580779bbc1394a9692b798a86.json',10),(360770,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(360838,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(360877,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(360910,'8309dc172cf91ff580779bbc1394a9692b798a86.json',2),(360968,'8309dc172cf91ff580779bbc1394a9692b798a86.json',2),(360969,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(360995,'8309dc172cf91ff580779bbc1394a9692b798a86.json',10),(361002,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361016,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361024,'8309dc172cf91ff580779bbc1394a9692b798a86.json',2),(361036,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361038,'8309dc172cf91ff580779bbc1394a9692b798a86.json',9),(361039,'8309dc172cf91ff580779bbc1394a9692b798a86.json',56),(361068,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361160,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361163,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361164,'8309dc172cf91ff580779bbc1394a9692b798a86.json',4),(361167,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361393,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361397,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361433,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361476,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361487,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361488,'8309dc172cf91ff580779bbc1394a9692b798a86.json',2),(361522,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361547,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361571,'8309dc172cf91ff580779bbc1394a9692b798a86.json',15),(361584,'8309dc172cf91ff580779bbc1394a9692b798a86.json',13),(361660,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361687,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361699,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361701,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361704,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361757,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361845,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361882,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361930,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(361970,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362021,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362024,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362038,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362074,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362084,'8309dc172cf91ff580779bbc1394a9692b798a86.json',2),(362210,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362218,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362267,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362287,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362289,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362290,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362301,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362313,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362344,'8309dc172cf91ff580779bbc1394a9692b798a86.json',2),(362412,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362435,'8309dc172cf91ff580779bbc1394a9692b798a86.json',2),(362444,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362448,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362468,'8309dc172cf91ff580779bbc1394a9692b798a86.json',4),(362478,'8309dc172cf91ff580779bbc1394a9692b798a86.json',16),(362480,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362513,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362516,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362555,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362619,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362655,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362696,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362700,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362709,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362714,'8309dc172cf91ff580779bbc1394a9692b798a86.json',1),(362791,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362814,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362827,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362898,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362899,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362900,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',7),(362901,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362902,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362923,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362924,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362926,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362949,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',26),(362951,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(362980,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363031,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363035,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363085,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363112,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363120,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363158,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363174,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(363175,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363181,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363185,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363186,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363197,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363203,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',20),(363204,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363230,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(363240,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(363294,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(363339,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363366,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363391,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',3),(363393,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363413,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363414,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363419,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',3),(363422,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363423,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363437,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363472,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363490,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363524,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',8),(363534,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363566,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363581,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363583,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363624,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363635,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363671,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363672,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363719,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(363745,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',3),(363766,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363814,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',7),(363818,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',3),(363835,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363857,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(363928,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364047,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364097,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364132,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364136,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364198,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(364313,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364327,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',4),(364489,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364781,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(364814,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(364936,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',6),(364971,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(364972,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(365142,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(365211,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',2),(365246,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(365288,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(365345,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(365454,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',3),(365501,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(365569,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(365603,'281b4272005dbc998e36b034b7fb528c9ba038cd.json',1),(365654,'4994fa72322bbf19120592304d92629226948d8e.json',4),(365655,'4994fa72322bbf19120592304d92629226948d8e.json',2),(365677,'4994fa72322bbf19120592304d92629226948d8e.json',1),(365706,'4994fa72322bbf19120592304d92629226948d8e.json',1),(365834,'4994fa72322bbf19120592304d92629226948d8e.json',1),(365851,'4994fa72322bbf19120592304d92629226948d8e.json',2),(365864,'4994fa72322bbf19120592304d92629226948d8e.json',1),(365908,'4994fa72322bbf19120592304d92629226948d8e.json',3),(365993,'4994fa72322bbf19120592304d92629226948d8e.json',2),(366006,'4994fa72322bbf19120592304d92629226948d8e.json',4),(366007,'4994fa72322bbf19120592304d92629226948d8e.json',4),(366009,'4994fa72322bbf19120592304d92629226948d8e.json',4),(366074,'4994fa72322bbf19120592304d92629226948d8e.json',2),(366089,'4994fa72322bbf19120592304d92629226948d8e.json',2),(366117,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366159,'4994fa72322bbf19120592304d92629226948d8e.json',2),(366161,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366190,'4994fa72322bbf19120592304d92629226948d8e.json',7),(366199,'4994fa72322bbf19120592304d92629226948d8e.json',3),(366221,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366228,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366236,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366240,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366303,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366338,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366340,'4994fa72322bbf19120592304d92629226948d8e.json',7),(366346,'4994fa72322bbf19120592304d92629226948d8e.json',7),(366383,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366471,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366499,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366503,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366527,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366536,'4994fa72322bbf19120592304d92629226948d8e.json',4),(366561,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366563,'4994fa72322bbf19120592304d92629226948d8e.json',4),(366591,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366630,'4994fa72322bbf19120592304d92629226948d8e.json',3),(366637,'4994fa72322bbf19120592304d92629226948d8e.json',5),(366668,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366733,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366848,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366868,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366869,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366873,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366874,'4994fa72322bbf19120592304d92629226948d8e.json',2),(366883,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366893,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366904,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366917,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366918,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366920,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366941,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366958,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366959,'4994fa72322bbf19120592304d92629226948d8e.json',2),(366960,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366961,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366967,'4994fa72322bbf19120592304d92629226948d8e.json',2),(366975,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366992,'4994fa72322bbf19120592304d92629226948d8e.json',1),(366995,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367000,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367002,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367006,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367007,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367011,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367033,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367034,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367039,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367045,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367054,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367056,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367058,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367065,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367066,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367079,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367082,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367087,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367102,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367107,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367124,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367127,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367141,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367146,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367149,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367161,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367190,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367218,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367219,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367220,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367230,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367231,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367248,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367249,'4994fa72322bbf19120592304d92629226948d8e.json',10),(367261,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367374,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367378,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367386,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367397,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367412,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367425,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367431,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367451,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367462,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367481,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367536,'4994fa72322bbf19120592304d92629226948d8e.json',3),(367580,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367581,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367582,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367586,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367589,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367590,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367602,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367606,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367609,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367614,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367623,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367634,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367638,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367639,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367640,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367655,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367658,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367659,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367678,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367698,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367702,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367762,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367771,'4994fa72322bbf19120592304d92629226948d8e.json',2),(367799,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367830,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367831,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367855,'4994fa72322bbf19120592304d92629226948d8e.json',1),(367903,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(367939,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(367946,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(367993,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368073,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368090,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368236,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',4),(368438,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368453,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',7),(368531,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',2),(368536,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368591,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368595,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368605,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368606,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368607,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368654,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368724,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368850,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368877,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',25),(368880,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',7),(368890,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(368900,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',2),(368906,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',2),(368915,'d47eb01f2085b582008bb1ff7c604355c5f4c582.json',1),(369049,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(369231,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',2),(369378,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(369428,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',5),(369465,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',3),(369480,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',3),(369512,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',2),(369655,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(369683,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(369745,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(369860,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(369905,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',2),(369922,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(369999,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370036,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370197,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370200,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370243,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370351,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',13),(370352,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',20),(370546,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',2),(370576,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370594,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370608,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',5),(370609,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',69),(370715,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(370730,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(371174,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(371202,'555bbbbd23da02c97c6a4238937e31b5781438a9.json',1),(371257,'bada3cc0577a226694617d77f95ac443c6431969.json',4),(371263,'bada3cc0577a226694617d77f95ac443c6431969.json',33),(371293,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371305,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(371333,'bada3cc0577a226694617d77f95ac443c6431969.json',21),(371335,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371343,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371353,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371354,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371365,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371397,'bada3cc0577a226694617d77f95ac443c6431969.json',5),(371401,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371408,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371415,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371421,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371427,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(371428,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(371429,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371436,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371437,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371438,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371439,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371450,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371467,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371494,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371585,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(371675,'bada3cc0577a226694617d77f95ac443c6431969.json',5),(371685,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(371701,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371874,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371879,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(371893,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(372000,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372036,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372061,'bada3cc0577a226694617d77f95ac443c6431969.json',5),(372062,'bada3cc0577a226694617d77f95ac443c6431969.json',4),(372115,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372230,'bada3cc0577a226694617d77f95ac443c6431969.json',5),(372304,'bada3cc0577a226694617d77f95ac443c6431969.json',4),(372318,'bada3cc0577a226694617d77f95ac443c6431969.json',3),(372319,'bada3cc0577a226694617d77f95ac443c6431969.json',3),(372335,'bada3cc0577a226694617d77f95ac443c6431969.json',35),(372343,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372371,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372389,'bada3cc0577a226694617d77f95ac443c6431969.json',5),(372399,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372413,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372436,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372437,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372448,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(372525,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372552,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372586,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372587,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372644,'bada3cc0577a226694617d77f95ac443c6431969.json',3),(372736,'bada3cc0577a226694617d77f95ac443c6431969.json',3),(372770,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372806,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372837,'bada3cc0577a226694617d77f95ac443c6431969.json',2),(372912,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(372996,'bada3cc0577a226694617d77f95ac443c6431969.json',3),(373074,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373082,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373124,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373125,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373178,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373288,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373298,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373374,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373406,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373437,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373494,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373555,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373602,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373603,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373605,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373606,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373607,'bada3cc0577a226694617d77f95ac443c6431969.json',1),(373638,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(373662,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373680,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373792,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373793,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373794,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373798,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373823,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373849,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373867,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373917,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373935,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373937,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(373999,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374065,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374072,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374097,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',4),(374154,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374229,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374311,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374318,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(374369,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374385,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374418,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374438,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(374468,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(374534,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374569,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374574,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374691,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374718,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374725,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374786,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374802,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374877,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374902,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',4),(374903,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374910,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374919,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374926,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374928,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374933,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(374937,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375000,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375010,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375012,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375056,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375111,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375162,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375182,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',6),(375199,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(375327,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375332,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(375340,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375350,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375374,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375388,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375407,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',40),(375409,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375440,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',6),(375461,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375478,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(375614,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375642,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375643,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375675,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375769,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375847,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375877,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375879,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375892,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375896,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375915,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(375925,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',4),(375928,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',8),(375998,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376005,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376151,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376155,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376177,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(376238,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376272,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376273,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376315,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',3),(376463,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376483,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376531,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(376545,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376600,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(376762,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376847,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376873,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(376899,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(377025,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(377034,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(377142,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',2),(377171,'b6eed04e43ad653c06247b10ab05d03bdccf04bc.json',1),(377238,'fb3c146b1766b4aa33df1637742bf93159967289.json',3),(377252,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377255,'fb3c146b1766b4aa33df1637742bf93159967289.json',21),(377259,'fb3c146b1766b4aa33df1637742bf93159967289.json',19),(377279,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377320,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377345,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377390,'fb3c146b1766b4aa33df1637742bf93159967289.json',26),(377396,'fb3c146b1766b4aa33df1637742bf93159967289.json',49),(377416,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377461,'fb3c146b1766b4aa33df1637742bf93159967289.json',3),(377469,'fb3c146b1766b4aa33df1637742bf93159967289.json',3),(377500,'fb3c146b1766b4aa33df1637742bf93159967289.json',3),(377513,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377530,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377552,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377619,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377734,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377866,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377907,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(377971,'fb3c146b1766b4aa33df1637742bf93159967289.json',4),(378204,'fb3c146b1766b4aa33df1637742bf93159967289.json',6),(378417,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(378422,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(378424,'fb3c146b1766b4aa33df1637742bf93159967289.json',3),(378441,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(378652,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(378723,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(378749,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(378813,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(378819,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(378842,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(378946,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(378971,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(379022,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(379096,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(379175,'fb3c146b1766b4aa33df1637742bf93159967289.json',7),(379176,'fb3c146b1766b4aa33df1637742bf93159967289.json',7),(379185,'fb3c146b1766b4aa33df1637742bf93159967289.json',7),(379241,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379402,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379412,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379413,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379493,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(379564,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379565,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379713,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379931,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(379979,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380118,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380123,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380234,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380239,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380253,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380303,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380352,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380389,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380447,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380456,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380458,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380470,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380488,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380492,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380495,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380501,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380548,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(380550,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(380584,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380587,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380590,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(380620,'fb3c146b1766b4aa33df1637742bf93159967289.json',3),(380650,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380663,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380676,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380679,'fb3c146b1766b4aa33df1637742bf93159967289.json',2),(380694,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380698,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380709,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380713,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380714,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380742,'fb3c146b1766b4aa33df1637742bf93159967289.json',3),(380754,'fb3c146b1766b4aa33df1637742bf93159967289.json',1),(380771,'fb3c146b1766b4aa33df1637742bf93159967289.json',1);
/*!40000 ALTER TABLE `word_to_paper` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2022-04-14 17:28:40
